{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [],
   "source": [
    "import json\n",
    "from os import listdir\n",
    "from os.path import isfile, join"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [],
   "source": [
    "dirbase='E:\\\\Spring 2020\\\\ML1\\\\final\\\\CORD-19-research-challenge\\\\biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\pdf_json'\n",
    "def read_file(filename):\n",
    "    input_file_text = open(filename ,encoding='utf-8')\n",
    "    with input_file_text as f:\n",
    "        data = json.load(f)\n",
    "    return data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [],
   "source": [
    "def read_directory_files(directory):\n",
    "    file_texts = []\n",
    "    files = [f for f in listdir(directory) if isfile(join(directory, f))]\n",
    "    for f in files:\n",
    "        file_text = read_file(join(directory, f))\n",
    "        print(file_text)\n",
    "        file_texts.append({\"file\":f, \"content\": file_text })\n",
    "    return file_texts\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "IOPub data rate exceeded.\n",
      "The notebook server will temporarily stop sending output\n",
      "to the client in order to avoid crashing it.\n",
      "To change this limit, set the config variable\n",
      "`--NotebookApp.iopub_data_rate_limit`.\n",
      "\n",
      "Current values:\n",
      "NotebookApp.iopub_data_rate_limit=1000000.0 (bytes/sec)\n",
      "NotebookApp.rate_limit_window=3.0 (secs)\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "{'paper_id': 'ac45d1c2865c21a16912e73a857060bacce4c95a', 'metadata': {'title': 'High sensitivity detection of coronavirus SARS-CoV-2 using multiplex PCR and a multiplex-PCR-based metagenomic method', 'authors': [{'first': 'Chenyu', 'middle': [], 'last': 'Li', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Paragon Genomics Inc', 'location': {'postCode': '94545', 'settlement': 'Hayward', 'region': 'CA', 'country': 'USA'}}, 'email': ''}, {'first': '1#', 'middle': [], 'last': '', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'David', 'middle': ['N'], 'last': 'Debruyne', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Paragon Genomics Inc', 'location': {'postCode': '94545', 'settlement': 'Hayward', 'region': 'CA', 'country': 'USA'}}, 'email': ''}, {'first': 'Julia', 'middle': [], 'last': 'Spencer', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Paragon Genomics Inc', 'location': {'postCode': '94545', 'settlement': 'Hayward', 'region': 'CA', 'country': 'USA'}}, 'email': ''}, {'first': 'Vidushi', 'middle': [], 'last': 'Kapoor', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Paragon Genomics Inc', 'location': {'postCode': '94545', 'settlement': 'Hayward', 'region': 'CA', 'country': 'USA'}}, 'email': ''}, {'first': 'Lily', 'middle': ['Y'], 'last': 'Liu', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Paragon Genomics Inc', 'location': {'postCode': '94545', 'settlement': 'Hayward', 'region': 'CA', 'country': 'USA'}}, 'email': ''}, {'first': 'Bo', 'middle': [], 'last': 'Zhou', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Stanford University', 'location': {'postCode': '94305', 'region': 'CA', 'country': 'USA'}}, 'email': ''}, {'first': 'Lucie', 'middle': [], 'last': 'Lee', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Paragon Genomics Inc', 'location': {'postCode': '94545', 'settlement': 'Hayward', 'region': 'CA', 'country': 'USA'}}, 'email': ''}, {'first': 'Rounak', 'middle': [], 'last': 'Feigelman', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Paragon Genomics Inc', 'location': {'postCode': '94545', 'settlement': 'Hayward', 'region': 'CA', 'country': 'USA'}}, 'email': ''}, {'first': 'Grayson', 'middle': [], 'last': 'Burdon', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Paragon Genomics Inc', 'location': {'postCode': '94545', 'settlement': 'Hayward', 'region': 'CA', 'country': 'USA'}}, 'email': ''}, {'first': 'Jeffrey', 'middle': [], 'last': 'Liu', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Paragon Genomics Inc', 'location': {'postCode': '94545', 'settlement': 'Hayward', 'region': 'CA', 'country': 'USA'}}, 'email': ''}, {'first': 'Alejandra', 'middle': [], 'last': 'Oliva', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Paragon Genomics Inc', 'location': {'postCode': '94545', 'settlement': 'Hayward', 'region': 'CA', 'country': 'USA'}}, 'email': ''}, {'first': 'Adam', 'middle': [], 'last': 'Borcherding', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'BGI-Shenzhen', 'location': {'postCode': '518083', 'settlement': 'Shenzhen', 'country': 'China'}}, 'email': ''}, {'first': 'Hongdong', 'middle': [], 'last': 'Tan', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'BGI-Shenzhen', 'location': {'postCode': '518083', 'settlement': 'Shenzhen', 'country': 'China'}}, 'email': ''}, {'first': 'Alexander', 'middle': ['E'], 'last': 'Urban', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Stanford University', 'location': {'postCode': '94305', 'region': 'CA', 'country': 'USA'}}, 'email': ''}, {'first': 'Guoying', 'middle': [], 'last': 'Liu', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Paragon Genomics Inc', 'location': {'postCode': '94545', 'settlement': 'Hayward', 'region': 'CA', 'country': 'USA'}}, 'email': ''}, {'first': 'Zhitong', 'middle': [], 'last': 'Liu', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Paragon Genomics Inc', 'location': {'postCode': '94545', 'settlement': 'Hayward', 'region': 'CA', 'country': 'USA'}}, 'email': 'zhitong@paragongenomics.com'}]}, 'abstract': [{'text': 'Many detection methods have been used or reported for the diagnosis and/or surveillance of SARS-CoV-2. Among them, reverse transcription polymerase chain reaction (RT-PCR) is the most sensitive, claiming detection of about 5 copies of viruses. However, it has been reported that only 47-59% of the positive cases were identified by RT-PCR, probably due to loss or degradation of virus RNA in the sampling process, or even mutation of the virus genome. Therefore, developing highly sensitive methods is imperative to ensure robust detection capabilities. With the goal of improving sensitivity and accommodate various application settings, we developed a multiplex-PCR-based method comprised of 172 pairs of specific primers, and demonstrated its efficiency to detect SARS-CoV-2 at low copy numbers. The assay produced clean characteristic target peaks of defined sizes, which allowed for direct identification of positives by electrophoresis. In addition, optional sequencing can provide further confirmation as well as phylogenetic information of the identified virus(es) for specific strain discrimination, which will be of paramount importance for surveillance purposes that represent a global health imperative. Finally, we also developed in parallel a multiplex-PCR-based metagenomic method that is amenable to detect SARS-CoV-2, with the additional benefit of its potential for uncovering mutational diversity and novel pathogens at low sequencing depth.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}], 'body_text': [{'text': 'A severe epidemic coronavirus (SARS-CoV-2) infection, now just characterized as a pandemic by the World Health Organization (WHO), started in December of 2019 in Wuhan China and quickly spread to many countries in the world [1] [2] [3] . It has caused over 7,000 deaths as of mid-March and made daily impacts on various aspects of societal life around the globe 4 . SARS-CoV-2 is a coronavirus with positive-sense, singlestranded RNA about 30kb in length. The genome of SARS-CoV-2 is currently under careful investigation [5] [6] [7] [8] [9] and being extensively modeled according to the Chinese National Center for Biological information (2019 New Coronavirus Information Database, https://bigd.big.ac.cn/ncov). Of note, SARS-CoV-2 exhibits over 99% sequence similarity among many sequenced isolates, and is also highly similar to other coronaviruses 5,9 . Present methods for detecting SARS-CoV-2 have been reported and discussed 10, 11 , including RT-PCR, serological testing 12 and reverse transcription-loop-mediated isothermal amplification 13, 14 . Currently, RT-PCR is considered the gold standard of diagnosis for SARS-CoV-2 due to its ease of use and high sensitivity. RT-PCR has been reported to detect SARS-CoV-2 in saliva 15 , pharyngeal swab, blood, anal swab 16 , urine, stool 17 , and sputum specimens 18 . In laboratory conditions, the RT-PCR methodology has been shown to detect 4-8 copies of virus, through amplification of targets in the Orf1ab, E and N viral genes, at 95% confidence intervals [19] [20] [21] . However, only about 47-59% of the positive cases were identified by RT-PCR, and 75% of RT-PCR negative results were actually found to be positive, with repeated tests required 17, [22] [23] [24] . In addition, there is evidence suggesting that heat inactivation of clinical samples causes loss of virus particles, thereby hindering the efficiency of downstream diagnosis 25 .', 'cite_spans': [{'start': 224, 'end': 227, 'text': '[1]', 'ref_id': 'BIBREF0'}, {'start': 228, 'end': 231, 'text': '[2]', 'ref_id': 'BIBREF1'}, {'start': 232, 'end': 235, 'text': '[3]', 'ref_id': None}, {'start': 362, 'end': 363, 'text': '4', 'ref_id': 'BIBREF3'}, {'start': 522, 'end': 525, 'text': '[5]', 'ref_id': 'BIBREF4'}, {'start': 526, 'end': 529, 'text': '[6]', 'ref_id': 'BIBREF5'}, {'start': 530, 'end': 533, 'text': '[7]', 'ref_id': 'BIBREF6'}, {'start': 534, 'end': 537, 'text': '[8]', 'ref_id': 'BIBREF7'}, {'start': 538, 'end': 541, 'text': '[9]', 'ref_id': 'BIBREF8'}, {'start': 853, 'end': 858, 'text': '5,9 .', 'ref_id': None}, {'start': 933, 'end': 936, 'text': '10,', 'ref_id': 'BIBREF9'}, {'start': 937, 'end': 939, 'text': '11', 'ref_id': 'BIBREF10'}, {'start': 980, 'end': 982, 'text': '12', 'ref_id': 'BIBREF11'}, {'start': 1048, 'end': 1051, 'text': '13,', 'ref_id': 'BIBREF12'}, {'start': 1052, 'end': 1054, 'text': '14', 'ref_id': 'BIBREF13'}, {'start': 1236, 'end': 1238, 'text': '15', 'ref_id': 'BIBREF14'}, {'start': 1275, 'end': 1277, 'text': '16', 'ref_id': 'BIBREF15'}, {'start': 1293, 'end': 1295, 'text': '17', 'ref_id': 'BIBREF16'}, {'start': 1319, 'end': 1321, 'text': '18', 'ref_id': 'BIBREF17'}, {'start': 1516, 'end': 1520, 'text': '[19]', 'ref_id': 'BIBREF18'}, {'start': 1521, 'end': 1525, 'text': '[20]', 'ref_id': 'BIBREF19'}, {'start': 1526, 'end': 1530, 'text': '[21]', 'ref_id': 'BIBREF20'}, {'start': 1709, 'end': 1712, 'text': '17,', 'ref_id': 'BIBREF16'}, {'start': 1713, 'end': 1717, 'text': '[22]', 'ref_id': 'BIBREF21'}, {'start': 1718, 'end': 1722, 'text': '[23]', 'ref_id': 'BIBREF22'}, {'start': 1723, 'end': 1727, 'text': '[24]', 'ref_id': 'BIBREF23'}, {'start': 1904, 'end': 1906, 'text': '25', 'ref_id': None}], 'ref_spans': [], 'section': 'Introduction'}, {'text': \"Therefore, it is necessary to develop robust, sensitive, specific and highly quantitative methods for the delivery of reliable diagnostic assays 26, 27 . The urgency to develop an effective surveillance method that can be easily used in a variety of laboratory settings is underlined by the wide and rapid spreading of SARS-CoV-2 [28] [29] [30] . In addition, such method should also distinguish SARS-CoV-2 from other respiratory pathogens such as influenza virus, parainfluenza virus, adenovirus, respiratory syncytial virus, rhinovirus, human metapneumovirus, SARS coronavirus, etc., as well as mycoplasma pneumoniae, chlamydia pneumonia and bacterial pneumonia [31] [32] [33] [34] . Furthermore, providing nucleotide sequence information through next generation sequencing (NGS) will prove to be essential for the surveillance of SARS-CoV-2's evolution [35] [36] [37] [38] . Indeed, SARS-CoV-2 phylogenetic studies through genome sequence analysis have provided better understanding of the transmission origin, time and routes, which has guided policy-making and management procedures 8, 36, 37, [39] [40] [41] .\", 'cite_spans': [{'start': 145, 'end': 148, 'text': '26,', 'ref_id': 'BIBREF26'}, {'start': 149, 'end': 151, 'text': '27', 'ref_id': 'BIBREF27'}, {'start': 330, 'end': 334, 'text': '[28]', 'ref_id': 'BIBREF28'}, {'start': 335, 'end': 339, 'text': '[29]', 'ref_id': 'BIBREF29'}, {'start': 340, 'end': 344, 'text': '[30]', 'ref_id': 'BIBREF30'}, {'start': 664, 'end': 668, 'text': '[31]', 'ref_id': 'BIBREF31'}, {'start': 669, 'end': 673, 'text': '[32]', 'ref_id': None}, {'start': 674, 'end': 678, 'text': '[33]', 'ref_id': 'BIBREF34'}, {'start': 679, 'end': 683, 'text': '[34]', 'ref_id': 'BIBREF35'}, {'start': 856, 'end': 860, 'text': '[35]', 'ref_id': 'BIBREF36'}, {'start': 861, 'end': 865, 'text': '[36]', 'ref_id': 'BIBREF37'}, {'start': 866, 'end': 870, 'text': '[37]', 'ref_id': 'BIBREF38'}, {'start': 871, 'end': 875, 'text': '[38]', 'ref_id': 'BIBREF39'}, {'start': 1088, 'end': 1090, 'text': '8,', 'ref_id': 'BIBREF7'}, {'start': 1091, 'end': 1094, 'text': '36,', 'ref_id': 'BIBREF37'}, {'start': 1095, 'end': 1098, 'text': '37,', 'ref_id': 'BIBREF38'}, {'start': 1099, 'end': 1103, 'text': '[39]', 'ref_id': 'BIBREF40'}, {'start': 1104, 'end': 1108, 'text': '[40]', 'ref_id': 'BIBREF41'}, {'start': 1109, 'end': 1113, 'text': '[41]', 'ref_id': 'BIBREF42'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'Here, we describe the development of a highly sensitive and robust detection assay incorporating the use of multiplex PCR technology to identify SARS-CoV-2. Theoretically, the multiplex PCR strategy, by amplifying hundreds of targets, has significantly higher sensitivity than RT-PCR and may even detect DNA molecules resulting from degraded virus genome fragments. Multiplex PCR has been shown to be an efficient and low-cost method to detect Plasmodium falciparum infections, with high coverage (median 99%), specificity (99.8%) and sensitivity. Moreover, this solution can be tailored to simultaneously address multiple questions of interest within various epidemiological settings 42 . Similar to a recently described metagenomic approach for SARS-CoV-2 identification 43 , we also establish a user-friendly multiplex-PCRbased metagenomic method that is not only able to detect SARS-CoV-2, but could also be applied for the identification of significant sequence mutations within known viruses and to uncover novel pathogens with a limited sequencing depth of approximately 1 million reads.', 'cite_spans': [{'start': 685, 'end': 687, 'text': '42', 'ref_id': 'BIBREF43'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'Several RT-PCR methods for detecting SARS-CoV-2 have been reported to date 15, [19] [20] [21] . Among them, two groups reported the detection of 4-5 copies of the virus 19, 21 . To investigate the opportunity for further improvement on the sensitivity of RT-PCR, we built a mathematical model to estimate the limit of detection (LOD) for SARS-CoV-2. The reported RT-PCR amplicon lengths are around 78-158bp, and the SARS-CoV-2 genome is 29,903bp (NC_045512.2). Thus, in this mathematical modeling, we chose 100bp amplicon length and 30kb SARS-CoV-2 genome size for the estimation. With the assumption of 99% RT-PCR efficiency 20 , we found that RT-PCR assays could detect 4.8 copies of SARS-CoV-2 at 95% probability ( Fig. 1A) , which is consistent with the experimental results obtained in a previous report 19 . In this model, the probability of RT-PCR assays to detect one copy of SARS-CoV-2 is only 26% (Supplemental Fig. 1 ). This finding may explain, at least in part, the low reported 47-56% detection rates of SARS-CoV-2 in positive samples by RT-PCR 17, 22 . We further found that the LOD appears to be independent of the virus genome size. For genomes of 4 to 100kb, the detection limit remains 4.8 copies at 95% probability.', 'cite_spans': [{'start': 75, 'end': 78, 'text': '15,', 'ref_id': 'BIBREF14'}, {'start': 79, 'end': 83, 'text': '[19]', 'ref_id': 'BIBREF18'}, {'start': 84, 'end': 88, 'text': '[20]', 'ref_id': 'BIBREF19'}, {'start': 89, 'end': 93, 'text': '[21]', 'ref_id': 'BIBREF20'}, {'start': 169, 'end': 172, 'text': '19,', 'ref_id': 'BIBREF18'}, {'start': 173, 'end': 175, 'text': '21', 'ref_id': 'BIBREF20'}, {'start': 626, 'end': 628, 'text': '20', 'ref_id': 'BIBREF19'}, {'start': 809, 'end': 811, 'text': '19', 'ref_id': 'BIBREF18'}, {'start': 1059, 'end': 1062, 'text': '17,', 'ref_id': 'BIBREF16'}, {'start': 1063, 'end': 1065, 'text': '22', 'ref_id': 'BIBREF21'}], 'ref_spans': [{'start': 718, 'end': 726, 'text': 'Fig. 1A)', 'ref_id': 'FIGREF0'}, {'start': 907, 'end': 927, 'text': '(Supplemental Fig. 1', 'ref_id': 'FIGREF0'}], 'section': 'Mathematical model of RT-PCR'}, {'text': 'One way to elevate the sensitivity is to amplify multiple targets on the same virus genome in a multiplex PCR reaction, thereby increasing the frequency of occurrence in the mathematical model. Amplifying multiple targets has the advantage of potentially detecting fragments of degraded virus genome while withstanding sequencing variations, thus allowing for the detection of upcoming mutants. The amplification efficiency of multiplex PCR is critical for LOD. We estimated that the efficiency of our multiplex PCR technology is about 26% by using Unique Molecular Identifier (UMI)-labeled primers to count the amplified products after NGS sequencing (Supplemental Fig. 2 and Supplemental Table 1 ). However, the amplification efficiency could be lower, and not all amplicons amplified successfully if the template used is one single strand of cDNA. Thus, more amplicons are potentially required for multiplex PCR to detect limited copies of viruses.', 'cite_spans': [], 'ref_spans': [{'start': 666, 'end': 672, 'text': 'Fig. 2', 'ref_id': None}, {'start': 690, 'end': 697, 'text': 'Table 1', 'ref_id': None}], 'section': 'Mathematical model of RT-PCR'}, {'text': 'We thus designed a panel of 172 pairs of multiplex PCR primers for the sensitive detection of SARS-CoV-2 ( Fig. 1B) . The average amplicon length is 99bp. The amplicons run across the entire SARS-CoV-2 genome with a 76bp gap (76±10bp) between each amplicon. Since the observed efficiency of multiplex PCR is about 26% in amplifying the four DNA strands of a pair of human chromosomes, we assumed an efficiency of 6% in amplifying a single-strand of cDNA. In addition, it has already been reported that 79% of variants are recovered when directly amplifying 600 amplicons from a single cell using our technology 44 . Therefore, we assume that 80% of amplicons would be amplified successfully. Using the same mathematical model described above, we estimated that our specific SARS-CoV-2-designed panel can detect 1.15 copies of the virus at 95% probability (Fig. 1C) . Again, the LOD is independent of virus genome size.', 'cite_spans': [{'start': 611, 'end': 613, 'text': '44', 'ref_id': 'BIBREF45'}], 'ref_spans': [{'start': 107, 'end': 115, 'text': 'Fig. 1B)', 'ref_id': 'FIGREF0'}, {'start': 855, 'end': 864, 'text': '(Fig. 1C)', 'ref_id': 'FIGREF0'}], 'section': 'Mathematical model of multiplex-PCR-based detection method'}, {'text': 'We also designed a second pool of 171 multiplex PCR primers. These primers overlap with those in the previous 172-primer pool. Together, these two overlapping pools of primers deliver full coverage of the entire virus genome. If using both pools in detection, the calculated detection limit is 0.29 copies at 95% probability. ', 'cite_spans': [], 'ref_spans': [], 'section': 'Mathematical model of multiplex-PCR-based detection method'}, {'text': 'The workflow was designed so that the multiplex PCR products are further amplified in a secondary PCR, while sample indexes and NGS sequencing primers are added (Fig. 1D) . The PCR products were first analyzed by electrophoresis to identify potential positives. Since dozens of amplicons could be amplified from a single copy of SARS-CoV-2, electrophoresis peaks with defined peak sizes were expected. Multiplex PCR amplifies SARS-CoV-2, as well as other coronaviruses due to their high sequence similarities. In that context, electrophoresis analysis provides a fast and sensitive indication of infection from that family of viruses. In addition, the generated library allows for further investigation through NGS sequencing to provide definitive identification of the specific virus family member.', 'cite_spans': [], 'ref_spans': [{'start': 161, 'end': 170, 'text': '(Fig. 1D)', 'ref_id': 'FIGREF0'}], 'section': 'Detecting limited copies of SARS-CoV-2'}, {'text': 'Two plasmids, containing the full sequence of S and N genes of SARS-CoV-2, respectively, were used to validate our multiplex PCR method. 28 amplicons are expected to be amplified within our 172amplicon panel. To simulate the use of real clinical samples, these two plasmids were spiked into the cDNA made from human total RNA. The copy number of each plasmid was determined by droplet-based digital PCR in QX200 from Bio-Rad® 45 . The two plasmids were diluted from approximately 9,000 copies to below one copy, and were amplified in multiplex PCR reactions. The library peaks of expected sizes were obtained from 8,900 to 2.8 copies of plasmids ( Fig. 2A) . Quantification of peaks demonstrated a wide dynamic range from 1 to about 1,000 copies of plasmids (Fig. 2B) . The yield of the libraries started to saturate when the copy number was 1,700. It is possible that the saturation point could be even lower when all of the 172 amplicons are amplified from positive clinical samples, and the library peak could be observed with even fewer copies of virus. In contrast, the detected quantities of a single target on N gene by RT-PCR rapidly dropped when using 2.85 copies (Fig. 2B and Supplemental Fig. 3) .', 'cite_spans': [], 'ref_spans': [{'start': 648, 'end': 656, 'text': 'Fig. 2A)', 'ref_id': None}, {'start': 758, 'end': 767, 'text': '(Fig. 2B)', 'ref_id': None}, {'start': 1173, 'end': 1206, 'text': '(Fig. 2B and Supplemental Fig. 3)', 'ref_id': None}], 'section': 'Detecting limited copies of SARS-CoV-2'}, {'text': 'Estimated from the aforementioned mathematical model, 28 amplicons have a 16% chance to detect one single copy. To test this hypothesis, we amplified about one copy of plasmid in multiplex PCR reactions. The theoretical calculation gives a 66% probability to sample 1.1 copies, and a 12% chance to detect them based on a multiplex PCR efficiency of 6%. In reality, we experimentally observed a significantly higher 56% probability to detect 1.1 copies (Fig. 2C) . These results suggest that the efficiency of multiplex PCR is actually higher than the previously estimated 6% when single-stranded cDNA was amplified.', 'cite_spans': [], 'ref_spans': [{'start': 452, 'end': 461, 'text': '(Fig. 2C)', 'ref_id': None}], 'section': 'Detecting limited copies of SARS-CoV-2'}, {'text': 'When the amplified products were sequenced, we found that the recovered reads were within a range of about 20-fold relative depth with about 1.4 to 2.8 plasmids, and uniformly distributed across the GC range (Fig. 2D) . When detecting down to 1.4 copies of plasmids, only the reads of one amplicon were about 100-fold lower than the average. Approximately 96% of the amplicons were recovered with 14 copies of plasmids, 77% with 2.8 copies, and 37% with 0.6 copies (Fig. 2E) .', 'cite_spans': [], 'ref_spans': [{'start': 208, 'end': 217, 'text': '(Fig. 2D)', 'ref_id': None}, {'start': 465, 'end': 474, 'text': '(Fig. 2E)', 'ref_id': None}], 'section': 'Detecting limited copies of SARS-CoV-2'}, {'text': '(A) Two plasmids, containing S and N genes of SARS-CoV-2, respectively, were diluted in human cDNA and amplified in multiplex PCR with pool 1 (172 pairs of primers). The number of plasmid copies per reaction, determined by ddPCR, were from 8,900 to 0.6. The resulting products obtained after multiplex PCR were resolved by electrophoresis. The specific amplification products (the library) can still be seen with 2.8 copies of plasmids. (B) The library yields can be detected down to 0.6 copies of plasmids (n=4) by multiplex PCR (black line), while only down to 2.8 copies can still be detected by RT-PCR (> 4.5-fold difference) (red', 'cite_spans': [], 'ref_spans': [], 'section': 'Figure 2. Detection of SARS-CoV-2 gene-containing plasmids by electrophoresis and sequencing'}, {'text': \"In order to discover highly mutated viruses and unknown pathogens, we subsequently developed a userfriendly multiplex-PCR-based metagenomic method. In this method, random hexamer-adapters were used to amplify DNA or cDNA targets in a multiplex PCR reaction. The large amounts of non-specific amplification products were removed by using Paragon Genomics' proprietary background removing reagent, thus resolving a library suitable for sequencing. For RNA samples, our reverse transcription reagents were additionally used to convert RNA into cDNA, resulting in significantly reduced amount of human ribosomal RNA species.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Metagenomic method design for novel pathogens'}, {'text': 'We sequenced a library made with 4,500 copies of N and S gene-containing plasmids spiked into 10 ng of human gDNA, which roughly represents 3,300 haploid genomes. Even though the molar ratios of viral targets and human haploid genomes were comparable, N and S genes which encompass about 4kb of targets, were a negligible fraction of the 3 billion base pairs of a human genome. If every region of the human genome were amplified and sequenced at 0.6 million reads per sample, only one read of viral target would be recovered. In fact, our results showed that 16% of the recovered bases, or 13% of the recovered reads, were within the viral N and S genes ( Fig. 3A and Supplemental Table 2 ). 80% and 78% of SARS-CoV-2 and mitochondrial targets were covered, respectively (Fig. 3B) , and the base coverage was significantly higher than human targets (Fig. 3C) . In contrast, only 0.08% of regions in human chromosomes were amplified. Furthermore, the human exonic regions were preferentially amplified (Fig.  3D) . This suggested that the random hexamers deselected a large portion of the human genome, while favorably amplifying regions that were more \"random\" in base composition. Indeed, long gaps and absence of coverage in very large repetitive regions were observed in human chromosomes (Fig. 3E) . On the contrary, the gaps in SARS-CoV-2 and mitochondrial regions were significantly shorter (Fig. 3F) , whereas the amplified targets overlapped and were longer than human targets (Fig. 3G) .', 'cite_spans': [], 'ref_spans': [{'start': 656, 'end': 663, 'text': 'Fig. 3A', 'ref_id': None}, {'start': 681, 'end': 688, 'text': 'Table 2', 'ref_id': None}, {'start': 771, 'end': 780, 'text': '(Fig. 3B)', 'ref_id': None}, {'start': 849, 'end': 858, 'text': '(Fig. 3C)', 'ref_id': None}, {'start': 1001, 'end': 1011, 'text': '(Fig.  3D)', 'ref_id': None}, {'start': 1292, 'end': 1301, 'text': '(Fig. 3E)', 'ref_id': None}, {'start': 1397, 'end': 1406, 'text': '(Fig. 3F)', 'ref_id': None}, {'start': 1485, 'end': 1494, 'text': '(Fig. 3G)', 'ref_id': None}], 'section': 'Metagenomic method design for novel pathogens'}, {'text': 'The coverage was from 1000-to 10,000-fold for S and N genes, and 30-to 500-fold for the mitochondrial chromosome (Fig. 3F) . The coverage is roughly within a 10-fold range, as is observed in human chromosomes (Fig. 3E) . Therefore, increasing sequencing depth might not significantly improve the coverage. This 10-fold difference in coverage has been routinely observed with our multiplex PCR technology (Supplemental Table 3 and Supplemental Fig. 4) . We were able to detect 80% of the regions in S and N genes in libraries generated using 4,500 copies of plasmids, with an average base coverage of 5,000 for a total sequencing depth of 0.6 million reads. A few targets of 150 to 200 bp in length in S and N genes were preferentially amplified. Even when copy number went down to a few copies (3-14), these targets were still detected (Fig.4) . They represented 14% and 11% of the target regions when 14 and 2.8 copies of plasmids were amplified, respectively.', 'cite_spans': [], 'ref_spans': [{'start': 113, 'end': 122, 'text': '(Fig. 3F)', 'ref_id': None}, {'start': 209, 'end': 218, 'text': '(Fig. 3E)', 'ref_id': None}, {'start': 418, 'end': 425, 'text': 'Table 3', 'ref_id': None}, {'start': 430, 'end': 450, 'text': 'Supplemental Fig. 4)', 'ref_id': None}, {'start': 836, 'end': 843, 'text': '(Fig.4)', 'ref_id': None}], 'section': 'Metagenomic method design for novel pathogens'}, {'text': '(A) Random hexamer-adapters were used in multiplex PCR to amplify 4,500 copies of plasmids in the background of 3,300 haploid human gDNA molecules. The resulting libraries were sequenced at an average of 0.6 million total reads. Of the total bases recovered, 16% were on SARS-CoV-2 S and N genes. (B) 80% of S and N genes, and 78% of the human mitochondrial chromosome were amplified with >= 1X coverage, while only 0.08% of the human chromosomes were. (C) On average, S and N genes were covered at 2,346X, mitochondria at 77X, and human chromosomes at 20X. (D) Human exons were relatively overamplified about 4-fold higher compared to their actual ratio within the genome. (E) Gaps and long regions of absence of amplification were observed for human chromosomes. An example shown here is chromosome Y. Small gaps were additionally found in the enlarged cluster of amplification. The long absent region (red double arrow) overlapped with the repetitive regions on Y chromosome. (F) Representation of the recovered regions in S and N genes and the human mitochondrial chromosome. The coverage was from 1,000-to 10,000-fold for S and N genes, and 30-to 500-fold for the mitochondrial chromosome. (G) The length of the majority of chromosomal amplification products are clustered around 100bp, while the amplified regions for the S and N genes, as well as the mitochondrial chromosome, were significantly longer.', 'cite_spans': [], 'ref_spans': [], 'section': 'Figure 3. Multiplex PCR-based metagenomic method for the detection of SARS-CoV-2 genes'}, {'text': '. CC-BY-NC-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.12.988246 doi: bioRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'Figure 3. Multiplex PCR-based metagenomic method for the detection of SARS-CoV-2 genes'}, {'text': 'Even when low number of plasmids were used (3 to 14 copies), several targets of 150 to 200 bp in S and N genes were still detected at a total sequencing depth of about 1 million reads. These targets represented 19%, 14%, and 11% of S and N genes when 71, 14 and 2.8 copies of plasmids were used, respectively.', 'cite_spans': [], 'ref_spans': [], 'section': 'Figure 4. Metagenomic method for the detection of limited copies of SARS-CoV-2 genes'}, {'text': '. CC-BY-NC-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.12.988246 doi: bioRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'Figure 4. Metagenomic method for the detection of limited copies of SARS-CoV-2 genes'}, {'text': 'Tremendous efforts have been made from around the world to provide a fast and reliable diagnostic method for SARS-CoV-2. RT-PCR is currently the preferential and most frequently used detection strategy. Yet, we found that the LOD for RT-PCR sits at around 5 copies with 95% confidence, independent of the size of the target genome. Consequently, to overcome the instability of RNA and the genomic sequence variability of virus genome due to evolution, more sensitive and robust methods are required.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Here, we report the development of a multiplex PCR assay for high-sensitivity identification of SARS-CoV-2 infection. With 172 pairs of primers, this method enables the detection of low copy numbers, potentially degraded fragments of the viral genome, or even a mutated variant, with high confidence. The 172 pairs of primers cover about 56% of the genome of SARS-CoV-2. When detecting limited copies of virus, fewer targets are successfully amplified. In the case of one copy of virus, we estimate that 20% of the viral genome, or 6kb of sequences, could be amplified. These are sufficient to produce a conspicuous peak in electrophoresis for an initial indication of positives. The use of NGS sequencing can provide nucleic acid-level information to further confirm the identification, and provide additional evidence for phylogenetic analysis. In addition, our method is robust, accurate and easily performed from different levels of expertise in various laboratory settings.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Furthermore, we also propose a metagenomic method for the potential identification of unknown pathogens. Metagenomic technology usually requires about 20 to 100 million of sequencing reads in order to detect minuscule numbers of targets embedded in the massive amounts of human background. However, our method appears to selectively amplify target sequences. This bias permits the obtention of around 16% of target bases by sequencing at a depth of about 1 million total reads. It is thus especially suitable for the detection of novel pathogens and highly genetically unstable pathogens, such as influenza viruses. The exact mechanism of deselecting human sequences is currently under investigation. The random hexamer preference, chromosomal structure, sequence composition, target length, circularity, methylation status, telomeric and centromeric regions, as well as the edge effect, may influence such outcome. Ultimately, the observed preferential amplification towards more \"random\" sequences could provide us with an advantageous edge for the continuous improvement of this method.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': \"The Universal Human Reference RNA was from Agilent Technologies, Inc. (Cat#74000). The plasmids containing either S or N gene of SARS-CoV-2 (pUC-S and pUC-N, respectively) were purchased from Sangon Biotech, Shanghai, China. The PCR primers used in ddPCR and RT-PCR reactions for S gene are 5'-TGTACTTGGACAATCAAAAAGAGTTGAT and 5'-AGGAGCAGTTGTGAAGTTCTTTTC; for N gene are 5'-GGGGAACTTCTCCTGCTAGAAT and 5'-CAGACATTTTGCTCTCAAGCTG, respectively. 343 pairs of multiplex PCR primers covering the entire genome of SARS-CoV-2 (the panel) were designed by Paragon Genomics, Inc. and separated into two pools. Pool 1, containing 172 pairs of primers, was used in the detection of SARS-CoV-2 by multiplex PCR.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Sample preparation'}, {'text': '50ng of Universal Human Reference RNA was converted into cDNA using random primers and SuperScript™ IV Reverse Transcriptase by following the supplier recommended method (Thermo Fisher Scientific, Cat# 18090050). After reverse transcription, cDNA was purified with 2.4X volume of magnetic beads, and washed twice with 70% ethanol. Finally, the purified cDNA was dissolved in 1X TE buffer and used per multiplex PCR reaction.', 'cite_spans': [], 'ref_spans': [], 'section': 'Reverse transcription'}, {'text': 'Panel design is based on the SARS-CoV-2 sequence NC_045512.2 (https://www.ncbi.nlm.nih.gov/nuccore/NC_045512.2/). In total, 343 primer pairs, distributed into two pools, were selected by a proprietary panel design pipeline to cover the whole genome except for 92 bases at its ends. Primers were optimized to preferentially amplify the SARS-CoV-2 cDNA versus background human cDNA or genomic DNA. They were also optimized to uniformly amplify the covered genome.', 'cite_spans': [], 'ref_spans': [], 'section': 'Multiplex PCR panel design'}, {'text': \"Plasmids pUC-S and pUC-N were combined with human cDNA and used in each multiplex PCR reaction. Paragon Genomics' CleanPlex® multiplex PCR reagents and protocol were used. Briefly, a 10µl multiplex PCR reaction was made by combining 5X mPCR mix, 10X Pool 1 of the panel, plasmid pUC-S, pUC-N and cDNA. The reaction was run in a thermal cycler (95°C for 10min, then 98°C for 15sec, 60°C for 5min for 10 cycles), then terminated by the addition of 2µl of stop buffer. The reaction was then purified by 29µl of magnetic beads, followed by a secondary PCR with a pair of primers for 25 cycles. The secondary PCR added sample indexes and sequencing adapters, allowing for sequencing of the resulting products by high throughput sequencing. A final bead purification was performed after the secondary PCR, followed by library interrogation using a Bioanalyzer 2100 instrument with Agilent High Sensitivity DNA Kit (Agilent Technologies, Inc. Part# 5067-4626). The copyright holder for this preprint (which was not peer-reviewed) is the .\", 'cite_spans': [], 'ref_spans': [], 'section': 'Multiplex PCR'}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'Early Transmission Dynamics in Wuhan, China, of Novel Coronavirus-Infected Pneumonia', 'authors': [{'first': 'Q', 'middle': [], 'last': 'Li', 'suffix': ''}], 'year': 2020, 'venue': 'N Engl J Med', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF1': {'ref_id': 'b1', 'title': 'A pneumonia outbreak associated with a new coronavirus of probable bat origin', 'authors': [{'first': 'P', 'middle': [], 'last': 'Zhou', 'suffix': ''}], 'year': 2020, 'venue': 'Nature', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF3': {'ref_id': 'b3', 'title': 'Economic Impacts of Wuhan 2019-nCoV on China and the World', 'authors': [{'first': 'F', 'middle': ['K'], 'last': 'Ayittey', 'suffix': ''}], 'year': 2020, 'venue': 'J Med Virol', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF4': {'ref_id': 'b4', 'title': 'Genome Composition and Divergence of the Novel Coronavirus (2019-nCoV) Originating in China', 'authors': [{'first': 'A', 'middle': [], 'last': 'Wu', 'suffix': ''}], 'year': 2020, 'venue': 'Cell Host Microbe', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF5': {'ref_id': 'b5', 'title': 'A Novel Coronavirus from Patients with Pneumonia in China', 'authors': [{'first': 'N', 'middle': [], 'last': 'Zhu', 'suffix': ''}], 'year': 2019, 'venue': 'N Engl J Med', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF6': {'ref_id': 'b6', 'title': 'Full-genome evolutionary analysis of the novel corona virus (2019-nCoV) rejects the hypothesis of emergence as a result of a recent recombination event', 'authors': [{'first': 'D', 'middle': [], 'last': 'Paraskevis', 'suffix': ''}], 'year': 2020, 'venue': 'Infect Genet Evol', 'volume': '79', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF7': {'ref_id': 'b7', 'title': 'Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding', 'authors': [{'first': 'R', 'middle': [], 'last': 'Lu', 'suffix': ''}], 'year': 2020, 'venue': 'Lancet', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF8': {'ref_id': 'b8', 'title': 'Genomic variance of the 2019-nCoV coronavirus', 'authors': [{'first': 'C', 'middle': [], 'last': 'Ceraolo', 'suffix': ''}, {'first': 'F', 'middle': ['M'], 'last': 'Giorgi', 'suffix': ''}], 'year': 2020, 'venue': 'J Med Virol', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF9': {'ref_id': 'b9', 'title': 'Measures for diagnosing and treating infections by a novel coronavirus responsible for a pneumonia outbreak originating in Wuhan', 'authors': [{'first': 'F', 'middle': [], 'last': 'Yu', 'suffix': ''}], 'year': 2020, 'venue': 'China. Microbes Infect', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF10': {'ref_id': 'b10', 'title': 'Recent advances in the detection of respiratory virus infection in humans', 'authors': [{'first': 'N', 'middle': [], 'last': 'Zhang', 'suffix': ''}], 'year': 2020, 'venue': 'J Med Virol', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF11': {'ref_id': 'b11', 'title': 'Molecular and serological investigation of 2019-nCoV infected patients: implication of multiple shedding routes', 'authors': [{'first': 'W', 'middle': [], 'last': 'Zhang', 'suffix': ''}], 'year': 2020, 'venue': 'Emerg Microbes Infect', 'volume': '9', 'issn': '1', 'pages': '386--389', 'other_ids': {}}, 'BIBREF12': {'ref_id': 'b12', 'title': 'Rapid Detection of Novel Coronavirus (COVID-19) by Reverse Transcription-Loop-Mediated Isothermal Amplification. medRxiv', 'authors': [{'first': 'L', 'middle': ['E'], 'last': 'Lamb', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF13': {'ref_id': 'b13', 'title': 'Rapid colorimetric detection of COVID-19 coronavirus using a reverse transcriptional loop-mediated isothermal amplification (RT-LAMP) diagnostic plat-form: iLACO. medRxiv', 'authors': [{'first': 'L', 'middle': [], 'last': 'Yu', 'suffix': ''}], 'year': 2002, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF14': {'ref_id': 'b14', 'title': 'Consistent detection of 2019 novel coronavirus in saliva', 'authors': [{'first': 'K', 'middle': ['K'], 'last': 'To', 'suffix': ''}], 'year': 2020, 'venue': 'Clin Infect Dis', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF15': {'ref_id': 'b15', 'title': 'Detectable 2019-nCoV viral RNA in blood is a strong indicator for the further clinical severity', 'authors': [{'first': 'W', 'middle': [], 'last': 'Chen', 'suffix': ''}], 'year': 2020, 'venue': 'Emerg Microbes Infect', 'volume': '9', 'issn': '1', 'pages': '469--473', 'other_ids': {}}, 'BIBREF16': {'ref_id': 'b16', 'title': 'Comparison of different samples for 2019 novel coronavirus detection by nucleic acid amplification tests', 'authors': [{'first': 'C', 'middle': [], 'last': 'Xie', 'suffix': ''}], 'year': 2020, 'venue': 'Int J Infect Dis', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF17': {'ref_id': 'b17', 'title': 'Comparison of throat swabs and sputum specimens for viral nucleic acid detection in 52 cases of novel coronavirus (SARS-Cov-2) infected pneumonia (COVID-19). medRxiv', 'authors': [{'first': 'C', 'middle': [], 'last': 'Lin', 'suffix': ''}], 'year': 2002, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF18': {'ref_id': 'b18', 'title': 'Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR', 'authors': [{'first': 'V', 'middle': ['M'], 'last': 'Corman', 'suffix': ''}], 'year': 2020, 'venue': 'Euro Surveill', 'volume': '', 'issn': '3', 'pages': '', 'other_ids': {}}, 'BIBREF19': {'ref_id': 'b19', 'title': 'Molecular Diagnosis of a Novel Coronavirus (2019-nCoV) Causing an Outbreak of Pneumonia', 'authors': [{'first': 'D', 'middle': ['K W'], 'last': 'Chu', 'suffix': ''}], 'year': 2020, 'venue': 'Clin Chem', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF20': {'ref_id': 'b20', 'title': 'Development of Genetic Diagnostic Methods for Novel Coronavirus 2019 (nCoV-2019) in Japan', 'authors': [{'first': 'K', 'middle': [], 'last': 'Shirato', 'suffix': ''}], 'year': 2020, 'venue': 'Jpn J Infect Dis', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF21': {'ref_id': 'b21', 'title': 'Correlation of Chest CT and RT-PCR Testing in Coronavirus Disease 2019 (COVID-19) in China: A Report of 1014 Cases', 'authors': [{'first': 'T', 'middle': [], 'last': 'Ai', 'suffix': ''}], 'year': 2020, 'venue': 'Radiology', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF22': {'ref_id': 'b22', 'title': 'Chest CT for Typical 2019-nCoV Pneumonia: Relationship to Negative RT-PCR Testing', 'authors': [{'first': 'X', 'middle': [], 'last': 'Xie', 'suffix': ''}], 'year': 2020, 'venue': 'Radiology', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF23': {'ref_id': 'b23', 'title': 'Comparison of the clinical characteristics between RNA positive and negative patients clinically diagnosed with', 'authors': [{'first': 'Y', 'middle': ['Y'], 'last': 'Li', 'suffix': ''}], 'year': 2019, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF26': {'ref_id': 'b26', 'title': 'Racing towards the development of diagnostics for a novel coronavirus (2019-nCoV)', 'authors': [{'first': 'Dennis', 'middle': [], 'last': 'Lo', 'suffix': ''}, {'first': 'Y', 'middle': ['M'], 'last': '', 'suffix': ''}, {'first': 'R', 'middle': ['W K'], 'last': 'Chiu', 'suffix': ''}], 'year': 2020, 'venue': 'Clin Chem', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF27': {'ref_id': 'b27', 'title': 'Clinical and biochemical indexes from 2019-nCoV infected patients linked to viral loads and lung injury', 'authors': [{'first': 'Y', 'middle': [], 'last': 'Liu', 'suffix': ''}], 'year': 2020, 'venue': 'Sci China Life Sci', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF28': {'ref_id': 'b28', 'title': 'First cases of coronavirus disease 2019 (COVID-19) in France: surveillance, investigations and control measures', 'authors': [{'first': 'S', 'middle': [], 'last': 'Bernard Stoecklin', 'suffix': ''}], 'year': 2020, 'venue': 'Euro Surveill', 'volume': '', 'issn': '6', 'pages': '', 'other_ids': {}}, 'BIBREF29': {'ref_id': 'b29', 'title': 'Novel Coronavirus Outbreak in Wuhan, China, 2020: Intense Surveillance Is Vital for Preventing Sustained Transmission in New Locations', 'authors': [{'first': 'R', 'middle': ['N'], 'last': 'Thompson', 'suffix': ''}], 'year': None, 'venue': 'J Clin Med', 'volume': '2020', 'issn': '2', 'pages': '', 'other_ids': {}}, 'BIBREF30': {'ref_id': 'b30', 'title': 'Laboratory readiness and response for novel coronavirus (2019-nCoV) in expert laboratories in 30 EU/EEA countries', 'authors': [{'first': 'C', 'middle': [], 'last': 'Reusken', 'suffix': ''}], 'year': 2020, 'venue': 'Euro Surveill', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF31': {'ref_id': 'b31', 'title': 'Guidelines for the Diagnosis and Treatment of Novel Coronavirus (2019-nCoV) Infection by the National Health Commission (Trial Version 5', 'authors': [{'first': 'L', 'middle': [], 'last': 'Lin', 'suffix': ''}, {'first': 'T', 'middle': ['S'], 'last': 'Li', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF32': {'ref_id': 'b32', 'title': '100(0): p. E001. 32. [Diagnosis and clinical management of 2019 novel coronavirus infection: an operational recommendation of Peking Union Medical College Hospital', 'authors': [{'first': 'Zhonghua Yi Xue Za', 'middle': [], 'last': 'Zhi', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF34': {'ref_id': 'b34', 'title': 'Genome Detective Coronavirus Typing Tool for rapid identification and characterization of novel coronavirus genomes', 'authors': [{'first': 'S', 'middle': [], 'last': 'Cleemput', 'suffix': ''}], 'year': 2020, 'venue': 'Bioinformatics', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF35': {'ref_id': 'b35', 'title': \"Diagnosis, treatment, and prevention of 2019 novel coronavirus infection in children: experts' consensus statement\", 'authors': [{'first': 'K', 'middle': [], 'last': 'Shen', 'suffix': ''}], 'year': 2020, 'venue': 'World J Pediatr', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF36': {'ref_id': 'b36', 'title': 'Emerging novel Coronavirus (2019-nCoV) -Current scenario, evolutionary perspective based on genome analysis and recent developments', 'authors': [{'first': 'Y', 'middle': ['S'], 'last': 'Malik', 'suffix': ''}], 'year': 2020, 'venue': 'Vet Q', 'volume': '', 'issn': '', 'pages': '1--12', 'other_ids': {}}, 'BIBREF37': {'ref_id': 'b37', 'title': \"Potential of large 'first generation' human-to-human transmission of 2019-nCoV\", 'authors': [{'first': 'X', 'middle': [], 'last': 'Li', 'suffix': ''}], 'year': 2020, 'venue': 'J Med Virol', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF38': {'ref_id': 'b38', 'title': 'Transmission dynamics and evolutionary history of 2019-nCoV', 'authors': [{'first': 'X', 'middle': [], 'last': 'Li', 'suffix': ''}], 'year': 2020, 'venue': 'J Med Virol', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF39': {'ref_id': 'b39', 'title': 'Homologous recombination within the spike glycoprotein of the newly identified coronavirus may boost cross-species transmission from snake to human', 'authors': [{'first': 'W', 'middle': [], 'last': 'Ji', 'suffix': ''}], 'year': 2020, 'venue': 'J Med Virol', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF40': {'ref_id': 'b40', 'title': 'The 2019-new coronavirus epidemic: Evidence for virus evolution', 'authors': [{'first': 'D', 'middle': [], 'last': 'Benvenuto', 'suffix': ''}], 'year': 2020, 'venue': 'J Med Virol', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF41': {'ref_id': 'b41', 'title': 'The global spread of 2019-nCoV: a molecular evolutionary analysis', 'authors': [{'first': 'D', 'middle': [], 'last': 'Benvenuto', 'suffix': ''}], 'year': 2020, 'venue': 'Pathog Glob Health', 'volume': '', 'issn': '', 'pages': '1--4', 'other_ids': {}}, 'BIBREF42': {'ref_id': 'b42', 'title': 'The first two cases of 2019-nCoV in Italy: Where they come from', 'authors': [{'first': 'M', 'middle': [], 'last': 'Giovanetti', 'suffix': ''}], 'year': 2020, 'venue': 'J Med Virol', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF43': {'ref_id': 'b43', 'title': 'Sensitive, highly multiplexed sequencing of microhaplotypes from the <em&gt', 'authors': [{'first': 'S', 'middle': ['K'], 'last': 'Tessema', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF44': {'ref_id': 'b44', 'title': 'RNA based mNGS approach identifies a novel human coronavirus from two individual pneumonia cases in 2019 Wuhan outbreak', 'authors': [{'first': 'L', 'middle': [], 'last': 'Chen', 'suffix': ''}], 'year': 2020, 'venue': 'Emerg Microbes Infect', 'volume': '9', 'issn': '1', 'pages': '313--319', 'other_ids': {}}, 'BIBREF45': {'ref_id': 'b45', 'title': 'Amplicon-Based Targeted Sequencing of Single Circulating Tumor Cells', 'authors': [{'first': 'N', 'middle': [], 'last': 'Ericson', 'suffix': ''}], 'year': None, 'venue': 'Journal of Molecular Diagnostics', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}}, 'ref_entries': {'FIGREF0': {'text': 'Mathematical model, primer design and workflow(A) A mathematical model of RT-PCR based on Poisson process. The LOD is 4.8 copies of virus at 95% probability. (B) Two overlapping pools of multiplex PCR primers, shown on the right of the genome of SARS-CoV-2, were designed to span the entire virus genome. Pool 1, containing 172 pairs of primers, covers 56.9% of the viral genome and was used in the detection. Pool 2 contains 171 pairs of primers and covers 56.4% of the genome. Both pools are used to cover the full length of the genome. (C) A mathematical model of multiplex PCR with pool 1 of the primers. The LOD is 1.15 copies of virus at 95% probability. (D) The workflow of the multiplex PCR method. The prepared libraries can be detected using high-resolution electrophoresis, and sequenced together with other samples using high-throughput sequencing.', 'latex': None, 'type': 'figure'}, 'FIGREF1': {'text': 'al., Comparison of four digital PCR platforms for accurate quantification of DNA copy number of a certified plasmid DNA reference material. Scientific Reports, 2015.', 'latex': None, 'type': 'figure'}}, 'back_matter': [{'text': 'The authors would like to thank Jian Xu of MGI, BGI-Shenzhen for technical support in MGI sequencing, Dr. Jin Billy Li of Stanford University for the coordination of academic support, Dr. Alexander E. Urban of Stanford University for suggestions in preparing the manuscript, and Dr. Zihuai He of Stanford University for discussion on a potential mathematical model of the metagenomic method.', 'cite_spans': [], 'ref_spans': [], 'section': 'Acknowledgements'}]}\n",
      "{'paper_id': 'ac4a3bbe27272cd6780e7168906b0bf1413fc749', 'metadata': {'title': 'Innate Immune Priming by cGAS as a Preparatory Countermeasure Against 1 RNA Virus Infection 2 3', 'authors': [{'first': 'Michael', 'middle': ['T'], 'last': 'Parker', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Yale University School of Medicine', 'location': {'addrLine': 'New 7 Haven', 'settlement': 'Connecticut', 'country': 'USA'}}, 'email': ''}, {'first': 'Smita', 'middle': [], 'last': 'Gopinath', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Yale University School of Medicine', 'location': {'addrLine': '9 Connecticut', 'settlement': 'New Haven', 'country': 'USA'}}, 'email': ''}, {'first': 'Corey', 'middle': ['E'], 'last': 'Perez', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Yale University', 'location': {'settlement': 'New Haven', 'region': 'Connecticut', 'country': 'USA'}}, 'email': ''}, {'first': 'Melissa', 'middle': ['M'], 'last': 'Linehan', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Yale University School of Medicine', 'location': {'addrLine': '9 Connecticut', 'settlement': 'New Haven', 'country': 'USA'}}, 'email': ''}, {'first': 'Jason', 'middle': ['M'], 'last': 'Crawford', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Yale University School of Medicine', 'location': {'addrLine': 'New 7 Haven', 'settlement': 'Connecticut', 'country': 'USA'}}, 'email': ''}, {'first': 'Akiko', 'middle': [], 'last': 'Iwasaki', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Yale University School of Medicine', 'location': {'addrLine': '9 Connecticut', 'settlement': 'New Haven', 'country': 'USA'}}, 'email': ''}, {'first': 'Brett', 'middle': ['D'], 'last': 'Lindenbach', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Yale University School of Medicine', 'location': {'addrLine': 'New 7 Haven', 'settlement': 'Connecticut', 'country': 'USA'}}, 'email': ''}]}, 'abstract': [{'text': 'The detection of nucleic acids by pattern recognition receptors is an ancient and 27 conserved component of the innate immune system. Notably, RNA virus genomes 28 are sensed by mammalian cytosolic RIG-I-like receptors, thereby activating 29 interferon-stimulated gene (ISG) expression to restrict viral replication. However, 30 recent evidence indicates that the cGAS-STING DNA sensing pathway also 31 protects against RNA viruses. So far, the mechanisms responsible for DNA sensing 32 of RNA viruses, which replicate without known DNA intermediates, remain unclear. 33', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'By using cGAS gene knockout and reconstitution in human and mouse cell 34', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': \", we discovered that DNA sensing and cGAMP synthase activities are 35 required for cGAS-mediated restriction of vesicular stomatitis virus and Sindbis 36 virus. The level of cGAMP produced in response to RNA virus infection was below 37 the threshold of detection, suggesting that only transient and/or low levels of 38 cGAMP are produced during RNA virus infections. To clarify the DNA ligands that 39 activate cGAS activity, we confirmed that cGAS binds mitochondrial DNA in the 40 cytosol of both uninfected and infected cells; however, the amount of 41 cGAS-associated mitochondrial DNA did not change in response to virus infection. 42 Rather, a variety of pre-existing cytosolic DNAs, including mitochondrial DNA and 43 endogenous cDNAs, may serve as stimuli for basal cGAS activation. Importantly, 44 cGAS knockout and reconstitution experiments demonstrated that cGAS drives 45 low-level ISG expression at steady state. We propose that cGAS-STING restricts 46 RNA viruses by promoting a preparatory immune activation state within cells, likely 47 primed by endogenous cellular DNA ligands. 48 . Importance 49 Many medically important RNA viruses are restricted by the cGAS-STING 50 DNA-sensing pathway of innate immune activation. Since these viruses replicate 51 without DNA intermediates, it is unclear what DNA ligand(s) are responsible for 52 triggering this pathway. We show here that cGAS's DNA binding and signaling 53 activities are required for RNA virus restriction, similar to the mechanisms by which it 54 restricts DNA viruses. Furthermore, we confirmed that cGAS continuously binds host 55 DNA, which was unaffected by RNA virus infection. Finally, cGAS expression 56\", 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'correlated with the low-level expression of interferon-stimulated genes in uninfected 57 cells, both in vitro and in vivo. We propose that cGAS-mediated sensing of 58 endogenous DNA ligands contributes to RNA virus restriction by establishing a 59 baseline of innate immune activation. doi: bioRxiv preprint 61 A key feature of innate immunity is the detection of pathogen-associated 62 molecular patterns (PAMPs) by pattern recognition receptors (PRRs) (1). For 63 mammalian cells, viral nucleic acids are detected by distinct PRRs, triggering 64 interferon-stimulated gene (ISG) expression to set up an antiviral state. During RNA 65 virus infections, uncapped and double-stranded RNAs are detected in the cytosol by 66 the PRRs retinoic acid-inducible gene I (RIG-I) and related RIG-I-like receptors 67 (RLRs). However, the recent discovery of the cGAS-STING cytosolic DNA sensing 68 pathway, and the observation that it can also restrict RNA viruses (2), reveals a need 69 to further investigate the mechanisms of nucleic acid sensing during RNA virus 70 infection. 71 The stimulator of interferon genes (STING) is an endoplasmic reticulum-and 72 mitochondrial-bound protein that spontaneously activates ISG expression when 73 overexpressed (2). Although STING is involved in DNA sensing, STING -/mice and 74 mouse endothelial fibroblasts (MEFs) are more permissive for vesicular stomatitis 75 virus (VSV), a negative-stand RNA virus (2, 3). Additionally, studies in MEFs 76 deficient in three prime repair exonuclease 1 (TREX1), a nuclease important for the 77 turnover of cytosolic retroelement cDNAs (4), have described enhanced antiviral 78 phenotypes in response to a wide array of RNA viruses and retroviruses, 79', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'presumably due to the accumulation of DNA in the cytosol (5, 6). It appears that this 80 DNA-based restriction is broad, as many RNA viruses have evolved mechanisms to 81 subvert the cGAS-STING pathway, including flaviviruses (7-9), hepaciviruses (10, 82', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'STING does not directly interact with cytosolic DNA, but functions as an innate 84 immune adaptor protein to transduce signals between cyclic GMP-AMP synthase 85 (cGAS) and Tank-binding kinase 1, which subsequently phosphorylates the 86 transcription factor interferon regulatory factor 3 (IRF3) to initiate an ISG response doi: bioRxiv preprint 157', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': \"158 growth of VSV-GFP, VSVΔM51A-GFP, and SINV-GFP, while THP-1 KO cells or 159 THP-1 KO cells reconstituted with inactive hcGAS-HA3x mutants showed little or no 160 restriction (Figure 3D-3F). As observed previously in MEFs, VSVΔM51A-GFP was 161 more potently restricted than VSV-GFP, but unlike in MEFs, infected fewer cells 162 expressing mutant cGAS. This was also true for SINV-GFP, albeit restriction with 163 WT hcGAS-HA3x was extremely potent, comparatively. It is unclear whether these 164 modest decreases in infection of the cGAS mutants was due to hcGAS-HA3x 165 overexpression in THP-1 cells, residual hcGAS activities, or normal clonal 166 variation of cells. Nevertheless, these results are most consistent with an integral 167 role for cGAS DNA binding and cGAMP synthase activities in RNA virus restriction. 168 Detection of cGAMP produced in response to DNA transfection but not 169 RNA virus infection. Because cGAMP synthesis activity was essential for RNA 170 virus restriction, we next sought to identify cGAMP produced in response to RNA . 676 2013. The papain-like protease of porcine epidemic diarrhea virus negatively 677 regulates type I interferon pathway by acting as a viral deubiquitinase. The 678 Journal of General Virology 94:1554-1567. 679 JL, Mar KB, Richardson RB, Ratushny AV, Litvak V, Dabelic R, 704 Manicassamy B, Aitchison JD, Aderem A, Elliott RM, Garcia-Sastre A, 705 Racaniello V, Snijder EJ, Yokoyama WM, Diamond MS, Virgin HW, Rice 706 CM. 2014. Pan-viral specificity of IFN-induced genes reveals new roles for 707 cGAS in innate immunity. Nature 505:691-695. 708 . CC-BY-NC-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/434027 doi: bioRxiv preprint Flamand A. 1990. Genetic evidence for multiple functions of the matrix 761 protein of vesicular stomatitis virus. J Gen Virol 71 ( Pt 4):991-996. 762 39. Li L, Yin Q, Kuss P, Maliga Z, Millan JL, Wu H, Mitchison TJ. 2014. 763 Hydrolysis of 2'3'-cGAMP by ENPP1 and design of nonhydrolyzable analogs. 764 P, Shu H-B. 2008. The Adaptor Protein MITA Links Virus-Sensing Receptors 794 to IRF3 Transcription Factor Activation. Immunity 29:538-550. 795 48. Castanier C, Garcin D, Vazquez A, Arnoult D. 2010. Mitochondrial 796 dynamics regulate the RIG-I-like receptor antiviral pathway. EMBO Rep 797 11:133-138. 798 49. Horner SM, Liu HM, Park HS, Briley J, Gale M. 2011. 799 Mitochondrial-associated endoplasmic reticulum membranes (MAM) form 800 innate immune synapses and are targeted by hepatitis C virus. Proceedings 801 of the National Academy of Sciences 108:14590-14595. 802 50. Schoggins JW, MacDuff DA, Imanaka N, Gainey MD, Shrestha B, Eitson 803 JL, Mar KB, Richardson RB, Ratushny AV, Litvak V, Dabelic R, 804 Manicassamy B, Aitchison JD, Aderem A, Elliott RM, Garcia-Sastre A, 805 Racaniello V, Snijder EJ, Yokoyama WM, Diamond MS, Virgin HW, Rice 806 CM. 2014. Pan-viral specificity of IFN-induced genes reveals new roles for 807 cGAS in innate immunity. Nature 505:691-695. 808 51. Liu Y, Liu Y, Wu J, Roizman B, Zhou GG. 2018. Innate responses to gene 809 knockouts impact overlapping gene networks and vary with respect to 810 resistance to viral infection. Proceedings of the National Academy of 811 Sciences 115:E3230. 812 52. Schoggins JW, Wilson SJ, Panis M, Murphy MY, Jones CT, Bieniasz P, 813 Rice CM. 2011. A diverse range of gene products are effectors of the type I 814 interferon antiviral response. Nature 472:481-485. 815 53.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}], 'body_text': [{'text': \"Introduction mechanisms and may restrict RNA virus replication independent of IRF3 activation 89 (20) . 90 cGAS is a nucleic acid-binding protein specific for dsDNA and DNA:RNA hybrids 91 that also has nucleotidyl transferase activity (21) (22) (23) (24) . DNA binding induces structural 92 changes to form the cGAS active site, which synthesizes a non-canonical 5´-2'-and 93 5´-3'-linked cyclic dinucleotide known as cyclic guanosine monophosphate-94 adenosine monophosphate (cGAMP) (25-28). cGAMP is a diffusible secondary 95 messenger that specifically binds to STING with high affinity (K D ~4 nM), thereby 96 inducing a downstream innate immune response (29) (30) (31) (32) . 97\", 'cite_spans': [{'start': 97, 'end': 101, 'text': '(20)', 'ref_id': None}, {'start': 235, 'end': 239, 'text': '(21)', 'ref_id': None}, {'start': 240, 'end': 244, 'text': '(22)', 'ref_id': None}, {'start': 245, 'end': 249, 'text': '(23)', 'ref_id': None}, {'start': 250, 'end': 254, 'text': '(24)', 'ref_id': None}, {'start': 659, 'end': 663, 'text': '(29)', 'ref_id': None}, {'start': 664, 'end': 668, 'text': '(30)', 'ref_id': None}, {'start': 669, 'end': 673, 'text': '(31)', 'ref_id': None}, {'start': 674, 'end': 678, 'text': '(32)', 'ref_id': None}], 'ref_spans': [], 'section': ''}, {'text': 'For RNA viruses that replicate in the cytosol without a DNA intermediate, the 98 specific ligands that activate cGAS remain unclear. At present, the prevailing 99 hypothesis is that RNA viruses induce release of mitochondrial DNA (mtDNA) into 100 the cytosol, thereby activating innate immune responses (7, (33) (34) (35) (36) . However, it is 101 unclear whether mitochondrial damage is a conserved feature of RNA virus 102 infection, nor is it clear that cGAS-STING activation follows the same pathway for 103 both RNA and DNA viruses. 104 In this study, we investigated whether the DNA binding and cGAMP synthesis 105 activities of human cGAS (hcGAS) are required for RNA virus restriction. While both 106 activities were required, the amount of cGAMP produced during virus infection was 107 too low to detect. We also confirmed that hcGAS binds mtDNA in both uninfected 108 and infected cells but did not observe increased cytosolic or cGAS-associated 109 mtDNA in response to RNA virus infection. We found that cGAS stimulated 110 smoldering, low-level innate immune activation, most likely in response to 111 endogenous DNA ligands, suggesting that cGAS-STING can passively restrict 112 incoming RNA viruses. 113 . CC-BY-NC-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/434027 doi: bioRxiv preprint virus infection or, as a positive control, DNA transfection, by using liquid 172', 'cite_spans': [{'start': 303, 'end': 306, 'text': '(7,', 'ref_id': None}, {'start': 307, 'end': 311, 'text': '(33)', 'ref_id': None}, {'start': 312, 'end': 316, 'text': '(34)', 'ref_id': None}, {'start': 317, 'end': 321, 'text': '(35)', 'ref_id': None}, {'start': 322, 'end': 326, 'text': '(36)', 'ref_id': None}, {'start': 517, 'end': 541, 'text': 'RNA and DNA viruses. 104', 'ref_id': None}], 'ref_spans': [], 'section': ''}, {'text': 'were used in these experiments because this cell line lacks endogenous cGAS 174 expression and could be reconstituted with WT or mutant hcGAS-HA3x; however, 175 unlike MEFs and THP-1 KO cells, HEK 293E cells could be efficiently transfected 176 with DNA and readily scaled up for isolation of cGAMP from cytosolic extracts. As 177 shown in Figure 4A , a unique UHPLC peak (~5 minutes elution) was observed after 178 transfecting WT hcGAS-HA3x-expressing HEK 293E cells with salmon sperm DNA; 179 MS analysis confirmed that this peak corresponded to cGAMP (Figures 4B and 4C) . published cGAMP bioassays (21). Again, we were unable to detect cGAMP in 194 lysates from VSV-infected or SINV-infected THP-1 cells expressing WT hcGAS, 195 while a synthetic cGAMP control led to robust phosphorylation of IRF3 ( Figure 4I ). 196', 'cite_spans': [{'start': 555, 'end': 574, 'text': '(Figures 4B and 4C)', 'ref_id': None}, {'start': 720, 'end': 729, 'text': 'WT hcGAS,', 'ref_id': None}, {'start': 730, 'end': 733, 'text': '195', 'ref_id': None}], 'ref_spans': [{'start': 340, 'end': 349, 'text': 'Figure 4A', 'ref_id': None}, {'start': 806, 'end': 815, 'text': 'Figure 4I', 'ref_id': None}], 'section': 'chromatography-mass spectrophotometry (LC-MS) and LC-MS/MS. HEK 293E cells 173'}, {'text': 'To validate that cell-derived cGAMP could be detected by this assay, a time-course 197 experiment was conducted by transfecting HEK 293E cells expressing WT hcGAS 198 with salmon sperm DNA, revealing the time-dependent increase in cGAMP (Figure 199 4J). Furthermore, we found that transfected cGAMP was rapidly turned over within 200 . CC-BY-NC-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/434027 doi: bioRxiv preprint hours (Fig. 4K ), most likely via the ENPP1 phosphodiesterase previously reported to 201 turnover cGAMP in mammalian cells (39) . Collectively, these results indicate that if 202 cGAMP is produced in response to RNA virus infection, it may be produced at levels 203 below the limit of our detection and/or rapidly turned over. 204 cGAS binds mitochondrial DNA at steady state and during RNA virus 205 infection. Given that cGAS DNA binding activity was also required for RNA virus 206 restriction, we sought to identify DNA ligands of cGAS during RNA virus infection. 207', 'cite_spans': [{'start': 154, 'end': 166, 'text': 'WT hcGAS 198', 'ref_id': None}, {'start': 237, 'end': 248, 'text': '(Figure 199', 'ref_id': None}, {'start': 672, 'end': 676, 'text': '(39)', 'ref_id': None}], 'ref_spans': [{'start': 555, 'end': 563, 'text': '(Fig. 4K', 'ref_id': None}], 'section': 'chromatography-mass spectrophotometry (LC-MS) and LC-MS/MS. HEK 293E cells 173'}, {'text': 'First, we identified conditions to specifically co-immunoprecipitate cGAS and 208 mtDNA, a known DNA ligand (34). As shown in Figure 5A To more broadly assess cytosolic and hcGAS-bound DNAs, we developed deep 220 sequencing libraries from cytosolic extracts or after immunoprecipitation of WT 221 hcGAS-HA3x. The first one-third of the mitochondrial genome was specifically 222 enriched in cytosolic preps from both uninfected and VSV-GFP-infected MEFs 223 ( Figure 5F ). Similarly, mtDNA was also highly enriched after immunoprecipitation of 224 hcGAS-HA3x, although there was a bias for the latter three-quarters of the genome 225 ( Figure 5G ). Importantly, there was no obvious difference in mtDNA pulldown 226 between uninfected and infected cells. Collectively, these data indicate that VSV 227 does not induce cytosolic release of mtDNA to stimulate cGAS activation. Consistent 228 with this, VSV-GFP replicated equally well in LMTK cells and mtDNA-depleted LMTK 229 . CC-BY-NC-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/434027 doi: bioRxiv preprint \\uf072 0 cells (41), which express cGAS and STING ( Figure 5H ). Collectively, these data 230 suggest that mtDNA is dispensable for cGAS-mediated restriction of an RNA virus. 231', 'cite_spans': [], 'ref_spans': [{'start': 126, 'end': 135, 'text': 'Figure 5A', 'ref_id': None}, {'start': 459, 'end': 468, 'text': 'Figure 5F', 'ref_id': None}, {'start': 635, 'end': 644, 'text': 'Figure 5G', 'ref_id': None}, {'start': 1236, 'end': 1245, 'text': 'Figure 5H', 'ref_id': None}], 'section': 'chromatography-mass spectrophotometry (LC-MS) and LC-MS/MS. HEK 293E cells 173'}, {'text': 'Although VSV is a negative-strand RNA virus that replicates solely via RNA 232', 'cite_spans': [], 'ref_spans': [], 'section': 'chromatography-mass spectrophotometry (LC-MS) and LC-MS/MS. HEK 293E cells 173'}, {'text': 'intermediates, it has been reported that VSV-specific cDNAs can arise in infected 233 cells, presumably through reverse transcriptase (RT) activity encoded by 234 endogenous retroelement(s) (42). We therefore investigated whether such viral 235 cDNAs arose during VSV-GFP infections in our laboratory. Indeed, VSV N 236 gene-specific cDNAs were generated in infected cells, although in extremely low 237 abundance, ~1 copy/10 4 cells ( Figure 5I ). The cDNA origin of the N gene template 238 was confirmed by nuclease treatment ( Figure 5J ), by its sensitivity to tenofovir, an 239 RT inhibitor that had no effect on VSV replication ( Figure S3A ), and by its enhanced 240 expression in cells devoid of TREX1 nuclease ( Figure S3B ). We also identified 241 virus-specific cDNAs in cells infected with yellow fever virus (YFV), a positive-strand 242 RNA virus ( Figure S3C ), suggesting that cDNA formation is a general feature of RNA 243 virus infections. Finally, to determine whether cDNA formation was specific to 244 virus-infected cells or to viral transcripts, we examined whether cDNA forms of an 245 abundant housekeeping gene, GAPDH, arose in uninfected cells. Indeed, 246 splice-dependent GAPDH cDNAs were identified in low abundance by qPCR ( Figure  247 5K). Importantly, VSV or retroelement cDNAs were not detected in deep sequencing 248 analyses of whole cytosol or cGAS-HA immunoprecipitations, likely due to their low 249', 'cite_spans': [], 'ref_spans': [{'start': 436, 'end': 445, 'text': 'Figure 5I', 'ref_id': None}, {'start': 530, 'end': 539, 'text': 'Figure 5J', 'ref_id': None}, {'start': 636, 'end': 646, 'text': 'Figure S3A', 'ref_id': None}, {'start': 721, 'end': 731, 'text': 'Figure S3B', 'ref_id': None}, {'start': 862, 'end': 872, 'text': 'Figure S3C', 'ref_id': None}, {'start': 1255, 'end': 1266, 'text': 'Figure  247', 'ref_id': None}], 'section': 'chromatography-mass spectrophotometry (LC-MS) and LC-MS/MS. HEK 293E cells 173'}, {'text': \"Collectively, our results indicate that cGAS binds mtDNA in both infected and 251 uninfected cells, and that VSV infection does not induce the release of mtDNA into 252 the cytosol or increase cGAS-bound mtDNA. Additionally, viral and cellular 253 mRNA-specific cDNAs can be detected, but are of extremely low abundance, less 254 than one copy per 10 4 cells. Taken together, these results suggest that steady state 255 levels of cytosolic DNA, rather than virus-induced DNAs, may provide ligands for 256 cGAS-mediated restriction of RNA virus replication. 257 . CC-BY-NC-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/434027 doi: bioRxiv preprint cGAS primes smoldering baseline ISG expression. Based on the above 258 results, we hypothesized that cGAS may serve to program baseline levels of innate 259 immune activation rather than strictly in response to RNA virus infection. To address 260 this, we analyzed ISRE-driven luciferase expression in uninfected THP-1 cells To examine whether cGAS drives basal levels of innate immune activation in 279 vivo, we examined ISG expression in vaginal tissue from uninfected WT B6J mice or 280 in mice defective for several innate immunity pathways. As shown in Fig. 7 Although cGAS was previously reported to restrict RNA viruses (50), it has been 301 widely assumedthough unproventhat this restriction depends on cGAS's DNA 302 binding and cGAMP synthase activities. Here, we used genetic knockout and 303 transgenic replacement to determine that both DNA binding and cGAMP synthase 304 activities are essential for cGAS-mediated restriction of RNA viruses. One caveat to 305 this approach is that gene knockout can have far-reaching network-level effects on 306 transcription, which are just beginning to be unearthed (51). A second caveat is that 307 reconstituted cGAS was slightly overexpressed in THP-1 cells, which, at least for WT 308 cGAS, can induce ISG expression (50, 52) and may have exaggerated the 309 response. Nevertheless, our results in THP-1 cells were consistent with results 310 obtained from MEFs (Figure 3 ), which did not overexpress cGAS. Taken together, 311 these data establish that DNA binding and cGAMP synthase activities are required 312 for cGAS-mediated RNA virus restriction. 313 . CC-BY-NC-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/434027 doi: bioRxiv preprint Despite the essential role of cGAMP synthase activity and demonstrated 314 detection of cGAMP synthesized after DNA transfection, we were unable to detect 315 cGAMP production in response to VSV-GFP infection. Our results are consistent 316 with results recently reported by Franz et al., who were also unsuccessful in 317 detecting cGAMP production in . While Franz and colleagues 318 concluded that cGAMP is not produced in response to VSV infection, we also 319 considered the possibility that cGAMP levels may be below the limit of detection 320 and/or rapidly degraded. Whereas cGAMP synthesis is readily detected in response 321\", 'cite_spans': [], 'ref_spans': [{'start': 1334, 'end': 1340, 'text': 'Fig. 7', 'ref_id': None}, {'start': 2193, 'end': 2202, 'text': '(Figure 3', 'ref_id': None}], 'section': 'abundance. 250'}, {'text': 'to DNA transfection, this may simply reflect the wide dynamic range of cGAS in 322 response to overloading the cytosol with transfected DNA. Moreover, it has been 323 exceedingly difficult to detect cGAMP after virus infections, even for DNA viruses. 324', 'cite_spans': [], 'ref_spans': [], 'section': 'abundance. 250'}, {'text': \"to cytomegalovirus infection was cell type-dependent, despite active cGAS-STING 326 expression. Where cGAMP was detected, levels were on the order of 5 fmol/10 4 327 cells, or ~3x10 5 molecules/cell, which was slightly above their assay's limit of 328 detection (53). As our biochemical and biological assays were both less sensitive 329 than that of Paijo et al., we surmise that the synthesis of cGAMP in response to RNA 330 virus infection is below the limit of detection and/or may be rapidly turned over. 331 Alternatively, continuous low-level production of cGAMP in response to endogenous 332 DNA ligands may be more relevant to RNA virus restriction. Clearly, cGAMP assays 333 with improved sensitivity are needed to discern between these possibilities. 334\", 'cite_spans': [], 'ref_spans': [], 'section': 'For instance, Paijo et al. reported that the detection of cGAMP produced in response 325'}, {'text': 'Because cGAS DNA binding activity was required for VSV restriction, we 335 examined whether VSV introduces cGAS DNA ligands into the cytosol. Prior work 336 has shown that the cytosolic release of mtDNA activates the cGAS-STING pathway 337 (33-35); moreover, infection with HSV-1, a DNA virus, or dengue virus, an RNA 338 virus, reportedly causes cytosolic release of mtDNA (34, 40). An emerging concept 339 is that mammalian cells may regulate the efflux of mtDNA into the cytosol in 340 response to stress, supported by a role for the Bax/Bak pore in mtDNA release as 341 well as mitochondrial inner membrane release mechanisms via permeabilization and 342 . CC-BY-NC-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/434027 doi: bioRxiv preprint herniation (33-35, 54, 55) . In contrast, the levels of cytosolic mtDNA and 343 cGAS-associated mtDNA did not increase during VSV infection. Moreover, 344 cGAS-STING-mediated VSV restriction was intact in \\uf072 0 cells, which lack mtDNA, 345 consistent with similar experiments reported by Franz et al. (20) . Real-time 346 examination of mitochondrial dynamics may be needed to clarify the role of mtDNA 347 release during RNA virus infections. 348', 'cite_spans': [{'start': 885, 'end': 900, 'text': '(33-35, 54, 55)', 'ref_id': None}, {'start': 1160, 'end': 1177, 'text': 'Franz et al. (20)', 'ref_id': None}], 'ref_spans': [], 'section': 'For instance, Paijo et al. reported that the detection of cGAMP produced in response 325'}, {'text': 'Given that cGAS recognizes RNA:DNA hybrids (22), as well as a recent report of 349 VSV cDNAs (42), we also quantitated viral cDNAs produced during VSV infection. 350', 'cite_spans': [], 'ref_spans': [], 'section': 'For instance, Paijo et al. reported that the detection of cGAMP produced in response 325'}, {'text': 'We confirmed that rare viral and cellular cDNAs are indeed produced, most likely by 351 an endogenous cellular RT; however, the abundance of any given cDNA was 352 incredibly low, ~1 copy per 10 4 cells. This was less than the amount of VSV N-gene 353 cDNA previous reported by Shimizu, et al. (42) , which we attribute to the enhanced 354 specificity of our hydrolysis probe-based assay vs. SYBR green assays. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/434027 doi: bioRxiv preprint models where cGAS responds to RNA virus-induced release of mtDNA. Additional 372 work will be needed to definitively identify the relevant DNA ligands that activate 373 cGAS; we suggest that pre-existing baseline stimuli should be considered. 374', 'cite_spans': [{'start': 278, 'end': 286, 'text': 'Shimizu,', 'ref_id': None}, {'start': 287, 'end': 298, 'text': 'et al. (42)', 'ref_id': None}], 'ref_spans': [], 'section': 'For instance, Paijo et al. reported that the detection of cGAMP produced in response 325'}, {'text': 'While ISG expression served as a convenient and sensitive readout of baseline 375 cGAS-STING activation in our studies, it should be noted that we did not 376 demonstrate that low-level ISG expression directly contributes to RNA virus 377 restriction. On the surface, our results may seem at odds with those of Franz et al., 378 who recently reported that STING restricts RNA viruses, including VSV, in an 379 ISG-independent manner (20). However, we do not exclude the possibility that 380 smoldering cGAS activation may also contribute to ISG-independent mechanisms of 381 virus restriction via STING. 382', 'cite_spans': [], 'ref_spans': [], 'section': 'For instance, Paijo et al. reported that the detection of cGAMP produced in response 325'}, {'text': 'In summary, we propose that cGAS may become activated in response to RNA 383 virus infection, such as by virus-induced mtDNA release, but also contributes to RNA 384 virus restriction via constitutive, low-level innate immune activation, likely via 385 recognition of endogenous DNA ligands (Fig. 8) . Pilot experiments showed that THP-1-derived macrophages were more 414 permissive for SINV-GFP than undifferentiated THP-1 monocytes. Differentiation of 415 . CC-BY-NC-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/434027 doi: bioRxiv preprint THP-1 monocytes to macrophages was performed by plating cells at a concentration 416 of 5x10 5 cells/mL in fresh media and incubating for three days with 100 ng/mL 417 phorbol myristrate acetate (PMA; Invivogen). Adherent monolayers were washed 418 once with DPBS, dissociated with 0.05% trypsin/EDTA, resuspended in fresh media, 419 counted, and seeded for experimentation. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/434027 doi: bioRxiv preprint 4°C to remove insoluble material. Protein concentrations were quantified by using a 445 BCA protein assay kit (Thermo Scientific). Equal amounts of protein were separated 446 on 4-12% Bis-Tris Bolt SDS-PAGE gels (Thermo Scientific) and transferred to PVDF 447 membranes by using a Pierce Fast Semi-Dry Blotter. Immunoblotting was performed 448 by 30 minutes of blocking with either 5% milk (American Bio) or SuperBlock (Thermo 449 Scientific) followed by primary antibody and then secondary antibody (2 hours and 1 450 hour at room temperature, respectively), diluted in the same blocking solution. Blots 451', 'cite_spans': [], 'ref_spans': [{'start': 291, 'end': 299, 'text': '(Fig. 8)', 'ref_id': None}], 'section': 'For instance, Paijo et al. reported that the detection of cGAMP produced in response 325'}, {'text': 'were developed by using SuperSignal Pico or Femto chemiluminescence substrate 452 kits (Thermo Scientific) and imaged on a GE ImageQuant LAS 4000. Precision Plus 453 protein standards (Bio-Rad) were used to estimate protein molecular weights. 454', 'cite_spans': [], 'ref_spans': [], 'section': 'For instance, Paijo et al. reported that the detection of cGAMP produced in response 325'}, {'text': 'The following primary antibodies were used for western blotting analysis: Rabbit 455 anti-HA (1:1,000, Abcam #ab9110), rabbit anti-pIRF3 (1:1,000, Abcam #76493), 456 rabbit anti-cGAS (1:500, CST #15102s), rabbit anti-TOM40 (1:5,000, Santa Cruz 457 #H-300), rabbit anti-Calreticulin (1:5,000, Abcam #ab2907), rabbit anti-Lamin B1 458', 'cite_spans': [], 'ref_spans': [], 'section': 'For instance, Paijo et al. reported that the detection of cGAMP produced in response 325'}, {'text': '(1:1,000, Abcam #ab16048), and mouse anti-β-actin (1:10,000, Sigma #A1978). 459', 'cite_spans': [], 'ref_spans': [], 'section': 'For instance, Paijo et al. reported that the detection of cGAMP produced in response 325'}, {'text': 'The following secondary antibodies were used for western blotting analysis: Goat To perform immunoprecipitation, lysates were pre-cleared with 2 μg/mL rabbit 472 sera and two incubations with 50 μL protein A-magnetic beads (Pierce). Samples 473 . CC-BY-NC-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/434027 doi: bioRxiv preprint were then rotated overnight with 2 µg HA antibody, and complexes were captured 474', 'cite_spans': [], 'ref_spans': [], 'section': 'For instance, Paijo et al. reported that the detection of cGAMP produced in response 325'}, {'text': 'with Protein A-magnetic beads. Washing was performed as follows: 2x with RIPA, 2x 475 with high salt RIPA (500 mM NaCl), 1x with IP-wash buffer (0.5 M LiCl, 1% NP-40, 476 1% deoxycholate, 100 mM Tris-HCl pH 8.0), and 2x with T 10 E 1 (10 mM Tris-HCl pH For RT-qPCR of mouse tissue-derived mRNAs, SYBR Green qPCR reactions 497 were run in triplicate with gene specific primers ( Table 1 ). The Ct values were 498 averaged, internally normalized against housekeeping gene HPRT, then normalized 499 to B6J control mice by using the ∆∆Ct method of comparison. Fold-expression was 500 estimated assuming one doubling per cycle (fold expression = 2 -∆∆Ct ). 501 . CC-BY-NC-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/434027 doi: bioRxiv preprint Site-directed mutagenesis of hcGAS was performed by using appropriate 512 primers (Table 1) and PfuTurbo (Agilent Technologies), as previously described 513', 'cite_spans': [], 'ref_spans': [{'start': 378, 'end': 385, 'text': 'Table 1', 'ref_id': 'TABREF1'}, {'start': 953, 'end': 962, 'text': '(Table 1)', 'ref_id': 'TABREF1'}], 'section': 'For instance, Paijo et al. reported that the detection of cGAMP produced in response 325'}, {'text': 'with XhoI and NotI. 515', 'cite_spans': [], 'ref_spans': [], 'section': '(70). Mutants were sequenced and subcloned back into the pMXs-IRES-puro vector 514'}, {'text': 'Gene knockout was performed in cell culture by using Cas9 to induce 516 non-homologous end-joining repair. Briefly, gRNAs-specific oligos (Table 1) Tris-HCl, pH 7.4, 10 mM KCl, 1.5 mM MgCl 2 ). Cells were lysed via nitrogen 541 cavitation in a cell disruptor (Parr Instrument Company). Lysates were clarified at 542 1,000 x g for 5 minutes, then 16,100 x g for 10 minutes, retaining the supernatants. 543', 'cite_spans': [], 'ref_spans': [{'start': 138, 'end': 147, 'text': '(Table 1)', 'ref_id': 'TABREF1'}], 'section': '(70). Mutants were sequenced and subcloned back into the pMXs-IRES-puro vector 514'}, {'text': 'Resulting supernatants were digested for 1 hour with benzonase (0.5 U/μL; Fisher 544 Scientific) at 37°C, 1 hour with proteinase K (0.5 U/μL; Invitrogen) at 55°C, 545', 'cite_spans': [], 'ref_spans': [], 'section': '(70). Mutants were sequenced and subcloned back into the pMXs-IRES-puro vector 514'}, {'text': 'heat-inactivated at 95°C for 10 min, and spun for 5 minutes at 16,100 x g, retaining 546 the final supernatant (S1). 547', 'cite_spans': [], 'ref_spans': [], 'section': '(70). Mutants were sequenced and subcloned back into the pMXs-IRES-puro vector 514'}, {'text': 'To detect cGAMP bioactivity, 2 µL of S1 sample, synthetic cGAMP (positive 548 controls), or DPBS (negative controls) were incubated with 10 6 THP-1 cells, 2 mM 549 ATP, 1.5 ng/µL SLO (a kind gift from Dr. N. Andrews, University of Maryland) and 550 media in 8 µL (total volume). After 1.5 hours at 30°C, reactions were lysed with an 551 equal volume of RIPA buffer and processed for phosphorylated IRF3 western blot, 552 as above. 553', 'cite_spans': [], 'ref_spans': [], 'section': '(70). Mutants were sequenced and subcloned back into the pMXs-IRES-puro vector 514'}, {'text': 'To detect cGAMP by liquid chromatography and mass spectrophotometry, 554 trypsinized cells were washed once with DPBS, pelleted at 1,000 x g and then frozen 555 at -20°C. To extract cGAMP, 5x10 6 cells/mL were resuspended three times in 556 extraction buffer (40% acetonitrile, 40% methanol, 20% ddH 2 O) for 20 minutes, 557 spinning after each extraction at 16,100 x g and keeping the supernatant. 558 and incubated for 10 minutes at 4°C with rotation. A succession of 4°C spins was 573 performed, retaining the supernatant for each step: 3x 1,000 x g for 3 minutes, 1x 574 16,100 x g for 10 minutes, 1x 100,000 x g for 1 hour on a 0.34 M sucrose cushion 575 (SW41 Ti rotor, Beckman). The final supernatant was then processed for western 576 blotting, above, and DNA purification, below. 577 DNA purification and phi29 amplification. DNA was isolated from total cytosol 578 by treating samples with RNase A and RNase T1 (Ambion) for 1 hour at 37°C, 579 digesting with Proteinase K for 1 hour at 55°C, and heat inactivating at 95°C for 15 580 minutes. DNAs were then purified with the QiaQuick purification kit (Qiagen). To 581 isolate DNA from immunoprecipitates, crosslinks were reversed by adding 5 M NaCl 582 The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/434027 doi: bioRxiv preprint ', 'cite_spans': [], 'ref_spans': [], 'section': '(70). Mutants were sequenced and subcloned back into the pMXs-IRES-puro vector 514'}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'Approaching the asymptote? Evolution and 623 revolution in immunology', 'authors': [{'first': 'C', 'middle': ['A'], 'last': 'Janeway', 'suffix': ''}], 'year': 1989, 'venue': 'Cold Spring Harb Symp Quant Biol 54 Pt', 'volume': '1', 'issn': '', 'pages': '1--13', 'other_ids': {}}, 'BIBREF1': {'ref_id': 'b1', 'title': 'STING is an endoplasmic reticulum adaptor 625 that facilitates innate immune signalling', 'authors': [{'first': 'H', 'middle': [], 'last': 'Ishikawa', 'suffix': ''}, {'first': 'G', 'middle': ['N'], 'last': 'Barber', 'suffix': ''}], 'year': 2008, 'venue': 'Nature', 'volume': '455', 'issn': '', 'pages': '674--678', 'other_ids': {}}, 'BIBREF2': {'ref_id': 'b2', 'title': 'STING regulates intracellular', 'authors': [{'first': 'H', 'middle': [], 'last': 'Ishikawa', 'suffix': ''}, {'first': 'Z', 'middle': [], 'last': 'Ma', 'suffix': ''}, {'first': 'G', 'middle': ['N'], 'last': 'Barber', 'suffix': ''}], 'year': 2009, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF3': {'ref_id': 'b3', 'title': 'DNA-mediated, type I interferon-dependent innate immunity', 'authors': [], 'year': None, 'venue': 'Nature', 'volume': '628', 'issn': '', 'pages': '788--792', 'other_ids': {}}, 'BIBREF4': {'ref_id': 'b4', 'title': 'Trex1 prevents 630 cell-intrinsic initiation of autoimmunity', 'authors': [{'first': 'D', 'middle': ['B'], 'last': 'Stetson', 'suffix': ''}, {'first': 'J', 'middle': ['S'], 'last': 'Ko', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Heidmann', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Medzhitov', 'suffix': ''}], 'year': 2008, 'venue': 'Cell', 'volume': '134', 'issn': '', 'pages': '587--598', 'other_ids': {}}, 'BIBREF6': {'ref_id': 'b6', 'title': 'Trex1 regulates', 'authors': [{'first': 'B', 'middle': [], 'last': 'Levine', 'suffix': ''}, {'first': 'E', 'middle': ['K'], 'last': 'Wakeland', 'suffix': ''}, {'first': 'M', 'middle': ['A'], 'last': 'Lee-Kirsch', 'suffix': ''}, {'first': 'Yan', 'middle': ['N'], 'last': '', 'suffix': ''}], 'year': 2013, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF7': {'ref_id': 'b7', 'title': 'lysosomal biogenesis and interferon-independent activation of antiviral 634 genes', 'authors': [], 'year': None, 'venue': 'Nature Immunology', 'volume': '14', 'issn': '', 'pages': '61--71', 'other_ids': {}}, 'BIBREF9': {'ref_id': 'b9', 'title': 'The cytosolic exonuclease TREX1 inhibits the innate immune 637 response to human immunodeficiency virus type 1', 'authors': [{'first': 'J', 'middle': [], 'last': '', 'suffix': ''}], 'year': 2010, 'venue': 'Nature Immunology', 'volume': '638', 'issn': '', 'pages': '1005--1013', 'other_ids': {}}, 'BIBREF13': {'ref_id': 'b13', 'title': 'Type I IFN Production in Infected Cells by Cleaving Human STING', 'authors': [], 'year': None, 'venue': 'PLoS', 'volume': '643', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF16': {'ref_id': 'b16', 'title': 'Hepatitis C virus NS4B blocks the interaction of STING and TBK1 to evade 653 host innate immunity', 'authors': [], 'year': None, 'venue': 'Journal of Hepatology', 'volume': '59', 'issn': '', 'pages': '52--58', 'other_ids': {}}, 'BIBREF20': {'ref_id': 'b20', 'title': 'Hepatitis C virus NS4B protein targets STING and abrogates RIG-I-658 mediated type I interferon-dependent innate immunity', 'authors': [{'first': 'M', 'middle': [], 'last': '', 'suffix': ''}], 'year': 2013, 'venue': 'Hepatology', 'volume': '57', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF22': {'ref_id': 'b22', 'title': 'Regulation of IRF-3-Dependent 661', 'authors': [{'first': 'Tseng C-Tk', 'middle': [], 'last': 'Cj', 'suffix': ''}, {'first': 'S', 'middle': ['C'], 'last': 'Baker', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Li', 'suffix': ''}], 'year': 2007, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF23': {'ref_id': 'b23', 'title': 'Innate Immunity by the Papain-like Protease Domain of the SARS 662', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF25': {'ref_id': 'b25', 'title': 'SARS 664 coronavirus papain-like protease inhibits the type I interferon signaling 665 pathway through interaction with the STING-TRAF3-TBK1 complex', 'authors': [{'first': 'X', 'middle': [], 'last': 'Chen', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Yang', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Zheng', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Yang', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Xing', 'suffix': ''}, {'first': 'Z', 'middle': [], 'last': 'Chen', 'suffix': ''}], 'year': 2014, 'venue': 'Protein 666 & Cell', 'volume': '5', 'issn': '', 'pages': '369--381', 'other_ids': {}}, 'BIBREF28': {'ref_id': 'b28', 'title': 'Deubiquitinating and Interferon Antagonism Activities of Coronavirus 670', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF32': {'ref_id': 'b32', 'title': 'A diverse range of gene products are effectors of the type I 710 interferon antiviral response', 'authors': [{'first': 'C', 'middle': ['M'], 'last': 'Rice', 'suffix': ''}], 'year': 2011, 'venue': 'Nature', 'volume': '472', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF35': {'ref_id': 'b35', 'title': 'Activated by Double-Stranded DNA-Induced Oligomerization', 'authors': [], 'year': None, 'venue': 'Immunity', 'volume': '714', 'issn': '', 'pages': '1019--1031', 'other_ids': {}}, 'BIBREF37': {'ref_id': 'b37', 'title': 'The Cytosolic DNA Sensor cGAS Forms an 717', 'authors': [{'first': 'Chen', 'middle': [], 'last': 'Zhijian', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': '', 'suffix': ''}], 'year': 2014, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF38': {'ref_id': 'b38', 'title': 'Oligomeric Complex with DNA and Undergoes Switch-like Conformational 718', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF41': {'ref_id': 'b41', 'title': 'Enzymes in Innate Immunity', 'authors': [{'first': '', 'middle': [], 'last': 'Second-Messenger', 'suffix': ''}], 'year': None, 'venue': 'Cell Reports', 'volume': '3', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF43': {'ref_id': 'b43', 'title': 'Structural mechanism of cytosolic DNA 724 sensing by cGAS', 'authors': [{'first': 'V', 'middle': [], 'last': 'Hornung', 'suffix': ''}, {'first': 'K-P', 'middle': [], 'last': 'Hopfner', 'suffix': ''}], 'year': 2013, 'venue': 'Nature', 'volume': '498', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF45': {'ref_id': 'b45', 'title': \"cGAS produces a 2'-5'-linked cyclic 727 dinucleotide second messenger that activates STING\", 'authors': [{'first': 'K-P', 'middle': [], 'last': 'Ludwig', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Hornung', 'suffix': ''}, {'first': 'V', 'middle': [], 'last': '', 'suffix': ''}], 'year': 2013, 'venue': 'Nature', 'volume': '498', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF46': {'ref_id': 'b46', 'title': 'Cyclic 729', 'authors': [{'first': 'X', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Shi', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Wu', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Sun', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Chen', 'suffix': ''}, {'first': 'Z', 'middle': ['J'], 'last': 'Chen', 'suffix': ''}], 'year': 2013, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF47': {'ref_id': 'b47', 'title': 'Containing Mixed Phosphodiester Linkages Is An Endogenous 730 High Affinity Ligand for STING', 'authors': [{'first': '', 'middle': [], 'last': 'Gmp-Amp', 'suffix': ''}], 'year': None, 'venue': 'Molecular cell', 'volume': '731', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['51:10.1016/j.molcel.2013.1005.1022']}}, 'BIBREF49': {'ref_id': 'b49', 'title': 'The innate immune DNA 738 sensor cGAS produces a non-canonical cyclic-di-nucleotide that activates 739 human STING', 'authors': [{'first': 'M', 'middle': [], 'last': 'Hayakawa', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Hammond', 'suffix': ''}, {'first': 'M', 'middle': ['C'], 'last': 'Vance', 'suffix': ''}, {'first': 'R', 'middle': ['E'], 'last': '', 'suffix': ''}], 'year': 2013, 'venue': 'Cell reports', 'volume': '3', 'issn': '', 'pages': '1355--1361', 'other_ids': {}}, 'BIBREF52': {'ref_id': 'b52', 'title': 'Apoptotic Caspases 743', 'authors': [{'first': 'E', 'middle': [], 'last': 'Matthew', 'suffix': ''}, {'first': 'Huang', 'middle': [], 'last': 'David', 'suffix': ''}, {'first': 'C', 'middle': ['S'], 'last': '', 'suffix': ''}, {'first': 'Kile', 'middle': [], 'last': 'Benjamin', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': '', 'suffix': ''}], 'year': 2014, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF53': {'ref_id': 'b53', 'title': 'Suppress mtDNA-Induced STING-Mediated Type I IFN Production', 'authors': [], 'year': None, 'venue': 'Cell', 'volume': '744', 'issn': '', 'pages': '1549--1562', 'other_ids': {}}, 'BIBREF56': {'ref_id': 'b56', 'title': 'Mitochondrial DNA stress 748 primes the antiviral innate immune response', 'authors': [{'first': '', 'middle': [], 'last': 'Jr', 'suffix': ''}, {'first': 'R', 'middle': ['E'], 'last': 'Means', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Iwasaki', 'suffix': ''}, {'first': 'G', 'middle': ['S'], 'last': 'Shadel', 'suffix': ''}], 'year': 2015, 'venue': 'Nature', 'volume': '520', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF59': {'ref_id': 'b59', 'title': 'Apoptotic caspases prevent the induction of 752 type I interferons by mitochondrial DNA', 'authors': [{'first': 'A', 'middle': [], 'last': 'Iwasaki', 'suffix': ''}, {'first': 'R', 'middle': ['A'], 'last': 'Flavell', 'suffix': ''}], 'year': 2014, 'venue': 'Cell', 'volume': '159', 'issn': '', 'pages': '1563--1577', 'other_ids': {}}, 'BIBREF61': {'ref_id': 'b61', 'title': 'Dengue virus activates cGAS through 755 the release of mitochondrial DNA', 'authors': [{'first': 'T', 'middle': [], 'last': 'Chawla', 'suffix': ''}, {'first': 'C', 'middle': ['K'], 'last': 'Tang', 'suffix': ''}, {'first': 'E', 'middle': ['E'], 'last': 'Ooi', 'suffix': ''}], 'year': 2017, 'venue': 'Scientific Reports', 'volume': '7', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF62': {'ref_id': 'b62', 'title': 'DNA tumor virus 757 oncogenes antagonize the cGAS-STING DNA-sensing pathway', 'authors': [{'first': 'L', 'middle': [], 'last': 'Lau', 'suffix': ''}, {'first': 'E', 'middle': ['E'], 'last': 'Gray', 'suffix': ''}, {'first': 'R', 'middle': ['L'], 'last': 'Brunette', 'suffix': ''}, {'first': 'D', 'middle': ['B'], 'last': 'Stetson', 'suffix': ''}], 'year': 2015, 'venue': 'Science', 'volume': '758', 'issn': '', 'pages': '568--571', 'other_ids': {}}, 'BIBREF67': {'ref_id': 'b67', 'title': 'Dengue virus NS2B 769 protein targets cGAS for degradation and prevents mitochondrial DNA 770 sensing during infection', 'authors': [{'first': 'Garcia-Sastre', 'middle': [], 'last': 'Lc', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Fernandez-Sesma', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': '', 'suffix': ''}], 'year': 2017, 'venue': 'Nat Microbiol', 'volume': '2', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF69': {'ref_id': 'b69', 'title': 'Generation of ρ(0) cells utilizing a mitochondrially targeted 773 restriction endonuclease and comparative analyses', 'authors': [{'first': 'P', 'middle': [], 'last': 'Seibel', 'suffix': ''}], 'year': 2008, 'venue': 'Nucleic Acids Research', 'volume': '774', 'issn': '', 'pages': '44--44', 'other_ids': {}}, 'BIBREF71': {'ref_id': 'b71', 'title': 'Characterisation of cytoplasmic DNA 777 complementary to non-retroviral RNA viruses in human cells', 'authors': [{'first': '', 'middle': [], 'last': 'Jd', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Takeuchi', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Hoshino', 'suffix': ''}], 'year': 2014, 'venue': 'Scientific 778 Reports', 'volume': '4', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF72': {'ref_id': 'b72', 'title': 'Vesicular Stomatitis Virus Matrix Protein Inhibits 780', 'authors': [{'first': 'B', 'middle': ['L'], 'last': 'Black', 'suffix': ''}, {'first': 'D', 'middle': ['S'], 'last': 'Lyles', 'suffix': ''}], 'year': 1992, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF73': {'ref_id': 'b73', 'title': 'Host Cell-Directed Transcription of Target Genes In Vivo', 'authors': [], 'year': None, 'venue': 'J Virol', 'volume': '781', 'issn': '', 'pages': '4058--4064', 'other_ids': {}}, 'BIBREF74': {'ref_id': 'b74', 'title': 'Effect of vesicular stomatitis virus 783 matrix protein on host-directed translation in vivo', 'authors': [{'first': 'B', 'middle': ['L'], 'last': 'Black', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Brewer', 'suffix': ''}, {'first': 'D', 'middle': ['S'], 'last': 'Lyles', 'suffix': ''}], 'year': 1994, 'venue': 'Journal of Virology', 'volume': '784', 'issn': '', 'pages': '555--560', 'other_ids': {}}, 'BIBREF76': {'ref_id': 'b76', 'title': 'Induces Type I Interferons through the RIG-I Pathway', 'authors': [{'first': 'Dna', 'middle': [], 'last': 'Cytosolic', 'suffix': ''}], 'year': None, 'venue': '787 Cell', 'volume': '138', 'issn': '', 'pages': '576--591', 'other_ids': {}}}, 'ref_entries': {'FIGREF0': {'text': '180Moreover, cGAMP was not observed in untransfected cells expressing WT cGAS or 181 in DNA-transfected cells expressing a catalytically inactive form of hcGAS (Figure 182', 'latex': None, 'type': 'figure'}, 'FIGREF1': {'text': '4D). Surprisingly, cGAMP remained below detectable levels after 5 hours of 183 VSV-GFP infection at a MOI of 10 (Figure 4D), suggesting that detectable levels of 184 cGAMP were not produced in response to RNA virus infection.185 While the LC/MS technique provides exquisite specificity for identifying cGAMP 186 in complex cytosolic extracts, cGAMP biological assays may be more sensitive. 187 Indeed, our UPLC-MS configuration reliably detected nanogram amounts of cGAMP 188 spiked into cytosolic extract (Fig. 4E), which equates to >1 million molecules of 189 cGAMP per cell. We therefore established a bioassay for cGAMP-mediated IRF-3 190 activation in streptolysin O-(SLO)-permeabilized cells (Figure 4F). This bioassay 191 was shown to be dependent on STING activation (Fig. 4G) and had a limit of 192 detection (L.O.D.) of ~5 x 10 -4 µg/µl (~0.74 µM) cGAMP (Figure 4H), in line with other 193', 'latex': None, 'type': 'figure'}, 'FIGREF2': {'text': ', mtDNA was specifically 209 enriched by HA-immunoprecipitation from cells expressing WT hcGAS-HA3x, but 210 not from cells expressing the K384E DNA binding mutant. It should be noted that this 211 experiment is representative of many iterations performed at different scales. Given 212 prior links between virus infection, mitochondrial stress, and cGAS-mtDNA 213 interaction (34, 40), we next asked whether VSV altered the amount of 214 cGAS-associated mtDNA. Surprisingly, VSV-GFP infection had no impact on the 215 amount of cGAS-associated mtDNA (Fig. 5B), which led us to isolate cytosolic DNA 216 (Figure 5C) to quantitate mtDNA content with and without infection. Unexpectedly, 217 VSV-GFP infection had no impact on either the total amount of cellular mtDNA (Fig. 218 5D) or cytosolic mtDNA (Fig. 5E). 219', 'latex': None, 'type': 'figure'}, 'FIGREF3': {'text': '261 devoid of cGAS expression or reconstituted with WT or mutant hcGAS-HA3x (Figure 262', 'latex': None, 'type': 'figure'}, 'FIGREF4': {'text': '6A). These experiments suggested that WT hcGAS-HA3x significantly enhances 263 baseline ISG induction compared to the parental cGAS KO line and hcGAS mutants. 264 Further experiments showed that WT hcGAS-HA3x also stimulated greater 265 ISRE-driven luciferase production during infection of THP-1 cells with VSV-GFP and 266 SINV-GFP (Figures 6B, C). It should be noted that VSV-GFP ISG levels were not 267 appreciably different from the control, likely due to the transcriptional repression 268 capability of the M protein (43, 44). 269 To confirm our ISRE-luciferase findings, we used RT-qPCR to quantify ISG 270 transcripts known to be induced by the cGAS-STING DNA sensing pathway (Figures 271 6D-6G). These results show that cGAS KO significantly reduced basal expression of 272 Mx1 and CXCL10 in uninfected cells, but not of IFIT1, which was not expressed 273 basally. Importantly, cells reconstituted with WT hcGAS-HA3x expressed 274 significantly higher levels of IFIT1 and CXCL10 mRNA, while cells expressing 275 inactive hcGAS-3xHA mutants did not. As these experiments were conducted in 276 cells that slightly overexpressed hcGAS (Figure S2), ISG upregulation likely reflects 277 reinforced, native patterns of expression. 278', 'latex': None, 'type': 'figure'}, 'FIGREF5': {'text': '355 Nevertheless, the low abundance of VSV cDNAs is inconsistent with a model 356 whereby RNA virus cDNAs play a significant role in stimulating population-wide 357 innate immune responses. These findings, however, do highlight the constant 358 synthesis and turnover of cDNAs within the mammalian cytosol. Consistent with this, 359 deficiencies in the TREX1 nuclease lead to cytosolic accumulation of DNA, including 360 retroelement cDNAs, causing chronic cGAS stimulation and autoimmunity in the 361 form of Aicardi-Goutières syndrome (4, 56, 57). 362 Given that cGAS may be continuously stimulated by endogenous DNA ligands, 363 and that candidate DNA ligands were unchanged during VSV infection, we 364 examined whether cGAS contributes to a pre-existing baseline of innate immune 365 activation. Indeed, low level cGAS-dependent ISG expression was observed even in 366 the absence of viral infection and was significantly decreased in cGAS KO cells, 367 consistent with prior examples of the cGAS-STING pathway altering ISG baseline 368expression(2, 50, 58, 59). These results support the hypothesis that cGAS 369 contributes to RNA virus restriction by establishing smoldering, baseline-levels of 370 constitutive innate immune activation. This is an important distinction from other 371', 'latex': None, 'type': 'figure'}, 'FIGREF6': {'text': 'research. All mice were maintained, bred, and handled in our facility in 388 compliance with federal and institutional policies under protocols approved by the 389 Yale Animal Care and Use Committee. C57BL/6J, B6(C)-Cgas tm1d(EUCOMM)Hmgu /J 390 (cGAS -/-) mice (50) and B6(Cg)-Ifnar1tm1.2Ees/J (Ifnar1 -/-) mice (60) were 391 purchased from Jackson Laboratory. Irf3 -/-Irf7 -/mice (61) were a generous gift by 392 Dr. T. Taniguchi (University of Tokyo) and Mavs -/mice (62) were a generous gift by 393 Dr. Z. Chen (University of Texas, Southwestern). 394 Primary MEFs were isolated from day 14.5 embryos (E14.5) as previously 395 described (63). Vaginal tissue was harvested from six-to twelve-week old female 396 mice synchronized in diestrus via subcutaneous injection with 2 mg Depo-Provera in 397 the neck scruff; mice were sacrificed and vaginal tissue harvested ten days after 398 Depo-Provera treatment. 399 Cell cultures. All cells were maintained at low passage by using a seed-lot 400 system and routinely tested for mycoplasma contamination. HEK 293E cells were 401 obtained from Dr. W. Mothes (Yale); SW13 cells were obtained from Dr. C. Rice 402 (Rockefeller University); BHK-21 cells were obtained from Dr. D. Brackney (State of 403 Connecticut Agriculture Research Station); LMTK and LMTK \\uf072 0 cells were obtained 404 from Dr. G. Shadel (Yale). HEK293E, SW13, BHK-21 and MEF cells were 405 maintained at 37°C and 5% CO 2 in complete growth medium (DMEM containing 2 406 mM L-glutamine [Invitrogen], 10% heat-inactivated fetal calf serum [FCS, Omega 407 Scientific], and 0.1 mM non-essential amino acids [NEAA; Invitrogen]). LMTK and 408LMTK \\uf072 0 cells were maintained in DMEM as above supplemented with 100 μg/mL 409 sodium pyruvate (Invitrogen) and 50 μg/mL uridine (Sigma) 410 THP-1-Lucia ISG cells (Invivogen) were maintained at 37°C and 5% CO 2 in 411 RPMI 1640 containing 2 mM L-glutamine, 10% FCS, 0.1 mM NEAA, 10 U/mL 412 penicillin/streptomycin, 100 μg/mL normocin, and 100 μg/mL zeocin. 413', 'latex': None, 'type': 'figure'}, 'FIGREF7': {'text': 'Viruses expressing green fluorescent protein (GFP) were used to 421 facilitate monitoring of virus infections. rVSV-p1-eGFP (VSV-GFP) (64) and 422 rVSV-ΔM51-p5-eGFP (VSVΔM51A-GFP) (65) were kind gifts from Drs. J. Rose and 423 A. van den Pol (Yale), respectively. SINV G100-eGFP (SINV-GFP) (66) was a kind 424 gift from Dr. M. Heise (University of North Carolina, Chapel Hill). VSV and SINV 425 stocks were generated via low multiplicity of infection (MOI = 0.01) passage in 426 BHK-21 cells. Herpes simplex virus 1 KOS-eGFP (HSV-GFP) (67), kindly provided 427 by Dr. P. Desai (Johns Hopkins Medical School), was propagated by low MOI 428 passage (MOI 0.01) in Vero cells, and harvested at 60 hours post-infection. 429 HSV-GFP was prepped by three cycles of freezing (-80°C) and thawing (37°C), 430 clarification (1,500 x g for 15 minutes at 4°C), addition of 10% FCS and 7% dimethyl 431 sulfoxide, aliquoted, and stored at -80°C. 432 Plaque assay and fluorescent cell counting. Plaque assays were developed 433 by using semi-solid overlays (DMEM, 10% FCS, 1.6% LE agarose). When plaque 434 formation was evident, cells were fixed with 3% formaldehyde, agarose plugs were 435 removed, and cells stained with 0.1% crystal violet in 20% ethanol. Plaque forming 436 units per mL (pfu/mL) were calculated by counting the number of colonies formed 437 and multiplying this count by the dilution factor. 438 To prepare GFP-expressing cells for cytometry, cells were trypsinized, washed 439 with DPBS, and fixed in DPBS containing 1% PFA. Fluorescent cells were counted 440 on an Accuri C6 Flow Cytometer (Becton-Dickinson). 441 Protein analysis. For western blotting, cells were lysed in RIPA buffer (50 mM 442Tris pH 8.0, 150 mM NaCl, 1% Triton X-100, 0.5% sodium deoxycholate, 0.1% SDS) 443 containing protease inhibitor cocktail, followed by a 20-minute spin at 16,100 x g and 444', 'latex': None, 'type': 'figure'}, 'FIGREF8': {'text': '460anti-rabbit horseradish peroxidase (1:5,000, Jackson ImmunoResearch 461 #111-035-144), and goat anti-mouse horseradish peroxidase (1:5,000, Jackson 462 #115-035-146). 463To immunoprecipitate cGAS-DNA complexes, hcGAS-HA3x-expressing cells 464were fixed in DPBS containing 0.5% paraformaldehyde (5 minutes, room 465 temperature), then quenched with 125 mM glycine. All subsequent steps were 466 performed at 4°C. After two washes with DPBS, cells were lysed for 30 minutes in 467 ice-cold RIPA, followed by a 20-minute spin at 16,100 x g. Clarified lysates were 468 sonicated with four cycles of 10 seconds on and 30 seconds off at 20% amplitude on 469 a Sonifier 450 (Branson Ultrasonics). Samples were spun for 20 minutes at 16,100 x 470 g and supernatants were retained. 471', 'latex': None, 'type': 'figure'}, 'FIGREF9': {'text': \"EDTA). Bound complexes were eluted with 0.1 M glycine-HCl, pH 2.5, 478 samples were neutralized with 1 M Tris-HCl pH 8.0 (0.1 M final), and eluted 479 protein-nucleic acid complexes were then processed for western blotting or deep 480 sequencing (see Nucleic acid purification, below). 481 PCR, qPCR, and RT-PCR. Standard PCRs were performed with Phusion DNA 482 polymerase or Taq DNA polymerase (NEB). Unless otherwise noted, cycling was 483 performed for 35 cycles with primers listed inTable 1.484 For RT-PCR, RNAs were extracted from cells by using TRIzol Reagent (Life 485 Technologies) or the RNeasy extraction kit (Qiagen). Viral RNA was extracted from 486 cell culture media with the QiAmp Viral RNA Mini Kit (Qiagen). cDNA synthesis was 487 performed by using random hexamer or gene-specific primers with the Transcriptor 488 First Strand cDNA Synthesis Kit (Roche). 489 For qPCR of cell culture-derived cDNAs, primers were designed by using the 490 ProbeFinder software (Roche) for compatibility with Roche Universal Probe Library 491 (UPL) hydrolysis probes. Assays were performed in a LightCycler 96 or LightCycler 492 480 (Roche), as per manufacturer's instructions, with primers and UPL probes listed 493 inTable 1. All reactions were performed in duplicate and quantified by comparison to 494 standard curves created with cloned amplicons diluted (10 2 -10 7 copies) in ddH 2 O 495 supplemented with 50 ng/µL carrier DNA and run in parallel. 496\", 'latex': None, 'type': 'figure'}, 'FIGREF10': {'text': 'YO-2143 and cloning into pMXs-mcGAS-HA3x-IRES-puro (gift of 504 Dr. G. Shadel, Yale University) via common XhoI and NotI sites. To facilitate 505 reconstitution of puromycin-resistant cGAS knockout (KO) cells with hcGAS, the Pac 506 gene in pMXs-hcGAS-HA3x-IRES-puro was replaced with the Bsd gene, amplified 507 from pMICU-APEX2 (68) (Addgene plasmid # 79057). In addition, the 518-bp NotI-508 BlpI fragment of hcGAS was recoded (BlueHeron) to avoid editing of the 509 reconstituted hcGAS. For transient expression of hcGAS in primary MEFs, 510 hcGAS-3xHA was cloned into pLenti-puro (69) (Addgene plasmid # 39481).', 'latex': None, 'type': 'figure'}, 'FIGREF12': {'text': 'were 517 chosen from published datasets (71) or designed with gRNA Designer (72) and 518 cloned into pLentiCRISPR (73) (Addgene plasmid # 51760). 519 Lentiviruses and retroviruses. Lentiviruses and retroviruses were packaged in 520 HEK 293E cells by co-transfection with appropriate HIV-or MLV-Gag/Pol and VSV G 521 packaging constructs. Forty-eight hours post-transfection, packaged vector stocks 522 were clarified (16,100 x g), passed through a 0.45 µm filter, and supplemented with 8 523 µg/mL polybrene (Sigma) and 20 mM HEPES (Life Technologies). Target cells were 524 transduced by spinoculation, selected with 3 µg/mL puromycin, and screened for 525 expression or knockout via genomic PCR and sequencing and/or western blotting. 526 To develop clonal cultures, adherent cells were isolated by using sterile 8 mm Pyrex 527 cloning cylinders and expanded. Clonal phenotypes were screened via western 528 blotting or qPCR. Promega) to destroy virus infectivity. Lucia activity in 20 µL samples was 532 measured on a Centro LB 960 plate reader (Berthold) by integrating over 10 533 seconds. 534 cGAMP extraction and assays. Cells were infected for five hours with 535 VSV-GFP or SINV-GFP (MOI 3-10), transfected for five hours with salmon sperm 536 DNA (2 µg/mL final concentration) and Transit LT-1 (Mirus), or left untreated. 537 cGAMP was extracted based on established methods (21). Briefly, cells were 538 dissociated with trypsin, washed with DPBS, gently pelleted and resuspended at a 539 concentration of 1x10 7 cells/mL in ice-cold cGAMP homogenization buffer (10 mM 540', 'latex': None, 'type': 'figure'}, 'FIGREF13': {'text': 'Phenomenex) on an iFunnel 6550 Q-TOF / MS (Agilent). Samples were run in 562 negative mode with the following parameters: Buffer A = 0.1% formic acid; Buffer B = 563 acetonitrile, 0.1% formic acid; gradient cycles: 0 -4% B over 10 minutes, 4% B -564 100% B over 5 minutes, 5 minutes wash with 100% B; UV detection at 260 nm, m/z 565 scans from 150-1,000. cGAMP was observed between 4-7.5 minutes in extracted 566 ion chromatographs at an observed mass of 673.085 m/z; this was confirmed to be 567 cGAMP by MS/MS ion fragmentation patterns. 568Preparation of cytosolic nucleic acid extracts. Cells were trypsinized and 569 resuspended in an equal volume of fresh media, then spun at 1,000 x g for 5 minutes 570 at room temperature. After washing once with DPBS, cells were resuspended in 571 cytosolic extraction buffer (50 mM HEPES pH 7.4, 150 mM NaCl, 25 µg/mL digitonin) 572', 'latex': None, 'type': 'figure'}, 'FIGREF14': {'text': \"final) and shaking overnight at 65°C, then digesting RNA and protein, as 583 above. For isothermal DNA amplification, 1-40 ng of DNA was annealed to 584 exo-resistant random hexamer primers (Molecular Cloning Laboratories) and 585 amplified overnight at 30°C with phi29 DNA polymerase (NEB), followed by a 65°C 586 inactivation step. DNA was extracted with the QiaQuick purification kit. preparation, sequencing, and analysis. Phi29-amplified 588 samples were sonicated, as above, to achieve DNA fragments of 200-to500-bp.  589    DNAs were end-repaired with T4 DNA polymerase (NEB), T4 polynucleotide kinase 590 (NEB), and Klenow DNA polymerase (NEB) at 20°C for 30 minutes, then purified via 591 QiaQuick. A-tailing was performed with Klenow fragment (3'-5' exo[-], NEB) before 592 ligation of TruSeq Adapters (Illumina) and amplification with Phusion DNA 593 Polymerase (NEB) and Illumina TruSeq primer cocktails. Size-selected libraries 594(350-to 450-bp) were excised from LMP agarose, purified with a Gel Extraction Kit 595 (Qiagen), re-amplified by PCR for 18 cycles, then purified. Triplicate libraries were 596 sequenced on a HiSeq4000 (Illumina) by the Yale Center for Genome Analysis with 597 a 150-bp paired-end protocol, 100 million reads/sample. 598Software and Statistics. Unless otherwise noted, statistical significance was 599 estimated by using the Student's t-test with Holm-Sidak correction for multiple 600 comparisons. Standard p-value indicators were used throughout the manuscript: * 601 indicates p<0.05; ** indicates p<0.01; *** indicates p<0.001; and **** indicates 602 p<0.0001. Data were graphed by using Graphpad Prism software (version 7.0a). 603Pixel densities were analyzed in ImageJ2 (74) and images were prepared for 604 presentation with Photoshop and Illustrator CS4 (Adobe). Next-generation 605 sequencing results were mapped to the mm10 mouse genome by using the 606 Burrows-Wheeler aligner (BWA) (75) and TopHat(76)to look for raw and gapped 607 alignment, respectively. Alignments were assessed for content of genomic DNA and 608 mtDNA with the integrated genome browser software (77). Ding Q, Gaska JM, Douam F, Wei L, Kim D, Balev M, Heller B, Ploss A. 648 2018. Species-specific disruption of STING-dependent antiviral cellular 649 defenses by the Zika virus NS2B3 protease. Proc Natl Acad Sci U S Holm CK, Rahbek SH, Gad HH, Bak RO, Jakobsen MR, Jiang Z, Hansen 684 AL, Jensen SK, Sun C, Thomsen MK, Laustsen A, Nielsen CG, 685 Severinsen K, Xiong Y, Burdette DL, Hornung V, Lebbink RJ, Duch M, 686 Fitzgerald KA, Bahrami S, Mikkelsen JG, Hartmann R, Paludan SR. 687 2016. Influenza A virus targets a cGAS-independent STING pathway that 688 controls enveloped RNA viruses. Nature Communications 7:10680. 689 19. Liu S, Cai X, Wu J, Cong Q, Chen X, Li T, Du F, Ren J, Wu YT, Grishin 690 NV, Chen ZJ. 2015. Phosphorylation of innate immune adaptor proteins 691 MAVS, STING, and TRIF induces IRF3 activation. Science 347:aaa2630. 692 20. Franz KM, Neidermyer WJ, Tan Y-J, Whelan SPJ, Kagan JC. 2018. 693 STING-dependent translation inhibition restricts RNA virus replication. 694 Proceedings of the National Academy of Sciences 115:E2058. 695 21. Wu J, Sun L, Chen X, Du F, Shi H, Chen C, Chen ZJ. 2013. Cyclic 696 GMP-AMP is an endogenous second messenger in innate immune signaling 697 by cytosolic DNA. Science 339:826-830. 698 22. Mankan AK, Schmidt T, Chauhan D, Goldeck M, Höning K, Gaidt M, 699 Kubarenko AV, Andreeva L, Hopfner K-P, Hornung V. 2014. Cytosolic 700 RNA:DNA hybrids activate the cGAS-STING axis. The EMBO Journal 701 33:2937-2946. 702 23. Schoggins JW, MacDuff DA, Imanaka N, Gainey MD, Shrestha BMcArthur K, Whitehead LW, Heddleston JM, Li L, Padman BS, Oorschot 824 V, Geoghegan ND, Chappaz S, Davidson S, San Chin H, Lane RM, 825 Dramicanin M, Saunders TL, Sugiana C, Lessene R, Osellame LD, Chew 826 T-L, Dewson G, Lazarou M, Ramm G, Lessene G, Ryan MT, Rogers KL, 827 van Delft MF, Kile BT. 2018. BAK/BAX macropores facilitate mitochondrial 828 herniation and mtDNA efflux during apoptosis. Science 359. 829 56. Yang Y-G, Lindahl T, Barnes DE. 2007. Trex1 exonuclease degrades 830 ssDNA to prevent chronic checkpoint activation and autoimmune disease. 831 Cell 131:873-886. 832 57. Gray EE, Treuting PM, Woodward JJ, Stetson DB. 2015. Cutting Edge: 833 cGAS Is Required for Lethal Autoimmune Disease in the Trex1-Deficient 834 Mouse Model of Aicardi-Goutieres Syndrome. J Immunol 195:1939-1943. 835 58. Cheng J, Liao Y, Zhou L, Peng S, Chen H, Yuan Z. 2016. Amplified RLR 836 signaling activation through an interferon-stimulated gene-endoplasmic 837 reticulum stress-mitochondrial calcium uniporter protein loop. Scientific 838 Reports 6:20158. 839 59. Gopinath S, Kim MV, Rakib T, Wong PW, van Zandt M, Barry NA, Kaisho 840 T, Goodman AL, Iwasaki A. 2018. Topical application of aminoglycoside 841 antibiotics enhances host resistance to viral infections in a 842 microbiota-independent manner. Nature Microbiology 3:611-621. 843 60. Prigge JR, Hoyt TR, Dobrinen E, Capecchi MR, Schmidt EE, Meissner N. 844 2015. Type I IFNs Act upon Hematopoietic Progenitors To Protect and 845 Maintain Hematopoiesis during Pneumocystis Lung Infection in Mice. J 846 Immunol 195:5347-5357. 847 61. Honda K, Yanai H, Negishi H, Asagiri M, Sato M, Mizutani T, Shimada N, 848 Ohba Y, Takaoka A, Yoshida N, Taniguchi T. 2005. IRF-7 is the master 849 Sun Q, Sun L, Liu HH, Chen X, Seth RB, Forman J, Chen ZJ. 2006. The 852 specific and essential role of MAVS in antiviral innate immune responses. 853 Immunity 24:633-642. 854 63. Ye J. 2007. Reliance of host cholesterol metabolic pathways for the life cycle 855 of hepatitis C virus. PLoS Pathog 3:e108. 856 64. Ramsburg E, Publicover J, Buonocore L, Poholek A, Robek M, Palin A, 857 Rose JK. 2005. A vesicular stomatitis virus recombinant expressing 858 granulocyte-macrophage colony-stimulating factor induces enhanced T-cell 859 responses and is highly attenuated for replication in animals. J Virol 860 79:15043-15053. 861 65. van den Pol AN, Davis JN. 2013. Highly attenuated recombinant vesicular 862 stomatitis virus VSV-12'GFP displays immunogenic and oncolytic activity. J 863 Virol 87:1019-1034. 864 66. Suthar MS. 2007. Molecular pathogenesis of the sindbis-group virus strain 865 ar86. Ph.D. University of North Carolina at Chapel Hill, Chapel Hill, NC. 866 67. Desai P, Person S. 1998. Incorporation of the green fluorescent protein into 867 the herpes simplex virus type 1 capsid. J Virol 72:7563-7568. 868 68. Lam SS, Martell JD, Kamer KJ, Deerinck TJ, Ellisman MH, Mootha VK, 869 Ting AY. 2015. Directed evolution of APEX2 for electron microscopy and 870 proximity labeling. Nat Methods 12:51-54. 871 69. Guan B, Wang TL, Shih Ie M. 2011. ARID1A, a factor that promotes 872 formation of SWI/SNF-mediated chromatin remodeling, is a tumor suppressor 873 in gynecologic cancers. Cancer Res 71:6718-6727. 874 70. Zheng L, Baumann U, Reymond JL. 2004. An efficient one-step 875 site-directed and site-saturation mutagenesis protocol. Nucleic Acids Res 876 32:e115. Wang T, Wei JJ, Sabatini DM, Lander ES. 2014. Genetic screens in human 878 cells using the CRISPR-Cas9 system. Science 343:80-84. 879 72. Doench JG, Hartenian E, Graham DB, Tothova Z, Hegde M, Smith I, 880 Sullender M, Ebert BL, Xavier RJ, Root DE. 2014. Rational design of highly 881 active sgRNAs for CRISPR-Cas9-mediated gene inactivation. Nat Biotechnol 882 32:1262-1267. 883 73. Shalem O, Sanjana NE, Hartenian E, Shi X, Scott DA, Mikkelsen TS, 884 Heckl D, Ebert BL, Root DE, Doench JG, Zhang F. 2014. Genome-scale 885 CRISPR-Cas9 knockout screening in human cells. Science 343:84-87. 886 74. Rueden CT, Schindelin J, Hiner MC, DeZonia BE, Walter AE, Arena ET, 887 Eliceiri KW. 2017. ImageJ2: ImageJ for the next generation of scientific 888 image data. BMC Bioinformatics 18:529. 889 75. Li H, Durbin R. 2010. Fast and accurate long-read alignment with 890 Burrows-Wheeler transform. Bioinformatics 26:589-595. 891 76. Trapnell C, Pachter L, Salzberg SL. 2009. TopHat: discovering splice 892 junctions with RNA-Seq. Bioinformatics 25:1105-1111. 893 77. Freese NH, Norris DC, Loraine AE. 2016. Integrated genome browser: 894 visual analytics platform for genomics. Bioinformatics 32:2089-2095. WT cGAS and hcGAS restrict RNA virus infection in immortalized 899 MEFs. Single-step growth curve of (A) VSV-GFP and (B) SINV-GFP production from 900 MEFs infected at MOI 3, as assessed by plaque assay on SW13 cells. (C) VSV-GFP 901 stocks were titered by plaque assay on WT, KO, and KO+WT cells. 902 hcGAS-HA3x variants display differential restriction of RNA virus 904 infection. (A) The structure of hcGAS (grey) in complex with ATP (pink) and DNA 905 (yellow-purple), as rendered from PDB 6CTA; residues K384 (blue), K407 (green), 906 and E225/D227 (orange) are shown. (B) Plaque assay of VSV-GFP produced by 907 SV40 T-immortalized MEF KO cells reconstituted with hcGAS-HA3x variants and 908 infected (MOI 3) for 8 hours; western blot of hcGAS expression is shown below. 909 hcGAS variants display differential restriction of RNA virus infection. 911 Primary KO MEFs were transduced to express hcGAS variants with lentiviral vectors 912 and infected with (A) VSV-GFP, (B) VSVΔM51A-GFP, or (C) SINV-GFP; % infected 913 cells was determined by flow cytometry in relation to empty vector-transduced KO 914 MEFs (Empty). WT THP-1, THP-1 cGAS knockout (KO), and THP-1 cGAS KO cells 915 reconstituted with WT (+WT) mutant forms of hcGAS (K384E, K407E, or E/D) were 916 differentiated with PMA and infected with (D) VSV-GFP, (E) VSVΔM51A-GFP, or (F) 917 SINV-GFP; % infected cells was determined by flow cytometry in relation to THP-VSV-GFP infection does not induce detectable cGAMP production. 921 (A) UHPLC profiles showing a time-course of cGAMP production after transfecting 922 salmon sperm DNA into hcGAS-3xHA-expressing HEK 293E cells. The yellow box 923 represents the peak elution range of synthetic cGAMP observed in pilot 924 experiments. (B) Mass chromatogram of the eluted cGAMP peak after transfecting 925 DNA hcGAS-3xHA-expressing HEK 293E cells. Known ionization products of 926 DNA (ß-gluc) were determined by qPCR in uninfected and 954 VSV-GFP-infected MEF cells. (E) The mtDNA content was determined in cytosolic 955 uninfected and VSV-GFP-infected MEF cells. (F) Deep sequencing of 956 cytosolic extracts from uninfected and VSV-GFP-infected MEFs revealed the 957 presence of mtDNA. (G) Deep sequencing of cGAS-immunoprecipitates from 958 uninfected and VSV-GFP-infected reveal abundant mtDNA. (H) Time course of 959 VSV-GFP infection in LMTK and LMTK \\uf072 0 cells; cGAS and STING expression were 960 confirmed by western blot (inset). (I) Detection of VSV cDNA in virus-infected cells. N 961 gene-specific primers and probes were used to quantitate VSV cDNAs. GAPDH was 962 used as a control for cellular target DNA. (J) VSV cDNAs are sensitive to DNase I. 963 Cytosolic extracts were incubated with the indicated nucleases, cleaned up, and 964 subjected to qPCR. (K) Detection of GAPDH cDNA. Total cellular DNA was 965 subjected to qPCR with genomic DNA (gDNA)-and splice dependent 966 (cDNA)-specific primer and probe sets. 967 cGAS primes basal ISG expression in steady state cell cultures. 969 ISRE-driven luciferase production in (A) uninfected, (B) VSV-GFP infected, and (C) 970 and SINV-GFP infected THP-1 KO cells lines with or without WT or mutant 971 hcGAS-3xHA expression. RT-qPCR of (D) MX1, (E) IFIT1, and (F) CXCL10 972 expression in uninfected WT THP-1, THP-1 KO, or THP-1 KO cells expressing WT 973 or mutant hcGAS-HA3x. 974 cGAS primes basal ISG expression in vivo. Vaginal tissue was 976 collected from uninfected female mice of the indicated genotypes, synchronized in 977 diestrus. Expression levels of USP18, Mx1, and Rsad2 were quantitated by 978 RT-qPCR and normalized to B6J mice. 979 Model of cGAS-mediated restriction of RNA virus infection. 981 982Figure S1. hcGAS recoding, gRNA binding site, and residues targeted for 983 mutations. Site-directed mutagenesis was utilized to generate three mutants 984\", 'latex': None, 'type': 'figure'}, 'FIGREF15': {'text': 'Figure 2', 'latex': None, 'type': 'figure'}, 'FIGREF16': {'text': 'Figure 3', 'latex': None, 'type': 'figure'}, 'FIGREF17': {'text': 'Figure 4', 'latex': None, 'type': 'figure'}, 'FIGREF18': {'text': 'Figure 6', 'latex': None, 'type': 'figure'}, 'FIGREF19': {'text': 'Figure 7', 'latex': None, 'type': 'figure'}, 'FIGREF20': {'text': 'Figure 8', 'latex': None, 'type': 'figure'}, 'TABREF0': {'text': ', low basal 281 levels of USP18, Mx1, and Rsad2 expression were observed in B6J mice, but were 282 significantly decreased in IFNAR1 -/mice, demonstrating that basal ISG expression 283 depends on IFNAR signaling. Importantly, cGAS -/mice had significant decreases in 284 basal Mx1 and Rsad2 expression, similar in degree to reduced basal USP18 and 285Rsad2 expression observed in IRF3/7 -/mice. In contrast, MAVS had little effect on 286Altogether, these results suggest that cGAS primes cells to express smoldering 288 levels of ISG expression and that the DNA binding and catalytic activity are integral 289While the RLR-MAVS and cGAS-STING pathways are important, respectively, 293 for restricting RNA and DNA virus infections, there is considerable crosstalk and 294 redundancy between these two pathways. For instance, mammalian RNA 295 polymerase III can transcribe A-T-rich DNA in the cytosol, producing uncapped 296RNAs that trigger 46). In addition, STING can physically associate with 297 RIG-I and MAVS and may act as a cofactor in RNA sensing (47-49). More recently, 298 STING has been shown to inhibit RNA virus replication, independent of ISG 299', 'latex': None, 'type': 'table'}, 'TABREF1': {'text': 'Oligonucleotide primers and probes used in these studies.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/434027 doi: bioRxiv preprint Homo sapiens; m, Mus musculus; F, forward; R, reverse BY-NC-ND 4.0 International license author/funder. It is made available under aThe copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.', 'latex': None, 'type': 'table'}}, 'back_matter': [{'text': 'We thank Drs. H. Ramanathan, D. DiMaio, and P. Cresswell for', 'cite_spans': [], 'ref_spans': [], 'section': 'Acknowledgments.'}, {'text': '(brown). The K384E and K407E mutations disrupt the DNA-binding ability of the 985 cGAS, while the E225A/D227A mutation ablates cGAMP catalytic activity. The 5´ 986 518-bp of hcGAS were codon optimized (red) to improve expression and to generate 987 a sequence resistant to CRISPR/Cas9 targeting. The gRNA binding site (blue) was 988 modified at 8 residues. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/434027 doi: bioRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'annex'}]}\n",
      "{'paper_id': 'ac81102667b0d56edeb8ab0044765dc49a19f374', 'metadata': {'title': 'Determination of host cell proteins constituting the molecular microenvironment of 1 coronavirus replicase complexes by proximity-labeling 2', 'authors': [{'first': '', 'middle': [], 'last': 'V&apos;kovski Philip', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Gerber', 'middle': [], 'last': 'Markus', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Kelly', 'middle': [], 'last': 'Jenna', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Pfaender', 'middle': [], 'last': 'Stephanie', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Ebert', 'middle': [], 'last': 'Nadine', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Braga', 'middle': ['Lagache'], 'last': 'Sophie', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Bern', 'location': {'country': 'Switzerland'}}, 'email': ''}, {'first': 'Simillion', 'middle': [], 'last': 'Cedric', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Bern', 'location': {'country': 'Switzerland'}}, 'email': ''}, {'first': 'Portmann', 'middle': [], 'last': 'Jasmine', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Stalder', 'middle': [], 'last': 'Hanspeter', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Gaschen', 'middle': [], 'last': 'Véronique', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Bern', 'location': {'addrLine': 'Länggassstrasse 122', 'postCode': '3012', 'settlement': 'Vetsuisse Faculty, Bern', 'country': 'University, Switzerland'}}, 'email': ''}, {'first': 'Bruggmann', 'middle': [], 'last': 'Remy', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Bern', 'location': {'addrLine': '11 Switzerland'}}, 'email': ''}, {'first': 'Stoffel', 'middle': [], 'last': 'Michael', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Bern', 'location': {'addrLine': 'Länggassstrasse 122', 'postCode': '3012', 'settlement': 'Vetsuisse Faculty, Bern', 'country': 'University, Switzerland'}}, 'email': ''}, {'first': 'Heller', 'middle': [], 'last': 'Manfred', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Bern', 'location': {'settlement': 'Bern', 'country': 'Switzerland'}}, 'email': ''}, {'first': 'Ronald', 'middle': [], 'last': '', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Thiel', 'middle': [], 'last': 'Volker', 'suffix': '', 'affiliation': {}, 'email': 'volker.thiel@vetsuisse.unibe.ch'}, {'first': 'Volker', 'middle': [], 'last': 'Thiel', 'suffix': '', 'affiliation': {}, 'email': ''}]}, 'abstract': [{'text': 'Positive-sense RNA viruses hijack intracellular membranes that provide niches for viral RNA 23 synthesis and a platform for interactions with host proteins. However, little is known about host 24 factors at the interface between replicase complexes and the host cytoplasm. We engineered a 25 biotin ligase into a coronaviral replication/transcription complex (RTC) and identified >500 26 host proteins constituting the RTC microenvironment. siRNA-silencing of each host factor demonstrated importance of vesicular trafficking pathways, ubiquitin-dependent 28 and autophagy-related processes, and translation initiation factors. Notably, detection of 29 translation initiation factors at the RTC was instrumental to visualize and demonstrate active 30 translation proximal to replication complexes of several coronaviruses. 31', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'Collectively, we establish a spatial link between viral RNA synthesis and diverse host factors 32 of unprecedented breadth. Our data may serve as a paradigm for other positive-strand RNA 33 viruses and provide a starting point for a comprehensive analysis of critical virus-host 34 interactions that represent targets for therapeutic intervention.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'Positive-strand RNA viruses replicate at membranous structures that accommodate the viral 37 replication complex and facilitate RNA synthesis in the cytosol of infected host cells (1-5). 38', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'Rewiring host endomembranes is hypothesized to provide a privileged microenvironment 39 physically separated from the cytosol, thereby ensuring adequate concentrations of 40 macromolecules for viral RNA synthesis, preventing recognition of replication intermediates 41 such as double-stranded RNA (dsRNA) by cytosolic innate immune receptors (6, 7), and 42 providing a platform that facilitates molecular interactions with host cell proteins. 43', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': \"Ultrastructural studies have reported the origin, nature, and extent of membrane modifications 44 induced by coronaviruses (order Nidovirales, family Coronaviridae), which materialize as an 45 ER-derived network of interconnected double-membrane vesicles (DMVs) and convoluted 46 membranes (CM) in perinuclear regions of infected cells to which the viral 47 replication/transcription complex (RTC) is anchored (4, 8, 9). The RTC is generated by 48 translation of the genomic RNA into two large polyproteins that are extensively auto-49 proteolytically processed by viral proteases to give rise to 16 processing end-products, termed 50 non-structural proteins (nsps) 1-16. Nsp1 is rapidly cleaved from the polyproteins and not 51 considered an integral component of the coronaviral RTC, but interferes with host cell 52 translation by inducing degradation of cellular mRNAs (10-12). Although it has not yet been 53 formally demonstrated, the remaining nsps (2-16) are thought to comprise the RTC and harbor 54 multiple enzymes and functions, such as de-ubiquitination, proteases, helicase, polymerase, 55 exo-and endonuclease, and N7-and 2'O-methyltransferases (13-18). Many of these functions 56 have been studied using reverse genetic approaches, which revealed their importance in virus-57 host interactions (19-23). In most cases phenotypes were described via loss-of-function 58 mutagenesis, however, in the context of virus infection, the specific interactions of RTC 59 components with host cell factors remain largely unknown. 60 . CC-BY-NC-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/417907 doi: bioRxiv preprint 4 A number of individual host cell proteins have been shown to impact coronavirus replication 61 by using various screening methods, such as genome-wide siRNA, kinome, and yeast-two-62 hybrid screens (24-28). Likewise, genome-wide CRISPR-based screens have been applied to 63 other positive-stranded RNA viruses, such as flaviviruses, and identified critical host proteins 64 required for replication (29, 30). Some of these proteins were described in the context of 65 distinct ER processes, such as N-linked glycosylation, ER-associated protein degradation 66 (ERAD), and signal peptide insertion and processing. Although individual proteins identified 67 by these screens may interact with viral replication complexes, they likely constitute only a 68 small fraction of the global replicase microenvironment. 69\", 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'To capture the full breadth of host cell proteins and cellular pathways that are spatially 70 associated with viral RTCs, we employed a proximity-based labeling approach involving a 71 promiscuous E. coli-derived biotin ligase (BirAR118G). BirAR118G biotinylates proximal (<10 72 nm) proteins in live cells without disrupting intracellular membranes or protein complexes, and 73 hence, does not rely on high affinity protein-protein interactions, but is able to permanently tag 74 transient interactions (31). Covalent protein biotinylation allows stringent lysis and washing 75 conditions during affinity purification and subsequent mass spectrometric identification of 76 captured factors. By engineering a recombinant MHV harboring BirAR118G as an integral 77 component of the RTC we identified >500 host proteins reflecting the molecular 78 microenvironment of MHV replication structures. siRNA-mediated silencing of each of these 79 factors highlighted, amongst others, the functional importance of vesicular ER-Golgi apparatus 80 trafficking pathways, ubiquitin-dependent and autophagy-related catabolic processes, and 81 translation initiation factors. Importantly, the detection of active translation in close proximity 82 to the viral RTC highlighted the critical involvement of translation initiation factors during 83 coronavirus replication. Collectively, the determination of the coronavirus RTC-associated 84 microenvironment provides a functional and spatial link between conserved host cell processes 85 . CC-BY-NC-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/417907 doi: bioRxiv preprint 5 and viral RNA synthesis, and highlights potential targets for the development of novel antiviral 86 agents. 87 88 Results 89 Engineering the BirAR118-biotin ligase into the MHV replicase transcriptase complex 90 To insert the promiscuous biotin ligase BirAR118G as an integral subunit of the MHV RTC, we 91 used a vaccinia virus-based reverse genetic system (32, 33) to generate a recombinant MHV 92 harboring a myc-tagged BirAR118G fused to nsp2. This strategy was recently employed by 93 Freeman et al. for a fusion of green fluorescent protein (GFP) with nsp2 (34). MHV-BirAR118G-94 nsp2 retained the cleavage site between nsp1 and BirAR118G, while a deleted cleavage site 95', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'between BirAR118G and nsp2 ensured the expression of a BirAR118G-nsp2 fusion protein (Fig.  96   1a) . 97 MHV-BirAR118G-nsp2 replicated to comparable peak titers and replication kinetics as the 98 parental wild-type MHV-A59 (Fig. 1b) . MHV-EGFP-nsp2, which was constructed in parallel 99 and contained the coding sequence of EGFP (34) instead of BirAR118G, was used as a control 100 and also reached wild-type virus peak titers, with slightly reduced viral titers at 9 hours post-101 infection (h.p.i.) compared to MHV-A59 and MHV-BirAR118G-nsp2 (Fig. 1b) . 102', 'cite_spans': [], 'ref_spans': [{'start': 85, 'end': 100, 'text': '(Fig.  96   1a)', 'ref_id': None}, {'start': 224, 'end': 233, 'text': '(Fig. 1b)', 'ref_id': None}, {'start': 546, 'end': 555, 'text': '(Fig. 1b)', 'ref_id': None}], 'section': 'Abstract'}, {'text': 'To confirm the accommodation of BirAR118G within the viral RTC, MHV-A59-, MHV-103', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'BirAR118G-nsp2-, and mock-infected L929 fibroblasts were visualized using indirect 104 immunofluorescence microscopy. BirAR118G-nsp2 remained strongly associated with the MHV 105 RTC, as indicated by the co-localization of BirAR118G-nsp2 with established markers of the 106 MHV replicase, such as nsp2/3 and nsp8 ( Fig. 1c; Supplemental Fig. S1 ). This observation 107 corroborates previous studies demonstrating that nsp2, although not required for viral RNA 108 synthesis, co-localizes with other nsps of the coronaviral RTC (35-37). Importantly, by 109 . CC-BY-NC-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/417907 doi: bioRxiv preprint 6 supplementing the culture medium with biotin, we could readily detect biotinylated proteins 110 with fluorophore-coupled streptavidin that appeared close to the MHV RTC in MHV-111', 'cite_spans': [], 'ref_spans': [{'start': 315, 'end': 344, 'text': 'Fig. 1c; Supplemental Fig. S1', 'ref_id': None}], 'section': 'Abstract'}, {'text': \"BirAR118G-nsp2-infected cells, demonstrating efficient proximity-dependent biotinylation of 112 RTC-proximal host factors ( Fig. 1c; Supplemental Fig. S1 ). Furthermore, to define the 113 localization of the nsp2 fusion protein at the ultrastructural level, we replaced the BirAR118G 114 biotin ligase with the APEX2 ascorbate peroxidase to generate recombinant MHV-APEX2-115 nsp2. APEX2 mediates the catalysis of 3,3'-diaminobenzidine (DAB) into an insoluble 116 polymer that can be readily observed by electron microscopy (38). As shown in figure 1d , 117 APEX2-catalized DAB polymer deposition was readily detectable at characteristic coronavirus 118 replication compartments, such as DMVs and CM, categorically demonstrating that the nsp2 119 fusion proteins localize to known sites of coronavirus replication (4, 8). 120 Importantly, our collective results establish that the recombinant MHV-BirAR118G-nsp2 121 replicates with comparable kinetics to wild-type MHV-A59, expresses a functional BirAR118G 122 biotin ligase that is tightly associated with the MHV RTC, and that biotinylated, RTC-proximal 123 proteins can be readily detected in MHV-BirAR118G-nsp2 infected cells. 124 125\", 'cite_spans': [], 'ref_spans': [{'start': 124, 'end': 153, 'text': 'Fig. 1c; Supplemental Fig. S1', 'ref_id': None}, {'start': 542, 'end': 551, 'text': 'figure 1d', 'ref_id': None}], 'section': 'Abstract'}, {'text': 'To further demonstrate the efficiency and specificity of BirAR118G-mediated biotinylation we 127 assessed, by western blot analysis, fractions of biotinylated proteins derived from MHV-A59-128 , MHV-BirAR118G-nsp2-, or non-infected cells that were grown with or without the addition of 129 biotin (Fig. 2a,b) . A characteristic pattern of endogenously biotinylated proteins was observed 130 under all conditions where no exogenous biotin was added to the culture medium (Fig. 2b) . 131', 'cite_spans': [], 'ref_spans': [{'start': 297, 'end': 308, 'text': '(Fig. 2a,b)', 'ref_id': None}, {'start': 470, 'end': 479, 'text': '(Fig. 2b)', 'ref_id': None}], 'section': 'Abstract'}, {'text': 'The same pattern was detectable in non-infected and wild-type MHV-A59-infected cells when 132 the culture medium was supplemented with biotin, suggesting that the addition of biotin in the 133 . CC-BY-NC-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/417907 doi: bioRxiv preprint 7 absence of the BirAR118G biotin ligase does not recognizably change the fraction of 134 endogenously biotinylated proteins. In contrast, we observed a greatly increased fraction of 135 biotinylated proteins in lysates derived from MHV-BirAR118G-nsp2-infected cells treated with 136', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'biotin. This result demonstrates that virus-mediated expression of the BirAR118G biotin ligase 137 results in efficient biotinylation when biotin is added to the culture medium. Moreover, we 138 could readily affinity purify, enrich, and recover the fraction of biotinylated proteins under 139 stringent denaturing lysis and washing conditions by using streptavidin-coupled magnetic 140 beads (Fig. 2b) . 141', 'cite_spans': [], 'ref_spans': [{'start': 393, 'end': 402, 'text': '(Fig. 2b)', 'ref_id': None}], 'section': 'Abstract'}, {'text': 'Affinity purified proteins derived from biotin-treated MHV-A59-and MHV-BirAR118G-nsp2-142 infected cells were subjected to mass spectrometric analysis (n=3). Liquid chromatography 143 tandem-mass spectrometry (LC-MS/MS) was performed from in-gel digested samples and log-144 transformed label free quantification (LFQ) levels were used to compare protein enrichment 145 between samples (Fig. 2c) . Overall, 1381 host proteins were identified, of which 513 were 146 statistically significantly enriched in MHV-BirAR118G-nsp2-infected samples over MHV-A59-147 infected samples. These host proteins represent a comprehensive repertoire of RTC-proximal 148 factors throughout MHV infection (Fig. 2c, table S1 ). Importantly, viral replicase gene 149 products nsp2-10 and nsp12-16, and the nucleocapsid protein were significantly enriched in 150 fractions derived from MHV-BirAR118G-nsp2-infected cells (Fig. 2c, d) . This is in agreement 151 with studies demonstrating co-localization and interactions amongst individual nsps, and with 152 studies showing association of the nucleocapsid protein with the coronavirus RTC (8,[39][40][41]. 153', 'cite_spans': [], 'ref_spans': [{'start': 386, 'end': 395, 'text': '(Fig. 2c)', 'ref_id': None}, {'start': 686, 'end': 704, 'text': '(Fig. 2c, table S1', 'ref_id': None}, {'start': 898, 'end': 910, 'text': '(Fig. 2c, d)', 'ref_id': None}], 'section': 'Abstract'}, {'text': 'It also highlights the specificity and effectiveness of the labeling approach in live cells and is 154 the first experimental evidence showing that collectively these viral nsps and the nucleocapsid 155 (N) protein are subunits of the coronavirus RTC. Furthermore, these results corroborate 156 previous reports that nsp1 is likely not an integral component of the coronavirus RTC (10-12, 157 42). Amongst the \"not detected\" or \"not enriched\" viral proteins are (i) nsp11, which is a short 158 . CC-BY-NC-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/417907 doi: bioRxiv preprint number of host proteins from our candidate list in MHV-infected cells. Accordingly, we 172 identified RTC-proximal host proteins displaying a pronounced co-localization with the MHV 173 RTC, such as the ER protein reticulon 4 (rtn4; Fig. 2e ), and host proteins where co-localization 174 by indirect immunofluorescence microscopy was not readily detectable, such as the eukaryotic 175 translation initiation factor 3E (eIF3E; Fig. 2e ). However, in the latter case, a more sensitive 176 detection technique, such as a proximity ligation assay that relies on proximity-dependent 177 antibody-coupled DNA probe amplification (43), demonstrated proximity of eIF3E and 178 dsRNA in MHV-infected cells (Fig. 2f) . 179', 'cite_spans': [], 'ref_spans': [{'start': 942, 'end': 949, 'text': 'Fig. 2e', 'ref_id': None}, {'start': 1135, 'end': 1142, 'text': 'Fig. 2e', 'ref_id': None}, {'start': 1406, 'end': 1415, 'text': '(Fig. 2f)', 'ref_id': None}], 'section': 'Abstract'}, {'text': 'Collectively, our results show that the approach of integrating a promiscuous biotin ligase as 180 an integral subunit into a coronavirus RTC revealed a comprehensive list of host cell proteins 181 that comprises the RTC microenvironment. The efficacy and specificity of our approach is best 182 illustrated by the fact that we were able to identify all expected viral components of the MHV 183 . CC-BY-NC-ND 4.0', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}], 'body_text': [{'text': '8 peptide of only 14 amino acids at the carboxyterminus of polyprotein 1a with a yet unassigned 159 role or function in coronavirus replication, (ii) the structural proteins spike (S) protein, 160 envelope (E) protein, and membrane (M) protein, which mainly localize to sites of viral 161 assembly before being incorporated into newly-formed viral particles, and (iii) all accessory 162 proteins (NS2a, HE, ORF4, ORF5a). Altogether, these results validate the proximity-163 dependent biotinylation approach and demonstrate the specific and exclusive labeling of MHV-164 RTC-associated proteins (Fig. 2d) . 165', 'cite_spans': [], 'ref_spans': [{'start': 594, 'end': 603, 'text': '(Fig. 2d)', 'ref_id': None}], 'section': ''}, {'text': 'The BirAR118G biotin ligase biotinylates proteins in its close proximity that must not necessarily 166 have tight, prolonged, or direct interaction (31). Therefore, the identified RTC-proximal host 167 proteins, recorded over the entire duration of the MHV replication cycle, likely include proteins 168 that display a prolonged co-localization with the MHV RTC, proteins that may locate only 169 transiently in close proximity to the RTC, and proteins of which only a minor fraction of the 170 cellular pool may associate with the RTC. To this end, we assessed the localization of a limited 171 proteins, were not amongst the significantly enriched proteins. Our data further suggest that 185 the RTC microenvironment may be highly dynamic and likely also contains proteins that are 186 only transiently present in the microenvironment or only comprise a sub-fraction of the cellular 187 pool in close proximity to the MHV RTC. terms were highly significant (p-value <0.005) (Fig. 3a , Table S2 ). Additional analysis using 195 network analysis, using STRING, revealed at least 4 distinct clusters of interacting factors 208 within this category (Fig. 3b) . Cluster I, protein transport, comprised nuclear transport 209 receptors at nuclear pore complexes, such as importins and transportins. Interestingly, this 210 cluster also contained Sec63, which is part of the Sec61 translocon (44) and has been implicated 211 in protein translocation across ER membranes. The list of RTC-proximal factors also included 212 signal recognition particles SRP54a and SRP68 (Table S2) In order to assess the potential functional relevance of RTC-proximal factors identified in our 266 MHV-BirAR118G-nsp2-mediated proximity-dependent screen, we designed a custom siRNA 267 library individually targeting the expression of each of the 513 identified RTC-proximal host 268 proteins. siRNA-treated L929 cells were infected (MOI=0.05, n=4) with a recombinant MHV 269 vesicular transport between the ER, Golgi apparatus and endosomal compartments for the 283 establishment of replication organelles, several factors contained within these pathways 284 impaired MHV replication as exemplified by the siRNA-mediated silencing of Kif11, Snx9, 285 Dnm11, Scfd1, Ykt6, Stx5a, Clint1, Aak11, or Vapa (Fig. 4b) . Consistently, ER-associated 286 protein sorting complexes associated with the ribosome and newly synthetized proteins (Naca, 287 BTF3, SRP54a, SRP68) that were revealed in the GO enrichment analysis (Fig. 3a, table S2) , 288 also appear to be required for efficient MHV replication (Fig. 4b) . 289 Furthermore, we also observed significantly reduced MHV replication upon silencing of core 290 elements of the 26S and 20S proteasome complex (Psmd1 and Psmc2, and Psmb3, 291 respectively), suggesting a crucial role of the ubiquitin-proteasome pathway for efficient CoV 292 replication (27, 52). Indeed, this finding may provide a link to the described coronavirus RTC-293 encoded de-ubiquitination activity residing in nsp3 that has been implicated in innate immune 294 evasion (16, 17, 54). 295', 'cite_spans': [], 'ref_spans': [{'start': 976, 'end': 984, 'text': '(Fig. 3a', 'ref_id': None}, {'start': 987, 'end': 995, 'text': 'Table S2', 'ref_id': None}, {'start': 1147, 'end': 1156, 'text': '(Fig. 3b)', 'ref_id': None}, {'start': 1562, 'end': 1572, 'text': '(Table S2)', 'ref_id': None}, {'start': 2276, 'end': 2285, 'text': '(Fig. 4b)', 'ref_id': None}, {'start': 2487, 'end': 2506, 'text': '(Fig. 3a, table S2)', 'ref_id': 'FIGREF9'}, {'start': 2570, 'end': 2579, 'text': '(Fig. 4b)', 'ref_id': None}], 'section': ''}, {'text': 'Most interestingly, this custom siRNA screen identified a crucial role of the host protein 296 synthesis apparatus that was associated with the MHV RTC as indicated by the proximity-297 dependent proteomic screen (Fig. 3a, c) . Silencing of ribosomal proteins Rpl13a and Rls24d1 298 and several subunits of the eIF3 complex resulted in greatly reduced MHV replication and 299 scored with highest significance in the siRNA screen, suggesting that proximity of the host cell 300 translation machinery to the viral RTC likely has functional importance for coronavirus 301 replication (Fig. 4b) . 302 303 Active translation near sites of viral mRNA synthesis 304 . CC-BY-NC-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/417907 doi: bioRxiv preprint initiation factors, we extended these results in independent assays. For this, we selected all host 306 factors assigned to the category \"translation\" (Fig. 3a) and assessed virus replication following 307 siRNA-mediated silencing of each factor. Measurement of luciferase activity after MHV-Gluc 308 infection confirmed initial findings obtained by screening the entire siRNA library of MHV 309 RTC-proximal factors (Fig. 4c) . Specifically for Rpl13a, and eIFs 3i, 3f, and 3e viral replication 310 was reduced to levels comparable to our controls Ceacam1a (MHV receptor) and Gbf1 (24). 311', 'cite_spans': [], 'ref_spans': [{'start': 213, 'end': 225, 'text': '(Fig. 3a, c)', 'ref_id': None}, {'start': 581, 'end': 590, 'text': '(Fig. 4b)', 'ref_id': None}, {'start': 1025, 'end': 1034, 'text': '(Fig. 3a)', 'ref_id': None}, {'start': 1291, 'end': 1300, 'text': '(Fig. 4c)', 'ref_id': None}], 'section': ''}, {'text': 'Consistently, cell-associated viral mRNA levels (Fig. 4d ) and viral titers ( The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/417907 doi: bioRxiv preprint coronaviruses, such as HCoV-229E or the highly pathogenic MERS-CoV (Fig. 6) . The HCoV-331 229E RTC, which was detected with an antiserum directed against nsp8, appeared as small and 332 dispersed perinuclear puncta during early infection and eventually converged into larger 333 perinuclear structures later in infection. Consistent with findings obtained for MHV, we 334 observed a striking co-localization of the HCoV-229E RTC with sites of active translation 335 during the early phase of the infection (Fig. 6, S4) . The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/417907 doi: bioRxiv preprint defined temporal roles during particular phases of the viral life cycle and proteins that did not 380 yet attract our attention in previous screens because of functional redundancies. We therefore 381 expect that this approach will find wide application in the field of virus-host interaction, target 382 identification for virus inhibition, and provides a starting point to reveal similarities and 383 differences between replication strategies of a broad range of viruses. 384', 'cite_spans': [], 'ref_spans': [{'start': 48, 'end': 56, 'text': '(Fig. 4d', 'ref_id': None}, {'start': 276, 'end': 284, 'text': '(Fig. 6)', 'ref_id': None}, {'start': 716, 'end': 728, 'text': '(Fig. 6, S4)', 'ref_id': None}], 'section': ''}, {'text': \"One novel finding that arose immediately from our RTC-proximity screen is the demonstration 385 of a close spatial association of host cell translation with the coronavirus RTC. Indeed, the 386 biotin ligase-based proteomic screen identified a number of translation initiation factors, most 387 prominently several eIF3 subunits that were found to have functional importance for viral 388 replication, and numerous ribosome-and translation-associated proteins within the coronavirus 389 RTC microenvironment (Fig. 3, 4) . In addition, the presence of subunits of the signal uncorrected Dunn's test) was used to test the statistical significance of reduced viral replication 641 (mean < 95% as compared to scramble control, n=216). The R package ggplot2 was used to 642 create the bubble plot (Fig 4B) . The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/417907 doi: bioRxiv preprint\", 'cite_spans': [], 'ref_spans': [{'start': 508, 'end': 519, 'text': '(Fig. 3, 4)', 'ref_id': None}, {'start': 792, 'end': 800, 'text': '(Fig 4B)', 'ref_id': None}], 'section': ''}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'Endoplasmic Reticulum: The Favorite Intracellular biotinylation. MHV-BirAR118G-nsp2, MHV-A59-or non-infected L929 fibroblasts were 1045 cultured in medium supplemented with 67μM biotin. Cells were fixed 12 hours post infection 1046 (h.p.i.) and processed for immunofluorescence analysis', 'authors': [{'first': 'I', 'middle': [], 'last': 'Romero-Brey', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Bartenschlager', 'suffix': ''}], 'year': 2016, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF1': {'ref_id': 'b1', 'title': 'Nuclei are counterstained with DAPI. Z-projection of deconvolved z-stacks 1049 acquired with a DeltaVision Elite High-Resolution imaging system are shown', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF2': {'ref_id': 'b2', 'title': 'Control cells were treated, prior to puromycin incubation, with 355 μM 1070 cycloheximide and 208 μM Emetin for 30 min to block protein synthesis. Western blots were 1071 probed using anti-puromycin antibodies to assess puromycin incorporation into polypeptides 1072 and normalized to actin levels. Error bars represent the mean standard deviation of three 1073 independent experiments', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF3': {'ref_id': 'b3', 'title': 'Supplemental Figure 4. Visualization of active translation during HCoV-229E infections', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF4': {'ref_id': 'b4', 'title': 'Cells were pulsed 1077 with cycloheximide, emetine and puromycin for 5min to label translating ribosomes and 1078 subjected to a coextraction/fixation procedure to remove free puromycin', 'authors': [], 'year': None, 'venue': 'Huh7 cells were infected with HCoV-229E (MOI=1) for 12', 'volume': '15', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF5': {'ref_id': 'b5', 'title': 'non-pulsed cells were used as control. Cells were labelled using anti-nsp8 (HCoV-229E) and', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}}, 'ref_entries': {'FIGREF0': {'text': 'classification of RTC-proximal host factors 190 To categorize functionally-related proteins from the list of RTC-proximal host proteins and 191 identify enriched biological themes in the dataset, we performed a functional classification of 192 RTC-proximal factors using Gene Ontology (GO) enrichment analysis. 86 GO biological 193 process (BP) terms were significantly enriched in the dataset (p-value <0.05), of which 32 194', 'latex': None, 'type': 'figure'}, 'FIGREF1': {'text': 'proteins that promote the transfer 213 of newly synthetized integral membrane proteins or secreted proteins across translocon 214 complexes. Furthermore, the list contained Naca and BTF3, which prevent the translocation of 215 non-secretory proteins towards the ER lumen (45, 46). Interestingly, genome-wide CRISPR 216 screens have identified proteins involved in biosynthesis of membrane and secretory proteins 217 as required for flavivirus replication (29, 30), suggesting similarities between flaviruses and 218 coronaviruses concerning the requirement of ER-associated protein sorting complexes for viral 219 replication. 220 repertoire of candidate proteins implicated in the coronavirus replication cycle, and contains 259 several factors that have previously been reported to impact the replication of other positive-260 sense RNA viruses. Importantly, since our screening approach was tailored to detect host 261 factors associated with the coronavirus RTC, it provides a spatial link of these factors to the 262 site of viral RNA synthesis. 263 264 Identification of proviral factors within the coronavirus RTC microenvironment 265', 'latex': None, 'type': 'figure'}, 'FIGREF2': {'text': 'Fig. 4e) were reduced 312 upon siRNA silencing of these factors. Although the silencing of a subset of host translation 313 factors severely restricted MHV replication, effective knockdown of these factors (Fig. S3c) 314 did not affect cell viability (Fig. S3b, d) and only moderately affected host cell translation 315 levels (Fig. 4f, S3e). This data demonstrates that the reduced viral replication observed after 316 siRNA knockdown is not due to a general impairment of host translation. 317 Subsequently, we aimed to visualize the localization of active translation during virus infection 318 by puromycin incorporation into nascent polypeptides on immobilized ribosomes 319 (ribopuromycylation) followed by fluorescence imaging using antibodies directed against 320 puromycin (55). In non-infected L929 cells, ribopuromycylation resulted in an expected 321 diffuse, mainly cytosolic, staining pattern interspersed with punctate structures indicative of 322 translation localized to dedicated subcellular cytosolic locations (Fig. 5). In striking contrast, 323 MHV-infected L929 cells displayed a pronounced enrichment of actively translating ribosomes 324 near the viral RTC as indicated by the strong overlap between the viral replicase and the 325 ribopuromycylation stain. Interestingly, active translation in vicinity of the RTC was strongest 326 during the early phase of infection at 6 h.p.i., and was observed until 8 h.p.i., before gradually 327 decreasing as the infection advanced along with the appearance of typical syncytia formation 328 indicative of cytopathic effect (CPE). 329', 'latex': None, 'type': 'figure'}, 'FIGREF3': {'text': \"The co-localization gradually decreased as 336 the infection reached the late phase with upcoming signs of CPE. Finally, we further 337 demonstrated that active translation is localized to the site of MERS-CoV RNA synthesis as 338 dsRNA puncta highly overlapped with the ribopuromycylation stain in MERS-CoV-infected 339Huh7 cells(Fig. 6). Collectively, these results not only confirm the spatial link between 340 individual components of the host cell translation machinery and coronavirus replication 341 compartments as identified by proximity-dependent biotinylation using MHV-BirAR118G-nsp2, 342 but they also demonstrate that active translation is taking place in close proximity to the study, we made use of a recently developed system based on proximity-dependent 347 biotinylation of host factors in living cells (31). By engineering a promiscuous biotin ligase 348 (BirAR118G) as an integral component of the coronavirus replication complex, we provide a 349 novel approach to define the molecular mircoenvironment of viral replication complexes that 350 is applicable to many other RNA and DNA viruses.351 We show that nsp2 fusion proteins encoded by recombinant MHV-APEX2-nsp2 and MHV-352 BirAR118G-nsp2, are indeed part of the RTC and localize to characteristic coronavirus 353 . Notably, in MHV-BirAR118G-nsp2-infected cells the 357 detection of biotinylated coronavirus replicase gene products nsp2-10, nsp12-16, and the 358 nucleocapsid protein by mass spectrometry demonstrates that these proteins are in close 359 proximity during infection. This extends previous immunofluorescence and electron 360 microscopic studies that were limited by the availability of nsp-specific antibodies and could 361 only show localization of individual nsps to coronavirus replicative structures (4, 8, 35-37). 362 Moreover, the close proximity of BirAR118G-nsp2 to MHV replicative enzymes, such as the 363 RNA-dependent RNA polymerase (nsp12), the NTPase/helicase (nsp13), the 5'-cap 364 methyltransferases (nsp14, nsp16), the proof-reading exonuclease (nsp14), in MHV-BirAR118G-365 nsp2-infected cells further suggest close proximity of nsp2 to the site of viral RNA synthesis. 366 We thus propose that nsp2-16 and the nucleocapsid protein collectively constitute a functional 367 coronavirus replication and transcription complex in infected cells. 368 The analysis of the host proteome enriched at MHV replication sites revealed a comprehensive 369 list of host proteins that constitute the coronavirus RTC microenvironment. This included 370 several individual factors and host cell pathways, especially transport mechanisms involving 371 vesicle-mediated trafficking, which have previously been shown to assist coronavirus 372 replication. (24, 26, 27, 49, 52). Moreover, numerous coronavirus RTC-proximal host proteins 373 and pathways also have documented roles in the life cycle of other, more intensively studied, 374 positive-stranded RNA viruses, suggesting considerable commonalities and conserved virus-375 host interactions at the replication complexes of a broad range of RNA viruses (29, 30, 56). 376 Importantly, our list of RTC-proximal proteins by far exceeds the number of host cell proteins 377 currently known to interact with viral replication complexes and the vast majority of MHV 378\", 'latex': None, 'type': 'figure'}, 'FIGREF4': {'text': '390 recognition particle in proximity to the coronavirus RTC and their functional relevance for 391 viral replication is indicative of an importance for the translation of membrane proteins. 392 recent genome-wide CRISPR screens demonstrated the functional importance of proteins 404 involved in biosynthesis of membrane and secretory proteins, further supporting a pivotal role 405 of the ER-associated translation machinery for virus replication (30). 406 Compartmentalization of cellular translation to sites of viral RNA synthesis has been described 407 for dsRNA viruses of the orthoreovirus family, which replicate and assemble in distinct 408 cytosolic inclusions known as viral factories to which the host translation machinery is 409 recruited (58). The data presented here indicate that coronaviruses have evolved a similar 410 strategy by compartmentalizing and directing viral RNA synthesis to sites of ER-associated 411 translation. Likewise, this strategy has a number of advantages. Coronaviruses would not 412 require sophisticated transport mechanisms that direct viral mRNA to distantly located 413 ribosomes. A close spatial association of viral RNA synthesis and translation during early post-414 entry events would rather allow for remodeling the ER-associated translation machinery to 415 ensure translation of viral mRNA in a protected microenvironment. Viruses have evolved 416 diverse mechanisms to facilitate translation of their mRNAs including highly diverse internal nsp2 -mediated labelling approach to identify RTC-proximal cellular processes that 429 significantly contribute to viral replication. Indeed, the ability of BirAR118G to label viral and 430 host factors independently of high affinity and prolonged molecular interactions enables the 431 establishment of a comprehensive repertoire reflecting the history of protein association with 432 the viral RTC, recorded during the entire course of infection. In future studies it will be 433 important to provide an \"RTC-association map\" with temporal resolution. Like we have seen 434 for translation initiation factors in this study, association of host cell proteins with the viral 435 RTC might not persist throughout the entire replication cycle but might be of importance only 436 transiently or during specific phases of the replication cycle. Given its short labelling time, 437 APEX2 indeed offers this possibility to dissect protein recruitment to the viral RTC in a time-438 resolved manner, i.e. to detect RTC-associated host proteins at specific time points post 439 infection. This will ultimately result in a dynamic, high resolution molecular landscape of 440 virus-host interactions at the RTC and provide an additional impetus to elucidate critical virus-441 host interactions that take place at the site of viral RNA synthesis. These interactions should 442 be exploited in the development of novel strategies to combat virus infection, based on 443 conserved mechanisms of interactions at replication complexes of a broad range of positive1774 was used for the generation of MHV-BirAR118G-nsp2 and MHV-APEX2-nsp2. The 491 resulting VV were used to generate full-length cDNA genomic fragments by restriction 492 digestion of the VV backbone. Rescue of MHV-GFP-nsp2, MHV-BirAR118G-nsp2 and MHV-493 APEX2-nsp2 was performed by electroporation of capped in vitro transcribed recombinant 494 genomes into a BHK-21-derived cell line stably expressing the nucleocapsid (N) protein 495 layered on permissive 17Cl1 mouse fibroblasts. Recombinant MHV viruses were plaque-496 purified three times and purified viruses were passaged three times for stock preparations. All 497 plasmid sequences, VV sequences and recombinant MHV sequences were confirmed by PCR 498 or RT-PCR sequencing. Viruses were propagated on 17Cl1 cells and virus stocks were titrated 499 by plaque assay on L929 cells. 500 501 Viral replication assay 502 L929 cells were infected with MHV-A59, MHV-GFP-nsp2, MHV-BirAR118G-nsp2 or MHV-503 APEX2-nsp2 in quadruplicate at an MOI=1. Virus inoculum was removed 2 h.p.i., cells were 504 washed with PBS and fresh medium was added. Viral supernatants were collected at the 505 indicated time point and titrated by plaque assay on L929 cells. Titers reported are the averages 506 of three independent experiments ± standard error of the mean (SEM). 507 with 4% (v/v) neutral buffered formalin before being washed three additional times. CB) for 60 min and incubated 60 min with the indicated primary antibodies diluted in 516 CB (polyclonal anti-MHV-nsp2/3 or nsp8 (gift from S. Baker), 1:200 (35, 67); anti-myc, 517 1:8000 Cell Signalling 2276). Cells were washed three times with CB and incubated for 60 min 518 with donkey-derived, AlexaFluor488-conjugated anti-mouse IgG (H+L) and donkey-derived, 519 AlexaFluor647-conjugated anti-rabbit IgG (H+L) (Jackson Immunoresearch). Cells were 520 additionally labelled with streptavidin conjugated to AlexaFluor 594 (Molecular Probes) to 521 detect biotinylated proteins. Coverslips were mounted on slides using immunofluorescence detection of viral and host proteins, L929 cells were grown 525 on glass coverslips in 24-well plates and infected with MHV-A59 or MHV-BirAR118G-nsp2 526 (MOI=1). At the indicated time point, cells were fixed with 4% (v/v) formalin and processed 527 using primary monoclonal antibodies directed against dsRNA (J2 Mab, English Scientific and 528 Consulting) or myc-tab (Cell signalling 2276) and polyclonal antibodies recognizing eIF3E 529 (Sigma, HPA023973) or RTN4 (Nogo A+B, Abcam 47085) as well as secondary donkey-530 derived, AlexaFluor488-conjugated anti-mouse and AlexaFluor647-conjugated anti-rabbit IgG 531 (H+L), as described above. 532 For proximity ligation assays, L929 cells were seeded in 24-well plates on glass coverslips and 533 infection with MHV-A59 or MHV-BirAR118G-nsp2 (MOI=1). At the indicated time point, cells 534 were washed with PBS, fixed with 4% (v/v) formalin and permeabilized with 0.1% (v/v) Triton 535 X-100. Proximity ligation was performed as recommended by the manufacturer (Duolink In 536 Situ detection reagents Red, Sigma) using monoclonal antibodies directed against dsRNA (were imaged by acquiring 0.2 μm stacks over 10 μm using a DeltaVision Elite 541 High-Resolution imaging system (GE Healthcare Life Sciences) equipped with a 60x or 100x 542 oil immersion objective (1.4 NA). Images were deconvolved using the integrated softWoRx 543 software and processed using Fiji (ImageJ). 544 545 Biotinylation assay -western blot -mass spectrometry 546 L929 cells were infected with MHV-A59 or MHV-BirAR118G-nsp2, and for comparison 547 MHVH277A and MHVH227A-BirAR118G-nsp2, at an MOI=1 in medium supplemented with 67 μM 548 biotin (Sigma B4501). At 15 h.p.i., cells were washed three times with PBS and lysed in ice-549 cold buffer containing 50 mM TRIS-Cl pH 7.4, 500 mM NaCl, 0.2% (w/v) SDS, 1 mM DTT 550 and 1x protease inhibitor (cOmplete Mini, Roche). Cells were scraped off the flask and 551 transferred to tubes. Cells were kept on ice until the end of the procedure. Triton X-100 was 552 added to each sample to a final concentration of 2%. Samples were sonicated for two rounds 553 of 20 pulses with a Branson Sonifier 250 (30% constant, 30% power). Equal volumes of 50 554 mM TRIS-Cl were added to each sample and samples were centrifuged at 4°C for 10 min at 555 18000 x g. Supernatants were incubated with magnetic beads on a rotator at 4°C overnight 556 (800 μl Dynabeads per sample, MyOne Streptavidin C1, Life Technologies) that were 557 previously washed with lysis buffer diluted 1:1 with 50 mM TRIS-Cl. Beads were washed 558 twice with buffer 1 (2% (w/v) SDS), once with buffer 2 (0.1% (w/v) deoxycholic acid, 1% 559(v/v) Triton X-100, 1 mM EDTA, 500 mM NaCl, 50 mM HEPES pH 7.5), once with buffer 3 560 (0.5% w/v deoxycholic acis, 0.5% NP40, 1 mM EDTA, 250 mM LiCl, 10 mM TRIS-Cl pH 561 SDS-PAGE and western blot analysis, cells were cultured in 6-well plates and lysates were 564 prepared and affinity purified as described above. Proteins were separated on 10% (w/v) SDS-565 polyacrylamide gels (Bio-Rad), and proteins were electroblotted on nitrocellulose membranes 566 (Amersham Biosciences, GE Healthcare) in a Mini Trans-Blot cell (Bio-Rad). Membranes 567 were incubated in a protein-free blocking buffer (Advansta) and biotinylated proteins were 568 probed by incubation with horseradish peroxidase-conjugated Streptavidin (Dako). Proteins 569 were visualized using WesternBright enhanced chemiluminescence horseradish peroxidase 570 substrate (Advansta) according to the manufacturer\\'s protocol. 571For mass spectrometry analysis, lysates and affinity purification were performed as described 572 above from 4*10 7 cells cultured in 150 cm 2 tissue culture flasks. Proteins were separated 1 cm 573 into a 10% (w/v) SDS-polyacrylamide gel. A Coomassie stain was performed and 4x 2 mm 574 bands were cut with a scalpel. Proteins on gel samples were reduced, alkylated and digested under Ubuntu against a forward + reverse Mus musculus (UniprotKB version 2016_04Lysine and on protein N-term, and deamidation of Glutamine and Asparagine. 590Parent and fragment mass tolerances were set to 10 ppm and 0.4 Da, respectively. Matches on 591 the reversed sequence database were used to set a Z-score threshold, where 1% false 592 discoveries (FDR) on the peptide spectrum match level had to be expected. Protein 593 identifications were only accepted, when two unique peptides fulfilling the 1% FDR criterion 594 spectrometry(70-73). GO BP terms with a p-value <0.05 were considered to be terms that were 612 significantly enriched in the dataset. Additional analysis of significant GO terms was conducted 613 using AmiGO and revealed that the top 32 GO BP terms (p-value <0.005) were predominantly 614 associated with five broad functional categories (cell-cell adhesion, transport, cell organization, 615 translation, and catabolic processes)(74). Alternatively, enrichment analysis was performed 616 using SetRank (data not shown), a recently described algorithm that circumvents pitfalls of 617 commonly used approaches and thereby reduces the amount of false-positive hits (75) and the 618 following databases were searched for significant gene sets: BIOCYC (76), GO (72), ITFP 619 (77), KEGG (78), PhosphoSitePlus (79), REACTOME (80), and WikiPathways (81). Both 620 independent approaches lead to highly similar results and consistently complement results 621 obtained upon GO Cellular Components analysis. 622 STRING functional protein association networks were generated using RTC-proximal host 623 proteins found within each of the five broad functional categories. Default settings were used 624 Assay Kit (ThermoFisher Scientific). Experiments were carried out in 4 independent replicates 638 and both cytotoxicity values and luciferase counts were normalized to the corresponding non-639 targeting scrambled control of each plate. A one-way ANOVA (Kruskal-Wallis test, 640', 'latex': None, 'type': 'figure'}, 'FIGREF5': {'text': \"were transfected with 10 nM siRNA as described above. 48 h post-transfection, cell 646 viability was assessed using the CytoTox 96 Non-Radioactive Cytotoxicity Assay (Promega) 647 and visually inspected by automated phase-contrast microscopy using an EVOS FL Auto 2 648Imaging System equipped with a 4x air objective. Cells were infected with MHV-Gluc 649 (MOI=0.05), washed with PBS 3 h.p.i. and incubated for 9 additional hours. Gaussia luciferase 650 activity, viral titers and cell viability were measured from the supernatant as described above. 651One-way ANOVAs (ordinary one-way ANOVA, uncorrected Fisher's LSD test) were used to 652 test the statistical significance.653 Total cellular RNA was isolated from cells using the NucleoMag® RNA Kit (Machery Nagel, 654 Switzerland) on a KingFisher™ Flex Purification System (Thermo Fisher Scientific, 655 Switzerland) according to the manufacture's instructions. The QuantiTect Probe RT-PCR Kit 656 (Qiagen, Switzerland) was used according to the manufactures instructions for measuring the 657 cell associated viral RNA levels with primers and probe specific to the MHV genome fragment 658 coding the nucleocapsid gene (Table S4). Primers and Probe for mouse Glyceraldehyde 3-659 phosphate dehydrogenase (GAPDH) where obtained from ThermoFisher Scientific 660 (Mm03302249_g1, Catalog Number: 4331182). The MHV levels were normalized to GAPDH 661 Ct Tbp) (84). One-way ANOVA (ordinary one-way 667 ANOVA, uncorrected Fisher's LSD test) was used to test the statistical significance. 668 669 Total cellular translation 670 siRNA-based silencing was performed as described above. 48 h post-transfection, control cells 671 were incubated with 355 M cycloheximide (Sigma) and 208 M Emetin (Sigma) for 30 min 672 to block protein synthesis. Cells were treated with 3 M puromycin for 60 min followed by 673 three PBS washes(85). Total cell lysates were prepared using M-PER mammalian protein 674 extraction reagent (Thermo Scientific) supplemented with protease inhibitors (Mini, Roche). Cells were blocked for 30 min in CB, and immunostained as 692 described above using polyclonal anti-MHV-nsp2/3 (gift from S. Baker), polyclonal anti-693 HCoV-229E-nsp8 (gift from J. Ziebuhr), or monoclonal anti-dsRNA (J2 MAB, English and 694 Scientific Consulting) as primary antibodies to detect MHV, HCoV-229E and ZIKV 695 replication complexes, respectively. Donkey-derived, AlexaFluor488-conjugated anti-mouse 696 or anti-rabbit IgG (H+L) were used as secondary antibodies. Additionally, ribosome-bound 697 puromycin was detected using a monoclonal AlexaFluor647-conjugated anti-puromycin 698 antibody (clone 12D10, Merk Millipore). Slides were mounted, imaged and processed as 699 described above. 700 701 DAB staining and transmission electron microscopy 702 L929 fibroblasts were seeded in 24-well plates and infected with MHV-APEX2-nsp2, MHV-703 A59, or non-infected for 10 h. 3,3-diaminobenzidine (DAB) stains were performed as described 704 previously (38). Briefly, cells were fixed at 10 h.p.i. using warm 2% (v/v) glutaraldehyde in 705 100 mM sodium cacodylate, pH 7.4, supplemented with 2 mM calcium chloride (cacodylate 706 buffer) and placed on ice for 60 min. The following incubations were performed on ice in ice-707 cold buffers unless stated otherwise. Cells were washed 3x with sodium cacodylate buffer, 708 quenched with 20 mM glycine in cacodylate buffer for 5 min. before 3 additional washes with 709 cacodylate buffer. Cells were stained in cacodylate buffer containing 0.5 mg/ml DAB and 10 710 mM H2O2 for 20 min until DAB precipitates were visible by light microscopy. Cells were 711 30 min at room temperature or overnight at 4 °C. After three washes in 716 cacodylate buffer for 10 min each, cells were post-fixed with 1% OsO4 (Chemie Brunschwig, 717 Basel, Switzerland) in 0.1 M cacodylate buffer for 1 h at 4 °C and again washed three times 718 with cacodylate buffer. Thereafter, cells were dehydrated in an ascending ethanol series (70%, 719 80%, 90%, 94%, 100% (v/v) for 20 min each) and embedded in Epon resin, a mixture of Epoxy 720 embedding medium, dodecenyl succinic anhydride (DDSA) and methyl nadic anhydride 721 (MNA) (Sigma Aldrich, Buchs, Switzerland). Ultrathin sections of 90 nm were then obtained 722 with diamond knives (Diatome, Biel, Switzerland) on a Reichert-Jung Ultracut E (Leica, 723 Heerbrugg, Switzerland) and collected on collodion-coated 200-mesh copper grids (Electron 724 Figure legends\", 'latex': None, 'type': 'figure'}, 'FIGREF6': {'text': 'Characterization of the recombinant MHV-BirAR118G-nsp2. (a) Genome 734 organization of recombinant MHV-BirAR118G-nsp2. The positive-sense RNA genome of MHV 735 contains a 5\\' cap and a 3\\' poly(A) tail. ORF1a and ORF1b encode the viral replication and 736 transcription complex (nsp1-16). myc-BirAR118G was inserted as an N-terminal fusion with 737 nsp2 within ORF1a. The cleavage site between nsp1 and myc-BirAR118G was retained (black 738 arrow) while a deleted cleavage site between BirAR118G and nsp2 ensured the release of a 739 BirAR118G-nsp2 fusion protein from the pp1a polyprotein. The cleavage site between nsp2 and 740 nsp3 was also retained. (b) Viral replication kinetics of recombinant MHV-BirAR118G-nsp2 741 were compared to wild-type MHV-A59 and recombinant MHV-GFP-nsp2. Murine L929 742 fibroblasts were infected at a multiplicity of infection (MOI) of 1 plaque forming unit (pfu) per 743 cell. Viral supernatants were collected at the indicated time points, titrated by plaque assay and 744 expressed in pfu per ml. Data points represent the mean and SEM of three independent 745 experiments, each performed in quadruplicate. (c) Immunofluorescence analysis of MHV-Determination of the coronavirus RTC-proximal proteome (aMHV-BirAR118G-nsp2, MHV-A59 (parental wild-type strain) or non-infected in 761 medium with and without supplementation of 67 μM biotin. Cells were lysed 15 h.p.i. and 762 biotinylated factors were subjected to affinity purification using streptavidin-coupled magnetic 763 beads. Total cell lysates and affinity-purified fractions were separated by SDS-PAGE and 764 analysed by western blot probed with horse radish peroxidase (HRP)-coupled Streptavidin. (c) 765 Host and viral factors identified by LC-MS/MS. 4*107 L929 cells were infected with MHV-766 BirAR118G-nsp2 or MHV-A59 in medium supplemented with 67μM biotin. 15 h.p.i., lysates 767 were affinity purified and LC-MS/MS was performed from in-gel digested samples. MS 768 identification of biotinylated proteins was performed in three independent biological replicates. 769 to the level of viral replication impairment. Colors correspond to the functional categories 807 highlighted in Figure 3. Light grey bubbles (below the dashed line) represent host proteins that 808 did not significantly impact MHV replication (p-value > 0.05). (c, d, e, f) Silencing of RTC-809 proximal components of the cellular translation machinery. Upon 48h siRNA silencing of 810 factors assigned to the category \"translation\" (Figure 3), L929 fibroblasts were infected with 811 MHV-Gluc (MOI=0.05, n=3). Luciferase activity (c), cell-associated viral RNA levels (d) and 812 viral titers (e) were assessed at 12 h.p.i.. (f) Western blot quantification of total cellular 813 translation following silencing of a subset of the host translation apparatus. Upon 48h siRNA-814 silencing, L929 fibroblasts were pulsed with 3 μM puromycin for 60 min. Control cells were 815 treated, prior to puromycin incubation, with 355 μM cycloheximide and 208 μM Emetin for 816 30 min to block protein synthesis. Cell lysates were separated by SDS-PAGE and Western 817 blots were probed using anti-puromycin antibodies to assess puromycin incorporation into 818 polypeptides and normalized to actin levels. Error bars represent the mean standard deviationActive translation near sites of MHV mRNA synthesis. Visualization of active 822 translation in MHV-infected L929 fibroblasts. Cells infected with MHV-A59 (MOI=1) or non-823 infected cells were cultured for 6, 8, 10 and 12 hours and pulsed with cycloheximide, emetine', 'latex': None, 'type': 'figure'}, 'FIGREF7': {'text': 'Active translation near sites of HCoV-229E and MERS-CoV mRNA synthesis. 833 Visualization of active translation during hCoV-229E and MERS-CoV infections. Huh7 cells 834 were infected with HCoV-229E and MERS-CoV (MOI=1) for 12 h and 6 h, respectively. Cells 835 were pulsed with cycloheximide, emetine and puromycin for 5min to label translating 836 ribosomes and subjected to a coextraction/fixation procedure to remove free puromycin. Non-837 infected and/or non-pulsed cells were used as control. Cells were labelled using anti-nsp8 838 (HCoV-229E) or dsRNA (MERS-CoV) and anti-puromycin antibodies. Nuclei are 839 counterstained with DAPI. Z-projection of deconvolved z-stacks acquired with a DeltaVision 840 Elite High-Resolution imaging system are shown. Intensity profiles of magnified regions are 841 shown. Scale bar: 20 μm. with hepatitis C virus replication. PLoS Pathog 8(12):e1003056. and limits apoptosis in macrophages. Proc Natl Acad Sci U S A', 'latex': None, 'type': 'figure'}, 'FIGREF9': {'text': 'Figure S2', 'latex': None, 'type': 'figure'}, 'FIGREF11': {'text': 'Figure S3', 'latex': None, 'type': 'figure'}, 'FIGREF12': {'text': 'Figure S4', 'latex': None, 'type': 'figure'}, 'TABREF0': {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.BY-NC-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/417907 doi: bioRxiv preprint', 'latex': None, 'type': 'table'}}, 'back_matter': []}\n",
      "{'paper_id': 'ac8c98a982e2b5aec7061105538e56fcaedeaf85', 'metadata': {'title': 'Epidemiological Tools that Predict Partial Herd Immunity to SARS Coronavirus 2', 'authors': [{'first': 'Yasuhiko', 'middle': [], 'last': 'Kamikubo', 'suffix': '', 'affiliation': {}, 'email': 'atkamikubo.yasuhiko.7u@kyoto-u.ac.jp'}, {'first': 'Atsushi', 'middle': [], 'last': 'Takahashi', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Y', 'middle': ['K'], 'last': '', 'suffix': '', 'affiliation': {}, 'email': ''}]}, 'abstract': [], 'body_text': [{'text': 'The outbreak of SARS coronavirus 2 (SARS-CoV-2), which occurred in Wuhan, China in December 2019, has caused a worldwide pandemic of coronavirus disease 2019 . However, there is a lack of epidemiological tools to guide effective public policy development. Here we present epidemiological evidence that SARS-CoV-2 S type exited Wuhan or other epicenters in China earlier than L type and conferred partial resistance to the virus on infected populations. Analysis of regional disparities in incidence has revealed that a sharp decline in influenza epidemics is a useful surrogate indicator for the undocumented spread of SARS-CoV-2. The biggest concern in the world is knowing when herd immunity has been achieved and scheduling a time to regain the living activities of each country. This study provides a useful tool to guide the development of local policies to contain the virus.', 'cite_spans': [], 'ref_spans': [], 'section': 'SUMMARY'}, {'text': 'The pandemic of SARS coronavirus 2 (SARS-CoV-2) 1-4 fills the world with fear and confusion, threatening the collapse of health care and the global financial crisis.', 'cite_spans': [], 'ref_spans': [], 'section': 'SUMMARY'}, {'text': \"Flattening the epidemic curve to avoid loss of healthcare capacity is a major global strategy, but has the downside of slowing the achievement of full herd immunity and sacrificing economic activity. Once the herd immunity is completed, the government can confidently decide when to lift the restriction. Therefore, the government must predict trends in the outbreak of SARS-CoV-2. There is no waiting in a pandemic and no time to wait for scientific confirmation. At such times epidemiology exists to determine public health policy first and foremost. Unfortunately, no government knows epidemiological methods for predicting their own SARS-CoV-2 status. Here, we developed the world's first influenza-based epidemiological method as a useful proxy to detect the spread of SARS-CoV-2 and the establishment of partial herd immunity in countries. \", 'cite_spans': [], 'ref_spans': [], 'section': 'SUMMARY'}, {'text': 'The quantification and modeling of the epidemiological curve was performed under contract with Shin Nippon Biomedical Laboratories, Ltd. (Tokyo, Japan). Analyses of Spearman correlation coefficient were performed with the use of the Statcel4 add-in package (OMS Publishing, Tokorozawa, Japan) for Microsoft Excel.', 'cite_spans': [], 'ref_spans': [], 'section': 'STATISITICAL ANALYSIS'}, {'text': 'A population genetic study of 103 SARS-CoV-2 genomes has identified two viral subtypes designate S and L. 5 The S type is the ancestral version of the virus. Notably, only 1 out of 27 (3.7%) viral strains isolated from Wuhan was S type, while 10 out of 31 (32.3%) from other places in China and 18 out of 45 (40.0%) outside the country were S type. The L type has accumulated more mutations than the S type and the orf1ab replicase/transcriptase gene of L type has a codon that is more preferred for translation than S type, suggesting that the L type replicates and/or transmits faster than the S type.', 'cite_spans': [{'start': 106, 'end': 107, 'text': '5', 'ref_id': 'BIBREF4'}], 'ref_spans': [], 'section': 'RESULTS'}, {'text': 'However, it remains to be determined whether the L type is more virulent than the S type.', 'cite_spans': [], 'ref_spans': [], 'section': 'RESULTS'}, {'text': 'We examined whether viral subtypes isolated from countries outside China are correlated with patterns of SARS-CoV-2 epidemics. As shown in Table 1 , lower ratio of isolated S type is associated with delayed spread of SARS-CoV-2. Moreover, the epidemic seems to be subsiding in areas where S type virus is predominant. These results prompted us to hypothesize that (i) S type SARS-CoV-2 exit Wuhan or other epicenter in China earlier than L type virus without recognition by infectious disease surveillance systems of China and other countries; (ii) the infection by S type induced herd immunity that provides at least partial protection against spread of SARS-CoV-2.', 'cite_spans': [], 'ref_spans': [{'start': 139, 'end': 146, 'text': 'Table 1', 'ref_id': None}], 'section': 'RESULTS'}, {'text': 'is likely that the L type took over the S type because of higher replicative potential and/or faster transmission. Asymptomatic ratio of Japanese nationals evacuated from Wuhan, who are more likely to be infected by the L type, is 30.8% (95% confidence interval: 7.7-53.8%). 6 In contrast, 51.4% (318/619) of RT-PCR confirmed cases among passengers in the cruise ship Diamond Princess, an outbreak traced to a Hong Kong passenger who is most likely to have S type virus, were asymptomatic (https://www.niid.go.jp/niid/en/2019-ncov-e/9417-covid-dp-fe-02.html). Although the subtypes of the virus remain to be clarified, the higher asymptomatic ratio among cruise ship passengers supports the view that infection by S type is more likely to be asymptomatic than that by L type and easier to avoid recognition by surveillance systems.', 'cite_spans': [{'start': 275, 'end': 276, 'text': '6', 'ref_id': 'BIBREF5'}], 'ref_spans': [], 'section': 'Clonal competition between L and S subtypes should have taken place in Wuhan and it'}, {'text': 'Medical doctors in Japan have noticed that influenza epidemic suddenly faded this winter season. Type I interferons induced by S type SARS-CoV-2 may interfere with infection by influenza virus. We therefore performed regional disparity studies comparing influenza warning reports from prefectures in Japan with current positive January 2020, suggesting that only S form entered those prefectures.', 'cite_spans': [], 'ref_spans': [], 'section': 'Clonal competition between L and S subtypes should have taken place in Wuhan and it'}, {'text': 'The maximum number of infected cases in Japan has been reported in . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Clonal competition between L and S subtypes should have taken place in Wuhan and it'}, {'text': 'The copyright holder for this preprint . https://doi.org/10.1101/2020.03.25.20043679 doi: medRxiv preprint Hokkaido. Fig. 1 suggests that Hokkaido was not or incompletely exposed to S type and the current epidemic is due to the L type. Mathematical modeling has shown that undocumented infections of SARS-CoV-2 were the infection source for most documented cases. 7 Because children infected with SARS-CoV-2 are asymptomatic 8 or mildly symptomatic, 4 Europe has become the center of the pandemic. But why mortality rates vary from country to country remains enigmatic. We arranged influenza epidemic curves of European countries in descending order of COVID-19 mortality (Supplementary Figs. 1 and 2). There seems to be a pattern in the curve related to COVID-19 case fatality rates (CFR). Depending on the likelihood of S-type SRS-CoV-2 transmission, a scoring system was developed to model the mortality-related curve patterns. As shown in Fig. 3 , this score correlates with the SARS-CoV-2 CFR in each country, implying that the extent of S type SARS-CoV-2 transmission determines the severity of the current infection. This scoring system is useful as a country-specific COVID-19 severity risk score. As shown in Fig. 4 , the risk score reveals geographic effects on the spread of the', 'cite_spans': [{'start': 364, 'end': 365, 'text': '7', 'ref_id': 'BIBREF6'}, {'start': 450, 'end': 451, 'text': '4', 'ref_id': 'BIBREF3'}], 'ref_spans': [{'start': 117, 'end': 123, 'text': 'Fig. 1', 'ref_id': 'FIGREF3'}, {'start': 687, 'end': 694, 'text': 'Figs. 1', 'ref_id': 'FIGREF3'}, {'start': 943, 'end': 949, 'text': 'Fig. 3', 'ref_id': 'FIGREF5'}, {'start': 1218, 'end': 1224, 'text': 'Fig. 4', 'ref_id': 'FIGREF6'}], 'section': '(which was not peer-reviewed)'}, {'text': 'Our epidemiological analyses provided evidence for herd immunity against the S type SARS-CoV-2 that exited Wuhan or other epicenter in China earlier than the L type.', 'cite_spans': [], 'ref_spans': [], 'section': 'DISCUSSION'}, {'text': 'Undocumented spread of SARS-CoV-2 can be monitored through influenza surveillance systems based on viral competition. The findings have led to the development of an epidemiological tool that predicts severity of COVID-19 in each region and its application in Europe.', 'cite_spans': [], 'ref_spans': [], 'section': 'DISCUSSION'}, {'text': 'Because the L type is likely to transmit faster than the S type, the basic reproduction number of the L type (R0 L ) is expected to be higher than the S type (R0 S ).', 'cite_spans': [], 'ref_spans': [], 'section': 'DISCUSSION'}, {'text': 'When herd immunity to the S type is established,, the proportion of the population with immunity (pS) is 1 -1/R0 S . 9 If R0 L = 2.2 and R0 S = 2.17, which are plausible values, 10 pL = 1 -1/R0 L = 0.55 (55%) of the population will be infected when a non-immune population is exposed to the L type. When a population with herd immunity to the S type is exposed to the L type, pL -pS = 1/R0 S -1/ R0 L = 0.006 (0.6%) will be infected, consistent with the positive rates of SARS-CoV-2 RT-PCR in some Japan prefectures (Fig, 1) . If 80% of a population is exposed to the S type and then the whole population is exposed to the L type, pL -0.8 x pS = 1 -1/R0 L -0.8 (1 -1/ R0 L ) = 0.114 (11.4%) is infected, consistent with the positive rates observed in Aichi and Hokkaido in Japan ( Fig. 1) . Thus, the spread of S type before the L type confers partial herd immunity to the L type.', 'cite_spans': [], 'ref_spans': [{'start': 516, 'end': 524, 'text': '(Fig, 1)', 'ref_id': 'FIGREF3'}, {'start': 781, 'end': 788, 'text': 'Fig. 1)', 'ref_id': 'FIGREF3'}], 'section': 'DISCUSSION'}, {'text': 'Until March 9, 2020, the Japanese government did not restrict entry of tourists from areas of China outside Hubei Province which might have unintentionally allowed selective influx of S form to Japan. S type influx into Ireland and the Balkans may be due to travel restrictions in those countries restricted to infected individuals. In contrast to other areas in Japan, epidemic is ongoing in Nagoya City in Aichi prefecture and Sapporo City in Hokkaido, where the spread of the S form may have been incomplete as discussed above. In the United States of America and most European countries, travel from across China was restricted from the beginning of February, blocking the inflow of the S form from China. There is deep concern that serious outbreaks of SARS-CoV-2 in the United States and Europe may have been caused by the spread of the L form preceding the S form, by which up to 55% of the population may be infected as calculate above.', 'cite_spans': [], 'ref_spans': [], 'section': 'DISCUSSION'}, {'text': 'China have been shown to be effective, 7 they incur massive costs such as economic losses. 11 Conversely, herd immunity policy will face substantial mortality among high risk population. 12, 13 We propose that a policy that expects herd immunity are dangerous in the areas indicated in brown, red, and magenta in Fig. 4 . In areas marked in yellow, policies that balance social closure and population immunity are desirable. In the areas marked in green, in contrast, no social isolation may be required except for individuals over 60 and at high risk, such as those with diabetes, coronary heart disease, high blood pressure, chronic obstructive pulmonary disease, cerebrovascular disease, and cancer. 12,13 Thus, with the use of the new epidemiological tool, governments may be able to predict SARS-CoV-2 morbidity and mortality in their countries and use them in public health policies. Applying this scoring system to small geographies should allow for finer-grained policy creation.', 'cite_spans': [{'start': 191, 'end': 193, 'text': '13', 'ref_id': None}], 'ref_spans': [{'start': 313, 'end': 319, 'text': 'Fig. 4', 'ref_id': 'FIGREF6'}], 'section': 'Although traditional public health outbreak response tactics implemented in'}, {'text': 'Quantitative analysis for SARS-CoV-2-specific IgM and IgG antibodies in the blood of healthy persons may reveal herd immunity against S type SARS-CoV-2. Such seroepidemiological data will predict the effectiveness and risk of herd immunity . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Although traditional public health outbreak response tactics implemented in'}, {'text': 'The copyright holder for this preprint . https://doi.org/10.1101/2020.03.25.20043679 doi: medRxiv preprint policies. Detection of \"S type SARS-CoV-2 herd immunity\" might also help schedule a timing to lift travel restrictions, event restraints, closure of schools, social distancing, and contact precautions. However, collection of sera and analyses require significant logistic effort, time, and cost, 6 which developing countries may not be able to cover.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'Moreover, to determine the normal range of the test, the test values in unaffected individuals must be determined. For diseases with many subclinical infections, such as SARS-CoV-2, it may be difficult to determine the cutoff value, 9 especially if the subclinical infection is prevalent in the majority of the population. Meanwhile, quick epidemiological tools should be useful for policy making that balances herd immunity approaches with containment policies. The approach used in this study will lay the foundation for further developing epidemiological tools to support political decisions on infection control.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'The copyright holder for this preprint . . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. ', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'A new coronavirus associated with human respiratory disease in China', 'authors': [{'first': 'F', 'middle': [], 'last': 'Wu', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Zhao', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Yu', 'suffix': ''}], 'year': 2020, 'venue': 'Nature', 'volume': '579', 'issn': '', 'pages': '265--274', 'other_ids': {}}, 'BIBREF1': {'ref_id': 'b1', 'title': 'A pneumonia outbreak associated with a new coronavirus of probable bat origin', 'authors': [{'first': 'P', 'middle': [], 'last': 'Zhou', 'suffix': ''}, {'first': 'X', 'middle': ['L'], 'last': 'Yang', 'suffix': ''}, {'first': 'X', 'middle': ['G'], 'last': 'Wang', 'suffix': ''}], 'year': 2020, 'venue': 'Nature', 'volume': '579', 'issn': '', 'pages': '270--273', 'other_ids': {}}, 'BIBREF2': {'ref_id': 'b2', 'title': 'A Novel Coronavirus from Patients with Pneumonia in China', 'authors': [{'first': 'N', 'middle': [], 'last': 'Zhu', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Wang', 'suffix': ''}], 'year': 2019, 'venue': 'N Engl J Med', 'volume': '382', 'issn': '', 'pages': '727--760', 'other_ids': {}}, 'BIBREF3': {'ref_id': 'b3', 'title': 'Characteristics of pediatric SARS-CoV-2 infection and potential evidence for persistent fecal viral shedding', 'authors': [{'first': 'Y', 'middle': [], 'last': 'Xu', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Zhu', 'suffix': ''}], 'year': 2020, 'venue': 'Nature Medicine', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF4': {'ref_id': 'b4', 'title': 'On the origin and continuing evolution of SARS-CoV-2', 'authors': [{'first': 'X', 'middle': [], 'last': 'Tang', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Wu', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Li', 'suffix': ''}], 'year': 2020, 'venue': 'National Science Review', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF5': {'ref_id': 'b5', 'title': 'Estimation of the asymptomatic ratio of novel coronavirus infections (COVID-19)', 'authors': [{'first': 'H', 'middle': [], 'last': 'Nishiura', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Kobayashi', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Suzuki', 'suffix': ''}], 'year': 2020, 'venue': 'Int J Infect Dis', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF6': {'ref_id': 'b6', 'title': 'Substantial undocumented infection facilitates the', 'authors': [{'first': 'R', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'Pei', 'middle': ['S'], 'last': 'Chen', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': '', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}}, 'ref_entries': {'FIGREF0': {'text': 'rate of RT-PCR for SARS-CoV-2. Only 15 prefectures were selected where more than 400 samples were examined because positive rates in other prefectures show Poisson distribution and are not reliable as an indicator of incidence. As shown in Fig. 1, the presence of sharp declines in influenza warning reports correlated with lower incidence of SARS-CoV-2 infection. From early November 2019 to 23 January 2020, about 1,840,000 tourists from China entered Japan, suggesting that unrecognized spread of S type SARS-CoV-2 interfered with influenza epidemic. Interestingly, the timing of decline suggests that S type SARS-CoV-2 spread from 23 December 2019 in Tokyo, Kanagawa, Saitama, and Hiroshima; from 20 January 2020 in Fukuoka and Niigata; from 27 January in Kyoto, Shizuoka, and Ibaraki; and from 2 February in Wakayama.No outbreak has been recognized in Japan before February 2020 and the Chinese government implemented blockades in Wuhan and other cities in Hubei Province on 23', 'latex': None, 'type': 'figure'}, 'FIGREF1': {'text': 'we surmise that children are major sources of SARS-CoV-2 spread. Hokkaido is divided into several General Promotion Bureau jurisdictions. Incidents of influenza-like illness in kindergartens reported from health centers in the jurisdictions were compared with current prevalence of SARS-CoV-2. As shown in Fig. 2, strong negative correlation was observed (Spearman correlation coefficient ρ= -0.73), suggesting that undocumented infection SARS-CoV-2 in Hokkaido suppressed influenza-like illness among small children. Influenza-like illnesses among older children did not reflect the epidemic (ρ= -0.14 in elementary school, 0.07 in junior high school, and -0.04 in high school), possibly because influenza-like symptoms of COVID-19 in these age groups 4 confounded the analyses.', 'latex': None, 'type': 'figure'}, 'FIGREF2': {'text': 'Figure legends', 'latex': None, 'type': 'figure'}, 'FIGREF3': {'text': 'SARS-CoV-2 Trend Curves in Japan Prefectures.The number of influenza warning reports from November 11, 2019 to March 8, 2020has been plotted. The positive rate (%) of RT-PCR for SARS-CoV-2 as of March 16, 2020 is shown for each prefecture.', 'latex': None, 'type': 'figure'}, 'FIGREF4': {'text': 'Correlation between Influenza in Kindergartens and SARS-CoV-2Infection.The correlation diagram shows the prevalence (%) of influenza-like illness in kindergartens in each district in Hokkaido and the number of individuals who were RT-PCR positive for SARS-CoV-2. Spearman correlation coefficient ρ = -0.73.', 'latex': None, 'type': 'figure'}, 'FIGREF5': {'text': 'Correlation between the Risk Score and COVID-19 Mortality.Risk scores and case fatality rate (CFR) of COVID-19 in European countries calculated from data in the Worldometer were plotted. Spearman correlation coefficient ρ = 0.65.', 'latex': None, 'type': 'figure'}, 'FIGREF6': {'text': 'European Map.Distribution of risk scores of various strengths. Data on influenza epidemics were not available for blank areas.', 'latex': None, 'type': 'figure'}, 'TABREF0': {'text': 'Epidemiological data were obtained from the Worldometer (https://www.worldometers.info/coronavirus/), the COVID Tracking Project(https://covidtracking.com/data/), and websites operated by Su Wei (https://covid-2019.live/), the Infectious Disease Surveillance Center, National Institute of Infectious Diseases, Japan (https://nesid4g.mhlw.go.jp/Hasseidoko/Levelmap/flu/2019_2020/trend.html), the Hokkaido government (http://www.pref.hokkaido.lg.jp/hf/kth/kak/hasseijoukyou.htm), and the World Health Organization (https://app.powerbi.com/view?r=eyJrIjoiZTIxMzAwMzYtZWE4NC00YTU2LWE3M', 'latex': None, 'type': 'table'}, 'TABREF1': {'text': '(which was not peer-reviewed) The copyright holder for this preprint. https://doi.org/10.1101/2020.03.25.20043679 doi: medRxiv preprint Table1. Epidemic Data and Virus Subtypes. Number of cases, the number of death, CFR (%): case fatality rate, virus subtype, %S (the % of S type virus), and epidemic curve are shown according to countries.', 'latex': None, 'type': 'table'}}, 'back_matter': []}\n",
      "{'paper_id': 'acb0e11e5763dbf7588ec435751a2ab705a783f1', 'metadata': {'title': 'Insights into Cross-species Evolution of Novel Human Coronavirus 2019-nCoV and Defining Immune Determinants for Vaccine Development', 'authors': [{'first': 'Arunachalam', 'middle': [], 'last': 'Ramaiah', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of California', 'location': {'postCode': '92697', 'settlement': 'Irvine', 'region': 'CA', 'country': 'United States'}}, 'email': ''}, {'first': 'Vaithilingaraja', 'middle': [], 'last': 'Arumugaswami', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of California', 'location': {'postCode': '90095', 'settlement': 'Los Angeles', 'region': 'CA', 'country': 'United States'}}, 'email': ''}]}, 'abstract': [{'text': 'Novel Coronavirus (nCoV) outbreak in the city of Wuhan, China during December 2019, has now spread to various countries across the globe triggering a heightened containment effort. This human pathogen is a member of betacoronavirus genus carrying 30 kilobase of single positive-sense RNA genome. Understanding the evolution, zoonotic transmission, and source of this novel virus would help accelerating containment and prevention efforts. The present study reported detailed analysis of 2019-nCoV genome evolution and potential candidate peptides for vaccine development. This nCoV genotype might have been evolved from a bat-CoV by accumulating non-synonymous mutations, indels, and recombination events. Structural proteins Spike (S), and Membrane (M) had extensive mutational changes, whereas Envelope (E) and Nucleocapsid (N) proteins were very conserved suggesting differential selection pressures exerted on 2019-nCoV during evolution. Interestingly, 2019-nCoV Spike protein contains a 39 nucleotide sequence insertion relative to SARSlike bat-SL-CoVZC45/2017. Furthermore, we identified eight high binding affinity (HBA) CD4 T-cell epitopes in the S, E, M and N proteins, which can be commonly recognized by HLA-DR alleles of Asia and Asia-Pacific Region population. These immunodominant epitopes can be incorporated in universal subunit CoV vaccine. Diverse HLA types and variations in the epitope binding affinity may contribute to the wide range of immunopathological outcomes of circulating virus in humans. Our findings emphasize the requirement for continuous surveillance of CoV strains in live animal markets to better understand the viral adaptation to human host and to develop practical solutions to prevent the emergence of novel pathogenic CoV strains.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}], 'body_text': [{'text': '. CC-BY-NC 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.29.925867 doi: bioRxiv preprint Current 2019 novel Coronavirus (2019-nCoV) outbreak in China and its subsequent spread to multiple countries across four continents precipitated a major global health crisis. Uncovering the 2019-nCoV genome evolution and immune determinants of its proteome would be crucial for outbreak containment and preventative efforts.', 'cite_spans': [], 'ref_spans': [], 'section': 'INTRODUCTION'}, {'text': 'Coronaviruses are single positive-stranded RNA viruses belonging to Coronaviridae family 1 , which are divided into four genera: Alpha-, Beta-, Delta-and Gammacoronavirus. CoVs are commonly found in many species of animals, including bats, camels and humans. Occasionally, the animal CoVs can acquire genetic mutations by errors during genome replication or recombination mechanism, which can further expand their tropism to humans. The first human CoVs were discovered in the mid-1960s. A total of six human CoV types were identified to be responsible for causing human respiratory ailments, which include two alpha CoVs (229E; NL63), and four beta CoVs (OC43; HKU1; SARS-CoV; MERS-CoV) 2 . Typically, these CoVs cause asymptomatic infection or severe acute respiratory illness, including fever, cough, and shortness of breath. However, other symptoms such as gastroenteritis and neurological diseases of varying severity have also been reported [3] [4] [5] .', 'cite_spans': [{'start': 947, 'end': 950, 'text': '[3]', 'ref_id': None}, {'start': 951, 'end': 954, 'text': '[4]', 'ref_id': 'BIBREF2'}, {'start': 955, 'end': 958, 'text': '[5]', 'ref_id': 'BIBREF3'}], 'ref_spans': [], 'section': 'INTRODUCTION'}, {'text': 'The recent two CoV human outbreaks were caused by betacoronaviruses, SARS-CoV The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.29.925867 doi: bioRxiv preprint While the 2019-CoV appears to be a zoonotic infectious disease, the specific animal species and other reservoirs need to be clearly defined. Identification of the source animal species for this outbreak would facilitate global public health authorities to inspect the trading route and the movement of wild and domestic animals to Wuhan and taking control measures to limit the spread of this disease 7 . The 2019-nCoV outbreak has a fatality rate of 2% with significant impact on global health and economy. As of 29 January 2020, the 2019-nCoV has spread to fifteen countries including Asia-Pacific region (Thailand, Japan, The Republic of Korea, Malaysia, Cambodia, Sri Lanka, Nepal, Vietnam, Singapore, and Australia), United Arab Emirates, France, Germany, Canada and the United States with 6065 confirmed cases, in which 5997 reported from China 8 .', 'cite_spans': [], 'ref_spans': [], 'section': 'INTRODUCTION'}, {'text': 'Here, we report a detailed genetic analysis of 2019-nCoV evolution based on 47 genome sequences available in Global Initiative on Sharing All Influenza Database (GISAID) (https://www.gisaid.org/) (Supplemental Table 1 ) and one genome sequence from GenBank. We further defined potential epitope targets in the structural proteins (SPs) for vaccine design.', 'cite_spans': [], 'ref_spans': [{'start': 210, 'end': 217, 'text': 'Table 1', 'ref_id': None}], 'section': 'INTRODUCTION'}, {'text': 'Phylogenetic tree construction. We employed the following methodologies for genetic analyses. The genome sequences of 2019-nCoV were obtained on 29 January 2020 along with genetically related 10 SARS-CoV/SARS-CoV-like virus genome sequences that we identified through the BLASTN search in NCBI database. We also included 10 sequences of MERS-CoV strains and one outgroup alphacoronavirus sequence. 2019-nCoV genome (30 Kb in size) codes for a large non-structural polyprotein ORF1ab that is proteolytically cleaved to generate 15/16 proteins, and four structural proteins -Spike (S), Envelope (E), Membrane (M) and Nucleocapsid (N), as well as five accessory proteins (ORF3a, ORF6, ORF7a, ORF8 and ORF10). The genome sequences were aligned using the MAFFTv7 program 9 , which were exported to MEGA-X 10 for Neighbor-Joining (NJ) phylogenetic tree construction with 1000 bootstrap support and genetic . CC-BY-NC 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.29.925867 doi: bioRxiv preprint distance calculations. Structural genes/proteins were aligned with closely related sequences using Clustal Omega Program 11 .', 'cite_spans': [], 'ref_spans': [], 'section': 'MATERIALS AND METHODS'}, {'text': 'High Binding Affinity (HBA) epitope identification. In order to identify the potential target peptides for vaccine design, the structural protein sequences of highly conserved and annotated representative 2019-nCoV strain Wuhan-Hu-1 (MN908947.3) were used to predict the CD4 T-cell epitopes (TCEs) with the Immune Epitope Database and Analysis Resource (IEDB) consensus tool by IEDB recommended prediction method 12, 13 . To identify the most immunodominant (IMD) CD4 epitopes, we employed the computational approach described previously 14 . For all the possible 15-mer peptides from the SPs, the binding affinity was predicted using six most prevalent Human Leukocyte Antigen (HLA; MHC Class-II DR) alleles (http://allelefrequencies.net/) that cover most of the ethnic population in China (PRC), Thailand (TH) and Japan (JPN) (Supplemental Table 2 ). Since, the prevalence of dominant HLA-DR types may vary in human populations from different countries, the prevalent six HLA-DR alleles covering Asia-Pacific region 14 and Global population data sets have been used for binding affinity predictions to assess the generality of those predicted epitopes with high binding affinities (HBA). We calculated the average binding affinity score for each predicted 15mer peptides against all the predominant HLA-DR types from each country/region using a sliding window approach ( Figure 2 ). This approach is more efficient as the epitopes predicted with average HBA (IC <=10 nM), indicate that those epitopes are more likely to bind and enhance a cellular immune response in the studied population. (Table 1C) . Whereas, M protein shared 98.6% identity with one amino acid insertion (position 4, S) and N protein showed 99% identity with 4 non-synonymous changes (P37S, S215G, S243G, Q267A).', 'cite_spans': [{'start': 413, 'end': 416, 'text': '12,', 'ref_id': 'BIBREF9'}, {'start': 417, 'end': 419, 'text': '13', 'ref_id': 'BIBREF10'}], 'ref_spans': [{'start': 842, 'end': 849, 'text': 'Table 2', 'ref_id': None}, {'start': 1373, 'end': 1381, 'text': 'Figure 2', 'ref_id': 'FIGREF2'}, {'start': 1593, 'end': 1603, 'text': '(Table 1C)', 'ref_id': 'TABREF1'}], 'section': 'MATERIALS AND METHODS'}, {'text': 'Envelope protein shares 100% identity.', 'cite_spans': [], 'ref_spans': [], 'section': 'RESULTS AND DISCUSSION'}, {'text': 'The genome alignment between the novel and bat (bat/Yunnan/RaTG13/2013) coronaviruses showed 2 of 7 insertions (Table 1A) In summary, the present study provided a detailed genetic analysis of 2019-nCoV genome evolution and potential universal epitopes for subunit vaccine development.', 'cite_spans': [], 'ref_spans': [{'start': 111, 'end': 121, 'text': '(Table 1A)', 'ref_id': 'TABREF0'}], 'section': 'RESULTS AND DISCUSSION'}, {'text': '. CC-BY-NC 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.29.925867 doi: bioRxiv preprint This 2019-nCoV strain might be evolved from the bat-CoV by accumulating favorable genetic changes for human infection. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.29.925867 doi: bioRxiv preprint Table 1 -Summary of authors, source laboratories and metadata from GISAID for 2019-nCoV genomes analyzed in this study. Table 2 -Amino acid sequences of high confidence mean HBA (IC <= 10nM) CD4 T-cell epitopes from functionally important regions of the four structural proteins (S, E, M and N). The details of protein name, starting and ending positions of epitopes in the given protein, and epitope sequences are provided. The unique epitope sequences for each protein and the universal epitopes commonly recognized by HLA-DR alleles of all five population sets (highlighted in red) are listed at the end of each table.', 'cite_spans': [], 'ref_spans': [{'start': 484, 'end': 491, 'text': 'Table 1', 'ref_id': None}, {'start': 604, 'end': 611, 'text': 'Table 2', 'ref_id': None}], 'section': 'RESULTS AND DISCUSSION'}, {'text': '. CC-BY-NC 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.29.925867 doi: bioRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'Supplemental'}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'Clinical features of patients infected with 2019 novel coronavirus in Wuhan', 'authors': [{'first': '', 'middle': [], 'last': 'Huang', 'suffix': ''}], 'year': 2020, 'venue': 'China. The Lancet', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1016/S0140-6736(20)30183-5']}}, 'BIBREF2': {'ref_id': 'b2', 'title': 'Isolation and characterization of a bat SARS-like coronavirus that uses the ACE2 receptor', 'authors': [{'first': 'X', 'middle': ['Y'], 'last': 'Ge', 'suffix': ''}], 'year': 2013, 'venue': 'Nature', 'volume': '503', 'issn': '', 'pages': '535--538', 'other_ids': {}}, 'BIBREF3': {'ref_id': 'b3', 'title': 'Molecular mechanisms of coronavirus RNA capping and methylation', 'authors': [{'first': 'Y', 'middle': [], 'last': 'Chen', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Guo', 'suffix': ''}], 'year': 2016, 'venue': 'Virol. Sin', 'volume': '31', 'issn': '', 'pages': '3--11', 'other_ids': {}}, 'BIBREF4': {'ref_id': 'b4', 'title': 'The continuing 2019-nCoV epidemic threat of novel coronaviruses to global health -The latest 2019 novel coronavirus outbreak in Wuhan', 'authors': [{'first': 'D', 'middle': ['S'], 'last': 'Hui', 'suffix': ''}], 'year': 2020, 'venue': 'China. Int. J. Infect. Dis', 'volume': '91', 'issn': '', 'pages': '264--266', 'other_ids': {}}, 'BIBREF5': {'ref_id': 'b5', 'title': 'WHO. The Situation Report -9', 'authors': [], 'year': 2020, 'venue': '', 'volume': '29', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF6': {'ref_id': 'b6', 'title': 'MAFFT online service: multiple sequence alignment, interactive sequence choice and visualization', 'authors': [{'first': 'K', 'middle': [], 'last': 'Katoh', 'suffix': ''}], 'year': 2019, 'venue': 'Brief. Bioinform', 'volume': '20', 'issn': '', 'pages': '1160--1166', 'other_ids': {}}, 'BIBREF7': {'ref_id': 'b7', 'title': 'MEGA X: Molecular Evolutionary Genetics Analysis across computing platforms', 'authors': [{'first': 'S', 'middle': [], 'last': 'Kumar', 'suffix': ''}], 'year': 2018, 'venue': 'Mol. Biol. Evol', 'volume': '35', 'issn': '', 'pages': '1547--1549', 'other_ids': {}}, 'BIBREF8': {'ref_id': 'b8', 'title': 'Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega', 'authors': [{'first': 'F', 'middle': [], 'last': 'Sievers', 'suffix': ''}], 'year': 2011, 'venue': 'Mol. Syst. Biol', 'volume': '7', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF9': {'ref_id': 'b9', 'title': 'A systematic assessment of MHC class II peptide binding predictions and evaluation of a consensus approach', 'authors': [{'first': 'P', 'middle': [], 'last': 'Wang', 'suffix': ''}], 'year': 2008, 'venue': 'PLoS Comput. Biol', 'volume': '4', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF10': {'ref_id': 'b10', 'title': 'The immune epitope database (IEDB) 3.0. Nucl. Acids Res', 'authors': [{'first': 'R', 'middle': [], 'last': 'Vita', 'suffix': ''}], 'year': 2014, 'venue': '', 'volume': '43', 'issn': '', 'pages': '405--412', 'other_ids': {}}}, 'ref_entries': {'FIGREF0': {'text': 'Severe Acute Respiratory Syndrome-CoV) and MERS-CoV (Middle East Respiratory Syndrome-CoV). In 2002, SARS-CoV outbreak was first reported in China, which spread globally causing hundreds of deaths with a 11% mortality rate. In 2012, MERS-CoV was first recognized in Saudi Arabia and subsequently in other countries, which has a fatality rate of 37%. Since December 2019, the seventh novel human betacoronavirus (2019-nCoV) causing pneumonia outbreak was reported by the World Health Organization. This novel virus is linked with an outbreak of febrile respiratory illness in the city of Wuhan, Hubei Province, China with an epidemiological association to the Huanan Seafood Wholesale Market selling live farm and wild animals', 'latex': None, 'type': 'figure'}, 'FIGREF1': {'text': 'Phylogenetic analysis reveals 2019-nCoV closely related to Bat CoV. Genome based phylogenetic tree analysis revealed a major cluster of large clade was formed with humans 2019-nCoV strains, SARS-CoV-like strains from bats and human SARS-CoV viruses (Figure 1A) with 100 bootstrap support. All 2019-nCoV strains were clustered together with 96% bootstrap support, but separated from bat/bat-SARS like human SARS-CoVs clusters, indicating that 2019-nCoV belongs to a novel genotype. A small cluster was formed by MERS-CoV strains, which was genetically unrelated to the strains circulating in the current outbreak. This result indicated that the causative agent of current 2019-2020 outbreak is a betacoronavirus that is genetically and evolutionarily related to bat CoVs: bat/Yunnan/RaTG13/2013 and bat/SARS-CoVlike-viruses. Moreover, the result suggests that the evolutionary pressures exerted on 2019-nCoV, as well as bat and human SARS-coronaviruses were different. Interestingly, strains from the current outbreak were clearly segregated from bat SARS-CoV-like viruses (isolated in 2015 and 2017), but located adjacent to the bat/Yunnan/RaTG13/2013 virus isolated in 2013. Besides, both 2019-nCoV and bat CoV clusters share a common ancestor (square), which is supported by 92% bootstrap replications in the phylogeny. The ancestors of both human SARS-CoV and 2019-nCoV share a common ancestor. Overall our data indicates that the 2019-nCoV is not closely clustering with either human SARS-CoV or bat-SARS-CoV like strains but shares a common ancestor. The mean group genetic distance among 2019-nCoV strains was zero, indicating high genetic conservation among currently circulating viruses. Similar findings were observed in human SARS and MERS isolates, but bat SARS-like isolates showed a genetic distance of 0.04. The current circulating strains have close genetic associations with the 2013 bat CoV, bat/Yunnan/RaTG13/2013 (EPI-ISL-402131). The 2019-nCoV Wuhan-Hu-1 and bat/Yunnan/RaTG13/2013 strains share 96% sequence identity (genetic distance 0.04) with 1141 variable nucleotide substitutions and 48 INDELs (Figure 1C and D; Table 1A). While, bat-SL-CoVZC45/2017 virus and SARS human-BJ182-12/2008 virus from other clusters had 88% and 80% sequence identity (distance 0.23 and 0.46), 3542 and 5879 nucleotide substitutions and 137 and 535 INDELs, respectively with 2019-nCoV Wuhan-Hu-1, suggesting that bat/Yunnan/RaTG13/2013 strain is likely the most recent ancestor of currently circulating 2019-nCoV strains. This most recent ancestral CoV strain from bat species underwent significant evolution during 2013 to 2019 possibly by genetic shift before spreading to humans (Figure 1B).However, it is likely that CoVs from other animal species may also be a source for this', 'latex': None, 'type': 'figure'}, 'FIGREF2': {'text': \"in Spike gene, whereas the alignment of 2019-nCoV with distantly related SARS-like bat-SL-CoVZC45/2017 showed 9 of 15 insertions and 2 of 5 deletions (Figure 1C; Table 1B) in Spike gene. Interestingly, the Spike gene has a larger 39 nucleotide insertion (5'-aAT GGT GTT GAA GGT TTT AAT TGT TAC TTT CCT TTA CAA Tca-3') at genomic location 23022. Our alignment result suggests a possible occurrence of recombination and nonsynonymous changes in Spike gene of 2019-nCoV which can alter the preference of viral attachment to host receptor. The non-synonymous changes in Nucleocapsid gene likely differentially regulate its binding to virus RNA genome and viral replication. Taken together, our findings implicate that overall genetic differences facilitated the novel coronavirus to adapt to humans from bat via altered cell receptor binding, changes in viral genome replication and better evading host immune detection. Epitopes in 2019-CoV structural proteins are differentially recognized by HLA-DR alleles. Host immune responses orchestrated by HLA molecules determine the outcome of infectious diseases. Antigen presentation by class II MHC isoforms play a critical role in inducing CD4+ Th1 and Th2 cell-mediated immune responses. It is important to understand the nature of immune selection pressure and evolution of immune escape viral mutants. Since the current outbreak is originated from Wuhan city of China, we set out to study the 2019-nCoV epitopes displayed by HLA-DR alleles predominantly present in the ethnic populations of China (PRC), Thailand (TH), Japan (JPN) and Asia-Pacific Region (APR). Thus, we calculated the average binding affinity of 134 unique average HBA TCEs were identified in 16 islands/hotspots across the structural protein sequences (Figure 2; Supplemental Table 2). The lowest and highest numbers of average HBA TCEs were identified in nucleocapsid protein (n=11), and spike protein (n=78), respectively. This result reveals that mutations in spike protein may contribute to changes in viral virulence observed in 2019-nCoV. The overall binding affinity patterns of HLA-DR alleles from different countries were similar, however there are some exceptions: i) no HBA epitope was identified in nucleocapsid protein for TH, JPN and APR dominant HLA-DR allele, while this protein exhibited 3 and 8 TCEs for PRC and Global population set alleles, and ii) HLA-DR alleles for APR and TH recognized lower (n=15) and higher (n=65) number of HBA epitopes, respectively. Nevertheless, delineating common epitopes is crucial for designing universally potent subunit vaccine. Thus, we investigated common epitopes that are recognized by all dominant HLA-DR alleles prevalent in the five ethnic populations. Comprehensively, 8 unique HBA epitopes distributed across Spike (n=2; 232-GINITRFQTLLALHR-246; and 233-INITRFQTLLALHRS-247), Envelope (n=3; 55-SFYVYSRVKNLNSSR-69; 56-FYVYSRVKNLNSSRV-70; and 57-YVYSRVKNLNSSRVP-71) and Membrane (SupplementalTable 2). The high affinity epitopes identified in the functionally important regions of Spike and Membrane proteins could be critical in modulating immune responses to circulating 2019-nCoV. Many of the low binding affinity regions in the structural proteins may favor viruses to evade host antiviral immunity. Our analysis suggests that a subunit vaccine containing the eight immunodominant HLA-DR epitopes can induce effective antiviral Tcell and antibody responses in different ethnic populations.\", 'latex': None, 'type': 'figure'}, 'FIGREF3': {'text': 'Phylogenetic relationships of novel 2019-nCoV viral strains and the possible route of viral evolution and disease transmission. (A) NJ bootstrap consensus tree was reconstructed based on the genome sequences with 1000 bootstrap replications. 2019-nCoV strains (n=48 including one bat strain) from current outbreak, SARS-like-CoV (n=2), SARS-CoV (n=8) and MERS-CoV (n=10) viruses are colored in red, orange, yellow and green, respectively. The alphacoronavirus (MN056942) was used as an outgroup control (blue). (B) Evolution of a 2019-nCoV virus from bat/unknown animals to humans and spread among humans. (Yunnan/RaTG13/2013 (EPI-ISL-402131) (left), or bat SARS-CoV-like (MG772933) viruses (middle); or human SARS-CoV (EU371564) (right). All these viruses were reported from China. (D) The proportion of conserved, variable (Single Nucleotide Polymorphism) and INDELs in 2019-nCoV was uncovered by aligning it to bat-CoV (left), bat SARS-CoV-like (middle) and human SARS-CoV (right).', 'latex': None, 'type': 'figure'}, 'FIGREF4': {'text': \"Comparison of 2019-nCoV structural proteins' epitopes with mean HBA to predominant HLA-DR alleles from China (PRC), Thailand (TH), Japan (JPN), Asia-Pacific region (APR) and Global population. The mean binding affinity score was calculated from the binding affinity (score) of each predicted 15-mer HBA peptides (sliding window approach) from four structural proteins with six predominant HLA-DR types from China (PRC), Thailand (TH), Japan (JPN), Asia-Pacific region (APR) and Global population sets. Functional islands with mean HBA TCEs (IC <= 10nM) in the structural proteins were located in the horizontal orange color. The epitopes having Table 1 -Details of the insertions and deletions (INDELs) in the genome of 2019-nCoV (MN908947.3) relative to A) bat/Yunnan/RaTG13/2013 (GISAID #EPI_ISL_402131) and B) bat-SL-CoVZC45/2017 (MG772933.1) viruses. C) List of non-synonymous changes and INDELs in the four structural proteins of 2019-nCoV relative to bat/Yunnan/RaTG13/2013.\", 'latex': None, 'type': 'figure'}, 'TABREF0': {'text': 'BY-NC 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.29.925867 doi: bioRxiv preprint Table 1B.', 'latex': None, 'type': 'table'}, 'TABREF1': {'text': 'Protein Non-synonymous / amino acid change Insertion position and amino acids S S32F, L50S, I76T, P218Q, D324E, T346R, T372A, T403R, K439N, H440N, I441L, A443S, E445V, F449Y, A459S, K478T, Q483V, T484E, L486F, Y490F, Y493Q, R494S, Y498Q, D501N, H505Y, N519H, A604T, S1125N, I1228V BY-NC 4.0 International license author/funder. It is made available under aThe copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.29.925867 doi: bioRxiv preprint SUPPLEMENTAL TABLES', 'latex': None, 'type': 'table'}}, 'back_matter': [{'text': 'We gratefully acknowledge the authors, the originating and submitting Laboratories for their sequence and metadata shared through GISAID and GenBank, on which this research is based.', 'cite_spans': [], 'ref_spans': [], 'section': 'ACKNOWLEDGEMENTS'}, {'text': 'The authors declare that they have no conflict of interest.', 'cite_spans': [], 'ref_spans': [], 'section': 'CONFLICT OF INTEREST'}]}\n",
      "{'paper_id': 'acd5cb2ba08da5c0f6bc310d74d497d38aa86be2', 'metadata': {'title': 'Enhanced replication of mouse adenovirus type 1 following virus-induced degradation of protein 3 kinase R (PKR) 4 5 Running Title: MAV-1 degrades PKR during infection 6 7', 'authors': [{'first': 'Danielle', 'middle': ['E'], 'last': 'Goodman', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Michigan', 'location': {'settlement': 'Ann Arbor', 'country': 'USA'}}, 'email': ''}, {'first': 'Carla', 'middle': ['D'], 'last': 'Pretto', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Michigan', 'location': {'settlement': 'Ann Arbor', 'country': 'USA'}}, 'email': ''}, {'first': 'Tomas', 'middle': ['A'], 'last': 'Krepostman', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Michigan', 'location': {'settlement': 'Ann Arbor', 'country': 'USA'}}, 'email': ''}, {'first': 'Kelly', 'middle': ['E'], 'last': 'Carnahan', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Michigan', 'location': {'settlement': 'Ann Arbor', 'country': 'USA'}}, 'email': ''}, {'first': 'Katherine', 'middle': ['R'], 'last': 'Spindler', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Michigan', 'location': {'settlement': 'Ann Arbor', 'country': 'USA'}}, 'email': ''}]}, 'abstract': [{'text': 'word count: 225 22 Importance word count: 144 23 . CC-BY-NC-ND 4.0 International license is made available under a', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}], 'body_text': [{'text': 'Protein kinase R (PKR) plays a major role in activating host immunity during infection 25 by sensing dsRNA produced by viruses. Once activated by dsRNA, PKR phosphorylates the 26 translation factor eIF2α, halting cellular translation. Many viruses have methods of inhibiting 27 PKR activation or its downstream effects, circumventing protein synthesis shutdown. These 28 include sequestering dsRNA or producing proteins that bind to and inhibit PKR activation. Here 29 we describe our finding that in multiple cell types, PKR was depleted during mouse adenovirus 30 type 1 (MAV-1) infection. MAV-1 did not appear to be targeting PKR at a transcriptional or 31 translational level because total PKR mRNA levels and levels of PKR mRNA bound to 32 polysomes were unchanged or increased during MAV-1 infection. However, inhibiting the 33 proteasome reduced the PKR depletion seen in MAV-1-infected cells, whereas inhibiting the 34 lysosome had no effect. This suggests that proteasomal degradation alone is responsible for PKR 35 degradation during MAV-1 infection. Time course experiments indicate that the degradation 36 occurs early after infection. Infecting cells with UV-inactivated virus prevented PKR 37 degradation, whereas inhibiting viral DNA replication did not. Together these results suggest that 38 an early viral gene is responsible. Degradation of PKR is a rare mechanism to oppose PKR 39 activity, and it has only been described in six RNA viruses. To our knowledge, this is the first 40 example of a DNA virus counteracting PKR by degrading it. 41 42 Importance 43', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract 24'}, {'text': 'The first line of defense in cells during viral infection is the innate immune system, which 44 is activated by different viral products. PKR is a part of this innate immune system and is 45 induced by interferon and activated by dsRNA produced by DNA and RNA viruses. PKR is such 46 discovered that the PKR -/-MEF lines used are not complete PKR knockouts (52). There are two 116 categories of PKR -/-MEFs derived from knockout mice: N-PKR -/-MEFs and C-PKR -/-MEFs 117 (52). The PKR -/-MEFs derived from mice created in the Weissmann lab (53) are designated N-118 PKR -/-MEFs, because the C-terminal fragment of PKR is still expressed and can be detected by 119 immunoblot when there is IFN induction (52). The fragment has the kinase catalytic activity of 120 PKR, but it does not bind dsRNA (52). The PKR -/-MEFs derived from mice created in the Bell 121 lab (54) are designated as C-PKR -/-MEFs, because the N-terminal fragment of PKR is still 122 expressed and can be detected by immunoblot with specific PKR antibodies (52). The fragment 123 is catalytically inactive, but it can still bind dsRNA (52). Susceptibility of these PKR -/-MEFs to 124 specific viruses may be dependent on the PKR mutation and the mechanism used by each virus 125 to circumvent PKR. 126', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract 24'}, {'text': 'To determine whether PKR plays an important role during MAV-1 infection, we tested 128 the susceptibility of both PKR -/-MEF lines to MAV-1 infection. We infected wild type MEFs, 129 N-PKR -/-MEFs, and C-PKR -/-MEFs with MAV-1 at an MOI of 1 PFU/cell and collected cell 130 pellets at 48 and 72 hpi. DNA was purified from the cell pellets and analyzed for MAV-1 131', 'cite_spans': [], 'ref_spans': [], 'section': '127'}, {'text': 'To determine whether MAV-1 affects PKR during infection, we infected several cell 139 types and analyzed PKR protein expression. We infected immortalized C57BL/6 MEFs, 140 C57BL/6 primary peritoneal macrophages, and CMT93 cells (mouse rectal carcinoma cells) with 141 MAV-1 at an MOI of 10 and collected cell lysates 24, 48, and 72 hpi. We analyzed cell lysates 142', 'cite_spans': [], 'ref_spans': [], 'section': 'Mouse PKR is depleted during MAV-1 infection. 138'}, {'text': 'for the presence of PKR by immunoblot using a polyclonal antibody that detects mouse PKR. 143', 'cite_spans': [], 'ref_spans': [], 'section': 'Mouse PKR is depleted during MAV-1 infection. 138'}, {'text': 'We probed blots with antibodies to actin as a loading control. To our surprise, in C57BL/6 144 MEFs, PKR was almost completely depleted from lysates 24 hpi and remained depleted through 145 72 hpi ( Fig. 2A and B) . PKR was also depleted compared to mock infection at 24 and 48 hpi in 146 C57BL/6 MEFs infected at MOI 2 and 5 (Fig. S1 ). We also observed depletion of PKR in other 147 cell types. In CMT93 cells, PKR was nearly undetectable at 24 hpi ( Fig. 2A) . In C57BL/6 148 primary peritoneal macrophages, PKR was decreased at 48 hpi compared to mock lysates, and 149 absent in infected lysates at 72 hpi ( Fig. 2A ). This indicates that MAV-1 causes PKR depletion 150 during infection. 151', 'cite_spans': [], 'ref_spans': [{'start': 199, 'end': 213, 'text': 'Fig. 2A and B)', 'ref_id': None}, {'start': 326, 'end': 334, 'text': '(Fig. S1', 'ref_id': None}, {'start': 453, 'end': 461, 'text': 'Fig. 2A)', 'ref_id': None}, {'start': 612, 'end': 619, 'text': 'Fig. 2A', 'ref_id': None}], 'section': 'Mouse PKR is depleted during MAV-1 infection. 138'}, {'text': 'To determine whether kinase activity of PKR is important for the depletion, we assayed 153 infection of MEFs expressing a mutant form of mouse PKR with a point mutation in the kinase 154 domain (K271R) (55). These cells, designated K271R SV40-MEFs, showed an even more rapid 155 depletion of PKR than in WT SV40-MEFs (Fig. 2C) . At 24 hpi, in K271R SV40-MEFs, 28% of 156 PKR remained, compared to 60% in the WT SV40-MEFs. The fraction remaining at 72 hpi in K271R SV40-MEFs suggests that the upper band of the PKR doublet usually seen in wild-type 161 cells is a phospho-PKR band, because only the lower band of the PKR doublet is seen in kinase-162 dead mutant K271R SV40-MEFs. The data in Fig. 2A and C thus indicate that both PKR and 163 phospho-PKR are depleted during MAV-1 infection. 164', 'cite_spans': [], 'ref_spans': [{'start': 317, 'end': 326, 'text': '(Fig. 2C)', 'ref_id': None}, {'start': 691, 'end': 698, 'text': 'Fig. 2A', 'ref_id': None}], 'section': '152'}, {'text': 'To determine the mechanism of PKR depletion, we first assayed whether reduction in 167 PKR protein during MAV-1 infection was due to reduced PKR mRNA steady-state levels. We 168 mock-infected or infected C57BL/6 MEFs and primary peritoneal macrophages at an MOI of 10 169 and collected cell lysates at 24, 48, and 72 hpi. We synthesized cDNA from RNA purified from 170 these cell lysates and assayed for PKR mRNA by qPCR. In C57BL/6 MEFs, PKR mRNA levels 171', 'cite_spans': [], 'ref_spans': [], 'section': 'MAV-1 does not cause PKR depletion by reducing steady-state levels of PKR mRNA 166'}, {'text': 'were similar between mock and infected lysates at 24 hpi (Fig. 3A) , a time point in which PKR 172 protein levels were already greatly depleted in the infected lysates compared to mock lysates 173 (Fig. 2) . Although PKR mRNA levels were depleted 33% at 48 hpi and 40% 72 hpi in MAV-1-174', 'cite_spans': [], 'ref_spans': [{'start': 57, 'end': 66, 'text': '(Fig. 3A)', 'ref_id': 'FIGREF17'}, {'start': 197, 'end': 205, 'text': '(Fig. 2)', 'ref_id': None}], 'section': 'MAV-1 does not cause PKR depletion by reducing steady-state levels of PKR mRNA 166'}, {'text': 'infected lysates compared to mock lysates, this does not correlate to the 84% and 94% reduction, 175 respectively, in PKR protein levels at those time points (Fig. 2B ). In C57BL/6 primary peritoneal 176 macrophages, PKR mRNA levels in the infected lysates were 2-3 times higher than levels in 177 mock lysates at all three time points assayed (Fig. 3B) , even though PKR protein levels were 178 almost completely depleted in infected lysates at 72 hpi (Fig. 2) . This is evidence that MAV-1 is 179 not causing PKR protein depletion by reducing PKR steady-state mRNA levels during infection. 180', 'cite_spans': [], 'ref_spans': [{'start': 158, 'end': 166, 'text': '(Fig. 2B', 'ref_id': None}, {'start': 344, 'end': 353, 'text': '(Fig. 3B)', 'ref_id': 'FIGREF17'}, {'start': 453, 'end': 461, 'text': '(Fig. 2)', 'ref_id': None}], 'section': 'MAV-1 does not cause PKR depletion by reducing steady-state levels of PKR mRNA 166'}, {'text': 'Because MAV-1 did not reduce PKR mRNA steady-state levels, we determined whether 185 MAV-1 causes PKR depletion by reducing translation of its mRNA. We first assayed total PKR 186 mRNA bound to ribosomes during infection. C57BL/6 MEFs were mock infected or infected at 187 an MOI of 5, and lysates were collected at 48 hpi in the presence of cycloheximide to keep the 188 mRNA bound to the ribosomes (56). Lysates were centrifuged through 25% sucrose to pellet 189 ribosomes, and RNA was purified from the pellets. The purified RNAs were used to generate 190 cDNA, which we assayed for PKR mRNA by qPCR. As a control for pelleting of ribosomes, we 191 assayed the pellets and sucrose cushion supernatants by immunoblot with antibodies to 192 ribosomal protein RPL7. We confirmed that RPL7 was only present in the pellets and not the 193 supernatants (Fig. S2 ). There was no significant difference between the amount of PKR mRNA 194 in the ribosome pellet of mock infected lysates compared to MAV-1-infected lysates (Fig. 4A) . 195 196 To confirm the results seen in total mRNA bound to ribosomes, we also centrifuged cell 197 extracts on sucrose gradients to generate polysome profiles. This enabled us to analyze levels of 198 PKR mRNA associated with actively-translating ribosomes during infection. C57BL/6 MEFs 199 were mock infected or infected at an MOI of 2, and lysates were collected at 24 hpi in the 200 presence of cycloheximide, as above. RNA content for mock and infected lysates was estimated 201 by NanoDrop spectrophotometry, and equivalent OD amounts of RNA were centrifuged on 10-202 50% sucrose gradients to sediment 40S and 60S ribosomal subunits, 80S ribosomes 203 (monosomes), and polyribosomes (polysomes). A typical polysome profile was obtained (Fig. 204 4B) . RNA was purified from fractions containing monosomes and polysomes and then used to 205 generate cDNA, which we assayed for PKR mRNA by qPCR (Fig. 4C) . As a control, GAPDH 206 mRNA was measured by qPCR, and PKR mRNA levels in each fraction were normalized to the 207 GAPDH mRNA content. When the data for percentage of PKR mRNA bound to ribosomes was 208 pooled into monosome and polysome fractions and analyzed (Fig. 4D) , 90.1% and 91.8% were 209 bound to polysomes (fractions 7-19) for mock and infected samples, respectively, compared to 210 9.9% and 8.2% bound to monosomes (fractions 1-6). We performed two additional polysome 211 gradient analyses. The pooled data from all three (Fig. 4E ) were similar to the data for Fig. 4D , 212', 'cite_spans': [{'start': 1028, 'end': 1035, 'text': '195 196', 'ref_id': None}, {'start': 1771, 'end': 1784, 'text': '(Fig. 204 4B)', 'ref_id': None}], 'ref_spans': [{'start': 850, 'end': 858, 'text': '(Fig. S2', 'ref_id': None}, {'start': 1016, 'end': 1025, 'text': '(Fig. 4A)', 'ref_id': 'FIGREF14'}, {'start': 1928, 'end': 1937, 'text': '(Fig. 4C)', 'ref_id': 'FIGREF14'}, {'start': 2200, 'end': 2209, 'text': '(Fig. 4D)', 'ref_id': 'FIGREF14'}, {'start': 2475, 'end': 2483, 'text': '(Fig. 4E', 'ref_id': 'FIGREF14'}, {'start': 2515, 'end': 2522, 'text': 'Fig. 4D', 'ref_id': 'FIGREF14'}], 'section': 'MAV-1 infection effects on PKR translation 184'}, {'text': 'i.e., 82.7% and 92.7% of PKR mRNA were bound to mock and infected polysomes, respectively, 213 compared to 17.3% and 7.3% bound to monosomes. Thus, PKR protein depletion during MAV-1 214 infection does not appear to stem from a decrease in PKR mRNA translation. 215', 'cite_spans': [], 'ref_spans': [], 'section': 'MAV-1 infection effects on PKR translation 184'}, {'text': \"We also assayed whether PKR mRNA might have a signal that would reduce its 217 translation during MAV-1 infection. We constructed a plasmid that positioned sequence 218 corresponding to the 5' UTR of PKR mRNA upstream of a reporter nanoluciferase gene (57), 219 transfected it into C57BL/6 MEFs, then infected with MAV-1. Compared to cells transfected 220 with a control plasmid with the human \\uf062-globin 5' UTR upstream of the reporter nanoluciferase, 221 there was no significant difference in luciferase activity between mock infected and infected 222 samples (Fig. S3 ). These data suggest that MAV-1 is not affecting PKR translation through 223 interaction with the 5' UTR of PKR. The data are consistent with the ribosome pellet and 224 polysome data that MAV-1 infection does not reduce PKR mRNA translation. 225 226 PKR is depleted by proteasomal degradation during MAV-1 infection. 227\", 'cite_spans': [], 'ref_spans': [{'start': 561, 'end': 569, 'text': '(Fig. S3', 'ref_id': 'FIGREF17'}], 'section': '216'}, {'text': 'There are two main proteolysis pathways in cells, proteasomal degradation and lysosomal 228 degradation (58). To determine whether MAV-1 depletes PKR by either protein degradation 229 pathway, we first assayed whether PKR is lysosomally degraded as follows. CMT93 cells were 230 mock infected or infected with MAV-1 and treated at the time of infection with the lysosome 231 inhibitors ammonium chloride or chloroquine, or water (as a control). At 24 hpi, we collected 232 lysates and analyzed them by immunoblot with antibodies to PKR. In the presence of the 233 lysosomal degradation inhibitors, PKR was depleted by 24 hpi (Fig. 5A) , indicating that 234 lysosomal degradation was not the cause of PKR depletion during MAV-1 infection. We 235 confirmed that the inhibitor treatment did block lysosomal degradation by incubating cells with 236 dye-quenched bovine serum albumin (DQ BSA) in addition to the lysosomal inhibitors. DQ BSA 237 is self-quenched until it is digested in the lysosome (59, 60), and imaging confirmed that cells 238 treated with lysosome inhibitors did not fluoresce, but cells treated with the vehicle control 239 (H 2 O) did, as expected (Fig. 5B) . 240 241 Next, we examined whether proteasomal degradation is responsible for the degradation of 242 PKR by using proteasome inhibitors MG132 and bortezomib. These inhibit proteasome activity 243 by binding to the active sites in the 20S subunit and blocking the proteolytic activity (61-63). We 244 mock infected or infected C57BL/6 MEFs with MAV-1 and treated with MG132 or bortezomib 245 in DMSO at the time of infection. At 24 hpi, we collected lysates and analyzed them by 246 immunoblot for PKR protein levels. While PKR was depleted in the control DMSO-treated 247 MAV-1-infected cells as expected, PKR protein was present in the MG132-and bortezomib-248 treated cells at levels comparable to mock infected cells ( Fig. 6A and B ). To rule out the 249 possibility that PKR was present (not depleted) because the virus infection itself was inhibited by 250 MG132 or bortezomib, we assayed viral replication of MAV-1 with MG132 and bortezomib 251 treatment by qPCR of viral DNA. Viral replication was equivalent in all three treatment groups 252 ( Fig. S4) , indicating that the treatments did not affect the ability of the virus to productively 253 infect the cells. Taken together, these data indicate that MAV-1 infection results in PKR 254 depletion by causing PKR to be degraded by the proteasome during infection. 255', 'cite_spans': [], 'ref_spans': [{'start': 625, 'end': 634, 'text': '(Fig. 5A)', 'ref_id': 'FIGREF22'}, {'start': 1165, 'end': 1174, 'text': '(Fig. 5B)', 'ref_id': 'FIGREF22'}, {'start': 1898, 'end': 1911, 'text': 'Fig. 6A and B', 'ref_id': 'FIGREF23'}, {'start': 2229, 'end': 2237, 'text': 'Fig. S4)', 'ref_id': 'FIGREF14'}], 'section': '216'}, {'text': 'A signal for proteasomal degradation is the conjugation of ubiquitin to a protein (64, 65). 257', 'cite_spans': [], 'ref_spans': [], 'section': '256'}, {'text': 'We examined whether PKR is ubiquitinated by immunoblot. We detected ubiquitination of a 258 positive control, mouse p53, which is degraded in the presence of MAV-1 proteins (66). 259', 'cite_spans': [], 'ref_spans': [], 'section': '256'}, {'text': 'However, even with the use of epitope-tagged ubiquitin (67) and MG132 treatment, we were 260 unable to detect PKR ubiquitination during infection (Fig. S5 ). This is consistent with an 261 inability to detect PKR ubiquitination when it is degraded during RVFV infection (32). Although 262 RVFV NSs is known to recruit an E3 ligase to PKR, the authors reported that ubiquitinated PKR 263 is undetectable. Therefore, the cellular degradation signal for PKR remains unclear. 264', 'cite_spans': [], 'ref_spans': [{'start': 146, 'end': 154, 'text': '(Fig. S5', 'ref_id': 'FIGREF22'}], 'section': '256'}, {'text': 'We investigated when proteasomal degradation of PKR occurs. Early viral proteins are 267 expressed prior to viral DNA replication, which is then followed by late viral protein expression. 268', 'cite_spans': [], 'ref_spans': [], 'section': 'PKR is actively depleted early in infection. 266'}, {'text': 'First, we examined the kinetics of PKR degradation to determine whether an early or late viral 269 protein was likely responsible. We mock infected and infected CMT93 cells with MAV-1 at an 270 MOI of 10 and collected lysates every six hours for 24 hours and analyzed them by immunoblot 271 with antibodies to PKR or MAV-1 early region 1A (E1A) protein, the first viral protein made 272 during infection (68). In infected cells, PKR degradation was first detected at 12 hpi (Fig. 7A) , 273 and quantitation of five independent experiments showed that ~20% of the starting levels of PKR 274 protein remained at 24 hpi ( Fig 7B) . 275 . CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/584680 doi: bioRxiv preprint', 'cite_spans': [], 'ref_spans': [{'start': 474, 'end': 483, 'text': '(Fig. 7A)', 'ref_id': 'FIGREF24'}, {'start': 619, 'end': 626, 'text': 'Fig 7B)', 'ref_id': 'FIGREF24'}], 'section': 'PKR is actively depleted early in infection. 266'}, {'text': 'In parallel, to determine the half-life of PKR in uninfected CMT93 cells, we treated 277 CMT93 cells with cycloheximide to halt protein translation and thus production of new PKR. 278', 'cite_spans': [], 'ref_spans': [], 'section': 'PKR is actively depleted early in infection. 266'}, {'text': 'We collected lysates every six hours for 24 hours and analyzed by immunoblot with antibodies 279 to PKR. After 24 hours of cycloheximide treatment, approximately 90% of the starting levels of The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/584680 doi: bioRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'PKR is actively depleted early in infection. 266'}, {'text': 'We addressed whether viral DNA replication is needed for PKR degradation. We mock 299 infected or infected CMT93 cells with MAV-1 at an MOI of 10 and treated them with cytosine 300 arabinoside (araC) at the time of infection to inhibit DNA synthesis (69, 70) . This would allow 301 the virus to infect the cell and produce early viral proteins, but would inhibit viral DNA 302 replication and prevent late protein synthesis. We collected lysates at 20 and 40 hpi and analyzed 303 them by immunoblot. We confirmed that araC treatment resulted in no late protein synthesis by 304 performing an immunoblot for late virion proteins (Fig. S8 ). In samples treated with araC, PKR 305 degradation was seen at 20 and 40 hpi ( We have demonstrated here that PKR is antiviral in MAV-1 infections of cultured cells. 312', 'cite_spans': [{'start': 250, 'end': 254, 'text': '(69,', 'ref_id': None}, {'start': 255, 'end': 258, 'text': '70)', 'ref_id': None}], 'ref_spans': [{'start': 628, 'end': 636, 'text': '(Fig. S8', 'ref_id': None}], 'section': 'PKR is actively depleted early in infection. 266'}, {'text': 'Surprisingly, MAV-1 infection of primary and established cultured cells depleted PKR. The 313 depletion was not due to reduced steady-state levels or reduced translation of PKR mRNA. 314', 'cite_spans': [], 'ref_spans': [], 'section': 'PKR is actively depleted early in infection. 266'}, {'text': 'Instead, we showed that PKR depletion is inhibited by proteasome inhibitors, implicating 315 proteasomal degradation of PKR. Several lines of evidence suggest that the degradation is due to 316 a viral early function. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/584680 doi: bioRxiv preprint antiviral activity during infection (10, 72), MAV-1 does not produce such VA RNAs, and how 322 MAV-1 infection is affected by PKR is first described in this report. When we infected PKR -/-323 MEFs with MAV-1, viral DNA yields were 5 to 6 times higher than viral DNA yields from wild 324 type MEFs (Fig. 1) , indicating that PKR plays an antiviral role during MAV-1 infection. At 48 325 hpi, the viral DNA yield from the N-MEFs was nearly 4 times higher than the C-MEFs, but by 326 72 hpi the viral DNA yields from both types of PKR -/-MEFs were similar to each other and 327', 'cite_spans': [], 'ref_spans': [{'start': 665, 'end': 673, 'text': '(Fig. 1)', 'ref_id': None}], 'section': 'PKR is actively depleted early in infection. 266'}, {'text': 'significantly increased compared to wild type MEFs (Fig. 1) . This difference in viral replication 328 kinetics between the two types of PKR -/-MEFs may be due to differences in expression level and 329 activity of the PKR fragments reportedly produced by them; we have not assayed PKR fragment 330 production in our cells. 331', 'cite_spans': [], 'ref_spans': [{'start': 51, 'end': 59, 'text': '(Fig. 1)', 'ref_id': None}], 'section': 'PKR is actively depleted early in infection. 266'}, {'text': 'We examined PKR protein levels during MAV-1 infection and found that PKR was 333 depleted from the cells as early as 12 hpi (Fig. 7) . Depletion was seen in a wide variety of cell 334 types, including immortalized C57BL/6 MEFs, primary C57BL/6 peritoneal macrophages, and 335 CMT93 mouse colon carcinoma cells. Once depleted, PKR protein levels never returned to mock 336 infected PKR protein levels during infection. Activation (transautophosphorylation) of PKR (6-337 9) was not required for this depletion, because kinase-dead mouse PKR was also depleted from 338 The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/584680 doi: bioRxiv preprint macrophages during MAV-1 infection (Fig. 3) , times when the PKR protein levels were depleted 345 ( Fig. 2A) . While PKR mRNAs in C57BL/6 MEFs were depleted 33% at 48 hpi and 40% at 72 346 hpi compared to mock lysates, this is not sufficient to explain the 84% and 94% reduction, 347 respectively, in PKR protein levels at those time points (Fig. 2B ). More likely, the reduction in 348 The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/584680 doi: bioRxiv preprint', 'cite_spans': [], 'ref_spans': [{'start': 124, 'end': 132, 'text': '(Fig. 7)', 'ref_id': 'FIGREF24'}, {'start': 751, 'end': 759, 'text': '(Fig. 3)', 'ref_id': 'FIGREF17'}, {'start': 816, 'end': 824, 'text': 'Fig. 2A)', 'ref_id': None}, {'start': 1057, 'end': 1065, 'text': '(Fig. 2B', 'ref_id': None}], 'section': '332'}, {'text': 'We considered whether reduced PKR levels were due to reduced PKR protein translation. 368', 'cite_spans': [], 'ref_spans': [], 'section': '332'}, {'text': \"There was no change in the total amount of PKR mRNA bound to ribosomes during infection 369 compared to uninfected cells, nor was there a significant change in the amount of actively 370 translating PKR mRNA during infection (Fig. 4) . We also found that the 5' UTR of mouse PKR 371 This indicates that PKR is not degraded by lysosomal degradation during viral infection, but by 378 proteasomal degradation. Though PKR degradation was due to proteasome activity during 379 MAV-1 infection, we were unable to demonstrate PKR ubiquitination, although we did detect 380 ubiquitination of mouse p53 (Fig. S5 ). This inability to demonstrate PKR ubiquitination could be 381 explained if at any given moment there were only low levels of ubiquitinated PKR present in the 382 cell. Perhaps, increasing the time under MG132 treatment could increase the amounts of 383 ubiquitinated proteins enough so that PKR ubiquitination could be seen. However, our inability 384 to detect ubiquitinated PKR is consistent with a similar inability to identify PKR ubiquitination 385 by RVFV NSs, even though NSs is known to recruit an E3 ligase to PKR (32). Alternatively, it is 386 possible that in MAV-1 infection, PKR is degraded in a ubiquitin-independent manner, possibly 387 because of intrinsic disordered regions of PKR or binding of regulating proteins to PKR that 388 target proteins to the proteasome (74, 75). 389 . CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/584680 doi: bioRxiv preprint\", 'cite_spans': [], 'ref_spans': [{'start': 225, 'end': 233, 'text': '(Fig. 4)', 'ref_id': 'FIGREF14'}, {'start': 595, 'end': 603, 'text': '(Fig. S5', 'ref_id': 'FIGREF22'}], 'section': '332'}, {'text': 'Our experiments indicate that MAV-1 actively depletes PKR early in infection. Ongoing 390 experiments are focused on determining the MAV-1 early protein(s) responsible for PKR 391 degradation. Two possibilities are E4 proteins, the homologs of hAd E4orf6 and E4orf3, which 392 we originally termed E4orfa/b and E4orfa/c, respectively (76). In human adenovirus, E4orf6 393 interacts with another early hAd protein, E1B 55K, to participate in an E3 ligase complex that 394 ubiquitinates and degrades p53 via proteasomal degradation (77, 78). When MAV-1 E4orf6, 395 E1B 55K, and mouse p53 are introduced by transfection into human cells, all three proteins 396 interact and mouse p53 is degraded (66). If MAV-1 E4orf6 and E1B 55K form a similar complex 397 in mouse cells, it may also degrade PKR. We have preliminary evidence that mouse p53 is 398 ubiquitinated in C57BL/6 MEFs during MAV-1 infection, which suggests that the mouse p53 399 degradation seen in human cells could be paralleled by degradation of endogenous mouse p53 400 and mouse PKR in mouse cells, mediated by MAV-1 E4orf6 and E1B 55K during infection. 401', 'cite_spans': [], 'ref_spans': [], 'section': '332'}, {'text': 'Another hAd E4 protein, E4orf3, causes proteasomal degradation of transcriptional intermediary 402 factor 1\\uf067 (79) and general transcription factor II-I (80) in a manner independent of hAd E4orf6 403 and E1B 55K. E4orf3 has SUMO E3 ligase and E4 elongase activity and induces sumoylation of 404 general transcription factor II-I, leading to its proteasome-dependent degradation (80). MAV-1 405', 'cite_spans': [], 'ref_spans': [], 'section': '332'}, {'text': 'E4orf3 may similarly have sumoylation activity that results ultimately in proteasome-dependent 406 PKR degradation. Another possibility of a viral protein involved in PKR degradation is the 407 protease encoded by MAV-1. The hAd protease is encapsidated in virions and proteolytically 408 processes viral proteins IIIa, VI, VII, VIII, mu, and TP (81-84). However, we think it is unlikely 409 that the MAV-1 protease degrades PKR, because we showed that UV-inactivated virus was 410 unable to degrade PKR. We assume that UV treatment would not destroy the MAV-1 protease 411 . CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/584680 doi: bioRxiv preprint activity, just as HSV-1 VP16 activity is not altered by UV-inactivation of HSV-1 (85), but we 412', 'cite_spans': [], 'ref_spans': [], 'section': '332'}, {'text': 'have not tested this directly. 413', 'cite_spans': [], 'ref_spans': [], 'section': '332'}, {'text': 'In summary, we demonstrated that PKR has an antiviral role during MAV-1 infection in 415 vitro, because when PKR is mutated, viral replication in MEFs is significantly higher compared 416 to wild type MEFs. Analysis of global PKR steady-state protein levels during infection showed containing 5% or 10% heat-inactivated fetal bovine serum (FBS), respectively, before use. 432', 'cite_spans': [], 'ref_spans': [], 'section': '414'}, {'text': 'Primary peritoneal macrophages were obtained from 6-10 week old C57BL/6 mice purchased 433 from Jackson Laboratory (#000664) as described (87). Briefly, 6-10 week old C57BL/6 mice 434 . CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/584680 doi: bioRxiv preprint were injected intraperitoneally with 1.2 mL 3% thioglycolate and euthanized 3-5 days later. The 435', 'cite_spans': [], 'ref_spans': [], 'section': '414'}, {'text': 'abdominal skin was carefully removed, exposing the peritoneum, which was then injected with 5 436 mL of sterile phosphate-buffered saline (PBS). The abdomen was massaged gently, then the PBS 437 containing the peritoneal macrophages was carefully withdrawn. The macrophages were 438 centrifuged at 100 x g for 4 minutes, red blood cells lysed in lysis buffer (0.15 M ammonium 439 chloride, 1 mM potassium bicarbonate, and 0.1 mM EDTA disodium salt) for 2 minutes at room 440 temperature, centrifuged at 100 x g for 4 minutes, washed twice in PBS, resuspended in DMEM 441 + 5% heat-inactivated FBS, and plated in 6 well plates. WT and PKR -/-MEFs (termed PKR WT 442', 'cite_spans': [], 'ref_spans': [], 'section': '414'}, {'text': 'MEFs and N-PKR -/-MEFs, respectively, throughout this paper) were obtained from Robert 443', 'cite_spans': [], 'ref_spans': [], 'section': '414'}, {'text': 'Silverman, Cleveland Clinic (88) Scientific #78410) was added to the plate. The cells were allowed to lyse at room temperature for 466 10 minutes before being harvested and centrifuged at 4˚C at 14,000 x g for 10 minutes to remove 467 debris. Equivalent amounts of protein, determined by a BCA assay (Pierce BCA Protein Assay 468', 'cite_spans': [], 'ref_spans': [], 'section': '414'}, {'text': 'Kit, Thermo Scientific #23227), were acetone precipitated by incubating with 4x volume ice cold 469 acetone overnight at -20˚C. Precipitated proteins were pelleted at 4˚C at 13,000 x g for 10 470 minutes and the pellets were dried for 30 minutes at room temperature. Pellets were resuspended 471 in 10 µL Pierce TM RIPA lysis buffer (Thermo Scientific #89900), 3.25 µL NuPAGE 4x LDS 472', 'cite_spans': [], 'ref_spans': [], 'section': '414'}, {'text': 'Sample Buffer (Invitrogen Cat #NP0007), and 1.25 µL 1M DTT. Samples were incubated at 473 37˚C for 10 minutes and then loaded into a well of an 8% acrylamide gel (8.3 cm wide x 7.3 cm 474 high x 0.1 cm thick) with a 2.5% stacking gel, electrophoresed for 30 minutes at 50 V and 85 475 minutes at 150 V, and then transferred to a PVDF membrane (BioRad #1620177) for 1 hour at 476 100V at 4˚C. Blots were blocked in 5% bovine serum albumin (BSA, Sigma A7906) in tris-477 buffered saline (BioRad #1706435) and 0.1% Tween 20 (Sigma P1379). Blots were probed with 478 primary antibodies to detect mouse PKR (Santa Cruz D-20 sc-708, 1:2000, or B-10 sc-6282, 479 . CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/584680 doi: bioRxiv preprint 1:200), mouse actin (Santa Cruz sc-1616-R, 1:1000), MAV-1 E1A (AKO-7-147, 1:1000, 480 described previously (68)), or MAV-1 late viral proteins (AKO 1-103, 1:1000, described 481 previously (91, 92)). Secondary antibodies used were IRDye 800CW anti-rabbit (Li-Cor 925-482 32213, 1:15,000) or IgG peroxidase-conjugated anti-mouse (Jackson Immuno 515-035-062, 483 1:20,000). Blots were visualized by LI-COR Odyssey imaging (LI-COR Biosciences) or 484 enhanced chemiluminescent substrates (Pierce ECL Western Blotting Substrate #32106) and X-485 ray film (Dot Scientific #BDB810). Densitometric quantification was performed on .tif files 486 using ImageJ software from NIH (93). 487', 'cite_spans': [{'start': 1125, 'end': 1157, 'text': '(Li-Cor 925-482 32213, 1:15,000)', 'ref_id': None}], 'ref_spans': [], 'section': '414'}, {'text': 'To attempt to demonstrate PKR ubiquitination status during MAV-1 infection, C57BL/6 489 MEFs were transfected with GFP-or HA-epitope tagged ubiquitin plasmids (Addgene #11928 490 and #18712, respectively) 24 hours before infection. We used Polyplus jetPRIME transfection 491 reagent (Polyplus #114-15) with 10 µg plasmid DNA and 30 µL jetPRIME reagent per 10 cm 492 plate. At 12 hpi (36 hours post transfection), we treated mock and infected C57BL/6 MEFs with 493 10 µM MG132 (Sigma M7449) for 6 hours before collecting lysates at 18 hpi in HCN buffer (50 494 mM HEPES, 150 mM NaCl, 2 mM CaCl 2, 1% Triton X-100 (Sigma T9284), 1x protease 495 inhibitors (Protease Inhibitor Cocktail Kit, Thermo Scientific #78410), and 5 mM N-496 ethylmalemide). The lysates were split into two aliquots, and 3 µg PKR (D-20 sc-708, 497 discontinued) or 3 µg isotype rabbit polyclonal antibody (Jackson Immuno #011-000-002) was 498 added to lysates. After rocking samples overnight at 4˚C, 20 µL protein A agarose suspension 499 (Calbiochem/Millipore #IP02-1.5ML) was added to each and samples were rocked at 4˚C for 2 500 hours. After incubation, agarose was washed 3 times with 1 mL HCN buffer, resuspended in 40 501 µL 2x Laemmli buffer (BioRad #161-0737) with 5% 2-mercaptoethanol (Sigma M6250), and 502 . CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/584680 doi: bioRxiv preprint boiled for 10 minutes. Lysate supernatants remaining after the initial PKR immunoprecipitation 503 were then immunoprecipitated again using the same procedure but with 4 µg anti-p53 (DO-1, 504', 'cite_spans': [], 'ref_spans': [], 'section': '488'}, {'text': \"Santa Cruz sc-126) or 4 µg isotype mouse monoclonal antibodies (ThermoFisher Scientific #02-505 6200). Immunoprecipitated proteins were immunoblotted for GFP-or HA-epitope tagged 506 ubiquitin with antibodies for GFP (1:3,000, Roche #11814460001) or HA (1:4,000, Abcam 507 Cells were washed twice with room temperature PBS and harvested by scraping into 516 PBS, centrifuging at 100 x g for 4 minutes at 4˚C, and resuspending in PBS. Total cellular DNA 517 was purified using the Invitrogen PureLink DNA Purification Kit (Thermo Scientific #K1820-518 02) and quantitated by a NanoDrop Spectrophotometer. 10 ng of total cellular DNA was 519 analyzed by qPCR using custom primers specific to MAV-1 E1A (mE1Agenomic Fwd: 5' GCA 520 CTC CAT GGC AGG ATT CT 3' and mE1Agenomic Rev 5' GGT CGA AGC AGA CGG TTC 521 TTC 3') and the results were normalized to GAPDH, which was analyzed using a GAPDH-522 specific primer/probe set (ThermoFisher Scientific, Mm99999915_g1, #4331182). The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/584680 doi: bioRxiv preprint mRNA analysis by qPCR 526 Cells were harvested by scraping into media, centrifuging at 100 x g for 4 min at 4˚C, and 527 washing the cell pellet three times with ice-cold PBS. RNA was purified using the Qiagen 528\", 'cite_spans': [], 'ref_spans': [], 'section': '488'}, {'text': 'RNeasy Mini Kit (Qiagen #74134) and stored at -80˚C. 125 ng of RNA per sample was used to 529 make cDNA using the High Capacity cDNA Reverse Transcription Kit (Applied Biosystems 530 #4368814), and 2 µL of the cDNA was analyzed by qPCR using a primer/probe set specific to 531 mouse PKR sequence (Thermo Fisher, Mm01235643_m1, #4331182). The results were 532 normalized to GAPDH, which was analyzed using a GAPDH-specific primer/probe set 533 Kit, Thermo Scientific #78410) was added to the plate. The cells were allowed to lyse at room 542 temperature for 10 minutes before being harvested and centrifuged at 4˚C at 14,000 x g for 10 543 minutes to remove debris. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/584680 doi: bioRxiv preprint (final concentration, Sigma C6628) was added to the media. At 24 hpi, at room temperature, 549 cells were washed once with PBS and Pierce TM RIPA lysis buffer (Thermo Scientific #89900) 550 with 1x protease inhibitors (Protease Inhibitor Cocktail Kit, Thermo Scientific #78410) was 551 added to the plate. The cells were allowed to lyse at room temperature for 10 minutes before 552 being harvested and centrifuged at 4˚C at 14,000 x g for 10 minutes to remove debris. 553', 'cite_spans': [], 'ref_spans': [], 'section': '488'}, {'text': 'The DQ BSA assay was performed as described (60). Briefly, C57BL/6 MEFs and 555 CMT93 cells were plated at 1.5 x 10 5 cells/plate or 3 x 10 5 cells/plate, respectively, in MatTek 556', 'cite_spans': [], 'ref_spans': [], 'section': '554'}, {'text': 'Glass Bottom Microwell Dishes (Part No: P35G-1.5-14C) with 2 mL of DMEM + 10% or 5% 557 FBS, respectively. The next day, the cell media was treated with 10 µL water, 10 mM NH 4 Cl 558 (final concentration), or 60 µM chloroquine (final concentration, Sigma C6628). Four hours after 559 adding inhibitors, DQ Red BSA (Invitrogen Cat #D12051) was added to the media to a final 560 concentration of 5 µg/mL in 2 mL DMEM + 10% or 5% FBS, respectively. At 24 hours post 561 treatment, the cells were imaged on a Nikon TE300 inverted microscope equipped with a 562 mercury arc lamp, Plan-Apochromat 60x, 1.4 NA objective, cooled digital CCD camera (Quantix 563', 'cite_spans': [], 'ref_spans': [], 'section': '554'}, {'text': 'Photometrics, Tucson, AZ), and a temperature-controlled stage, set at 37˚C. To image the DQ-564 BSA, we used an excitation filter centered at 572 nm and an emission filter centered at 635 nm. 565', 'cite_spans': [], 'ref_spans': [], 'section': '554'}, {'text': 'The exposure time was the same for all images. 566 567', 'cite_spans': [], 'ref_spans': [], 'section': '554'}, {'text': 'Ribosomes were pelleted as described (94). Briefly, C57BL/6 MEFs were plated on 10 569 cm plates, 3x10 5 cells per plate. The next day, the cells (~90% confluent) were infected with 570', 'cite_spans': [], 'ref_spans': [], 'section': 'Ribosome pelleting 568'}, {'text': 'MAV-1 at an MOI of 5. C57BL/6 MEF lysates were collected at 48 hpi by scraping the cells in 571 . CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/584680 doi: bioRxiv preprint ice-cold PBS containing 100 µg/mL cycloheximide (Sigma C7698), pelleting, and resuspending 572 in lysis buffer, which was 10 mM HEPES pH 7.5, 100 mM KCl, 5 mM MgCl 2 , 4 mM DTT, 573 0.5% NP-40, 100 µg/mL cycloheximide, 20 U/mL RNasin (Promega # N2511), 10% sucrose, 574 and 1x protease inhibitors (Protease Inhibitor Cocktail Kit, Thermo Scientific #78410). Cells 575 were lysed by passage through a chilled 26G needle five times and cleared by centrifugation for 576 10 min at 21,000 x g at 4˚C. 400 µL of cleared lysate (10 OD 260nm units) was layered onto 25% 577 sucrose and centrifuged 29,500 rpm in an SW41 rotor (107,458 rcf average) for 4 hours at 4˚C. (ThermoFisher Scientific Cat# AM2238), and 1% Triton X-100 (Sigma T9284). Cells were lysed 594 by passaging through a chilled 26G needle ten times, vortexing for 30 seconds, and then 595 incubating on ice for 5 min. Lysates were cleared by centrifugation for 5 min at 14,000 x g at 596 4˚C. 500 µL of cleared lysate (10 OD 260nm ) was layered onto a 10-50% sucrose gradient and 597 centrifuged at 35,000 rpm in an SW41 rotor (151,000 rcf) for 3 hours at 4˚C. After 598 centrifugation, gradients were pumped out of the top with a Brandel BR-188 Density Gradient 599', 'cite_spans': [], 'ref_spans': [], 'section': 'Ribosome pelleting 568'}, {'text': 'Fractionation System with a continuous reading of the OD 254 nm . From 24-34 fractions (350-500 600 µL) were collected. RNA was purified from selected fractions immediately using the Qiagen 601', 'cite_spans': [], 'ref_spans': [], 'section': 'Ribosome pelleting 568'}, {'text': 'RNeasy Mini Kit (Qiagen #74134) and stored at -80˚C until analysis by RT-qPCR. 602 603', 'cite_spans': [], 'ref_spans': [], 'section': 'Ribosome pelleting 568'}, {'text': 'Data were analyzed with GraphPad Prism 7.0 software. For qPCR and densitometry 605 analyses, the data were analyzed by individual Mann-Whitney tests. A value of P < 0.5 was 606 considered significant. 607 608', 'cite_spans': [], 'ref_spans': [], 'section': 'Statistical analyses 604'}, {'text': 'We thank Robert Silverman, Gokhan Hotamisligil, and Anthony Sadler for PKR mutant 610 and wild type cells. We thank Katelyn Green, Becky Billmire, and Peter Todd for the use of and 611 assistance with the Brandel BR-188 Density Gradient Fractionation System and for providing the 612 nanoluciferase and firefly luciferase plasmids. We thank Zachary Mendel and Joel Swanson for 613 assistance with imaging the DQ BSA assay. We thank Jason Weinberg, Michael Imperiale, 614 The cell pellets were collected and RNA was isolated. cDNA was generated from the RNA, and 904 qPCR was used to quantitate PKR mRNA levels. Each graph contains 5-7 replicates for each with MAV-1 (MAV) at an MOI of 5 or mock infected (mock) and collected at 48 hpi. Cells were 909 lysed, and cleared lysates from three 10 cm plates were layered onto 25% sucrose and 910 centrifuged to pellet ribosomes. RNA was purified from the pellets, cDNA was generated from 911 the RNA, and qPCR was used to quantitate the PKR mRNA levels. The graph contains 9 912 The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/584680 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/584680 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/584680 doi: bioRxiv preprint MAV-1 (MAV) at an MOI of 5 or mock infected (mock) and collected at 48 hpi. Cells were lysed, and cleared lysates from three 10 cm plates were layered onto 25% sucrose and centrifuged to pellet ribosomes. RNA was purified from the pellets, cDNA was generated from the RNA, and qPCR was used to quantitate the PKR mRNA levels. The graph contains 9 replicates for each time point, The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/584680 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. Figure 5 . PKR is not detectably ubiquitinated during MAV-1 infection. CMT93 cells were transfected with a GFP-ubiquitin plasmid using standard Polyplus transfection protocols. At 24 hpt, the cells were infected with MAV-1 (MAV) at an MOI of 5 or mock infected (mock) and treated with 10 µM MG132 at 6 hpi. Cell lysates were collected at 12 hpi and immunoprecipitated with (A) PKR (D-20) or an isotype control antibody or (B) p53 (DO-1) or an isotype control antibody. Immunoprecipitated samples were analyzed by immunoblot with GFP, ubiquitin, isotype, PKR (B-10), or p53 (1801) The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/584680 doi: bioRxiv preprint Supplemental Figure 6 . Viral DNA replication can be detected at 24 hpi by qPCR. CMT93 cells were infected with MAV-1 (MAV) at an MOI of 10 and collected every 6 hours for 24 hours. DNA was purified from cell pellets and analyzed for MAV-1 genome copies by qPCR. Graph is representative of four to five biological replicates per treatment group. Error bars are standard error of the mean (SEM). The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/584680 doi: bioRxiv preprint', 'cite_spans': [], 'ref_spans': [{'start': 2089, 'end': 2097, 'text': 'Figure 5', 'ref_id': 'FIGREF22'}, {'start': 2831, 'end': 2839, 'text': 'Figure 6', 'ref_id': 'FIGREF23'}], 'section': 'Acknowledgments 609'}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'Viral proteins targeting host protein kinase 657 R to evade an innate immune response: A mini review', 'authors': [{'first': 'E', 'middle': [], 'last': 'Dzananovic', 'suffix': ''}, {'first': 'S', 'middle': ['A'], 'last': 'Mckenna', 'suffix': ''}, {'first': 'T', 'middle': ['R'], 'last': 'Patel', 'suffix': ''}], 'year': 2018, 'venue': 'Biotechnol Genet Eng Rev', 'volume': '34', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF1': {'ref_id': 'b1', 'title': 'Identification of a conserved motif that is necessary for 660 binding of the vaccinia virus E3L gene products to double-stranded RNA', 'authors': [{'first': 'H', 'middle': ['W'], 'last': 'Chang', 'suffix': ''}, {'first': 'Bl ;', 'middle': [], 'last': 'Jacobs', 'suffix': ''}, {'first': 'A', 'middle': ['D'], 'last': 'Rice', 'suffix': ''}, {'first': 'P', 'middle': ['C'], 'last': 'Turner', 'suffix': ''}, {'first': 'J', 'middle': ['E'], 'last': 'Embury', 'suffix': ''}, {'first': 'L', 'middle': ['L'], 'last': 'Moldawer', 'suffix': ''}, {'first': 'H', 'middle': ['V'], 'last': 'Baker', 'suffix': ''}, {'first': 'R', 'middle': ['W'], 'last': 'Moyer', 'suffix': ''}], 'year': 1993, 'venue': 'Virology', 'volume': '661', 'issn': '', 'pages': '537--584', 'other_ids': {}}, 'BIBREF3': {'ref_id': 'b3', 'title': 'NSs protein of Rift Valley fever virus induces the specific 704 degradation of the double-stranded RNA-dependent protein kinase', 'authors': [{'first': 'H', 'middle': [], 'last': 'Unger', 'suffix': ''}, {'first': 'F', 'middle': [], 'last': 'Weber', 'suffix': ''}], 'year': 2009, 'venue': 'J Virol', 'volume': '83', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF4': {'ref_id': 'b4', 'title': 'Rift Valley 706 fever virus NSs protein promotes post-transcriptional downregulation of protein kinase 707 PKR and inhibits eIF2alpha phosphorylation', 'authors': [{'first': 'T', 'middle': [], 'last': 'Ikegami', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Narayanan', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Won', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Kamitani', 'suffix': ''}, {'first': 'C', 'middle': ['J'], 'last': 'Peters', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Makino', 'suffix': ''}], 'year': 2009, 'venue': 'PLoS Pathog', 'volume': '5', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF5': {'ref_id': 'b5', 'title': 'Black TL, Safer B, Hovanessian A, Katze MG. 1989. The cellular 68,000-Mr protein 714 kinase is highly autophosphorylated and activated yet significantly degraded during 715 poliovirus infection: Implications for translational regulation', 'authors': [{'first': 'R', 'middle': [], 'last': 'Mudhasani', 'suffix': ''}, {'first': 'J', 'middle': ['P'], 'last': 'Tran', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Retterer', 'suffix': ''}, {'first': 'K', 'middle': ['P'], 'last': 'Kota', 'suffix': ''}, {'first': 'C', 'middle': ['A'], 'last': 'Whitehouse', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Bavari', 'suffix': ''}, {'first': 'T', 'middle': ['L'], 'last': 'Black', 'suffix': ''}, {'first': 'G', 'middle': ['N'], 'last': 'Barber', 'suffix': ''}, {'first': 'M', 'middle': ['G'], 'last': 'Katze', 'suffix': ''}], 'year': 1993, 'venue': 'PLoS Pathog', 'volume': '12', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF6': {'ref_id': 'b6', 'title': 'Esterase D enhances type I interferon signal transduction to suppress foot-and-718 mouth disease virus replication', 'authors': [{'first': 'W', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'Z', 'middle': [], 'last': 'Zhu', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Cao', 'suffix': ''}, {'first': 'F', 'middle': [], 'last': 'Yang', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Mao', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Liu', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Zheng', 'suffix': ''}], 'year': 2016, 'venue': 'Mol Immunol', 'volume': '717', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF7': {'ref_id': 'b7', 'title': 'Foot-and-mouth disease virus induces lysosomal degradation of host protein kinase 721 PKR by 3C proteinase to facilitate virus replication', 'authors': [{'first': 'C', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'Z', 'middle': [], 'last': 'Zhu', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Du', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Cao', 'suffix': ''}, {'first': 'F', 'middle': [], 'last': 'Yang', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Feng', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Liu', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Zheng', 'suffix': ''}], 'year': None, 'venue': 'Virology', 'volume': '720', 'issn': '2017', 'pages': '', 'other_ids': {}}, 'BIBREF9': {'ref_id': 'b9', 'title': 'Middle East 724 respiratory coronavirus accessory protein 4a inhibits PKR-mediated antiviral stress 725 responses', 'authors': [{'first': 'P', 'middle': ['J'], 'last': 'Bredenbeek', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Kikkert', 'suffix': ''}, {'first': 'R', 'middle': ['J'], 'last': 'De Groot', 'suffix': ''}, {'first': 'F', 'middle': ['J'], 'last': 'Van Kuppeveld', 'suffix': ''}], 'year': 2016, 'venue': 'PLoS Pathog', 'volume': '12', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF10': {'ref_id': 'b10', 'title': 'Rapid 727 decrease in the levels of the double-stranded RNA-dependent protein kinase during virus 728 infections', 'authors': [{'first': 'A', 'middle': ['G'], 'last': 'Hovanessian', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Galabru', 'suffix': ''}, {'first': 'E', 'middle': [], 'last': 'Meurs', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Buffet-Janvresse', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Svab', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Robert', 'suffix': ''}], 'year': 1987, 'venue': 'Virology', 'volume': '159', 'issn': '', 'pages': '126--162', 'other_ids': {}}, 'BIBREF11': {'ref_id': 'b11', 'title': 'dsRNA binding domain of PKR is 730 proteolytically released by enterovirus A71 to facilitate viral replication', 'authors': [{'first': 'Y', 'middle': ['H'], 'last': 'Chang', 'suffix': ''}, {'first': 'K', 'middle': ['S'], 'last': 'Lau', 'suffix': ''}, {'first': 'R', 'middle': ['L'], 'last': 'Kuo', 'suffix': ''}, {'first': 'J', 'middle': ['T'], 'last': 'Horng', 'suffix': ''}], 'year': 2017, 'venue': 'Front Cell Infect 731 Microbiol', 'volume': '7', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF12': {'ref_id': 'b12', 'title': 'Toscana virus NSs protein promotes degradation of 733 double-stranded RNA-dependent protein kinase', 'authors': [{'first': 'B', 'middle': [], 'last': 'Kalveram', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Ikegami', 'suffix': ''}], 'year': 2013, 'venue': 'J Virol', 'volume': '87', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF13': {'ref_id': 'b13', 'title': 'NSs virulence factor of 735 Rift Valley fever virus engages the F-box proteins FBXW11 and beta-TRCP1 to degrade 736 the antiviral protein kinase PKR', 'authors': [{'first': 'M', 'middle': [], 'last': 'Kainulainen', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Lau', 'suffix': ''}, {'first': 'C', 'middle': ['E'], 'last': 'Samuel', 'suffix': ''}, {'first': 'V', 'middle': [], 'last': 'Hornung', 'suffix': ''}, {'first': 'F', 'middle': [], 'last': 'Weber', 'suffix': ''}], 'year': 2016, 'venue': 'J Virol', 'volume': '90', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF14': {'ref_id': 'b14', 'title': 'Susceptibility and signs associated with mouse 738 adenovirus type 1 infection of adult outbred Swiss mice', 'authors': [{'first': 'S', 'middle': ['C'], 'last': 'Kring', 'suffix': ''}, {'first': 'C', 'middle': ['S'], 'last': 'King', 'suffix': ''}, {'first': 'K', 'middle': ['R'], 'last': 'Spindler', 'suffix': ''}], 'year': 1995, 'venue': 'J Virol', 'volume': '69', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF15': {'ref_id': 'b15', 'title': 'Mouse adenoviruses', 'authors': [{'first': 'K', 'middle': ['R'], 'last': 'Spindler', 'suffix': ''}, {'first': 'M', 'middle': ['L'], 'last': 'Moore', 'suffix': ''}, {'first': 'A', 'middle': ['N'], 'last': 'Cauthen', 'suffix': ''}], 'year': 2007, 'venue': 'The mouse 740 in biomedical research', 'volume': '2', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF16': {'ref_id': 'b16', 'title': 'Adenovirus myocarditis in mice. An electron 742 microscopic study', 'authors': [{'first': 'Z', 'middle': ['R'], 'last': 'Blailock', 'suffix': ''}, {'first': 'E', 'middle': ['R'], 'last': 'Rabin', 'suffix': ''}, {'first': 'J', 'middle': ['L'], 'last': 'Melnick', 'suffix': ''}], 'year': 1968, 'venue': 'Exp Mol Pathol', 'volume': '9', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF17': {'ref_id': 'b17', 'title': 'Proinflammatory effects of interferon gamma in 745 mouse adenovirus 1 myocarditis', 'authors': [{'first': 'M', 'middle': ['K'], 'last': 'Mccarthy', 'suffix': ''}, {'first': 'M', 'middle': ['C'], 'last': 'Procario', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Twisselmann', 'suffix': ''}, {'first': 'J', 'middle': ['E'], 'last': 'Wilkinson', 'suffix': ''}, {'first': 'A', 'middle': ['J'], 'last': 'Archambeau', 'suffix': ''}, {'first': 'Michele', 'middle': [], 'last': 'De', 'suffix': ''}, {'first': 'S', 'middle': ['M'], 'last': 'Day', 'suffix': ''}, {'first': 'J', 'middle': ['B'], 'last': 'Weinberg', 'suffix': ''}], 'year': 2015, 'venue': 'J Virol', 'volume': '89', 'issn': '', 'pages': '468--79', 'other_ids': {}}, 'BIBREF18': {'ref_id': 'b18', 'title': 'Mouse adenovirus type 747 1 causes a fatal hemorrhagic encephalomyelitis in adult C57BL/6 but not BALB/c mice', 'authors': [{'first': 'J', 'middle': ['D'], 'last': 'Guida', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Fejer', 'suffix': ''}, {'first': 'L', 'middle': ['A'], 'last': 'Pirofski', 'suffix': ''}, {'first': 'C', 'middle': ['F'], 'last': 'Brosnan', 'suffix': ''}, {'first': 'M', 'middle': ['S'], 'last': 'Horwitz', 'suffix': ''}], 'year': 1995, 'venue': 'J 748 Virol', 'volume': '69', 'issn': '', 'pages': '7674--81', 'other_ids': {}}, 'BIBREF19': {'ref_id': 'b19', 'title': 'Adenovirus virus-associated RNA and translation control', 'authors': [{'first': 'M', 'middle': ['B'], 'last': 'Mathews', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Shenk', 'suffix': ''}], 'year': 1991, 'venue': 'J Virol', 'volume': '750', 'issn': '', 'pages': '5657--62', 'other_ids': {}}, 'BIBREF20': {'ref_id': 'b20', 'title': 'Completion of the 752 DNA sequence of mouse adenovirus type 1: Sequence of E2B, L1, and L2 (18-51 map 753 units)', 'authors': [{'first': 'J', 'middle': ['D'], 'last': 'Meissner', 'suffix': ''}, {'first': 'G', 'middle': ['N'], 'last': 'Hirsch', 'suffix': ''}, {'first': 'E', 'middle': ['A'], 'last': 'Larue', 'suffix': ''}, {'first': 'R', 'middle': ['A'], 'last': 'Fulcher', 'suffix': ''}, {'first': 'K', 'middle': ['R'], 'last': 'Spindler', 'suffix': ''}], 'year': 1997, 'venue': 'Virus Res', 'volume': '51', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF21': {'ref_id': 'b21', 'title': 'Essential role for the dsRNA-dependent protein kinase PKR in innate 756 immunity to viral infection', 'authors': [{'first': 'S', 'middle': [], 'last': 'Balachandran', 'suffix': ''}, {'first': 'P', 'middle': ['C'], 'last': 'Roberts', 'suffix': ''}, {'first': 'L', 'middle': ['E'], 'last': 'Brown', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Truong', 'suffix': ''}, {'first': 'A', 'middle': ['K'], 'last': 'Pattnaik', 'suffix': ''}, {'first': 'D', 'middle': ['R'], 'last': 'Archer', 'suffix': ''}, {'first': 'D', 'middle': ['F'], 'last': 'Stojdl', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Abraham', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Knowles', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Marius', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Brasey', 'suffix': ''}, {'first': 'B', 'middle': ['D'], 'last': 'Lichty', 'suffix': ''}, {'first': 'E', 'middle': ['G'], 'last': 'Brown', 'suffix': ''}], 'year': 2000, 'venue': 'Immunity', 'volume': '13', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF22': {'ref_id': 'b22', 'title': 'The murine double-stranded RNA-dependent protein kinase 759 PKR is required for resistance to vesicular stomatitis virus', 'authors': [{'first': 'N', 'middle': [], 'last': 'Sonenberg', 'suffix': ''}, {'first': 'J', 'middle': ['C'], 'last': 'Bell', 'suffix': ''}], 'year': 2000, 'venue': 'J Virol', 'volume': '74', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF23': {'ref_id': 'b23', 'title': 'Blockade 761 of interferon induction and action by the E3L double-stranded RNA binding proteins of 762 vaccinia virus', 'authors': [{'first': 'Y', 'middle': [], 'last': 'Xiang', 'suffix': ''}, {'first': 'R', 'middle': ['C'], 'last': 'Condit', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Vijaysri', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Jacobs', 'suffix': ''}, {'first': 'B', 'middle': ['R'], 'last': 'Williams', 'suffix': ''}, {'first': 'R', 'middle': ['H'], 'last': 'Silverman', 'suffix': ''}], 'year': 2002, 'venue': 'J Virol', 'volume': '76', 'issn': '', 'pages': '5251--5260', 'other_ids': {}}, 'BIBREF24': {'ref_id': 'b24', 'title': 'Functional characterization of pkr gene products 764 expressed in cells from mice with a targeted deletion of the N terminus or C terminus 765 domain of PKR', 'authors': [{'first': 'D', 'middle': [], 'last': 'Baltzis', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'A', 'middle': ['E'], 'last': 'Koromilas', 'suffix': ''}], 'year': 2002, 'venue': 'J Biol Chem', 'volume': '277', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF26': {'ref_id': 'b26', 'title': 'Deficient signaling in mice devoid of double-stranded RNA-768 dependent protein kinase', 'authors': [{'first': 'C', 'middle': [], 'last': 'Weissmann', 'suffix': ''}], 'year': 1995, 'venue': 'EMBO J', 'volume': '14', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF27': {'ref_id': 'b27', 'title': '771 Characterization of transgenic mice with targeted disruption of the catalytic domain of 772 the double-stranded RNA-dependent protein kinase, PKR', 'authors': [{'first': 'N', 'middle': [], 'last': 'Abraham', 'suffix': ''}, {'first': 'D', 'middle': ['F'], 'last': 'Stojdl', 'suffix': ''}, {'first': 'P', 'middle': ['I'], 'last': 'Duncan', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Methot', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Ishii', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Dube', 'suffix': ''}, {'first': 'B', 'middle': ['C'], 'last': 'Vanderhyden', 'suffix': ''}, {'first': 'H', 'middle': ['L'], 'last': 'Atkins 770', 'suffix': ''}, {'first': 'D', 'middle': ['A'], 'last': 'Gray', 'suffix': ''}, {'first': 'M', 'middle': ['W'], 'last': 'Mcburney', 'suffix': ''}, {'first': 'A', 'middle': ['E'], 'last': 'Koromilas', 'suffix': ''}, {'first': 'E', 'middle': ['G'], 'last': 'Brown', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Sonenberg', 'suffix': ''}, {'first': 'J', 'middle': ['C'], 'last': 'Bell', 'suffix': ''}], 'year': 1999, 'venue': 'J Biol Chem', 'volume': '274', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF28': {'ref_id': 'b28', 'title': 'The 774 kinase activity of PKR represses inflammasome activity', 'authors': [{'first': 'H', 'middle': ['C'], 'last': 'Yim', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Yu', 'suffix': ''}, {'first': 'C', 'middle': ['L'], 'last': 'White', 'suffix': ''}, {'first': 'P', 'middle': ['W'], 'last': 'Faber', 'suffix': ''}, {'first': 'B', 'middle': ['R'], 'last': 'Williams', 'suffix': ''}, {'first': 'A', 'middle': ['J'], 'last': 'Sadler', 'suffix': ''}], 'year': 2016, 'venue': 'Cell Res', 'volume': '26', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF29': {'ref_id': 'b29', 'title': 'Inhibition of protein synthesis in reticulocytes 776 by antibiotics. II. The site of action of cycloheximide, streptovitacin A and pactamycin', 'authors': [{'first': 'L', 'middle': [], 'last': 'Felicetti', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Colombo', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Baglioni', 'suffix': ''}], 'year': 1966, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF31': {'ref_id': 'b31', 'title': 'Todd 779 PK. 2016. CGG repeat-associated non-AUG translation utilizes a cap-dependent scanning 780 mechanism of initiation to produce toxic proteins', 'authors': [{'first': 'M', 'middle': ['G'], 'last': 'Kearse', 'suffix': ''}, {'first': 'K', 'middle': ['M'], 'last': 'Green', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Krans', 'suffix': ''}, {'first': 'C', 'middle': ['M'], 'last': 'Rodriguez', 'suffix': ''}, {'first': 'A', 'middle': ['E'], 'last': 'Linsalata', 'suffix': ''}, {'first': 'A', 'middle': ['C'], 'last': 'Goldstrohm', 'suffix': ''}], 'year': None, 'venue': 'Mol Cell', 'volume': '62', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF32': {'ref_id': 'b32', 'title': 'Protein Degradation, The Cell: A Molecular Approach', 'authors': [{'first': 'G', 'middle': ['M'], 'last': 'Cooper', 'suffix': ''}], 'year': 2000, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF34': {'ref_id': 'b34', 'title': 'DQ-Red BSA Trafficking Assay in Cultured Cells to 784 Assess Cargo Delivery to Lysosomes', 'authors': [{'first': 'R', 'middle': [], 'last': 'Marwaha', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Sharma', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Palm', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Park', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Wright', 'suffix': ''}, {'first': 'N', 'middle': ['N'], 'last': 'Pavlova', 'suffix': ''}, {'first': 'D', 'middle': ['A'], 'last': 'Tuveson', 'suffix': ''}, {'first': 'C', 'middle': ['B'], 'last': 'Thompson', 'suffix': ''}], 'year': 2015, 'venue': 'Cell', 'volume': '162', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF35': {'ref_id': 'b35', 'title': 'MG132, a proteasome inhibitor', 'authors': [{'first': 'N', 'middle': [], 'last': 'Guo', 'suffix': ''}, {'first': 'Z', 'middle': [], 'last': 'Peng', 'suffix': ''}], 'year': 2013, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF37': {'ref_id': 'b37', 'title': 'Bortezomib as the first 791 proteasome inhibitor anticancer drug: current status and future perspectives', 'authors': [{'first': 'D', 'middle': [], 'last': 'Chen', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Frezza', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Schmitt', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Kanwar', 'suffix': ''}, {'first': 'Q', 'middle': ['P'], 'last': 'Dou', 'suffix': ''}], 'year': 2011, 'venue': 'Curr Cancer 792 Drug Targets', 'volume': '11', 'issn': '', 'pages': '239--53', 'other_ids': {}}, 'BIBREF39': {'ref_id': 'b39', 'title': 'ATP-dependent 797 conjugation of reticulocyte proteins with the polypeptide required for protein 798 degradation', 'authors': [{'first': 'H', 'middle': [], 'last': 'Ovaa', 'suffix': ''}, {'first': 'P', 'middle': ['J'], 'last': 'Galardy', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Ciechanover', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Heller', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Elias', 'suffix': ''}, {'first': 'A', 'middle': ['L'], 'last': 'Haas', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Hershko', 'suffix': ''}], 'year': 1980, 'venue': 'Proc Natl Acad Sci', 'volume': '2', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF40': {'ref_id': 'b40', 'title': 'Proposed role of ATP in 800 protein breakdown: conjugation of protein with multiple chains of the polypeptide of 801 ATP-dependent proteolysis', 'authors': [{'first': 'A', 'middle': [], 'last': 'Hershko', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Ciechanover', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Heller', 'suffix': ''}, {'first': 'A', 'middle': ['L'], 'last': 'Haas', 'suffix': ''}, {'first': 'I', 'middle': ['A'], 'last': 'Rose', 'suffix': ''}], 'year': 1980, 'venue': 'Proc Natl Acad Sci', 'volume': '77', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF41': {'ref_id': 'b41', 'title': 'Using the E4orf6-Based E3 ubiquitin ligase as a tool to analyze the evolution 804 of adenoviruses', 'authors': [{'first': 'T', 'middle': [], 'last': 'Gilson', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Blanchette', 'suffix': ''}, {'first': 'M', 'middle': ['Z'], 'last': 'Ballmann', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Papp', 'suffix': ''}, {'first': 'J', 'middle': ['J'], 'last': 'Penzes', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Benko', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Harrach', 'suffix': ''}], 'year': 2016, 'venue': 'J Virol', 'volume': '90', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF42': {'ref_id': 'b42', 'title': 'A deubiquitinase 806 negatively regulates retro-translocation of nonubiquitinated substrates', 'authors': [{'first': 'K', 'middle': ['M'], 'last': 'Bernardi', 'suffix': ''}, {'first': 'J', 'middle': ['M'], 'last': 'Williams', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Inoue', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Schultz', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Tsai', 'suffix': ''}], 'year': 2013, 'venue': 'Mol Biol Cell', 'volume': '807', 'issn': '', 'pages': '3545--56', 'other_ids': {}}, 'BIBREF43': {'ref_id': 'b43', 'title': 'Interaction of mouse 809 adenovirus type 1 early region 1A protein with cellular proteins pRb and p107', 'authors': [{'first': 'K', 'middle': [], 'last': 'Smith', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Ying', 'suffix': ''}, {'first': 'A', 'middle': ['O'], 'last': 'Ball', 'suffix': ''}, {'first': 'C', 'middle': ['W'], 'last': 'Beard', 'suffix': ''}, {'first': 'K', 'middle': ['R'], 'last': 'Spindler', 'suffix': ''}], 'year': 1996, 'venue': 'Virology', 'volume': '810', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF44': {'ref_id': 'b44', 'title': 'Inhibitors of DNA synthesis (aphidicolin and araC/HU) prevent the 812 recovery of RNA synthesis after UV-irradiation', 'authors': [{'first': 'L', 'middle': ['V'], 'last': 'Mayne', 'suffix': ''}], 'year': 1984, 'venue': 'Mutat Res', 'volume': '131', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF45': {'ref_id': 'b45', 'title': 'A study of the mechanisms 814 of cytotoxicity of Ara-C on three human leukemic cell lines', 'authors': [{'first': 'J', 'middle': [], 'last': 'Zittoun', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Marquet', 'suffix': ''}, {'first': 'J', 'middle': ['C'], 'last': 'David', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Maniey', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Zittoun', 'suffix': ''}], 'year': 1989, 'venue': 'Cancer Chemother 815 Pharmacol', 'volume': '24', 'issn': '', 'pages': '251--256', 'other_ids': {}}, 'BIBREF46': {'ref_id': 'b46', 'title': 'Interferon action in 817 triply deficient mice reveals the existence of alternative antiviral pathways', 'authors': [{'first': 'A', 'middle': [], 'last': 'Zhou', 'suffix': ''}, {'first': 'J', 'middle': ['M'], 'last': 'Paranjape', 'suffix': ''}, {'first': 'S', 'middle': ['D'], 'last': 'Der', 'suffix': ''}, {'first': 'B', 'middle': ['R'], 'last': 'Williams', 'suffix': ''}, {'first': 'R', 'middle': ['H'], 'last': 'Silverman', 'suffix': ''}], 'year': 1999, 'venue': 'Virology', 'volume': '818', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF47': {'ref_id': 'b47', 'title': 'Structure, function, and evolution of adenovirus-associated 820 RNA: a phylogenetic approach', 'authors': [{'first': 'Y', 'middle': [], 'last': 'Ma', 'suffix': ''}, {'first': 'M', 'middle': ['B'], 'last': 'Mathews', 'suffix': ''}], 'year': 1996, 'venue': 'J Virol', 'volume': '70', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF48': {'ref_id': 'b48', 'title': 'PKR, a p53 target gene, plays a crucial role in 822 the tumor-suppressor function of p53', 'authors': [{'first': 'C', 'middle': ['H'], 'last': 'Yoon', 'suffix': ''}, {'first': 'E', 'middle': ['S'], 'last': 'Lee', 'suffix': ''}, {'first': 'D', 'middle': ['S'], 'last': 'Lim', 'suffix': ''}, {'first': 'Y', 'middle': ['S'], 'last': 'Bae', 'suffix': ''}], 'year': 2009, 'venue': 'Proc Natl Acad Sci', 'volume': '106', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF49': {'ref_id': 'b49', 'title': 'Proteasomes can degrade a significant 824 proportion of cellular proteins independent of ubiquitination', 'authors': [{'first': 'J', 'middle': ['M'], 'last': 'Baugh', 'suffix': ''}, {'first': 'E', 'middle': ['G'], 'last': 'Viktorova', 'suffix': ''}, {'first': 'E', 'middle': ['V'], 'last': 'Pilipenko', 'suffix': ''}], 'year': 2009, 'venue': 'J Mol Biol', 'volume': '386', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF50': {'ref_id': 'b50', 'title': 'Proteasomes and protein conjugation across domains of life', 'authors': [{'first': 'J', 'middle': [], 'last': 'Maupin-Furlow', 'suffix': ''}], 'year': 2011, 'venue': 'Nat 826 Rev Microbiol', 'volume': '10', 'issn': '', 'pages': '100--111', 'other_ids': {}}, 'BIBREF51': {'ref_id': 'b51', 'title': 'Transcription mapping of mouse adenovirus type 828 1 early region 4', 'authors': [{'first': 'S', 'middle': ['C'], 'last': 'Kring', 'suffix': ''}, {'first': 'A', 'middle': ['O'], 'last': 'Ball', 'suffix': ''}, {'first': 'K', 'middle': ['R'], 'last': 'Spindler', 'suffix': ''}, {'first': 'E', 'middle': [], 'last': 'Querido', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Blanchette', 'suffix': ''}, {'first': 'Q', 'middle': [], 'last': 'Yan', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Kamura', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Morrison', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Boivin', 'suffix': ''}, {'first': 'W', 'middle': ['G'], 'last': 'Kaelin', 'suffix': ''}], 'year': 1992, 'venue': 'Virology', 'volume': '190', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF52': {'ref_id': 'b52', 'title': 'Degradation of p53 by adenovirus 831 E4orf6 and E1B55K proteins occurs via a novel mechanism involving a Cullin-832 containing complex', 'authors': [{'first': 'R', 'middle': ['C'], 'last': 'Conaway', 'suffix': ''}, {'first': 'J', 'middle': ['W'], 'last': 'Conaway', 'suffix': ''}, {'first': 'P', 'middle': ['E'], 'last': 'Branton', 'suffix': ''}], 'year': 2001, 'venue': 'Genes Dev', 'volume': '15', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF53': {'ref_id': 'b53', 'title': 'Analysis of the 834 adenovirus E1B-55K-anchored proteome reveals its link to ubiquitination machinery', 'authors': [{'first': 'J', 'middle': ['N'], 'last': 'Harada', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Shevchenko', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Shevchenko', 'suffix': ''}, {'first': 'D', 'middle': ['C'], 'last': 'Pallas', 'suffix': ''}, {'first': 'A', 'middle': ['J'], 'last': 'Berk', 'suffix': ''}], 'year': 2002, 'venue': 'J 835 Virol', 'volume': '76', 'issn': '', 'pages': '9194--206', 'other_ids': {}}}, 'ref_entries': {'FIGREF0': {'text': '280 PKR protein remained (Fig. 7A bottom, and B). Comparing the results from MAV-1 infection 281 (Fig. 7A top, and 7B) and cycloheximide treatment of uninfected cells (Fig. 7A bottom, and 7B), 282 we conclude that MAV-1 was actively depleting PKR protein early in infection. E1A was 283 detected by immunoblot at 18 hpi (Fig. 7C), whereas viral DNA replication was first detected at 284 24 hpi in CMT93 cells (Fig. S6). Thus the 18 hpi timepoint is considered an early time point 285 during MAV-1 infection of CMT93 cells, prior to DNA replication, suggesting the involvement 286 of an early viral protein in PKR depletion. 287 288 An early viral function is required for PKR depletion by MAV-1 289 To determine whether viral gene expression or DNA replication are required for PKR 290 degradation during infection, we infected C57BL/6 MEFs and CMT93 cells with UV-inactivated 291 MAV-1 (which does not replicate viral DNA, Fig. S7). We infected cells at an MOI of 10 with 292 WT MAV-1 or UV-inactivated MAV-1 and analyzed lysates from 24 and 48 hpi by immunoblot 293 for PKR protein levels. In both cell types, while PKR was degraded by 24 hpi in the cells 294 infected with WT MAV-1, PKR protein levels were unaffected at both time points in cells 295infected with UV-inactivated MAV-1(Fig. 8A). This suggested that either gene expression or 296 DNA replication was required for PKR degradation during MAV-', 'latex': None, 'type': 'figure'}, 'FIGREF1': {'text': 'Fig. 8B), indicating that DNA replication was not 306 required for PKR degradation. Taken together, the results ofFig. 8are consistent with early viral 307 gene expression prior to DNA replication being involved in induction of PKR degradation by 308', 'latex': None, 'type': 'figure'}, 'FIGREF2': {'text': 'an IFN-inducible gene product that is an important component of the innate 319 immune response (1, 49). However, not all viruses have increased virulence in PKR -/-MEFs, 320including EMCV and vaccinia virus (51, 71). While hAds produce VA RNAs that inhibit PKR 321', 'latex': None, 'type': 'figure'}, 'FIGREF3': {'text': 'MEFs during infection (Fig. 2C). Both PKR and phospho-PKR were depleted in 339 all cell types examined. 340 341 We examined several possibilities that could explain depletion of PKR protein, including 342 PKR mRNA levels and alterations in translation. PKR mRNA levels remained unchanged during 343 MAV-1 infection in C57BL/6 MEFs at 24 hpi and were increased in primary C57BL/', 'latex': None, 'type': 'figure'}, 'FIGREF4': {'text': 'PKR steady-state mRNA levels at the late infection time points can be attributed to other effects 349 from viral infection, including the degradation or inhibition of proteins that induce PKR 350 expression. For example, p53 is capable of binding to the PKR promoter and inducing its 351 expression (73), but MAV-1 proteins cause p53 proteolysis (66). Together our results in 352 C57BL/6 MEFs and macrophages suggest the virus does not cause PKR protein depletion by 353 reducing PKR steady-state mRNA levels. 354 355 The differences in total PKR mRNA levels during infection between C57BL/6 MEFs and 356 primary peritoneal macrophages (Fig. 3) is possibly due to the fact that macrophages are an 357 immune cell, while MEFs are not. PKR protein took almost 3 times as long to be completely 358 degraded during infection in macrophages compared to MEFs (72 hours versus 24 hours) (Fig. 3592A). Total PKR mRNA levels were 2 to 3 times higher in MAV-1-infected macrophages 360 compared to mock infected macrophages, unlike the MEFs where total PKR mRNA levels were 361 unchanged or reduced 33-40% during MAV-1-infection compared to mock infected MEFs. Since 362 PKR is an IFN-stimulated gene (1, 2), higher levels of total PKR mRNA seen during infection in 363 the macrophages suggests IFN induction. This suggests that the immune response mounted by 364 the macrophages was greater than the immune response in the MEFs, and could help explain 365 why PKR took longer to degrade in macrophages compared to MEFs.', 'latex': None, 'type': 'figure'}, 'FIGREF5': {'text': 'placed upstream of a reporter gene produced the same amount of reporter with and without 372 MAV-1 infection. These data indicate that there are not translational effects of MAV-1 infection 373 on PKR protein levels that could explain the depleted PKR levels we observed. 374 375Inhibiting lysosomal degradation resulted in no change in PKR depletion in infected cells 376(Fig. 5A), but adding proteasome inhibitors preserved PKR protein within cells (Fig. 6A and B).', 'latex': None, 'type': 'figure'}, 'FIGREF7': {'text': \"417 complete PKR depletion by 72 hpi in multiple cell types, including immortalized and primary 418 cells, with even faster kinetics in some. PKR transcription and translation were not decreased by 419 MAV-1 infection, whereas proteasomal inhibition prevented PKR degradation. Taken together, 420 these data suggest that MAV-1 causes PKR to be proteasomally degraded at a post-translational 421 level. This work provides new insight into possible mechanisms of adenovirus inhibition of PKR 422 by DNA viruses. PKR degradation may be induced by other adenoviruses that do not produce 423 VA RNA, which includes all animal adenoviruses except primate adenoviruses and one type of 424 , virus, and infections 429 CMT93 cells (CCL-223) and C57BL/6 MEFs (SCRC-1008) were obtained from the 430 American Type Culture Collection and passaged in Dulbecco's modified eagle media (DMEM) 431\", 'latex': None, 'type': 'figure'}, 'FIGREF8': {'text': '#ab9110). No ubiquitin signal was detected from the PKR immunoprecipitations, though the 508 positive control, p53, showed ubiquitin signal with both epitope-tagged ubiquitins (data not 509 shown). Blots were also probed for PKR (1:200, PKR B-10 sc-6282), p53 (1:200, anti-p53 510 sc-98), and IRDye 800CW anti-mouse (Li-Cor 925-32212, 1:15,000) to confirm the 511 immunoprecipitations were successful, and signals for both proteins were detected (data not 512 shown). 513 514 Viral DNA yield analysis by qPCR 515', 'latex': None, 'type': 'figure'}, 'FIGREF10': {'text': 'ThermoFisher Scientific, Mm99999915_g1, #4331182). Arbitrary units were calculated as 534 follows: Mean C T PKR -mean C T GAPDH = \\uf044C T for sample. Arbitrary unit = 2^-\\uf044C T . 535 536 Proteasome inhibition 537 C57BL/6 MEFs were infected at an MOI of 10, and DMSO, 1 µM MG132 (Sigma 538 M7449), or 1 µM bortezomib (Selleckchem #S1013) were added to the media after a 1 hour 539 adsorption. At 24 hpi, cells were washed once with room temperature PBS, and Pierce TM RIPA 540 lysis buffer (Thermo Scientific #89900) with 1x protease inhibitors (Protease Inhibitor Cocktail 541', 'latex': None, 'type': 'figure'}, 'FIGREF11': {'text': 'Lysosome inhibition and DQ BSA assay 546CMT93 cells were infected at an MOI of 10. After a 1 hour adsorption, 10 µL water, 10 547 mM NH 4 Cl (final concentration, Baker Chemical Company #0660-1), or 60 µM chloroquine 548', 'latex': None, 'type': 'figure'}, 'FIGREF12': {'text': '578After pelleting, the supernatant was removed with a micropipet and 350 µL of 4˚C Buffer RLT 579Plus (from Qiagen RNeasy Mini Kit) was added to the pellet to collect the RNA. RNA was 580 purified immediately using the Qiagen RNeasy Mini Kit (Qiagen #74134) and stored atwere plated on 10 cm plates, 2 x 10 6 cells per plate. The next day, the 585 cells were infected with MAV-1 at an MOI of 2. Following a standard protocol (95), 5 minutes 586 prior to collection, cycloheximide was added at a final concentration of 100 µg/mL and 587 incubated at 37˚C. Cells were collected at 24 hpi by scraping in ice-cold PBS containing 100 588 µg/mL cycloheximide, pelleting, and resuspending in 500 µL lysis buffer (20 mM Tris-Cl, 150 589 mM NaCl, 15 mM MgCl 2 , 8% glycerol, 20 IU/mL SUPERase•In (ThermoFisher Scientific Cat# 590 AM2696), 80 IU/mL Murine RNAse Inhibitor (New England BioLabs Cat# M0314S), 0.1 591 mg/mL heparin (Sigma H3393-50), 0.1mg/mL cycloheximide, 1 mM DTT, 1x protease inhibitor 592 (Protease Inhibitor Cocktail Kit, Thermo Scientific #78410), 20 IU/mL Turbo DNAse 593', 'latex': None, 'type': 'figure'}, 'FIGREF13': {'text': 'Viral DNA yield is increased in PKR -/mouse embryonic fibroblasts. PKR WT MEFs 882 (WT), N-PKR -/-MEFs (N-), and C-PKR -/-MEFs (C-) were infected with MAV-1 at an MOI 1 883 and collected at 48 and 72 hpi. DNA was purified from cell pellets and analyzed for MAV-1 884 genome copies by qPCR. Graph is representative of three experiments, 14 biological replicates 885 per cell line per time point. Error bars are standard error of the mean (SEM). Mouse PKR is depleted during MAV-1 infection. (A) Cells (indicated at left) were 889 infected with MAV-1 (MAV) at an MOI of 10 or mock infected (mock). Cell lysates were 890 collected at the indicated times and analyzed by immunoblot with antibodies for PKR (B-10 for 891 C57BL/6 MEFs and D-20 for C57BL/6 primary peritoneal macrophages and CMT93 cells) and 892 actin. Blots are representative of a minimum of three independent experiments per cell line. (B) 893 Densitometry quantitation of five C57BL/6 MEFs immunoblots from A. Error bars are SEM. 894 N=5 for 24 hpi, n=4 for 48 hpi, and n=2 for 72 hpi. **P ≤ 0.01. (C) SV40-MEFs or kinase-dead 895 (K271R) SV40-MEFs were infected with MAV-1 at an MOI of 10. Cell lysates were 896 immunoblotted as in A with PKR D-20. Numbers below are the proportion of PKR protein for 897 each time point, normalized to actin and the mock PKR protein levels from the corresponding 898 time point. 899 MAV-1 does not cause PKR depletion by reducing steady state levels of PKR mRNA 901 at times when the protein levels are already reduced. MEFs (A) or isolated primary peritoneal 902 macrophages (B) were harvested and infected with MAV-1 at an MOI of 10 or mock infected. 903', 'latex': None, 'type': 'figure'}, 'FIGREF14': {'text': 'MAV-1 infection does not affect PKR translation. (A) C57BL/6 MEFs were infected 908', 'latex': None, 'type': 'figure'}, 'FIGREF15': {'text': \"replicates for each time point, pooled from 3 independent experiments. Error bars show the 913 SEM. (B) C57BL/6 MEFs were infected with MAV-1 (MAV) at an MOI of 2 or mock infected 914 (mock). Cells were collected at 25 hpi and lysed; cleared lysates were layered onto 10-50% 915 sucrose gradients and centrifuged. Gradients were collected from the top and pumped through a 916 UV spectrophotometer, and 34 fractions were collected. The gradients are displayed with the 917 bottom fractions to the right. The UV trace of the first 10 fractions (including 40S and 60S 918 ribosomal subunits) is not shown. (C) RNA was purified from each fraction of the gradients in B. 919 cDNA was generated from the RNA, and qPCR was used to quantitate PKR mRNA in each 920 fraction and displayed as the percent of total PKR mRNA associated with ribosomes. B and C 921 are results from one representative experiment of 3 independent experiments. (D) Percentage of 922 total PKR mRNA associated with monosomes (fractions 1-6) and polysomes (fractions 7-19) 923 from the trial displayed in B and C were pooled for mock and infected samples. The percentages 924 represented by each bar are displayed below each bar. (E) Pooled monosome and polysome data 925 as described in D from three independent experiments. Error bars show the SEM. The 926 PKR is not depleted by lysosomal degradation during MAV-1 infection. (A) CMT93 930 cells were infected with MAV-1 (MAV) at an MOI of 10 or mock infected (mock) and treated 931 with 10 mM ammonium chloride or 60 µM chloroquine to inhibit lysosomal degradation, or 932 water, as a control. Cell lysates were analyzed by immunoblot with antibodies for PKR (D-20) 933 and actin. Blots are representative of three independent experiments. (B) Inhibitors were tested 934 for activity using a DQ BSA assay; the DQ BSA molecule will only fluoresce if lysosomal 935 degradation is functional. Uninfected cells were treated as indicated and imaged by fluorescence 936 microscopy. 937 PKR is depleted by proteasomal degradation during MAV-1 infection. (A) C57BL/6 939 MEFs were infected with MAV-1 (MAV) at an MOI of 10 or mock infected (mock) and treated 940 with DMSO (vehicle for inhibitors), 1 µM MG132, or 1 µM bortezomib. Cell lysates were were 941 analyzed by immunoblot with antibodies for PKR (D-20) and actin. Blots are representative of 942 four independent experiments. (B) Densitometry quantitation of four independent experiments. 943 Treatment with bortezomib significantly inhibited PKR depletion in MAV-1 infected cells, PKR is actively depleted early in infection. (A) CMT93 cells were infected with 947 MAV-1 (MAV) at an MOI of 10 or mock infected (mock) (top), or uninfected cells were treated 948 with 50 µg/mL cycloheximide (CHX, bottom) to inhibit elongation of protein synthesis. Cell 949 with antibodies for E1A and actin. Blots are representative of four replicates 953 from two independent experiments. The E1A blot image was uniformly adjusted to a brightness 954 of 30 and a contrast of 5 in Adobe Photoshop. 955 Early gene expression is required for PKR depletion by MAV-1. (A) Cells (as 957 indicated at left) were infected with WT MAV-1 (WT MAV) or UV-inactivated MAV-1 (UV 958 MAV) at an MOI of 10, or mock infected (mock). Cell lysates were analyzed by immunoblot 959 with antibodies for PKR (D-20) and actin. Two independent wells were infected for each 960 condition at both time points. (B) CMT93 cells were infected with WT MAV-1 (MAV) at an 961 MOI of 10 or mock infected (mock). Infected cells were also treated (+) or not (-) with 20 µg/mL 962 cytosine arabinasine (AraC), an inhibitor of DNA synthesis. Cell lysates were analyzed with 963 antibodies for PKR and actin. immunoblot with antibodies for PKR (B-10) and actin. Densitometry quantitation 970 using ImageJ is listed below each lane; for each time point, the infected samples were 971 normalized to the mock. The PKR blot image was uniformly adjusted to a brightness of Ribosome pelleting. To confirm that most ribosomes ended up in the 975 pellet after centrifugation through sucrose (Fig. 4A), a sample of the pellet and the corresponding 976 supernatant were analyzed by immunoblot with antibodies for RPL7 (ribosomal protein L7, PKR mRNA 5' UTR does not result in altered reporter protein levels 980 upon MAV-1 infection. (A) C57BL/6 MEFs or (B) CMT93 cells were co-transfected with 981 pmPKR5UTRfullNL or AUG-NL-3xFLAG and pGL4.13 using jetPRIME reagents (Polyplus 982 #114-15) using the standard Polyplus protocol, with 200 ng total of plasmid and 300 µL of 983 jetPRIME reagent per 35 mm well. At 24 hours after transfection, the cells were infected with 984 MAV-1 at an MOI of 10. At 24 hpi, cells were lysed in 70 µL/well Glo Lysis Buffer (Promega 985 Corp.). After lysing, 25 µL of each lysed sample and 25 µL of OneGlo or NanoGlo (Promega 986Corp.) was added to two wells in a black 96-well plate (Fisher Scientific #07-000-634). After 5 987 minutes, the plate was read on a Promega GloMax luminometer. Relative light units from the 988\", 'latex': None, 'type': 'figure'}, 'FIGREF16': {'text': 'Viral DNA yield is increased in PKR -/mouse embryonic fibroblasts. PKR WT MEFs (WT), N-PKR -/-MEFs (N-), and C-PKR -/-MEFs (C-) were infected with MAV-1 at an MOI 1 and collected at 48 and 72 hpi. DNA was purified from cell pellets and analyzed for MAV-1 genome copies by qPCR. Graph is representative of three experiments, 14 biological replicates per cell line per time point. Error bars are standard error of the mean (SEM). *P ≤ 0.05, ***P ≤ 0.0002, Mouse PKR is depleted during MAV-1 infection. (A) Cells (indicated at left) were infected with MAV-1 (MAV) at an MOI of 10 or mock infected (mock). Cell lysates were collected at the indicated times and analyzed by immunoblot with antibodies for PKR (B-10 for C57BL/6 MEFs and D-20 for C57BL/6 primary peritoneal macrophages and CMT93 cells) and actin. Blots are representative of a minimum of three independent experiments per cell line. (B) Densitometry quantitation of five C57BL/6 MEFs immunoblots from A. Error bars are SEM. N=5 for 24 hpi, n=4 for 48 hpi, and n=2 for 72 hpi. **P ≤ 0.01. (C) SV40-MEFs or kinase-dead (K271R) SV40-MEFs were infected with MAV-1 at an MOI of 10. Cell lysates were immunoblotted as in A with PKR D-20. Numbers below are the proportion of PKR protein for each time point, normalized to actin and the mock PKR protein levels from the corresponding time point. A C mock MAV mock MAV mock MAV mock MAV mock MAV mock', 'latex': None, 'type': 'figure'}, 'FIGREF17': {'text': 'MAV-1 does not cause PKR depletion by reducing steady state levels of PKR mRNA at times when the protein levels are already reduced. MEFs (A) or isolated primary peritoneal macrophages (B) were harvested and infected with MAV-1 at an MOI of 10 or mock infected. The cell pellets were collected and RNA was isolated. cDNA was generated from the RNA, and qPCR was used to quantitate PKR mRNA levels. Each graph contains 5-7 replicates for each time point from three pooled experiments. Error bars show the SEM. *P ≤ 0.05 and **P ≤ 0.01.', 'latex': None, 'type': 'figure'}, 'FIGREF18': {'text': 'Figure 4.', 'latex': None, 'type': 'figure'}, 'FIGREF19': {'text': 'MAV-1 infection does not affect PKR translation. (A) C57BL/6 MEFs were infected with', 'latex': None, 'type': 'figure'}, 'FIGREF20': {'text': 'pooled from 3 independent experiments. Error bars show the SEM. (B) C57BL/6 MEFs were infected with MAV-1 (MAV) at an MOI of 2 or mock infected (mock). Cells were collected at 25 hpi and lysed; cleared lysates were layered onto 10-50% sucrose gradients and centrifuged. Gradients were collected from the top and pumped through a UV spectrophotometer, and 34 fractions were collected. The gradients are displayed with the bottom fractions to the right. The UV trace of the first 10 fractions (including 40S and 60S ribosomal subunits) is not shown. (C) RNA was purified from each fraction of the gradients in B. cDNA was generated from the RNA, and qPCR was used to quantitate PKR mRNA in each fraction and displayed as the percent of total PKR mRNA associated with ribosomes. B and C are results from one representative experiment of 3 independent experiments. (D) Percentage of total PKR mRNA associated with monosomes (fractions 1-6) and polysomes (fractions 7-19) from the trial displayed in B and C were pooled for mock and infected samples. The percentages represented by each bar are displayed below each bar. (E) Pooled monosome and polysome data as described in D from three independent experiments. Error bars show the SEM. The percentages represented by each bar are displayed below each bar. There were no significant differences between mock and infected samples (A, E).', 'latex': None, 'type': 'figure'}, 'FIGREF22': {'text': 'PKR is not depleted by lysosomal degradation during MAV-1 infection. (A) CMT93 cells were infected with MAV-1 (MAV) at an MOI of 10 or mock infected (mock) and treated with 10 mM ammonium chloride or 60 µM chloroquine to inhibit lysosomal degradation, or water, as a control. Cell lysates were analyzed by immunoblot with antibodies for PKR (D-20) and actin. Blots are representative of three independent experiments. (B) Inhibitors were tested for activity using a DQ BSA assay; the DQ BSA molecule will only fluoresce if lysosomal degradation is functional.Uninfected cells were treated as indicated and imaged by fluorescence microscopy.', 'latex': None, 'type': 'figure'}, 'FIGREF23': {'text': 'PKR is depleted by proteasomal degradation during MAV-1 infection. (A) C57BL/6 MEFs were infected with MAV-1 (MAV) at an MOI of 10 or mock infected (mock) and treated with DMSO (vehicle for inhibitors), 1 µM MG132, or 1 µM bortezomib. Cell lysates were were analyzed by immunoblot with antibodies for PKR (D-20) and actin. Blots are representative of four independent experiments. (B) Densitometry quantitation of four independent experiments.Treatment with bortezomib significantly inhibited PKR depletion in MAV-1 infected cells,', 'latex': None, 'type': 'figure'}, 'FIGREF24': {'text': 'PKR is actively depleted early in infection. (A) CMT93 cells were infected with MAV-1 (MAV) at an MOI of 10 or mock infected (mock) (top), or uninfected cells were treated with 50 µg/mL cycloheximide (CHX, bottom) to inhibit elongation of protein synthesis. Cell lysates were analyzed by immunoblot with antibodies for PKR (D-20) and actin. Blots are representative of five independent experiments. (B) Densitometry quantitation of five independent experiments, **P ≤ 0.01. (C) CMT93 cell lysates from A were analyzed with a second immunoblot with antibodies for E1A and actin. Blots are representative of four replicates from two independent experiments. The E1A blot image was uniformly adjusted to a brightness of 30 and a contrast of 5 in Adobe Photoshop.', 'latex': None, 'type': 'figure'}, 'FIGREF25': {'text': \"Early gene expression is required for PKR depletion by MAV-1. (A) Cells (as indicated at left) were infected with WT MAV-1 (WT MAV) or UV-inactivated MAV-1 (UV MAV) at an MOI of 10, or mock infected (mock). Cell lysates were analyzed by immunoblot with antibodies for PKR (D-20) and actin. Two independent wells were infected for each condition at both time points. (B) CMT93 cells were infected with WT MAV-1 (MAV) at an MOI of 10 or mock infected (mock).Infected cells were also treated (+) or not (-) with 20 µg/mL cytosine arabinasine (AraC), an inhibitor of DNA synthesis. Cell lysates were analyzed with antibodies for PKR and actin. PKR is depleted at MOIs of 2, 5, and 10. C57BL/6 MEFs were infected with MAV-1 at an MOI of 2, 5, or 10 (2, 5, 10) or mock infected (mock). Cell lysates were analyzed by immunoblot with antibodies for PKR (B-10) and actin. Densitometry quantitation using ImageJ is listed below each lane; for each time point, the infected samples were normalized to the mock.The PKR blot image was uniformly adjusted to a brightness of 150 in Adobe Photoshop. Ribosome pelleting. To confirm that most ribosomes ended up in the pellet after centrifugation through sucrose(Fig. 4A), a sample of the pellet and the corresponding supernatant were analyzed by immunoblot with antibodies for RPL7 (ribosomal protein L7, PKR mRNA 5' UTR does not result in altered reporter protein levels upon MAV-1 infection. (A) C57BL/6 MEFs or (B) CMT93 cells were co-transfected with pmPKR5UTRfullNL or AUG-NL-3xFLAG and pGL4.13 using jetPRIME reagents (Polyplus #114-15) using the standard Polyplus protocol, with 200 ng total of plasmid and 300 µL of jetPRIME reagent per 35 mm well. At 24 hours after transfection, the cells were infected with MAV-1 at an MOI of 10. At 24 hpi, cells were lysed in 70 µL/well Glo Lysis Buffer (Promega Corp.). After lysing, 25 µL of each lysed sample and 25 µL of OneGlo or NanoGlo (Promega Corp.) was added to two wells in a black 96-well plate (Fisher Scientific #07-000-634). After 5 minutes, the plate was read on a Promega GloMax luminometer. Relative light units from the pmPKR5UTRfullNL plasmid were normalized to the firefly luciferase and positive control plasmids. Graphs are representative MG132 and bortezomib treatment do not affect MAV-1 replication at 24hpi. C57BL/6 MEFs were infected with MAV-1 at an MOI of 10 and treated with DMSO (vehicle for inhibitors), 1 µM MG132 or bortezomib, and collected at 24 hpi. DNA was purified from cell pellets and analyzed for MAV-1 genome copies by qPCR. Graph is representative of five biological replicates per treatment group. Error bars are standard error of the mean (SEM). *P ≤ 0\", 'latex': None, 'type': 'figure'}, 'FIGREF27': {'text': 'UV-inactivated virus does not replicate viral DNA. (A) C57BL/6 MEFs or (B) CMT93 cells were infected with WT MAV-1 (WT) or UV-inactivated MAV-1 (UV) at an MOI of 10 and collected at indicated times. DNA was purified from cell pellets and analyzed for MAV-1 genome copies by qPCR. Graphs are representative of three to four biological replicates per treatment group. Error bars are standard error of the mean (SEM). AraC treatment inhibited late protein expression. CMT93 cells were infected with WT MAV-1 (MAV) at an MOI of 10 or mock infected (mock). Infected cells were also treated (+) or not (-) with 20 µg/mL cytosine arabinasine (araC), an inhibitor of DNA synthesis. Cell lysates were analyzed with antibodies for late virion proteins (AKO1-103, 1:1000). Arrowheads indicate late viral proteins.', 'latex': None, 'type': 'figure'}, 'TABREF0': {'text': 'and were passaged in DMEM containing 10% heat-inactivated 444 FBS before use. PKR -/-MEFs stably transfected with empty vector (termed C-PKR -/-MEFs 445 throughout this paper) were obtained from Dr. Gokhan Hotamisligil, Harvard University (89) 446 and were passaged in DMEM containing 10% heat-inactivated FBS before use. WT (SV40-447 MEFs) and K271R PKR mutant (K271R SV40-MEFs) MEFs were obtained from Anthony 448 Sadler, Hudson Institute of Medical Research (55) and were passaged in DMEM containing 10% 449 heat-inactivated FBS before use. 450Wild type mouse adenovirus type 1 (MAV-1) stock was prepared and titrated on mouse 452 NIH 3T6 fibroblasts as described previously(90). WT MAV-1 was UV-inactivated by UV-453 treating 200 µL of virus for 10 min at 800 mJ/cm 2 . UV inactivation was confirmed by qPCR and 454The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/584680 doi: bioRxiv preprint For infections, media was removed from cells and adsorption was performed in 0.4 mL of 457 inocula for 6-well plate 35-mm wells (unless otherwise noted) for 1 hour at 37˚C at the indicated 458MOIs (PFU/cell). After 60 minutes, 2 mL of DMEM+5% FBS was added without removing 459 inocula; this time was designated as 0 hpi. For araC experiments, 20 µg/mL araC (Sigma C1768) 460 was added at 0 hpi and replenished every 12-16 hours. 461At room temperature, cells were washed once with PBS and Pierce TM RIPA lysis buffer 464 (Thermo Scientific #89900) with 1x protease inhibitors (Protease Inhibitor Cocktail Kit, Thermo 465', 'latex': None, 'type': 'table'}, 'TABREF1': {'text': 'Christiane Wobus, Billy Tsai, Andrew Mehle, Mitch Ledwith, Mike Mathews, Chuck Samuel, 615 and Britt Glaunsinger for helpful discussions. We thank Michael Imperiale and Christiane 616 Wobus for comments on the manuscript. 617 Saphire EO, Basler CF. 2006. Ebola virus VP35 protein binds double-stranded RNA and 676 inhibits alpha/beta interferon production induced by RIG-I signaling. J Virol 80:5168-78. 677 21. Poppers J, Mulvey M, Khoo D, Mohr I. 2000. Inhibition of PKR activation by the 678 proline-rich RNA binding domain of the herpes simplex virus type 1 Us11 protein. J', 'latex': None, 'type': 'table'}, 'TABREF2': {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101', 'latex': None, 'type': 'table'}, 'TABREF3': {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the author/funder.It . https://doi.org/10.1101/584680 doi: bioRxiv preprint PKR actin mock MAV mock MAV mock MAV mock MAV mock MAV', 'latex': None, 'type': 'table'}, 'TABREF4': {'text': 'mock MAV mock MAV mock MAV mock MAV mock MAVThe copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/584680 doi: bioRxiv preprint', 'latex': None, 'type': 'table'}, 'TABREF6': {'text': 'antibodies. Input lane (mock input) contains 0.008 volume of mock infected lysate (relative to volume in immunoprecipitations). For (B) GFP and p53 were probed on separate duplicate blots.', 'latex': None, 'type': 'table'}}, 'back_matter': []}\n",
      "{'paper_id': 'acd8d28ae75dc1ec5fea2d767e3158545011c1f7', 'metadata': {'title': 'Title: 2 The Role of Interleukin-1 cytokine family (IL-1β, IL-37) and interleukin-12 cytokine 3 family (IL-12, IL-35) in eumycetoma infection pathogenesis. Authors 7 Corresponding Author', 'authors': [{'first': 'Amir', 'middle': [], 'last': 'Abushouk', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Amre', 'middle': [], 'last': 'Nasr', 'suffix': '', 'affiliation': {}, 'email': 'nasra@ksau-hs.edu.sa'}, {'first': 'Emad', 'middle': [], 'last': 'Masuadi', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Research Unit', 'location': {'country': 'King Saud'}}, 'email': 'e.mail:22masuadie@ksau-hs.edu.sa'}, {'first': 'Gamal', 'middle': [], 'last': 'Allam', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Taif University', 'location': {'addrLine': '25 Saudi Arabia', 'settlement': 'Taif'}}, 'email': 'e.mail:g.allam@tu.edu.sa'}, {'first': 'Emmanuel', 'middle': ['E'], 'last': 'Siddig', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Khartoum', 'location': {'settlement': 'Khartoum', 'country': 'Sudan'}}, 'email': ''}, {'first': 'Ahmed', 'middle': ['H'], 'last': 'Fahal', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Khartoum', 'location': {'settlement': 'Khartoum', 'country': 'Sudan'}}, 'email': ''}]}, 'abstract': [{'text': \"Mycetoma is a neglected tropical disease, endemic in many tropical and subtropical regions, 49 characterised by massive deformity and disability and can be fatal if untreated early and 50 appropriately. Interleukins (IL) -35 and IL-37 are newly discovered cytokines that play an 51 important role in suppressing the immune system. However, the expression of these 52 interleukins in patients with Madurella mycetomatis (M. mycetomatis) induced eumycetoma 53 has not yet been explored. This study aims to determine the levels of the IL-1 family (IL-1β, 54 IL-37) and IL-12 family (IL-12, IL-35) in a group of these patients and the association 55 between these cytokines levels and the patients' demographic characteristics. The present, 56 a case-control study was conducted at the Mycetoma Research Centre, Soba University 57 Hospital, University of Khartoum, Sudan and it included 140 individuals. They were divided 58 into two groups; group I: healthy controls [n = 70; median age 25 years (range 12 to 70 59 years)]. Group II: mycetoma patients [n = 70 patients; median age 25 (range 13 to 70 60 years)]. Cytokines levels were measured in sera using enzyme-linked immunosorbent assay 61 (ELISA). 62 There was no significant correlation between the IL-1β and IL-12 levels and the lesions size 63 and disease duration, whereas levels of IL-37 and IL-35 were significantly correlated with 64 that. The analysis of the risk factors of higher circulatory levels of IL-37 in patients of 65 mycetoma showed a significant negative association with IL-1β cytokine, where a unit 66 increment in IL-1β will decrease the levels of IL-37 by 35.28 pg/ml. The levels of IL-37 67 among the patients with a duration of mycetoma infection ≤ one year had significantly 68 decreased by an average of 18.45 compared to patients with a mycetoma infection's 69 duration of ≥ 5years (reference group). Furthermore, the risk factors of higher levels of IL-35 . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/502609 doi: bioRxiv preprint 70 in mycetoma patients revealed a significant negative association with IL-12, as a unit 71 increment in IL-12 decreases the levels of IL-35by 8.99 pg/ml (p < 0.001). Levels of IL-35 72 among the patients with duration of mycetoma infection ≤ one year had significantly 73 decreased (p-value = 0.002) on average by 41.82 compared to patients with a duration of 74 mycetoma infection ≥ five years (reference group). In conclusion, this study indicates that 75 both IL-35 and IL-37 are negatively associated with the levels of IL-1β and IL-12 in 76 eumycetoma mycetoma infection; and high levels of IL-37 and IL-35 may have a negative 77 impact on disease progression. 78 79 Authors Summary 80 Mycetoma is a progressive chronic granulomatous fungal or bacterial infection that may 81 result in massive destruction of subcutaneous tissues, muscles and bones. Mycetoma is 82 a neglected disease which is endemic in many tropical and subtropical areas. If the 83 disease is not treated properly, eventually it ends up with amputation and adverse 84 medical, health and socioeconomic effects on patients and the community.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': \"85 Previous data suggested a crucial role of adaptive immunity in host resistance to 86 causative agents and the disease progression. The recently identified IL -35 and IL-37 87 cytokines revealed an important role in immune suppression. Nevertheless, the 88 expression of these interleukins in patients with mycetoma has not yet been 89 investigated. Therefore, the present case-control study aimed to determine the levels of 90 IL-1 family (IL-1β, IL-37) and IL-12 family (IL-12, IL-35) in these patients and the 91 association between these cytokines levels and the patients' demographic 92 characteristics. . CC-BY 4.0 International license is made available under a\", 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/502609 doi: bioRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'The results of this study showed that the levels of IL-37 and IL-35 were consistently 94 positively correlated with different diameters of mycetoma lesions as well as its duration. 95 However, the levels of IL-1β and IL-12 were consistently negatively correlated with 96 different diameters of lesions and the duration of mycetoma infection. The analysis of 97 the risk factors of higher circulatory levels of IL-37 in patients of mycetoma showed a 98 significant negative association with IL-1β cytokine. Furthermore, the risk factors of 99 higher levels of IL-35 in patients of mycetoma revealed a significant negative 100 association with IL-12. These findings uncover a possible the role of IL-35 and IL-37 in 101', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}], 'body_text': [{'text': 'Mycetoma is a chronic granulomatous subcutaneous inflammatory disease, caused by 108 certain bacteria (actinomycetoma) or fungi (eumycetoma). This infection progresses to affect pre-existing Th2 environment caused by schistosomiasis promotes the development of 140 mycetoma as patients with mycetoma were significantly more positive for schistosoma 141 antibodies than healthy endemic controls [9] . These findings suggested that Th2 like 142 response and anti-inflammatory/immunosuppressive cytokines could have a negative impact 143 on mycetoma development and disease progression. 144 IL-1 is a polypeptide which has two forms; IL-1α and IL-1β. It is involved in the acute-phase 145 response and is accountable for several alterations that are related to the onset of various 146 medical disorders [15, 16] . It is demonstrated recently that higher levels of IL-1β cytokine 147 are strongly associated with surgically treated mycetoma patients, in comparison to those 148 treated without surgery [17] . It is known that IL-1β is a pro-inflammatory cytokine that is 149 involved in cell death coordination [18] . IL-1β cytokine is cleaved into the mature, active and disease severity in murine aspergillosis [29] . In addition to its role in innate immunity; IL-174 37 plays a pivotal role in regulating adaptive immunity by inducing regulatory T (T reg ) cells 175 and impairing activation of effector T-cell responses [30] . To our knowledge, there has been 176 no study to report the relationship between IL-37 and mycetoma pathogenesis so far. poorly understood. However IL-35 is recognised as a typical anti-inflammatory cytokine, and 189 the predominant mechanism of suppression is associated with its ability to suppress T cell (Table 2) . On the other hand, the 287 levels of cytokine IL-1β were constitutively negatively correlated with IL-37 and IL-35, ( Table   288 2). Furthermore, the levels of cytokine IL-37 were constitutively positively correlated with IL-289 35 (Table 2) . However, the levels of cytokine IL-37 and IL-35 were constitutively negatively 290 correlated with IL-12 (Table 2) . (Table 3) . Whereas, levels of IL-37 were constitutively positively correlated with IL-35 (Table   . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/502609 doi: bioRxiv preprint 3). However, the levels of IL-37 and IL-35 were constitutively negatively correlated with IL-300 12, (Table 3) . Table  (Table 8) . Furthermore, the circulating levels of IL-37 had significantly decreased with 369 increasing age groups [(19-24 years; p-value = 0.0100), 25-29 years; p-value =0.030 and 370 (30-39) years; p-value = 0.022)] (Table 8) . Interestingly, levels of IL-37 among mycetoma 371 patients showed a statistically significant difference between Itraconazole compared to 372 Ketoconazole, p < 0.001, (Table 8) . (Table 9 ). Additionally, the circulating levels of IL-35 showed no 384 significant association with the different age groups (Table 9) . Interestingly, treatment with 385 Itraconazole significantly increased circulating levels of IL-35 among mycetoma patients 386 compared to treatment with Ketoconazole, p-value <0.001, (Table 9) . The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/502609 doi: bioRxiv preprint a mycetoma infection duration ≥ 5years (reference group). More investigations are needed 506 to explore the mechanism by which IL-35 and IL-37 contribute in the mycetoma infection 507 outcomes. This will help in understanding the role of these cytokines IL-35 and IL-37 in the 508 pathogenesis of mycetoma, and may exploit it as a potential therapeutic target to prevent 509 mycetoma diseases recurrence. . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/502609 doi: bioRxiv preprint Table 2 . Correlations between the serum levels of (IL-1β, IL-37, IL-12 and IL-35) The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/502609 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/502609 doi: bioRxiv preprint', 'cite_spans': [{'start': 349, 'end': 352, 'text': '141', 'ref_id': None}, {'start': 394, 'end': 397, 'text': '[9]', 'ref_id': None}, {'start': 584, 'end': 587, 'text': '144', 'ref_id': None}, {'start': 801, 'end': 805, 'text': '[15,', 'ref_id': None}, {'start': 806, 'end': 809, 'text': '16]', 'ref_id': None}, {'start': 999, 'end': 1003, 'text': '[17]', 'ref_id': None}, {'start': 1068, 'end': 1071, 'text': '149', 'ref_id': None}, {'start': 1108, 'end': 1112, 'text': '[18]', 'ref_id': None}, {'start': 1210, 'end': 1214, 'text': '[29]', 'ref_id': None}, {'start': 1364, 'end': 1367, 'text': '175', 'ref_id': None}, {'start': 1422, 'end': 1426, 'text': '[30]', 'ref_id': None}], 'ref_spans': [{'start': 1737, 'end': 1746, 'text': '(Table 2)', 'ref_id': None}, {'start': 1867, 'end': 1878, 'text': 'Table   288', 'ref_id': 'TABREF9'}, {'start': 1982, 'end': 1991, 'text': '(Table 2)', 'ref_id': None}, {'start': 2099, 'end': 2108, 'text': '(Table 2)', 'ref_id': None}, {'start': 2111, 'end': 2120, 'text': '(Table 3)', 'ref_id': 'TABREF5'}, {'start': 2201, 'end': 2211, 'text': '(Table   .', 'ref_id': None}, {'start': 2520, 'end': 2529, 'text': '(Table 3)', 'ref_id': 'TABREF5'}, {'start': 2532, 'end': 2548, 'text': 'Table  (Table 8)', 'ref_id': 'TABREF9'}, {'start': 2758, 'end': 2767, 'text': '(Table 8)', 'ref_id': 'TABREF9'}, {'start': 2937, 'end': 2946, 'text': '(Table 8)', 'ref_id': 'TABREF9'}, {'start': 2949, 'end': 2957, 'text': '(Table 9', 'ref_id': 'TABREF10'}, {'start': 3075, 'end': 3084, 'text': '(Table 9)', 'ref_id': 'TABREF10'}, {'start': 3271, 'end': 3280, 'text': '(Table 9)', 'ref_id': 'TABREF10'}, {'start': 4046, 'end': 4053, 'text': 'Table 2', 'ref_id': None}, {'start': 4097, 'end': 4128, 'text': '(IL-1β, IL-37, IL-12 and IL-35)', 'ref_id': None}], 'section': '107'}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'Use of filter paper for the collection 658 and analysis of human whole blood specimens', 'authors': [{'first': 'J', 'middle': ['V'], 'last': 'Mei', 'suffix': ''}, {'first': 'J', 'middle': ['R'], 'last': 'Alexander', 'suffix': ''}, {'first': 'B', 'middle': ['W'], 'last': 'Adam', 'suffix': ''}, {'first': 'W', 'middle': ['H'], 'last': 'Hannon', 'suffix': ''}], 'year': 2001, 'venue': 'J Nutr', 'volume': '131', 'issn': '5', 'pages': '1631--1637', 'other_ids': {}}, 'BIBREF1': {'ref_id': 'b1', 'title': 'Comparison of available methods to elute 660 serum from dried blood spot samples for measles serology', 'authors': [{'first': 'S', 'middle': [], 'last': 'Mercader', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Featherstone', 'suffix': ''}, {'first': 'W', 'middle': ['J'], 'last': 'Bellini', 'suffix': ''}], 'year': 2006, 'venue': 'J Virol Methods', 'volume': '661', 'issn': '1', 'pages': '140--149', 'other_ids': {}}, 'BIBREF2': {'ref_id': 'b2', 'title': 'The use of the dried blood spot sample in epidemiological 663 studies', 'authors': [{'first': 'S', 'middle': ['P'], 'last': 'Parker', 'suffix': ''}, {'first': 'W', 'middle': ['D'], 'last': 'Cubitt', 'suffix': ''}], 'year': 1999, 'venue': 'J Clin Pathol', 'volume': '52', 'issn': '9', 'pages': '633--639', 'other_ids': {}}, 'BIBREF3': {'ref_id': 'b3', 'title': 'Helsinki Declaration revisited over concerns about human subjects. 665', 'authors': [{'first': 'B', 'middle': [], 'last': 'Sibbald', 'suffix': ''}], 'year': 2003, 'venue': 'CMAJ', 'volume': '169', 'issn': '10', 'pages': '', 'other_ids': {}}, 'BIBREF4': {'ref_id': 'b4', 'title': 'Differential activity 667 of innate defense antimicrobial peptides against Nocardia species', 'authors': [{'first': 'S', 'middle': [], 'last': 'Rieg', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Meier', 'suffix': ''}, {'first': 'E', 'middle': [], 'last': 'Fähnrich', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Huth', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Wagner', 'suffix': ''}, {'first': 'W', 'middle': ['V'], 'last': 'Kern', 'suffix': ''}], 'year': 2010, 'venue': 'BMC Microbiol', 'volume': '668', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF5': {'ref_id': 'b5', 'title': 'Immunity to fungi', 'authors': [{'first': 'S', 'middle': [], 'last': 'Leibundgut-Landmann', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Wüthrich', 'suffix': ''}, {'first': 'T', 'middle': ['M'], 'last': 'Hohl', 'suffix': ''}], 'year': None, 'venue': 'Curr Opin', 'volume': '670', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF8': {'ref_id': 'b8', 'title': 'MyD88-deficient Hydra reveal an ancient function of TLR signaling in sensing 673 bacterial colonizers', 'authors': [], 'year': 2012, 'venue': 'Proc Natl Acad Sci U S A', 'volume': '109', 'issn': '47', 'pages': '19374--19383', 'other_ids': {}}, 'BIBREF10': {'ref_id': 'b10', 'title': 'Suppression of inflammation and acquired immunity by 685 IL-37', 'authors': [{'first': 'G', 'middle': [], 'last': 'Cavalli', 'suffix': ''}, {'first': 'C', 'middle': ['A'], 'last': 'Dinarello', 'suffix': ''}], 'year': 2018, 'venue': 'Immunol Rev', 'volume': '281', 'issn': '1', 'pages': '179--190', 'other_ids': {}}, 'BIBREF11': {'ref_id': 'b11', 'title': 'Impact 687 of Fungi on Immune Responses', 'authors': [{'first': 'P', 'middle': [], 'last': 'Conti', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Tettamanti', 'suffix': ''}, {'first': 'F', 'middle': [], 'last': 'Mastrangelo', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Ronconi', 'suffix': ''}, {'first': 'I', 'middle': [], 'last': 'Frydas', 'suffix': ''}, {'first': 'S', 'middle': ['K'], 'last': 'Kritas', 'suffix': ''}], 'year': 2018, 'venue': 'Clin Ther', 'volume': '40', 'issn': '6', 'pages': '885--888', 'other_ids': {}}, 'BIBREF13': {'ref_id': 'b13', 'title': 'MyD88-mediated inflammatory responses in human aortic valve interstitial cells', 'authors': [], 'year': None, 'venue': 'Mol', 'volume': '690', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF16': {'ref_id': 'b16', 'title': 'Upregulated in Patients with Tuberculosis and Induces Macrophages Towards an 693 M2-like Phenotype', 'authors': [], 'year': 2015, 'venue': 'Scand J Immunol', 'volume': '82', 'issn': '4', 'pages': '370--379', 'other_ids': {}}, 'BIBREF17': {'ref_id': 'b17', 'title': 'Obstacles and opportunities for understanding macrophage 695 polarization', 'authors': [{'first': 'P', 'middle': ['J'], 'last': 'Murray', 'suffix': ''}, {'first': 'T', 'middle': ['A'], 'last': 'Wynn', 'suffix': ''}], 'year': 2011, 'venue': 'J Leukoc Biol', 'volume': '89', 'issn': '4', 'pages': '557--63', 'other_ids': {}}, 'BIBREF18': {'ref_id': 'b18', 'title': 'Autocrine 697 to diagnose early onset sepsis in neonates', 'authors': [{'first': 'T', 'middle': [], 'last': 'Schreiber', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Ehlers', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Heitmann', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Rausch', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Mages', 'suffix': ''}, {'first': 'P', 'middle': ['J'], 'last': 'Murray', 'suffix': ''}], 'year': 2016, 'venue': 'Clin Chim Acta', 'volume': '462', 'issn': '', 'pages': '90--95', 'other_ids': {}}, 'BIBREF19': {'ref_id': 'b19', 'title': 'IL-35 inhibits HBV antigen-706 specific IFN-γ-producing CTLs in vitro', 'authors': [{'first': 'X', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Tian', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Dong', 'suffix': ''}, {'first': 'Q', 'middle': [], 'last': 'Zhu', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Han', 'suffix': ''}], 'year': 2015, 'venue': 'Clin Sci', 'volume': '129', 'issn': '5', 'pages': '395--404', 'other_ids': {}}, 'BIBREF21': {'ref_id': 'b21', 'title': 'Nocardia brasiliensis: in vitro and in vivo growth response to steroid sex 709 hormones', 'authors': [{'first': 'P', 'middle': [], 'last': '', 'suffix': ''}], 'year': 1995, 'venue': 'Mycopathologia', 'volume': '132', 'issn': '2', 'pages': '79--85', 'other_ids': {}}, 'BIBREF22': {'ref_id': 'b22', 'title': 'A new 711 model for management of mycetoma in the Sudan', 'authors': [{'first': 'A', 'middle': [], 'last': 'Fahal', 'suffix': ''}, {'first': 'Ame', 'middle': [], 'last': 'Hassan', 'suffix': ''}, {'first': 'M', 'middle': ['E'], 'last': 'Abdel-Rahman', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Alshambaty', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Hashim', 'suffix': ''}], 'year': 2014, 'venue': 'PLoS Negl Trop Dis', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF23': {'ref_id': 'b23', 'title': 'The Surgical Treatment of Mycetoma', 'authors': [{'first': 'S', 'middle': ['H'], 'last': 'Suleiman', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Wadaella El', 'suffix': ''}, {'first': 'A', 'middle': ['H'], 'last': 'Fahal', 'suffix': ''}], 'year': None, 'venue': 'PLoS', 'volume': '714', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF25': {'ref_id': 'b25', 'title': 'Predictors of Post-716 operative Mycetoma Recurrence Using Machine-Learning Algorithms: The 717 Mycetoma Research Center Experience', 'authors': [{'first': 'A', 'middle': [], 'last': 'Wadal', 'suffix': ''}, {'first': 'T', 'middle': ['A'], 'last': 'Elhassan', 'suffix': ''}, {'first': 'H', 'middle': ['A'], 'last': 'Zein', 'suffix': ''}, {'first': 'M', 'middle': ['E'], 'last': 'Abdel-Rahman', 'suffix': ''}, {'first': 'A', 'middle': ['H'], 'last': 'Fahal', 'suffix': ''}], 'year': 2016, 'venue': 'PLoS Negl Trop Dis', 'volume': '718', 'issn': '10', 'pages': '', 'other_ids': {}}}, 'ref_entries': {'FIGREF0': {'text': \"150 form primarily by inflammasome-dependent caspase activity [18]. It is possible that mature 151 IL-1β secretion by macrophages activates IL-1R1 on macrophages, fibroblasts and epithelial 152 cells, inducing production of the CXC chemokine CXCL1/KC, which binds to CXCR2 on 153 neutrophils and mediates recruitment of neutrophils from peripheral blood to stimulate 154 inflammation at the site of mycetoma invasion. Therefore, these higher levels of IL-1β 155 cytokine advocate a crucial role in M. mycetomatis pathogenesis. 156 IL-37, which is a member of the IL-1 family, has emerged as a potent anti-inflammatory degenerative intervertebral discs [23] and Graves' disease (GD) [24]. IL-37 protein level in 164 PBMCs and dendritic cells (DCs) is up-regulated when stimulated by Toll-like receptor (TLR) 165 ligands or pro-inflammatory cytokines [25]. In vitro, overexpression of IL-37 in macrophages 166 or epithelial cells greatly inhibits the production of major pro-inflammatory cytokines such as 167 TNF-α, IL-1α, IL-1β, IL-6, IFN-γ and macrophage inflammatory protein 2 [25, 26]. In vivo, IL-168 37 transgene protects mice from lipopolysaccharide-induced shock and chemical-induced 169 colitis [19, 27]. IL-37 interferes with the innate protective anti-Candida host response by 170 reducing the production of pro-inflammatory cytokines and suppressing neutrophil 171 recruitment in response to Candida infection, resulting in increased susceptibility to 172 disseminated candidiasis [28]. Moreover, IL-37 markedly reduced inflammasome activation 173\", 'latex': None, 'type': 'figure'}, 'FIGREF2': {'text': '190 proliferation and effector functions [33, 38]. Given the direct immunosuppressive effect of IL-191 35, many studies have been conducted to evaluate its role in the development of several 192 diseases. IL-35 can suppress several types of chronic inflammatory diseases such as 193 inflammatory bowel disease [26], and decreased the severity of collagen-induced arthritis in 194 animals via enhancement of IL-10 production [39] and suppression of Th17 cells [40]. In an 195 asthma model, intra-tracheal instillation of IL-35 decreased disease severity by diminishing 196 the Th2 cell counts [41] and by reducing the production of IL-17 [42]. In bacterial infections, 197 Shen and associates [35] found that mice without IL-35 expression demonstrated an 198 improved resistance to infection with the intracellular bacterial pathogen Salmonella 199 typhimurium. In addition, IL-35 has been increased in the serum of adults and children with 200 sepsis, and administration of anti-IL-35 p35 antibodies diminished dissemination of the 201 bacteria in septic animals [43]. Similarly, tuberculous patients exhibited an increase in serum Sample collection 231 One hundred μl of blood was collected on Whatman qualitative filter paper, Grade 1, circles, 232 diam. 42.5 mm (Sigma-Aldrich Chemical Co., St. Louis, MO, USA) for the determination of 233 cytokines. The use of filter paper dried whole blood spots (DBS) for specimen collection was 234 preferred to facilitate collection, storage and transportation of specimens and it is in line with 235 the World Health Organization recommendations and also used in several previous studies 236 [46-48]. Sera were extracted from filter-paper samples as described previously in details 237 [17].', 'latex': None, 'type': 'figure'}, 'FIGREF3': {'text': \"1β, IL-37, IL-12 and IL-35 measurement 240 IL-1β, IL-37, and IL-12 were measured in the sera using commercially available enzyme-241 linked immunosorbent assay (ELISA) kits (abcam®, Cambridge, UK). Serum levels of IL-35 242 were estimated using a sandwich ELISA commercial kit (Colorful Gene Biological 243 Technology, Wuhan, China). Cytokine assays were performed in duplicates according to the 244 manufacturers' protocols. The sensitivity of Human ELISA kits for IL-1β, IL-37, IL-12 and IL-245 35 cytokines was 0.5 pg/ml.\", 'latex': None, 'type': 'figure'}, 'FIGREF4': {'text': 'were managed by SPSS version 24.0 statistical software for Windows (IBM© 249 SPSS© statistics) and appropriate statistical tests were used. The results are expressed as 250 mean ± standard deviation (SD) or median with interquartile range (IQR). Spearman 251 correlation test was used to evaluate the associations between serum IL-37 levels and 252 laboratory values as well as serum cytokine levels. For non-parametric data, comparisons', 'latex': None, 'type': 'figure'}, 'FIGREF5': {'text': 'between cytokine levels (IL-1β, IL-37, IL-12 and IL-35) stratified by the 283 lesion diameter among mycetoma patients and control group 284 The levels of cytokines (IL-1β, IL-37, IL-12 and IL-35) were constitutively correlated among 285 mycetoma patients with different lesions diameters. The levels of IL-1β were constitutively 286 positively correlated with IL-12 and lesions diameter,', 'latex': None, 'type': 'figure'}, 'FIGREF6': {'text': \"between cytokine levels (IL-1β, IL-37, IL-12 and IL-35) stratified by the 294 disease duration and control group 295 In the patients' group, the levels of cytokines (IL-1β, IL-37, IL-12 and IL-35) were 296 constitutively correlated with the duration of mycetoma infection. Levels of IL-1β showed a 297 consistent positive correlation with IL-12 and negative correlation with IL-37 and IL-35, 298\", 'latex': None, 'type': 'figure'}, 'FIGREF7': {'text': 'Analysis of the serum levels of IL-1β, IL-37, IL-12 and IL-35 within the different lesion 302 diameters among mycetoma patients and control groups 303 Circulating serum cytokine levels were determined in all mycetoma patients and were 304 compared between the different lesion diameters among mycetoma patients and healthy 305 controls. Overall, there was a significant difference in the levels of all studied cytokines 306 between the four groups (Three levels of lesion diameter and the healthy controls), (', 'latex': None, 'type': 'figure'}, 'FIGREF8': {'text': 'for circulating IL-35 among mycetoma patients. 375 The analysis of the risk factors of higher levels of IL-35 in patients of mycetoma revealed a 376 significant negative association with IL-12; as a unit increment in IL-12 decreases the levels 377 of IL-35 by 8.99 pg/ml, p-value < 0.001 (Table 9). Levels of IL-35 among the patients with a 378 mycetoma with a disease duration of ≤ 1 year had significantly decreased, p value= 0.002), 379 on average by 41.82 pg/ml compared to patients with a disease duration ≥ 5years 380 (reference group) (Table 9). However, patients with an infection duration of 2-4 years and ≥ 381 five years showed no significant difference in IL-35 levels, p-value = 0.391. Furthermore, 382 there was no significant difference (p-value = 0.49) in IL-35 levels between male and female 383 mycetoma sufferers', 'latex': None, 'type': 'figure'}, 'FIGREF9': {'text': \"represents a major health problem in many tropical and subtropical393    areas, there are no prevention or control measures for this neglected disease[22, 23]. In 394 mycetoma endemic areas, most individuals have antibodies against the causative agents.395    However only a few develop the disease [4]. Few researchers believed that patients who 396 develop mycetoma seem to be deficient in their cell-mediated immunity[6]. Hence, we 397 aimed to investigate the profiles of the pro-inflammatory (IL-1β and IL-12) and the anti-398 inflammatory immunosuppressive (IL-37 and IL-35) cytokines among mycetoma patients 399 and their association with disease characteristics. As far as we know, this is the first study 400 addressing the relation between immunosuppressive cytokines and mycetoma infection. Our 401 data showed that eumycetoma patients presented higher circulating levels of IL-1β, IL-12, 402 IL-37 and IL-35 compared to controls. Moreover, serum levels of IL-1β and IL-12 were 403 significantly decreased with increasing lesions' diameter and disease duration, whereas 404 levels of IL-37 and IL-35 were significantly higher with increasing lesions' diameter and 405 disease duration. These findings indicate that immunosuppressive cytokines like IL-37 and 406 IL-35, which could suppress cell-mediated immune responses, may have a negative impact 407 on the disease progression. 408 Data of the current study clearly showed the serum levels of IL-1β and IL-12 in eumycetoma 409 patients with different lesion size and disease duration were positively correlated with each 410 other, and negatively correlated with IL-37. It has been demonstrated that the first line of the 411 innate immune response against mycetoma infection is by phagocytes, from which 412 macrophages represent the major phagocytic cells [50]. In general, protective immunity to 413 fungal infections [51] involves activation of Toll-like receptors (TLRs) generating 53]. IL-1β and other pro-inflammatory cytokines are produced early 416 in response to fungal infections and promote phagocytosis and other means of the innate 417 immune response [54, 55]. Following inflammatory stimuli, several cell types including 418 immune and non-immune cells produce IL-37, as a protective mechanism to prevent 419 runaway inflammation and excessive tissue damage [56]. IL-37 directly inhibits generation of 420 pro-inflammatory cytokines and down-regulates macrophage cytokine release, and therefore 421 innate immunity [57, 58].Moreover, IL-37 induces macrophages towards an M2-like 422 phenotype [59]. M1 macrophages are the most critical effector cells in the innate immune 423 defence system and are characterised by high expression levels of iNOS, subsequent NO 424 production and secretion of pro-inflammatory cytokines, such as IL-1β and IL-12 [60]. 425 However, M2 macrophages secrete anti-inflammatory cytokines, such as IL-10 [61] and 426 express arginase 1, which inhibits NO production, thus rendering these cells ineffective in 427 killing infectious agents including fungal agents [61, 62]. Furthermore, DCs expressing IL-37 428 secreted higher levels of IL-10 and reduced levels of IL-1β and IL-12. Therefore, the 429 presence of IL-37 in DCs impairs their function in prime T cells and promotes their ability to 430induce Treg cells that produce IL-10, which is also a potent anti-inflammatory cytokine[30].\", 'latex': None, 'type': 'figure'}, 'FIGREF10': {'text': \"have consistently shown higher circulatory levels of IL-37 in patients of 433 mycetoma which is negatively associated with IL-1β, as a unit increment in IL-1β decreases 434 the levels of IL-37 by 35.28 pg/ml. Based on the aforementioned data, we can speculate that 435 IL-37 could play a role in damping inflammatory response in mycetoma infection which leads 436 to disease progression and this is not in the patient's favour.\", 'latex': None, 'type': 'figure'}, 'FIGREF12': {'text': 'collaboration and generous hospitality during a decade of samples 515 collection. The authors are thankful to Dr Hamza A., Dundee University, the UK for 516 improving the English language and proofreading the Manuscript.', 'latex': None, 'type': 'figure'}, 'FIGREF14': {'text': 'El Hassan AM, Fahal AH, Ahmed AO, Ismail A, Veress B. The immunopathology of 539 actinomycetoma lesions caused by Streptomyces somaliensis. Trans R Soc Trop 540 Med Hyg. 2001;95(1):89-92. 541 6. Mahgoub ES, Gumaa SA, El Hassan AM. Immunological status of mycetoma 542 patients. Bull Soc Pathol Exot Filiales. 1977;70(1):48-54. 543 7. Al Dawi AF, Mustafa MI, Fahal AH, Hassan AME, Bakhiet SM, Mahgoub ES, et al.544 The association of HLA-DRB1 and HLA-DQB1 alleles and the occurrence of 545 eumycetoma in Sudanese patients. Khartoum Medical Journal.2013; 6(3):923 -929. 13-rich environment correlate with inflammation severity in experimental 548 actinomycetoma by Nocardia brasiliensis. Pathog Dis. 2013;67(3):214-20. . Figueiredo RT, Fernandez PL, Dutra FF, González Y, Lopes LC, Bittencourt VC, et 553 al. TLR4 recognizes Pseudallescheria boydii conidia and purified rhamnomannans. 554 J Biol Chem. 2010; 285(52):40714-23. 555 11. Salinas-Carmona MC, Torres-López E. Role of passive humoral immunity in 556 experimental mycetoma by Nocardia brasiliensis. Ann N Y Acad Sci. 1996 ;797:263-557 5. 558 12. Welsh O, Vera-Cabrera L, Welsh E, Salinas MC. Actinomycetoma and advances in 559 its treatment. Clin Dermatol. 2012;30(4):372-81. 560 13. Deem RL, Doughty FA, Beaman BL. Immunologically specific direct T lymphocyte-561 mediated killing of Nocardia asteroides. J Immunol. 1983;130(5):2401-6. 562 14. Trevino-Villarreal JH, Vera-Cabrera L, Valero-Guillén PL, Salinas-Carmona MC. 563 Nocardia brasiliensis cell wall lipids modulate macrophage and dendritic responses 564 that favor development of experimental actinomycetoma in BALB/c mice. Infect 565 Immun. 2012; 80(10):3587-601. 566 15. Di Iorio A, Ferrucci L, Sparvieri E, Cherubini A, Volpato S, Corsi A, et al. Serum IL-567 1beta levels in health and disease: a population-based study. Cytokine. the antifungal azoles itraconazole, fluconazole, ketoconazole and 721 miconazole on cytokine gene expression in human lymphoid cells.', 'latex': None, 'type': 'figure'}, 'TABREF0': {'text': 'Interleukin-12 (IL-12) is frequently denoted as a B cell cytokine, although it is mainly formed178 by innate immune cells, comprising epithelial cells, DCs, and macrophages [31]. IL-12 is a multimer that plays a fundamental role in immune regulation and is extensively involved in 180 infections. It binds to the heterodimeric IL-12 receptor, which is principally present in T cells 181 and on natural killer (NK) cells [31]. IL-12 induces Th1 responses [32], which consequently 182 increase the cytotoxic cytokines, in addition to IFN-γ by T cells [31, 32]. IL-35 is a recently identified heterodimeric cytokine which belongs to the IL-12 cytokine family, composed of the subunits of IL-27; β chain Epstein-Barr-virus (EBV)-induced gene 3 (Ebi3) and IL-12 α chain p35 [33, 34]. IL-35 is a potent immunosuppressive cytokine 186 produced by T regs, regulatory B cells (B regs ) [35], DCs [36], and to a lesser extent, by endothelial cells, smooth muscle cells, and monocytes [37]. The biological effect of IL-35 is', 'latex': None, 'type': 'table'}, 'TABREF1': {'text': 'between the groups were performed using the Kruskal-Wallis test. One-way ANOVA was used for parametric data. General linear models were used to assess the risk factors for circulating IL-37 pg/ml and IL-35 among mycetoma patients with different disease duration and lesions size of mycetoma infection adjusted with other variables. A test with a p-value <0.05 was considered statistically significant.This study was approved by the Ethics Committee of Soba University Hospital, Khartoum, Sudan. Written informed consent was taken from all the participants before enrolment in the study. The work described here was performed in accordance with the Declaration of This study included 70 confirmed mycetoma patients and 70 healthy controls; gender and age-matched. Fifty-six patients (80%) were males and 14 (20%) were females. Their ages ranged between 12 and 77 years, and the median age was 25.5 years. The common age patients and 12 (17.1%) healthy controls] were farmers. Due to the prolonged illness and disability, 12 (17.1%) patients had lost their jobs while only five (7.1%) healthy controls were unemployed. A total of six individuals [5 (7.1%) patients and 1(1.4%) healthy controls] of the patients were housewives. A total of 13 individuals [1(1.4%) patients and 12 (17.1%) healthy', 'latex': None, 'type': 'table'}, 'TABREF2': {'text': 'Fahal A, Mahgoub ES, Hassan AME, Abdel-Rahman ME. Mycetoma in the Sudan: an update from the mycetoma research centre, University of Khartoum, Sudan.PLoS Negl Trop Dis. 2015; 9: e0003679.', 'latex': None, 'type': 'table'}, 'TABREF5': {'text': 'Correlations between the serum levels of (IL-1β, IL-37, IL-12 and IL-35) and the duration of', 'latex': None, 'type': 'table'}, 'TABREF6': {'text': 'Analysis of the serum cytokine levels of (IL-1β, IL-37, IL-12 and IL-35) within the different *P values are derived from non-parametric method; Kruskal Wallis test.The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/502609 doi: bioRxiv preprint', 'latex': None, 'type': 'table'}, 'TABREF7': {'text': 'Analysis of the serum cytokine levels of (IL-1β, IL-37, IL-12 and IL-35) among mycetoma patient stratified by different duration of mycetoma infection compared to controls group 803 804 *P values are derived from non-parametric method; Kruskal Wallis test.The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/502609 doi: bioRxiv preprint', 'latex': None, 'type': 'table'}, 'TABREF8': {'text': 'Risk factors for circulating cytokines IL-37 pg/ml in mycetoma patients with different lesion ‡B (95%CI) adjusted with lesion diameter, gender, age groups and medical treatments.The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/502609 doi: bioRxiv preprintThe copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/502609 doi: bioRxiv preprint', 'latex': None, 'type': 'table'}, 'TABREF9': {'text': 'Risk factors for circulating cytokines IL-37 pg/ml among mycetoma patients with different duration of mycetoma infection 832 833 ‡B (95%CI) adjusted with Duration of mycetoma infection, gender, age groups and medical treatments.', 'latex': None, 'type': 'table'}, 'TABREF10': {'text': 'Risk factors for circulating cytokines IL-35 pg/ml among mycetoma patients with different839 ‡B (95%CI) adjusted with Duration of mycetoma infection, gender, age groups and medical treatments.', 'latex': None, 'type': 'table'}}, 'back_matter': [{'text': 'The authors are grateful to the patients, controls and their families and the staff at the 513 Mycetoma Research Centre at Soba University Hospital, University of Khartoum-Sudan, for', 'cite_spans': [], 'ref_spans': [], 'section': '512'}]}\n",
      "{'paper_id': 'ad146e228bda4e5a3d56b28a3a1acbd01e0b323e', 'metadata': {'title': 'Modelling the epidemic 2019-nCoV event in Italy: a preliminary note', 'authors': [{'first': 'Marco', 'middle': ['Claudio'], 'last': 'Traini', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Università degli Studi di Trento', 'location': {'addrLine': 'Via Sommarive 14', 'postCode': 'I-38123', 'settlement': 'Trento-Povo', 'country': 'Italy'}}, 'email': ''}, {'first': 'Carla', 'middle': [], 'last': 'Caponi', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Azienda Ospedaliero-Universitaria Piazzale Gambuli 1', 'location': {'postCode': '06132', 'settlement': 'Perugia', 'country': 'Italy'}}, 'email': ''}, {'first': 'Giuseppe', 'middle': [], 'last': 'Vittorio', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'De', 'middle': [], 'last': 'Socio', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Azienda Ospedaliero-Universitaria Piazzale Gambuli 1', 'location': {'postCode': '06132', 'settlement': 'Perugia', 'country': 'Italy'}}, 'email': ''}]}, 'abstract': [], 'body_text': [{'text': 'Late December 2019 health facilities reported cluster of patients with pneumonia of unknown origin epidemiologically linked to a seafood and wet animal wholesale market in Wuhan China. A new previous unknown betacoronavirus was discovered and identified as the etiologic agent of this new pneumonia diagnosed in Wuhan [1] . Coronaviruses (CoV) are a large family of viruses that cause illness ranging from the common cold to more severe diseases. The new virus was provisory named 2019-nCoV by World Health Organizzation on 12 January 2020 [2] and sun after the Coronavirus Study Group (CSG) based on phylogeny, taxonomy and established practice, formally recognized this virus as a sister to severe acute respiratory syndrome coronaviruses (SARS-CoVs) of the species Severe acute respiratory syndrome-related coronavirus and designates it as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) [3] . The outbreak of the SARS-CoV-2 due to the global spread has been defined pandemia on March 11, 2020. In Europe Italy is becoming a particularly alarming and interesting place to study the evolution of the epidemic also thanks to the rapid reaction of the Italian Health organizations and the relevant control measure to prevent transmission adopted. Based on Chinese experience and the estimation of transmission rate published by Tang B et al.. [4] . Here we develop models for the evolution of the SARS-CoV-2 during the early stages of transmission in Italy; models which may be useful for inference, forecasting or scenario analysis. Despite the fact that epidemic is changing rapidly and our results have been considered preliminary, the models we are using are considered strongly predictive and useful for the interpretation of such an unexpected event in a country like Italy.', 'cite_spans': [{'start': 318, 'end': 321, 'text': '[1]', 'ref_id': 'BIBREF0'}, {'start': 904, 'end': 907, 'text': '[3]', 'ref_id': 'BIBREF1'}, {'start': 1356, 'end': 1359, 'text': '[4]', 'ref_id': 'BIBREF2'}], 'ref_spans': [], 'section': 'I. INTRODUCTION'}, {'text': 'A. The SEIR model and its limiting approximation', 'cite_spans': [], 'ref_spans': [], 'section': 'II. DETERMINISTIC MODELS'}, {'text': 'In the SEIR framework [5] , individuals in the population are classified according to their infectious status: Susceptible (S), Exposed (E) (infected but not infectious), Infectious (I) and Recovered (R). In the case of a new infection (as in the present case) the population has no prior immunity, consequently the population starts out at the disease-free equilibrium (S ≈ N, E = 0, I = 0, R = 0), where N is the total population size, and the dynamics are determined by the following equations describing the rates of change of each simplified classes:', 'cite_spans': [{'start': 22, 'end': 25, 'text': '[5]', 'ref_id': 'BIBREF3'}], 'ref_spans': [], 'section': 'II. DETERMINISTIC MODELS'}, {'text': '(1)', 'cite_spans': [], 'ref_spans': [], 'section': 'II. DETERMINISTIC MODELS'}, {'text': 'After introduction of an infectious individual (and if the basic reproductive ratio, R 0 = β/γ, is greater than 1), the infection develops through the population by contacts between susceptible and infected individuals to sustain further transmission. Neglecting background birth and death processes (because of the short period analyzed w.r.t. the typical vital periods), the population eventually reaches the following state (S = S ∞ , E = 0, I = 0,', 'cite_spans': [], 'ref_spans': [], 'section': 'II. DETERMINISTIC MODELS'}, {'text': 'The SIR model is a limiting case of the previous framework and assumes E + I → I at any time, without dis-. CC-BY 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'II. DETERMINISTIC MODELS'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.14.20034884 doi: medRxiv preprint tinguishing the classes E and I. The previous equations reduce to', 'cite_spans': [], 'ref_spans': [], 'section': 'II. DETERMINISTIC MODELS'}, {'text': 'In Fig. 1 we show the available data [6, 7] for the initial period February 20th -March 11th (21 days) and their comparison with the simplest SIR approximation (5), (6) , and (7) (iteratively solved by means of a MATLAB code) for a first intuitive analysis. The continuous black curve shows the solution assuming a I(0) = 3 number of infectious individuals at day=0, exactly equivalent to the official detected cases at the same day. No attention is paid to a best fit of the data, also because they are rather sparse, however the quality of the simple (SIR) model is evident also at the first stages of the epidemic. The parameter values sound also reasonable: with a recovering average time T r = 7 days (γ = 1/T r ) and R 0 = 3.4 and no further corrections: a brilliant example of the predictive force of the SIR model. However Fig. 1 has a more specific aim: showing the rather delicate rôle played by a possible underestimation of the infectious individual at the early stages of the infection outbreak. In fact one can assumes a different (possible) scenario based on the difficulty of a precise estimation of the initial number of infectious individuals. Such a possible scenario is illustrated by the the full blue curve showing a prediction which results from a misleading estimation of I(0). The effective number of initial infectious is 30 times larger (I(0) = 90), but the curve has been translated to a fictitiousĪ(0) = 3, the reveled number, from the dashed mathematical result. A change in R 0 from R 0 = 3.4 to R 0 = 1.7 would give a reason- able fit of the early stage of the outbreak. The effect is rather well known as discussed in ref. [8] in the case of the epidemic event in China. In that case a proposed exponential fit of the precocious data (January 10th to January 24th) results in a values of R 0 ≈ 3.58 which is reduced to R 0 ≈ 2.24 is the associated initial condition on I(0) is multiplied by an increasing factor from 2 to 8. In Fig. 2 we show the time behavior of the infectious population within the simple SIR model of Eqs. (5)-(7) (T r = 1/γ = 7 days, R 0 = 3.4) both in logarithmic (upper panel) and linear scale (lower panel). The data are the data of Fig. 1 .', 'cite_spans': [{'start': 37, 'end': 40, 'text': '[6,', 'ref_id': None}, {'start': 41, 'end': 43, 'text': '7]', 'ref_id': None}, {'start': 165, 'end': 168, 'text': '(6)', 'ref_id': None}, {'start': 1656, 'end': 1659, 'text': '[8]', 'ref_id': 'BIBREF6'}], 'ref_spans': [{'start': 3, 'end': 9, 'text': 'Fig. 1', 'ref_id': 'FIGREF0'}, {'start': 831, 'end': 837, 'text': 'Fig. 1', 'ref_id': 'FIGREF0'}, {'start': 1961, 'end': 1967, 'text': 'Fig. 2', 'ref_id': 'FIGREF1'}, {'start': 2190, 'end': 2196, 'text': 'Fig. 1', 'ref_id': 'FIGREF0'}], 'section': 'II. DETERMINISTIC MODELS'}, {'text': 'The exponential behavior predicted by the model in the growing part of the outbreak description is evident and can be compared with the recent fit to the data by . CC-BY 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'II. DETERMINISTIC MODELS'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.14.20034884 doi: medRxiv preprint Bucci and Marinari in ref. [9] . While the present simplified SIR model predict a doubling infectious population each (roughly) 2 days, the Roma fit [9] gives an estimation of 2.6 days (see Fig. 3 ).', 'cite_spans': [{'start': 171, 'end': 174, 'text': '[9]', 'ref_id': 'BIBREF7'}, {'start': 293, 'end': 296, 'text': '[9]', 'ref_id': 'BIBREF7'}], 'ref_spans': [{'start': 334, 'end': 340, 'text': 'Fig. 3', 'ref_id': None}], 'section': 'II. DETERMINISTIC MODELS'}, {'text': 'The model [4] is a generalized SEIR-type epidemiological model which incorporates appropriate compartments relevant to intervention such as quarantine, isolation and treatment. The population is stratified in Susceptible ', 'cite_spans': [{'start': 10, 'end': 13, 'text': '[4]', 'ref_id': 'BIBREF2'}], 'ref_spans': [], 'section': 'B. A Quarantined model with isolation'}, {'text': 'The authors of ref. [4] have calibrated the model on the data of the 2019-nCoV as emerged in Wuhan the last two months, therefore in a situations that has the same basic parameters. In table I The results are shown in Figs 4 and 5. The drastic reduction of the infectious population is evident as well as the long tail of the distribution.', 'cite_spans': [{'start': 20, 'end': 23, 'text': '[4]', 'ref_id': 'BIBREF2'}], 'ref_spans': [{'start': 185, 'end': 192, 'text': 'table I', 'ref_id': 'TABREF0'}], 'section': 'B. A Quarantined model with isolation'}, {'text': 'The quarantine imposed to the infectious individuals, the effects of an intensive contact tracing and isolation reduce the maximum value of the distribution by a factor ∼ 26 (see Fig. 5 ).', 'cite_spans': [], 'ref_spans': [{'start': 179, 'end': 185, 'text': 'Fig. 5', 'ref_id': 'FIGREF3'}], 'section': 'B. A Quarantined model with isolation'}, {'text': 'The present calculation strongly supports the interventions and restrictions adopted in Italy to reduce the outbreak of the infectious population of the 2019-nCoV. . CC-BY 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'III. CONCLUSIONS'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.14.20034884 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'III. CONCLUSIONS'}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'A Novel Coronavirus from Patients with Pneumonia in China', 'authors': [{'first': 'N', 'middle': [], 'last': 'Zhu', 'suffix': ''}], 'year': 2020, 'venue': 'China Novel Coronavirus Investigating and Research Team', 'volume': '382', 'issn': '', 'pages': '727--733', 'other_ids': {'PMID': ['31978945']}}, 'BIBREF1': {'ref_id': 'b1', 'title': 'Severe acute respiratory syndrome-related coronavirus: The species and its viruses a statement of the Coronavirus Study Group', 'authors': [{'first': 'A', 'middle': ['E'], 'last': 'Gorbalenya', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.02.07.937862']}}, 'BIBREF2': {'ref_id': 'b2', 'title': 'Estimation of the Transmission Risk of the 2019-nCoV and Its Implication for Public Health Interventions', 'authors': [{'first': 'Biao', 'middle': [], 'last': 'Tang', 'suffix': ''}], 'year': 2020, 'venue': 'J. Clin. Med', 'volume': '9', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.3390/jcm9020462']}}, 'BIBREF3': {'ref_id': 'b3', 'title': 'Modeling Infectious Diseases in Humans and Animals, Matthew James Keeling and Pejman Rohani', 'authors': [], 'year': 2008, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF6': {'ref_id': 'b6', 'title': 'Preliminary estimation of the basic reproduction number of novel coronavirus (2019-nCoV) in China, from 2019 to 2020: A data-driven analysis in the early phase of the outbreak', 'authors': [{'first': 'Shi', 'middle': [], 'last': 'Zhao', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '92', 'issn': '', 'pages': '214--217', 'other_ids': {'DOI': ['10.1016/j.ijid.2020.01.050']}}, 'BIBREF7': {'ref_id': 'b7', 'title': \"Considerazioni sull'evoluzione in corso dell'epidemia da nuovo coronavirus SARS-COV-2 in Italia\", 'authors': [{'first': 'Enrico', 'middle': ['M'], 'last': 'Bucci', 'suffix': ''}, {'first': 'Enzo', 'middle': [], 'last': 'Marinari', 'suffix': ''}], 'year': 2020, 'venue': 'Patto Trasversale per la Scienza', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}}, 'ref_entries': {'FIGREF0': {'text': '(color on line). The data cases as detected in Italy in the period February 20th (day=0) and March 11th (day = 20) (Ministero della Salute[6,7]), compared with the prediction of few deterministic models, see text.', 'latex': None, 'type': 'figure'}, 'FIGREF1': {'text': '(color on line) upper panel Behavior (in logarithmic scale) of the number of infectious individuals in Italy within the SIR model and the comparison with available data till March 11th, 2020. lower panel: As in the upper panel but in linear scale to better appreciate the global behavior and the effective time to recover.', 'latex': None, 'type': 'figure'}, 'FIGREF2': {'text': '(color on line). The model predictions of Fig. 2 are compared with an exponential fit (full red line) I ∼ exp (t days /2.994) and compared with the fit proposed in ref.[9] (dashed red line). (color on line). Number of infectious individuals for the 2019-nCoV outbreak in Italy. The SEIR model predictions (full black line) are compared with the simplified results of the SIR model of Fig. 3 as well as the proposed exponential fit of Bucci and Marinari [9] (see Figure caption). The maximum values are drastically reduced, at the cost of a long tail in the distribution.', 'latex': None, 'type': 'figure'}, 'FIGREF3': {'text': '(color on line). The time evolution of the infectious population in the Italian outbreak. The linear scale emphasize the drastic reduction induced by isolation, quarantine and intense tracing as described by the recent formulation of a SEIR model based on the Wuhan database (see ref.[4]).(S), exposed (E), infectious but not yet symptomatic (pre-symptomatic) (A), infectious with symptoms (I), hospitalized (H) and recovered (R). Further stratification includes quarantined susceptible (S q ), isolated exposed (E q ) and isolated infected (I q ) compartments. The equations of the model are explicitly shown in Eqs. (8)-(15).', 'latex': None, 'type': 'figure'}, 'FIGREF4': {'text': 'dS dt = −(βc + cq(1 − β)) S (I + θ A) N + λ S q ; (8)', 'latex': None, 'type': 'figure'}, 'FIGREF5': {'text': 'the values of the model parameters as tuned by Tang et al. on the outbreak event at Wuhan are summarized and the initial conditions imposed to the populations localized in Wuhan and in Italy.', 'latex': None, 'type': 'figure'}, 'TABREF0': {'text': 'Parameters of the SEIR model description of the Wuhan outbreak. The contact rate Probability transmission has been reduced by a factor 0.14 to describe the data of the Italian event. In addition the value θ = 1 in Eqs. (8)-(15) has been assumed to be to reduce the model dependence.', 'latex': None, 'type': 'table'}, 'TABREF1': {'text': '', 'latex': None, 'type': 'table'}}, 'back_matter': [{'text': 'We summarize in the table I of the present appendix the parameters of the SEIR-type model as proposed by Tang et al. in ref. [4] . Table II is devoted to summarize the initial conditions imposed to the SEIR solutions in the numerical calculation.', 'cite_spans': [{'start': 125, 'end': 128, 'text': '[4]', 'ref_id': 'BIBREF2'}], 'ref_spans': [{'start': 131, 'end': 139, 'text': 'Table II', 'ref_id': None}], 'section': 'Appendix A: Parameters and tables'}]}\n",
      "{'paper_id': 'adb09e4e5c7331b2aa661b3d3bb0a643e00d11bc', 'metadata': {'title': 'Evidence of recombination in coronaviruses implicating pangolin origins of nCoV- 2019', 'authors': [{'first': 'Matthew', 'middle': ['C'], 'last': 'Wong', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Baylor College of Medicine', 'location': {'postCode': '77030', 'settlement': 'Houston', 'region': 'TX', 'country': 'USA'}}, 'email': ''}, {'first': 'Sara', 'middle': ['J Javornik'], 'last': 'Cregeen', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Baylor College of Medicine', 'location': {'postCode': '77030', 'settlement': 'Houston', 'region': 'TX', 'country': 'USA'}}, 'email': ''}, {'first': 'Nadim', 'middle': ['J'], 'last': 'Ajami', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'The University of Texas MD Anderson Cancer Center', 'location': {'postCode': '77030', 'settlement': 'Houston', 'region': 'Texas', 'country': 'USA'}}, 'email': ''}, {'first': 'Joseph', 'middle': ['F'], 'last': 'Petrosino', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Baylor College of Medicine', 'location': {'postCode': '77030', 'settlement': 'Houston', 'region': 'TX', 'country': 'USA'}}, 'email': 'jpetrosi@bcm.edu'}]}, 'abstract': [{'text': 'A novel coronavirus (nCoV-2019) was the cause of an outbreak of respiratory illness detected in Wuhan, Hubei Province, China in December of 2019. Genomic analyses of nCoV-2019 determined a 96% resemblance with a coronavirus isolated from a bat in 2013 (RaTG13); however, the receptor binding motif (RBM) of these two genomes share low sequence similarity. This divergence suggests a possible alternative source for the RBM coding sequence in nCoV-2019. We identified high sequence similarity in the RBM between nCoV-2019 and a coronavirus genome reconstructed from a viral metagenomic dataset from pangolins possibly indicating a more complex origin for nCoV-', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}], 'body_text': [{'text': 'An outbreak of respiratory illness caused by a novel coronavirus (nCoV-2019) 1 , first identified in Wuhan China, has resulted in over twenty thousand confirmed cases worldwide (WHO Situation Report, February 5 2020). Data available to date has led to the hypothesis that the outbreak strain originated from bats. The nCoV-2019 has been reported to share 96% sequence identity to the RaTG13 genome, a coronavirus isolated from an intermediate horseshoe bat (Rhinolophus affinis) 2 . RaTG13 is the most closely related coronavirus genome to nCoV-2019, however other viral strains may have contributed to its genomic content. To explore this hypothesis, we examined several viral metagenomic datasets for potential coronaviruses that could harbor RBMs closer to nCoV-2019. We report evidence of a coronavirus recovered from a Malayan pangolin (Manis javanica) viral metagenomic dataset 3 that shares a higher sequence identity at a crucial segment of the genome involved in host infection ( Figure 1 ). This result indicates a potential recombination event and a more complex origin for the nCoV-2019 strain.', 'cite_spans': [{'start': 77, 'end': 78, 'text': '1', 'ref_id': 'BIBREF0'}], 'ref_spans': [{'start': 989, 'end': 997, 'text': 'Figure 1', 'ref_id': 'FIGREF0'}], 'section': ''}, {'text': 'The coronavirus spike protein is a trimeric protein that mediates virus entry into the host cell. It is composed of two domains: S1 and S2. The S1 domain contains two subdomains: S1-NTD and S1-CTD, also known as the receptor binding domain (RBD). The S2 domain encodes the stalk of the spike protein and is highly conserved across the SARS-like coronaviruses 4 . In order to infect a host cell, the RBD must first bind to a surface protein on the host cell. In the case of SARS-CoV and nCoV-2019, this is the angiotensin converting enzyme 2 (ACE2) receptor, expressed in human airway epithelia as well as lung parenchyma among other tissues. Following RBD binding, the S1/S2 junction must be cleaved by a surface protease 4 . Zhou et al. (2020) 2 have reported noticeable decreased amino acid homology between the RBD of the spike protein in RaTG13 and nCoV-2019. A further analysis of the RBD in RaTG13 showed the majority of the divergence was restricted to amino acid residues 435 to 510 (75% nucleotide identity and 78% amino acid identity). This region corresponds to the receptor binding motif (RBM), which is responsible for host specificity. Such divergence at the RBM could indicate the possibility of an alternate source for the RBM coding sequence in nCoV-2019. To explore this, we examined several viral metagenomic datasets for potential coronaviruses that could harbor RBMs closer to nCoV-2019.', 'cite_spans': [{'start': 359, 'end': 360, 'text': '4', 'ref_id': 'BIBREF3'}], 'ref_spans': [], 'section': ''}, {'text': 'Using VirMAP 5 , we aimed to reconstruct coronavirus genomes to compare recovered RBM sequences to that of nCoV-2019.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'Viral metagenomic datasets of hosts that could potentially harbor coronaviruses were downloaded from the NCBI BioProject database. These included PRJNA573298 3 (pangolin), PRJNA597258 (fruit bats) and PRJNA379515 (bats). The coronavirus genome (recovered from bat, GenBank accession number MG772933.1 6 ) reported to be the most homologous to the nCoV-2019 outbreak strain prior to the publication the RaTG13 (GenBank accession number MN996532) was also downloaded.', 'cite_spans': [], 'ref_spans': [], 'section': 'Data used in the study'}, {'text': 'Sequences from each project were trimmed (minimum quality score 20 and minimum length 75bp) and filtered for Illumina adapter and PhiX presence using bbduk.sh v38.71 7 . Prior to analysis all sequences were filtered against a the relevant host reference genome (pangolin ManJav1.0, PRJNA256023 or bat mRhiFer1_v1.p, PRJNA489106) using bbmap.sh v38.71(k=15 and minid=0.9) 7 . After trimming and filtering, sequence reads were processed using VirMAP 5 to obtain and identify coronavirus genomes present in the viral metagenomic dataset. Two coronavirus genomes sharing high homology to nCoV-2019 were recovered from the pangolin dataset in two samples (SRR10168377 and SRR10168378).', 'cite_spans': [], 'ref_spans': [], 'section': 'Analysis'}, {'text': 'The two genomes were merged using the easymerge.pl subcommand from VirMAP to create the final pangolin-associated coronavirus (Pangolin-CoV) genome.', 'cite_spans': [], 'ref_spans': [], 'section': 'Analysis'}, {'text': 'All further analysis was done using four coronaviruses genome sequences: Pangolin-CoV, RaTG13, Bat-CoV and nCoV-2019. They were pairwise aligned on NCBI using the blastn and blastx author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Analysis'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.07.939207 doi: bioRxiv preprint algorithms 8, 9 and sequence homology plots were calculated using RDP4 beta 99 10 . Distances of the selected coronavirus genomes relative to nCov-2019 were calculated using a 300bp window sliding every 10bp across the entire genome length. Sequence homology plots were visualized using R version 3.6.2 11 and the ggplot2 visualization package 12 . The RBM segments as well as the Pro-Arg-Arg-Ala (PRRA) amino acid insertion region were excised from the full genome and pairwise aligned on NCBI using blastn and blastx. Multiple alignments of the both these regions were done using MAFFT v7.313 13 and visualized with MEGAX v10.1.7 14 .', 'cite_spans': [{'start': 153, 'end': 155, 'text': '8,', 'ref_id': 'BIBREF7'}, {'start': 156, 'end': 157, 'text': '9', 'ref_id': 'BIBREF8'}, {'start': 221, 'end': 223, 'text': '10', 'ref_id': 'BIBREF9'}, {'start': 486, 'end': 488, 'text': '12', 'ref_id': 'BIBREF11'}, {'start': 737, 'end': 739, 'text': '13', 'ref_id': 'BIBREF12'}], 'ref_spans': [], 'section': 'Analysis'}, {'text': 'Of the three viral metagenomic datasets inspected, only the pangolin dataset contained coronavirus genomes sharing significant homology with the outbreak strain. Two samples in this dataset yielded two partial coronavirus genomes that overlapped >8.4kb at >99% nucleotide identity with each other.', 'cite_spans': [], 'ref_spans': [], 'section': 'Results'}, {'text': 'After merging, the final Pangolin-CoV draft genome was 90.5% similar to nCoV-2019 (median nucleotide identity). Figure 1 shows the sequence homology of Pangolin-CoV, RaTG13-CoV and Bat-CoV relative to nCoV-2019 ( Figure 1A shows the homology across the entire genome length). The multiple alignments across the RBM segments revealed an 89% nucleotide ( Figure 1B ) and 98% amino acid identity ( Figure 1C ) of the Pangolin-CoV compared to nCoV-2019. The sequence homology plots showed that the Pangolin-CoV was more similar to nCoV-2019 at the RBD region than the RaTG13-CoV and Bat-CoV ( Figure 1B) . The multiple alignment comparing the PRRA insertion region at the S1/S2 junction shows the insertion present in nCoV-2019 and missing in the other three CoV genomes ( Figure 2 ).', 'cite_spans': [], 'ref_spans': [{'start': 112, 'end': 120, 'text': 'Figure 1', 'ref_id': 'FIGREF0'}, {'start': 213, 'end': 222, 'text': 'Figure 1A', 'ref_id': 'FIGREF0'}, {'start': 353, 'end': 362, 'text': 'Figure 1B', 'ref_id': 'FIGREF0'}, {'start': 395, 'end': 404, 'text': 'Figure 1C', 'ref_id': 'FIGREF0'}, {'start': 589, 'end': 599, 'text': 'Figure 1B)', 'ref_id': 'FIGREF0'}, {'start': 769, 'end': 777, 'text': 'Figure 2', 'ref_id': 'FIGREF1'}], 'section': 'Results'}, {'text': 'and MERS 16 are believed to have arisen from coronaviruses recombining in civet cats and camels, respectively. While early indications for nCoV-2019 showed that the virus may have jumped directly author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Recent coronavirus zoonotic transmissions have involved non-bat intermediates. Indeed, both SARS 15'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.07.939207 doi: bioRxiv preprint from bats to humans 2 , here we present evidence that nCoV-2019 likely arose from yet another recombined coronavirus in an intermediate host. A dataset describing the viral diversity in Malayan pangolins 3 yielded a coronavirus genome that shares 89% nucleotide ( Figure 1A and 1B) and 98% amino acid ( Figure 1C ) identity across the same RBM segment with nCoV-2019. Additionally, the RaTG13 genome only shares one out of five key amino acids involved in receptor binding motif 2 , whereas the pangolin coronavirus shares all five key residues ( Figure 1C) . It therefore appears that the RBM in nCoV-2019 was introduced during a recombination event between the strain of coronavirus found in the pangolin and RaTG13. Given the heavy selective pressure under which the RBM is constrained, it is unlikely that RaTG13 acquired the mutations found in the pangolin coronavirus genome through random chance. We posit that the homology at the RBM between nCoV-2019 and a pangolin coronavirus may have arisen from a recombination event based on sequence analysis presented herein.', 'cite_spans': [], 'ref_spans': [{'start': 406, 'end': 415, 'text': 'Figure 1A', 'ref_id': 'FIGREF0'}, {'start': 445, 'end': 454, 'text': 'Figure 1C', 'ref_id': 'FIGREF0'}, {'start': 689, 'end': 699, 'text': 'Figure 1C)', 'ref_id': 'FIGREF0'}], 'section': 'Recent coronavirus zoonotic transmissions have involved non-bat intermediates. Indeed, both SARS 15'}, {'text': 'Amino acid alignment between pangolin coronavirus and nCoV-2019 at the S1-NTD subdomain results in low sequence similarity. However, increased similarity is observed from the RBD through the S2 domain ( Figure 1A-B) . This suggests a possible recombination event between a bat coronavirus and the pangolin coronavirus at the S1-NTD and RBD junction. While there is high conservation at the amino-acid level between the pangolin coronavirus and nCoV-2019, this is not reflected in the nucleotide identity, suggesting that the proposed recombination event occurred in the more distant past, allowing subsequent time for genetic drift. One such mutation could be the insertion of the amino acid residues PRRA near the S1/S2 junction which induces a furin cleavage motif. The secondary requirement for entry into the host cell is proteolytic cleavage of the S1/S2 domains in order to mediate membrane fusion post ACE2 binding 4 , and introduction of the furin cleavage motif may be another factor in nCoV-2019 being able to replicate in humans ( Figure 2 ). author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [{'start': 203, 'end': 215, 'text': 'Figure 1A-B)', 'ref_id': 'FIGREF0'}, {'start': 1042, 'end': 1050, 'text': 'Figure 2', 'ref_id': 'FIGREF1'}], 'section': 'Recent coronavirus zoonotic transmissions have involved non-bat intermediates. Indeed, both SARS 15'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.07.939207 doi: bioRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'Recent coronavirus zoonotic transmissions have involved non-bat intermediates. Indeed, both SARS 15'}, {'text': 'The main limitations of this analysis are the lack of additional viral datasets arising from pangolins or the availability of coronavirus genomes isolated from pangolins that could be used Close up of the S1 domain with surrounding regions. Grey boxes highlight the S1-NTD subdomain and the S1-CTD or RBD subdomain. In the RBD subdomain the homology of the Pangolin-CoV genome exceeds that of RaTG13-CoV as compared to the nCoV-2019 strain. Panel C: Multiple alignment of the receptor biding motif at the amino acid level. Boxes highlight the key amino acid residues involved in ACE2 receptor binding. The nCoV-2019 and Pangolin-CoV strains are identical at all five amino acid residues, whereas RaTG13-CoV strain differs at four out of five compared to nCoV-2019. Labels: nCoV-2019 -outbreak strain; Pangolin-CoV -pangolin coronavirus identified in this study; RaTG13-CoV -R. affinis coronavirus; Bat-CoV -bat coronavirus (MG772933.1) author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Recent coronavirus zoonotic transmissions have involved non-bat intermediates. Indeed, both SARS 15'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.07.939207 doi: bioRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'Recent coronavirus zoonotic transmissions have involved non-bat intermediates. Indeed, both SARS 15'}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'A new coronavirus associated with human respiratory disease in China', 'authors': [{'first': 'F', 'middle': [], 'last': 'Wu', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Zhao', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Yu', 'suffix': ''}, {'first': 'Y', 'middle': ['M'], 'last': 'Chen', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'Z', 'middle': ['G'], 'last': 'Song', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Hu', 'suffix': ''}], 'year': 2020, 'venue': 'Nature', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF1': {'ref_id': 'b1', 'title': 'A pneumonia outbreak associated with a new coronavirus of probable bat origin', 'authors': [{'first': 'P', 'middle': [], 'last': 'Zhou', 'suffix': ''}, {'first': 'X', 'middle': ['L'], 'last': 'Yang', 'suffix': ''}, {'first': 'X', 'middle': ['G'], 'last': 'Wang', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Hu', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Zhang', 'suffix': ''}], 'year': 2020, 'venue': 'Nature', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF2': {'ref_id': 'b2', 'title': 'Viral Metagenomics Revealed Sendai Virus and Coronavirus Infection of Malayan Pangolins (Manis javanica)', 'authors': [{'first': 'P', 'middle': [], 'last': 'Liu', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Chen', 'suffix': ''}, {'first': 'J', 'middle': ['P'], 'last': 'Chen', 'suffix': ''}], 'year': 2019, 'venue': 'Viruses', 'volume': '11', 'issn': '11', 'pages': '', 'other_ids': {}}, 'BIBREF3': {'ref_id': 'b3', 'title': 'Structure, Function, and Evolution of Coronavirus Spike Proteins', 'authors': [{'first': 'F', 'middle': [], 'last': 'Li', 'suffix': ''}], 'year': 2016, 'venue': 'Annual Review of Virology', 'volume': '3', 'issn': '1', 'pages': '237--261', 'other_ids': {}}, 'BIBREF4': {'ref_id': 'b4', 'title': 'Maximal viral information recovery from sequence data using VirMAP', 'authors': [{'first': 'N', 'middle': ['J'], 'last': 'Ajami', 'suffix': ''}, {'first': 'M', 'middle': ['C'], 'last': 'Wong', 'suffix': ''}, {'first': 'M', 'middle': ['C'], 'last': 'Ross', 'suffix': ''}, {'first': 'R', 'middle': ['E'], 'last': 'Lloyd', 'suffix': ''}, {'first': 'J', 'middle': ['F'], 'last': 'Petrosino', 'suffix': ''}], 'year': 2018, 'venue': 'Nature Communications', 'volume': '9', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF5': {'ref_id': 'b5', 'title': 'Genomic characterization and infectivity of a novel SARS-like coronavirus in Chinese bats. Emerging microbes and infections', 'authors': [{'first': 'D', 'middle': [], 'last': 'Hu', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Zhu', 'suffix': ''}, {'first': 'Ai', 'middle': ['L'], 'last': 'He', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Ye', 'suffix': ''}, {'first': 'F', 'middle': [], 'last': '', 'suffix': ''}], 'year': 2018, 'venue': '', 'volume': '7', 'issn': '', 'pages': '1--10', 'other_ids': {}}, 'BIBREF6': {'ref_id': 'b6', 'title': 'BBMap Short Read Aligner', 'authors': [{'first': 'B', 'middle': [], 'last': 'Bushnell', 'suffix': ''}], 'year': 2016, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF7': {'ref_id': 'b7', 'title': 'Basic local alignment search tool', 'authors': [{'first': 'S', 'middle': ['F'], 'last': 'Altschul', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Gish', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Miller', 'suffix': ''}, {'first': 'E', 'middle': ['W'], 'last': 'Myers', 'suffix': ''}, {'first': 'D', 'middle': ['J'], 'last': 'Lipman', 'suffix': ''}], 'year': 1990, 'venue': 'Journal of Molecular Biology', 'volume': '215', 'issn': '', 'pages': '403--410', 'other_ids': {}}, 'BIBREF8': {'ref_id': 'b8', 'title': 'BLAST+: architecture and applications', 'authors': [{'first': 'C', 'middle': [], 'last': 'Camacho', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Coulouris', 'suffix': ''}, {'first': 'V', 'middle': [], 'last': 'Avagyan', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Ma', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Papadopoulos', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Bealer', 'suffix': ''}, {'first': 'T', 'middle': ['L'], 'last': 'Madden', 'suffix': ''}], 'year': 2008, 'venue': 'BMC Bioinformatics', 'volume': '10', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF9': {'ref_id': 'b9', 'title': 'RDP4: Detection and analysis of recombination patterns in virus genomes', 'authors': [{'first': 'D', 'middle': ['P'], 'last': 'Martin', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Murrell', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Golden', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Khoosal', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Muhire', 'suffix': ''}], 'year': 2015, 'venue': 'Virus Evolution', 'volume': '1', 'issn': '5', 'pages': '', 'other_ids': {}}, 'BIBREF10': {'ref_id': 'b10', 'title': 'R: A language and environment for statistical computing. R Foundation for Statistical Computing', 'authors': [{'first': '', 'middle': [], 'last': 'R Core Team', 'suffix': ''}], 'year': 2019, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF11': {'ref_id': 'b11', 'title': 'ggplot2: Elegant Graphics for Data Analysis', 'authors': [{'first': 'H', 'middle': [], 'last': 'Wickham', 'suffix': ''}], 'year': 2016, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF12': {'ref_id': 'b12', 'title': 'MAFFT Multiple Sequence Alignment Software Version 7: Improvements in Performance and Usability', 'authors': [{'first': 'K', 'middle': [], 'last': 'Katoh', 'suffix': ''}, {'first': 'D', 'middle': ['M'], 'last': 'Standley', 'suffix': ''}], 'year': 2013, 'venue': 'Molecular Biology and Evolution', 'volume': '30', 'issn': '4', 'pages': '772--780', 'other_ids': {}}, 'BIBREF13': {'ref_id': 'b13', 'title': ':25049. author/funder. All rights reserved. No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the', 'authors': [{'first': 'Z', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Shen', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Gu', 'suffix': ''}], 'year': 2016, 'venue': 'Scientific Reports', 'volume': '6', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.02.07.939207']}}}, 'ref_entries': {'FIGREF0': {'text': 'for multiple genome alignments with nCoV-2019. In addition, the current analysis only focuses on the spike protein of nCoV-2019 without exploring additional recombination events that may lead to changes in host tropism. Nonetheless, the identification of a possible recombination event in the host-receptor binding protein of nCoV-2019, suggesting an intermediate host, supports a call for continuous surveillance of species known to be infected by this group of viruses, including pangolins, as well as all small mammals sold in such wet markets as the Huanan Seafood Wholesale Market in Wuhan to better understand and characterize zoonotic transmission events. Nucleotide and amino acid identity across three coronavirus genomes relative to the outbreak strain (nCoV-2019). Panel A: Distances of selected coronavirus genomes relative to nCov-2019 across the entire genome length. The gaps in the Pangolin-CoV are a result of the incompleteness of the genome used for comparison. Grey box indicates S1 and S2 domains of the spike protein. Panel B:', 'latex': None, 'type': 'figure'}, 'FIGREF1': {'text': 'Multiple alignment at the amino acid level of PRRA insertion region and flanking areas across three coronavirus genomes relative to the outbreak strain (nCoV-2019). Box show the PRRA insertion present in nCoV-2019 and missing in the other three CoV genomes.', 'latex': None, 'type': 'figure'}}, 'back_matter': [{'text': 'Research reported in this publication was supported by the National Institute Of Allergy And Infectious ', 'cite_spans': [], 'ref_spans': [], 'section': 'Acknowledgements'}]}\n",
      "{'paper_id': 'adb3a6501ad731eda95d0a1a182a793a0dcd58b0', 'metadata': {'title': 'Quantifying bias of COVID-19 prevalence and severity estimates in Wuhan, China that depend on reported cases in international travelers', 'authors': [{'first': 'Rene', 'middle': [], 'last': 'Niehus', 'suffix': '', 'affiliation': {}, 'email': 'rniehus@hsph.harvard.edu.pablo'}, {'first': 'Pablo', 'middle': ['M'], 'last': 'De Salazar', 'suffix': '', 'affiliation': {}, 'email': 'pablom@hsph.harvard.edu.marc'}, {'first': 'Aimee', 'middle': ['R'], 'last': 'Taylor', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Marc', 'middle': [], 'last': 'Lipsitch', 'suffix': '', 'affiliation': {}, 'email': 'mlipsitc@hsph.harvard.edu'}]}, 'abstract': [{'text': 'Risk of COVID-19 infection in Wuhan has been estimated using imported case counts of international travelers, often under the assumption that all cases in travelers are ascertained. Recent work indicates variation among countries in detection capacity for imported cases. Singapore has historically had very strong epidemiological surveillance and contact-tracing capacity and has shown in the COVID-19 epidemic evidence of a high sensitivity of case detection. We therefore used a Bayesian modeling approach to estimate the relative imported case detection capacity for other countries compared to that of Singapore.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': \"We estimate that the global ability to detect imported cases is 38% (95% HPDI 22% -64%) of Singapore's capacity. Equivalently, an estimate of 2.8 (95% HPDI 1.5 -4.4) times the current number of imported cases, could have been detected, if all countries had had the same detection capacity as Singapore. Using the second component of the Global Health Security index to stratify likely case-detection capacities, we found that the ability to detect imported cases relative to Singapore among high surveillance locations is 40% (95% HPDI 22% -67%), among intermediate surveillance locations it is 37% (95% HPDI 18% -68%), and among low surveillance locations it is 11% (95% HPDI 0% -42%). Using a simple mathematical model, we further find that treating all travelers as if they were residents (rather than accounting for the brief stay of some of these travelers in Wuhan) can modestly contribute to underestimation of prevalence as well. We conclude that estimates of case counts in Wuhan based on assumptions of perfect detection in travelers . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. : medRxiv preprint\", 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'term can be well approximated as We plot this approximation of V given doubling times . V ≈ 1 − e −(r+γ)d aligned with 4 , a range of durations of detectable infection and a range of lengths of stay 2 ( Figure 2 ).', 'cite_spans': [], 'ref_spans': [{'start': 201, 'end': 211, 'text': '( Figure 2', 'ref_id': None}], 'section': 'Abstract'}, {'text': \"We estimate that the global ability to detect imported cases is 38% ( 95% HPDI 22% -64%) of Singapore's capacity. Equivalently, an estimate of 2.8 (95% HPDI 1.5 -4.4) times the current number of imported cases, could have been detected, if all countries had had the same detection capacity as Singapore, which leads to 1.8 (95% CI 0.5 -3.4) undetected cases per detected case. The ability to detect imported cases among high surveillance countries is 40% (95% HPDI 22% -67%), among intermediate surveillance countries it is 37% (95% HPDI 18% -68%), and among low surveillance countries it is 11% (95% HPDI 0% -42%). : medRxiv preprint 7\", 'cite_spans': [], 'ref_spans': [{'start': 68, 'end': 69, 'text': '(', 'ref_id': None}], 'section': 'Abstract'}, {'text': 'In this paper, we have aimed to test two assumptions underlying the estimation of incidence at the epicentre of the SARS-Cov2 outbreak. The first of these is that the capacity for detection of international imported cases is 100% sensitive and specific across locations. While we know of no reason to doubt specificity of detection, we tested the assumption of perfect sensitivity. Based on our previous estimates 5 we assumed Singapore has the highest capacity of surveillance with respect to COVID-19. We regressed the cumulative cases against Wuhan-to-location air travel volume considering Singapore to have the greatest detection capacity and estimating the relative underdetection compared to Singapore in the remaining 190 locations classified according to the GHS 2 index 9 . While it is unlikely that this index reflects the true ranking of countries for any specific epidemic, such as the current one of COVID-19, we assume that it captures different levels of surveillance capacity. We therefore grouped the remaining locations into three big classes high, medium, and low surveillance capacity instead of using exact ranking.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'We estimated that detection of exported cases from Wuhan worldwide is 38% (95% HPDI 22%-64%) as sensitive as it has been in Singapore. Put another way, this implies that the true number of cases in travelers is at least 2.8 (95% HPDI 1.5 -4.4) times the number that has been detected.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': \"Equivalently, for each detected case there are at least 1.8 (95% CI 0.5-3.4) undetected cases. If the model is correct, this is an upper bound on the detection frequency because (1) Singapore's detection is probably not 100% efficient. Singapore had as of 12 February 2020 eight\", 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}], 'body_text': [{'text': '2 may be underestimated by several fold, and severity correspondingly overestimated by several fold.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'Undetected cases are likely in countries around the world, with greater risk in countries of low detection capacity and high connectivity to the epicenter of the outbreak.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'During the outbreak of a new virus SARS-Cov2 and its associated disease COVID-19, infection in travelers has been used to estimate the risk of infection in Wuhan, Hubei Province, China, the epicenter of the outbreak 1 . This approach is similar to that used for the the 2009 influenza pandemic where infections in tourists returning from Mexico were used to estimate the time-specific risk of infection (incidence or cumulative incidence) with the novel pandemic H1N1 influenza strain in Mexico (or parts thereof). The idea was that surveillance for the novel virus was not intense during the early days of the pandemic in Mexico, the source country, and that detection would be far more sensitive in travelers leaving Mexico, who would be screened when returning home as a means of preventing introductions of cases into destination countries 2, 3 . Reports that health systems in Wuhan are overwhelmed and many cases are not being counted have led to the use of outgoing traveler data to estimate the time-specific risk of COVID-19 in Wuhan 4 . This estimate, in turn, has been used to estimate the cumulative incidence of infection by a certain date in Wuhan, and from there (often assuming exponential growth and no appreciable depletion of susceptibles) the cumulative number of cases. Two important assumption underlie this calculation: i) that the detection of cases in the destination country has been 100% sensitive and specific, whether they are detected at the airport (prevalent cases with symptoms) or later after arrival at their destination (cases that were incubating during travel); ii) that travelers have the same prevalence of infection as the average resident of Hubei , so the prevalence inferred in travelers may be directly applied in Hubei. Here we consider the extent to which these two assumptions are justified. We conclude that the first assumption is strongly inconsistent with observed data, resulting in potentially substantial underestimates of prevalence in Hubei and corresponding overestimates of case-severity measures that are normalised by case counts.', 'cite_spans': [{'start': 844, 'end': 846, 'text': '2,', 'ref_id': 'BIBREF1'}, {'start': 847, 'end': 848, 'text': '3', 'ref_id': 'BIBREF2'}, {'start': 1043, 'end': 1044, 'text': '4', 'ref_id': 'BIBREF3'}], 'ref_spans': [], 'section': 'Introduction:'}, {'text': 'We previously showed that there was variability between locations in the world in the relationship between the number of travelers from Wuhan to each international destination and the number of imported cases detected in that destination. On average, for countries presumed to have high surveillance capacity, one imported case reported over the period 8th January to 4th February was associated with each additional 14 passengers/day historical travel volume 5 . However there was variation around this average. Among countries with substantial travel volume, Singapore showed the highest ratio of detected imported cases to daily travel volume, a ratio of one case per 5 daily travelers. Singapore is historically known for exceptionally sensitive detection of cases, for example in SARS 6 , and has had extremely detailed case reporting during the COVID-19 outbreak 7 . We therefore use Singapore in this work as the upper limit of case detection capacity. And we estimate this capacity of other locations relative to Singapore.', 'cite_spans': [{'start': 869, 'end': 870, 'text': '7', 'ref_id': 'BIBREF6'}], 'ref_spans': [], 'section': 'Introduction:'}, {'text': \"Regarding the second assumption, we demonstrate that the point prevalence of infection may be lower in visitors who have stayed only briefly in the source population (Wuhan) than in residents. All else equal, the discrepancy between resident and visitor prevalence is most pronounced if the visitors' durations 3 of stay are shorter, for slower-growing epidemics, and for longer durations of detectable infection; conversely, visitors are more similar to residents in their prevalence of infections if they stay longer, if the epidemic is growing faster, and if the duration of detectable infection is shorter. We quantify this discrepancy as a function of these features using a simple model.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction:'}, {'text': 'Data From a total of 195 worldwide locations (reflecting mainly countries without taking any position on territorial claims), we included n =194, which excludes the epicenter China. Data on imported cases aggregated by location were obtained from the WHO technical report dated 4th February 2020 1 (a zero case count was assumed for all locations not listed). We used case counts up to the 4th February, because after this date the number of exported cases from Hubei province drops rapidly 1 , likely due to the Hubei-wide lockdowns.', 'cite_spans': [], 'ref_spans': [], 'section': 'Methods:'}, {'text': \"We defined imported cases as those with known travel history from China (of those, 83% had travel history from Hubei province, and 17% from unknown locations in China 1 ). Estimates on daily air travel volume were obtained from Lai et al. 8 Singapore as a special case for surveillance of COVID-19, and we assign it it's own category of highest-achieved surveillance. This study did not include human subjects, used publicly available data, and therefore no ethical approval was required.\", 'cite_spans': [{'start': 239, 'end': 240, 'text': '8', 'ref_id': 'BIBREF7'}], 'ref_spans': [], 'section': 'Methods:'}, {'text': 'We consider the detection of 18 cases by 4th February 2020 in Singapore 1 to reflect the highest surveillance capacity among all locations, and estimate the probability of detection in other countries relative to Singapore according to the following model. We model the case detection across i = 1,..., n . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Estimating detection probability relative to Singapore'}, {'text': 'The copyright holder for this preprint . https://doi.org/10.1101/2020.02.13.20022707 doi: medRxiv preprint worldwide locations, where the n = 194 locations are indexed with Singapore being i = 1, followed by the rest of the locations in order of decreasing GHS 2 index. We assume that the observed case count across the n locations follows a Poisson distribution, and that the expected case count is linearly proportional to the daily air travel volume and a random variable, reflecting the capacity of detecting cases relative ', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Testing the effect of length of stay in point prevalence of travelers'}, {'text': 'The copyright holder for this preprint . https://doi.org/10.1101/2020.02.13.20022707 doi: medRxiv preprint In 2009, during the influenza pandemic which originated in Mexico, it was assumed that most travelers leaving Mexico were tourists, or other temporary visitors, with relatively short stays in Mexico, and that the risk that they were infected represented a cumulative hazard over the period of their stay 2,3 . The basic assumption was that short term visitors faced the same hazard of infection as residents of Mexico, but, given the shorter stay, they had a somewhat lower prevalence of infection when returning to their home country. Many estimates in 2019-20 for COVID19 have instead made the assumption of equal prevalence in travelers leaving Wuhan and in residents, which is equivalent to assuming either that all travellers are Wuhan residents, or that all visitors had stayed long enough during the epidemic that their prevalence was similar to that of residents.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'To quantify the difference between these two scenarios -assuming that all travellers are short term visitors versus assuming that all travellers are residents or long term visitors -we considered a simple not in the intensity of exposure. Under these assumptions, the ratio of prevalence in visitors to that in residents, which we call V , would be . Once the number of cases is substantial, this', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'We further find that the prevalence ratio between temporary visitors and residents approaches 1 as the epidemic growth rate, the duration of stay, and the recovery rate increase and it approaches zero for short duration of stay, long duration of infection, and slower epidemic growth ( Figure 2 ). ', 'cite_spans': [], 'ref_spans': [{'start': 284, 'end': 294, 'text': '( Figure 2', 'ref_id': 'FIGREF4'}], 'section': '(which was not peer-reviewed)'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion:'}, {'text': 'The copyright holder for this preprint . https://doi.org/10.1101/2020.02.13.20022707 doi: medRxiv preprint 8 countries has relied largely on symptoms and travel history, so the number of asymptomatic or low-severity cases missed by such a strategy is unknown.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'The second assumption we tested is that the true prevalence in travelers is similar to that of residents. It may be different for either of two reasons, one of which we attempt to quantify here. It could be less if those who travel for some reason are less well integrated into the social mixing that produces infection, for example if they tend to have stayed in certain parts of the city or in hotels. This aspect could conceivably increase or decrease prevalence in travelers relative to residents. Here we quantify a second difference, which is that some travelers (whom we refer to as visitors) will have been in the city only for a short time and had less exposure to the infection than residents. This effect, we find, is most pronounced when the epidemic is growing slowly, when the visitors have stayed only briefly, and when the duration of detectable infection is short. We find that for plausible parameters for COVID-19, prevalence in visitors staying only 3 days could be as little as half that of residents, but for longer stays of over a week the visitor prevalence should be 80% or more that of residents. Assuming that the traveler population is a mix of visitors of various durations and residents, this suggests that underestimation of source population prevalence due to the presence of short-stay visitors could be appreciable but more modest than the effect of imperfect detection.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'These findings that detected cases in travelers likely underrepresent the source population prevalence have two important implications for public health response to SARS-CoV2. First, this finding has implications for approaches to case burden and severity estimation which use cases in travelers to impute cases in Wuhan, which are then compared (for severity estimation) against deaths in Wuhan. If the true number of cases in travelers is higher than previously thought, this implies more cases in Wuhan and a larger denominator, resulting in reduced estimates of severity compared to estimates assuming perfect detection in travelers. Future studies should account for our evolving understanding of detection capacity when estimating case numbers and severity in source population on the basis of traveler case numbers.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'Second, the scenario where the virus has been imported from Wuhan and remained undetected in various worldwide locations is a plausible one, at least until the city lockdown (23rd January 2020), and one might speculate that detection capacity remained limited beyond this period as travelers infected elsewhere in China continued to leave China. Based on our model, the risk of undetected circulation correlates both with air travel connectivity and (inversely) to outbreak detection capacity, but could have happened in virtually any location worldwide leading to the potential risk of self-sustained transmission, which may be an early stage of a global pandemic.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'WHO. 2019-nCoV situation report -15', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF1': {'ref_id': 'b1', 'title': 'Pandemic potential of a strain of influenza A (H1N1): early findings', 'authors': [{'first': 'C', 'middle': [], 'last': 'Fraser', 'suffix': ''}], 'year': 2009, 'venue': 'Science', 'volume': '324', 'issn': '', 'pages': '1557--1561', 'other_ids': {}}, 'BIBREF2': {'ref_id': 'b2', 'title': 'Use of cumulative incidence of novel influenza A/H1N1 in foreign travelers to estimate lower bounds on cumulative incidence in Mexico', 'authors': [{'first': 'M', 'middle': [], 'last': 'Lipsitch', 'suffix': ''}], 'year': 2009, 'venue': 'PLoS One', 'volume': '4', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF3': {'ref_id': 'b3', 'title': 'Nowcasting and forecasting the potential domestic and international spread of the 2019-nCoV outbreak originating in Wuhan, China: a modelling study', 'authors': [{'first': 'J', 'middle': ['T'], 'last': 'Wu', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Leung', 'suffix': ''}, {'first': 'G', 'middle': ['M'], 'last': 'Leung', 'suffix': ''}], 'year': 2020, 'venue': 'Lancet', 'volume': '', 'issn': '', 'pages': '30260--30269', 'other_ids': {'DOI': ['10.1016/S0140-6736(20)30260-9']}}, 'BIBREF4': {'ref_id': 'b4', 'title': 'Using predicted imports of 2019-nCoV cases to determine locations that may not be identifying all imported cases. Infectious Diseases (except HIV/AIDS', 'authors': [{'first': 'De', 'middle': [], 'last': 'Salazar', 'suffix': ''}, {'first': 'P', 'middle': ['M'], 'last': 'Niehus', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Taylor', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Buckee', 'suffix': ''}, {'first': 'C', 'middle': ['O'], 'last': 'Lipsitch', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': '', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.02.04.20020495']}}, 'BIBREF5': {'ref_id': 'b5', 'title': 'Public health measures implemented during the SARS outbreak in Singapore', 'authors': [{'first': 'L', 'middle': [], 'last': 'James', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Shindo', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Cutter', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Ma', 'suffix': ''}, {'first': 'S', 'middle': ['K'], 'last': 'Chew', 'suffix': ''}], 'year': 2003, 'venue': 'Public Health', 'volume': '120', 'issn': '', 'pages': '20--26', 'other_ids': {}}, 'BIBREF6': {'ref_id': 'b6', 'title': 'Updates on Covid-19 Local Situation', 'authors': [{'first': '', 'middle': [], 'last': 'Ministry Of Health', 'suffix': ''}, {'first': '', 'middle': [], 'last': 'Singapore', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF7': {'ref_id': 'b7', 'title': 'Assessing spread risk of Wuhan novel coronavirus within and beyond China', 'authors': [{'first': 'S', 'middle': [], 'last': 'Lai', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF8': {'ref_id': 'b8', 'title': 'John Hopkins Bloomberg School of Public Health', 'authors': [], 'year': 2019, 'venue': 'Global health Security Index', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF9': {'ref_id': 'b9', 'title': 'Stan: A probabilistic programming language', 'authors': [{'first': 'B', 'middle': [], 'last': 'Carpenter', 'suffix': ''}], 'year': 2017, 'venue': 'J. Stat. Softw', 'volume': '76', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF11': {'ref_id': 'b11', 'title': 'Rank-normalization, folding, and localization: An improved ˆ R for assessing convergence of MCMC', 'authors': [{'first': 'A', 'middle': [], 'last': 'Vehtari', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Gelman', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Simpson', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Carpenter', 'suffix': ''}, {'first': 'P.-C', 'middle': [], 'last': 'Bürkner', 'suffix': ''}], 'year': 2019, 'venue': '', 'volume': '18', 'issn': '', 'pages': '', 'other_ids': {'arXiv': ['arXiv:1903.08008']}}}, 'ref_entries': {'FIGREF0': {'text': '. They are based on historical (February 2018) data from the International Air Travel Association and include estimates for the 27 locations that are most connected to Wuhan. They capture the daily average number of passengers traveling via direct and indirect flight itineraries from Wuhan to destinations outside of China. For all 167 locations not listed by Lai et al. 8 , we set the daily air travel volume to 1.5 passengers per day, which is one half of the minimum reported by Lai et al. Surveillance capacity was assessed using the Global Health Security Index 9 , which is an assessment of health security across 195 countries agreeing to the International Health Regulations (IHR [2005]). Specifically, we use the second category of the index, Early Detection and Reporting Epidemics of Potential International Concern, henceforth referred to as simply the GHS 2 index. We classify locations with GHS 2 index above the 80th percentile as high surveillance locations, those with GHS 2 index below the 20th percentile as low surveillance locations, with all others classified as locations with intermediate surveillance. We treat', 'latex': None, 'type': 'figure'}, 'FIGREF1': {'text': \"i denotes the reported case count in the i -th location, λ i denotes the expected case count in the i -th location, β denotes the regression coefficient, x i denotes the daily air travel volume of the i -th location, and denotes a proportion of Singapore's detection capacity, with being a realisation of for θ level θ level[i] θ level location i . We assume that there are three different levels: low, medium and high. For each we assign a uniform prior over [0,1], and for log( β ) we assign a weakly θ , θ , θ } θ level ∈ { low med high informative Normal prior with mean zero and standard deviation 50. Having fit the model (see details below) we approximate the distribution of the average detection probability, realisations of which, , are θ global obtained by transforming draws from the posterior distributions of . Specifically, we take the , θ , θ θ low med high weighted mean of the posterior draws of for i = 2,... n where weights are proportional to daily , θ , θ θ low med high air travel volume, x i . Exclusion of Singapore ( i =1) enables the estimation of the global detection probability relative to Singapore. Conversely,is an estimate of the multiplier of the case count that could have 1/θ global been detected globally under a capacity equivalent to that of Singapore. We discuss the mean and 95% highest posterior density interval (HPDI) of the numerical approximation of the posterior distribution of , as well as the mean and 95% HPDI of the numerical approximation of the posterior distribution of θ global . Note that the latter two are not reciprocals of the former two because the inverse of a mean is not 1/θ global equal to the mean of the inverse, and similarly for the HPDIs. We fit this model using Stan software (v2.19.1) 10 and we draw 80,000 samples from the posterior using four independent chains (20,000 samples each), each with a burn-in of 500. Diagnostic plots of the MCMC sampler for each of the inferred variables ( and ) are shown in Supplementary , θ , θ θ low med high β All analyses are fully reproducible with the code available online (https://github.com/c2-d2/detect_prob_corona2019).\", 'latex': None, 'type': 'figure'}, 'FIGREF2': {'text': 'model of an exponentially growing epidemic, in which the hazard of infection at time is increasing t (t) λ exponentially at rate . At the beginning of the epidemic, which we call time 0, the hazard of infection is r and thereafter . Then the point prevalence of infection at time in residents who have Wuhan for the duration of the epidemic will be the probability that they have become infected and not recovered by time , assuming that the cumulative hazard remains small enough by that point that u there has been no appreciable depletion of susceptibles: for a visitor who had only been in Wuhan for days prior to departure would be d , assuming that they differ from residents only in the duration of exposure,', 'latex': None, 'type': 'figure'}, 'FIGREF3': {'text': 'Posterior distributions of detection probabilities relative to Singapore. The bottom panel shows the posterior distributions of . The top panel is a density plot of . HPDI: Highest , θ , θ θ low med high θ global Posterior Density Interval.', 'latex': None, 'type': 'figure'}, 'FIGREF4': {'text': 'Ratio of infection prevalence in temporary visitors relative to that in residents ( V ). Plot shows V over a range of durations of visit ( d ) and a range of durations of detectable infection ( ). In the upper γ −1 panel the doubling time of the epidemic is 5 days and in the lower panel it is 7 days. The ratios are given as numbers (rounded to two decimals) with lighter areas as V approaches 1.', 'latex': None, 'type': 'figure'}}, 'back_matter': [{'text': 'We thank Shengjie Lai for providing the air travel data, and we thank Nicholas Jewell and Stephen Kissler for valuable feedback along the way. ', 'cite_spans': [], 'ref_spans': [], 'section': 'Acknowledgments'}, {'text': 'Role of the funding sourceThe funding bodies of this study had no role in the study design, data analysis and interpretation, or writing of the manuscript. The corresponding authors had full access to all the data in the study and had final responsibility for the decision to submit for publication.', 'cite_spans': [], 'ref_spans': [], 'section': '9'}, {'text': 'RN, PMS, and ML designed the study. All authors performed the analysis, reviewed and approved the final manuscript.', 'cite_spans': [], 'ref_spans': [], 'section': 'Contributors'}, {'text': 'Marc Lipsitch has received consulting fees from Merck. All other authors declare no competing interests.', 'cite_spans': [], 'ref_spans': [], 'section': 'Declaration of interests'}]}\n",
      "{'paper_id': 'ae216f7eedbbce7713ae035c05e12ef507ca3efe', 'metadata': {'title': 'Title: Critical Care for Patients with Severe Covid-19 in Sichuan Province, China--A Provincial Cohort Study Novel coRonavirus pneumonia Infected critically ill Patients in Sichuan provincE (SUNRISE) Investigators', 'authors': [{'first': 'Xuelian', 'middle': [], 'last': 'Liao', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Sichuan University', 'location': {'settlement': 'Chengdu'}}, 'email': ''}, {'first': 'Hong', 'middle': [], 'last': 'Chen', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Bo', 'middle': [], 'last': 'Wang', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Sichuan University', 'location': {'settlement': 'Chengdu'}}, 'email': ''}, {'first': ';', 'middle': ['Xiaodong'], 'last': 'Jin', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Sichuan University', 'location': {'settlement': 'Chengdu'}}, 'email': ''}, {'first': 'Zhen', 'middle': [], 'last': 'Li', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Tongji University School of Medicine', 'location': {'settlement': 'Shanghai'}}, 'email': ''}, {'first': 'D', 'middle': ['*'], 'last': '', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Zhongwei', 'middle': [], 'last': 'Zhang', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Sichuan University', 'location': {'settlement': 'Chengdu'}}, 'email': ''}, {'first': 'Weimin', 'middle': [], 'last': 'Li', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Sichuan University', 'location': {'settlement': 'Chengdu'}}, 'email': ''}, {'first': 'Jin', 'middle': [], 'last': 'Tang', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Jian', 'middle': [], 'last': 'Wang', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': ';', 'middle': ['Rui'], 'last': 'Shi', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Sichuan University', 'location': {'settlement': 'Chengdu'}}, 'email': ''}, {'first': ';', 'middle': ['Xiangde'], 'last': 'Zhen', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Dazhou Central Hospital', 'location': {'settlement': 'Dazhou'}}, 'email': ''}, {'first': 'Maojuan', 'middle': [], 'last': 'Wang', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Xianying', 'middle': [], 'last': 'Lei', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Yu', 'middle': [], 'last': 'Gong', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Suining Central Hospital', 'location': {'settlement': 'Suining'}}, 'email': ''}, {'first': 'Sheng', 'middle': [], 'last': 'Lv', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Panzhihua Central Hospital', 'location': {'settlement': 'Panzhihua'}}, 'email': ''}, {'first': 'Chao', 'middle': [], 'last': 'Jia', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': ';', 'middle': ['Li'], 'last': 'Chen', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Juan', 'middle': [], 'last': 'Shang', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Nanchong Central Hospital', 'location': {'settlement': 'Nanchong'}}, 'email': ''}, {'first': 'Min', 'middle': [], 'last': 'Yang', 'suffix': '', 'affiliation': {'laboratory': \"People's hospital of ya'an, Yaan 18 Department of Pulmonary and Critical Care Medicine, The People's Hospital of Leshan\", 'institution': '', 'location': {'settlement': 'Leshan'}}, 'email': ''}, {'first': ';', 'middle': [], 'last': 'Hailong', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Wei', 'middle': [], 'last': '18', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Yuanjun', 'middle': [], 'last': 'Zhang', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': ';', 'middle': ['Xiong'], 'last': 'Yang', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': ';', 'middle': ['Huaqiang'], 'last': 'Shen', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Bazhong Central Hospital', 'location': {'postCode': '22', 'settlement': 'Bazhong, Neijiang'}}, 'email': ''}, {'first': 'Xianhua', 'middle': [], 'last': 'Xiao', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': ';', 'middle': ['Jie'], 'last': 'Yang', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Sichuan University', 'location': {'settlement': 'Chengdu'}}, 'email': ''}, {'first': ';', 'middle': ['Chang'], 'last': 'Liu', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Sichuan University', 'location': {'settlement': 'Chengdu'}}, 'email': ''}, {'first': 'Qin', 'middle': [], 'last': 'Wu', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Sichuan University', 'location': {'settlement': 'Chengdu'}}, 'email': ''}, {'first': 'Wen', 'middle': [], 'last': 'Wang', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Jin', 'middle': [], 'last': 'Yang', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Sichuan University', 'location': {'settlement': 'Chengdu'}}, 'email': ''}, {'first': 'Wanhong', 'middle': [], 'last': 'Yin', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Sichuan University', 'location': {'settlement': 'Chengdu'}}, 'email': ''}, {'first': ';', 'middle': ['Xiaoqi'], 'last': 'Xie', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Sichuan University', 'location': {'settlement': 'Chengdu'}}, 'email': ''}, {'first': 'Yongming', 'middle': [], 'last': 'Tian', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Sichuan University', 'location': {'settlement': 'Chengdu'}}, 'email': ''}, {'first': 'Huan', 'middle': [], 'last': 'Liu', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Sichuan University', 'location': {'settlement': 'Chengdu'}}, 'email': ''}, {'first': 'Bingxin', 'middle': [], 'last': 'Shuai', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Sichuan University', 'location': {'settlement': 'Chengdu', 'region': 'West China Hospital'}}, 'email': ''}, {'first': 'Wei', 'middle': [], 'last': 'Zhang', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Sichuan University', 'location': {'settlement': 'Chengdu'}}, 'email': ''}, {'first': 'Ph', 'middle': ['D ;'], 'last': 'Xiangrong Song', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Sichuan University', 'location': {'settlement': 'Chengdu'}}, 'email': ''}, {'first': 'Ph', 'middle': ['D'], 'last': '', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Yan', 'middle': [], 'last': 'Kang', 'suffix': '', 'affiliation': {}, 'email': 'kangyan@scu.edu.cn'}, {'first': 'Xiaodong', 'middle': [], 'last': 'Jin', 'suffix': '', 'affiliation': {}, 'email': ''}]}, 'abstract': [{'text': 'Data regarding critical care for patients with severe Covid-19 are limited. We aimed to describe the clinical course and critical care implemented for this patient population at the provincial level in Sichuan, China.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'In this population-based multicenter cohort study, conducted from January 16 to March 15, 2020, all microbiologically confirmed Covid-19 patients who met the national severe or critical criteria were included and followed-up until discharge, death, or the end of the study.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'Out of 539 confirmed Covid-19 patients, 81 severe cases (15.0%) were identified. The median (IQR) age was 50 (39-65) years, 37% were female, and 53.1% had chronic comorbidities. Among the five predefined criteria for severe illness, low PaO2:FiO2 ratio (<300 mmHg), low pulse oxygen saturation (≤93%), and dyspnea were the most commonly reported, accounting for 87.7%, 66.7% and 27.2% of the severe cases, respectively. The median period from the onset of symptoms to the first hospitalization was 3 (1-6) days. Seventy-seven patients (95.06%) were admitted to hospitals being able to provide critical care by day 1. By day 28, 53 (65.4%) were discharged, 3 (3.7%) were deceased, and 25 (30.9%) were still hospitalized. Conventional oxygen therapy, administered to 95.1% of the patients, was the most commonly used respiratory support 5 and met 62.7% of the respiratory support needed, followed by high-flow nasal cannula (19.5%) and noninvasive mechanical ventilation (10%).', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'Early identification, hospitalization, and provision of critical care to severe Covid-19 patients may improve prognosis. Sufficient conventional oxygen equipment should be prioritized and implemented without delay.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': '6', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}], 'body_text': [{'text': 'The 2019 novel coronavirus disease has spread globally since its outbreak in December 2019 and was declared a pandemic by the World Health Organization on March 11. Out of over 200,000 patients affected worldwide, approximately 19% are expected to progress to severe or critical disease, which poses a high mortality rate up to 8.1%. 1,2 However, little data has been published regarding critical care for Covid-19.', 'cite_spans': [], 'ref_spans': [], 'section': 'INTRODUCTION'}, {'text': 'Early studies from Wuhan investigated the epidemiological and clinical characteristics of patients affected by . The in-hospital mortality rate ranged from 4.9% to 15%. [3] [4] [5] A recent single-center study reported a 28-day mortality of 61% among 52 critically ill patients who were admitted to the intensive care unit (ICU) and experienced considerable respiratory compromise. 6 Several studies reported a few severe cases of Covid-19, with one in Zhejiang Province 7 , five in Shanxi Province 8 , China. No severe cases were reported in Singapore 9 . The clinical characteristics of 173 patients with severe Covid-19, using data from 30 provinces in China, were described by Guan et al. 2 However, the clinical course, critical care, and patient prognosis of severe Covid-19 remain largely unexplored. 10, 11 Population-based studies regarding these subjects may provide important data to reduce the mortality rate among patients with severe Covid-19. 7 Sichuan province located in the south west of China, with a total area of 486 thousand square kilometers and 83 million population. 12 Since the first reported case of Covid-19 on January 16, 2020, a total of 539 cases have been confirmed in Sichuan. Due to the urban-rural disparities in healthcare, provision of critical care for severely ill patients was centralized by dispatching ICU staff and resources to designated hospitals.', 'cite_spans': [{'start': 169, 'end': 172, 'text': '[3]', 'ref_id': 'BIBREF2'}, {'start': 173, 'end': 176, 'text': '[4]', 'ref_id': 'BIBREF3'}, {'start': 177, 'end': 180, 'text': '[5]', 'ref_id': 'BIBREF4'}, {'start': 382, 'end': 383, 'text': '6', 'ref_id': 'BIBREF5'}, {'start': 693, 'end': 694, 'text': '2', 'ref_id': 'BIBREF1'}, {'start': 808, 'end': 811, 'text': '10,', 'ref_id': 'BIBREF9'}, {'start': 812, 'end': 814, 'text': '11', 'ref_id': 'BIBREF10'}, {'start': 958, 'end': 959, 'text': '7', 'ref_id': 'BIBREF6'}, {'start': 1092, 'end': 1094, 'text': '12', 'ref_id': None}], 'ref_spans': [], 'section': 'INTRODUCTION'}, {'text': 'In this multicenter cohort study (StUdy of 2019 Novel coRonavirus pneumonia Infected critically ill patients in Sichuan provincE, SUNRISE), we aimed to analyze the epidemiology, clinical course and critical care for severe Covid-19 at the provincial level.', 'cite_spans': [], 'ref_spans': [], 'section': 'INTRODUCTION'}, {'text': 'To ensure high quality care despite region-based healthcare disparities, the Sichuan Provincial Department of Health (SPDH) dispatched ICU physicians, nurses, respiratory therapists and resources to designated hospitals. A designated hospital was considered eligible for severe Covid-19 patients if it was covered by ICU physicians and provided advanced respiratory support which was defined as the use of high flow nasal cannula (HFNC), non-invasive (NIV), or invasive ventilation (IV). Remote multidisciplinary consultation was arranged to discuss complex patients daily. The number and severity classification 1 of Covid-19 patients in all designated hospitals were required to be reported daily. 8 ', 'cite_spans': [{'start': 700, 'end': 701, 'text': '8', 'ref_id': 'BIBREF7'}], 'ref_spans': [], 'section': 'Setting'}, {'text': 'This provincial multicenter cohort study (http://www.chictr.org.cn/index.aspx, ChiCTR2000029758) was initiated by investigators in West China Hospital (WCH) in Sichuan Province. 13 The study involved 21 hospitals designated for severely affected patients, of which 18 were directly caring for patients included in the present study.', 'cite_spans': [{'start': 178, 'end': 180, 'text': '13', 'ref_id': 'BIBREF11'}], 'ref_spans': [], 'section': 'Study Design'}, {'text': 'Data were prospectively collected for patients who were still in the hospital after study enrollment, and otherwise retrospectively collected between January 16 and March 15.', 'cite_spans': [], 'ref_spans': [], 'section': 'Study Design'}, {'text': 'The study timeline is shown in the supplementary file. The study protocol was approved by the Ethics Committee of the West China Hospital and the participating hospitals.', 'cite_spans': [], 'ref_spans': [], 'section': 'Study Design'}, {'text': \"Informed consent was obtained from the patient or the patient's legally authorized representative.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Study Design'}, {'text': 'Confirmed Covid-19 patients meeting any of the following criteria were included as severe cases: 1) dyspnea or respiratory frequency ≥30/min; 2) pulse oxygen saturation (SPO2) ≤93% without oxygen therapy in resting state; 3) PaO2:FiO2 ratio <300; 4) lung infiltrates >50% within 24-48 hours; 5) respiratory failure, septic shock, and/or multiple organ dysfunction. 1 The date of enrollment of each patient was considered day 1 (D1). Covid-19 was diagnosed using real time reverse transcription polymerase chain reaction (RT-PCR) of nasal, pharyngeal swab or sputum specimens by the local Centers for Disease Control (CDC). Patients were followed up from D1 to the end of the study.', 'cite_spans': [{'start': 365, 'end': 366, 'text': '1', 'ref_id': 'BIBREF0'}], 'ref_spans': [], 'section': 'Participants and follow-up'}, {'text': 'Patient characteristics were demonstrated according to the outcome by D28, while the main support methods used were analyzed throughout the study period.', 'cite_spans': [], 'ref_spans': [], 'section': 'Participants and follow-up'}, {'text': 'In the first stage, 13 designated hospitals were included in our study on February 6. In the second stage, another 8 designated hospitals were included on February 12 after obtaining permission from the SPDH to access information regarding daily reported cases of Covid-19. Detailed demographic, epidemiological, clinical and laboratory data were recorded using the electronic data capture and analysis system (EDC) (more details are provided in the supplementary file). Data entry was completed by physicians and nurses who were trained on the use of EDC and were working in the designated hospitals. Data quality was overseen by a team of senior ICU physicians and statisticians.', 'cite_spans': [], 'ref_spans': [], 'section': 'Data collection'}, {'text': 'In the special circumstances as stated above, conventional definitions of ICU admission or discharge will not be used because mildly and severely ill patients were treated in the same ward in some cases. We defined D1 as the day when patient met the criteria for severe illness. Patients were categorized into three groups according to clinical outcomes by D28: rapid recovery (RR), prolonged recovery (PR) and no recovery (NR). 1) RR: patient fully meeting the discharge criteria before D28, with normal body temperature ≥3 days, obvious improvement in respiratory symptoms and pulmonary imaging, and twice-negative nucleic acid tests (sampling interval being at least 24 hours) on respiratory samples; 2) PR: patient partially meeting the discharge criteria on D28 and still requiring hospitalization but without advanced respiratory support; 3) NR: death or the patient still in need of advanced respiratory support on D28.', 'cite_spans': [], 'ref_spans': [], 'section': 'Definitions'}, {'text': \"Data management, manipulation, and analysis were conducted by a professional epidemiologist and statistician, from a third Clinical Research Center, who did not participate in data collection. To ensure the high quality of database, all missing data and outliers detected were checked by two clinicians independently in the medical records. Then the verified data were collected and transferred for data completion or correction. No imputation was made for missing data. Data are expressed as median (IQR) for continuous variables and number (%) for categorical variables. For continuous variables, Wilcoxon test was applied to assess the difference between patients in one group and others, Kruskal-Wallis test to assess the overall difference among the RR, PR and NR groups. For categorical variables, χ2 test or Fisher's exact test were performed according to the distribution of data. Two-sided tests with a significance level of 0.05 were applied. Needs for different respiratory support throughout the study period were assessed using person-day that denoted the use by one person in one day. All the analyses were conducted using R software version 3.6.2 11 (R Foundation for Statistical Computing). We used the tydyverse 14 package of to manage daily data and plot the daily use of main support and the pheatmap 15 package to plot the heatmap.\", 'cite_spans': [{'start': 1162, 'end': 1164, 'text': '11', 'ref_id': 'BIBREF10'}], 'ref_spans': [], 'section': 'Statistics'}, {'text': 'Province met the criteria for severe Covid-19 and were included in the study, accounting for 15.0% (81/539) of the total provincial patient population. Out of 226 planned designated hospitals, 18 (8.0%) participated in the study. All patients were centralized to the designated hospitals, and 77 patients (95.1%) were admitted by D1 ( Figure 1 ). The median (IQR) durations from the onset of symptoms to the first hospitalization, RT-PCR confirmation and the diagnosis of severe condition were 3 (1-6), 7 (5-10), and 9 (6-11) days, respectively.', 'cite_spans': [], 'ref_spans': [{'start': 335, 'end': 343, 'text': 'Figure 1', 'ref_id': 'FIGREF1'}], 'section': 'Eighty-one patients from 13 cities and 2 minority autonomous prefectures in Sichuan'}, {'text': 'The median age (IQR) of the patients was 50 (39-65) years and 37.0% were female. Of these patients, 37 (45.7%) had travel history to Wuhan, 19 (23.5%) were part of family clusters and 19 (23.5%) had no known contact history. Chronic comorbidities were observed among 43 (53.1%) patients, in which diabetes, hypertension and chronic pulmonary diseases were the most common, accounting for 22.2%, 18.5% and 13.6%, respectively. Forty-one patients (50.6 %) were with an BMI ≥24 kg/m 2 ( Table 1) .', 'cite_spans': [], 'ref_spans': [{'start': 484, 'end': 492, 'text': 'Table 1)', 'ref_id': 'TABREF4'}], 'section': 'Epidemiology and clinical characteristics of the severe Covid-19 patients on D1'}, {'text': 'Only 42 (51.9%) patients had fever on D1 although 69 (85.2%) had a history of fever.', 'cite_spans': [], 'ref_spans': [], 'section': 'Epidemiology and clinical characteristics of the severe Covid-19 patients on D1'}, {'text': 'Among the five predefined criteria for severe illness, PaO2:FiO2 ratio, SPO2, and dyspnea criteria were the most commonly reported, accounting for 87.7%, 66.7% and 27.2%, respectively. All chest images of the patients (80 with CT scan and 1 with chest X-ray) showed bilateral lesions, but only 4 (4.9%) were diagnosed using the imaging criterion. Seventy-six patients (93.8%) were administrated by respiratory support, including 51 (67.9%) by conventional oxygen therapy (COT) through nasal catheter or mask, 13 (16.1%) by NIV, and 8 (9.9%) by HFNC. No patient was intubated or needed ECMO. Six patients developed acute kidney injury (AKI) 16 and 5 met the criteria for septic shock 17 ( Table 1) .', 'cite_spans': [{'start': 640, 'end': 642, 'text': '16', 'ref_id': 'BIBREF13'}], 'ref_spans': [{'start': 688, 'end': 696, 'text': 'Table 1)', 'ref_id': 'TABREF4'}], 'section': 'Epidemiology and clinical characteristics of the severe Covid-19 patients on D1'}, {'text': 'All patients were followed up and reached the predefined endpoints. Among 81 patients, 53 (65.4%) were regarded as RR and admitted before D28. Eighteen (22.2%) patients were regarded as PR, including 13 still in need of conventional oxygen therapy and 5 awaiting negative results of RT-PCR on D28. Ten patients (12.3%) were in the NR group, including 3 deaths and 7 still in need of advanced respiratory support on D28.', 'cite_spans': [], 'ref_spans': [], 'section': 'Patient follow-up and clinical outcomes'}, {'text': 'Patients in the NR group tended to be older (p<0.001), had higher Apache II 18 (p=0.01) 13 and SOFA scores 19 (p<0.001), were more likely to have comorbidities (p=0.02) including AKI (p=0.004) than the other groups. C-reaction protein (p=0.02) was higher while lymphocyte (p=0.03) and platelet (p=0.005) were lower in the NR group ( Table   1 ). Among the 3 deceased, one was a 64-year-old female with scleroderma, pulmonary fibrosis and diabetes, one was an 80-year-old female with hypertension and coronary heart disease. Both patients developed severe respiratory failure and died of multi-organ dysfunction. The third patient was a 73-year-old male with hypertension and end-stage renal disease and died from circulatory failure. By March 15, among those who were still hospitalized on D28, 15 were discharged, 9 patients were still hospitalized, and an 81-year-old male died of end-stage chronic pulmonary disease on D45.', 'cite_spans': [{'start': 88, 'end': 90, 'text': '13', 'ref_id': 'BIBREF11'}], 'ref_spans': [{'start': 333, 'end': 342, 'text': 'Table   1', 'ref_id': 'TABREF4'}], 'section': 'Patient follow-up and clinical outcomes'}, {'text': 'Daily respiratory support given to each patient from D1 to D28 were shown in Figure   2 . Of the 81 patients diagnosed severe, 79 (97.5%) used COT, 31 (38.3%) used HFNC, 22 (27.2%) used NIV, 10 (12.3%) used IV, and 1 (1.2%) used ECMO ( Table 2) . Thirtyfour patients (42% of 81) used only COT among which 79.4% discharged before D28.', 'cite_spans': [{'start': 170, 'end': 172, 'text': '22', 'ref_id': 'BIBREF19'}], 'ref_spans': [{'start': 77, 'end': 87, 'text': 'Figure   2', 'ref_id': 'FIGREF7'}, {'start': 236, 'end': 244, 'text': 'Table 2)', 'ref_id': 'TABREF0'}], 'section': 'Respiratory support'}, {'text': 'In the 25 patients who started with COT and needed escalation to advanced respiratory support methods, 12 (48.0%) were discharged by D28.', 'cite_spans': [], 'ref_spans': [], 'section': 'Respiratory support'}, {'text': 'Among patients who were discharged within 28 days, the median duration (IQR) of hospitalization was 18 (14) (15) (16) (17) (18) (19) (20) (21) (22) (23) (24) days. Only COT, HFNC, and NIV were used as 14 respiratory support, for a median duration of 10 (5-14), 0 (0-4), and 0 (0-0) days, respectively. As for PR patients, COT, HFNC, NIV, and IV were used for 18 (7) (8) (9) (10) (11) (12) (13) (14) (15) (16) (17) (18) (19) (20) (21) (22) (23) (24) , 0 (0-7), 0 (0-3), and 0 (0-3) days, respectively. Concerning about the NR patients, the median duration for COT, HFNC, NIV and IV were 3 (1-10), 2.5 (0-11), 0 (0-3), and 2 (0-16) days, respectively. ECMO was used for one day by one patient.', 'cite_spans': [{'start': 103, 'end': 107, 'text': '(14)', 'ref_id': 'BIBREF12'}, {'start': 108, 'end': 112, 'text': '(15)', 'ref_id': None}, {'start': 113, 'end': 117, 'text': '(16)', 'ref_id': 'BIBREF13'}, {'start': 118, 'end': 122, 'text': '(17)', 'ref_id': 'BIBREF14'}, {'start': 123, 'end': 127, 'text': '(18)', 'ref_id': 'BIBREF15'}, {'start': 128, 'end': 132, 'text': '(19)', 'ref_id': 'BIBREF16'}, {'start': 133, 'end': 137, 'text': '(20)', 'ref_id': 'BIBREF17'}, {'start': 138, 'end': 142, 'text': '(21)', 'ref_id': 'BIBREF18'}, {'start': 143, 'end': 147, 'text': '(22)', 'ref_id': 'BIBREF19'}, {'start': 148, 'end': 152, 'text': '(23)', 'ref_id': 'BIBREF20'}, {'start': 153, 'end': 157, 'text': '(24)', 'ref_id': 'BIBREF21'}, {'start': 362, 'end': 365, 'text': '(7)', 'ref_id': 'BIBREF6'}, {'start': 366, 'end': 369, 'text': '(8)', 'ref_id': 'BIBREF7'}, {'start': 370, 'end': 373, 'text': '(9)', 'ref_id': 'BIBREF8'}, {'start': 374, 'end': 378, 'text': '(10)', 'ref_id': 'BIBREF9'}, {'start': 379, 'end': 383, 'text': '(11)', 'ref_id': 'BIBREF10'}, {'start': 384, 'end': 388, 'text': '(12)', 'ref_id': None}, {'start': 389, 'end': 393, 'text': '(13)', 'ref_id': 'BIBREF11'}, {'start': 394, 'end': 398, 'text': '(14)', 'ref_id': 'BIBREF12'}, {'start': 399, 'end': 403, 'text': '(15)', 'ref_id': None}, {'start': 404, 'end': 408, 'text': '(16)', 'ref_id': 'BIBREF13'}, {'start': 409, 'end': 413, 'text': '(17)', 'ref_id': 'BIBREF14'}, {'start': 414, 'end': 418, 'text': '(18)', 'ref_id': 'BIBREF15'}, {'start': 419, 'end': 423, 'text': '(19)', 'ref_id': 'BIBREF16'}, {'start': 424, 'end': 428, 'text': '(20)', 'ref_id': 'BIBREF17'}, {'start': 429, 'end': 433, 'text': '(21)', 'ref_id': 'BIBREF18'}, {'start': 434, 'end': 438, 'text': '(22)', 'ref_id': 'BIBREF19'}, {'start': 439, 'end': 443, 'text': '(23)', 'ref_id': 'BIBREF20'}, {'start': 444, 'end': 448, 'text': '(24)', 'ref_id': 'BIBREF21'}], 'ref_spans': [], 'section': 'Respiratory support'}, {'text': 'To describe the needs for respiratory support throughout the study period, we also analyzed the usage of respiratory support along with the daily number of newly diagnosed patients with Covid-19 disease in Sichuan, newly diagnosed severe cases, and patients with severe disease hospitalized (Figure 3) . In total, all forms of respiratory support were used 1579 person-day, of which COT took up 62.7 % (990 person-day), HFNC 19.3% (305 person-day), NIV 9.4% (149 person-day), IV 8.5%', 'cite_spans': [], 'ref_spans': [{'start': 291, 'end': 301, 'text': '(Figure 3)', 'ref_id': None}], 'section': 'Respiratory support'}, {'text': '(134 person-day), and ECMO 0.06% (1 person-day).', 'cite_spans': [], 'ref_spans': [], 'section': 'Respiratory support'}, {'text': 'The peak needs of respiratory support, which lasted for 20 days and paralleled with hospitalization needs for severely ill patients, had a significant lag of 9 days behind the peak of newly diagnosed patients in Sichuan. During the most demanding days for respiratory support measures, 47 patients with severe disease needed access. Of these demands, 55.3% were COT, 21.3% were HFNC, 12.8% were NIV, and 10.6% were IV.', 'cite_spans': [], 'ref_spans': [], 'section': 'Respiratory support'}, {'text': 'Other support measures and medical treatment 15 Some patients needed life-saving measures such as renal replacement (5 patients, 6.2%), vasopressors (5, 6.2%), and blood transfusion (6, 7.4%). Other support methods included the prone position, partial parenteral nutrition therapy, analgesics and sedatives, used by 16 (19.8%), 17 (21.0%), and 13 (16.1%) patients, respectively ( ', 'cite_spans': [{'start': 45, 'end': 47, 'text': '15', 'ref_id': None}], 'ref_spans': [], 'section': 'Respiratory support'}, {'text': 'To our knowledge, it is the first time that the clinical course and critical care of severe Covid-19 patient was demonstrated in detail from the first day when recognized as severe illness to 28-day outcome. Among all the Covid-19 patients Sichuan, only 0.56% (3/539) deceased; even among the severe Covid-19 patients, the rate was 3.7% (3/81), lower than the mortality rates reported elsewhere worldwide 1-6 . Some features of the severe Covid-19 patients were observed in our study: the median duration from the onset of symptoms to hospitalization and D1 was 3 (1-6) and 9 (6-11) days, 16 respectively. Seventy-seven (95.06%) patients were admitted to the designated hospitals with ability to provide critical care by D1. Low PaO2:FO2 ratio was observed among 87.8% of the patients, the highest among the five severity criteria. Conventional oxygen therapy was the most commonly used respiratory support method given to 95.1% of the patients and met 62.7% of all the respiratory support needed.', 'cite_spans': [{'start': 589, 'end': 591, 'text': '16', 'ref_id': 'BIBREF13'}], 'ref_spans': [], 'section': 'DISCUSSION'}, {'text': \"Limited data has been reported regarding the importance of PaO2:FiO2 ratio in the identification of severe Covid-19. Results in our study suggested that PaO2:FiO2 ratio was a sensitive indicator for early identification of severe Covid-19. On D1 87.8% of the patients in our study had an PaO2:FiO2 ratio lower than 300 mmHg, while the SPO2 was 96 (93-97) %, and 72.8% without symptoms of dyspnea. With the early identification of the severe illness, 93.8% patients were receiving various respiratory support from or before D1, with a median PaO2:FO2 ratio of 204 mmHg. In the previous studies, 4,6 the median PaO2:FO2 ratio on ICU admission reported were much lower (103.8 and 136 mmHg), while many of the studies didn't report the data. [1] [2] [3] 5, 20 It is reasonable to hypothesis that hypoxemia may participate in the inflammation and multiple organ injury 21 if the hypoxic compensatory period was ignored or not timely treated due to limited oxygen support equipment when confronting a surge of patient flow. 22,23 Therefore, we comprehensively analyzed the respiratory support used during the study period. If severely ill patients can be recognized earlier, basic oxygen therapy and close monitoring may suffice, as in the 42% of the patients in our study. This scenario is quite different from that in Wuhan, where more than 80% of patients needed advanced respiratory support. 4, 6 Advanced respiratory support equipment is life-saving but a large availability of conventional oxygen therapy is also necessary. Prioritizing and providing basic care for severe Covid-19 patients should be considered by healthcare providers in early stage.\", 'cite_spans': [{'start': 738, 'end': 741, 'text': '[1]', 'ref_id': 'BIBREF0'}, {'start': 742, 'end': 745, 'text': '[2]', 'ref_id': 'BIBREF1'}, {'start': 746, 'end': 749, 'text': '[3]', 'ref_id': 'BIBREF2'}, {'start': 750, 'end': 752, 'text': '5,', 'ref_id': 'BIBREF4'}, {'start': 753, 'end': 755, 'text': '20', 'ref_id': 'BIBREF17'}, {'start': 1390, 'end': 1392, 'text': '4,', 'ref_id': 'BIBREF3'}, {'start': 1393, 'end': 1394, 'text': '6', 'ref_id': 'BIBREF5'}], 'ref_spans': [], 'section': 'DISCUSSION'}, {'text': \"Resources and timing are of critical importance to critical care for severe Covid-19 patients. All the severe Covid-19 patients were centralized to the 18 designated hospitals covering with ICU staff and advanced organ support methods, allowing all patients to receive critical care, albeit at a lower quality than usual. 24 While in other study, only 19.1% of the severe cases were treated in the ICU. 2 In addition, the median duration from symptom onset to admission for critical care in our study was 9 days, while data from Wuhan were 9.5-12 days. 4, 6, 20 The time lag of provision of critical care may have significant impact on patient prognosis. Regarding the duration from symptom onset to hospital visit time, there's obvious difference between studies from Wuhan and outside Wuhan where the mortality rate was much lower than Wuhan. The reported median hospitalization duration was 5-11 days in Wuhan, 3,4,20 while 2-4.5 days in Zhejiang province, 7 Beijing 10 and our study. In the 41 cases study from Wuhan 3 , a significant difference was observed for this duration between ICU and non-ICU patients (8 vs . 1 day, p=0.002) . In our study, because most of our patients were admitted 18 to hospital early, the duration was similar in the RR, PR and NR group (median: 3, 4, and 4.5 days, respectively). Studies with larger sample size including patients from different cities may further clarify the association between admission timing and patient outcomes.\", 'cite_spans': [{'start': 322, 'end': 324, 'text': '24', 'ref_id': 'BIBREF21'}, {'start': 403, 'end': 404, 'text': '2', 'ref_id': 'BIBREF1'}, {'start': 553, 'end': 555, 'text': '4,', 'ref_id': 'BIBREF3'}, {'start': 556, 'end': 558, 'text': '6,', 'ref_id': 'BIBREF5'}, {'start': 559, 'end': 561, 'text': '20', 'ref_id': 'BIBREF17'}, {'start': 1114, 'end': 1119, 'text': '(8 vs', 'ref_id': None}, {'start': 1197, 'end': 1199, 'text': '18', 'ref_id': 'BIBREF15'}], 'ref_spans': [{'start': 1120, 'end': 1137, 'text': '. 1 day, p=0.002)', 'ref_id': 'FIGREF1'}], 'section': 'DISCUSSION'}, {'text': 'Our study has the following limitations. First, a small part of the data was collected retrospectively, leading to potential incompleteness and inaccuracy for some variables.', 'cite_spans': [], 'ref_spans': [], 'section': 'DISCUSSION'}, {'text': 'We mitigated this limitation by designating a team of researchers to verify and complete the data. Second, our study was mainly descriptive due to the small sample size; however, this limitation did not detract from our main objective in a relatively well- ', 'cite_spans': [], 'ref_spans': [], 'section': 'DISCUSSION'}, {'text': 'On behalf of all authors, the corresponding author states that there is no conflict of interest.', 'cite_spans': [], 'ref_spans': [], 'section': 'CONFLICTS OF INTEREST'}, {'text': 'Hospital.', 'cite_spans': [], 'ref_spans': [], 'section': 'The project was supported by Project of Novel Coronavirus Pneumonia in West China'}, {'text': 'We would like to thank Ms. Haixin Miao, Mr. Desong Qiu from Sichuan Zhikang Technology CO., Chengdu, for their help of establishing the electronic data capture and analysis system; We thank Ms. Yu Ma, Mr. Biwei Zhan from Chengdu Urban Planning 25 Information Technic Center for the localization and mapping for the designated hospital.', 'cite_spans': [], 'ref_spans': [], 'section': 'ACKNOWLEDGEMENT'}, {'text': 'We also thank Yi Liu, Ph D. from Shanghai, for his assistance in data visualization. ', 'cite_spans': [], 'ref_spans': [], 'section': 'ACKNOWLEDGEMENT'}, {'text': 'Yes -no. ', 'cite_spans': [], 'ref_spans': [], 'section': 'Coexisting disorders'}, {'text': 'Acute respiratory distress syndrome-no. Daily organ support, including respiratory support and renal replacement therapy, for each of the 81 patients with severe Covid-19 are shown from the day diagnosed severe (D1) to death, discharged from hospital, or D28. ', 'cite_spans': [], 'ref_spans': [], 'section': 'Main organ dysfunction'}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'Characteristics of and important lessons from the coronavirus disease 2019 (COVID-19) outbreak in China: summary of a report of 72 314 cases from the Chinese Center for Disease Control and Prevention', 'authors': [{'first': 'Z', 'middle': [], 'last': 'Wu', 'suffix': ''}, {'first': 'J', 'middle': ['M'], 'last': 'Mcgoogan', 'suffix': ''}], 'year': 2020, 'venue': 'JAMA', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF1': {'ref_id': 'b1', 'title': 'Clinical Characteristics of Coronavirus Disease 2019 in China', 'authors': [{'first': 'W', 'middle': [], 'last': 'Guan', 'suffix': ''}, {'first': 'Z', 'middle': [], 'last': 'Ni', 'suffix': ''}, {'first': 'Hu', 'middle': [], 'last': 'Yu', 'suffix': ''}], 'year': 2020, 'venue': 'NEJM', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF2': {'ref_id': 'b2', 'title': 'Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China', 'authors': [{'first': 'C', 'middle': [], 'last': 'Huang', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Li', 'suffix': ''}], 'year': 2020, 'venue': 'Lancet', 'volume': '395', 'issn': '', 'pages': '497--506', 'other_ids': {}}, 'BIBREF3': {'ref_id': 'b3', 'title': 'Clinical characteristics of 138 hospitalized patients with 2019 novel coronavirus-infected pneumonia in Wuhan, China', 'authors': [{'first': 'D', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Hu', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Hu', 'suffix': ''}], 'year': 2020, 'venue': 'JAMA', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF4': {'ref_id': 'b4', 'title': 'Epidemiological and clinical characteristics of 99 cases of 2019 novel coronavirus pneumonia in Wuhan, China: a descriptive study', 'authors': [{'first': 'N', 'middle': [], 'last': 'Chen', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Zhou', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Dong', 'suffix': ''}], 'year': 2020, 'venue': 'Lancet', 'volume': '395', 'issn': '', 'pages': '507--513', 'other_ids': {}}, 'BIBREF5': {'ref_id': 'b5', 'title': 'Clinical course and outcomes of critically ill patients with SARS-CoV-2 pneumonia in Wuhan, China: a single-centered, retrospective, observational study', 'authors': [{'first': 'X', 'middle': [], 'last': 'Yang', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Yu', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Xu', 'suffix': ''}], 'year': 2020, 'venue': 'Lancet Respir Med', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF6': {'ref_id': 'b6', 'title': 'Clinical findings in a group of patients infected with the 2019 novel coronavirus (SARS-Cov-2) outside of Wuhan, China: retrospective case series', 'authors': [{'first': 'X', 'middle': ['W'], 'last': 'Xu', 'suffix': ''}, {'first': 'X', 'middle': ['X'], 'last': 'Wu', 'suffix': ''}, {'first': 'X', 'middle': ['G'], 'last': 'Jiang', 'suffix': ''}], 'year': 2020, 'venue': 'BMJ', 'volume': '368', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF7': {'ref_id': 'b7', 'title': 'Epidemiological Characteristics of 2019-ncoV Infections in Shaanxi', 'authors': [{'first': 'Y', 'middle': [], 'last': 'Yao', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Tian', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Zhou', 'suffix': ''}], 'year': 2020, 'venue': 'Eur Respir J', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF8': {'ref_id': 'b8', 'title': 'Epidemiologic features and clinical course of patients infected with SARS-CoV-2 in Singapore', 'authors': [{'first': 'B', 'middle': ['E'], 'last': 'Young', 'suffix': ''}, {'first': 'Xiang', 'middle': [], 'last': 'Ong', 'suffix': ''}, {'first': 'S', 'middle': ['W'], 'last': 'Kalimuddin', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': '', 'suffix': ''}], 'year': 2020, 'venue': 'JAMA', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF9': {'ref_id': 'b9', 'title': 'Characteristics of COVID-19 infection in Beijing', 'authors': [{'first': 'S', 'middle': ['J'], 'last': 'Tian', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Hu', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Lou', 'suffix': ''}], 'year': 2020, 'venue': 'J Infection', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF10': {'ref_id': 'b10', 'title': 'Clinical characteristics of novel coronavirus cases in tertiary hospitals in Hubei Province', 'authors': [{'first': 'K', 'middle': [], 'last': 'Liu', 'suffix': ''}, {'first': 'Y', 'middle': ['Y'], 'last': 'Fang', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Deng', 'suffix': ''}], 'year': 2020, 'venue': 'CMJ', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF11': {'ref_id': 'b11', 'title': 'Novel coronavirus infection during the 2019-2020 epidemic: preparing intensive care units-the experience in Sichuan Province', 'authors': [{'first': 'X', 'middle': ['L'], 'last': 'Liao', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'Yan', 'middle': [], 'last': 'Kang', 'suffix': ''}], 'year': 2020, 'venue': 'China. Intensive Care Med', 'volume': '46', 'issn': '2', 'pages': '357--60', 'other_ids': {}}, 'BIBREF12': {'ref_id': 'b12', 'title': 'Welcome to the Tidyverse', 'authors': [{'first': 'H', 'middle': [], 'last': 'Wickham', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Averick', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Bryan', 'suffix': ''}], 'year': 2019, 'venue': '15. Kolde R. pheatmap: Pretty Heatmaps', 'volume': '4', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF13': {'ref_id': 'b13', 'title': 'Diagnosis, evaluation, and management of acute kidney injury: a KDIGO summary', 'authors': [{'first': 'J', 'middle': ['A'], 'last': 'Kellum', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Lameire', 'suffix': ''}, {'first': 'Kagw', 'middle': [], 'last': 'Group', 'suffix': ''}], 'year': 2013, 'venue': 'Critical care', 'volume': '17', 'issn': '1', 'pages': '204--204', 'other_ids': {}}, 'BIBREF14': {'ref_id': 'b14', 'title': 'The Third International Consensus Definitions for Sepsis and Septic Shock (Sepsis-3)', 'authors': [{'first': 'M', 'middle': [], 'last': 'Singer', 'suffix': ''}, {'first': 'C', 'middle': ['S'], 'last': 'Deutschman', 'suffix': ''}, {'first': 'C', 'middle': ['W'], 'last': 'Seymour', 'suffix': ''}], 'year': 2016, 'venue': 'JAMA', 'volume': '315', 'issn': '8', 'pages': '801--810', 'other_ids': {}}, 'BIBREF15': {'ref_id': 'b15', 'title': 'APACHE II: a severity of disease classification system', 'authors': [{'first': 'W', 'middle': ['A'], 'last': 'Knaus', 'suffix': ''}, {'first': 'D', 'middle': ['E'], 'last': 'Draper', 'suffix': ''}, {'first': 'D', 'middle': ['P'], 'last': 'Wagner', 'suffix': ''}], 'year': 1985, 'venue': 'Crit Care Med', 'volume': '13', 'issn': '10', 'pages': '818--829', 'other_ids': {}}, 'BIBREF16': {'ref_id': 'b16', 'title': 'The SOFA (Sepsis-related Organ Failure Assessment) score to describe organ dysfunction/failure. On behalf of the Working Group on Sepsis-Related Problems of the European Society of Intensive Care Medicine', 'authors': [{'first': 'J', 'middle': ['L'], 'last': 'Vincent', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Moreno', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Takala', 'suffix': ''}], 'year': 1996, 'venue': 'Intensive Care Med', 'volume': '22', 'issn': '7', 'pages': '707--710', 'other_ids': {}}, 'BIBREF17': {'ref_id': 'b17', 'title': 'Clinical course and risk factors for mortality of adult inpatients with COVID-19 in Wuhan, China: A retrospective cohort study', 'authors': [{'first': 'F', 'middle': [], 'last': 'Zhou', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Yu', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Du', 'suffix': ''}], 'year': 2020, 'venue': 'Lancet', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF18': {'ref_id': 'b18', 'title': 'Hypoxia and inflammation', 'authors': [{'first': 'H', 'middle': ['K'], 'last': 'Eltzschig', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Carmeliet', 'suffix': ''}], 'year': 2011, 'venue': 'N Engl J Med', 'volume': '364', 'issn': '', 'pages': '656--65', 'other_ids': {}}, 'BIBREF19': {'ref_id': 'b19', 'title': 'Critical care utilization for the COVID-19 outbreak in Lombardy, Italy: early experience and forecast during an emergency response', 'authors': [{'first': 'G', 'middle': [], 'last': 'Grasselli', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Pesenti', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Cecconi', 'suffix': ''}], 'year': 2020, 'venue': 'JAMA', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF20': {'ref_id': 'b20', 'title': 'Potential association between COVID-19 mortality and health-care resource availability', 'authors': [{'first': 'Y', 'middle': [], 'last': 'Ji', 'suffix': ''}, {'first': 'Z', 'middle': [], 'last': 'Ma', 'suffix': ''}, {'first': 'M', 'middle': ['P'], 'last': 'Peppelenbosch', 'suffix': ''}], 'year': 2020, 'venue': 'Lancet Glob Health', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF21': {'ref_id': 'b21', 'title': 'Potential Value of Regionalized Intensive Care for Mechanically Ventilated Medical Patients', 'authors': [{'first': 'J', 'middle': ['M'], 'last': 'Kahn', 'suffix': ''}, {'first': 'W', 'middle': ['T'], 'last': 'Linde-Zwirble', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Wunsch', 'suffix': ''}], 'year': 2008, 'venue': 'Am J Respir Crit Care Med', 'volume': '177', 'issn': '', 'pages': '285--291', 'other_ids': {}}, 'BIBREF22': {'ref_id': 'b22', 'title': 'All authors contributed to the study design, conduction, data collection, writing, examination and approval of the final manuscript', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}}, 'ref_entries': {'FIGREF0': {'text': 'controlled clinical environment. Third, the findings reported in our study might not be generalizable to populations with completely different impact of the pandemic and government strategies. Covid-19 patients with severe illness should be early identified and closely monitored in a critical care setting. Sufficient conventional oxygen equipment should be well prepared and implemented as first-line treatment without delay to patients with risk of hypoxia. Lei, Yu Gong, Chao Jia, Sheng Lv, Li Chen, Juan Shang, Min Yang, Hailong Wei, Yuanjun Zhang, Xiong Yang, Huaqiang Shen, Xianhua Xiao were responsible for the local research progress and ensure the data quality; , Xuelian Liao and Jie Yang analysed data or performed statistical analysis; Yan Kang, Xuelian Liao, Zhen Li, Bo Wang and Xiaodong Jin wrote the manuscript and had primary responsibility for final content.', 'latex': None, 'type': 'figure'}, 'FIGREF1': {'text': 'Clinical Characteristics of the severe Covid-19 patients, according to outcome by', 'latex': None, 'type': 'figure'}, 'FIGREF3': {'text': 'BMI was classified into 4 groups according to the criteria for Chinese population: underweight (<18.5 kg/m2), normal (18.5-23.9 kg/m2), overweight (24-27.9 kg/m2), and obese (≥18 kg/m2).', 'latex': None, 'type': 'figure'}, 'FIGREF4': {'text': 'd. Data were missing for the measurement of C-reactive protein in 12 patients(14.8)  and D-Dimer in 4 patients (4.9%).', 'latex': None, 'type': 'figure'}, 'FIGREF5': {'text': 'Figure legend', 'latex': None, 'type': 'figure'}, 'FIGREF6': {'text': 'Distribution and referral routes of severe Covid-19 cases in Sichuan Province. Distribution of all the 81 patients with severe Covid-19 from 13 cities and 2 minority autonomous prefectures of Sichuan province were shown in the figure. The blue dot denotes the 18 designated hospital for severe cases, pink dot denotes 30 designated hospital for non-severe cases, and green dot denotes non-designated hospital. The red and green arrow represent the route taken by inpatient and outpatient cases, respectively. Centralization mode of the severe cases is clearly demonstrated.', 'latex': None, 'type': 'figure'}, 'FIGREF7': {'text': 'Daily organ support for 81 patients with severe Covid-19 from D1 to D28.', 'latex': None, 'type': 'figure'}, 'FIGREF8': {'text': 'Daily respiratory support needs for patients with severe Covid-19 from January 16 to March 15. The bar plot shows, for each calendar day, counts of the respiratory support used for patients with severe Covid-19. Daily number of newly diagnosed patients with Covid-19 disease in Sichuan, newly diagnosed severe cases, and cumulative severe cases hospitalized are shown in lines. Distribution and referral routes of severe Covid-19 cases in Sichuan Province.', 'latex': None, 'type': 'figure'}, 'FIGREF9': {'text': 'Daily organ support for 81 patients with severe Covid-19 from D1 to D28. Daily respiratory support needs for patients with severe Covid-19 from January 16 to March 15.', 'latex': None, 'type': 'figure'}, 'TABREF0': {'text': 'Various drug treatments were given despite the absence of effective evidence-based antiviral and immunomodulates. Antiviral agents lopinavir/ritnonavir and abidol were used in 71 (87.7%) and 25 (30.9%) patients, respectively. Regarding immunomodulates, nebulized recombinant human interferon α2b, corticosteroid and thymosin α were used by 57 (70.4%), 44 (54.3%) and 33 (40.7%) patients, respectively. Antibiotics were used for prevention in 58 (71.6%) patients.', 'latex': None, 'type': 'table'}, 'TABREF1': {'text': '', 'latex': None, 'type': 'table'}, 'TABREF4': {'text': 'Other scenarios included having visited a designated hospital for Covid-19 patients or being in another city with confirmed Covid-19 cases during the past 14 days. c. Other symptoms included muscle soreness, loss of appetite, diarrhea, abdominal pain, headache, dizziness, nausea, vomiting.', 'latex': None, 'type': 'table'}, 'TABREF5': {'text': 'Treatment for severe Covid-19 throughout the study period, according to outcome by D28 Respiratory Supportno. (%) Conventional oxygen therapy through mask.', 'latex': None, 'type': 'table'}}, 'back_matter': []}\n",
      "{'paper_id': 'ae4f1ac93bb55da1231f08eb1654f19352ab1a3f', 'metadata': {'title': 'Viral etiology of Acute Respiratory Infections in Hospitalized', 'authors': [{'first': 'Olga', 'middle': [], 'last': 'Kurskaya', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Tatyana', 'middle': [], 'last': 'Ryabichenko', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Novosibirsk State Medical 13 University', 'location': {'settlement': 'Novosibirsk', 'country': 'Russia'}}, 'email': ''}, {'first': 'Natalya', 'middle': [], 'last': 'Leonova', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Weifeng', 'middle': [], 'last': 'Shi', 'suffix': '', 'affiliation': {'laboratory': 'Key Laboratory of Etiology and Epidemiology of Emerging Infectious Diseases in', 'institution': 'Taishan Medical College', 'location': {'addrLine': 'Taian, Shandong', 'country': 'China 18'}}, 'email': ''}, {'first': 'Hongtao', 'middle': [], 'last': 'Bi', 'suffix': '', 'affiliation': {'laboratory': 'Qinghai Key Laboratory of Tibetan Medicine Pharmacology and Safety', 'institution': '', 'location': {'settlement': 'Xining', 'country': 'China'}}, 'email': ''}, {'first': 'Kirill', 'middle': [], 'last': 'Sharshov', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Eugenia', 'middle': [], 'last': 'Kazachkova', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Ivan', 'middle': [], 'last': 'Sobolev', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Elena', 'middle': [], 'last': '', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': '', 'middle': [], 'last': 'Prokopyeva', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Tatyana', 'middle': [], 'last': 'Kartseva', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Novosibirsk State Medical 13 University', 'location': {'settlement': 'Novosibirsk', 'country': 'Russia'}}, 'email': ''}, {'first': 'Alexander', 'middle': [], 'last': 'Alekseev', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Alexander', 'middle': [], 'last': '', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': '', 'middle': [], 'last': 'Shestopalov', 'suffix': '', 'affiliation': {}, 'email': ''}]}, 'abstract': [], 'body_text': [{'text': \"human bocavirus (BoV) are the most common viral agents associated with ARIs, 32 accounting for around 70 % of ARIs [3, 4] . The frequency of mixed respiratory viral 33 detection varies from10%to 30% in hospitalized children [5] [6] [7] . In addition, several 34 new human respiratory viruses have been described in recent years, including human 35 metapneumovirus [8, 9] , human bocavirus, and novel human coronaviruses, 36 including severe acute respiratory syndrome coronavirus (SARS-CoV) [10], human 37 coronaviruses NL63 (HCoV-NL63), HKU1 (HCoV-HKU1) [11] , and Middle East 38 respiratory syndrome coronavirus (MERS -CoV) [12] . 39 Although the majority of ARIs are associated with respiratory viruses, antibiotics December 2013 to April 2017 were enrolled in the investigation. There were 824 110 males (52.8%) and 736 females (47.2%), and the patient's ages ranged from 3 111 months to 15 years. The majority of them (43.2%) were between 1 and 3 years old.\", 'cite_spans': [{'start': 77, 'end': 79, 'text': '32', 'ref_id': 'BIBREF45'}, {'start': 115, 'end': 118, 'text': '[3,', 'ref_id': None}, {'start': 119, 'end': 121, 'text': '4]', 'ref_id': None}, {'start': 165, 'end': 167, 'text': '33', 'ref_id': None}, {'start': 224, 'end': 227, 'text': '[5]', 'ref_id': None}, {'start': 228, 'end': 231, 'text': '[6]', 'ref_id': None}, {'start': 232, 'end': 235, 'text': '[7]', 'ref_id': None}, {'start': 259, 'end': 261, 'text': '34', 'ref_id': 'BIBREF50'}, {'start': 364, 'end': 367, 'text': '[8,', 'ref_id': None}, {'start': 368, 'end': 370, 'text': '9]', 'ref_id': None}, {'start': 421, 'end': 423, 'text': '36', 'ref_id': None}, {'start': 503, 'end': 505, 'text': '37', 'ref_id': 'BIBREF55'}, {'start': 555, 'end': 559, 'text': '[11]', 'ref_id': 'BIBREF0'}, {'start': 626, 'end': 630, 'text': '[12]', 'ref_id': None}, {'start': 633, 'end': 635, 'text': '39', 'ref_id': None}], 'ref_spans': [], 'section': ''}, {'text': 'The age distribution is shown in Table 1 . The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/353037 doi: bioRxiv preprint 7 120 Among 1560 samples, 1128 (72.3 %) were found positive for at least one virus, and 121 432 (27.7%) were negative for all respiratory viruses tested (Table 1) . There was no 122 significant difference in the incidence of respiratory viral infection between boys 123 (601/824; 72.9%) and girls (527/736; 71.6%) (χ 2 = 0.345, p>0.05). The respiratory 124 virus positive rate appeared to decrease with age. The lowest positive rate was 125 observed in the age group of more than 6 years old (167/302; 55.3%), while the 126 positive rates in age groups less than 6 years old were more than 70% (Table 1) .', 'cite_spans': [], 'ref_spans': [{'start': 33, 'end': 40, 'text': 'Table 1', 'ref_id': None}, {'start': 327, 'end': 336, 'text': '(Table 1)', 'ref_id': None}, {'start': 768, 'end': 777, 'text': '(Table 1)', 'ref_id': None}], 'section': '112'}, {'text': 'Statistically significant difference was observed between the age group of more than 128 6 years old and other age groups (χ 2 = 54.113, p<0.01). No statistically significant 129 difference was observed among the age groups less than 1 year old, 1 -3 years, and HBoV with the detection rates higher than 5.0%. The positivity rates of HCoV and 143 HAdV were lower than 5.0% (Fig 2) . Among detected respiratory viruses, HRV, HPIV, HCoV, and HAdV did not have 155 statistically significant difference in the distribution among the different age groups.', 'cite_spans': [], 'ref_spans': [{'start': 373, 'end': 380, 'text': '(Fig 2)', 'ref_id': 'FIGREF8'}], 'section': '127'}, {'text': 'HMPV was detected in age group less than 1 year old much more frequently than in 157 children older than 1 year (χ 2 = 6.627, p<0.05). HBoV was significantly more 158 frequently observed in children younger than 3 years old compare with children of 159 4 -15 years old (χ 2 = 28.523, p<0.005). The incidence of RSV decreased 160 significantly with increasing age (p < 0.05) dropping from 35.1% in children 161 younger than 1 year old to 5.3% in the school-age children (7 -15 years old group), 162 while the reverse relationship was observed for IFV (Fig 3) . The data was analyzed with regard to the seasonality. Figure 4 illustrates the monthly [32] . In contrast, we collected samples only during epidemic 241 seasons of ARIs, so the positivity rate in our study was considerably higher.', 'cite_spans': [{'start': 647, 'end': 651, 'text': '[32]', 'ref_id': 'BIBREF45'}], 'ref_spans': [{'start': 550, 'end': 557, 'text': '(Fig 3)', 'ref_id': None}, {'start': 614, 'end': 622, 'text': 'Figure 4', 'ref_id': 'FIGREF12'}], 'section': '156'}, {'text': 'Furthermore, acute respiratory infections can be caused by viruses that are not yet 243 known, as well as bacteria that have not been included in these studies [13] . 244 We found that the prevalence of respiratory viruses did not differ between boys and under the 5 years of age, that is associated with immaturity of the immune system 291 and, thus, greater susceptibility to infection [44] . In our study, we significantly more 292 frequently detected cases of simultaneous infection with two or more viruses in 293 children under 7 years of age compared with children of school age (12.2% versus 294 3.4%), while there was no significant difference in the incidence of viral co-infection 295 between the age groups of 0 -1 year, 1-3 years and 4 -6 years. . CC-BY 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/353037 doi: bioRxiv preprint', 'cite_spans': [{'start': 160, 'end': 164, 'text': '[13]', 'ref_id': None}, {'start': 167, 'end': 170, 'text': '244', 'ref_id': None}, {'start': 388, 'end': 392, 'text': '[44]', 'ref_id': 'BIBREF71'}], 'ref_spans': [], 'section': '242'}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'The novel human coronaviruses NL63 345 and HKU1', 'authors': [{'first': 'K', 'middle': [], 'last': 'Pyrc', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Berkhout', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Van Der Hoek', 'suffix': ''}], 'year': 2006, 'venue': 'J Virol', 'volume': '81', 'issn': '7', 'pages': '3051--3058', 'other_ids': {}}, 'BIBREF1': {'ref_id': 'b1', 'title': 'Coronaviruses: Important Emerging Human 347', 'authors': [{'first': 'C', 'middle': ['M'], 'last': 'Coleman', 'suffix': ''}, {'first': 'M', 'middle': ['B'], 'last': 'Frieman', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF3': {'ref_id': 'b3', 'title': 'Epidemiological and clinical 350 characteristics of respiratory viral infections in children in', 'authors': [{'first': 'W', 'middle': [], 'last': 'Dong', 'suffix': ''}, {'first': 'Q', 'middle': [], 'last': 'Chen', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Hu', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'He', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Liu', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Yan', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF5': {'ref_id': 'b5', 'title': '354 et al. Seasonal influenza in adults and children-diagnosis, treatment', 'authors': [{'first': 'S', 'middle': ['A'], 'last': 'Harper', 'suffix': ''}, {'first': 'J', 'middle': ['S'], 'last': 'Bradley', 'suffix': ''}, {'first': 'J', 'middle': ['A'], 'last': 'Englund', 'suffix': ''}, {'first': 'T', 'middle': ['M'], 'last': 'File', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Gravenstein', 'suffix': ''}, {'first': 'F', 'middle': ['G'], 'last': 'Hayden', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF6': {'ref_id': 'b6', 'title': 'institutional outbreak management: clinical practice 356 guidelines of the Infectious Diseases Society of America', 'authors': [], 'year': 2009, 'venue': 'Clin Infect Dis', 'volume': '357', 'issn': '', 'pages': '1003--1035', 'other_ids': {'DOI': ['10.1086/598513']}}, 'BIBREF7': {'ref_id': 'b7', 'title': 'Reducing antibiotic use in influenza: challenges and rewards', 'authors': [{'first': 'D', 'middle': [], 'last': 'Low', 'suffix': ''}], 'year': None, 'venue': 'Clin', 'volume': '359', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF11': {'ref_id': 'b11', 'title': 'Genetic and biological 363 characterization of avian influenza H5N1 viruses isolated from wild birds 364 and poultry in Western Siberia', 'authors': [{'first': 'S', 'middle': [], 'last': 'Zolotykh', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Shestopalov', 'suffix': ''}, {'first': 'I', 'middle': [], 'last': 'Drozdov', 'suffix': ''}], 'year': 2010, 'venue': 'Arch Virol', 'volume': '155', 'issn': '7', 'pages': '1145--1150', 'other_ids': {}}, 'BIBREF12': {'ref_id': 'b12', 'title': 'The copyright holder for this preprint (which was not peer-reviewed) is the', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/353037']}}, 'BIBREF15': {'ref_id': 'b15', 'title': 'Influenza A(H5N8) Virus in Wild Aquatic Birds', 'authors': [], 'year': 2016, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF18': {'ref_id': 'b18', 'title': 'Monitoring of influenza viruses in Western Siberia', 'authors': [{'first': 'K', 'middle': [], 'last': '', 'suffix': ''}, {'first': 'Shestopalov', 'middle': ['A'], 'last': '', 'suffix': ''}], 'year': 2013, 'venue': 'Infect Genet Evol', 'volume': '372', 'issn': '', 'pages': '177--87', 'other_ids': {'DOI': ['37310.1016/j.meegid.2013.08.025.Epub']}}, 'BIBREF20': {'ref_id': 'b20', 'title': 'Virological evaluation of avian influenza virus persistence in 376 natural and anthropic ecosystems of Western Siberia (Novosibirsk Region, 377 summer 2012)', 'authors': [{'first': 'A', 'middle': [], 'last': 'Shestopalov', 'suffix': ''}], 'year': 2014, 'venue': '', 'volume': '9', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['37810.1371/journal.pone.0100859']}}, 'BIBREF21': {'ref_id': 'b21', 'title': 'High incidence of 380 multiple viral infections identified in upper respiratory tract infected 381 children under three years of age in', 'authors': [{'first': 'G', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Hu', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Bao', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Zhou', 'suffix': ''}], 'year': 2012, 'venue': 'PLoS One', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF23': {'ref_id': 'b23', 'title': 'Viral etiology of 384 acute respiratory infection in Gansu Province', 'authors': [{'first': 'G', 'middle': [], 'last': 'Huang', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Yu', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Mao', 'suffix': ''}, {'first': 'Z', 'middle': [], 'last': 'Zhu', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'Z', 'middle': [], 'last': 'Jiang', 'suffix': ''}], 'year': 2011, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF25': {'ref_id': 'b25', 'title': 'The copyright holder for this preprint (which was not peer-reviewed) is the', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/353037']}}, 'BIBREF26': {'ref_id': 'b26', 'title': 'Respiratory 387 virus infections among children in South China', 'authors': [{'first': 'X', 'middle': ['Y'], 'last': 'Cai', 'suffix': ''}, {'first': 'Q', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'G', 'middle': ['Y'], 'last': 'Lin', 'suffix': ''}, {'first': 'Z', 'middle': ['W'], 'last': 'Cai', 'suffix': ''}, {'first': 'C', 'middle': ['X'], 'last': 'Lin', 'suffix': ''}, {'first': 'P', 'middle': ['Z'], 'last': 'Chen', 'suffix': ''}], 'year': 2014, 'venue': 'J Med Virol', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF28': {'ref_id': 'b28', 'title': 'Aetiology of Acute Respiratory Tract Infections in Hospitalised 391 Children in Cyprus PLoS One', 'authors': [{'first': 'J', 'middle': [], 'last': 'Richter', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Panayiotou', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Tryfonos', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Koptides', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Koliou', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Kalogirou', 'suffix': ''}], 'year': 2016, 'venue': '', 'volume': '11', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['39210.1371/journal.pone.0147041']}}, 'BIBREF29': {'ref_id': 'b29', 'title': 'Epidemiology and 394 seasonality of respiratory viral infections in hospitalized children in Kuala 395', 'authors': [{'first': 'C', 'middle': ['S'], 'last': 'Khor', 'suffix': ''}, {'first': 'I', 'middle': ['C'], 'last': 'Sam', 'suffix': ''}, {'first': 'P', 'middle': ['S'], 'last': 'Hooi', 'suffix': ''}, {'first': 'K', 'middle': ['F'], 'last': 'Quek', 'suffix': ''}, {'first': 'Y', 'middle': ['F'], 'last': 'Chan', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF30': {'ref_id': 'b30', 'title': 'Malaysia: a retrospective study of 27 years', 'authors': [{'first': '', 'middle': [], 'last': 'Lumpur', 'suffix': ''}], 'year': 2012, 'venue': 'BMC Pediatr', 'volume': '12', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1186/1471-2431-12-32']}}, 'BIBREF31': {'ref_id': 'b31', 'title': 'Molecular 398 epidemiology of respiratory viruses in virus-induced asthma', 'authors': [{'first': 'H', 'middle': [], 'last': 'Tsukagoshi', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Ishioka', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Noda', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Kozawa', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Kimura', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF33': {'ref_id': 'b33', 'title': 'Detection and typing by molecular techniques of respiratory viruses in 402 children hospitalized for acute respiratory infection in', 'authors': [{'first': 'A', 'middle': [], 'last': 'Pierangeli', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Gentile', 'suffix': ''}, {'first': 'M', 'middle': ['P'], 'last': 'Di', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Pagnotti', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Scagnolari', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Trombetti', 'suffix': ''}], 'year': None, 'venue': 'J Med', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF36': {'ref_id': 'b36', 'title': 'Viral aetiology of influenza-like illness in Belgium during the 406 influenza A(H1N1)2009 pandemic', 'authors': [{'first': 'M', 'middle': [], 'last': '', 'suffix': ''}], 'year': 2012, 'venue': 'Eur J Clin Microbiol Infect Dis', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF38': {'ref_id': 'b38', 'title': 'The copyright holder for this preprint (which was not peer-reviewed) is the', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/353037']}}, 'BIBREF39': {'ref_id': 'b39', 'title': 'McNeal 409 MM, et al. Etiology and seasonality of viral respiratory infections in rural 410 Honduran children', 'authors': [{'first': 'E', 'middle': ['P'], 'last': 'Schlaudecker', 'suffix': ''}, {'first': 'J', 'middle': ['P'], 'last': 'Heck', 'suffix': ''}, {'first': 'E', 'middle': ['T'], 'last': 'Macintyre', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Martinez', 'suffix': ''}, {'first': 'C', 'middle': ['N'], 'last': 'Dodd', 'suffix': ''}], 'year': 2012, 'venue': 'Pediatr Infect Dis J', 'volume': '31', 'issn': '', 'pages': '1113--1121', 'other_ids': {'DOI': ['41110.1097/INF.0b013e31826052eb'], 'PMID': ['22653487']}}, 'BIBREF40': {'ref_id': 'b40', 'title': 'The epidemiology 413 and etiology of influenza-like illness in Chinese children from', 'authors': [{'first': 'J', 'middle': [], 'last': 'Peng', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Kong', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Guo', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Liu', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Zhu', 'suffix': ''}], 'year': 2008, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF42': {'ref_id': 'b42', 'title': 'Epidemiological analysis of respiratory viral 416 etiology for influenza-like illness during 2010 in Zhuhai, China', 'authors': [{'first': 'H', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'Q', 'middle': [], 'last': 'Wei', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Tan', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Wang', 'suffix': ''}], 'year': 2013, 'venue': 'Virol J', 'volume': '417', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1186/1743-422X-10-143']}}, 'BIBREF44': {'ref_id': 'b44', 'title': 'The features of ARVD etiological structure in 420 different age and professional population groups in Saint-Petersburg during 421 2013-2014 epidemic season', 'authors': [{'first': 'M', 'middle': ['A'], 'last': 'Mikhailova', 'suffix': ''}], 'year': 2014, 'venue': 'Journal of Infectology', 'volume': '6', 'issn': '3', 'pages': '62--70', 'other_ids': {}}, 'BIBREF45': {'ref_id': 'b45', 'title': 'Viral etiology of influenza-424 like illnesses in Huizhou', 'authors': [{'first': 'X', 'middle': [], 'last': 'Ju', 'suffix': ''}, {'first': 'Q', 'middle': [], 'last': 'Fang', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Xu', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Liang', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Ke', 'suffix': ''}], 'year': 2009, 'venue': 'Arch Virol', 'volume': '425', 'issn': '8', 'pages': '2003--2013', 'other_ids': {'DOI': ['10.1007/s00705-014-2035-1.Epub']}}, 'BIBREF47': {'ref_id': 'b47', 'title': 'Aetiology of influenza-like illness in adults includes parainfluenzavirus type 428', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF49': {'ref_id': 'b49', 'title': 'The copyright holder for this preprint (which was not peer-reviewed) is the', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/353037']}}, 'BIBREF50': {'ref_id': 'b50', 'title': 'Nationwide surveillance of 18 respiratory viruses in patients with 432 influenzalike illnesses. A pilot feasibility study in the French Sentinel 433 network', 'authors': [{'first': 'A', 'middle': [], 'last': 'Falchi', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Turbelin', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Andreoletti', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Arena', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Blanchon', 'suffix': ''}, {'first': 'I', 'middle': [], 'last': 'Bonmarin', 'suffix': ''}], 'year': 2011, 'venue': 'J Med Virol', 'volume': '83', 'issn': '8', 'pages': '1451--1458', 'other_ids': {'DOI': ['10.1002/jmv.22113.434']}}, 'BIBREF52': {'ref_id': 'b52', 'title': 'Viral etiologies of acute respiratory infections among hospitalized 437', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF53': {'ref_id': 'b53', 'title': 'Ho Chi Minh City', 'authors': [], 'year': 2004, 'venue': '', 'volume': '6', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1371/journal.pone.0018176']}}, 'BIBREF54': {'ref_id': 'b54', 'title': 'A 3-year 440 prospective study of the epidemiology of acute respiratory viral infections in 441 hospitalized children in Shenzhen, China Influenza Other Respir Viruses', 'authors': [{'first': 'Y', 'middle': [], 'last': 'He', 'suffix': ''}, {'first': 'G', 'middle': ['Y'], 'last': 'Lin', 'suffix': ''}, {'first': 'Q', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'X', 'middle': ['Y'], 'last': 'Cai', 'suffix': ''}, {'first': 'Y', 'middle': ['H'], 'last': 'Zhang', 'suffix': ''}, {'first': 'C', 'middle': ['X'], 'last': 'Lin', 'suffix': ''}], 'year': 2014, 'venue': '', 'volume': '442', 'issn': '', 'pages': '443--51', 'other_ids': {'DOI': ['10.1111/irv.12257.Epub']}}, 'BIBREF55': {'ref_id': 'b55', 'title': 'Possible Risk Factors, Viral Codetection Rates, and 445 Seasonality of Respiratory Syncytial Virus Among Children With Lower 446', 'authors': [{'first': 'R', 'middle': ['Q'], 'last': 'Gurgel', 'suffix': ''}, {'first': 'P', 'middle': ['G'], 'last': 'Bezerra', 'suffix': ''}, {'first': 'Duarte', 'middle': [], 'last': 'Mdo', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Moura', 'suffix': ''}, {'first': 'A', 'middle': ['Á'], 'last': 'Souza', 'suffix': ''}, {'first': 'E', 'middle': ['L'], 'last': 'Silva', 'suffix': ''}, {'first': 'L', 'middle': ['S'], 'last': '', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF56': {'ref_id': 'b56', 'title': 'Respiratory Tract Infection in Northeastern Brazil', 'authors': [], 'year': 2016, 'venue': '', 'volume': '447', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1097/MD.0000000000003090']}}, 'BIBREF58': {'ref_id': 'b58', 'title': 'Respiratory Viruses Associated Hospitalization among Children Aged< 5', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF60': {'ref_id': 'b60', 'title': 'The copyright holder for this preprint (which was not peer-reviewed) is the', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/353037']}}, 'BIBREF62': {'ref_id': 'b62', 'title': 'Viral aetiology influenza like illnesses in Santa', 'authors': [{'first': 'L', 'middle': [], 'last': 'Thirion-Perrier', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF64': {'ref_id': 'b64', 'title': 'Understanding the symptoms of the common cold and influenza', 'authors': [{'first': 'R', 'middle': [], 'last': 'Eccles', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF66': {'ref_id': 'b66', 'title': 'On the outcome of the epidemic season of 459 influenza and ARIs', 'authors': [], 'year': 2018, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF68': {'ref_id': 'b68', 'title': 'Viral Co-Infections in Pediatric Patients 463 Hospitalized with Lower Tract Acute Respiratory Infections', 'authors': [{'first': 'N', 'middle': [], 'last': 'Torres', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Salas', 'suffix': ''}], 'year': 2015, 'venue': 'PLoS One', 'volume': '464', 'issn': '9', 'pages': '', 'other_ids': {'DOI': ['10.1371/journal.pone.0136526.465']}}, 'BIBREF70': {'ref_id': 'b70', 'title': 'Disease Severity of Respiratory Viral Co-Infection versus Single Viral 468 Infection: A Systematic Review and Meta-Analysis', 'authors': [], 'year': 2014, 'venue': 'PLoS One', 'volume': '9', 'issn': '6', 'pages': '', 'other_ids': {'DOI': ['10.1371/journal.pone.0099392']}}, 'BIBREF71': {'ref_id': 'b71', 'title': 'Surveillance of 16 471 respiratory viruses in patients with influenza-like illness in Nanjing', 'authors': [{'first': 'X', 'middle': [], 'last': 'Huo', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Qin', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Qi', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Zu', 'suffix': ''}, {'first': 'F', 'middle': [], 'last': 'Tang', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Li', 'suffix': ''}], 'year': None, 'venue': 'China. J', 'volume': '472', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF73': {'ref_id': 'b73', 'title': 'International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/353037']}}}, 'ref_entries': {'FIGREF0': {'text': 'causing considerable morbidity and mortality among people of all age 26 groups [1]. Children are on average infected two to three times more frequently than 27 adults. [2]. There are more than 200 respiratory viruses that can cause ARIs.28 Respiratory syncytial virus (RSV), human rhinovirus (HRV), human 29 metapneumovirus (HMPV), human parainfluenza virus (PIV), human enterovirus 30 (EV), influenza virus (IFV), human coronavirus (CoV), adenovirus (ADV), and 31', 'latex': None, 'type': 'figure'}, 'FIGREF1': {'text': \"40 are often used in the clinical treatment of ARIs. As children with ARTIs often have 41 similar clinical symptoms, studying the clinical characteristics of children with diseases. This territory was involved in the circulation of A/H5N1 and A/H5N8 48 avian influenza viruses in 2005 -2017 [16, 17]. These viruses were spread by wild 49 birds' migration. Western Siberia is a place of crossing of birds' migratory flyways 50 wintering in different regions of the world: Europe, Africa, Middle East, Central 51 Asia, Hindustan, and South East Asia. Therefore, there is high probability of 52 emergence of reassortant strains between human and animal influenza viruses, as 53 well as emergence of local outbreaks of human morbidity caused by uncommon 54 variants of influenza viruses. Furthermore, Novosibirsk is the largest transport hub 55 in this part of Russia with numerous international connections, that is important for 56 the spread of ARIs [18, 19]. 57 The prevalence of respiratory viruses among children with ARIs differs in different 58 regions and varies over time [20-24]. Thus, to better understand the epidemiology 59 of Acute Respiratory Infections in Russia, we investigated etiology of ARIs in 60 children admitted to Novosibirsk Children's Municipal Clinical Hospital in 2013 -61 2017.\", 'latex': None, 'type': 'figure'}, 'FIGREF3': {'text': '77 epidemiological and clinical information including case history, symptoms, physical 78 signs, and examination were included in a standardized questionnaire. The samples 79 were placed immediately in viral transport medium (Eagle MEM, BSA and 80 antibiotics) and stored at 4-8ºC prior transportation to the laboratory (not more than 81 24 hours). Detection of respiratory viruses was performed immediately after delivery 82 to the laboratory. All specimens were tested for 15 common respiratory viruses, 83 including influenza virus types A, B (IFVA and IFVB), human parainfluenza virus 84 (HPIV) types 1-4, human respiratory syncytial virus (HRSV), human 85 metapneumovirus (HMPV), four human coronaviruses (HCo), human rhinovirus acids were extracted from nasal and throat swabs using RNA/DNA 90 extraction kit «RIBO-sorb» (Interlabservice, Russia) according to the 91 manufacturer\\'s instructions. The extracted viral nucleic acid was immediately used 92 to perform the reaction of reverse transcription using commercial kit \"REVERTA-93 L\" (Interlabservice, Russia).', 'latex': None, 'type': 'figure'}, 'FIGREF4': {'text': \"respiratory viruses, including HPIV 1-4, HRSV, HMPV, HCoV-OC43, 96 HCoV-229E, HCoVNL63, HCoV-HKU1, HRV, HAdV, and HBoV was performed 97 using a RT-PCR Kit «AmpliSens ARVI-screen-FL» (Interlabservice, Russia), and 98 IFVA and IFVB virus detection was performed using a RT-PCR Kit « AmpliSens 99 Influenza virus A/B-FL» (Interlabservice, Russia) according to the manufacturer's 100 instructions. Positive and negative controls were included in each run.\", 'latex': None, 'type': 'figure'}, 'FIGREF7': {'text': 'the samples, single infections accounted for 61.9% (965/1560), while co-132 infections accounted for 10.4% (163/1560) with the lowest rate of incidence in 133 children more than 6 years old compared to children younger than 6 years (Viral co-infection rate in different age groups. IFV were the most frequently detected viruses with high incidence of 140 23.0% (358/1560) and 22.1% (344/1560), respectively, among all patients with 141 ARIs. HRV was detected in 15.1% (235/1560), followed by HMPV, HPIV and', 'latex': None, 'type': 'figure'}, 'FIGREF8': {'text': 'Detection rates of viral pathogens in single and co-infections in children was analyzed with regard to the age and gender distribution of virus 150 infection. Of the enrolled patients, 824 (52.8%) were male and 736 (47.2%) were 151 female. All the patients were grouped into four age groups with different positive 152 rate of viral infections. No difference in the etiological distribution of viral 153 pathogens was observed between males and females.', 'latex': None, 'type': 'figure'}, 'FIGREF10': {'text': 'Fig 3. The distribution of respiratory viruses in different age groups.', 'latex': None, 'type': 'figure'}, 'FIGREF11': {'text': '168 distribution of the most frequently detected viruses (HRSV, HRV and IFV) from 169 2013 to 2017. We have observed no considerable activity of Influenza viruses in 170 2013 -2014 epidemic season, but increasing activity detected in subsequent years 171 with peaks in February 2015, February 2016 and January and February 2017. HRSV 172 exhibited marked peaks during each season: in January 2014, March 2015, 173 December 2015 and March 2017. For HRV monthly distribution was relatively 174 constant with only one clear peak in October -November 2014(Fig 4).', 'latex': None, 'type': 'figure'}, 'FIGREF12': {'text': 'Monthly distribution of HRSV, HRV and IFV. infections with two or more viruses were detected in 163 out of the 1128 (14.5%) 180 positive samples (Table 1). Dual infections accounted for 11.4% (129/1128) of all 181 positive samples and three viruses were detected in 3.1% (34/1128) of positive 182 samples. Most co-infected patients were 0-6 years of age (12.2%, 153/1258) versus 183 children older than 6 years (3.4%, 10/292). No significant difference was found for 184 incidence of co-infections between the age group less than 1 year (13.2%, 43/325), 185 1 -3 years (12.9%, 87/674), and 4 -6 years (8.9%, 23/259). The most common each individual virus detected varied significantly. Viruses appearing most 189 often in co-infections were DNA-viruses -HAdV and HBoV -in 52.7% (29/55) of 190 cases of adenovirus detection and in 45.1% (41/91) of cases of bocavirus detection. 191 HRV was the most often co-infected with HBoV (34.1%, 14/41) and HAdV (31%, 192 9/29). We have not detected any case of simultaneous infection of HAdV and HBoV. 193 All occurring combinations of viruses are shown in table 2.', 'latex': None, 'type': 'figure'}, 'FIGREF13': {'text': \"Detection of single and co-infection cases among 1560 children with 196 ARI in Novosibirsk Municipal Clinical Hospital from 2013 one of the most frequently detected viruses among children with ARIs with 200 detection rate 22.1% (344/1560). The lowest detection rate of IFV was in the less 201 than 1 year old age group (5.8%, 19/325). The incidence of IFV increased 202 significantly with increasing patients' age (p-value < 0.0001) showing 32.6% 203 (183/561) in children older than 3 years. 204 During the study period the lowest influenza activity was investigated in 2013 -205 2014 with positivity rate 6.9% of all positive samples. In 2014 -2015 influenza virus 206 detection was 17.2% while in 2015 -2016 and 2016 -2017 the detection rates were 207 much higher and amounted to 32.2% and 30.2% respectively. Influenza А(H3N2) 208 virus was predominant in 2013 -2014 and 2014 -2015 accounting for 57.9% and 209 69.8% of all influenza virus detections while in 2015 -2016 82% of influenza 210 viruses were A(H1N1) pdm09. We have not detected any influenza А(H3N2) 211 viruses during 2015 -2016 epidemic season. In 2016 -2017 influenza type B 212 detections (52%) predominating over type A (48%). Of influenza A viruses, all of 213 them were A(H3N2) viruses (Fig 5).\", 'latex': None, 'type': 'figure'}, 'FIGREF14': {'text': \"infections are a serious health and economic problem, causing 219 high morbidity and significant economic losses due to temporary disability 220 payments and medical costs. Children are the most susceptible group to the 221 development of the disease. ARIs can lead to serious diseases such as bronchiolitis 222 and pneumonia and sometimes even cause death in infants and children worldwide 223 [25]. Nevertheless, most of the data on the epidemiological features and etiological 224 structure of ARIs were from more-developed countries, and less is known about the 225 etiology of ARIs in Russia. In the present study, we examined the viral etiology of 226 ARIs in hospitalized children in Novosibirsk by Real-Time RT-PCR assay. 227 We detected at least one of the tested viruses in 72.3% (1128/1560) of the samples. 228 In similar studies conducted in different regions of the world, the virus detection rate 229 ranged from less than 50% to 75% [26-28]. For example, in studies performed in 230 China, the proportion of positive samples in children with ARIs ranged from 37.6% 231 to 78.7% [14,16,29,30]. The previous study of respiratory infections among children 232 in European part of Russia revealed the 71.5% detection rate of respiratory viruses 233 [31]. 234 The percentage of the respiratory viruses' detection varies in different years in 235 different regions, which may be associated with climatic and environmental factors, seasonality of sampling can also lead to differences in the level of viruses' 238 detection in various studies. Thus, Ju X. et al. carried out a study continuously from 239 July 2011 through July 2013 and found 48.66 % of samples to be positive for at least 240 one respiratory virus\", 'latex': None, 'type': 'figure'}, 'FIGREF15': {'text': \"245 girls (72.9% and 71.6% respectively), which confirms the absence of a gender-based 246 susceptibility to respiratory viral infections [13]. However, we observed a decrease 247 in the respiratory viruses' detection rate with age, with the lowest detection rate in 248 school-age children compared to children under 7 years of age (55.3% versus 76.4%, 249 respectively). These data are consistent with findings obtained in other regions of 250 Russia [31] and it may be due to decreased sensitivity to respiratory virus infections 251 in older children. 252 Etiology of ARIs and the respiratory virus prevalence varies in different studies. In 253the United States, IFV, HRSV and HPIV were the most frequently detected[33].254 Studies conducted in France have shown that metapneumovirus and respiratory 255 syncytial virus are the most common [34]. IFV, HRSV and HRV were the most 256 commonly detected respiratory viruses among children with ARIs in most regions 257 of China [13]. The study of ARIs etiologic structure showed that HRSV, HRV, HPIV study the most common viruses detected were HRSV and IFV, followed by 261 HRV. The age distribution of ARIs indicated that children under 3 years old were 262 more likely to be infected by HRSV confirmed the importance of RSV in children 263 with ARIs, especially in children < 4 years of age [35-38]. We observed a high rate 264 of RSV-detections in 2013 -2014 (44.4%), while in 2016 -2017 it was less than 265 10% which could be due to the annual variation in the circulation pattern of RSV. 266 Such year-to-year variation in the epidemiological patterns of viral infections 267 confirms importance of the long-term study of the ARIs epidemiology [39]. 268 Influenza virus is one of the major causative agents of respiratory disease in humans 269 and leads to a more severe disease than the common cold which is caused by a 270 different type of virus [40]. In temperate countries influenza outbreaks usually occur 271 during the winter season. Finally, in our study, IFV (344/1560, 22.1%) was the 272 second most frequent detected pathogen with markable seasonality in winter months. 273 During the 2013-2014 epidemic season, influenza virus detection rate was low -274 6.9% of all respiratory viruses, which was in accordance with the official influenza 275 surveillance results of Ministry of Health, Russia [41]. In 2014-2015, influenza 276 viruses were detected in 17.2% among all respiratory viruses. Herewith, the 277 percentage of influenza A virus accounted for 73.3% and influenza B virus -26.7% 278 of all detected influenza viruses. At the same time, in Russia influenza B was the 279 main etiological agent, accounting for 50.6% of all detected influenza viruses. In cases [41]. In our study influenza A and B viruses were detected with 284 approximately the same frequency (48% and 52% respectively). 285 With the introduction of molecular techniques, the detection of multiple co-infecting 286 viruses has become common, though the prevalence of each virus varies between 287 studies [42]. In our study detection rate of viral co-infection was 14.5% among the 288 positive samples. According to the previous reports, the incidence of viral co-289 infection in children can reach 30% [43]. Most often co-infection found in children290\", 'latex': None, 'type': 'figure'}, 'FIGREF16': {'text': 'Thus, in our study we investigated the etiological structure of acute respiratory viral297 infections in hospitalized children in Novosibirsk, Russia, and evaluated age and 298 seasonal distribution of the various respiratory viruses. Systematic monitoring of 299 respiratory viruses is necessary to better understand the structure of respiratory 300 infections. Such studies are important for the improvement and optimization of.', 'latex': None, 'type': 'figure'}, 'FIGREF17': {'text': 'H, Zheng Yu, Deng J, Wang W, Liu P, Yang F, et al. Prevalence of 311 respiratory viruses among children hospitalized from respiratory infections 312 in Shenzhen, China. Virol J. 2016; 13:39. DOI 10.1186/s12985-016-0493-7 313 2. Williams BG, Gouws E, Boschi-Pinto C, Bryce J, Dye C. Estimates of 314 world-wide distribution of child deaths from acute respiratory infections. 315 Lancet Infect Dis. 2002; 2:25-32; 316 3. Kusel MM, de Klerk NH, Holt PG, Kebadze T, Johnston SL, Sly PD. Role 317 of respiratory viruses in acute upper and lower respiratory tract illness in the 318 first year of life: a birth cohort study. Pediatr Infect Dis J.', 'latex': None, 'type': 'figure'}, 'FIGREF18': {'text': 'Brittain-Long R, Nord S, Olofsson S, Westin J, Anderson LM, Lindh M. 321 Multiplex real-time PCR for detection of respiratory tract infections. J Clin 322 Virol. 2008; 41:53-56; C, Mihai ME, Alexandrescu VI, Orăşeanu D, Zapucioiu C, Ivanciuc 331 AE, et al., Single and multipathogen viral infections in hospitalized children 332 with acute respiratory infections. Roum Arch Microbiol Immunol. den Hoogen BG, de Jong JC, Groen J, Kuiken T, de Groot R, Fouchier 335 RA, et al. A newly discovered human pneumovirus isolated from young 336 children with respiratory tract disease. Nat Med. 2001 Jun;7(6):719-24; 337 9. Feuillet F, Lina B, Rosa-Calatrava M, Boivin G. Ten years of human 338 metapneumovirus research. J Clin Virol. 2012 Feb;53(2):97-105. doi: 339 10.1016/j.jcv.2011.10.002. Epub 2011 Nov 9; 340 10. Drosten C, Günther S, Preiser W, van der Werf S, Brodt HR, Becker S, et 341 al. Identification of a novel coronavirus in patients with severe acute 342 respiratory syndrome. N Engl J Med. 2003 May 15;348(20):1967-76. Epub 343 2003 Apr 10;', 'latex': None, 'type': 'figure'}, 'TABREF0': {'text': 'All aspects of the study were approved by the Ethics Committee of the Federal State Budgetary Institution \"Research Center of Clinical and Experimental Medicine\" (2013-23). Accordingly, written informed consent was obtained from parents prior to sample taking.Patients and specimensParticipants enrolled to the study were children 0-15 years of age within 3 days of illness onset and had at least two of the following symptoms: fever, sore throat, cough, rhinorrhea, nasal congestion, sputum, shortness of breath, lung auscultation abnormalities, tachypnea, and chest pain. Paired nasal and throat swabs werecollected from each patient admitted to Novosibirsk Children\\'s Municipal Clinical Hospital by hospital nurses. A total of 1560 samples collected during four epidemic seasons of 2013 -2017 (October -April) were enrolled to the study. The', 'latex': None, 'type': 'table'}, 'TABREF1': {'text': 'Two-tailed chi-square test (two by two table) was performed to compare the infection rates for respiratory viruses among different age groups. P-value <0.05 was Totally, 1560 samples collected from patients with ARI during the period from', 'latex': None, 'type': 'table'}, 'TABREF2': {'text': \"CC-BY 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101diagnostic tactics, as well as measures for the control and prevention of the 302 respiratory viral infections. We thank the clinicians of Novosibirsk Children's Municipal Clinical Hospital for their assistance with sample collection. We thank Dr. Galina Skosyreva and Elena Timofeeva (Department of propaedeutic of childhood diseases, Novosibirsk State Medical University, Novosibirsk, Russia) for their assistance with sample collection.\", 'latex': None, 'type': 'table'}}, 'back_matter': []}\n",
      "{'paper_id': 'ae61300f4e7a12a68baf114f199eb51940c7aad3', 'metadata': {'title': 'Prediction of the Epidemic of COVID-19 Based on Quarantined Surveillance in China', 'authors': [{'first': 'Rui', 'middle': [], 'last': 'Li', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Wuhan Red Cross Hospital', 'location': {'settlement': 'Wuhan'}}, 'email': ''}, {'first': 'Wenliang', 'middle': [], 'last': 'Lu', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Xifei', 'middle': [], 'last': 'Yang', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Peihua', 'middle': [], 'last': 'Feng', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': \"Xi'an Jiaotong University\", 'location': {'settlement': \"Xi'an,Shaanxi\", 'country': 'China'}}, 'email': ''}, {'first': 'Ozarina', 'middle': [], 'last': 'Muqimova', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Wuhan University', 'location': {'addrLine': 'Wuhan,Hubei Province', 'country': 'China'}}, 'email': ''}, {'first': 'Xiaoping', 'middle': [], 'last': 'Chen', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Gang', 'middle': [], 'last': 'Wei', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'M', 'middle': ['D'], 'last': '', 'suffix': '', 'affiliation': {}, 'email': ''}]}, 'abstract': [{'text': 'To predict the epidemic of COVID-19 based on quarantined surveillance from real world in China by modified SEIR model different from the previous simply mathematical model.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'We forecasted the epidemic of COVID-19 based on current clinical and epidemiological data and built a modified SEIR model to consider both the infectivity during incubation period and the influence on the epidemic from strict quarantined measures.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}], 'body_text': [{'text': 'The peak time of the curve for the infected newly diagnosed as', 'cite_spans': [], 'ref_spans': [], 'section': 'Results'}, {'text': 'should substantially present on Feb.5,2020(in non-Hubei areas) and Feb. 19,2020(in Hubei) .It is estimated that the peak of the curve of the cumulative confirmed cases will appear in non-Hubei areas on Mar.3,2020 and in Hubei province on Mar.10,2020,and the total number of the patients diagnosed as COVID-19 is 18,000 in non-Hubei areas and 78,000-96,000 in Hubei.The Chinese', 'cite_spans': [{'start': 72, 'end': 89, 'text': '19,2020(in Hubei)', 'ref_id': None}], 'ref_spans': [], 'section': 'Results'}, {'text': 'epidemic can be completetly controlled in May,2020.', 'cite_spans': [], 'ref_spans': [], 'section': 'Results'}, {'text': 'COVID-19 is only a local outbreak in Hubei Province,China.It can be probably avoided the pandemic of global', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusions'}, {'text': 'cases rise with the great efforts by Chinese government and its people.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusions'}, {'text': 'All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Background and Objective'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02. 27.20027169 doi: medRxiv preprint Huanggang and Xiaogan(accounting for 27.0% together),and with 55% of the outflow outside of Hubei Province,including Henan, Guangdong,Zhejiang,Hunan,Jiangxi and Anhui together.The remainder population was 9 million in Wuhan,which was the most severe epidemic region.The strict blocked and isolated measures had been taken in other cities in Hubei before the local outbreak of SARS-CoV-2,which had not caused fulminating infection similar to Wuhan.In Hubei and other areas with serious epidemic situation,Chinese government had taken the strict blocked and isolated control,including the infected,the suspiciously infected individuals and the susceptible intimately contacted with the infected and the suspiciously infected individuals,involved the independent community buildings and villages,which were separated and quarantined in a unified way.On Jan.28,2020,JAMA magazine interviewed Professor NIH of the United States,and affirmed that the Chinese government adopted the strategy of closure and quarantine against the epidemic.', 'cite_spans': [{'start': 111, 'end': 144, 'text': '27.20027169 doi: medRxiv preprint', 'ref_id': None}], 'ref_spans': [], 'section': 'Background and Objective'}, {'text': 'Many academics tried to estimate key epidemiological characteristics and possible outcomes of the epidemic by means of transmission model mathematically.On Jan.24,2020,Jonathan M. Read 3 estimated the basic reproduction index(R0) for SARS-CoV-2 to be between 3.6 and 4.0,then and COVID-19 would outbreak further both in other Chinese cities and in international travel destinations sucn as Thailand,Japan,Taiwan,Hong Kong and South Korea at an sharply increasing rate.According to his model,he predicted the number of infected individuals in Wuhan to be greater than 190 thousand by Feb. 14,2020 although with a 99% effective reduction as a result of trave restrictions,the size of the epidemic outside of Wuhan might only be reduced by 24.9%.His model suggested that travel limitation from and to Wuhan city were unlikely to be effective in halting transmission across China.', 'cite_spans': [], 'ref_spans': [], 'section': 'Background and Objective'}, {'text': 'Professor Joseph T Wu 4 published a paper in the Lancet on Jan. 31,2020.The R0 of SARS-CoV-2 was estimated to be 2.68 by MCMC method,and the SEIR model was used to predict the epidemic.', 'cite_spans': [], 'ref_spans': [], 'section': 'Background and Objective'}, {'text': 'However,there were some deficiencies in his study:1)the main population outflow from Wuhan was not outside of Hubei Province:his study focused on several large cities (Chongqing,Beijing,Shanghai,Guangzhou and Shenzhen),instead of the moderate-sized urban All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Background and Objective'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the ', 'cite_spans': [], 'ref_spans': [], 'section': 'Background and Objective'}, {'text': 'All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Design and Methods'}, {'text': 'author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Design and Methods'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02. 27.20027169 doi: medRxiv preprint There is no quarantined measure in the early stage of the epidemic,so R0,as the basic reproductive index,is often high for SARS-CoV-2, 6,7 and there is a significant difference between in Hubei and in non-Hubei areas,but both decrease gradually with the strengthening of isolated measures. 8, 9 We forecasted the epidemic of COVID-19 by modified SEIR model 10,11 and current clinical and epidemiological data on COVID-19,involved in two practical factors to correct the model:1)considering the infectivity during incubation period 12 ,2)considering the influence on the epidemic from strict quarantined measures.It had been confirmed that both the latent and the infected are infectious.We assumed that both were with the same infectivity. According to the epidemic data before Feb.1,2020,it indicated that the probability from the exposed to the infected was 1.7%, 13,14 and the average incubation period was 7 days,and the average hospital stay was 14 days. 15, 16 The average daily contact rate of the infected was 6,referring to the statistical data from CCDC in the early stage of SARS epidemic in 2003. 17,18, 19 However,the population density in Hubei Province was 2.3 times the national average, so the daily contact rate in Hubei was 13.8,and the average daily contact rate of the latent was twice as the infected.The calculation method was R0 = average daily contact rate× probability of the exposed to the infected ×infection period. The average infection period was 21 days. Therefore, R0 was 4.9-9.9 in Hubei,but 2. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 111, 'end': 144, 'text': '27.20027169 doi: medRxiv preprint', 'ref_id': None}, {'start': 438, 'end': 439, 'text': '9', 'ref_id': 'BIBREF10'}, {'start': 1109, 'end': 1111, 'text': '16', 'ref_id': 'BIBREF14'}, {'start': 1261, 'end': 1263, 'text': '19', 'ref_id': 'BIBREF17'}], 'ref_spans': [], 'section': 'Design and Methods'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.20027169 doi: medRxiv preprint days, and aimed to obtain the possible total number of the infected individuals diagnosed as COVID-19 in Hubei and in non-Hubei areas and the time elimilating the epidemic of SARS-CoV-2 in Hubei and in non-Hubei areas.The modified SEIR model was as listed below( Table 1) . author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [{'start': 407, 'end': 415, 'text': 'Table 1)', 'ref_id': 'TABREF2'}], 'section': 'Design and Methods'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02. 27.20027169 doi: medRxiv preprint The ultimate population outflow from Wuhan on Jan. 23,2020 was the most source of transmission,based on the incubation period of 1-14 days,so the peak time of the curve for the infected newly diagnosed as', 'cite_spans': [{'start': 111, 'end': 144, 'text': '27.20027169 doi: medRxiv preprint', 'ref_id': None}], 'ref_spans': [], 'section': 'Design and Methods'}, {'text': 'should substantially present on Feb.5,2020(in non-Hubei areas) and Feb.19,2020(in Hubei) for the intensive blocked and isolated measures resulting in population mobility accessible to zero during the Spring Festival (Figure 4,Figure 6 ).It is estimated that the peak of the curve of the cumulative confirmed cases will appear in non-Hubei areas on Mar.3,2020 and in Hubei province on Mar.10,2020,and the total number of the patients diagnosed as COVID-19 is 18,000 in non-Hubei areas and 78,000-96,000 in Hubei.The Chinese', 'cite_spans': [], 'ref_spans': [{'start': 216, 'end': 234, 'text': '(Figure 4,Figure 6', 'ref_id': 'FIGREF7'}], 'section': 'Design and Methods'}, {'text': 'epidemic can be completetly controlled in May,2020 (Figure 5,Figure 7) . It can be probably avoided the pandemic of global', 'cite_spans': [], 'ref_spans': [{'start': 51, 'end': 70, 'text': '(Figure 5,Figure 7)', 'ref_id': 'FIGREF8'}], 'section': 'Design and Methods'}, {'text': 'cases rise with the great efforts by Chinese government and its people. 26,27', 'cite_spans': [], 'ref_spans': [], 'section': 'Design and Methods'}, {'text': 'From the above,it could be seen that our study was principally based on the epidemiological author/funder, who has granted medRxiv a license to display the preprint in perpetuity. ', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion and Conclusions'}, {'text': 'cases rise with the great efforts by Chinese government and its people. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion and Conclusions'}, {'text': 'author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion and Conclusions'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.20027169 doi: medRxiv preprint q Quarantined rate of the exposed individuals 5×10 -7 υ 1 Rate of removing the quarantined( observed medically) who are not 1/14(0.07) infected but intimate contact with the exposed or the infected υ 2 Convertion speed rate of the exposed persons to the infected persons 1/7(0.14) author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion and Conclusions'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02. 27.20027169 doi: medRxiv preprint ', 'cite_spans': [{'start': 111, 'end': 144, 'text': '27.20027169 doi: medRxiv preprint', 'ref_id': None}], 'ref_spans': [], 'section': 'Discussion and Conclusions'}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'A Novel coronavirus from patients with pneumonia in china', 'authors': [{'first': 'N', 'middle': [], 'last': 'Zhu', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Wang', 'suffix': ''}], 'year': 2019, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF2': {'ref_id': 'b2', 'title': 'MRC Centre for Global Infectious Disease Analysis:Wuhan coronavirus reports 1-3', 'authors': [{'first': 'N', 'middle': [], 'last': 'Imai', 'suffix': ''}, {'first': 'Cori', 'middle': ['A'], 'last': 'Dorigatti', 'suffix': ''}, {'first': 'I', 'middle': [], 'last': '', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF3': {'ref_id': 'b3', 'title': 'Novel coronavirus 2019-nCoV: Early estimation of epidemiological parameters and epidemic predictions. medRxiv2020', 'authors': [{'first': 'J', 'middle': ['M'], 'last': 'Read', 'suffix': ''}, {'first': 'J', 'middle': ['R'], 'last': 'Bridgen', 'suffix': ''}, {'first': 'D', 'middle': ['A'], 'last': 'Cummings', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['org/10.1101/2020.01.23.20018549']}}, 'BIBREF4': {'ref_id': 'b4', 'title': 'Nowcasting and Forecasting the Potential Domestic and International Spread of the 2019-nCoV Outbreak Originating in Wuhan, China: A Modelling Study', 'authors': [{'first': 'T', 'middle': [], 'last': 'Joseph', 'suffix': ''}, {'first': 'Kathy', 'middle': [], 'last': 'Wu', 'suffix': ''}, {'first': '', 'middle': [], 'last': 'Leung', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Gabriel', 'suffix': ''}, {'first': '', 'middle': [], 'last': 'Leung', 'suffix': ''}], 'year': 2020, 'venue': 'Lancet', 'volume': '', 'issn': '', 'pages': '30260--30269', 'other_ids': {'DOI': ['10.1016/S0140-6736(20']}}, 'BIBREF5': {'ref_id': 'b5', 'title': 'A mathematical model for simulating the transmission of Wuhan novel coronavirus', 'authors': [{'first': 'T', 'middle': [], 'last': 'Chen', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Rui', 'suffix': ''}, {'first': 'Q', 'middle': [], 'last': 'Wang', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '2020', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.01.19.911669v1']}}, 'BIBREF6': {'ref_id': 'b6', 'title': 'Early transmission dynamics in Wuhan,China,of novel coronavirus-infected pneumonia', 'authors': [{'first': 'Q', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Guan', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Wu', 'suffix': ''}], 'year': 2020, 'venue': 'N Engl J Med', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1056/NEJMoa2001316']}}, 'BIBREF7': {'ref_id': 'b7', 'title': 'Preparedness and proactive infection control measures against the emerging Wuhan coronavirus pneumonia in China', 'authors': [{'first': 'V', 'middle': ['C C'], 'last': 'Cheng', 'suffix': ''}, {'first': 'S', 'middle': ['C'], 'last': 'Wong', 'suffix': ''}], 'year': 2020, 'venue': 'J. Hosp. Infect', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1016/j.jhin.2020.01.010']}}, 'BIBREF8': {'ref_id': 'b8', 'title': 'Transmission dynamics of 2019 novel coronavirus (2019-nCoV)', 'authors': [{'first': 'T', 'middle': [], 'last': 'Liu', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Hu', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Kang', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '2020', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF10': {'ref_id': 'b10', 'title': 'Preliminary estimation of the basic reproduction number of novel coronavirus contact in Germany', 'authors': [{'first': 'S', 'middle': [], 'last': 'Zhao', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Ran', 'suffix': ''}, {'first': 'S', 'middle': ['S'], 'last': 'Musa', 'suffix': ''}], 'year': None, 'venue': 'N. Engl. J. Med', 'volume': '2020', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1056/NEJMc2001468']}}, 'BIBREF11': {'ref_id': 'b11', 'title': 'Clinical features of patients infected with 2019 novel coronavirus in Wuhan', 'authors': [{'first': 'C', 'middle': [], 'last': 'Huang', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Li', 'suffix': ''}], 'year': 2020, 'venue': 'China. Lancet', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1016/S0140-6736']}}, 'BIBREF12': {'ref_id': 'b12', 'title': 'Modelling the epidemic trend of the 2019 novel coronavirus outbreak in China', 'authors': [{'first': 'M', 'middle': [], 'last': 'Shen', 'suffix': ''}, {'first': 'Z', 'middle': [], 'last': 'Peng', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Xiao', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '2020', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.01.23.916726']}}, 'BIBREF13': {'ref_id': 'b13', 'title': 'Dynamically modeling SARS and other newly emerging respiratory illnesses: past, present, and future', 'authors': [{'first': 'C', 'middle': ['T'], 'last': 'Bauch', 'suffix': ''}, {'first': 'J', 'middle': ['O'], 'last': 'Lloyd-Smith', 'suffix': ''}, {'first': 'M', 'middle': ['P'], 'last': 'Coffee', 'suffix': ''}], 'year': 2005, 'venue': 'Epidemiology', 'volume': '', 'issn': '', 'pages': '791--801', 'other_ids': {}}, 'BIBREF14': {'ref_id': 'b14', 'title': 'Model parameters and outbreak control for SARS. Emerging Infectious Diseases', 'authors': [{'first': 'G', 'middle': [], 'last': 'Chowell', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Castillo-Chavez', 'suffix': ''}, {'first': 'P', 'middle': ['W'], 'last': 'Fenimore', 'suffix': ''}], 'year': 2004, 'venue': '', 'volume': '10', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF15': {'ref_id': 'b15', 'title': 'New SARS like virus in China triggers alarm', 'authors': [{'first': 'J', 'middle': [], 'last': 'Cohen', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Normile', 'suffix': ''}], 'year': 2020, 'venue': 'Science', 'volume': '367', 'issn': '', 'pages': '234--235', 'other_ids': {}}, 'BIBREF16': {'ref_id': 'b16', 'title': 'Incubation periods of acute respiratory viral infections: a systematic review', 'authors': [{'first': 'J', 'middle': [], 'last': 'Lessler', 'suffix': ''}, {'first': 'N', 'middle': ['G'], 'last': 'Reich', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Brookmeyer', 'suffix': ''}], 'year': 2009, 'venue': 'Lancet Infect Dis', 'volume': '9', 'issn': '5', 'pages': '70069--70075', 'other_ids': {'DOI': ['10.1016/S1473-3099(09']}}, 'BIBREF17': {'ref_id': 'b17', 'title': 'Superspreading and the effect of individual variation on disease emergence', 'authors': [{'first': 'J', 'middle': ['O'], 'last': 'Lloyd-Smith', 'suffix': ''}, {'first': 'S', 'middle': ['J'], 'last': 'Schreiber', 'suffix': ''}, {'first': 'P', 'middle': ['E'], 'last': 'Kopp', 'suffix': ''}], 'year': 2005, 'venue': 'Nature', 'volume': '438', 'issn': '', 'pages': '355--59', 'other_ids': {}}, 'BIBREF18': {'ref_id': 'b18', 'title': 'Transmission dynamics and control of severe acute respiratory syndrome', 'authors': [{'first': 'M', 'middle': [], 'last': 'Lipsitch', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Cohen', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Cooper', 'suffix': ''}], 'year': 2003, 'venue': 'Science', 'volume': '300', 'issn': '', 'pages': '1966--70', 'other_ids': {}}, 'BIBREF19': {'ref_id': 'b19', 'title': 'Real-time tentative assessment of the epidemiological characteristics of novel coronavirus infections in Wuhan,China, as at 22', 'authors': [{'first': 'P', 'middle': [], 'last': 'Wu', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Hao', 'suffix': ''}, {'first': 'E', 'middle': ['H'], 'last': 'Lau', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '25', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.2807/1560-7917.ES.2020.25.3.2000044']}}, 'BIBREF20': {'ref_id': 'b20', 'title': 'Estimation of the Transmission Risk of the 2019-nCoV and Its Implication for Public Health Interventions', 'authors': [{'first': 'Biao', 'middle': [], 'last': 'Tang', 'suffix': ''}, {'first': 'Xia', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'Qian', 'middle': [], 'last': 'Li', 'suffix': ''}], 'year': 2020, 'venue': 'J Clin Med', 'volume': '9', 'issn': '2', 'pages': '', 'other_ids': {'DOI': ['10.3390/jcm9020462']}}, 'BIBREF21': {'ref_id': 'b21', 'title': 'Transmission dynamics of the etiological agent of SARS in Hong Kong: impact of public health interventions', 'authors': [{'first': 'S', 'middle': [], 'last': 'Riley', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Fraser', 'suffix': ''}, {'first': 'C', 'middle': ['A'], 'last': 'Donnelly', 'suffix': ''}], 'year': 2003, 'venue': 'Science', 'volume': '300', 'issn': '', 'pages': '1961--66', 'other_ids': {}}, 'BIBREF22': {'ref_id': 'b22', 'title': 'The 2019-nCoV Outbreak Joint Field Epidemiology Investigation Team, Li Q. An outbreak of NCIP (2019-nCoV) infection in China-Wuhan', 'authors': [], 'year': 2019, 'venue': '', 'volume': '2', 'issn': '', 'pages': '79--80', 'other_ids': {}}, 'BIBREF23': {'ref_id': 'b23', 'title': 'The effect of public health measures on the 1918 influenza pandemic in U.S. cities', 'authors': [{'first': 'M', 'middle': ['C'], 'last': 'Bootsma', 'suffix': ''}, {'first': 'N', 'middle': ['M'], 'last': 'Ferguson', 'suffix': ''}], 'year': 2007, 'venue': 'Proc Natl Acad Sci', 'volume': '104', 'issn': '', 'pages': '7588--93', 'other_ids': {}}, 'BIBREF24': {'ref_id': 'b24', 'title': 'The continuing 2019 nCoV epidemic threat of novel coronaviruses to global health The latest 2019 novel coronavirus outbreak in Wuhan', 'authors': [{'first': 'D', 'middle': ['S'], 'last': 'Hui', 'suffix': ''}, {'first': 'E', 'middle': ['E I'], 'last': 'Azhar', 'suffix': ''}, {'first': 'T', 'middle': ['A'], 'last': 'Madani', 'suffix': ''}], 'year': 2020, 'venue': 'China. Int. J. Infect. Dis', 'volume': '91', 'issn': '', 'pages': '264--266', 'other_ids': {}}}, 'ref_entries': {'FIGREF0': {'text': '65% in the outflow population from Wuhan;2)the L (T) value referred to the spring transport data in 2019:the comprehensive traffic interdiction in Hubei Province and other severe provinces isolated strictly,after Jan.23,2020,the spring transportation passenger flow L (T) should be 0;3)underestimated the strength of the strict isolated measures and the initial and maintaining time by Chinese government:since Jan. 23,both community buildings in cities and villages in rural areas as the separated units, including the unified respective isolation of the infected,the suspiciously infected individuals and the susceptible closely contacted with the infected and the suspiciously infected individuals;4)evaluating R0:there was significant differences between in Hubei and in non-Hubei areas available to the observed data by MCMC model 5 5)overestimate the spread scope of the epidemic:from a global perspective,although all continents were scattered,but most of them are imported,and the number of the infected outside of China only accounts for 0.6% together globally by at 12:00 on Feb.17, 2020;the number of the infected in Hubei accounted for 82.4% of the total number in China, and the number of the infected in Wuhan accounted for 70.7% of the total number in Hubei;6)overestimated the spread of the epidemic:the number of the infected newly diagnosed as COVID-19 in non-Hubei areas had declined steadily for 12 consecutive days since Feb. 4, 2020.In Hubei,because clinical diagnosis had also been considered as the diagnosis criterion (still not announcing the RNA result of SARS-CoV-2) since Feb. 12,most of the suspiciously infected individuals in the inventory were diagnosed as COVID-19 on Feb.13,and the number of the infected newly diagnosed increased dramatically,but the number of the suspiciously infected individuals reduced by more than 80%.However,the number of the infected newly diagnosed in Hubei had reduced slowly since Feb.9 in case of the basis of RNA result.From the above,it can be concluded that the epidemic of COVID-19 is still a local public health emergency in Hubei Province of China currently. The above data from real world in China thew a hit on their predictions on the COVID-19 epidemic simply taking advantage of epidemiological model but ignoring quarantined measures and different R0.', 'latex': None, 'type': 'figure'}, 'FIGREF1': {'text': 'Hubei areas.However,R0 would gradually decrease with the progress of the epidemic for the sake of strengthened isolation and blocking measures. 20 The median value of R0 was 7.5 in Hubei.According to the statistical data from Hubei Provincial Center for Disease Control and Prevention (http://wjw.hubei.gov.cn/),there were 27 patients diagnosed as COVID-19 and 0 death in Wuhan on Dec.31, 2019.Since nobody approached to obtain the quantity of the latents during epidemic period early, 21 we had to deduce the quantity of the latents by the subsequent epidemiological data,the probability from the latent to the infected,which was 14.0%.Consequently,the possible quantity of the latents was 208 individuals in Hubei on Dec. 31,2019.Assuming that there was only one hidden infected individual with COVID-19 in non-Hubei area at that time,the quantity of the latents was 8 individuals in non-Hubei areas.We forecasted the epidemic of COVID-19 both in Hubei and in non-Hubei areas by modified SEIR model 22 by the software K-SEIR simulator (v1.0 ),combined with the epidemic data of the last 45 All rights reserved. No reuse allowed without permission.', 'latex': None, 'type': 'figure'}, 'FIGREF2': {'text': 'E,I,R,S m, E m represent the susceptible individuals,exposed individuals,infected individuals,convalescent individuals,susceptible individuals quarantined for being observed medically and the exposed individuals quarantined respectively.The convertion speed rate on the occasion and strength of blocking and isolation. 23,24 The strength of blocking and isolation included the intensity and scope of containment(Figure 2,Figure 3).Theoretically,the maximal reducing number of the possible infected individuals could be calculated by the formula:1 -1/R(1 -1/7.5) × 100%, 25 about 86.7%.In China,in the early stage of', 'latex': None, 'type': 'figure'}, 'FIGREF3': {'text': '45 days and modified SEIR model,which was quite different from other absolutely theoretical models,mainly as follows:1) our study was based on period was still infectious;2)evaluating R0 was on the basis of the big data of epidemic situation,R0 was 4.9-9.9(the median R0 7.5)in Hubei,but 2.5-4.8(the median R0 3.7) in non-Hubei areas;3)the occasion and intensity of the blocked and isolated measures by Chinese government were unprecedented and efficient in scale,which prevented Hubei Province in China;5)our insufficiency: whether there was a significant difference in infectivity between the latent and the infected and it required to be further confirmed in clinical practice; the blocked and isolated measures had been implemented differently in different areas due to different economic,cultural and management levels; the maintaining time of the blocked and isolated measures is still in suspense; whether the epidemic will relapse owing to the surveillance paralysis and the complicated population migration caused by returning to work in a large number of enterprises; for SEIR model by itself,the limitations on considering little on other vital factors,such as the improvement of prevention,treatment experience and All rights reserved. No reuse allowed without permission.', 'latex': None, 'type': 'figure'}, 'FIGREF4': {'text': 'The total count of the infected individuals in Hubei Province accounts for 82.4% of the total number of the infected cases in China although its total population simply accounts for 4.2% of the total population in mainland China.', 'latex': None, 'type': 'figure'}, 'FIGREF5': {'text': 'Contact rate c indicates the intensity of containment.The figure clarifies the influence on the total count of COVID-19 simulating the different c value(1.0×c,1.5×c,2.0×c),and the lower c value signifies stern surveillance and better halting transmission.', 'latex': None, 'type': 'figure'}, 'FIGREF6': {'text': 'Quarantined rate of the exposed individuals q implies the scope of containment.The figure clarifies the influence on the total count of COVID-19 simulating the different q value(1.0×q,0.9×q,0.8×q,0.5×q),and the lower q value(0.5q) indicates deficient quarantine on the infected and exposed individuals gives rise to doubling nearly in the total number of COVID-19 compared with 1.0q.', 'latex': None, 'type': 'figure'}, 'FIGREF7': {'text': 'The peak time of the curve for the cases newly diagnosed as COVID-19 should substantially present on Feb.5,2020 in non-Hubei areas.', 'latex': None, 'type': 'figure'}, 'FIGREF8': {'text': 'The peak of the curve of the cumulative confirmed cases will appear in non-Hubei areas on Mar.3,2020 and the total number of the cases diagnosed as COVID-19 is approximately 18,000 in non-Hubei areas.', 'latex': None, 'type': 'figure'}, 'FIGREF9': {'text': 'The peak time of the curve for the cases newly diagnosed as COVID-19 should substantially present on Feb. 19,2020 in Hubei.', 'latex': None, 'type': 'figure'}, 'FIGREF10': {'text': 'The peak of the curve of the cumulative confirmed cases will appear on Mar.10,2020,and the total number of the cases diagnosed as COVID-19 is about 78,000-96,000 in Hubei.', 'latex': None, 'type': 'figure'}, 'FIGREF11': {'text': 'of recovery for the asymptomatic infected individuals 0.30 MCMC:Markov chain Monte Carlo;CCDC:Chinese Center for Disease Control and Prevention All rights reserved. No reuse allowed without permission.', 'latex': None, 'type': 'figure'}, 'TABREF1': {'text': 'control measures,the advent of new medication,building new hospitals to shorten waiting time in hospital,are not considered,in addition,the setting of each coefficient is still subjective based on clinical practice. Substantially,COVID-19 is only a local outbreak in Hubei Province,China.It can be probably avoided the pandemic of global', 'latex': None, 'type': 'table'}, 'TABREF2': {'text': 'Parameters estimate for in China according to MCMC and CCDC', 'latex': None, 'type': 'table'}}, 'back_matter': []}\n",
      "{'paper_id': 'ae66f22f886d3b019934ae237c11a5fb53dbefb2', 'metadata': {'title': 'Prediction of receptorome for human-infecting virome', 'authors': [{'first': 'Zheng', 'middle': [], 'last': 'Zhang', 'suffix': '', 'affiliation': {'laboratory': 'Hunan Provincial Key Laboratory of Medical Virology', 'institution': 'Hunan University', 'location': {'postCode': '410082', 'settlement': 'Changsha', 'country': 'China'}}, 'email': ''}, {'first': 'Sifan', 'middle': [], 'last': 'Ye', 'suffix': '', 'affiliation': {'laboratory': 'Hunan Provincial Key Laboratory of Medical Virology', 'institution': 'Hunan University', 'location': {'postCode': '410082', 'settlement': 'Changsha', 'country': 'China'}}, 'email': ''}, {'first': 'Aiping', 'middle': [], 'last': 'Wu', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Chinese Academy of Medical Sciences & Peking Union Medical College', 'location': {'postCode': '100005', 'settlement': 'Beijing'}}, 'email': ''}, {'first': 'Taijiao', 'middle': [], 'last': 'Jiang', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Chinese Academy of Medical Sciences & Peking Union Medical College', 'location': {'postCode': '100005', 'settlement': 'Beijing'}}, 'email': ''}, {'first': 'Yousong', 'middle': [], 'last': 'Peng', 'suffix': '', 'affiliation': {'laboratory': 'Hunan Provincial Key Laboratory of Medical Virology', 'institution': 'Hunan University', 'location': {'postCode': '410082', 'settlement': 'Changsha', 'country': 'China'}}, 'email': ''}]}, 'abstract': [{'text': 'The virus receptor is key for viral infection of host cells. Identification of the virus receptor is still challenging at present. Our previous study has shown that human virus receptor proteins have some unique features including high level of N-glycosylation, high number of interaction partners and high expressions. Here, we built a random-forest model to identify human virus receptorome from human cell membrane proteins with accepted accuracy based on combination of unique features of human virus receptors and protein sequences. A total of 729 human cell membrane proteins were predicted to constitute the receptorome for the human-infecting virome.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'Combination of the random-forest model with protein-protein interactions between human and viruses predicted in previous studies further predicted', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}], 'body_text': [{'text': 'Receptor-binding is the first step for viral infection of host cells. Proteins are considered to be better receptors for viruses due to their higher binding affinity and specificity than carbohydrate and lipid (Baranowski et al., 2001; Casasnovas, 2013; Dimitrov, 2004; Li, 2015; Wang, 2002) . Lots of human virus receptors has been identified. The selection of proteins by viruses as receptors is not a random process.', 'cite_spans': [{'start': 210, 'end': 235, 'text': '(Baranowski et al., 2001;', 'ref_id': 'BIBREF0'}, {'start': 236, 'end': 253, 'text': 'Casasnovas, 2013;', 'ref_id': 'BIBREF1'}, {'start': 254, 'end': 269, 'text': 'Dimitrov, 2004;', 'ref_id': 'BIBREF4'}, {'start': 270, 'end': 279, 'text': 'Li, 2015;', 'ref_id': 'BIBREF11'}, {'start': 280, 'end': 291, 'text': 'Wang, 2002)', 'ref_id': 'BIBREF19'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'Previous studies have shown that proteins that were abundant in the host cell surface or had relatively low affinity for their natural ligands were preferred by viruses as receptors (Dimitrov, 2004; Wang, 2002) . Based on a collection of 119 mammalian virus receptors, our recent study further showed that human virus protein receptors have higher level of N-glycosylation, higher number of interaction partners in the human protein-protein interaction (PPI) network and high expressions in 32 common human tissues compared to other cell membrane proteins (Zhang et al., 2019) . These studies could facilitate identification of human virus receptors.', 'cite_spans': [{'start': 182, 'end': 198, 'text': '(Dimitrov, 2004;', 'ref_id': 'BIBREF4'}, {'start': 199, 'end': 210, 'text': 'Wang, 2002)', 'ref_id': 'BIBREF19'}, {'start': 556, 'end': 576, 'text': '(Zhang et al., 2019)', 'ref_id': 'BIBREF24'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'Identifying receptors for viruses is challenging. Currently, several experimental approaches have been developed in identifying virus receptors. The first approach is to select candidate membrane proteins that bind to the virus receptor-binding proteins (RBPs) by affinity purification and mass spectroscopy (Free et al., 2009; Ryu, 2016) .', 'cite_spans': [{'start': 308, 'end': 327, 'text': '(Free et al., 2009;', 'ref_id': 'BIBREF5'}, {'start': 328, 'end': 338, 'text': 'Ryu, 2016)', 'ref_id': 'BIBREF17'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'The second approach is to select candidate membrane proteins to which the monoclonal antibodies specific could block the virus entry (Minor et al., 1984; Ryu, 2016) . The third approach is to identify the virus receptor by functional cloning selection based on the cDNA expression library (Ryu, 2016) . The protein which could enable viral infection of the non-susceptible cell after transfection would be considered as receptor candidate. However, identifying virus receptor is still time-consuming and difficult at author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [{'start': 133, 'end': 153, 'text': '(Minor et al., 1984;', 'ref_id': 'BIBREF13'}, {'start': 154, 'end': 164, 'text': 'Ryu, 2016)', 'ref_id': 'BIBREF17'}, {'start': 289, 'end': 300, 'text': '(Ryu, 2016)', 'ref_id': 'BIBREF17'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https: //doi.org/10.1101 //doi.org/10. /2020 present. Besides, limitations on scalability have hampered large-scale identification of viral receptors. The development of computational methods for identifying the human virus receptors is in great need.', 'cite_spans': [{'start': 85, 'end': 102, 'text': '//doi.org/10.1101', 'ref_id': None}, {'start': 103, 'end': 122, 'text': '//doi.org/10. /2020', 'ref_id': None}], 'ref_spans': [], 'section': 'Introduction'}, {'text': \"Previous studies have developed computational models for predicting PPIs between viruses and hosts which could help identification of virus receptors (Lasso et al., 2019; Yan et al., 2019) . For example, Lasso et al. developed an in silico computational framework (P-HIPSTer) that employs structural information to predict more than 280,000 PPIs between 1,001 human-infecting viruses and humans, and made a series of new findings about human-virus interactions (Lasso et al., 2019) . Numerous PPIs between viral RBPs and human cell membrane proteins could be used to identify virus receptors. Here, we developed a computational model to predict the receptorome of the human-infecting virome based on the features of human virus receptors and protein sequences. Combination of the model with Lasso's work further predicted receptors for 693 human-infecting viruses. The study would greatly facilitate identification of human virus receptors.\", 'cite_spans': [{'start': 150, 'end': 170, 'text': '(Lasso et al., 2019;', 'ref_id': 'BIBREF10'}, {'start': 171, 'end': 188, 'text': 'Yan et al., 2019)', 'ref_id': 'BIBREF21'}, {'start': 461, 'end': 481, 'text': '(Lasso et al., 2019)', 'ref_id': 'BIBREF10'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'A total of 91 human virus protein receptors were obtained from the viralReceptor database (available at http://www.computationalbiology.cn:5000/viralReceptor) developed in our previous study (Zhang et al., 2019) . Human cell membrane proteins and human membrane proteins were obtained from the UniProtKB/Swiss-Prot database on November 24 th , 2017. The human proteins with the words of \"cell membrane\" and \"cell surface\" in the field of \"Subcellular location\" were considered to be human cell author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [{'start': 191, 'end': 211, 'text': '(Zhang et al., 2019)', 'ref_id': 'BIBREF24'}], 'ref_spans': [], 'section': 'Human viral receptors, human cell membrane proteins and human membrane proteins'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.967885 doi: bioRxiv preprint membrane proteins. The human proteins with the words of \"membrane\" in the field of \"Subcellular location\" were considered to be human membrane proteins. A total of 2,161 human cell membrane proteins and 5,187 human membrane proteins were obtained.', 'cite_spans': [], 'ref_spans': [], 'section': 'Human viral receptors, human cell membrane proteins and human membrane proteins'}, {'text': 'The N-glycosylation sites of human proteins were obtained from the UniprotKB/Swiss-Prot database. For those proteins without annotation of the N-glycosylation site in the UniprotKB/Swiss-Prot database, the N-glycosylation sites were predicted with NetNGlyc 1.0 (available at http://www.cbs.dtu.dk/services/NetNGlyc/) (Gupta et al., 2004) . The level of N-glycosylation for a protein was defined as the number of N-glycosylation sites per hundred amino acids.', 'cite_spans': [{'start': 317, 'end': 337, 'text': '(Gupta et al., 2004)', 'ref_id': 'BIBREF7'}], 'ref_spans': [], 'section': 'The protein features and protein sequence features'}, {'text': 'To calculate the node degree of human proteins in the human PPI network, firstly, the human PPIs with combined scores greater than 400 were extracted from the STRING database (version 10.5) (Szklarczyk et al., 2015) and formed the human PPI network.', 'cite_spans': [{'start': 190, 'end': 215, 'text': '(Szklarczyk et al., 2015)', 'ref_id': 'BIBREF18'}], 'ref_spans': [], 'section': 'The protein features and protein sequence features'}, {'text': 'Then, the node degree was calculated with the functions of degree in the R package igraph (version 1.2.4.2) (Csardi and Nepusz, 2006) .', 'cite_spans': [{'start': 108, 'end': 133, 'text': '(Csardi and Nepusz, 2006)', 'ref_id': 'BIBREF3'}], 'ref_spans': [], 'section': 'The protein features and protein sequence features'}, {'text': 'The expression levels of human genes in 32 common human tissues were obtained from the Expression Atlas database (Petryszak et al., 2016) on February 6 th , 2018.', 'cite_spans': [{'start': 113, 'end': 137, 'text': '(Petryszak et al., 2016)', 'ref_id': 'BIBREF15'}], 'ref_spans': [], 'section': 'The protein features and protein sequence features'}, {'text': 'Because there were strong correlations between the expressions of genes in different tissues, the principal component analysis (PCA) method was used to reduce the correlations with the function of PCA in the package scikit-learn (version 0.21.3) (Pedregosa et al., 2011) in Python (version 3.6.7). Only the first principal component was used to measure the expression level of human genes, which explained 95% of the author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [{'start': 246, 'end': 270, 'text': '(Pedregosa et al., 2011)', 'ref_id': 'BIBREF14'}], 'ref_spans': [], 'section': 'The protein features and protein sequence features'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.967885 doi: bioRxiv preprint total variance.', 'cite_spans': [], 'ref_spans': [], 'section': 'The protein features and protein sequence features'}, {'text': 'The frequencies of k-mers with one or two amino acids were calculated for each human protein with a Python script.', 'cite_spans': [], 'ref_spans': [], 'section': 'The protein features and protein sequence features'}, {'text': 'To distinguish human virus receptors from other human cell membrane proteins with machine learning models, the human virus receptors were taken as positive samples, and the human membrane proteins were taken as negative samples after excluding the human cell membrane proteins since the human cell membrane proteins possibly contain virus receptors unidentified yet.', 'cite_spans': [], 'ref_spans': [], 'section': 'Machine learning modeling'}, {'text': 'Because not all human proteins were observed in the human PPI network used or had observed expressions in the data used, only the human proteins with all three protein features, i.e., the N-glycosylation level, node degree and expression in common human tissues, were used in the modeling. Besides, the sequence redundancy in both human virus receptor proteins and human membrane proteins were removed using CD-HIT (version 4.8.1) (Fu et al., 2012) at 70% identity level. Finally, a total of 88 human virus receptors and 1,593 human membrane proteins was used in the machine learning modeling.', 'cite_spans': [{'start': 431, 'end': 448, 'text': '(Fu et al., 2012)', 'ref_id': 'BIBREF6'}], 'ref_spans': [], 'section': 'Machine learning modeling'}, {'text': 'The random-forest (RF) model is an ensemble machine learning technique using multiple decision trees and can handle data with high variance and high bias, while the risk of over-fitting can be significantly reduced by averaging multiple trees. Therefore, we chose the RF model to distinguish human virus receptors from other human cell membrane proteins. Because the number of positive samples, i.e., human viral author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Machine learning modeling'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.967885 doi: bioRxiv preprint receptors, was much smaller than that of negative samples, i.e., human membrane proteins, the function of BalanceRandomForest in the package imbalanced-learn (version 0.5.0) (Chen et al., 2004) in Python was used to deal with the imbalanced positive and negative samples with the parameter of n_estimators set to be 100.', 'cite_spans': [{'start': 316, 'end': 335, 'text': '(Chen et al., 2004)', 'ref_id': 'BIBREF2'}], 'ref_spans': [], 'section': 'Machine learning modeling'}, {'text': 'Five times of five-fold cross-validations were used to evaluate the predictive performances of the RF model with the function of StratifiedKFold in the package scikit-learn in Python. The predictive performances of the RF model were evaluated by the area under receiver operating characteristics curve (AUC), accuracy, sensitivity and specificity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Machine learning modeling'}, {'text': 'The predicted interactions between human-infecting viruses and human proteins were To evaluate the ability of the RF model in helping identify virus receptors, 25 pairs of experimentally-validated interactions between viral RBPs and receptors, and the predicted PPIs between these viral RBPs and human cell membrane proteins, were author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Validation of the RF model in ranking receptor candidates'}, {'text': \"The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.967885 doi: bioRxiv preprint extracted from the P-HIPSTer database. For each viral RBP, the predicted RBP-interacting human cell membrane proteins were ranked by either the LR provided in Lasso's work, or the predicted score provided by the RF model. Then, the ranks of real receptors were analyzed. The rank percentage of the real receptor was calculated by dividing the rank by the number of RBP-interacting human cell membrane proteins.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Validation of the RF model in ranking receptor candidates'}, {'text': 'When ranking the RBP-interacting proteins by the RF model, high sequence similarity between the RBP-interacting proteins and human proteins in the modeling (both the positive and negative samples) may over-estimate the ability of the RF model. To reduce such effect, for each pair of viral RBP and receptor, the predicted RBP-interacting human proteins were clustered with human proteins used in the modeling using CD-HIT at 50% identity level. All proteins which were clustered with RBP-interacting proteins were excluded in the modeling.', 'cite_spans': [], 'ref_spans': [], 'section': 'Validation of the RF model in ranking receptor candidates'}, {'text': 'Our previous studies have shown that human virus protein receptors have high level of N-glycosylation, high number of interaction partners in the human PPI network, and high expressions in 32 common human tissues (Zhang et al., 2019) . To identify the potential receptors of human-infecting virome, we firstly built a RF model to distinguish the human virus receptor proteins from other human membrane proteins based on these protein features. The RF model built based on individual protein feature achieved an AUC ranging from 0.50 to 0.66 in five-fold cross-validations (Table 1) .', 'cite_spans': [{'start': 213, 'end': 233, 'text': '(Zhang et al., 2019)', 'ref_id': 'BIBREF24'}], 'ref_spans': [{'start': 572, 'end': 581, 'text': '(Table 1)', 'ref_id': 'TABREF0'}], 'section': 'Development of random-forest models for predicting receptorome of human-infecting virome'}, {'text': 'Combination of all three features improved the RF model much, with both the AUC and prediction accuracy equaling to 0.75 (Table 1) . author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [{'start': 121, 'end': 130, 'text': '(Table 1)', 'ref_id': 'TABREF0'}], 'section': 'Development of random-forest models for predicting receptorome of human-infecting virome'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10. 1101 For comparison, we also developed RF models to distinguish the human virus from other human membrane proteins based on protein sequences. The frequencies of k-mers with length of one and two amino acids were used as features in the modeling.', 'cite_spans': [{'start': 98, 'end': 102, 'text': '1101', 'ref_id': None}], 'ref_spans': [], 'section': 'Development of random-forest models for predicting receptorome of human-infecting virome'}, {'text': 'The RF model based on combination of 20 amino-acid frequencies achieved an AUC 0.68. When using the frequencies of k-mers with two amino acids as features, only the top N (N=1 to 400) most important features were used in the modeling to reduce over-fitting. Figure S1 showed that the AUC of the RF models increased rapidly as increase of N from 1 to 10. Then, it increased slowly or even decreased when more ten features were used. Therefore, only top ten features were used in the modeling to balance the model performance and complexity. The RF model based on top ten features achieved an AUC of 0.72.', 'cite_spans': [], 'ref_spans': [{'start': 258, 'end': 267, 'text': 'Figure S1', 'ref_id': 'FIGREF2'}], 'section': 'Development of random-forest models for predicting receptorome of human-infecting virome'}, {'text': 'To further improve the model for predicting receptorome of human-infecting virome, we incorporated the protein features and the top ten most important k-mers of two amino acids in the modeling. The RF model achieved an AUC of 0.77. The prediction accuracy, sensitivity and specificity of the model equaled 0.80, 0.73 and 0.80, respectively ( Table 1) . The model combining both the protein features and k-mers was used for further analysis. ', 'cite_spans': [], 'ref_spans': [{'start': 342, 'end': 350, 'text': 'Table 1)', 'ref_id': 'TABREF0'}], 'section': 'Development of random-forest models for predicting receptorome of human-infecting virome'}, {'text': 'Based on the RF model, we predicted the receptorome from human cell membrane proteins. A score ranging from 0 to 1 was assigned to each human cell membrane protein. Proteins with high scores are more likely to be virus receptors. A total of 729 proteins with scores greater than 0.5 were considered to constitute the receptorome of human-infecting virome. All human cell membrane proteins and the relevant scores were provided in Table S1 .', 'cite_spans': [], 'ref_spans': [{'start': 430, 'end': 438, 'text': 'Table S1', 'ref_id': 'TABREF0'}], 'section': 'Prediction of receptorome for human-infecting virome'}, {'text': 'Then, we further investigated the prediction of virus-receptor interactions. In previous study, Lasso et al. predicted 282, 528 pairs of PPIs between human and 1,001 human-infecting viruses (Lasso et al., 2019) . Based on their study, 9,395 pairs of PPIs between 718 viral RBPs from 693 human-infecting viruses, and 314 human cell membrane proteins were extracted for further analysis (see Table S2 ). A viral RBP was predicted to interact with 1 to 65 human cell membrane proteins, with a median of 10. For each viral RBP, the RBP-interacting cell membrane proteins were ranked by the score provided by the RF model to select the most likely receptor (Table S2) .', 'cite_spans': [{'start': 96, 'end': 123, 'text': 'Lasso et al. predicted 282,', 'ref_id': None}, {'start': 124, 'end': 127, 'text': '528', 'ref_id': None}, {'start': 190, 'end': 210, 'text': '(Lasso et al., 2019)', 'ref_id': 'BIBREF10'}], 'ref_spans': [{'start': 390, 'end': 398, 'text': 'Table S2', 'ref_id': None}, {'start': 652, 'end': 662, 'text': '(Table S2)', 'ref_id': None}], 'section': 'Prediction of virus-receptor interactions for human-infecting viruses'}, {'text': 'author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Prediction of virus-receptor interactions for human-infecting viruses'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi. org/10.1101 org/10. /2020 To validate the ranking by the RF model, 25 pairs of experimentally-validated interactions between viral RBPs and receptors were extracted. For each pair of these interactions, the rank of the real receptor among the predicted RBP-interacting proteins was obtained, and the related rank percentage was calculated (Materials and Methods). Seven real receptors were ranked in top one by the RF model (Table 2) .', 'cite_spans': [{'start': 91, 'end': 102, 'text': 'org/10.1101', 'ref_id': None}, {'start': 103, 'end': 116, 'text': 'org/10. /2020', 'ref_id': None}], 'ref_spans': [{'start': 515, 'end': 524, 'text': '(Table 2)', 'ref_id': None}], 'section': 'Prediction of virus-receptor interactions for human-infecting viruses'}, {'text': 'Besides, nearly 70% (17/25) of real receptors were ranked in top three. On average, the real receptors had a rank percentage of 0.21 among all RBP-interacting human cell membrane proteins, suggesting that the real receptors would be ranked in the top 21% of all candidates by the RF model.', 'cite_spans': [], 'ref_spans': [], 'section': 'Prediction of virus-receptor interactions for human-infecting viruses'}, {'text': \"For comparison, we also ranked the RBP-interacting proteins by the LR provided in Lasso's work. In 12 pairs of viral RBP-receptor interaction with LRs, no real receptors were ranked in top one and only two real receptors were ranked in top three when ranking RBP-interacting human cell membrane proteins by the LR. On average, the median rank percentage of real receptors was 0.43 when ranking by the LR, while that was 0.15 by the RF model (Table 2) . \", 'cite_spans': [], 'ref_spans': [{'start': 441, 'end': 450, 'text': '(Table 2)', 'ref_id': None}], 'section': 'Prediction of virus-receptor interactions for human-infecting viruses'}, {'text': 'Previous studies have shown that the ACE2 protein which is the receptor of 2019-nCoV (Hoffmann et al., 2020; Zhou et al., 2020) had a low level of expression in the lung and the upper respiratory tract (Qi et al., 2020; Zhang et al., 2020) . It is author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [{'start': 85, 'end': 108, 'text': '(Hoffmann et al., 2020;', 'ref_id': None}, {'start': 109, 'end': 127, 'text': 'Zhou et al., 2020)', 'ref_id': 'BIBREF26'}, {'start': 202, 'end': 219, 'text': '(Qi et al., 2020;', 'ref_id': 'BIBREF16'}, {'start': 220, 'end': 239, 'text': 'Zhang et al., 2020)', 'ref_id': None}], 'ref_spans': [], 'section': 'Prediction of candidate alternative receptors for 2019-nCoV'}, {'text': \"The copyright holder for this preprint (which was not peer-reviewed) is the . https: //doi.org/10.1101 //doi.org/10. /2020 possible that 2019-nCoV has alternative receptors. We investigated the prediction of the alternative receptors for 2019-nCoV. Lasso's study have predicted PPIs between 28 human cell membrane proteins which were members of receptorome of human-infecting viruses, and the spike protein of two coronaviruses, including Severe Acute Respiratory Syndrome-CoV and Middle East Respiratory Syndrome-CoV. We supposed that the 2019-nCoV is very likely to use these spike-interacting proteins as alternative receptors. These spike-interacting proteins were ranked by the score provided by the RF model (Figure 1 ). The expression levels of these spike-interacting proteins in 32 common human tissues were shown in Figure 1 . Most of them had high expressions than ACE2 in the lung, such as APP, EZR, CD4 and so on. (Lasso et al., 2019) . A variable number of human cell membrane proteins were predicted to interact with viral RBPs. To further select the potential receptors for viruses, both the LR and the RF model could be used to rank the viral RBP-interacting human cell membrane proteins. The RF model was found to rank the real receptors better than the LR in a small validation dataset, suggesting that the RF model may perform better than the LR in selecting the real receptors from the predicted RBP-interacting human cell membrane proteins.\", 'cite_spans': [{'start': 85, 'end': 102, 'text': '//doi.org/10.1101', 'ref_id': None}, {'start': 103, 'end': 122, 'text': '//doi.org/10. /2020', 'ref_id': None}, {'start': 927, 'end': 947, 'text': '(Lasso et al., 2019)', 'ref_id': 'BIBREF10'}], 'ref_spans': [{'start': 714, 'end': 723, 'text': '(Figure 1', 'ref_id': 'FIGREF2'}, {'start': 826, 'end': 834, 'text': 'Figure 1', 'ref_id': 'FIGREF2'}], 'section': 'Prediction of candidate alternative receptors for 2019-nCoV'}, {'text': \"Combination of the RF model and the RBP-interacting human cell membrane proteins predicted in Lasso's work enables prediction of receptors for 693 human-infecting viruses (Table S2 ). Further efforts are needed to validate these predicted receptors.\", 'cite_spans': [], 'ref_spans': [{'start': 171, 'end': 180, 'text': '(Table S2', 'ref_id': None}], 'section': 'Prediction of candidate alternative receptors for 2019-nCoV'}, {'text': 'The 2019-nCoV is responsible for the on-going outbreak of COVID-19 (Wu et al., 2020; Zhou et al., 2020) . Previous studies have shown that the virus used ACE2 as the receptor (Hoffmann et al., 2020; Zhou et al., 2020) . However, several studies have author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [{'start': 67, 'end': 84, 'text': '(Wu et al., 2020;', 'ref_id': 'BIBREF20'}, {'start': 85, 'end': 103, 'text': 'Zhou et al., 2020)', 'ref_id': 'BIBREF26'}, {'start': 175, 'end': 198, 'text': '(Hoffmann et al., 2020;', 'ref_id': None}, {'start': 199, 'end': 217, 'text': 'Zhou et al., 2020)', 'ref_id': 'BIBREF26'}], 'ref_spans': [], 'section': 'Prediction of candidate alternative receptors for 2019-nCoV'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10. 1101 found that ACE2 had low expressions in the lung and the upper respiratory tract (Qi et al., 2020; Zhang et al., 2020) . It is possibly that the 2019-nCoV may use co-receptor or alternative receptors to facilitate its infection. A previous study investigated the candidate co-receptors of 2019-nCoV, such as ANPEP, ENPEP and DPP4 (Qi et al., 2020) . We further supposed that the 2019-nCoV possibly have alternative receptors.', 'cite_spans': [{'start': 98, 'end': 102, 'text': '1101', 'ref_id': None}, {'start': 183, 'end': 200, 'text': '(Qi et al., 2020;', 'ref_id': 'BIBREF16'}, {'start': 201, 'end': 220, 'text': 'Zhang et al., 2020)', 'ref_id': None}, {'start': 432, 'end': 449, 'text': '(Qi et al., 2020)', 'ref_id': 'BIBREF16'}], 'ref_spans': [], 'section': 'Prediction of candidate alternative receptors for 2019-nCoV'}, {'text': 'Based on the predicted PPIs between coronaviruses and humans in previous work, we predicted several candidate alternative receptors for 2019-nCoV. Most candidate alternative receptors had higher expressions than ACE2 in the lung. Nevertheless, more efforts are needed to validate these candidate alternative receptors.', 'cite_spans': [], 'ref_spans': [], 'section': 'Prediction of candidate alternative receptors for 2019-nCoV'}, {'text': 'There are some limitations to this study. Firstly, the number of human virus receptor proteins was much less than that of human membrane proteins in the modeling, which may hinder accurate modeling. The under-sampling method was used to deal with the imbalance problem. Secondly, the performance of the RF model was modest in discriminating the human virus receptor proteins from human membrane proteins.', 'cite_spans': [], 'ref_spans': [], 'section': 'Prediction of candidate alternative receptors for 2019-nCoV'}, {'text': \"More efforts are needed to improve the model. Thirdly, the RF model can not be used to identify receptors for a specific human-infecting virus. Instead, it predicts the receptorome for the human-infecting virome. Combining the RF model with the model of PPI predictions such as Lasso's work can help identify virus-receptor interactions.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Prediction of candidate alternative receptors for 2019-nCoV'}, {'text': 'In conclusion, this study for the first time built a computational model for predicting the receptorome of the human-infecting virome. It could facilitate identification of human virus receptors in either computational or experimental studies.', 'cite_spans': [], 'ref_spans': [], 'section': 'Prediction of candidate alternative receptors for 2019-nCoV'}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'Evolution of cell recognition by viruses', 'authors': [{'first': 'E', 'middle': [], 'last': 'Baranowski', 'suffix': ''}, {'first': 'C', 'middle': ['M'], 'last': 'Ruiz-Jarabo', 'suffix': ''}, {'first': 'Domingo', 'middle': [], 'last': '', 'suffix': ''}, {'first': 'E', 'middle': [], 'last': '', 'suffix': ''}], 'year': 2001, 'venue': 'Science', 'volume': '292', 'issn': '', 'pages': '1102--1105', 'other_ids': {}}, 'BIBREF1': {'ref_id': 'b1', 'title': 'Virus-receptor interactions and receptor-mediated virus entry into host cells', 'authors': [{'first': 'J', 'middle': ['M'], 'last': 'Casasnovas', 'suffix': ''}], 'year': 2013, 'venue': 'Structure and Physics of Viruses', 'volume': '', 'issn': '', 'pages': '441--466', 'other_ids': {}}, 'BIBREF2': {'ref_id': 'b2', 'title': 'Using random forest to learn imbalanced data', 'authors': [{'first': 'C', 'middle': [], 'last': 'Chen', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Liaw', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Breiman', 'suffix': ''}], 'year': 2004, 'venue': '', 'volume': '110', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF3': {'ref_id': 'b3', 'title': 'The igraph software package for complex network research. InterJournal, complex systems 1695', 'authors': [{'first': 'G', 'middle': [], 'last': 'Csardi', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Nepusz', 'suffix': ''}], 'year': 2006, 'venue': '', 'volume': '', 'issn': '', 'pages': '1--9', 'other_ids': {}}, 'BIBREF4': {'ref_id': 'b4', 'title': 'Virus entry: molecular mechanisms and biomedical applications', 'authors': [{'first': 'D', 'middle': ['S'], 'last': 'Dimitrov', 'suffix': ''}], 'year': 2004, 'venue': 'Nature Reviews Microbiology', 'volume': '2', 'issn': '', 'pages': '109--122', 'other_ids': {}}, 'BIBREF5': {'ref_id': 'b5', 'title': 'Identifying novel protein-protein interactions using co-immunoprecipitation and mass spectroscopy', 'authors': [{'first': 'R', 'middle': ['B'], 'last': 'Free', 'suffix': ''}, {'first': 'L', 'middle': ['A'], 'last': 'Hazelwood', 'suffix': ''}, {'first': 'D', 'middle': ['R'], 'last': 'Sibley', 'suffix': ''}], 'year': 2009, 'venue': 'Current protocols in neuroscience', 'volume': '46', 'issn': '', 'pages': '5--28', 'other_ids': {}}, 'BIBREF6': {'ref_id': 'b6', 'title': 'CD-HIT: accelerated for clustering the next-generation sequencing data', 'authors': [{'first': 'L', 'middle': [], 'last': 'Fu', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Niu', 'suffix': ''}, {'first': 'Z', 'middle': [], 'last': 'Zhu', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Wu', 'suffix': ''}, {'first': 'Li', 'middle': [], 'last': '', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': '', 'suffix': ''}], 'year': 2012, 'venue': 'Bioinformatics', 'volume': '28', 'issn': '', 'pages': '3150--3152', 'other_ids': {}}, 'BIBREF7': {'ref_id': 'b7', 'title': 'Prediction of N-glycosylation sites in human proteins', 'authors': [{'first': 'R', 'middle': [], 'last': 'Gupta', 'suffix': ''}, {'first': 'E', 'middle': [], 'last': 'Jung', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Brunak', 'suffix': ''}], 'year': 2004, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF9': {'ref_id': 'b9', 'title': 'The novel coronavirus 2019 (2019-nCoV) uses the SARS-coronavirus receptor ACE2 and the cellular protease TMPRSS2 for entry into target cells. bioRxiv', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF10': {'ref_id': 'b10', 'title': 'A structure-informed atlas of human-virus interactions', 'authors': [{'first': 'G', 'middle': [], 'last': 'Lasso', 'suffix': ''}, {'first': 'S', 'middle': ['V'], 'last': 'Mayer', 'suffix': ''}, {'first': 'E', 'middle': ['R'], 'last': 'Winkelmann', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Chu', 'suffix': ''}, {'first': 'O', 'middle': [], 'last': 'Elliot', 'suffix': ''}, {'first': 'J', 'middle': ['A'], 'last': 'Patino-Galindo', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Park', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Rabadan', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Honig', 'suffix': ''}, {'first': 'S', 'middle': ['D'], 'last': 'Shapira', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '178', 'issn': '', 'pages': '1526--1541', 'other_ids': {}}, 'BIBREF11': {'ref_id': 'b11', 'title': 'Journal author/funder. All rights reserved. No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the', 'authors': [{'first': 'F', 'middle': [], 'last': 'Li', 'suffix': ''}], 'year': 2015, 'venue': 'doi: bioRxiv preprint of virology', 'volume': '89', 'issn': '', 'pages': '1954--1964', 'other_ids': {'DOI': ['10.1101/2020.02.27.967885']}}, 'BIBREF12': {'ref_id': 'b12', 'title': 'ViralZone: recent updates to the virus knowledge resource', 'authors': [{'first': 'P', 'middle': [], 'last': 'Masson', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Hulo', 'suffix': ''}, {'first': 'E', 'middle': [], 'last': 'De Castro', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Bitter', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Gruenbaum', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Essioux', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Bougueleret', 'suffix': ''}, {'first': 'I', 'middle': [], 'last': 'Xenarios', 'suffix': ''}, {'first': 'Le', 'middle': [], 'last': 'Mercier', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': '', 'suffix': ''}], 'year': 2012, 'venue': 'Nucleic acids research', 'volume': '41', 'issn': '', 'pages': '579--583', 'other_ids': {}}, 'BIBREF13': {'ref_id': 'b13', 'title': 'Monoclonal antibodies which block cellular receptors of poliovirus', 'authors': [{'first': 'P', 'middle': [], 'last': 'Minor', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Pipkin', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Hockley', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Schild', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Almond', 'suffix': ''}], 'year': 1984, 'venue': 'Virus research', 'volume': '1', 'issn': '', 'pages': '203--212', 'other_ids': {}}, 'BIBREF14': {'ref_id': 'b14', 'title': 'Scikit-learn: Machine learning in Python', 'authors': [{'first': 'F', 'middle': [], 'last': 'Pedregosa', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Varoquaux', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Gramfort', 'suffix': ''}, {'first': 'V', 'middle': [], 'last': 'Michel', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Thirion', 'suffix': ''}, {'first': 'O', 'middle': [], 'last': 'Grisel', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Blondel', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Prettenhofer', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Weiss', 'suffix': ''}, {'first': 'V', 'middle': [], 'last': 'Dubourg', 'suffix': ''}], 'year': 2011, 'venue': 'Journal of machine learning research', 'volume': '12', 'issn': '', 'pages': '2825--2830', 'other_ids': {}}, 'BIBREF15': {'ref_id': 'b15', 'title': 'Expression Atlas update-an integrated database of gene and protein expression in humans, animals and plants', 'authors': [{'first': 'R', 'middle': [], 'last': 'Petryszak', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Keays', 'suffix': ''}, {'first': 'Y', 'middle': ['A'], 'last': 'Tang', 'suffix': ''}, {'first': 'N', 'middle': ['A'], 'last': 'Fonseca', 'suffix': ''}, {'first': 'E', 'middle': [], 'last': 'Barrera', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Burdett', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Füllgrabe', 'suffix': ''}, {'first': 'A', 'middle': ['M'], 'last': 'Fuentes', 'suffix': ''}, {'first': '.-P', 'middle': [], 'last': 'Jupp', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Koskinen', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': '', 'suffix': ''}], 'year': 2016, 'venue': 'Nucleic acids research', 'volume': '44', 'issn': '', 'pages': '746--752', 'other_ids': {}}, 'BIBREF16': {'ref_id': 'b16', 'title': 'Single cell RNA sequencing of 13 human tissues identify cell types and receptors of human coronaviruses', 'authors': [{'first': 'F', 'middle': [], 'last': 'Qi', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Qian', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'Z', 'middle': [], 'last': 'Zhang', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF17': {'ref_id': 'b17', 'title': 'Molecular virology of human pathogenic viruses', 'authors': [{'first': 'W.-S', 'middle': [], 'last': 'Ryu', 'suffix': ''}], 'year': 2016, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF18': {'ref_id': 'b18', 'title': 'STRING v10: protein-protein interaction networks, integrated over the tree of life', 'authors': [{'first': 'D', 'middle': [], 'last': 'Szklarczyk', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Franceschini', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Wyder', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Forslund', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Heller', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Huerta-Cepas', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Simonovic', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Roth', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Santos', 'suffix': ''}, {'first': 'K', 'middle': ['P'], 'last': 'Tsafou', 'suffix': ''}], 'year': 2015, 'venue': 'Nucleic acids research', 'volume': '43', 'issn': '', 'pages': '447--452', 'other_ids': {}}, 'BIBREF19': {'ref_id': 'b19', 'title': 'Protein recognition by cell surface receptors: physiological receptors versus virus interactions', 'authors': [{'first': 'J', 'middle': [], 'last': 'Wang', 'suffix': ''}], 'year': 2002, 'venue': 'Trends in biochemical sciences', 'volume': '27', 'issn': '', 'pages': '122--126', 'other_ids': {}}, 'BIBREF20': {'ref_id': 'b20', 'title': 'Genome Composition and Divergence of the Novel Coronavirus (2019-nCoV) Originating in China', 'authors': [{'first': 'A', 'middle': [], 'last': 'Wu', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Peng', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Huang', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Ding', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Niu', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Meng', 'suffix': ''}, {'first': 'Z', 'middle': [], 'last': 'Zhu', 'suffix': ''}, {'first': 'Z', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'Wang', 'middle': [], 'last': '', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': '', 'suffix': ''}], 'year': 2020, 'venue': 'Cell Host & Microbe', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF21': {'ref_id': 'b21', 'title': 'IILLS: predicting virus-receptor interactions based on similarity and semi-supervised learning', 'authors': [{'first': 'C', 'middle': [], 'last': 'Yan', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Duan', 'suffix': ''}, {'first': 'F.-X', 'middle': [], 'last': 'Wu', 'suffix': ''}, {'first': 'Wang', 'middle': [], 'last': '', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': '', 'suffix': ''}], 'year': 2019, 'venue': 'BMC bioinformatics', 'volume': '20', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF23': {'ref_id': 'b23', 'title': 'The digestive system is a potential route of 2019-nCov infection: a bioinformatics analysis based on single-cell transcriptomes', 'authors': [], 'year': None, 'venue': 'BioRxiv', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF24': {'ref_id': 'b24', 'title': 'Cell membrane proteins with high n-glycosylation, high expression and multiple interaction partners are preferred by mammalian viruses as receptors', 'authors': [{'first': 'Z', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'Z', 'middle': [], 'last': 'Zhu', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Chen', 'suffix': ''}, {'first': 'Z', 'middle': [], 'last': 'Cai', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Xu', 'suffix': ''}, {'first': 'Z', 'middle': [], 'last': 'Tan', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Wu', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Ge', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Guo', 'suffix': ''}, {'first': 'Z', 'middle': [], 'last': 'Tan', 'suffix': ''}], 'year': 2019, 'venue': 'Bioinformatics', 'volume': '35', 'issn': '', 'pages': '723--728', 'other_ids': {}}, 'BIBREF25': {'ref_id': 'b25', 'title': 'No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.02.27.967885']}}, 'BIBREF26': {'ref_id': 'b26', 'title': 'A pneumonia outbreak associated with a new coronavirus of probable bat origin', 'authors': [{'first': 'P', 'middle': [], 'last': 'Zhou', 'suffix': ''}, {'first': 'X.-L', 'middle': [], 'last': 'Yang', 'suffix': ''}, {'first': 'X.-G', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Hu', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'H.-R', 'middle': [], 'last': 'Si', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Zhu', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'C.-L', 'middle': [], 'last': 'Huang', 'suffix': ''}], 'year': 2020, 'venue': 'Nature', 'volume': '', 'issn': '', 'pages': '1--4', 'other_ids': {}}, 'BIBREF27': {'ref_id': 'b27', 'title': 'No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.02.27.967885']}}}, 'ref_entries': {'FIGREF0': {'text': 'obtained from the database of P-HIPSTer (available at http://phipster.org/) (Lasso et al., 2019) on November 1 st , 2019. A total of 9,395 pairs of interactions between viral RBPs and human cell membrane proteins with the likelihood ratio (LR) ≥ 100 were extracted for further analysis, which included 718 viral RBPs and 314 human cell membrane proteins. The RBPs of human-infecting viruses were compiled from three sources: the ViralZone database (Masson et al., 2012), the UniprotKB database during which viral proteins annotated with GO terms \"viral entry into host cell\" or \"virion attachment the host cell\", and the literatures related to viral RBPs. Viruses of the same viral family were supposed to use the same RBP. For example, all coronaviruses were supposed to use the spike protein as RBP.', 'latex': None, 'type': 'figure'}, 'FIGREF1': {'text': \"The ranks of real virus receptors among the RBP-interacting human cell membrane proteins by LR and RF score. The median rank percentage of real virus receptors among RBP-interacting human cell membrane proteins, and the number of real virus receptors among top one, three and five ranks were summarized at the bottom. # The number in brackets referred to those when only considering the 12 pairs of viral RBP-receptor interaction with LRs. * No LR was provided in Lasso's work since there were resolved complex structures between the RBP and the receptor. MACV, Machupo mammarenavirus; HAdV-A, Human mastadenovirus A; HAdV-C, Human mastadenovirus C; HAdV-E, Human mastadenovirus E;HAdV-D, Human author/funder. All rights reserved. No reuse allowed without permission.\", 'latex': None, 'type': 'figure'}, 'FIGREF2': {'text': 'The predicted alternative receptors (in the left side) of 2019-nCoV and their expressions in 32 human tissues (in the bottom). The predicted alternative receptors were ranked by the RF score. The expression level was measured by transcripts per million (TPM) and was colored according to the legend in the top right. The white referred to no data available. The lung was highlighted by an arrow. The ACE2 was marked by an asterisk.', 'latex': None, 'type': 'figure'}, 'TABREF0': {'text': 'The predictive performances of RF models using different sets of features. N-gly, N-glycosylation; PPI, node degree in human PPI network; Expression, expressions in 32 human tissues. Acc, accuracy; Sen, Sensitivity; Spe, Specificity.', 'latex': None, 'type': 'table'}, 'TABREF1': {'text': '', 'latex': None, 'type': 'table'}, 'TABREF3': {'text': 'author/funder. All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.967885 doi: bioRxiv preprint Discussion Identification of receptors for human-infecting viruses is critical for understanding the interactions between viruses and human. Our previous studies have shown that human virus receptors have some unique features compared to other cell membrane proteins: high level of N-glycosylation, high number of interaction partners and high expressions. This study further built a RF model for identifying human virus receptors from human cell membrane proteins with accepted accuracy. Based on the RF model, the receptorome for human-infecting virome were predicted which included a total of 729 human cell membrane proteins. They could facilitate identification of human virus receptors. Previous studies by Lasso et al. developed computational models for predicting PPIs between human-infecting viruses and humans', 'latex': None, 'type': 'table'}}, 'back_matter': [{'text': 'This work was supported by the National Key Plan for Scientific Research and author/funder. All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.967885 doi: bioRxiv preprint The authors have declared that no competing interests exist.', 'cite_spans': [], 'ref_spans': [], 'section': 'Acknowledgements'}]}\n",
      "{'paper_id': 'ae6fc64042b050df93ebb8f8045892952f18510f', 'metadata': {'title': 'Assessing the impact of a symptom-based mass screening 1 and testing intervention during a novel infectious disease', 'authors': [{'first': 'Yang', 'middle': [], 'last': 'Ge', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Brian', 'middle': ['Kenneth'], 'last': 'Mckay', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Andreas', 'middle': [], 'last': 'Handel', 'suffix': '', 'affiliation': {}, 'email': ''}]}, 'abstract': [{'text': 'A symptom-based mass screening and testing intervention (MSTI) can identify a large fraction', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}], 'body_text': [{'text': 'part of containing infectious disease outbreaks [1, 2] . A symptom-based mass screening and 23 testing intervention (MSTI), is an approach that can identify a significant fraction of infected 24', 'cite_spans': [{'start': 48, 'end': 51, 'text': '[1,', 'ref_id': 'BIBREF0'}, {'start': 52, 'end': 54, 'text': '2]', 'ref_id': None}], 'ref_spans': [], 'section': ''}, {'text': 'individuals. In the current novel coronavirus (COVID-19) outbreak, China is using an MSTI 25 strategy in an attempt to identify as many cases as possible [3] . While the concept of identify, 26 isolate, and treat is not new in terms of controlling an outbreak [4] the scales at which it is 27 currently being used is. A problem for a situation such as the COVID-19 outbreak is that many 28 novel pathogens show symptoms that are similar to common infections, e.g., general respiratory 29 symptoms. Since general respiratory symptoms are common, a MSTI can quickly overload the 30 health system [5] . Strained health facilities mean reduced care for individuals with other health 31 needs. In addition, such a situation might directly lead to a worsening of the outbreak, due to 32 increased transmission at testing sites [6, 7] . We explore how the latter risk affects the potential 33 benefit of MSTI. 34', 'cite_spans': [{'start': 154, 'end': 157, 'text': '[3]', 'ref_id': None}, {'start': 594, 'end': 597, 'text': '[5]', 'ref_id': None}, {'start': 821, 'end': 824, 'text': '[6,', 'ref_id': None}, {'start': 825, 'end': 827, 'text': '7]', 'ref_id': None}], 'ref_spans': [], 'section': ''}, {'text': 'We consider a scenario where a novel pathogen has entered a population and causes symptoms 35 that are non-specific and similar to circulating pathogens, e.g., general respiratory symptoms. 36 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.20.20025973 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'The default is to ask individuals with symptoms to stay home and follow general precautionary 37 measures [8] . An alternative option is to implement MSTI, which requires testing anyone with 38 symptoms specific for the novel pathogen at a healthcare facility. individual is infected with the novel pathogen (probability ), we assume that they are correctly 49 identified and placed under isolation and treatment, where they have reduced transmission and 50 mortality risks, namely and . Importantly, if the health system is strained, those patients 51 arriving at the testing facility without being infected with the novel pathogen (probability 1 − ) 52 may become infected by another person in the testing facility. We denote this probability of 53 becoming infected with . Since these individuals test negative, we assume they will be sent 54 home and then have the same transmission and mortality risk as individuals asked to remain at 55 home. Thus, the total transmission potential for a person undergoing screening is = + 56', 'cite_spans': [{'start': 106, 'end': 109, 'text': '[8]', 'ref_id': None}], 'ref_spans': [], 'section': ''}, {'text': '(1 − ) and the total mortality risk is = + (1 − ). 57 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.20.20025973 doi: medRxiv preprint = / = / + (1 − ) / and = / = / 0 + (1 − ) / . If that 59 expression is less than 1, it suggests that MSTI reduces transmission and mortality. If it is larger 60 than 1, the better strategy would be to not implement MSTI. 61', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'As a best-case scenario, we consider a situation where isolation and treatment following a While hard numbers for the quantities and for specific pathogens or outbreaks are 86 impossible to obtain, we used previously published data to obtain very rough estimates for and 87 ranges for the COVID-19 outbreak, as well as the 2014 Ebola outbreak and a combination of 88 previous measles outbreaks (see SM for details). Those estimates are shown as colored areas in 89 the figure. For both measles and COVID-19, our estimates suggest that depending on the specific 90 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.20.20025973 doi: medRxiv preprint setting (e.g. a specific country), MSTI might or might not be beneficial. It seems more clearly 91 beneficial for Ebola, mainly due to Ebola-related symptoms that make the disease easier to 92 discriminate from other pathogens, thus leading to a higher . 93', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'For MSTI to be beneficial, one needs to minimize and maximize [9] . An approach to reduce 94 could be the use of dedicated testing sites separate from the usual healthcare facilities [10] . 95', 'cite_spans': [{'start': 62, 'end': 65, 'text': '[9]', 'ref_id': None}, {'start': 183, 'end': 187, 'text': '[10]', 'ref_id': None}], 'ref_spans': [], 'section': ''}, {'text': 'Staff at those sites can be trained to follow protocols that reduce transmission risk. One could 96 also ask symptomatic individuals to call a phone number and schedule a test, instead of allowing 97 individuals to self-report at any time. With a scheduling system, crowding can be reduced, and 98 when individuals show up at the scheduled time, they can be processed rapidly, thus reducing 99 transmission risk [6] . 100 can be increased by having a more specific case definition (ideally without losing sensitivity). 101', 'cite_spans': [{'start': 412, 'end': 415, 'text': '[6]', 'ref_id': None}], 'ref_spans': [], 'section': ''}, {'text': 'This would reduce the overall pool of individuals with symptoms and increases among those 102 targeted for testing. More refined case definitions, screening by experts using telemedicine 103 approaches [11], or rapid home tests [12] could all be options which reduce the pool of those 104 considered at risk of being infected with the novel pathogen, thus increasing . This will also 105 reduce the total number of individuals going to testing sites, likely reducing . 106', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'Overall, our analysis suggests that MSTI can be useful if the probability of transmission at 107 testing sites is less than the probability that a symptomatic person is infected with the novel 108 pathogen. Both and will likely vary between settings and thus should be evaluated for a 109 specific setting, e.g. a specific country, if MSTI is considered as a potential control strategy. 110 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.20.20025973 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': ''}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'Factors that make an infectious disease 112 outbreak controllable', 'authors': [{'first': 'C', 'middle': [], 'last': 'Fraser', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Riley', 'suffix': ''}, {'first': 'R', 'middle': ['M'], 'last': 'Anderson', 'suffix': ''}, {'first': 'N', 'middle': ['M'], 'last': 'Ferguson', 'suffix': ''}], 'year': 2004, 'venue': 'Proceedings of the National Academy of Sciences', 'volume': '101', 'issn': '', 'pages': '6146--6151', 'other_ids': {}}, 'BIBREF1': {'ref_id': 'b1', 'title': 'Comparing nonpharmaceutical interventions for 114 containing emerging epidemics', 'authors': [{'first': 'C', 'middle': ['M'], 'last': 'Peak', 'suffix': ''}, {'first': 'L', 'middle': ['M'], 'last': 'Childs', 'suffix': ''}, {'first': 'Y', 'middle': ['H'], 'last': 'Grad', 'suffix': ''}, {'first': 'C', 'middle': ['O'], 'last': 'Buckee', 'suffix': ''}], 'year': 2017, 'venue': 'Proceedings of the National Academy of Sciences', 'volume': '115', 'issn': '', 'pages': '4023--4028', 'other_ids': {}}, 'BIBREF2': {'ref_id': 'b2', 'title': 'Transcript of the press conference on February', 'authors': [], 'year': None, 'venue': '', 'volume': '4', 'issn': '', 'pages': '2020--117', 'other_ids': {}}, 'BIBREF3': {'ref_id': 'b3', 'title': \"Health Commission of the People's Republic of China\", 'authors': [], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF4': {'ref_id': 'b4', 'title': 'Accessed 5', 'authors': [], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF5': {'ref_id': 'b5', 'title': 'When is quarantine a useful control strategy for 121 emerging infectious diseases?', 'authors': [{'first': 'T', 'middle': [], 'last': 'Day', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Park', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Madras', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Gumel', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Wu', 'suffix': ''}], 'year': 2006, 'venue': 'American Journal of Epidemiology', 'volume': '163', 'issn': '', 'pages': '479--485', 'other_ids': {}}, 'BIBREF6': {'ref_id': 'b6', 'title': 'How to maintain surveillance for novel 123 influenza a h1n1 when there are too many cases to count', 'authors': [{'first': 'M', 'middle': [], 'last': 'Lipsitch', 'suffix': ''}, {'first': 'F', 'middle': ['G'], 'last': 'Hayden', 'suffix': ''}, {'first': 'B', 'middle': ['J'], 'last': 'Cowling', 'suffix': ''}, {'first': 'G', 'middle': ['M'], 'last': 'Leung', 'suffix': ''}], 'year': 2009, 'venue': 'The Lancet', 'volume': '374', 'issn': '', 'pages': '1209--1211', 'other_ids': {}}, 'BIBREF7': {'ref_id': 'b7', 'title': 'Potential for airborne transmission of infection in the 125 waiting areas of healthcare premises: Stochastic analysis using a monte carlo model', 'authors': [{'first': 'C', 'middle': ['B'], 'last': 'Beggs', 'suffix': ''}, {'first': 'S', 'middle': ['J'], 'last': 'Shepherd', 'suffix': ''}, {'first': 'K', 'middle': ['G'], 'last': 'Kerr', 'suffix': ''}], 'year': 2010, 'venue': 'BMC 126 infectious diseases', 'volume': '10', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF8': {'ref_id': 'b8', 'title': 'Transmission characteristics of mers and sars in the 128 healthcare setting: A comparative study', 'authors': [{'first': 'G', 'middle': [], 'last': 'Chowell', 'suffix': ''}, {'first': 'F', 'middle': [], 'last': 'Abdirizak', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Lee', 'suffix': ''}], 'year': 2015, 'venue': 'BMC medicine', 'volume': '13', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF9': {'ref_id': 'b9', 'title': 'Interventions for the interruption or reduction of the 130 spread of respiratory viruses', 'authors': [{'first': 'T', 'middle': [], 'last': 'Jefferson', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Foxlee', 'suffix': ''}, {'first': 'Del', 'middle': [], 'last': 'Mar', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': '', 'suffix': ''}], 'year': 2007, 'venue': 'Cochrane database of systematic reviews', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF10': {'ref_id': 'b10', 'title': 'Evaluation of the benefits and risks of 132 introducing ebola community care centers, sierra leone', 'authors': [{'first': 'A', 'middle': ['J'], 'last': 'Kucharski', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Camacho', 'suffix': ''}, {'first': 'F', 'middle': [], 'last': 'Checchi', 'suffix': ''}], 'year': 2015, 'venue': 'Emerging Infectious Diseases', 'volume': '133', 'issn': '', 'pages': '393--399', 'other_ids': {}}, 'BIBREF11': {'ref_id': 'b11', 'title': 'Health care facility and community strategies for 135 patient care surge capacity', 'authors': [{'first': 'J', 'middle': ['L'], 'last': 'Hick', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Hanfling', 'suffix': ''}, {'first': 'J', 'middle': ['L'], 'last': 'Burstein', 'suffix': ''}], 'year': 2004, 'venue': 'Annals of emergency medicine', 'volume': '44', 'issn': '', 'pages': '253--261', 'other_ids': {}}, 'BIBREF12': {'ref_id': 'b12', 'title': 'Use of telemedicine technologies in the 137 management of infectious diseases: A review', 'authors': [{'first': 'P', 'middle': [], 'last': 'Parmar', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Mackie', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Varghese', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Cooper', 'suffix': ''}], 'year': 2015, 'venue': 'Clinical Infectious Diseases', 'volume': '60', 'issn': '', 'pages': '1084--1094', 'other_ids': {}}, 'BIBREF13': {'ref_id': 'b13', 'title': 'Home testing pilot launched in london to cut hospital visits and 139 ambulance use', 'authors': [{'first': 'E', 'middle': [], 'last': 'Mahase', 'suffix': ''}, {'first': '', 'middle': [], 'last': 'Coronavirus', 'suffix': ''}], 'year': 2020, 'venue': 'BMJ', 'volume': '368', 'issn': '', 'pages': '', 'other_ids': {}}}, 'ref_entries': {'FIGREF0': {'text': '39 At any given time during an outbreak, a proportion, , of those showing symptoms will be 40 infected with the novel pathogen. For symptomatic patients who stay home, the expected number 41 of transmissions of the novel pathogen caused by that person is the average number of 42 transmissions caused by someone who is infected with that pathogen (reproductive number, R), 43 multiplied by the probability the person is infected with the new pathogen, = . Similarly, 44 such a patient has a mortality risk due to the novel pathogen of = , where is the 45 probability of death if a person enters a healthcare facility under regular circumstances, i.e., 46 when they feel bad enough to seek medical care. 47 For individuals who go through an MSTI, there are two possible scenarios. If the symptomatic 48', 'latex': None, 'type': 'figure'}, 'FIGREF1': {'text': '62 positive test are perfect, with = = 0 (other scenarios are explored in the SM). In that case, 63 TRR and MRR are the same and given by (1 − ) / . To keep this ratio below 1, the risk of 64 MSTI associated infection needs to be < /(1 − ).65', 'latex': None, 'type': 'figure'}, 'FIGREF2': {'text': 'shows a heatmap for TRR or MRR (on a log scale) for different values of and . The 66 solid line is given by = /(1 − ), at which TRR/MRR=1 (thus their log is 0). This line 67 marks the threshold at which MSTI changes from being beneficial to not being beneficial. 68 For small , this is essentially a linear relationship, ≈ . This means that if the fraction of 69 those infected with the novel pathogen among those with symptoms is low, MSTI is only 70suitable if the risk of infection associated with MSTI is less than the fraction of syptomatic who 71 carry the novel pathogen. will be low if the overall pathogen prevalence is low (e.g. early in an 72 outbreak) and if symptoms are so broad that only a small fraction of those individuals with 73 symptoms are infected with the novel pathogen. As increases, a larger MSTI associated 74 infection risk is acceptable. Once reaches a value of ≥ 0.5, MSTI becomes useful even if = 75 1, i.e. MSTI associated infection is certain. If such a high is reached because the novel 76 pathogen is widespread, MSTI is likely logistically unfeasible. If instead a high can be reached 77 by better discrimination of symptoms, MSTI can be a useful strategy.', 'latex': None, 'type': 'figure'}}, 'back_matter': []}\n",
      "{'paper_id': 'aeb0c3a2994cf6075961a930c84245563aedea3e', 'metadata': {'title': 'Simulation-based Estimation of the Spread of COVID-19 in Iran', 'authors': [{'first': 'Navid', 'middle': [], 'last': 'Ghaffarzadegan', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Virginia Tech', 'location': {}}, 'email': 'navidg@vt.edu'}, {'first': 'Hazhir', 'middle': [], 'last': 'Rahmandad', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Sloan School of Management', 'location': {}}, 'email': 'hazhir@mit.edu'}]}, 'abstract': [{'text': 'This is a preprint version, not peer-reviewed yet, and readers should treat it accordingly. 2 Summary Background: The COVID-19 disease has turned into a global pandemic with unprecedented challenges for the global community. Understanding the state of the disease and planning for future trajectories relies heavily on data on the spread and mortality. Yet official data coming from various countries are highly unreliable: symptoms similar to common cold in majority of cases and limited screening resources and delayed testing procedures may contribute to underestimation of the burden of disease. Anecdotal and more limited data are available, but few have systematically combined those with official statistics into a coherent view of the epidemic. This study is a modeling-in-real-time of the emerging outbreak for understanding the state of the disease. Our focus is on the case of the spread of disease in Iran, as one of the epicenters of the disease in the first months of 2020.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'We develop a simple dynamic model of the epidemic to provide a more reliable picture of the state of the disease based on existing data. Building on the generic SEIR (Susceptible, Exposed, Infected, and Recovered) framework we incorporate two behavioral and logistical considerations. First we capture the endogenous changes in contact rate (average contact per person) as more death are reported. As a result the reproduction number changes endogenously in the model. Second we differentiate reported and true cases by including simple formulations for how only a fraction of cases might be diagnosed, and how that fraction changes in response to epidemic\\'s progression. In estimating the model we use both the official data as well as the discovered infected travelers and unofficial medical community estimates and triangulate these sources to build a more complete picture. Calibration is completed by forming a likelihood function for observing the actual time series data conditional on model parameters, and conducting a Markov Chain Monte Carlo simulations. The model is used to estimate current \"true\" cases of infection and death. We analyze the future trajectory of the disease under six conditions related to the seasonal effects and policy measures targeting social distancing.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'The model closely replicates the past data but also shows the true number of cases is likely far larger. We estimate about 493,000 current infected cases (90% CI: 271K-810K) as 1 A Persian summary of the article is provided as an appendix. \\u202bاست\\u202c \\u202bموجود\\u202c \\u202bانتها\\u202c \\u202bضمیمه\\u202c \\u202bدر\\u202c \\u202bفارسی\\u202c \\u202bبه\\u202c \\u202bمقاله\\u202c \\u202bخالصه\\u202c', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}], 'body_text': [{'text': 'of March 20th, 2020. Our estimate for cumulative cases of infection until that date is 916,000 (90% CI: 508K, 1.5M), and for total death is 15,485 (90% CI: 8.4K, 25.8K). These numbers are significantly (more than one order of magnitude) higher than official statistics. The trajectory of the epidemic until the end of June could take various paths depending on the impact of seasonality and policies targeting social distancing. In the most optimistic scenario for seasonal effects, depending on policy measures, 1.6 million Iranians (90% CI: 0.9M-2.6M) are likely to get infected, and death toll will reach about 58,000 cases (90% CI: 32K-97K), while in the more pessimistic scenarios, death toll may exceed 103,000 cases (90% CI: 56K-172K).', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'Our results suggest that the number of cases and deaths may be over an order of magnitude larger than official statistics in Iran. Absent extended testing capacity other countries may face a significant under-count of existing cases and thus be caught off guard about the actual toll of the epidemic.', 'cite_spans': [], 'ref_spans': [], 'section': 'Implication:'}, {'text': 'The 2019 novel Coronavirus (SARS-Cov-2), the pathogen that causes COVID-19 infection, is exposing the world to one of its largest global health challenges of recent time. The challenge is multi-faceted, from understanding a quickly evolving situation to responding with limited information and under extreme pressures on the healthcare system. From public understanding to policy choices, much depends on the data about the epidemics spread and models that integrate such data into actionable policies (Kaplan, Craft et al. 2002 , Thompson, Tebbens et al. 2008 ). Yet the official data is highly uncertain, with large variations in quality depending on the country reporting it, and regularly offering lower bounds that have unknown error bounds compared to the reality on the ground. Other sources of data, often based on smaller samples, travel screening, and anecdotal evidence may offer relevant hints, but there are not easily generalized or combined with official data. The current paper focuses on using a standard dynamic epidemiological model as a tool for incorporating various sources of data into a unified estimation of the trajectory of disease for the country of Iran.', 'cite_spans': [{'start': 502, 'end': 528, 'text': '(Kaplan, Craft et al. 2002', 'ref_id': 'BIBREF8'}, {'start': 529, 'end': 560, 'text': ', Thompson, Tebbens et al. 2008', 'ref_id': 'BIBREF13'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'Current information (as of mid-March 2020) point to the spread of COVID-19 starting from a food market in Wuhan China in mid-November 2019. The epidemic was not officially detected until the early 2020, and spread rapidly, mostly in China, for January and February of 2020. In response significant public health resources were mobilized in China, entire cities and regions were put under quarantine, and economic activity was reduced to a fraction. By mid-February the speed of contagion was slowing down in China, offering hope to many that the epidemic would be contained before becoming a global pandemic. Since then, however, rapid rise of detected cases in many other countries have dimmed the hopes for containing the contagion at origins. The situation may also be much worse than official statistics portray. For example, in late February, the United States had only a handful of endogenous cases, i.e. identified as not being imported from China. Yet genetic sequencing of the virus among one of those cases tied it back to a traveler from China who entered the country six weeks before and was thought to have not infected anybody (Fink and Baker 2020) . This implies that the epidemic might have been spreading in the U.S. for six weeks without being detected. Cases of death from COVID-19 might have also been popping up, but attributed to flu instead. The situation may be worse elsewhere. Among countries in south and east Asia with significant economic and travel ties to China, several, including India, Pakistan, Indonesia, Malaysia, Vietnam, Myanmar, Bangladesh, and Philippines have reported limited cases (as of early March) despite having a total population of over two billion. That seems unlikely when many smaller countries or those with further ties from China already show sustained epidemics, unless temperature plays an important role in infectivity, a possibility that requires further investigation (Wang, Jiang et al. 2020) . Thus official statistics are not a reliable gauge for understanding the actual extent of the epidemic.', 'cite_spans': [{'start': 1141, 'end': 1162, 'text': '(Fink and Baker 2020)', 'ref_id': 'BIBREF5'}, {'start': 1929, 'end': 1954, 'text': '(Wang, Jiang et al. 2020)', 'ref_id': 'BIBREF15'}], 'ref_spans': [], 'section': 'Background and Motivation'}, {'text': 'The situation in Iran, the country with the third largest number of official deaths from the disease by late March 2020, is instructive. The first official cases were reported on February 19th, 2020, in the city of Qom (Wikipedia 2020) . Later some anonymous sources reported on observing cases suspected of the disease much earlier, as early as January 2020 (IranInternational 2020). Yet in the absence of test kits to confirm the virus and lagging government response, those reports gained little traction. From February 19 th on the disease spread rapidly in different cities.', 'cite_spans': [{'start': 219, 'end': 235, 'text': '(Wikipedia 2020)', 'ref_id': 'BIBREF16'}], 'ref_spans': [], 'section': 'Background and Motivation'}, {'text': '. CC-BY-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Background and Motivation'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https: //doi.org/10.1101 //doi.org/10. /2020 How far will this epidemic go and what policies should be put in place to control and mitigate the risks very much depends on understanding the true magnitude of the epidemic.', 'cite_spans': [{'start': 85, 'end': 102, 'text': '//doi.org/10.1101', 'ref_id': None}, {'start': 103, 'end': 122, 'text': '//doi.org/10. /2020', 'ref_id': None}], 'ref_spans': [], 'section': 'Background and Motivation'}, {'text': 'While large and alarming, current reports for Iran, and many other countries, may well be underestimating the actual size of the pandemic. Any under-estimation is worrisome, but early in the exponential growth phase of an epidemic such errors could be extremely costly. The risk of under-estimation is partly driven by the characteristics of the COVID-19 such as a potentially large population of unrecognized patients with mild symptoms (at least 80% of the cases have symptoms not very different from common cold or flu) (Novel Coronavirus Pneumonia Emergency Response Epidemiology 2020). Some heuristics are used to correct for such biases. For example the ratio of deaths to those recovered at the beginning of March was 6.3% (Worldmeters 2020), yet most reports put the death rate at lower rates (Fauci, Lane et al. 2020, Wu and McGoogan 2020) , implying that some 70% of cases go undetected. A formal study puts that number at 86% for the early stage of the epidemic in China (Li, Pei et al. 2020 ). Yet, the country-specific biases in measurement and reporting may exceed those due to the nature of the disease. To illustrate, in the next section we provide a quick survey of various clues related to the magnitude of epidemic in Iran, which we then build on in our analysis.', 'cite_spans': [{'start': 801, 'end': 833, 'text': '(Fauci, Lane et al. 2020, Wu and', 'ref_id': None}, {'start': 834, 'end': 848, 'text': 'McGoogan 2020)', 'ref_id': 'BIBREF19'}, {'start': 982, 'end': 1002, 'text': '(Li, Pei et al. 2020', 'ref_id': 'BIBREF9'}], 'ref_spans': [], 'section': 'Background and Motivation'}, {'text': \"There have been a few clues from Iran which may inform efforts to estimate the true cases. First, is the number of cases identified among travelers arriving in other countries from Iran. Screening of passengers from high-risk countries in airports is more reliable than most countrylevel screening statistics. One study in pre-print estimated 18,300 total cases of infected individuals in Iran by February 25 (Tuite, Bogoch et al. 2020 ). The method relied on estimation of cases in the whole country based on three diagnosed cases of infection upon individuals' arrival from Iran in various international airports and the likelihood of such an incident given approximately 7,500 daily outbound passengers from Iran during those early days (Fraser, Donnelly et al. 2009 ). There is much uncertainty associated with these estimates, a wide 95% confidence interval of 3,770 -53,470, partly because the method assumes many cases of infected travelers have gone undetected. Nevertheless the range was orders of magnitude larger than official statistics at the time. Later reports on the number of infected travelers from Iran rose rapidly, to 97 cases by February 28 (RadioFarda 2020). An article in The Atlantic, offered a series of back-of-the-envelope calculations which included several simplifying assumptions, estimating 2 million accumulated cases of infected individuals by March 9 th (Wood 2020) .\", 'cite_spans': [{'start': 409, 'end': 435, 'text': '(Tuite, Bogoch et al. 2020', 'ref_id': 'BIBREF14'}, {'start': 740, 'end': 769, 'text': '(Fraser, Donnelly et al. 2009', 'ref_id': 'BIBREF6'}, {'start': 1389, 'end': 1400, 'text': '(Wood 2020)', 'ref_id': 'BIBREF17'}], 'ref_spans': [], 'section': \"Clues to the magnitude of Iran's epidemic\"}, {'text': \"One may expect the death statistics to be more reliable. But test kits for identifying have been in short supply, and post-mortem testing may not have been a priority of officials in Iran, so potentially many cases are missed. For example on March 1st a health official in Golestan, a state with 1.9 million population in Iran, reported 594 cases of Coronavirus in the state based on CT-scan outcomes (IranInternational 2020). He complained the cases were not yet counted in the official tally because the central authorities had not provided test kits to the state and are reluctant to accept other diagnosis methods. A BBC Persian report on February 28 th used interviews with an unspecified number of hospitals in Iran to put the death from the disease at 210, an order of magnitude larger than official numbers at the time (BBCPersian 2020). Another news agency quoted similar sources for a total of 416 deaths by March 1 st (IranInternational 2020) and 5000 on March 18 th . On February 24 th a member of Iran's . CC-BY-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\", 'cite_spans': [], 'ref_spans': [], 'section': \"Clues to the magnitude of Iran's epidemic\"}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10. 1101 parliament reported 50 deaths only in Qom, a city of 1.2 million (Wikipedia 2020) . Several reports of government officials contracting the disease have also been released, including reports on infection of 20 member of the Iranian parliament (a body of 270 members), as well as several deaths among officials (BBCPersian 2020). More informal observations shared on social media offer a peek into grim conditions in hospitals, with large numbers of patients suffering from COVID-19 symptoms and death rates that are far larger than official statistics suggest and include many deaths at homes.', 'cite_spans': [{'start': 98, 'end': 102, 'text': '1101', 'ref_id': None}, {'start': 168, 'end': 184, 'text': '(Wikipedia 2020)', 'ref_id': 'BIBREF16'}], 'ref_spans': [], 'section': \"Clues to the magnitude of Iran's epidemic\"}, {'text': 'We develop a dynamic simulation model of the spread of the disease in Iran to estimate the likely trajectory of the disease that is consistent with the evidence summarized above. We start with the traditional SEIR (for Susceptible, Exposed, Infectious, and Recovered stocks representing population groups) model and incorporate feedbacks regulating endogenous changes in contact rate, screening, diagnosis, and reporting in response to risk perception and other relevant factors. Thus not only reported statistics, but also the effective reproduction number, Re, are endogenously generated and can change as people respond to the epidemic. The model is very simple on other fronts: it assumes perfect mixing for the whole population of Iran and includes no disaggregation of population into different groups, nor any travel patterns across population groups. The focus being on a single country, the only link with the pandemic is when the cases were first seeded in Iran (supposedly from China). We use this model, along with various strands of data reported above, to weave together an estimate of disease trajectory so far, and offer projections for expected future trajectory. Given the rapid development of this model and various uncertainties not included in the analysis, the results should be seen as an assessment to enhance the overall picture of the epidemic, but not as reliable point estimates. With these limitations in mind, we estimate over 916,000 (90% CI: 508K, 1.5M) cumulative cases of the disease in Iran as of March 20th, with over 15,485 (90% CI: 8.4K, 25.8K) deaths. We thus estimate that only 2.1% of cases and 9.2% of deaths are officially attributed to COVID-19, with the rest going undetected. The confidence intervals around our estimations are relatively wide due to data limitations and our conservative assumptions in estimating those intervals. Nevertheless these results point to extreme gaps between official data and actual trajectory of disease, which may lead to slow response and under-appreciation of risks of the diseases in the coming months.', 'cite_spans': [], 'ref_spans': [], 'section': 'Reconciling various clues using a dynamic model'}, {'text': \"Model Structure- Figure 1 offers a simple representation of the model's structure. Model equations, and parameter values are documented in the supplementary material. The model belongs to the family of the infectious disease models knowns as SEIR (Susceptible, Exposed, Infectious, and Recovered). Main state variables of the model are shown in the Figure as boxes, with flows between them represented explicitly. The infected population is first asymptomatic and later becomes symptomatic. The inflow to the asymptomatic population mostly comes from the susceptible population. First cases of infection are injected to the exposed stock which then trigger dynamics of infection. These first cases are critical for starting the epidemic, but once several seeds of infection are planted, later importation of cases in largely inconsequential to the overall trajectory (Chinazzi, Davis et al. 2020 ). The symptomatic infected population will follow two different paths of recovery or death. In practice different symptomatic subpopulations . CC-BY-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\", 'cite_spans': [{'start': 867, 'end': 895, 'text': '(Chinazzi, Davis et al. 2020', 'ref_id': None}], 'ref_spans': [{'start': 17, 'end': 25, 'text': 'Figure 1', 'ref_id': 'FIGREF4'}], 'section': 'Methods'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10. 1101 have different risks both in severity of illness and mortality, but to avoid proliferation of parameters we use general population averages.', 'cite_spans': [{'start': 98, 'end': 102, 'text': '1101', 'ref_id': None}], 'ref_spans': [], 'section': 'Methods'}, {'text': \"The model includes the 'Infection' reinforcing loop (R1) which regulates the spread of disease from infected to susceptible, and creates the initial exponential growth in the number of cases. We also assume that asymptomatic patients (captured in the 'Exposed' stock in Figure 1 ) might be infective (Bai, Yao et al. 2020 ), though at a lower infectivity than symptomatic infected. As in the generic SEIR model the disease will continue as long as the reproduction number, the number of secondary cases from each infection, remains above one.\", 'cite_spans': [{'start': 300, 'end': 321, 'text': '(Bai, Yao et al. 2020', 'ref_id': 'BIBREF0'}], 'ref_spans': [{'start': 270, 'end': 278, 'text': 'Figure 1', 'ref_id': 'FIGREF4'}], 'section': 'Figure-1: A simplified representation of the model'}, {'text': 'To this basic epidemiological model we add two endogenous mechanisms that are key to understanding the observed trajectories in official data. First, we formulate contact rate to be endogenously changing in response to perceived risk of infection (a function of death statistics). The impact of perceived risk on contact rate captures not only the endogenous changes in social interactions, gatherings, self-isolation of suspected cases, and hygiene, but also government mandated closure of events, schools, and businesses to name a few. For example in Iran schools and sport and cultural events were closed down after the seriousness of the epidemic was recognized in late February. Moreover people started to reduce their discretionary interactions in public spaces, cut down on social and family gatherings, and followed hand washing and mask wearing recommendations more closely. We assume public risk perception depends on the number of recent reported cases of death. This balancing feedback (loop B) can bring down contact rate, potentially enough to slow down the epidemic, and captures in a simple formulation how a few other countries, namely China and South Korea, have brought down their case counts after the initial exponential growth. Second, we explicitly model the endogenous changes in screening and reporting of cases over time. Here, the increased understanding of risks leads to mobilization of screening resources that further expand the case detection and official statistics in a reinforcing process (loop R2). Nevertheless even with good screening many mild cases will go undiagnosed and not counted in official statistics. . CC-BY-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Figure-1: A simplified representation of the model'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10. 1101 Estimation-We have time series data for official reports of death, recovered, and cumulative number of infection over time. Besides the official data, we use a few unofficial data points including three observations about number of Iranian passengers diagnosed with COVID-19 upon arrival in international airports, and three unofficial estimations from BBC and IranInetrnational news sources about total cases of death from COVID 19. The model includes several biological parameters (such as the asymptomatic period of the disease or average time to recover) which we specify based on prior literature; population size and travel scope are also input using existing data. Eleven uncertain parameters remain that are estimated using the above data. Five of the parameters are used to specify how official measurement and reporting relates to \"true\" values of infection and death, three parameters estimate public reaction to the reports, and two parameters are for mortality rate among two different groups of patients. Finally, the arrival of first cases of the virus is estimated as a separate parameter.', 'cite_spans': [{'start': 98, 'end': 102, 'text': '1101', 'ref_id': None}], 'ref_spans': [], 'section': 'Figure-1: A simplified representation of the model'}, {'text': 'Our calibration method is mainly based on forming a likelihood function for observing the actual time series data conditional on model parameters. We then conduct a Markov Chain Monte Carlo (MCMC) simulation to estimate the joint posterior distribution of the model parameters subject to observed data. We define a likelihood function for change over time (net-flow) of official reports on death, recovered and infection assuming they are count events drawn from model-predicted rates (Poisson distribution). We use a similar Poisson distribution assumption for number of infected passengers and unofficial reports of death as well, since they both fit well into a count measure framework. The MCMC method searches over the feasible ranges for the uncertain parameters and accepts various combinations of parameters that are consistent with the observed data. Similar methods are used frequently in estimating dynamic models of epidemics. For example Wu and colleagues used a similar likelihood based MCMC method for estimating the basic parameters of the COVID-19 epidemic and its potential for spread beyond China (Li, Pei et al. 2020 , Wu, Leung et al. 2020 . FOur prior experience with the use of MCMC methods in nonlinear dynamic models highlights the risk that model mis-specifications may lead to confidence intervals that are too tight. We therefore downscale the likelihood function to expand the confidence intervals (details in the online appendix) to err on the side of caution in assessing uncertainties and structural nuances not explicitly modeled.', 'cite_spans': [{'start': 1116, 'end': 1136, 'text': '(Li, Pei et al. 2020', 'ref_id': 'BIBREF9'}, {'start': 1137, 'end': 1160, 'text': ', Wu, Leung et al. 2020', 'ref_id': 'BIBREF18'}], 'ref_spans': [], 'section': 'Figure-1: A simplified representation of the model'}, {'text': 'We first provide results from an out of sample prediction exercise intended to build additional confidence in our method, then discuss the detailed results based on full sample.', 'cite_spans': [], 'ref_spans': [], 'section': 'Results'}, {'text': \"Our data are limited (three data series for 30 days plus a handful of data points for other variables) and thus proper out of sample prediction test is limited in its scope. Nevertheless, we calibrate the model with the first 15 days of data using the methods discussed above, and test the model's ability in replicating the rest of the data points. Figure 2 shows the results, comparing simulation outcomes (median and 90% confidence interval) with data.\", 'cite_spans': [], 'ref_spans': [{'start': 350, 'end': 358, 'text': 'Figure 2', 'ref_id': 'FIGREF0'}], 'section': 'Out of sample prediction test'}, {'text': '. CC-BY-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Out of sample prediction test'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint The model performs better in replicating cumulative cases of infection and death than recovered. While any model that may create exponential growth trend might perform reasonably under this tests, our model is constrained by many structural features of contagion and behavioral response dynamics, and the match is despite those constraints. In the next section, we recalibrate the model with the entire data of 30 days.', 'cite_spans': [], 'ref_spans': [], 'section': 'Out of sample prediction test'}, {'text': \"We follow the procedure described in the modeling section and calibrate our model using multiple data sources. The results from MCMC is a joint distribution of parameter values for 11 estimated parameters through calibration. After about 2 million simulation runs, we received about 1.6 million acceptable points from posterior joint distribution of parameters, and used top 90% of the distribution of payoff values. We randomly selected 3% of those samples and used those to simulate the trajectories for various model variables. Figure 3 -a compares simulation results for official confirmed cases of death, recovered, and cumulative cases of infection compared with the data. RMSPE values of simulation for the last 10 days of reported recovered, death, and cumulative infected are 16%, 26%, and 5%, and RMSE's are 877, 116, and 794. Simulation runs almost perfectly correlate with data. Table A1 in the Appendix reports estimated parameters along with those adopted from the literature. Figure 3 -b shows how simulation runs replicate unofficial statistics including the number of infected travelers diagnosed before air travel patterns changed substantially, and medical community's counts of cumulative death reported by media sources.\", 'cite_spans': [], 'ref_spans': [{'start': 531, 'end': 539, 'text': 'Figure 3', 'ref_id': 'FIGREF2'}, {'start': 891, 'end': 899, 'text': 'Table A1', 'ref_id': 'TABREF3'}, {'start': 991, 'end': 999, 'text': 'Figure 3', 'ref_id': 'FIGREF2'}], 'section': 'Parameter estimation'}, {'text': 'The simulation results reported in Figure 2 . CC-BY-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [{'start': 35, 'end': 43, 'text': 'Figure 2', 'ref_id': 'FIGREF0'}], 'section': 'Status of COVID-19 in Iran as of March 20 2020'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020. 03.22.20040956 doi: medRxiv preprint infected (panel e) until March 20, 2020, starting from December 31 st , 2019 (day 0 in simulation). As discussed in detail in the modeling assumptions (above) and limitations (below), there are several structural uncertainties that can affect our estimations. Some of those uncertainties are quantified further in section 3.4. With these sources of uncertainty in mind, we find total infected cases at the time of this writing may be closer to 916,000, than the reported 19,000. Cumulative deaths by March 20th, over 15000, may also be an order of magnitude higher than official statistics and might have almost tripled in the last 10 days of the analysis. There is also much uncertainty in these numbers: our baseline estimate for cumulative infected to date may be as low as 508,000 and as high as 1.5 million. Similarly cumulative deaths is between 8400 and 25,800. . CC-BY-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 108, 'end': 144, 'text': '03.22.20040956 doi: medRxiv preprint', 'ref_id': None}], 'ref_spans': [], 'section': 'Status of COVID-19 in Iran as of March 20 2020'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https: //doi.org/10.1101 //doi.org/10. /2020 (e) (f) The wide confidence intervals are partly due to the nature of the data: not having found any data on the rate of testing in Iran it is very hard to fully know the underlying diffusion patterns based only on the formal statistics. The travel and informal estimates of death partially address these limitations, but much uncertainty remains. Our conservative assumption on downscaling the likelihood function may also have contributed to this wide interval. Nevertheless, even our lower bound are 26 times the official statistics for total infections and 6 times larger than the death statistics.', 'cite_spans': [{'start': 85, 'end': 102, 'text': '//doi.org/10.1101', 'ref_id': None}, {'start': 103, 'end': 122, 'text': '//doi.org/10. /2020', 'ref_id': None}], 'ref_spans': [], 'section': 'Status of COVID-19 in Iran as of March 20 2020'}, {'text': 'There are some glimmers of hope in our estimates as well. The drop in contact rate in response to perceived risk, which we have captured in our model, might have reduced Re significantly, to just below 1 as of late March (panel f). That reduction, if confirmed by data in the coming days, would help bring down the number of new cases. Thus, Figure 3 -e shows the number of currently infected cases may well be reaching its peak at the end of the simulation. Note that this number is smaller than cumulative cases to date (panel c).', 'cite_spans': [], 'ref_spans': [{'start': 342, 'end': 350, 'text': 'Figure 3', 'ref_id': 'FIGREF2'}], 'section': 'Status of COVID-19 in Iran as of March 20 2020'}, {'text': 'The epidemic seems to have raged with a reproduction number close to 2.72 (90% CI: 2.57-2.92 CI) for more than six weeks before the behavioral and policy interventions have slowed down the diffusion rate starting in late February. Some of this drop may also be attributed to weather, but current research is mixed on the impact of weather (Luo, Majumder et al. 2020 , Wang, Jiang et al. 2020 ) so we did not include it in our estimation (but would explore it in forward projection in 3.5). Those factors, we estimate, have already brought down the aggregate contact rate to 26% (90% CI: 22%-31%) of pre-epidemic levels. This number may just succeed in bringing down the reproduction number below 1, which is the necessary condition for containment of the epidemic. . CC-BY-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 339, 'end': 365, 'text': '(Luo, Majumder et al. 2020', 'ref_id': 'BIBREF10'}, {'start': 366, 'end': 391, 'text': ', Wang, Jiang et al. 2020', 'ref_id': 'BIBREF15'}], 'ref_spans': [], 'section': 'Status of COVID-19 in Iran as of March 20 2020'}, {'text': \"is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10. 1101 Whereas the case fatality rate in Iran, based on official statistics until March 20 th , is 18.4%, our estimated case fatality rate for the Iranian population is 3.7 (0.4)% (standard errors for estimates are in parentheses). This estimate is consistent with, and slightly higher than, the 2.3% reported by Wu and McGoogan (2020) based on tracing confirmed cases in China. If correct, the magnitude of epidemic in Iran would be much larger than that in China so far, and limits to healthcare services may explain the increased death rate. This finding is reinforced by the limited coverage of testing in Iran that is indicated in our estimation. Many cases go unconfirmed, and at its maximum, formal testing is covering only 2.5 (0.9)% of infections and 26 (17)% of deaths. These estimates are consistent with qualitative media reports from Iran on very limited availability of testing, the doctors' need for getting authorization before conducting tests, and the multi-day delays in receiving test results that render them ineffective in the clinical decision making process. We estimate that hospital sources of news organizations who have offered alternative versions of actual death statistics in Iran have had a much wider coverage of true death statistics (42 (15)% of true values). Their count offers a lower bound for true deaths, is also close to our lower confidence bound, and suggests that the true number of cumulative cases, even with a case fatality rate as high as 3.7%, would not be lower than 380,000. Nevertheless, we estimate that even those medical community reports offer coverage significantly below 100%, consistent with the less-than-perfect coverage of hospital reports by media sources, and anecdotal evidence that hospital system has been overwhelmed in most hot spots of the disease and many patients have died at home and been buried with no testing or proper association of cause of death to COVID-19.\", 'cite_spans': [{'start': 98, 'end': 102, 'text': '1101', 'ref_id': None}, {'start': 409, 'end': 431, 'text': 'Wu and McGoogan (2020)', 'ref_id': 'BIBREF19'}], 'ref_spans': [], 'section': 'Status of COVID-19 in Iran as of March 20 2020'}, {'text': \"The model used prior estimates from the literature for three important parameters: the total duration of illness (d=14 days), the asymptomatic period (τ=4 days), and the fractional infectivity of the asymptomatic period (θ=0.25). We assess our projections' sensitivity to these structural assumptions by re-estimating the model for different values of each parameter. In this analysis we find a new set of best-fitting parameters given each different assumption on disease parameters. In Table 1 results are summarized as percentage changes in key projections (cumulative infections and cumulative death until March 20 th , 2020) compared to baseline bestfitting parameters. Results suggest these structural parameters are important in identifying the true magnitude of the epidemic. Specifically, a smaller fractional infectivity during asymptomatic period (θ), a longer asymptomatic period (τ), or a shorter duration of illness all significantly increase the projections for total cases and deaths. Reductions in the estimates are less significant when parameters move in the opposite direction, though a very high level of θ would bring down the . CC-BY-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\", 'cite_spans': [], 'ref_spans': [{'start': 488, 'end': 495, 'text': 'Table 1', 'ref_id': 'TABREF3'}], 'section': 'Sensitivity to disease parameters'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https: //doi.org/10.1101 //doi.org/10. /2020 estimated cumulative cases notably. Current evidence for transmission during asymptomatic period is rather limited, and thus our baseline projections may be somewhat conservative. Nevertheless, given our conservative treatment of the confidence intervals all these baseline projections with different disease parameters remain within the 90% bounds of our baseline projections.', 'cite_spans': [{'start': 85, 'end': 102, 'text': '//doi.org/10.1101', 'ref_id': None}, {'start': 103, 'end': 122, 'text': '//doi.org/10. /2020', 'ref_id': None}], 'ref_spans': [], 'section': 'Sensitivity to disease parameters'}, {'text': 'We run the model until July 1st of 2020 under six scenarios (three alternative assumptions about the impact of seasonality times two policy measures). Specifically, our scenarios about seasonality include no effect (status quo), moderate effect (infectivity of the virus decreases linearly from April 1 st and halves by June 1 st , then stays the same for the rest of the simulation), and very strong mitigating effect (infectivity of the virus decreases from April 1 st to a quarter of its base value by June 1 st , then stays the same for the rest of the simulation). For policy, our focus is on contact rate, and we include two conditions of status quo and aggressive efforts to decrease contact rate by half what it would be otherwise. These six conditions provide intuitions for a potentially wide range of cases. Both intervention and weather impacts are assumed to include rather strong options, which is to offer a feel for the range of possibilities, but not necessarily be representative of most likely scenarios. Moreover, we also note that the wide cone of uncertainty in the baseline simulations will continue and expand in future projections.', 'cite_spans': [], 'ref_spans': [], 'section': 'Future trajectory of the critical measures'}, {'text': 'Keeping the uncertainty considerations in mind (but not graphed due to clutter), Figure 4 shows results based on best-fit parameters. If our best case scenario on the reduction of contacts and benefits of seasonality for containment hold, the number of infected cases will peak soon at approximately 494,000 (90% CI: 274K-813K), and will go down later on. This optimistic scenario will still lead to over a million infections and some 58,000 deaths (90% CI: 32K-97K) by the end of June. The less aggressive scenarios point to continued spread of the epidemic. A reduction in contagion is realized only when reproduction number remains below one as a result of natural contact reduction in response to recent deaths, government interventions that reduce contacts, and seasonality. Among these three, the first (behavioral response) is endogenous to recent death rate. When death rate comes under control (due to the combination of all those factors), our model assumes the contract rate rebounds, weakening the first channel. Economic pressures, normalization of death, and behavioral modeling after the more risk seeking (or those already recovered and thus presumably immune) could all weaken the behavioral response when the perceived risks have faded. The increased contact would then, with a delay, bring up the death rate close to levels sustaining a reproduction number around or slightly above one. This creates a strong attractor in the dynamics where in steady state contact rate is high enough to sustain the contagion but not so high to lead to the rapid infection of all the population. Therefore it is only with strong weather and/or government interventions that reduced reproduction number brings the epidemic under control. This dynamic offers a cautionary tale against declaring victory early in the fight against the epidemic. However the actual magnitude of such rebound effect is not known in this setting and our data offers no guidance on the relevant parameters. Therefore these results are only qualitatively suggestive but not quantitatively reliable.', 'cite_spans': [], 'ref_spans': [{'start': 81, 'end': 89, 'text': 'Figure 4', 'ref_id': None}], 'section': 'Future trajectory of the critical measures'}, {'text': '. CC-BY-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Future trajectory of the critical measures'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . CC-BY-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Future trajectory of the critical measures'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10. 1101 Figure 4-Simulation-based estimation of the trajectory of cumulative recovered (a), death (b), cumulative infected (c), current infected (d), Reproduction number (e) and Contact rate (f), until the end of June 2020, under 6 conditions: (Three seasonality effect conditions (S1-S3) X two policy effects (P1 and P2)). Note: Each graph shows the median value of than 3000 simulation runs. 90% CI would give a wider range of results. The reproduction number starts from zero given that the disease have slightly different starting days at different simulations, and then reaches its initial steady state value.', 'cite_spans': [{'start': 98, 'end': 102, 'text': '1101', 'ref_id': None}], 'ref_spans': [], 'section': 'Future trajectory of the critical measures'}, {'text': 'In this paper we provided a more sobering picture of the COVID-19 outbreak in Iran using a dynamic model that goes beyond official statistics. Integrating data from various sources we suspect the current official statistics are at least an order of magnitude below actual spread of the epidemic, and even in optimistic scenarios the burden of disease will be large and lasting for many months. Implementing, and sustaining, strong policies that target social distancing offers the main hope for containing the epidemic.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'The gap between official statistics and our estimates may be due to various complications in measurement as well as policy choices. The availability of testing infrastructure has been a major bottleneck for detecting cases in Iran. Citizens with suspicious symptoms have had no easy way for getting tested; most tests are conducted on hospitalized patients, require specific authorization, and are processed in a few centralized labs adding significant delays to the process. Not only mild cases are missed, but also many critical patients have been unable to access the care they need due to hospital congestion. The fact that majority of cases have mild symptoms similar to flu (Wu and McGoogan 2020) adds to the risk of under-counting even if the testing capacity was ample. In fact many mild or moderately sick patients may have preferred to stay home than risk being infected upon visiting a congested hospital, further reducing demand for testing.', 'cite_spans': [{'start': 680, 'end': 702, 'text': '(Wu and McGoogan 2020)', 'ref_id': 'BIBREF19'}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Many of the insights form this analysis are applicable to the spread of the disease in other regions. Specifically, many countries have low test rates which may be hiding the magnitude of the epidemic and increasing the risk of unpleasant surprises for policy makers and the public.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'This paper is based on modeling-in-real-time of an emerging outbreak. The problem is rapidly changing, and timely estimation is critical for the purpose of our analysis. Therefore we have made several simplifying assumptions that readers should be aware of in interpreting our results. First, consistent with prior findings (Rahmandad and Sterman 2008) we have focused on capturing behavioral feedbacks in contacts and testing and have adopted a formal estimation process, but have done so at the cost of abstracting away from much detail complexity and heterogeneity in populations and risks. Second, we have left out explicit treatment of healthcare resources and their impact on the burden of disease. Third the model is scoped only around Iran --we ignore potential effects of global spread of the disease on Iran, including the risk of reintroduction of cases in future. Forth, we assumed that the recovered population are immune during our simulation time period of six months. Fifth, we ignore mutations in the virus which may change its contagiousness and case fatality rate. Despite these limits we hope the paper offers a more accurate assessment of the risks and scope of the epidemic for Iran and beyond. We also hope other researchers build on this work using the publicly available data, models, and replication instructions provided in the online appendix.', 'cite_spans': [{'start': 324, 'end': 352, 'text': '(Rahmandad and Sterman 2008)', 'ref_id': 'BIBREF12'}], 'ref_spans': [], 'section': 'Discussion'}, {'text': '. CC-BY-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10. 1101 . CC-BY-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 98, 'end': 102, 'text': '1101', 'ref_id': None}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10. 1101 ', 'cite_spans': [{'start': 98, 'end': 102, 'text': '1101', 'ref_id': None}], 'ref_spans': [], 'section': 'Discussion'}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'Presumed Asymptomatic Carrier Transmission of COVID-19', 'authors': [{'first': 'Y', 'middle': [], 'last': 'Bai', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Yao', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Wei', 'suffix': ''}, {'first': 'F', 'middle': [], 'last': 'Tian', 'suffix': ''}, {'first': 'D', 'middle': ['Y'], 'last': 'Jin', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Chen', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Wang', 'suffix': ''}], 'year': 2020, 'venue': 'JAMA', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF1': {'ref_id': 'b1', 'title': 'Corona virus has at least claimed 210 lives in Iran', 'authors': [{'first': '', 'middle': [], 'last': 'Bbcpersian', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF2': {'ref_id': 'b2', 'title': 'Who, among political and government figures, have contracted the corona virus?', 'authors': [{'first': '', 'middle': [], 'last': 'Bbcpersian', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF4': {'ref_id': 'b4', 'title': 'Covid-19 -Navigating the Uncharted', 'authors': [{'first': 'A', 'middle': ['S'], 'last': 'Fauci', 'suffix': ''}, {'first': 'H', 'middle': ['C'], 'last': 'Lane', 'suffix': ''}, {'first': 'R', 'middle': ['R'], 'last': 'Redfield', 'suffix': ''}], 'year': 2020, 'venue': 'N Engl J Med', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF5': {'ref_id': 'b5', 'title': 'Coronavirus May Have Spread in U.S. for Weeks', 'authors': [{'first': 'S', 'middle': [], 'last': 'Fink', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Baker', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF6': {'ref_id': 'b6', 'title': 'Pandemic Potential of a Strain of Influenza A (H1N1): Early Findings', 'authors': [{'first': 'C', 'middle': [], 'last': 'Fraser', 'suffix': ''}, {'first': 'C', 'middle': ['A'], 'last': 'Donnelly', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Cauchemez', 'suffix': ''}, {'first': 'W', 'middle': ['P'], 'last': 'Hanage', 'suffix': ''}, {'first': 'M', 'middle': ['D'], 'last': 'Van Kerkhove', 'suffix': ''}, {'first': 'T', 'middle': ['D'], 'last': 'Hollingsworth', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Griffin', 'suffix': ''}, {'first': 'R', 'middle': ['F'], 'last': 'Baggaley', 'suffix': ''}, {'first': 'H', 'middle': ['E'], 'last': 'Jenkins', 'suffix': ''}, {'first': 'E', 'middle': ['J'], 'last': 'Lyons', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Jombart', 'suffix': ''}, {'first': 'W', 'middle': ['R'], 'last': 'Hinsley', 'suffix': ''}, {'first': 'N', 'middle': ['C'], 'last': 'Grassly', 'suffix': ''}, {'first': 'F', 'middle': [], 'last': 'Balloux', 'suffix': ''}, {'first': 'A', 'middle': ['C'], 'last': 'Ghani', 'suffix': ''}, {'first': 'N', 'middle': ['M'], 'last': 'Ferguson', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Rambaut', 'suffix': ''}, {'first': 'O', 'middle': ['G'], 'last': 'Pybus', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Lopez-Gatell', 'suffix': ''}, {'first': 'C', 'middle': ['M'], 'last': 'Alpuche-Aranda', 'suffix': ''}, {'first': 'I', 'middle': ['B'], 'last': 'Chapela', 'suffix': ''}, {'first': 'E', 'middle': ['P'], 'last': 'Zavala', 'suffix': ''}, {'first': 'D', 'middle': ['M E'], 'last': 'Guevara', 'suffix': ''}, {'first': 'F', 'middle': [], 'last': 'Checchi', 'suffix': ''}, {'first': 'E', 'middle': [], 'last': 'Garcia', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Hugonnet', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Roth', 'suffix': ''}, {'first': 'W', 'middle': ['R P A'], 'last': 'Coll', 'suffix': ''}], 'year': 2009, 'venue': '', 'volume': '324', 'issn': '', 'pages': '1557--1561', 'other_ids': {}}, 'BIBREF7': {'ref_id': 'b7', 'title': 'The number of deaths from Corona virus in Iran reach 416', 'authors': [{'first': '', 'middle': [], 'last': 'Iraninternational', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF8': {'ref_id': 'b8', 'title': 'Emergency response to a smallpox attack: The case for mass vaccination', 'authors': [{'first': 'E', 'middle': ['H'], 'last': 'Kaplan', 'suffix': ''}, {'first': 'D', 'middle': ['L'], 'last': 'Craft', 'suffix': ''}, {'first': 'L', 'middle': ['M'], 'last': 'Wein', 'suffix': ''}], 'year': 2002, 'venue': 'Proceedings of the National Academy of Sciences', 'volume': '99', 'issn': '16', 'pages': '10935--10940', 'other_ids': {}}, 'BIBREF9': {'ref_id': 'b9', 'title': 'Substantial undocumented infection facilitates the rapid dissemination of novel coronavirus (COVID-19)', 'authors': [{'first': 'R', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Pei', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Chen', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Song', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Yang', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Shaman', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF10': {'ref_id': 'b10', 'title': 'The epidemiological characteristics of an outbreak of 2019 novel coronavirus diseases (COVID-19) in China', 'authors': [{'first': 'W', 'middle': [], 'last': 'Luo', 'suffix': ''}, {'first': 'M', 'middle': ['S'], 'last': 'Majumder', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Liu', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Poirier', 'suffix': ''}, {'first': 'K', 'middle': ['D'], 'last': 'Mandl', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Lipsitch', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Santillana', 'suffix': ''}], 'year': 2020, 'venue': 'medRxiv. Novel Coronavirus Pneumonia Emergency Response Epidemiology', 'volume': '41', 'issn': '', 'pages': '145--151', 'other_ids': {}}, 'BIBREF11': {'ref_id': 'b11', 'title': 'WHO Chief Says 97 Cases of Coronavirus In 11 Countries Originated From Iran', 'authors': [{'first': '', 'middle': [], 'last': 'Radiofarda', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF12': {'ref_id': 'b12', 'title': 'Heterogeneity and network structure in the dynamics of diffusion: Comparing agent-based and differential equation models', 'authors': [{'first': 'H', 'middle': [], 'last': 'Rahmandad', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Sterman', 'suffix': ''}], 'year': 2008, 'venue': 'Management Science', 'volume': '54', 'issn': '5', 'pages': '998--1014', 'other_ids': {}}, 'BIBREF13': {'ref_id': 'b13', 'title': 'The risks, costs, and benefits of possible future global policies for managing polioviruses', 'authors': [{'first': 'K', 'middle': ['M'], 'last': 'Thompson', 'suffix': ''}, {'first': 'R', 'middle': ['J D'], 'last': 'Tebbens', 'suffix': ''}, {'first': 'M', 'middle': ['A'], 'last': 'Pallansch', 'suffix': ''}, {'first': 'O', 'middle': ['M'], 'last': 'Kew', 'suffix': ''}, {'first': 'R', 'middle': ['W'], 'last': 'Sutter', 'suffix': ''}, {'first': 'R', 'middle': ['B'], 'last': 'Aylward', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Watkins', 'suffix': ''}, {'first': 'H', 'middle': ['E'], 'last': 'Gary', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Alexander', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Jafari', 'suffix': ''}, {'first': 'S', 'middle': ['L'], 'last': 'Cochi', 'suffix': ''}], 'year': 2008, 'venue': 'American Journal of Public Health', 'volume': '98', 'issn': '7', 'pages': '1322--1330', 'other_ids': {}}, 'BIBREF14': {'ref_id': 'b14', 'title': 'Estimation of COVID-2019 burden and potential for international dissemination of infection from Iran', 'authors': [{'first': 'A', 'middle': ['R'], 'last': 'Tuite', 'suffix': ''}, {'first': 'I', 'middle': [], 'last': 'Bogoch', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Sherbo', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Watts', 'suffix': ''}, {'first': 'D', 'middle': ['N'], 'last': 'Fisman', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Khan', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF15': {'ref_id': 'b15', 'title': 'Temperature significant change COVID-19 Transmission in 429 cities', 'authors': [{'first': 'M', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Jiang', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Gong', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Luo', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Guo', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Zheng', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Yang', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Zeng', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF16': {'ref_id': 'b16', 'title': '2020 coronavirus outbreak in Iran', 'authors': [{'first': '', 'middle': [], 'last': 'Wikipedia', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF17': {'ref_id': 'b17', 'title': 'Iran Has Far More Coronavirus Cases Than It Is Letting On', 'authors': [{'first': 'G', 'middle': [], 'last': 'Wood', 'suffix': ''}], 'year': 2020, 'venue': 'The Atlantics. Worldmeters', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF18': {'ref_id': 'b18', 'title': 'Nowcasting and forecasting the potential domestic and international spread of the 2019-nCoV outbreak originating in Wuhan, China: a modelling study', 'authors': [{'first': 'J', 'middle': ['T'], 'last': 'Wu', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Leung', 'suffix': ''}, {'first': 'G', 'middle': ['M'], 'last': 'Leung', 'suffix': ''}], 'year': 2020, 'venue': 'Lancet', 'volume': '395', 'issn': '', 'pages': '689--697', 'other_ids': {}}, 'BIBREF19': {'ref_id': 'b19', 'title': 'Characteristics of and Important Lessons From the Coronavirus Disease 2019 (COVID-19) Outbreak in China: Summary of a Report of 72314 Cases From the Chinese Center for Disease Control and Prevention', 'authors': [{'first': 'Z', 'middle': [], 'last': 'Wu', 'suffix': ''}, {'first': 'J', 'middle': ['M'], 'last': 'Mcgoogan', 'suffix': ''}], 'year': 2020, 'venue': 'JAMA', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}}, 'ref_entries': {'FIGREF0': {'text': 'Comparison of the simulation predications of reports (mean and 90% confidence interval (CI)) with data. The model is calibrated for the first 15 days of data and testing against the last 15 days of data.Note: RMSPE: Root mean square percentage error, RMSE: Root mean square error.', 'latex': None, 'type': 'figure'}, 'FIGREF2': {'text': 'Base run simulations: Replication of data for confirmed cases (a), unofficial reports and infected passengers (b); simulation-based estimation of cumulative infected (c), death (d), current infected population (e), and reproduction number. Panels c-e include median and 90% confidence intervals. Note: Cum Inf.: official reports for cumulative cases of infected;, Inf. Passengers: Infected passengers diagnosed at international airports; U. dead: Unofficial reports about cases on death; CI: confidence interval.', 'latex': None, 'type': 'figure'}, 'FIGREF4': {'text': 'Feb 2-Mar 1-Apr 1-May 31-May 30-Jun Contact rate (', 'latex': None, 'type': 'figure'}, 'TABREF3': {'text': 'Impact of alternative disease parameters. Each row reports percentage change in cumulative infection and death compared to baseline given alternative assumptions on fractional infectivity of asymptomatic period (θ), asymptomatic period (τ), and duration of illness (d).', 'latex': None, 'type': 'table'}, 'TABREF4': {'text': '. https://doi.org/10.1101/2020.03.', 'latex': None, 'type': 'table'}}, 'back_matter': [{'text': 'We are thankful to Mohammad Akbarpour, Narges Doratoltaj, Babak Heydari, Hamed Ghoddusi, and Tse Yang Lim for their thoughtful feedback on various drafts of this paper.', 'cite_spans': [], 'ref_spans': [], 'section': 'Acknowledgments:'}]}\n",
      "{'paper_id': 'aef0734eb2538d3624d650a4ae20c3f66cace5f2', 'metadata': {'title': 'Chronic hM3Dq signaling in microglia ameliorates neuroinflammation in male mice Abbreviated title: hM3Dq signaling in microglia', 'authors': [{'first': 'William', 'middle': [], 'last': 'Binning1', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Aja', 'middle': ['E'], 'last': 'Hogan-Cann1', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Diana', 'middle': ['Yae'], 'last': 'Sakae4', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Matthew', 'middle': [], 'last': 'Maksoud1', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Valeriy', 'middle': [], 'last': 'Ostapchenko4', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Mohammed', 'middle': [], 'last': 'Al-Onaizi4', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Sara', 'middle': [], 'last': 'Matovic4', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Wei-Yang', 'middle': [], 'last': 'Lu1', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Marco', 'middle': ['A M'], 'last': 'Prado1', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Vania', 'middle': ['F'], 'last': 'Prado', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Wataru', 'middle': [], 'last': '', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Marco', 'middle': ['A M'], 'last': 'Prado', 'suffix': '', 'affiliation': {}, 'email': ''}]}, 'abstract': [], 'body_text': [{'text': 'Nothing to declare.', 'cite_spans': [], 'ref_spans': [], 'section': 'Declarations of interest:'}, {'text': 'Microglia are the resident immune cells of the central nervous system. They account for 5-10% of the total cells in the brain and are necessary for maintaining brain homeostasis (Li and Barres, 2018) . Microglia are distributed throughout the CNS and dynamically survey the environment by extending and retracting their highly ramified processes. Upon environmental alterations, microglia quickly modify their activation state and adjust their range of effects which include phagocytosis of apoptotic cells, modulation of neuronal activity, elimination of synapses, production of neuromodulatory factors that support synaptic plasticity as well as release of cytokines to modulate inflammation in the adult brain (Kettenmann et al., 2011; Li and Barres, 2018; Wake et al., 2011) .', 'cite_spans': [{'start': 178, 'end': 199, 'text': '(Li and Barres, 2018)', 'ref_id': 'BIBREF30'}, {'start': 713, 'end': 738, 'text': '(Kettenmann et al., 2011;', 'ref_id': 'BIBREF22'}, {'start': 739, 'end': 759, 'text': 'Li and Barres, 2018;', 'ref_id': 'BIBREF30'}, {'start': 760, 'end': 778, 'text': 'Wake et al., 2011)', 'ref_id': 'BIBREF60'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'It is increasingly recognized that microglia may be neurotoxic or exert beneficial roles depending on their activation status and release of mediators (Li and Barres, 2018) . Among the multiple signals that operate on microglia to modulate their functional state and coordinate their diverse repertoire of functions are neurotransmitters. Microglia express numerous types of neurotransmitter receptors such as those for glutamate, GABA, serotonin, adrenaline and acetylcholine (Kettenmann et al., 2011; Li and Barres, 2018) .', 'cite_spans': [{'start': 151, 'end': 172, 'text': '(Li and Barres, 2018)', 'ref_id': 'BIBREF30'}, {'start': 477, 'end': 502, 'text': '(Kettenmann et al., 2011;', 'ref_id': 'BIBREF22'}, {'start': 503, 'end': 523, 'text': 'Li and Barres, 2018)', 'ref_id': 'BIBREF30'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'Cholinergic signaling is involved in the modulation of central and peripheral inflammation (Lehner et al., 2019; Pavlov and Tracey, 2015) , however, we still have limited information about how microglia respond to activation of cholinergic receptors in vivo. Microglia have been reported to express the α3, α5, α6, α7 and β4 nicotinic channel subunits (Liu et al., 2009; Rock et al., 2008; Shytle et al., 2004; Suzuki et al., 2006) . Expression of muscarinic receptors on cultured microglia has also been demonstrated (Pannell et al., 2016; Zhang et al., 1998) . In cultured microglia, acetylcholine seems to exert mainly an anti-inflammatory effect on microglia. For instance, acetylcholine and nicotine inhibited TNF-α release in response to LPS stimulation (Shytle et al., 2004) and nicotine reduced free radical production in response to β-amyloid (Moon et al., 2008) .', 'cite_spans': [{'start': 91, 'end': 112, 'text': '(Lehner et al., 2019;', 'ref_id': 'BIBREF28'}, {'start': 113, 'end': 137, 'text': 'Pavlov and Tracey, 2015)', 'ref_id': 'BIBREF48'}, {'start': 352, 'end': 370, 'text': '(Liu et al., 2009;', 'ref_id': 'BIBREF31'}, {'start': 371, 'end': 389, 'text': 'Rock et al., 2008;', 'ref_id': 'BIBREF53'}, {'start': 390, 'end': 410, 'text': 'Shytle et al., 2004;', 'ref_id': 'BIBREF54'}, {'start': 411, 'end': 431, 'text': 'Suzuki et al., 2006)', 'ref_id': 'BIBREF55'}, {'start': 518, 'end': 540, 'text': '(Pannell et al., 2016;', 'ref_id': 'BIBREF46'}, {'start': 541, 'end': 560, 'text': 'Zhang et al., 1998)', 'ref_id': 'BIBREF63'}, {'start': 760, 'end': 781, 'text': '(Shytle et al., 2004)', 'ref_id': 'BIBREF54'}, {'start': 852, 'end': 871, 'text': '(Moon et al., 2008)', 'ref_id': 'BIBREF38'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'How muscarinic receptors influence microglia function is poorly understood. On BV-2 microglia cells, acute activation of Gq-associated Designer Receptors Exclusively Activated by Designer Drugs (DREADDs) induced the production of pro-inflammatory mediators while activation of Giassociated DREADDs inhibited lipopolysaccharide-and CCL2-induced inflammatory signaling (Grace et al., 2018) . The influence of muscarinic receptors might be more important for the modulation of \"activated microglia\" than \"resting microglia\" as their expression is largely increased in microglia isolated from stroke and Alzheimer\\'s disease mouse models when compared to microglia isolated from healthy mice (Pannell et al., 2016) . To note, most of these studies were done in vitroand microglia have been shown to respond differently when analyzed in vitro or in vivo (Aguzzi et al., 2013; Bohlen et al., 2017; Yamasaki, 2014) .', 'cite_spans': [{'start': 367, 'end': 387, 'text': '(Grace et al., 2018)', 'ref_id': 'BIBREF17'}, {'start': 687, 'end': 709, 'text': '(Pannell et al., 2016)', 'ref_id': 'BIBREF46'}, {'start': 848, 'end': 869, 'text': '(Aguzzi et al., 2013;', 'ref_id': 'BIBREF0'}, {'start': 870, 'end': 890, 'text': 'Bohlen et al., 2017;', 'ref_id': 'BIBREF6'}, {'start': 891, 'end': 906, 'text': 'Yamasaki, 2014)', 'ref_id': 'BIBREF62'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'Here, we took advantage of DREADD technology to investigate the effects of M3 muscarinic Gq-mediated signaling on microglial function in vivo. We used the human modified muscarinic type 3 DREADD (hM3Dq), which is not activated by acetylcholine (Nichols and Roth, 2009 ) but can be activated by clozapine N-oxide (CNO) and its metabolites (Dong et al., 2010; Gomez et al., 2017) , to explore the function of M3 muscarinic Gq-signaling in vivo in these cells.', 'cite_spans': [{'start': 244, 'end': 267, 'text': '(Nichols and Roth, 2009', 'ref_id': 'BIBREF44'}, {'start': 338, 'end': 357, 'text': '(Dong et al., 2010;', 'ref_id': 'BIBREF12'}, {'start': 358, 'end': 377, 'text': 'Gomez et al., 2017)', 'ref_id': 'BIBREF15'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'Using Cre/lox technology (Parkhurst et al., 2013) we generated a mouse line in which hM3Dq (Zhu et al., 2016) is selectively expressed in microglia. We found that acute activation of the hM3Dq pathway in microglia increased phagocytosis, increased the production of pro-inflammatory cytokines in the brain, but did not cause overt behavioral changes. In contrast, chronic activation of this signaling pathway attenuated the increase in brain cytokine production and sickness behavior triggered by low dose of lipopolysaccharide (LPS) injected peripherally. Our experiments provide a proof of principle that chronic M3 muscarinic Gq-linked signaling, and possibly other Gq-coupled GPCRs, can trigger immunological memory that helps to decrease inflammatory responses in the brain.', 'cite_spans': [{'start': 25, 'end': 49, 'text': '(Parkhurst et al., 2013)', 'ref_id': 'BIBREF47'}, {'start': 91, 'end': 109, 'text': '(Zhu et al., 2016)', 'ref_id': 'BIBREF64'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'CX3CR1CreER mice (Jackson Laboratory, B6.129P2(Cg)-Cx3cr1tm2.1(cre/ERT2)Litt/WganJ) expressing tamoxifen-activated Cre-recombinase in CX3CR1-positive cells (Parkhurst et al., 2013) , were obtained from Jackson Laboratories. R26-hM3Dq/mCitrine mice (Zhu et al., 2016) (B6N;129-Tg(CAG-CHRM3*,-mCitrine)1Ute/J) expressing a CAG promotor-driven HA-tagged hM3Dq with an upstream floxed STOP cassette were a generous gift of Ute Hochgeschwender (Central Michigan University) and Bryan Roth (University of North Carolina). CX3CR1CreER mice were crossed with R26-hM3Dq/mCitrine mice to create CX3CR1CreER(+/-)-hM3Dq(+/-) mice, which upon treatment with tamoxifen express hM3Dq in CX3CR1-positive cells (hM3Dq-positive mice).', 'cite_spans': [{'start': 156, 'end': 180, 'text': '(Parkhurst et al., 2013)', 'ref_id': 'BIBREF47'}, {'start': 248, 'end': 266, 'text': '(Zhu et al., 2016)', 'ref_id': 'BIBREF64'}], 'ref_spans': [], 'section': 'Animals'}, {'text': 'CX3CR1CreER(+/-)-hM3Dq(+/-) mice treated with vehicle (hM3Dq-negative mice) were used as controls. CX3CR1CreER(+/-) or hM3Dq(+/-) littermate controls were used as additional hM3Dqnegative controls. To avoid expression of h3MDq in circulating myeloid-derived CX3CR1 positive cells, experiments were performed no sooner than 1 month following tamoxifen treatment (P90) to allow turnover of peripheral myeloid-derived cells, as previously described (Parkhurst et al., 2013) .', 'cite_spans': [{'start': 446, 'end': 470, 'text': '(Parkhurst et al., 2013)', 'ref_id': 'BIBREF47'}], 'ref_spans': [], 'section': 'Animals'}, {'text': 'All procedures were conducted in accordance with guidelines of the Canadian Council of Animal Care and approved by the University of Western Ontario Animal Use Subcommittee (protocol 2016-103 and 2016-104) . The study also complied with the ARRIVE guidelines (Animal Research: Reporting In Vivo Experiments). Animals were housed maximum of four by cage (28x18 author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [{'start': 173, 'end': 205, 'text': '(protocol 2016-103 and 2016-104)', 'ref_id': None}], 'ref_spans': [], 'section': 'Animals'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.27.921809 doi: bioRxiv preprint cm) and had access to rodent chow (Envigo 7913) and water ad libitum. They were maintained on a 12h/12h light-dark cycle in non-SPF conditions.', 'cite_spans': [], 'ref_spans': [], 'section': 'Animals'}, {'text': 'Experimental groups consisted of 4-6 month old CNO-treated CX3CR1CreER(+/-)-hM3Dq(+/-) mice.', 'cite_spans': [], 'ref_spans': [], 'section': 'Experimental Design'}, {'text': 'Vehicle-treated CX3CR1CreER(+/-)-hM3Dq(+/-) animals were used as controls. CX3CR1CreER(+/-) littermate males were used as additional controls. Mice were randomly assigned to a treatment group and investigators were blinded during experimental procedures.', 'cite_spans': [], 'ref_spans': [], 'section': 'Experimental Design'}, {'text': 'Tamoxifen (Sigma, cat#T56648) was dissolved at 100 mg/ml in 10% ethanol/90% corn oil. To induce DREADD expression, tamoxifen (75 mg/kg; i.p.) was injected once per day for 5 consecutive days. For cell culture assays, the tamoxifen active metabolite 4-hydroxytamoxifen (4-OHT, Sigma, cat#H6278) was used instead of tamoxifen. 4-OHT was dissolved in 100% ethanol and used at concentration of 5 µM. Clozapine N-Oxide (CNO, Tocris, cat#4936) was dissolved in 0.5% dimethylsulfoxide (DMSO). Calcium dye Rhod-2 AM (Abcam, cat#ab142780) was used at a final concentration of 5 µM in 1% DMSO. Store-operated Ca2+ channel (SOCC) inhibitor SKF-96365 (Santa Cruz, cat#130495-35-1) was diluted in water and used at 20 µM. E. coli 0111:B4 lipopolysaccharide (LPS) (Sigma-Aldrich, cat#L3012, Ec#297-473-0, lot#WI150506) was dissolved in saline and used at a dose of 0.1 mg/kg, i.p. This dose of LPS was chosen because it is the minimum effective dose to elicit pro-inflammatory cytokine response and sickness behavior in mice (Berg et al., 2004) . author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [{'start': 1011, 'end': 1030, 'text': '(Berg et al., 2004)', 'ref_id': 'BIBREF3'}], 'ref_spans': [], 'section': 'Drugs'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.27.921809 doi: bioRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'Drugs'}, {'text': 'Primary antibodies used include anti-HA (1:1000, BioLegend, cat#901501, RRID:AB_2563417), anti-Iba1 (1:1000, Abcam, cat#ab5076, RRID:AB_2224403), anti-GFAP (1:1000, Abcam, cat#ab7260, RRID:AB_296804), and anti-NeuN (1:1000, Cell Signaling, cat#D3S3I, RRID:AB_2630395). Secondary antibodies used include Alexa Fluor 555 goat anti-mouse (1:1000, ThermoFisher, cat#A32727, RRID: AB_2535783) and Alexa Fluor 633 goat anti-rabbit (1:1000, ThermoFisher, cat#A-21070, RRID:AB_10372583).', 'cite_spans': [], 'ref_spans': [], 'section': 'Antibodies'}, {'text': 'CX3CR1CreER-hM3Dq mice underwent intracardiac perfusion with 1X PBS for 5 min and 4% PFA in PBS for 5 min. Brains were extracted and postfixed in 4% PFA at 4°C for 24 h. Brains were then cut into 40-µm thick sections using a vibratome (VT1200S, Leica). Staining for Iba1, HA, NeuN or GFAP was performed as described previously (Kolisnyk et al., 2016) using Alexa Fluor 555/633tagged secondary antibodies and Hoechst. Sections were mounted on glass slides using Fluorescent-G mounting medium (Electron Microscopy Sciences, cat#17984-25) and imaged using an SP8 confocal microscope (Leica) equipped with a 20x/0.75 dry or a 40x/1.3 oil objective. At least 4 sections were imaged for each animal (N=5) for each condition, with at least 6 fields imaged for each section. Images were analyzed using ImageJ Cell Counter plugin. 075-CL) containing 10% heat-inactivated fetal bovine serum (Gibco, cat#10438026) and 1× penicillin/streptomycin (Gibco, cat#15140-122) . Cells were plated on 75-cm2 tissue culture flasks coated with poly-D-lysine (10µg/mL, Sigma Aldrich, cat#P9155) and incubated at 37 °C in 5% CO2 atmosphere. The culture medium was changed after 3 days and then every 5 days until confluency (12-18 days in vitro).', 'cite_spans': [{'start': 327, 'end': 350, 'text': '(Kolisnyk et al., 2016)', 'ref_id': 'BIBREF25'}, {'start': 934, 'end': 956, 'text': '(Gibco, cat#15140-122)', 'ref_id': None}], 'ref_spans': [], 'section': 'Immunohistochemistry'}, {'text': 'To obtain pure microglia cultures, flasks of mixed glial cultures were shaken at 200 rpm for 2 h at 37 °C. Floating microglia were collected by centrifugation (380g for 10 min) and seeded at ~1×105 cells/ml on poly-D-lysine-coated dishes containing 50% old/50% new media. On the following day, culture medium was supplemented with 5 µM 4-OHT and incubated for 48 h for optimal induction of hM3Dq expression. Approximately 5×105 cells were obtained per brain used.', 'cite_spans': [], 'ref_spans': [], 'section': 'Immunohistochemistry'}, {'text': 'Immunocytochemistry was done on microglia plated on poly-D-lysine-coated glass coverslips at a density of ~1×104 cells per well. Cells were obtained from three independent cultures of three different litters to confirm HA expression. Cells were fixed with 4% paraformaldehyde (PFA) in PBS. After permeabilization with 0.5% Triton X-100 and blocking with 5% bovine serum albumin (BSA)/0.1% Triton X-100, cultures were incubated with primary antibodies (anti-HA, anti-Iba1, and anti-GFAP) overnight at 4 °C. Cells were then washed with PBS and incubated for 1 hour with secondary antibodies (Alexa Fluor 555 goat anti-mouse and Alexa Fluor 633 goat anti-rabbit, 1:500) and Hoechst (1:2000) . Coverslips were then mounted on glass slides using Immumount (Thermo Fisher) medium and imaged using an FV1000 confocal microscope (Olympus) equipped with a 20x/0.75, 30x/1.05 or 60x/1.35 objective. Images were analyzed using ImageJ (NIH, Bethesda, MD) Cell Counter plugin. author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [{'start': 671, 'end': 687, 'text': 'Hoechst (1:2000)', 'ref_id': None}], 'ref_spans': [], 'section': 'Immunocytochemistry'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.27.921809 doi: bioRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'Immunocytochemistry'}, {'text': 'For Ca2+ imaging, microglia were plated on poly-D-lysine-coated glass-bottom dishes (MatTek).', 'cite_spans': [], 'ref_spans': [], 'section': 'Calcium imaging'}, {'text': 'Ca2+ imaging was performed as described previously (Beraldo et al., 2016) with some modifications. Cultures were washed with Krebs-Ringer HEPES (KRH) buffer (129 mM NaCl, 5 mM NaHCO3, 4.8 mM KCl, 1.2 mM KH2PO4, 1.2 mM MgCl2, 2.8 mM glucose, 10 mM HEPES) supplied with 1 mM CaCl2, incubated for 45 min with 5 µM of red spectrum fluorescent dye Rhod-2 AM at 37 °C, washed again 3 times with calcium-free KRH and incubated in KRH with or without calcium for 10 min at 37 °C. Intracellular Ca2+ was recorded with a FV1000 confocal microscope equipped with a 30x/1.05 objective. The imaging field was chosen to contain cells expressing hM3Dq as shown by mCitrine fluorescence (ex488/em505-550). Time kinetics were done with a 2.5-second step. After 100 s of background recording, 3 µM CNO, 1 mM ATP and 2 µM ionomycin or 10 µM thapsigargin (for KRH with or without calcium, respectively) were added consecutively to the dish. Rhod-2 fluorescence was registered using 546 nm excitation laser and 560-660 nm emission filter. ROIs were assigned and quantified for each cell in the imaging field using ImageJ (NIH, Bethesda, MD). Integrated fluorescence was normalized to the average signal during background recording. At least three independent cultures were obtained from three different litters and at least 50 cells were imaged for each condition.', 'cite_spans': [{'start': 51, 'end': 73, 'text': '(Beraldo et al., 2016)', 'ref_id': 'BIBREF1'}], 'ref_spans': [], 'section': 'Calcium imaging'}, {'text': 'Control cells and 4-OHT-treated cells on poly-D-lysine-coated 35-mm petri dishes were kept in bath solution containing (in mM): 130 NaCl, 5 KCl, 3 MgCl2, 2 CaCl2, 10 HEPES and 5 glucose.', 'cite_spans': [], 'ref_spans': [], 'section': 'Voltage-clamp recordings'}, {'text': 'For patch-clamp recordings, KCl in the bath solution was replaced with NaCl (containing 135 mM author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Voltage-clamp recordings'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.27.921809 doi: bioRxiv preprint NaCl). Patch electrodes were pulled from thin-wall glass tubes (World Precision Instruments, Inc.; Sarasota, FL) using a PP-830 pipette puller (Narishige, Japan). The electrode was filled with intracellular solution containing (in mM): 140 Cs-gluconate, 8 NaCl, 1 MgCl2, 10 Cs-BAPTA, and 10 HEPES following a previously described protocol (Zweifach and Lewis, 1996) with modifications. The pH of all solutions was adjusted to 7.4 using NaOH and/or KOH, and the osmolarity was adjusted to ~300 mOsm. Under voltage-clamp conditions, whole-cell patch recordings were performed using a Multiclamp 700B amplifier (Molecular Devices) by a person blinded to the treatment compounds. After measuring membrane capacitance of the test cell, voltage ramps (from -100mV to +50mV over 150 ms) were repeatedly applied to the cell every 10 s, revealing the current-voltage relationship. When stable recordings of baseline transmembrane current were achieved, the test cells were perfused with 3 µM CNO for 6 min, followed by application of 20 µM SKF-96365. The recorded electrical signal was digitized at 10kHz, low-pass at filtered (1 kHz), acquired on-line using the program pClamp (Molecular Devices), and saved in a computer for off-line analysis. The current density (pA/pF) at different voltages for each test cell was averaged and plotted for report. At least 15 cells from three different litters were analyzed for each testing condition.', 'cite_spans': [{'start': 481, 'end': 507, 'text': '(Zweifach and Lewis, 1996)', 'ref_id': 'BIBREF65'}], 'ref_spans': [], 'section': 'Voltage-clamp recordings'}, {'text': 'Microglia were plated on glass coverslips coated with poly-D-lysine (~2×104 cells/glass) and incubated in DMEM/F12 (Multicell, cat#319-075-CL) containing 10% heat-inactivated fetal bovine serum (Gibco, cat#10438026) and 1× penicillin/streptomycin (Gibco, cat#15140-122) for 1 day. After approximately 24 hours, 5 µM 4-OHT was added and incubated for another 2 days.', 'cite_spans': [], 'ref_spans': [], 'section': 'Phagocytosis Assay'}, {'text': 'Phagocytosis assay was performed as described previously (de Jong et al., 2008) . Briefly, medium containing 0.05% (v/v) of 1.0 µm diameter blue fluorescent FluoSpheres carboxylate-modified author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [{'start': 57, 'end': 79, 'text': '(de Jong et al., 2008)', 'ref_id': 'BIBREF10'}], 'ref_spans': [], 'section': 'Phagocytosis Assay'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.27.921809 doi: bioRxiv preprint microspheres (ThermoFisher, cat#F8814) was added to the wells for 45 min. Microglia cultures were then washed gently with ice cold PBS and fixed with 4% PFA for 20 min.', 'cite_spans': [], 'ref_spans': [], 'section': 'Phagocytosis Assay'}, {'text': 'Immunocytochemistry was then performed for HA and Iba1 as described above. Microglia cultures were prepared from three separate litters, both male and female pups were used for cultures.', 'cite_spans': [], 'ref_spans': [], 'section': 'Phagocytosis Assay'}, {'text': 'Number of cells, number of FluoSpheres, and number of cells containing FluoSpheres were quantified using ImageJ Cell Counter plugin.', 'cite_spans': [], 'ref_spans': [], 'section': 'Phagocytosis Assay'}, {'text': \"Total RNA was extracted from the hippocampus using Aurum Total RNA mini-kit (Bio-Rad, cat#732-6870) and phenol/chloroform mix (Invitrogen, cat#15593-031). cDNA was synthesized from 2 µg of RNA using High Capacity cDNA RT kit (Applied Biosystems, cat#4368814). qPCR was performed in a CFX96 Real-time system (Bio-Rad) using SensiFAST SYBR NO-ROX kit (Invitrogen, cat#BIO-98020) and specific primer sets (shown below) for TNF-α, IL-1β, IL-6, and RPL13a as the reference gene. Experimental groups consisted of at least 7 animals. Data were analyzed using the comparative threshold cycle (Ct) method and results were expressed as fold of change from control. Specific oligonucleotide primers for qPCR are as follows: TNF-α, forward, 5'-CTTCTGTCTACTGAACTTCGGG-3', reverse, 5'-CAGGCTTGTCACTCGAATTTTG-3';\", 'cite_spans': [], 'ref_spans': [], 'section': 'RNA extraction and quantitative PCR'}, {'text': 'author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'RNA extraction and quantitative PCR'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.27.921809 doi: bioRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'RNA extraction and quantitative PCR'}, {'text': 'For all behavioral experiments, mice were injected with vehicle or CNO (1.0 mg/kg, i.p.) 30 min before the trial. All experiments were completed with 4-6 month old male CX3CR1CreER(+/-)-hM3Dq(+/-) mice during the light period (between 10:00 and 16:00). Three cohorts consisting of at least 8 mice per group were tested over three consecutive days. The first cohort was tested for open field and social preference. The second cohort was used for light-dark box, elevated plus maze, and forced swim test. The third cohort was used for social memory. Experimenter was blind to treatment.', 'cite_spans': [], 'ref_spans': [], 'section': 'Behavior'}, {'text': \"Open field activity was assessed as previously described (Janickova et The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.27.921809 doi: bioRxiv preprint Bonferroni post hoc analysis was used. For all open field experiments assessing LPS-induced neuroinflammatory behavior, student's t-test was used for analysis.\", 'cite_spans': [{'start': 57, 'end': 70, 'text': '(Janickova et', 'ref_id': None}], 'ref_spans': [], 'section': 'Open Field'}, {'text': \"Light-dark test was performed as previously described (Depino et al., 2008) . Mice were automatically recorded with VersaMax Tracking System (X-Y and Z sensors) in an open field box (40.64×40.64 cm2) containing a dark enclosure (Opto-Varimex-5 Light/Dark Box, 0150-LDB) within the top half of the box. Mice were placed under the hole of the dark enclosure and allowed to freely move for 5 min. Data are represented as total time the mice spent in the half without the dark enclosure. The crossover measurementwhich records time spent in one region until the animal's body fully enters the other regionwas used to prevent mice poking their heads out of the dark enclosure from being tracked as light side activity. Student's t-test was used for analysis.\", 'cite_spans': [{'start': 54, 'end': 75, 'text': '(Depino et al., 2008)', 'ref_id': 'BIBREF11'}], 'ref_spans': [], 'section': 'Light-Dark Box'}, {'text': 'To assess anxiety-like behavior, the elevated plus maze was performed as previously described (Janickova et al., 2019) . Additional details are as follows. A maze with two open arms and two enclosed arms (open arms: 30×5 cm, surrounded by a 0.5-cm-high border; closed arms: 30×5 cm, surrounded by 19-cm-high walls) (Med Associates Inc., St Albans, USA) was used. Mice were placed on the central platform facing an open arm and were allowed to freely explore for 5 min.', 'cite_spans': [{'start': 94, 'end': 118, 'text': '(Janickova et al., 2019)', 'ref_id': 'BIBREF20'}], 'ref_spans': [], 'section': 'Elevated Plus Maze'}, {'text': \"ANY-maze tracking software (Stoelting Co.) recorded locomotor activity during the test. Total time mice spent in the open arms of the maze was compared using a Student's t-test.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Elevated Plus Maze'}, {'text': 'Forced swim test was completed similarly to a previously described procedure (Beraldo et al., 2015) . A plastic beaker (diameter, 15 cm; height, 25 cm) filled with 16 cm of water at 25˚C was author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [{'start': 77, 'end': 99, 'text': '(Beraldo et al., 2015)', 'ref_id': 'BIBREF2'}], 'ref_spans': [], 'section': 'Forced Swim Test'}, {'text': \"The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.27.921809 doi: bioRxiv preprint placed in the center of a table underneath a camera with ANY-maze software. Mice were placed in the beaker of water, and immobility/mobility was recorded for 6 min. The first min of the trial was discarded to account for the habituation phase. Data expressed as time of immobility during the last 5 min of the trial were analyzed using Student's t-test.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Forced Swim Test'}, {'text': \"Sociability was assessed as previously described (Kwon et al., 2006) . Briefly, testing was done in a three-chambered apparatus (15×90×18.5 cm divided into three chambers of 15×29 cm and separated by dividers with a central 3.8×3.8 cm door) (Med Associates, St. Albans, VT, USA) that offers the subject a choice between a social stimulus and an object. Mice were first habituated for 10 min to the three-chamber apparatus and two empty metal mesh cylinders at each end (Leite et al., 2016) . Mice were then locked in the center of the chamber for 5 min, during which a novel object (green lego block) was placed under the cylinder at one end, and a juvenile mouse of different background was placed under the cylinder at the other end. During the 10 min task, time spent investigating the object and the juvenile mouse was defined as the time a mouse spent sniffing through the holes of either cylinder and was manually scored by an investigator blind to the treatment. In baseline experiments, data were analyzed using student's t-test. In LPS experiments, data were analyzed using repeated measures two-way ANOVA with Bonferroni post hoc analysis.\", 'cite_spans': [{'start': 49, 'end': 68, 'text': '(Kwon et al., 2006)', 'ref_id': 'BIBREF27'}, {'start': 469, 'end': 489, 'text': '(Leite et al., 2016)', 'ref_id': 'BIBREF29'}], 'ref_spans': [], 'section': 'Social preference'}, {'text': \"Social memory was completed as previously described (Prado et al., 2006) . Briefly, mice were singly housed 4 days prior to the experiment to establish territorial dominance. Immediately before testing, mice were habituated to an empty metal mesh cylinder for 10 min in their home cage. To test social interaction, a juvenile intruder was then placed in the test subject's home cage under the author/funder. All rights reserved. No reuse allowed without permission.\", 'cite_spans': [{'start': 52, 'end': 72, 'text': '(Prado et al., 2006)', 'ref_id': 'BIBREF51'}], 'ref_spans': [], 'section': 'Social Memory'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.27.921809 doi: bioRxiv preprint holed cylinder, allowing for olfactory investigation. Time of investigation was defined as the time a mouse spent sniffing through the holes of the cylinder and was manually recorded by an investigator blind to treatment. The task consisted of five trials, each 5 min long and separated with 15 min intertrial intervals. The same juvenile intruder was used for the first four trials and a new juvenile intruder was introduced in trial five to assess social memory. Time spent investigating the juvenile intruder during each of the 5 trials was compared using repeated measures two-way ANOVA with multiple comparisons with Bonferroni post hoc analysis.', 'cite_spans': [], 'ref_spans': [], 'section': 'Social Memory'}, {'text': 'In order to control for potential recognition deficits, an experimental design similar to social memory was used (Prado et al., 2006) . Initially, a 10-min habituation phase was given where the mice explored an empty holed acrylic cylinder in the test cage. Mice then underwent five consecutive 3 min trials with 10 min intertrial intervals. A green rectangular lego block was placed under the cylinder in the first four trials, whereas a red rectangular lego block was used in the last trial. The time spent sniffing the acrylic cylinder (within 5cm; defined by Any-maze software) was measured in each of the five trials. Results are graphed as time (seconds) the mice spent investigating the cylinder during each trial and were analyzed using two-way ANOVA with repeated measures with Bonferroni post hoc analysis.', 'cite_spans': [{'start': 113, 'end': 133, 'text': '(Prado et al., 2006)', 'ref_id': 'BIBREF51'}], 'ref_spans': [], 'section': 'Object Recognition'}, {'text': 'Olfaction was assessed as previously described (Prado et al., 2006) . Briefly, to test for olfactory deficits, mice were exposed to the same scent (vanilla extract) for four consecutive trials and were exposed to another scent (caramel extract) on the fifth trial. Each trial was 3 min in duration with 10 min intertrial intervals. A 5×5 cm2 paper towel was soaked with 3 large drops of either vanilla author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [{'start': 47, 'end': 67, 'text': '(Prado et al., 2006)', 'ref_id': 'BIBREF51'}], 'ref_spans': [], 'section': 'Olfaction Test'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10. 1101 /2020 or caramel extract and placed in a small petri dish with several small holes in its cover to allow for olfaction but prevent physical interaction with the paper towel. The paper towel was replaced with a newly soaked one before each trial to maintain a consistent level of scent during each trial. Time of sniffing was manually recorded by an investigator blind to treatment. Time spent sniffing during each trial was compared using two-way ANOVA with multiple comparisons.', 'cite_spans': [{'start': 98, 'end': 102, 'text': '1101', 'ref_id': None}, {'start': 103, 'end': 108, 'text': '/2020', 'ref_id': None}], 'ref_spans': [], 'section': 'Olfaction Test'}, {'text': \"All statistical analyses were performed using GraphPad software (Prism). For two-group comparisons, Student's unpaired t-test was used when samples displayed a normal distribution. For multiple groups comparisons, one-way ANOVA was used followed by Tukey's post hoc test when appropriate. For multiple factor comparisons, two-way ANOVA was used with Bonferroni's post hoc test. Data are expressed as a mean ± standard error of the mean (SEM).\", 'cite_spans': [], 'ref_spans': [], 'section': 'Statistical Analysis'}, {'text': 'author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Statistical Analysis'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.27.921809 doi: bioRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'Statistical Analysis'}, {'text': 'To examine how M3 muscarinic receptors influence microglia function we used the Cre-loxP strategy to generate mice expressing the human modified M3 muscarinic DREADD (hM3Dq) selectively in microglia. Specifically, we crossed the tamoxifen inducible Cre mouse line CX3CR1-CreER (Parkhurst et al., 2013) to the R26-hM3Dq/mCitrine mice (Zhu et al., 2016) to generate CX3CR1CreER(+/-)-hM3Dq(+/-), which express hM3Dq under control of the strong ubiquitous promoter CAG separated by the DREADD coding sequence by a floxed STOP cassette (Fig. 1A) .', 'cite_spans': [{'start': 277, 'end': 301, 'text': '(Parkhurst et al., 2013)', 'ref_id': 'BIBREF47'}, {'start': 333, 'end': 351, 'text': '(Zhu et al., 2016)', 'ref_id': 'BIBREF64'}], 'ref_spans': [{'start': 531, 'end': 540, 'text': '(Fig. 1A)', 'ref_id': 'FIGREF8'}], 'section': 'mouse brain'}, {'text': 'The hM3Dq carries an HA-tag, which allows detailed spatial localization of the receptor in the cell and is bicistronic, as it is followed by a self-cleaving P2Apeptide and a mCitrine reporter (Zhu et al., 2016) . Tamoxifen-induced Cre expression results in recombination (allowing removal of the STOP cassette) and persistent expression of hM3Dq in long-lived microglia. Conversely, shortlived myeloid cells (monocytes and inflammatory macrophages), which are rapidly replenished by CX3CR1-bone marrow precursors, do not carry Cre-mediated recombination 30 days after tamoxifen injection (Parkhurst et al., 2013) . Thus, to permit Cre-dependent expression of hM3Dq almost exclusively in microglia, all in vivo experiments were conducted at least one month after tamoxifen treatment.', 'cite_spans': [{'start': 192, 'end': 210, 'text': '(Zhu et al., 2016)', 'ref_id': 'BIBREF64'}, {'start': 588, 'end': 612, 'text': '(Parkhurst et al., 2013)', 'ref_id': 'BIBREF47'}], 'ref_spans': [], 'section': 'mouse brain'}, {'text': 'Brain sections co-stained for HA to label hM3Dq and markers of microglia (Iba1), neurons (NeuN) or astrocytes (GFAP, Fig. 1B) showed that hM3Dq was expressed mainly at the cell surface of Iba1-positive microglia, while virtually no co-localization with NeuN or GFAP was observed.', 'cite_spans': [], 'ref_spans': [{'start': 117, 'end': 125, 'text': 'Fig. 1B)', 'ref_id': 'FIGREF8'}], 'section': 'mouse brain'}, {'text': 'A month after tamoxifen injection, 84.2% of counted cells were positive for both HA and Iba1, while 12.5% of cells were Iba1-positive only and 3.3% of cells were HA-positive only (n= 796 author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'mouse brain'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.27.921809 doi: bioRxiv preprint cells from three different mice). HA expression was not observed in either non-tamoxifen treated CX3CR1CreER(+/-)-hM3Dq(+/-) nor in CX3CR1CreER(+/-) littermate control mice (data not shown).', 'cite_spans': [], 'ref_spans': [], 'section': 'mouse brain'}, {'text': 'To investigate the effect of hM3Dq activation on microglia, we initially isolated primary microglia from CX3CR1CreER(+/-)-hM3Dq(+/-) mice and treated cultures with the active tamoxifen metabolite 4-OHT to drive CreER-dependent hM3Dq expression ( Fig. 2A) . Immunofluorescence analysis showed colocalization of the microglia marker Iba1 with HA-tagged hM3Dq (Fig. 2B ).', 'cite_spans': [], 'ref_spans': [{'start': 246, 'end': 254, 'text': 'Fig. 2A)', 'ref_id': 'FIGREF9'}, {'start': 357, 'end': 365, 'text': '(Fig. 2B', 'ref_id': 'FIGREF9'}], 'section': 'Primary microglia cultures obtained from CX3CR1CreER(+/-)-hM3Dq(+/-) mice express functional DREADDs'}, {'text': 'CX3CR1CreER(+/-) littermates (Fig. 2B) .', 'cite_spans': [], 'ref_spans': [{'start': 29, 'end': 38, 'text': '(Fig. 2B)', 'ref_id': 'FIGREF9'}], 'section': 'HA expression was not detected in 4-OHT-treated control microglia cultures prepared from'}, {'text': 'Activation of h3MDq has been shown to increase intracellular Ca2+ signaling in cells (Pei et al., 2008; Urban and Roth, 2015) . We used CNO (3 µM) to activate hM3Dq in Rhod2-loaded microglia isolated from CX3CR1CreER(+/-)-hM3Dq(+/-) mice and test for hM3Dq-mediated changes in intracellular Ca2+ signaling. hM3Dq-positive microglia cells were identified by the expression of mCitrine (Zhu et al., 2016) . Application of CNO increased intracellular Ca2+ levels in the presence (Fig. 2C , green traces) and in the absence (Fig. 2D , green traces) of extracellular Ca2+ in hM3Dq-positive microglia cells, indicating mobilization from intracellular Ca2+ stores. Microglia isolated from CX3CR1CreER(+/-) littermate control mice did not show a CNO-induced increase in Ca2+ signaling (Fig. 2C, D; Fig.2E , F, black traces). However, both positive and negative cells responded to ATP and ionomycin or thapsigargin, used as positive controls (Fig. 2C-F) . These experiments indicate that hM3Dq expression in microglia allows selective activation of Ca2+ signaling pathways by CNO. author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [{'start': 85, 'end': 103, 'text': '(Pei et al., 2008;', 'ref_id': 'BIBREF49'}, {'start': 104, 'end': 125, 'text': 'Urban and Roth, 2015)', 'ref_id': 'BIBREF59'}, {'start': 384, 'end': 402, 'text': '(Zhu et al., 2016)', 'ref_id': 'BIBREF64'}], 'ref_spans': [{'start': 476, 'end': 484, 'text': '(Fig. 2C', 'ref_id': 'FIGREF9'}, {'start': 520, 'end': 528, 'text': '(Fig. 2D', 'ref_id': 'FIGREF9'}, {'start': 777, 'end': 796, 'text': '(Fig. 2C, D; Fig.2E', 'ref_id': 'FIGREF9'}, {'start': 933, 'end': 944, 'text': '(Fig. 2C-F)', 'ref_id': 'FIGREF9'}], 'section': 'HA expression was not detected in 4-OHT-treated control microglia cultures prepared from'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.27.921809 doi: bioRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'HA expression was not detected in 4-OHT-treated control microglia cultures prepared from'}, {'text': 'The release of Ca2+ from intracellular stores by hM3Dq is known to activate store-operated calcium channels (SOCC) that can replenish endoplasmic reticulum Ca2+ stores (Prakriya and Lewis, 2015) . To test whether CNO-induced Ca2+ signaling activates SOC entry through a specific channel, we made voltage-clamp recordings in primary microglia from CX3CR1CreER(+/-)-hM3Dq(+/-) mice in a whole-cell configuration. By applying voltage ramps (from -100mV to +50mV over 150ms) to test cells, we found that activation of h3MDq by CNO evoked a current appearing only at negative membrane potentials (Figs. 2G and 2H; F(2,36)=7.663, p=0.0017, oneway ANOVA), which has been previously related to SOCCs (Michaelis et al., 2015) . This CNO- Elevation of intracellular Ca2+ has been shown to be important for many microglial functions including phagocytosis (Heo et al., 2015; Hoffmann et al., 2003; Toescu et al., 1998 ).', 'cite_spans': [{'start': 168, 'end': 194, 'text': '(Prakriya and Lewis, 2015)', 'ref_id': 'BIBREF52'}, {'start': 692, 'end': 716, 'text': '(Michaelis et al., 2015)', 'ref_id': 'BIBREF36'}, {'start': 845, 'end': 863, 'text': '(Heo et al., 2015;', 'ref_id': 'BIBREF18'}, {'start': 864, 'end': 886, 'text': 'Hoffmann et al., 2003;', 'ref_id': 'BIBREF19'}, {'start': 887, 'end': 906, 'text': 'Toescu et al., 1998', 'ref_id': 'BIBREF57'}], 'ref_spans': [], 'section': 'HA expression was not detected in 4-OHT-treated control microglia cultures prepared from'}, {'text': 'Thus, we tested whether in vitro activation of hM3Dq by CNO facilitates phagocytosis ( The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.27.921809 doi: bioRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'HA expression was not detected in 4-OHT-treated control microglia cultures prepared from'}, {'text': 'As alterations in the intracellular calcium concentration have been shown to be important for modulation of cytokine expression by microglia (Goghari et al., 2000; Hoffmann et al., 2003; McLarnon et al., 2001; Tvrdik and Kalani, 2017) , we investigated whether acute activation of hM3Dq alters microglia cytokine expression under normal conditions as well as during LPSinduced neuroinflammation. To test this, we used qPCR to measure mRNA expression of proinflammatory cytokines in the hippocampus of CX3CR1CreER(+/-)-hM3Dq(+/-) mice that were treated with CNO or vehicle prior to receiving either LPS or saline injection (Fig. 3A) . Acute ', 'cite_spans': [{'start': 141, 'end': 163, 'text': '(Goghari et al., 2000;', 'ref_id': 'BIBREF14'}, {'start': 164, 'end': 186, 'text': 'Hoffmann et al., 2003;', 'ref_id': 'BIBREF19'}, {'start': 187, 'end': 209, 'text': 'McLarnon et al., 2001;', 'ref_id': 'BIBREF35'}, {'start': 210, 'end': 234, 'text': 'Tvrdik and Kalani, 2017)', 'ref_id': 'BIBREF58'}], 'ref_spans': [{'start': 622, 'end': 631, 'text': '(Fig. 3A)', 'ref_id': None}], 'section': 'Acute activation of hM3Dq increases mRNA expression of pro-inflammatory cytokines in the brain'}, {'text': 'Activation of pro-inflammatory cytokine expression in the brain by systemic inflammatory challenges induces symptoms of sickness behavior with a number of behavioral modifications author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Activation of hM3Dq in microglia does not cause overt behavioral change'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi. org/10.1101 org/10. /2020 including decreased locomotion, decreased exploratory behavior, increased anxiety, anorexia, and weight loss (Dantzer, 2001; Tchessalova et al., 2018) . To examine whether increased expression of pro-inflammatory cytokines by hM3Dq activation in microglia also leads to behavioral changes we conducted a series of behavioral analyses where over 3 days mice were treated with CNO (1.0 mg/kg; i.p.) or vehicle (0.5% DMSO in saline) 30 min prior to individual behavioral tests (Fig.   4A ). author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [{'start': 91, 'end': 102, 'text': 'org/10.1101', 'ref_id': None}, {'start': 103, 'end': 116, 'text': 'org/10. /2020', 'ref_id': None}, {'start': 226, 'end': 241, 'text': '(Dantzer, 2001;', 'ref_id': 'BIBREF8'}, {'start': 242, 'end': 267, 'text': 'Tchessalova et al., 2018)', 'ref_id': 'BIBREF56'}], 'ref_spans': [{'start': 591, 'end': 601, 'text': '(Fig.   4A', 'ref_id': 'FIGREF10'}], 'section': 'Activation of hM3Dq in microglia does not cause overt behavioral change'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi. org/10.1101 org/10. /2020 We also investigated whether activation of hM3Dq with CNO alters social behaviors. The three-chamber social preference task showed that hM3Dq signaling in microglia does not change sociability, as time spent investigating another rodent as compared to time spent investigating a plastic block was similar between CNO-and vehicle-treated CX3CR1CreER(+/-)-hM3Dq(+/-) mice F(4,56)=1.156, p=0.3403) . These results suggest that, within the parameters we used, activation of hM3Dq in microglia does not affect normal mouse behavior.', 'cite_spans': [{'start': 91, 'end': 102, 'text': 'org/10.1101', 'ref_id': None}, {'start': 103, 'end': 116, 'text': 'org/10. /2020', 'ref_id': None}], 'ref_spans': [{'start': 487, 'end': 511, 'text': 'F(4,56)=1.156, p=0.3403)', 'ref_id': 'FIGREF8'}], 'section': 'Activation of hM3Dq in microglia does not cause overt behavioral change'}, {'text': 'inflammatory cytokines in the brain author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Chronic stimulation of hM3Dq in microglia attenuates LPS-induced increase of pro-'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10. 1101 /2020 Repetitive peripheral administration of LPS has been shown to induce immune memory in microglia that can then modify their response to inflammatory stimuli (Wendeln et al., 2018) . To test the possibility that repetitive activation of hM3Dq could affect the subsequent microglial responses to LPS, we treated CX3CR1CreER(+/-)-hM3Dq(+/-) mice for 4 days with vehicle or CNO', 'cite_spans': [{'start': 98, 'end': 102, 'text': '1101', 'ref_id': None}, {'start': 103, 'end': 108, 'text': '/2020', 'ref_id': None}, {'start': 265, 'end': 287, 'text': '(Wendeln et al., 2018)', 'ref_id': 'BIBREF61'}], 'ref_spans': [], 'section': 'Chronic stimulation of hM3Dq in microglia attenuates LPS-induced increase of pro-'}, {'text': '(1 mg/kg), then challenged them with LPS 2 hours before collecting brain tissues for evaluation of mRNA expression of the pro-inflammatory cytokines TNF-α, IL-1β, or IL-6 (Fig. 5A ).', 'cite_spans': [], 'ref_spans': [{'start': 171, 'end': 179, 'text': '(Fig. 5A', 'ref_id': 'FIGREF11'}], 'section': 'Chronic stimulation of hM3Dq in microglia attenuates LPS-induced increase of pro-'}, {'text': 'Interestingly, with the 4-days CNO treatment, the chronic activation of microglial hM3Dq alone no longer increased mRNA for inflammatory cytokine (TNF-α: Fig mice that do not express hM3Dq (CX3CR1CreER(+/-)) and were submitted to the same protocol CNO had no effect (Fig. 5E-G) , validating our findings that modulation of LPS-induced cytokine mRNA was attributable to hM3Gq signaling.', 'cite_spans': [], 'ref_spans': [{'start': 154, 'end': 157, 'text': 'Fig', 'ref_id': None}, {'start': 266, 'end': 277, 'text': '(Fig. 5E-G)', 'ref_id': 'FIGREF11'}], 'section': 'Chronic stimulation of hM3Dq in microglia attenuates LPS-induced increase of pro-'}, {'text': 'To examine whether prior hM3Dq activation in microglia influences LPS-induced behavioral changes, we treated CX3CR1CreER(+/-)-hM3Dq(+/-) mice with CNO for 4 consecutive author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Chronic stimulation of hM3Dq in microglia prevents LPS-induced sickness-like behavior'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.27.921809 doi: bioRxiv preprint days, challenged them with LPS (0.1 mg/kg, i.p.) on the 4th day and then performed different behavioral tests 2 h and 24 h after LPS administration (Fig. 6A) . As control groups we used CX3CR1CreER(+/-)-hM3Dq(+/-) mice that received vehicle instead of CNO. Weight analysis during the 5 days showed that activation of hM3Dq with CNO alone did not lead to significant change in weight. On the other hand, both CNO-and vehicle-treated mice showed significant weight loss after LPS injection ( Fig. 6B ; time: F(4,80)=51.43, p<0.0001, two-way ANOVA repeated measures).', 'cite_spans': [], 'ref_spans': [{'start': 290, 'end': 299, 'text': '(Fig. 6A)', 'ref_id': None}, {'start': 632, 'end': 639, 'text': 'Fig. 6B', 'ref_id': None}], 'section': 'Chronic stimulation of hM3Dq in microglia prevents LPS-induced sickness-like behavior'}, {'text': 'Mice in which hM3Dq was activated by CNO showed longer social interaction in the threechamber social preference task than vehicle-treated mice when examined at 2 hour after LPS indicating that, as expected (Corona et al., 2010) , peripheral LPS injection decreased social interaction. Importantly, exploratory activity of CNO-treated mice injected with LPS (338.00 \\uf0b1 20.51; Fig 6C; N=11) was very similar to that of mice that did not receive LPS, with or without CNO (319.8 \\uf0b1 18.37; Fig 4G; N=20) . These results suggest that chronic activation of hM3Dq in microglia protects against LPS-induced suppression of social interaction. Twenty-four hours after LPS injection, vehicle-injected mice showed increased social interaction (Fig. 6C) , consistent with the time course of recovery of social interaction following low-dose LPS injection (Corona et al., 2010) . At the 24h time point, there was no difference between vehicle-and CNO-treated groups.', 'cite_spans': [{'start': 206, 'end': 227, 'text': '(Corona et al., 2010)', 'ref_id': 'BIBREF7'}, {'start': 839, 'end': 860, 'text': '(Corona et al., 2010)', 'ref_id': 'BIBREF7'}], 'ref_spans': [{'start': 374, 'end': 381, 'text': 'Fig 6C;', 'ref_id': None}, {'start': 483, 'end': 496, 'text': 'Fig 4G; N=20)', 'ref_id': 'FIGREF9'}, {'start': 728, 'end': 737, 'text': '(Fig. 6C)', 'ref_id': None}], 'section': 'Chronic stimulation of hM3Dq in microglia prevents LPS-induced sickness-like behavior'}, {'text': 'A similar protective effect was observed when we tested mice for locomotor activity in the open author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Chronic stimulation of hM3Dq in microglia prevents LPS-induced sickness-like behavior'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https: //doi.org/10.1101 //doi.org/10. /2020 field 24h after LPS injection (compare Fig. 6D and 4B ). Together, these data suggest that chronic activation of hM3Dq in microglia protects against LPS-induced sickness behavior.', 'cite_spans': [{'start': 85, 'end': 102, 'text': '//doi.org/10.1101', 'ref_id': None}, {'start': 103, 'end': 122, 'text': '//doi.org/10. /2020', 'ref_id': None}], 'ref_spans': [{'start': 162, 'end': 176, 'text': 'Fig. 6D and 4B', 'ref_id': 'FIGREF10'}], 'section': 'Chronic stimulation of hM3Dq in microglia prevents LPS-induced sickness-like behavior'}, {'text': 'Using the Cre-lox system to drive specific and inducible expression of hM3Dq in microglia, we dissected the action of muscarinic M3 receptor (the hM3Dq progenitor) signaling in these cells in vivo, with implications for other GPCRs that are also Gq-associated. Our results demonstrate that acute activation of microglial hM3Dq-signaling enhances the production of pro-inflammatory cytokines in the brain. In contrast, chronic activation of this pathway attenuates LPS-mediated cytokine expression and sickness behavior.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Tamoxifen-induced Cre expression in CX3CR1-positive cells allowed persistent expression of hM3Dq in long-lived microglia but not in short-lived myeloid cells. Previous validation of the CX3CR1CreER-dependent manipulation of gene expression in microglia using the reporter Rosa26-STOP-DsRed mice showed that in the absence of tamoxifen, very few (~0.3%) CX3CR1-positive cells expressed DsRed. In contrast, 5 days after tamoxifen treatment, the majority of CX3CR1positive cells were positive for DsRed. Also, 30 days following tamoxifen treatment, more than 90% of microglia still expressed DsRed (Parkhurst et al., 2013) , whereas less than 2% of CX3CR1positive monocytes in the blood showed the expression due to their rapid turnover. Importantly, because microglia are long-lived (Fuger et al., 2017) , they continue to express the recombined gene (Parkhurst et al., 2013) . Our results are compatible with these previous observations, as we detected hM3Dq in microglia up to almost a year after tamoxifen treatment (not shown). Also, we did not author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [{'start': 595, 'end': 619, 'text': '(Parkhurst et al., 2013)', 'ref_id': 'BIBREF47'}, {'start': 781, 'end': 801, 'text': '(Fuger et al., 2017)', 'ref_id': 'BIBREF13'}, {'start': 849, 'end': 873, 'text': '(Parkhurst et al., 2013)', 'ref_id': 'BIBREF47'}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https: //doi.org/10.1101 //doi.org/10. /2020 detect expression of hM3Dq in astrocytes or neurons. These results confirm that the mouse model we generated expresses hM3Dq specifically in microglia.', 'cite_spans': [{'start': 85, 'end': 102, 'text': '//doi.org/10.1101', 'ref_id': None}, {'start': 103, 'end': 122, 'text': '//doi.org/10. /2020', 'ref_id': None}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Microglia express several GPCRs that associate to Gq-signaling heterotrimeric G proteins, such as group 1 metabotropic glutamate receptors (mGluR1, mGluR5) (Biber et al., 1999) , purinergic receptors (P2Y1, P2Y2, P2Y4, P2Y6, P2Y11) (Koizumi et al., 2013) , adrenergic (\\uf0611AR) (Mori et al., 2002) and the M3 muscarinic receptor (Pannell et al., 2016) . Although these receptors are activated by different ligands, they are all able to elicit Gq pathway activation. Gq proteins are known to mobilize calcium through inositol 1,4,5-triphosphate-regulation of intracellular stores and diacylglycerol-dependent protein kinase C activation (Mizuno and Itoh, 2009 ). Our in vitro experiments showed that CNO was able to elicit an increase in microglial intracellular calcium concentration, only in hM3Dq-positive cells. This increase was seen in both calcium-free media and media containing calcium, indicating that hM3Dq was functional and the source of calcium influx is, at least in part, from intracellular stores. The functionality of hM3Dq and the signaling pathway activated were also confirmed by patch-clamp experiments showing that hM3Dq activation by CNO induced an increase in conductance in microglia that was abolished by the SOCC inhibitor.', 'cite_spans': [{'start': 156, 'end': 176, 'text': '(Biber et al., 1999)', 'ref_id': 'BIBREF4'}, {'start': 232, 'end': 254, 'text': '(Koizumi et al., 2013)', 'ref_id': 'BIBREF23'}, {'start': 275, 'end': 294, 'text': '(Mori et al., 2002)', 'ref_id': 'BIBREF39'}, {'start': 326, 'end': 348, 'text': '(Pannell et al., 2016)', 'ref_id': 'BIBREF46'}, {'start': 633, 'end': 655, 'text': '(Mizuno and Itoh, 2009', 'ref_id': 'BIBREF37'}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Our results also showed that activation of hM3Dq by CNO increased phagocytosis in cultured microglia. Conversely, activation of muscarinic receptors by carbachol led to increased chemotaxis and decreased phagocytosis in microglia (Pannell et al., 2016) . It is possible that the difference in results stems from the fact that while CNO allows for specific activation of M3 muscarinic receptors, carbachol, a non-selective cholinergic agonist, activates all subtypes of muscarinic and also nicotinic receptors. To note, activation of P2Y6 receptor, a Gq-linked GPCR author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [{'start': 230, 'end': 252, 'text': '(Pannell et al., 2016)', 'ref_id': 'BIBREF46'}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https: //doi.org/10.1101 //doi.org/10. /2020 that responds to UDP, also increased microglia phagocytosis (Koizumi et al., 2007; Neher et al., 2014) .', 'cite_spans': [{'start': 85, 'end': 102, 'text': '//doi.org/10.1101', 'ref_id': None}, {'start': 103, 'end': 122, 'text': '//doi.org/10. /2020', 'ref_id': None}, {'start': 183, 'end': 205, 'text': '(Koizumi et al., 2007;', 'ref_id': 'BIBREF24'}, {'start': 206, 'end': 225, 'text': 'Neher et al., 2014)', 'ref_id': 'BIBREF41'}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Microglia sense exogenous and endogenous inflammatory signals and orchestrate subsequent neuroinflammation which triggers a number of behavioral changes (Biesmans et al., 2013) . Accumulating evidence indicate that the microglial response to inflammatory stimuli can be very plastic (Neher and Cunningham, 2019) . Upon previously encountering an inflammatory stimulus, microglia can adjust their activation state and elicit amplified responses to a secondary inflammatory insult, a phenomenon referred to as microglial pre-conditioning or \"priming\" (Neher and Cunningham, 2019) . Microglia can also develop long-lasting molecular reprograming that suppresses microglial activation upon a secondary insult, a phenotype referred to as \"innate immune memory\" or tolerance (Netea et al., 2016; Netea et al., 2011) . Microglia have recently been shown to display immune training and tolerance in response to repeated daily peripheral stimulation with LPS. One low-dose intraperitoneal injection of LPS led to increased pro-inflammatory cytokine levels in the brain in response to the second injection. Conversely, upon the fourth daily LPS injection microglia barely responded, showing decreased cytokine levels in the brain, an indication of immune tolerance (Wendeln et al., 2018) . We showed that acute activation of hM3Dq in microglia strongly induced brain TNF-α, IL-1β and IL-6 mRNA expression, however, this \"proinflammatory\" effect was completely lost after repeated (4 days) hM3Dq activation, suggesting that repeated activation of muscarinic M3 receptors in microglia can result in microglial tolerance.', 'cite_spans': [{'start': 153, 'end': 176, 'text': '(Biesmans et al., 2013)', 'ref_id': 'BIBREF5'}, {'start': 283, 'end': 311, 'text': '(Neher and Cunningham, 2019)', 'ref_id': 'BIBREF40'}, {'start': 549, 'end': 577, 'text': '(Neher and Cunningham, 2019)', 'ref_id': 'BIBREF40'}, {'start': 769, 'end': 789, 'text': '(Netea et al., 2016;', 'ref_id': 'BIBREF42'}, {'start': 790, 'end': 809, 'text': 'Netea et al., 2011)', 'ref_id': 'BIBREF43'}, {'start': 1255, 'end': 1277, 'text': '(Wendeln et al., 2018)', 'ref_id': 'BIBREF61'}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Recurrent hM3Dq activation also diminished LPS-induced cytokine production and prevented LPS-induced sickness behavior, raising an intriguing possibility that hM3Dq-signaling alone could lead to microglial immune memory formation. Remarkably, pharmacological studies using muscarinic M3-knockout mice showed that in the periphery, M3-signaling is important for immune author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.27.921809 doi: bioRxiv preprint memory formation (Darby et al., 2015) . M3-knockout mice showed reduced CD4+ T cell activation and cytokine production, which resulted in impaired ability to resolve a secondary infection (Darby et al., 2015) .', 'cite_spans': [{'start': 159, 'end': 179, 'text': '(Darby et al., 2015)', 'ref_id': 'BIBREF9'}, {'start': 330, 'end': 350, 'text': '(Darby et al., 2015)', 'ref_id': 'BIBREF9'}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'It is important to note that neither chronic nor acute activation of hM3Dq in microglia affected normal mouse behavior, indicating that hM3Dq activation per se does not lead to sickness behavior or other deleterious effects. Thus, these results suggest that chemogenetics can be a powerful approach to modulate microglia and modify neuroinflammation.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Activation of microglia using DREADDs has been reported previously using viral constructs (Grace et al., 2016; Grace et al., 2018) . In these studies, hM3Dq or hM4Di DREADDs were expressed in spinal cord microglia by intrathecal administration of AAV9 vector constructs.', 'cite_spans': [{'start': 90, 'end': 110, 'text': '(Grace et al., 2016;', 'ref_id': 'BIBREF16'}, {'start': 111, 'end': 130, 'text': 'Grace et al., 2018)', 'ref_id': 'BIBREF17'}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'We avoided using this approach because microglia can respond to viral infection (Kunitsyna et al., 2016; Martianova et al., 2019) , which may be a confounder in these types of studies. In virusinfected animals, CNO-induced activation of hM3Dq was able to elicit allodynia in male rats (Grace et al., 2018) . In the present study, CX3CR1CreER(+/-)-hM3Dq(+/-) mice showed no visual signs of pain following either chronic or acute CNO treatment, based on the basal behavioral performance that was not different from controls. However, because we did not specifically test allodynia, we cannot compare our observations to their results. In agreement with our observations with acute CNO treatment, Grace and colleagues also showed increased pro-inflammatory cytokine synthesis when activating hM3Dq signaling acutely in BV-2 microglia cells.', 'cite_spans': [{'start': 80, 'end': 104, 'text': '(Kunitsyna et al., 2016;', 'ref_id': 'BIBREF26'}, {'start': 105, 'end': 129, 'text': 'Martianova et al., 2019)', 'ref_id': 'BIBREF32'}, {'start': 285, 'end': 305, 'text': '(Grace et al., 2018)', 'ref_id': 'BIBREF17'}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Our data suggest that activation of the muscarinic Gq pathway could be involved in inflammatory cytokine production and immune memory formation, which can likely be extended to other Gq-activating GPCRs. We provide a proof of principle that the use of chemogenetics in author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.27.921809 doi: bioRxiv preprint vivo can be a powerful approach to modulate microglia and regulate cytokine production and the resulting sickness behavior. This approach allows the manipulation of microglial activity in vivo and the ability to study the cross-talk between microglia and neurons in different settings, including during development and in mouse models of neurodegenerative diseases. Transmembrane current density (pA/pF) graph for CX3CR1CreER-hM3Dq(+/-) microglia in control conditions (baseline), and after being sequentially exposed to 3 µM CNO and 20 µM SKF-96365. author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10. 1101 /2020 Inset displays the CNO-evoked current obtained by baseline subtraction. H) Normalized current amplitude of CX3CR1CreER-hM3Dq(+/-) microglia (when VM = -80mV) in conditions of control, and being exposed to CNO and SKF-96365 sequentially. I) Representative images of HA (red) showing phagocytosis of blue FluoSpheres (white) by vehicle-treated or CNO-treated CX3CR1CreER-hM3Dq(+/-) primary microglia. J) Quantification of the number of FluoSpheres per hM3Dq-positive microglia in cultures treated with vehicle or 5 µM CNO. K) Percentage of hM3Dq-positive microglia with FluoSpheres in cultures treated with vehicle or 5 µM CNO. Scale bar 20µm. *p<0.05, **p<0.01, ***p<0.001, ****p<0.0001. All data are represented as mean ± SEM.', 'cite_spans': [{'start': 98, 'end': 102, 'text': '1101', 'ref_id': None}, {'start': 103, 'end': 108, 'text': '/2020', 'ref_id': None}, {'start': 377, 'end': 382, 'text': '(red)', 'ref_id': None}], 'ref_spans': [], 'section': 'Figure Legends'}, {'text': 'A) Experimental approach and timeline for hippocampus gene expression experiments. CNO (1 mg/kg; i.p.); LPS (0.1 mg/kg; i.p.). B-D) qPCR measurements of mRNA expression of TNF-α (B), IL-1β (C), and IL-6 (D) in hippocampus of hM3Dq-positive mice following vehicle or CNO treatment prior to saline or LPS injection. All data are expressed as folds of change from the vehicle/saline-treated group and are represented as mean ± SEM. ns p>0.05, ****p<0.0001. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.27.921809 doi: bioRxiv preprint assess object recognition. J) Time spent investigating an enclosed scent to test olfaction. Data are represented as mean ± SEM. ns p>0.05. CX3CR1CreER(+/-)-hM3Dq(+/-) mice 2 hr after LPS or saline injection. E-G) qPCR measurements of mRNA expression of TNF-α (E), IL-1β (F), and IL-6 (G) in hippocampus of CX3CR1CreER(+/-) (hM3Dq-negative) mice. All data are expressed as folds of change from the vehicle/saline-treated group and are represented as mean ± SEM. ns p>0.05, *p<0.05, *** p<0.001, **** p<0.0001.', 'cite_spans': [], 'ref_spans': [], 'section': 'Figure 3. Acute microglial hM3Dq activation by CNO increases cytokine expression in the brain.'}, {'text': 'A) Experimental approach and timeline for CNO treatment, LPS challenge and behavior assessments. CNO (1 mg/kg; i.p.); LPS (0.1 mg/kg; i.p.). B) Body weight decreased after LPS injection (day 4) for both CNO-and vehicle-treated mice (n=11 per treatment, **** in comparison to days 1-4). C) Time spent investigating juvenile mouse in a three-chamber social preference task at 2h and 24h after LPS treatment. D) Locomotor activity in an open field 24h after LPS treatment.', 'cite_spans': [], 'ref_spans': [], 'section': 'Figure 6. Chronic stimulation of hM3Dq in microglia prevents LPS-induced sickness behavior.'}, {'text': 'All data are represented as mean ± SEM. * p<0.05, ** p<0.01, **** p<0.0001.', 'cite_spans': [], 'ref_spans': [], 'section': 'Figure 6. Chronic stimulation of hM3Dq in microglia prevents LPS-induced sickness behavior.'}, {'text': 'hM3Dq mutant mice were a generous gift of Ute Hochgeschwender and Bryan Roth. We thank Matt Cowan, Jue Fan and Sanda Raulic for assistance with mice and behavioral experiments. author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Acknowledgements:'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.27.921809 doi: bioRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'Acknowledgements:'}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'Microglia: scapegoat, saboteur, or something else?', 'authors': [{'first': 'A', 'middle': [], 'last': 'Aguzzi', 'suffix': ''}, {'first': 'B', 'middle': ['A'], 'last': 'Barres', 'suffix': ''}, {'first': 'M', 'middle': ['L'], 'last': 'Bennett', 'suffix': ''}], 'year': 2013, 'venue': 'Science', 'volume': '339', 'issn': '', 'pages': '156--161', 'other_ids': {}}, 'BIBREF1': {'ref_id': 'b1', 'title': 'Regulation of Amyloid ‚âà√≠‚Äö√¢¬ß Oligomer Binding to Neurons and Neurotoxicity by the Prion Protein-mGluR5 Complex', 'authors': [{'first': 'F', 'middle': ['H'], 'last': 'Beraldo', 'suffix': ''}, {'first': 'V', 'middle': ['G'], 'last': 'Ostapchenko', 'suffix': ''}, {'first': 'F', 'middle': ['A'], 'last': 'Caetano', 'suffix': ''}, {'first': 'A', 'middle': ['L'], 'last': 'Guimaraes', 'suffix': ''}, {'first': 'G', 'middle': ['D'], 'last': 'Ferretti', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Daude', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Bertram', 'suffix': ''}, {'first': 'K', 'middle': ['O'], 'last': 'Nogueira', 'suffix': ''}, {'first': 'J', 'middle': ['L'], 'last': 'Silva', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Westaway', 'suffix': ''}, {'first': 'N', 'middle': ['R'], 'last': 'Cashman', 'suffix': ''}, {'first': 'V', 'middle': ['R'], 'last': 'Martins', 'suffix': ''}, {'first': 'V', 'middle': ['F'], 'last': 'Prado', 'suffix': ''}, {'first': 'M', 'middle': ['A'], 'last': 'Prado', 'suffix': ''}], 'year': 2016, 'venue': 'J Biol Chem', 'volume': '291', 'issn': '', 'pages': '21945--21955', 'other_ids': {}}, 'BIBREF2': {'ref_id': 'b2', 'title': 'Hyperactivity and attention deficits in mice with decreased levels of stress-inducible phosphoprotein 1 (STIP1)', 'authors': [{'first': 'F', 'middle': ['H'], 'last': 'Beraldo', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Thomas', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Kolisnyk', 'suffix': ''}, {'first': 'P', 'middle': ['H'], 'last': 'Hirata', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'De Jaeger', 'suffix': ''}, {'first': 'A', 'middle': ['C'], 'last': 'Martyn', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Fan', 'suffix': ''}, {'first': 'D', 'middle': ['F'], 'last': 'Goncalves', 'suffix': ''}, {'first': 'M', 'middle': ['F'], 'last': 'Cowan', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Masood', 'suffix': ''}, {'first': 'V', 'middle': ['R'], 'last': 'Martins', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Gros', 'suffix': ''}, {'first': 'V', 'middle': ['F'], 'last': 'Prado', 'suffix': ''}, {'first': 'M', 'middle': ['A'], 'last': 'Prado', 'suffix': ''}], 'year': 2015, 'venue': 'Disease models & mechanisms', 'volume': '8', 'issn': '', 'pages': '1457--1466', 'other_ids': {}}, 'BIBREF3': {'ref_id': 'b3', 'title': 'Alpha-tocopherol attenuates lipopolysaccharide-induced sickness behavior in mice', 'authors': [{'first': 'B', 'middle': ['M'], 'last': 'Berg', 'suffix': ''}, {'first': 'J', 'middle': ['P'], 'last': 'Godbout', 'suffix': ''}, {'first': 'K', 'middle': ['W'], 'last': 'Kelley', 'suffix': ''}, {'first': 'R', 'middle': ['W'], 'last': 'Johnson', 'suffix': ''}], 'year': 2004, 'venue': 'Brain Behav Immun', 'volume': '18', 'issn': '', 'pages': '149--157', 'other_ids': {}}, 'BIBREF4': {'ref_id': 'b4', 'title': 'Expression and signaling of group I metabotropic glutamate receptors in astrocytes and microglia', 'authors': [{'first': 'K', 'middle': [], 'last': 'Biber', 'suffix': ''}, {'first': 'D', 'middle': ['J'], 'last': 'Laurie', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Berthele', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Sommer', 'suffix': ''}, {'first': 'T', 'middle': ['R'], 'last': 'Tölle', 'suffix': ''}, {'first': 'P', 'middle': ['J'], 'last': 'Gebicke-Härter', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Van Calker', 'suffix': ''}, {'first': 'H', 'middle': ['W'], 'last': 'Boddeke', 'suffix': ''}], 'year': 1999, 'venue': 'J Neurochem', 'volume': '72', 'issn': '', 'pages': '1671--1680', 'other_ids': {}}, 'BIBREF5': {'ref_id': 'b5', 'title': 'Systemic immune activation leads to neuroinflammation and sickness behavior in mice', 'authors': [{'first': 'S', 'middle': [], 'last': 'Biesmans', 'suffix': ''}, {'first': 'T', 'middle': ['F'], 'last': 'Meert', 'suffix': ''}, {'first': 'J', 'middle': ['A'], 'last': 'Bouwknecht', 'suffix': ''}, {'first': 'P', 'middle': ['D'], 'last': 'Acton', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Davoodi', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'De Haes', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Kuijlaars', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Langlois', 'suffix': ''}, {'first': 'L', 'middle': ['J'], 'last': 'Matthews', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Ver Donck', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Hellings', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Nuydens', 'suffix': ''}], 'year': 2013, 'venue': 'Mediators Inflamm', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF6': {'ref_id': 'b6', 'title': 'Diverse Requirements for Microglial Survival, Specification, and Function Revealed by Defined-Medium Cultures', 'authors': [{'first': 'C', 'middle': ['J'], 'last': 'Bohlen', 'suffix': ''}, {'first': 'F', 'middle': ['C'], 'last': 'Bennett', 'suffix': ''}, {'first': 'A', 'middle': ['F'], 'last': 'Tucker', 'suffix': ''}, {'first': 'H', 'middle': ['Y'], 'last': 'Collins', 'suffix': ''}, {'first': 'S', 'middle': ['B'], 'last': 'Mulinyawe', 'suffix': ''}, {'first': 'B', 'middle': ['A'], 'last': 'Barres', 'suffix': ''}], 'year': 2017, 'venue': 'Neuron', 'volume': '94', 'issn': '', 'pages': '759--773', 'other_ids': {}}, 'BIBREF7': {'ref_id': 'b7', 'title': 'Fractalkine receptor (CX3CR1) deficiency sensitizes mice to the behavioral changes induced by lipopolysaccharide', 'authors': [{'first': 'A', 'middle': ['W'], 'last': 'Corona', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Huang', 'suffix': ''}, {'first': 'J', 'middle': ['C'], 'last': 'O&apos;connor', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Dantzer', 'suffix': ''}, {'first': 'K', 'middle': ['W'], 'last': 'Kelley', 'suffix': ''}, {'first': 'P', 'middle': ['G'], 'last': 'Popovich', 'suffix': ''}, {'first': 'J', 'middle': ['P'], 'last': 'Godbout', 'suffix': ''}], 'year': 2010, 'venue': 'J Neuroinflammation', 'volume': '7', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF8': {'ref_id': 'b8', 'title': 'Cytokine-induced sickness behavior: where do we stand?', 'authors': [{'first': 'R', 'middle': [], 'last': 'Dantzer', 'suffix': ''}], 'year': 2001, 'venue': 'Brain, behavior, and immunity', 'volume': '15', 'issn': '', 'pages': '7--24', 'other_ids': {}}, 'BIBREF9': {'ref_id': 'b9', 'title': 'The M3 muscarinic receptor is required for optimal adaptive immunity to helminth and bacterial infection', 'authors': [{'first': 'M', 'middle': [], 'last': 'Darby', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Schnoeller', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Vira', 'suffix': ''}, {'first': 'F', 'middle': ['J'], 'last': 'Culley', 'suffix': ''}, {'first': 'F', 'middle': [], 'last': 'Culley', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Bobat', 'suffix': ''}, {'first': 'E', 'middle': [], 'last': 'Logan', 'suffix': ''}, {'first': 'F', 'middle': [], 'last': 'Kirstein', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Wess', 'suffix': ''}, {'first': 'A', 'middle': ['F'], 'last': 'Cunningham', 'suffix': ''}, {'first': 'F', 'middle': [], 'last': 'Brombacher', 'suffix': ''}, {'first': 'M', 'middle': ['E'], 'last': 'Selkirk', 'suffix': ''}, {'first': 'W', 'middle': ['G'], 'last': 'Horsnell', 'suffix': ''}], 'year': 2015, 'venue': 'PLoS Pathog', 'volume': '11', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF10': {'ref_id': 'b10', 'title': 'Expression of CXCL4 in microglia in vitro and in vivo and its possible signaling through CXCR3', 'authors': [{'first': 'E', 'middle': ['K'], 'last': 'De Jong', 'suffix': ''}, {'first': 'A', 'middle': ['H'], 'last': 'De Haas', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Brouwer', 'suffix': ''}, {'first': 'H', 'middle': ['R'], 'last': 'Van Weering', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Hensens', 'suffix': ''}, {'first': 'I', 'middle': [], 'last': 'Bechmann', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Pratley', 'suffix': ''}, {'first': 'E', 'middle': [], 'last': 'Wesseling', 'suffix': ''}, {'first': 'H', 'middle': ['W'], 'last': 'Boddeke', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Biber', 'suffix': ''}], 'year': 2008, 'venue': 'Journal of neurochemistry', 'volume': '105', 'issn': '', 'pages': '1726--1736', 'other_ids': {}}, 'BIBREF11': {'ref_id': 'b11', 'title': 'GABA homeostasis contributes to the developmental programming of anxiety-related behavior', 'authors': [{'first': 'A', 'middle': ['M'], 'last': 'Depino', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Tsetsenis', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Gross', 'suffix': ''}], 'year': 2008, 'venue': 'Brain Res', 'volume': '1210', 'issn': '', 'pages': '189--199', 'other_ids': {}}, 'BIBREF12': {'ref_id': 'b12', 'title': 'A chemical-genetic approach for precise spatio-temporal control of cellular signaling', 'authors': [{'first': 'S', 'middle': [], 'last': 'Dong', 'suffix': ''}, {'first': 'J', 'middle': ['A'], 'last': 'Allen', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Farrell', 'suffix': ''}, {'first': 'B', 'middle': ['L'], 'last': 'Roth', 'suffix': ''}], 'year': 2010, 'venue': 'Mol Biosyst', 'volume': '6', 'issn': '', 'pages': '1376--1380', 'other_ids': {}}, 'BIBREF13': {'ref_id': 'b13', 'title': \"Microglia turnover with aging and in an Alzheimer's model via long-term in vivo single-cell imaging\", 'authors': [{'first': 'P', 'middle': [], 'last': 'Fuger', 'suffix': ''}, {'first': 'J', 'middle': ['K'], 'last': 'Hefendehl', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Veeraraghavalu', 'suffix': ''}, {'first': 'A', 'middle': ['C'], 'last': 'Wendeln', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Schlosser', 'suffix': ''}, {'first': 'U', 'middle': [], 'last': 'Obermuller', 'suffix': ''}, {'first': 'B', 'middle': ['M'], 'last': 'Wegenast-Braun', 'suffix': ''}, {'first': 'J', 'middle': ['J'], 'last': 'Neher', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Martus', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Kohsaka', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Thunemann', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Feil', 'suffix': ''}, {'first': 'S', 'middle': ['S'], 'last': 'Sisodia', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Skodras', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Jucker', 'suffix': ''}], 'year': 2017, 'venue': 'Nat Neurosci', 'volume': '20', 'issn': '', 'pages': '1371--1376', 'other_ids': {}}, 'BIBREF14': {'ref_id': 'b14', 'title': 'Acute application of interleukin-1beta induces Ca(2+) responses in human microglia', 'authors': [{'first': 'V', 'middle': [], 'last': 'Goghari', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Franciosi', 'suffix': ''}, {'first': 'S', 'middle': ['U'], 'last': 'Kim', 'suffix': ''}, {'first': 'Y', 'middle': ['B'], 'last': 'Lee', 'suffix': ''}, {'first': 'J', 'middle': ['G'], 'last': 'Mclarnon', 'suffix': ''}], 'year': 2000, 'venue': 'Neurosci Lett', 'volume': '281', 'issn': '', 'pages': '83--86', 'other_ids': {}}, 'BIBREF15': {'ref_id': 'b15', 'title': 'Chemogenetics revealed: DREADD occupancy and activation via converted clozapine', 'authors': [{'first': 'J', 'middle': ['L'], 'last': 'Gomez', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Bonaventura', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Lesniak', 'suffix': ''}, {'first': 'W', 'middle': ['B'], 'last': 'Mathews', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Sysa-Shah', 'suffix': ''}, {'first': 'L', 'middle': ['A'], 'last': 'Rodriguez', 'suffix': ''}, {'first': 'R', 'middle': ['J'], 'last': 'Ellis', 'suffix': ''}, {'first': 'C', 'middle': ['T'], 'last': 'Richie', 'suffix': ''}, {'first': 'B', 'middle': ['K'], 'last': 'Harvey', 'suffix': ''}, {'first': 'R', 'middle': ['F'], 'last': 'Dannals', 'suffix': ''}, {'first': 'M', 'middle': ['G'], 'last': 'Pomper', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Bonci', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Michaelides', 'suffix': ''}], 'year': 2017, 'venue': 'Science', 'volume': '357', 'issn': '', 'pages': '503--507', 'other_ids': {}}, 'BIBREF16': {'ref_id': 'b16', 'title': 'Morphine paradoxically prolongs neuropathic pain in rats by amplifying spinal NLRP3 inflammasome activation', 'authors': [{'first': 'P', 'middle': ['M'], 'last': 'Grace', 'suffix': ''}, {'first': 'K', 'middle': ['A'], 'last': 'Strand', 'suffix': ''}, {'first': 'E', 'middle': ['L'], 'last': 'Galer', 'suffix': ''}, {'first': 'D', 'middle': ['J'], 'last': 'Urban', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'M', 'middle': ['V'], 'last': 'Baratta', 'suffix': ''}, {'first': 'T', 'middle': ['J'], 'last': 'Fabisiak', 'suffix': ''}, {'first': 'N', 'middle': ['D'], 'last': 'Anderson', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Cheng', 'suffix': ''}, {'first': 'L', 'middle': ['I'], 'last': 'Greene', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Berkelhammer', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'A', 'middle': ['L'], 'last': 'Ellis', 'suffix': ''}, {'first': 'H', 'middle': ['H'], 'last': 'Yin', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Campeau', 'suffix': ''}, {'first': 'K', 'middle': ['C'], 'last': 'Rice', 'suffix': ''}, {'first': 'B', 'middle': ['L'], 'last': 'Roth', 'suffix': ''}, {'first': 'S', 'middle': ['F'], 'last': 'Maier', 'suffix': ''}, {'first': 'L', 'middle': ['R'], 'last': 'Watkins', 'suffix': ''}], 'year': 2016, 'venue': 'Proc Natl Acad Sci U S A', 'volume': '113', 'issn': '', 'pages': '3441--3450', 'other_ids': {}}, 'BIBREF17': {'ref_id': 'b17', 'title': 'DREADDed microglia in pain: Implications for spinal inflammatory signaling in male rats', 'authors': [{'first': 'P', 'middle': ['M'], 'last': 'Grace', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'K', 'middle': ['A'], 'last': 'Strand', 'suffix': ''}, {'first': 'M', 'middle': ['V'], 'last': 'Baratta', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'E', 'middle': ['L'], 'last': 'Galer', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Yin', 'suffix': ''}, {'first': 'S', 'middle': ['F'], 'last': 'Maier', 'suffix': ''}, {'first': 'L', 'middle': ['R'], 'last': 'Watkins', 'suffix': ''}], 'year': 2018, 'venue': 'Experimental neurology', 'volume': '304', 'issn': '', 'pages': '125--131', 'other_ids': {}}, 'BIBREF18': {'ref_id': 'b18', 'title': 'Regulation of phagocytosis and cytokine secretion by store-operated calcium entry in primary isolated murine microglia', 'authors': [{'first': 'D', 'middle': ['K'], 'last': 'Heo', 'suffix': ''}, {'first': 'H', 'middle': ['M'], 'last': 'Lim', 'suffix': ''}, {'first': 'J', 'middle': ['H'], 'last': 'Nam', 'suffix': ''}, {'first': 'M', 'middle': ['G'], 'last': 'Lee', 'suffix': ''}, {'first': 'J', 'middle': ['Y'], 'last': 'Kim', 'suffix': ''}], 'year': 2015, 'venue': 'Cell Signal', 'volume': '27', 'issn': '', 'pages': '177--186', 'other_ids': {}}, 'BIBREF19': {'ref_id': 'b19', 'title': 'Elevation of basal intracellular calcium as a central element in the activation of brain macrophages (microglia): suppression of receptor-evoked calcium signaling and control of release function', 'authors': [{'first': 'A', 'middle': [], 'last': 'Hoffmann', 'suffix': ''}, {'first': 'O', 'middle': [], 'last': 'Kann', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Ohlemeyer', 'suffix': ''}, {'first': 'U', 'middle': ['K'], 'last': 'Hanisch', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Kettenmann', 'suffix': ''}], 'year': 2003, 'venue': 'J Neurosci', 'volume': '23', 'issn': '', 'pages': '4410--4419', 'other_ids': {}}, 'BIBREF20': {'ref_id': 'b20', 'title': 'Selective decrease of cholinergic signaling from pedunculopontine and laterodorsal tegmental nuclei has little impact on cognition but markedly increases susceptibility to stress', 'authors': [{'first': 'H', 'middle': [], 'last': 'Janickova', 'suffix': ''}, {'first': 'O', 'middle': [], 'last': 'Kljakic', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Rosborough', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Raulic', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Matovic', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Gros', 'suffix': ''}, {'first': 'L', 'middle': ['M'], 'last': 'Saksida', 'suffix': ''}, {'first': 'T', 'middle': ['J'], 'last': 'Bussey', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Inoue', 'suffix': ''}, {'first': 'V', 'middle': ['F'], 'last': 'Prado', 'suffix': ''}, {'first': 'M', 'middle': ['A M'], 'last': 'Prado', 'suffix': ''}], 'year': 2019, 'venue': 'FASEB J', 'volume': '33', 'issn': '', 'pages': '7018--7036', 'other_ids': {}}, 'BIBREF21': {'ref_id': 'b21', 'title': 'Deletion of the vesicular acetylcholine transporter from pedunculopontine/laterodorsal tegmental neurons modifies gait', 'authors': [{'first': 'H', 'middle': [], 'last': 'Janickova', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Rosborough', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Al-Onaizi', 'suffix': ''}, {'first': 'O', 'middle': [], 'last': 'Kljakic', 'suffix': ''}, {'first': 'M', 'middle': ['S'], 'last': 'Guzman', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Gros', 'suffix': ''}, {'first': 'M', 'middle': ['A'], 'last': 'Prado', 'suffix': ''}, {'first': 'V', 'middle': ['F'], 'last': 'Prado', 'suffix': ''}], 'year': 2017, 'venue': 'Journal of neurochemistry', 'volume': '140', 'issn': '', 'pages': '787--798', 'other_ids': {}}, 'BIBREF22': {'ref_id': 'b22', 'title': 'Physiology of microglia', 'authors': [{'first': 'H', 'middle': [], 'last': 'Kettenmann', 'suffix': ''}, {'first': 'U', 'middle': ['K'], 'last': 'Hanisch', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Noda', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Verkhratsky', 'suffix': ''}], 'year': 2011, 'venue': 'Physiological reviews', 'volume': '91', 'issn': '', 'pages': '461--553', 'other_ids': {}}, 'BIBREF23': {'ref_id': 'b23', 'title': 'Purinergic receptors in microglia: functional modal shifts of microglia mediated by P2 and P1 receptors', 'authors': [{'first': 'S', 'middle': [], 'last': 'Koizumi', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Ohsawa', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Inoue', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Kohsaka', 'suffix': ''}], 'year': 2013, 'venue': 'Glia', 'volume': '61', 'issn': '', 'pages': '47--54', 'other_ids': {}}, 'BIBREF24': {'ref_id': 'b24', 'title': 'UDP acting at P2Y6 receptors is a mediator of microglial phagocytosis', 'authors': [{'first': 'S', 'middle': [], 'last': 'Koizumi', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Shigemoto-Mogami', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Nasu-Tada', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Shinozaki', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Ohsawa', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Tsuda', 'suffix': ''}, {'first': 'B', 'middle': ['V'], 'last': 'Joshi', 'suffix': ''}, {'first': 'K', 'middle': ['A'], 'last': 'Jacobson', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Kohsaka', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Inoue', 'suffix': ''}], 'year': 2007, 'venue': 'Nature', 'volume': '446', 'issn': '', 'pages': '1091--1095', 'other_ids': {}}, 'BIBREF25': {'ref_id': 'b25', 'title': 'Cholinergic Regulation of hnRNPA2/B1 Translation by M1 Muscarinic Receptors', 'authors': [{'first': 'B', 'middle': [], 'last': 'Kolisnyk', 'suffix': ''}, {'first': 'M', 'middle': ['A'], 'last': 'Al-Onaizi', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Xu', 'suffix': ''}, {'first': 'G', 'middle': ['M'], 'last': 'Parfitt', 'suffix': ''}, {'first': 'V', 'middle': ['G'], 'last': 'Ostapchenko', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Hanin', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Soreq', 'suffix': ''}, {'first': 'M', 'middle': ['A'], 'last': 'Prado', 'suffix': ''}, {'first': 'V', 'middle': ['F'], 'last': 'Prado', 'suffix': ''}], 'year': 2016, 'venue': 'J Neurosci', 'volume': '36', 'issn': '', 'pages': '6287--6296', 'other_ids': {}}, 'BIBREF26': {'ref_id': 'b26', 'title': 'Lentiviral Transduction of Neurons in Adult Brain: Evaluation of Inflammatory Response and Cognitive Effects in Mice', 'authors': [{'first': 'T', 'middle': ['A'], 'last': 'Kunitsyna', 'suffix': ''}, {'first': 'O', 'middle': ['I'], 'last': 'Ivashkina', 'suffix': ''}, {'first': 'M', 'middle': ['A'], 'last': 'Roshchina', 'suffix': ''}, {'first': 'K', 'middle': ['A'], 'last': 'Toropova', 'suffix': ''}, {'first': 'K', 'middle': ['V'], 'last': 'Anokhin', 'suffix': ''}], 'year': 2016, 'venue': 'Bull Exp Biol Med', 'volume': '161', 'issn': '', 'pages': '316--319', 'other_ids': {}}, 'BIBREF27': {'ref_id': 'b27', 'title': 'Pten regulates neuronal arborization and social interaction in mice', 'authors': [{'first': 'C', 'middle': ['H'], 'last': 'Kwon', 'suffix': ''}, {'first': 'B', 'middle': ['W'], 'last': 'Luikart', 'suffix': ''}, {'first': 'C', 'middle': ['M'], 'last': 'Powell', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Zhou', 'suffix': ''}, {'first': 'S', 'middle': ['A'], 'last': 'Matheny', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'S', 'middle': ['J'], 'last': 'Baker', 'suffix': ''}, {'first': 'L', 'middle': ['F'], 'last': 'Parada', 'suffix': ''}], 'year': 2006, 'venue': 'Neuron', 'volume': '50', 'issn': '', 'pages': '377--388', 'other_ids': {}}, 'BIBREF28': {'ref_id': 'b28', 'title': 'Forebrain Cholinergic Signaling Regulates Innate Immune Responses and Inflammation', 'authors': [{'first': 'K', 'middle': ['R'], 'last': 'Lehner', 'suffix': ''}, {'first': 'H', 'middle': ['A'], 'last': 'Silverman', 'suffix': ''}, {'first': 'M', 'middle': ['E'], 'last': 'Addorisio', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Roy', 'suffix': ''}, {'first': 'M', 'middle': ['A'], 'last': 'Al-Onaizi', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Levine', 'suffix': ''}, {'first': 'P', 'middle': ['S'], 'last': 'Olofsson', 'suffix': ''}, {'first': 'S', 'middle': ['S'], 'last': 'Chavan', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Gros', 'suffix': ''}, {'first': 'N', 'middle': ['M'], 'last': 'Nathanson', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Al-Abed', 'suffix': ''}, {'first': 'C', 'middle': ['N'], 'last': 'Metz', 'suffix': ''}, {'first': 'V', 'middle': ['F'], 'last': 'Prado', 'suffix': ''}, {'first': 'M', 'middle': ['A M'], 'last': 'Prado', 'suffix': ''}, {'first': 'K', 'middle': ['J'], 'last': 'Tracey', 'suffix': ''}, {'first': 'V', 'middle': ['A'], 'last': 'Pavlov', 'suffix': ''}], 'year': 2019, 'venue': 'Frontiers in immunology', 'volume': '10', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF29': {'ref_id': 'b29', 'title': 'Vesicular acetylcholine transporter knock down-mice are more susceptible to inflammation, c-Fos expression and sickness behavior induced by lipopolysaccharide', 'authors': [{'first': 'H', 'middle': ['R'], 'last': 'Leite', 'suffix': ''}, {'first': 'O', 'middle': ['C'], 'last': 'Oliveira-Lima', 'suffix': ''}, {'first': 'L', 'middle': ['M'], 'last': 'Pereira', 'suffix': ''}, {'first': 'V', 'middle': ['E M'], 'last': 'Oliveira', 'suffix': ''}, {'first': 'V', 'middle': ['F'], 'last': 'Prado', 'suffix': ''}, {'first': 'M', 'middle': ['A M'], 'last': 'Prado', 'suffix': ''}, {'first': 'G', 'middle': ['S'], 'last': 'Pereira', 'suffix': ''}, {'first': 'A', 'middle': ['R'], 'last': 'Massensini', 'suffix': ''}], 'year': 2016, 'venue': 'Brain Behav Immun', 'volume': '57', 'issn': '', 'pages': '282--292', 'other_ids': {}}, 'BIBREF30': {'ref_id': 'b30', 'title': 'Microglia and macrophages in brain homeostasis and disease', 'authors': [{'first': 'Q', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'B', 'middle': ['A'], 'last': 'Barres', 'suffix': ''}], 'year': 2018, 'venue': 'Nat Rev Immunol', 'volume': '18', 'issn': '', 'pages': '225--242', 'other_ids': {}}, 'BIBREF31': {'ref_id': 'b31', 'title': 'Nicotinic acetylcholine receptor subunits in rhesus monkey retina', 'authors': [{'first': 'J', 'middle': [], 'last': 'Liu', 'suffix': ''}, {'first': 'A', 'middle': ['M'], 'last': 'Mcglinn', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Fernandes', 'suffix': ''}, {'first': 'A', 'middle': ['H'], 'last': 'Milam', 'suffix': ''}, {'first': 'C', 'middle': ['E'], 'last': 'Strang', 'suffix': ''}, {'first': 'M', 'middle': ['E'], 'last': 'Andison', 'suffix': ''}, {'first': 'J', 'middle': ['M'], 'last': 'Lindstrom', 'suffix': ''}, {'first': 'K', 'middle': ['T'], 'last': 'Keyser', 'suffix': ''}, {'first': 'R', 'middle': ['A'], 'last': 'Stone', 'suffix': ''}], 'year': 2009, 'venue': 'Investigative ophthalmology & visual science', 'volume': '50', 'issn': '', 'pages': '1408--1415', 'other_ids': {}}, 'BIBREF32': {'ref_id': 'b32', 'title': 'Activation of microglia associated with lentiviral transduction: A semiautomated method of assessment', 'authors': [{'first': 'E', 'middle': [], 'last': 'Martianova', 'suffix': ''}, {'first': 'V', 'middle': ['A'], 'last': 'Aniol', 'suffix': ''}, {'first': 'A', 'middle': ['O'], 'last': 'Manolova', 'suffix': ''}, {'first': 'A', 'middle': ['A'], 'last': 'Kvichansky', 'suffix': ''}, {'first': 'N', 'middle': ['V'], 'last': 'Gulyaeva', 'suffix': ''}], 'year': 2019, 'venue': 'Acta Histochem', 'volume': '121', 'issn': '', 'pages': '368--375', 'other_ids': {}}, 'BIBREF33': {'ref_id': 'b33', 'title': 'Novel strains of mice deficient for the vesicular acetylcholine transporter: insights on transcriptional regulation and control of locomotor behavior', 'authors': [{'first': 'C', 'middle': [], 'last': 'Martins-Silva', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'De Jaeger', 'suffix': ''}, {'first': 'M', 'middle': ['S'], 'last': 'Guzman', 'suffix': ''}, {'first': 'R', 'middle': ['D'], 'last': 'Lima', 'suffix': ''}, {'first': 'M', 'middle': ['S'], 'last': 'Santos', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Kushmerick', 'suffix': ''}, {'first': 'M', 'middle': ['V'], 'last': 'Gomez', 'suffix': ''}, {'first': 'M', 'middle': ['G'], 'last': 'Caron', 'suffix': ''}, {'first': 'M', 'middle': ['A'], 'last': 'Prado', 'suffix': ''}, {'first': 'V', 'middle': ['F'], 'last': 'Prado', 'suffix': ''}], 'year': 2011, 'venue': 'PloS one', 'volume': '6', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF34': {'ref_id': 'b34', 'title': 'Preparation of separate astroglial and oligodendroglial cell cultures from rat cerebral tissue', 'authors': [{'first': 'K', 'middle': ['D'], 'last': 'Mccarthy', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'De Vellis', 'suffix': ''}], 'year': 1980, 'venue': 'J Cell Biol', 'volume': '85', 'issn': '', 'pages': '890--902', 'other_ids': {}}, 'BIBREF35': {'ref_id': 'b35', 'title': 'Acute actions of tumor necrosis factor-alpha on intracellular Ca(2+) and K(+) currents in human microglia', 'authors': [{'first': 'J', 'middle': ['G'], 'last': 'Mclarnon', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Franciosi', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'J', 'middle': ['H'], 'last': 'Bae', 'suffix': ''}, {'first': 'H', 'middle': ['B'], 'last': 'Choi', 'suffix': ''}, {'first': 'S', 'middle': ['U'], 'last': 'Kim', 'suffix': ''}], 'year': 2001, 'venue': 'Neuroscience', 'volume': '104', 'issn': '', 'pages': '1175--1184', 'other_ids': {}}, 'BIBREF36': {'ref_id': 'b36', 'title': 'STIM1, STIM2, and Orai1 regulate store-operated calcium entry and purinergic activation of microglia', 'authors': [{'first': 'M', 'middle': [], 'last': 'Michaelis', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Nieswandt', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Stegner', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Eilers', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Kraft', 'suffix': ''}], 'year': 2015, 'venue': 'Glia', 'volume': '63', 'issn': '', 'pages': '652--663', 'other_ids': {}}, 'BIBREF37': {'ref_id': 'b37', 'title': 'Functions and regulatory mechanisms of Gq-signaling pathways', 'authors': [{'first': 'N', 'middle': [], 'last': 'Mizuno', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Itoh', 'suffix': ''}], 'year': 2009, 'venue': 'Neurosignals', 'volume': '17', 'issn': '', 'pages': '42--54', 'other_ids': {}}, 'BIBREF38': {'ref_id': 'b38', 'title': 'Activation of nicotinic acetylcholine receptor prevents the production of reactive oxygen species in fibrillar beta amyloid peptide (1-42)-stimulated microglia', 'authors': [{'first': 'J', 'middle': ['H'], 'last': 'Moon', 'suffix': ''}, {'first': 'S', 'middle': ['Y'], 'last': 'Kim', 'suffix': ''}, {'first': 'H', 'middle': ['G'], 'last': 'Lee', 'suffix': ''}, {'first': 'S', 'middle': ['U'], 'last': 'Kim', 'suffix': ''}, {'first': 'Y', 'middle': ['B'], 'last': 'Lee', 'suffix': ''}], 'year': 2008, 'venue': 'Exp Mol Med', 'volume': '40', 'issn': '', 'pages': '11--18', 'other_ids': {}}, 'BIBREF39': {'ref_id': 'b39', 'title': 'Effects of norepinephrine on rat cultured microglial cells that express alpha1, alpha2, beta1 and beta2 adrenergic receptors', 'authors': [{'first': 'K', 'middle': [], 'last': 'Mori', 'suffix': ''}, {'first': 'E', 'middle': [], 'last': 'Ozaki', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Yang', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Yokoyama', 'suffix': ''}, {'first': 'I', 'middle': [], 'last': 'Takeda', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Maeda', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Sakanaka', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Tanaka', 'suffix': ''}], 'year': 2002, 'venue': 'Neuropharmacology', 'volume': '43', 'issn': '', 'pages': '1026--1034', 'other_ids': {}}, 'BIBREF40': {'ref_id': 'b40', 'title': 'Priming Microglia for Innate Immune Memory in the Brain', 'authors': [{'first': 'J', 'middle': ['J'], 'last': 'Neher', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Cunningham', 'suffix': ''}], 'year': 2019, 'venue': 'Trends Immunol', 'volume': '40', 'issn': '', 'pages': '358--374', 'other_ids': {}}, 'BIBREF41': {'ref_id': 'b41', 'title': 'Inhibition of UDP/P2Y6 purinergic signaling prevents phagocytosis of viable neurons by activated microglia in vitro and in vivo', 'authors': [{'first': 'J', 'middle': ['J'], 'last': 'Neher', 'suffix': ''}, {'first': 'U', 'middle': [], 'last': 'Neniskyte', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Hornik', 'suffix': ''}, {'first': 'G', 'middle': ['C'], 'last': 'Brown', 'suffix': ''}], 'year': 2014, 'venue': 'Glia', 'volume': '62', 'issn': '', 'pages': '1463--1475', 'other_ids': {}}, 'BIBREF42': {'ref_id': 'b42', 'title': 'Trained immunity: A program of innate immune memory in health and disease', 'authors': [{'first': 'M', 'middle': ['G'], 'last': 'Netea', 'suffix': ''}, {'first': 'L', 'middle': ['A'], 'last': 'Joosten', 'suffix': ''}, {'first': 'E', 'middle': [], 'last': 'Latz', 'suffix': ''}, {'first': 'K', 'middle': ['H'], 'last': 'Mills', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Natoli', 'suffix': ''}, {'first': 'H', 'middle': ['G'], 'last': 'Stunnenberg', 'suffix': ''}, {'first': 'L', 'middle': ['A'], 'last': 'O&apos;neill', 'suffix': ''}, {'first': 'R', 'middle': ['J'], 'last': 'Xavier', 'suffix': ''}], 'year': None, 'venue': 'Science', 'volume': '352', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF43': {'ref_id': 'b43', 'title': 'Trained immunity: a memory for innate host defense', 'authors': [{'first': 'M', 'middle': ['G'], 'last': 'Netea', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Quintin', 'suffix': ''}, {'first': 'J', 'middle': ['W'], 'last': 'Van Der Meer', 'suffix': ''}], 'year': 2011, 'venue': 'Cell Host Microbe', 'volume': '9', 'issn': '', 'pages': '355--361', 'other_ids': {}}, 'BIBREF44': {'ref_id': 'b44', 'title': 'Engineered G-protein Coupled Receptors are Powerful Tools to Investigate Biological Processes and Behaviors', 'authors': [{'first': 'C', 'middle': ['D'], 'last': 'Nichols', 'suffix': ''}, {'first': 'B', 'middle': ['L'], 'last': 'Roth', 'suffix': ''}], 'year': 2009, 'venue': 'Front Mol Neurosci', 'volume': '2', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF45': {'ref_id': 'b45', 'title': 'The Ca2+ release-activated Ca2+ current (I(CRAC)) mediates store-operated Ca2+ entry in rat microglia', 'authors': [{'first': 'L', 'middle': [], 'last': 'Ohana', 'suffix': ''}, {'first': 'E', 'middle': ['W'], 'last': 'Newell', 'suffix': ''}, {'first': 'E', 'middle': ['F'], 'last': 'Stanley', 'suffix': ''}, {'first': 'L', 'middle': ['C'], 'last': 'Schlichter', 'suffix': ''}], 'year': 2009, 'venue': 'Channels (Austin)', 'volume': '3', 'issn': '', 'pages': '129--139', 'other_ids': {}}, 'BIBREF46': {'ref_id': 'b46', 'title': \"The subpopulation of microglia expressing functional muscarinic acetylcholine receptors expands in stroke and Alzheimer's disease\", 'authors': [{'first': 'M', 'middle': [], 'last': 'Pannell', 'suffix': ''}, {'first': 'M', 'middle': ['A'], 'last': 'Meier', 'suffix': ''}, {'first': 'F', 'middle': [], 'last': 'Szulzewsky', 'suffix': ''}, {'first': 'V', 'middle': [], 'last': 'Matyash', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Endres', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Kronenberg', 'suffix': ''}, {'first': 'V', 'middle': [], 'last': 'Prinz', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Waiczies', 'suffix': ''}, {'first': 'S', 'middle': ['A'], 'last': 'Wolf', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Kettenmann', 'suffix': ''}], 'year': 2016, 'venue': 'Brain structure & function', 'volume': '221', 'issn': '', 'pages': '1157--1172', 'other_ids': {}}, 'BIBREF47': {'ref_id': 'b47', 'title': 'Microglia promote learning-dependent synapse formation through brain-derived neurotrophic factor', 'authors': [{'first': 'C', 'middle': ['N'], 'last': 'Parkhurst', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Yang', 'suffix': ''}, {'first': 'I', 'middle': [], 'last': 'Ninan', 'suffix': ''}, {'first': 'J', 'middle': ['N'], 'last': 'Savas', 'suffix': ''}, {'first': 'J', 'middle': ['R'], 'last': 'Yates', 'suffix': ''}, {'first': 'J', 'middle': ['J'], 'last': 'Lafaille', 'suffix': ''}, {'first': 'B', 'middle': ['L'], 'last': 'Hempstead', 'suffix': ''}, {'first': 'D', 'middle': ['R'], 'last': 'Littman', 'suffix': ''}, {'first': 'W', 'middle': ['B'], 'last': 'Gan', 'suffix': ''}], 'year': 2013, 'venue': 'Cell', 'volume': '155', 'issn': '', 'pages': '1596--1609', 'other_ids': {}}, 'BIBREF48': {'ref_id': 'b48', 'title': 'Neural circuitry and immunity', 'authors': [{'first': 'V', 'middle': ['A'], 'last': 'Pavlov', 'suffix': ''}, {'first': 'K', 'middle': ['J'], 'last': 'Tracey', 'suffix': ''}], 'year': 2015, 'venue': 'Immunologic research', 'volume': '63', 'issn': '', 'pages': '38--57', 'other_ids': {}}, 'BIBREF49': {'ref_id': 'b49', 'title': 'Engineered GPCRs as tools to modulate signal transduction', 'authors': [{'first': 'Y', 'middle': [], 'last': 'Pei', 'suffix': ''}, {'first': 'S', 'middle': ['C'], 'last': 'Rogan', 'suffix': ''}, {'first': 'F', 'middle': [], 'last': 'Yan', 'suffix': ''}, {'first': 'B', 'middle': ['L'], 'last': 'Roth', 'suffix': ''}], 'year': 2008, 'venue': 'Physiology (Bethesda)', 'volume': '23', 'issn': '', 'pages': '313--321', 'other_ids': {}}, 'BIBREF50': {'ref_id': 'b50', 'title': 'Expression of interleukin-1 receptors and their role in interleukin-1 actions in murine microglial cells', 'authors': [{'first': 'E', 'middle': [], 'last': 'Pinteaux', 'suffix': ''}, {'first': 'L', 'middle': ['C'], 'last': 'Parker', 'suffix': ''}, {'first': 'N', 'middle': ['J'], 'last': 'Rothwell', 'suffix': ''}, {'first': 'G', 'middle': ['N'], 'last': 'Luheshi', 'suffix': ''}], 'year': 2002, 'venue': 'J Neurochem', 'volume': '83', 'issn': '', 'pages': '754--763', 'other_ids': {}}, 'BIBREF51': {'ref_id': 'b51', 'title': 'Mice deficient for the vesicular acetylcholine transporter are myasthenic and have deficits in object and social recognition', 'authors': [{'first': 'V', 'middle': ['F'], 'last': 'Prado', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Martins-Silva', 'suffix': ''}, {'first': 'B', 'middle': ['M'], 'last': 'De Castro', 'suffix': ''}, {'first': 'R', 'middle': ['F'], 'last': 'Lima', 'suffix': ''}, {'first': 'D', 'middle': ['M'], 'last': 'Barros', 'suffix': ''}, {'first': 'E', 'middle': [], 'last': 'Amaral', 'suffix': ''}, {'first': 'A', 'middle': ['J'], 'last': 'Ramsey', 'suffix': ''}, {'first': 'T', 'middle': ['D'], 'last': 'Sotnikova', 'suffix': ''}, {'first': 'M', 'middle': ['R'], 'last': 'Ramirez', 'suffix': ''}, {'first': 'H', 'middle': ['G'], 'last': 'Kim', 'suffix': ''}, {'first': 'J', 'middle': ['I'], 'last': 'Rossato', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Koenen', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Quan', 'suffix': ''}, {'first': 'V', 'middle': ['R'], 'last': 'Cota', 'suffix': ''}, {'first': 'M', 'middle': ['F'], 'last': 'Moraes', 'suffix': ''}, {'first': 'M', 'middle': ['V'], 'last': 'Gomez', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Guatimosim', 'suffix': ''}, {'first': 'W', 'middle': ['C'], 'last': 'Wetsel', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Kushmerick', 'suffix': ''}, {'first': 'G', 'middle': ['S'], 'last': 'Pereira', 'suffix': ''}, {'first': 'R', 'middle': ['R'], 'last': 'Gainetdinov', 'suffix': ''}, {'first': 'I', 'middle': [], 'last': 'Izquierdo', 'suffix': ''}, {'first': 'M', 'middle': ['G'], 'last': 'Caron', 'suffix': ''}, {'first': 'M', 'middle': ['A'], 'last': 'Prado', 'suffix': ''}], 'year': 2006, 'venue': 'Neuron', 'volume': '51', 'issn': '', 'pages': '601--612', 'other_ids': {}}, 'BIBREF52': {'ref_id': 'b52', 'title': 'Store-Operated Calcium Channels', 'authors': [{'first': 'M', 'middle': [], 'last': 'Prakriya', 'suffix': ''}, {'first': 'R', 'middle': ['S'], 'last': 'Lewis', 'suffix': ''}], 'year': 2015, 'venue': 'Physiol Rev', 'volume': '95', 'issn': '', 'pages': '1383--1436', 'other_ids': {}}, 'BIBREF53': {'ref_id': 'b53', 'title': 'Potentiation of HIV-1 expression in microglial cells by nicotine: involvement of transforming growth factor-beta 1', 'authors': [{'first': 'R', 'middle': ['B'], 'last': 'Rock', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Gekker', 'suffix': ''}, {'first': 'R', 'middle': ['N'], 'last': 'Aravalli', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Hu', 'suffix': ''}, {'first': 'W', 'middle': ['S'], 'last': 'Sheng', 'suffix': ''}, {'first': 'P', 'middle': ['K'], 'last': 'Peterson', 'suffix': ''}], 'year': 2008, 'venue': 'Journal of neuroimmune pharmacology : the official journal of the Society on NeuroImmune Pharmacology', 'volume': '3', 'issn': '', 'pages': '143--149', 'other_ids': {}}, 'BIBREF54': {'ref_id': 'b54', 'title': 'Cholinergic modulation of microglial activation by alpha 7 nicotinic receptors', 'authors': [{'first': 'R', 'middle': ['D'], 'last': 'Shytle', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Mori', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Townsend', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Vendrame', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Sun', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Zeng', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Ehrhart', 'suffix': ''}, {'first': 'A', 'middle': ['A'], 'last': 'Silver', 'suffix': ''}, {'first': 'P', 'middle': ['R'], 'last': 'Sanberg', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Tan', 'suffix': ''}], 'year': 2004, 'venue': 'Journal of neurochemistry', 'volume': '89', 'issn': '', 'pages': '337--343', 'other_ids': {}}, 'BIBREF55': {'ref_id': 'b55', 'title': 'Microglial alpha7 nicotinic acetylcholine receptors drive a phospholipase C/IP3 pathway and modulate the cell activation toward a neuroprotective role', 'authors': [{'first': 'T', 'middle': [], 'last': 'Suzuki', 'suffix': ''}, {'first': 'I', 'middle': [], 'last': 'Hide', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Matsubara', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Hama', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Harada', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Miyano', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Andra', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Matsubayashi', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Sakai', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Kohsaka', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Inoue', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Nakata', 'suffix': ''}], 'year': 2006, 'venue': 'Journal of neuroscience research', 'volume': '83', 'issn': '', 'pages': '1461--1470', 'other_ids': {}}, 'BIBREF56': {'ref_id': 'b56', 'title': 'Neuroimmune Activation Drives Multiple Brain States', 'authors': [{'first': 'D', 'middle': [], 'last': 'Tchessalova', 'suffix': ''}, {'first': 'C', 'middle': ['K'], 'last': 'Posillico', 'suffix': ''}, {'first': 'N', 'middle': ['C'], 'last': 'Tronson', 'suffix': ''}], 'year': 2018, 'venue': 'Front Syst Neurosci', 'volume': '12', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF57': {'ref_id': 'b57', 'title': 'Long-term activation of capacitative Ca2+ entry in mouse microglial cells', 'authors': [{'first': 'E', 'middle': ['C'], 'last': 'Toescu', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'M√∂ller', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Kettenmann', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Verkhratsky', 'suffix': ''}], 'year': 1998, 'venue': 'Neuroscience', 'volume': '86', 'issn': '', 'pages': '925--935', 'other_ids': {}}, 'BIBREF58': {'ref_id': 'b58', 'title': 'In Vivo Imaging of Microglial Calcium Signaling in Brain Inflammation and Injury', 'authors': [{'first': 'P', 'middle': [], 'last': 'Tvrdik', 'suffix': ''}, {'first': 'M', 'middle': ['Y S'], 'last': 'Kalani', 'suffix': ''}], 'year': 2017, 'venue': 'Int J Mol Sci', 'volume': '18', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF59': {'ref_id': 'b59', 'title': 'DREADDs (designer receptors exclusively activated by designer drugs): chemogenetic tools with therapeutic utility', 'authors': [{'first': 'D', 'middle': ['J'], 'last': 'Urban', 'suffix': ''}, {'first': 'B', 'middle': ['L'], 'last': 'Roth', 'suffix': ''}], 'year': 2015, 'venue': 'Annual review of pharmacology and toxicology', 'volume': '55', 'issn': '', 'pages': '399--417', 'other_ids': {}}, 'BIBREF60': {'ref_id': 'b60', 'title': 'Functions of microglia in the central nervous system--beyond the immune response', 'authors': [{'first': 'H', 'middle': [], 'last': 'Wake', 'suffix': ''}, {'first': 'A', 'middle': ['J'], 'last': 'Moorhouse', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Nabekura', 'suffix': ''}], 'year': 2011, 'venue': 'Neuron Glia Biol', 'volume': '7', 'issn': '', 'pages': '47--53', 'other_ids': {}}, 'BIBREF61': {'ref_id': 'b61', 'title': 'Innate immune memory in the brain shapes neurological disease hallmarks', 'authors': [{'first': 'A', 'middle': ['C'], 'last': 'Wendeln', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Degenhardt', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Kaurani', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Gertig', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Ulas', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Jain', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Wagner', 'suffix': ''}, {'first': 'L', 'middle': ['M'], 'last': 'Häsler', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Wild', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Skodras', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Blank', 'suffix': ''}, {'first': 'O', 'middle': [], 'last': 'Staszewski', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Datta', 'suffix': ''}, {'first': 'T', 'middle': ['P'], 'last': 'Centeno', 'suffix': ''}, {'first': 'V', 'middle': [], 'last': 'Capece', 'suffix': ''}, {'first': 'M', 'middle': ['R'], 'last': 'Islam', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Kerimoglu', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Staufenbiel', 'suffix': ''}, {'first': 'J', 'middle': ['L'], 'last': 'Schultze', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Beyer', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Prinz', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Jucker', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Fischer', 'suffix': ''}, {'first': 'J', 'middle': ['J'], 'last': 'Neher', 'suffix': ''}], 'year': 2018, 'venue': 'Nature', 'volume': '556', 'issn': '', 'pages': '332--338', 'other_ids': {}}, 'BIBREF62': {'ref_id': 'b62', 'title': 'Microglia in vivo and in vitro', 'authors': [{'first': 'R', 'middle': [], 'last': 'Yamasaki', 'suffix': ''}], 'year': 2014, 'venue': 'Clinical and Experimental Neuroimmunology', 'volume': '5', 'issn': '', 'pages': '114--116', 'other_ids': {}}, 'BIBREF63': {'ref_id': 'b63', 'title': 'Cholinergic agonists increase intracellular Ca2+ in cultured human microglia', 'authors': [{'first': 'L', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'J', 'middle': ['G'], 'last': 'Mclarnon', 'suffix': ''}, {'first': 'V', 'middle': [], 'last': 'Goghari', 'suffix': ''}, {'first': 'Y', 'middle': ['B'], 'last': 'Lee', 'suffix': ''}, {'first': 'S', 'middle': ['U'], 'last': 'Kim', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Krieger', 'suffix': ''}], 'year': 1998, 'venue': 'Neuroscience letters', 'volume': '255', 'issn': '', 'pages': '33--36', 'other_ids': {}}, 'BIBREF64': {'ref_id': 'b64', 'title': 'Cre-dependent DREADD (Designer Receptors Exclusively Activated by Designer Drugs) mice', 'authors': [{'first': 'H', 'middle': [], 'last': 'Zhu', 'suffix': ''}, {'first': 'D', 'middle': ['K'], 'last': 'Aryal', 'suffix': ''}, {'first': 'R', 'middle': ['H'], 'last': 'Olsen', 'suffix': ''}, {'first': 'D', 'middle': ['J'], 'last': 'Urban', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Swearingen', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Forbes', 'suffix': ''}, {'first': 'B', 'middle': ['L'], 'last': 'Roth', 'suffix': ''}, {'first': 'U', 'middle': [], 'last': 'Hochgeschwender', 'suffix': ''}], 'year': 2016, 'venue': 'Genesis', 'volume': '54', 'issn': '', 'pages': '439--446', 'other_ids': {}}, 'BIBREF65': {'ref_id': 'b65', 'title': 'Calcium-dependent potentiation of store-operated calcium channels in T lymphocytes', 'authors': [{'first': 'A', 'middle': [], 'last': 'Zweifach', 'suffix': ''}, {'first': 'R', 'middle': ['S'], 'last': 'Lewis', 'suffix': ''}], 'year': 1996, 'venue': 'J Gen Physiol', 'volume': '107', 'issn': '', 'pages': '597--610', 'other_ids': {}}}, 'ref_entries': {'FIGREF0': {'text': 'al., 2017; Martins-Silva et al., 2011) with some modification. VersaMax Tracking System (X-Y and Z sensors) automatically recorded activity of mice in an open field box (40.64×40.64 cm2) (Omnitech Electronics Inc., Columbus, USA) that was separated into 4 equal quadrants using acrylic plexiglass. Recordings were done without an investigator present in the room. Two mice were placed in each open field box, one in the lower left corner and one in the top right corner of the box. The task was completed on two consecutive days with the exact same criteria. Locomotor activity is expressed as the total distance (cm) that mice travelled during the trial, while time in center is measured as the total time (seconds) that mice spent within the inner (8×8 cm2) region of their quadrant. All open field data (locomotor activity and time in center) are expressed as a sum of the first 60 min of the trial. For all open field experiments assessing baseline activity, a repeated measures two-way ANOVA with author/funder. All rights reserved. No reuse allowed without permission.', 'latex': None, 'type': 'figure'}, 'FIGREF1': {'text': \"evoked inward current in hM3Dq-expressing microglia was abolished in the presence of the SOCC inhibitor SKF-96365 (20 µM, Fig. 2G). To better demonstrate the current potentiated by CNO (p=0.0361, Tukey's multiple comparisons test) and inhibited by SKF (p=0.0024), Fig. 2H shows the current at -80 mV of Fig 2G. Subtracting the control current from the CNO-evoked currentdisplayed an inwardly rectifying current(Fig. 2G, inset) that is characteristic of store-operated Ca2+ release-activated Ca2+ (CRAC) currents(Ohana et al., 2009).\", 'latex': None, 'type': 'figure'}, 'FIGREF2': {'text': \"Fig. 2I-K). Treatment with CNO increased the number of FluoSpheres per hM3Dq-positive cell (Fig. 1J; t(18)=4.586, p=0.0002, Student's t-test) as well as the percentage of hM3Dq-positive cells containing FluoSpheres (Fig. 2K; t(18)=5.133, p<0.0001, Student's t-test) when compared to vehicle treatment. Taken together, these experiments show that microglia from CX3CR1CreER(+/-)-hM3Dq(+/-) mice express hM3Dq and that activation of this receptor by CNO triggers intracellular calcium increase and changes in microglial function.author/funder. All rights reserved. No reuse allowed without permission.\", 'latex': None, 'type': 'figure'}, 'FIGREF3': {'text': \"administration of CNO under non-inflammatory conditions (no LPS injection groups) increased mRNA levels of TNF-α (Fig. 3B; two-way ANOVA main effect of CNO: F(1,26)=101.5, p<0.0001, post-hoc Tukey's analysis p<0.0001), IL-1β (Fig. 3C; two-way ANOVA main effect of CNO: F(1,26)=17.94, p<0.0001, post-hoc Tukey's analysis p<0.0001) and IL-6 (Fig. 3D; , two-way ANOVA main effect of CNO: F(1,26)= 44.61, p<0.0001, post-hoc Tukey's analysis p<0.0001). Although LPS treatment increased cytokine mRNA levels, a prior activation of hM3Dq by CNO (30 min before LPS injection, Fig. 3A) did not alter mRNA levels of TNF-α, IL-1β and IL-6 (Fig. 3B; TNF-α: two-way ANOVA LPS: F(1,26)= 44.22, p<0.0001, post-hoc Tukey's analysis p=0.21; Fig. 3C; IL-1β: LPS: F(1,26)= 50.97, p<0.0001, post-hoc Tukey's analysis p=0.96; and Fig. 3D; IL-6: two-way ANOVA LPS: F(1,26)= 69.62, p<0.0001, post-hoc Tukey's analysis p=0.54), possibly due to ceiling effects.\", 'latex': None, 'type': 'figure'}, 'FIGREF4': {'text': \"Investigation of spontaneous activity in the open field using automated locomotor boxes showed no significant difference between CX3CR1CreER(+/-)-hM3Dq(+/-) mice treated with CNO or vehicle on the distance travelled in 60 min, both on day 1 and day 2, indicating that hM3Dq activation does not interfere with locomotor activity (Fig. 4B; no effect of CNO on distance travelled (F(1,28)=3.06, p=0.995); no interaction between CNO treatment and time (F(1,28)= 0.237, p=0.630). Additionally, both CNO-and vehicle-treated CX3CR1CreER(+/-)-hM3Dq(+/-) mice showed the expected decrease in locomotor activity on the 2nd test day due to habituation (Fig. 4B; significant effects of day on the total distance travelled; F(1,28)=5.947, p=0.0213, two-way ANOVA repeated measures). We tested for changes in anxiety-like behavior using different paradigms. Time spent in the center vs. the periphery of the open field (Fig. 4C) was not affected by activation of hM3Dq (No main effect of CNO treatment: F(1,28)=1.045, p=0.3153; No treatment x day interaction: F(1,28)=1.031, p=0.3187, two-way ANOVA repeated measures). Likewise, we found that activation of hM3Dq with CNO did not affect the time mice spent exploring a novel unprotected environment in the light-dark box (light compartment) (Fig. 4D; t(15)=0.14, p=0.8907, Student's t-test) or the elevated plus maze (open arms) (Fig. 4E; t(15)=0.0075, p=0.9941, Student's t-test). Additionally, we did not detect any effect of hM3Dq activation in the forced swim test (Fig. 4F; t(15)=0.48, p=0.6391, Student's t-test).\", 'latex': None, 'type': 'figure'}, 'FIGREF5': {'text': 'Fig. 4G; t(39)=0.93, p=0.359,. To investigate social memory, we tested mice on a habituation-dishabituation paradigm using a mouse intruder. CNO-and vehicle-treated CX3CR1CreER(+/-)-hM3Dq(+/-) mice showed significantly more exploration of the intruder during first contact and decreased exploration time with subsequent exposure to the same juvenile, suggesting that both cohorts could equally remember the initial intruder. Upon changing to an unfamiliar mouse (fifth trial), both CNO-and vehicle-treated-mice explored the new intruder as much as they explored the original intruder during the first contact, confirming their preference for social novelty (Fig. 4H; No effect of CNO treatment: F(1,28)=0.399, p=0.533; a significant effect of trial on time of interaction: F(4,112)=21.84, p<0.0001, two-way ANOVA repeated measures). A similar paradigm was used to investigate object and olfactory memory and, likewise, no difference between CNO-and vehicle-treated CX3CR1CreER(+/-)-hM3Dq(+/-) mice was observed (Fig. 4I; object recognition memory: two-way ANOVA repeated measures no main effect of CNO treatment: F(1,13)=0.1905, p=0.6697; no treatment x trial interaction: F(4,52)= 0.8272, p=0.5139; Fig. 4J; olfactory memory: no main effect of CNO treatment: F(1,14)=0.004324, p=0.9485; no treatment x trial interaction:', 'latex': None, 'type': 'figure'}, 'FIGREF6': {'text': \". 5B, two-way ANOVA no main effect of CNO: F(1,37)=3.86, p=0.057; post-hoc Tukey's analysis; IL-1β: Fig. 5C, two-way ANOVA no main effect of CNO: F(1,37)=2.69, p=0.11; post-hoc Tukey's analysis; and IL-6: Fig. 5D, two-way ANOVA no main effect of CNO: F(1,38)= 1.89, p=0.18; post-hoc Tukey's analysis). Remarkably, in LPS-treated mice, chronic activation of hM3Dq with CNO significantly attenuated LPS-induced mRNA levels of TNF-α (Fig. 5B; two-way ANOVA main effect of LPS: F(1,37)=18.14, p=0.0001; post-hoc Tukey's analysis Vehicle-LPS x CNO-LPS p=0.02), IL-1β (Fig. 5C; two-way ANOVA main effect of LPS: F(1,37)=74.53, p<0001; post-hoc Tukey's analysis Vehicle-LPS x CNO-LPS p=0.02), and IL-6 (Fig. 5D; two-way ANOVA main effect of LPS: F(1,38)=46.67, p<0.0001; post-hoc Tukey's analysis Vehicle-LPS x CNO-LPS p=0.047). In littermate control\", 'latex': None, 'type': 'figure'}, 'FIGREF7': {'text': 'injection (Fig. 6C; main effect of CNO treatment: F(1,20)=8.946, p=0.0072, main effect of time time: F(1,20)=5.46, p=0.029, interaction: F(1,20)=1.318, p= 0.2646, two-way ANOVA). Comparison of social interaction data from Figures 4G and 6C shows that vehicle-treated CX3CR1CreER(+/-)-hM3Dq(+/-) mice that did not receive LPS explored the juvenile mice much longer (319.8 \\uf0b118.37; Fig 4G; N=20) than vehicle-treated mice injected with LPS (193.72 \\uf0b1 39.02; Fig. 6C; N=11),', 'latex': None, 'type': 'figure'}, 'FIGREF8': {'text': 'hM3Dq is expressed selectively in Iba1-positive microglia in CX3CR1CreER(+/-)-hM3Dq(+/-) mouse brain. A) Schematic of the targeting strategy used to obtain tamoxifeninducible Cre-dependent expression of hM3Dq selectively in microglia. B) Representative confocal images of hippocampus of tamoxifen-treated CX3CR1CreER-hM3Dq(+/-) mice showing distribution of HA (red) and Iba1, NeuN or GFAP (green). Right panels show images of left panels merged with Hoechst nuclear stain (blue). Arrows indicate examples of HA-positive cells.', 'latex': None, 'type': 'figure'}, 'FIGREF9': {'text': 'Microglia from CX3CR1CreER-hM3Dq(+/-) mice express functional hM3Dq upon tamoxifen treatment. A) Experimental approach and timeline of in vitro experiments. B) Representative images of immunostaining for Iba1 (green) and HA (red) in microglia culture treated with 4-OHT. C-D) Kinetics of relative Rhod-2 fluorescence in cells in response to 3 µM CNO, 1 mM ATP, and 2 µM ionomycin (iono) or 10 µM thapsigargin (thaps) in 4-OHT-treated cultures in KRH buffer (C) or calcium-free KRH buffer (D). E-F) Kinetics of relative Rhod-2 fluorescence in cells in non-treated cultures in KRH buffer (E) or calcium-free KRH buffer (F). G)', 'latex': None, 'type': 'figure'}, 'FIGREF10': {'text': 'Microglial hM3Dq activation by CNO does not cause overt behavioral change. A)Timeline of CNO treatment and behavior assessments. CNO (1.0 mg/kg; i.p.) or vehicle wasinjected 30 min before each behavioral task. B-C) Total distance travelled (B) and time spent in the center (C) of an open field on two consecutive days. D) Time spent in the light side of a lightdark box. E) Time spent in the open arm of an elevated plus maze (EPM). F) Time spent immobile during minutes 2-6 in forced swim test (FST). G) Time spent investigating a juvenile mouse in a three-chamber social preference task (Sociability). H) Time spent interacting with an intruder mouse for social memory assessment. 1 trial = 5 min. I) Time spent investigating a novel object to author/funder. All rights reserved. No reuse allowed without permission.', 'latex': None, 'type': 'figure'}, 'FIGREF11': {'text': 'Chronic microglial hM3Dq activation by CNO attenuates LPS-induced increase of pro-inflammatory cytokines in the brain. A) Experimental approach and timeline for hippocampus gene expression experiments. CNO (1 mg/kg; i.p.); LPS (0.1 mg/kg; i.p.). B-D) qPCR measurements of mRNA expression of TNF-α (B), IL-1β (C), and IL-6 (D) in hippocampus of', 'latex': None, 'type': 'figure'}}, 'back_matter': []}\n",
      "{'paper_id': 'af266fac8970a7960e96630a67d91bec5dda0335', 'metadata': {'title': 'Estimating the generation interval for COVID-19 based on symptom onset data', 'authors': [{'first': 'Ganyani', 'middle': [], 'last': 'Tapiwa', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Hasselt University', 'location': {'settlement': 'Hasselt', 'country': 'Belgium'}}, 'email': ''}, {'first': 'Kremer', 'middle': [], 'last': 'Cécile', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Hasselt University', 'location': {'settlement': 'Hasselt', 'country': 'Belgium'}}, 'email': ''}, {'first': 'Chen', 'middle': [], 'last': 'Dongxuan', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'National Institute for Public Health and the Environment', 'location': {'settlement': 'Bilthoven', 'country': 'The Netherlands'}}, 'email': ''}, {'first': 'Torneri', 'middle': [], 'last': 'Andrea', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Hasselt University', 'location': {'settlement': 'Hasselt', 'country': 'Belgium'}}, 'email': ''}, {'first': 'Faes', 'middle': [], 'last': 'Christel', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Hasselt University', 'location': {'settlement': 'Hasselt', 'country': 'Belgium'}}, 'email': ''}, {'first': 'Wallinga', 'middle': [], 'last': 'Jacco', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'National Institute for Public Health and the Environment', 'location': {'settlement': 'Bilthoven', 'country': 'The Netherlands'}}, 'email': ''}, {'first': 'Hens', 'middle': [], 'last': 'Niel', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Hasselt University', 'location': {'settlement': 'Hasselt', 'country': 'Belgium'}}, 'email': ''}]}, 'abstract': [{'text': 'Estimating key infectious disease parameters from the COVID-19 outbreak is quintessential for modelling studies and guiding intervention strategies.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'Whereas different estimates for the incubation period distribution and the serial interval distribution have been reported, estimates of the generation interval for COVID-19 have not been provided.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'Methods: We used outbreak data from clusters in Singapore and Tianjin, China to estimate the generation interval from symptom onset data while acknowledging uncertainty about the incubation period distribution and the underlying transmission network. From those estimates we obtained the proportions pre-symptomatic transmission and reproduction numbers.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'Results: The mean generation interval was 5.20 (95%CI 3.78-6.78) days for Singapore and 3.95 (95%CI 3.01-4.91) days for Tianjin, China when relying on a previously reported incubation period with mean 5.2 and SD 2.8 days. The proportion of pre-symptomatic transmission was 48% (95%CI 32-67%) for Singapore and 62% (95%CI 50-76%) for Tianjin, China. Estimates of the reproduction number based on the generation interval distribution were slightly higher than those based on the serial interval distribution.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'Conclusions: Estimating generation and serial interval distributions from outbreak 1 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint .', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}], 'body_text': [{'text': 'In order to plan intervention strategies aimed at bringing disease outbreaks such as the COVID-19 outbreak under control as well as to monitor disease outbreaks, public health officials depend on insights about key disease transmission parameters which are typically obtained from mathematical or statistical modelling. Examples of key parameters include the reproduction number (average number of infections caused by an infectious individual) and distributions of the generation interval (time between infection events in an infector-infectee pair), serial interval (time between symptom onsets in an infector-infectee pair), and incubation period (time between moment of infection and symptom onset) [1] . Estimates of the reproduction number together with the generation interval distribution can provide insight into the speed with which a disease will spread.', 'cite_spans': [{'start': 703, 'end': 706, 'text': '[1]', 'ref_id': 'BIBREF0'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'On the other hand, estimates of the incubation period distribution can help guide determining appropriate quarantine periods.', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'As soon as line lists were made available, statistical and mathematical modelling was used to quantify these key epidemiological parameters. Li et al. [2] estimated the basic reproduction number (using a renewal equation) to be 2.2 (95% CI 1.4-3.9), the serial interval distribution to have a mean of 7.5 days (95% CI 5.5-19) based on 6 observations, and the incubation period distribution to have a mean of 5.2 days (95% CI 4.1-7.0) based on 10 observations. Other studies estimated the incubation period distribution to have a mean of 6.4 days (95% CI 5.6-7.7) [3] , median of 5 days (95% CI 4.0-5.8) [4] , mean of 5.2 days (range 1.8-12.4 days) [5] , and a mean of 4.8 days (range 2-11 days) [6] .', 'cite_spans': [{'start': 151, 'end': 154, 'text': '[2]', 'ref_id': 'BIBREF1'}, {'start': 563, 'end': 566, 'text': '[3]', 'ref_id': 'BIBREF2'}, {'start': 603, 'end': 606, 'text': '[4]', 'ref_id': 'BIBREF4'}, {'start': 648, 'end': 651, 'text': '[5]', 'ref_id': 'BIBREF5'}, {'start': 695, 'end': 698, 'text': '[6]', 'ref_id': 'BIBREF6'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'When the incubation period does not change over the course of the epidemic, the expected values of the serial and generation interval distributions are expected to be equal but their variances to be different [7] . It has recently been shown that ignoring the difference between the serial and generation interval can lead to biased estimates of the reproduction number. More specifically, when the serial interval distribution has larger variance than the generation interval distribution, using the serial interval as a proxy for 2 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 209, 'end': 212, 'text': '[7]', 'ref_id': 'BIBREF7'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.05.20031815 doi: medRxiv preprint the generation interval will lead to an underestimation of the basic reproduction number R. When R is underestimated, this may lead to prevention policies that are insufficient to stop disease spread [7] .', 'cite_spans': [{'start': 344, 'end': 347, 'text': '[7]', 'ref_id': 'BIBREF7'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'The most well-known method to estimate the serial interval distribution from line list data is the likelihood-based estimation method proposed by Wallinga and Teunis [8] . In 2012, Hens et al. [9] proposed using the Expectation-Maximisation (EM) algorithm to estimate the generation interval distribution from incomplete line list data based on the method by [8] and allowing for auxiliary information to be used in assigning potential infector-infectee pairs. Te Beest et al [10] used a Markov chain Monte Carlo (MCMC) approach as an alternative to the EM-algorithm, to facilitate taking into account uncertainty related to the dates of symptom onset. In this paper, we use an MCMC approach to estimate, next to the serial interval distribution, the generation interval distribution upon specification of the incubation period distribution. We compare the impact of differences amongst previous estimates of the incubation period distribution for COVID-19 and analyse data on clusters of confirmed cases from Singapore (January 21 to February 26) and Tianjin, China (January 14 to February 27).', 'cite_spans': [{'start': 166, 'end': 169, 'text': '[8]', 'ref_id': 'BIBREF8'}, {'start': 193, 'end': 196, 'text': '[9]', 'ref_id': 'BIBREF9'}, {'start': 359, 'end': 362, 'text': '[8]', 'ref_id': 'BIBREF8'}, {'start': 476, 'end': 480, 'text': '[10]', 'ref_id': 'BIBREF10'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'The data used in this paper consist of symptom onset dates and cluster information for confirmed cases in Singapore and Tianjin, China.', 'cite_spans': [], 'ref_spans': [], 'section': 'Data'}, {'text': 'As of February 26th, 91 confirmed COVID-19 cases had been reported in Singapore.', 'cite_spans': [], 'ref_spans': [], 'section': 'Data'}, {'text': 'Detailed information on age, sex, known travel history, time of symptom onset, and known contacts is available for 54 of these cases (link: https://www.moh.gov.sg/newshighlights/, last accessed February 26th). For cases with no infector information available, it is assumed that they could have been infected by any other case within the same cluster. Cases known to be Chinese/Wuhan nationals or known to have been in close contact with a Chinese/Wuhan national are labeled as index cases. All other cases are assumed to have been infected locally.', 'cite_spans': [], 'ref_spans': [], 'section': 'Data'}, {'text': 'As of February 27th, 135 confirmed cases had been reported by the Tianjin municipal 3 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Data'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.05.20031815 doi: medRxiv preprint health commission. Data on these cases are available in official daily reports (link: http://wsjk.tj.gov.cn/col/col87/index.html, last accessed February 27th) and include age, sex, relationship to other known cases, and travel history to risk areas in and outside Hubei province, China. In these data, 114 cases can be traced to one of 16 clusters. The largest cluster consisting of 45 cases can be traced to a shopping mall in Baodi district.', 'cite_spans': [], 'ref_spans': [], 'section': 'Data'}, {'text': 'Through contact investigations, potential transmission links were identified for cases who had close contacts. Travel history information was used to identify some individuals as import cases. For cases with no infector information available, it is assumed that they could have been infected by any other case within the same cluster. ', 'cite_spans': [], 'ref_spans': [], 'section': 'Data'}, {'text': 'Assuming the incubation period is independent of the infection time, Z i can be rewritten as a convolution of the generation interval for individual i and the difference between the incubation period of individual i and the incubation period of its infector v(i) [7] , i.e.,', 'cite_spans': [{'start': 263, 'end': 266, 'text': '[7]', 'ref_id': 'BIBREF7'}], 'ref_spans': [], 'section': 'Model'}, {'text': 'The random variables X i and δ i are positive and are both assumed to be independent and identically distributed, i.e., X i ∼ f (x; Θ 1 ) and δ i ∼ k(δ; Θ 2 ), so that Y i ∼ g(y i ; Θ 2 ).', 'cite_spans': [], 'ref_spans': [], 'section': 'Model'}, {'text': 'have the same mean and that the latter has a larger variance and can be negative.', 'cite_spans': [], 'ref_spans': [], 'section': 'Equation (1) implies that both the generation interval and serial interval distributions'}, {'text': 'The observed serial interval, z i , can be expressed in terms of the latent variables as', 'cite_spans': [], 'ref_spans': [], 'section': 'Equation (1) implies that both the generation interval and serial interval distributions'}, {'text': '. The density function h(.) is given by [11] ,', 'cite_spans': [{'start': 40, 'end': 44, 'text': '[11]', 'ref_id': 'BIBREF11'}], 'ref_spans': [], 'section': 'Equation (1) implies that both the generation interval and serial interval distributions'}, {'text': 'In general, h(z; Θ 1 , Θ 2 ) and g(y; Θ 2 ) have no closed form for arbitrary choices of f (x; Θ 1 ) 4 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Equation (1) implies that both the generation interval and serial interval distributions'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.05.20031815 doi: medRxiv preprint and k(δ; Θ 2 ). Monte Carlo methods [12] can be used to estimate h(z; Θ 1 , Θ 2 ) as follows,', 'cite_spans': [{'start': 180, 'end': 184, 'text': '[12]', 'ref_id': 'BIBREF13'}], 'ref_spans': [], 'section': 'Equation (1) implies that both the generation interval and serial interval distributions'}, {'text': 'where y j is the j th Monte Carlo sample drawn from g(y; Θ 2 ). When all infector-infectee pairs are observed, the likelihood function is given by,', 'cite_spans': [], 'ref_spans': [], 'section': 'Equation (1) implies that both the generation interval and serial interval distributions'}, {'text': 'To account for uncertainty in the transmission links we resort to a Bayesian framework in which missing links are imputed [10] (see Subsection 2.3). The likelihood function is then given by L Θ, v(i) missing |z i , v(i) .', 'cite_spans': [{'start': 122, 'end': 126, 'text': '[10]', 'ref_id': 'BIBREF10'}], 'ref_spans': [], 'section': 'Equation (1) implies that both the generation interval and serial interval distributions'}, {'text': 'We use the Bayesian method described in te Beest et al. [10] for parameter estimation.', 'cite_spans': [{'start': 56, 'end': 60, 'text': '[10]', 'ref_id': 'BIBREF10'}], 'ref_spans': [], 'section': 'Estimation'}, {'text': 'This method proceeds in two steps. The first step updates the missing links v(i) missing and the second step updates the parameter vector Θ 1 , i.e., the parameters of the generation interval distribution. We assume that both the generation interval and the incubation period are gamma distributed, i.e., f (x; Θ 1 ) ≡ Γ(α 1 , β 1 ) and k(δ; Θ 2 ) ≡ Γ(α 2 , β 2 ).', 'cite_spans': [], 'ref_spans': [], 'section': 'Estimation'}, {'text': 'The parameter vector Θ 2 is fixed to (α 2 = 3.45, β 2 = 0.66), corresponding to an incubation period with a mean of 5.2 and standard deviation (SD) of 2.8 days [5] .', 'cite_spans': [{'start': 160, 'end': 163, 'text': '[5]', 'ref_id': 'BIBREF5'}], 'ref_spans': [], 'section': 'Estimation'}, {'text': 'Minimally informative uniform priors are assigned to the parameters of the generation interval distribution, i.e., α 1 ∼ U (0, 30) and β 1 ∼ U (0, 20). For cases with multiple potential infectors, the possible links v(i) missing are assigned equal prior probabilities. The missing links are updated using an independence sampler, whereas Θ 1 is updated using a random-walk Metropolis-Hastings algorithm with a uniform proposal distribution [12] .', 'cite_spans': [{'start': 440, 'end': 444, 'text': '[12]', 'ref_id': 'BIBREF13'}], 'ref_spans': [], 'section': 'Estimation'}, {'text': 'We evaluate the posterior distribution using 3 000 000 iterations of which the first 500 000 are discarded as burn-in. Thinning is applied by taking every 200 th iteration. The serial interval distribution is obtained by simulating 1 000 000 draws from h(z;Θ 1 ,Θ 2 ).', 'cite_spans': [], 'ref_spans': [], 'section': 'Estimation'}, {'text': 'All analyses were performed using R, datasets and code are available on GitHub (https://github.com/cecilekremer/COVID19).', 'cite_spans': [], 'ref_spans': [], 'section': 'Estimation'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '5'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.05.20031815 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': '5'}, {'text': 'The proportion of pre-symptomatic transmission is calculated as p = P (X i < δ v(i) ), i.e., pre-symptomatic transmission occurs when the generation interval is shorter than the incubation period of the infector. This proportion was obtained by simulating values from the estimated generation interval and incubation period distributions (assuming a mean incubation time of 5.2 days [5] ).', 'cite_spans': [{'start': 383, 'end': 386, 'text': '[5]', 'ref_id': 'BIBREF5'}], 'ref_spans': [], 'section': 'Implications'}, {'text': 'The reproduction number is calculated as R = e rµ− 1 2 r 2 σ 2 , where r denotes the exponential growth rate estimated from the initial phase of the outbreak, and µ and σ 2 are the mean and variance of the generation or serial interval distribution [13] .', 'cite_spans': [{'start': 249, 'end': 253, 'text': '[13]', 'ref_id': 'BIBREF14'}], 'ref_spans': [], 'section': 'Implications'}, {'text': 'Confidence intervals for p and R are calculated by evaluating p and R at each iteration of the converged MCMC chain, i.e., at each mean/variance pair of the posterior generation/serial interval distribution. The 95% confidence intervals are given by the 2.5% and 97.5% quantiles of the resulting distributions.', 'cite_spans': [], 'ref_spans': [], 'section': 'Implications'}, {'text': 'As sensitivity analyses, we investigate the robustness of our estimates of the generation interval distribution to the choice of different incubation period distributions. In particular, we fix Θ 2 to (α 2 = 7.74, β 2 = 1.21) and (α 2 = 4.36, β 2 = 0.91), corresponding to an incubation period with mean and SD (6.4, 2.3) days [3] and (4.8, 2.6) days [6] , respectively.', 'cite_spans': [{'start': 327, 'end': 330, 'text': '[3]', 'ref_id': 'BIBREF2'}, {'start': 351, 'end': 354, 'text': '[6]', 'ref_id': 'BIBREF6'}], 'ref_spans': [], 'section': 'Sensitivity analyses'}, {'text': 'In our main (i.e., baseline) analyses, missing serial intervals were only allowed to be positive, i.e., the symptom onset time of the infector has to occur before that of the infectee. However, given that pre-symptomatic transmission is possible, this can be deemed an unrealistic assumption. Therefore, we assess the impact of allowing for negative serial intervals on our estimates of the generation interval distribution.', 'cite_spans': [], 'ref_spans': [], 'section': 'Sensitivity analyses'}, {'text': 'To further assess the robustness of the estimated generation interval distribution, for each dataset, we fit the model to data from the largest cluster. In the Tianjin dataset, the largest cluster is the shopping mall cluster consisting of 45 cases. In the Singapore dataset this is the Grace Assembly of God cluster consisting of 25 cases.', 'cite_spans': [], 'ref_spans': [], 'section': 'Sensitivity analyses'}, {'text': '6 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Sensitivity analyses'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.05.20031815 doi: medRxiv preprint 3 Results Table 1 shows parameter estimates of the generation and serial interval distributions for each dataset, assuming an incubation period with mean 5.2 and SD 2.8 days. The mean generation time is estimated to be 5.2 days (95%CI, 3.78 -6.78) for the Singapore data, and 3.95 days (95%CI, 3.01 -4.91) for the Tianjin data. As expected, the estimated means of the generation interval and serial interval distributions are approximately equal but the latter has a larger variance. . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [{'start': 154, 'end': 161, 'text': 'Table 1', 'ref_id': 'TABREF0'}], 'section': 'Sensitivity analyses'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.05.20031815 doi: medRxiv preprint it is estimated to be 2.57 days. Table 5 shows the proportions of pre-symptomatic transmission and reproduction numbers for each dataset. Pre-symptomatic transmission is higher when allowing for negative serial intervals for cases with no known infector. The reproduction number is lower when estimated using the serial interval compared to when using the generation interval. ', 'cite_spans': [], 'ref_spans': [{'start': 177, 'end': 184, 'text': 'Table 5', 'ref_id': 'TABREF7'}], 'section': 'Sensitivity analyses'}, {'text': 'We estimated the generation time to have a mean of 5.20 (95%CI 3.78-6.78) days and a standard deviation of 1.72 (95%CI 0.91-3.93) days for the Singapore data, and a mean 9 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint As expected, the proportion of pre-symptomatic transmission increases from 48% (95%CI 32-67%) in the baseline scenario to 66% (95%CI 45-84%) when allowing for negative serial intervals, for the Singapore data, and from 62% (95%CI 50-76%) to 77% (95%CI 65-87%) for the Tianjin data. Hence, a substantial proportion of transmission appears to occur before symptom onset, which is an important point to consider when planning intervention strategies. We also estimated R 0 , solely to illustrate the bias that occurs when using the serial interval as a proxy for the generation interval [7] . Whereas the impact was limited for our analyses, estimates based on the generation interval are larger and should be preferred to inform intervention policies. Indeed, as expected, the reproduction number was underestimated when using the serial interval distribution which is more variable than the generation interval distribution.', 'cite_spans': [{'start': 660, 'end': 663, 'text': '[7]', 'ref_id': 'BIBREF7'}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Tindale et al. [14] recently estimated the mean serial interval for COVID-19 to be 4.56 (95%CI 2.69 -6.42) days for Singapore and 4.22 (95%CI 3.43 -5.01) days for Tianjin.', 'cite_spans': [{'start': 15, 'end': 19, 'text': '[14]', 'ref_id': 'BIBREF15'}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Although these estimates are different from the ones we report, they fall within the uncertainty ranges we obtained. An important advantage of our method is that we are able to infer the generation interval distribution while allowing serial intervals to be negative. Our estimates of R are smaller than the ones reported by Tindale et al. [14] , because we use a different estimate of the growth rate r (0.04 for Singapore and 0.12 for 10 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 340, 'end': 344, 'text': '[14]', 'ref_id': 'BIBREF15'}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.05.20031815 doi: medRxiv preprint Tianjin as obtained from the initial exponential growth phase in each dataset, compared to 0.15 used by [14] ).', 'cite_spans': [{'start': 248, 'end': 252, 'text': '[14]', 'ref_id': 'BIBREF15'}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Another advantage of our method is that we can derive a proper variance estimate for the generation interval, in contrast to using a too large variance estimate that is obtained when using the serial interval as a proxy for the generation interval. Furthermore, in theory we do not need to condition on the order of symptom onset times. However, when the data does not provide sufficient information on directionality of transmission, this lack of auxiliary information may cause problems for estimation.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'This study does have some limitations. First, we rely on previous estimates for the incubation period. However, sensitivity analyses show that changing the incubation period distribution does not have a big impact on our estimates of the generation interval distribution. Second, we do not account for incomplete or possible changes in reporting over the course of the epidemic. Third, we do not acknowledge changes in contact processes and thus behavioral change, which could shape realised generation interval distributions as well as serial interval distributions (unpublished work). Fourth, we do not account for contraction of the generation interval because of depletion of susceptibles.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Future work should take into account these shortcomings.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Infection control for the COVID-19 epidemic relies on case-based measures such as finding cases and tracing contacts. A variable that determines how effective these case-based measures are is the proportion of pre-symptomatic transmission. Our estimates of this proportion are high, ranging from 0.48 to 0.77. This implies that the effectiveness of case finding and contact tracing in preventing COVID-19 infections will be considerably smaller compared to the effectiveness in preventing SARS or MERS infections, where presymptomatic transmission did not play an important role (see e.g. [15] ). It is unlikely that these measures alone will suffice to control the COVID-19 epidemic. Additional measures, such as social distancing, are required. ', 'cite_spans': [{'start': 589, 'end': 593, 'text': '[15]', 'ref_id': 'BIBREF16'}], 'ref_spans': [], 'section': 'Discussion'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '11'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.05.20031815 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': '11'}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'Handbook of infectious disease data analysis', 'authors': [{'first': 'L', 'middle': [], 'last': 'Held', 'suffix': ''}], 'year': 2019, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF1': {'ref_id': 'b1', 'title': 'Early transmission dynamics in Wuhan, China, of novel coronavirs-infected pneumonia', 'authors': [{'first': 'Q', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Guan', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Wu', 'suffix': ''}], 'year': 2020, 'venue': 'The New England Journal of Medicine', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1056/NEJMoa2001316']}}, 'BIBREF2': {'ref_id': 'b2', 'title': 'Incubation period of 2019 novel coronavirus (2019-nCoV) infections among travellers from Wuhan', 'authors': [{'first': 'J', 'middle': ['A'], 'last': 'Backer', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Klinkenberg', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Wallinga', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '20--28', 'other_ids': {}}, 'BIBREF3': {'ref_id': 'b3', 'title': 'Eurosurveillance 25', 'authors': [], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.2807/1560-7917.ES.2020.25.5.2000062']}}, 'BIBREF4': {'ref_id': 'b4', 'title': 'Epidemiological characteristics of novel coronavirus infection: A statistical analysis of publicly available case data', 'authors': [{'first': 'N', 'middle': ['M'], 'last': 'Linton', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Kobayashi', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Yang', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.01.26.20018754']}}, 'BIBREF5': {'ref_id': 'b5', 'title': 'Evolving epidemiology of novel coronavirus diseases 2019 and possible interruption of local transmission outside Hubei Province in China: a descriptive and modeling study', 'authors': [{'first': 'J', 'middle': [], 'last': 'Zhang', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.02.21.20026328']}}, 'BIBREF6': {'ref_id': 'b6', 'title': 'Transmission dynamics of 2019 novel coronavirus (2019-nCoV)', 'authors': [{'first': 'T', 'middle': [], 'last': 'Liu', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Hu', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Lin', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.01.25.919787']}}, 'BIBREF7': {'ref_id': 'b7', 'title': 'Estimation in emerging epidemics: biases and remedies', 'authors': [{'first': 'T', 'middle': [], 'last': 'Britton', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Scalia-Tomba', 'suffix': ''}], 'year': 2019, 'venue': 'Journal of The Royal Society Interface', 'volume': '16', 'issn': '', 'pages': '', 'other_ids': {'arXiv': ['doi:arXiv:1803.01688']}}, 'BIBREF8': {'ref_id': 'b8', 'title': 'Different Epidemic Curves for Severe Acute Respiratory Syndrome Reveal Similar Impacts of Control Measures', 'authors': [{'first': 'J', 'middle': [], 'last': 'Wallinga', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Teunis', 'suffix': ''}], 'year': 2004, 'venue': 'American Journal of Epidemiology', 'volume': '160', 'issn': '', 'pages': '509--516', 'other_ids': {'DOI': ['10.1093/aje/kwh255']}}, 'BIBREF9': {'ref_id': 'b9', 'title': 'Robust Reconstruction and Analysis of Outbreak Data: Influenza A(H1N1)v Transmission in a School-based Population', 'authors': [{'first': 'N', 'middle': [], 'last': 'Hens', 'suffix': ''}], 'year': 2012, 'venue': 'American Journal of Epidemiology', 'volume': '176', 'issn': '', 'pages': '196--203', 'other_ids': {'DOI': ['10.1093/aje/kws006']}}, 'BIBREF10': {'ref_id': 'b10', 'title': 'Estimating the generation interval of influenza A (H1N1) in a range of social settings', 'authors': [{'first': 'D', 'middle': ['E'], 'last': 'Beest', 'suffix': ''}], 'year': 2013, 'venue': 'In: Epidemiology', 'volume': '24', 'issn': '', 'pages': '244--250', 'other_ids': {}}, 'BIBREF11': {'ref_id': 'b11', 'title': 'Introduction to the Theory of Statistics. McGraw-hill', 'authors': [{'first': 'A', 'middle': ['M'], 'last': 'Mood', 'suffix': ''}], 'year': 1950, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF12': {'ref_id': 'b12', 'title': 'author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.03.05.20031815']}}, 'BIBREF13': {'ref_id': 'b13', 'title': 'Monte Carlo Statistical Methods', 'authors': [{'first': 'C', 'middle': ['P'], 'last': 'Robert', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Casella', 'suffix': ''}], 'year': 2004, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1007/978-1-4757-4145-2']}}, 'BIBREF14': {'ref_id': 'b14', 'title': 'How generation intervals shape the relationship between growth rates and reproductive numbers', 'authors': [{'first': 'J', 'middle': [], 'last': 'Wallinga', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Lipsitch', 'suffix': ''}], 'year': 2007, 'venue': 'Proceedings of the Royal Society B: Biological Sciences', 'volume': '274', 'issn': '', 'pages': '599--604', 'other_ids': {'DOI': ['10.1098/rspb.2006.3754']}}, 'BIBREF15': {'ref_id': 'b15', 'title': 'Transmission interval estimates suggest pre-symptomatic spread of COVID-19', 'authors': [{'first': 'L', 'middle': ['C'], 'last': 'Tindale', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Coombe', 'suffix': ''}, {'first': 'J', 'middle': ['E'], 'last': 'Stockdale', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF16': {'ref_id': 'b16', 'title': 'Public Health Interventions and SARS Spread', 'authors': [{'first': 'D', 'middle': ['M'], 'last': 'Bell', 'suffix': ''}], 'year': 2003, 'venue': 'Emerging Infectious Diseases', 'volume': '10', 'issn': '', 'pages': '1900--1906', 'other_ids': {'DOI': ['10.3201/eid1011.040729']}}, 'BIBREF17': {'ref_id': 'b17', 'title': 'author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.03.05.20031815']}}}, 'ref_entries': {'FIGREF0': {'text': '. . . , n, denote t i the time of infection for individual i, t v(i) the time of infection for the infector of individual i, δ i the incubation period for individual i, and δ v(i) the incubation period for the infector of individual i. The serial interval (Z i ) for case i is a linear combination of latent variables, i.e., Z i', 'latex': None, 'type': 'figure'}, 'FIGREF1': {'text': 'the Tianjin data. These mean estimates increase only slightly when increasing the mean incubation period. For the Singapore data, allowing the serial interval to be negative decreases the estimated mean generation time from 5.20 days, when restricting missing serial intervals to be positive, to 3.86 (95%CI 2.22-5.60) days when allowing them to be negative. For the Tianjin data, the baseline estimate of the mean generation time (3.95 days) is about the same than when allowing serial intervals to be negative in the Singapore data. However, in the Tianjin data there were already some negative serial intervals among the reported links, which may explain this lower estimate. When allowing for negative serial intervals in the Tianjin data, the mean generation timedecreased to 2.90 (1.85-4.12) days. These are the first estimates of the generation interval for COVID-19. Sensitivity analyses show that the assumptions made about the incubation period have only moderate impact on the results. On the other hand, assumptions made about the underlying transmission network (e.g. acknowledging possibly negative serial intervals) have a large impact on our results.', 'latex': None, 'type': 'figure'}, 'FIGREF2': {'text': \"Acknowledgements NH acknowledges funding from the European Research Council (ERC) under the European Unionâs Horizon 2020 research and innovation programme (grant agreement 682540 -TransMID). CF, NH and JW acknowledge funding from the European Union's SC1-PHE-CORONAVIRUS-2020 programme.\", 'latex': None, 'type': 'figure'}, 'TABREF0': {'text': 'Parameter estimates and credible intervals of generation and serial interval distributions for Singapore and Tianjin datasets. The estimates are obtained using reported information on infector-infectee pairs and assuming an incubation period with mean 5.2 and SD 2.8 days. GI: generation interval; SI: serial interval.', 'latex': None, 'type': 'table'}, 'TABREF1': {'text': 'shows parameter estimates of the generation and serial interval distributions for author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'latex': None, 'type': 'table'}, 'TABREF2': {'text': 'Parameter estimates and credible intervals of generation and serial interval distributions for Singapore and Tianjin datasets. The estimates are obtained under different assumptions for the incubation period, under the baseline scenario where missing SI are only allowed to be positive. GI: generation interval; SI: serial interval', 'latex': None, 'type': 'table'}, 'TABREF3': {'text': 'shows parameter estimates of the generation and serial interval distributions obtained when allowing for negative serial intervals in case there is no known infector.Compared to baseline analyses (Table 1), estimates of the mean generation time are smaller when allowing for negative serial intervals. The mean generation time is 3.86 days for Singapore and 2.90 days for Tianjin.', 'latex': None, 'type': 'table'}, 'TABREF4': {'text': 'Parameter estimates and credible intervals of generation and serial interval distributions for Singapore and Tianjin datasets when allowing serial intervals to be', 'latex': None, 'type': 'table'}, 'TABREF5': {'text': 'shows parameter estimates obtained when we fit the model to data from the largest cluster. We only show results for the Tianjin dataset, as for the Singapore data', 'latex': None, 'type': 'table'}, 'TABREF6': {'text': 'Parameter estimates and credible intervals of generation and serial interval distributions for the largest cluster in the Tianjin dataset. The estimates are obtained under different scenarios for the serial interval, assuming an incubation period with mean', 'latex': None, 'type': 'table'}, 'TABREF7': {'text': '', 'latex': None, 'type': 'table'}}, 'back_matter': []}\n",
      "{'paper_id': 'af5b90184fc31ef419b382d1160f55651206933e', 'metadata': {'title': 'Electrophysiology in the time of coronavirus: coping with the great wave', 'authors': [{'first': 'Jia', 'middle': [], 'last': 'Li', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': \"The Second Affiliated Hospital and Yuying Children's Hospital of Wenzhou Medical University\", 'location': {'addrLine': 'No.109 Xueyuan West Road', 'postCode': '325000', 'settlement': 'Lucheng District, Wenzhou, Zhejiang', 'country': 'PR China ('}}, 'email': ''}, {'first': 'Patrizio', 'middle': [], 'last': 'Mazzone', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': \"Unità Operativa di Aritmologia ed Elettrofisiologia cardiaca dell'IRCCS Ospedale San Raffaele di Milano\", 'location': {}}, 'email': ''}, {'first': 'Lisa', 'middle': ['Wm'], 'last': 'Leung', 'suffix': '', 'affiliation': {'laboratory': ') Cardiology Clinical Academic Group', 'institution': \"St George's Hospital\", 'location': {'postCode': 'SW17 0QT', 'settlement': 'London'}}, 'email': ''}, {'first': 'Weiqian', 'middle': [], 'last': 'Lin', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': \"The Second Affiliated Hospital and Yuying Children's Hospital of Wenzhou Medical University\", 'location': {'addrLine': 'No.109 Xueyuan West Road', 'postCode': '325000', 'settlement': 'Lucheng District, Wenzhou, Zhejiang', 'country': 'PR China ('}}, 'email': ''}, {'first': 'Giuseppe', 'middle': ['D &apos;'], 'last': 'Angelo', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': \"Unità Operativa di Aritmologia ed Elettrofisiologia cardiaca dell'IRCCS Ospedale San Raffaele di Milano\", 'location': {}}, 'email': ''}, {'first': 'Jun', 'middle': [], 'last': 'Ma', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': \"The Second Affiliated Hospital and Yuying Children's Hospital of Wenzhou Medical University\", 'location': {'addrLine': 'No.109 Xueyuan West Road', 'postCode': '325000', 'settlement': 'Lucheng District, Wenzhou, Zhejiang', 'country': 'PR China ('}}, 'email': ''}, {'first': 'Jin', 'middle': [], 'last': 'Li', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': \"The Second Affiliated Hospital and Yuying Children's Hospital of Wenzhou Medical University\", 'location': {'addrLine': 'No.109 Xueyuan West Road', 'postCode': '325000', 'settlement': 'Lucheng District, Wenzhou, Zhejiang', 'country': 'PR China ('}}, 'email': ''}, {'first': 'Zaki', 'middle': [], 'last': 'Akhtar', 'suffix': '', 'affiliation': {'laboratory': ') Cardiology Clinical Academic Group', 'institution': \"St George's Hospital\", 'location': {'postCode': 'SW17 0QT', 'settlement': 'London'}}, 'email': ''}, {'first': 'Yuechun', 'middle': [], 'last': 'Li', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': \"The Second Affiliated Hospital and Yuying Children's Hospital of Wenzhou Medical University\", 'location': {'addrLine': 'No.109 Xueyuan West Road', 'postCode': '325000', 'settlement': 'Lucheng District, Wenzhou, Zhejiang', 'country': 'PR China ('}}, 'email': ''}, {'first': 'Paolo', 'middle': ['E'], 'last': 'Della Bella', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': \"Unità Operativa di Aritmologia ed Elettrofisiologia cardiaca dell'IRCCS Ospedale San Raffaele di Milano\", 'location': {}}, 'email': ''}, {'first': 'Jiafeng', 'middle': [], 'last': 'Lin', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': \"The Second Affiliated Hospital and Yuying Children's Hospital of Wenzhou Medical University\", 'location': {'addrLine': 'No.109 Xueyuan West Road', 'postCode': '325000', 'settlement': 'Lucheng District, Wenzhou, Zhejiang', 'country': 'PR China ('}}, 'email': ''}, {'first': 'Mark', 'middle': ['M'], 'last': 'Gallagher', 'suffix': '', 'affiliation': {'laboratory': ') Cardiology Clinical Academic Group', 'institution': \"St George's Hospital\", 'location': {'postCode': 'SW17 0QT', 'settlement': 'London'}}, 'email': 'mark.gallagher@stgeorges.nhs.uk'}, {'first': 'Mark', 'middle': [], 'last': 'Gallagher', 'suffix': '', 'affiliation': {}, 'email': ''}]}, 'abstract': [], 'body_text': [{'text': 'Informed consent was obtained from all participants/interviewees who took part in this study. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'CONSENT'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.30.20044776 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'CONSENT'}, {'text': 'The COVID-19 pandemic has caused suffering and death across the world since late 2019. 1, 2 Physicians and journalists encountering the arrival of the epidemic in an area typically describe a wave, a tide or a tsunami of disease. It has had a widely publicised impact on the economy of afflicted areas. Medical services have suffered massive disruption. Even services far removed from respiratory medicine and critical care have suffered because of the diversion of resources needed to support the victims of the epidemic and through the illness of staff. As a resource-intensive speciality that deals predominantly with non-emergency cases, interventional electrophysiology is particularly vulnerable to disruption.', 'cite_spans': [{'start': 87, 'end': 89, 'text': '1,', 'ref_id': 'BIBREF0'}, {'start': 90, 'end': 91, 'text': '2', 'ref_id': 'BIBREF2'}], 'ref_spans': [], 'section': 'BACKGROUND'}, {'text': 'While the pandemic disrupts the delivery of routine electrophysiology services, COVID-19 is associated with cardiac complications which could bring an additional burden of acute problems to electrophysiology. [3] [4] [5] The relative importance of the reduction in elective cases and any increase in emergency work is undefined.', 'cite_spans': [{'start': 209, 'end': 212, 'text': '[3]', 'ref_id': 'BIBREF4'}, {'start': 213, 'end': 216, 'text': '[4]', 'ref_id': 'BIBREF5'}, {'start': 217, 'end': 220, 'text': '[5]', 'ref_id': 'BIBREF6'}], 'ref_spans': [], 'section': 'BACKGROUND'}, {'text': 'Wenzhou, Milan and London are each cities with a metropolitan area population of more than 4 million people; each has each been struck by the pandemic. During the initial months of 2020, Wenzhou experienced the greatest concentration of COVID-19 cases of any Chinese city outside the Hubei province. 6 Following an intense program of public health interventions 7 , new cases declined and have now vanished.', 'cite_spans': [{'start': 300, 'end': 301, 'text': '6', 'ref_id': None}], 'ref_spans': [], 'section': 'BACKGROUND'}, {'text': 'As of March 27 th 2020, Wenzhou has had no new case of COVID-19 for the last 38 consecutive days. With this, many sectors have begun to return toward normality with stringent precautions. Milan was close to the centre of the Italian outbreak of COVA-19 and had to suspend all activity the first week in March. 8 London, like the rest of the United Kingdom has seen an abrupt rise in the disease incidence only since the second week of March 2020. 9 The disruption for cardiac electrophysiology labs has begun.', 'cite_spans': [{'start': 310, 'end': 311, 'text': '8', 'ref_id': 'BIBREF9'}, {'start': 447, 'end': 448, 'text': '9', 'ref_id': 'BIBREF10'}], 'ref_spans': [], 'section': 'BACKGROUND'}, {'text': 'We reviewed the catheter lab records of electrophysiology laboratories in a centre that experienced a high burden of COVID-19 early and near the origin of the pandemic and one in which the onset was delayed. The impact on workflow resulting from the redirection of staff and resources was correlated with the case-load of All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'METHODS'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.30.20044776 doi: medRxiv preprint COVID-19. We charted the burden of emergency procedures performed to look for evidence of any augmentation of these arising from COVID-19, and we interviewed the front-line cardiologists for information about arrhythmic complications encountered in the COVID-19 population. We looked for instances of COVID-19 infection acquired in hospital by electrophysiology patients and staff. We documented the protocols used to permit the resumption of activity after the first wave of the epidemic and examined the success of anti-infective precautions.', 'cite_spans': [], 'ref_spans': [], 'section': 'METHODS'}, {'text': 'The first diagnosis of COVID-19 in Wenzhou Medical University was in mid-January ', 'cite_spans': [], 'ref_spans': [], 'section': 'RESULTS'}, {'text': 'In each of the centres, routine activity of the EP lab was suspended within a week of the first locally diagnosed COVID-19 case in the hospital (table 1). The primary reason for the prompt suspension in each case was the concern that continuing to admit patients for elective procedures would expose patients to the risk of infection from undiagnosed COVID-19 patients already in the hospital.', 'cite_spans': [], 'ref_spans': [], 'section': 'Effect on routine work'}, {'text': 'We encountered no instance of an arrhythmia occurring as a definable consequence of COVID-19 that required ablation as an emergency procedure. A small number of patients required emergency ablation during the period of restricted activity (table 2, figure 2), but this represented less than 5% of the normal workload of the centres.', 'cite_spans': [], 'ref_spans': [{'start': 239, 'end': 248, 'text': '(table 2,', 'ref_id': 'TABREF1'}], 'section': 'Electrophysiological emergencies during the COVID-19 crisis'}, {'text': 'The distribution of cases was similar to that encountered in normal times with supraventricular tachycardia, ventricular tachycardia and atrial fibrillation all represented.', 'cite_spans': [], 'ref_spans': [], 'section': 'Electrophysiological emergencies during the COVID-19 crisis'}, {'text': 'All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Electrophysiological emergencies during the COVID-19 crisis'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.30.20044776 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'Electrophysiological emergencies during the COVID-19 crisis'}, {'text': 'One patient in London acquired COVID-19, apparently in hospital after her ablation.', 'cite_spans': [], 'ref_spans': [], 'section': 'Nosocomial Infection'}, {'text': 'She died of pneumonia associated with the condition at 18 days after the ablation.', 'cite_spans': [], 'ref_spans': [], 'section': 'Nosocomial Infection'}, {'text': 'Two electrophysiologists in London including a co-author of this paper acquired it and recovered without complication. Three nurses in a cardiology ward in Milan were infected, but none of the electrophysiologists. In Wenzhou, no staff member was infected.', 'cite_spans': [], 'ref_spans': [], 'section': 'Nosocomial Infection'}, {'text': 'Of the centres involved, only Wenzhou has been able to resume routine activity. This 1) Patients underwent a pre-assesment visit the day before hospital admission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Resumption of normal activity'}, {'text': 'At this visit, a nasal swab was analysed by PCR for evidence of the virus.', 'cite_spans': [], 'ref_spans': [], 'section': 'Resumption of normal activity'}, {'text': '2) Patients with a satisfactory screening visit were admitted on the morning of the scheduled procedure and underwent a second nasal swab. If the PCR was negative for the virus, the procedure went ahead as planned.', 'cite_spans': [], 'ref_spans': [], 'section': 'Resumption of normal activity'}, {'text': 'In all centres, outpatient activity was moved to a telephone-based or internet-based format in all possible cases from the time that widespread local transmission of the virus was recognised. Only those who were clinically urgent and were unable to to use a telephone or required an instrumental test were allowed to keep their physical All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Outpatient activity'}, {'text': 'author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Outpatient activity'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.30.20044776 doi: medRxiv preprint appointments. Outpatient investigations were cancelled unless clinically urgent.', 'cite_spans': [], 'ref_spans': [], 'section': 'Outpatient activity'}, {'text': 'Patients who had access to mobile-phone associated ECG recording were encouraged to use these and transmit recordings in preparation for telephone consultations.', 'cite_spans': [], 'ref_spans': [], 'section': 'Outpatient activity'}, {'text': 'As part of social distancing, all EP educational meetings were suspended in all centres, and all major international meetings have been cancelled. Electronic learning has been put into practice across several medical disciplines and improving web-based educational tools. Major medical examinations such as the Membership exam (MRCP) and university entrance exams have been delayed indefinitely.', 'cite_spans': [], 'ref_spans': [], 'section': 'EP Education'}, {'text': 'The EP team in Wenzhou and that in London defaulted to emergency mode at the arrival of the first COVID cases. With elective work being cancelled, the electrophysiologists concentrated on urgent inpatient intervention including pacemaker implantation to minimist the time that any patient had to spend in hospital. The centres differed in their policies on redistributing staff to other duties.', 'cite_spans': [], 'ref_spans': [], 'section': 'EP team redeployment'}, {'text': 'We have no gift to set a statesman right\" William Yates 10 The electrophysiologist, like the poet is a luxury that society may dispense with when times are exceptionally difficult. Our work reduces or resolves troublesome symptoms; in narrow subgroups of the population that we serve, we may increase life expectancy, albeit marginally. In achieving this we expend resources that may be better used elsewhere in difficult times and we bring patients into a healthcare All rights reserved. No reuse allowed without permission.', 'cite_spans': [{'start': 50, 'end': 58, 'text': 'Yates 10', 'ref_id': None}], 'ref_spans': [], 'section': '\"I think it better that in times like these A poet\\'s mouth be silent, for in truth'}, {'text': 'author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '\"I think it better that in times like these A poet\\'s mouth be silent, for in truth'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.30.20044776 doi: medRxiv preprint environment that has suddenly become dangerous. We have no expertise that is relevant to the management of an epidemic but we have qualities that might be useful: Being delicate and unnecessary we may serve as an indicator of the health of our surroundings.', 'cite_spans': [], 'ref_spans': [], 'section': '\"I think it better that in times like these A poet\\'s mouth be silent, for in truth'}, {'text': 'The results presented show that in representative centres from several of the worst- ', 'cite_spans': [], 'ref_spans': [], 'section': '\"I think it better that in times like these A poet\\'s mouth be silent, for in truth'}, {'text': \"Severe acute respiratory syndrome coronarvirus-2 (SARS-CoV-2), the agent responsible for COVID-19 has an affinity for angiotensin converting enzyme 2 (ACE2) receptors. 11 This is central to the pathophysiology of the condition, leading to pneumonia and in critical stages, associated fulminant acute respiratory distress syndrome (ARDS) and multi-organ failure. 12 The primary affected organ is the lung but cardiovascular injury is also common. There is enhanced expression of ACE-2 in those with cardiovascular conditions, possibly accounting for the apparent greater severity of COVID-19 related illness in these patients. In one study, a majority (58%) of those with severe infection had a background of hypetension, and non-specifically 44% had a history of 'arrhythmia'. 13 Binding of the virus to ACE2 receptors in the lung and the heart initiates the acute inflammatory response. Pneumonia results, but also evidence of myocarditis; those with a rise in high-sensitivity Troponin I are more likely to require admission to intensive care. 14 There is a strong association between a history of hypertension and mortality from COVID-19. Systolic blood pressure was significantly higher in those with COVID-19 treated in intensive care units compared to those not requiring this level of care (144 versus 122, P<0.001). 14 This may relate to up-regulation of ACE2 receptors in those receiving ACE inhibiting drugs. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\", 'cite_spans': [{'start': 168, 'end': 170, 'text': '11', 'ref_id': 'BIBREF12'}, {'start': 362, 'end': 364, 'text': '12', 'ref_id': 'BIBREF13'}, {'start': 777, 'end': 779, 'text': '13', 'ref_id': 'BIBREF14'}], 'ref_spans': [], 'section': 'COVID-19 and the cardiovascular system'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.30.20044776 doi: medRxiv preprint A history of ischaemic heart disease also predicts mortality in COVID-19. Any acute inflammatory condition can destabilise coronary plaque and triger acute coronary syndromes. A minority of patients with COVID-19 may present acutely with cardiac sounding symptoms. 13 It is not clear whether the apparent high mortality in patients with ischaemic heart disease represents an effect on ACE2 receptor expression, a vulnerability to ischaemic complications of the systemic illness or a bias against such patients when ventilatiors must be rationed. The pathophysiology of COVID-19 and its routes of systemic transmission remain incompletely understood. Observations had been noted of its neuroinvasive potential via peripheral neuronal trans-synaptic route and viral preponderance in organs with low expression of ACE2. 15 Brugada syndrome is a concern as the prolonged febrile illness characteristic of COVID-19 has the propensity to trigger arrhythmias. No such case has been reported, but like patients with heart failure and ischaemic heart disease, those with Brugada syndrome should take particular care to isolate themselves from likely sources of infection.', 'cite_spans': [{'start': 409, 'end': 411, 'text': '13', 'ref_id': 'BIBREF14'}, {'start': 961, 'end': 963, 'text': '15', 'ref_id': 'BIBREF16'}], 'ref_spans': [], 'section': 'COVID-19 and the cardiovascular system'}, {'text': 'No specific pharmacological therapy exists, but drugs being trialled in treating aspects of COVID-19 include anti-retrovirals and hydrochloroquine. These drugs may cause cardiotoxicitiy and prolong corrected QT intervals. No instance of a lifethreatening arrhythmia from this source has been reported.', 'cite_spans': [], 'ref_spans': [], 'section': 'COVID-19 and the cardiovascular system'}, {'text': 'The risk of COVID-19 transmission to healthcare workers is high, with over 8% reported in Italy. 16 Ophthalmologists have been disproportionately affected, probably because of the close contact with the patients in whom conjunctivitis has been a presenting symptom. 17 Intensivists are at risk due to their inevitable extensive contact with severely affected patients and due to aerosol generation during intubation. Cardiology, including ablation and trans-esophageal echo also involves aerosol generating procedures (AGP) and close proximity to the patient.', 'cite_spans': [{'start': 97, 'end': 99, 'text': '16', 'ref_id': None}, {'start': 266, 'end': 268, 'text': '17', 'ref_id': 'BIBREF17'}], 'ref_spans': [], 'section': 'Transmission to Healthcare providers'}, {'text': 'Under normal circumstances, a patient entering the cardiac catheter lab will encounter approximately 10 healthcare professionals before moving to a ward for recovery. The combination of numerous close contacts and fomites creates a risk of transmission. It makes sense to defer procedures whenever possible. Emergency All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Transmission to Healthcare providers'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the ', 'cite_spans': [], 'ref_spans': [], 'section': 'Transmission to Healthcare providers'}, {'text': 'Many patients scheduled for EP procedures fall into categories at high risk of death if exposed to COVID-19. In the London cohort, the average age is 65, and >60% are male. 18 Co-existing diabetes, hypertension or heart failure are common. All of these, particularly hypertension are strong risk factors for COVID-19 related mortality. It was therefore obligatory to halt the performance of routine ablation procedures until the risk recedes.', 'cite_spans': [{'start': 173, 'end': 175, 'text': '18', 'ref_id': 'BIBREF20'}], 'ref_spans': [], 'section': 'EP patient cohort'}, {'text': 'Not all EP patients can wait on indefinitely. Guidelines published by NHS England, 19 include narrow criteria for ablation in this crisis, recommending that it is limited to cases of rapidly conducted pre-excited atrial fibrillation, heart failure secondary to tachycardia, and ventricular tachycardia that is controllable with medication. All of the ablations conducted in all 3 centres during the period of restricted activity met these criteria ( figure 2, table 2) .', 'cite_spans': [{'start': 83, 'end': 85, 'text': '19', 'ref_id': 'BIBREF21'}], 'ref_spans': [{'start': 450, 'end': 468, 'text': 'figure 2, table 2)', 'ref_id': None}], 'section': 'EP patient cohort'}, {'text': 'Strict guidelines were put in place in each centre to minimise the risk to patients and staff in cases where ablation had to be performed. Local protocols mirored published All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Catheter lab protocols'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.30.20044776 doi: medRxiv preprint consensus documents. 20 Strict isolation measures were applied before and after any procedure. During the procedure, staff wore PPE.', 'cite_spans': [{'start': 165, 'end': 167, 'text': '20', 'ref_id': None}], 'ref_spans': [], 'section': 'Catheter lab protocols'}, {'text': 'The reduction in electrophysiology caseload related to the pandemic will effect the EP industry. Companies which supply equipment have shut offices and distribution networks. Loss of revenue will last for many months. Companies may prosper and may help the fight against the pandemic by switching production to items of relevance to that fight: Ventilators and associated equipment, viral test kits and thermometers.', 'cite_spans': [], 'ref_spans': [], 'section': 'EP Industry'}, {'text': 'In China, the official recognition of the seriousness of the outbreak and the take-over by central government of direct responsibility for its handling coincided with the introduction of an integrated set of measures. 7 Best publicised was the lock-down of society and industry to achieve extreme social distancing. The less-publicised adjunct to the lock-down was the introduction of a system for screening all persons with fever or with a history of contact with known cases, so that they could be has not yet been developed.', 'cite_spans': [{'start': 218, 'end': 219, 'text': '7', 'ref_id': 'BIBREF8'}], 'ref_spans': [], 'section': 'Public Health Measures'}, {'text': 'The experience from Wenzhou shows that with energetic public health intervention, the epidemic may be suppressed, and normal life may resume with precautions. In the longer term, a vaccine may permit us to dispense with these precautions. The less stringent interventions chosen by the UK and other North European countries All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Public Health Measures'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.30.20044776 doi: medRxiv preprint risk allowing a lingering epidemic that drags on for so long that by the time a vaccine comes, production and supply chains will have fallen into habitual disuse and operators will have lost familiarity with their craft. Many patients will also have died by then.', 'cite_spans': [], 'ref_spans': [], 'section': 'Public Health Measures'}, {'text': 'In the Spring of 2020, as in the Summer of 1914, it feels as though \"the lamps are going out all over Europe\", but there is reason for hope. Vigorous public health measures have suppressed the epidemic in parts of Asia, permitting a resumption of some normal routine activity with stringent precautions. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'CONCLUSION'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.30.20044776 doi: medRxiv preprint In each of the electrophysiology labs, routine activity was cancelled as soon as the country detected significant numbers of COVID-19 cases. During the period of suspension of routine activity, very few emergency procedures were performed. Of the 3 centres, only Wenzhou had been able to resume normal activity. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'TABLES'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.30.20044776 doi: medRxiv preprint Figure 2 : Impact of the COVID-19 epidemic on workflow in electrophysiology.', 'cite_spans': [], 'ref_spans': [{'start': 144, 'end': 152, 'text': 'Figure 2', 'ref_id': None}], 'section': 'TABLES'}, {'text': 'The pie-charts indicate the mean number of each category of each procedure carried out per week in 2020 in each centre in the period before COVID-19 arrived, in the period of restricted activity and in the case of Wenzhou after the resumption of normal activity. The workload during the period of crisis represents the mean number of cases per week per centre across all 3 centres, and represents a fall of more than 95% compared to pre-crisis activity without a clear-cut alteration in the breakdown of procedure types.', 'cite_spans': [], 'ref_spans': [], 'section': 'TABLES'}, {'text': 'None declared. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'CONFLICTS OF INTEREST'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.30.20044776 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'CONFLICTS OF INTEREST'}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and coronarvirus', 'authors': [{'first': 'C', 'middle': ['C'], 'last': 'Lai', 'suffix': ''}, {'first': 'T', 'middle': ['P'], 'last': 'Shih', 'suffix': ''}, {'first': 'W', 'middle': ['C'], 'last': 'Ko', 'suffix': ''}, {'first': 'H', 'middle': ['J'], 'last': 'Tang', 'suffix': ''}, {'first': 'P', 'middle': ['R'], 'last': 'Hsueh', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF1': {'ref_id': 'b1', 'title': 'COVID-19): The epidemic and the challenges', 'authors': [], 'year': None, 'venue': 'Int J Antimicrob Agents. 2020', 'volume': '55', 'issn': '3', 'pages': '', 'other_ids': {'DOI': ['10.1016/j.ijantimicag.2020.105924']}}, 'BIBREF2': {'ref_id': 'b2', 'title': 'A novel coronavirus outbreak of global health concern', 'authors': [{'first': 'C', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'P', 'middle': ['W'], 'last': 'Horby', 'suffix': ''}, {'first': 'F', 'middle': ['G'], 'last': 'Hayden', 'suffix': ''}, {'first': 'G', 'middle': ['F'], 'last': 'Gao', 'suffix': ''}], 'year': None, 'venue': 'Lancet', 'volume': '2020', 'issn': '', 'pages': '470--473', 'other_ids': {}}, 'BIBREF4': {'ref_id': 'b4', 'title': 'Sars-cOv-2: a potential novel etiology of fulminant myocarditis', 'authors': [{'first': 'C', 'middle': [], 'last': 'Chen', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Zhou', 'suffix': ''}, {'first': 'D', 'middle': ['W'], 'last': 'Wang', 'suffix': ''}], 'year': 2020, 'venue': 'Herz', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1007/s00059-020-04909-z']}}, 'BIBREF5': {'ref_id': 'b5', 'title': 'Coronarvirus Fulminant Myocarditis Saved With Glucocorticoid and Human Immunoglobulin', 'authors': [{'first': 'H', 'middle': [], 'last': 'Hu', 'suffix': ''}, {'first': 'F', 'middle': [], 'last': 'Ma', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Wei', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Fang', 'suffix': ''}], 'year': None, 'venue': 'Eur Heart J. 2020', 'volume': '190', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1093/eurheartj/ehaa190']}}, 'BIBREF6': {'ref_id': 'b6', 'title': 'Cardiovascular Considerations for Patients, Health Care Workers, and Health Systems During the Coronarvirus Disease 2019 (COVID-19) Pandemic', 'authors': [{'first': 'E', 'middle': [], 'last': 'Driggin', 'suffix': ''}, {'first': 'M', 'middle': ['V'], 'last': 'Madhavan', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Bikdeli', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Chuich', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Laracy', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Bondi-Zoccai', 'suffix': ''}], 'year': None, 'venue': 'J Am Coll Cardiol', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1016/j.jacc.2020.03.031']}}, 'BIBREF7': {'ref_id': 'b7', 'title': 'doi: medRxiv preprint 6. Clinical findings in a group of patients infected with the 2019 novel coronarvirus (SARS-CoV-2) outside of Wuhan, China: a retrospective case series', 'authors': [], 'year': 2020, 'venue': 'BMJ', 'volume': '368', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.03.30.20044776']}}, 'BIBREF8': {'ref_id': 'b8', 'title': \"China's local governments are combating COVID-19 with unprecedented responses-from a Wenzhou governance perspective\", 'authors': [{'first': 'F', 'middle': [], 'last': 'Gong', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Xiong', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Ziao', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Lin', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Liu', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Wang', 'suffix': ''}], 'year': 2020, 'venue': 'Front Med', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['Doi:10.1007/s11684-020-0755-z']}}, 'BIBREF9': {'ref_id': 'b9', 'title': 'COVID-19 pandemic: perspectives on an unfolding crisis', 'authors': [{'first': 'A', 'middle': [], 'last': 'Spinelli', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Pellino', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '2020', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1002/bjs.11627']}}, 'BIBREF10': {'ref_id': 'b10', 'title': 'World Health Organisation (WHO) coronarvirus disease (COVID-19', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF11': {'ref_id': 'b11', 'title': 'In: Edith Wharton, The Book of the Homeless', 'authors': [{'first': 'W', 'middle': ['B'], 'last': 'Yates', 'suffix': ''}], 'year': 1916, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF12': {'ref_id': 'b12', 'title': 'Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus orgins and receptor binding', 'authors': [{'first': 'R', 'middle': [], 'last': 'Lu', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Zhao', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Niu', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Yang', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Wu', 'suffix': ''}], 'year': 2020, 'venue': 'Lancet', 'volume': '395', 'issn': '', 'pages': '565--574', 'other_ids': {}}, 'BIBREF13': {'ref_id': 'b13', 'title': 'Epidemiological and clinical characteristics of 99 cases of 2019 novel coronarvirus pneumonia in Wuhan, China: a descriptive study', 'authors': [{'first': 'N', 'middle': [], 'last': 'Chen', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Zhou', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Dong', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Qu', 'suffix': ''}, {'first': 'F', 'middle': [], 'last': 'Gong', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Han', 'suffix': ''}], 'year': None, 'venue': 'Lancet. 2020', 'volume': '395', 'issn': '', 'pages': '30211--30218', 'other_ids': {'DOI': ['Doi:10.1016/s0140-6736(20']}}, 'BIBREF14': {'ref_id': 'b14', 'title': 'COVID-19 and the cardiovascular system', 'authors': [{'first': 'Y', 'middle': [], 'last': 'Zheng', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Ma', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Xiang', 'suffix': ''}], 'year': 2020, 'venue': 'Nat Rev Cardiol', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1038/s41569-020-0360-5']}}, 'BIBREF15': {'ref_id': 'b15', 'title': 'Clinical features of patients infected with 2019 novel coronavirus in Wuhan', 'authors': [{'first': 'C', 'middle': [], 'last': 'Huang', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Ren', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Zhao', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Hu', 'suffix': ''}], 'year': 2020, 'venue': 'China. Lancet', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1016/S0140-6736(20)30183-5']}}, 'BIBREF16': {'ref_id': 'b16', 'title': 'The neuroinvasive potential of SARS-CoV2 may play a role in respiratory failure of COVID-19 patients', 'authors': [{'first': 'Y', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Bai', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Hashikawa', 'suffix': ''}], 'year': 2020, 'venue': 'J Med Virol', 'volume': '1', 'issn': '4', 'pages': '', 'other_ids': {'DOI': ['10.1002/jmv.25728']}}, 'BIBREF17': {'ref_id': 'b17', 'title': 'Stepping up infection control measures in ophthalmology during the novel coronavirus outbreak: an experience from All rights reserved', 'authors': [{'first': 'T', 'middle': [], 'last': 'Lai', 'suffix': ''}, {'first': 'E', 'middle': [], 'last': 'Tang', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Chau', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Fung', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Li', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF18': {'ref_id': 'b18', 'title': 'author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint (which was not peer-reviewed) is the', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.03.30.20044776']}}, 'BIBREF19': {'ref_id': 'b19', 'title': \"Graefe's Archive for Clinical and Experimental Ophthalmology\", 'authors': [{'first': 'Hong', 'middle': [], 'last': 'Kong', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1007/s00417-020-04641-8']}}, 'BIBREF20': {'ref_id': 'b20', 'title': 'Improving Esophageal Protection during AF ablation: the IMPACT study', 'authors': [{'first': 'L', 'middle': ['W'], 'last': 'Leung', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Bajpai', 'suffix': ''}, {'first': 'Z', 'middle': [], 'last': 'Zuberi', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Norman', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Kaba', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.01.30.20019158']}}, 'BIBREF21': {'ref_id': 'b21', 'title': 'NHS England Publication; 001559 version 1. Clinical guide for the management of cardiology patients during the coronavirus pandemic. 2020; 1-5. specialty-guide-cardiolgy-coronavirus-v1-20-march', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF23': {'ref_id': 'b23', 'title': \"Catheterization Laboratory Considerations During the Coronavirus (COVID-19) Pandemic: From ACC's Interventional Council and SCAI\", 'authors': [], 'year': 2020, 'venue': 'J Am Coll Cardiol', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1016/j.jacc.2020.03.021']}}, 'BIBREF24': {'ref_id': 'b24', 'title': 'All rights reserved. No reuse allowed without permission', 'authors': [], 'year': 1925, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF25': {'ref_id': 'b25', 'title': 'author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint (which was not peer-reviewed) is the', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.03.30.20044776']}}}, 'ref_entries': {'FIGREF0': {'text': \"from the Research Ethics Committee (REC) London according to the type of study. Institutional ethical approval obtained at the centres of: St. George's Hospital NHS Foundation Trust, London, UK; Local Health Authority Ethical Approval was obtained in: The Second Affiliated Hospital and Yuying Children's Hospital of Wenzhou Medical University in Wenzhou, PR China and San Raffaele in Milan, Italy.\", 'latex': None, 'type': 'figure'}, 'FIGREF1': {'text': \"In Lomardy a large outbreak became evident on February 22nd, while in St George's Hospital, London cases began to arrive in large number from the second week in March. In all cases, there was a sharp downturn in activity within days after the recognition of widespread COVA-19 transmission in the area (figure 1).\", 'latex': None, 'type': 'figure'}, 'FIGREF2': {'text': 'became possible because of a country-wide suppression of the epidemic produced by an intensive program of public health interventions across the country. This included a period of lock-down accompanied by intensive contact-tracing and use of isolation. Mobile technology played an integral part. Electronic tracking of each individual was conducted with a traffic-light system to inform the person and those in close proximity of their health and travel status. Having achieved the virtual elimination of virus transmission in most of the country by late February, routine activity resumed with stringent precautions:', 'latex': None, 'type': 'figure'}, 'FIGREF3': {'text': 'affected countries, the arrival of COVID-19 caused a complete cessation of routine electrophysiological intervention. A small number of urgent ablations were performed in patients without infection. Although arrhythmias were encountered in patients hospitalised with COVID-19, these were all either terminal events in the context of cardiomyopathy resulting from the condition or were related to the metabolic and haemodynamic stress arising from infection and mechanical ventilation. No instance was encountered of an arrhythmia arising from COVID-19 requiring urgent ablation.', 'latex': None, 'type': 'figure'}, 'FIGREF4': {'text': 'was already experienced in infection control protocols due to their experience of Severe Acute Respiratory Syndrome (SARS) in 2002-2003, and a rapid and effective response took place once the problem was recognised. In contrast, without any recent experience to draw on, Italy and the UK had been relatively unprepared for the scale of the events. Like China, Hong Kong was culturally sensitised by experience of SARS and produced swift and effective intervention to protect the public and healthcare workers, avoiding the spread of infection in healthcare settings. This involved a 3level hierarchy of control measures: administrative control, environmental control and the use of PPE. 17 Rigorous screening of the prospective patients took place to avoid bringing the virus into the uninfected healthcare environment.', 'latex': None, 'type': 'figure'}, 'FIGREF5': {'text': \"isolated in designated facilities. The use of electronic technology and surveillance served as an adjunct to these measures. The short-term impact of China's program of public health measures is evident from the progression of the epidemic (figure 1). In China, a single decisive act brought all necessary measures into being at a time when deaths were in low double figures. In Italy and in the UK as across Europe, distancing measures were introduced piecemeal and later in the course of events, and the infrastructure for the identification of infected individuals and the tracing, testing and isolation of contacts\", 'latex': None, 'type': 'figure'}, 'FIGREF6': {'text': 'Chronology of events in the first 87 days of 2020', 'latex': None, 'type': 'figure'}, 'TABREF0': {'text': 'Effects of COVID-19 on electrophysiology staff and services.', 'latex': None, 'type': 'table'}, 'TABREF1': {'text': 'Effects of the pandemic on electrophysiology activity. All rights reserved. No reuse allowed without permission.', 'latex': None, 'type': 'table'}, 'TABREF2': {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.30.20044776 doi: medRxiv preprint Arrhythmias ablated in COVID-19 patients All rights reserved. No reuse allowed without permission.', 'latex': None, 'type': 'table'}}, 'back_matter': []}\n",
      "{'paper_id': 'af68aa340ddee30dd67567eeb667f29b997f2f1c', 'metadata': {'title': 'Development and external validation of a prognostic multivariable model on admission for hospitalized patients with COVID-19', 'authors': [{'first': 'Jianfeng', 'middle': [], 'last': 'Xie', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Southeast University', 'location': {'postCode': '210009', 'settlement': 'Nanjing', 'country': 'China'}}, 'email': ''}, {'first': 'Daniel', 'middle': [], 'last': 'Hungerford', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Liverpool', 'location': {'postCode': 'L69 7BE', 'settlement': 'Liverpool', 'country': 'UK'}}, 'email': ''}, {'first': 'Hui', 'middle': [], 'last': 'Chen', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'The First Affiliated Hospital of Soochow University', 'location': {'postCode': '215000', 'settlement': 'Suzhou', 'country': 'China'}}, 'email': ''}, {'first': 'Simon', 'middle': ['T'], 'last': 'Abrams', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Liverpool', 'location': {'postCode': 'L69 7BE', 'settlement': 'Liverpool', 'country': 'UK'}}, 'email': ''}, {'first': 'Shusheng', 'middle': [], 'last': 'Li', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Huazhong University of Science and Technology', 'location': {'postCode': '430030', 'settlement': 'Wuhan', 'country': 'China'}}, 'email': ''}, {'first': 'M', 'middle': ['D'], 'last': '', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Guozheng', 'middle': [], 'last': 'Wang', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Southeast University', 'location': {'postCode': '210009', 'settlement': 'Nanjing', 'country': 'China'}}, 'email': ''}, {'first': 'Yishan', 'middle': [], 'last': 'Wang', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Capital Medical University', 'location': {'postCode': '100020', 'settlement': 'Beijing', 'country': 'China'}}, 'email': ''}, {'first': 'Hanyujie', 'middle': [], 'last': 'Kang', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Capital Medical University', 'location': {'postCode': '100020', 'settlement': 'Beijing', 'country': 'China'}}, 'email': ''}, {'first': 'Laura', 'middle': [], 'last': 'Bonnett Phd', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Liverpool', 'location': {'postCode': 'L69 3GL', 'settlement': 'Liverpool', 'country': 'UK'}}, 'email': ''}, {'first': 'Ruiqiang', 'middle': [], 'last': 'Zheng', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Medical College of Yangzhou University', 'location': {'postCode': '225001', 'settlement': 'Yangzhou', 'country': 'China'}}, 'email': ''}, {'first': 'Xuyan', 'middle': [], 'last': 'Li', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Huazhong University of Science and Technology', 'location': {'postCode': '430030', 'settlement': 'Wuhan', 'country': 'China'}}, 'email': ''}, {'first': '，zhaohui', 'middle': [], 'last': 'Tong', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Haibo', 'middle': ['Qiu'], 'last': 'Phd', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Capital Medical University', 'location': {'postCode': '100020', 'settlement': 'Beijing', 'country': 'China'}}, 'email': ''}, {'first': 'Cheng-Hock', 'middle': [], 'last': 'Toh', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Southeast University', 'location': {'postCode': '210009', 'settlement': 'Nanjing', 'country': 'China'}}, 'email': ''}, {'first': 'Haibo', 'middle': [], 'last': 'Qiu', 'suffix': '', 'affiliation': {}, 'email': ''}]}, 'abstract': [{'text': 'Since the outbreak of COVID-19, there has been a pressing need for development of a prognostic tool that is easy for clinicians to use. Recently, a Lancet publication showed that in a cohort of 191 patients with COVID-19, age, SOFA score and D-dimer measurements were associated with mortality. No other publication involving prognostic factors or models has been identified to date.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'In our cohorts of 444 patients from two hospitals, SOFA scores were low in the majority of patients on admission. The relevance of D-dimer could not be verified, as it is not included in routine laboratory tests. In this study, we have established a multivariable clinical prediction model using a development cohort of 299 patients from one hospital. After backwards selection, four variables, including age, lymphocyte count, lactate dehydrogenase and SpO2 remained in the model to predict mortality. This has been validated internally and externally with a cohort of 145 patients from a different hospital. Discrimination of the model was excellent in both internal (c=0·89) and external (c=0·98) validation. Calibration plots showed excellent agreement between predicted and observed probabilities of mortality after recalibration of the model to account for underlying differences in the risk profile of the datasets. This . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': '(which was not peer-reviewed) The copyright holder for this preprint .', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}], 'body_text': [{'text': 'Background: COVID-19 pandemic has developed rapidly and the ability to stratify the most vulnerable patients is vital. However, routinely used severity scoring systems are often low on diagnosis, even in non-survivors. Therefore, clinical prediction models for mortality are urgently required.', 'cite_spans': [], 'ref_spans': [], 'section': 'Summary'}, {'text': 'We developed and internally validated a multivariable logistic regression model to predict inpatient mortality in COVID-19 positive patients using data collected retrospectively from Tongji Hospital, Wuhan (299 patients). External validation was conducted using a retrospective cohort from Jinyintan Hospital, Wuhan (145 patients). Nine variables commonly measured in these acute settings were considered for model development, including age, biomarkers and comorbidities. Backwards stepwise selection and bootstrap resampling were used for model development and internal validation. We assessed discrimination via the C statistic, and calibration using calibration-in-the-large, calibration slopes and plots. 4 demonstrated that the model is able to make reliable predictions in patients from different hospitals. In addition, these variables agree with pathological mechanisms and the model is easy to use in all types of clinical settings.', 'cite_spans': [], 'ref_spans': [], 'section': 'Methods:'}, {'text': 'After further external validation in different countries the model will enable better risk stratification and more targeted management of patients with COVID-19. With the nomogram, this model that is based on readily available parameters can help clinicians to stratify COVID-19 patients on diagnosis to use limited healthcare resources effectively and improve patient outcome.', 'cite_spans': [], 'ref_spans': [], 'section': 'Implication of all the available evidence'}, {'text': 'Since the outbreak of coronavirus disease 2019 in December 2019 in China, there have been over 200,000 confirmed cases, with 10-20% developing severe COVID-19 and approximately 5% requiring intensive care 1, 2 . Although the number of daily cases has significantly reduced in China because of intensive control procedures, the numbers in the rest of world are continuing to increase. More than 10,000 deaths have been reported but the estimated mortality rate (3%) is much lower than Severe Acute Respiratory Syndrome (SARS) are not life-threatening, stratification of these patients becomes increasingly important as large populations are expected to be infected globally.', 'cite_spans': [{'start': 205, 'end': 207, 'text': '1,', 'ref_id': 'BIBREF0'}, {'start': 208, 'end': 209, 'text': '2', 'ref_id': 'BIBREF1'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'The deaths and morbidity from COVID-19 infection are primarily due to respiratory failure, although a few people have died of multiple organ failure (MOF) or chronic comorbidities 4,5 .', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'Therefore, reduced oxygen saturation is the major indicator of disease severity. However, symptoms at onset are relatively mild and a substantial proportion of patients have no obvious symptoms prior to the development of respiratory failure 4, 5 . Clinically, it is difficult to stratify patients who may develop lethal COVID-19 until respiratory failure develops. Since early identification and effective treatment can reduce mortality and morbidity as well as relieve resource shortages, there is an urgent need for effective prediction models to identify patients who are most likely to develop respiratory failure and poor outcomes.', 'cite_spans': [{'start': 242, 'end': 244, 'text': '4,', 'ref_id': 'BIBREF3'}, {'start': 245, 'end': 246, 'text': '5', 'ref_id': 'BIBREF5'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'A robust, highly discriminatory and validated clinical prognosis model is required for stratification of these patients. Currently, very few reports propose reliable prediction models, which have been constructed using Transparent Reporting of a multivariable prediction model for Individual Prognosis Or Diagnosis (TRIPOD) guidance with internal and external validation 6 . One report based on a cohort of 191 COVID patients in early stages of the pandemic showed that age, Sequential Organ Failure Assessment (SOFA) score and D-dimer are independent risk factors of mortality 7 . However, D-dimer is not a routinely available clinical measurement and SOFA can be relatively low in COVID-19 patients on admission, but in the aforementioned report some patients in late stage of COVID-19 were transferred to its study hospitals 7 . This might explain the difference in SOFA scores.', 'cite_spans': [{'start': 828, 'end': 829, 'text': '7', 'ref_id': 'BIBREF7'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'In this study, we aimed to employ routine, widely available clinical test data to predict mortality of COVID-19 hospitalised patients using multivariable regression analysis in a cohort of 299 cases. The model has been established and validated with a new cohort of 145 patients with COVID-19 from a different hospital. Through this study, we hope that the experience from the outbreak in China will assist the rest of the world in better stratification of COVID-19 patients to achieve better outcomes for patients.', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'This was a retrospective observational study performed in officially designated treatment centers for confirmed patients with COVID-19 in Wuhan at the center of the outbreak in China.', 'cite_spans': [], 'ref_spans': [], 'section': 'Study design and patients'}, {'text': 'The protocol was approved by the local Institutional Ethics Committee (Approval Number: KY-2020-10.02). First cohort of 299 patients admitted in Tongji hospital within January and February 2020 were enrolled into this study for model development. Second cohort of 151 patients admitted to Jinyintan hospital who were included in a previous study were reused for model validation. All patients were diagnosed by positive tests of novel coronavirus nucleic acids (SARS-CoV-2), according to WHO interim guidance 8 . Only patients who were discharged from hospital or had died were included in this study. Six patients in the second cohort, who died very quickly after admission and did not received any laboratory testing, were excluded. Patients younger than 18 years of age were also excluded.', 'cite_spans': [], 'ref_spans': [{'start': 70, 'end': 87, 'text': '(Approval Number:', 'ref_id': None}], 'section': 'Study design and patients'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Data collection'}, {'text': '(which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03. 28.20045997 doi: medRxiv preprint Clinical data, including demographic information, chronic comorbidities, biomarkers of infection and other laboratory tests were collected from the local Online Medical System. The time from onset of symptoms to hospital admission was also recorded. SOFA was calculated within 24h of admission. For patients who did not have arterial blood gas analysis, peripheral capillary oxygen saturation (SpO2) to estimate the arterial oxygen partial pressure (PaO2) and the recorded fraction of inspired oxygen (FiO2) were used. The SOFA score for the respiratory system was calculated according to PaO2/FiO2 (P/F) ratio when patients received non-invasive or invasive mechanical ventilation. All patients were closely followed until they died or were discharged from the hospital. Hospital mortality and length of hospital stay were also recorded.', 'cite_spans': [{'start': 104, 'end': 137, 'text': '28.20045997 doi: medRxiv preprint', 'ref_id': None}], 'ref_spans': [], 'section': 'Data collection'}, {'text': 'Data were fully anonymized before data cleaning and analyses. We followed the TRIPOD guidance for development, validation and reporting of multivariable prediction models 6 .', 'cite_spans': [{'start': 171, 'end': 172, 'text': '6', 'ref_id': 'BIBREF6'}], 'ref_spans': [], 'section': 'Statistical analysis'}, {'text': 'Analyses were performed in R version 3.5.1 9 . The development model was built using the data from Tongji Hospital. We performed logistic regression analysis with the outcome variable defined as mortality. Variables were selected a priori based on previous clinically-related studies, completeness across both sites, clinical knowledge and practicality of measurement in acute medical emergencies. Variables were excluded if they had high collinearity with global scores. The number of predictors was restricted based on the total number of outcomes in the development dataset 10 . Analysis was conducted for the following nine variables: age, hypertension, diabetes, SpO2, systolic blood pressure, albumin, lactate dehydrogenase (LDH), lymphocyte count and platelet count. Organ-specific markers, such as alanine transaminase (ALT) and blood urea nitrogen (BUN) were not included in the original development model because of collinearity with global markers, particularly LDH. SOFA score was not included for this reason, as explained in Results. D-dimer, interleukin (IL)-6 and cardiac troponin I (cTnI) were excluded from the analysis because they were not routinely tested in the cohorts. LDH . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 577, 'end': 579, 'text': '10', 'ref_id': 'BIBREF10'}], 'ref_spans': [], 'section': 'Statistical analysis'}, {'text': '(which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03. 28.20045997 doi: medRxiv preprint was selected as a general marker for cell injury or death and since blood gas analysis were not commonly used in these hospitals, SpO2 was selected to reflect respiratory function and was modelled as a continuous variable to maximize statistical power 11 . Albumin was selected as a marker for the acute phase response instead of C reactive protein (CRP) because of CRP collinearity with LDH. Age is known risk factor for mortality from severe respiratory infections, including COVID-19. Lymphocyte count and platelet counts were reported in a previous publication of COVID-19 and selected 7, 12 . In this study, lymphocyte count was log transformed due to extreme values. Systolic blood pressure (SBP), diabetes and hypertension were selected based on previous publications 7 . All nine selected variables were input into the multivariable model and backwards stepwise selection was performed with improvement in goodness-of-fit assessed by a reduction in the Akaike Information Criterion (AIC). A nomogram of the selected final model was used for graphical representation of the prediction model and was produced with R function regplotthis was chosen to offer maximum use of the prediction model in clinical practice without the need for internet access 13, 14 .', 'cite_spans': [{'start': 104, 'end': 115, 'text': '28.20045997', 'ref_id': None}, {'start': 390, 'end': 392, 'text': '11', 'ref_id': 'BIBREF11'}, {'start': 728, 'end': 730, 'text': '7,', 'ref_id': 'BIBREF7'}, {'start': 731, 'end': 733, 'text': '12', 'ref_id': 'BIBREF12'}, {'start': 1395, 'end': 1398, 'text': '13,', 'ref_id': 'BIBREF13'}, {'start': 1399, 'end': 1401, 'text': '14', 'ref_id': 'BIBREF14'}], 'ref_spans': [], 'section': 'Statistical analysis'}, {'text': 'Model performance was assessed via measures of discrimination and calibration. To assess discrimination, the C statistic was used. For calibration, patients in the development database were split into deciles, ordered by their probability of death. For each decile, the mean of the predicted probability of death was calculated and compared with the mean observed probability of death. Calibration plots were constructed including locally estimated scatterplot smoothing (loess) regression lines 15 .', 'cite_spans': [{'start': 496, 'end': 498, 'text': '15', 'ref_id': 'BIBREF15'}], 'ref_spans': [], 'section': 'Statistical analysis'}, {'text': 'Internal validity was assessed with bootstrapping (1000 replications) of the entire model building process including backwards stepwise selection of all potential predictors. is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Statistical analysis'}, {'text': 'The copyright holder for this preprint . https://doi.org/10.1101/2020.03. 28.20045997 doi: medRxiv preprint calculation of optimism and optimism adjusted discrimination and calibration statistics. The final development model coefficients were then adjusted for optimism using the optimism adjusted calibration slope for shrinkage 15 .', 'cite_spans': [{'start': 74, 'end': 85, 'text': '28.20045997', 'ref_id': None}, {'start': 330, 'end': 332, 'text': '15', 'ref_id': 'BIBREF15'}], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'To assess external validity, the final optimism-adjusted model was applied to an independent data set, from Jinyintan hospital. External validity of the model was assessed using the C statistic, calibration-in-the-large, calibration slope and the calibration plot. Recalibration was conducted updating the intercept, and the intercept and slope.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'The development cohort included 299 patients who were admitted in Tongji Hospital. Data completeness is presented in supplementary Table 1 . There were 155 deaths, the median age of patients was 65 years (IQR: 54-73) and 48·2% were male ( Table 1 ). The validation cohort that included 145 patients was younger: age 56 years (IQR 47-68) with a higher percentage of males (65·5%) than in the developmental cohort (P<0·001). The median (IQR) time from onset of symptoms to hospital admission was no different in the developmental 10 [7] [8] [9] [10] [11] [12] [13] [14] and validation 10 [7] [8] [9] [10] [11] [12] [13] cohorts. More patients had hypertension (42·6% vs 26·9%) and heart failure (4·4% vs 0.7%) in the development than validation cohorts, whilst diabetes (18·5% vs 11·7%), chronic obstructive pulmonary disease (COPD) (5·0% vs 3·4%) and others (6·4% vs 4·8) showed no statistical difference. SOFA scores were statistically higher in the development than validation cohorts (2·0 [2·0-4·0] vs 1·0 [0·0-3·0]) but both were very low compared to common critical illnesses. Table 2 shows that non-survivors were significantly older than survivors (69 years (IQR 61·75 to 75·00) vs. 52·50 (IQR 43·75-64·00), p<0·001). Patients with hypertension, showed a significantly higher mortality rate (p<0·001).', 'cite_spans': [{'start': 531, 'end': 534, 'text': '[7]', 'ref_id': 'BIBREF7'}, {'start': 535, 'end': 538, 'text': '[8]', 'ref_id': 'BIBREF8'}, {'start': 539, 'end': 542, 'text': '[9]', 'ref_id': 'BIBREF9'}, {'start': 543, 'end': 547, 'text': '[10]', 'ref_id': 'BIBREF10'}, {'start': 548, 'end': 552, 'text': '[11]', 'ref_id': 'BIBREF11'}, {'start': 553, 'end': 557, 'text': '[12]', 'ref_id': 'BIBREF12'}, {'start': 558, 'end': 562, 'text': '[13]', 'ref_id': 'BIBREF13'}, {'start': 563, 'end': 567, 'text': '[14]', 'ref_id': 'BIBREF14'}, {'start': 586, 'end': 589, 'text': '[7]', 'ref_id': 'BIBREF7'}, {'start': 590, 'end': 593, 'text': '[8]', 'ref_id': 'BIBREF8'}, {'start': 594, 'end': 597, 'text': '[9]', 'ref_id': 'BIBREF9'}, {'start': 598, 'end': 602, 'text': '[10]', 'ref_id': 'BIBREF10'}, {'start': 603, 'end': 607, 'text': '[11]', 'ref_id': 'BIBREF11'}, {'start': 608, 'end': 612, 'text': '[12]', 'ref_id': 'BIBREF12'}, {'start': 613, 'end': 617, 'text': '[13]', 'ref_id': 'BIBREF13'}], 'ref_spans': [{'start': 131, 'end': 138, 'text': 'Table 1', 'ref_id': 'TABREF0'}, {'start': 239, 'end': 246, 'text': 'Table 1', 'ref_id': 'TABREF0'}, {'start': 1081, 'end': 1088, 'text': 'Table 2', 'ref_id': 'TABREF2'}], 'section': 'Patient cohorts'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Model development'}, {'text': 'The copyright holder for this preprint . https://doi.org/10.1101/2020.03. 28.20045997 doi: medRxiv preprint In the developmental cohort (Tongji Hospital), 268/299 patients had complete data for all nine variables. The final multivariable model included age (adjusted OR 1·054; 95% CI 1·028 to 1·083), LDH (adjusted OR 1·004; 95% CI 1·002 to 1·006), log lymphocyte count (adjusted OR 0·296; 95% CI 0·148 to 0·541) and SpO2 (adjusted OR 0·897; 95% CI 0·831 to 0·958) ( Table 3 ). The final model showed excellent discrimination (C statistic = 0·89) and was well calibrated (slope =1 and calibration-in-the-large =0·00), as shown in the calibration plot (Table   4 and Figure 1 ). The final model for mortality is presented as a nomogram and an example of how to use the nomogram is presented for a patient aged 59, with LDH of 482, SpO2 of 85% and lymphocyte count of 0·64 ( Figure 2 ).', 'cite_spans': [{'start': 74, 'end': 107, 'text': '28.20045997 doi: medRxiv preprint', 'ref_id': None}], 'ref_spans': [{'start': 467, 'end': 474, 'text': 'Table 3', 'ref_id': 'TABREF4'}, {'start': 651, 'end': 661, 'text': '(Table   4', 'ref_id': None}, {'start': 666, 'end': 674, 'text': 'Figure 1', 'ref_id': 'FIGREF6'}, {'start': 873, 'end': 881, 'text': 'Figure 2', 'ref_id': None}], 'section': '(which was not peer-reviewed)'}, {'text': 'Interval validation was run using bootstrap resampling and showed a small amount of overfitting. Bootstrapping with backwards stepwise selection provided a shrinkage factor of 0.900. After adjusting for overfitting the final model retained high discrimination (C statistic= 0·880) and calibration-in-the-large of 0·006 (Table 4 ). The shrinkage factor was applied to the β coefficients of the multivariable model to provide optimism adjusted coefficients (Table 3) .', 'cite_spans': [], 'ref_spans': [{'start': 319, 'end': 327, 'text': '(Table 4', 'ref_id': None}, {'start': 455, 'end': 464, 'text': '(Table 3)', 'ref_id': 'TABREF4'}], 'section': 'Internal validation'}, {'text': 'External validation is required because the accuracy of a predictive model will always be high if it is validated on the development cohort used to generate the model 16, 17 . In this study, we applied our final prediction model with optimism adjusted coefficients to the validation dataset.', 'cite_spans': [{'start': 167, 'end': 170, 'text': '16,', 'ref_id': 'BIBREF16'}, {'start': 171, 'end': 173, 'text': '17', 'ref_id': 'BIBREF17'}], 'ref_spans': [], 'section': 'External validation'}, {'text': 'There were 145 patients in the validation dataset of whom 69 patients (47·6%) died. Of the 145 patients, 127 had complete data for the four variables in the final developmental model (age, lymphocyte count, SpO2 and LDH) The C statistic for the discrimination of the developed model in the independent data was 0·980 (0·958 to 1·000) and the model had reasonable calibration (slope and calibration-in-the-large) ( Table 5 ). Figure 3 suggests some miscalibration of the model in the independent data with over-estimation of risk of mortality in . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [{'start': 414, 'end': 421, 'text': 'Table 5', 'ref_id': 'TABREF5'}, {'start': 425, 'end': 433, 'text': 'Figure 3', 'ref_id': 'FIGREF7'}], 'section': 'External validation'}, {'text': 'The copyright holder for this preprint . https://doi.org/10.1101/2020.03. 28.20045997 doi: medRxiv preprint low-risk individuals and under-estimation of risk in high-risk individuals. After recalibration of the intercept, the discrimination of the model was the same (as expected given that recalibration does not influence the ranking of individuals according to the model) as was the calibration slope. Calibration-in-the-large was 0·00 (-0·477 to 0·471). The calibration plots still suggest mis-calibration of the model (Figure 3c ). When both intercept and slope were recalibrated the calibration slope was 1·00 (0·675 to 1·477) and the plot then demonstrated good calibration (Figure 3d ).', 'cite_spans': [{'start': 74, 'end': 85, 'text': '28.20045997', 'ref_id': None}], 'ref_spans': [{'start': 523, 'end': 533, 'text': '(Figure 3c', 'ref_id': 'FIGREF7'}, {'start': 681, 'end': 691, 'text': '(Figure 3d', 'ref_id': 'FIGREF7'}], 'section': '(which was not peer-reviewed)'}, {'text': 'Most routine scoring systems, such as SOFA, which are very useful in the intensive care unit is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'The copyright holder for this preprint . https://doi.org/10.1101/2020.03. 28.20045997 doi: medRxiv preprint of the final model after optimization can be used by clinicians to give a prediction of mortality and thus inform treatment choice and guide patient and family counselling.', 'cite_spans': [{'start': 74, 'end': 85, 'text': '28.20045997', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'Old age is a major factor of mortality from infection 21 . This is consistent with the current data from China on COVID-19 with less than 1% in children <10 years old, increasing to 2%, 4% and 10% in age 11-40, 41-70, and >70, respectively. However, the majority of people infected are 30-70 years old (approximately 80% of the total cases), thereby making it difficult to stratify the majority of cases according to age. Patient comorbidities, including hypertension, diabetes, did not significantly contribute to the model, although mortality rates were significantly higher in patients with hypertension.', 'cite_spans': [{'start': 54, 'end': 56, 'text': '21', 'ref_id': 'BIBREF21'}], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'The pathological feature of COVID-19 is mainly that of a viral pneumonia with alveolar oedema and blockage of small bronchi to compromise gas exchange within the lungs.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'Therefore, PaO2 will drop when severe pneumonia develops. Since arterial blood gas (ABG) analysis is an invasive and complex process, which requires arterial blood taking and use of a ABG analyzer, SpO2 measurement is much more used for continuous analysis of blood oxygen saturation in patients to estimate the arterial oxygen partial pressure (PaO2) 22 . Moreover, many mobile phone brands contain built-in sensors and SpO2 can be measured by patients themselves.', 'cite_spans': [{'start': 352, 'end': 354, 'text': '22', 'ref_id': 'BIBREF22'}], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'In our cohorts, many patients had no blood gas data and therefore SpO2 was used as a variable in our model. Although SpO2 is not as accurate as PaO2 in critical illnesses, it could give the first indication of compromised lung gas exchange in the early stages of COVID-19, which would then be verified by ABG measurement upon admission to hospital.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'particularly in sepsis 23, 24 . CD4 and CD8 T cell apoptosis causes prolonged infection by promoting virus survival 25 , and induces immune suppression to increase mortality in patients with sepsis 26 . In this study, we found a dramatic difference in lymphocyte counts between survivors and non-survivors. In non-survivors, the peripheral lymphocyte counts were very low . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 23, 'end': 26, 'text': '23,', 'ref_id': 'BIBREF23'}, {'start': 27, 'end': 29, 'text': '24', 'ref_id': 'BIBREF24'}, {'start': 116, 'end': 118, 'text': '25', 'ref_id': 'BIBREF25'}, {'start': 198, 'end': 200, 'text': '26', 'ref_id': 'BIBREF26'}], 'ref_spans': [], 'section': 'Lymphopenia is very common in patients with influenza virus infection and bacterial infection,'}, {'text': 'The copyright holder for this preprint . https://doi.org/10.1101/2020.03. 28.20045997 doi: medRxiv preprint on admission, indicating that extensive lymphocyte death occurred following infection. LDH is released into circulation after cell injury or death 27 and serves as an index of the extent of cellular damage by the virus or the host immune response 28 .', 'cite_spans': [{'start': 74, 'end': 85, 'text': '28.20045997', 'ref_id': None}, {'start': 255, 'end': 257, 'text': '27', 'ref_id': 'BIBREF27'}, {'start': 355, 'end': 357, 'text': '28', 'ref_id': 'BIBREF28'}], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'This study has several limitations. At this stage in the global outbreak of COVID-19, no published studies have developed and validated prediction models for assessing the probability of mortality in hospitalized patients. A most recent report proposed SOFA score, age and Ddimers as the important prognostic markers 7 . However, in our cohorts, the SOFA score was very low although there is statistical difference between survivors and non-survivors. This difference may be due to some patients being treated in other hospitals prior to transfer in the previously reported study 7 . In addition, SOFA score relies on blood gas analysis, which cannot be easily done continually, in clinics and in under-resourced settings, thus making it difficult for clinicians to use readily. In addition, since D-dimer is not routinely requested, too many patients in our cohorts had no test performed, making it is difficult for us to verify published results. Organ-specific injury markers were not included in the model, including cTnI due to missing data (not routinely measured), as well as ALT and BUN due to high co-linearity with other variables within the model. Therefore, the choice of potential prognostic factors could not be informed by a systematic review of existing prediction models. Similarly, there are no published estimates of discrimination and calibration to compare this study to. Additionally, the event rate is limited, especially in the independent dataset where at least 100 events would be preferable 29 . A multicenter study with a larger cohort for model development and also for validation would add more power, reduce bias, and make the findings more generically applicable.', 'cite_spans': [{'start': 580, 'end': 581, 'text': '7', 'ref_id': 'BIBREF7'}, {'start': 1518, 'end': 1520, 'text': '29', 'ref_id': 'BIBREF29'}], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'There is also inherent bias in the recruitment of patients because the Jinyintan hospital was the first officially designated receiving hospital in Wuhan for all COVID-19 cases at the start of the outbreak. The recruited patients were mainly self-referrals, but some were referred by other . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'The copyright holder for this preprint . https://doi.org/10.1101/2020.03. 28.20045997 doi: medRxiv preprint hospitals immediately after a positive diagnostic test. Self-referred patients are more likely to have severe symptoms that cause them to seek emergency medical help. Likewise, other hospitals are more likely to test patients who have more pronounced symptoms, thus creating an inherent bias in the patient sample. Therefore, the cohort may not accurately represent patients with mild or asymptomatic COVID-19. These results need to be further validated with patients who are hospitalized for different severities of illness. In conclusion, traditional early warning scores and organ injury markers do not help to predict severity and mortality in patients with COVID-19. Here, we have developed and validated a clinical prediction model using readily available clinical markers which is able to accurately predict mortality in people with COVID-19. This model can therefore be used in current pandemic regions to identify patients who are at risk of severe disease at an early enough stage to initialize intensive supportive treatment and to guide discussions between clinicians, patients and families. It has the potential to be used throughout the world following additional validation . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 74, 'end': 85, 'text': '28.20045997', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'The copyright holder for this preprint . https://doi.org/10.1101/2020.03. 28.20045997 doi: medRxiv preprint in relevant worldwide datasets, with recalibration as necessary to account for differing risk profiles of patients. Role of the funding source: The sponsors had not been involved in study design, data collection, data analysis, data interpretation, or writing of the report. The corresponding authors had full access to all the data and had final responsibility for this study. DH is funded outside the scope of work by an NIHR Post-Doctoral Fellowship (PDF-2018-11-ST2-006). . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 74, 'end': 107, 'text': '28.20045997 doi: medRxiv preprint', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'The copyright holder for this preprint . https://doi.org/10.1101/2020.03. 28 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 74, 'end': 76, 'text': '28', 'ref_id': 'BIBREF28'}], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'The copyright holder for this preprint . https://doi.org/10.1101/2020.03. 28 1·000 (0·773 to 1·264)) 0·100 0·900 1·00 (0·675 to 1·477)', 'cite_spans': [{'start': 74, 'end': 76, 'text': '28', 'ref_id': 'BIBREF28'}], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': '(which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03. 28.20045997 doi: medRxiv preprint is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 104, 'end': 115, 'text': '28.20045997', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': '(which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03. 28.20045997 doi: medRxiv preprint Figure 2 Nomogram of the final development model. The nomogram presents the distribution of continuous variables and for categorical variables the relative contribution of variable to the model is reflected in the size. The distribution of the final score is shown for the development data and the red dots represent a randomly selected patient from the development dataset with probability of mortality of 0.77.', 'cite_spans': [{'start': 104, 'end': 115, 'text': '28.20045997', 'ref_id': None}], 'ref_spans': [{'start': 138, 'end': 146, 'text': 'Figure 2', 'ref_id': None}], 'section': '(which was not peer-reviewed)'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': '(which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03. 28.20045997 doi: medRxiv preprint . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 104, 'end': 115, 'text': '28.20045997', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': '(which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03. 28.20045997 doi: medRxiv preprint ', 'cite_spans': [{'start': 104, 'end': 137, 'text': '28.20045997 doi: medRxiv preprint', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'WHO. Coronavirus disease 2019 (COVID-19) Situation Report. 2020', 'authors': [], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF1': {'ref_id': 'b1', 'title': 'A Novel Coronavirus from Patients with Pneumonia in China', 'authors': [{'first': 'N', 'middle': [], 'last': 'Zhu', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Wang', 'suffix': ''}], 'year': 2019, 'venue': 'N Engl J Med', 'volume': '382', 'issn': '8', 'pages': '727--760', 'other_ids': {}}, 'BIBREF2': {'ref_id': 'b2', 'title': 'Coronavirus Infections-More Than Just the Common Cold', 'authors': [{'first': 'C', 'middle': ['I'], 'last': 'Paules', 'suffix': ''}, {'first': 'H', 'middle': ['D'], 'last': 'Marston', 'suffix': ''}, {'first': 'A', 'middle': ['S'], 'last': 'Fauci', 'suffix': ''}], 'year': 2020, 'venue': 'JAMA', 'volume': '323', 'issn': '8', 'pages': '707--715', 'other_ids': {}}, 'BIBREF3': {'ref_id': 'b3', 'title': 'Clinical Characteristics of 138 Hospitalized Patients With', 'authors': [{'first': 'D', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Hu', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Hu', 'suffix': ''}], 'year': 2019, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF4': {'ref_id': 'b4', 'title': 'Novel Coronavirus-Infected Pneumonia in Wuhan, China', 'authors': [], 'year': 2020, 'venue': 'JAMA', 'volume': '323', 'issn': '11', 'pages': '1061--1070', 'other_ids': {}}, 'BIBREF5': {'ref_id': 'b5', 'title': 'Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China', 'authors': [{'first': 'C', 'middle': [], 'last': 'Huang', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Li', 'suffix': ''}], 'year': 2020, 'venue': 'Lancet', 'volume': '395', 'issn': '', 'pages': '497--506', 'other_ids': {}}, 'BIBREF6': {'ref_id': 'b6', 'title': 'Transparent reporting of a multivariable prediction model for individual prognosis or diagnosis (TRIPOD): the TRIPOD statement', 'authors': [{'first': 'G', 'middle': ['S'], 'last': 'Collins', 'suffix': ''}, {'first': 'J', 'middle': ['B'], 'last': 'Reitsma', 'suffix': ''}, {'first': 'D', 'middle': ['G'], 'last': 'Altman', 'suffix': ''}, {'first': 'K', 'middle': ['G'], 'last': 'Moons', 'suffix': ''}], 'year': 2015, 'venue': 'BMJ', 'volume': '350', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF7': {'ref_id': 'b7', 'title': 'Clinical course and risk factors for mortality of adult inpatients with COVID-19 in Wuhan, China: a retrospective cohort study', 'authors': [{'first': 'F', 'middle': [], 'last': 'Zhou', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Yu', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Du', 'suffix': ''}], 'year': 2020, 'venue': 'Lancet', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF8': {'ref_id': 'b8', 'title': 'Laboratory testing for coronavirus disease (COVID-19) in suspected human cases', 'authors': [], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF9': {'ref_id': 'b9', 'title': 'R: A language and environment for statistical computing. R Foundation for Statistical Computing', 'authors': [{'first': 'R', 'middle': ['C'], 'last': 'Team', 'suffix': ''}], 'year': 2019, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF10': {'ref_id': 'b10', 'title': 'A simulation study of the number of events per variable in logistic regression analysis', 'authors': [{'first': 'P', 'middle': [], 'last': 'Peduzzi', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Concato', 'suffix': ''}, {'first': 'E', 'middle': [], 'last': 'Kemper', 'suffix': ''}, {'first': 'T', 'middle': ['R'], 'last': 'Holford', 'suffix': ''}, {'first': 'A', 'middle': ['R'], 'last': 'Feinstein', 'suffix': ''}], 'year': 1996, 'venue': 'J Clin Epidemiol', 'volume': '49', 'issn': '12', 'pages': '1373--1382', 'other_ids': {}}, 'BIBREF11': {'ref_id': 'b11', 'title': 'The cost of dichotomising continuous variables', 'authors': [{'first': 'D', 'middle': ['G'], 'last': 'Altman', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Royston', 'suffix': ''}], 'year': 2006, 'venue': 'BMJ', 'volume': '332', 'issn': '7549', 'pages': '', 'other_ids': {}}, 'BIBREF12': {'ref_id': 'b12', 'title': 'Functional exhaustion of antiviral lymphocytes in COVID-19 patients', 'authors': [{'first': 'M', 'middle': [], 'last': 'Zheng', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Gao', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Wang', 'suffix': ''}], 'year': 2020, 'venue': 'Cel& Mol Immu', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF13': {'ref_id': 'b13', 'title': 'Guide to presenting clinical prediction models for use in clinical settings', 'authors': [{'first': 'L', 'middle': ['J'], 'last': 'Bonnett', 'suffix': ''}, {'first': 'Kie', 'middle': [], 'last': 'Snell', 'suffix': ''}, {'first': 'G', 'middle': ['S'], 'last': 'Collins', 'suffix': ''}, {'first': 'R', 'middle': ['D'], 'last': 'Riley', 'suffix': ''}], 'year': 2019, 'venue': 'BMJ', 'volume': '365', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF14': {'ref_id': 'b14', 'title': 'Enhanced Regression Nomogram Plot. R package version 10', 'authors': [{'first': 'Marshall', 'middle': ['R'], 'last': 'Regplot', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF15': {'ref_id': 'b15', 'title': 'Clinical prediction models', 'authors': [{'first': 'E', 'middle': ['W'], 'last': 'Steyerberg', 'suffix': ''}], 'year': 2019, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF16': {'ref_id': 'b16', 'title': 'External validation is necessary in prediction research: a clinical example', 'authors': [{'first': 'S', 'middle': ['E'], 'last': 'Bleeker', 'suffix': ''}, {'first': 'H', 'middle': ['A'], 'last': 'Moll', 'suffix': ''}, {'first': 'E', 'middle': ['W'], 'last': 'Steyerberg', 'suffix': ''}], 'year': 2003, 'venue': 'J Clin Epidemiol', 'volume': '56', 'issn': '9', 'pages': '826--858', 'other_ids': {}}, 'BIBREF17': {'ref_id': 'b17', 'title': 'External validation of the Intensive Care National Audit & Research Centre (ICNARC) risk prediction model in critical care units in Scotland', 'authors': [{'first': 'D', 'middle': ['A'], 'last': 'Harrison', 'suffix': ''}, {'first': 'N', 'middle': ['I'], 'last': 'Lone', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Haddow', 'suffix': ''}], 'year': 2014, 'venue': 'BMC Anesthesiol', 'volume': '14', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF18': {'ref_id': 'b18', 'title': 'Prognostic Accuracy of the SOFA Score, SIRS Criteria, and qSOFA Score for In-Hospital Mortality Among Adults With Suspected Infection Admitted to the Intensive Care Unit', 'authors': [{'first': 'E', 'middle': ['P'], 'last': 'Raith', 'suffix': ''}, {'first': 'A', 'middle': ['A'], 'last': 'Udy', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Bailey', 'suffix': ''}], 'year': 2017, 'venue': 'JAMA', 'volume': '317', 'issn': '3', 'pages': '290--300', 'other_ids': {}}, 'BIBREF19': {'ref_id': 'b19', 'title': 'Prognostic Accuracy of Sepsis-3 Criteria for In-Hospital Mortality Among Patients With Suspected Infection Presenting to the Emergency Department', 'authors': [{'first': 'Y', 'middle': [], 'last': 'Freund', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Lemachatti', 'suffix': ''}, {'first': 'E', 'middle': [], 'last': 'Krastinova', 'suffix': ''}], 'year': 2017, 'venue': 'JAMA', 'volume': '317', 'issn': '3', 'pages': '301--309', 'other_ids': {}}, 'BIBREF20': {'ref_id': 'b20', 'title': 'Quick Sepsis-related Organ Failure Assessment, Systemic Inflammatory Response Syndrome, and Early Warning Scores for Detecting Clinical Deterioration in Infected Patients outside the Intensive Care Unit', 'authors': [{'first': 'M', 'middle': ['M'], 'last': 'Churpek', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Snyder', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Han', 'suffix': ''}], 'year': 2017, 'venue': 'Am J Respir Crit Care Med', 'volume': '195', 'issn': '7', 'pages': '906--917', 'other_ids': {}}, 'BIBREF21': {'ref_id': 'b21', 'title': 'Causes of Death. Our World in Data', 'authors': [{'first': 'H', 'middle': [], 'last': 'Ritchie', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Roser', 'suffix': ''}], 'year': 2019, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF22': {'ref_id': 'b22', 'title': 'Accuracy of pulse oximetry in detection of oxygen saturation in patients admitted to the intensive care unit of heart surgery: comparison of finger, toe, forehead and earlobe probes', 'authors': [{'first': 'S', 'middle': [], 'last': 'Seifi', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Khatony', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Moradi', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Abdi', 'suffix': ''}, {'first': 'F', 'middle': [], 'last': 'Najafi', 'suffix': ''}], 'year': 2018, 'venue': 'BMC Nurs', 'volume': '17', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF23': {'ref_id': 'b23', 'title': 'Persistent lymphopenia after diagnosis of sepsis predicts mortality', 'authors': [{'first': 'A', 'middle': ['M'], 'last': 'Drewry', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Samra', 'suffix': ''}, {'first': 'L', 'middle': ['P'], 'last': 'Skrupky', 'suffix': ''}, {'first': 'B', 'middle': ['M'], 'last': 'Fuller', 'suffix': ''}, {'first': 'S', 'middle': ['M'], 'last': 'Compton', 'suffix': ''}, {'first': 'R', 'middle': ['S'], 'last': 'Hotchkiss', 'suffix': ''}], 'year': 2014, 'venue': 'Shock', 'volume': '42', 'issn': '5', 'pages': '383--91', 'other_ids': {}}, 'BIBREF24': {'ref_id': 'b24', 'title': 'Lymphocytosis and Lymphopenia Induced by Imported Infectious Diseases: A Controlled Cross-Sectional Study of 17,229 Diseased German Travelers Returning from the Tropics and Subtropics', 'authors': [{'first': 'K', 'middle': ['H'], 'last': 'Herbinger', 'suffix': ''}, {'first': 'I', 'middle': [], 'last': 'Hanus', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Beissner', 'suffix': ''}], 'year': 2016, 'venue': 'Am J Trop Med Hyg', 'volume': '94', 'issn': '6', 'pages': '1385--91', 'other_ids': {}}, 'BIBREF25': {'ref_id': 'b25', 'title': 'CD4 T cells promote CD8 T cell immunity at the priming and effector site during viral encephalitis', 'authors': [{'first': 'T', 'middle': ['W'], 'last': 'Phares', 'suffix': ''}, {'first': 'S', 'middle': ['A'], 'last': 'Stohlman', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Hwang', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Min', 'suffix': ''}, {'first': 'D', 'middle': ['R'], 'last': 'Hinton', 'suffix': ''}, {'first': 'C', 'middle': ['C'], 'last': 'Bergmann', 'suffix': ''}], 'year': 2012, 'venue': 'J Virol', 'volume': '86', 'issn': '5', 'pages': '2416--2443', 'other_ids': {}}, 'BIBREF26': {'ref_id': 'b26', 'title': 'Pathological alteration and therapeutic implications of sepsis-induced immune cell apoptosis', 'authors': [{'first': 'C', 'middle': [], 'last': 'Cao', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Yu', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Chai', 'suffix': ''}], 'year': 2019, 'venue': 'Cell Death Dis', 'volume': '10', 'issn': '10', 'pages': '', 'other_ids': {}}, 'BIBREF27': {'ref_id': 'b27', 'title': 'Analysis of Cell Viability by the Lactate Dehydrogenase Assay', 'authors': [{'first': 'P', 'middle': [], 'last': 'Kumar', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Nagarajan', 'suffix': ''}, {'first': 'P', 'middle': ['D'], 'last': 'Uchil', 'suffix': ''}], 'year': 2018, 'venue': 'Cold Spring Harb Protoc', 'volume': '', 'issn': '6', 'pages': '', 'other_ids': {}}, 'BIBREF28': {'ref_id': 'b28', 'title': 'Diagnostic and prognostic value of very high serum lactate dehydrogenase in admitted medical patients', 'authors': [{'first': 'A', 'middle': [], 'last': 'Erez', 'suffix': ''}, {'first': 'O', 'middle': [], 'last': 'Shental', 'suffix': ''}, {'first': 'J', 'middle': ['Z'], 'last': 'Tchebiner', 'suffix': ''}], 'year': 2014, 'venue': 'Isr Med Assoc J', 'volume': '16', 'issn': '7', 'pages': '439--482', 'other_ids': {}}, 'BIBREF29': {'ref_id': 'b29', 'title': 'Sample size considerations for the external validation of a multivariable prognostic model: a resampling study', 'authors': [{'first': 'G', 'middle': ['S'], 'last': 'Collins', 'suffix': ''}, {'first': 'E', 'middle': ['O'], 'last': 'Ogundimu', 'suffix': ''}, {'first': 'D', 'middle': ['G'], 'last': 'Altman', 'suffix': ''}], 'year': 2016, 'venue': 'Stat Med', 'volume': '35', 'issn': '2', 'pages': '214--240', 'other_ids': {}}}, 'ref_entries': {'FIGREF0': {'text': 'The final model included age, lymphocyte count, lactate dehydrogenase and SpO2 as independent predictors of mortality. Discrimination of the model was excellent in both internal (c=0·89) and external (c=0·98) validation. Internal calibration was excellent (calibration slope=1). External validation showed some over-prediction of risk in low-risk individuals and under-prediction of risk in high-risk individuals prior to recalibration. Recalibration of the intercept and slope led to excellent performance of the model in independent data. Interpretation: COVID-19 is a new disease and behaves differently from common critical illnesses. This study provides a new prediction model to identify patients with lethal COVID-19. Its practical reliance on commonly available parameters should improve usage of limited healthcare resources and patient survival rate.', 'latex': None, 'type': 'figure'}, 'FIGREF1': {'text': '9·6%, 774 died of 8096 infected) and Middle East Respiratory Syndrome (MERS) in 2012 (34·4%, 858 died of 2494 infected) 3 . Although the vast majority of cases of COVID-19', 'latex': None, 'type': 'figure'}, 'FIGREF2': {'text': 'Bootstrapped samples were created by drawing random samples with replacement from the development database. The prediction model was fitted on each bootstrap sample and tested on the original sample. Overfitting was assessed in each bootstrap replication. This allowed', 'latex': None, 'type': 'figure'}, 'FIGREF3': {'text': 'ICU) for sepsis and other critical illnesses 18-20 , were very low in COVID-19 patients on admission with median SOFA score of 2. However, 155/299 and 69/145 patients in the development and validation cohorts died in the hospital, respectively. This poor outcome strongly indicates that these routine scoring systems used in general wards and ICU cannot accurately assess the severity and predict the mortality of patients with COVID-19. We have therefore used a hospitalized study population from Wuhan Tongji Hospital to develop a clinical prediction model using available demographic and clinical indicators. The most informative factors in COVID-19 positive patients were age, LDH, lymphocyte count and SpO2. The developed statistical model had good discrimination and minimal model optimism. In external validation using an independent cohort from Jinyintan hospital, the model had excellent discrimination but needed recalibrating to account for the different underlying risk profile in the independent dataset. Specifically, the development and validation cohorts are temporally distinct as the validation cohort were from earlier in the COVID-19 outbreak. In addition, they are from different sites within Wuhan and the demographics are distinct, with patients admitted to the Jinyintan hospital being younger with a male majority and less patients with hypertension and low SOFA scores (1.0 [0-3] vs 2.0 [2.0-4.0]. However, the nomogram', 'latex': None, 'type': 'figure'}, 'FIGREF4': {'text': 'Despite these limitations, the obvious advantage of the model developed here is that the panel of variables used to build the model are basic demographics and values from accessible clinical tests that are recorded routinely in the acute clinical setting. With the vivid nomogram, this model will enable the majority of clinicians to assess the severity of patient illness without difficulty, particular in very busy clinical settings. We were also able to externally validate our model in an independent dataset. The discrimination of the model was high in both internal and external validation showing that the model is able to accurately differentiate high and low risk individuals. The prediction models are based on and validated in Chinese hospitalized COVID-19 positive populations and should therefore be applicable to other sites within China and may also be translatable into other sites during the global outbreak. However, separate validation will be required in other patient populations before widespread application.', 'latex': None, 'type': 'figure'}, 'FIGREF5': {'text': 'This study was supported by following funding: Ministry of Science and Technology (2020YFC0841300 and 2020YFC0843700) of China, National Institute for Health Research (NIHR) (PDF-2018-11-ST2-006), British Heart Foundation (BHF) (PG/16/65/32313) and Liverpool University Hospitals NHS Foundation Trust in UK. We also thank all the patients and medical staff involved. Author Contribution: JX, HC, SL, GW, BD and HQ had the idea for and designed the study; JX, HC, SL, YW, HK, RZ, XL and ZT collected the clinical data. DH, STA and LB did the statistical analysis and wrote part of the manuscript. GW and CHT wrote the manuscript. All authors contributed to acquisition, analysis, or interpretation of data. All authors revised the report and approved the final version before submission. Declaration of Interests: All the authors declare no conflict of interest.', 'latex': None, 'type': 'figure'}, 'FIGREF6': {'text': 'Calibration plot of the final development model.', 'latex': None, 'type': 'figure'}, 'FIGREF7': {'text': 'Calibration plots of external validation. A) Calibration without shrinkage B) Calibration with shrinkage of final model coefficients. C) Recalibration with intercept updated. D) Recalibration with slope and intercept updated.', 'latex': None, 'type': 'figure'}, 'TABREF0': {'text': 'Basic demographics and clinical features of patients in the developmental and validation datasets.(For continuous variables median and IQR are reported)', 'latex': None, 'type': 'table'}, 'TABREF2': {'text': 'Differences in demographics between COVID-19 patients who survived and died in developmental and', 'latex': None, 'type': 'table'}, 'TABREF3': {'text': \"Data are presented as median (IQR), or n (%). P values were calculated using Wilcoxon rank-sum test, χ 2 or Fisher's exact test. ALT= Alanine aminotransferase. AST= Aspartate aminotransferase. BUN=Blood urea nitrogen. COPD=chronic obstructive disease. CRP= C-reactive protein. hs-cTnI= high-sensitivity cardiac troponin I. INR= international normalized ratio. LDH= lactate dehydrogenase. SOFA= sequential organ failure assessment. SPO2= pulse oxygen saturation. WBC=white blood cell.\", 'latex': None, 'type': 'table'}, 'TABREF4': {'text': 'Final multivariable model in development dataset and optimism adjusted β coefficients', 'latex': None, 'type': 'table'}, 'TABREF5': {'text': 'External validation and recalibration: model performance', 'latex': None, 'type': 'table'}}, 'back_matter': []}\n",
      "{'paper_id': 'af7d038be406d73b373f05a9cdfaf62d98e26cbb', 'metadata': {'title': 'Mathematical modeling of COVID-19 transmission and mitigation strategies in the population of Ontario, Canada', 'authors': [{'first': 'Ashleigh', 'middle': ['R'], 'last': 'Tuite', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Toronto', 'location': {'region': 'Ontario', 'country': 'Canada'}}, 'email': 'ashleigh.tuite@utoronto.ca'}, {'first': 'David', 'middle': ['N'], 'last': 'Fisman', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Toronto', 'location': {'region': 'Ontario', 'country': 'Canada'}}, 'email': ''}, {'first': 'Amy', 'middle': ['L'], 'last': 'Greer', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Guelph', 'location': {'addrLine': '155 College Street', 'postCode': 'M5T 3M7', 'settlement': 'Guelph, Toronto', 'region': 'Ontario, Ontario', 'country': 'Canada'}}, 'email': ''}, {'first': 'Ashleigh', 'middle': [], 'last': 'Tuite', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Guelph', 'location': {'addrLine': '155 College Street', 'postCode': 'M5T 3M7', 'settlement': 'Guelph, Toronto', 'region': 'Ontario, Ontario', 'country': 'Canada'}}, 'email': ''}]}, 'abstract': [{'text': 'We evaluated how non-pharmaceutical interventions could be used to control the COVID-19 pandemic and reduce the burden on the healthcare system.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'Using an age-structured compartmental model of COVID-19 transmission in the population of Ontario, Canada, we compared a base case with limited testing, isolation, and quarantine to scenarios with: enhanced case finding; restrictive social distancing measures; or a combination of enhanced case finding and less restrictive social distancing. Interventions were either implemented for fixed durations or dynamically cycled on and off, based on projected ICU bed occupancy. We present median and credible intervals (CrI) from 100 replicates per scenario using a two-year time horizon.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': \"We estimated that 56% (95% CrI: 42-63%) of the Ontario population would be infected over the course of the epidemic in the base case. At the epidemic peak, we projected 107,000 (95% CrI: 60,760-149,000) cases in hospital and 55,500 (95% CrI: 32,700-75,200) cases in ICU. For fixed duration scenarios, all interventions were projected to delay and reduce the height of the epidemic peak relative to the base case, with restrictive social distancing estimated to have the greatest effect. Longer duration interventions were more effective. Dynamic interventions were projected to reduce the proportion of the population infected at the end of the two-year period. Dynamic social distancing interventions could reduce the median number of cases in ICU below current estimates of Ontario's ICU capacity.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'Interpretation: Without significant social distancing or a combination of moderate social distancing with enhanced case finding, we project that ICU resources would be overwhelmed. Dynamic social distancing could maintain health system capacity and also allow periodic psychological and economic respite for populations.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}], 'body_text': [{'text': 'The COVID-19 pandemic represents a global public health emergency unparalleled in recent time. In the two months since the initial World Health Organization report describing the COVID-19 outbreak concentrated in Wuhan City, China (1), the number of confirmed cases has risen sharply from 282 to more than 330,0000, with 14,510 reported deaths across all regions of the globe (2). The first imported case of COVID-19 in Ontario, Canada was reported on January 25, 2020 and community transmission was first documented on March 1, 2020 in British Columbia, Canada (3).', 'cite_spans': [], 'ref_spans': [], 'section': 'Background'}, {'text': 'This pathogen represents a significant challenge for public health, pandemic planning, and healthcare systems. The SARS-CoV-2 virus is highly transmissible (4) (5) (6) (7) . It causes moderate to severe clinical outcomes in approximately 20% of all recognized infected individuals (5, 8, 9) . In the absence of a vaccine, public health responses have focused on the use of non-pharmaceutical interventions (NPIs) (10) . These NPIs include: (1) \"case-based\" measures such as testing, contact tracing, isolation, and quarantine, and (2) \"non-case-based\" measures such as reducing the probability of transmission given an effective contact (e.g. hand hygiene, and cough etiquette), and social distancing measures to reduce the contact rate in the population. Social distancing minimizes opportunities for person-to-person transmission of the virus to occur. These social distancing measures include some combination of school closure, teleworking, cancellation of group activities and events, and a general overall reduction in community contacts. While they are expected to be effective in reducing transmission of SARS-CoV-2, they are also associated with substantial economic costs and social disruption.', 'cite_spans': [{'start': 156, 'end': 159, 'text': '(4)', 'ref_id': 'BIBREF4'}, {'start': 160, 'end': 163, 'text': '(5)', 'ref_id': 'BIBREF5'}, {'start': 164, 'end': 167, 'text': '(6)', 'ref_id': 'BIBREF6'}, {'start': 168, 'end': 171, 'text': '(7)', 'ref_id': 'BIBREF7'}, {'start': 281, 'end': 284, 'text': '(5,', 'ref_id': 'BIBREF5'}, {'start': 285, 'end': 287, 'text': '8,', 'ref_id': None}, {'start': 288, 'end': 290, 'text': '9)', 'ref_id': None}, {'start': 413, 'end': 417, 'text': '(10)', 'ref_id': 'BIBREF11'}], 'ref_spans': [], 'section': 'Background'}, {'text': '. CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Background'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.24.20042705 doi: medRxiv preprint Epidemiological models can contribute important insight for public health decision-makers by allowing for the examination of a variety of \"what-if\" scenarios. The Canadian Pandemic Influenza Plan (CPIP) for the Health Sector (the backbone of which informs COVID-19 pandemic preparedness and response) identifies two main objectives for responding to a pandemic: (1) to minimize serious morbidity and mortality, and (2) to minimize societal disruption (11) . The overarching goal of pandemic response is to find a combination of NPIs that would minimize the number of cases requiring in-patient medical care (e.g. hospitalization and/or ICU admission), and deaths, while also minimizing the level of societal disruption.', 'cite_spans': [{'start': 506, 'end': 509, 'text': '(1)', 'ref_id': None}, {'start': 595, 'end': 599, 'text': '(11)', 'ref_id': 'BIBREF12'}], 'ref_spans': [], 'section': 'Background'}, {'text': 'Societal disruption measures could include outcomes such as the overall duration of time that the intervention needs to be to achieve the associated reductions in morbidity and mortality. A challenge for pandemic response using NPIs is that, in a fully susceptible population, although NPIs may slow disease transmission while they are in place, once the intervention is lifted (or compliance with the intervention becomes low), the transmission of the pathogen rebounds rapidly (10, 12) . In the case of COVID-19, it may not be possible to minimize morbidity and mortality, and societal and economic disruption at the same time.', 'cite_spans': [{'start': 479, 'end': 483, 'text': '(10,', 'ref_id': 'BIBREF11'}, {'start': 484, 'end': 487, 'text': '12)', 'ref_id': 'BIBREF13'}], 'ref_spans': [], 'section': 'Background'}, {'text': 'Given these considerations, we used a transmission dynamic model of COVID-19 to explore the potential impact of case-based, and non-case-based NPIs in the population of Ontario, Canada.', 'cite_spans': [], 'ref_spans': [], 'section': 'Background'}, {'text': 'Our analysis focusses on identifying strategies that keep the number of projected severe cases (hospitalizations, and ICU admissions) within a range that would not overwhelm the Ontario healthcare system, while also considering the amount of time these interventions would be in place.', 'cite_spans': [], 'ref_spans': [], 'section': 'Background'}, {'text': '. CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Background'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.24.20042705 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'Background'}, {'text': \"We developed an age-structured compartmental model that describes COVID-19 transmission in the province of Ontario, Canada. We used a modified 'Susceptible-Exposed-Infectious-Recovered' framework that incorporated additional compartments to account for public health interventions, different severities of clinical symptoms, and hospitalization risk. An overview of the model compartments and movements between them is provided in Figure 1 and model equations and additional details are provided in the Technical Appendix. The model was run for a period of two years and we assumed that recovered individuals remain immune from reinfection for the duration of the epidemic. Individuals remained infectious until they recovered or were hospitalized; we did not model transmission within healthcare settings. For simplicity, we assumed that all deaths occurred in cases requiring intensive care. We included cases in hospital and requiring intensive care to estimate health care requirements over the course of the epidemic. The model was constructed in R (13) .\", 'cite_spans': [{'start': 1054, 'end': 1058, 'text': '(13)', 'ref_id': 'BIBREF14'}], 'ref_spans': [{'start': 431, 'end': 439, 'text': 'Figure 1', 'ref_id': 'FIGREF1'}], 'section': 'Model overview'}, {'text': 'The model was stratified by 5-year age groups using 2019 population estimates (14) . Contacts within and between age groups were based on the POLYMOD study (15) , using contact data specific for the United Kingdom. The model was further stratified by health status to account for differential vulnerability to severe infection among those with underlying health conditions. We obtained comorbidity estimates by age from the Canadian Community Health Survey (16) for Ontario and included the following conditions: hypertension, heart disease, asthma, stroke, diabetes, and cancer. For younger age groups (<12 years) we used estimates from Moran et al. (17) .', 'cite_spans': [{'start': 78, 'end': 82, 'text': '(14)', 'ref_id': 'BIBREF15'}, {'start': 156, 'end': 160, 'text': '(15)', 'ref_id': 'BIBREF16'}, {'start': 457, 'end': 461, 'text': '(16)', 'ref_id': 'BIBREF17'}, {'start': 651, 'end': 655, 'text': '(17)', 'ref_id': 'BIBREF18'}], 'ref_spans': [], 'section': 'Model parameters'}, {'text': '. CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Model parameters'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint ', 'cite_spans': [], 'ref_spans': [], 'section': 'Model parameters'}, {'text': 'Testing was assumed to move individuals with non-severe symptoms from the infectious to isolated compartments. Isolated cases were assumed to have reduced transmission compared to non-isolated cases. Social distancing measures were assumed to reduce the number of contacts per day across the entire population. Details of parameters that were varied under different interventions are included in Table 2 . For the base case, we assumed that there was a degree of testing and isolation occurring and that a proportion of exposed cases were quarantined. We then added in additional control measures: (i) enhanced testing and contact tracing; (ii) restrictive social distancing measures; and (iii) a combination of enhanced testing and contract tracing, along with less restrictive social distancing than in (ii). We considered two approaches to implementing interventions: (i) fixed durations and (ii) a dynamic approach with interventions turned on and off based on the number of cases requiring ICU care in the population. We focused on ICU capacity, since this is expected to be most limited resource during the COVID-19 epidemic. Prior to emergence of COVID-19, Ontario had approximately 1300 ICU beds with associated ventilators; but 90% were occupied by individuals with non-. CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [{'start': 396, 'end': 403, 'text': 'Table 2', 'ref_id': 'TABREF2'}], 'section': 'Interventions'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.24.20042705 doi: medRxiv preprint COVID-19 illness; in mid-March 2020, the Ontario government made 300 additional ventilatorassociated ICU beds available (for a total of 430 unoccupied beds). As such, we used 200 COVID-19 cases in the ICU (across all of Ontario) as a threshold for turning the intervention on.', 'cite_spans': [], 'ref_spans': [], 'section': 'Interventions'}, {'text': 'This value was based on ~50% saturation of available beds combined with the recognition that there is a lag between cases acquiring infection and requiring intensive care, such that one would expect ICU needs to grow rapidly once initial COVID-19 cases present for care.', 'cite_spans': [], 'ref_spans': [], 'section': 'Interventions'}, {'text': 'Key model outputs included final epidemic attack rates (% of population infected at the end of the 2-year period), prevalence of hospitalizations and ICU use, and deaths. For comparison we', 'cite_spans': [], 'ref_spans': [], 'section': 'Outputs'}, {'text': 'show the maximum and current ICU capacity per 10,000 population relative to model projections. For the dynamic intervention scenarios, we also calculated the amount of time over the 2-year model period during which the intervention was implemented, as a measure of intervention intensity. We present model outputs as medians and credible intervals (CrI) from 100 model replicates per intervention.', 'cite_spans': [], 'ref_spans': [], 'section': 'Outputs'}, {'text': '. CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Outputs'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.24.20042705 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'Outputs'}, {'text': 'In the model base case, with limited testing, isolation and quarantine, we estimated that 56% (95% CrI: 42-63%) of the Ontario population would be infected over the course of the epidemic.', 'cite_spans': [], 'ref_spans': [], 'section': 'Base case'}, {'text': 'This would include cases of all severities. Attack rates were projected to be highest in those aged 5-14 years (77%, 95% CrI: 63-83%) and 15-49 years (63%, 95%CrI: 48-71). Lower attack rates were projected in individuals aged less than 5 years (50%, 95% CrI: 37-58%) and adults aged 50-69 years (47%, 95% CrI: 34-55) and greater than 70 years (30%, 95% CrI: 21-36). An example of the outbreak trajectory across model simulations is presented in Figure 2 . At the peak of the epidemic, in the absence of any resource constraints to provide care (i.e., assuming all cases requiring medical care receive it), we projected 107,000 (95% CrI: 60,760-149,000) cases in hospital and 55,500 (95% CrI: 32,700-75,200) cases in ICU. The high prevalence of cases in ICU reflects the mean length of ICU stay associated with COVID-19 infection in other countries.', 'cite_spans': [], 'ref_spans': [{'start': 445, 'end': 453, 'text': 'Figure 2', 'ref_id': 'FIGREF2'}], 'section': 'Base case'}, {'text': \"All of the interventions considered were projected to delay the epidemic peak and reduce the number of cases requiring ICU care at the peak (Figure 3) . The effectiveness of the interventions scaled with intervention duration. For all interventions, when the intervention duration was 6 months or less, there was no appreciable difference on final attack rate. With 12 and 18 months of heightened response measures, the proportion of the population infected at the end of the 2year period was reduced and, in some simulations, the prevalence of cases requiring intensive care fell below Ontario's current capacity for all or part of the time period. The largest effect was observed for the restrictive social distancing intervention. The combination intervention, with enhanced case detection and less aggressive social distancing was projected to substantially . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\", 'cite_spans': [], 'ref_spans': [{'start': 140, 'end': 150, 'text': '(Figure 3)', 'ref_id': 'FIGREF3'}], 'section': 'Fixed duration interventions'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.24.20042705 doi: medRxiv preprint reduce attack rates when implemented for 18 months, while enhanced case detection in the absence of social distancing measures had a more modest effect, on average. There was substantial variability in model projections, due to model stochasticity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Fixed duration interventions'}, {'text': \"We also explored dynamic interventions that were turned on and off in response to the current state of the epidemic. Dynamic interventions were projected to be effective for reducing the proportion of the population infected at the end of the two-year period, with potentially shorter durations of social distancing than the fixed duration approach (Figure 4) . For example, when implemented dynamically, 13 months of social distancing, cycled on and off, reduced the mean overall attack rate to 2%. For the social distancing alone and combination intervention scenarios, we observed atypical epidemic curves, with the number of cases increasing and decreasing repeatedly over time. In these scenarios, the median number of cases in ICU was reduced below current estimates of Ontario's ICU capacity.\", 'cite_spans': [], 'ref_spans': [{'start': 349, 'end': 359, 'text': '(Figure 4)', 'ref_id': 'FIGREF4'}], 'section': 'Dynamic interventions'}, {'text': '. CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Dynamic interventions'}, {'text': \"is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.24.20042705 doi: medRxiv preprint 1 0 Interpretation COVID-19 poses an extraordinary challenge to societies. While severe illness, particularly in older individuals, is frequent enough to overwhelm a society's ICU capacity (19) , mild unrecognized illness (particularly in younger individuals) contributes to spread (20) , and epidemics may only be recognized when superspreader events occur (21) , often in vulnerable settings like health care facilities (22) . In contrast to SARS (23), the high frequency of mild cases means that strategies which focus on case identification and isolation alone are likely to fail (22) . Control strategies for COVID-19 thus need to balance competing risks: the risks of mortality and health system collapse, on the one hand, against economic risks and attendant hardships (and health consequences) on the other. In this work, we evaluated plausible strategies for attenuating the COVID-19 epidemic in Ontario, Canada. We focussed on ICU resources for two reasons: first, because this component of most health systems represents a scarce resource prone to being saturated; and secondly because such saturation results in abrupt surges in casefatality, as individuals with acute respiratory distress syndrome will die quickly without the . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\", 'cite_spans': [{'start': 333, 'end': 337, 'text': '(19)', 'ref_id': 'BIBREF20'}, {'start': 426, 'end': 430, 'text': '(20)', 'ref_id': 'BIBREF21'}, {'start': 502, 'end': 506, 'text': '(21)', 'ref_id': 'BIBREF22'}, {'start': 566, 'end': 570, 'text': '(22)', 'ref_id': 'BIBREF23'}, {'start': 728, 'end': 732, 'text': '(22)', 'ref_id': 'BIBREF23'}], 'ref_spans': [], 'section': 'Dynamic interventions'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.24.20042705 doi: medRxiv preprint 1 1 capacity for mechanical ventilation. In broad terms, we find that prolonged social distancing is the preferred strategy for maintaining ICU resources, but an extreme fixed duration of social distancing is required to prevent the epidemic from overwhelming ICU capacity. That said, social distancing, even without reducing overall outbreak size, has the added benefit of delaying the epidemic peak, which gains time that can be used to build health system capacity and identify therapies and vaccines.', 'cite_spans': [{'start': 144, 'end': 147, 'text': '1 1', 'ref_id': None}], 'ref_spans': [], 'section': 'Dynamic interventions'}, {'text': 'In contrast to fixed-duration social distancing, we find that dynamic social distancing, with interventions turned on and off as needed, based on ICU capacity crossing a given threshold, represents a more effective, and likely more palatable, control strategy. Social distancing can be relaxed, but this inevitably results in resurgent disease in the population, requiring reinstatement. Nonetheless, dynamic social distancing is projected to maintain ICU capacity, and dramatically reduces overall attack rates, while at the same time requiring less total social distancing time than would be required by a fixed duration strategy of comparable effectiveness. Furthermore, dynamic social distancing has the potential to allow populations, and the economy, to \"come up for air\" at intervals, which may make this strategy more sustainable. We also found that a combination approach, with less restrictive social distancing along with enhanced case isolation and quarantine, could have a similar effect in the dynamic scenario as more restrictive social distancing alone. It is plausible that, as testing capacity increases, a combination approach that is less reliant on social distancing, will strike the right balance between disease control and societal disruption (30).', 'cite_spans': [], 'ref_spans': [], 'section': 'Dynamic interventions'}, {'text': 'Ontario, and lack of information on ICU occupancy by COVID-19 patients, makes it challenging . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Like any model, ours has limitations. At the time of writing, limitations in testing capacity in'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.24.20042705 doi: medRxiv preprint to know where exactly on the epidemic curve we currently find ourselves. Any model involves trade-offs between simplicity and realism, and in the current work we have not attempted to model social distancing measures in a highly realistic way, but rather generically as reductions in contact frequency. Our understanding of the natural history of SARS-CoV-2 infection continues to evolve and the precise role of pre-symptomatic and subclinical transmission is uncertain. Social distancing becomes a more important control measure in the face of incomplete case ascertainment due to asymptomatic or mildly symptomatic cases. The model does not include seasonality; it is possible that transmission will attenuate in the summer (31), resulting in a decline in cases that would be expected to resurge with the return of colder weather. All of these factors mean that the quantitative findings are subject to uncertainty.', 'cite_spans': [], 'ref_spans': [], 'section': 'Like any model, ours has limitations. At the time of writing, limitations in testing capacity in'}, {'text': 'Nonetheless, the qualitative insights around the role of social distancing, the relatively long intervention durations required to bend the epidemic curve, and the potential use of cyclic interventions can be used to policy-makers and decision makers, along with emerging empirical evidence from other countries, to consider the best approaches for epidemic control over the coming months. Lastly, we have not modeled the fact that that abrupt surges in death resulting from full ICUs would, result in lower demands for ICU beds. Our goal here is to inform policy so that such outcomes are avoided to the extent possible.', 'cite_spans': [], 'ref_spans': [], 'section': 'Like any model, ours has limitations. At the time of writing, limitations in testing capacity in'}, {'text': 'In summary, we have modelled plausible contours of the COVID-19 epidemic in Ontario, Canada, with a focus on maintenance of ICU resources. In the absence of significant social distancing or a combination of moderate social distancing with enhanced case detection and isolation, we project that ICU resources would be quickly overwhelmed, a conclusion consistent with that in other modeling work (12) , as well as current events in Italy and Spain. On a more positive note, we project that dynamic social distancing, that reacts to changes in ICU . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 395, 'end': 399, 'text': '(12)', 'ref_id': 'BIBREF13'}], 'ref_spans': [], 'section': 'Like any model, ours has limitations. At the time of writing, limitations in testing capacity in'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.24.20042705 doi: medRxiv preprint occupancy, could maintain health system capacity and also allow periodic psychological and economic respite for populations. 1 5 24.', 'cite_spans': [{'start': 269, 'end': 272, 'text': '1 5', 'ref_id': None}], 'ref_spans': [], 'section': 'Like any model, ours has limitations. At the time of writing, limitations in testing capacity in'}, {'text': 'Wang COVID-19 transmission model structure. Exposed cases can be either quarantined or not; quarantined cases would represent those who were identified via contact tracing. Hospitalized cases are assumed to be no longer infectious to others due to recognition of infection and are included in the model to estimate healthcare requirements. The model is stratified by age group and presence/absence of comorbidities.', 'cite_spans': [], 'ref_spans': [], 'section': 'Like any model, ours has limitations. At the time of writing, limitations in testing capacity in'}, {'text': '. CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Like any model, ours has limitations. At the time of writing, limitations in testing capacity in'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.24.20042705 doi: medRxiv preprint . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Like any model, ours has limitations. At the time of writing, limitations in testing capacity in'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.24.20042705 doi: medRxiv preprint CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Like any model, ours has limitations. At the time of writing, limitations in testing capacity in'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.24.20042705 doi: medRxiv preprint CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Like any model, ours has limitations. At the time of writing, limitations in testing capacity in'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.24.20042705 doi: medRxiv preprint . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Like any model, ours has limitations. At the time of writing, limitations in testing capacity in'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.24.20042705 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'Like any model, ours has limitations. At the time of writing, limitations in testing capacity in'}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'World Health Organization', 'authors': [], 'year': None, 'venue': 'Novel Coronavirus (2019-nCoV) Situation Report', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF1': {'ref_id': 'b1', 'title': '2. World Health Organization', 'authors': [], 'year': 2020, 'venue': 'Coronavirus DIsease (COVID-19) Situation Report', 'volume': '22', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF3': {'ref_id': 'b3', 'title': 'Epidemiological data from the COVID-19 outbreak in Canada', 'authors': [], 'year': 2020, 'venue': 'Accessed', 'volume': '22', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF4': {'ref_id': 'b4', 'title': 'Report of the WHO-China Joint Mission on Coronavirus Disease 2019 (COVID-19)', 'authors': [], 'year': 2020, 'venue': 'Accessed', 'volume': '22', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF5': {'ref_id': 'b5', 'title': 'The Novel Coronavirus Pneumonia Emergency Response Epidemiology Team. The epidemiological characteristics of an outbreak of 2019 novel coronavirus disease (COVID-19) -China', 'authors': [], 'year': 2020, 'venue': 'China CDC Weekly', 'volume': '2', 'issn': '', 'pages': '1--10', 'other_ids': {}}, 'BIBREF6': {'ref_id': 'b6', 'title': 'Early transmission dynamics in Wuhan, China, of novel coronavirus-infected pneumonia', 'authors': [{'first': 'Q', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Guan', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Wu', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Zhou', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Tong', 'suffix': ''}], 'year': 2020, 'venue': 'N Engl J Med', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF7': {'ref_id': 'b7', 'title': 'Early dynamics of transmission and control of COVID-19: a mathematical modelling study', 'authors': [{'first': 'A', 'middle': ['J'], 'last': 'Kucharski', 'suffix': ''}, {'first': 'T', 'middle': ['W'], 'last': 'Russell', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Diamond', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Liu', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Edmunds', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Funk', 'suffix': ''}], 'year': 2020, 'venue': 'Lancet Infect Dis', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF9': {'ref_id': 'b9', 'title': 'Estimating clinical severity of COVID-19 from the transmission dynamics in Wuhan, China. Nature Medicine', 'authors': [], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF10': {'ref_id': 'b10', 'title': 'Clinical course and mortality risk of severe COVID-19', 'authors': [{'first': 'P', 'middle': [], 'last': 'Weiss', 'suffix': ''}, {'first': 'D', 'middle': ['R'], 'last': 'Murdoch', 'suffix': ''}], 'year': 2020, 'venue': 'Lancet', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF11': {'ref_id': 'b11', 'title': 'Strategies for mitigating an influenza pandemic', 'authors': [{'first': 'N', 'middle': ['M'], 'last': 'Ferguson', 'suffix': ''}, {'first': 'D', 'middle': ['A'], 'last': 'Cummings', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Fraser', 'suffix': ''}, {'first': 'J', 'middle': ['C'], 'last': 'Cajka', 'suffix': ''}, {'first': 'P', 'middle': ['C'], 'last': 'Cooley', 'suffix': ''}, {'first': 'D', 'middle': ['S'], 'last': 'Burke', 'suffix': ''}], 'year': 2006, 'venue': 'Nature', 'volume': '442', 'issn': '7101', 'pages': '448--52', 'other_ids': {}}, 'BIBREF12': {'ref_id': 'b12', 'title': 'Canadian Pandemic Influenza Preparedness Task G. Canadian Pandemic Influenza Preparedness: Health sector planning guidance', 'authors': [{'first': 'B', 'middle': [], 'last': 'Henry', 'suffix': ''}], 'year': 2018, 'venue': 'Can Commun Dis Rep', 'volume': '44', 'issn': '1', 'pages': '6--9', 'other_ids': {}}, 'BIBREF13': {'ref_id': 'b13', 'title': 'Impact of non-pharmaceutical interventions (NPIs) to reduce COVID-19 mortality and healthcare demand', 'authors': [{'first': 'N', 'middle': ['M'], 'last': 'Ferguson', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Laydon', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Nedjati-Gilani', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Imai', 'suffix': ''}, {'first': 'Ainslie', 'middle': ['K'], 'last': 'Baguelin', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': '', 'suffix': ''}], 'year': 2020, 'venue': 'Accessed', 'volume': '22', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF14': {'ref_id': 'b14', 'title': 'R: A language and environment for statistical computing. R Foundation for Statistical Computing', 'authors': [{'first': '', 'middle': [], 'last': 'R Core Team', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF15': {'ref_id': 'b15', 'title': 'Population estimates on July 1st', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.25318/1710000501-eng.2019']}}, 'BIBREF16': {'ref_id': 'b16', 'title': 'Social contacts and mixing patterns relevant to the spread of infectious diseases', 'authors': [{'first': 'J', 'middle': [], 'last': 'Mossong', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Hens', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Jit', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Beutels', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Auranen', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Mikolajczyk', 'suffix': ''}], 'year': 2008, 'venue': 'PLoS Med', 'volume': '5', 'issn': '3', 'pages': '', 'other_ids': {}}, 'BIBREF17': {'ref_id': 'b17', 'title': 'Canadian Community Health Survey, public use microdata files', 'authors': [], 'year': 2016, 'venue': 'Accessed', 'volume': '15', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF18': {'ref_id': 'b18', 'title': 'Influenza vaccination rates in Ontario children: implications for universal childhood vaccination policy', 'authors': [{'first': 'K', 'middle': [], 'last': 'Moran', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Maaten', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Guttmann', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Northrup', 'suffix': ''}, {'first': 'J', 'middle': ['C'], 'last': 'Kwong', 'suffix': ''}], 'year': 2009, 'venue': 'Vaccine', 'volume': '27', 'issn': '17', 'pages': '2350--2355', 'other_ids': {}}, 'BIBREF19': {'ref_id': 'b19', 'title': 'Temporal changes in Ebola transmission in Sierra Leone and implications for control requirements: a real-time modelling study', 'authors': [{'first': 'A', 'middle': [], 'last': 'Camacho', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Kucharski', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Aki-Sawyerr', 'suffix': ''}, {'first': 'M', 'middle': ['A'], 'last': 'White', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Flasche', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Baguelin', 'suffix': ''}], 'year': 2015, 'venue': 'PLoS Curr', 'volume': '7', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF20': {'ref_id': 'b20', 'title': 'The early phase of the COVID-19 outbreak in', 'authors': [{'first': 'D', 'middle': [], 'last': 'Cereda', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Tirani', 'suffix': ''}, {'first': 'F', 'middle': [], 'last': 'Rovida', 'suffix': ''}, {'first': 'V', 'middle': [], 'last': 'Demicheli', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Ajelli', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Poletti', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF21': {'ref_id': 'b21', 'title': 'Substantial undocumented infection facilitates the rapid dissemination of novel coronavirus (SARS-CoV2)', 'authors': [{'first': 'R', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'Pei', 'middle': ['S'], 'last': 'Chen', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Song', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Yang', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': '', 'suffix': ''}], 'year': 2020, 'venue': 'Science', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF22': {'ref_id': 'b22', 'title': 'Secondary attack rate and superspreading events for SARS-CoV-2', 'authors': [{'first': 'Y', 'middle': [], 'last': 'Liu', 'suffix': ''}, {'first': 'R', 'middle': ['M'], 'last': 'Eggo', 'suffix': ''}, {'first': 'A', 'middle': ['J'], 'last': 'Kucharski', 'suffix': ''}], 'year': 2020, 'venue': 'Lancet', 'volume': '395', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF23': {'ref_id': 'b23', 'title': 'Identifying and Interrupting Superspreading Events-Implications for Control of Severe Acute Respiratory Syndrome Coronavirus 2', 'authors': [{'first': 'T', 'middle': ['R'], 'last': 'Frieden', 'suffix': ''}, {'first': 'C', 'middle': ['T'], 'last': 'Lee', 'suffix': ''}], 'year': 2020, 'venue': 'Emerg Infect Dis', 'volume': '26', 'issn': '6', 'pages': '', 'other_ids': {}}, 'BIBREF24': {'ref_id': 'b24', 'title': 'Transmission dynamics and control of severe acute respiratory syndrome', 'authors': [{'first': 'M', 'middle': [], 'last': 'Lipsitch', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Cohen', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Cooper', 'suffix': ''}, {'first': 'J', 'middle': ['M'], 'last': 'Robins', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Ma', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'James', 'suffix': ''}], 'year': 2003, 'venue': 'Science', 'volume': '300', 'issn': '5627', 'pages': '1966--70', 'other_ids': {}}}, 'ref_entries': {'FIGREF0': {'text': 'As such, population-level interventions, with their attendant economic costs, have been used to prevent health systems from collapsing (24). While events in China, Singapore, Hong Kong and elsewhere have demonstrated that COVID-19 epidemics can be contained (24-27), the seeding of epidemics in countries around the globe, many with weak health systems (28), means that reintroduction of COVID-19 will continue to occur for some time. As successful containment efforts maintain a large number of susceptible individuals in populations, vulnerability to repeated epidemics is likely to persist until a COVID-19 vaccine is developed and manufactured at scale; or until large fractions of the population are infected and either die or develop immunity (29).', 'latex': None, 'type': 'figure'}, 'FIGREF1': {'text': 'J, et al. Interventions to mitigate early spread of SARS-CoV-2 in Singapore: a modelling study. Lancet Infect Dis , DOI:https://doiorg/101016/S1473-3099(20)30162-6. 2020. 26. Cowling BJ, Ali ST, Ng TWY, Tsang TK, Li JCM, Fong MW, et al. Impact assessment of non-pharmaceutical interventions against COVID-19 and influenza in Hong Kong: an observational study. medRxiv. 2020:2020.03.12.20034660. 27. Lai S, Ruktanonchai NW, Zhou L, Prosper O, Luo W, Floyd JR, et al. Effect of non-pharmaceutical interventions for containing the COVID-19 outbreak in China. medRxiv. 2020:2020.03.03.20029843. 28. Tuite AR, Bogoch, II, Sherbo R, Watts A, Fisman D, Khan K. Estimation of Coronavirus Disease 2019 (COVID-19) Burden and Potential for International Dissemination of Infection From Iran. Ann Intern Med. 2020. 29. Pandemic Influenza Outbreak Research Modelling Team, Fisman D. Modelling an influenza pandemic: A guide for the perplexed. CMAJ. 2009;181(3-4):171-3. 30. Lee VJ, Chiew CJ, Khong WX. Interrupting transmission of COVID-19: lessons from containment efforts in Singapore. Journal of Travel Medicine. 2020. 31. Kissler SM, Tedijanto C, Goldstein E, Grad YH, Lipsitch M. Projecting the transmission dynamics of SARS-CoV-2 through the post-pandemic period. medRxiv. 2020:2020.03.04.20031112. 32. Bi Q, Wu Y, Mei S, Ye C, Zou X, Zhang Z, et al. Epidemiology and transmission of COVID-19 in Shenzhen China: analysis of 391 cases and 1,286 of their close contacts. medRxiv 2020030320028423; doi: https://doiorg/101101/2020030320028423. 2020. 33. Tindale L, Coombe M, Stockdale J, Garlock E, Lau W, Saraswat M, et al. Transmission interval estimates suggest pre-symptomatic spread of COVID-19. medRxiv 2020030320029983; doi: https://doiorg/101101/2020030320029983. 2020. 34. Woelfel R, Corman V, Guggemos W, Seilmaier M, Zange S, Mueller M, et al. Clinical presentation and virological assessment of hospitalized cases of coronavirus disease 2019 in a travel-associated transmission cluster. medRxiv 2020030520030502; doi: https://doiorg/101101/2020030520030502. 2020. 35. Wang D, Hu B, Hu C, Zhu F, Liu X, Zhang J, et al. Clinical characteristics of 138 hospitalized patients with 2019 novel coronavirus-infected pneumonia in Wuhan, China. JAMA. 2020. 36. Yang X, Yu Y, Xu J, Shu H, Xia J, Liu H, et al. Clinical course and outcomes of critically ill patients with SARS-CoV-2 pneumonia in Wuhan, China: a single-centered, retrospective, observational study. Lancet Respir Med. 2020.', 'latex': None, 'type': 'figure'}, 'FIGREF2': {'text': 'Projected COVID-19 epidemic trajectory for the base case model with minimal intervention. Daily incident cases per 1000 population are presented. The line represents the median value of 100 model simulations and the shaded area indicates', 'latex': None, 'type': 'figure'}, 'FIGREF3': {'text': 'Projected ICU bed requirements and attack rates for fixed duration interventions. (A) Prevalent cases requiring intensive care are shown for intervention durations of 1, 3, 6, 12, and 18 months. Maximum and current ICU capacity in Ontario are indicated by the dashed horizontal lines. Median values are presented. (B) Modelprojected percent of the population infected over the 2-year time period. Attack rates include all infections, regardless of severity. Note that the slight variability in epidemic size for the no additional intervention base case reflects model stochasticity across simulations. 9 .', 'latex': None, 'type': 'figure'}, 'FIGREF4': {'text': 'Projected ICU bed requirements and attack rates for dynamic interventions. (A) Prevalent cases requiring intensive care are shown for the base case and three intervention scenarios. Interventions are turned on and off (returning to base case parameter values), depending on the number of COVID-19 cases in the ICU. Maximum and current ICU capacity in Ontario are indicated by the dashed horizontal lines. Median values are presented. (B) Zoomed view of prevalent ICU cases to show the dynamics for the enhanced social distancing and combination scenarios. (C) Modelprojected estimates of percent of the population infected over the 2-year time period. Attack rates include all incident infections, regardless of severity. Median and 95% CrI are shown. The amount of time the dynamic interventions are in place is shown on the x-axis. Points indicate the median duration and lines the 95% CrI for each scenario. 0 .', 'latex': None, 'type': 'figure'}, 'TABREF0': {'text': 'Parameters describing the natural history and clinical course of infection were derived from published studies (Table 1, full details in Technical Appendix). To capture variability in transmission, specifically the observation that the basic reproductive number (R0) is overdispersed, with some cases transmitting to many others (superspreader events), while many other cases transmit much less, we have added volatility to the transmission term(18). The model was initiated with 750 prevalent cases (based on 150 reported cases in Ontario on March 19, 2020 and an assumed reporting rate of 20%), that were randomly distributed across the infectious compartments.', 'latex': None, 'type': 'table'}, 'TABREF1': {'text': 'Model parameters.. CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.24.. CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.24.20042705 doi: medRxiv preprint', 'latex': None, 'type': 'table'}, 'TABREF2': {'text': 'Model scenarios details.', 'latex': None, 'type': 'table'}}, 'back_matter': [{'text': 'The authors wish to thank Gabrielle Brankston, Shannon French, Tanya Rossi, and Matthew Van Camp from the Department of Population Medicine, University of Guelph for helping to compile data on population demographics and chronic conditions. The authors also acknowledge the feedback of colleagues from local public units across the province of Ontario.', 'cite_spans': [], 'ref_spans': [], 'section': 'Acknowledgements'}]}\n",
      "{'paper_id': 'b026bbd2c2afbbf5f36cfdae973657aaca4153a7', 'metadata': {'title': 'Modeling COVID-19 epidemics in an Excel spreadsheet: Democratizing 1 the access to first-hand accurate predictions of epidemic outbreaks 2 3', 'authors': [{'first': 'Mario', 'middle': [], 'last': 'Moisés Alvarez', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Tecnologico de Monterrey', 'location': {'postCode': '64849', 'settlement': 'Monterrey', 'region': 'NL', 'country': 'México'}}, 'email': ''}, {'first': 'Everardo', 'middle': [], 'last': 'González-González', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Tecnologico de Monterrey', 'location': {'postCode': '64849', 'settlement': 'Monterrey', 'region': 'NL', 'country': 'México'}}, 'email': ''}, {'first': 'Grissel', 'middle': [], 'last': 'Trujillo-De', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Tecnologico de Monterrey', 'location': {'postCode': '64849', 'settlement': 'Monterrey', 'region': 'NL', 'country': 'México'}}, 'email': ''}, {'first': 'Santiago', 'middle': [], 'last': '', 'suffix': '', 'affiliation': {}, 'email': ''}]}, 'abstract': [{'text': 'COVID-19, the first pandemic of this decade and the second in less than 15 years, has 15 harshly taught us that viral diseases do not recognize boundaries; however, they truly do 16 discriminate between aggressive and mediocre containment responses. 17', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'We present a simple epidemiological model that is amenable to implementation in Excel 18 spreadsheets and sufficiently accurate to reproduce observed data on the evolution of the 19 COVID-19 pandemics in different regions (i.e., Italy, Spain, and New York City (NYC)). 20', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'We also show that the model can be adapted to closely follow the evolution of COVID-19 21 in any large city by simply adjusting two parameters related to (a) population density and 22', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': '(b) aggressiveness of the response from a society/government to epidemics. Moreover, we 23', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'show that this simple epidemiological simulator can be used to assess the efficacy of the 24 response of a government/society to an outbreak. 25 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint .', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}], 'body_text': [{'text': 'In equation (7), Dd=P s /A, where A is the surface area of the region subject to analysis. In 144 this formulation, Dd is the total number of inhabitants of the region who are susceptible to 145 infection, while Dd ref is a value of demographic density in a densely populated urban area 146 that the model uses as a reference. In this work, the demographic density of the city of 147 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint ', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': '167', 'cite_spans': [], 'ref_spans': [], 'section': 'Selection of relevant epidemiological parameters for COVID-19'}, {'text': 'The number of asymptomatic inhabitants was calculated under the assumption that only 168 ~30.0% of the infected population develops symptomatology (2.5 asymptomatic subjects 169 per 1.0 symptomatic subject). This assumption should be regarded as speculative, since 170 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. We have selected these three data sets to illustrate that the evolution of the epidemic has a 204 local flavor that mainly depends on the number of initial infected persons, the demographic 205 density, and the set of containment measures taken by government officials and society. 206 line should be observed, and the slope of that line denotes the specific rate (µ) of the 212 epidemic spreading. Note that COVID-19 has exhibited a wide range of spreading rates in 213 different countries (from ~0.3 to ~0.9 day -1 ). Note also that µ is related to the doubling time 214 (t d ), often reported in population and epidemiological studies, by the equation t d =Ln 2/ µ. 215 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Selection of relevant epidemiological parameters for COVID-19'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.23.20041590 doi: medRxiv preprint Therefore, ranges of doubling times between 0.75 and 2.45 days are observed just among 216 these three regional cases. ', 'cite_spans': [], 'ref_spans': [], 'section': 'Selection of relevant epidemiological parameters for COVID-19'}, {'text': 'Different exponential stages, perfectly distinguishable by their exhibition of different slopes 230 (Table 2) , may be observed within the same time series. For instance, the outbreak in NYC 231 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [{'start': 100, 'end': 109, 'text': '(Table 2)', 'ref_id': 'TABREF1'}], 'section': '229'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.23.20041590 doi: medRxiv preprint ( Figure 1B ; blue symbols) was first described by an extremely high slope (µ o = 0.926 day -232', 'cite_spans': [], 'ref_spans': [{'start': 146, 'end': 155, 'text': 'Figure 1B', 'ref_id': 'FIGREF3'}], 'section': '229'}, {'text': 'The last point is extremely important, since two drastically different slopes can be observed 235 before and after a package of adequate measures within the same territory. In addition, two 236 localities that experienced similar initial specific epidemic rates may exhibit dramatically Validation and predictions 249 We have run different scenarios to validate the predictive capabilities of our epidemic 250 model for COVID-19. Overall, the model is capable of closely reproducing the progression 251 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 314, 'end': 317, 'text': '249', 'ref_id': None}], 'ref_spans': [], 'section': '229'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.23.20041590 doi: medRxiv preprint of reported cases for urban areas of more than 5 × 10 6 inhabitants (i.e., Iran, the city of 252', 'cite_spans': [], 'ref_spans': [], 'section': '229'}, {'text': 'Tehran in Iran, Spain, and NYC). We found that, adapting the model to a particular locality Figure 2A ). Note 296 that his suggest that the measures of social distancing imposed in NYC were equivalent to 297 decrease the effective demographic density to 50%. At the end of March, after this 298 adjustment, our model forecasts a peak of infections of nearly 5,000 new cases per day (less 299 than half than the prediction before social distancing), and a cumulative number of 1 × 10 6 300 citizens infected. For instance, our results suggest that, for an urban area such as NYC, imposing measures 322 that guarantee a social distance equivalent to a decrease in demographic density of 50% 323 will delay the peak of maximum number of infections by 25 days and will decrease its 324 intensity from 23000 to 9000 new cases of infection per day. In turn, this implies a lower 325 demand for hospital beds per day during the epidemics and may mark the difference 326 between a manageable crisis and a public health catastrophe 5,30 . 327 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. ', 'cite_spans': [], 'ref_spans': [{'start': 92, 'end': 101, 'text': 'Figure 2A', 'ref_id': 'FIGREF7'}], 'section': '229'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. April. The simulation may be used to forecast the demand of beds during the month of 359', 'cite_spans': [], 'ref_spans': [], 'section': '338'}, {'text': 'April 2020, which is estimated to exhibit a peak of nearly 50000. 360 361 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. Estimation of the number of beds needed during the month of April 2020 in Monterrey, 371 based on the number of patients that will require hospitalization according to the model 372', 'cite_spans': [], 'ref_spans': [], 'section': '338'}, {'text': 'predictions. 373 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '338'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.23.20041590 doi: medRxiv preprint China indicate that the percentage of patients with severe symptoms may be 14%, with a 376 5% of critical cases 32 . In prospective, the total number of beds in the Mexican public health 377 sector is estimated in 20,000 (for the whole country). 378 379 Concluding remarks 380 We used a set of differential equations, recent epidemiological data regarding the evolution 381 of COVID-19 infection in a reduced set of regions (i.e., Spain, Iran, and NYC), and basic 382 information on the characteristics of COVID-19 infection (i.e., time from infection to 383 recovery, case mortality rate) to accurately recreate the onset of the COVID-19 in two 384 urban areas with different demographic characteristics (i.e., NYC and Monterrey, México). 385', 'cite_spans': [{'start': 417, 'end': 420, 'text': '380', 'ref_id': None}], 'ref_spans': [], 'section': '338'}, {'text': 'We showed that the model can be adapted to closely follow the evolution of COVID-19 in 386 densely populated urban areas by simply adjusting two parameters related to (a) population 387 density and (b) aggressiveness of the response from a society/government to epidemics. 388', 'cite_spans': [], 'ref_spans': [], 'section': '338'}, {'text': 'Scenarios such as those currently unfolding in Iran, Italy, or Spain emphasize the 389 importance of planning ahead during epidemic events. The availability of a simple model 390 may be highly enabling for local governments, physicians, civil organizations, and citizens 391 as they struggle in their endeavor to accurately forecast the progression of an epidemic and 392 formulate a plan of action. As previously stated, the use of simple/user-friendly models to 393 evaluate in (practically) real time the effectiveness of containment strategies or programs 394 may be a powerful tool for analyzing and facing epidemic events 6,12 . This contribution 395 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '338'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.23.20041590 doi: medRxiv preprint shows the prediction potential of an extremely simple simulation tool that can be used by 396 practically any citizen with basic training in Excel. 397', 'cite_spans': [], 'ref_spans': [], 'section': '338'}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'First Case of 2019 Novel Coronavirus in the United States', 'authors': [{'first': 'M', 'middle': ['L'], 'last': 'Holshue', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF2': {'ref_id': 'b2', 'title': 'Global spread of COVID-19 and pandemic potential', 'authors': [{'first': 'C', 'middle': ['R'], 'last': 'Macintyre', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF4': {'ref_id': 'b4', 'title': 'Estimating the reproductive number and the outbreak size of 416 14', 'authors': [{'first': 'S', 'middle': ['C'], 'last': 'Choi', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Ki', 'suffix': ''}, {'first': 'E', 'middle': [], 'last': 'Prompetchara', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Ketloy', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Palaga', 'suffix': ''}], 'year': 2006, 'venue': 'COVID-19 and potential vaccines: Lessons learned from SARS and', 'volume': '443', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF5': {'ref_id': 'b5', 'title': 'A pneumonia outbreak associated with a new coronavirus of probable 449 old-style public health measures in the novel coronavirus (2019-nCoV) outbreak |', 'authors': [{'first': 'P', 'middle': [], 'last': 'Zhou', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}}, 'ref_entries': {'FIGREF0': {'text': 'system, all equations depend on R Infected-s. Here, we propose a simple formulation for 114 the evaluation of R Infected-s at the onset of a local epidemic event. correctly describes the evolution of the number of newly infected during the 138 initial stage of the epidemic episode. For later times, the rate of new infections is corrected 139 by a term that depends on the demographic density (Dd) of the region.', 'latex': None, 'type': 'figure'}, 'FIGREF1': {'text': 'subjects estimated for other pandemic events. For instance, in the 176 context of pandemic influenza A/H1N1/2009, up to 20-40% of the population in urban 177 areas (i.e., Monterrey, México, and Pittsburgh, USA) 20,21 exhibited specific antibodies 178 regardless of experiencing symptoms, while the fraction of confirmed symptomatic 179 infections was lower than less than 10%. 180 In addition, the average time of sickness was set at 14 days in our simulations, within the 181 range reported from 14 to 32 days 22 , with a median time to recovery of 21 days 23 . 182 Therefore, the number of patients recovered (R) is calculated as a fraction of 0.977 of those 183 infected 14 days previously. Similarly, asymptomatic patients are only removed from the 184 pool of susceptible after full recovery. Note that, in the current version of our model, 185 asymptomatic patients are not considered part of the population capable of transmitting 186 COVID-19, despite recently reported evidence that suggests that asymptomatic subjects (or 187 minimally symptomatic patients) may exhibit similar viral loads 24 to those of symptomatic 188 patients and may be active transmitters of the disease 2,25 . The number of deceased patients 189 was calculated as 0.023 of those infected 14 days before. This mortality percentage (case 190 fatality rate) lies within the range reported in recent literature for COVID-19 9,26-28 . The 191 time lapse of 14 days between the onset of disease and death was statistically estimated by 192 Linton et al. in a recent report 29 . set of parameters described before, allows the calculation of all populations (I s, X s , 195 X as , D, R, and P s ) every hour. Note that this model enables the description of the 196 progressive exhaustion of the epidemic, as expected by the progressive depletion of the 197 susceptible population. Next, we discuss criteria for selection of the values of µ o based on 198 the initial behavior of the COVID-19 Pandemic at different urban areas around the globe. 199 200 Estimation of specific epidemic rate values 201 Figure 1A shows the progression on the number of COVID-19 positive cases in different 202 regions, namely Spain (mainly Madrid), Iran (mainly Tehran), and New York City (NYC). 203', 'latex': None, 'type': 'figure'}, 'FIGREF2': {'text': '1B shows the natural log of the cumulative number of infections over time for the 207 same set of countries. This simple plotting strategy is highly useful for analyzing the local 208 rate of progression of the pandemic. If the local epidemic progression is consistent with a 209 simple first order exponential model where dI/dt = µ*I, then the integral form of this 210 equation renders the linear equation: ln I/I o = µ*t. During the exponential phase, a straight 211', 'latex': None, 'type': 'figure'}, 'FIGREF3': {'text': 'Epidemiological data related to the onset of a COVID-19 pandemic in different 220 regions. (A) Cumulative number of positive cases of COVID-19 infection in Spain (yellow 221 circles), Iran (green squares), and NYC (blue triangles) during the first days after the outbreak. (B) 222 Natural logarithm of the cumulative number of positive cases of COVID-19 infection in Spain 223 (yellow circles), Iran (green squares), and NYC (blue triangles). (C) Cumulative number of positive 224 cases of COVID-19 infection in Italy (blue squares) and South Korea (red circles). (D) Natural 225 logarithm of the cumulative number of positive cases of COVID-19 infection in Italy (blue squares) 226 and South Korea (red circles). Two clearly distinctive exponential stages are observed in the case 227 of South Korean progression.', 'latex': None, 'type': 'figure'}, 'FIGREF4': {'text': '237 different evolutions as a function of the initial response of government and society (Figure 238', 'latex': None, 'type': 'figure'}, 'FIGREF5': {'text': '1C,D). For instance, while the COVID-19 epidemics in Italy and South Korea exhibited 239 practically equal µ o values, the Italian outbreak has maintained the same growth rate 240 throughout 20 days, while South Korea has set an example by effectively and rapidly 241 lowering the specific epidemic rate to nearly 0 in just two weeks. 242 243', 'latex': None, 'type': 'figure'}, 'FIGREF6': {'text': '253 is straightforward and only requires (a) the calculation of the population and the surface 254 area of the urban area, and (b) the selection of a t d value (time to doubling the name of 255 infections). Note that our model is formulated in terms of values of the specific epidemic 256 growth rate (µ o for the onset of the epidemic and µ for later times). However, expressing 257 the specific epidemic rate in terms of doubling time (t d =Ln 2/µ) is more practical and 258 simpler to communicate and understand (Table 2). 259 260 The selection of µ o (t d ) can be easily done by fitting the prediction to the initial set of 261 reported cases of infection. In our experience, four to five reliable data points are needed 262 for a good fit. For instance, Figure 2 shows the predicted trend of the pandemic in NYC 263 during the first 28 days of March, 2020. In addition, we set (Dd/Dd ref =1.90), since the 264 population density in NYC is 1.90-fold higher than that in Madrid. A value of t d = 2.25 was 265 also set for the first week of this simulation. Later, at day 7 (March 7), we reset the value of 266 t d to 3.75 to reflect the modification of the slope of the local epidemic event in NYC (Figure 267', 'latex': None, 'type': 'figure'}, 'FIGREF7': {'text': '1d), due to the implemented measures of containment. Based on this exercise, we foresee 268 that this simple modeling tool can be used to evaluate the efficacy of containment 269 strategies. In other words, the value of µ o required in the simulation to adapt the predicted 270 data to the actual trend of the local epidemic provides an indicator of the local rate of 271 spreading of the pandemic. Progression of the COVID-19 Pandemic in NYC. (A) Initial evolution of the number 275 of positive cases of COVID-19 in NYC. Actual data points, as officially reported, are shown using 276 red circles. Simulation predictions are described by the blue dotted line. (B) Model prediction of 277 the total number of symptomatic patients through the months of March and April. (C) Model 278 prediction of new cases of COVID-19 during the period from March 1 to May 20, 2020 if no 279 further containment actions are adopted.', 'latex': None, 'type': 'figure'}, 'FIGREF8': {'text': 'of the effectiveness of that set of measures. This can be extremely 283 useful when assessing the efficacy of control of epidemics. For example, for NYC, this 284 simple model states that the set of containment measures adopted during the first week of 285 March in NYC diminished the specific rate of the epidemic by increasing the doubling time 286 of infections from a value of 2.25 to 3.75 days. 287 The ability to make close predictions of the progression of cases in a particular region has 288 profound and enabling implications. For example, in March 15 th , our simulations predicted 289 that, in absence of more aggressive containment measures (yellow trend in Figure 2A), the 290 peak of infections in NYC will be reached by April 10, 2010, after reaching the 291 unprecedented value of 11,000 new cases per day, and a cumulative number of 1 × 10 6 292 citizens infected. However, we observed a deviation from this prediction by the third week 293 of March that we attribute to the stringent measures of social distancing established in NYC 294 earlier that week. Accordingly, we multiplied the value of (Dd/Dd ref ) in our simulations by 295 a factor of 0.50 to properly fit the new trend on actual cases (blue trend in', 'latex': None, 'type': 'figure'}, 'FIGREF9': {'text': '-19 epidemics 30,31 . Next, we evaluate the effect of different degrees of social 307 distancing on the shape of the epidemic curve for NYC, one of the most densely urban 308 areas worldwide. This evaluation is straightforward, since the formulation of our model 309 explicitly considers the demographic density of the region as the most important modifier 310 of the rate of progression of the epidemics. 311 In the Excel implementation of the model, we multiply the demographic ration (Dd/Dd ref ) 312 by 0.75 to calculate the impact of distancing measures that would diminish social contact 313 by 25%. Similarly, we multiply (Dd/Dd ref ) by 0.50 to simulate the effect of a scenario of 314 social distancing that would diminish close social interaction by 50%. Figure 4 shows the 315 effect of three different degrees/levels of social distancing on the cumulative number of 316 infections (Figure 3A) and on the number of new cases of infection per day (Figure 3B). 317 Social distancing has a clear buffering effect on the epidemics, delaying the occurrence of 318 the peak of infections and distributing the number of cases across a longer time span. This 319 is remarkably important as it provides time for proper attention to patients with severe 320 symptomatology 5 . 321', 'latex': None, 'type': 'figure'}, 'FIGREF10': {'text': 'Prediction of the effect of social distancing on the progression of the COVID-19 330 pandemics in New York City (NYC). (A) Model prediction of the total number of symptomatic 331 patients from March 1 to May 31, 2020 for different scenarios of social distancing: social 332 distancing as in March 20, 2010 (blue line, current prediction); social distancing effective on March 333 20, whereby the effective demographic density in NYC is reduced by 25% (light green line); social 334 distancing effective on March 20 whereby the effective demographic density in NYC is reduced by 335 50% (red line); and social distancing effective on March 10, whereby the effective demographic 336 density in NYC is reduced by 25% (dark green line). (B) Model prediction for the number of new 337 infections per day for each of the scenarios of social distancing described.', 'latex': None, 'type': 'figure'}, 'FIGREF11': {'text': 'have accurately predicted the nine subsequent actual values, as officially 350 reported from March 19 to March 28 (Figure 4 A). Monitoring actual data, while 351 comparing with model predictions, enables real time assessment of the effectiveness of the 352 containment measures. In turn, this empowers officials, scientists, health care providers, 353 and citizens. Moreover, friendly and widely available mathematical modeling enables 354 rational planning. For instance, according to the pandemic scenario predicted for 355 Monterrey, in the absence of further containment measures and stricter social distancing, 356 the total number of symptomatic infected will surpass 650,000 persons (Figure 4B), and the 357 number of new infections per day (Figure 4C) will exhibit a peak of 2000 by the end of 358', 'latex': None, 'type': 'figure'}, 'FIGREF12': {'text': 'Progression of the COVID-19 pandemic in the metropolitan area of 364 Monterrey, Nuevo León, Mexico. (A) Initial evolution of the number of positive cases of 365 COVID-19 in the metropolitan area of Monterrey. Actual data points, as officially 366 reported, are shown using red circles. Simulation predictions are described by the blue 367 dotted line. (B) Model prediction of the total number of symptomatic patients from March 368 1 to May 20, 2020. (C) Model prediction of new cases of COVID-19 during the period 369 from March 1 to May 20, 2020 if no further containment actions are adopted. (D) 370', 'latex': None, 'type': 'figure'}, 'TABREF0': {'text': 'population susceptible to infection, a counter is needed to continuously update the number 149 of recovered patients, asymptomatic patients, and deaths. Therefore, at each time step 150 during the numerical integration, the susceptible population is updated by subtracting the 151 number of number of recovered patients, asymptomatic patients, and deaths. Note that in 152our Excel spreadsheet, we use Dd/Dd ref = density_Factor (Supplemental Excel File 1). 153Defining an expression for R Infected-s enables stepwise numerical integration, for example by 154 the Euler method. We have implemented this solution in a spreadsheet. To that aim, 155 differential equations (1) and(7)should be converted into their corresponding equations of 156ΔX s = {R Infected-s -R Recovered -R Death } ΔtFor all the simulation results presented here, we set Δt=1h= 1/24 day. We have solved this 162 differential set, step by step, updating the values of R Infected-s , R Recovered , R Death , and P s , 163 according to equations (2´) to (5´). The ratio (Dd/Dd ref ) is also recalculated at each time 164 step using the updated value of P s from equation (5´). 165', 'latex': None, 'type': 'table'}, 'TABREF1': {'text': 'Specific infection rates (µ o ) and associated doubling times (t d ) for COVID-19', 'latex': None, 'type': 'table'}}, 'back_matter': [{'text': 'The authors declare no competing interests. 409 410', 'cite_spans': [], 'ref_spans': [], 'section': '408'}]}\n",
      "{'paper_id': 'b0746b57a6c8f3cd9e52a27b0d708d069134b502', 'metadata': {'title': 'Title: Bat coronavirus phylogeography in the western Indian Ocean Running title: Bat coronavirus in the western Indian Ocean Author affiliations', 'authors': [{'first': 'Léa', 'middle': [], 'last': 'Joffrin', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Steven', 'middle': ['M'], 'last': 'Goodman', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'David', 'middle': ['A'], 'last': 'Wilkinson', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Beza', 'middle': [], 'last': 'Ramasindrazana', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Er-3', 'middle': [], 'last': 'Lagadec', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Yann', 'middle': [], 'last': 'Gomard', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Gildas', 'middle': [], 'last': 'Le Minter', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Andréa', 'middle': [], 'last': 'Dos', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'M', 'middle': ['Corrie'], 'last': 'Santos', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Ra-4', 'middle': [], 'last': 'Schoeman', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Pablo', 'middle': [], 'last': 'Sookhareea', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Simon', 'middle': [], 'last': 'Tortosa', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Eduardo', 'middle': ['S'], 'last': 'Julienne', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Patrick', 'middle': [], 'last': 'Gudo', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': '', 'middle': [], 'last': 'Mavingui', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Camille', 'middle': [], 'last': 'Lebarbenchon', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': '(', 'middle': ['L'], 'last': 'Joffrin', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'B', 'middle': [], 'last': 'Ra-10', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'E', 'middle': [], 'last': 'Lagadec', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Y', 'middle': [], 'last': 'Gomard', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'G', 'middle': [], 'last': 'Le Minter', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'D', 'middle': ['A'], 'last': 'Wilkinson', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'P', 'middle': [], 'last': 'Tortosa', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'P', 'middle': [], 'last': 'Mavingui', 'suffix': '', 'affiliation': {}, 'email': ''}]}, 'abstract': [], 'body_text': [{'text': 'The burden of emerging infectious diseases has significantly increased over the last decades 41 and is recognized as a major global health concern. In 2018, the World Health Organization 42 (WHO) established the \"Blueprint priority disease list\", identifying viruses such as Ebola, Lassa 43 fever, Middle East Respiratory Syndrome (MERS), and Nipah fever as significant threats to 44 international biosecurity (1). This list also highlights the potential pandemic risk from the emer-45 gence of currently unknown zoonotic pathogens, collectively referring to these unknown threats 46 as \"disease X\" (1). Investigation of the potential zoonotic pathogens in wild animals, particu-47 larly vertebrates, is thus critical for emerging infectious diseases preparedness and responses. 48 Bats represent nearly 1,400 species and live on all continents except Antarctica. They pro- 49 vide key ecosystem services such as crop pest regulation, pollination, seed dispersal, and soil Our study area spans geographic locations across the islands of the western Indian 58 Ocean (WIO) and southeastern continental Africa (SECA) (Figure 1 ). These land areas have 59 diverse geological origins that have influenced the process of bat colonization and species dis-60 tributions. The ecological settings and species diversity on these islands for bats are notably 61 different. On Madagascar, more than 45 bat species are known to occur, of which more than 80 62 author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [{'start': 93, 'end': 95, 'text': '41', 'ref_id': None}, {'start': 381, 'end': 383, 'text': '44', 'ref_id': None}, {'start': 581, 'end': 583, 'text': '46', 'ref_id': None}, {'start': 779, 'end': 781, 'text': '48', 'ref_id': None}, {'start': 874, 'end': 876, 'text': '49', 'ref_id': None}], 'ref_spans': [{'start': 1114, 'end': 1123, 'text': '(Figure 1', 'ref_id': 'FIGREF21'}], 'section': '40'}, {'text': \"The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/742866 doi: bioRxiv preprint 4 % are endemic to the island. The smaller studied islands of the WIO, Mauritius, Mayotte, Re-63 union Island, and Mahé (Seychelles), host reduced bat species diversity (e.g. three species on 64 Reunion Island), whereas SECA supports a wide range of bat species. To date, several studies 65 have identified bat-infecting CoVs in countries of continental Africa, including Zimbabwe (4), 66 South Africa (5), and Kenya (6). CoVs have also been reported in fruit bat (Pteropodidae) pop-67 ulations of Madagascar, where beta-coronaviruses belonging to the D-subgroup were identified 68 in the two bat species Eidolon dupreanum, and Pteropus rufus (7). 69 In this study, we investigated the presence of CoVs in over 1,000 individual bats be- Appendix Table S1 . 86 author/funder. All rights reserved. No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/742866 doi: bioRxiv preprint 6 7: 5'-CCA-TCA-TCA-GAT-AGA-ATC-ATC-ATA-3' (14 119 We tested the effect of bat family, species, sex, as well as sampling location and roosting 120 habitat (caves, outdoor or buildings) on the probability for detecting CoV RNA using Pearson 121 χ² tests. The effect of bat age (adult vs juvenile) and female reproductive status (pregnant vs not \", 'cite_spans': [{'start': 323, 'end': 325, 'text': '64', 'ref_id': None}, {'start': 419, 'end': 421, 'text': '65', 'ref_id': None}, {'start': 517, 'end': 519, 'text': '66', 'ref_id': None}, {'start': 779, 'end': 781, 'text': '69', 'ref_id': None}, {'start': 1142, 'end': 1145, 'text': '119', 'ref_id': None}], 'ref_spans': [{'start': 868, 'end': 885, 'text': 'Appendix Table S1', 'ref_id': None}], 'section': '40'}, {'text': 'In total, 88 of the 1,099 bat samples tested positive for CoV by RT-PCR (mean detection 152 rate ± 95% confidence interval: 8.0% ± 1.6%). The prevalence of positive bats was different 153 according to the sampling locations (χ² = 70.2; p<0.001), with a higher prevalence in Mozam-154 bique (19.6% ± 4.7%) than on all WIO islands (4.2% ± 1.2%) (Figure 2) . A significant differ-155 ence in the prevalence of positive bats was also detected between families (χ² = 44.2; p<0.001;', 'cite_spans': [], 'ref_spans': [{'start': 343, 'end': 353, 'text': '(Figure 2)', 'ref_id': None}], 'section': '151'}, {'text': 'Appendix Figure S1 ). The highest prevalence was observed in the families Nycteridae (28.6 % 157 ± 23.6%), Rhinolophidae (24.6% ± 10.5%), Hipposideridae (11.9% ± 6.9%), and Rhinonycter-158 idae (10.7% ± 5.5%). Bat species (χ² = 156.27; p<0.001; Appendix Figure S2 ) and roosting 159 author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [{'start': 9, 'end': 18, 'text': 'Figure S1', 'ref_id': 'FIGREF21'}, {'start': 254, 'end': 263, 'text': 'Figure S2', 'ref_id': None}], 'section': '156'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/742866 doi: bioRxiv preprint 8 habitat (χ² = 23.76; p<0.001; Figure 3 ) had a significant effect on the probability of CoVs de-160 tection, but not host sex (χ² = 2.32; p = 0.13). For Mozambican Hipposideros caffer and Mops 161 condylurus, bat age (adult vs juvenile) had no effect on the probability of CoVs detection (χ² = 162 0.1; p = 0.8 and χ² = 0.1; p = 0.7, respectively). For Chaerephon pusillus, female reproductive 163 status (pregnant or not pregnant) had no effect on the probability of CoVs detection (χ² = 2.50; CoVs. For α-CoVs, all sequences detected in our study of members of the family Molossidae 183 author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [{'start': 163, 'end': 171, 'text': 'Figure 3', 'ref_id': None}], 'section': '156'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/742866 doi: bioRxiv preprint 9 formed a highly supported monophyletic group, including CoV sequences from Molossidae Table S4 ). All', 'cite_spans': [], 'ref_spans': [{'start': 219, 'end': 227, 'text': 'Table S4', 'ref_id': None}], 'section': '156'}, {'text': 'CoVs found in Miniopteridae clustered in a monophyletic group, including Miniopteridae CoVs 189 sequences from Africa, Asia, and Oceania (Appendix Table S4 ). The great majority of α-CoVs Table S4 ). Hipposideridae α-CoVs mainly clustered into a single monophyletic group.', 'cite_spans': [], 'ref_spans': [{'start': 147, 'end': 155, 'text': 'Table S4', 'ref_id': None}, {'start': 188, 'end': 196, 'text': 'Table S4', 'ref_id': None}], 'section': '188'}, {'text': 'Hipposideridae CoV sequences from this study clustered with a 229E Human CoV-related bat 198 sequence found in Hipposideros vittatus from Kenya ( Figure 7 ) and demonstrated 93% simi-199 larity to 229E Human CoV (Appendix Table S4 ).', 'cite_spans': [], 'ref_spans': [{'start': 146, 'end': 154, 'text': 'Figure 7', 'ref_id': 'FIGREF29'}, {'start': 222, 'end': 230, 'text': 'Table S4', 'ref_id': None}], 'section': '197'}, {'text': 'Regarding the β-CoVs, two sequences obtained from Nycteris thebaica clustered in the Table S4 ). Rousettus madagascariensis CoV clustered with 204 Pteropodidae CoVs belonging to the D-subgroup of β-CoVs ( Figure 9 ). BLAST queries against 205 the NCBI database showed 98% nucleotide identity between CoV sequences from Rousettus 206 author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [{'start': 85, 'end': 93, 'text': 'Table S4', 'ref_id': None}, {'start': 205, 'end': 213, 'text': 'Figure 9', 'ref_id': None}], 'section': '200'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/742866 doi: bioRxiv preprint madagascariensis and a β-D CoV sequence detected in Eidolon helvum from Kenya (Appendix 207   Table S4 ). Rhinacovirus, and Nobecovirus) and three that could not be classified according to this taxo-245 nomic scheme hence representing unclassified subgenera (we propose \"Molacovirus\", \"Nycbe-246 covirus\", and \"Rhinacovirus2\").', 'cite_spans': [], 'ref_spans': [{'start': 219, 'end': 233, 'text': '207   Table S4', 'ref_id': None}], 'section': '200'}, {'text': 'In the context of WIO islands, we expect a strong geographical influence on CoVs diversity 248 rather than on host specificity, with independent evolution of CoVs on each island because of 249 spatial isolation and endemism. Anthony et al. (32) suggested that the dominant evolutionary 250 mechanism for African CoVs was host switching. In our study, congruence between host and 251 viral phylogenies suggests a strong signal for co-evolution between WIO bats and their associ-252 ated CoVs. This strongly suggests that individual bat species harbor specific CoV lineages, but 253 CoV transmission between different bat species may nevertheless occur infrequently. However, 254 author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [{'start': 225, 'end': 244, 'text': 'Anthony et al. (32)', 'ref_id': None}, {'start': 577, 'end': 580, 'text': '253', 'ref_id': None}], 'ref_spans': [], 'section': '247'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/742866 doi: bioRxiv preprint the geographical influence seems to occur within family specific groups, as in the WIO Molos-255 sidae CoV clade. Endemism resulting from geographic isolation may then have favored viral 256 diversification within family specific viral lineages.', 'cite_spans': [], 'ref_spans': [], 'section': '247'}, {'text': 'Although co-evolution could be the dominant mechanism, host-switching may take place The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/742866 doi: bioRxiv preprint 13 decade, they were both detected in bats in Kenya, and in Ghana, Gabon, Kenya, and Zimbabwe, 279 respectively (4,6,38,39). Moreover, CoVs notably similar to both NL63 and 229E have been contact with contaminated environments such as bat guano may also increase infection risk.', 'cite_spans': [{'start': 262, 'end': 268, 'text': 'Kenya,', 'ref_id': None}, {'start': 269, 'end': 282, 'text': 'and in Ghana,', 'ref_id': None}, {'start': 283, 'end': 289, 'text': 'Gabon,', 'ref_id': None}, {'start': 290, 'end': 296, 'text': 'Kenya,', 'ref_id': None}, {'start': 297, 'end': 310, 'text': 'and Zimbabwe,', 'ref_id': None}, {'start': 311, 'end': 314, 'text': '279', 'ref_id': None}], 'ref_spans': [], 'section': '257'}, {'text': 'Given the strong co-evolution of CoV and bats in the WIO, there is the potential of human 306 populations being exposed to CoV spillovers, which in part depends on contact with day roost The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/742866 doi: bioRxiv preprint Acknowledgments 320 We are very grateful to S. Muradrasoli for recommendations on the optimization of the multi-321 probe PCR protocol as well as for providing PCR controls and to C. Cordonin for providing bat The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/742866 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/742866 doi: bioRxiv preprint', 'cite_spans': [{'start': 334, 'end': 337, 'text': '320', 'ref_id': None}], 'ref_spans': [], 'section': '305'}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'striata _China_2007_NC011550', 'authors': [{'first': 'Munia_Cov_Hku13_3514_', 'middle': [], 'last': 'Lo', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF1': {'ref_id': 'b1', 'title': 'sp _China_2007_NC016991', 'authors': [{'first': 'White_Eye_Cov_Hku16_', 'middle': [], 'last': 'Zo', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF2': {'ref_id': 'b2', 'title': 'Bulbul_coronavirus_HKU11_934_ Py.jocosus _China_2007_FJ376619', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF3': {'ref_id': 'b3', 'title': 'Wigeon_COV_HKU20_ Ma.penelope _China_2008_NC016995 Night_Heron_COV_HKU19_ Ny.nycticorax _China_2007_NC_016994', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF4': {'ref_id': 'b4', 'title': 'Duck_coronavirus_isolate_DK_CH_HN_ZZ2004_ An.platyrhynchos _China_2004_JF705860', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF5': {'ref_id': 'b5', 'title': 'Infectious_bronchitis_virus_isolate_YN_ Ga.gallusdomesticus _China_2005_JF893452', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF6': {'ref_id': 'b6', 'title': 'Turkey_coronavirus_strain_TCoV_VA_74_03_ Me.gallopavo _GQ427173', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF7': {'ref_id': 'b7', 'title': 'sp _KF793826 Beluga_Whale_coronavirus_SW1_ De.leucas _NC010646', 'authors': [{'first': 'Bottlenose_Dolphin_Coronavirus_Hku22_Isolate_Cf090331_', 'middle': [], 'last': 'Tu', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF8': {'ref_id': 'b8', 'title': 'MERS_COV_2c_EMC_Human_ Ho.sapiens sapiens_2012_JX869059', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF9': {'ref_id': 'b9', 'title': 'MERS_COV_KFU_HKU_19Dam_ Ca.dromedarius _Saudi_Arabia_2013_KJ650296', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF10': {'ref_id': 'b10', 'title': 'MERS_COV_Hu_Riyadh_KSA_12160_Human_ Ho.sapiens sapiens_South_Korea_2016_KX154688 MERS_COV_Korea_Seoul_SNU1_035_Human_ Ho.sapiens sapiens_South_Korea_2015_KU308549 MERS_COV_D2731.3_14_ Ca.dromedarius _United_Arab_Emirates_KT751244 MERS_COV_ Ca', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF12': {'ref_id': 'b12', 'title': 'BtCoV_8_724_ Pi.pygmaeus _Romania_2009_KC243390', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF14': {'ref_id': 'b14', 'title': 'Bat_Coronavirus_HKU5_ Pi.abramus _China_2007_NC009020 Bat_coronavirus_16BF104_ Ep.serotinus _South_Korea_2016_KY432468', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF15': {'ref_id': 'b15', 'title': 'Bat_Coronavirus_HKU4_ Ty.pachypus _NC009019', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF16': {'ref_id': 'b16', 'title': 'Bat_CoV_FMNH_228899_Mozambique_ Ny.thebaica _2015_MN183194 Bat_CoV_FMNH_228903_Mozambique_ Ny.thebaica _2015_MN183195 Bat_CoV_FMNH_228900_Mozambique_ Ny.thebaica _2015_MN183193', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF17': {'ref_id': 'b17', 'title': 'Bat_CoV_FMNH_228901_Mozambique_ Ny.thebaica _2015_MN183196', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF18': {'ref_id': 'b18', 'title': 'BtCoV_KW2E_F53_ Ny.gambiensis _Ghana_2011_JX899384 BtCoV_KW2E_F93_ Ny.gambiensis _Ghana_2010_JX899383 BtCoV_KW2E_F82_ Ny.gambiensis _Ghana_2011_JX899382', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF19': {'ref_id': 'b19', 'title': 'BtCoV_KCR180_ Pt.parnellii _CostaRica_2012_KC633193 BtCoV_KCR230_ Pt.parnellii _CostaRica_2010_JQ731779', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF20': {'ref_id': 'b20', 'title': 'BtCoV_KCR15_ Pt.parnellii _Costa_Rica_2012_KC633195 BtCoV_KCR155_ Pt.parnellii _CostaRica_2012_KC633194', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF21': {'ref_id': 'b21', 'title': 'Bat_coronavirus_49_ Pt.davyi _Mexico_2012_KC886322', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF22': {'ref_id': 'b22', 'title': 'SARS_like_CoV_SL_CoVZXC21_ Rh.sinicus _China_2015_MG772934 Bat_SARS_COV_HKU3_ Rh.sinicus _China_2006_DQ022305', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF23': {'ref_id': 'b23', 'title': 'Bat_SARS_coronavirus_Rm1_ Rh.macrotis _China_2004_DQ412043', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF24': {'ref_id': 'b24', 'title': 'Bat_SARS_CoV_Rf1_ Rh.ferrumequium _China_2004_DQ412042', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF25': {'ref_id': 'b25', 'title': 'SARS_coronavirus_Tor2_Human_ Ho.sapiens sapiens_Canada_2003_NC004718 SARS_coronavirus_SZ3_Civet_ Pa.larvata HongKong_2003_AY304486 SARS_COV_HKU39849_Human_ Ho.sapiens sapiens_United_Kingdom_2003_JN854286', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF26': {'ref_id': 'b26', 'title': 'SARS_bat_coronavirus_Is39_ Rh.sp _Japan_2013_AB889995', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF28': {'ref_id': 'b28', 'title': 'SARS_related_BtCoV_ Rh.ferrumequinum _Luxembourg_2016_KY502395', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF29': {'ref_id': 'b29', 'title': 'Betacoronavirus_187632_2_ Rh.hipposideros _Italy_2012_KF500952', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF30': {'ref_id': 'b30', 'title': 'commersoni _Nigeria_2008_HQ166910 Bat_coronavirus_292_ Hi.caffer _Gabon_2009_JX174638', 'authors': [{'first': 'Zaria_Bat_Coronavirus_', 'middle': [], 'last': 'Hi', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF32': {'ref_id': 'b32', 'title': 'sp _NC_003045 Sable_Antelope_Cov_ Hi.niger _USA_2003_EF424621', 'authors': [{'first': 'Bovine_Cov_', 'middle': [], 'last': 'Bo', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF33': {'ref_id': 'b33', 'title': 'Human_coronavirus_OC43_ Ho.sapiens sapiens_AY391777 Rabbit_COV_HKU14_ Or.cuniculus _China_2006_NC_017083', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF34': {'ref_id': 'b34', 'title': 'MHV_COV_StrainA59_ Mu.musculus _USA_2014_MF618253 Human_coronavirus_HKU1_N11_genotype_A_ Ho.sapiens _DQ415908', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF35': {'ref_id': 'b35', 'title': 'pusillus _Congo_2014_KX285087 PREDICT_CD116088_ Ep.franqueto _Congo_2014_KX285095 PREDICT_GVF_CM_ECO70527_ Ep.gambianus _Cameroon_2013_KX285008 PREDICT_AATHA_ Ep.sp _Tanzania_2013_KX285299', 'authors': [{'first': 'Mi', 'middle': [], 'last': 'Predict_Cd116004_', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF36': {'ref_id': 'b36', 'title': 'Bat_CoV_UADBA_50625_Madagascar_ Ro.madagascariensis _2014_MN183191 Bat_CoV_UADBA_50636_Madagascar_ Ro.madagascariensis _2014_MN183192 Bat_coronavirus_KY24_ Ei.helvum _Kenya_2006_HQ728482', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF37': {'ref_id': 'b37', 'title': 'helvum _Kenya_2007_GU065433', 'authors': [{'first': 'Btky89_', 'middle': [], 'last': 'Ei', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF38': {'ref_id': 'b38', 'title': 'Bat_coronavirus_HKU9_ Ro.leschenaultii _China_NC009022 Bat_coronavirus_KY06_ Ro.aegyptiacus _Kenya_2006_HQ728484', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF39': {'ref_id': 'b39', 'title': 'Bat_coronavirus_HKU9_PREDICT_LAP11_K0006_ Eo.spelaea _Laos_2011_KX284911', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF40': {'ref_id': 'b40', 'title': 'Bat_coronavirus_Diliman1525G2_ Cy.brachyotis _Philippines_2008_AB539082 Bat_coronavirus_GLCC1_ Cy.sphinx _China_2005_KU182962', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF41': {'ref_id': 'b41', 'title': 'Bat_coronavirus_2265_ Pt.jagori _Philippines_2010_AB683971', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF42': {'ref_id': 'b42', 'title': 'PREDICT_CoV_22_LAP11_D0063_ Eo.spelaea _Laos_2011_KX284906 Bat_coronavirus_NWDC188_ Me.kusnotoi _China_2013_KU182986', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF43': {'ref_id': 'b43', 'title': 'Bat_coronavirus_RK074_ Eo.spelea _China_2010_KX520659', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF44': {'ref_id': 'b44', 'title': 'Bat_coronavirus_1573_ St.lilium _Brazil_2012_KT717392', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF45': {'ref_id': 'b45', 'title': 'scrofa _China_2015_KX083668', 'authors': [{'first': 'Transmissible_Gastroenteritis_Virus_', 'middle': [], 'last': 'Su', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF46': {'ref_id': 'b46', 'title': 'domesticus _USA_2014_KR270796', 'authors': [{'first': 'Porcine_Respiratory_Coronavirus_', 'middle': [], 'last': 'Su', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF47': {'ref_id': 'b47', 'title': 'Canine_coronavirus_strain_171_ Ca.lupus familiaris _Germany_1971_KC175339 Feline_coronavirus_UU15_ Fe.catus _Netherlands_2007_FJ938057', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF48': {'ref_id': 'b48', 'title': 'vison _USA_1998_HM245925', 'authors': [{'first': 'Mink_Coronavirus_', 'middle': [], 'last': 'Mu', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF50': {'ref_id': 'b50', 'title': 'sp _China_MF094687', 'authors': [{'first': 'Sads_Related_Coronavirus_', 'middle': [], 'last': 'Rh', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF51': {'ref_id': 'b51', 'title': 'sp _China_2004_DQ249235', 'authors': [{'first': '−1_', 'middle': [], 'last': 'Bat_Coronavirus_Hku2', 'suffix': ''}, {'first': '', 'middle': [], 'last': 'Rh', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF52': {'ref_id': 'b52', 'title': 'Bat_coronavirus_MLHJC4_ Rh.sinicus _China_2012_KU182954', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF54': {'ref_id': 'b54', 'title': 'BtRf_AlphaCoV_YN2012_ Rh.ferrumequinum _China_2012_KJ473808', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF55': {'ref_id': 'b55', 'title': 'Bat_CoV_FMNH_228954_Mozambique_ Rh.lobatus _2015_MN183189 Bat_CoV_FMNH_228963_Mozambique_ Rh.rhodesiae _2015_MN183190', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF56': {'ref_id': 'b56', 'title': 'mossambicus _2015_MN183158 Bat_CoV_FMNH_228978_Mozambique_ Mi.mossambicus _2015_MN183157 Bat_CoV_FMNH_228949_Mozambique_ Rh.rhodesiae _2015_MN183159 Bat_CoV_FMNH_228980_Mozambique_ Mi.mossambicus _2015_MN183160', 'authors': [{'first': 'Mi', 'middle': [], 'last': 'Bat_Cov_Fmnh_228969_Mozambique_', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF57': {'ref_id': 'b57', 'title': 'natalensis _Kenya_2006_HQ728484 Bat_CoV_FMNH_228971_Mozambique_ Mi.mossambicus _2015_MN183161', 'authors': [{'first': 'Mi', 'middle': [], 'last': 'Miniopterus Bat_Coronavirus_Ky27_', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF58': {'ref_id': 'b58', 'title': 'sp _HongKong_2008_EU420138', 'authors': [{'first': 'Mi', 'middle': [], 'last': 'Miniopterus Bat_Coronavirus_1a_', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF59': {'ref_id': 'b59', 'title': 'fuliginosus _China_2013_KJ473803', 'authors': [{'first': 'Mi', 'middle': [], 'last': 'Btmf_Alphacov_Hub2013_', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF60': {'ref_id': 'b60', 'title': 'magnater _China_2005_DQ666337', 'authors': [{'first': 'Mi', 'middle': [], 'last': 'Bat_Coronavirus_1a_', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF61': {'ref_id': 'b61', 'title': 'fuliginosus _China_2011_KJ473795', 'authors': [{'first': 'Mi', 'middle': [], 'last': 'Btmf_Alphacov_Ah2011_', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF62': {'ref_id': 'b62', 'title': 'sp _HongKong_2008_EU420137', 'authors': [{'first': 'Mi', 'middle': [], 'last': 'Bat_Coronavirus_1b_', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF63': {'ref_id': 'b63', 'title': 'magnater _DQ249226', 'authors': [{'first': 'Mi', 'middle': [], 'last': 'Bat_Coronavirus_Hku7_', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF64': {'ref_id': 'b64', 'title': 'Bat_coronavirus_Anlong_171_ Mi.schreibersii _China_2012_KF294275', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF65': {'ref_id': 'b65', 'title': 'Bat_coronavirus_CoV132_ Mi.australis _Australia_2007_EU834954 Bat_coronavirus_CoV100_ Rh.megaphyllus _Australia_2007_EU834953', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF66': {'ref_id': 'b66', 'title': 'pusillus _China_2004_DQ249228', 'authors': [{'first': 'Mi', 'middle': [], 'last': 'Bat_Coronavirus_Hku8_', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF68': {'ref_id': 'b68', 'title': 'Alphacoronavirus_ Ny.lasiopterus _Spain_2007_HQ184055 BNM98_30_ Ny.leisleri _Bulgaria_2008_GU190239', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF69': {'ref_id': 'b69', 'title': 'Pip1_Cr_ Pi.pipistrellus _France_2014_KT345294', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF71': {'ref_id': 'b71', 'title': 'BtNv_AlphaCoV_SC2013_ Ny.velutinus _China_2013_KJ473809', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF72': {'ref_id': 'b72', 'title': 'Alphacoronavirus_Iprima_ Pi.kuhlii _Spain_2007_HQ184058', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF73': {'ref_id': 'b73', 'title': 'condylurus _2015_MN183181 Bat_CoV_FMNH_229266_Mozambique_ Mo.condylurus _2015_MN183180 Bat_CoV_FMNH_229296_Mozambique_ Mo.condylurus _2015_MN183179 Bat_CoV_FMNH_229303_Mozambique_ Mo.condylurus _2015_MN183182', 'authors': [{'first': 'Bat_Cov_Fmnh_229293_Mozambique_', 'middle': [], 'last': 'Mo', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF74': {'ref_id': 'b74', 'title': 'Chaerephon_bat_coronavirus_KY22_ Ch.sp _Kenya_2006_HQ728486 BtCoV_NCL_MCO1_ Mo.condylurus _South_Africa_2012_KF843853 PREDICT_GVF_CM_ECO70102_ Mo.condylurus _Cameroon_2013_KX284982', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF75': {'ref_id': 'b75', 'title': 'Bat_CoV_MAY004_Mayotte_ Ch.pusillus _2014_MN183177 Bat_CoV_MAY033_Mayotte_ Ch.pusillus _2014_MN183178 Bat_CoV_MAY015_Mayotte_ Ch.pusillus _2014_MN183173 Bat_CoV_MAY051_Mayotte_ Ch.sp _2014_MN183174', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF76': {'ref_id': 'b76', 'title': 'Bat_CoV_MAY025_Mayotte_ Ch.pusillus _2014_MN183175 Bat_CoV_MAY027_Mayotte_ Ch.pusillus _2014_MN183176', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF77': {'ref_id': 'b77', 'title': 'jugularis _2013_MN183187 Bat_CoV_FMNH_222698_Madagascar_ Mo.jugularis _2013_MN183185 Bat_CoV_UADBA_33735_Madagascar_ Mo.jugularis _2013_MN183186 Bat_CoV_UADBA_33728_Madagascar_ Mo.jugularis _2013_MN183183', 'authors': [{'first': 'Bat_Cov_Fmnh_222691_Madagascar_', 'middle': [], 'last': 'Mo', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF78': {'ref_id': 'b78', 'title': 'jugularis _2013_MN183184 Bat_CoV_RB369_Reunion_ Mo.francoismoutoui_2015_MN183188', 'authors': [{'first': 'Bat_Cov_Uadba_33733_Madagascar_', 'middle': [], 'last': 'Mo', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF80': {'ref_id': 'b80', 'title': 'Bat_coronavirus_POA_ Uknown _Brazil_2012_144_KC110778', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF81': {'ref_id': 'b81', 'title': 'BtMs_AlphaCoV_GS2013_ My.sp _China_2013_KJ473810', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF82': {'ref_id': 'b82', 'title': 'BtRf_AlphaCoV_HuB2013_ Rh.ferrumequinum _China_2013_KJ473807', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF83': {'ref_id': 'b83', 'title': 'BtCoV_4307_1_ Rh.affinis _China_2013_KP876527', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF84': {'ref_id': 'b84', 'title': 'Bat_COV_HKU10_175A_ Ro.leschenaultii _China_2005_JQ989271', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF85': {'ref_id': 'b85', 'title': 'Bat_COV_HKU10_TLC1347A_ Hi.pomona _China_2010_JQ989273', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF86': {'ref_id': 'b86', 'title': 'Bat_coronavirus_2231_ Hi.diadema _Philippines_2010_AB683971', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF87': {'ref_id': 'b87', 'title': 'Bat_CoV_FMNH_228956_Mozambique_ Rh.rhodesiae _2015_MN183149 Bat_CoV_FMNH_228955_Mozambique_ Rh.rhodesiae _2015_MN183151 Bat_CoV_FMNH_228962_Mozambique_ Rh.rhodesiae _2015_MN183147 Bat_CoV_FMNH_228957_Mozambique_ Rh.rhodesiae _2015_MN183150 Bat_CoV_FMNH_228945_Mozambique_ Rh.rhodesiae _2015_MN183146 Bat_CoV_FMNH_228953_Mozambique_ Rh.rhodesiae _2015_MN183148', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF88': {'ref_id': 'b88', 'title': 'Bat_CoV_FMNH_228952_Mozambique_ Rh.rhodesiae _2015_MN183153 Bat_CoV_FMNH_228939_Mozambique_ Rh.lobatus _2015_MN183154 Bat_CoV_FMNH_228951_Mozambique_ Rh.sp _2015_MN183152', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF89': {'ref_id': 'b89', 'title': 'Bat_CoV_FMNH_228940_Mozambique_ Rh.lobatus _2015_MN183155 Bat_CoV_FMNH_228941_Mozambique_ Rh.lobatus _2015_MN183156 Kenya bat_coronavirus_BtKY83_ Rh.sp _Kenya_2007_GU065427', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF90': {'ref_id': 'b90', 'title': 'Bat_coronavirus_KY43_ Ca.cor _Kenya_2006_HQ728480', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF92': {'ref_id': 'b92', 'title': 'BtCoV_KP565_ Ar.jamaicensis _Panama_2010_JQ731786 Bat_coronavirus_Aju21_ Ar.jamaicensis _Costa_Rica_2012_KC779226', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF93': {'ref_id': 'b93', 'title': 'BtCoV_KP256_ Ar.jamaicensis _Panama_2010_JQ731784', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF94': {'ref_id': 'b94', 'title': 'BtCoV_LUX15_A_59_ My.emarginatus _Luxembourg_2015_KY502386 BtCoV_LUX15_A_46_ My.emarginatus _Luxembourg_2015_KY502383', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF95': {'ref_id': 'b95', 'title': 'Alphacoronavirus_13rs384_31_ My.blythii _Italy_2012_KF312400', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF96': {'ref_id': 'b96', 'title': 'Bat_coronavirus_Anlong_60__ Ia.io _China_2013_KY770857', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF97': {'ref_id': 'b97', 'title': 'Bat_coronavirus_Anlong_46_ Rh.pusillus _China_2013_KY770855', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF98': {'ref_id': 'b98', 'title': 'currentium _Brazil_2009_JQ731800 BtCoV_BRA182_ Mo.rufus _Brazil_2009_JQ731799', 'authors': [{'first': 'Btcov_Brap103_', 'middle': [], 'last': 'Mo', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF99': {'ref_id': 'b99', 'title': 'pygmaeus _Germany_2007_EU375867 Bat_coronavirus_VM105_ My.dasycneme _Netherlands_2006_GQ259968', 'authors': [{'first': '', 'middle': [], 'last': 'Bat_Coronavirus_D5', 'suffix': ''}, {'first': '', 'middle': [], 'last': 'Pi', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF100': {'ref_id': 'b100', 'title': 'Bat_coronavirus_Moxy4235_IT_16_ My.oxygnathus _Italy_2016_KY780395 Bat_coronavirus_HKU6_ My.pilosus _China_2004_DQ249224', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF101': {'ref_id': 'b101', 'title': 'CoV_75_55_L01_R1_CoV_BAT_VN_ Sc.kuhlii _Vietnam_KX092163', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF102': {'ref_id': 'b102', 'title': 'Rocky_Mountain_Bat_Coronavirus_11_ My.lucifugus _USA_2011_EF544563 Bat_coronavirus_ My.lucifugus _Canada_2010_KY799179', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF103': {'ref_id': 'b103', 'title': 'domesticus _1993_NC003436', 'authors': [{'first': 'Pedv_Porcine_Epidemic_Cov_', 'middle': [], 'last': 'Su', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF104': {'ref_id': 'b104', 'title': 'Camel229E_CoV_AC04_ Ca.dromedarius _SouthKorea_2014_KT253327', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF105': {'ref_id': 'b105', 'title': 'pacos _USA_2008_JQ410000 229E_related_bat_CoV_BtCov_KY229E_8_ Hi.vittatus _Kenya_2010_KY073748', 'authors': [{'first': 'Alpaca_Respiratory_Coronavirus_Isolate_Ca08_1_', 'middle': [], 'last': 'Vi', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF106': {'ref_id': 'b106', 'title': 'caffer _2015_MN183170 Bat_CoV_FMNH_228926_Mozambique_ Hi.caffer _2015_MN183171 Bat_CoV_FMNH_228933_Mozambique_ Hi.caffer _2015_MN183172', 'authors': [{'first': 'Bat_Cov_Fmnh_228981_Mozambique_', 'middle': [], 'last': 'Hi', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF107': {'ref_id': 'b107', 'title': '229E_related_bat_CoV_BtCoV_AT1A_F45_ Hi.abae _Ghana_2010_KT253259 Human_coronavirus_229E_ Ho.sapiens sapiens_NC002645 229E_related_bat_CoV_BtCoV_KW1C_F161_ Hi.ruber _Ghana_2010_KT253264', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF108': {'ref_id': 'b108', 'title': 'Bat_CoV_FMNH_229241_Mozambique_ Tr.afer _2015_MN183166 Bat_CoV_FMNH_229212_Mozambique_ Tr.afer _2015_MN183164', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF109': {'ref_id': 'b109', 'title': 'afer _2015_MN183167', 'authors': [{'first': 'Bat_Cov_Fmnh_228892_Mozambique_', 'middle': [], 'last': 'Tr', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF110': {'ref_id': 'b110', 'title': 'NL63_related_bat_coronavirus_BtKYNL63_15_ Tr.afer _Kenya_2008_KY073746 Bat_CoV_FMNH_228887_Mozambique_ Tr.afer _2015_MN183162', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF111': {'ref_id': 'b111', 'title': 'Bat_CoV_FMNH_228891_Mozambique_ Tr.afer _2015_MN183163 Bat_CoV_FMNH_229232_Mozambique_ Tr.afer _2015_MN183165', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF112': {'ref_id': 'b112', 'title': 'Bat_CoV_FMNH_228888_Mozambique_ Tr.afer _2015_MN183168', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF113': {'ref_id': 'b113', 'title': 'Bat_CoV_FMNH_229243_Mozambique_ Tr.afer _2015_MN183169 NL63_related_bat_coronavirus_BtKYNL63_9a_ Tr.afer _Kenya_2010_KY073744 Human_Cov_NL63_ Ho.sapiens sapiens_Netherlands_2004_NC005831', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF114': {'ref_id': 'b114', 'title': 'Human_coronavirus_NL63_isolate_HCOV_005_ Ho.sapiens sapiens_ Kenya_2009_KP112154 Bat_coronavirus_1CO7BA_ Gl.soricina _Trinidad_2007_EU769558', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF115': {'ref_id': 'b115', 'title': 'Bat_coronavirus_strain_328_11_ Mo.molossus _Brazil_2011_KX094982 BtCoV_KP816_ Ph.discolor _Panama_2011_JQ731782', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF116': {'ref_id': 'b116', 'title': 'rotundus _Brazil_2013_KU552072', 'authors': [{'first': 'Bat_Coronavirus_4292_', 'middle': [], 'last': 'De', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF117': {'ref_id': 'b117', 'title': 'BtCoV_KP534_ Ar.jamaicensis _Panama_2010_JQ731787', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF118': {'ref_id': 'b118', 'title': 'BtCoV_BRA118_ Ca.perspicillata _Brazil_2009_JQ731796', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF119': {'ref_id': 'b119', 'title': 'Bat_coronavirus_1599_ Ca.perspicillata _Brazil_2012_KT717385 Bat_cov_309_ Ca.castanea _Costa_Rica_2014_KM215147 BtCoV_KCR253_ Ca', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}}, 'ref_entries': {'FIGREF0': {'text': '50 fertilization. Bats are also recognized as reservoirs of many zoonotic pathogens. Several bat-51 borne coronaviruses (CoVs) have recently emerged in humans and livestock with sometimes 52 major impacts to public health. For instance, in 2003, the Severe Acute Respiratory Syndrome 53 (SARS) outbreak in China spread to 30 countries, infecting 8,096 people and leading to 774 54 deaths in less than a year (2). The Middle East Respiratory Syndrome (MERS) is caused by a 55 camel-associated CoV that likely originated from bats, and in 2003 afflicted humans in Saudi 56 Arabia and elsewhere, infecting 2,442 people with 842 associated deaths worldwide (3).', 'latex': None, 'type': 'figure'}, 'FIGREF2': {'text': '70 longing to 39 species and sampled on five islands (Madagascar, Mauritius, Mayotte, Reunion 71 Island, and Mahé) and one continental area (Mozambique). Based on molecular screening and 72 partial sequencing of the RNA-dependent RNA polymerase gene, we (i) estimated CoV preva-73 lence in the regional bat populations, (ii) assessed CoVs genetic diversity, and (iii) identified 74 potential association between bat families and CoVs and evolutionary drivers leading to these from vouchered bat specimens during previous studies in Mozam-80 bique (February to May 2015), Mayotte (November to December 2014), Reunion Island (Feb-81 ruary 2015), Seychelles (February to March 2014), Mauritius (November 2012) and Madagas-82 car (October to November 2014) were tested (8-11) (Technical Appendix). We also collected 83 additional swab samples from several synanthropic bat species on Madagascar, in January 2018 84 (Technical Appendix). A total of 1,099 bats were tested (Figure 1). Details on sample types, bat 85 families, species, and locations are provided in', 'latex': None, 'type': 'figure'}, 'FIGREF3': {'text': \"Time (RT) PCR (13). The primer set with Locked Nucleic Acids (LNA; underlined position in 98 probe sequences) was purchased from Eurogentec (Seraing, Belgium): 11-FW: 5'-TGA-TGA-99 TGS-NGT-TGT-NTG-YTA-YAA-3' and 13-RV: 5'-GCA-TWG-TRT-GYT-GNG-ARC-100 ARA-ATT-C-3'. Three probes were used: probe I (ROX): 5'-TTG-TAT-TAT-CAG-AAT-101 GGY-GTS-TTY-AT-3', probe II (FAM): 5'-TGT-GTT-CAT-GTC-WGA-RGC-WAA-ATG-102 TT-3', and probe III (HEX): 5'-TCT-AAR-TGT-TGG-GTD-GA-3'. RT-PCR was performed 103 with ABsolute Blue QPCR Mix low ROX 1X (Thermo Fisher Scientific, Waltham, MA, USA) 104 and 2.5 µL of cDNA under the following thermal conditions: 95 °C for 15 min, 95 °C for 30 s, 105 touchdowns from 56 °C to 50°C for 1 min and 50 cycles with 95 °C for 30 s and 50 °C for 60 106 s in a CFX96 Touch ™ Real-Time PCR Detection System (Bio-Rad, Hercules, CA, USA).\", 'latex': None, 'type': 'figure'}, 'FIGREF4': {'text': \"Because of the limited size of the sequence generated from the RT-PCR, a second PCR108 targeting 440 bp of the RdRp gene was performed with 5 µL of cDNA of each positive sample, 109 with the following primer set: IN-6: 5'-GGT-TGG-GAC-TAT-CCT-AAG-TGT-GA-3' and IN-110 author/funder. All rights reserved. No reuse allowed without permission.\", 'latex': None, 'type': 'figure'}, 'FIGREF5': {'text': '122 pregnant) was investigated for species in which we detected at least 15 CoV positive individu-123 als. Finally, the effect of sampling during the wet (February) or dry (May) season was investi-124 gated in Mozambique. Analyses were conducted with R v3.5.1 software (15).', 'latex': None, 'type': 'figure'}, 'FIGREF6': {'text': 'with the second PCR system (14) were edited with the Chromas 128 Lite Software package version 2.6.4 (16). We explored CoV diversity of the sequences with 129 pairwise identity values obtained from seqidentity function in R bio3d package v2.3-4 (17) and 130 identified the most similar CoV RdRp sequences referenced in GenBank using BLASTN 131 2.2.29+. An alignment was then generated using the 51 nucleotide sequences obtained in this 132 study and 151 reference CoV sequences from a large diversity of host family and geographic 133 origins (Europe, Asia, Oceania, America and Africa), with CLC Sequence viewer 8.0 Software 134 author/funder. All rights reserved. No reuse allowed without permission.', 'latex': None, 'type': 'figure'}, 'FIGREF7': {'text': \"Host-virus associations were investigated using the phylogeny of WIO bats and their 138 associated CoVs. Bat phylogeny was generated from an alignment of 1,030 bp of mitochondrial 139 Cytochrome b (Cyt b) gene sequences downloaded from GenBank and sequenced for this study 140 (AppendixTable S2), for each CoV positive bat species with available sequence data. Finally,141 bat and pruned CoV phylogenies based on each 393 bp RdRp unique sequence fragment were 142 generated by Neighbor-Joining with 1,000 bootstrap iterations, using CLC Sequence viewer 143 8.0 Software (CLC Bio, Aarhus, Denmark)(20). Phylogenetic congruence was tested (21) to 144 assess the significance of the coevolutionary signal between bat host species and CoVs se-145 quences, using ParaFit with 999 permutations in the 'ape' package v5.0 in R 3.5.1 (22). Tan-146 glegram representations of the co-phylogeny were visualized using the Jane software v4.01 147 (23).\", 'latex': None, 'type': 'figure'}, 'FIGREF9': {'text': '88 positive samples, we obtained 77 partial RdRp sequences using the RT-PCR170 detection system (179 bp) and 51 longer partial RdRp sequences using the standard PCR system 171 (440 bp): this latter was subsequently used for phylogenetic analyses. Details of the sequenced 172 CoV-positive samples are given in Appendix Table S3. Pairwise comparison of these 51 se-173 quences revealed 28 unique sequences, and sequences similarities ranging from 60.2% to 174 99.8%. The lowest sequence similarity was found in Mozambique (60.2% to 99.8%), then in 175 Madagascar (64.0% to 99.8%). No genetic variation was observed for samples from Mayotte 176 and Reunion Island.', 'latex': None, 'type': 'figure'}, 'FIGREF10': {'text': 'indicated that WIO bats harbor a high diversity of both α and β-180 CoVs, with conserved clade groups clustering mostly by bat family(Figure 4). Specifically, 25 181 sequences were identified as α-CoVs, and three sequences were genetically related to the β-182', 'latex': None, 'type': 'figure'}, 'FIGREF11': {'text': 'Mozambique), Mormopterus francoismoutoui (Reunion Island), Chaerephon pusillus and 186 Chaerephon sp. (Mayotte), and Mormopterus jugularis (Madagascar) shared 90% -98% nu-187 cleotide similarity with a CoV detected in Chaerephon sp. in Kenya (Appendix', 'latex': None, 'type': 'figure'}, 'FIGREF12': {'text': '190 detected in Rhinolophidae bats clustered in two monophyletic groups (Figure 4); one with Af-191 rican Rhinolophidae CoVs and one with Asian Rhinolophidae CoVs. We additionally detected 192 one CoV from Rhinolophus rhodesiae, which was 100% similar to a Miniopteridae CoV from 193 this study. Rhinonycteridae CoVs formed a single monophyletic group with NL63 Human 194 CoVs. The Rhinonycteridae CoVs detected clustered with NL63-related bat sequences found 195 in Triaenops afer in Kenya (Figure 6) and showed 85% similarity to NL63 Human CoVs (Ap-196 pendix', 'latex': None, 'type': 'figure'}, 'FIGREF13': {'text': '-subgroup of β-CoVs together with other CoVs previously reported in African Nycteris sp. 202 bats (Figure 8). The sequences showed 88% nucleotide identity to a β-C CoV found in Nycteris 203 gambiensis in Ghana (Appendix', 'latex': None, 'type': 'figure'}, 'FIGREF14': {'text': 'phylogeny tests were conducted using 11 Cyt b sequences obtained from the 11 211 CoVs positive bat species and 27 partial CoV RdRp sequences (440 bp). Results supported co-212 evolution between the WIO bats and their CoVs (provide evidence for a high diversity of CoVs in bat populations on WIO islands. The 217 overall prevalence of CoV positive bats was consistent with other studies from continental Af-218 rica (5) and island systems in the Australasian region (24). However, in the case of WIO islands, 219 prevalence rates showed considerable variations based on bat family, species, landmass, and 220 season. 221 We found a significant effect of the roosting habitat on the probability of CoV detection, 222 with higher detection in bats occupying caves and buildings than outdoor habitats. Bats use 223 different types of day roosts sites, including caves, rock crevices, tree cavities, forest vegetation, 224 or synanthropic structures (25). Critical for this study, roosting site choice could be a risk factor 225 for infection of bats (26). This was indeed the case with higher detection rates for bats using 226 caves and buildings, as compared to other roost sites. Shelters like caves or buildings (bridges, 227 houses, etc.) may protect excreted viral particles from rainfall, temperature, humidity variation, 228 and ultraviolet radiation (27,28). Thus, these roost types may facilitate the maintenance and 229 transmission of viruses between syntopic species. Moreover, the accumulation of guano in these is well known for some bat CoVs subgenera (30,31). For example, β-C 235 CoVs are largely associated with Vespertilionidae bats, whereas β-D CoVs are found mostly in 236 Pteropodidae (31,32). In our study, we showed that WIO bats harbor genetic structured CoVs, 237 of both α-CoV and β-CoV subclades, clustering mostly by bat family. In the new CoV taxon-238 omy based on full genomes proposed by the International Committee of Taxonomy of Viruses 239 (ICTV), α-CoVs and β-CoVs are split in subgenera mostly based on host families (33), reflected 240 in the subgenera names (e.g. Rhinacovirus for a Rhinolophidae α-CoV cluster, Minuacovirus 241 for a Miniopteridae α-CoV cluster, Hibecovirus for an Hipposideridae β-CoV cluster). Alt-242 hough our classification was based on a partial sequence of the RdRp region, we identified 243 sequences from samples belonging to four of these subgenera (Minuacovirus, Duvinacovirus, 244', 'latex': None, 'type': 'figure'}, 'FIGREF15': {'text': '258 in certain situations. For example, in Mozambique we found a potential Miniopteridae CoV in 259 a Rhinolophidae bat co-roosting with Miniopteridae in the same cave. These host-switching 260 events could be favored when several bat species roost in the same portion of a cave in sym-261 patria (34). A similar scenario was described on Australia where Miniopteridae CoV was de-262 tected in Rhinolophidae bats (24). These infrequent host-switching events show that spillovers 263 can happen but suggest that viral transmission is not maintained independently within the spill-264 over host. Thus, the co-evolution of virus and host might have resulted in strong adaptation of 265 the CoVs to the specific bat species. In addition, viral factors (mutation rate, recombination 266 propensity, replication ability in the cytoplasm, changes in the ability to bind host cells), envi-267 ronmental factors (climate variation, habitat degradation, decrease of bat preys), and phyloge-268 netic relatedness of host species are also critical for the viral establishment in a novel host (35). 269 Nevertheless, apparent evidence of host switching as a dominant mechanism of CoV evolution 270 could be an artifact of a lack of data for some potential bat hosts, leading to incomplete phylo-271 genetic reconstructions (32). 272 Several bat CoVs we identified in Rhinonycteridae and Hipposideridae from Mozambique 273 had between 85% and 93% nucleotide sequence similarity with NL63 Human CoVs and 229E 274 Human CoVs, respectively. These two human viruses are widely distributed in the world and 275 associated with mild to moderate respiratory infection in humans (36). Tao et al. established 276 that the NL63 Human CoVs and 229E Human CoVs have a zoonotic recombinant origin from 277 their most recent common ancestor, estimated to be about 1,000 years ago (37). During the past 278 author/funder. All rights reserved. No reuse allowed without permission.', 'latex': None, 'type': 'figure'}, 'FIGREF16': {'text': '280 described in Kenya and Mozambique bats, suggesting an East African bats origin of these hu-281 man viruses. Intermediate hosts are important in the spillover of CoVs, despite gaps in direct 282 and indirect transmission routes of bat infectious agents to secondary hosts (40). This hypothe-283 sis has been formulated for the 229E Human CoV, with an evolutionary origin in Hippo-284 sideridae bats and with camelids as intermediate hosts (39). Similarly, the spillover of NL63 285 from Rhinonycteridae bats to humans might have occurred through a currently unidentified 286 intermediate host (6,41,42). Because receptor recognition by viruses is the first essential cellular 287 step to infect host cells, CoVs may have spilt over into humans from bats through an interme-288 diate host possibly due to mutations on spike genes (6,43). Further investigations of CoVs in 289 Kenyan and Mozambican livestock and hunted animals could potentially provide information 290 on the complete evolutionary and emergence history of these viruses before their establishment 291 in humans. 292 MERS-like CoV, with high sequence similarity (>85%) to human and camel strains of 293 MERS-CoV, have been detected in Neoromicia capensis in South Africa and Pipistrellus cf. 294 hesperidus in Uganda, suggesting a possible origin of camel MERS-CoV in Vespertilionidae 295 bats(5,32,44). This family has been widely studied, with 30% of all reported bat CoVs se-296 quences from the past 20 years coming from vespertilionids (45), including MERS-like CoVs.297No members of this family were positive for CoV in our study, which may be associated with 298 the low number of individuals sampled; additional material is needed to explore potential 299 MERS-like CoV on Madagascar.', 'latex': None, 'type': 'figure'}, 'FIGREF17': {'text': 'CoV transmission risk also depends on ecological factors and human actions such as en-301 croachment, landscape uses, and cultural traditions. On Madagascar, for example, bats are 302 author/funder. All rights reserved. No reuse allowed without permission.', 'latex': None, 'type': 'figure'}, 'FIGREF18': {'text': '307sites and different ecological contexts, as well as the bat families present. Moreover, there is 308 evidence of a relationship between biodiversity loss and the risk of infectious disease emer-309 gence (48).', 'latex': None, 'type': 'figure'}, 'FIGREF19': {'text': 'At a worldwide scale, epidemiological studies with serological surveys on human popula-311 tions are necessary to investigate potential CoV transmission from bats to humans (49,50) and 312 should be considered on WIO islands. Even if CoV prevalence in bats is higher on Madagascar 313 than other oceanic islands around the world, the high CoV genetic diversity in Malagasy bats 314 could complicate the development of detection protocols in human populations. On oceanic 315 islands, however, the development of serological tests for particular coronaviruses found in 316 endemic bat species is achievable, and underlines the need to develop virus surveillance in local 317 populations based on results from bat studies.', 'latex': None, 'type': 'figure'}, 'FIGREF20': {'text': \"Cyt b sequences. L. Biscornet, C. Dionisio, L. Domergue, M. Dietrich, T. Mbohoahy, T. 323 Nekena, J. Rakotoarivelo, M. Rakotomanga, C. F. Rakotondramanana, and A. Randrenjarison 324 are thanked for their assistance in the field. We also thank S. Bos and A. Hoarau for their help 325 in the laboratory, and K. Dellagi and H. Pascalis for the development and the management of 326 the 'partenariat Mozambique-Réunion dans la recherche en santé: pour une approche intégrée 327 d'étude des maladies infectieuses à risque épidémique (MoZaR)' research program. research was funded by the 'Pathogènes associés à la Faune Sauvage Océan Indien (FS-331 OI)', the 'Leptospirose Ocean Indien (LeptOI)', the 'Paramyxovirus Océan Indien (Para-332 myxOI)', and the 'Partenariat Mozambique-Réunion dans la recherche en santé : pour une ap-333 proche intégrée d'étude des maladies infectieuses à risque épidémique (MoZaR)' programs 334 (Fond Européen de Développement Régional, Programme Opérationnel de Coopération Terri-335 toriale). Fieldwork on Mayotte was funded by the 'Centre National de la Recherche Scien-336 tifique' (Projets Exploratoires Premier Soutien BATMAN). Molecular analyses were finan-337 cially supported by tutorship institutions of the UMR PIMIT. LJ is a recipient of a 'Région 338 Réunion, European Regional Development Funds (FEDER 2014-2020)' PhD fellowship. BR's 339 post-doctoral fellowship was supported by the 'Run Emerge' European Union's Seventh 340 Framework Program (FP7/2007-2013; Grant agreement NO 263958), the 'Fonds de Coopé-341 author/funder. All rights reserved. No reuse allowed without permission.\", 'latex': None, 'type': 'figure'}, 'FIGREF21': {'text': 'M, Pfukenyi DM, Boué V, Talignani L, Chiweshe N, Diop F, et al. Circula-359 tion of alphacoronavirus, betacoronavirus and paramyxovirus in Hipposideros bat spe-360 cies in Zimbabwe. Infect Genet Evol. 2018 Mar 1;58:253-7.3615. Ithete NL, Stoffberg S, Corman VM, Cottontail VM, Richards LR, Schoeman MC, et 362 al. Close relative of human Middle East respiratory syndrome coronavirus in bat, South 363 Africa. Emerg Infect Dis. 2013;19(10):1697-9. 364 6. Tao Y, Shi M, Chommanard C, Queen K, Zhang J, Markotter W, et al. Surveillance of 365 bat coronaviruses in Kenya identifies relatives of human coronaviruses NL63 and 229E 366 and their recombination history. J Virol. 2017;91(5):e01953-16. 367 7. Razanajatovo NH, Nomenjanahary LA, Wilkinson DA, Razafimanahaka JH, Goodman 368 SM, Jenkins RK, et al. Detection of new genetic variants of betacoronaviruses in en-369 demic frugivorous bats of Madagascar. Virol J. 2015 Dec 12;12al. Highly diverse morbillivirus-related paramyxoviruses in wild fauna of the South-375 western Indian Ocean islands: evidence of exchange between introduced and endemic 376 small mammals. J Virol. 2014;88(15):8268-77. 377 10. Gomard Y, Dietrich M, Wieseke N, Ramasindrazana B, Lagadec E, Goodman SM, et 378 al. Malagasy bats shelter a considerable genetic diversity of pathogenic Leptospira sug-379 gesting notable host-specificity patterns. FEMS Microbiol Ecol. 2016;92(4):fiw037. 380 11. Mélade J, Wieseke N, Ramasindrazana B, Flores O, Lagadec E, Gomard Y, et al. Muradrasoli S, Mohamed N, Hornyák Á, Fohlman J, Olsen B, Belák S, et al. Broadly 387 targeted multiprobe QPCR for detection of coronaviruses: coronavirus is common 388 among mallard ducks (Anas platyrhynchos). J Virol Methods. 2009;159(2):277-87. 389 14. Poon LLM, Chu DKW, Chan KH, Wong OK, Ellis TM, Leung YHC, et al. Identifica-390 tion of a novel coronavirus in bats. J Virol. 2005 Feb;79(4):2001-9. 391 15. R Core Team. R: A language and environment for statistical computing. R Foundation 392 for Statistical Computing, Vienna, Austria. 2018. 393 16. Technelysium Pty. Ltd. Chromas Lite Software. 2018.394 17. Grant BJ, Rodrigues APC, ElSawy KM, McCammon JA, Caves LSD. Bio3d: an R 395 package for the comparative analysis of protein structures. Bioinformatics. Keane TM, Creevey CJ, Pentony MM, Naughton TJ, Mclnerney JO. Assessment of 400 methods for amino acid matrix selection and their use on empirical data shows that ad 401 hoc assumptions for choice of matrix are not justified. BMC Evol Biol. Conow C, Fielder D, Ovadia Y, Libeskind-Hadas R. Jane: a new tool for the cophylog-409 eny reconstruction problem. Algorithms Mol Biol. 2010 Feb 3;5:16. 410 24. Smith CS, de Jong CE, Meers J, Henning J, Wang L-F, Field HE. Coronavirus infec-411 tion and diversity in bats in the Australasian region. Ecohealth. Kunz T, Lumsden L. Ecology of cavity and foliage roosting bats. In: Kunz TH, Fenton 414 M, editors. Bat Ecology. Chicago, USA: University of Chicago Press, 2005; 2006. p. 415 24. 416 26. Mühldorfer K, Speck S, Kurth A, Lesnik R, Freuling C, Mü Ller T, et al. Diseases and 417 causes of death in European bats: dynamics in disease susceptibility and infection rates. 418 PLoS One. 2011;6(12):e29773. 419 27. Chan KH, Peiris JSM, Lam SY, Poon LLM, Yuen KY, Seto WH. The effects of tem-420 perature and relative humidity on the viability of the SARS coronavirus. Adv Virol. et al. Group C betacoronavirus in bat guano fertilizer, Thailand. Emerg Infect Dis.', 'latex': None, 'type': 'figure'}, 'FIGREF22': {'text': \"426 2019;13(8):1349-51. 427 30. Leopardi S, Holmes EC, Gastaldelli M, Tassoni L, Priori P, Scaravelli D, et al. Inter-428 play between co-divergence and cross-species transmission in the evolutionary history Ziebuhr J, Baric RS, Baker S, de Groot RJ, Drosten C, Gulyaeva A, et al. ICTV Report 435 2017.013S. 2017. 436 34. Luis AD, Hayman DTS, O'Shea TJ, Cryan PM, Gilbert AT, Pulliam JRC, et al. A com-437 parison of bats and rodents as reservoirs of zoonotic viruses: are bats special? Proc R 438 Soc B Biol Sci. 2013;280(1756):20122753. 439 35. Parrish CR, Holmes EC, Morens DM, Park E-C, Burke DS, Calisher CH, et al. Cross-440 species virus transmission and the emergence of new epidemic diseases. Microbiol Mol 441 Biol Rev. 2008 Sep;72(3):457-70. 442 36. Dijkman R, van der Hoek L. Human coronaviruses 229E and NL63: close yet still so 443 far. J Formos Med Assoc. 2009;108:270-9. 444 37. Pyrc K, Dijkman R, Deng L, Jebbink MF, Ross HA, Berkhout B, et al. Mosaic struc-445 ture of human coronavirus NL63, one thousand years of evolution. J Mol Biol. 2006 446 Dec 15;364(5):964-73. 447 38. Pfefferle S, Oppong S, Drexler JF, Gloza-Rausch F, Ipsen A, Seebens A, et al. Distant 448 relatives of Severe Acute Respiratory Syndrome coronavirus and close relatives of hu-449 man coronavirus 229E in bats, Ghana. Emerg Infect Dis. 2009;15:1377-84. 450 39. Corman VM, Baldwin HJ, Fumie Tateno A, Zerbinati RM, Annan A, Owusu M, et al. 451 Evidence for an ancestral association of human coronavirus 229E with bats. J Virol. 452 2015;89(23):11858-70. 453 40. Joffrin L, Dietrich M, Mavingui P, Lebarbenchon C. Bat pathogens hit the road: But 454 which one? PLoS One. 2018;14(8):e1007134. 455 41. El-Duah P, Meyer B, Sylverken A, Owusu M, Gottula LT, Yeboah R, et al. Develop-456 ment of a whole-virus ELISA for serological evaluation of domestic livestock as possi-457 ble hosts of human coronavirus NL63. Viruses. 2019;11(43):v11010043. 458 42. Huynh J, Li S, Yount B, Smith A, Sturges L, Olsen JC, et al. Evidence supporting a zo-459 onotic origin of human coronavirus strain NL63. J Virol. 2012 Dec 1;86(23):12816-25. 460 43. Cui J, Li F, Shi Z-L. Origin and evolution of pathogenic coronaviruses. Nat Rev Micro-461 biol. 2019;17(March 2019):181-92. 462 44. Geldenhuys M, Mortlock M, Weyer J, Bezuidt O, Seamark ECJ, Kearney T, et al. A 463 metagenomic viral discovery approach identifies potential zoonotic and novel mamma-464 lian viruses in Neoromicia bats within South Africa. PLoS One. 2018;13(3):e0194527. 465 45. Chen L, Liu B, Yang J, Jin Q. DBatVir: the database of bat-associated viruses. Data-466 base (Oxford). 2014;2014:bau021. 467 46. Hobbs JJ. People and caves in Madagascar. Focus Geogr. 2010;46(3):1-7. 468 47. Fernández-Llamazares Á, López-Baucells A, Rocha R, Andriamitandrina SM, Andria-469 tafika ZE, Burgas D, et al. Are sacred caves still safe havens for the endemic bats of 470 Madagascar? Oryx. 2019;25(2):271-5. 471 48. Wilkinson DA, Marshall JC, French NP, Hayman DTS. Habitat fragmentation, biodi-472 versity loss and the risk of novel infectious disease emergence. J R Soc Interface.\", 'latex': None, 'type': 'figure'}, 'FIGREF23': {'text': '. Meyer B, Drosten C, Müller MA. Serological assays for emerging coronaviruses: Chal-475 lenges and pitfalls. Virus Res. 2014 Dec 19;194:175-83.', 'latex': None, 'type': 'figure'}, 'FIGREF25': {'text': 'Geographic distribution of the tested samples. N: number of bats sampled for each Mean CoV prevalence (± 95% confidence interval) in bats in the western Indian 500 Ocean. Pairwise test; ***: p<0.001; NS: p>0.05, not significant.', 'latex': None, 'type': 'figure'}, 'FIGREF26': {'text': 'Mean CoV prevalence (± 95% confidence interval) as function of the type of roost-503 ing habitat. Pairwise test; ***: p<0.001; **: p<0.01; NS: p>0.05, not significant. Maximum Likelihood (ML) consensus tree derived from 202 coronavirus (CoV) 518 RNA-dependent RNA-polymerase partial nucleotide sequences (393 bp). Colored circles at 519 the end of branches indicate bat family origin. Sequences in bold refer to bat CoVs detected in 520 this study. Only bootstrap values >0.7 are indicated on the tree. Scale bar indicates mean 521 number of nucleotide substitutions per site. The tree was generated with the General Time Re-522 versible evolutionary model (GTR+I+Г, I = 0.18, α = 0.64) and 1000 bootstrap replicates.', 'latex': None, 'type': 'figure'}, 'FIGREF28': {'text': 'Detail of the alpha CoV clade. Molossidae CoVs generated in the study are indi-525 cated in bold. This sub-tree is a zoom on Molossidae CoV clade from the tree depicted in Fig-526 ure 4. Only bootstrap values >0.7 are indicated on the tree. Scale bar indicates mean number 527 of nucleotide substitutions per site. Detail of the alpha CoV clade. NL63-like CoVs generated in the study are indicated 531 in bold. This sub-tree is a zoom on NL63 CoV clade from the tree depicted in Figure 4. Only 532 bootstrap values >0.7 are indicated on the tree. Scale bar indicates mean number of nucleotide 533 substitutions per site.', 'latex': None, 'type': 'figure'}, 'FIGREF29': {'text': 'Detail of the alpha CoV clade. 229E-like CoVs generated in the study are indicated 536 in bold. This sub-tree is a zoom on NL63 CoV clade from the tree depicted in Figure 4. Only 537 bootstrap values >0.7 are indicated on the tree. Scale bar indicates mean number of nucleotide 538 substitutions per site.', 'latex': None, 'type': 'figure'}, 'FIGREF30': {'text': 'Detail of the beta-C CoV clade. CoVs generated in the study are indicated in bold.541This sub-tree is a zoom on beta-C CoV clade from the tree depicted inFigure 4. Only boot-542 strap values >0.7 are indicated on the tree. Scale bar indicates mean number of nucleotide 543 substitutions per site.', 'latex': None, 'type': 'figure'}, 'FIGREF31': {'text': 'Detail of the beta-D CoV. CoVs generated in the study are indicated in bold. This 546 sub-tree is a zoom on beta-D CoV clade from the tree depicted in Figure 4. Only bootstrap 547 values >0.7 are indicated on the tree. Scale bar indicates mean number of nucleotide substitu-Tanglegram representing host-virus co-evolution between bats of the western In-552 dian Ocean and their associated CoVs. Phylogeny of bats (left) was constructed with an align-553 ment of 11 Cyt b sequences of 1,030 bp by Neighbor-Joining with 1,000 bootstrap iterations. 554 Pruned phylogeny of SWIO bats CoVs (right) was constructed with an alignment of 27 555 unique sequences of 393 bp from SWIO bats CoVs, by Neighbor-Joining with 1,000 boot-556 strap iterations.', 'latex': None, 'type': 'figure'}, 'TABREF0': {'text': 'RNA was extracted from 140 μL of each sample using the QIAamp Viral RNA mini kit 89 (QIAGEN, Valencia, California, USA), and eluted in 60 μL of Qiagen AVE elution buffer. For bat organs, approximately 1 mm 3 of tissue (either lungs or intestines) was placed in 750 µL of DMEM medium and homogenized in a TissueLyser II (Qiagen, Hilden, Germany) for 2 min at 25 Hz using 3 mm tungsten beads, prior to the RNA extraction. Reverse transcription was performed on 10 μL of RNA using the ProtoScript II Reverse Transcriptase and Random Primer 6 (New England BioLabs, Ipswich, MA, USA) under the following thermal conditions: 70 °C for 95 5 min, 25 °C for 10 min, 42 °C for 50 min, and 65 °C for 20 min (12). cDNAs were tested for 96 the presence of the RNA-dependent RNA-polymerase (RdRp) gene using a multi-probe Real-', 'latex': None, 'type': 'table'}, 'TABREF1': {'text': '). PCRs were performed with the GoTaq G2 Hot Start Green Master Mix (Promega, Madison, WI, USA) in an Applied Biosystems 2720 Thermal Cycler (Thermo Fisher Scientific, Waltham, MA, USA). After electrophoresis in a 1.5% agarose gel stained with 2% GelRed (Biotium, Hayward, CA, USA), the remaining amplicons of the expected size were directly sequenced on both strands by Genoscreen 115 (Lille, France). All generated sequences were deposited in GenBank under the accession num-', 'latex': None, 'type': 'table'}, 'TABREF2': {'text': '(CLC Bio, Aarhus, Denmark). A phylogenetic tree was obtained by maximum likelihood using MEGA Software v10.0.4 (18), with 1,000 bootstrap iterations, and the best evolutionary model for our dataset as selected by modelgenerator v0.85 (19).', 'latex': None, 'type': 'table'}, 'TABREF3': {'text': 'hunted for commercial or personal consumption (46,47). Further, certain Malagasy ethnic groups have cultural rituals associated with caves, including those with bat roosts (47). Indirect', 'latex': None, 'type': 'table'}, 'TABREF4': {'text': \"ration Régionale, Prefecture de La Réunion', and The Field Museum of Natural History, Chicago, through the Ralph and Marian Falk Medical Research Trust. CL is supported by a 'Chaire mixte : Université de La Réunion -INSERM'. The funding agencies were not involved in the study design, implementation or publishing of this study, and the research presented herein represents the opinions of the authors but not necessarily the opinions of the funding agencies.\", 'latex': None, 'type': 'table'}, 'TABREF5': {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the. https://doi.org/10.1101/742866 doi: bioRxiv preprint 19 50. Wang L-F, Anderson DE. Viruses in bats and potential spillover to animals and humans. Curr Opin Virol. 2019;34:79-89. Data Accessibility: -DNA sequences: Genbank accessions MN183146 to MN183273 Author Contributions: CL and LJ conceived and designed the study. BR, CL, DAW, EL, LJ, PT, SMG and YG collected biological material on Madagascar. SMG, RS and YG collected biological material on Mauritius. BR, EL and GLM, collected biological material on Mayotte. SMG, GLM, ADS and MCS, collected biological material in Mozambique. BR, DAW, EL and GLM collected biological material on Reunion Island. EL, GLM, SJ and YG collected biological material in the Seychelles. LJ performed the molecular analyses. CL and DAW analyzed the data. CL and LJ wrote the paper. SMG, ESG, MP, PM and SJ contributed to the project management in Malagasy, Mauritian, Seychelles, Mozambican, and French institutions. All authors edited, read, and approved the final manuscript.author/funder. All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/742866 doi: bioRxiv preprint', 'latex': None, 'type': 'table'}, 'TABREF6': {'text': 'author/funder. All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/742866 doi: bioRxiv preprint', 'latex': None, 'type': 'table'}}, 'back_matter': []}\n",
      "{'paper_id': 'b07bb8f6466cb60a903182960e1e57f5cdbc7780', 'metadata': {'title': 'The Population Attributable Fraction (PAF) of cases due to gatherings and groups with relevance to COVID-19 mitigation strategies', 'authors': [{'first': 'Ellen', 'middle': [], 'last': 'Brooks-Pollock', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Jonathan', 'middle': ['M'], 'last': 'Read', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Thomas', 'middle': [], 'last': 'House', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Graham', 'middle': ['F'], 'last': 'Medley', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Matt', 'middle': ['J'], 'last': 'Keeling', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Leon', 'middle': [], 'last': 'Danon', 'suffix': '', 'affiliation': {}, 'email': ''}]}, 'abstract': [{'text': 'Many countries have banned groups and gatherings as part of their response to the pandemic caused by the coronavirus, SARS-CoV-2. Although there are outbreak reports involving mass gatherings, the contribution to overall transmission is unknown.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'We used data from a survey of social contact behaviour that specifically asked about contact with groups to estimate the Population Attributable Fraction (PAF) due to groups as the relative change in the Basic Reproduction Number when groups are prevented.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'We estimate that PAF due to groups of 50+ people is 2.2% (95%CI 1.1%, 3.6%); the PAF due to groups of 20+ people is 6.4% (5.0%, 8.0%); the PAF due to groups of 10+ is 11.3% (9.9%, 13.0%) Interpretation Large groups of individuals have a small epidemiological impact; small and medium sized groups between 10 and 50 people have a larger impact on an epidemic.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}], 'body_text': [{'text': 'Preventing social contacts and mass gatherings has been used worldwide in the response to reduce transmission communicable diseases, including to reduce transmission of the coronavirus, SARS-CoV-2. As of March 2020, As of March 2020, multiple countries have banned all gatherings of 1,000 people or more; with some countries such as the Czech Republic and the USA banning much smaller groups.', 'cite_spans': [], 'ref_spans': [], 'section': 'INTRODUCTION'}, {'text': 'Given knowledge of transmission mechanisms, bringing together large numbers of people into the same space should prove conducive for the spread of close-contact infectious diseases. Indeed, mass gatherings have been associated with outbreaks of communicable diseases such as measles [1] , influenza [2] and meningitis [3] . And public health agencies, including the World Health Organization (WHO), have specific guidance for preventing disease outbreaks at mass gatherings [4] . Factors such as age of participant [1] , zoonotic transmission and presence of animals [5] , crowding [6, 7] , lack of sanitation [7] , location and event duration [6] are associated with the reporting of mass gathering-related outbreaks.', 'cite_spans': [{'start': 283, 'end': 286, 'text': '[1]', 'ref_id': 'BIBREF0'}, {'start': 299, 'end': 302, 'text': '[2]', 'ref_id': 'BIBREF1'}, {'start': 318, 'end': 321, 'text': '[3]', 'ref_id': 'BIBREF2'}, {'start': 474, 'end': 477, 'text': '[4]', 'ref_id': 'BIBREF3'}, {'start': 515, 'end': 518, 'text': '[1]', 'ref_id': 'BIBREF0'}, {'start': 567, 'end': 570, 'text': '[5]', 'ref_id': 'BIBREF4'}, {'start': 582, 'end': 585, 'text': '[6,', 'ref_id': 'BIBREF5'}, {'start': 586, 'end': 588, 'text': '7]', 'ref_id': 'BIBREF6'}, {'start': 610, 'end': 613, 'text': '[7]', 'ref_id': 'BIBREF6'}, {'start': 644, 'end': 647, 'text': '[6]', 'ref_id': 'BIBREF5'}], 'ref_spans': [], 'section': 'INTRODUCTION'}, {'text': 'Despite the evidence of the importance of mass gatherings for disease transmission from intuition and individual outbreaks, the population-level impact of different mass gathering policies has not been established. While systematic reviews have identified outbreak reports involving mass gatherings [5, 6] , the overall impact of mass gatherings could not be quantitatively assessed. A detailed modelling study of disease transmission in the state of Georgia, USA, found that in extreme scenarios when 25% of the population participated in a 2day long gathering shortly before the epidemic peak, peak prevalence could increase by up to 10% [8] . More realistic scenarios resulted in minimal population-level changes [8] .', 'cite_spans': [{'start': 299, 'end': 302, 'text': '[5,', 'ref_id': 'BIBREF4'}, {'start': 303, 'end': 305, 'text': '6]', 'ref_id': 'BIBREF5'}, {'start': 640, 'end': 643, 'text': '[8]', 'ref_id': 'BIBREF7'}, {'start': 716, 'end': 719, 'text': '[8]', 'ref_id': 'BIBREF7'}], 'ref_spans': [], 'section': 'INTRODUCTION'}, {'text': \"Here, we use representative data on individuals' daily social contacts, including group contacts, to estimate the Population Attributable Fraction (PAF) due to mass gatherings and large numbers of contacts.\", 'cite_spans': [], 'ref_spans': [], 'section': 'INTRODUCTION'}, {'text': \"The Social Contact Survey (SCS) [9, 10] collected data on social contacts from 5,388 participants between 2009 and 2010 in the UK. Participants were asked to enumerate other people with whom they had had contact over the course of a single day. Contacts were defined as those with whom participants had a face-to-face conversation within 3 metres and/or physically touched skin-on-skin. Participants were able report individual contacts and up to five groups of contacts, for instance church groups, weddings, large work functions or multiple contacts at work. The 'groups' question was designed to aid participants in reporting multiple similar contacts. Group contacts were defined in the same way as individual contacts, i.e. if a person attended a concert with 1,000 people, but only spoke to 5 people, the number of recorded group contacts would be 5. Participants were asked whether members of the group knew each other.\", 'cite_spans': [{'start': 32, 'end': 35, 'text': '[9,', 'ref_id': 'BIBREF8'}, {'start': 36, 'end': 39, 'text': '10]', 'ref_id': 'BIBREF9'}], 'ref_spans': [], 'section': 'Social contact data'}, {'text': 'As well as the number of contacts, participants were asked to estimate the length of time spent with each contact or group of contacts as either: less than 10 minutes, 11-30 minutes, 31-60 minutes or over 60 minutes.', 'cite_spans': [], 'ref_spans': [], 'section': 'Social contact data'}, {'text': 'The SCS data are available to download at http://wrap.warwick.ac.uk/54273/.', 'cite_spans': [], 'ref_spans': [], 'section': 'Social contact data'}, {'text': 'The Population Attributable Fraction (PAF) is a quantity borrowed from non-communicable disease epidemiology. The PAF due with a risk factor is the percentage of disease burden or mortality that can be attributed to the presence of that increased risk. In previous work, we demonstrated that for infectious diseases, the PAF can be estimated as the percentage change in the Basic Reproduction Number (average number of secondary cases per infectious case in an otherwise susceptible population [11] ) in the absence of the risk factor [12] .', 'cite_spans': [{'start': 494, 'end': 498, 'text': '[11]', 'ref_id': 'BIBREF10'}, {'start': 535, 'end': 539, 'text': '[12]', 'ref_id': 'BIBREF11'}], 'ref_spans': [], 'section': 'Reproduction Numbers and Population Attributable Fraction'}, {'text': 'Here, we treat participation in a group as the risk factor and calculate the Basic Reproduction Number with and without groups of various sizes. For each participant, we use their contact reports to calculate their individual Basic Reproduction Number, ! . We assumed that ! is proportional to the number of individuals reported in the contact, \" , ( \" = 1 for single contacts, \" > 1 for groups) multiplied by the duration of each contact, \" :', 'cite_spans': [], 'ref_spans': [], 'section': 'Reproduction Numbers and Population Attributable Fraction'}, {'text': 'CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Reproduction Numbers and Population Attributable Fraction'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.20.20039537 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'Reproduction Numbers and Population Attributable Fraction'}, {'text': 'The duration of each contact is taken as the mid-point of each time interval, i.e. 5 minutes, 20 minutes, 45 minutes and 6 hours, as recorded by the participant. For group contacts with more than 20 individuals, we observe a saturation of contact duration, therefore we divide the contact duration by the number of individuals in the group. 95% Confidence Intervals in R0 were calculated using bootstrapping.', 'cite_spans': [], 'ref_spans': [], 'section': 'Reproduction Numbers and Population Attributable Fraction'}, {'text': 'We calculated the PAF for groups of size or greater is calculated as the percentage change in the Basic Reproduction Number:', 'cite_spans': [], 'ref_spans': [], 'section': 'Reproduction Numbers and Population Attributable Fraction'}, {'text': 'We investigate the PAF for groups of greater than 10, and up to groups greater than 100, in increments of 10. We investigated differences between groups that knew each other and groups that did not know each other.', 'cite_spans': [], 'ref_spans': [], 'section': 'Reproduction Numbers and Population Attributable Fraction'}, {'text': '48,001 unique contacts were reported by 5,388 participants. Of those, 42,945 (89%) were individual contacts and 5,056 groups were reported (accounting for 11% of reported contacts). The median and mean number of contacts per person was 11.5 and 27.0, range 1 to 3,011.', 'cite_spans': [], 'ref_spans': [], 'section': 'Impact of groups on numbers of contacts per person'}, {'text': '2,427 (45%) of participants reported group contacts. The majority of groups reported (3,860; 76%) were groups of people who knew each other. 2,979 (59%) groups had 10 or fewer members; the median and mean reported group size was 9 and 20.3 individuals respectively.', 'cite_spans': [], 'ref_spans': [], 'section': 'Impact of groups on numbers of contacts per person'}, {'text': 'Restricting contacts to groups of size 50 or less, reduces the median and mean number of individual contacts per person to 11.0 and 18.8; restricting contact to groups of size 20 or less, reduces the median and mean number of contacts per person to 10.0 and 14.1; restricting contacts to groups of size 10 or less, reduces the median and mean number of contacts per person to 9 and 11.0. Figure 1 shows the degree distribution (number of contacts) per person with and without contacts associated with groups of size greater than 10.', 'cite_spans': [], 'ref_spans': [{'start': 388, 'end': 396, 'text': 'Figure 1', 'ref_id': None}], 'section': 'Impact of groups on numbers of contacts per person'}, {'text': 'The PAF due to groups decreased with increasing group size. For the largest groups with more than 100 individuals the PAF100 is estimated at 0.8% (0.3%, 1.7%). The PAF50 is estimated at 2.2% (95% Confidence Interval of the mean: 1.1%, 3.6%); the PAF20 is 6.4% (5.0%, 8.0%); the PAF10 is 11.4% (9.9%, 13.0%) (figure 2).', 'cite_spans': [], 'ref_spans': [], 'section': 'Population Attributable Fraction (PAF)'}, {'text': 'The pattern of decreasing PAF with increasing group size is seen for both groups of individuals who are known to each other and groups of individuals who are unknown to each other. The PAF due to groups of 10+ known to each other is estimated at 8.4% (7.4%, 9.4%) and due to groups of 50+ known to each other is estimated at 0.8% (0.5%, 1.3%). The remaining contribution to R0 is due to contact with individuals.', 'cite_spans': [], 'ref_spans': [], 'section': 'Population Attributable Fraction (PAF)'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Population Attributable Fraction (PAF)'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.20.20039537 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'Population Attributable Fraction (PAF)'}, {'text': 'The low estimated impact of large groups on ! is due to the relative frequency with which they are reported in the Social Contact Survey. These results highlight the relative importance of medium-size groups of between 10 and 20 individuals.', 'cite_spans': [], 'ref_spans': [], 'section': 'Population Attributable Fraction (PAF)'}, {'text': 'In this paper, we analysed social contact data in the context of infectious disease transmission and gatherings. Our findings suggest that large groups of individuals have a relatively small impact on an epidemic. This is due to the relative rarity of large-scale gatherings and the sublinear scaling between number of contacts and infectivity.', 'cite_spans': [], 'ref_spans': [], 'section': 'DISCUSSION'}, {'text': 'The Social Contact Survey (SCS) is one of a number of social contact surveys that have been conducted to quantify the impact of social mixing on disease transmission [13] [14] [15] [16] [17] . The SCS specifically asked about groups of similar contacts. These groups are not necessarily public or mass gatherings, and represented groups that both knew each other and those that did not. The SCS asked about contacts on a single day, therefore did not capture multi-day events; simulation studies have shown that prolonged mass gatherings were necessary to alter the course of an epidemic [8] . Our analysis was based on social contact data collected between 2009 and 2010; contact patterns may have altered in the past decade. We also did not account for individuals changing their behaviour if group activities were cancelled.', 'cite_spans': [{'start': 166, 'end': 170, 'text': '[13]', 'ref_id': 'BIBREF12'}, {'start': 171, 'end': 175, 'text': '[14]', 'ref_id': 'BIBREF13'}, {'start': 176, 'end': 180, 'text': '[15]', 'ref_id': 'BIBREF14'}, {'start': 181, 'end': 185, 'text': '[16]', 'ref_id': 'BIBREF15'}, {'start': 186, 'end': 190, 'text': '[17]', 'ref_id': None}, {'start': 588, 'end': 591, 'text': '[8]', 'ref_id': 'BIBREF7'}], 'ref_spans': [], 'section': 'DISCUSSION'}, {'text': 'In the context of COVID-19 mitigation, this analysis considered one aspect of gatherings: the impact on an epidemic. However, there may be other valid reasons for preventing mass events, such as policing and managing resources. Our analysis implicitly assumes that infection is already present in the population as is the case for COVID-19; for other diseases, mass gatherings can be associated with increased global travel which can bring new strains into an area or result in out-of-season outbreaks, which were also not captured here.', 'cite_spans': [], 'ref_spans': [], 'section': 'DISCUSSION'}, {'text': 'Our findings illustrate the difficult choices that are necessary to limit COVID-19 spread. Meetings of large groups of more than 100 individuals are relatively infrequent, and their prohibition may have a limited impact on the epidemic. More epidemiologically relevant are groups of 10 to 20 people, as they occur more frequently and could potentially have a larger impact on transmission; they may also involve inter-generational family groups. This analysis was designed to aid policy-making and should be considered against alternative control strategies so that the most effective measures can be implemented in the long term.', 'cite_spans': [], 'ref_spans': [], 'section': 'DISCUSSION'}, {'text': 'Funding source EBP was partly supported by the National Institute for Health Research Health Protection Research Unit (NIHR HPRU) in Evaluation of Interventions. The views expressed are those of the author(s) and not necessarily those of the NHS, the NIHR or the Department of Health. The NIHR had no role in writing the manuscript or the decision to publish.', 'cite_spans': [], 'ref_spans': [], 'section': 'Declaration of Interests None'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Declaration of Interests None'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint Contacts with all groups . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Declaration of Interests None'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.20.20039537 doi: medRxiv preprint All groups Groups that know each other . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Declaration of Interests None'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.20.20039537 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'Declaration of Interests None'}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'Measles outbreaks at mass gathering mostly occur at youth events', 'authors': [{'first': 'V', 'middle': ['T'], 'last': 'Hoang', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Gautret', 'suffix': ''}], 'year': 2020, 'venue': 'Lancet Infect Dis', 'volume': '20', 'issn': '19', 'pages': '30685--30686', 'other_ids': {'DOI': ['10.1016/S1473-3099']}}, 'BIBREF1': {'ref_id': 'b1', 'title': 'Influenza outbreaks during world youth day 2008 mass gathering', 'authors': [{'first': 'C', 'middle': ['C'], 'last': 'Blyth', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Foo', 'suffix': ''}, {'first': 'S', 'middle': ['J'], 'last': 'Van Hal', 'suffix': ''}, {'first': 'A', 'middle': ['C'], 'last': 'Hurt', 'suffix': ''}, {'first': 'I', 'middle': ['G'], 'last': 'Barr', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Mcphie', 'suffix': ''}], 'year': 2010, 'venue': 'Emerg Infect Dis', 'volume': '16', 'issn': '', 'pages': '809--824', 'other_ids': {'DOI': ['10.3201/eid1605.091136']}}, 'BIBREF2': {'ref_id': 'b2', 'title': 'Prevention of meningococcal disease at mass gatherings: Lessons from the Hajj and Umrah', 'authors': [{'first': 'S', 'middle': [], 'last': 'Yezli', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Gautret', 'suffix': ''}, {'first': 'A', 'middle': ['M'], 'last': 'Assiri', 'suffix': ''}, {'first': 'B', 'middle': ['D'], 'last': 'Gessner', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Alotaibi', 'suffix': ''}], 'year': 2018, 'venue': 'Vaccine', 'volume': '36', 'issn': '', 'pages': '4603--4612', 'other_ids': {'DOI': ['10.1016/j.vaccine.2018.06.030']}}, 'BIBREF3': {'ref_id': 'b3', 'title': 'Global mass gatherings : implications and opportunities for global health security Report by the Secretariat', 'authors': [], 'year': 2011, 'venue': '', 'volume': '86', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF4': {'ref_id': 'b4', 'title': 'Mass gatherings and respiratory disease outbreaks in the united states-should we be worried? results from a systematic literature review and analysis of the national outbreak reporting system', 'authors': [{'first': 'J', 'middle': ['J'], 'last': 'Rainey', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Phelps', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Shi', 'suffix': ''}], 'year': 2016, 'venue': 'PLoS One', 'volume': '11', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1371/journal.pone.0160378']}}, 'BIBREF5': {'ref_id': 'b5', 'title': 'Could influenza transmission be reduced by restricting mass gatherings? Towards an evidence-based policy framework', 'authors': [{'first': 'D', 'middle': ['A'], 'last': 'Ishola', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Phin', 'suffix': ''}], 'year': 2011, 'venue': 'J Epidemiol Glob Health', 'volume': '1', 'issn': '', 'pages': '33--60', 'other_ids': {'DOI': ['10.1016/j.jegh.2011.06.004']}}, 'BIBREF6': {'ref_id': 'b6', 'title': 'Global perspectives for prevention of infectious diseases associated with mass gatherings', 'authors': [{'first': 'I', 'middle': [], 'last': 'Abubakar', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Gautret', 'suffix': ''}, {'first': 'G', 'middle': ['W'], 'last': 'Brunette', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Blumberg', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Johnson', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Poumerol', 'suffix': ''}], 'year': 2012, 'venue': 'Lancet Infect Dis', 'volume': '12', 'issn': '11', 'pages': '70246--70254', 'other_ids': {'DOI': ['10.1016/S1473-3099']}}, 'BIBREF7': {'ref_id': 'b7', 'title': 'The impact of mass gatherings and holiday traveling on the course of an influenza pandemic: A computational model', 'authors': [{'first': 'P', 'middle': [], 'last': 'Shi', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Keskinocak', 'suffix': ''}, {'first': 'J', 'middle': ['L'], 'last': 'Swann', 'suffix': ''}, {'first': 'B', 'middle': ['Y'], 'last': 'Lee', 'suffix': ''}], 'year': 2010, 'venue': 'BMC Public Health', 'volume': '10', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1186/1471-2458-10-778']}}, 'BIBREF8': {'ref_id': 'b8', 'title': 'Social encounter networks: collective properties and disease transmission', 'authors': [{'first': 'L', 'middle': [], 'last': 'Danon', 'suffix': ''}, {'first': 'T', 'middle': ['A'], 'last': 'House', 'suffix': ''}, {'first': 'J', 'middle': ['M'], 'last': 'Read', 'suffix': ''}, {'first': 'M', 'middle': ['J'], 'last': 'Keeling', 'suffix': ''}], 'year': 2012, 'venue': 'J R Soc Interface', 'volume': '9', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF9': {'ref_id': 'b9', 'title': 'Social encounter networks: Characterizing great Britain', 'authors': [{'first': 'L', 'middle': [], 'last': 'Danon', 'suffix': ''}, {'first': 'J', 'middle': ['M'], 'last': 'Read', 'suffix': ''}, {'first': 'T', 'middle': ['A'], 'last': 'House', 'suffix': ''}, {'first': 'M', 'middle': ['C'], 'last': 'Vernon', 'suffix': ''}, {'first': 'M', 'middle': ['J'], 'last': 'Keeling', 'suffix': ''}], 'year': 2013, 'venue': 'Proc R Soc B Biol Sci', 'volume': '280', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1098/rspb.2013.1037']}}, 'BIBREF10': {'ref_id': 'b10', 'title': 'Infectious diseases of humans: dynamics and control', 'authors': [{'first': 'R', 'middle': [], 'last': 'Anderson', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'May', 'suffix': ''}], 'year': 1992, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF11': {'ref_id': 'b11', 'title': 'Defining the population attributable fraction for infectious diseases', 'authors': [{'first': 'E', 'middle': [], 'last': 'Brooks-Pollock', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Danon', 'suffix': ''}], 'year': 2017, 'venue': 'Int J Epidemiol', 'volume': '46', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1093/ije/dyx055']}}, 'BIBREF12': {'ref_id': 'b12', 'title': 'Social contacts and mixing patterns relevant to the spread of infectious diseases', 'authors': [{'first': 'J', 'middle': [], 'last': 'Mossong', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Hens', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Jit', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Beutels', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Auranen', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Mikolajczyk', 'suffix': ''}], 'year': 2008, 'venue': 'PLoS Med', 'volume': '5', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1371/journal.pmed.0050074']}}, 'BIBREF13': {'ref_id': 'b13', 'title': 'Mining social mixing patterns for infectious disease models based on a two-day population survey in Belgium', 'authors': [{'first': 'N', 'middle': [], 'last': 'Hens', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Goeyvaerts', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Aerts', 'suffix': ''}, {'first': 'Z', 'middle': [], 'last': 'Shkedy', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Van Damme', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Beutels', 'suffix': ''}], 'year': 2009, 'venue': 'BMC Infect Dis', 'volume': '9', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF14': {'ref_id': 'b14', 'title': 'Measured dynamic social contact patterns explain the spread of H1N1v influenza', 'authors': [{'first': 'Ktd', 'middle': [], 'last': 'Eames', 'suffix': ''}, {'first': 'N', 'middle': ['L'], 'last': 'Tilston', 'suffix': ''}, {'first': 'E', 'middle': [], 'last': 'Brooks-Pollock', 'suffix': ''}, {'first': 'W', 'middle': ['J'], 'last': 'Edmunds', 'suffix': ''}], 'year': 2012, 'venue': 'PLoS Comput Biol', 'volume': '8', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1371/journal.pcbi.1002425']}}, 'BIBREF15': {'ref_id': 'b15', 'title': 'Social mixing patterns in rural and urban areas of Southern China', 'authors': [{'first': 'J', 'middle': ['M'], 'last': 'Read', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Lessler', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Riley', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'L', 'middle': ['J'], 'last': 'Tan', 'suffix': ''}, {'first': 'K', 'middle': ['O'], 'last': 'Kwok', 'suffix': ''}], 'year': 2014, 'venue': 'Proc R Soc B Biol Sci', 'volume': '281', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1098/rspb.2014.0268']}}}, 'ref_entries': {'FIGREF0': {'text': 'The Population Attributable Fraction (PAF) against group size. The filled black circles are all groups, empty blue circles are groups of people who are known to each other.', 'latex': None, 'type': 'figure'}, 'TABREF0': {'text': '. https://doi.org/10.1101/2020.03.20.20039537 doi: medRxiv preprint [17] Kwok KO, Cowling B, Wei V, Riley S, Read JM. Temporal variation of human encounters and the number of locations in which they occur: A longitudinal study of Hong Kong residents. J R Soc Interface 2018;15. doi:10.1098/rsif.2017.0838.Figure 1: The number of social contacts per person with and without groups of 10+.Contacts removing groups of 10+', 'latex': None, 'type': 'table'}}, 'back_matter': []}\n",
      "{'paper_id': 'b0899a264af548a89d649154aec569889717b295', 'metadata': {'title': 'Stochastic challenges to interrupting helminth transmission', 'authors': [{'first': 'Robert', 'middle': ['J'], 'last': 'Hardwick', 'suffix': '', 'affiliation': {'laboratory': 'London Centre for Neglected Tropical Disease Research (LCNTDR)', 'institution': 'Imperial College London', 'location': {'postCode': 'WC2 1PG', 'settlement': 'London', 'country': 'UK'}}, 'email': ''}, {'first': 'Marleen', 'middle': [], 'last': 'Werkman', 'suffix': '', 'affiliation': {'laboratory': 'London Centre for Neglected Tropical Disease Research (LCNTDR)', 'institution': 'Imperial College London', 'location': {'postCode': 'WC2 1PG', 'settlement': 'London', 'country': 'UK'}}, 'email': ''}, {'first': 'James', 'middle': ['E'], 'last': 'Truscott', 'suffix': '', 'affiliation': {'laboratory': 'London Centre for Neglected Tropical Disease Research (LCNTDR)', 'institution': 'Imperial College London', 'location': {'postCode': 'WC2 1PG', 'settlement': 'London', 'country': 'UK'}}, 'email': ''}, {'first': 'Roy', 'middle': ['M'], 'last': 'Anderson', 'suffix': '', 'affiliation': {'laboratory': 'London Centre for Neglected Tropical Disease Research (LCNTDR)', 'institution': 'Imperial College London', 'location': {'postCode': 'WC2 1PG', 'settlement': 'London', 'country': 'UK'}}, 'email': ''}]}, 'abstract': [{'text': \"Predicting the effect of different programmes designed to control both the morbidity induced by helminth infections and parasite transmission is greatly facilitated by the use of mathematical models of transmission and control impact. In such models, it is essential to account for as many sources of uncertainty-natural, or otherwise -to ensure robustness in prediction and to accurately depict variation around an expected outcome. In this paper, we investigate how well the standard deterministic models match the predictions made using individual-based stochastic simulations. We also explore how well concepts which derive from deterministic models, such as 'breakpoints' in transmission, apply in the stochastic world. Employing an individual based stochastic model framework we also investigate how transmission and control are affected by the migration of infected people into a defined community. To give our study focus we consider the control of soil-transmitted helminths (STH) by mass drug administration (MDA), though our methodology is readily applicable to the other helminth species such as the schistosome parasites and the filarial worms. We show it is possible to define a 'stochastic breakpoint' where much noise surrounds the expected deterministic breakpoint. We also discuss the concept of the 'interruption of transmission' independent of the 'breakpoint' concept where analyses of model behaviour illustrate the current limitations of deterministic models to account for the 'fade-out' or transmission extinction behaviour in simulations. The analyses based on migration confirm a relationship between the infected human migration rate per unit of time and the death rate of infective stages that are released into the free-living environment (eggs or larvae depending on the STH species) that create the reservoir of infection which in turn determines the likelihood that control activities aim at chemotherapeutic treatment of the human host will eliminate transmission. The development of a new stochastic simulation code for STH in the form of a publicly-available open-source python package which includes features to incorporate many population stratifications, different control interventions including mass drug administration (with defined frequency, coverage levels and compliance patterns) and inter-village human migration is also described. : medRxiv preprint\", 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'Following past publications [12, 23] , the STH transmission model we introduce here is a stochastic individual-based analogue to the deterministic original given in Refs. [26, 11, 7] . : medRxiv preprint ∂ ∂Z G jn [Z(t), t] = λ jn Λ j (t) [Z(t) − 1] G jn [Z(t), t] , whose solution is given by Eq. (A.4) when one replaces Z(t ) − 1 with the inverted form of the equation above.', 'cite_spans': [{'start': 28, 'end': 32, 'text': '[12,', 'ref_id': 'BIBREF10'}, {'start': 33, 'end': 36, 'text': '23]', 'ref_id': 'BIBREF21'}, {'start': 171, 'end': 175, 'text': '[26,', 'ref_id': 'BIBREF24'}, {'start': 176, 'end': 179, 'text': '11,', 'ref_id': 'BIBREF9'}, {'start': 180, 'end': 182, 'text': '7]', 'ref_id': 'BIBREF5'}], 'ref_spans': [], 'section': 'Abstract'}], 'body_text': [{'text': 'Helminthiases are a class of the neglected tropical diseases (NTDs) that affect many hundreds of millions of humans and animals worldwide. One group of worms, the soiltransmitted helminths (STH) are especially prevalent are transmitted through the ingestion of eggs (for Ascaris lumbricoides or Trichuris trichuria) or larvae (in the case of the hookworms: Necator americanus and Ancylostoma duodenale). They are estimated to affect 1.45 billion people at present [1, 2] .', 'cite_spans': [{'start': 464, 'end': 467, 'text': '[1,', 'ref_id': 'BIBREF0'}, {'start': 468, 'end': 470, 'text': '2]', 'ref_id': None}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'In recent years, mathematical models of both infectious disease transmission [3, 4, 5] and intervention impact have been widely used in infectious disease epidemiological studies and public health policy formulation [6, 7, 8, 9, 10] . Their use in the study of the NTDs is more recent, where the variety of approaches implemented have included both deterministic [11, 7] and individual-based stochastic simulation models [12, 13, 14, 15] . Much progress in model formulation, * r.hardwick@imperial.ac.uk parameter estimation and application has been made over the last 20 years. Model-based analyses are increasingly serving an important role in policy formulation and the evaluation of different control policies [16, 17, 18] as is well illustrated by the activities of the Bill and Melinda Gates Foundation funded NTD Modelling consortium [19] . Furthermore, following the London Declaration in 2010, which stimulated the expansion of large mass drug administration (MDA) programmes under the direction of World Health Organization (WHO) guidelines on treatment strategies [17, 20, 21] , mathematical models have played an increasing role in determining how best to design and evaluate MDA programmes [22, 23, 24] .', 'cite_spans': [{'start': 77, 'end': 80, 'text': '[3,', 'ref_id': 'BIBREF1'}, {'start': 81, 'end': 83, 'text': '4,', 'ref_id': 'BIBREF2'}, {'start': 84, 'end': 86, 'text': '5]', 'ref_id': 'BIBREF3'}, {'start': 216, 'end': 219, 'text': '[6,', 'ref_id': 'BIBREF4'}, {'start': 220, 'end': 222, 'text': '7,', 'ref_id': 'BIBREF5'}, {'start': 223, 'end': 225, 'text': '8,', 'ref_id': 'BIBREF6'}, {'start': 226, 'end': 228, 'text': '9,', 'ref_id': 'BIBREF7'}, {'start': 229, 'end': 232, 'text': '10]', 'ref_id': 'BIBREF8'}, {'start': 363, 'end': 367, 'text': '[11,', 'ref_id': 'BIBREF9'}, {'start': 368, 'end': 370, 'text': '7]', 'ref_id': 'BIBREF5'}, {'start': 421, 'end': 425, 'text': '[12,', 'ref_id': 'BIBREF10'}, {'start': 426, 'end': 429, 'text': '13,', 'ref_id': 'BIBREF11'}, {'start': 430, 'end': 433, 'text': '14,', 'ref_id': 'BIBREF12'}, {'start': 434, 'end': 437, 'text': '15]', 'ref_id': 'BIBREF13'}, {'start': 714, 'end': 718, 'text': '[16,', 'ref_id': 'BIBREF14'}, {'start': 719, 'end': 722, 'text': '17,', 'ref_id': 'BIBREF15'}, {'start': 723, 'end': 726, 'text': '18]', 'ref_id': None}, {'start': 841, 'end': 845, 'text': '[19]', 'ref_id': 'BIBREF17'}, {'start': 1075, 'end': 1079, 'text': '[17,', 'ref_id': 'BIBREF15'}, {'start': 1080, 'end': 1083, 'text': '20,', 'ref_id': 'BIBREF18'}, {'start': 1084, 'end': 1087, 'text': '21]', 'ref_id': 'BIBREF19'}, {'start': 1203, 'end': 1207, 'text': '[22,', 'ref_id': 'BIBREF20'}, {'start': 1208, 'end': 1211, 'text': '23,', 'ref_id': 'BIBREF21'}, {'start': 1212, 'end': 1215, 'text': '24]', 'ref_id': 'BIBREF22'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': \"Two key concepts emerge from analyses of deterministic models. The first is the existence of a 'breakpoint' in transmission created by the dioecious nature of the worms (a male and female must be present in the same host to ensure the production of fertile eggs and pass to the external environment and form the pool of infectious material) which creates an unstable equilibrium separating the stable endemic infection equilibrium Basic reproductive number Mean worm burden S t a b l e e n d e m i c e q .\", 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': \"Uns tabl e 'bre akp oint ' eq. from the other stable equilibrium of parasite extinction. Independent of this breakpoint, a further epidemiological situation arises where only one stable equilibrium exists; namely, parasite and transmission extinction where the rate of infection is always too low to sustain the parasites in the human host. In this case transmission is interrupted, since an adult female worm in the human host on average, produces too few offspring to ensure one of her offspring matures in the human host to perpetuate the lifecycle. For many viral and bacterial infections (the microparasites), this is the situation where the basic reproduction number, R 0 < 1, where R 0 describes the generation of secondary cases. For the macroparasitic helminths, the concept of R 0 needs modification since it describes the average number of female offspring, produced by female worms, that survive to reproductive maturity by infecting a new human host and maturing within it [7] . The concept of R 0 is further modified by two density dependent processes: acting on fecundity as a population regulatory factor and acting via the dioecious nature of the worm. Because of the action of these two processes, the crtical R 0 value to sustain transmission is greater than unity in value. These two concepts are illustrated diagrammatically in Fig. 1 . Note, in particular, how the darker red shaded region -which demarcates the R 0 < 1 region -is replaced with a lighter shaded red 'breakpoint' region once these density dependent processes have been introduced.\", 'cite_spans': [{'start': 986, 'end': 989, 'text': '[7]', 'ref_id': 'BIBREF5'}], 'ref_spans': [{'start': 1349, 'end': 1355, 'text': 'Fig. 1', 'ref_id': 'FIGREF0'}], 'section': 'Introduction'}, {'text': 'The importance of these notions when we move from the deterministic world to a stochastic one is linked to explaining what the interruption of transmission really appears as in a noisy world full of variation and chance events. Even when the deterministic breakpoint is crossed, when its value is low due to high degrees of parasite aggregation in the human host (see Ref. [7] ), stochastic noise may induce fluctuations where bounce back or extinction occur [24, 25] . Similarly, even when transmission is interrupted when the average female worm produces too few offspring to ensure the continuation of the life cycle, stochastic noise may not always result in parasite extinction in a defined location. In this paper, we shall analyze these situations with the use of a full individual-based stochastic model and certain approximations to the fully-simulated outcomes. In particular what is novel in our approach, is both looking at noise around the breakpoint and the transmission interruption state, and assessing how the migration of infected humans influences the likelihood of achieving effective control or even transmission interruption by MDA.', 'cite_spans': [{'start': 373, 'end': 376, 'text': '[7]', 'ref_id': 'BIBREF5'}, {'start': 459, 'end': 463, 'text': '[24,', 'ref_id': 'BIBREF22'}, {'start': 464, 'end': 467, 'text': '25]', 'ref_id': 'BIBREF23'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'At present the demand is for increasing complexity in models to describe all known biological and epidemiological complexities including, for example: differing patterns of compliance to treatment; infected human migration patterns in and out of defined control activity regions; and various heterogeneities in human behaviour that influence transmission. In such circumstances, the temptation is to move to ever more complex simulation models, with a concomitant growth in parameters and the associated problems in measurement and estimation. In this paper we adopt a somewhat different approach, seeking to address the following fundamental questions:', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': '• How well can deterministic models match the predictions made using stochastic simulations?', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': \"• Do the concepts such as 'breakpoints' in transmission still apply when stochasticity is introduced?\", 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': '• How are transmission and control affected by infected human migration in a world of stochasticity?', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'In addressing these questions here we seek to ascertain the uncertainties that have the greatest impact on forecasts of NTD control initiatives both to enhance the quality of predictions and focus attention on what needs to be measured to improve accuracy.', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'The results described in this paper are underpinned by various analytical and numerical methods. We focus attention on stochastic models for STH infection and control, with epidemiological parameters set to those of the two hookworm species (Necator americanus and Ancylostoma duodenale). However, the methodologies developed are generalizable to other human helminth infections.', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'In Sec. 2 we introduce the mathematical model and formalism within which the first two main questions posed above are answered. In Sec. 3 we then extend this formalism to include a model for MDA control and infected human migration between reservoirs of infection, which we then use to answer the third question above. Lastly, in Sec. 4 we conclude with a summary of our findings and a discussion of future work.', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'For a recent review of stochastic STH models and their predictions concerning tramsmission dynamics and control impact, see Ref. [23] .', 'cite_spans': [{'start': 129, 'end': 133, 'text': '[23]', 'ref_id': 'BIBREF21'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': \"Consider an ensemble of Poisson walkers, each representing individual worm hosts within a community, and each assigned a worm uptake rate λ jn Λ j (t) (or the 'force of infection') for the nth individual within the j-th age bin, where one initially draws human predispositions to worm uptake from a unit-mean and 1/k j -variance gamma distribution 1\", 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'and one evolves the contribution to the force of infection in the j-th age bin according to the following differential equation', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'where it is important to note that the quantity which accounts for egg input into the reservoir from sexually-reproducing worms', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'depends on the total number of worms w jn (as opposed to the just the number of females) in the n-th individual within the j-th age bin. Throughout we shall assume that the standard exponential relationship for STH which takes into account the density dependent fecundity of worms z ≡ e −γ [7, 11, 26] is set to γ = 0.08, unless otherwise stated. Now let us build a corresponding Poisson process in the opposing direction to model the worm and human deaths, given by rates µ 1 and µ. In such a model, the total mean worm burden m(t) is defined as a weighted sum over the mean worm burdens within each age bin m j (t) (with N a age bins in total) -which itself is the average over the individual worm burdens', 'cite_spans': [{'start': 290, 'end': 293, 'text': '[7,', 'ref_id': 'BIBREF5'}, {'start': 294, 'end': 297, 'text': '11,', 'ref_id': 'BIBREF9'}, {'start': 298, 'end': 301, 'text': '26]', 'ref_id': 'BIBREF24'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'and can hence take different values, depending on the realisation. The stochastic jump equation, which governs its time evolution over a population of N p people, takes the following form', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'where the worm uptake U j (t) and (worm and human) death D j (t) are given by the following Poisson processes summed from n = 1 to N j individuals within the j-th age bin', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'where ∆t i ≡ t i − t i−1 , τ is a pre-specified timescale short enough such that no event is expected to take place.', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'We have developed a new stochastic simulation code for helminth transmission based on the model defined above. This code comes in the form of a publicly-available open-source python class: helmpy, which includes features to incorporate any population stratifications, models of control with MDA, inter-cluster human migration (building from earlier work in Refs. [27, 28] ) and an interactive notebook tutorial for calculating all of the quantities discussed in this paper. 2 This code will also be extended to include the other helminths in the future.', 'cite_spans': [{'start': 363, 'end': 367, 'text': '[27,', 'ref_id': 'BIBREF25'}, {'start': 368, 'end': 371, 'text': '28]', 'ref_id': 'BIBREF26'}], 'ref_spans': [], 'section': 'Our computational implementation in brief'}, {'text': 'The individual-based stochastic model implemented in the helmpy code follows a similar pattern to the Gillespie algorithm [29] , however we also adopt a separate methodology for the mean field expansion (which we shall introduce later) to compare the with the former, which relies on a multidimensional Langevin solver, i.e., numerically evolves many coupled drift-diffusion processes simultaneously -see, e.g., Ref. [30] .', 'cite_spans': [{'start': 122, 'end': 126, 'text': '[29]', 'ref_id': 'BIBREF27'}, {'start': 417, 'end': 421, 'text': '[30]', 'ref_id': 'BIBREF28'}], 'ref_spans': [], 'section': 'Our computational implementation in brief'}, {'text': 'The code is written in a (mostly) vectorised implementation of the python programming language. To provide a benchmark for the performance, we note here that a single individual-based stochastic realisation of the code which implements multiple MDA treatment rounds and migration between 40 clusters of 500 people (each cluster corresponding to a different infectious reservoir) requires ∼ 20 minutes to run for 100 years on a laptop with a 1.7 GHz Intel Core i5 processor and 4GB of memory.', 'cite_spans': [], 'ref_spans': [], 'section': 'Our computational implementation in brief'}, {'text': 'Stochastic individual-based simulations of STH transmission seek to incorporate a range of uncertainties into the populationlevel dynamics of the host-parasite interaction. It is interesting to note that these may typically be categorised into either one, or all, of three important biological/epidemiological components:', 'cite_spans': [], 'ref_spans': [], 'section': 'The approximate worm burden distribution'}, {'text': \"1. Uncertainty in predispositions of people (i.e. the specific chance of a particular group of people having their predispositions to STH infection drawn from a probability distribution), i.e., finite population uncertainty induced by specific samples of realisations. 2. Uncertainty in precisely when new infections, resulting in new worm acquisition, or worm/host deaths, resulting in worm loss, occur, i.e., dynamical uncertainty about precisely when events occur. 3. Uncertainty in the demographic parameters and initial conditions of the given human community (or 'system') under consideration, i.e., either prior or posterior uncertainty on the initial conditions.\", 'cite_spans': [], 'ref_spans': [], 'section': 'The approximate worm burden distribution'}, {'text': \"One may average over the above forms of uncertainty either individually, or in combination. In particular, to obtain the dynamics corresponding to deterministic models of STH transmission, one must average over the first two -which we shall hereafter refer to as an 'ensemble average'. The ensemble average of Eq. (2) may be written as\", 'cite_spans': [], 'ref_spans': [], 'section': 'The approximate worm burden distribution'}, {'text': 'where the result', 'cite_spans': [], 'ref_spans': [], 'section': 'The approximate worm burden distribution'}, {'text': 'has been derived in numerous Refs. [26, 11, 7] . Note that we have also defined the ensemble average over m j (t) as M j (t) ≡ E[m j (t)]. Performing an equivalent ensemble average over N j m j , the mean-field evolution equation for M j (t) is given by', 'cite_spans': [{'start': 35, 'end': 39, 'text': '[26,', 'ref_id': 'BIBREF24'}, {'start': 40, 'end': 43, 'text': '11,', 'ref_id': 'BIBREF9'}, {'start': 44, 'end': 46, 'text': '7]', 'ref_id': 'BIBREF5'}], 'ref_spans': [], 'section': 'The approximate worm burden distribution'}, {'text': \"matching the standard theory of the mean or 'deterministic' disease model [7] . Note that we have also used Eq. (1) to iden-\", 'cite_spans': [{'start': 74, 'end': 77, 'text': '[7]', 'ref_id': 'BIBREF5'}], 'ref_spans': [], 'section': 'The approximate worm burden distribution'}, {'text': 'assuming that λ jn and Λ(t) may be treated as independent random variables. Note that this latter assumption is especially poor when the number of infected individuals (predominately controlled by k and/or treatment) is low since a limited number of individuals asserting their influence on the reservoir will lead to stronger cross-', 'cite_spans': [], 'ref_spans': [], 'section': 'The approximate worm burden distribution'}, {'text': 'In contrast to the simplicity of computing the dynamical equations for M j (t) above, calculating the time evolution of V j (t) ≡ Var[m j (t)] will require knowledge of the overall distribution.', 'cite_spans': [], 'ref_spans': [], 'section': 'The approximate worm burden distribution'}, {'text': \"It is straightforward to derive a master equation which governs the out-of-equilibrium behaviour of the n-th individual's (in the j-th age bin) worm burden distribution as a function of time P(w jn , t). This is d dt P(w jn , t) = − λ jn Λ j (t) + (µ + µ 1 )w jn P(w jn , t)\", 'cite_spans': [], 'ref_spans': [], 'section': 'The approximate worm burden distribution'}, {'text': 'where one recognises the solution as an inhomogeneous Poisson walker w jn (t) ∼ Pois[w jn (t); I jn (t)] with intensity (for the explicit derivation, see Appendix A)', 'cite_spans': [], 'ref_spans': [], 'section': 'The approximate worm burden distribution'}, {'text': '(16) Therefore, for an ensemble of N j independent walkers (hence individuals) the corresponding probability mass function is also that of a Poisson distribution', 'cite_spans': [], 'ref_spans': [], 'section': 'The approximate worm burden distribution'}, {'text': 'where we have also used the definition of the mean worm burden in the j-th age bin given by Eq. (4) . Note that in the case where Λ j (t) is roughly constant for all time, the timescale for the distribution to achieve stationarity is ∆t stat 1/(µ + µ 1 ).', 'cite_spans': [{'start': 96, 'end': 99, 'text': '(4)', 'ref_id': 'BIBREF2'}], 'ref_spans': [], 'section': 'The approximate worm burden distribution'}, {'text': 'Note that by assuming that the reservoir of infection Λ(t) in Eq. (16) is deterministic in time and averaging over λ jn according to Eq. (1), one recovers the well-established [7] negative binomial distribution for worms within hosts as the result of a gamma-poisson mixture. In reality, of course, the sample mean estimate for the number of infectious stages entering the reservoir in Eq.', 'cite_spans': [{'start': 176, 'end': 179, 'text': '[7]', 'ref_id': 'BIBREF5'}], 'ref_spans': [], 'section': 'The approximate worm burden distribution'}, {'text': '(2) will induce stochastic fluctuations in Λ(t). Estimating the amplitude of these fluctuations will be essential to evaluating the distribution of worms within hosts in the stochastic individual-based simulation.', 'cite_spans': [], 'ref_spans': [], 'section': 'The approximate worm burden distribution'}, {'text': 'To begin with, we must first understand the variation in Λ j (t) that is induced from the finite sample of N j individuals assigned with an initial value of λ jn . The implicit solution to Eq. (2) is given by', 'cite_spans': [], 'ref_spans': [], 'section': 'The approximate worm burden distribution'}, {'text': 'Between ensemble realisations of Eq. (18), the only source of random variation is the sample variance in individual worm burdens when computing the value of the double sum', 'cite_spans': [], 'ref_spans': [], 'section': 'The approximate worm burden distribution'}, {'text': 'As discussed in Ref. [28] , the extremely short reservoir timescale 1/µ 2 compared to the timescales of the other processes, e.g., 1/µ and 1/µ 1 , allows for an accurate approximation 4 to Eq. (18) to be made where one may coarse-grain (integrate) over time such that', 'cite_spans': [{'start': 21, 'end': 25, 'text': '[28]', 'ref_id': 'BIBREF26'}, {'start': 193, 'end': 197, 'text': '(18)', 'ref_id': None}], 'ref_spans': [], 'section': 'The approximate worm burden distribution'}, {'text': \"(20) If one now also assumes that w jn is drawn from an approximately negative binomial distribution then the distribution of N a j=1 N j samples drawn from the individual's egg input distribution is approximately also a negative binomial with updated mean and variance [28] such that\", 'cite_spans': [{'start': 270, 'end': 274, 'text': '[28]', 'ref_id': 'BIBREF26'}], 'ref_spans': [], 'section': 'The approximate worm burden distribution'}, {'text': 'where one may compute the variance of the egg input distribution through evaluation of its second moment, yielding', 'cite_spans': [], 'ref_spans': [], 'section': 'The approximate worm burden distribution'}, {'text': 'By combining Eqs. (1), (17) , (20) and (21) we may infer that the full distribution over ensemble realisations of P(w jn , t) may be approximated by', 'cite_spans': [{'start': 23, 'end': 27, 'text': '(17)', 'ref_id': 'BIBREF15'}, {'start': 30, 'end': 34, 'text': '(20)', 'ref_id': 'BIBREF18'}], 'ref_spans': [], 'section': 'The approximate worm burden distribution'}, {'text': 'where we have marginalised over all possible reservoir config-urationsȲ (within the negative binomial approximation) and individual uptake rates λ jn . In Appendix B we calculate the approximate ensemble mean and variance of the mean worm burden constructed from summed samples from Eq. (25) . Sampling directly from Eq. (25), one may effectively reconstruct an accurate ensemble of realisations at any specified point in time as long as the negative binomial approximation of the reservoir is accurate. However, accurate temporal correlations between the sum of individuals will require alternative methods.', 'cite_spans': [{'start': 287, 'end': 291, 'text': '(25)', 'ref_id': 'BIBREF23'}], 'ref_spans': [], 'section': 'The approximate worm burden distribution'}, {'text': 'In order to obtain a better approximation to the temporal correlations of the system, we shall perform a mean field expansion which allows us to calculate an accurate approximation to the stochastic noise around the mean worm burden of the deterministic STH model. Let us now rewrite the value of N j m j in separate components which depend differently on the system size N j , to obtain', 'cite_spans': [], 'ref_spans': [], 'section': 'A mean field expansion of the system'}, {'text': 'where ξ j (t) denotes a fluctuation in each age bin at each moment in time.', 'cite_spans': [], 'ref_spans': [], 'section': 'A mean field expansion of the system'}, {'text': 'Note that in Eq. (26) we have performed a temporal average (as opposed to ensemble average) over the states of the Poisson walkers, each with intensity I jn (t) -see Eq. (17) . The full ensemble of realisations for the system, which includes a random set of values chosen for λ jn , is not strictly ergodic. Hence, in order to assess the temporally correlated behaviour, in this section we shall use the temporal average (with notation · ) to refer to drawing the random values of λ jn a priori for a given realisation of the system -averaging over only the uncertainty component 2. as discussed at the beginning of Sec. 2.2. The full ensemble may then be restored subsequently by a second averaging over finite population samples (component 1. of Sec. 2.2) a posteriori.', 'cite_spans': [{'start': 170, 'end': 174, 'text': '(17)', 'ref_id': 'BIBREF15'}], 'ref_spans': [], 'section': 'A mean field expansion of the system'}, {'text': 'In Appendix C we perform a van Kampen [31] expansion to obtain a Fokker-Planck equation for the evolution of the distribution over fluctuations ξ j in time, which is', 'cite_spans': [{'start': 38, 'end': 42, 'text': '[31]', 'ref_id': 'BIBREF29'}], 'ref_spans': [], 'section': 'A mean field expansion of the system'}, {'text': 'which has the following stationary solutioñ', 'cite_spans': [], 'ref_spans': [], 'section': 'A mean field expansion of the system'}, {'text': 'The exact form of the force of infection Λ j (t) is not generally known, however in the limit where the mean-field equations apply, the following estimator may be intuited which uses the ensemble average', 'cite_spans': [], 'ref_spans': [], 'section': 'A mean field expansion of the system'}, {'text': 'Using the approximation in Eq. (30) we may thus use the solution to the mean field Eqs. (10) and (12) to stochastically evolve trajectories of the mean worm burden with temporal correlations.', 'cite_spans': [{'start': 88, 'end': 92, 'text': '(10)', 'ref_id': 'BIBREF8'}], 'ref_spans': [], 'section': 'A mean field expansion of the system'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '5'}, {'text': 'The copyright holder for this preprint . https://doi.org/10.1101/2019. 12.17.19013490 doi: medRxiv preprint ', 'cite_spans': [{'start': 71, 'end': 85, 'text': '12.17.19013490', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'C1 R 0 = 3.5; k = 0.3; M(t 0 ) = 2', 'cite_spans': [], 'ref_spans': [], 'section': 'Configuration Parameter choices (t 0 is initial time)'}, {'text': \"Having established the main mathematical preliminaries, in the analysis of our results it will be convenient to refer to specific configurations of the human heminth system (according to initial choices of transmission parameters) to illustrate our results more effectively. These configurations shall always include the following choices corresponding to the hookworm timescales of: the adult worm death rate µ 1 = 1/2 per year (or an average life span of 2 years); the eggs/larvae death rate in the infectious reservoir µ 2 = 26 per year (or an average life span of 2 weeks) and a density-dependent worm fecundity power of γ = 0.08 [32] . We also have provided the particular choices of parameters in the configurations C1 and C2 of Table 1 . Configuration C1 has been chosen with parameters such that the system is far from the unstable 'breakpoint' equilibrium as illustrated in Fig. 1 , conversely, configuration C2 has been chosen such that the system is much closer to the breakpoint for a clear comparison.\", 'cite_spans': [{'start': 634, 'end': 638, 'text': '[32]', 'ref_id': 'BIBREF30'}], 'ref_spans': [{'start': 735, 'end': 742, 'text': 'Table 1', 'ref_id': 'TABREF0'}, {'start': 883, 'end': 889, 'text': 'Fig. 1', 'ref_id': 'FIGREF0'}], 'section': 'Deterministic models versus stochastic simulations'}, {'text': 'In Sec. 2.2, we stated that deterministic models of STH transmission are obtained by averaging over both of the uncertainties in predispositions to infection and when events precisely occur in time. As such, the first of our results regarding the relationship between the deterministic models of STH transmission and stochastic simulations is not unexpected. The deterministic predictions for the dynamics of the mean worm burden [7] and the mean of the mean worm burdens derived from individual-based simulations exactly match -see, for instance, the good match between dashed lines plotted in the left and right top panels of Fig. 2 , where we have plotted the dynamics of the mean worm burdens in a community with parameters in configuration C1 (see Table 1 ) for a range of population sizes. The left and right columns of this figure correspond to the numerical solution to the fully individual-based stochastic simulation that we defined at the beginning of Sec. 2 and the mean field model with the stochastic noise approximation using Eqs. (27) and (28), respectively.', 'cite_spans': [{'start': 430, 'end': 433, 'text': '[7]', 'ref_id': 'BIBREF5'}, {'start': 1046, 'end': 1050, 'text': '(27)', 'ref_id': 'BIBREF25'}], 'ref_spans': [{'start': 628, 'end': 634, 'text': 'Fig. 2', 'ref_id': 'FIGREF3'}, {'start': 753, 'end': 760, 'text': 'Table 1', 'ref_id': 'TABREF0'}], 'section': 'Deterministic models versus stochastic simulations'}, {'text': \"Considering the same pair of plots (top row) in Fig. 2 in more detail we find that, for a given range of parameters, the two methods used to generate the 68% credible intervals about each mean value also agree to very good accuracy. This agreement is not always the case, however, as we illustrate with the same pair of plots but with different transmission parameters on the bottom row of Fig. 2 . Given configuration C2 (see Table 1 ), in which the endemic equilibrium value is situated closer to the unstable 'breakpoint' equilibrium, we find that an important phenomenon which exists in the individual-based stochastic simulation is not present in the deterministic model -this phenomenon is known as 'fade-out' in the parasite population. 'Fade-out' is where transmission is interrupted due to chance events, even when the underlying transmission success (the value of R 0 ) is above the level that deterministic models predict will result in parasite persistence. In other words, values just above the transmission breakpoint of the deterministic model may indeed move to parasite extinction, due to chance effects.\", 'cite_spans': [], 'ref_spans': [{'start': 48, 'end': 54, 'text': 'Fig. 2', 'ref_id': 'FIGREF3'}, {'start': 390, 'end': 396, 'text': 'Fig. 2', 'ref_id': 'FIGREF3'}, {'start': 427, 'end': 434, 'text': 'Table 1', 'ref_id': 'TABREF0'}], 'section': 'Deterministic models versus stochastic simulations'}, {'text': \"Ordinarily in disease transmission models, the stochastic noise approximation using Eqs. (27) and (28) may be used to compute similar 'fade-out' effects to great accuracy, see, e.g., in an SIR model [33] . However, the presence of the unstable 'breakpoint' equilibrium leads to a stronger fade-out effect as one can observe in the drift towards a zero mean worm burden (transmission interruption) of the lower 68% credible interval contour (and mean) in the bottom row plots of Fig. 2 with population sizes of 100 and 350, and which markedly reduces the accuracy of the mean field stochastic approximation. A simple Gaussian approximation notwithstanding, fade-out is a very important phenomenon of considerable practical relevance to the prospect of helminth transmission interruption. Note that the probability of a fade-out event also increases from very small to large as village population size decreases, as is also shown by the results of Fig. 2 . The inability of the stochastic noise approximation model to capture the fade-out phenomenon is of considerable interest and is requires a more detailed mathematical description of the stochastic dynamics of the reservoir of infection. Such work is planned for investigation in future work.\", 'cite_spans': [{'start': 89, 'end': 93, 'text': '(27)', 'ref_id': 'BIBREF25'}, {'start': 199, 'end': 203, 'text': '[33]', 'ref_id': 'BIBREF31'}], 'ref_spans': [{'start': 478, 'end': 484, 'text': 'Fig. 2', 'ref_id': 'FIGREF3'}, {'start': 946, 'end': 952, 'text': 'Fig. 2', 'ref_id': 'FIGREF3'}], 'section': 'Deterministic models versus stochastic simulations'}, {'text': 'In our analysis in this section, although we ignored age structure, its inclusion does not influence the general insights although it will influence quantitative detail. For hookworm, the absence of age structure is not too serious, even in quantitative terms since field studies suggest that the force of infection is often independent of age [32] . To illustrate this point, in Fig. 3 we have plotted the C1 configuration (see Table 1 ) with relevant hookworm age stucture included, where the difference between ', 'cite_spans': [{'start': 344, 'end': 348, 'text': '[32]', 'ref_id': 'BIBREF30'}], 'ref_spans': [{'start': 380, 'end': 386, 'text': 'Fig. 3', 'ref_id': 'FIGREF5'}, {'start': 429, 'end': 436, 'text': 'Table 1', 'ref_id': 'TABREF0'}], 'section': 'Deterministic models versus stochastic simulations'}, {'text': \"STHs are dioecious and therefore both male and female parasites must be in the same host for the female worm to be fertilized and produce viable eggs or larvae which sustain transmission. Past research in the context of the deterministic models of transmission has demonstrated that there are three apparent equilibria: a stable endemic state, parasite extinction, and an unstable state termed the 'transmission breakpoint' which lies between the stable state and parasite extinction [34, 35, 36] . The existence of such a breakpoint can be intuited by the above limitations of STH reproduction when it becomes difficult to find a male and female worm pair within an individual host and it represents a clear target for control policies which aim to achieve transmission interruption. As we also discussed earlier in Sec. 1 and illustrated in Fig. 1 , this (dashed) breakpoint curve in the phase plane of Fig. 1 acts as a separatrix between 6 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\", 'cite_spans': [{'start': 484, 'end': 488, 'text': '[34,', 'ref_id': 'BIBREF32'}, {'start': 489, 'end': 492, 'text': '35,', 'ref_id': 'BIBREF33'}, {'start': 493, 'end': 496, 'text': '36]', 'ref_id': 'BIBREF34'}], 'ref_spans': [{'start': 843, 'end': 849, 'text': 'Fig. 1', 'ref_id': 'FIGREF0'}, {'start': 905, 'end': 911, 'text': 'Fig. 1', 'ref_id': 'FIGREF0'}], 'section': \"The 'breakpoint' of stochastic simulations\"}, {'text': 'The copyright holder for this preprint (27) and (28)) have also been run for comparison (right column). The top row corresponds configuration C1 in Table 1 .', 'cite_spans': [], 'ref_spans': [{'start': 148, 'end': 155, 'text': 'Table 1', 'ref_id': 'TABREF0'}], 'section': '(which was not peer-reviewed)'}, {'text': 'The bottom row corresponds configuration C2 in Table 1. 7 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. Table 1 . In this instance, the population has been split into two equal-sized components of ages 0-14 and 15+, where we the worm uptake rate in the case of the 15+ group has been set to exactly 3 times that of the 0-14 group. The observed lack of an effect may be restricted to hookworm infections where the force of infection is often constant across age classes. This is not the case for Ascaris and Trichuris infections [23] . the attractor basins of the stable (endemic) and exctinction equilibria.', 'cite_spans': [{'start': 646, 'end': 650, 'text': '[23]', 'ref_id': 'BIBREF21'}], 'ref_spans': [{'start': 47, 'end': 55, 'text': 'Table 1.', 'ref_id': 'TABREF0'}, {'start': 222, 'end': 229, 'text': 'Table 1', 'ref_id': 'TABREF0'}], 'section': '(which was not peer-reviewed)'}, {'text': \"Extensive past numerical analyses based on using the individual-based simulation code [37, 25] strongly indicate that the uncertainties which are inherent in individual-based simulations dominate the transmission dynamics near the deterministic breakpoint in transmission. This is especially true if the breakpoint is close to the endemic equilibrium of parasite extinction (as opposed to the stable equilibrium of endemic infection) as a consequence of high degree of parasite aggregation in the human population [38, 7] , as is illustrated by the appearance of the fade-out effect between the top left and bottom left plots of Fig. 2 . Furthermore, in the previous section, we highlighted some limitations in the standard formulation of deterministic models to account for the specific phenomena of population 'fade-out' (or spontaneous transmission interruption) which are present in the real world due to chance effects in population growth and decay especially when host population densities are low. Such effects are also captured by individualbased stochastic simulation models. It may hence be reasonable to ask whether concepts which exist in the deterministic framework are present in the stochastic simulations.\", 'cite_spans': [{'start': 86, 'end': 90, 'text': '[37,', 'ref_id': 'BIBREF35'}, {'start': 91, 'end': 94, 'text': '25]', 'ref_id': 'BIBREF23'}, {'start': 514, 'end': 518, 'text': '[38,', 'ref_id': 'BIBREF36'}, {'start': 519, 'end': 521, 'text': '7]', 'ref_id': 'BIBREF5'}], 'ref_spans': [{'start': 629, 'end': 635, 'text': 'Fig. 2', 'ref_id': 'FIGREF3'}], 'section': '(which was not peer-reviewed)'}, {'text': \"Conclusions drawn from hookworm simulation studies in Refs. [25, 37] for a range of population sizes study thresholds in prevalence in the range 0.5%-2% below which the probability of transmission interruption is assessed. In particular, attaining a prevalence below the threshold of ∼ 1% for population sizes 100-1000 leads to interruption of transmission with a high probability. In this section, we provide a theoretical argument for the existence of this threshold as a 'stochastic breakpoint' which works in a similar way to the behaviour present in deterministic models, but accounting for some unavoidable uncertainty.\", 'cite_spans': [{'start': 60, 'end': 64, 'text': '[25,', 'ref_id': 'BIBREF23'}, {'start': 65, 'end': 68, 'text': '37]', 'ref_id': 'BIBREF35'}], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': \"Let us define a net 'grower' as an individual who, given a particular reservoir of infection and adult worm death rate, is able to accumulate more than one worm over time in a consistent manner (an individual who is predisposed to infection). Such individuals are crucial to the survival of the parasite in a defined community as they provide future material to the reservoir of infective stages in the habitat which subsequentially may grow the number of human hosts who become net growers of infection themselves, and so on. One can typically state that if the number of human hosts in which the parasite population grows (net growers) is one or more in a given cluster or community of people, then the chances of parasite extinction in the long-term are low.\", 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'Immediately after, e.g., treatment through an MDA programme, only a limited number of infected individuals remain in each age bin N inf and hence are able to contribute to the reservoir in the following timesteps (until further relaxation of the system towards some equilibrium). Using a modified version of Eq. (20) we may infer the following estimate of the infectious reservoir contribution by those individuals into the j-th age bin', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'Hence, by further assuming that the fluctuations in the reservoir of infection are well-approximated by the negative binomial distribution, we may evaluate the expected maximum fraction E( f >1 max ) -of all individuals in the cluster -whose ratio between their uptake rate λ jn Λ j (t) inf and the death rate of a single worm (µ + µ 1 ) exceeds 1 through the following integral motivated by the form of Eq. (25)', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': \"where NB inf [Ȳ; E(Ȳ), Var(Ȳ)] denotes the reservoir egg count negative binomial distribution corresponding toȲ| inf and Θ(·) is a Heaviside function. Given Eq. (32) he expected number of net growers is therefore simply N p E( f >1 max ). Let us now define the prevalence of infection p as the fraction of the total population who are infected. We therefore deduce that p = N inf /N p . Using Eqs. (25) and (32), we have plotted a calculation of both the probability of the number of net growers for communities with configurations C1 and C2 (see Table 1 ) in Fig. 4 and the expected number of net growers as a function of the prevalence in Fig. 5 for a wide range of system configurations. In both sets of plots we consider population numbers of 100, 350 and 1000, as indicated. Fig. 4 demonstrates that for system configurations further from the unstable 'breakpoint' equilibrium -configuration C1 in the right column of plots -the probability of exceeding a net number of growers larger than 1 becomes large once the 1% 8 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\", 'cite_spans': [{'start': 398, 'end': 402, 'text': '(25)', 'ref_id': 'BIBREF23'}], 'ref_spans': [{'start': 547, 'end': 554, 'text': 'Table 1', 'ref_id': 'TABREF0'}, {'start': 560, 'end': 566, 'text': 'Fig. 4', 'ref_id': 'FIGREF6'}, {'start': 641, 'end': 647, 'text': 'Fig. 5', 'ref_id': None}, {'start': 780, 'end': 786, 'text': 'Fig. 4', 'ref_id': 'FIGREF6'}], 'section': '(which was not peer-reviewed)'}, {'text': \"The copyright holder for this preprint The relative probability (the probability mass divided by the maximum value) of a number of individuals who will gain more than one worm, or, 'net growers' given that effectively only one individual is contributing to the reservoir of infection per unit time for two different system configurations (given by the left and right columns). This probability has been calculated using Eq. (25) and the condition for a net grower -as discussed in Sec. 2.5. The right columns have the same parameters of the system as those of configuration C1 in Table 1 (which is assumed to have attained an endemic equilibrium mean worm burden). The left columns have the same parameters of the system as those of configuration C2 in Table 1 (which is also assumed to have attained an endemic equilibrium mean worm burden).\", 'cite_spans': [], 'ref_spans': [{'start': 580, 'end': 587, 'text': 'Table 1', 'ref_id': 'TABREF0'}, {'start': 753, 'end': 760, 'text': 'Table 1', 'ref_id': 'TABREF0'}], 'section': '(which was not peer-reviewed)'}, {'text': \"prevalence threshold, with the contrary being true below this threshold. The plots in the left column -which correspond to configuration C2 -the probability for the number of net growers to be above 1 remains low even for a prevalence of 2%, which is to be expected due to this configuration's proximity to the unstable 'breakpoint' equilibrium. Furthermore, from Fig. 5 , it is immediately apparent that the expected number of growers is extremely unlikely to exceed 1 if the prevalence of infection is lower than ∼ 1% in all cases of population number. Hence, this threshold essentially acts as a 'stochastic breakpoint', below which the probability of interrupting transmission in stochastic individual-based simulations becomes very likely. Note that caveats to our analysis here include varying the population number well beyond the prior limits used in our short sensitivity analysis -in such situations it is possible for apparent thresholds to go much lower or higher, e.g., consider the small, but non-negligible probabilities of sustaining transmission below a prevalence of 1% for much larger population numbers than those studied here in Ref. [37] . It is important to also mention that the approximate bound we obtain on the prevalence as a 'stochastic breakpoint' applies as more of a strict bound and hence does not exclude the possibility (or even high likelihood in some specific configurations) of elimination at a higher prevalence of 2%. Our discussion here is therefore complementary (and not contradictory) to, e.g., Ref. [25] .\", 'cite_spans': [{'start': 1155, 'end': 1159, 'text': '[37]', 'ref_id': 'BIBREF35'}, {'start': 1544, 'end': 1548, 'text': '[25]', 'ref_id': 'BIBREF23'}], 'ref_spans': [{'start': 364, 'end': 370, 'text': 'Fig. 5', 'ref_id': None}], 'section': '(which was not peer-reviewed)'}, {'text': 'We expect that similar conclusions apply in settings where parasite interruption is predicted since the basic reproduction number (R 0 ) is too low to sustain transmission (see Fig. 1 ). If the mean worm load is just below this critical R 0 value to sustain transmission in regions where the only stable state is parasite extinction, there will be a probability distribution of situations where parasite persistence occurs for a while due to chance events which may obscure the precise location of this threshold.', 'cite_spans': [], 'ref_spans': [{'start': 177, 'end': 183, 'text': 'Fig. 1', 'ref_id': 'FIGREF0'}], 'section': '(which was not peer-reviewed)'}, {'text': \"Our short analysis in this section has provided some theoretical justification for the existence of a 'stochastic breakpoint' which has been identified to be an important target for transmission interruption from previous hookworm simulation studies [25, 37] and our analysis may also be extended to other parasites, which shall be subject of future work.\", 'cite_spans': [{'start': 250, 'end': 254, 'text': '[25,', 'ref_id': 'BIBREF23'}, {'start': 255, 'end': 258, 'text': '37]', 'ref_id': 'BIBREF35'}], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': \"In the stochastic model we have built, the effect of people ageing over time has been neglected so far. Assuming that the effective number of people in each age bin remains unchanged, the passing of individuals between age bins may be fully accounted for as a reservoir pulse of the same form as Eq. (33) which introduces new egg pulses drawn from the ( j − 1)-th age bin and removes eggs from the j-th bin into the ( j + 1)-th bin. The reservoir fluctuation model described above demonstrates how ageing can can only affect the system by a significant variation in k j between bins -furthermore, such fluctuations are can be shown to be damped significantly when the age bins are much wider than 1/(µ + µ 1 ) due to the relaxation timescale of each individual Poisson walker (see, e.g., Eq. (17)). In light of this fact, we have elected to avoid a direct description of ageing in this work without loss of generality in our Figure 5 : The expected number of individuals who will gain more than one worm, or, 'net growers' given that effectively only one individual is contributing to the reservoir of infection per unit time for a range of different system configurations. These values have been obtained by randomly sampling configurations of Eq. (32) with flat priors defined over the following intervals in one cluster without age structure for illustration: the basic reproduction number R 0 ∈ [2, 4]; the logarithim (base 10) of the worm aggregation parameter log 10 k ∈ [−4, 0]; the initial mean worm burden M(t 0 ) ∈ [2, 5] and an assumed density-dependent worm fecundity power of γ = 0.08. The range of system configurations sampled corresponds to much wider range than those given in Fig. 4. \", 'cite_spans': [], 'ref_spans': [{'start': 925, 'end': 933, 'text': 'Figure 5', 'ref_id': None}, {'start': 1694, 'end': 1701, 'text': 'Fig. 4.', 'ref_id': 'FIGREF6'}], 'section': 'A note on modifying the reservoir to account for ageing'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '10'}, {'text': 'The copyright holder for this preprint .', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'argumentation, while leaving future versions of our stochastic simulation to include these sub-dominant effects. Past published individual-based stochastic models of helminth transmission have included a dynamically ageing population [23] .', 'cite_spans': [{'start': 234, 'end': 238, 'text': '[23]', 'ref_id': 'BIBREF21'}], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'Up until this point we have not considered processes which may modify the standard STH transmission model. Applying control measures through MDA and the effect of infected human migration between clusters are both processes which modify the dynamics. The impact of MDA has been covered in various recent publications [39, 25, 37] , whereas, infected human migration in the context of STH transmission has only recently been addressed in the context of its impact on achieving breakpoints in transmission [27, 28] . In the context of STH infections it is necessary for infected people to be present at a location long enough to contribute to the infectious reservoir.', 'cite_spans': [{'start': 317, 'end': 321, 'text': '[39,', 'ref_id': 'BIBREF37'}, {'start': 322, 'end': 325, 'text': '25,', 'ref_id': 'BIBREF23'}, {'start': 326, 'end': 329, 'text': '37]', 'ref_id': 'BIBREF35'}, {'start': 504, 'end': 508, 'text': '[27,', 'ref_id': 'BIBREF25'}, {'start': 509, 'end': 512, 'text': '28]', 'ref_id': 'BIBREF26'}], 'ref_spans': [], 'section': 'The effect of infected human migration'}, {'text': 'In this paper, we incorporate the migration model for STH described in Ref. [28] into our individual-based stochastic simulation. We shall use this model to describe the consequences that infected human migration has on human helminth transmission using illustrative examples. In Ref. [28] the relative size of the migration rate of individual egg counts into or out of a region (e.g., some number of people per year) compared to the death rate of infectious stages within the reservoir, µ 2 (µ 2 = 26 per year for hookworm), was found to be have important implications for the effect of migration on transmission. Hence, in all of the results on migration in this section, the migration rate shall be quoted as a factor of µ 2 to investigate the importance of the ratio between these two parameters in the fully individualbased stochastic simulation.', 'cite_spans': [{'start': 76, 'end': 80, 'text': '[28]', 'ref_id': 'BIBREF26'}, {'start': 285, 'end': 289, 'text': '[28]', 'ref_id': 'BIBREF26'}], 'ref_spans': [], 'section': 'The effect of infected human migration'}, {'text': 'The migration of infected individuals between clustered communities represents a combination of both the finite population and dynamical uncertainty described at the beginning of Sec. 2.2. A simple modification to the model we have introduced, which builds from the work in Ref. [28] , may be considered to incorporate the effects of human migration into the transmission dynamics: consider a compound Poisson process j (t) which perturbs Eq. (2) in the following way 3 3 As was also shown in Ref. [28] , when the reservoir timescale µ 2 is not as fast, Eq. (34) should receive a non-Markovian modification ∝ 1 − e µ 2 (t−t 0 ) to account for the slower decay of pulses in the reservoir.', 'cite_spans': [{'start': 279, 'end': 283, 'text': '[28]', 'ref_id': 'BIBREF26'}, {'start': 470, 'end': 471, 'text': '3', 'ref_id': 'BIBREF1'}, {'start': 498, 'end': 502, 'text': '[28]', 'ref_id': 'BIBREF26'}], 'ref_spans': [], 'section': 'The effect of infected human migration'}, {'text': 'and where r + and r − are the ingoing and outgoing the migratory rates, respectively, and we have coarse-grained in time to obtain the j (t) fluctuation amplitude with respect to the mean worm burden -as in Eq. (20) . With the effect of migration defined in this way, the parameters of Eq. (35) may be randomly selected from any of the corresponding available age bins, and the ensemble mean updated to account for any out-of-equilibrium behaviour. Using a similar set of pulses, in Ref. [28] it was further shown that the migratory rate must exceed or be around the same order as the death rate of infectious material in the reservoir, i.e., r + , r − µ 2 , for the migration to affect the dynamics significantly.', 'cite_spans': [{'start': 211, 'end': 215, 'text': '(20)', 'ref_id': 'BIBREF18'}, {'start': 488, 'end': 492, 'text': '[28]', 'ref_id': 'BIBREF26'}], 'ref_spans': [], 'section': 'The effect of infected human migration'}, {'text': 'In Ref. [28] it is also assumed that the reservoir pulses which arise from migration are uniform across the full ensemble of realisations: though this captures the inherent variability due to migration effects, it does not marginalise entirely over the full ensemble of possible fluctuations which are captured in this work. It is necessary in the finite population limit, then, for such processes to be modelled by a full stochastic individualbased simulation, where we leave the possibility of an analytic description for future work.', 'cite_spans': [{'start': 8, 'end': 12, 'text': '[28]', 'ref_id': 'BIBREF26'}], 'ref_spans': [], 'section': 'The effect of infected human migration'}, {'text': 'In light of the apparent necessity for a numerical approach, we also point out that when using a fully individual-based simulation one may randomly draw from the worm burdens of the individuals and compute an egg pulse amplitude using Eq. (3) -in doing so potentially capturing the out-of-equilibrium behaviour of the reservoir during, e.g., rounds of treatment. For the latter reason we shall adopt this approach when generating our numerical results throughout. We further note that such an approach has also been advocated in similar models such as those in Ref. [27] .', 'cite_spans': [{'start': 566, 'end': 570, 'text': '[27]', 'ref_id': 'BIBREF25'}], 'ref_spans': [], 'section': 'The effect of infected human migration'}, {'text': \"In the bottom left panel of Fig. 2 we provided an example of the fade-out effect which appears in standard individual-based simulations. In metapopulation disease transmission models it is well-known that migration effects can stabilise local tranmission, evading effects such as fade-out even when the population sizes are small -see, e.g., Ref. [40] . By including migration from a cluster with configuration C1 (see Table 1 ) into the cluster depicted in the plot with configuration C2 (with 350 people each), in Fig. 6 we demonstrate that a similar stabilisation of the fade-out effect can also occur for helminth transmission models, despite the especially strong fade-out induced from the presence of the unstable 'breakpoint' equilibrium. As targets for helminth elimination trials drive the prevalence in regions closer to (and beyond) the unstable equilibrium, this is an important possibility to take note of with practical implications should migration within a given region be of the same order or higher as is indicated in Fig. 2 . Note that in order for this effect to be important, the migration rate needs to exceed the death rate of eggs/larvae in the infectious reservoir, i.e., r + , r − µ 2 , which is an early confirmation of the conclusions drawn from Ref. [28] , but we shall now investigate the importance of migration rates relative to µ 2 in more detail by examining another 11 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\", 'cite_spans': [{'start': 347, 'end': 351, 'text': '[40]', 'ref_id': 'BIBREF38'}, {'start': 1279, 'end': 1283, 'text': '[28]', 'ref_id': 'BIBREF26'}], 'ref_spans': [{'start': 28, 'end': 34, 'text': 'Fig. 2', 'ref_id': 'FIGREF3'}, {'start': 419, 'end': 426, 'text': 'Table 1', 'ref_id': 'TABREF0'}, {'start': 516, 'end': 522, 'text': 'Fig. 6', 'ref_id': 'FIGREF7'}, {'start': 1036, 'end': 1042, 'text': 'Fig. 2', 'ref_id': 'FIGREF3'}], 'section': 'The effect of infected human migration'}, {'text': 'The copyright holder for this preprint Table 1 into a cluster (of 350 people) with the same transmission parameters as configuration C2 in Table 1 , the latter of which is displayed. Three different migration rates of individuals are used as scales of the death rate of eggs/larvae in the infectious reservoir µ 2 and are depicted by different shades of blue colour.', 'cite_spans': [], 'ref_spans': [{'start': 39, 'end': 46, 'text': 'Table 1', 'ref_id': 'TABREF0'}, {'start': 139, 'end': 146, 'text': 'Table 1', 'ref_id': 'TABREF0'}], 'section': '(which was not peer-reviewed)'}, {'text': \"new consequence of migration. Migration into a cluster may also occur from an untreated external source (UES), i.e., from a location with a defined negative binomial distribution of worms within hosts and population number that is left untreated in the endemic state. In such instances, it is also possible that the region may have previously achieved elimination which is then reversed -an 'outbreak' scenario.\", 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'In Fig. 7 we have plotted the prevalence of worms within hosts after a period of 100 years for a previously-eliminated cluster experiencing UES migration for a range of migration rates (as a ratio of the eggs or larvae death rate for comparison). The transmission parameters chosen for the UES match a cluster of the indicated population size and configuration C1 from Table 1 from which migrants move to a cluster with the same indicated population size and transmission parameters from configuration C2 in the left column of plots (configuration C1 in the right column of plots), where the latter cluster has been initialised without any initial infections. From these plots one can immediately see that a discrete transition occurs in prevalence after a critical UES migration rate has been reached and that this value changes according to population size. This transition marks the point at which the STH transmission is sustained endemically, and hence the UES migration rate that this corresponds to is of critical importance to policy makers and implementers of STH control programmes since achieving elimination above this critical point in migration rate is likely impossible without treating the external source of migrants simultaneously.', 'cite_spans': [], 'ref_spans': [{'start': 3, 'end': 9, 'text': 'Fig. 7', 'ref_id': 'FIGREF8'}, {'start': 369, 'end': 376, 'text': 'Table 1', 'ref_id': 'TABREF0'}], 'section': '(which was not peer-reviewed)'}, {'text': 'For the larger population numbers (350 and 1000), Fig. 7 also confirms the conclusions drawn in Ref. [28] -that critical mi-gration rates to affecting transmission must exceed a threshold of at least µ 2 . Note, however that the precise number of individuals required to be contributing to the reservoir per unit time can be substantially modified when the population number is changed and the apparent prevalence required to trigger the outbreak transition can be much higher than 1% -typically 10%. Therefore, it is important that we state that the stochastic breakpoint that we derived in Sec. 2.5 only applies to situations where the prevalence of infection is lowered due to treatment, as opposed to being raised from a zero worm state due to UES migration. Such a difference in breakpoint prevalence value required between these two scenarios is due to the greater homogeneity in worm loads of individuals in the case of UES migration, as opposed to the case of post-treatment where a limited number of individuals have a large number or worms -the latter of which is more consistent with the negative binomial reservoir approximation used to derive the stochastic breakpoint in Sec. 2.5. We plan to investigate this apparent asymmetry in how the stochastic breakpoint applies for individual-based simulations in future work.', 'cite_spans': [{'start': 101, 'end': 105, 'text': '[28]', 'ref_id': 'BIBREF26'}], 'ref_spans': [{'start': 50, 'end': 56, 'text': 'Fig. 7', 'ref_id': 'FIGREF8'}], 'section': '(which was not peer-reviewed)'}, {'text': 'Note here that for configuration C1 and a small population number of 100, in Fig. 7 (top right panel) we see that transmission may transition to the endemic state spontaneously with a much lower migration rate than µ 2 -such an effect is due to the additional variance induced by finite population effects and this represents a potentially important caveat to the migration rate threshold we have proposed above.', 'cite_spans': [], 'ref_spans': [{'start': 77, 'end': 83, 'text': 'Fig. 7', 'ref_id': 'FIGREF8'}], 'section': '(which was not peer-reviewed)'}, {'text': 'To demonstrate the difficulty in achieving STH transmission interruption with an MDA control programme when migration of infected humans between clustered communities is in place, even when treating all communities at once, in Fig. 8 we plot the positive predictive value (PPV) of elimination after 100 years having reached at least a given prevalence threshold or below (as indicated on the horizontal axis) in a cluster with a C2 configuration (see Table 1 ) after 3 rounds of treatment (assuming 100% efficacy). Treatment rounds have been applied in years 15, 16 and 17 simultaneously to clusters with configurations C1 and C2 (with population sizes of 350 each), where the dashed black horizontal line at a PPV of 1 shows that elimination is essentially certain in the C2 configuration after treatment if no migration occurs between the two clusters. Once again, however, we find that this PPV drops sharply as the migration rate between clusters is increased in rate relative to the eggs/larvae death rate in the reservoir, µ 2 .', 'cite_spans': [], 'ref_spans': [{'start': 227, 'end': 233, 'text': 'Fig. 8', 'ref_id': 'FIGREF9'}, {'start': 451, 'end': 458, 'text': 'Table 1', 'ref_id': 'TABREF0'}], 'section': 'Accounting for treatment with random compliance'}, {'text': \"The control of helminth infections worldwide will require a greater degree of monitoring and evaluation throughout 2020 to 2030 as treatment programmes strive to get to the point where transmission interruption thresholds have been achieved. This leads to an increased focus of research into predicting the likelihood of 'bounce back' once the transmission breakpoint has been achieved. Such breakpoints are variously defined, but in Table 1 from which migrants move to a cluster with the same indicated population size and transmission parameters from configuration C2 in the left column of plots (configuration C1 in the right column of plots), where the latter cluster has been initialised without any initial infections. Note that we are using the death rate of eggs/larvae in the infectious reservoir, µ 2 , as a unit with which to measure the relative migration rate, where µ 2 = 26 per year for hookworm. Dashed vertical black lines indicate when the migration rate equals µ 2 .\", 'cite_spans': [], 'ref_spans': [{'start': 434, 'end': 441, 'text': 'Table 1', 'ref_id': 'TABREF0'}], 'section': 'Concluding discussion'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '13'}, {'text': 'The copyright holder for this preprint Table 1 ) given that a prevalence threshold -which is varied on the horizontal axis -has been crossed immediately post-MDA. In three of the shown cases the C2 cluster is also experiencing migration from another cluster with a C1 configuration (once again, see Table 1 ) where the migration rates are indicated (using units of µ 2 = 26 per year for hookworm). The treatment has been applied to both clusters and is 3 rounds of MDA with a random compliance pattern of individual behaviours assumed at 60% coverage (and 100% efficacy) in years 15, 16 and 17 of the overall 100. general for human helminths, the indication from modelling studies is that a true prevalence (based on the use of a very sensitive diagnostic) of less than 1% moves the dynamics of the system to the interruption of transmission. Defining the breakpoint and designing monitoring and evaluation programmes requires the use of stochastic individual-based models to determine this likelihood in terms of PPV and NPV values of the probability of transmission elimination or bounce back. It is therefore highly desirable to account for as many sourcesnatural, or otherwise -of uncertainty to ensure a high degree of confidence in the predictions made by such models. The effect of infected human migration in and out of health implementation units on control outcomes, is but one important example of a key source of heterogeneity.', 'cite_spans': [], 'ref_spans': [{'start': 39, 'end': 46, 'text': 'Table 1', 'ref_id': 'TABREF0'}, {'start': 299, 'end': 306, 'text': 'Table 1', 'ref_id': 'TABREF0'}], 'section': '(which was not peer-reviewed)'}, {'text': 'In this paper, we have sought to answer a number of questions of importance to providing reliable predictions for the design of public health programmes for the control of human helminth infections and their monitoring and evaluation.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'Averaged quantities, e.g., the mean worm burden, are matched well by the deterministic model to the individualbased simulations. This is an expected result conformed by past work [12] . Stochastic versions of the deterministic models employing mean-field approximations may be derived to predict how fluctuations in the averaged quantities vary over time. The mean-field approximation, however, appears to fail to predict specific important phenomena accurately which exist in an individual-based simulation; namely, fade-out effects.', 'cite_spans': [{'start': 179, 'end': 183, 'text': '[12]', 'ref_id': 'BIBREF10'}], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'Our analyses suggest that deterministic models of STH (and other helminth) transmission provide important general guidelines to making predictions of what level of control, such as MDA coverage, and what patterns of individual compliance to treatment will move the system close to or below the deterministic breakpoint in transmission. However, once in this region, chance effects can result in either transmission elimination (fade out) or bounce back even when above or below the breakpoint. In practice, around the unstable equilibrium, stochastic noise induces a degree of uncertainty in outcome which must be described as a probability of a certain outcome being achieved. Deriving these probability descriptions in terms of, for example PPV and NPV values, requires the use of an individual-based stochastic model.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': \"Our analyses suggest that public health workers will benefit from determining the transmission threshold -or 'breakpoint' -for transmission interruption using individual-based simulations, and that theoretical calculations suggest a true prevalence target of below ∼ 1% to have an extremely high probability of interruption in a wide variety of scenarios for all the major helminth infections of humans. The important public health message, however, is that in a stochastic world where chance events play a central role, nothing is certain. As such, outcomes should be expressed as probability events. Even below the 1% threshold, however, bounce back can occur due to chance events, where its likelihood will depend on host and parasite population size.\", 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'Infected human migration, like treatment, is an effect which requires individual-based simulation to assess its importance in sustaining high levels of control in the real world where much heterogeneity exists within and between health intervention population units in most regions of endemic infection. Analyses suggest that sufficiently rapid migration from regions with prevalent infection can shift units in which high MDA coverage is achieved to a situation of sustained transmission. This suggests that external processes such as inter-cluster migration could play an important role in sustaining STH transmission.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'The models suggest the existence of a near-universal critical migration rate (up to the population number caveat discussed in Sec. 3.1) corresponding to the death rate of eggs or larvae in the case of STH infections in the infectious reservoir exists, above which, simulations exhibit various behaviours. These include stabilization of fade-out effects such that infection persist; migration from a neighboring setting generating infection bounce back in locations where infection has been well controlled; and lower PPVs for transmission elimination, proceeding treatment achieving a target prevalence of infection threshold is likely even if all clusters are being treated when migration between locations is occurring. The key result is that migration from an untreated external source (UES) is capable of causing bounce back even in areas with effective MDA coverage to get transmission close to the breakpoint once its rate equals or exceeds the death rate of eggs or larvae in the infectious reservoir. For hookworm, this is typically around 26 infected individuals per year. As a consequence of the result above, if the rate of infected migration exceeds or equals the defined threshold, the PPV for elimination may be lowered even if all clusters are treated.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'Finally, public health workers will hopefully benefit from open access to simulation models, such as our public code, helmpy, in order to examine the effects of infected human migration on the effectiveness of their control policies for STH and other helminths, though some simpler approximate descriptions exist under certain assumptions [28] . In future work, we plan also to extend the helmpy package to include features for likelihood-based inference for parameter estimation from epidemiological data in given settings.', 'cite_spans': [{'start': 339, 'end': 343, 'text': '[28]', 'ref_id': 'BIBREF26'}], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'The models suggest the existence of a near-universal critical migration rate (up to the population number caveat discussed in Sec. 3.1) corresponding to the death rate of eggs or larvae in the case of STH infections in the infectious reservoir exists, above which, simulations exhibit various behaviours. These include stabilization of fade-out effects such that infection persist; migration from an neighboring setting generating infection bounce back in locations where infection has been well controlled; and lower PPVs for transmission elimination, proceeding treatment achieving a target prevalence of infection threshold is likely even if all clusters are being treated when migration between locations is occurring. The challenges for future work on migration impact on the likelihood of achieving transmission breakpoints, lie in good measurement of both the heterogeneity of MDA coverage over time in adjacent health implementation units and migration patterns between such units. Since the mean of Eq. (27) is an averaged quantity it does not fluctuate as a random variable, hence we may now rewrite the probability distribution in terms of the other remaining random variable which now characterises the fluctuations of the system around the temporal average λ jn Λ j (t) + (µ + µ 1 )', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'where the following van Kampen [31] (or Taylor) expansion of the distributions on the right hand side can be made which is controlled by powers of the system size N j P ξ j ± N −1/2 j , t =P ξ j , t ± N −1/2 j ∂ ∂ξ jP (ξ j , t) + (2N j ) −1 ∂ 2 ∂ξ 2 jP (ξ j , t) ± . . . . (C.8)', 'cite_spans': [{'start': 31, 'end': 35, 'text': '[31]', 'ref_id': 'BIBREF29'}], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'By equating Eq. (C.6) and Eq. (C.7) with the expansion above, we may match terms of the same system size order to obtain constraint equations. At O(N j ) there is a cancellation of terms, followed by O(N 1/2 j ) where the mean-field results of Eqs. (12) , (13) 18 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 249, 'end': 253, 'text': '(12)', 'ref_id': 'BIBREF10'}], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': '(which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2019. 12.17.19013490 doi: medRxiv preprint and (14) are consistently reproduced after a posteriori marginalisation over λ jn values. Lastly, at O(N 0 j ), one obtains a linear noise approximation for the fluctuations in the form of a Fokker-Planck equation', 'cite_spans': [{'start': 101, 'end': 137, 'text': '12.17.19013490 doi: medRxiv preprint', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'which also corresponds to the following Langevin equation for the individual realisations dξ j dt = −(µ + µ 1 )ξ j (t) + η j (t)', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'where η j (t) is a zero-mean unit-amplitude Gaussian random variable with the temporal averages of η j (t) = 0, η j (t)η j (t ) = δ(t − t ).', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'Global numbers of infection and disease burden of soil transmitted helminth infections in 2010', 'authors': [{'first': 'R', 'middle': ['L'], 'last': 'Pullan', 'suffix': ''}, {'first': 'J', 'middle': ['L'], 'last': 'Smith', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Jasrasaria', 'suffix': ''}, {'first': 'S', 'middle': ['J'], 'last': 'Brooker', 'suffix': ''}], 'year': 2014, 'venue': 'Parasites & vectors', 'volume': '7', 'issn': '1', 'pages': '', 'other_ids': {}}, 'BIBREF1': {'ref_id': 'b1', 'title': 'The prevention of malaria', 'authors': [{'first': 'R', 'middle': [], 'last': 'Ross', 'suffix': ''}], 'year': 1910, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF2': {'ref_id': 'b2', 'title': 'A contribution to the mathematical theory of epidemics', 'authors': [{'first': 'W', 'middle': ['O'], 'last': 'Kermack', 'suffix': ''}, {'first': 'A', 'middle': ['G'], 'last': 'Mckendrick', 'suffix': ''}], 'year': 1927, 'venue': 'Proceedings of the royal society of london. Series A', 'volume': '115', 'issn': '772', 'pages': '700--721', 'other_ids': {}}, 'BIBREF3': {'ref_id': 'b3', 'title': 'A simple stochastic epidemic', 'authors': [{'first': 'N', 'middle': ['T'], 'last': 'Bailey', 'suffix': ''}], 'year': 1950, 'venue': 'Biometrika', 'volume': '', 'issn': '', 'pages': '193--202', 'other_ids': {}}, 'BIBREF4': {'ref_id': 'b4', 'title': 'The population dynamics of malaria', 'authors': [{'first': 'J', 'middle': ['L'], 'last': 'Aron', 'suffix': ''}, {'first': 'R', 'middle': ['M'], 'last': 'May', 'suffix': ''}], 'year': 1982, 'venue': 'The population dynamics of infectious diseases: theory and applications', 'volume': '', 'issn': '', 'pages': '139--179', 'other_ids': {}}, 'BIBREF5': {'ref_id': 'b5', 'title': 'Infectious Diseases of Humans: Dynamics and Control', 'authors': [{'first': 'R', 'middle': [], 'last': 'Anderson', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'May', 'suffix': ''}], 'year': 1992, 'venue': 'Dynamics and Control', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF6': {'ref_id': 'b6', 'title': 'Transmission of influenza: implications for control in health care settings', 'authors': [{'first': 'R', 'middle': ['A'], 'last': 'Weinstein', 'suffix': ''}, {'first': 'C', 'middle': ['B'], 'last': 'Bridges', 'suffix': ''}, {'first': 'M', 'middle': ['J'], 'last': 'Kuehnert', 'suffix': ''}, {'first': 'C', 'middle': ['B'], 'last': 'Hall', 'suffix': ''}], 'year': 2003, 'venue': 'Clinical infectious diseases', 'volume': '37', 'issn': '8', 'pages': '1094--1101', 'other_ids': {}}, 'BIBREF7': {'ref_id': 'b7', 'title': 'Transmission dynamics and control of severe acute respiratory syndrome', 'authors': [{'first': 'M', 'middle': [], 'last': 'Lipsitch', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Cohen', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Cooper', 'suffix': ''}, {'first': 'J', 'middle': ['M'], 'last': 'Robins', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Ma', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'James', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Gopalakrishna', 'suffix': ''}, {'first': 'S', 'middle': ['K'], 'last': 'Chew', 'suffix': ''}, {'first': 'C', 'middle': ['C'], 'last': 'Tan', 'suffix': ''}, {'first': 'M', 'middle': ['H'], 'last': 'Samore', 'suffix': ''}], 'year': 2003, 'venue': 'Science', 'volume': '300', 'issn': '', 'pages': '1966--1970', 'other_ids': {}}, 'BIBREF8': {'ref_id': 'b8', 'title': 'Effectiveness of a triple-drug regimen for global elimination of lymphatic filariasis: a modelling study', 'authors': [{'first': 'M', 'middle': ['A'], 'last': 'Irvine', 'suffix': ''}, {'first': 'W', 'middle': ['A'], 'last': 'Stolk', 'suffix': ''}, {'first': 'M', 'middle': ['E'], 'last': 'Smith', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Subramanian', 'suffix': ''}, {'first': 'B', 'middle': ['K'], 'last': 'Singh', 'suffix': ''}, {'first': 'G', 'middle': ['J'], 'last': 'Weil', 'suffix': ''}, {'first': 'E', 'middle': [], 'last': 'Michael', 'suffix': ''}, {'first': 'T', 'middle': ['D'], 'last': 'Hollingsworth', 'suffix': ''}], 'year': 2017, 'venue': 'The Lancet Infectious Diseases', 'volume': '17', 'issn': '4', 'pages': '451--458', 'other_ids': {}}, 'BIBREF9': {'ref_id': 'b9', 'title': 'Population dynamics of human helminth infections: control by chemotherapy', 'authors': [{'first': 'R', 'middle': ['M'], 'last': 'Anderson', 'suffix': ''}, {'first': 'R', 'middle': ['M'], 'last': 'May', 'suffix': ''}], 'year': 1982, 'venue': 'Nature', 'volume': '297', 'issn': '5867', 'pages': '', 'other_ids': {}}, 'BIBREF10': {'ref_id': 'b10', 'title': 'Community control of helminth infections of man by mass and selective chemotherapy', 'authors': [{'first': 'R', 'middle': [], 'last': 'Anderson', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Medley', 'suffix': ''}], 'year': 1985, 'venue': 'Parasitology', 'volume': '90', 'issn': '4', 'pages': '629--660', 'other_ids': {}}, 'BIBREF11': {'ref_id': 'b11', 'title': 'Schistosim: a microsimulation model for the epidemiology and control of schistosomiasis', 'authors': [{'first': 'S', 'middle': ['De'], 'last': 'Vlas', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Van Oortmarssen', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Gryseels', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Polderman', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Plaisier', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Habbema', 'suffix': ''}], 'year': 1996, 'venue': 'The American journal of tropical medicine and hygiene', 'volume': '55', 'issn': '5', 'pages': '170--175', 'other_ids': {}}, 'BIBREF12': {'ref_id': 'b12', 'title': 'Onchosim: a model and computer simulation program for the transmission and control of onchocerciasis', 'authors': [{'first': 'A', 'middle': [], 'last': 'Plaisier', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Van Oortmarssen', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Habbema', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Remme', 'suffix': ''}, {'first': 'E', 'middle': [], 'last': 'Alley', 'suffix': ''}], 'year': 1990, 'venue': 'Computer methods and programs in biomedicine', 'volume': '31', 'issn': '1', 'pages': '43--56', 'other_ids': {}}, 'BIBREF13': {'ref_id': 'b13', 'title': 'The lymfasim simulation program for modeling lymphatic filariasis and its control', 'authors': [{'first': 'A', 'middle': [], 'last': 'Plaisier', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Subramanian', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Das', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Souza', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Lapa', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Furtado', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Van Der Ploeg', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Habbema', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Van Oortmarssen', 'suffix': ''}], 'year': 1998, 'venue': 'Methods of information in medicine', 'volume': '37', 'issn': '01', 'pages': '97--108', 'other_ids': {}}, 'BIBREF14': {'ref_id': 'b14', 'title': 'Human hookworm infection in the 21st century', 'authors': [{'first': 'S', 'middle': [], 'last': 'Brooker', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Bethony', 'suffix': ''}, {'first': 'P', 'middle': ['J'], 'last': 'Hotez', 'suffix': ''}], 'year': 2004, 'venue': 'Advances in Parasitology', 'volume': '58', 'issn': '', 'pages': '197--288', 'other_ids': {'DOI': ['10.1016/S0065-308X(04)58004-1']}}, 'BIBREF15': {'ref_id': 'b15', 'title': 'Accelerating work to overcome the global impact of neglected tropical diseases : a roadmap for implementation : executive summary, World Health Organization', 'authors': [], 'year': 2012, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF17': {'ref_id': 'b17', 'title': 'Ntd modelling consortium webpage', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF18': {'ref_id': 'b18', 'title': 'Preventive chemotherapy in human helminthiasis : coordinated use of anthelminthic drugs in control interventions : a manual for health professionals and programme managers, World Health Organization', 'authors': [{'first': 'D', 'middle': ['W T'], 'last': 'Crompton', 'suffix': ''}], 'year': 2006, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF19': {'ref_id': 'b19', 'title': 'Ending the neglect and reaching 2020 goals', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF20': {'ref_id': 'b20', 'title': 'The coverage and frequency of mass drug administration required to eliminate persistent transmission of soil-transmitted helminths', 'authors': [{'first': 'R', 'middle': [], 'last': 'Anderson', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Truscott', 'suffix': ''}, {'first': 'T', 'middle': ['D'], 'last': 'Hollingsworth', 'suffix': ''}], 'year': None, 'venue': 'Philosophical Transactions of the Royal Society B: Biological Sciences', 'volume': '369', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF21': {'ref_id': 'b21', 'title': 'Chapter three -soil-transmitted helminths: Mathematical models of transmission, the impact of mass drug administration and transmission elimination criteria', 'authors': [{'first': 'J', 'middle': [], 'last': 'Truscott', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Turner', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Farrell', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Anderson', 'suffix': ''}], 'year': 2016, 'venue': 'Mathematical Models for Neglected Tropical Diseases', 'volume': '94', 'issn': '', 'pages': '133--198', 'other_ids': {'DOI': ['10.1016/bs.apar.2016.08.002']}}, 'BIBREF22': {'ref_id': 'b22', 'title': 'Assessing the interruption of the transmission of human helminths with mass drug administration alone: optimizing the design of cluster randomized trials', 'authors': [{'first': 'R', 'middle': [], 'last': 'Anderson', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Farrell', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Turner', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Walson', 'suffix': ''}, {'first': 'C', 'middle': ['A'], 'last': 'Donnelly', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Truscott', 'suffix': ''}], 'year': 2017, 'venue': 'Parasites & Vectors', 'volume': '10', 'issn': '1', 'pages': '', 'other_ids': {'DOI': ['10.1186/s13071-017-1979-x']}}, 'BIBREF23': {'ref_id': 'b23', 'title': 'Identifying optimal threshold statistics for elimination of hookworm using a stochastic simulation model', 'authors': [{'first': 'J', 'middle': ['E'], 'last': 'Truscott', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Werkman', 'suffix': ''}, {'first': 'J', 'middle': ['E'], 'last': 'Wright', 'suffix': ''}, {'first': 'S', 'middle': ['H'], 'last': 'Farrell', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Sarkar', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Ásbjörnsdóttir', 'suffix': ''}, {'first': 'R', 'middle': ['M'], 'last': 'Anderson', 'suffix': ''}], 'year': 2017, 'venue': 'Parasites & Vectors', 'volume': '10', 'issn': '1', 'pages': '', 'other_ids': {'DOI': ['10.1186/s13071-017-2256-8']}}, 'BIBREF24': {'ref_id': 'b24', 'title': 'The dynamics and control of direct life cycle helminth parasites', 'authors': [{'first': 'R', 'middle': ['M'], 'last': 'Anderson', 'suffix': ''}], 'year': 1980, 'venue': 'Vito Volterra Symposium on Mathematical Models in Biology', 'volume': '', 'issn': '', 'pages': '278--322', 'other_ids': {}}, 'BIBREF25': {'ref_id': 'b25', 'title': 'Human population movement can impede the elimination of soil-transmitted helminth transmission in regions with heterogeneity in mass drug administration coverage and transmission potential between villages: a metapopulation analysis', 'authors': [{'first': 'C', 'middle': [], 'last': 'Vegvari', 'suffix': ''}, {'first': 'J', 'middle': ['E'], 'last': 'Truscott', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Kura', 'suffix': ''}, {'first': 'R', 'middle': ['M'], 'last': 'Anderson', 'suffix': ''}], 'year': 2019, 'venue': 'Parasites & Vectors', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF26': {'ref_id': 'b26', 'title': \"The 'breakpoint'of soil-transmitted helminths with infected human migration\", 'authors': [{'first': 'R', 'middle': ['J'], 'last': 'Hardwick', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Vegvari', 'suffix': ''}, {'first': 'J', 'middle': ['E'], 'last': 'Truscott', 'suffix': ''}, {'first': 'R', 'middle': ['M'], 'last': 'Anderson', 'suffix': ''}], 'year': 2019, 'venue': 'Journal of theoretical biology', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF27': {'ref_id': 'b27', 'title': 'Exact stochastic simulation of coupled chemical reactions', 'authors': [{'first': 'D', 'middle': ['T'], 'last': 'Gillespie', 'suffix': ''}], 'year': 1977, 'venue': 'The journal of physical chemistry', 'volume': '81', 'issn': '25', 'pages': '2340--2361', 'other_ids': {}}, 'BIBREF28': {'ref_id': 'b28', 'title': 'Multiple spectator condensates from inflation', 'authors': [{'first': 'R', 'middle': ['J'], 'last': 'Hardwick', 'suffix': ''}], 'year': 2018, 'venue': 'JCAP 1805 (05)', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1088/1475-7516/2018/05/054'], 'arXiv': ['arXiv:1803.03521']}}, 'BIBREF29': {'ref_id': 'b29', 'title': 'Stochastic processes in physics and chemistry', 'authors': [{'first': 'N', 'middle': ['G'], 'last': 'Van Kampen', 'suffix': ''}], 'year': 1992, 'venue': '', 'volume': '1', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF30': {'ref_id': 'b30', 'title': 'Heterogeneity in transmission parameters of hookworm infection within the baseline data from the tumikia study in kenya', 'authors': [{'first': 'J', 'middle': ['E'], 'last': 'Truscott', 'suffix': ''}, {'first': 'A', 'middle': ['K'], 'last': 'Ower', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Werkman', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Halliday', 'suffix': ''}, {'first': 'W', 'middle': ['E'], 'last': 'Oswald', 'suffix': ''}, {'first': 'P', 'middle': ['M'], 'last': 'Gichuki', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Mcharo', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Brooker', 'suffix': ''}, {'first': 'S', 'middle': ['M'], 'last': 'Njenga', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Mwandariwo', 'suffix': ''}], 'year': 2019, 'venue': 'Parasites & vectors', 'volume': '12', 'issn': '1', 'pages': '1--13', 'other_ids': {}}, 'BIBREF31': {'ref_id': 'b31', 'title': 'Stochastic epidemics: the probability of extinction of an infectious disease at the end of a major outbreak', 'authors': [{'first': 'O', 'middle': ['A'], 'last': 'Van Herwaarden', 'suffix': ''}], 'year': 1997, 'venue': 'Journal of mathematical biology', 'volume': '35', 'issn': '7', 'pages': '793--813', 'other_ids': {}}, 'BIBREF32': {'ref_id': 'b32', 'title': 'The dynamics of helminth infections, with special reference to schistosomes', 'authors': [{'first': 'G', 'middle': [], 'last': 'Macdonald', 'suffix': ''}], 'year': 1965, 'venue': 'Transactions of The Royal Society of Tropical Medicine and Hygiene', 'volume': '59', 'issn': '5', 'pages': '489--506', 'other_ids': {'DOI': ['10.1016/0035-9203(65)90152-5']}}, 'BIBREF33': {'ref_id': 'b33', 'title': 'Thresholds and breakpoints in ecosystems with a multiplicity of stable states', 'authors': [{'first': 'R', 'middle': ['M'], 'last': 'May', 'suffix': ''}], 'year': 1977, 'venue': 'Nature', 'volume': '269', 'issn': '', 'pages': '471--477', 'other_ids': {'DOI': ['10.1038/269471a0']}}, 'BIBREF34': {'ref_id': 'b34', 'title': 'Helminth infections of humans: Mathematical models, population dynamics, and control', 'authors': [{'first': 'R', 'middle': ['M'], 'last': 'Anderson', 'suffix': ''}, {'first': 'R', 'middle': ['M'], 'last': 'May', 'suffix': ''}], 'year': 1985, 'venue': '', 'volume': '24', 'issn': '', 'pages': '1--101', 'other_ids': {'DOI': ['10.1016/S0065-308X(08)60561-8']}}, 'BIBREF35': {'ref_id': 'b35', 'title': 'Defining stopping criteria for ending randomized clinical trials that investigate the interruption of transmission of soil-transmitted helminths employing mass drug administration', 'authors': [{'first': 'M', 'middle': [], 'last': 'Werkman', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Toor', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Vegvari', 'suffix': ''}, {'first': 'J', 'middle': ['E'], 'last': 'Wright', 'suffix': ''}, {'first': 'J', 'middle': ['E'], 'last': 'Truscott', 'suffix': ''}, {'first': 'K', 'middle': ['H'], 'last': 'Ásbjörnsdóttir', 'suffix': ''}, {'first': 'A', 'middle': ['R'], 'last': 'Means', 'suffix': ''}, {'first': 'J', 'middle': ['L'], 'last': 'Walson', 'suffix': ''}, {'first': 'R', 'middle': ['M'], 'last': 'Anderson', 'suffix': ''}], 'year': 2018, 'venue': 'PLoS neglected tropical diseases', 'volume': '12', 'issn': '10', 'pages': '', 'other_ids': {}}, 'BIBREF36': {'ref_id': 'b36', 'title': 'Thresholds and breakpoints in ecosystems with a multiplicity of stable states', 'authors': [{'first': 'R', 'middle': ['M'], 'last': 'May', 'suffix': ''}], 'year': 1977, 'venue': 'Nature', 'volume': '269', 'issn': '5628', 'pages': '', 'other_ids': {}}, 'BIBREF37': {'ref_id': 'b37', 'title': 'What impact will the achievement of the current world health organisation targets for anthelmintic treatment coverage in children have on the intensity of soil transmitted helminth infections?', 'authors': [{'first': 'J', 'middle': [], 'last': 'Truscott', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Turner', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Anderson', 'suffix': ''}], 'year': 2015, 'venue': 'Parasites & Vectors', 'volume': '8', 'issn': '1', 'pages': '', 'other_ids': {'DOI': ['10.1186/s13071-015-1135-4']}}, 'BIBREF38': {'ref_id': 'b38', 'title': 'The emergence of the rescue effect from explicit within-and between-patch dynamics in a metapopulation', 'authors': [{'first': 'A', 'middle': [], 'last': 'Eriksson', 'suffix': ''}, {'first': 'F', 'middle': [], 'last': 'Elías-Wolff', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Mehlig', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Manica', 'suffix': ''}], 'year': None, 'venue': 'Proceedings of the royal society B: biological sciences', 'volume': '281', 'issn': '', 'pages': '', 'other_ids': {}}}, 'ref_entries': {'FIGREF0': {'text': \"Diagram illustrating the stable endemic equilibrium (solid black line), unstable 'breakpoint' equilibrium (dashed black line) and insufficient R 0 breakpoint. For reference, the transmission model illustrated is that of soiltransmitted helminth infections with the density dependent fecundity parameter γ = 0.08 and worm aggregation within hosts parameter k = 0.3.\", 'latex': None, 'type': 'figure'}, 'FIGREF1': {'text': 'γ = 0.08; µ = 1/70; µ 1 = 1/2; and µ 2 = 26 (last three are per year)', 'latex': None, 'type': 'figure'}, 'FIGREF2': {'text': 'Fig. 3 and the top left plot of Fig. 2 is very small.', 'latex': None, 'type': 'figure'}, 'FIGREF3': {'text': 'Two different examples (in both left and right panels) of the expected (dashed lines) and 68% credible regions (between the solid lines) of the possible mean worm burdens realised in time by the stochastic fully individual-based simulation method (left column) and the counterpart simulations using a mean field stochastic approximation method (see Eqs.', 'latex': None, 'type': 'figure'}, 'FIGREF4': {'text': 'changes in behaviour if the endemic equilibrium is the same', 'latex': None, 'type': 'figure'}, 'FIGREF5': {'text': 'An example demonstrating the relatively small effects of age structure on stochastic individual-based model with the original configuration of C1 in', 'latex': None, 'type': 'figure'}, 'FIGREF6': {'text': \"Figure 4: The relative probability (the probability mass divided by the maximum value) of a number of individuals who will gain more than one worm, or, 'net growers' given that effectively only one individual is contributing to the reservoir of infection per unit time for two different system configurations (given by the left and right columns). This probability has been calculated using Eq. (25) and the condition for a net grower -as discussed in Sec. 2.5. The right columns have the same parameters of the system as those of configuration C1 in Table 1 (which is assumed to have attained an endemic equilibrium mean worm burden). The left columns have the same parameters of the system as those of configuration C2 in Table 1 (which is also assumed to have attained an endemic equilibrium mean worm burden).\", 'latex': None, 'type': 'figure'}, 'FIGREF7': {'text': 'Three different examples (in both left and right panels) of the expected (dashed lines) and 68% credible regions (between the solid lines) of the possible mean worm burdens realised in time by the stochastic individual-based simulation method with the inclusion of infected human migration from a cluster (of 350 people) with the same transmission parameters as configuration C1 in', 'latex': None, 'type': 'figure'}, 'FIGREF8': {'text': 'The prevalence of worms within hosts after a period of 100 years for a previously-eliminated cluster experiencing migration from an untreated external source (UES) for a range of migration rates. The transmission parameters chosen for the UES match a cluster of the indicated population size and configuration C1 from', 'latex': None, 'type': 'figure'}, 'FIGREF9': {'text': 'The positive predictive value (PPV) for elimination (the latter is here defined as no remaining worms after 100 years) in a cluster with a C2 configuration (see', 'latex': None, 'type': 'figure'}, 'FIGREF10': {'text': \"j , t) + (µ + µ 1 )(N j m j + 1)P(N j m j + 1, t) + N j n=1 λ jn Λ j (t)P(N j m j − 1, t) . (C.1)Though this equation would almost-certainly represent a less efficient simulation than that of Eq. (15), it may nonetheless consistent results for a sufficiently short choice of numerical timestep.Let us now rewrite the local value of N j m j (which is regarded as constant in time -or a 'state variable' -by the master equation) in separate components which depend differently on the system size N j , to obtain N j m j = N j m j (t) + N\", 'latex': None, 'type': 'figure'}, 'FIGREF11': {'text': '[ξ j (t), t] . (C.6) Notice that Eq. (27) implies steps of N j m j ± 1 translate into fluctuations of ξ j ± N −1/2 j in the argument ofP. Using this mapping as well as Eq. (27), one finds that Eq. (C.1) may be rewritten as d dt P(N j m j , t)', 'latex': None, 'type': 'figure'}, 'TABREF0': {'text': 'The particular configurations with hookworm parameters[32] used for demonstration in this work.', 'latex': None, 'type': 'table'}}, 'back_matter': [{'text': \"The temporally inhomogenous process of an individual's worm uptake and death is obtained from the Markovian approximation to a summing over the possible histories in the following way P(w jn , t + ∆t) = s=−1,0,1 P(w jn + s, t)P(w jn , t + ∆t|w jn + s, t) = P(w jn − 1, t)λ jn Λ j (t)∆t + P(w jn + 1, t)(µ + µ 1 )(w jn + 1)∆t + P(w jn , t) 1 − λ jn Λ j (t)∆t − (µ + µ 1 )w jn ∆t + O(∆t 2 ) (A.1)Which matches Eq. (15) in the main text. We may also rewrite Eq. (A.2) using the Probability Generating Function (PGF) G jn (Z, t) ≡ ∞ w jn =0 Z w jn P(w jn , t) such thatBy method of characteristics, 4 the solution to Eq. (15) iswhere the initial intensity I jn (t 0 ) is to be set for each individual. One immediately recognises Eq. (A.4) as the PGF of an inhomogeneous Poisson walker with intensitywhich matches Eq. (16) in the main text. For an ensemble of N j independent walkers (hence individuals) in a given age bin we may obtain the PGF of the sum of their collective worm burdens G j (Z, t) by multiplication such thatwhere the corresponding probability mass function is therefore also that of a Poisson distributionwhere we have also used the definition of the mean worm burden in the j-th age bin given by Eq. (4) in the main text. Note that in the case where Λ j (t) is roughly constant for all time, the timescale for the distribution to achieve stationarity is ∆t stat 1/(µ + µ 1 ). 4 One solves first-order PDEs of the form in Eq. (A.3) by method of characteristics, i.e., choosing a curve Z(t) such thatGiven the fact that along the curve Z(t) one also obtains an effective ODE of the formAppendix B. Approximate ensemble mean and variance of the mean worm burdenThe approximate mean and variance of Λ j (t) may be computed by inserting Eqs. (22) and (23) into Eq. (20) and its square, yielding the following equationsUsing Eq. (17) in the main text and Eq. (B.1), we may therefore compute the ensemble mean of m j (t), matches the solution to Eq. (12) in the rapid equilibriation limit dE(Λ j )/dt → 0 and hence verifies the consistency of our approximative results in this section so far.The ensemble variance of m j (t) may also be deduced asfrom the properties of independent samples. 5 By making use of Eq. (1) and assuming the relative independence of λ jn and Λ(t) once again, we haveand hence one may obtain E[I jn (t) 2 ], which is given by 6If one assumes temporal stationarity such that E[Λ j (t)] = E(Λ j ) and E[λ 2 jn Λ j (t)Λ j (t )] = E(λ 2 jn Λ 2 j ), both time integrations in Eq. (B.7) are trivial and one obtains \", 'cite_spans': [{'start': 412, 'end': 416, 'text': '(15)', 'ref_id': 'BIBREF13'}, {'start': 1392, 'end': 1393, 'text': '4', 'ref_id': 'BIBREF2'}, {'start': 1753, 'end': 1757, 'text': '(22)', 'ref_id': 'BIBREF20'}, {'start': 1776, 'end': 1780, 'text': '(20)', 'ref_id': 'BIBREF18'}, {'start': 2200, 'end': 2201, 'text': '5', 'ref_id': 'BIBREF3'}], 'ref_spans': [], 'section': 'Appendix A. Derivation and solution of the master equation'}, {'text': 'We may re-derive Eq. (15) in terms of the sum over all worms carried by N j individuals in the j-th age bin N j m j (t) = N j n=1 w jn (t), taking a single step at each point in time, such that', 'cite_spans': [{'start': 21, 'end': 25, 'text': '(15)', 'ref_id': 'BIBREF13'}], 'ref_spans': [], 'section': 'Appendix C. The van Kampen expansion of the master equation'}]}\n",
      "{'paper_id': 'b117016a887f05386acc4387dab5bf8c769cf90e', 'metadata': {'title': 'Title Page 1 2 Simulating and Forecasting the Cumulative Confirmed Cases of SARS-CoV-2 in China by Boltzmann Function-based Regression Analyses 4 5 of SARS-CoV-2 in mainland China (including Hubei Province) will become minimal between', 'authors': [{'first': 'Xinmiao', 'middle': [], 'last': 'Fu', 'suffix': '', 'affiliation': {'laboratory': 'Provincial University Key Laboratory of Cellular Stress Response and Metabolic Regulation', 'institution': 'Fujian Normal University', 'location': {'addrLine': 'Fujian Province 350117', 'settlement': 'Fuzhou City', 'country': 'China'}}, 'email': ''}, {'first': 'Qi', 'middle': [], 'last': 'Ying', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Texas A&M University', 'location': {'addrLine': '11', 'postCode': '77843', 'settlement': 'College Station', 'region': 'TX', 'country': 'USA. 12'}}, 'email': ''}, {'first': 'Tieyong', 'middle': [], 'last': 'Zeng', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'The Chinese University of Hong Kong', 'location': {'postCode': '99999', 'settlement': 'Shatin, Hong Kong', 'region': 'NT', 'country': 'China'}}, 'email': ''}, {'first': 'Tao', 'middle': [], 'last': 'Long', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Nanjing Institute of Environmental Sciences', 'location': {'addrLine': 'Fujian 20 Province', 'postCode': '350117', 'settlement': 'Fuzhou', 'country': 'China, China'}}, 'email': ''}, {'first': 'Yan', 'middle': [], 'last': 'Wang', 'suffix': '', 'affiliation': {'laboratory': 'Provincial University Key Laboratory of Cellular Stress Response and Metabolic Regulation', 'institution': 'Fujian Normal University', 'location': {'addrLine': 'Fujian Province 350117', 'settlement': 'Fuzhou City', 'country': 'China'}}, 'email': ''}]}, 'abstract': [{'text': 'An ongoing outbreak of atypical pneumonia caused by the 2019 novel coronavirus 2) is hitting Wuhan City and has spread to other provinces/cities of China and overseas. It very 29 urgent to forecast the future course of the outbreak. Here, we provide an estimate of the potential 30 total number of confirmed cases in mainland China by applying Boltzmann-function based 31 regression analyses. We found that the cumulative number of confirmed cases from Jan 21 to Feb 32 14, 2020 for mainland China, Hubei Province, Wuhan City and other provinces were all well 33 fitted with the Boltzmann function (R 2 being close to 0.999). The potential total number of 34 confirmed cases in the above geographic regions were estimated at 95% confidence interval (CI) 35 as 79589 (71576, 93855), 64817 (58223, 77895), 46562 (40812, 57678) and 13956 (12748, 36 16092), respectively. Notably, our results suggest that the number of daily new confirmed cases', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}], 'body_text': [{'text': 'as well as in many foreign countries or regions including Japan, the Republic of Korea, Canada, 52 USA, and European countries [2] [3] [4] . This SARS-CoV-2 outbreak was declared as a public health 53 emergency of international concern by the World Health Organization (WHO) on Jan 30 [5] .', 'cite_spans': [{'start': 127, 'end': 130, 'text': '[2]', 'ref_id': None}, {'start': 131, 'end': 134, 'text': '[3]', 'ref_id': None}, {'start': 135, 'end': 138, 'text': '[4]', 'ref_id': None}, {'start': 285, 'end': 288, 'text': '[5]', 'ref_id': None}], 'ref_spans': [], 'section': ''}, {'text': 'Much research progress has been made in dissecting the evolution and origin of SARS-CoV-2 [6-56 8], as well as characterizing its clinical features [9] [10] [11] [12] [13] [14] [15] and epidemics [16] [17] [18] [19] in the past one and 57 half months . These efforts would significantly guide us to contain the SARS-CoV-2 epidemic. 58 While the outbreak is on-going, people raise grave concerns about the future trajectory of the 59 outbreak, especially given that the working and schooling time has been already dramatically 60 postponed after the Chinese Lunar New Year holiday was over (scheduled on Jan 31). It is highly 61 demanding to estimate the potential total number of confirmed cases, both nationally and locally.', 'cite_spans': [{'start': 148, 'end': 151, 'text': '[9]', 'ref_id': None}, {'start': 152, 'end': 156, 'text': '[10]', 'ref_id': None}, {'start': 157, 'end': 161, 'text': '[11]', 'ref_id': None}, {'start': 162, 'end': 166, 'text': '[12]', 'ref_id': None}, {'start': 167, 'end': 171, 'text': '[13]', 'ref_id': None}, {'start': 172, 'end': 176, 'text': '[14]', 'ref_id': None}, {'start': 177, 'end': 181, 'text': '[15]', 'ref_id': None}, {'start': 196, 'end': 200, 'text': '[16]', 'ref_id': None}, {'start': 201, 'end': 205, 'text': '[17]', 'ref_id': None}, {'start': 206, 'end': 210, 'text': '[18]', 'ref_id': None}, {'start': 211, 'end': 215, 'text': '[19]', 'ref_id': None}, {'start': 332, 'end': 334, 'text': '58', 'ref_id': None}], 'ref_spans': [], 'section': '55'}, {'text': 'Here we present Boltzmann function-based regression analyses on the data of confirmed cases of 80 Data were organized in Microsoft Excel and then incorporated into Microcal Origin software 81 (note: 2021 Jan 21 was set as day 1 and so on). The Boltzmann function was applied to data 82 simulation for each set of data regarding different geographic regions (e.g.., China, Hubei 83 Province and so on) and parameters of each function were obtained, with the potential total 84 number of confirmed cases being directly given by parameter A2. Estimation of critical dates was 85 performed by predicting the cumulative number of confirmed cases in the coming days post Feb 86 14, 2020, and the key dates were provisionally set when the number of daily new confirmed cases 87 is lower than 0.1% of the potential total number. The Boltzmann function for simulation is 88 expressed as follows:', 'cite_spans': [{'start': 95, 'end': 97, 'text': '80', 'ref_id': None}], 'ref_spans': [], 'section': '63'}, {'text': 'where C(x) is the cumulative number of confirmed cases at day x; A1, A2, x0, and dx are constants.', 'cite_spans': [], 'ref_spans': [], 'section': 'Data fitting with Boltzmann function and estimation of critical dates'}, {'text': 'In particular, A2 represents the estimated potential total number of confirmed cases of SARS-92 CoV-2. Details of derivation of the Boltzmann function for epidemic analysis are described in the 93 supporting information file. 95 A Monte Carlo technique is applied to assess the uncertainty in the estimated total number of 96 confirmed cases due to the uncertainty in the reported number cases. 1000 non-linear regressions 97 were performed with the same time series data but each data point in the time series was perturbed 98 by multiplying with a random scaling factor that represents the relative uncertainty. We assumed 99 that the relative uncertainty follows a single-sided normal distribution with a mean of 1.0 and a 100 standard deviation of 10%. This implies that all reported cases are positive but there is a tendency 101 to miss-reporting some positive cases so that the reported numbers represent a lower limit. The 102 resulting mean and 95% confidence interval (CI) were presented.', 'cite_spans': [{'start': 226, 'end': 228, 'text': '95', 'ref_id': None}], 'ref_spans': [], 'section': '91'}, {'text': 'In light of daily reported cases of SARS-CoV-2 since Jan 21, 2020, we decided to collect data for 107 analysis on the cumulative number of confirmed cases (initially from Jan 21 to Feb 10, 2020) in from Jan 21 to Feb 14 for Hubei Province by a fixed factor (refer to Table S1), assuming that 121 these newly added cases were linearly accumulative in those days. It is the same forth with the 122 data for Wuhan City.', 'cite_spans': [], 'ref_spans': [], 'section': '106'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '123'}, {'text': 'The copyright holder for this preprint . https://doi.org/10. 1101 Fitting data on the confirmed cases of SARS-CoV-2 to Boltzmann function and estimating 124 the potential total number of confirmed cases 125 Fitting analyses using Boltzmann function indicate that all sets of data were well fitted with the 126 function (all R 2 values being close to 0.999; Figs. 1, 2 and S1). Parameter A2 in the Boltzmann 127 function directly represents the potential total number of confirmed cases (refer to equation 1). 128 As summarized in Table 1 , the potential total number of confirmed cases for mainland China, 129 Hubei Province, Wuhan City, and other provinces were estimated as 72800±600, 59300±600, 130 42100±700 and 12800±100; respectively (also refer to Fig. 1) ; those for the six mostly influenced 131 provinces (Guangdong, Zhejiang, Henan, Hunan, Anhui and Jiangxi) were 1300±10, 1170±10, 132 1260±10, 1050±10, 1020±10 and 940±10, respectively (also refer to Fig. 2) ; those for the top-4 133 major cities (Beijing, Shanghai, Guangzhou and Shenzhen) were 394±4, 328±3, 337±3 and 397±4, 134 respectively (also refer to Fig. S1 ).', 'cite_spans': [{'start': 61, 'end': 65, 'text': '1101', 'ref_id': None}, {'start': 203, 'end': 206, 'text': '125', 'ref_id': None}, {'start': 509, 'end': 512, 'text': '128', 'ref_id': None}, {'start': 599, 'end': 605, 'text': 'China,', 'ref_id': None}, {'start': 606, 'end': 609, 'text': '129', 'ref_id': None}], 'ref_spans': [{'start': 530, 'end': 537, 'text': 'Table 1', 'ref_id': 'TABREF0'}, {'start': 755, 'end': 762, 'text': 'Fig. 1)', 'ref_id': 'FIGREF11'}, {'start': 963, 'end': 970, 'text': 'Fig. 2)', 'ref_id': 'FIGREF13'}, {'start': 1122, 'end': 1129, 'text': 'Fig. S1', 'ref_id': 'FIGREF11'}], 'section': '(which was not peer-reviewed)'}, {'text': 'In addition, we estimated the key date, on which the number of daily new confirmed cases is 137 lower than 0.1% of the potential total number of confirmed cases as defined by us subjectively.', 'cite_spans': [], 'ref_spans': [], 'section': '136'}, {'text': 'As summarized in Table 1 , the key dates for mainland China, Hubei Province, Wuhan City and 139 other provinces are Feb28 or Feb 27. It appears that it will take approximately two weeks for 140 mainland China to reach this state such that the number of daily new confirmed cases of SARS-', 'cite_spans': [], 'ref_spans': [{'start': 17, 'end': 24, 'text': 'Table 1', 'ref_id': 'TABREF0'}], 'section': '138'}, {'text': 'CoV-2 post the critical date is below 70.', 'cite_spans': [], 'ref_spans': [], 'section': '141'}, {'text': 'The above analyses were performed assuming that the released data on the confirmed cases are 144 precise. However, there is a tendency to miss-report some positive cases such that the reported 145 numbers represent a lower limit. One typical example indicating this uncertainty is the sudden 146 increase of more than 13 000 new confirmed cases in Hubei province on Feb 12 after clinical 147 features were officially accepted as a standard for infection confirmation. Another uncertainty 148 might result from insufficient kits for viral nucleic acid detection at the early stage of the outbreak. 149 We thus examined the effects of the uncertainty of the released data on the estimation of the 150 potential total number of confirmed cases using a Monte Carlo method (for detail, refer to the 151 Methods section). For simplicity, we assumed that the relative uncertainty of the reported data 152 follows a single-sided normal distribution with a mean of 1.0 and a standard deviation of 10%.', 'cite_spans': [{'start': 597, 'end': 600, 'text': '149', 'ref_id': None}], 'ref_spans': [], 'section': '143'}, {'text': 'Under the above conditions, the potential total numbers of confirmed cases of SARS-CoV-2 for 155 different regions were estimated (Figs. 3, S2 and S3) and summarized in Table 1 ', 'cite_spans': [], 'ref_spans': [{'start': 169, 'end': 176, 'text': 'Table 1', 'ref_id': 'TABREF0'}], 'section': '154'}, {'text': 'Such uncertainty analysis also allowed us to estimate the key dates at 95% CI. As summarized in 165 Table 1 , the key dates for mainland China, Hubei Province, Wuhan City, and other provinces 166 would fall in (2/28, 3/10), (2/27, 3/10), (2/28, 3/10) and (2/27, 3/13), respectively (also refer to ', 'cite_spans': [], 'ref_spans': [{'start': 100, 'end': 107, 'text': 'Table 1', 'ref_id': 'TABREF0'}], 'section': '164'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '172'}, {'text': 'The copyright holder for this preprint . https://doi.org/10. 1101 Data on the confirmed cases of 2013 SARS-CoV were well fitted to Boltzmann function 173 The ongoing SARS-CoV-2 outbreak has undoubtedly caused the memories of the SARS-CoV (Figs. 1, 2 and S1 ). More importantly, the data of 2003 SARS-CoV in China 203 and worldwide were also well fitted to the function (Fig. 4) . These results, in conjunction with 204 that Boltzmann function can be inferred from a few assumptions (for detail, refer to the Methods 205 section of the support information file), suggest that Boltzmann function is suitable for analyzing 206 the epidemics of coronaviruses like SARS-CoV and SARS-CoV-2. One advantage of this model 207 is that parameter A2 directly gives an estimate of the potential total numbers of confirmed cases.', 'cite_spans': [{'start': 61, 'end': 65, 'text': '1101', 'ref_id': None}, {'start': 150, 'end': 153, 'text': '173', 'ref_id': None}], 'ref_spans': [{'start': 238, 'end': 256, 'text': '(Figs. 1, 2 and S1', 'ref_id': 'FIGREF11'}, {'start': 369, 'end': 377, 'text': '(Fig. 4)', 'ref_id': 'FIGREF15'}], 'section': '(which was not peer-reviewed)'}, {'text': 'In addition, unlike traditional epidemiological models that require much more detailed data for indicating that the SARS-CoV-2 outbreak in China might not be as bad as thought. Notably, our 220 results also suggest that the number of daily new confirmed cases will become minimal between 221 Feb 28 and Mar 10 in mainland China (including Hubei Province) at 95% CI (Fig. 3A) . This trend, 222 if occur as predicted, may help citizens in China to release stress and anxiety, as there have been 223 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 389, 'end': 392, 'text': '222', 'ref_id': None}], 'ref_spans': [{'start': 365, 'end': 374, 'text': '(Fig. 3A)', 'ref_id': 'FIGREF14'}], 'section': '208'}, {'text': 'The copyright holder for this preprint . https://doi.org/10.1101/2020.02.16.20023564 doi: medRxiv preprint many provinces and/cities in China that have suspended public transportation systems and even 224 implemented house quarantines for all urban households [20] . In further support of these 225 estimates by the Boltzmann function, the newly released cumulative number of confirmed cases 226 in all the above geographic regions on Feb 15 and Feb 16 are very close to the predicted ones 227 (refer to Table S2 ). Consistently, parameters of the established Boltzmann functions by 228 regression analyses of the data from Jan 21 to Feb 16, 2020 (as presented in Fig. S4 ) are highly 229 similar to those made by the data from Feb 21 to Feb 14, 2020.', 'cite_spans': [{'start': 260, 'end': 264, 'text': '[20]', 'ref_id': None}], 'ref_spans': [{'start': 501, 'end': 512, 'text': 'to Table S2', 'ref_id': None}, {'start': 664, 'end': 671, 'text': 'Fig. S4', 'ref_id': 'FIGREF15'}], 'section': '(which was not peer-reviewed)'}, {'text': 'Nevertheless, our estimates based on the established Boltzmann functions are not absolutely 232 guaranteed, mainly because of the uncertainty of the reported data (Figs. 3, S2 and S3) . We 233 estimated the potential total numbers (refer to Table 1) under the assumption that the relative 234 uncertainty of the reported data follows a single-sided normal distribution with a mean of 1.0 and 235 a standard deviation of 10%, and this deviation may be underestimated. If the real uncertainty of 236 released data by health commissions is larger than 10%, the potential total numbers of confirmed 237 cases would accordingly increase, and the key dates will be postponed. Another limitation is that 238 this estimate is based on the assumption that the overall conditions are not changing. This might 239 not be true, given that in many regions the workers have started to return for work half a month ', 'cite_spans': [], 'ref_spans': [{'start': 163, 'end': 183, 'text': '(Figs. 3, S2 and S3)', 'ref_id': 'FIGREF13'}], 'section': '231'}, {'text': 'Ethical approval or individual consent was not applicable. 256', 'cite_spans': [], 'ref_spans': [], 'section': 'Ethics approval and consent to participate 255'}, {'text': 'All data and materials used in this work were publicly available, and also available based on request. 258', 'cite_spans': [], 'ref_spans': [], 'section': 'Availability of data and materials 257'}, {'text': 'The funding agencies had no role in the design and conduct of the study; collection, management, analysis, 260 and interpretation of the data; preparation, review, or approval of the manuscript; or decision to submit the 261 manuscript for publication. 262', 'cite_spans': [], 'ref_spans': [], 'section': 'Disclaimer 259'}, {'text': 'The authors declare that they have no conflict of interest. . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conflict of Interests 263'}, {'text': 'The copyright holder for this preprint . https://doi.org/10. 1101 cases of Hubei Province and Wuhan City were re-adjusted for data fitting due to the suddenly added cased 333 determined by clinical features (for detail, refer to the Results section and Table S1 ). 334 335 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. ', 'cite_spans': [{'start': 61, 'end': 65, 'text': '1101', 'ref_id': None}], 'ref_spans': [{'start': 253, 'end': 261, 'text': 'Table S1', 'ref_id': 'TABREF0'}], 'section': '(which was not peer-reviewed)'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. Figure 4 A B', 'cite_spans': [], 'ref_spans': [{'start': 164, 'end': 172, 'text': 'Figure 4', 'ref_id': 'FIGREF15'}], 'section': '356'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '356'}, {'text': '(which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10. 1101 ', 'cite_spans': [{'start': 91, 'end': 95, 'text': '1101', 'ref_id': None}], 'ref_spans': [], 'section': '356'}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'A novel coronavirus from patients with pneumonia in China', 'authors': [{'first': 'N', 'middle': [], 'last': 'Zhu', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Wang', 'suffix': ''}], 'year': None, 'venue': 'N Engl J Med', 'volume': '274', 'issn': '2020', 'pages': '', 'other_ids': {'DOI': ['10.1056/NEJMoa2001017']}}, 'BIBREF1': {'ref_id': 'b1', 'title': 'What to do next to control the 2019-nCoV epidemic?', 'authors': [{'first': 'F', 'middle': ['S'], 'last': 'Wang', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Zhang', 'suffix': ''}], 'year': 2020, 'venue': 'Lancet', 'volume': '395', 'issn': '', 'pages': '391--392', 'other_ids': {}}, 'BIBREF2': {'ref_id': 'b2', 'title': 'The Novel Coronavirus Originating in Wuhan, China: Challenges for 277 Global Health Governance', 'authors': [{'first': 'A', 'middle': ['L'], 'last': 'Phelan', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Katz', 'suffix': ''}, {'first': 'L', 'middle': ['O'], 'last': 'Gostin', 'suffix': ''}], 'year': 2020, 'venue': 'JAMA', 'volume': '278', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1001/jama.2020.1097']}}, 'BIBREF3': {'ref_id': 'b3', 'title': 'A novel coronavirus outbreak of global health concern', 'authors': [{'first': 'C', 'middle': [], 'last': 'Wang', 'suffix': ''}], 'year': 2020, 'venue': 'Lancet', 'volume': '', 'issn': '20', 'pages': '30185--30194', 'other_ids': {'DOI': ['10.1016/S0140-2796736']}}, 'BIBREF4': {'ref_id': 'b4', 'title': 'What next for the coronavirus response? Lancet, 2020', 'authors': [{'first': 'J', 'middle': [], 'last': 'Zarocostas', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '395', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF5': {'ref_id': 'b5', 'title': 'Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus 282 origins and receptor binding', 'authors': [{'first': 'R', 'middle': [], 'last': 'Lu', 'suffix': ''}], 'year': 2020, 'venue': 'Lancet', 'volume': '', 'issn': '', 'pages': '30251--30259', 'other_ids': {'DOI': ['10.1016/s0140-6736(20']}}, 'BIBREF6': {'ref_id': 'b6', 'title': 'A new coronavirus associated with human respiratory disease in China', 'authors': [{'first': 'F', 'middle': ['E'], 'last': 'Wu', 'suffix': ''}], 'year': 2020, 'venue': 'Nature', 'volume': '284', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1038/s41586-020-2008-3']}}, 'BIBREF7': {'ref_id': 'b7', 'title': 'A pneumonia outbreak associated with a new coronavirus of probable bat origin', 'authors': [{'first': 'P', 'middle': ['E'], 'last': 'Zhou', 'suffix': ''}], 'year': 2020, 'venue': 'Nature', 'volume': '286', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1038/s41586-020-2012-7']}}, 'BIBREF8': {'ref_id': 'b8', 'title': 'A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating 288 person-to-person transmission: a study of a family cluster', 'authors': [{'first': 'J', 'middle': ['F'], 'last': 'Chan', 'suffix': ''}], 'year': 2020, 'venue': 'Lancet', 'volume': '290', 'issn': '20', 'pages': '', 'other_ids': {'DOI': ['10.1016/S0140-2896736']}}, 'BIBREF9': {'ref_id': 'b9', 'title': 'Epidemiological and clinical characteristics of 99 cases of 2019 novel coronavirus 291 pneumonia in Wuhan, China: a descriptive study', 'authors': [{'first': 'N', 'middle': [], 'last': 'Chen', 'suffix': ''}], 'year': 2020, 'venue': 'Lancet', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1016/S0140-6736(20)30211-7.29211']}}, 'BIBREF10': {'ref_id': 'b10', 'title': 'Clinical characteristics of 2019 novel coronavirus infection in China', 'authors': [{'first': 'W', 'middle': ['J'], 'last': 'Guan', 'suffix': ''}, {'first': 'E', 'middle': ['A'], 'last': '', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '293', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.02.06.20020974']}}, 'BIBREF11': {'ref_id': 'b11', 'title': 'Clinical features of patients infected with 2019 novel coronavirus in Wuhan', 'authors': [{'first': 'C', 'middle': [], 'last': 'Huang', 'suffix': ''}], 'year': None, 'venue': 'China. Lancet', 'volume': '295', 'issn': '2020', 'pages': '', 'other_ids': {'DOI': ['10.1016/S0140-6736(20)30183-5']}}, 'BIBREF12': {'ref_id': 'b12', 'title': 'Early Transmission Dynamics in Wuhan, China, of Novel Coronavirus-Infected Pneumonia', 'authors': [{'first': 'Q', 'middle': [], 'last': 'Li', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF14': {'ref_id': 'b14', 'title': 'Epidemiological and clinical features of the 2019 novel coronavirus outbreak in China', 'authors': [{'first': 'Y', 'middle': [], 'last': 'Yang', 'suffix': ''}, {'first': 'E', 'middle': ['A'], 'last': '', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '299', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.02.10.20021675']}}, 'BIBREF15': {'ref_id': 'b15', 'title': 'Preliminary estimation of the basic reproduction number of novel coronavirus (2019-nCoV) 301 in China, from 2019 to 2020: A data-driven analysis in the early phase of the outbreak', 'authors': [{'first': 'S', 'middle': [], 'last': 'Zhao', 'suffix': ''}], 'year': 2020, 'venue': 'Int J Infect Dis', 'volume': '303', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1016/j.ijid.2020.01.050']}}, 'BIBREF16': {'ref_id': 'b16', 'title': 'A data driven time-dependent transmission rate for tracking an epidemic: a case 304 study of 2019-nCoV', 'authors': [{'first': 'N', 'middle': ['E'], 'last': 'Huang', 'suffix': ''}, {'first': 'F', 'middle': [], 'last': 'Qiao', 'suffix': ''}], 'year': 2020, 'venue': 'Science Bulletin', 'volume': '305', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1016/j.scib.2020.02.005']}}, 'BIBREF17': {'ref_id': 'b17', 'title': 'DATA-BASED ANALYSIS, MODELLING AND FORECASTING OF THE NOVEL 306 CORONAVIRUS (2019-NCOV) OUTBREAK', 'authors': [{'first': 'C', 'middle': ['E'], 'last': 'Anastassopoulou', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '307', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.02.11.20022186']}}, 'BIBREF18': {'ref_id': 'b18', 'title': 'Estimating the potential total number of novel Coronavirus (2019-308 nCoV) cases in Wuhan City', 'authors': [{'first': 'N', 'middle': [], 'last': 'Imai', 'suffix': ''}, {'first': 'D', 'middle': ['I'], 'last': 'Cori', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Riley', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Ferguson', 'suffix': ''}, {'first': 'N', 'middle': ['M'], 'last': '', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '309', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF19': {'ref_id': 'b19', 'title': 'Nowcasting and forecasting the potential domestic and international 310 spread of the 2019-nCoV outbreak originating in Wuhan, China: a modelling study', 'authors': [{'first': 'J', 'middle': ['T'], 'last': 'Wu', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Leung', 'suffix': ''}, {'first': 'G', 'middle': ['M'], 'last': 'Leung', 'suffix': ''}], 'year': 2020, 'venue': 'Lancet', 'volume': '', 'issn': '', 'pages': '30260--30269', 'other_ids': {'DOI': ['31110.1016/S0140-6736(20']}}, 'BIBREF20': {'ref_id': 'b20', 'title': 'Offline: 2019-nCoV-\"A desperate plea', 'authors': [{'first': 'R', 'middle': [], 'last': 'Horton', 'suffix': ''}], 'year': 2020, 'venue': 'Lancet', 'volume': '395', 'issn': '', 'pages': '', 'other_ids': {}}}, 'ref_entries': {'FIGREF0': {'text': 'of atypical pneumonia caused by the zoonotic 2019 novel coronavirus (SARS-CoV-47 2) is on-going in China [1]. As of Feb 12, 2020 (24:00, GMT+8), there have been 59901 48 confirmed patients and 1368 deaths from SARS-CoV-2 infection, in China, and the most affected 49 city, Wuhan, and related regions in Hubei province of China have reported 48206 confirmed 50 patients and 1310 deaths. Cases infected in Wuhan were also detected in other provinces of China 51', 'latex': None, 'type': 'figure'}, 'FIGREF1': {'text': '64 SARS-CoV-2 in China. Results indicate that the daily reported cumulative number of confirmed of health commissions of provinces, municipalities and major cities. Overseas data were not 76 included in our simulation due to the small number of confirmed cases. The cumulative number 77 of confirmed cases of 2003 SARS in China and worldwide were obtained from the official website 78 of WHO.79', 'latex': None, 'type': 'figure'}, 'FIGREF2': {'text': '108 several typical regions of China, including the center of the outbreak (i.e. Wuhan City and Hubei 109 Province), other top provinces ranking in the number of cases (i.e., Guangdong, Zhejiang, Henan, 110 Hunan, Anhui, and Jiang Provinces) and top-4 major cities in China (i.e., Beijing, Shanghai, 111 Guangzhou, Shenzhen).', 'latex': None, 'type': 'figure'}, 'FIGREF3': {'text': 'analysis on Feb 13, 2020, the number of new confirmed cases on Feb 12 in Hubei 114 Province and Wuhan City suddenly increased by 14840 and 13436, respectively, of which 13332 115 and 12364 are those confirmed by clinical features (note: all the number of confirmed cases 116 released by Feb 12 were counted according to the result of viral nucleic acid detection rather than 117 by referring to clinical features). Afterward, new confirmed cases determined by clinical criteria 118 in Hubei Province on Feb 13 and 4 are still up to 2052 and 1138. In view of these, we arbitrarily 119 distributed these suddenly added cases to the reported cumulative number of confirmed cases 120', 'latex': None, 'type': 'figure'}, 'FIGREF4': {'text': '. The potential 156 total numbers for China, Hubei Province, Wuhan City and other provinces were 79589 (95% CI 157 71576, 93855), 64817 (58223, 77895), 46562 (40812, 57678) and 13956 (12748, 16092), 158 respectively (also refer to Fig. 3); those for the six most affected provinces (Guangdong, Zhejiang, 159 Henan, Hunan, Anhui and Jiangxi) were 1415 (1324, 1550), 1269 (1204, 1364), 1372 (1271, 160 1559), 1140 (1050, 1279), 1110 (1021, 1273) and 1024 (948, 1146), respectively (also refer to 161 Fig. S2); those for top-4 major cities (Beijing, Shanghai, Guangzhou and Shenzhen) were 429 162 (395, 486), 356 (334, 388), 365 (346, 393) and 430 (407, 461), respectively (also refer to Fig. S3).', 'latex': None, 'type': 'figure'}, 'FIGREF5': {'text': '; those for the six provinces (Guangdong, Zhejiang, Henan, Hunan, Anhui and Jiangxi) also refer toFig. S2); those for top-4 major cities (Beijing, Shanghai, Guangzhou 170 and Shenzhen) were within (2/25, 3/11), (2/22, 3/01), (2/20, 2/28) and (2/17, 2/25), respectively 171 (also refer toFig. S3).', 'latex': None, 'type': 'figure'}, 'FIGREF6': {'text': 'were well fitted to Boltzmann function, we thus applied this function to fit the epidemic of 176 2003 SARS-CoV in China and worldwide. Results in Fig. 4 show that the cumulative numbers 177 of confirmed cases of 2003 SARS-CoV both in China and worldwide were fitted well with the 178 Boltzmann distribution function, with R 2 being 0.999 and 0.998, respectively. of significance to estimate the spread of the SARS-CoV-2 outbreak originating in Wuhan, 182 China. In particular, a precise estimation of the potential total number of infected cases and/or 183 confirmed cases is critical for optimal preparedness and implementation of infection control 184 practices made by governments nationally and locally. An earlier study published on 17 January 185 2020 [18] suggested that a total of 1,723 cases of SARS-CoV-2 in Wuhan City (95% CI: 427-186 4,471) had onset of symptoms by 12th January 2020, which was calculated based on the 187 internationally reported cases. Later, Wu et al [19] applied a susceptible-exposed-infectious-188 recovered (SEIR) metapopulation model to simulate the epidemics of the SARS-CoV-2 outbreak 189 across all major cities in China on the basis of the number of cases exported from Wuhan to cities 190 outside mainland China, and they estimated that 75 815 individuals (95% CI 37 304-130 330) 191 had been infected in Wuhan as of Jan 25, 2020. A recent study based on the Susceptible-Infected-192 Recovered-Dead (SIRD) model revealed that the cumulative number of infected cases will 193 surpass 68,000 (as a lower bound) and could reach 140,000 (with an upper bound of 290,000) by 194 February 29 [17]. Notably, Huang and Qiao proposed a simple and data driven model based 195 on the physics of natural growth algorithm and estimated the transmission rate of the SARS-196 CoV-2 outbreak [16]. The results of these modeling studies have provided important guidance 197for governments and health agencies to intensify preparedness and response efforts.', 'latex': None, 'type': 'figure'}, 'FIGREF7': {'text': 'applied Boltzmann function to analyze the reported confirmed cases in mainland China, 200 focusing Hubei Province and Wuhan City, the center of the outbreak, as well as those most 201 affected provinces and economic centers. Strikingly, all data sets were well fitted to the 202 Boltzmann function', 'latex': None, 'type': 'figure'}, 'FIGREF8': {'text': '209 analysis [14, 19], Boltzmann function-based regression analysis only needs the cumulative 210 number of confirmed cases, somehow as simple as the model proposed by Huang and Qiao [16]. 211 212 Furthermore, the established Boltzmann functions allow us to forecast the future course of the 213 epidemics of SARS-CoV-2 in different regions in the coming weeks. The potential total numbers 214 of confirmed cases in mainland China, in Hubei Province, in Wuhan City (the center of the 215 outbreak) and in other most affected provinces and top4 major cities, as summarized in', 'latex': None, 'type': 'figure'}, 'FIGREF9': {'text': \"240 post the Spring Festival holiday (schedulely ending on Feb 31), which may increase the SARS-241 CoV-2 infection. In this regard, it is noted that the daily number of new confirmed cases in past 242 a few days in several provinces and cities (e.g., Guangdong Province, Fig. 2A; Shanghai and 243 Shenzhen City, Fig. S1D) have increased a little bit more than predicted by the model. 244 245 Acknowledgments 246 We thank graduate students (Boyan Lv, Zhongyan Li, Zhongyu Chen, Yu Cheng, Mengmeng 247 Bian, Shuang Zhang, Zuqin Zhang, and Wei Yao; all from Prof. Xinmiao Fu's research group at 248 Fujian Normal University) for data collection from official websites of National and Health 249 Commission of China and of provincial health commissions in China. The authors would like to 250 acknowledge colleagues for helpful comments. This work is support by the National Natural 251 Science Foundation of China (No. 31972918 and 31770830 to XF).\", 'latex': None, 'type': 'figure'}, 'FIGREF10': {'text': 'and design: Xinmiao Fu 266 (II) Derivation of the Boltzmann function: Tieyong Zeng 267 (III) Data simulation and estimation: Xinmiao Fu 268 (IV) Uncertainty analyses: Qi Ying 269 (V) Data collection and organization: Tao Long and Yan Wang 270', 'latex': None, 'type': 'figure'}, 'FIGREF11': {'text': 'Fitting the cumulative number of confirmed cases from mainland China to 327 Boltzmann function 328 Cumulative number of confirmed cases of SARS-CoV-2 as of Feb 14, 2020, in mainland China (panel A), 329 in Hubei Province (panel B), in Wuhan City (panel C) and in other provinces (panel D) are shown as black 330 squares, and the simulation results from Boltzmann function are plotted as red short lines and parameters 331 of each established function are shown in inserts. Note: the reported cumulative number of confirmed 332', 'latex': None, 'type': 'figure'}, 'FIGREF12': {'text': 'Figure 1', 'latex': None, 'type': 'figure'}, 'FIGREF13': {'text': 'Fitting the cumulative number of confirmed cases from the six most affected 338 provinces to Boltzmann function 339 Cumulative number of confirmed cases of SARS-CoV-2 as of Feb 14, 2020, in provinces Guangdong 340 (panel A), in Zhejiang (panel B), in Henan (panel C), in Hunan (panel D), in Anhui (panel E) and in Jiangxi 341 (panel F) are shown as black squares, and the simulation results from Boltzmann function are plotted as 342 red short lines and parameters of each established function are shown in inserts.', 'latex': None, 'type': 'figure'}, 'FIGREF14': {'text': 'Analysis of the uncertainty of confirmed cases for mainland China, Hubei 346 Province, Wuhan City and other provinces. 347 Data of mainland China (panel A), of Hubei Province (panel B), of Wuhan City (panel C) and of 348 other provinces (panel D) were fitted to Boltzmann function assuming that the relative uncertainty 349 of the data follows a single-sided normal distribution with a mean of 1.0 and a standard deviation 350 of 10%. Original data are shown as circles; simulated results are presented as colored lines as 351 indicated. Inserts show key statistics. The key date is defined as the date when the number of daily 352 new confirmed cases is less than 0.1% of the potential total number. The low and high key dates 353 were determined by the simulated curve of confidence interval (CI) at 2.5% and 97.5%, 354 respectively.', 'latex': None, 'type': 'figure'}, 'FIGREF15': {'text': 'Fitting the cumulative number of confirmed cases for 2003 SARS in China and 358 worldwide to Boltzmann function 359 The cumulative number of confirmed cases of 2003 SARS in China (panel A) and worldwide (panel B) 360 are shown as black squares, and the simulation results from Boltzmann function are plotted as red short 361 lines and parameters of each established function are shown in inserts.', 'latex': None, 'type': 'figure'}, 'TABREF0': {'text': '216may provide valuable guidance for Chinese central and local governments to contain this outbreak at current critical stage. In particular, the numbers in mainland China and Hubei Province were', 'latex': None, 'type': 'table'}, 'TABREF2': {'text': 'Regression analysis results of confirmed cases of SARS-CoV-2 in China', 'latex': None, 'type': 'table'}}, 'back_matter': []}\n",
      "{'paper_id': 'b12f20ae397aa504f852fc25caeb3e3b0e5b1942', 'metadata': {'title': 'Advance of Novel Coronavirus Registration Clinical Trial', 'authors': [{'first': 'Gao', 'middle': [], 'last': 'Song', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': \"The Puer People's Hospital of Yunna City\", 'location': {'addrLine': '44 Zhenxing Rd', 'postCode': '665000', 'settlement': 'Puer', 'country': 'China'}}, 'email': ''}, {'first': 'Meng', 'middle': ['Qun'], 'last': 'Cheng', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': \"The Puer People's Hospital of Yunna City\", 'location': {'postCode': '665000', 'settlement': 'Puer', 'country': 'China'}}, 'email': ''}, {'first': 'Xian', 'middle': ['Wen'], 'last': 'Wei', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': \"The Puer People's Hospital of Yunna City\", 'location': {'addrLine': '44 Zhenxing Rd', 'postCode': '665000, 665000', 'settlement': 'Puer, Puer', 'country': 'China;, China'}}, 'email': ''}]}, 'abstract': [{'text': 'Coronavirus(COVID-19) registered in the China Clinical Trial Registry (ChiCTR), and provide data bases and information references for clinical treatment', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}], 'body_text': [{'text': \"Since the 1970s, more than 30 new types of infectious diseases, such as Ebola virus disease, human infection with highly pathogenic avian influenza, and Zika virus disease, have appeared in China, seriously threatening people's health and public health security [1, 2] . The Therefore, this study collates and analyzes the registration of COVID-19 clinical studies to provide an information-based data foundation and information reference for curbing the spread of COVID-19 epidemic.\", 'cite_spans': [{'start': 262, 'end': 265, 'text': '[1,', 'ref_id': None}, {'start': 266, 'end': 268, 'text': '2]', 'ref_id': None}], 'ref_spans': [], 'section': 'Background'}, {'text': 'Check the \"COVID-19 Clinical Research Index\" issued by the ChiCTR Center, search the ChiCTR database with the keywords \"Novel coronavirus\" or \"2019-nCoV\" and count the registration information as of February 24, 2020. Extract the following information: (1) Registration status (date of registration, date of completion, region of registration, etc.). author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Data collection'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi. org/10.1101 org/10. /2020 dataset.There was no disagreement between the two reviewers about the dataset with 232 studies.', 'cite_spans': [{'start': 91, 'end': 102, 'text': 'org/10.1101', 'ref_id': None}, {'start': 103, 'end': 116, 'text': 'org/10. /2020', 'ref_id': None}], 'ref_spans': [], 'section': 'Data collection'}, {'text': 'Basic research characteristics: research type, intervention method, funding source and research stage. It is defined as: (1) ', 'cite_spans': [], 'ref_spans': [], 'section': 'Definitions'}, {'text': 'Descriptive analysis of registration time trends and geographical distribution, the registration volume reflects the overall trend of COVID-19 clinical research. Descriptive statistics were calculated to investigate the percentage distribution of all of the items studied.The frequency calculation of multi-center research institutions is based on the statistics of the responsible units. The study duration was calculated by ChiCTR\\'s \"Study execute time\".Chi-square tests at the 95% significance level were then performed to study possible differences in design characteristics for research type, intervention method, funding source and research stage. In the chisquare test, we only statistically analyze the main features, excluding some feature analyses. Additionally, as 100.0% of the 232 studies had key inclusion/exclusion criteria, these items were ruled out in the chisquare tests. Statistical analysis was performed using SPSS22.0 software. P<0.05 was considered statistically significant.', 'cite_spans': [], 'ref_spans': [], 'section': 'Data analysis'}, {'text': 'All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Results'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https: //doi.org/10.1101 //doi.org/10. /2020 ', 'cite_spans': [{'start': 85, 'end': 102, 'text': '//doi.org/10.1101', 'ref_id': None}, {'start': 103, 'end': 122, 'text': '//doi.org/10. /2020', 'ref_id': None}], 'ref_spans': [], 'section': 'Results'}, {'text': \"A total of 232 COVID-19 clinical studies were registered with ChiCTR. Overall, the number of registrations has increased steadily ( Figure 1 ).The number of registrations before February 1 was small, and the number of registrations quickly increased to 8 on February 2, reaching a peak on February 14(n=18).On January 23, 2020, China's ChiCTR Center registered the first COVID-19 clinical study, and the intervention method was CMT treatment (ChiCTR2000029308).The first TCM clinical study was registered on January 27 (ChiCTR2000029381). The first BI study was registered on February 1 (ChiCTR2000029431). The BEI study was first registered on February 2.\", 'cite_spans': [], 'ref_spans': [{'start': 132, 'end': 140, 'text': 'Figure 1', 'ref_id': None}], 'section': 'COVID-19 clinical study registration'}, {'text': 'As of the statistical date, a total of 18 provinces and 4 municipalities nationwide participated in the COVID-19 clinical study ( Figure 2 ).The national registration volume ranked first in Hubei Province (n=69, 29.74%), followed by Guangdong Province (n=29, 12.50%), and Zhejiang Province and Sichuan Province tied for third place (n=23, 9.91%).Tongji Hospital, Tongji Medical College, Huazhong University of Science and Technology is the most registered hospital(n=16).', 'cite_spans': [], 'ref_spans': [{'start': 130, 'end': 138, 'text': 'Figure 2', 'ref_id': None}], 'section': 'COVID-19 clinical study registration'}, {'text': 'As shown in Table 1 , 67.67% of the studies were intervention studies and 25.00% were observational studies. 28.02% of the studies were treated with CMT, 12.93% were treated with BI, 28.02% were treated with TCM, and 8.62% were treated with BEI.The early stage accounted for 41.81% of all studies, and 20.69% of the studies were late stages. 53.88% of research is mainly funded by the public, and 57.76% of research is being recruited. However, 22.84% of the studies did not receive ethical approval, and 47.41% were not confirmed by the data management committee. Table 2 , only 6.47% of the studies were multicenter studies. 44.83% of the study participants were between 100 to 300, and only 3.45% of the studies were >1000. 43.53% of the studies were completed within 4-12 months, and only 2.16% were completed within 1 month. 75.0% of the studies used random assignments, but 70.69% of the studies lacked information on blinding. Table 3 ,the chisquare tests revealed significant differences in the design characteristics, depending on the Type of Study, for the items of participants number(P=0.000), study duration (P=0.008), study assignment (P=0.000) and blind method (P=0.000). The number of participants in 45.86% interventional and 43.10% observational studies was concentrated in the 100-300 range.46.50% interventional and 37.93% observational studies were completed within 4-12 months. 78.98% of the interventional All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [{'start': 12, 'end': 19, 'text': 'Table 1', 'ref_id': 'TABREF4'}, {'start': 565, 'end': 572, 'text': 'Table 2', 'ref_id': 'TABREF5'}, {'start': 934, 'end': 941, 'text': 'Table 3', 'ref_id': 'TABREF6'}], 'section': 'Heterogeneity analysis of study design features'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10. 1101 /2020 studies were randomly assigned to the group, and 58.62% of the observations were continuous. Only 7.01% of interventional studies were blinded.', 'cite_spans': [{'start': 98, 'end': 102, 'text': '1101', 'ref_id': None}, {'start': 103, 'end': 108, 'text': '/2020', 'ref_id': None}], 'ref_spans': [], 'section': 'As shown in'}, {'text': 'Significant differences could be seen in the dimensions of multicenter study (P=0.022), of participants numbe (P=0.000), study duration(P=0.000) and study assignment(P=0.001) for the four intervention methods.Only 1.54% of CMT studies are multi-center studies.', 'cite_spans': [], 'ref_spans': [], 'section': 'As shown in'}, {'text': 'Conversely, 20.00% of BEI studies are multi-center studies, accounting for the largest proportion.The number of participants mainly accounted for: 44.62% of CMT subjects were less than 100, 80.00% of BI subjects were less than 100, and 58.46% of TCM and 65.00% of BEI subjects were between 100 to 300. The study period mainly focused on: 53.85% of the CMT studies were completed within 4 to 12 months, while 40.00% of the BIs were within 1 to 3, 49.23% of the TCMs were within 4 to 12, and 35.00% of the BEI studies were longer than 12 month.For study assignments, 76.92% of CMT, 70.00% of BI, 66.15% of TCM, and 40.00% of BEI studies were randomized. Only 7.69% of CMT, 13.34% of BI, and 3.08% of TCM were blinded in the study.', 'cite_spans': [], 'ref_spans': [], 'section': 'As shown in'}, {'text': 'There were significant differences in the dimensions of study assignments (P=0.043) in the early and late study stage.60.82% of the early stages and 26.21% of the late stages were randomly assigned to the group. However, there were no significant differences in the main research dimensions of different funding sources (P>0.05).', 'cite_spans': [], 'ref_spans': [], 'section': 'As shown in'}, {'text': 'All drugs involved in CMT and BI treatment research are summarized, and the frequency of research is recorded (the same drug is only recorded once in the same clinical trial).68.97% of interventions were drug treatment (n=160). In the CMT study, the top three drugs studied by frequency: Chloroquine (n=16, 16.84%), Lopinavir/ritonavir (n=15, 15.79%), and I-IFN (n=11, 11.58%).', 'cite_spans': [], 'ref_spans': [], 'section': 'Analysis of the characteristics of CMT and BI research'}, {'text': 'In the BI study, the top three drugs in the study frequency were: cell therapy (n=15, 48.39%), plasma therapy (n=5, 16.13%), Thymosin (n=2, 6.45%) and M/P-AB (n=2, 6.45%).', 'cite_spans': [], 'ref_spans': [], 'section': 'Analysis of the characteristics of CMT and BI research'}, {'text': 'Among them, 32.26% of Cell therapy is mesenchymal stem cells, 9.68% of Cell therapy is Cord Blood Mononuclear Cell, 3.23% of Cell therapy is Menstrual Blood-derived Stem Cells, and 3.23% of Cell therapy is Natural Killer Cell. Figure 3、4 and Table 4 for a summary of the high research frequency drugs for CMT and BI.', 'cite_spans': [], 'ref_spans': [{'start': 227, 'end': 237, 'text': 'Figure 3、4', 'ref_id': None}, {'start': 242, 'end': 249, 'text': 'Table 4', 'ref_id': 'TABREF7'}], 'section': 'Analysis of the characteristics of CMT and BI research'}, {'text': 'Achieving information sharing in clinical trials is the key to accelerating the transformation of clinical resources and promoting scientific breakthroughs [7] . In the face of a public safety crisis such as an outbreak, the timely sharing of information is even more important, and it All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 156, 'end': 159, 'text': '[7]', 'ref_id': None}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10. 1101 /2020 can provide real-time guidance for preventing and controlling the epidemic, assessing development trends and the effects of intervention measures [8] .COVID-19\\'s \"war epidemic\" and \"epidemic prevention\" strategies are closely related to the important measure of improving the transparency of clinical trials, including sharing of the subject\\'s original data and standardizing trial design, registration, and implementation [9].', 'cite_spans': [{'start': 98, 'end': 102, 'text': '1101', 'ref_id': None}, {'start': 103, 'end': 108, 'text': '/2020', 'ref_id': None}, {'start': 255, 'end': 258, 'text': '[8]', 'ref_id': None}], 'ref_spans': [], 'section': 'Discussion'}, {'text': \"At present, China's clinical trial registration policy is to implement registration[10], which requires registration in accordance with the ChiCTR registration guidelines. The ChiCTR registration guidelines divide research types into 7 categories, such as interventional studies, preventive studies, diagnostic tests, and observational studies, etc. [11] . However, it is ambiguous that the registration guidelines classify studies by their nature and purpose.\", 'cite_spans': [{'start': 350, 'end': 354, 'text': '[11]', 'ref_id': None}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'According to international standards, it may be more clear to classify intervention and observation [12] .', 'cite_spans': [{'start': 100, 'end': 104, 'text': '[12]', 'ref_id': None}], 'ref_spans': [], 'section': 'Discussion'}, {'text': \"In order to regulate registration, more details should be provided on the randomization procedure and blind method.We found that most interventional studies use randomized allocation, but there are low blinding rates and lack of information.The possible reason is that it is difficult to implement blind method in epidemic situation, but it may also be incomplete information. More than half of the studies did not indicate blind method information.Missing information means that no information about the size of the study is available, which adversely affects the transparency of registration [13] .The results showed that more than half of the studies were not confirmed by the data management committee. Low blinding rate and lack of information may be related to it's. The detection or guidance of the data management committee should be further strengthened to improve the quality of research design [14] .\", 'cite_spans': [{'start': 594, 'end': 598, 'text': '[13]', 'ref_id': None}, {'start': 905, 'end': 909, 'text': '[14]', 'ref_id': None}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'It is worth noting that 22.84% of the studies failed ethical verification.All human-oriented clinical trials must in any case follow the \"Helsinki Declaration\", which has become a consensus of the medical community, and follow the basic ethical principle of \"doctors should provide medical care in the best interests of patients\" to ensure the subjects [15, 16] .The ChiCTR Center has issued a warning and requested the supplementary information of the failed units to be reviewed.Medical ethics review should strictly abide by the review work content of the \"Ethical Review Methods for Biomedical Research Involving', 'cite_spans': [{'start': 353, 'end': 357, 'text': '[15,', 'ref_id': None}, {'start': 358, 'end': 361, 'text': '16]', 'ref_id': None}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Humans\" [17] , and here are two suggestions: (1) undertake experimental projects that match actual capabilities; (2) design of experimental schemes An appropriate research method must be selected based on the nature of the problem to be solved.', 'cite_spans': [{'start': 8, 'end': 12, 'text': '[17]', 'ref_id': None}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'As of February 24, 2020, the overall number of COVID-19 registered clinical studies is on the rise.The possible reason is that China has already initiated a first-level response to All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': \"The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10. 1101 /2020 major public health emergencies, and conducting relevant clinical trials is critical to epidemic prevention and control. In addition, in order to cooperate with the epidemic prevention and control work, many national drug clinical trial institutions(GCP) in China have established detailed work guidelines and guidelines to effectively speed up clinical trial review.From the statistical analysis of China's geographical distribution, it is found that the number of registered provinces in Hubei Province ranks first, and the number of registered provinces close to it is higher than that of other provinces. The possible reason is that the COVID-19 epidemic was mainly concentrated in Wuhan, Hubei Province, China. There was no large-scale epidemic outbreak in remote provinces, and there were relatively few NCP patients.\", 'cite_spans': [{'start': 98, 'end': 102, 'text': '1101', 'ref_id': None}, {'start': 103, 'end': 108, 'text': '/2020', 'ref_id': None}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'We found significant differences in the dimension of study design characteristics for different study types.The results showed that compared to observational studies, most interventional studies had fewer than 100 participants. However, 8.62% of observational study designs were completed within 1 month. Such designs were not reflected in interventional studies, and most of the interventional studies were completed in 4-12 months.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Similarly, compared to observational studies, most interventional studies use randomized allocation. These results imply that different interventions have an impact on the design of clinical trials.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'The design of COVID-19 clinical trials should give priority to \"timeliness\". The trial sample size needs to consider the balance between clinical and statistical significance, and its estimated volume reflects the reliability and repeatability of the research results [18] .As public health emergencies do not allow more time for exploratory clinical research and there is no historical data, the sample size estimation of such trials may encounter unprecedented problems.Some scholars have proposed [19] , whether it is possible to reduce the required sample size by \"reducing the power\" or \"increasing the type I error rate\" to complete the clinical trial as soon as possible?But such an approach requires a variety of trade-offs between regulators, sponsors, and researchers.', 'cite_spans': [{'start': 268, 'end': 272, 'text': '[18]', 'ref_id': None}, {'start': 500, 'end': 504, 'text': '[19]', 'ref_id': None}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'In addition, the study period is also \"time-effective\". If a clinical study is long, it may not make much sense in terms of public health emergencies to respond to the outbreak. From the statistical data, it can be seen that the time limit for completing most of the clinical studies of COVID-19 is concentrated within 4-12 months, and very few studies are completed within 1 month. The clinical research to deal with the epidemic is to race against the virus and the epidemic. This time limit seems too long. Is it possible to conduct mid-term analysis or other adaptive, new design methods? Clinical trial statistics need to be continuously developed All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10. 1101 /2020 based on actual needs.', 'cite_spans': [{'start': 98, 'end': 102, 'text': '1101', 'ref_id': None}, {'start': 103, 'end': 108, 'text': '/2020', 'ref_id': None}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'In addition to the prevention and control of the epidemic, vaccines and drugs are the two major weapons to overcome the epidemic.The previous SARS-CoV epidemic, the 2009 influenza epidemic, and the Ebola epidemic in West Africa were basically approved for marketing at the end of the epidemic, with a lag [20, 21] .Similarly, the development of specific drugs is also very difficult.The \"old medicine\" is a new way to increase the indications, and conducting clinical research on anti-COVID-19 may be an important means to respond to the current epidemic.', 'cite_spans': [{'start': 305, 'end': 309, 'text': '[20,', 'ref_id': None}, {'start': 310, 'end': 313, 'text': '21]', 'ref_id': None}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'The \"New Coronary Virus Pneumonia Diagnosis and Treatment Scheme (Trial Version 6)\" (hereinafter referred to as the \"Scheme\") states that there are currently no effective drugs to treat NCP patients.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Drugs such as lopinavir / ritonavir, interferon, abidol, and chloroquine phosphate can be tried, but their effectiveness needs to be confirmed in clinical studies.In the registration trial, the main treatment was drug therapy, accounting for 68.97%. Among them, there are more CMT and TCM studies, and relatively few BI studies.The most frequently studied CMT is chloroquine phosphate.Chloroquine phosphate is an antimalarial drug marketed for many The \"Scheme\" emphasizes the important role of TCM. The treatment of the \"pathogenic factor\" the mechanism by which TCM treats the cause of COVID-19 [28] . In addition, its All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 597, 'end': 601, 'text': '[28]', 'ref_id': None}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.16.20034934 doi: medRxiv preprint multi-target regulation of the body\\'s immune system has a therapeutic effect on \"cytokine This study has some limitations. The data for this study were derived only from ChiCTR.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'There may be other databases containing COVID-19 clinical studies. The time limit for research registration is short, and relevant registration information has not been filled in, and it cannot truly reflect its true quantity and level. However, the innovation of this article is to summarize and analyze the registration of ChiCTR\\'s COVID-19 clinical research in a timely manner, reflecting the research status and development trend of clinical trials in this field, which is \"time-effective\".', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'There were problems of unclear classification of research types and irregular registration behavior. Also, within the studies researched, heterogeneity exists for various dimensions.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusion'}, {'text': 'Different research types, intervention methods, and research stages lead to significant differences in some dimensions of the COVID-19 study. Finally, statistical high-frequency research drugs can provide more treatment options and evidence-based evidence for the clinical practice. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusion'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusion'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https: //doi.org/10.1101 //doi.org/10. /2020 author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 85, 'end': 102, 'text': '//doi.org/10.1101', 'ref_id': None}, {'start': 103, 'end': 122, 'text': '//doi.org/10. /2020', 'ref_id': None}], 'ref_spans': [], 'section': 'Conclusion'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https: //doi.org/10.1101 //doi.org/10. /2020 author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 85, 'end': 102, 'text': '//doi.org/10.1101', 'ref_id': None}, {'start': 103, 'end': 122, 'text': '//doi.org/10. /2020', 'ref_id': None}], 'ref_spans': [], 'section': 'Conclusion'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10. 1101 /2020 author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 98, 'end': 102, 'text': '1101', 'ref_id': None}, {'start': 103, 'end': 108, 'text': '/2020', 'ref_id': None}], 'ref_spans': [], 'section': 'Conclusion'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10. 1101 /2020 author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 98, 'end': 102, 'text': '1101', 'ref_id': None}, {'start': 103, 'end': 108, 'text': '/2020', 'ref_id': None}], 'ref_spans': [], 'section': 'Conclusion'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the .', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusion'}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'Lopinavir/ritonavir A fixed dose combination for HIV/AID treatment', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF1': {'ref_id': 'b1', 'title': 'Lopinavir: protease inhibitor Ritonavir: protease inhibitor and inhibitor of CYP3A4 4 ChiCTR2000029387 ChiCTR2000029496 ChiCTR2000029759 ChiCTR2000029386 ChiCTR2000029468', 'authors': [], 'year': 2000, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF2': {'ref_id': 'b2', 'title': 'No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint (which was not peer-reviewed) is the', 'authors': [{'first': 'I-Ifn', 'middle': [], 'last': 'Antiviral', 'suffix': ''}], 'year': 2000, 'venue': 'regulate immunity 4 ChiCTR2000029387 ChiCTR2000029638 ChiCTR2000030000 ChiCTR2000029759 ChiCTR2000029386 ChiCTR2000029308 ChiCTR2000029600 ChiCTR2000029989 ChiCTR2000030013 ChiCTR2000030082 ChiCTR2000030117 BI Cell therapy Suppresses the excessive activation of the immune Early stage ChiCTR2000029606 ChiCTR2000029569 ChiCTR2000029572 All rights reserved', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.03.16.20034934']}}, 'BIBREF3': {'ref_id': 'b3', 'title': 'No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint (which was not peer-reviewed) is the', 'authors': [{'first': 'M/P-Ab', 'middle': [], 'last': '', 'suffix': ''}], 'year': None, 'venue': 'Specific binding Early stage ChiCTR2000030089 ChiCTR2000030012 All rights reserved', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.03.16.20034934doi:medRxivpreprint']}}}, 'ref_entries': {'FIGREF0': {'text': 'Novel coronavirus (COVID-19) is a new highly pathogenic coronavirus that broke out in Wuhan, China in December 2019. It can cause severe pneumonia and is listed as Public Health Emergency of International Concern (PHEIC) [3]. As of February 25, 2020, China has accumulated more than 70,000 confirmed cases of Novel coronavirus pneumonia(NCP), which have been reported internationally in 24 countries and 5 continents [4]. The timely conduct of clinical drug research and the promotion of clinical trial information sharing are critical to epidemic prevention and control.In May 2006, the World Health Organization (WHO) officially launched the International Clinical Trials Registry Platform (ICTRP). In July 2007, The ChiCTR was officially certified by the ICTRP as a first-level registrar, becoming the fourth primary registrar after the Australian/New Zealand Center, the American Center, and the British Center [5]. According to the Declaration of Helsinki and the regulations of National Medical Products Administration (NMPA), all medical research conducted in China must be registered in ChiCTR [6]. As of February 24, 2020, the ChiCTR Center has registered more than 200 COVID-19 clinical studies. It is of great significance to discuss the research status and development situation in this field in time, which is effective for conducting clinical research and accelerating the transformation of results.', 'latex': None, 'type': 'figure'}, 'FIGREF1': {'text': 'funds. (3) Recruitment status. (4) Ethical approval. (5) Data Management Committee. (6) Research types. (7) Research design. (8) Research stage. (9) Study time limit. (10) Number of participants. (11) Intervention methods. (12) Setting Blinding.Two reviewers (Gao Song and Mengqun Cheng) were transferred to SPSS and checked to determine the final All rights reserved. No reuse allowed without permission.', 'latex': None, 'type': 'figure'}, 'FIGREF2': {'text': 'years, and it has been reported in the literature[22] that it can interfere with the glycosylation of the ACE2 receptor, thereby exerting antiviral effects.In vitro experiments show that chloroquine phosphate has the activity of inhibiting COVID-19, and its half effective concentration (EC50) is 1.13 μ /moL[23].As of February 24, 16 clinical studies of chloroquine phosphate-based NCP have registered clinical trials with ChiCTR. Chloroquine phosphate may have clinical therapeutic value.Mesenchymal stem cells are the most frequently studied in immunotherapy (n=10).Immune abnormalities are the main reason for the progression of patients with severe new-type coronavirus pneumonia. MSC can regulate both the innate immune system and the acquired immune system, and has become the most promising treatment for immune disorders[24,25].MSC can regulate inflammation through a series of mechanisms, including inhibiting T cell hyperproliferation, inducing CD4 + CD25 + FoxP3 + regulatory T cell (Tregs) subsets, and also inhibiting B cell hyperproliferation, differentiation, and immunoglobulin production[26]. At the same time, it can also regulate the secretion of major inflammatory factors and anti-inflammatory factors[27].Immunotherapy of MSC may have clinical therapeutic value.', 'latex': None, 'type': 'figure'}, 'FIGREF3': {'text': 'storm\" [29]. Currently, relevant clinical studies have confirmed the effectiveness of TCM[30], but further validation by large-scale RCT is needed. Chinese scholars[31] have counted the research frequency of TCM, this article does not count. BEI research also plays an important role. The rapid implementation of BEI intervention strategies in the early stages of the epidemic can effectively control the spread of the epidemic[32-35]. The main strategies are: (1) Early effective risk communication, timely understanding of the relevant knowledge, knowledge, attitudes and behaviors of the public, and health education. (2) Restrict the travel of people in key areas and mobilize the whole society.', 'latex': None, 'type': 'figure'}, 'FIGREF4': {'text': 'Abbreviations COVID-19:Corona virus disease 2019; ChiCTR:China Clinical Trial Registry; PHEIC:Public Health Emergency of International Concern; NCP:Novel coronavirus pneumonia; ICTRP:International Clinical Trials Registry Platform; CMT:Chemical treatment; BI:Biologics and immunoregulatory drugs; TCM:Traditional Chinese Medicine Treatment; GCP:National Drug Clinical Trial Institutions; I-IFN:I-Interferon; PD-1:CD279; M/P-AB=Monoclonal/Polyclonal Antibodies; vMIP=Virus macrophage inflammatory protein; hFABP=Heart fatty acid binding protein; G-CSF=granulocyte colony stimulating factor; BPHP=biological preparation of human placenta; FDPG=Anti-aging Active Freeze-dried Powder Granules. All rights reserved. No reuse allowed without permission.', 'latex': None, 'type': 'figure'}, 'FIGREF5': {'text': \"The past 10 years of clinical trial registration in China: Status ans Challenge[J]. CHINESE JOURNAL OF EVIDENCE-BASED MEDICINE, 2018(6):522-525 6.ZHANG Ting, LI Wanling, CHEN Lei. Analysis of hypertension-related clinical trial registration in China based on ClinicalTrials.gov and Chinese Clinical Trial Registry database[J]. West China Medical Journal, 2019, 34(04):419-424. 7.Kochhar S, Knoppers B, Gamble C, et al. Clinical trial data sharing: here's the challenge[J]. BMJ open, 2019, 9(8): e032334. 8.Heymann D L. Data sharing and outbreaks: best practice exemplified[J]. The Lancet, 2020, 395(10223): 469-470. 9.Taichman D B , Sahni P , Pinborg A , et al. Data sharing statements for clinical trials[J]. Bmj Clinical Research, 2017, 357:j2372. 10.WU Taixiang, BIAN Zhaoxiang, LI Youping, SHANG Hongcail. Promoting standardization of clinical trial data management in China[J]. CHINESE JOURNAL OF EVIDENCE-BASED MEDICINE, 2018(6):532-537. 11.Yao C, Sugino K, Qian G. Current status of multinational clinical trials in east asia[J]. Chin J Evid-based Med, 2010, 10(2): 122-128. 12.Bhaskar S B. Clinical trial registration: A practical perspective[J]. Indian journal of anaesthesia, 2018, 62(1): 10. 13.Rochon P A, Stall N M, Savage R D, et al. Transparency in clinical trial reporting[J]. 2018. 14.Nourani A, Ayatollahi H, Dodaran M S. Clinical Trial Data Management Software: A Review of the Technical Features[J]. Reviews on recent clinical trials, 2019, 14(3): 160-172. 15.World Medical Association. Helsinki Declaration of the World Medical Association (2013 edition)[C].64th World Congress of Medical Associations, Brazil: Fortaleza,2013.10. 16.Chengchun Sun. Discussion on Ethical Problems and Countermeasures in Clinical Medical Research[J].Chinese Journal of Medical Scientific Research Management, 2016 (004): 244-247. 17.ZHANG Hai-hong. Discussion on the Core Ethical Issues in Global Health Emergencies[J]. Medicine & Philosophy, 2019 (14): 1. 18.Abbas I. Hybrid research simulation modeling for making decisions on sample size and power of randomized clinical trials considering expected net benefits[C]//2017 Winter Simulation Conference (WSC). IEEE, 2017: 2845-2856. All rights reserved. No reuse allowed without permission.\", 'latex': None, 'type': 'figure'}, 'FIGREF6': {'text': \"21.US Food and Drug Administration. First FDA-approved vaccine for the prevention of Ebola virus disease, marking a critical milestone in public health preparedness and response[J]. Press Announc, 2019. 22.Vincent M J, Bergeron E, Benjannet S, et al. Chloroquine is a potent inhibitor of SARS coronavirus infection and spread[J]. Virology journal, 2005, 2(1): 69. 23.Wang M, Cao R, Zhang L, et al. Remdesivir and chloroquine effectively inhibit the recently emerged novel coronavirus (2019-nCoV) in vitro[J]. Cell Research, 2020: 1-3. 24.Uccelli A, Moretta L, Pistoia V. Mesenchymal stem cells in health and disease[J]. Nature reviews immunology, 2008, 8(9): 726-736. 25.Nauta, A.J. and W.E. Fibbe, Immunomodulatory properties of mesenchymal stromal cells. Blood, The Journal of the American Society of Hematology, 2007. 110(10): p. 3499-3506. 26.Uccelli, A. and N.K. de Rosbo, The immunomodulatory function of mesenchymal stem cells: mode of action and pathways. Annals of the New York Academy of Sciences, 2015. 1351(1): p. 114-126. 27.Ben-Mordechai T, Palevski D, Glucksam-Galnoy Y, et al. Targeting macrophage subsets for infarct repair[J]. Journal of cardiovascular pharmacology and therapeutics, 2015, 20(1): 36-51. 28.ZOU Lu, YU Xiao, WU Yuqin, SUN Ding, WU Yin'gen, FANG Hong. Analysis and thinking on traditional Chinese medicine in preventing and treating severe cases of novel coronavirus pneumonia.[J/OL].Academic Journal of Shanghai University of Traditional\", 'latex': None, 'type': 'figure'}, 'FIGREF7': {'text': 'Hui D S, I Azhar E, Madani T A, et al. The continuing 2019-nCoV epidemic threat of novel coronaviruses to global health-The latest 2019 novel coronavirus outbreak in Wuhan, China[J]. International Journal of Infectious Diseases, 2020, 91: 264-266. 34.Cheng V C C, Wong S C, To K K W, et al. Preparedness and proactive infection control measures against the emerging Wuhan coronavirus pneumonia in China[J]. Journal of Hospital Infection, 2020. 35.QI Ye, CHEN Liu-huan, ZHANG Li, YANG Ying-ying, ZHAN Si-yi, FU Chuan-xi.Public practice, attitude and knowledge of Novel Coronavirus-Infected Pneumonia[J/OL].Journal of Tropical Medicine,2020(02):1-12[2020-02-28].http://kns.cnki.net/kcms/detail/44.1503.R.20200204.11 14.002.html. All rights reserved. No reuse allowed without permission.', 'latex': None, 'type': 'figure'}, 'TABREF0': {'text': 'Type of research: intervention, observation, prevention, diagnostic test, prognosis research. (2) Intervention methods: Chemical treatment (CMT); Biologics and immunoregulatory drugs (BI, such as cell therapy, antibodies and glucocorticoid); Traditional Chinese Medicine Treatment (TCM, such as Chinese herbal medicine, proprietary Chinese medicines); Behavioral intervention (BEI, such as cognitive, attitude, and behavioral interventions, exercise and psychological therapy, etc.); no (missing information or not applicable [NA]). (3) funding source: industry (company, company), public (hospital, university, government), self-funding. (4) Research stage: early stages (stage 0, I, 1/2, 2), late stage (stage 3 or 4), and non-applicable stage. (5) Ethical approval: yes, no. (6) Data Management Committee: yes, no. Study design characteristics: multicenter study, number of participants, study duration, study assignment, Blinding, and key inclusion/exclusion criteria. It is defined as: (1) Number of study locations: Single study location or Multiple study locations; (2) Number of participants: <100, 100 to 300, 301 to 1000,> 1000. (3) Study duration(month): <1, 1 to 3, 4 to 12,> 12. (4) Blind method: open, single blind, double blind, unspecified. (5) Study assignment: randomized, non-randomized, factorial grouping, continuous grouping, and others. (6) key inclusion/exclusion criteria: yes.', 'latex': None, 'type': 'table'}, 'TABREF1': {'text': \"National Health and Health Commission of the people's Republic of China. Update on the\", 'latex': None, 'type': 'table'}, 'TABREF4': {'text': 'Basic characteristics of COVID-19 registered clinical trialsAll rights reserved. No reuse allowed without permission.', 'latex': None, 'type': 'table'}, 'TABREF5': {'text': 'Design features of COVID-19 registered clinical trialAll rights reserved. No reuse allowed without permission.', 'latex': None, 'type': 'table'}, 'TABREF6': {'text': 'Design characteristic differences among Basic characteristics of COVID-19 registered clinical trials All rights reserved. No reuse allowed without permission.author/funder, who has granted medRxiv a license to display the preprint in perpetuity.The copyright holder for this preprint (which was not peer-reviewed) is the. https://doi.org/10.1101/2020.03.16.20034934 P＜0.05：Type of Study; b P＜0.05：Intervention model; c P＜0.05：stage; d P＜0.05：Primary sponsor All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.16.20034934', 'latex': None, 'type': 'table'}, 'TABREF7': {'text': 'CMT and BI studies in the clinical registry for the treatment of COVID-19 infection Figure1.Trend of registration time for COVID-19 clinical studies. Figure2.Regional distribution of registration institutions for COVID-19 clinical studies.(A)Provinces and municipalities registered for COVID-19 clinical research in China;(B)Registered Unit of Hubei Province. Figure3.Statistics chart of research frequency of CMT therapeutic drugs. Figure4.Statistics chart of research frequency of BI therapeutic drugs All rights reserved. No reuse allowed without permission.', 'latex': None, 'type': 'table'}}, 'back_matter': [{'text': 'Not applicable.', 'cite_spans': [], 'ref_spans': [], 'section': 'Acknowledgements'}]}\n",
      "{'paper_id': 'b19bcc6ecbec9356e42d8309b9def1270a107fce', 'metadata': {'title': 'High COVID-19 incidence among Norwegian travellers returned from Lombardy: implications for travel restrictions', 'authors': [{'first': 'Ola', 'middle': [], 'last': 'Brynildsrud', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Norwegian Institute of Public Health', 'location': {'settlement': 'Oslo', 'country': 'Norway'}}, 'email': ''}, {'first': 'Vegard', 'middle': [], 'last': 'Eldholm', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Norwegian Institute of Public Health', 'location': {'settlement': 'Oslo', 'country': 'Norway'}}, 'email': ''}]}, 'abstract': [], 'body_text': [{'text': 'On February 27th, three cases of COVID-19 were reported among Norwegians that had recently returned from Lombardy, Italy. Travellers from the region rapidly became the most common source of imported infections in the earliest stage of the Norwegian COVID-19 epidemic. The situation was exacerbated by the unfortunate temporal overlap between the Norwegian winter holidays and intense epidemic spread of COVID-19 in Northern Italy, resulting in a large number of infected travellers. Here we combined flight data on travels between Norway and Lombardy with patient-level data to determine the fraction of travellers returning to Norway that had been infected with SARS-CoV-2 .', 'cite_spans': [], 'ref_spans': [], 'section': 'Background'}, {'text': 'In Fig. 1 , daily numbers of diagnosed COVID-19 cases who had recently returned from Lombardy are shown together with the temporal distribution of passengers flying in each direction. We find that~3% of travellers who returned to Norway from Lombardy between February 21st and March 1st were eventually diagnosed with SARS-CoV-2 infection. We find a clear increase in the fraction of infected travellers during the period, from~1% on Feb 21st-25th with an exponential increase to~9% on March 1st (Fig. 2) . Similarly, we calculate the incidence rate per person-day among Norwegians in Lombardy to be roughly 3.4e-3 on Feb 21st-25th, up to 0.02 per person-day on March 1st.', 'cite_spans': [], 'ref_spans': [{'start': 3, 'end': 9, 'text': 'Fig. 1', 'ref_id': 'FIGREF1'}, {'start': 496, 'end': 504, 'text': '(Fig. 2)', 'ref_id': 'FIGREF2'}], 'section': 'Results and Discussion'}, {'text': 'These results confirm that COVID-19 infections were contracted at very high rates by travellers to Lombardy in late February.', 'cite_spans': [], 'ref_spans': [], 'section': 'Results and Discussion'}, {'text': '. CC-BY 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Results and Discussion'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint .', 'cite_spans': [], 'ref_spans': [], 'section': 'Results and Discussion'}, {'text': 'This period overlapped with the Norwegian winter vacation, and it is known that the vast majority of travellers were tourists heading for the Italian alps and ski resorts. In the same time-period, the number of daily reported cases in the Lombardy region rose from 15 to 369 in a population of ~10,000,000.', 'cite_spans': [], 'ref_spans': [], 'section': 'Results and Discussion'}, {'text': 'As the number of diagnosed cases in Lombardy in the early days of the epidemic is clearly not representative of the true COVID-19 prevalence at the time, we used daily numbers of fatalities to estimate the true case numbers at the time. Based on the number of deaths reported in Lombardy on March 13th (146), and assuming an average of 20 days ( Jung et al. 2020 ) from infection to death for fatal cases and a case fatality ratio of 1% ( Famulare 2020 ; Liu et al. 2020) , we estimate that the true number of new cases in Lombardy on February 22nd was ~14600, compared to 40 reported cases. This rough estimate suggests that as little as 1% of cases were detected.', 'cite_spans': [{'start': 344, 'end': 364, 'text': '( Jung et al. 2020 )', 'ref_id': 'BIBREF0'}, {'start': 437, 'end': 454, 'text': '( Famulare 2020 ;', 'ref_id': 'BIBREF2'}, {'start': 455, 'end': 471, 'text': 'Liu et al. 2020)', 'ref_id': 'BIBREF1'}], 'ref_spans': [], 'section': 'Results and Discussion'}, {'text': 'The pronounced uptick in infections diagnosed among passengers returning after February 25th is difficult to reconcile with the growth of the local epidemic (Fig. 2) . This might indicate that the infection-pressure increased quite abruptly at one or more tourist hubs such as ski-resorts or other localities. The very high infection rates seen among travelers returning to Norway indicates that tourist hubs might have been important early epicenters of the infection in Europe.', 'cite_spans': [], 'ref_spans': [{'start': 157, 'end': 165, 'text': '(Fig. 2)', 'ref_id': 'FIGREF2'}], 'section': 'Results and Discussion'}, {'text': 'Travellers returning to Norway from Lombardy contracted COVID-19 at extreme rates in the period spanning February 21st and March 1st, with a clear uptick in transmission in the middle of the period.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusion'}, {'text': 'The infection risk associated with international air travel is impossible to predict, as reported cases in a region might be only a fraction of the true case load, and also because the infection pressure in tourist destinations might be several fold higher than the overall pressure in a region. In Norway, this is also supported by extreme rates of infections among tourists returning from Austria in February and March, despite a low number of reported cases in the country at the time.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusion'}, {'text': 'The massive COVID-19 prevalence among travellers suggest that . CC-BY 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusion'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10. 1101 /2020 mandatory quarantine of returning travellers or suspension of non-essential international flights is essential if the aim is to control or suppress the COVID-19 pandemic.', 'cite_spans': [{'start': 98, 'end': 102, 'text': '1101', 'ref_id': None}, {'start': 103, 'end': 108, 'text': '/2020', 'ref_id': 'BIBREF2'}], 'ref_spans': [], 'section': 'Conclusion'}, {'text': 'Case statistics from the Lombardy region of Italy was collected from the official git of the Civil Protection department of Italy -Presidenza del Consiglio dei Ministeri -Dipartimanto della Protezione Civile ( https://github.com/pcm-dpc/COVID-19 ) .', 'cite_spans': [], 'ref_spans': [], 'section': 'Data collection'}, {'text': 'Flight data was kindly contributed by Avinor -a state-owned limited company that operates most civil airports in Norway. The data included all commercial direct flights between any Lombardian airport (all of which were Milanese) and an Avinor airport in the period Feb 1st -Mar 10th. The data also included passengers leaving an Avinor airport towards Milano with an indirect (\"connecting\") flight, but not those arriving from Lombardy via other airports. The latter was inferred by assuming equal proportions of direct vs indirect travel on flights to and from Lombardy. Notably, the data didn\\'t include information about non-Avinor-operated airports. This means that flights from Torp/Sandefjord airport are not included. A maximum of two flights per week were flying between Torp and Milano in the study period, and among patients that were diagnosed in Norway, none were reported to have been on this route. Travel by alternative means such as by car or train were assumed to be too uncommon to matter in our estimates.', 'cite_spans': [], 'ref_spans': [], 'section': 'Data collection'}, {'text': 'Patient-level information about Norwegian SARS-CoV-2 cases as well as epidemiological tracking was contributed by the Norwegian Institute of Public Health.', 'cite_spans': [], 'ref_spans': [], 'section': 'Data collection'}, {'text': 'Daily passenger-level travel counts were smoothed with a lowess function using a smoother span of 0.4. Daily number of confirmed cases in Norway was smoothed with a span of 0.66. From contact tracing of the patients that had been to Lombardy we established that they had returned by flight~5 days prior to receiving a positive diagnosis. However, since this information was not available for all cases, we proceeded as if all cases had . CC-BY 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Estimates using returning travelers'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10. 1101 /2020 returned home exactly 5 days before diagnosis. Based on contact information as well as previous estimates of incubation period (Lauer et al. 2020; Li et al. 2020 ) and time from symptom onset to diagnosis (Jung et al. 2020) , we assumed that they had been infected in Lombardy. For person-time, we assumed that all travelers stayed in Lombardy for four days. ', 'cite_spans': [{'start': 98, 'end': 102, 'text': '1101', 'ref_id': None}, {'start': 103, 'end': 108, 'text': '/2020', 'ref_id': 'BIBREF2'}, {'start': 236, 'end': 255, 'text': '(Lauer et al. 2020;', 'ref_id': None}, {'start': 256, 'end': 270, 'text': 'Li et al. 2020', 'ref_id': None}, {'start': 314, 'end': 332, 'text': '(Jung et al. 2020)', 'ref_id': 'BIBREF0'}], 'ref_spans': [], 'section': 'Estimates using returning travelers'}, {'text': 'All data used in the current study was anonymized prior to being obtained by the authors. Flight statistics was aggregated and could not be used to identify individuals. Daily number of cases by municipality with epidemiological origin is in the public domain and posted daily in reports from the Norwegian Institute of Public Health.', 'cite_spans': [], 'ref_spans': [], 'section': 'Ethical considerations'}, {'text': '. CC-BY 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Ethical considerations'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10. 1101 /2020 . CC-BY 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 98, 'end': 102, 'text': '1101', 'ref_id': None}, {'start': 103, 'end': 108, 'text': '/2020', 'ref_id': 'BIBREF2'}], 'ref_spans': [], 'section': 'Ethical considerations'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10. 1101 /2020 ', 'cite_spans': [{'start': 98, 'end': 102, 'text': '1101', 'ref_id': None}, {'start': 103, 'end': 108, 'text': '/2020', 'ref_id': 'BIBREF2'}], 'ref_spans': [], 'section': 'Ethical considerations'}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'Real Time Estimation of the Risk of Death from Novel Coronavirus (2019-nCoV) Infection: Inference Using Exported Cases', 'authors': [{'first': 'Sung', 'middle': ['-'], 'last': 'Jung', 'suffix': ''}, {'first': 'Andrei', 'middle': ['R'], 'last': 'Mok', 'suffix': ''}, {'first': 'Katsuma', 'middle': [], 'last': 'Akhmetzhanov', 'suffix': ''}, {'first': 'Natalie', 'middle': ['M'], 'last': 'Hayashi', 'suffix': ''}, {'first': 'Yichi', 'middle': [], 'last': 'Linton', 'suffix': ''}, {'first': 'Baoyin', 'middle': [], 'last': 'Yang', 'suffix': ''}, {'first': 'Tetsuro', 'middle': [], 'last': 'Yuan', 'suffix': ''}, {'first': 'Ryo', 'middle': [], 'last': 'Kobayashi', 'suffix': ''}, {'first': 'Hiroshi', 'middle': [], 'last': 'Kinoshita', 'suffix': ''}, {'first': '', 'middle': [], 'last': 'Nishiura', 'suffix': ''}], 'year': 2020, 'venue': 'medRxiv', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF1': {'ref_id': 'b1', 'title': 'Time-Varying Transmission Dynamics of Novel Coronavirus Pneumonia in China', 'authors': [{'first': 'Tao', 'middle': [], 'last': 'Liu', 'suffix': ''}, {'first': 'Jianxiong', 'middle': [], 'last': 'Hu', 'suffix': ''}, {'first': 'Jianpeng', 'middle': [], 'last': 'Xiao', 'suffix': ''}, {'first': 'Guanhao', 'middle': [], 'last': 'He', 'suffix': ''}, {'first': 'Min', 'middle': [], 'last': 'Kang', 'suffix': ''}, {'first': 'Zuhua', 'middle': [], 'last': 'Rong', 'suffix': ''}, {'first': 'Lifeng', 'middle': [], 'last': 'Lin', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.01.25.919787']}}, 'BIBREF2': {'ref_id': 'b2', 'title': '2019-nCoV: Preliminary Estimates of the Confirmed-Case-Fatality-Ratio and Infection-Fatality-Ratio, and Initial Pandemic Risk Assessment. v2.0\" Accessed', 'authors': [{'first': 'M', 'middle': [], 'last': 'Famulare', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF3': {'ref_id': 'b3', 'title': 'The Rate of Underascertainment of Novel Coronavirus (2019-nCoV) Infection: Estimation Using Japanese Passengers Data on Evacuation Flights', 'authors': [{'first': 'Hiroshi', 'middle': [], 'last': 'Nishiura', 'suffix': ''}, {'first': 'Tetsuro', 'middle': [], 'last': 'Kobayashi', 'suffix': ''}, {'first': 'Yichi', 'middle': [], 'last': 'Yang', 'suffix': ''}, {'first': 'Katsuma', 'middle': [], 'last': 'Hayashi', 'suffix': ''}, {'first': 'Takeshi', 'middle': [], 'last': 'Miyama', 'suffix': ''}, {'first': 'Ryo', 'middle': [], 'last': 'Kinoshita', 'suffix': ''}, {'first': 'Natalie', 'middle': ['M'], 'last': 'Linton', 'suffix': ''}], 'year': 2020, 'venue': 'Journal of Clinical Medicine Research', 'volume': '9', 'issn': '2', 'pages': '', 'other_ids': {'DOI': ['10.3390/jcm9020419']}}}, 'ref_entries': {'FIGREF0': {'text': 'Figure legends', 'latex': None, 'type': 'figure'}, 'FIGREF1': {'text': 'Air-travel between Norway and Lombardy, with reported cases in Norway among travellers returning from Lombardy. The green bars indicate confirmed cases in Norway linked to travel in Italy. The first 14 cases in Lombardy were detected on Feb 21st. The Norwegian winter vacation lasts for one week, but the exact week varies between regions. In total the vacation was spread over three weeks spanning February 21st -March 6th.', 'latex': None, 'type': 'figure'}, 'FIGREF2': {'text': 'COVID-19 infection-load among travellers returning from Lombardy. (Top) Estimated fraction of flight passengers returning to Norway infected with Sars-CoV-2. (Bottom) Cumulative number of notified cases in the Lombardy region.', 'latex': None, 'type': 'figure'}, 'TABREF0': {'text': 'Air travel between Norway and LombardySmoothed estimate of returning travelers later diagnosed with infection from Lombardia', 'latex': None, 'type': 'table'}}, 'back_matter': []}\n",
      "{'paper_id': 'b1a5096c2bfbb4ab9826c36e3961929a744134ca', 'metadata': {'title': 'Inferring generation-interval distributions from contact-tracing 2 data', 'authors': [{'first': 'Sang', 'middle': ['Woo'], 'last': 'Park', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'McMaster University', 'location': {'settlement': 'Hamilton', 'region': 'Ontario', 'country': '6 Canada'}}, 'email': ''}, {'first': 'David', 'middle': [], 'last': 'Champredon', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'McMaster University', 'location': {'settlement': 'Hamilton', 'region': 'Ontario', 'country': 'Canada'}}, 'email': ''}, {'first': 'Jonathan', 'middle': [], 'last': 'Dushoff', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'McMaster University', 'location': {'settlement': 'Hamilton', 'region': 'Ontario', 'country': 'Canada'}}, 'email': ''}]}, 'abstract': [{'text': 'Generation intervals, defined as the time between when an individual is infected and when 14 that individual infects another person, link two key quantities that describe an epidemic: 15 the reproductive number, R, and the rate of exponential growth, r. Generation intervals 16 are often measured through contact tracing by identifying who infected whom. We study 17 how observed intervals differ from \"intrinsic\" intervals that could be estimated by tracing 18 individual-level infectiousness, and identify both spatial and temporal effects, including cen-19 soring (due to observation time), and the effects of susceptible depletion at various spatial 20 scales. Early in an epidemic, we expect the variation in the observed generation intervals to 21 be mainly driven by the censoring and the population structure near the source of disease 22 spread; therefore, we predict that correcting observed intervals for the effect of temporal 23 censoring but not for spatial effects will provide a spatially informed \"effective\" generation-24 interval distribution, which will correctly link r and R. We develop and test statistical 25 methods for temporal corrections of generation intervals, and confirm our prediction using 26 individual-based simulations on an empirical network.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'An epidemic can be characterized by the exponential growth rate, r, and the reproductive 32 number, R. The reproductive number, defined as the average number of secondary cases 33 arising from a primary case, is of particular interest as it provides information about the 34 final size of an epidemic (1; 2). However, estimating the reproductive number directly from 35 disease natural history requires detailed knowledge which is not often available, particularly 36 early in an outbreak (3). Instead, the reproductive number can be indirectly estimated from 37 the exponential growth rate, which can be estimated from incidence data (4; 5; 6; 7; 8).', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'These two quantities are linked by generation-interval distributions (9; 10; 11; 12; 13). 39 A generation interval is defined as the time between when a person becomes infected and 40 when that person infects another person (10). Due to individual variation in infection time, 41 the observed generation-interval distribution can change depending on when and how it is 42 measured (10; 14; 15; 16). There are important distinctions to be made when estimating 43 generation intervals: intrinsic generation intervals measure the infectiousness of an infected 44 individual, while observed generation intervals refer to the time between actual infection 45 events. 46 While the intrinsic generation-interval distribution is often assumed to be fixed, the 47 shape of the observed generation-interval distribution depends on when and how intervals 48 are measured (14; 15; 17; 16; 18). When an epidemic is growing exponentially, as often occurs 49 near the beginning of an outbreak, the number of newly infected individuals will be large 50 relative to the number infected earlier on. A susceptible individual is thus relatively more 51 likely to be infected by a newly infected individual. Thus, \"backward\" generation intervals 52', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': '(which look at a cohort of individuals, and ask when their infectors were infected) will be 53 shorter on average than intrinsic generation intervals -the converse is true when an epidemic 54 is subsiding (15; 16; 18).', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'In practice, generation intervals are often measured by aggregating available information 56 from contact tracing. While an epidemic is ongoing, these measurements are \"censored\": we 57 don\\'t know what happens after the observation time. These censored intervals are similar 58 to backward intervals: there is a bias towards observing shorter intervals, which are more 59 likely to have concluded by the observation time.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'Realized generation intervals are also affected by spatial structure. In a population 61 that does not mix homogeneously, susceptibility will tend to decrease more quickly in the 62 neighbourhood of infected individuals than in the general population. This means that 63 contacts made late in an individual\\'s infection are more likely to be ineffective due to contacts 64 that were made earlier (because the contactee may have been infected already) As a result, 65 realized generation intervals will have shorter mean than the intrinsic generation-interval 66 distribution. This perspective allows us to reinterpret the finding of (19) that, given an 67 intrinsic generation interval and an observed growth rate, the reproductive number on various 68 network structures is always smaller on a network than would be predicted from homogeneous 69 mixing. These observations allow us to make the following prediction: removing the time-70 censoring bias from observed generation intervals will yield a spatially corrected \"effective\" 71 distribution that contains information about the population structure and will allow us to 72 2', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}], 'body_text': [{'text': 'correctly infer the reproductive number from the exponential growth rate.', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'In this study, we explore spatiotemporal variation in generation intervals measured 74 through contact tracing. We extend previous frameworks to study how the censored generation-75 interval distributions change over time and compare this distribution with the backward 76 generation-interval distribution. We classify spatial effects on generation intervals into three 77 levels (egocentric, local, and global) and discuss how they affect observed generation-interval 78 distributions. Finally, we compare two methods for accounting for temporal bias and test 79 our prediction using simulations. course of infection (e.g., duration of latent and infectious periods) and the level of infectious-88 ness, which can depend both on biological infectiousness as well as contact patterns. 89 Assuming that the individual properties are independent of risk of infection, the population-90 level kernel is given by integrating over these individual variations:', 'cite_spans': [{'start': 469, 'end': 471, 'text': '78', 'ref_id': None}, {'start': 785, 'end': 787, 'text': '89', 'ref_id': None}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'The population-level kernel describes the rate at which secondary infections are expected 92 to be caused by an an infected individual, on average. where f (a) represents a probability 93 density over a (possibly multi-dimensional) aspect space. 94 Assuming that a population mixes homogeneously, we can write:', 'cite_spans': [{'start': 246, 'end': 248, 'text': '94', 'ref_id': None}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'where R 0 = K(τ )dτ is the basic reproductive number (the expected number of secondary 96 cases caused by a randomly chosen infectious individual in a fully susceptible population), 97 and g(τ ) is the expected time distribution of those cases (the intrinsic generation-interval 98 distribution).', 'cite_spans': [{'start': 182, 'end': 184, 'text': '97', 'ref_id': None}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'In a homogeneously mixing population, current disease incidence at time t, i(t), is the 100 product of the current infectiousness of individuals infected in the past and the current 101 proportion of the population susceptible, S(t).', 'cite_spans': [{'start': 182, 'end': 185, 'text': '101', 'ref_id': None}], 'ref_spans': [], 'section': '99'}, {'text': '102', 'cite_spans': [], 'ref_spans': [], 'section': '99'}, {'text': 'where R = S(t)R 0 is the effective reproductive number. This model, referred to as the The density of new infections occurring at time t caused by infectors who were infected 116 at time t − τ is given by', 'cite_spans': [], 'ref_spans': [], 'section': '99'}, {'text': '4 . CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/683326 doi: bioRxiv preprint tion of infection that occurred τ time units before the reference time t and is proportional 119 to i(t − τ )g(τ ). The censored generation-interval distribution, c t (τ ), describes all cohorts 120 infected at time s < t; thus, the censored generation-interval distribution is just the average 121 of the backward distributions of these cohorts, weighted by incidence:', 'cite_spans': [], 'ref_spans': [], 'section': '99'}, {'text': 'For a single outbreak, the observed mean generation interval through contact tracing 123 will always be shorter than the mean intrinsic generation interval ( Figure 2 ). There are two 124 reasons for this phenomenon. First, longer generation intervals are more likely to be missed 125 due to right censoring (and short generation intervals are more likely to be observed). In 126 particular, when an epidemic is growing exponentially (i(t) = i(0) exp(rt)), the initial cen-127 sored (or backward) generation-interval distribution is just the intrinsic generation-interval 128 distribution discounted by the rate of exponential growth (18):', 'cite_spans': [], 'ref_spans': [{'start': 158, 'end': 166, 'text': 'Figure 2', 'ref_id': None}], 'section': '99'}, {'text': 'A deterministic simulation confirms that the censored generation-interval distribution has 130 the same mean as the backward generation-interval distribution during this period ( Figure 2 ).', 'cite_spans': [], 'ref_spans': [{'start': 179, 'end': 187, 'text': 'Figure 2', 'ref_id': None}], 'section': '99'}, {'text': 'Second, the decreasing number of susceptibles over the course of an epidemic makes long 132 infections less likely to occur (16). Overall, we expect naively using the observed generation-133 interval distribution, to underestimate the reproductive number. Infected individuals may contact the same susceptible individual multiple times, but only 136 the first effective contact gives rise to infection in a given individual (after this, they are no 137 longer susceptible). Therefore, we expect realized generation intervals to be shorter than 138 intrinsic generation intervals, on average, in a limited contact network.', 'cite_spans': [], 'ref_spans': [], 'section': '131'}, {'text': 'To explore the effect of multiple contacts on realized generation intervals, we first consider 140 the infection process from an \"egocentric\" point of view, taking into account infectious 141 contacts made by a single infector. We define the egocentric kernel as the rate at which 142 secondary infections are realized by a single primary case with aspect a in the absence of 143 other infectors:', 'cite_spans': [], 'ref_spans': [], 'section': '139'}, {'text': 'where k(τ ; a) is the individual-level intrinsic kernel and e −δ(a) τ 0 k(s;a)ds is the probability that 145 a susceptible acquaintance has not yet been contacted by a particular infected individual.', 'cite_spans': [], 'ref_spans': [], 'section': '139'}, {'text': 'The dilution term, δ(a), models how contacts are distributed among susceptible acquain-147 tances.', 'cite_spans': [], 'ref_spans': [], 'section': '146'}, {'text': 'Throughout this paper, we assume that there is a constant per-pair contact rate λ. In Figure 2 : Temporal variation in the mean observed generation interval. A deterministic Susceptible-Exposed-Infectious-Recovered (SEIR) model was simulated using Ebola-like parameters (20): mean latent period 1/σ = 5 days, mean infectious period 1/γ = 11.4 days, and the basic reproductive number R 0 = 2. The backward and the censored mean generation interval were calculated over the course of an epidemic. The dotted horizontal line represents the mean intrinsic generation interval.', 'cite_spans': [], 'ref_spans': [{'start': 86, 'end': 94, 'text': 'Figure 2', 'ref_id': None}], 'section': '148'}, {'text': '6 . CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/683326 doi: bioRxiv preprint the number of acquaintances: δ(a) = 1/N (a). This assumption can be relaxed by allowing 152 for asymmetry (19) or heterogeneity (27; 28) in contact rates; for brevity, we do not pursue 153 these directions here.', 'cite_spans': [], 'ref_spans': [], 'section': '148'}, {'text': 'The population-level egocentric kernel is found by integrating over individual variations:', 'cite_spans': [], 'ref_spans': [], 'section': '154'}, {'text': '(19) used this same kernel (also assuming a constant per-pair contact rate) to study the 157 effect of network structure on the estimate of reproductive number. The population-level 158 egocentric generation-interval distribution is:', 'cite_spans': [], 'ref_spans': [], 'section': '154'}, {'text': 'The population-level egocentric generation-interval distribution describes the distribution of :', 'cite_spans': [], 'ref_spans': [], 'section': '154'}, {'text': 'As the egocentric distribution always has a shorter mean than the intrinsic distribution,', 'cite_spans': [], 'ref_spans': [], 'section': '154'}, {'text': 'will always be smaller than R estimated from the intrinsic distribution; this generation- ', 'cite_spans': [], 'ref_spans': [], 'section': '165R'}, {'text': 'For example, consider a susceptible-exposed-infected-recovered (SEIR) model, which 169 assumes that latent and infectious periods are exponentially distributed. The intrinsic 170 generation-interval distribution that corresponds to this model can be written as (29):', 'cite_spans': [], 'ref_spans': [], 'section': '168'}, {'text': 'where 1/σ and 1/γ are mean latent and infectious periods, respectively. Assuming a constant 172 per-pair contact rate of λ for any pair, we obtain the following egocentric generation-interval 173 distribution:', 'cite_spans': [], 'ref_spans': [], 'section': '168'}, {'text': 'In this case, with fixed infectiousness during the infection period, the effect of accounting The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/683326 doi: bioRxiv preprint using this distribution to link r and R using the Euler-Lotka equation is unrealistic because 181 it requires that we know the per-pair contact rate. Instead, the per-pair contact rate can be 182 inferred from the growth rate r, assuming that mean and variance of the degree distribution The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/683326 doi: bioRxiv preprint times that occurs on most networks, however. Generation intervals are also shortened by 193 indirect connections: a susceptible individual can be infected through another route before 194 the focal individual makes infectious contacts. Simulations on a small homogeneous network 195 confirm this additional effect (Fig. 3, right panel) . 196 In general, spatial reduction in the mean generation interval can be viewed as an effect Since the right-censoring effect is a sampling bias, we typically want to correct for it.', 'cite_spans': [{'start': 1018, 'end': 1021, 'text': '196', 'ref_id': None}], 'ref_spans': [{'start': 994, 'end': 1015, 'text': '(Fig. 3, right panel)', 'ref_id': 'FIGREF8'}], 'section': '168'}, {'text': 'In contrast, spatial effects have the same effect on how the epidemic spreads as they do on 219 observed generation intervals. We therefore expect that starting from observed generation 220 intervals and correcting for the right-censoring effect, will allow us to estimate an \"effective\" 221 generation interval that accurately reflects dynamics of spread. When temporal correction 222 is performed early in an outbreak, during the exponential growth phase, the effective distri-223 bution should incorporate egocentric and local spatial effects but not the global effects; we 224 expect this distribution to correctly link r and R. We will call the temporally corrected ef-225 fective generation-interval distribution g eff (τ ). In a large homogeneously mixing population, Here, we investigate two methods for correcting for temporal bias in contact-tracing data 230 (see Methods for details). We refer to the first method as the population-level method as Using the observed generation-interval distributions (based on the first 1000 generation intervals) without correcting for right-censoring severely underestimates the reproductive number. Similarly, using the intrinsic generation-interval distribution overestimates the reproductive number because it fails to account for local spatial effects; the egocentric distribution corrects for this only partially. Both population-level and individual-level methods provide estimates of reproductive number that are consistent with the empirical estimates, which we define as the average number of secondary cases generated by the first 100 infected individuals, but the individual-level method is more precise. Boxplots are generated using 100 stochastic simulations of the SEIR model on an empirical network using Ebola-like parameters (20): mean latent period 1/σ = 5 days and mean infectious period 1/γ = 11.4 days. Per-pair contact rate λ = 0.08 days −1 is chosen to be sufficiently high such that differences are clear.', 'cite_spans': [], 'ref_spans': [], 'section': '218'}, {'text': '. CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/683326 doi: bioRxiv preprint epidemic is growing exponentially, right-censoring causes the observed generation interval to 233 be discounted by the exponential growth rate (7); hence, we can \"undo\" the censoring by 234 exponentially weighting the observed generation-interval distribution (17; 15; 18):', 'cite_spans': [], 'ref_spans': [], 'section': '10'}, {'text': 'where r is the exponential growth rate. 236 We refer to the second method as the individual-level method because it relies on indi- whereas changes due to spatial or network structure reflect the dynamics of the outbreak. We compare two methods for estimating the effective generation-interval distribution and 298 assume that the effective generation-interval distribution follows a gamma distribution. The 299 gamma approximation of the generation interval distribution has been widely used due to its 300 simplicity (31; 6; 32; 33; 34); we previously thought that summarizing the entire distribution 301 with two moments (mean and variance) is sufficient to understand the role of generation-302 interval distributions in linking r and R (13). However, further investigation of our methods 303 suggests that making a wrong distributional assumption can lead to biased estimates of 304 the mean and CV of a generation-interval distribution (Appendix A.3), even though the 305 gamma distribution \"looks\" indistinguishable from the shape of the intrinsic generation-306 interval distribution (derived from the SEIR model). These results are particularly alarming population structure can lead to subexponential growth (35; 36; 37; 38; 39; 40); we therefore expect our simulations on an empirical network to be better characterized by subexponential 314 growth models (40). However, our simulations suggest that the initial exponential growth 315 assumption still provides a viable approach for estimating the reproductive number.', 'cite_spans': [{'start': 40, 'end': 43, 'text': '236', 'ref_id': None}], 'ref_spans': [], 'section': '10'}, {'text': 'Contact tracing provides an effective way of collecting epidemiological data and control- The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/683326 doi: bioRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': '316'}, {'text': 'dI n dt = n I γ(I n−1 I n ) for n = 2, 3, . . . , n I', 'cite_spans': [], 'ref_spans': [], 'section': '316'}, {'text': 'where S is the number of susceptible individuals, E m is the number of exposed individuals in distribution. Assume that latent and infectious periods are exponentially distributed with 379 mean 1/σ and 1/γ, respectively. Assuming a constant per-pair contact rate of λ for any pair, 380 the egocentric generation-interval distribution can be written:', 'cite_spans': [], 'ref_spans': [], 'section': '316'}, {'text': 'Substituting into (11), we get', 'cite_spans': [], 'ref_spans': [], 'section': '316'}, {'text': 'where r is the exponential growth rate. Alternatively, the reproductive number can be 383 expressed based on the degree distribution (mean µ and variance σ 2 ) of a network:', 'cite_spans': [], 'ref_spans': [], 'section': '316'}, {'text': 'where κ = σ 2 /µ + µ − 1, referred to as the mean degree excess (52), describes the expected 385 number of susceptible individuals that an average infected individual will encounter early in 386 an outbreak. Combining the two equations, we get', 'cite_spans': [], 'ref_spans': [], 'section': '316'}, {'text': 'which completes the relationship between the growth rate and the egocentric reproductive The Population-level method estimates the effective generation-interval distribution by 392 reversing the inverse exponential weighting in the observed generation-interval distribution:', 'cite_spans': [], 'ref_spans': [], 'section': '316'}, {'text': 'where s i,j is the observed generation interval between infector j and infectee i, and θ is a (vec-406 tor) parameter of the generation-interval distribution g (and the corresponding cumulative 407 distribution function G eff ).', 'cite_spans': [], 'ref_spans': [], 'section': '316'}, {'text': 'The likelihood of the non-homogeneous poisson process can be written as:', 'cite_spans': [], 'ref_spans': [], 'section': '408'}, {'text': 'where N is the total number of infected individuals. Here, we estimate parameters R and θ We estimate the exponential growth rate r of an epidemic from daily incidence by modeling 416 the cumulative incidence c(t) with a logistic function (8). Fitting directly to cumulative 417 incidence can lead to overly confident results (55); instead we fit interval incidence x(t) = 418 c(t + ∆t) − c(t), where ∆t is 1 day, to daily incidence, assuming that daily incidence follows 419 a negative binomial distribution. We estimate the logistic parameters r, K, c 0 , and θ by 420 maximum likelihood. Fitting time window is defined from the last trough before the peak 421 of an epidemic to the first day after the peak of an epidemic. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/683326 doi: bioRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': '408'}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'Infectious diseases of humans: dynamics and control', 'authors': [{'first': 'R', 'middle': ['M'], 'last': 'Anderson', 'suffix': ''}, {'first': 'R', 'middle': ['M'], 'last': 'May', 'suffix': ''}], 'year': 1991, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF1': {'ref_id': 'b1', 'title': 'On the definition and the computa', 'authors': [{'first': 'O', 'middle': [], 'last': 'Diekmann', 'suffix': ''}, {'first': 'Jap', 'middle': [], 'last': 'Heesterbeek', 'suffix': ''}, {'first': 'J', 'middle': ['A'], 'last': 'Metz', 'suffix': ''}], 'year': 1990, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF2': {'ref_id': 'b2', 'title': 'Time variations in the generation time of an infectious disease: im-473 plications for sampling to appropriately quantify transmission potential', 'authors': [{'first': 'H', 'middle': [], 'last': 'Nishiura', 'suffix': ''}], 'year': 2010, 'venue': 'Math. Biosci', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF4': {'ref_id': 'b4', 'title': 'Intrinsic and realized generation intervals in 476 infectious-disease transmission', 'authors': [{'first': 'Champredon D', 'middle': [], 'last': 'Dushoff', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': '', 'suffix': ''}], 'year': 2015, 'venue': 'Proc. R. Soc. B', 'volume': '282', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF5': {'ref_id': 'b5', 'title': 'Some model based 478 considerations on observing generation times for communicable diseases', 'authors': [{'first': 'G', 'middle': [], 'last': 'Scalia Tomba', 'suffix': ''}, {'first': '', 'middle': [], 'last': 'Svenssonå', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Asikainen', 'suffix': ''}, {'first': 'Giesecke', 'middle': ['J'], 'last': '', 'suffix': ''}], 'year': 2010, 'venue': 'Math. Biosci', 'volume': '479', 'issn': '', 'pages': '24--31', 'other_ids': {}}, 'BIBREF6': {'ref_id': 'b6', 'title': 'Estimation in emerging epidemics: Biases and 481 remedies', 'authors': [{'first': 'T', 'middle': [], 'last': 'Britton', 'suffix': ''}, {'first': 'Scalia', 'middle': [], 'last': 'Tomba', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': '', 'suffix': ''}], 'year': 2019, 'venue': 'J. Royal Soc. Interface', 'volume': '16', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF7': {'ref_id': 'b7', 'title': 'Inferring 483 R0 in emerging epidemics-the effect of common population structure is small', 'authors': [{'first': 'P', 'middle': [], 'last': 'Trapman', 'suffix': ''}, {'first': 'F', 'middle': [], 'last': 'Ball', 'suffix': ''}, {'first': 'J', 'middle': ['S'], 'last': 'Dhersin', 'suffix': ''}, {'first': 'V', 'middle': ['C'], 'last': 'Tran', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Wallinga', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Britton', 'suffix': ''}], 'year': 2016, 'venue': 'J. Royal', 'volume': '484', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF9': {'ref_id': 'b9', 'title': 'Ebola virus disease in West Africa-the first 9 months 486 of the epidemic and forward projections', 'authors': [], 'year': 2014, 'venue': 'N. Engl. J. Med', 'volume': '371', 'issn': '', 'pages': '1481--1495', 'other_ids': {}}, 'BIBREF10': {'ref_id': 'b10', 'title': 'The influence of assumptions on generation time distributions in 488 epidemic models', 'authors': [{'first': '', 'middle': [], 'last': 'Svenssonå', 'suffix': ''}], 'year': 2015, 'venue': 'Math. Biosci', 'volume': '270', 'issn': '', 'pages': '81--89', 'other_ids': {}}, 'BIBREF11': {'ref_id': 'b11', 'title': 'The concept of R0 in epidemic theory', 'authors': [{'first': 'J', 'middle': [], 'last': 'Heesterbeek', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Dietz', 'suffix': ''}], 'year': 1996, 'venue': 'Stat. Neerl', 'volume': '490', 'issn': '', 'pages': '89--110', 'other_ids': {}}, 'BIBREF12': {'ref_id': 'b12', 'title': 'Mathematical epidemiology of infectious dis-492 eases: model building, analysis and interpretation', 'authors': [{'first': 'O', 'middle': [], 'last': 'Diekmann', 'suffix': ''}, {'first': 'Jap', 'middle': [], 'last': 'Heesterbeek', 'suffix': ''}], 'year': 2000, 'venue': '', 'volume': '5', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF13': {'ref_id': 'b13', 'title': 'Modelling strategies for minimizing the impact of an imported exotic 494 infection', 'authors': [{'first': 'M', 'middle': [], 'last': 'Roberts', 'suffix': ''}], 'year': 2004, 'venue': 'Proc. R. Soc. B', 'volume': '271', 'issn': '', 'pages': '2411--2415', 'other_ids': {}}, 'BIBREF14': {'ref_id': 'b14', 'title': 'An integral equation model for the control of a smallpox 496 outbreak', 'authors': [{'first': 'Aldis', 'middle': ['G'], 'last': 'Roberts', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': '', 'suffix': ''}], 'year': 2005, 'venue': 'Math. Biosci', 'volume': '195', 'issn': '', 'pages': '1--22', 'other_ids': {}}, 'BIBREF15': {'ref_id': 'b15', 'title': 'Relation between birth rates and death rates', 'authors': [{'first': 'A', 'middle': ['J'], 'last': 'Lotka', 'suffix': ''}], 'year': 1907, 'venue': 'Science', 'volume': '26', 'issn': '', 'pages': '21--22', 'other_ids': {}}, 'BIBREF16': {'ref_id': 'b16', 'title': 'Epidemics with two levels of mixing', 'authors': [{'first': 'F', 'middle': [], 'last': 'Ball', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Mollison', 'suffix': ''}, {'first': 'Scalia-Tomba', 'middle': [], 'last': '', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': '', 'suffix': ''}], 'year': 1997, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}}, 'ref_entries': {'FIGREF0': {'text': 'Comparison of individual-and population-level kernels. (Left) an individual-level kernel of an infected individual with latent period of 11.4 days followed by infectious period of 5 days. (Right) a population-level kernel of infected individuals with latent and infectious periods exponentially distributed with means of 11.4 and 5 days, respectively. Shaded areas under the curves are equal to individual-and population-level reproductive numbers, both of which are set to 2 in this example. Parameters are chosen to reflect the West African Ebola outbreak (20).', 'latex': None, 'type': 'figure'}, 'FIGREF1': {'text': 'renewal equation, can describe a wide range of epidemic models (22; 23; 24; 25; 12; 11). Over a 104 period of time where the susceptible proportion S remains roughly constant, we would expect 105 approximately exponential growth in incidence i(t); assuming i(t) = i(0) exp(rt) yields the 106 Euler-Lotka equation (26), which provides a direct link between speed and strength of an generation intervals are estimated through contact tracing during an outbreak, in-110 fection events that have not happened yet are not observed. This effect is called \"right-111 censoring\". We can understand the effect of right-censoring using backward generation-112 interval distributions (17; 15; 16; 18): the observed (censored) generation-interval distri-113 bution is a weighted average of backward generation-interval distributions (weighted by 114 incidence) up until the observation time.', 'latex': None, 'type': 'figure'}, 'FIGREF5': {'text': 'times at which secondary infections are realized by an average primary case; for convenience,161    we will often omit \"population-level\". Finally, the link between the growth rate and the162 egocentric reproductive number are linked by the egocentric generation-interval distribution 163 (and the Euler-Lotka equation) (19)', 'latex': None, 'type': 'figure'}, 'FIGREF6': {'text': 'interval-based argument provides a clear biological interpretation for the result presented by 167 (19).', 'latex': None, 'type': 'figure'}, 'FIGREF7': {'text': 'for pairwise contacts is the same as an increase in the recovery rate (by the amount of 176 the per-pair contact rate): infecting a susceptible contact is analogous to no longer being 177 infectious (since the contact cannot be infected again); therefore, the resulting egocentric 178 generation-interval distribution is equivalent to the intrinsic generation-interval distribution', 'latex': None, 'type': 'figure'}, 'FIGREF8': {'text': 'can be validated by simulating stochastic infection processes on a \"star\" 186 network (i.e, a single infected individual at the center connected to multiple susceptible 187 individuals who are not connected with each other). Simulations (Fig. 3) confirm that in this 188 case the distribution of contact times matches the intrinsic generation-interval distribution 189 (left panel), while the distribution of realized generation times (i.e., infection times) matches 190 the egocentric generation-interval distribution (middle panel). Spatial effects on realized generation intervals. Theoretical distributions and means are shown in color (and are the same in each panel, for reference). Simulated distributions and means are shown in black. (Left) the intrinsic generation-interval distribution corresponds to all contacts by a focal individual, regardless of whether the individual contacted is susceptible. (Middle) the egocentric generation-interval distribution corresponds to the distribution of all infectious contacts by the focal individual with susceptible individuals, in the case where the focal individual is the only possible infector (simulated on a star network).(Right) realized generation-interval distributions are shorter than egocentric distributions in general, because contacts can be wasted when susceptibles become infected through other routes (simulated on a homogeneous network). All figures were generated using 5000 stochastic simulations on a network with 5 nodes (1 infector and 4 susceptibles) with Ebola-like parameters (20): mean latent period 1/σ = 5 days and mean infectious period 1/γ = 11.4 days. Per-pair contact rate λ = 0.25 days −1 is chosen to be sufficiently high so that the differences between generation-interval distributions are clear. Each simulation is run until all individuals are either susceptible or has recovered.', 'latex': None, 'type': 'figure'}, 'FIGREF9': {'text': '197 of susceptible depletion and can be further classified into three levels: egocentric, local, and 198 global. Egocentric depletion, as discussed previously, is caused by an infected individual 199 making multiple contacts to the same individual. Local depletion refers to a depletion of 200 susceptible individuals in a household or neighborhood; we can think of these structures as 201 small homogeneous networks embedded in a larger population structure (and therefore we 202 can expect similar effects as we see in Fig. 3, right panel). Both the egocentric and local 203 depletion effects can be observed early in an epidemic, especially in a highly structured pop-204 ulation, even if most of the population remains susceptible. Finally, global depletion refers to 205 overall depeletion of susceptibility at the population level, and explains the reduction in re-206 alized compared to intrinsic generation intervals that occurs even in a wellintervals are sampled through contact tracing, there will be four effects 211 present in the sample: (1) right-censoring effect, (2) egocentric depletion effect, (3) local 212 depletion effect and (4) global depletion effect. We can correct explicitly for the egocentric 213 effect and, in the case of exponential growth, the right-censoring effect. While the other two 214 effects are difficult to measure, we can make qualitative predictions about their effects on 215 the realized generation intervals and reproductive numbers: both local and global depletion 216 effects reduce number of infections that occur and shorten generation intervals.', 'latex': None, 'type': 'figure'}, 'FIGREF11': {'text': '226the effective distribution is expected to be equivalent to the intrinsic distribution (since the 227 egocentric and local depletion are negligible in the absence of network structure, and global 228 depletion is negligible early in the epidemic).', 'latex': None, 'type': 'figure'}, 'FIGREF13': {'text': 'Comparison of estimates of reproductive number based on various methods.', 'latex': None, 'type': 'figure'}, 'FIGREF14': {'text': 'vidual contact information. We model each infection as a non-homogeneous Poisson process 238 arising from the infector (23); incorporating information about time of infection of an in-239 fector, time of infection of an infectee, and time since the beginning of an epidemic allows 240 us to explicitly model the censoring process in the observed generation intervals. For both 241 methods, mean and coefficient of variation (CV) of generation-interval distributions are es-242 timated by maximum likelihood; the inferred generation-interval distribution is then used to 243 estimate the reproductive number R from the observed growth rate r using the Euler-Lotka 244 equation. 245 To test these methods, we simulate 100 epidemics with Ebola-like parameters on an 246 empirical network (30) and compare the estimates of reproductive number with empirical 247 reproductive numbers, which we define as the average number of secondary cases generated 248 by the first 100 infected individuals (Fig. 4). As expected, calculating reproductive number 249 based on the intrinsic generation-interval distribution overestimates the empirical reproduc-250 tive number; estimates based on the egocentric generation-interval distribution (10) address 251 this problem only partially, as they do not account for indirect (local) spatial effects. Direct 252 estimates based on the observed generation intervals from contact tracing severely under-253 estimate the empirical estimates. While both population-and individual-level corrections 254 provide similar estimates to the empirical reproductive number on average, population-level 255 estimates are more variable as they are more sensitive to outliers in generation intervals and 256 our estimate of the growth rate. For smaller values of R, we expect the differences to become 257 smaller. In the electronic supplementary material, we present the same figure using smaller 258 R (Appendix A.1) and using Erlang-distributed latent periods (Appendix A.2), which better 259 corresponds to Ebola. Overall, our qualitative conclusions do not change. generation-interval distribution, which describes the expected time distribution 262 of secondary cases, provides a direct link between speed (exponential growth rate, r) and 263 strength (reproductive number, R) of an epidemic (12; 10; 21; 13). However, observed 264 generation-interval distributions can vary depending on how and when they are measured (15; 265 17; 16; 18); determining which distribution correctly links r and R can be challenging. Here, 266 we analyze how observed generation intervals, measured through contact tracing, differ from 267 intrinsic generation intervals. Changes due to temporal censoring reflect observation bias,268', 'latex': None, 'type': 'figure'}, 'FIGREF15': {'text': 'intervals are subject to right-censoring -it is not possible to trace in-272 dividuals that have not been infected yet. These right-censored distributions can be thought 273 of as averages of \"backward\" generation intervals (measured by tracing the infectors of a co-274 hort of infected individuals) (14; 15; 17; 16; 18). During an ongoing outbreak, the observed 275 generation-interval distribution will always have a shorter mean than the intrinsic-interval 276 distribution due to right-censoring. Early in the outbreak, censored intervals are expected 277 to match the early-outbreak backward intervals. Near the end of an outbreak, the effect of 278 right-censoring becomes negligible but the observed generation intervals are still shorter on 279 average than intrinsic generation intervals, because of depletion of the susceptible population.', 'latex': None, 'type': 'figure'}, 'FIGREF16': {'text': 'We think of susceptible depletion as operating on three levels: egocentric, local, and281 global. Egocentric susceptible depletion refers to the effect of an infected individual making 282 multiple contacts to the same susceptible individual. Accounting for the egocentric effect 283 allowed us to link the results by (19) to established results based on generation intervals. 284 Local susceptible depletion refers to the effect of other closely related infected individuals 285 (e.g., in a household or neighborhood) making multiple contacts to the same susceptible 286 individual. Global susceptible depletion refers to the overall decrease in the susceptible pool 287 in the population.', 'latex': None, 'type': 'figure'}, 'FIGREF17': {'text': 'Susceptible depletion happening at all three levels shortens realized generation intervals 289 but acts on different time scales. Egocentric and local depletion effects are present from 290 the beginning of an epidemic, even when depletion in the global susceptible population is 291 negligible, and can strongly affect the initial spread of an epidemic. Therefore, we predicted 292 the observed generation intervals during an exponential growth phase to contain informa-293 tion about the contact structure, allowing us to estimate the effective generation-interval 294 distribution by simply accounting for the right-censoring. Simulation studies confirmed our 295 prediction: using the effective generation-interval distribution provides the correct link be-296 tween r and R.', 'latex': None, 'type': 'figure'}, 'FIGREF19': {'text': '317ling an outbreak (41; 42; 43). In particular, using tracing information allows us to infer318 real-time estimates of the time-varying reproductive number (44; 45; 46; 47); generation-319 interval distributions, which can be either assumed or estimated, often play a central role 320 in analyzing tracing data. Our study illustrates that generation intervals measured through 321 contact-tracing data contain information about the underlying contact structure, which can 322 be implicitly reflected in the estimates of the reproductive number; this perspective can be 323 particularly useful for characterizing an epidemic because detailed information about the 324 contact structure is often unavailable. 325 The generation-interval distribution is a key, and often under-appreciated, component 326 of disease modeling and forecasting. Different definitions, and different measurement ap-327 proaches, produce different estimates of these distributions. We have shown that estimates 328 based on observed generation intervals (e.g., through contact tracing) differ in predictable 329 ways from estimates based on underlying measures of infectiousness (e.g., from shedding 330 studies). These predictable differences can arise from temporal effects, egocentric spatial 331 effects, local spatial (or network) effects and population-level effects. We have shown that 332 correcting observed intervals for temporal effects allows us to estimate a spatially informed 333 \"effective\" distribution, which accurately describes how disease spreads in a population. 334 Future studies should carefully consider how measurement influences estimated generation-335 interval distributions, and how these distributions influence the spread of disease. the effects on right-censoring on the observed generation intervals, we use the de-339 terministic Susceptible-Exposed-Infectious-Recovered (SEIR) model. The SEIR model de-340 scribes how disease spreads in a homogeneously mixing population; it assumes that infeceted 341 individuals become infectious after a latent period. We use a SE m I n R model, which extends 342 the SEIR model to have multiple equivalent stages in the latent and infectious periods. This 343gives latent and infectious periods with Erlang distributions (gamma distributions with in-344 teger shape parameters), which are often more realistic than the exponentially', 'latex': None, 'type': 'figure'}, 'FIGREF20': {'text': 'the m-th compartment, I n is the number of infectious individuals in the n-th compartment,348    and R is the number of recovered individuals. Parameters of the model are specified as 349 follows: N is the total population size, β is the transmission rate, 1/σ is the mean latent 350 period, n E is the number of latent compartments, 1/γ is the mean infectious period, and 351 n I is the number of infectious compartments. We show results based on more realistic an individual-based SEIR model on a contact network, using an algorithm 355 based on the \"Gillespie algorithm\" (50; 51). We begin by randomly selecting individuals 356 assumed to be infected at t = 0. For each infected individual i, we randomly draw the latent 357 period E i from an Erlang distribution with mean 1/σ and shape n E . We then construct358 the random infectious period and infectious contact times simultaneously as follows. For 359 each of the n I stages of the infectious period, we draw the number of effective contacts (be-360 fore transitioning to the next compartment) from a geometric distribution with probability 361 n I γ/(S i λ + n I γ), where S i is the number of susceptible acquaintances and λ is the per-pair 362 contact rate. We then choose the time between consecutive events (the chosen number of 363 contacts, followed by exit from the given stage of infection) from an exponential distribution 364 with rate S i λ + n I γ. For each contact, a contactee is uniformly sampled from the set of 365 susceptible acquaintances of the individual i. The infectious period I i is the sum of all of 366 these waiting times. 367 After repeating the contact process for all initially infected individuals, all contacts are 368 put into a sorted queue. The first person in the queue becomes infected, and the current 369 time is updated to infection time of this individual. Any subsequent contacts made to this 370 individual are removed from the queue because they will no longer be effective. We repeat 371 the contact process for this newly infected individual. Then, new contacts are added to 372 the sorted queue. The simulation continues until there are no more contacts are left in the', 'latex': None, 'type': 'figure'}, 'FIGREF22': {'text': 'by maximum likelihood. Since the estimate of R is sensitive to under-reporting, we use the411 estimated generation-interval distribution to infer the effective reproductive number from the 412 estimated growth rate (using the Euler-Lotka equation). (54) proposed a related approach 413 based on discretized incidence reports. 414 7.5 Measuring the exponential growth rate415', 'latex': None, 'type': 'figure'}, 'FIGREF23': {'text': 'Proc. R. Soc. B. 271, S364-S366. 519 [36] Chowell G, Viboud C, Hyman JM, and Simonsen L. 2015 The Western Africa Ebola 520 virus disease epidemic exhibits both global exponential and local polynomial growth 521 rates. PLoS Curr. 7. 522 [37] Chowell G and Viboud C. 2016 Is it growing exponentially fast?-impact of assum-523 ing exponential growth for characterizing and forecasting epidemics with initial near-524 exponential growth dynamics. Infect. Dis. Model. 1, 71-78. 525 [38] Chowell G, Viboud C, Simonsen L, and Moghadas SM. 2016 Characterizing the repro-526 duction number of epidemics with early subexponential growth dynamics. J. Royal Soc. 527 Interface. 13, 20160659. 528 [39] Kiskowski M and Chowell G. 2016 Modeling household and community transmission 529 of Ebola virus disease: epidemic growth, spatial dynamics and insights for epidemic 530 control. Virulence. 7, 163-173. 531 [40] Viboud C, Simonsen L, and Chowell G. 2016 A generalized-growth model to characterize 532 the early ascending phase of infectious disease outbreaks. Epidemics. 15, 27-37. 533 [41] Clarke J. 1998 Contact tracing for chlamydia: data on effectiveness. Int. J. STD. AIDS. Eames KT and Keeling MJ. 2003 Contact tracing and disease control. Proc. R. Soc. Cauchemez S, Boëlle PY, Thomas G, and Valleron AJ. 2006 Estimating in real time 542 the efficacy of measures to control emerging communicable diseases. Am. J. Epidemiol. 543 164, 591-597. 544 [45] Hens N, Calatayud L, Kurkela S, Tamme T, and Wallinga J. 2012 Robust reconstruction 545 and analysis of outbreak data: influenza A (H1N1) v transmission in a school-based 546 population. Am. J. Epidemiol. 176, 196-203. 547 [46] Jewell CP and Roberts GO. 2012 Enhancing Bayesian risk prediction for epidemics 548 using contact tracing. Biostatistics. 13, 567-579. 549 [47] Soetens L, Klinkenberg D, Swaan C, Hahné S, and Wallinga J. 2018 Real-time es-550 timation of epidemiologic parameters from contact tracing data during an emerging 551 infectious disease outbreak. Epidemiology. 29, 230-236. 552 [48] Anderson D and Watson R. 1980 On the spread of a disease with gamma distributed 553 latent and infectious periods. Biometrika. 67, 191-198. 554 [49] Bailey NT. 1964 Some stochastic models for small epidemics in large populations. J. 555 R. Stat. Soc. Ser. C. Appl. Stat. 13, 9-19. 556 [50] Bartlett MS. 1953 Stochastic processes or the statistics of change. J. R. Stat. Soc. Ser. 557 C. Appl. Stat. 2, 44-64. 558 [51] Gillespie DT. 1977 Exact stochastic simulation of coupled chemical reactions. J. Phys. 559 Chem. 81, 2340-2361. 560 [52] Newman ME. 2003 The structure and function of complex networks. SIAM Rev. Daley DJ and Vere-Jones D. 2007 An introduction to the theory of point processes: 563 volume II: general theory and structure. Springer Science & Business Media. 564 [54] White LF and Pagano M. 2008 A likelihood-based method for real-time estimation of 565 the serial interval and reproductive number of an epidemic. Stat. Med. 27, 2999-3016. 566 [55] King AA, de Celles MD, Magpantay FM, and Rohani P. 2015 Avoidable errors in the 567 modelling of outbreaks of emerging pathogens, with special reference to Ebola. Proc. 568 R. Soc. B. 282, 20150347. 569 [56] Leskovec J, Kleinberg J, and Faloutsos C. 2007 Graph evolution: Densification and 570 shrinking diameters. ACM T. Knowl. Discov. D. 1, 2.', 'latex': None, 'type': 'figure'}, 'TABREF0': {'text': 'Here, we show that the egocentric relationship between r and R derived by Trapman (19) (see the original source for detailed derivations) matches what would be calculated by applying the Euler-Lotka equation to the egocentric (rather than the intrinsic) generation-interval', 'latex': None, 'type': 'table'}, 'TABREF1': {'text': \"To simulate epidemics on a realistic network, we use the 'condensed matter physics'  network from the Stanford Large Network Dataset Collection (30). This graph describes coauthorship among anyone who submitted a paper to Condensed Matter category in the arXiv between January 1993 and April 2003 (56). It consists of 23133 nodes and 93497 edges. The 427 [28] Ball F and Neal P. 2002 A general model for stochastic SIR epidemics with two levels of mixing. Math. Biosci. 180, 73-102. [29] Champredon D, Dushoff J, and Earn DJD. 2018 Equivalence of the Erlang-distributed SEIR epidemic model and the renewal equation. SIAM J. Appl. Math. 78, 3258-3278. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/683326 doi: bioRxiv preprint lection. http://snap.stanford.edu/data. [31] McBryde E, Bergeri I, van Gemert C, Rotty J, Headley E, Simpson K, Lester R, Hellard 507 M, and Fielding JE. 2009 Early transmission characteristics of influenza A (H1N1) v in Australia: Victorian state, 16 May-3 June 2009. Euro Surveill. 14, 19363.[32] Roberts MG and Nishiura H. 2011 Early estimation of the reproduction number in the presence of imported cases: pandemic influenza H1N1-2009 in New Zealand. PLoS One. 512 [33] Trichereau J, Verret C, Mayet A, Manet G, Decam C, Meynard JB, Deparis X, and Migliani R. 2012 Estimation of the reproductive number for A (H1N1) pdm09 influenza among the French armed forces, September 2009-March 2010. J. Infect. 64, 628-630. [34] Nishiura H and Chowell G. 2015 Theoretical perspectives on the infectiousness of Ebola 516 virus disease. Theor. Biol. Med. Model. 12, 1. [35] Szendroi B and Csányi G. 2004 Polynomial epidemics and clustering in contact networks.\", 'latex': None, 'type': 'table'}}, 'back_matter': []}\n",
      "{'paper_id': 'b1b9d81726e56c2781ce6b63ae5015cf2f314c6f', 'metadata': {'title': 'Estimating the Efficacy of Quarantine and Traffic Blockage for the Epidemic Caused by 2019-nCoV (COVID-19)：A Simulation Analysis', 'authors': [{'first': 'Deqiang', 'middle': [], 'last': 'Li', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Southeast University', 'location': {'postCode': '210096', 'settlement': 'Nanjing', 'country': 'China'}}, 'email': 'deqiang_li@seu.edu.cn'}, {'first': 'Zhicheng', 'middle': [], 'last': 'Liu', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Southeast University', 'location': {'postCode': '210096', 'settlement': 'Nanjing', 'country': 'China'}}, 'email': 'zhichengliu@seu.edu.cn'}, {'first': 'Qinghe', 'middle': [], 'last': 'Liu', 'suffix': '', 'affiliation': {}, 'email': 'liuqinghe@seu.edu.cn'}, {'first': 'Zefei', 'middle': [], 'last': 'Gao', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Southeast University', 'location': {'postCode': '210096', 'settlement': 'Nanjing', 'country': 'China'}}, 'email': 'zf_gao@seu.edu.cn'}, {'first': 'Junkai', 'middle': [], 'last': 'Zhu', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Southeast University', 'location': {'postCode': '210096', 'settlement': 'Nanjing', 'country': 'China'}}, 'email': 'junkai_zhu@seu.edu.cn'}, {'first': 'Junyan', 'middle': [], 'last': 'Yang', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Southeast University', 'location': {'postCode': '210096', 'settlement': 'Nanjing', 'country': 'China'}}, 'email': ''}, {'first': 'Qiao', 'middle': [], 'last': 'Wang', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Southeast University', 'location': {'postCode': '210096', 'settlement': 'Nanjing', 'country': 'China'}}, 'email': 'qiaowang@seu.edu.cn'}, {'first': 'Q', 'middle': [], 'last': 'Wang', 'suffix': '', 'affiliation': {}, 'email': ''}]}, 'abstract': [{'text': 'Background: Since the 2019-nCoV outbreaks in Wuhan, China, the cumulative number of confirmed cases is increasing every day, and a large number of populations all over the world are at risk. The quarantine and traffic blockage can alleviate the risk of the epidemic and the infections, henceforth evaluating the efficacy of such actions is essential to inform policy makers and raise the public awareness of the importance of self-isolation and quarantine.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'We collected confirmed case data and the migration data, and introduced the quarantine factor and traffic blockage factor to the Flow-SEIR model. By varying the quarantine factor and traffic blockage factor, we simulated the change of the peak number and arrival time of infections, then the efficacy of these two intervation measures can be analyzed in our simulation. In our study, the self-protection at home is also included in quarantine.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'In the simulated results, the quarantine and traffic blockage are effective for epidemic control. For Hubei province, the current quarantine factor is estimaed to be 0.405, which means around 40.5% of suceptibles who are close contacting with are in quarantine, and the current traffic blockage factor is estimaed to be 0.66, which indicates around 34% of suceptibles who had flowed out from Hubei. For the other provinces outside Hubei, the current quarantine factor is estimated to be 0.285, and the current traffic blockage factor is estimated to be 0.26. With the quarantine and traffic blockage factor increasing, the number of infections decrease dramatically. We also simulated the start dates of quarantine and traffic blockage at four time points, the simulated results show that the early of warning is also effective for epidemic containing.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'However, provincial level traffic blockage can only alleviate 21.06% -22.38% of the peak number of infections. In general, the quarantine is much more effective than the traffic blockage control.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}], 'body_text': [{'text': 'Since the 2019-nCoV (COVID- 19) outbreaks in Wuhan, China, the cumulative number of confirmed cases is increasing every day, and a large number of populations all over the world are at risk. The recent study [1] [2] [3] [4] on the efficacy of traffic blockage for the COVID-19 indicated that the the population flows will certainly increase the cumulative number of cases and the quarantine and traffic blockage will lower this risk.', 'cite_spans': [{'start': 28, 'end': 31, 'text': '19)', 'ref_id': 'BIBREF19'}, {'start': 208, 'end': 211, 'text': '[1]', 'ref_id': 'BIBREF0'}, {'start': 212, 'end': 215, 'text': '[2]', 'ref_id': 'BIBREF1'}, {'start': 216, 'end': 219, 'text': '[3]', 'ref_id': 'BIBREF2'}, {'start': 220, 'end': 223, 'text': '[4]', 'ref_id': 'BIBREF4'}], 'ref_spans': [], 'section': 'Background'}, {'text': 'On January 24, 2020, J. M. Read et al [5] applied the transmission model and found that the reproduction number of infection is 3.8, and 72% -75% of the transmission needed to be restricted in order to inhibit the growing infection rate; In addition, they found that traffic blockage only slowed the spread of the epidemic by 24.9%.', 'cite_spans': [{'start': 38, 'end': 41, 'text': '[5]', 'ref_id': 'BIBREF5'}], 'ref_spans': [], 'section': 'Background'}, {'text': 'On January 27, 2020, B. Tang et al [6] proposed a deterministic compartmental model based on clinical progression of the epidemic. They showed that the reproduction number of infection is 6 .47 and the intervention measures can reduce the reproductive number of infection and the risk of transmission. The sensitivity results show that the number of infections in Beijing will decrease by 91.14% in 7 days with the intervation of travel.', 'cite_spans': [{'start': 35, 'end': 38, 'text': '[6]', 'ref_id': 'BIBREF6'}, {'start': 188, 'end': 189, 'text': '6', 'ref_id': 'BIBREF6'}], 'ref_spans': [], 'section': 'Background'}, {'text': 'On Feb.4, 2020, S. Ai et al [7] investigated both the effectiveness of closure of Wuhan and that of restricting the flow of people in and out the provinces (cities) in China.', 'cite_spans': [{'start': 28, 'end': 31, 'text': '[7]', 'ref_id': 'BIBREF7'}], 'ref_spans': [], 'section': 'Background'}, {'text': 'On Feb.9, 2020, X. Li et al [8] simulated the dynamics of the 2019-nCOV (COVID- 19) , on account of the strict control measures in Wuhan and Beijing. They showed that after the the traffic blockage on Jan.23,2020, the reproductive number in Wuhan will be 2.5, significantly lower than before.', 'cite_spans': [{'start': 28, 'end': 31, 'text': '[8]', 'ref_id': 'BIBREF8'}, {'start': 80, 'end': 83, 'text': '19)', 'ref_id': 'BIBREF19'}], 'ref_spans': [], 'section': 'Background'}, {'text': 'Unlike our previous work [9] on H7N9 avian influenza which was a retrospective analysis, we are now doing predictive analysis of COVID-19 in an emergency. Beside our Flow-SEIHR model based predition for the trends of COVID-19 [10] , this paper will focus on quantifying the quarantine factor and the traffic blockage factor such that we may numerically evaluate the effects of various policies. Specifically, we apply a Flow-SEIR model, a simplified version of Flow-SEIHR model proposed in [10] , which may enable us to perform a wide range of assessments nationwide.', 'cite_spans': [{'start': 25, 'end': 28, 'text': '[9]', 'ref_id': 'BIBREF9'}, {'start': 226, 'end': 230, 'text': '[10]', 'ref_id': 'BIBREF10'}, {'start': 490, 'end': 494, 'text': '[10]', 'ref_id': 'BIBREF10'}], 'ref_spans': [], 'section': 'Background'}, {'text': 'The outbreaks of COVID-19 occured around the Chinese Spring Festival, when a huge number of populatin flows in and out the origin outbreak area, Wuhan, which resulted in that the epidemic was severe in mainland of China and worldwide. The policy of traffic blockage was executed since Janurary 23 rd , 2020, after then the chance of cross-infection decreased sharply. Here we make the following assumptions regrading to the status quo of the epidemic and the SARS empirical parameters.', 'cite_spans': [], 'ref_spans': [], 'section': 'Methods'}, {'text': 'A1: The outbreak of COVID-19 started on the beginning of Dec., 2019.', 'cite_spans': [], 'ref_spans': [], 'section': 'Methods'}, {'text': 'A2: The flows in domestic provinces are concluded, the abroad flow-in is not considered. A3: Only the susceptible with no clinical symptoms can flow in and out, the exposed and infections will be quarantined.', 'cite_spans': [], 'ref_spans': [], 'section': 'Methods'}, {'text': 'A4: There is no super spreaders, in other words, the chance of infected was equal to those who are contact with the infections.', 'cite_spans': [], 'ref_spans': [], 'section': 'Methods'}, {'text': 'A5: The COVID-19 has weak seasonality in its transmission, unlike influenza.', 'cite_spans': [], 'ref_spans': [], 'section': 'Methods'}, {'text': 'A6: The recovery rate is time-dependent and will linearly increase.', 'cite_spans': [], 'ref_spans': [], 'section': 'Methods'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Methods'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.14.20022913 doi: medRxiv preprint A7: The empirical parameters on SARS will also work on COVID-19.', 'cite_spans': [], 'ref_spans': [], 'section': 'Methods'}, {'text': 'A8: Most people will follow the official instructions and stay at home after the traffic blockage.', 'cite_spans': [], 'ref_spans': [], 'section': 'Methods'}, {'text': 'We denote S, E, I, R,F in the flow-SEIR model [10] and the parameters as shown in Table 1 . In addition, we introduce the traffic blockage factor τ , and the quarantine factor ε , which stands for the quantifications of traffic blockage and quarantine, respectively. Since the people was urged to stay at home, this self-protection is also regarded as quarantine.', 'cite_spans': [{'start': 46, 'end': 50, 'text': '[10]', 'ref_id': 'BIBREF10'}], 'ref_spans': [{'start': 82, 'end': 89, 'text': 'Table 1', 'ref_id': 'TABREF0'}], 'section': 'Methods'}, {'text': 'Basically, we follow the standard SEIR [11] [12] [13] model to evaluate the state of infections. However, since the epidemic of COVID-19 appeared around the Chinese Spring Festival, the nationwide traffic should be modeled by the traffic network flow, in order to characterize the huge number of population flows in and out all provinces (cities), which formed the flow-SEHIR model in [10] . Here we simplify it into a Flow-SEIR model, and then estimate the influence of the migration and the efficacy of traffic blockage. The Flow-SEIR model is:', 'cite_spans': [{'start': 39, 'end': 43, 'text': '[11]', 'ref_id': 'BIBREF11'}, {'start': 44, 'end': 48, 'text': '[12]', 'ref_id': 'BIBREF12'}, {'start': 49, 'end': 53, 'text': '[13]', 'ref_id': 'BIBREF13'}, {'start': 385, 'end': 389, 'text': '[10]', 'ref_id': 'BIBREF10'}], 'ref_spans': [], 'section': 'Flow-SEIR model'}, {'text': 'where all the variables are presented in Table 1 . The migration data can represent the quantification of the susceptibles flow function F in the first equation of (1). In practice, we collect the migration data from the Baidu Migration API, which is available online. More precisely, the susceptibles flow to the ith province from other areas can be expressed as follows:', 'cite_spans': [], 'ref_spans': [{'start': 41, 'end': 48, 'text': 'Table 1', 'ref_id': 'TABREF0'}], 'section': 'Flow-SEIR model'}, {'text': 'where ji δ is the ratio of migration population to the whole population independent of the traffic blockage. In equation (2), the traffic blockage factor ߬ is introduced, which ranges from 0 to 1. Here ߬ tends to 0 implies free, and 1 blockage. According to our assumptions, the transmission with quarantine can be rewritten as:', 'cite_spans': [], 'ref_spans': [], 'section': 'Flow-SEIR model'}, {'text': 'is the number of susceptibles who are independent of quarantine, and k is a constant which will be estimated, ܴ is basic reproductive number [12, 13] . As the epidemic grows fastly, the public was aware of cross-infection, so the quarantine factor ε will increase to 1, thus ܳ is in decline gradually.', 'cite_spans': [{'start': 141, 'end': 145, 'text': '[12,', 'ref_id': 'BIBREF12'}, {'start': 146, 'end': 149, 'text': '13]', 'ref_id': 'BIBREF13'}], 'ref_spans': [], 'section': 'Flow-SEIR model'}, {'text': 'In appendix 1, all the parameters concerned in equaration (1) can be estimatied by fitting the confirmed data (cf. [16, 17] ). The practical considerations are also discussed there in appendix 1.', 'cite_spans': [{'start': 115, 'end': 119, 'text': '[16,', 'ref_id': 'BIBREF16'}, {'start': 120, 'end': 123, 'text': '17]', 'ref_id': 'BIBREF17'}], 'ref_spans': [], 'section': 'Flow-SEIR model'}, {'text': \"All the data we used is available online. The data of confirmed cases is released by the National Health Commission of China and each provinces' health commissions, and the migration data is from Baidu Migration API. For data source websites, please see Appendix 2\", 'cite_spans': [], 'ref_spans': [], 'section': 'Data source:'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Data source:'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.14.20022913 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'Data source:'}, {'text': 'Both the quarantine factors and traffic blockage factors are taken between 0 to 1. In Fig 2, the current quarantine factor of Hubei province is estimaed to be 0.405, which means around 40.5% of suceptibles who are close contacting with are in quarantine. If we increase the factor by 0.01, the peak number of infections will decrease by 5.53%. For the other provinces outside Hubei, the current quarantine factor is estimated to be 0.285, if we incease the factor by 0.01, the peak number of infections will decrease by 14.64%. On the other hand, when ߝ is close to 1, all the population is isolated from others, the peak value will decrease by 89.68%. However, when ߝ tends to 0, all the population flows are control free, and the peak value will increase by 20.4%.', 'cite_spans': [], 'ref_spans': [{'start': 86, 'end': 92, 'text': 'Fig 2,', 'ref_id': 'FIGREF1'}], 'section': 'The efficacy of quarantine and traffic blockage'}, {'text': 'As shown in Fig 3, the peak number of cases decrease as the traffic blockage factor increases. For Hubei province, the current traffic blockage factor is estimaed to be 0.66, which indicates around 34% of suceptibles who flowed out from Hubei, if we increase the factor by 0.1, the peak number of infections will decrease by 2.14%. For the provinces except Hubei, the current traffic blockage factor is estimated to be 0.26, if we incease the factor by 0.1, the peak number of infections will decrease by 4.01%. When the traffic blockage factor ߬ is close to 1, the target area is almost completely in blockage, the peak value will decrease by 21.06% -22.39% to the case that ߬ tends to 0. Clearly, self-isolation and quarantine is more effective than traffic blockage.', 'cite_spans': [], 'ref_spans': [{'start': 12, 'end': 18, 'text': 'Fig 3,', 'ref_id': 'FIGREF2'}], 'section': 'The efficacy of quarantine and traffic blockage'}, {'text': 'We estimated the trend of confirmed cases with quarantine at different time periods. As shown in Fig 4, the quarantine factor ߝ increase by 0.01, the peak arrival time in Hubei will arrive 0-2 days earlier, and the peak arrival time of other provinces outside Hubei will arrive 1-3 days earlier.', 'cite_spans': [], 'ref_spans': [{'start': 97, 'end': 103, 'text': 'Fig 4,', 'ref_id': 'FIGREF3'}], 'section': 'The efficacy of earlier warning of COVID-19'}, {'text': 'For Hubei Province, if the quarantine is 1 week delay, the peak value will increase by 8.50% -10.95%. Oppsitely, if the quarantine is 1 week earlier, the peak value will decrease by 24.26% -25.33%, and 2 weeks by 57.10% -57.46%. For the other provinces, if the quarantine is 1 week of delay, the peak value will increase by 20.92% -22.77%, but 1 (or 2, resp.) week earlier will decrease by 32.78% -35.60% (or 63.75% -67.13%, resp.).', 'cite_spans': [], 'ref_spans': [], 'section': 'The efficacy of earlier warning of COVID-19'}, {'text': 'Concretely, we estimated the trends of epidemic, assume that the traffic blockage started on Jan.', 'cite_spans': [], 'ref_spans': [], 'section': 'The efficacy of earlier warning of COVID-19'}, {'text': '9, Jan. 16, Jan. 23 and Jan. 30, respectively. In this experiment, we assumed that the traffic flow between provinces would be completely cut off. As shown in Fig 5, when the traffic blockage factor ߬ increase by 0.1 for Hubei, the peak arrival time will be earlier and the peak number of cases will be lower. If the traffic blockage is one week earlier, the peak number of cases only decreased by 3.10% -3.79%, and two weeks by 6.64% -8.08%. This illustrates that in the eve of city closure in Wuhan, the epidemic in Hubei province had been severe, and the transmission rate was quite high. However, in other provinces outside Hubei, if the traffic blockage is one week earlier, the peak number of cases will decrease by 15.30% -16.70%, and two weeks earlier, it will decrease by 31.74% -33.09% .', 'cite_spans': [], 'ref_spans': [{'start': 159, 'end': 165, 'text': 'Fig 5,', 'ref_id': 'FIGREF4'}], 'section': 'The efficacy of earlier warning of COVID-19'}, {'text': 'Finally, we estimated the daily change of number of cases nationwide. If there is one day earlier of traffic blockage, about 3,600 cases will eventually be reduced in the mainland of China, but one day delay might increase 1,800 cases.', 'cite_spans': [], 'ref_spans': [], 'section': 'The efficacy of earlier warning of COVID-19'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'The efficacy of earlier warning of COVID-19'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.14.20022913 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'The efficacy of earlier warning of COVID-19'}, {'text': 'In this study, we have used a Flow-SEIR model to numerically estimate the efficacy of the quarantine and traffic blockage. We introduced both the the quarantine factor and the traffic blockage factor, then the flow variable and transmission rate in the model will vary as the change of these two intervation factors, then the effectiveness of the traffic blockage and quarantine can be simulated and evaluated.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusions'}, {'text': 'In the simulated results, we conclude that if the masses take protective measures, the peak number of cases will decrease greatly by 89.68%. However, if the masses completely ignored self-isolation, the peak is predicted to increase by 20.40%. And if the external population flows to the target area is almost completely blocked, the peak number will decrease by 21.06%-22.38%.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusions'}, {'text': 'Earlier warning, timely traffic blockage and quarantine measures are extremely effective, especially for areas with slight epidemic situations. The experimental results also show that if there is one day earlier of traffic blockage, about 3,600 cases will eventually be reduced in the mainland China, and one day delay will result in additional 1,800 cases at risk.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusions'}, {'text': 'Applying the latest migration population data from Baidu Migration, and varying the traffic blockage factor, we can use the Flow-SEIR model to simulate the number of infections and then intuitively understand the efficacy of the traffic blockage.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'However, our study has several major limitations. First, the accuracy of results can be improved provided the source of data is more reliable and sufficient. Since in Hubei the medical resources are overburdened and the methods to diagnose the COVID-9 are not accurate, even the diagnosis changed several times, the underdiagnosis and misdiagnosis are invetiable, then the confirmed cases are underestimated. Secondly, there is heterogenity in transmission, it is not added in our study, and we also did not district the age distribution in our model.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'The study has numerically estimated the efficacy of traffic blockage and quarantine. We confirm by simulated quantification evidence that the traffic blockage is effective in controlling the epidemic of COVID-19, and the quarantine is a more effective way to help the public to prevent cross-infection. With the development of the epidemic, we have to admit the fact that there are still a number of population who are exposed to the COVID-19 without clinical symptoms [18] [19] [20] . Those population may carry the COVID-19 for several days and might infect others who are closely contacting with, which makes the potentional risk. Thus we encourage the people to be self-protective until the epidemic is totally under control.', 'cite_spans': [{'start': 469, 'end': 473, 'text': '[18]', 'ref_id': 'BIBREF18'}, {'start': 474, 'end': 478, 'text': '[19]', 'ref_id': 'BIBREF19'}, {'start': 479, 'end': 483, 'text': '[20]', 'ref_id': 'BIBREF20'}], 'ref_spans': [], 'section': 'Discussion'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.14.20022913 doi: medRxiv preprint , the x-axis is date, the y-axis is the number of confirmed cases.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.14.20022913 doi: medRxiv preprint 1 0 ߬ is set to take effect on four time: January 9, January 16, January 23, January 30. The x-axis is the date and the y-axis is the number of patients.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.14.20022913 doi: medRxiv preprint 1 Those who are exposed and being recovered.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'ε Rate of susceptibles who are in quarantine τ Rate of susceptibles who flowed out . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.14.20022913 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'According to our the assumption A1, firstly we estimated the number of potential suceptibles by the iteration process. We define the loss function as follows:', 'cite_spans': [], 'ref_spans': [], 'section': 'APPENDIX 1 Parameters estimation'}, {'text': 'where N is the length of the real data, which in practical is the count of days.', 'cite_spans': [], 'ref_spans': [], 'section': 'APPENDIX 1 Parameters estimation'}, {'text': '\\u202bݖ\\u202c is the nth real data value, and n=1,2,…,N , \\u202bݖ\\u202c is the nth predicted value, based on the model. We fitted the confirmed data from Jan.12 to Feb. 7 to get the optimal number of potential suceptibles by minimizing the loss value. The loss value is small enough to make the estimated results fitting the real data well. The parameters in the Flow-SEIR model can also be estimated in the same way.', 'cite_spans': [], 'ref_spans': [], 'section': 'APPENDIX 1 Parameters estimation'}, {'text': 'For the problem of high death rate in Hubei Province, we reduced the ratio of ', 'cite_spans': [], 'ref_spans': [], 'section': 'APPENDIX 1 Parameters estimation'}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'Early dynamics of transmission and control of 2019-nCoV: a mathematical modelling study', 'authors': [{'first': 'A', 'middle': ['J'], 'last': 'Kucharski', 'suffix': ''}, {'first': 'T', 'middle': ['W'], 'last': 'Russell', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Diamond', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.01.31.20019901']}}, 'BIBREF1': {'ref_id': 'b1', 'title': 'Risk assessment of novel coronavirus 2019-nCoV outbreaks outside China', 'authors': [{'first': 'P', 'middle': [], 'last': 'Boldog', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Tekeli', 'suffix': ''}, {'first': 'Z', 'middle': [], 'last': 'Vizi', 'suffix': ''}], 'year': 2002, 'venue': '', 'volume': '2020', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.02.04']}}, 'BIBREF2': {'ref_id': 'b2', 'title': 'Nowcasting and forecasting the potential domestic and international spread of the 2019-nCoV outbreak originating in Wuhan, China: a modelling study', 'authors': [{'first': 'J', 'middle': ['T'], 'last': 'Wu', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Leung', 'suffix': ''}, {'first': 'G', 'middle': ['M'], 'last': 'Leung', 'suffix': ''}], 'year': 2020, 'venue': 'The Lancet', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF4': {'ref_id': 'b4', 'title': 'Protective Population Behavior Change in Outbreaks of Emerging Infectious Disease', 'authors': [{'first': 'E', 'middle': ['K'], 'last': 'Lodge', 'suffix': ''}, {'first': 'A', 'middle': ['M'], 'last': 'Schatz', 'suffix': ''}, {'first': 'J', 'middle': ['M'], 'last': 'Drake', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '2020', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.01.27.921536']}}, 'BIBREF5': {'ref_id': 'b5', 'title': 'Novel coronavirus 2019-nCoV: early estimation of epidemiological parameters and epidemic predictions', 'authors': [{'first': 'J', 'middle': ['M'], 'last': 'Read', 'suffix': ''}, {'first': 'J', 'middle': ['R'], 'last': 'Bridgen', 'suffix': ''}, {'first': 'D', 'middle': ['A'], 'last': 'Cummings', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '2020', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.01.23.20018549']}}, 'BIBREF6': {'ref_id': 'b6', 'title': 'Estimation of the Transmission Risk of the 2019-nCoV and Its Implication for Public Health Interventions', 'authors': [{'first': 'B', 'middle': [], 'last': 'Tang', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'Q', 'middle': [], 'last': 'Li', 'suffix': ''}], 'year': 2020, 'venue': 'Journal of Clinical Medicine', 'volume': '9', 'issn': '2', 'pages': '', 'other_ids': {}}, 'BIBREF7': {'ref_id': 'b7', 'title': 'The lockdown of Hubei Province causing different transmission dynamics of the novel coronavirus (2019-nCoV) in Wuhan and Beijing', 'authors': [{'first': 'X', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Zhao', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Sun', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '2020', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.02.09.20021477']}}, 'BIBREF8': {'ref_id': 'b8', 'title': 'Population movement, city closure and spatial tran smission of the 2019-nCoV infection in China', 'authors': [{'first': 'S', 'middle': [], 'last': 'Ai', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Zhu', 'suffix': ''}, {'first': 'F', 'middle': [], 'last': 'Tian', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '2020', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.02.04.20020339']}}, 'BIBREF9': {'ref_id': 'b9', 'title': 'Effects of closing and reopening live poultry markets on the epidemic of human infection with avian influenza A virus', 'authors': [{'first': 'J', 'middle': [], 'last': 'Lu', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Liu', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Xia', 'suffix': ''}], 'year': 2016, 'venue': 'Journal of biomedical research', 'volume': '30', 'issn': '2', 'pages': '', 'other_ids': {}}, 'BIBREF10': {'ref_id': 'b10', 'title': 'Assessing the Tendency of 2019-nCoV (COVID-19) Outbreak in China', 'authors': [{'first': 'Q', 'middle': [], 'last': 'Liu', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'Z', 'middle': [], 'last': 'Liu', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '2020', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.02.09.20021444']}}, 'BIBREF11': {'ref_id': 'b11', 'title': 'Seasonality and period-doubling bifurcations in an epidemic model', 'authors': [{'first': 'J', 'middle': ['L'], 'last': 'Aron', 'suffix': ''}, {'first': 'I', 'middle': ['B'], 'last': 'Schwartz', 'suffix': ''}], 'year': 1984, 'venue': 'Journal of theoretical biology', 'volume': '110', 'issn': '4', 'pages': '665--79', 'other_ids': {}}, 'BIBREF12': {'ref_id': 'b12', 'title': 'Global dynamics of an SEIR epidemic model with vertical transmission', 'authors': [{'first': 'M', 'middle': ['Y'], 'last': 'Li', 'suffix': ''}, {'first': 'H', 'middle': ['L'], 'last': 'Smith', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Wang', 'suffix': ''}], 'year': 2001, 'venue': 'SIAM Journal on Applied Mathematics', 'volume': '62', 'issn': '1', 'pages': '58--69', 'other_ids': {}}, 'BIBREF13': {'ref_id': 'b13', 'title': 'Forecasting epidemics through nonparametric estimation of time-dependent transmission rates using the SEIR model', 'authors': [{'first': 'A', 'middle': [], 'last': 'Smirnova', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Chowell', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': '', 'suffix': ''}], 'year': 2019, 'venue': 'Bulletin of mathematical biology', 'volume': '81', 'issn': '11', 'pages': '4343--65', 'other_ids': {}}, 'BIBREF14': {'ref_id': 'b14', 'title': 'A new framework and software to estimate time-varying reproduction numbers during epidemics', 'authors': [{'first': 'A', 'middle': [], 'last': 'Cori', 'suffix': ''}, {'first': 'N', 'middle': ['M'], 'last': 'Ferguson', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Fraser', 'suffix': ''}], 'year': 2013, 'venue': 'American journal of epidemiology', 'volume': '178', 'issn': '9', 'pages': '1505--1517', 'other_ids': {}}, 'BIBREF15': {'ref_id': 'b15', 'title': 'Estimating in real time the efficacy of 8 measures to control emerging communicable diseases', 'authors': [{'first': 'S', 'middle': [], 'last': 'Cauchemez', 'suffix': ''}, {'first': 'P-Y', 'middle': [], 'last': 'Boëlle', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Thomas', 'suffix': ''}], 'year': 2006, 'venue': 'American journal of epidemiology', 'volume': '164', 'issn': '6', 'pages': '591--598', 'other_ids': {}}, 'BIBREF16': {'ref_id': 'b16', 'title': 'Model parameters and outbreak control for SARS', 'authors': [{'first': 'G', 'middle': [], 'last': 'Chowell', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Castillo-Chavez', 'suffix': ''}, {'first': 'P', 'middle': ['W'], 'last': 'Fenimore', 'suffix': ''}], 'year': 2004, 'venue': 'Emerging Infectious Diseases', 'volume': '10', 'issn': '7', 'pages': '', 'other_ids': {}}, 'BIBREF17': {'ref_id': 'b17', 'title': 'Mathematical models in population biology and epidemiology', 'authors': [{'first': 'F', 'middle': [], 'last': 'Brauer', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Castillo-Chavez', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Castillo-Chavez', 'suffix': ''}], 'year': 2012, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF18': {'ref_id': 'b18', 'title': 'Data-Based Analysis, Modelling and Forecasting of the novel Coronavirus (2019-nCoV) outbreak', 'authors': [{'first': 'C', 'middle': [], 'last': 'Anastassopoulou', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Russo', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Tsakris', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '2020', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.02.11.20022186']}}, 'BIBREF19': {'ref_id': 'b19', 'title': 'Epidemic size of novel coronavirus-infected pneumonia in the Epicenter Wuhan: using data of five-countries&#039; evacuation action', 'authors': [{'first': 'H', 'middle': [], 'last': 'Zhao', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Man', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Wang', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '2020', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.02.12.20022285']}}, 'BIBREF20': {'ref_id': 'b20', 'title': 'Estimating number of global importations of COVID-19 from Wuhan, risk of transmission outside mainland China and COVID-19 introduction index between countries outside mainland China', 'authors': [{'first': 'H', 'middle': [], 'last': 'Sun', 'suffix': ''}, {'first': 'B', 'middle': ['L'], 'last': 'Dickens', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Chen', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '2020', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.02.17.20024075']}}}, 'ref_entries': {'FIGREF0': {'text': 'Flow-SEIR model, based on the SEIR model, was proposed to to estimate the epidemic trend with a large number of population flowing in and out all provinces of China during the Chinese Spring Festival. And the efficacy of traffic blockage can be estimated in this model.', 'latex': None, 'type': 'figure'}, 'FIGREF1': {'text': 'Confirmed cases under differentߝ, the x-axis is date, the y-axis is number of confirmed cases.', 'latex': None, 'type': 'figure'}, 'FIGREF2': {'text': 'Confirmed cases under different ߬', 'latex': None, 'type': 'figure'}, 'FIGREF3': {'text': 'The quarantine factor ߝ is set to take effect on four time: January 23, January 30, Feburary 6, Feburary 13. The x-axis is the date and the y-axis is the number of patients.', 'latex': None, 'type': 'figure'}, 'FIGREF4': {'text': 'The traffic blockage factor', 'latex': None, 'type': 'figure'}, 'FIGREF5': {'text': 'are infectious and being recovered.', 'latex': None, 'type': 'figure'}, 'FIGREF7': {'text': 'For the situation that the medical resouces are limited in Wuhan, the number of infections will be larger than the released data, we enlarged the ratio of is data latency problem, because the epidemic has an incubation period and the infections can not be diagnosed timely. For the data latency problem, it can be assumed that the number of confirmed cases published on one day is the number of confirmed cases ݊ days ago, and ݊ is the number of delayed days in the ith province, Hubei, due to the above factors, the latency will be relatively long, which may reach 5-7 days. For other provinces, if patients can get treatment in time, the delay should only include the time waiting for the result of the diagnostic reagents, is about 1-2 day.', 'latex': None, 'type': 'figure'}, 'TABREF0': {'text': 'Description of the variables of the model and control factors Variable Description Number of those susceptible Number of those exposed Number of those infective Number of those recovered Population flows in and out Rate of those who are susceptible and being infectedRate of those who are exposed and being infectious.', 'latex': None, 'type': 'table'}}, 'back_matter': []}\n",
      "{'paper_id': 'b1ce6ca5960cb799403188ca303a364428f812d8', 'metadata': {'title': 'Direct nucleic acid analysis of mosquitoes for high fidelity species identification and 1 detection of Wolbachia using a cellphone 2 3', 'authors': [{'first': 'Sanchita', 'middle': [], 'last': 'Bhadra', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'The University of Texas', 'location': {}}, 'email': 'sanchitabhadra@utexas.edu'}, {'first': 'Timothy', 'middle': ['E'], 'last': 'Riedel', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'The University of Texas at Austin', 'location': {}}, 'email': ''}, {'first': 'Miguel', 'middle': ['A'], 'last': 'Saldaña', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Texas Medical Branch', 'location': {'settlement': 'Galveston'}}, 'email': ''}, {'first': 'Shivanand', 'middle': [], 'last': 'Hegde', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Texas Medical Branch', 'location': {'postCode': '77555', 'settlement': 'Galveston', 'region': 'TX', 'country': 'USA'}}, 'email': ''}, {'first': 'Nicole', 'middle': [], 'last': '', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': '', 'middle': [], 'last': 'Pederson', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'The University of Texas at Austin', 'location': {}}, 'email': ''}, {'first': 'Grant', 'middle': ['L'], 'last': 'Hughes', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Texas', 'location': {'postCode': '77555', 'settlement': 'Galveston', 'region': 'TX', 'country': 'USA'}}, 'email': ''}, {'first': 'Andrew', 'middle': ['D'], 'last': 'Ellington', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'The University of Texas', 'location': {}}, 'email': ''}]}, 'abstract': [{'text': 'Manipulation of natural mosquito populations using the endosymbiotic bacteria Wolbachia is being 21 investigated as a novel strategy to reduce the burden of mosquito-borne viruses. To evaluate the 22 efficacy of these interventions, it will be critical to determine Wolbachia infection frequencies in 23', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'Aedes aegypti mosquito populations. However, current diagnostic tools are not well-suited to fit this 24 need. Morphological methods cannot identify Wolbachia, immunoassays often suffer from low 25 sensitivity and poor throughput, while PCR and spectroscopy require complex instruments and 26 technical expertise, which restrict their use to centralized laboratories. To address this unmet need, 27', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'we have used loop-mediated isothermal amplification (LAMP) and oligonucleotide strand 28 displacement (OSD) probes to create a one-pot sample-to-answer nucleic acid diagnostic platform 29 for vector and symbiont surveillance. LAMP-OSD assays can directly amplify target nucleic acids 30 from macerated mosquitoes without requiring nucleic acid purification and yield specific single 31 endpoint yes/no fluorescence signals that are observable to eye or by cellphone camera. We 32 demonstrate cellphone-imaged LAMP-OSD tests for two targets, the Aedes aegypti cytochrome 33 oxidase I (coi) gene and the Wolbachia surface protein (wsp) gene, and show a limit of detection 34 of 4 and 40 target DNA copies, respectively. In a blinded test of 90 field-caught mosquitoes, the 35 coi LAMP-OSD assay demonstrated 98% specificity and 97% sensitivity in identifying Ae. aegypti 36 mosquitoes even after 3 weeks of storage without desiccant at 37 °C. Similarly, the wsp LAMP-37 OSD assay readily identified the wAlbB Wolbachia strain in field-collected Aedes albopictus 38 mosquitoes without generating any false positive signals. Modest technology requirements, 39 minimal execution steps, simple binary readout, and robust accuracy make the LAMP-OSD-to-40 cellphone assay platform well suited for field vector surveillance in austere or resource-limited 41 conditions. 42 43 All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'Mosquitoes spread many human pathogens and novel approaches are required to reduce the 45 burden of mosquito-borne disease. One promising approach is transferring Wolbachia into 46', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'Aedes aegypti mosquitoes where it blocks transmission of arboviruses like dengue, Zika and 47', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'Yellow fever viruses and spreads through mosquito populations. For effective evaluation of this 48 approach, regular surveillance of Wolbachia infections in Ae. aegypti is required, but current 49 diagnostic tools are not well suited to support these critical surveillance needs. To fill this need 50', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'we developed a simple, robust and inexpensive assay to identify Ae. aegypti mosquitoes and 51', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'Wolbachia using our unique one-pot assay platform, LAMP-OSD, which uses loop-mediated 52 isothermal amplification to amplify nucleic acid targets at a single temperature. Unlike other 53 LAMP-based tests, our assays assure accuracy by coupling amplification with novel nucleic acid 54 strand displacement (OSD) probes that hybridize to specific sequences in LAMP amplification 55 products and thereby generate simple yes/no readout of fluorescence readable by human eye 56 and by off-the-shelf cellphones. To facilitate field use, we developed our assays so they are 57 compatible with crushed mosquito homogenate as the template, meaning no nucleic acid 58 extraction is required. In blinded tests using field collected mosquitoes, LAMP-OSD-cellphone 59 tests performed robustly to identify 29 of 30 Ae. aegypti even after 3 weeks of storage at 37 °C 60 while producing only one false positive out of 60 non-specific mosquitoes. Similarly, our assay 61 could identify Wolbachia in field-caught Aedes albopictus without producing any false positives. 62', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'Our easy to use and easy to interpret assays should facilitate widespread field mosquito 63 surveillance with minimal instrumentation and high accuracy. 64 65 66 All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/291849 doi: bioRxiv preprint diseases [1]. Traditionally considered a problem for tropical regions, mosquitoes are increasingly 69 becoming a global public health challenge [2, 3] due to a changing global environment, urbanization, 70 increases in the global movement of populations, and the emergence of insecticide resistance [4]. 71', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': \"Estimates suggest nearly half the world's population is at risk for mosquito-borne diseases [5, 6], 72 and as such, there is an urgent need for novel approaches to reduce the burden of disease.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/291849 doi: bioRxiv preprint 4 traditional morphological-based approaches cannot detect associated symbionts or pathogens. 91 These limitations restrict widespread accessibility and necessitate sample preservation and 92 transport. On the other end of the spectrum, immunoassay-based tools for identifying pathogen-93 infected mosquitoes, such as VecTest TM dipsticks (Medical Analysis Systems Inc.), are portable 94 and inexpensive. However, these tests have poor sensitivity [23-25] and are not necessarily 95 available for distinguishing mosquito species or identifying Wolbachia endosymbionts. 96 Nucleic acid tests can provide the necessary sensitivity and versatility for identifying both 97 Wolbachia and mosquito species. However, since molecular testing is currently heavily reliant on 98 PCR [26-28], opportunities for field-based determinations are limited, leading to significant delays 99 and gaps in actionable surveillance. To support widespread vector surveillance inexpensive, 100', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'portable, nucleic acid diagnostic platforms are needed that rapidly produce accurate results without 101 requiring complex procedures, instruments, and laboratory infrastructure. In this regard, isothermal 102 nucleic acid amplification assays such as loop-mediated isothermal amplification (LAMP) have 103 begun to be employed because they do not require complex thermocycling instruments [29][30][31]. 104', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'However, although LAMP can rival PCR for sensitivity it often produces spurious amplicons, which 105 in turn lead to false positive readouts with non-specific reporters such as Mg 2+ precipitation or 106 fluorescent dye intercalation [32][33][34]. 107', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'To mitigate the spurious signals that arise with LAMP, we have previously applied principles that 108', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': \"were developed for nucleic acid strand exchange circuits [35][36][37][38] to the design of short hemiduplex 109 oligonucleotide strand displacement (OSD) probes for LAMP [39]. The single stranded 'toehold' 110 regions of OSD probes bind to LAMP amplicon loop sequences, and then signal via strand 111 exchange [40] that leads to separation of a fluorophore and quencher [39]. OSDs are the 112 functional equivalents of TaqMan probes and can specifically report single or multiplex LAMP 113 amplicons without interference from non-specific nucleic acids or inhibitors [39, 41]. OSDs 114 significantly enhance the diagnostic applicability of LAMP, allowing it to match the allelic specificity 115 All rights reserved. No reuse allowed without permission.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}], 'body_text': [{'text': 'One biocontrol countermeasure gaining traction for mosquito control is the release of Wolbachia-74 infected mosquitoes [7] [8] [9] . Wolbachia is a maternally-transmitted endosymbiont that can rapidly 75 become established in the natural mosquito populations and can inhibit a variety of pathogens, 76 including arboviruses, malaria parasites, and filarial nematodes [10] [11] [12] [13] [14] [15] . Wolbachia control 77 strategies are currently being deployed into the field to alter the capacity of Aedes aegypti to transmit 78 arboviruses or to suppress mosquito populations [16] [17] [18] . Surveillance of transinfected mosquitoes 79 as well as natural vector populations is crucial to evaluate the efficacy of these interventions [19] . 80', 'cite_spans': [{'start': 119, 'end': 122, 'text': '[7]', 'ref_id': None}, {'start': 123, 'end': 126, 'text': '[8]', 'ref_id': None}, {'start': 127, 'end': 130, 'text': '[9]', 'ref_id': None}, {'start': 367, 'end': 371, 'text': '[10]', 'ref_id': None}, {'start': 372, 'end': 376, 'text': '[11]', 'ref_id': None}, {'start': 377, 'end': 381, 'text': '[12]', 'ref_id': None}, {'start': 382, 'end': 386, 'text': '[13]', 'ref_id': None}, {'start': 387, 'end': 391, 'text': '[14]', 'ref_id': None}, {'start': 392, 'end': 396, 'text': '[15]', 'ref_id': None}, {'start': 577, 'end': 581, 'text': '[16]', 'ref_id': None}, {'start': 582, 'end': 586, 'text': '[17]', 'ref_id': None}, {'start': 587, 'end': 591, 'text': '[18]', 'ref_id': None}, {'start': 735, 'end': 739, 'text': '[19]', 'ref_id': None}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'However, most current screening methods rely on PCR, which is expensive and relies on 81 laboratory facilities. In addition to screening for Wolbachia infection, it would also be desirable to 82 identify the host mosquito species using these assays since different mosquito species differ in 83 their ability to transmit pathogens [20] . Knowledge of vector species, and prevalence and stability 84 of Wolbachia is essential for effective vector control and pre-emption of disease outbreaks with 85 public health measures [21] . 86', 'cite_spans': [{'start': 331, 'end': 335, 'text': '[20]', 'ref_id': None}, {'start': 522, 'end': 526, 'text': '[21]', 'ref_id': None}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'Unfortunately, mosquitoes are most commonly identified using morphological taxonomic keys. 87', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'This process can be tedious, and requires highly trained personnel and undamaged mosquitoes. 88', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'Alternative morphological methods such as the identification of morphometric wing characters 89', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': '[22] are low throughput and require microscopes and complex imaging instruments. Moreover, 90 were selected and maintained in an E. coli Top10 strain. Plasmid minipreps were prepared from 142 these strains using the Qiagen miniprep kit (Qiagen, Valencia, CA, USA). All target inserts were 143 verified by sequencing at the Institute of Cellular and Molecular Biology Core DNA Sequencing 144', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'Facility. 145', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'Wolbachia wAlbB and wPip strain wsp genes and Ae. aegypti coi gene sequences were obtained 147 from NCBI GenBank. Consensus signature sequences were derived following MUSCLE 148 The Primer Explorer v5 primer design software (Eiken Chemical Co., Japan) was used for 154 generating several potential LAMP primer sets composed of the outer primers F3 and B3 and the 155 inner primers FIP and BIP. Primer design was constrained to include at least a 40 bp gap between 156 the F1 and F2 or between the B1 and B2 priming sites. Primer specificity for targeted sequence 157 and a corresponding lack of significant cross-reactivity to other nucleic acids of human, vector or 158 pathogenic origin were further assessed using NCBI BLAST. 159', 'cite_spans': [], 'ref_spans': [], 'section': 'LAMP primer and OSD probe design 146'}, {'text': 'These primer sets were functionally tested in LAMP assays using zero to several hundred copies 160 of purified plasmids as templates. Amplification kinetics were measured in real time using the 161 All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'LAMP primer and OSD probe design 146'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/291849 doi: bioRxiv preprint fluorogenic intercalating dye Evagreen and the LightCycler 96 real-time PCR machine (Roche). 162', 'cite_spans': [], 'ref_spans': [], 'section': 'LAMP primer and OSD probe design 146'}, {'text': 'The fastest primer sets that detected the fewest template copies with negligible spurious reactivity 163 in the absence of templates were selected for further assay development. 164', 'cite_spans': [], 'ref_spans': [], 'section': 'LAMP primer and OSD probe design 146'}, {'text': 'Fluorogenic OSD probes were then designed to undergo toehold-mediated strand exchange with 165 these Ae. aegypti coi and the Wolbachia wsp LAMP amplicons. Of the two target derived loop 166 regions (between the F1c and F2, and the B1c and B2 primer binding sites) the regions between 167', 'cite_spans': [], 'ref_spans': [], 'section': 'LAMP primer and OSD probe design 146'}, {'text': 'F1c and F2 were chosen as wsp and coi OSD binding regions. Fluorogenic OSD probes were 168 designed using the NUPACK software and our previously described engineering principles [39] . 169', 'cite_spans': [{'start': 178, 'end': 182, 'text': '[39]', 'ref_id': None}], 'ref_spans': [], 'section': 'LAMP primer and OSD probe design 146'}, {'text': \"Briefly the hemiduplex OSDs were designed to display 11-12 nucleotide long single-stranded 170 toeholds on the longer, fluorophore-labeled strands. All free 3'-OH ends were blocked with 171 inverted dT to prevent extension by DNA polymerase. HCl, pH 7.5:0.1 mM EDTA, pH 8.0) immediately prior to use. Zero to several hundred copies of 181 synthetic plasmid templates were added to the LAMP reaction mixes followed by 90 min of 182 incubation at 65 °C. 183\", 'cite_spans': [], 'ref_spans': [], 'section': 'LAMP primer and OSD probe design 146'}, {'text': 'All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Real-time detection of LAMP amplicons using intercalating dyes 184'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/291849 doi: bioRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'Real-time detection of LAMP amplicons using intercalating dyes 184'}, {'text': 'LAMP reactions monitored in real time using OSD probes were assembled and analyzed as 195 above with the following changes. First, OSD probes were prepared by annealing 1 µM of the 196 fluorophore-labeled OSD strand with 5 µM of the quencher-labeled strand in 1X Isothermal buffer. Annealing was performed by denaturing the oligonucleotide mix at 95 °C for 1 min followed by 198 slow cooling at the rate of 0.1 °C/s to 25 °C. Excess annealed probe was stored at -20 °C. 199', 'cite_spans': [], 'ref_spans': [], 'section': 'Real-time detection of LAMP amplicons using OSD probes 194'}, {'text': 'Annealed OSD probes were added to the LAMP reactions at a final concentration of 100 nM of 200 the fluorophore-bearing strand. 201', 'cite_spans': [], 'ref_spans': [], 'section': 'Real-time detection of LAMP amplicons using OSD probes 194'}, {'text': 'LAMP-OSD assays intended for visual readout and smartphone imaging were assembled in 0.2 203 ml optically clear thin-walled tubes with low auto-fluorescence (Axygen, Union City, CA, USA). 204', 'cite_spans': [], 'ref_spans': [], 'section': 'Endpoint visual readout and smartphone imaging of LAMP-OSD assays 202'}, {'text': 'Following 90 min of amplification at 65 °C, LAMP-OSD reactions were incubated at 95 °C for 1 205 min followed by immediate transfer to room temperature and fluorescence imaging. Images were 206 acquired using an unmodified iPhone 6 and an UltraSlim-LED transilluminator (Syngene, 207', 'cite_spans': [], 'ref_spans': [], 'section': 'Endpoint visual readout and smartphone imaging of LAMP-OSD assays 202'}, {'text': 'Frederick, MD, USA). In some experiments, our previously described in-house 3D-printed imaging 208 All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Endpoint visual readout and smartphone imaging of LAMP-OSD assays 202'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/291849 doi: bioRxiv preprint insectary at the University of Texas medical Branch, Galveston, TX, USA. Four to seven day old 217 mosquitoes were collected and immediately frozen for shipment, storage and subsequent testing. 218', 'cite_spans': [], 'ref_spans': [], 'section': 'Endpoint visual readout and smartphone imaging of LAMP-OSD assays 202'}, {'text': 'To obtain blood fed insects, Aedes mosquitoes were starved for a period of 24 hours then offered 219 a sheep blood meal (Colorado Serum Company, Denver, CO, USA) using a hemotek membrane 220 system (Hemotek). Unfed mosquitoes were separated and mosquitoes that were engorged were 221 collected 24 hours post feeding and processed in the same manner as unfed mosquitoes. For 222 field collections, female mosquitoes were trapped using Fay prince trap (John W. Hock) baited 223 with CO2 in Galveston, Texas. 90 mosquitoes were morphologically identified and sorted into 224 three blinded groups that were stored at -20 °C, 4 °C and 37 °C, respectively for up to 3 weeks 225 prior to LAMP-OSD analysis. 226', 'cite_spans': [], 'ref_spans': [], 'section': 'Endpoint visual readout and smartphone imaging of LAMP-OSD assays 202'}, {'text': 'For LAMP-OSD analysis individual mosquitoes were prepared either in 1 cc syringes or in 1.5 ml 228 microcentrifuge tubes as follows. In-syringe preparation: The plunger was removed from a 1 cc 229 syringe and a 0.5 µM pore size 1/8 th inch diameter frit (catalog # 59037, Sigma-Aldrich, St. Louis, 230 MO, USA) was placed inside the syringe. A single mosquito was placed on top of the frit and 231 macerated thoroughly using the syringe plunger. 100 µl of water was aspirated into the syringe to 232 All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'LAMP-OSD analysis of mosquitoes 227'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/291849 doi: bioRxiv preprint fully re-suspend the macerated mosquito prior to evicting this mosquito-containing water from the 233 syringe into a microcentrifuge collection tube. A 2 µl aliquot of this sample was directly tested by 234 LAMP-OSD assays. In-tube preparation: A single mosquito was placed in a 1.5 ml microcentrifuge 235 tube and manually macerated using a disposable micropestle (Fisherbrand™ RNase-Free 236', 'cite_spans': [], 'ref_spans': [], 'section': 'LAMP-OSD analysis of mosquitoes 227'}, {'text': 'Disposable Pellet Pestles, Cat # 12-141-364, Fisher Scientific, Hampton, NH, USA). Each 237 macerated mosquito was resuspended in 100 µl water. A 2 µl 1:10 diluted aliquot of this mosquito 238 sample was directly assessed by LAMP-OSD analysis. 239', 'cite_spans': [], 'ref_spans': [], 'section': 'LAMP-OSD analysis of mosquitoes 227'}, {'text': \"The paired results of morphological identification and LAMP-OSD analysis were compared using 241 2x2 contingency tables. Sensitivity or true positive rate was calculated by using the formula 242 The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. With a single endpoint visual 'yes/no' readout of OSD fluorescence (either directly observed or 277 imaged using cellphone camera), the Ae. aegypti coi LAMP-OSD assay could reliably identify 278 the presence of as few as 4 copies of synthetic target DNA (Fig 1) . Similarly, the cellphone-279 imaged wsp LAMP-OSD assay produced bright visible fluorescence when presented with only 280 All rights reserved. No reuse allowed without permission.\", 'cite_spans': [], 'ref_spans': [{'start': 540, 'end': 547, 'text': '(Fig 1)', 'ref_id': 'FIGREF5'}], 'section': 'Statistical analysis 240'}, {'text': \"The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/291849 doi: bioRxiv preprint 40 copies of its target sequence. In the absence of target DNA, neither assay generated 281 spurious signal. 282 Our next goal was to demonstrate the ability of these LAMP-OSD assays to detect naturally 290 occurring target sequences in mosquitoes. At the same time, we wanted to ensure that 291 minimally processed samples would be compatible with our detection platform in order to 292 facilitate rapid in-field vector testing with fewest instruments and user-required steps. 293 Therefore, as an initial approach, we developed the 'in-syringe' method for rapid sample 294 preparation wherein individual mosquitoes were crushed inside 1 cc syringes using the syringe 295 plunger as a pestle. A small chromatography column frit placed inside the syringe served as a 296 pedestal that aided maceration and removed larger particulates when the macerated sample 297 was re-suspended in water and recovered. Small portions (up to 8% of a LAMP-OSD reaction) 298 of these macerated samples were added directly to LAMP-OSD reactions, which were then 299 incubated for 90 min at 65 °C to initiate and sustain amplification. 300\", 'cite_spans': [], 'ref_spans': [], 'section': 'Statistical analysis 240'}, {'text': 'Endpoint visual examination of these assays for the presence or absence of OSD fluorescence 301 revealed that our visually read LAMP-OSD system is compatible with direct analysis of crudely 302 All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Statistical analysis 240'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/291849 doi: bioRxiv preprint albopictus and Cu. quinquefasciatus mosquitoes, which are naturally infected with wAlbB and 307 wPip Wolbachia, respectively, but remained negative in the presence of unrelated Wolbachia 308 wMel and uninfected mosquitoes (Fig 2 and S4 Fig) . These results indicate that the Wolbachia wsp and Ae. aegypti coi LAMP-OSD assays are able 325 to specifically amplify and signal the presence of their target DNA directly from crudely crushed 326 All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [{'start': 368, 'end': 386, 'text': '(Fig 2 and S4 Fig)', 'ref_id': 'FIGREF10'}], 'section': 'Statistical analysis 240'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/291849 doi: bioRxiv preprint Furthermore, the large burden of non-specific nucleic acids as well as other molecular and 328 macroscopic components present in a crude mosquito sample did not compromise signal 329 accuracy. We also confirmed the absence of significant inhibition of amplification and signaling 330 by recapitulating the detection limit of synthetic DNA targets in a background of crude non-331 specific mosquito sample. The coi and wsp LAMP-OSD assays could detect 4 and 40 target 332 copies, respectively, even in the presence of 8% reaction volume of crude mosquito sample (S5 333 blood meal could be directly analyzed by visual LAMP-OSD without diminution of signal to noise 342 ratio (Fig 3) . 343 The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/291849 doi: bioRxiv preprint served as controls for monitoring background fluorescence. LAMP-OSD fluorescence was 350 imaged at endpoint using a cellphone. 351 albopictus, and Ochlerotatus species. The mosquitoes were divided into three groups of 30 356 individuals that were stored without desiccant at -20 °C, 4 °C, or 37 °C for 1, 2, or 3 weeks prior 357 to testing. To reduce mosquito processing cost, footprint, and time for this large study, we 358 further simplified sample preparation requirements by optimizing the \"in-tube\" mosquito 359 preparation method wherein each mosquito was crushed with a micropestle directly in a 360 microcentrifuge tube followed by resuspension in water and introduction in a LAMP-OSD 361', 'cite_spans': [], 'ref_spans': [{'start': 820, 'end': 827, 'text': '(Fig 3)', 'ref_id': 'FIGREF9'}], 'section': 'Statistical analysis 240'}, {'text': 'The visually read coi LAMP-OSD assay demonstrated an overall sensitivity (true positive rate) 363 of 97% and specificity (true negative rate) of 98% when compared to morphological typing of 364 field-caught mosquito species (Fig 4, S6, S7, S8) . On closer inspection of the data, it is evident 365 that even after three weeks of mosquito collection and storage at temperatures as high as 37 °C 366 the coi LAMP-OSD assay was correctly able to identify 29 out of 30 Ae. aegypti mosquitoes. 367', 'cite_spans': [], 'ref_spans': [{'start': 224, 'end': 243, 'text': '(Fig 4, S6, S7, S8)', 'ref_id': 'FIGREF10'}], 'section': 'reaction. 362'}, {'text': 'The single mosquito that the LAMP-OSD assay failed to identify had been stored at 37 °C for a 368', 'cite_spans': [], 'ref_spans': [], 'section': 'reaction. 362'}, {'text': 'week prior to testing. We ruled out lack of amplifiable nucleic acids or their incompatibility with 369 coi LAMP primers and OSD probe by PCR amplifying the relevant coi LAMP target and verifying 370 its sequence. 371 372 All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'reaction. 362'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/291849 doi: bioRxiv preprint mosquitoes stored at -20 °C for as long as 3 weeks. PCR analysis of the wsp-negative mosquito 394 using previously described primers (81F and 691R) and protocols [28] did not produce 395 amplicons suggesting that this individual was likely uninfected or had Wolbachia levels below 396 All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'reaction. 362'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/291849 doi: bioRxiv preprint the levels detectable by PCR. As the storage temperature was increased the frequency of also failed to produce wsp PCR amplicons. Since, 95-99% of Ae. albopictus mosquitoes in the 401 wild are typically found to be infected with Wolbachia can logically integrate multiple analytes; transform nucleic acids to glucose and human chorionic 443 gonadotrophin (hCG); adapt readout to beads, paperfluidics, glucose meters, pregnancy test 444 All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'reaction. 362'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/291849 doi: bioRxiv preprint exploited to engineer yes/no distinctions between strain-specific single nucleotide polymorphisms 450', 'cite_spans': [], 'ref_spans': [], 'section': 'reaction. 362'}, {'text': \"[39], and the same dual wsp assay could be rendered strain-specific by simply substituting an 451 OSD reporter specific to an alternate polymorphic loop sequence (S3 Fig) . By using our one-pot LAMP-OSD assay, macerated mosquito homogenates could be directly 455 analyzed and 'yes/no' visual readouts could be quickly ascertained with a cell phone in the field 456 without the requirement for laboratory equipment or technically training. Moreover, since our 457 assays can accurately analyze mosquitoes several days after capturethe coi LAMP-OSD assay 458 could for example identify mosquitoes after 3 weeks at 37 °C without desiccantmosquitoes from 459 remote collection outposts can potentially be analyzed even after delayed retrieval. The flexibility 460 of assay timing is further accommodated by the fact that lyophilized LAMP-OSD reaction mixes 461 that can be stored and deployed without cold chain [42] . These combined features make our 462 assay platform the best tool to date for expanding vector surveillance to resource poor settings 463\", 'cite_spans': [{'start': 908, 'end': 912, 'text': '[42]', 'ref_id': None}], 'ref_spans': [{'start': 162, 'end': 170, 'text': '(S3 Fig)', 'ref_id': 'FIGREF9'}], 'section': 'reaction. 362'}, {'text': '[72], especially in that the ease of use should allow minimally trained citizen scientists to 464 participate in otherwise sophisticated public health monitoring operations in the field. 465', 'cite_spans': [], 'ref_spans': [], 'section': 'reaction. 362'}, {'text': 'The development efforts that we have put into LAMP-OSD should now allow it to be generalized 466 to screening other microbes or mosquito phenotypes in field settings. For example, LAMP-based 467 assays have been developed to identify pathogens transmitted by mosquitoes and insecticide 468 resistance alleles [73] [74] [75] [76] , but these rely on purified nucleic acid as templates and non-specific 469 All rights reserved. No reuse allowed without permission.', 'cite_spans': [{'start': 309, 'end': 313, 'text': '[73]', 'ref_id': 'BIBREF31'}, {'start': 314, 'end': 318, 'text': '[74]', 'ref_id': None}, {'start': 319, 'end': 323, 'text': '[75]', 'ref_id': None}, {'start': 324, 'end': 328, 'text': '[76]', 'ref_id': None}], 'ref_spans': [], 'section': 'reaction. 362'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/291849 doi: bioRxiv preprint unique sequence-specific strand exchange probes. In addition, this technology could be used to 471 identify gut microbes and insect-specific viruses associated with mosquitoes, which is of growing 472 interest given that it is becoming clear that the microbiome can shape vector competence for 473 human pathogens [77] [78] [79] . Overall, we have demonstrated a versatile nucleic acid diagnostic 474 platform for rapid and accurate analyses of both insect vectors and symbionts, and that can now 475 be further configured for additional applications. Wolbachia symbiosis for vector and disease control. Acta Trop 132: S150-S163. Morphometric Wing Characters as a Tool for Mosquito Identification. Plos One 11. 540 All rights reserved. No reuse allowed without permission.', 'cite_spans': [{'start': 460, 'end': 464, 'text': '[77]', 'ref_id': None}, {'start': 465, 'end': 469, 'text': '[78]', 'ref_id': None}, {'start': 470, 'end': 474, 'text': '[79]', 'ref_id': None}], 'ref_spans': [], 'section': 'reaction. 362'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. and wAlbB Wolbachia-infected Ae. albopictus mosquitoes by coi and wsp LAMP-OSD assays, respectively. 2 µL of mosquito samples were either added directly to LAMP-OSD assays or subjected to 10-fold serial dilution in water prior to introduction in LAMP-OSD reactions. LAMP-OSD reactions without any macerated mosquito samples served as negative controls. Assays lacking LAMP primers but containing macerated mosquito samples served as controls for monitoring background fluorescence. (C) Specificity analysis of coi and wsp LAMP-OSD assays. The coi LAMP-OSD assay was challenged with 2 µL of Ae. albopictus in-syringe prepared crude mosquito sample. Similarly, the wsp LAMP-OSD assay was challenged with Ae. aegypti sample. In all experiments, OSD fluorescence was imaged at endpoint using a cellphone. Individual normal or blood-fed Ae. aegypti and Ae. albopictus mosquitoes were prepared by the in-syringe method. 2 µL of these mosquito samples were analyzed by coi and wsp LAMP-OSD assays. LAMP-OSD reactions without any macerated mosquito samples served as negative controls. Assays lacking LAMP primers but containing macerated mosquito samples served as controls for monitoring background fluorescence. LAMP-OSD fluorescence was imaged at endpoint using a cellphone.', 'cite_spans': [], 'ref_spans': [], 'section': 'reaction. 362'}, {'text': 'All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'reaction. 362'}, {'text': \"The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/291849 doi: bioRxiv preprint Mosquitoes stored for up to three weeks at the indicated temperatures were prepared by 'in-tube' crude processing. 2 µL of a 1:10 dilution of each mosquito sample was analyzed by wsp and coi LAMP-OSD assays. OSD fluorescence was imaged at endpoint using a cellphone. (A) Schematic depicting 'in-tube' method for rapid preparation of field-caught mosquitoes for LAMP-OSD analysis. (B) Comparison of coi LAMP-OSD results with morphological identification using 2X2 contingency table. (C) Tabulation of coi and wsp LAMP-OSD readout for each mosquito in the study. Ae. aegypti, Ae. albopictus and Ochlerotatus species are highlighted in blue, orange and white, respectively. 'Y' indicates a positive LAMP-OSD signal (bright fluorescence). 'N' indicates absence of any fluorescence signal in the LAMP-OSD reaction. Red boxes highlight LAMP-OSD tests that generated false positive or false negative outcomes.\", 'cite_spans': [], 'ref_spans': [], 'section': 'reaction. 362'}, {'text': 'All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'reaction. 362'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/291849 doi: bioRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'reaction. 362'}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'WHO (2014) A global brief on vector-borne diseases', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF1': {'ref_id': 'b1', 'title': 'Mosquito-borne 486 disease surveillance by the European Centre for Disease Prevention and Control', 'authors': [{'first': 'H', 'middle': [], 'last': 'Zeller', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Marrama', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Sudre', 'suffix': ''}, {'first': 'Van', 'middle': [], 'last': 'Bortel', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': '', 'suffix': ''}, {'first': 'Warns-Petit', 'middle': [], 'last': '', 'suffix': ''}, {'first': 'E', 'middle': [], 'last': '', 'suffix': ''}], 'year': 2013, 'venue': 'Clin', 'volume': '487', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF3': {'ref_id': 'b3', 'title': 'The global 489 distribution of the arbovirus vectors Aedes aegypti and Ae. albopictus. Elife 4. 490 44', 'authors': [{'first': 'Mug', 'middle': [], 'last': 'Kraemer', 'suffix': ''}, {'first': 'M', 'middle': ['E'], 'last': 'Sinka', 'suffix': ''}, {'first': 'K', 'middle': ['A'], 'last': 'Duda', 'suffix': ''}, {'first': 'Aqn', 'middle': [], 'last': 'Mylne', 'suffix': ''}, {'first': 'F', 'middle': ['M'], 'last': 'Shearer', 'suffix': ''}], 'year': 2009, 'venue': 'Nat Methods', 'volume': '6', 'issn': '', 'pages': '343--345', 'other_ids': {}}, 'BIBREF4': {'ref_id': 'b4', 'title': 'Basic local alignment 600 search tool', 'authors': [{'first': 'S', 'middle': ['F'], 'last': 'Altschul', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Gish', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Miller', 'suffix': ''}, {'first': 'E', 'middle': ['W'], 'last': 'Myers', 'suffix': ''}, {'first': 'D', 'middle': ['J'], 'last': 'Lipman', 'suffix': ''}], 'year': 1990, 'venue': 'J Mol Biol', 'volume': '215', 'issn': '', 'pages': '403--410', 'other_ids': {}}, 'BIBREF5': {'ref_id': 'b5', 'title': 'Molecular identification of 607 mosquitoes (Diptera: Culicidae) in southeastern Australia', 'authors': [{'first': 'S', 'middle': ['F'], 'last': 'Altschul', 'suffix': ''}, {'first': 'T', 'middle': ['L'], 'last': 'Madden', 'suffix': ''}, {'first': 'A', 'middle': ['A'], 'last': 'Schaffer', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'Z', 'middle': [], 'last': 'Zhang', 'suffix': ''}], 'year': 1997, 'venue': 'Nucleic Acids 603 Res', 'volume': '25', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF6': {'ref_id': 'b6', 'title': 'BOLD: The Barcode of Life Data System 610 (www.barcodinglife.org)', 'authors': [{'first': 'S', 'middle': [], 'last': 'Ratnasingham', 'suffix': ''}, {'first': 'Pdn', 'middle': [], 'last': 'Hebert', 'suffix': ''}], 'year': 2006, 'venue': 'Med Vet Entomol', 'volume': '7', 'issn': '', 'pages': '413--424', 'other_ids': {}}, 'BIBREF7': {'ref_id': 'b7', 'title': 'Identifying the Main Mosquito 614 Species in China Based on DNA Barcoding', 'authors': [{'first': 'G', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'C', 'middle': ['X'], 'last': 'Li', 'suffix': ''}, {'first': 'X', 'middle': ['X'], 'last': 'Guo', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Xing', 'suffix': ''}, {'first': 'Y', 'middle': ['D'], 'last': 'Dong', 'suffix': ''}], 'year': 2012, 'venue': 'Plos One', 'volume': '7', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF8': {'ref_id': 'b8', 'title': 'Size of Blood Meals of Aedes-Albopictus and Culex-Tritaeniorhynchus 624 (Diptera, Culicidae) Feeding on an Unrestrained Dog Infected with Dirofilaria-Immitis 625 (Spirurida, Filariidae)', 'authors': [{'first': 'E', 'middle': [], 'last': 'Konishi', 'suffix': ''}], 'year': 1989, 'venue': 'J Med Entomol', 'volume': '26', 'issn': '', 'pages': '535--538', 'other_ids': {}}, 'BIBREF9': {'ref_id': 'b9', 'title': 'An initial survey for Wolbachia', 'authors': [{'first': 'J', 'middle': ['L'], 'last': 'Rasgon', 'suffix': ''}, {'first': 'T', 'middle': ['W'], 'last': 'Scott', 'suffix': ''}], 'year': 2004, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF10': {'ref_id': 'b10', 'title': 'Rickettsiaceae) infections in selected california mosquitoes (Diptera : Culicidae)', 'authors': [], 'year': None, 'venue': 'J Med', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF13': {'ref_id': 'b13', 'title': 'Wolbachia infections in southeast Asian mosquitoes (Diptera : Culicidea)', 'authors': [], 'year': None, 'venue': 'J Med', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF16': {'ref_id': 'b16', 'title': 'Wolbachia in Aedes albopictus and Their Effects on Chikungunya Virus', 'authors': [], 'year': None, 'venue': 'Am J Trop Med', 'volume': '634', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF18': {'ref_id': 'b18', 'title': 'Non-destructive Detection and Identification of Two Strains of Wolbachia in Aedes 637 aegypti by Near-Infrared Spectroscopy', 'authors': [{'first': 'A', 'middle': [], 'last': 'Khoshmanesh', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Christensen', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Perez-Guaita', 'suffix': ''}, {'first': 'I', 'middle': [], 'last': 'Iturbe-Ormaetxe', 'suffix': ''}, {'first': 'O', 'middle': [], 'last': 'Neill', 'suffix': ''}, {'first': 'S', 'middle': ['L'], 'last': '', 'suffix': ''}], 'year': 2017, 'venue': 'Screening of Wolbachia Endosymbiont Infection in Aedes aegypti Mosquitoes', 'volume': '10', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF20': {'ref_id': 'b20', 'title': 'Loop-mediated isothermal amplification 649 (LAMP) for rapid identification of Anopheles gambiae and Anopheles arabiensis 650 mosquitoes', 'authors': [{'first': 'M', 'middle': [], 'last': 'Bonizzoni', 'suffix': ''}, {'first': 'Afrane', 'middle': [], 'last': '', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Yan', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': '', 'suffix': ''}], 'year': 2009, 'venue': 'Am J Trop Med Hyg', 'volume': '81', 'issn': '', 'pages': '1030--1034', 'other_ids': {}}, 'BIBREF21': {'ref_id': 'b21', 'title': 'Rapid molecular 652 assays for the detection of yellow fever virus in low-resource settings', 'authors': [{'first': 'C', 'middle': [], 'last': 'Escadafal', 'suffix': ''}, {'first': 'O', 'middle': [], 'last': 'Faye', 'suffix': ''}, {'first': 'A', 'middle': ['A'], 'last': 'Sall', 'suffix': ''}, {'first': 'Faye', 'middle': ['O'], 'last': 'Weidmann', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': '', 'suffix': ''}], 'year': 2014, 'venue': 'PLoS Negl Trop', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF22': {'ref_id': 'b22', 'title': 'Molecular logic and computing', 'authors': [], 'year': 2007, 'venue': 'Nature', 'volume': '655', 'issn': '2', 'pages': '399--410', 'other_ids': {}}, 'BIBREF23': {'ref_id': 'b23', 'title': 'Diagnostic applications of nucleic acid circuits', 'authors': [{'first': 'C', 'middle': [], 'last': 'Jung', 'suffix': ''}, {'first': 'A', 'middle': ['D'], 'last': 'Ellington', 'suffix': ''}], 'year': 2014, 'venue': 'Acc', 'volume': '657', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF26': {'ref_id': 'b26', 'title': 'Molecular Diagnostics: Coupling Isothermal Amplification and Strand Exchange Circuits 660 to', 'authors': [], 'year': None, 'venue': 'Glucometers. Sci Rep', 'volume': '5', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF27': {'ref_id': 'b27', 'title': 'Rational, modular adaptation of enzyme-free DNA 662 circuits to multiple detection methods', 'authors': [{'first': 'B', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'A', 'middle': ['D'], 'last': 'Ellington', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Chen', 'suffix': ''}], 'year': 2011, 'venue': 'Nucleic Acids Res', 'volume': '39', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF28': {'ref_id': 'b28', 'title': 'DNA circuits as amplifiers 664 for the detection of nucleic acids on a paperfluidic platform', 'authors': [{'first': 'P', 'middle': ['B'], 'last': 'Allen', 'suffix': ''}, {'first': 'S', 'middle': ['A'], 'last': 'Arshad', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'Chen', 'middle': ['X'], 'last': 'Ellington', 'suffix': ''}, {'first': 'A', 'middle': ['D'], 'last': '', 'suffix': ''}], 'year': 2012, 'venue': 'Lab Chip', 'volume': '12', 'issn': '', 'pages': '2951--2958', 'other_ids': {}}, 'BIBREF29': {'ref_id': 'b29', 'title': 'No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/291849']}}, 'BIBREF31': {'ref_id': 'b31', 'title': 'Loop-mediated 675 isothermal amplification applied to filarial parasites detection in the mosquito vectors', 'authors': [{'first': 'H', 'middle': [], 'last': 'Aonuma', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Yoshimura', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Perera', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Shinzawa', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Bando', 'suffix': ''}], 'year': 2009, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF32': {'ref_id': 'b32', 'title': 'Dirofilaria immitis as a study model. Parasites & Vectors 2', 'authors': [], 'year': None, 'venue': '', 'volume': '677', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF34': {'ref_id': 'b34', 'title': 'Plasmodium-carrying mosquitoes using loop-mediated isothermal amplification', 'authors': [], 'year': None, 'venue': 'Biochem', 'volume': '679', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF36': {'ref_id': 'b36', 'title': 'A multiplex microfluidic loop-681 mediated isothermal amplification array for detection of malaria-related parasites and 682 vectors', 'authors': [{'first': 'R', 'middle': [], 'last': 'Mao', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Ge', 'suffix': ''}, {'first': 'Z', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'R', 'middle': ['Z'], 'last': 'Hao', 'suffix': ''}, {'first': 'G', 'middle': ['H'], 'last': 'Zhang', 'suffix': ''}], 'year': 2018, 'venue': 'Acta Trop', 'volume': '178', 'issn': '', 'pages': '86--92', 'other_ids': {}}, 'BIBREF37': {'ref_id': 'b37', 'title': 'Detection of mutation by allele-684 specific loop-mediated isothermal amplification (AS-LAMP)', 'authors': [{'first': 'H', 'middle': [], 'last': 'Aonuma', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Badolo', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Okado', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Kanuka', 'suffix': ''}], 'year': 2013, 'venue': 'Methods Mol Biol', 'volume': '1039', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF38': {'ref_id': 'b38', 'title': 'The microbiome modulates arbovirus 687 transmission in mosquitoes', 'authors': [{'first': 'S', 'middle': [], 'last': 'Hegde', 'suffix': ''}, {'first': 'J', 'middle': ['L'], 'last': 'Rasgon', 'suffix': ''}, {'first': 'Gl ;', 'middle': [], 'last': 'Hughes', 'suffix': ''}, {'first': 'R', 'middle': ['G'], 'last': 'Saraiva', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Kang', 'suffix': ''}, {'first': 'M', 'middle': ['L'], 'last': 'Simoes', 'suffix': ''}, {'first': 'Y', 'middle': ['I'], 'last': 'Anglero-Rodriguez', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Dimopoulos', 'suffix': ''}], 'year': 2015, 'venue': 'Current Opinion in Virology', 'volume': '15', 'issn': '', 'pages': '97--102', 'other_ids': {}}, 'BIBREF39': {'ref_id': 'b39', 'title': 'Mosquito gut antiparasitic and antiviral immunity', 'authors': [], 'year': None, 'venue': 'Dev Comp Immunol', 'volume': '64', 'issn': '', 'pages': '53--64', 'other_ids': {}}, 'BIBREF40': {'ref_id': 'b40', 'title': 'No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/291849']}}, 'BIBREF42': {'ref_id': 'b42', 'title': 'Insect-specific viruses and their potential impact on arbovirus transmission', 'authors': [{'first': 'N', 'middle': [], 'last': 'Vasilakis', 'suffix': ''}, {'first': 'R', 'middle': ['B'], 'last': 'Tesh', 'suffix': ''}], 'year': 2015, 'venue': 'Current Opinion in Virology', 'volume': '15', 'issn': '', 'pages': '69--74', 'other_ids': {}}, 'BIBREF43': {'ref_id': 'b43', 'title': 'LAMP-OSD schematic. LAMP uses 2 inner (FIP and BIP) and 2 outer (F3 and B3) 697 primers specific to 6 blocks of target sequences designated as B3, B2, B1, F1c, F2c and F3c. F2 698 sequence in FIP (F1c-F2) initiates amplification by Bst DNA polymerase', 'authors': [], 'year': None, 'venue': 'S2 Fig', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF44': {'ref_id': 'b44', 'title': 'F1c sequence in FIP self-primes subsequent amplification. Similarly, BIP (B1c-B2) initiates DNA 700 synthesis by binding to B2c. F3 and B3 primer-initiated DNA synthesis displaces preceding inner 701 primer-initiated strands, which serve as templates for primer-initiated strand displacement DNA 702 synthesis', 'authors': [], 'year': None, 'venue': '3′-ends of the resulting single-stranded', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF45': {'ref_id': 'b45', 'title': 'Inner primers hybridize to the 704 single-stranded loops and initiate another round of strand displacement synthesis that opens the 705 original hairpin to form a concatemerized amplicon containing a self-priming 3′-end hairpin', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF46': {'ref_id': 'b46', 'title': 'The ensuing continuous amplification (initiated both by new inner primers and by self-priming) 707 generates increasingly complex, double-stranded concatameric amplicons containing self-708 priming hairpins and single-stranded loops to which the OSD probe hybridizes', 'authors': [{'first': 'V', 'middle': [')'], 'last': '', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF47': {'ref_id': 'b47', 'title': 'No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/291849']}}, 'BIBREF48': {'ref_id': 'b48', 'title': \"doi: bioRxiv preprint three weeks at the indicated temperatures were prepared by 'in-tube' crude processing. 2 µL of 743 a 1:10 dilution of each mosquito sample was analyzed by coi LAMP-OSD (A) and by wsp 744 LAMP-OSD (B) assays. OSD fluorescence was imaged at endpoint using a smartphone\", 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/291849']}}, 'BIBREF49': {'ref_id': 'b49', 'title': 'No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/291849']}}}, 'ref_entries': {'FIGREF0': {'text': 'MUltiple Sequence Comparison by Log-Expectation) alignment analysis of each gene set. 149 Target specificity of these signature sequences was evaluated by comparing them to respective 150 wsp or coi gene sets from phylogenetically-related strains and species such as Wolbachia wMel 151 and Ae. albopictus, respectively. Both MUSCLE alignment as well as NCBI BLAST [45, 46] 152 analysis were used for this in silico specificity analysis. 153', 'latex': None, 'type': 'figure'}, 'FIGREF2': {'text': \"TP/(TP+FN) where TP are true positive samples, and FN are false negative samples. Specificity 243 or true negative rate was calculated using the formula TN/(TN+FP) where TN are true negative 244 samples and FP are false positive samples. 245Results246Development of visually read LAMP-OSD assays to identify mosquito species and 247Wolbachia infection 248 LAMP uses two inner (FIP and BIP) and two outer (F3 and B3) primers specific to six consecutive 249 target sequences (B3, B2, B1, F1c, F2c and F3c)(S2 Fig) [47]. Bst DNA polymerase extends 250 these primers by strand displacement DNA synthesis to form 10 9 to 10 10 copies of 251 concatemerized amplicons with loops between the F1 and F2, B1 and B2, F1c and F2c, and B1c 252 and B2c regions. We use an additional fifth primer that binds to one of these loop regions and 253accelerates amplification[41]. OSD probes, with blocked 3'-ends that prevents spurious signaling 254 from polymerase-mediated extension, hybridize to the second loop region(S2 Fig).\", 'latex': None, 'type': 'figure'}, 'FIGREF4': {'text': '-OSD assay to amplify and detect a signature sequence in the mitochondrial 257 cytochrome c oxidase I (coi) gene. Each cell has multiple mitochondria and hence several 258 hundred copies of the coi gene, which should enable detection from a very small amount of 259 sample. Moreover, mitochondrial coi gene sequences are commonly used as barcodes for 260 molecular identification of animal species including distinction of mosquito species; our chosen 261 coi signature sequence was assigned to Ae. aegypti when queried against the Barcode of Life 262 Data Systems (BOLD; http://www.boldsystems.org/index.php) coi signature sequence database 263 [48-51]. 264 We developed a second visually read LAMP-OSD assay targeting the Wolbachia surface 265 protein (wsp) gene to identify Wolbachia-infected insects. The wsp gene is widely used as a 266 marker for strain typing and screening for infected insect vectors [28, 52]. We engineered our 267 wsp LAMP outer and inner primers to be complementary to, and hence amplify, two Wolbachia 268 strains, the wAlbB and the closely related wPip (S3 Fig). We deliberately designed our assay to 269 detect both strains in order to ensure that we could assess field-collected mosquitoes 270 irrespective of temporal and spatial variation in relative abundance of wAlbB-infected Ae. 271 albopictus and wPip-infected Cu. quinquefasciatus mosquitoes in our collection area [53, 54]. 272 Significant nucleic acid sequence variation should prevent amplification of the wsp gene from all 273 other Wolbachia groups (S3 Fig). To enable transduction of both wAlbB and wPip wsp LAMP 274 amplicons to visible fluorescence we designed an OSD probe that is specific to an identical loop 275 sequence present in both amplicons. 276', 'latex': None, 'type': 'figure'}, 'FIGREF5': {'text': 'Detection of Ae. aegypti coi gene and Wolbachia wAlbB wsp gene using visually 284 read LAMP-OSD assays. Indicated copies of recombinant plasmids bearing coi or wsp target 285 sequences were amplified by the coi-specific (A) or wsp-specific (B) LAMP-OSD assays, 286 respectively. OSD fluorescence was imaged at endpoint using a cellphone. 287 288 Direct LAMP-OSD analysis of crudely crushed mosquitoes 289', 'latex': None, 'type': 'figure'}, 'FIGREF6': {'text': 'Visually read LAMP-OSD analysis of crudely processed mosquito samples. (A) 311 Schematic depicting rapid preparation of crude mosquito samples for direct LAMP-OSD 312 analysis by performing in-syringe individual mosquito maceration. (B) Analysis of individual Ae. 313 aegypti and wAlbB Wolbachia-infected Ae. albopictus mosquitoes by coi and wsp LAMP-OSD 314 assays, respectively. 2 µL of mosquito samples were either added directly to LAMP-OSD 315 assays or subjected to 10-fold serial dilution in water prior to introduction in LAMP-OSD 316 reactions. LAMP-OSD reactions without any macerated mosquito samples served as negative 317 controls. Assays lacking LAMP primers but containing macerated mosquito samples served as 318 controls for monitoring background fluorescence. (C) Specificity analysis of coi and wsp LAMP-319 OSD assays. The coi LAMP-OSD assay was challenged with 2 µL of Ae. albopictus in-syringe 320 prepared crude mosquito sample. Similarly, the wsp LAMP-OSD assay was challenged with Ae. 321 aegypti sample. In all experiments, OSD fluorescence was imaged at endpoint using a 322 cellphone. 323 324', 'latex': None, 'type': 'figure'}, 'FIGREF7': {'text': 'Fig). 334', 'latex': None, 'type': 'figure'}, 'FIGREF8': {'text': '334Testing blood-fed mosquitoes with LAMP-OSD 335Mosquitoes feeding on blood meals have been reported to engorge on 1 nL to as much as 6 µL 336 of blood[55]. It is conceivable that the blood meal might confound visual LAMP-OSD 337 fluorescence analysis by contributing auto-fluorescence. To ascertain compatibility of visually 338 read LAMP-OSD with direct analysis of crudely prepared blood-engorged mosquitoes, we 339 challenged both coi and wsp LAMP-OSD assays with crude in-syringe preparations of blood 340 engorged Ae. aegypti and Ae. albopictus mosquitoes. Mosquitoes that had recently consumed a 341', 'latex': None, 'type': 'figure'}, 'FIGREF9': {'text': 'Effect of blood meal on LAMP-OSD analysis of crudely prepared individual 345 mosquitoes. Individual normal or blood-fed Ae. aegypti and Ae. albopictus mosquitoes were 346 prepared by the in-syringe method. 2 µL of these mosquito samples were analyzed by coi and 347 wsp LAMP-OSD assays. LAMP-OSD reactions without any macerated mosquito samples served 348 as negative controls. Assays lacking LAMP primers but containing macerated mosquito samples 349 All rights reserved. No reuse allowed without permission.', 'latex': None, 'type': 'figure'}, 'FIGREF10': {'text': \"Blinded LAMP-OSD analysis of field-caught mosquitoes. Mosquitoes stored for up to method for rapid preparation of field-caught mosquitoes for LAMP-OSD analysis. (B) 377 Comparison of coi LAMP-OSD results with morphological identification using 2X2 contingency 378 table. (C) Tabulation of coi and wsp LAMP-OSD readout for each mosquito in the study. Ae. 379 aegypti, Ae. albopictus and Ochlerotatus species are highlighted in blue, orange and white, 380 respectively. 'Y' indicates a positive LAMP-OSD signal (bright fluorescence). 'N' indicates 381 absence of any fluorescence signal in the LAMP-OSD reaction. Red boxes highlight LAMP-OSD 382 tests that generated false positive or false negative outcomes. 383 Of the 60 non-Ae. aegypti mosquitoes analyzed by coi LAMP-OSD, only one mosquito 385 generated a false positive signal. Sequence analysis of its coi gene ruled out mis-firing of the coi 386 LAMP-OSD assay. It is possible that this LAMP assay was inadvertently contaminated with a 387 small amount of a pre-formed Ae. aegypti amplicon. 388 The Wolbachia wAlbB/wPip wsp LAMP-OSD assay did not generate a positive signal from any 389 non-Ae. albopictus mosquito. This is expected since natural populations of Ae. aegypti and most 390 Ochlerotatus species are not infected with Wolbachia [56, 57]. However, ability of the wsp assay 391 to identify Wolbachia infection was influenced by the storage temperature of mosquitoes. The 392 wsp LAMP assay could readily identify Wolbachia infection in 3 out of 4 Ae. albopictus 393\", 'latex': None, 'type': 'figure'}, 'FIGREF11': {'text': \"[58], these results are suggestive of 402 nucleic acid deterioration in mosquitoes upon storage at high temperature. 403 Discussion 404 Mosquito control strategies that rely on the introduction of Wolbachia are now being deployed 405 around the world [7-9], and the surveillance of efficacy and spread require agile, field-based 406 methods for both mosquito and symbiont detection. Unfortunately, currently available tools for 407 mosquito diagnostics have several shortcomings. Morphological identification methods are 408 inherently low throughput, require extensive technical expertise, and cannot also readily identify 409 pathogens or biocontrol agents such as Wolbachia. Spectroscopy, such as near infrared 410 spectroscopy [59] and Fourier transform infrared spectroscopy [60], allow identification of 411 mosquito species, Wolbachia, and pathogens, but require expensive instruments and expertise that 412 are generally incompatible with low-resource settings. Immunoassays can detect pathogen-413 carrying vectors but are insensitive and cannot also identify vector species. Nucleic acid 414 amplification methods could potentially look at both vector and sybmiont sequences, but are heavily 415 reliant on PCR with the ensuing encumbrances of expensive instruments and trained operators, 416 again precluding widespread use. 417 Isothermal nucleic acid amplification assays would facilitate field-based vector monitoring, but most 418 reported approaches rely on nucleic acid purification and non-specific readout, and thus suffer from 419 laborious setup and the risk of false positives [61-63]. Probe-read isothermal methods such as the 420 assay (RPA) are more reliable but still require expensive and proprietary 421 reaction formulations and probes, which limits their flexibility and versatility in assay engineering. 422 Furthermore, most RPA applications for vector diagnostics [64] also depend on extensive sample 423 processing and nucleic acid purification prior to amplification. 424 These drawbacks led us to develop a simpler, more robust field-deployable assay based on loop-425 mediated isothermal amplification (LAMP) that can identify both mosquito species and specific 426 Wolbachia strains in mosquitoes. While LAMP assays have previously been developed for 427 Wolbachia detection by targeting the 16S rRNA gene for amplification [61], these assays required 428 extraction and purification of DNA prior to assay, and used non-specific readouts that were highly 429 prone to false positive signals. 430 To overcome these barriers to the use of LAMP, we have previously adopted methods that 431 originated in the field of nucleic acid computation: the use of strand exchange reactions that 432 initiate at complementary single-stranded 'toeholds' and progresses via branch migration. The 433 base-pairing predictability and programmability of strand exchange kinetics promotes the 434 construction of exquisitely sequence-specific oligonucleotide strand displacement (OSD) probes 435 for LAMP amplicons (S2 Fig), thereby greatly reducing the detection of non-specific amplification 436 background [39, 41, 42]. For instance, we recorded positive coi LAMP-OSD signals from field-437 caught Ae. aegypti mosquitos but did not detect signal from closely related Ae. albopictus and 438 Ochlerotatus species mosquitoes. Similarly, the wsp assay detected wAlbB and wPip, as 439 expected, but not wMel Wolbachia. 440 Strand exchange circuits have the additional advantage that they can be used to embed 441 algorithms and act as 'matter computers' [35-37, 65]. For example, strand exchange transducers 442\", 'latex': None, 'type': 'figure'}, 'FIGREF12': {'text': 'This might be 452 advantageous for strain discrimination if double infections were released during vector control 453 measures [70, 71]. 454', 'latex': None, 'type': 'figure'}, 'FIGREF13': {'text': 'SL, Xi ZY, Rasgon JL, Calvitti M, et al. (2014) Harnessing mosquito-500', 'latex': None, 'type': 'figure'}, 'FIGREF14': {'text': \"and O'Neill SL (2013) Beyond insecticides: new thinking on an ancient 502 problem. Nat Rev Microbiol 11: 181-193. 503 10. Dutra HLC, Rocha MN, Dias FBS, Mansur SB, Caragata EP, et al. (2016) Wolbachia 504 Blocks Currently Circulating Zika Virus Isolates in Brazilian Aedes aegypti Mosquitoes. 505 Cell Host & Microbe 19: 771-774. 506 11. Moreira LA, Iturbe-Ormaetxe I, Jeffery JA, Lu GJ, Pyke AT, et al. (2009) A Wolbachia 507 Symbiont in Aedes aegypti Limits Infection with Dengue, Chikungunya, and 508 Plasmodium. Cell 139: 1268-1278. 509 12. Bian GW, Joshi D, Dong YM, Lu P, Zhou GL, et al. (2013) Wolbachia Invades 510 Anopheles stephensi Populations and Induces Refractoriness to Plasmodium Infection. 511 Science 340: 748-751. 512 13. Kambris Z, Cook PE, Phuc HK and Sinkins SP (2009) Immune Activation by Life-513 Shortening Wolbachia and Reduced Filarial Competence in Mosquitoes. Hurk AF, Hall-Mendelin S, Pyke AT, Frentiu FD, McElroy K, et al. (2012) Impact 519 of Wolbachia on Infection with Chikungunya and Yellow Fever Viruses in the Mosquito 520 Vector Aedes aegypti. Plos Neglected Tropical Diseases 6. 521 16. Hoffmann AA, Montgomery BL, Popovici J, Iturbe-Ormaetxe I, Johnson PH, et al. (2011) 522 Successful establishment of Wolbachia in Aedes populations to suppress dengue 523 transmission. Nature 476: 454-U107. 524 17. O'Connor L, Plichart C, Sang AC, Brelsfoard CL, Bossin HC, et al. (2012) Open Release 525 of Male Mosquitoes Infected with a Wolbachia Biopesticide: Field Performance and 526 Infection Containment. Plos Neglected Tropical Diseases 6. 527 18. Walker T, Johnson PH, Moreira LA, Iturbe-Ormaetxe I, Frentiu FD, et al. (2011) The 528 wMel Wolbachia strain blocks dengue and invades caged Aedes aegypti populations. 529 Nature 476: 450-453. 530 19. Day JF and Shaman J (2011) Mosquito-Borne Arboviral Surveillance and the Prediction 531 of Disease Outbreaks. In: D. Ruzek, editor editors. Flavivirus Encephalitis. InTech. pp. 532 105-130. DOI: 110.5772/24068. 533 20. Ferraguti M, Martinez-de la Puente J, Roiz D, Ruiz S, Soriguer R, et al. (2016) Effects of 534 landscape anthropization on mosquito community composition and abundance. Sci Rep 535 6: 29002. 536 21. Vazquez-Prokopec GM, Chaves LF, Ritchie SA, Davis J and Kitron U (2010) Unforeseen 537 Costs of Cutting Mosquito Surveillance Budgets. Plos Neglected Tropical Diseases 4. 538 22. Wilke ABB, Christe RD, Multini LC, Vidal PO, Wilk-da-Silva R, et al. (2016) 539\", 'latex': None, 'type': 'figure'}, 'FIGREF15': {'text': 'YS, Kumar MR, Johnson RF, Hensley LE, et al. (2015) Real-time 590 sequence-validated loop-mediated isothermal amplification assays for detection of Fig 1. Detection of Ae. aegypti coi gene and Wolbachia wAlbB wsp gene using visually read LAMP-OSD assays. Indicated copies of recombinant plasmids bearing coi or wsp target sequences were amplified by the coi-specific (A) or wsp-specific (B) LAMP-OSD assays, respectively. OSD fluorescence was imaged at endpoint using a cellphone.', 'latex': None, 'type': 'figure'}, 'FIGREF16': {'text': 'Visually read LAMP-OSD analysis of crudely processed mosquito samples. (A) Schematic depicting rapid preparation of crude mosquito samples for direct LAMP-OSD analysis by performing in-syringe individual mosquito maceration. (B) Analysis of individual Ae. aegypti', 'latex': None, 'type': 'figure'}, 'FIGREF17': {'text': 'Effect of blood meal on LAMP-OSD analysis of crudely prepared individual mosquitoes.', 'latex': None, 'type': 'figure'}, 'FIGREF18': {'text': 'Blinded LAMP-OSD analysis of field-caught mosquitoes.', 'latex': None, 'type': 'figure'}, 'TABREF0': {'text': 'Single loop primers were designed to bind the second loop region (between B1c and B2 primer 173binding sites) of the wsp and coi LAMP amplicons and accelerate LAMP amplification. 174 LAMP assay with synthetic DNA template 175 LAMP assays were assembled in a total volume of 25 µl of 1X Isothermal buffer (NEB; 20 mM 176 Tris-HCl, 10 mM (NH4)2SO4, 50 mM KCl, 2 mM MgSO4, 0.1% Tween 20, pH 8.8 at 25°C). The 177 buffer was supplemented with 0.4 mM dNTPs, 0.8 M betaine, 2 mM additional MgCl2, 1.6 µM 178 each of FIP and BIP, 0.8 µM of loop primer, 0.4 µM each of F3 and B3 primers, and 16 units of 179 Bst 2.0 DNA polymerase. Plasmid DNA templates were serially diluted in TE buffer (10 mM Tris-180', 'latex': None, 'type': 'table'}, 'TABREF1': {'text': \"Detection of malaria sporozoites by standard ELISA and VecTest (TM) dipstick assay in 548 field-collected anopheline mosquitoes from a malaria endemic site in Ghana. Trop Med 549 Int Health 8: 1012-1017. 550 26. Lee SF, White VL, Weeks AR, Hoffmann AA and Endersby NM (2012) High-Throughput 551 PCR Assays To Monitor Wolbachia Infection in the Dengue Mosquito (Aedes aegypti) 552 and Drosophila simulans. Appl Environ Microbiol 78: 4740-4743. 553 27. de Oliveira CD, Goncalves DS, Baton LA, Shimabukuro PHF, Carvalho FD, et al. (2015) 554 Broader prevalence of Wolbachia in insects including potential human disease vectors. 555 Bull Entomol Res 105: 305-315. 556 28. Zhou WG, Rousset F and O'Neill S (1998) Phylogeny and PCR-based classification of 557 Wolbachia strains using wsp gene sequences. Proceedings of the Royal Society B-558 Biological Sciences 265: 509-515. 559 29. de Paz HD, Brotons P and Munoz-Almagro C (2014) Molecular isothermal techniques 560 for combating infectious diseases: towards low-cost point-of-care diagnostics. Expert 561 Rev Mol Diagn 14: 827-843. 562 30. Boyle DS, Lehman DA, Lillis L, Peterson D, Singhal M, et al. (2013) Rapid Detection of 563 HIV-1 Proviral DNA for Early Infant Diagnosis Using Recombinase Polymerase 564 Amplification. Mbio 4: pii: e00135-00113. 565 All rights reserved. No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/291849 doi: bioRxiv preprint recombinase polymerase amplification of HIV DNA. Lab Chip 12: 3082-3088. 567 32. Tomita N, Mori Y, Kanda H and Notomi T (2008) Loop-mediated isothermal amplification 568 (LAMP) of gene sequences and simple visual detection of products. Nature Protocols 3: 569 Sakurai T, Di Cello F, Grab DJ, Suzuki H, et al. (2003) Loop-mediated 571 isothermal amplification for detection of African trypanosomes. J Clin Microbiol 41: 5517-572 Kirstein OD, Hailu A and Warburg A (2016) Optimization of loop-mediated 574 isothermal amplification (LAMP) assays for the detection of Leishmania DNA in human 575 blood samples. Acta Trop 162: 20-26. 576 35. Zhang DY, Turberfield AJ, Yurke B and Winfree E (2007) Engineering entropy-driven 577 reactions and networks catalyzed by DNA. Science 318: 1121-1125. 578 36. Seelig G, Soloveichik D, Zhang DY and Winfree E (2006) Enzyme-free nucleic acid logic 579 circuits. Science 314: 1585-1588. 580 37. Yin P, Choi HM, Calvert CR and Pierce NA (2008) Programming biomolecular self-581 assembly pathways. Nature 451: 318-322. 582 38. Chakraborty K, Veetil AT, Jaffrey SR and Krishnan Y (2016) Nucleic Acid-Based 583 Nanodevices in Biological Imaging. Annual Review of Biochemistry, Vol 85 85: 349-373. 584 39. Jiang YS, Bhadra S, Li B, Wu YR, Milligan JN, et al. (2015) Robust strand exchange 585 reactions for the sequence-specific, real-time detection of nucleic acid amplicons. Anal 586 Chem 87: 3314-3320. 587 40. Zhang DY and Winfree E (2009) Control of DNA strand displacement kinetics using 588 toehold exchange. J Am Chem Soc 131: 17303-17314. 589 All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/291849 doi: bioRxiv preprint\", 'latex': None, 'type': 'table'}}, 'back_matter': [{'text': 'We would like to thank Dr. Holly Wichman (Department of Biological Sciences, Center for 478Modeling Complex Interactions, University of Idaho, Moscow, ID 83844-3051) for the generous 479 gift of wMel Wolbachia-infected Drosophila fruit flies. We would also like to thank the UTMB 480 insectary core for providing the lab-reared mosquitoes. 481', 'cite_spans': [], 'ref_spans': [], 'section': '477'}, {'text': 'All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'annex'}]}\n",
      "{'paper_id': 'b1e7752cbf528db99723a3a67b49b9b5c6e449af', 'metadata': {'title': 'Host shifts result in parallel genetic changes when viruses evolve in closely related species 1 2', 'authors': [{'first': 'Ben', 'middle': [], 'last': 'Longdon', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Exeter', 'location': {'addrLine': 'Penryn Campus', 'postCode': 'TR10 6 9FE', 'settlement': 'Biosciences', 'country': 'UK'}}, 'email': '*b.longdon2@exeter.ac.uk'}, {'first': 'Jonathan', 'middle': ['P'], 'last': 'Day', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Cambridge', 'location': {'postCode': 'CB2 3EH', 'settlement': 'Cambridge', 'country': 'UK'}}, 'email': ''}, {'first': 'Joel', 'middle': ['M'], 'last': 'Alves', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Cambridge', 'location': {'postCode': 'CB2 3EH', 'settlement': 'Cambridge', 'country': 'UK'}}, 'email': ''}, {'first': 'Sophia', 'middle': ['Cl'], 'last': 'Smith', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Cambridge', 'location': {'postCode': 'CB2 3EH', 'settlement': 'Cambridge', 'country': 'UK'}}, 'email': ''}, {'first': 'Thomas', 'middle': ['M'], 'last': 'Houslay', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Exeter', 'location': {'addrLine': 'Penryn Campus', 'postCode': 'TR10 6 9FE', 'settlement': 'Biosciences', 'country': 'UK'}}, 'email': ''}, {'first': 'John', 'middle': ['E'], 'last': '', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': '', 'middle': [], 'last': 'Mcgonigle', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Cambridge', 'location': {'postCode': 'CB2 3EH', 'settlement': 'Cambridge', 'country': 'UK'}}, 'email': ''}, {'first': 'Lucia', 'middle': [], 'last': 'Tagliaferri', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Cambridge', 'location': {'postCode': 'CB2 3EH', 'settlement': 'Cambridge', 'country': 'UK'}}, 'email': ''}, {'first': 'Francis', 'middle': ['M'], 'last': 'Jiggins', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Cambridge', 'location': {'postCode': 'CB2 3EH', 'settlement': 'Cambridge', 'country': 'UK'}}, 'email': ''}]}, 'abstract': [{'text': 'Host shifts, where a pathogen invades and establishes in a new host species, are a major source of 17 emerging infectious diseases. They frequently occur between related host species and often rely on 18 the pathogen evolving adaptations that increase their fitness in the novel host species. To 19 investigate genetic changes in novel hosts, we experimentally evolved replicate lineages of an RNA 20 virus (Drosophila C Virus) in 19 different species of Drosophilidae and deep sequenced the viral 21 genomes. We found a strong pattern of parallel evolution, where viral lineages from the same host 22', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'were genetically more similar to each other than to lineages from other host species. When we 23 compared viruses that had evolved in different host species, we found that parallel genetic changes 24 were more likely to occur if the two host species were closely related. This suggests that when a 25 virus adapts to one host it might also become better adapted to closely related host species. This 26 may explain in part why host shifts tend to occur between related species, and may mean that when 27 a new pathogen appears in a given species, closely related species may become vulnerable to the 28 new disease.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}], 'body_text': [{'text': 'Host shifts -where a pathogen jumps into and establishes in a new host species -are a major 32 source of emerging infectious diseases. RNA viruses seem particularly prone to host shift [1] [2] [3] [4] , with 33 HIV, Ebola virus and SARS coronavirus all having been acquired by humans from other host species 34 [5] [6] [7] . Whilst some pathogens may be pre-adapted to a novel host, there are increasing numbers of 35 examples demonstrating that adaptation to the new host occurs following a host shift [8, 9] . These 36 adaptations may allow a pathogen to enter host cells, increase replication rates, avoid or suppress 37 the host immune response, or optimise virulence or transmission [10, 11] . For example, in the 2013-38 2016 Ebola virus epidemic in West Africa, a mutation in the viral glycoprotein gene that arose early 39 in the outbreak and rose to high frequency was found to increase infectivity in human cells and 40 decrease infectivity in bats, which are thought to be the source of Ebola virus [12, 13] . Likewise, a 41 switch of a parvovirus from cats to dogs resulted in mutations in the virus capsid that allowed the 42 virus to bind to cell receptors in dogs, but resulted in the virus losing its ability to infect cats [14, 15] ', 'cite_spans': [{'start': 185, 'end': 188, 'text': '[1]', 'ref_id': None}, {'start': 189, 'end': 192, 'text': '[2]', 'ref_id': None}, {'start': 193, 'end': 196, 'text': '[3]', 'ref_id': None}, {'start': 197, 'end': 200, 'text': '[4]', 'ref_id': None}, {'start': 311, 'end': 314, 'text': '[5]', 'ref_id': None}, {'start': 315, 'end': 318, 'text': '[6]', 'ref_id': None}, {'start': 319, 'end': 322, 'text': '[7]', 'ref_id': None}, {'start': 503, 'end': 506, 'text': '[8,', 'ref_id': None}, {'start': 507, 'end': 509, 'text': '9]', 'ref_id': None}, {'start': 688, 'end': 692, 'text': '[10,', 'ref_id': None}, {'start': 693, 'end': 696, 'text': '11]', 'ref_id': None}, {'start': 1010, 'end': 1014, 'text': '[12,', 'ref_id': None}, {'start': 1015, 'end': 1018, 'text': '13]', 'ref_id': None}, {'start': 1240, 'end': 1244, 'text': '[14,', 'ref_id': None}, {'start': 1245, 'end': 1248, 'text': '15]', 'ref_id': None}], 'ref_spans': [], 'section': '31'}, {'text': 'In some instances adaptation to a novel host relies on specific mutations that arise repeatedly 45 whenever a pathogen switches to a given host. For example, in the jump of HIV-1 from chimps to 46 humans, codon 30 of the gag gene has undergone a change that increases virus replication in 47 humans, and this has occurred independently in all three HIV-1 lineages [5, 16] . Similarly, five 48 parallel mutations have been observed in the two independent epidemics of SARS coronavirus 49 following its jump from palm civets into humans [17] . Similar patterns have been seen in ', 'cite_spans': [{'start': 364, 'end': 367, 'text': '[5,', 'ref_id': None}, {'start': 368, 'end': 371, 'text': '16]', 'ref_id': None}, {'start': 535, 'end': 539, 'text': '[17]', 'ref_id': None}], 'ref_spans': [], 'section': '44'}, {'text': 'The host phylogeny is important for determining a pathogens ability to infect a novel host, with 59 pathogens tending to replicate most efficiently when they infect a novel host that is closely related 60 to their original host [2, [21] [22] [23] [24] [25] [26] [27] [28] [29] [30] [31] [32] [33] [34] . Here, we asked whether viruses acquire the same genetic changes 61 when evolving in the same and closely related host species. We experimentally evolved replicate 62 lineages of an RNA virus called Drosophila C Virus (DCV; Discistroviridae) in 19 species of 63 Drosophilidae that vary in their relatedness and shared a common ancestor approximately 40 million 64 years ago [35, 36] . We then sequenced the genomes of the evolved viral lineages and tested 65 whether the same genetic changes arose when the virus was evolved in closely related host species.', 'cite_spans': [{'start': 228, 'end': 231, 'text': '[2,', 'ref_id': None}, {'start': 232, 'end': 236, 'text': '[21]', 'ref_id': None}, {'start': 237, 'end': 241, 'text': '[22]', 'ref_id': None}, {'start': 242, 'end': 246, 'text': '[23]', 'ref_id': None}, {'start': 247, 'end': 251, 'text': '[24]', 'ref_id': None}, {'start': 252, 'end': 256, 'text': '[25]', 'ref_id': None}, {'start': 257, 'end': 261, 'text': '[26]', 'ref_id': None}, {'start': 262, 'end': 266, 'text': '[27]', 'ref_id': None}, {'start': 267, 'end': 271, 'text': '[28]', 'ref_id': None}, {'start': 272, 'end': 276, 'text': '[29]', 'ref_id': None}, {'start': 277, 'end': 281, 'text': '[30]', 'ref_id': None}, {'start': 282, 'end': 286, 'text': '[31]', 'ref_id': None}, {'start': 287, 'end': 291, 'text': '[32]', 'ref_id': None}, {'start': 292, 'end': 296, 'text': '[33]', 'ref_id': None}, {'start': 297, 'end': 301, 'text': '[34]', 'ref_id': None}, {'start': 677, 'end': 681, 'text': '[35,', 'ref_id': None}, {'start': 682, 'end': 685, 'text': '36]', 'ref_id': 'BIBREF183'}], 'ref_spans': [], 'section': '58'}, {'text': 'Parallel genetic changes occur in DCV lineages that have evolved in the same host species', 'cite_spans': [], 'ref_spans': [], 'section': '70'}, {'text': 'To examine how viruses evolve in different host species we serially passaged DCV in 19 species of 73 Drosophilidae. In total we infected 22,095 adult flies and generated 173 independent replicate 74 . CC-BY-NC 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/226175 doi: bioRxiv preprint lineages (6-10 per host species). We deep sequenced the evolved virus genomes to generate over 75 740,000 300bp sequence reads from each viral lineage. Out of 8989 sites, 584 contained a SNP with 76 a derived allele frequency >0.05 in at least one viral lineage, and 84 of these were tri-allelic. None of 77 these variants were found at an appreciable frequency in five sequencing libraries produced from 78 the ancestral virus, indicating that they had spread though populations during the experiment 79 ( Figure 1 ). In multiple cases these variants had nearly reached fixation (Figure 1 ).', 'cite_spans': [], 'ref_spans': [{'start': 918, 'end': 926, 'text': 'Figure 1', 'ref_id': 'FIGREF2'}, {'start': 991, 'end': 1000, 'text': '(Figure 1', 'ref_id': 'FIGREF2'}], 'section': '72'}, {'text': 'We next examined whether the same genetic changes occur in parallel when different populations 82 encounter the same host species. Of the 584 SNPs, 102 had derived allele frequencies >0.05 in at 83 least two viral lineages, and some had risen to high frequencies in multiple lineages (Figure 1 ). We 84 estimated the genetic differentiation between viral lineages by calculating F ST . We found that viral 85 lineages that had evolved within the same host were genetically more similar to each other than to 86 lineages from other host species (Figure 2 ; P<0.001). Furthermore, we found no evidence of 87 differences in substitution biases in the different host species (Fisher Exact Test: p=0.14; see 88 methods), suggesting that this pattern is not driven by changes in the types of mutations in different 89 host species.', 'cite_spans': [], 'ref_spans': [{'start': 284, 'end': 293, 'text': '(Figure 1', 'ref_id': 'FIGREF2'}, {'start': 544, 'end': 553, 'text': '(Figure 2', 'ref_id': 'FIGREF6'}], 'section': '81'}, {'text': 'To examine the genetic basis of parallel evolution, we individually tested whether each SNP in the 92 DCV genome showed a signature of parallel evolution among viral lineages passaged in the same 93 host species (i.e. we repeated the analysis in Figures 2 for each SNP). We identified 56 polymorphic 94 sites with a significant signal of parallel evolution within the same host species (P<0.05; significantly 95 parallel sites are shown with a red asterisk in Figure 1 ; the false discovery rate is estimated to be ', 'cite_spans': [], 'ref_spans': [{'start': 460, 'end': 468, 'text': 'Figure 1', 'ref_id': 'FIGREF2'}], 'section': '91'}, {'text': 'Mutations ordered by position in DCV genome 214  352  363  454  794  849  1206  1281  1290  1299  1331  1445  1529  1576  1589  1811  1829  1850  1901  1976  2018  2318  2509  2610  2804  2891  2930  3023  3111  3154  3291  3349  3491  3721  3761  3807  3824  3864  3884  3966  3989  3998  4118  4832  4934  5219  5453  5510  5552  5573  5696  5936  6008  6227  6235  6312  6495  6551  6719  6722  6787  6927  6990  7014  7016  7039  7084  7087  7388  7703  7711  7922  7923  7936  7939  7981  7992  7997  8000  8012  8041  8069  8070  8072  8073  8075  8076  8079  8084  8142  8144  8153  8154  8282  8285  8294  8392  8677  8761  8845  8869  8961 Mutations ordered by position in DCV genome ', 'cite_spans': [], 'ref_spans': [{'start': 44, 'end': 648, 'text': '214  352  363  454  794  849  1206  1281  1290  1299  1331  1445  1529  1576  1589  1811  1829  1850  1901  1976  2018  2318  2509  2610  2804  2891  2930  3023  3111  3154  3291  3349  3491  3721  3761  3807  3824  3864  3884  3966  3989  3998  4118  4832  4934  5219  5453  5510  5552  5573  5696  5936  6008  6227  6235  6312  6495  6551  6719  6722  6787  6927  6990  7014  7016  7039  7084  7087  7388  7703  7711  7922  7923  7936  7939  7981  7992  7997  8000  8012  8041  8069  8070  8072  8073  8075  8076  8079  8084  8142  8144  8153  8154  8282  8285  8294  8392  8677  8761  8845  8869  8961', 'ref_id': None}], 'section': '91'}, {'text': 'Viruses in closely related hosts are genetically more similar 120 121', 'cite_spans': [], 'ref_spans': [], 'section': '119'}, {'text': 'We investigated if viruses passaged through closely related hosts showed evidence of parallel 122 genetic changes. We calculated F ST between all possible pairs of viral lineages that had evolved in 123 different host species. We found that viral lineages from closely related hosts were more similar to 124 each other than viral lineages from more distantly related hosts ( Figure 3A ). This is reflected in a ', 'cite_spans': [], 'ref_spans': [{'start': 375, 'end': 384, 'text': 'Figure 3A', 'ref_id': 'FIGREF7'}], 'section': '119'}, {'text': 'When a pathogen infects a novel host species, it finds itself in a new environment to which it must 142 adapt [4, 8, 10, 44] . When DCV was passaged through different species of Drosophilidae, we found 143 the same genetic changes arose repeatedly in replicate viral lineages in the same host species. Such 144 repeatable parallel genetic changes to the same host environment are compelling evidence that 145 these changes are adaptive [20] . We then examined whether these same genetic changes might 146 occur in closely related host species, as these are likely to present a similar environment for the 147 virus. We found that viruses evolved in closely related hosts were more similar to each other than 148 viruses that evolved in more distantly related species. Therefore, mutations that evolve in one host 149 species frequently arise when the virus infects closely related hosts. This finding of parallel genetic 150 changes in closely related host species suggests that when a virus adapts to one host it might also 151 become better adapted to closely related host species.', 'cite_spans': [{'start': 110, 'end': 113, 'text': '[4,', 'ref_id': None}, {'start': 114, 'end': 116, 'text': '8,', 'ref_id': None}, {'start': 117, 'end': 120, 'text': '10,', 'ref_id': None}, {'start': 121, 'end': 124, 'text': '44]', 'ref_id': None}, {'start': 436, 'end': 440, 'text': '[20]', 'ref_id': None}], 'ref_spans': [], 'section': '141'}, {'text': 'Phylogenetic patterns of host adaptation may in part explain why pathogens tend to be more likely 154 to jump between closely related host species. This pattern is seen in nature, where host shifts tend 155 to occur most frequently between closely related hosts, and in laboratory cross-infection studies,', 'cite_spans': [], 'ref_spans': [], 'section': '153'}, {'text': 'where viruses tend to replicate more rapidly when the new host is related to the pathogens natural The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/226175 doi: bioRxiv preprint 7 acquired adaptations to their host species that benefitted them when they infected closely related 161 species. Our results demonstrate that this pattern is apparent at the level of specific nucleotides, 162 and can arise very shortly after a host shift. The function of these mutations is unknown, but in other 163 systems adaptations after host shifts have been found to enhance the ability of the virus to bind to 164 host receptors [11] , increase replication rates [16] or avoid the host immune response [8, 10, 45] .', 'cite_spans': [{'start': 668, 'end': 672, 'text': '[11]', 'ref_id': None}, {'start': 702, 'end': 706, 'text': '[16]', 'ref_id': None}, {'start': 741, 'end': 744, 'text': '[8,', 'ref_id': None}, {'start': 745, 'end': 748, 'text': '10,', 'ref_id': None}, {'start': 749, 'end': 752, 'text': '45]', 'ref_id': None}], 'ref_spans': [], 'section': '156'}, {'text': \"While the susceptibility of a novel host is correlated to its relatedness to the pathogens' original 167 host, it is also common to find exceptions to this pattern. This is seen both in nature when 168 pathogens shift between very distant hosts [46, 47] , and in laboratory cross-infection experiments 169 [33, 34] . This pattern is also seen in our data where we also observe parallel genetic changes 170 occurring between more distantly related hosts. For example, a mutation at position 8072 was not 171 only near fixation in most of the lineages infecting two closely related species, but also occurred at a 172 high frequency in replicate lineages in a phylogenetically distant host (Figure 1 ).\", 'cite_spans': [{'start': 245, 'end': 249, 'text': '[46,', 'ref_id': None}, {'start': 250, 'end': 253, 'text': '47]', 'ref_id': None}, {'start': 306, 'end': 310, 'text': '[33,', 'ref_id': None}, {'start': 311, 'end': 314, 'text': '34]', 'ref_id': None}], 'ref_spans': [{'start': 688, 'end': 697, 'text': '(Figure 1', 'ref_id': 'FIGREF2'}], 'section': '166'}, {'text': 'In conclusion, we have found that host relatedness can be important in determining how viruses 175 evolve when they find themselves in a new host. This study suggests that while some genetic 176 changes will be found only in specific hosts, we frequently see the same changes occurring in closely 177 related host species. These phylogenetic patterns suggest that mutations that adapt a virus to one 178 host may also adapt it to closely related host species. Therefore, there may be a knock-on effect, 179 where a host shift leaves closely related species vulnerable to the new disease.', 'cite_spans': [], 'ref_spans': [], 'section': '174'}, {'text': \"Virus production 185 186 DCV is a positive sense RNA virus in the family Discistroviridae that was isolated from D. 187 melanogaster, which it naturally infects in the wild [48, 49] . To minimise the amount of genetic 188 variation in the DCV isolate we used to initiate the experimental evolution study, we aimed to isolate 189 single infectious clones of DCV using a serial dilution procedure. DCV was produced in Schneider's \", 'cite_spans': [{'start': 173, 'end': 177, 'text': '[48,', 'ref_id': None}, {'start': 178, 'end': 181, 'text': '49]', 'ref_id': None}], 'ref_spans': [], 'section': '184'}, {'text': 'This processes was then repeated using the DCV samples from the first dilution series. One clone,', 'cite_spans': [], 'ref_spans': [], 'section': '201'}, {'text': 'B6A, was selected for amplification and grown in cell culture as described above. Media containing 203 DCV was removed and centrifuged at 3000 x g for 5 minutes at 4°C to pellet any remaining cell 204 . CC-BY-NC 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/226175 doi: bioRxiv preprint 8 debris, before being aliquoted and stored at -80°C. The Tissue Culture Infective Dose 50 (TCID 50 ) of 205 the DCV was 6.32 x 10 9 infectious particles per ml using the .', 'cite_spans': [], 'ref_spans': [], 'section': '202'}, {'text': 'Inoculating fly species', 'cite_spans': [], 'ref_spans': [], 'section': '207'}, {'text': 'We passaged the virus through 19 species of Drosophilidae, with 6-10 independent replicate 210 passages for each species. We selected species from across the phylogeny (that shared a common 211 ancestor approximately 40 million years ago [35, 36] ), but included clades of closely related species 212 that recently shared common ancestors less than 5 million years ago (Figure 1 ). All fly stocks were 213 reared at 22°C. Stocks of each fly species were kept in 250ml bottles at staggered ages. Flies were 214 collected and sexed, and males were placed on cornmeal medium for 4 days before inoculation. ', 'cite_spans': [{'start': 238, 'end': 242, 'text': '[35,', 'ref_id': None}, {'start': 243, 'end': 246, 'text': '36]', 'ref_id': 'BIBREF183'}], 'ref_spans': [{'start': 369, 'end': 378, 'text': '(Figure 1', 'ref_id': 'FIGREF2'}], 'section': '209'}, {'text': 'After passaging the virus, we sequenced evolved viral lineages from 19 host species, with a mean of 240 9 independent replicate lineages of the virus per species (range 6-10 replicates). cDNA was 241 synthesised using Invitrogen Superscript III reverse-transcriptase with random hexamer primers 242 (25°C 5mins, 50°C 50mins, 70°C 15mins). The genome of the evolved viruses, along with the initial 243 DCV ancestor (x5) were then amplified using Q5 high fidelity polymerase (NEB) in nine overlapping 244 PCR reactions (see supplementary Table S2 for PCR primers and cycle conditions). Primers covered ', 'cite_spans': [], 'ref_spans': [{'start': 522, 'end': 544, 'text': 'supplementary Table S2', 'ref_id': None}], 'section': '239'}, {'text': 'To generate a reference ancestral Drosophila C Virus sequence we amplified the ancestral starting 262 virus by PCR as above. PCR products were treated with exonuclease 1 and Antarctic phosphatase to ', 'cite_spans': [], 'ref_spans': [], 'section': '261'}, {'text': 'We examined the frequency of alternate alleles (single nucleotide polymorphisms: SNPs) in five 286 ancestral virus replicates (aliquots of the same virus stock that was used to found the evolved 287 lineages). SNPs in these ancestral viruses may represent pre-standing genetic variation, or may be 288 . CC-BY-NC 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/226175 doi: bioRxiv preprint sequencing errors. We found the mean SNP frequency was 0.000923 and the highest frequency of 289 any SNP was 0.043 across the ancestral viruses. We therefore included a SNP in our analyses if its 290 frequency was >0.05 in any of the evolved viral lineages. For all analyses we included all three alleles 291 at triallelic sites.', 'cite_spans': [], 'ref_spans': [], 'section': '285'}, {'text': 'Parallel evolution within species', 'cite_spans': [], 'ref_spans': [], 'section': '293'}, {'text': 'As a measure of genetic differentiation we estimated F ST between all the virus lineages based on the (1). H w was calculated separately for the two lineages being compared, and 302 the unweighted mean used in equation (1).', 'cite_spans': [], 'ref_spans': [], 'section': '295'}, {'text': 'To examine whether there had been parallel evolution among viral lineages that had evolved within 305 the same fly species, we calculated the mean F ST between lineages that had evolved in the same fly 306 species, and compared this to the mean F ST between lineages that had evolved in different fly 307 species. We tested whether this difference was statistically significant using a permutation test. The 308 fly species labels were randomly reassigned to the viral lineages, and we calculated the mean F ST 309 between lineages that had evolved in the same fly species. This was repeated 1000 times to 310 generate a null distribution of the test statistic, and this was then compared to the observed value.', 'cite_spans': [], 'ref_spans': [], 'section': '304'}, {'text': 'To identify individual SNPs with a signature of parallel evolution within species, we repeated this 313 procedure separately for each SNP.', 'cite_spans': [], 'ref_spans': [], 'section': '312'}, {'text': 'Parallel evolution between species', 'cite_spans': [], 'ref_spans': [], 'section': '315'}, {'text': 'We next examined whether viral lineages that had evolved in different fly species tended to be more 318 similar if the fly species were more closely related. Considering all pairs of viral lineages from 319 different host species, we correlated pairwise F ST with the genetic distance between the fly species.', 'cite_spans': [], 'ref_spans': [], 'section': '317'}, {'text': 'To test the significance of this correlation, we permuted the fly species over the Drosophila 321 phylogeny and recalculated the Pearson correlation coefficient. This was repeated 1000 times to 322 generate a null distribution of the test statistic, and this was then compared to the observed value.', 'cite_spans': [], 'ref_spans': [], 'section': '320'}, {'text': 'To identify individual SNPs whose frequencies were correlated with the genetic distance between 324 hosts we repeated this procedure separately for each SNP.', 'cite_spans': [], 'ref_spans': [], 'section': '323'}, {'text': 'We confirmed there was no relationship between rates of molecular evolution (SNP frequency) and The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/226175 doi: bioRxiv preprint previously [34] . We also examined the distribution of SNPs and whether they were synonymous or 331 non-synonymous (see supplementary results).', 'cite_spans': [{'start': 238, 'end': 242, 'text': '[34]', 'ref_id': None}], 'ref_spans': [], 'section': '326'}, {'text': 'To test whether there were systematic differences in the types of mutations occurring in the 334 different host species, we classified all the SNPs into the six possible types (A/G, A/T, A/C, G/T, G/C 335 and C/T). We then counted the number of times each type of SNP arose in each host species at a 336 frequency above 5% and in at least one biological replicate (SNPs in multiple biological replicates 337 were only counted once). This resulted in a contingency table with 6 columns and 19 rows. We 338 tested for differences between the species in the relative frequency of the 6 SNP types by simulation 339 [62] .', 'cite_spans': [{'start': 611, 'end': 615, 'text': '[62]', 'ref_id': None}], 'ref_spans': [], 'section': '333'}, {'text': 'Sequence data (fastq files) are available in the NCBI SRA (Accession: SRP119720). BAM files, data and 342 R scripts for analysis in the main text are available from the NERC data repository (funding ', 'cite_spans': [], 'ref_spans': [], 'section': '341'}, {'text': '. CC-BY-NC 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/226175 doi: bioRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': '426'}], 'bib_entries': {'BIBREF137': {'ref_id': 'b137', 'title': 'Diseases of humans and their domestic mammals: 365 pathogen characteristics, host range and the risk of emergence', 'authors': [{'first': 'S', 'middle': [], 'last': 'Cleaveland', 'suffix': ''}, {'first': 'M', 'middle': ['K'], 'last': 'Laurenson', 'suffix': ''}, {'first': 'L', 'middle': ['H'], 'last': 'Taylor', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF139': {'ref_id': 'b139', 'title': 'Phylogeny and geography predict pathogen community similarity in 369 wild primates and humans', 'authors': [{'first': 'T', 'middle': ['J'], 'last': 'Davies', 'suffix': ''}, {'first': 'A', 'middle': ['B'], 'last': 'Pedersen', 'suffix': ''}], 'year': None, 'venue': 'Proceedings of the Royal Society B-Biological Sciences', 'volume': '370', 'issn': '', 'pages': '1695--701', 'other_ids': {'DOI': ['10.1098/rspb.2008.0284']}}, 'BIBREF140': {'ref_id': 'b140', 'title': 'Risk factors for human disease emergence', 'authors': [{'first': 'L', 'middle': ['H'], 'last': 'Taylor', 'suffix': ''}, {'first': 'S', 'middle': ['M'], 'last': 'Latham', 'suffix': ''}, {'first': 'M', 'middle': ['E'], 'last': 'Woolhouse', 'suffix': ''}], 'year': None, 'venue': 'Philos', 'volume': '372', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF143': {'ref_id': 'b143', 'title': 'Emerging pathogens: the epidemiology and evolution 12 10', 'authors': [{'first': 'M', 'middle': ['E'], 'last': 'Woolhouse', 'suffix': ''}, {'first': 'D', 'middle': ['T'], 'last': 'Haydon', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Antia', 'suffix': ''}, {'first': 'P', 'middle': ['M'], 'last': 'Sharp', 'suffix': ''}, {'first': 'B', 'middle': ['H'], 'last': 'Hahn', 'suffix': ''}], 'year': None, 'venue': 'Transactions of the Royal Society B-Biological Sciences', 'volume': '379', 'issn': '', 'pages': '2487--94', 'other_ids': {'DOI': ['38010.1098/rstb.2010.0031'], 'PMID': ['16701375']}}, 'BIBREF144': {'ref_id': 'b144', 'title': 'Fruit bats as 382 reservoirs of Ebola virus', 'authors': [{'first': 'E', 'middle': ['M'], 'last': 'Leroy', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Kumulungui', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Pourrut', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Rouquet', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Hassanin', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Yaba', 'suffix': ''}], 'year': 2005, 'venue': 'Nature', 'volume': '438', 'issn': '7068', 'pages': '575--581', 'other_ids': {'DOI': ['10.1038/438575a'], 'PMID': ['383']}}, 'BIBREF145': {'ref_id': 'b145', 'title': 'Bats are natural reservoirs of SARS-385 like coronaviruses', 'authors': [{'first': 'W', 'middle': ['D'], 'last': 'Li', 'suffix': ''}, {'first': 'Z', 'middle': ['L'], 'last': 'Shi', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Yu', 'suffix': ''}, {'first': 'W', 'middle': ['Z'], 'last': 'Ren', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Smith', 'suffix': ''}, {'first': 'J', 'middle': ['H'], 'last': 'Epstein', 'suffix': ''}], 'year': 2005, 'venue': 'Science', 'volume': '310', 'issn': '5748', 'pages': '676--685', 'other_ids': {'DOI': ['10.1126/science.1118391'], 'PMID': ['386']}}, 'BIBREF146': {'ref_id': 'b146', 'title': 'Cross-species virus 388 transmission and the emergence of new epidemic diseases', 'authors': [{'first': 'C', 'middle': ['R'], 'last': 'Parrish', 'suffix': ''}, {'first': 'E', 'middle': ['C'], 'last': 'Holmes', 'suffix': ''}, {'first': 'D', 'middle': ['M'], 'last': 'Morens', 'suffix': ''}, {'first': 'E', 'middle': ['C'], 'last': 'Park', 'suffix': ''}, {'first': 'D', 'middle': ['S'], 'last': 'Burke', 'suffix': ''}, {'first': 'C', 'middle': ['H'], 'last': 'Calisher', 'suffix': ''}], 'year': None, 'venue': 'Microbiology and Molecular Biology', 'volume': '389', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF148': {'ref_id': 'b148', 'title': 'The potential for 391 respiratory droplet-transmissible A/H5N1 influenza virus to evolve in a mammalian host', 'authors': [{'first': 'C', 'middle': ['A'], 'last': 'Russell', 'suffix': ''}, {'first': 'J', 'middle': ['M'], 'last': 'Fonville', 'suffix': ''}, {'first': 'A', 'middle': ['E'], 'last': 'Brown', 'suffix': ''}, {'first': 'D', 'middle': ['F'], 'last': 'Burke', 'suffix': ''}, {'first': 'D', 'middle': ['L'], 'last': 'Smith', 'suffix': ''}, {'first': 'S', 'middle': ['L'], 'last': 'James', 'suffix': ''}], 'year': 2012, 'venue': 'Science', 'volume': '392', 'issn': '6088', 'pages': '1541--1548', 'other_ids': {'DOI': ['10.1126/science.1222526'], 'PMID': ['393'], 'PMCID': ['PMC3426314']}}, 'BIBREF149': {'ref_id': 'b149', 'title': 'The Evolution and Genetics of 395', 'authors': [{'first': 'B', 'middle': [], 'last': 'Longdon', 'suffix': ''}, {'first': 'M', 'middle': ['A'], 'last': 'Brockhurst', 'suffix': ''}, {'first': 'C', 'middle': ['A'], 'last': 'Russell', 'suffix': ''}, {'first': 'J', 'middle': ['J'], 'last': 'Welch', 'suffix': ''}, {'first': 'F', 'middle': ['M'], 'last': 'Jiggins', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF150': {'ref_id': 'b150', 'title': 'Virus Host Shifts', 'authors': [], 'year': 2014, 'venue': 'PLoS Pathog', 'volume': '10', 'issn': '11', 'pages': '', 'other_ids': {'DOI': ['e1004395.Epub2014/11/07.doi:39610.1371/journal.ppat.1004395'], 'PMID': ['25375777']}}, 'BIBREF151': {'ref_id': 'b151', 'title': 'The origins of new pandemic viruses: the acquisition of new host 398 ranges by canine parvovirus and influenza A viruses. Annual review of microbiology', 'authors': [{'first': 'C', 'middle': ['R'], 'last': 'Parrish', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Kawaoka', 'suffix': ''}], 'year': 2005, 'venue': '', 'volume': '59', 'issn': '', 'pages': '553--86', 'other_ids': {}}, 'BIBREF152': {'ref_id': 'b152', 'title': 'Epub 2005/09/13', 'authors': [{'first': 'W', 'middle': ['E'], 'last': 'Diehl', 'suffix': ''}, {'first': 'A', 'middle': ['E'], 'last': 'Lin', 'suffix': ''}, {'first': 'N', 'middle': ['D'], 'last': 'Grubaugh', 'suffix': ''}, {'first': 'L', 'middle': ['M'], 'last': 'Carvalho', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Kim', 'suffix': ''}, {'first': 'P', 'middle': ['P'], 'last': 'Kyawe', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '401', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1146/annurev.micro.59.030804.121059'], 'PMID': ['16153179']}}, 'BIBREF153': {'ref_id': 'b153', 'title': 'Glycoprotein with Increased Infectivity Dominated the 2013-2016 Epidemic', 'authors': [], 'year': 2016, 'venue': '', 'volume': '167', 'issn': '', 'pages': '1088--402', 'other_ids': {'DOI': ['10.1016/j.cell.2016.10.014'], 'PMID': ['27814506'], 'PMCID': ['PMC403']}}, 'BIBREF154': {'ref_id': 'b154', 'title': 'Human 405 Adaptation of Ebola Virus during the West African Outbreak', 'authors': [{'first': 'R', 'middle': ['A'], 'last': 'Urbanowicz', 'suffix': ''}, {'first': 'C', 'middle': ['P'], 'last': 'Mcclure', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Sakuntabhai', 'suffix': ''}, {'first': 'A', 'middle': ['A'], 'last': 'Sall', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Kobinger', 'suffix': ''}, {'first': 'M', 'middle': ['A'], 'last': 'Muller', 'suffix': ''}], 'year': 2016, 'venue': 'Cell', 'volume': '167', 'issn': '4', 'pages': '', 'other_ids': {'DOI': ['40610.1016/j.cell.2016.10.013'], 'PMID': ['27814505'], 'PMCID': ['PMC5101188']}}, 'BIBREF155': {'ref_id': 'b155', 'title': 'High rate of viral evolution associated with 408 the emergence of carnivore parvovirus', 'authors': [{'first': 'L', 'middle': ['A'], 'last': 'Shackelton', 'suffix': ''}, {'first': 'C', 'middle': ['R'], 'last': 'Parrish', 'suffix': ''}, {'first': 'U', 'middle': [], 'last': 'Truyen', 'suffix': ''}, {'first': 'E', 'middle': ['C'], 'last': 'Holmes', 'suffix': ''}], 'year': 2005, 'venue': 'Proc Natl Acad Sci', 'volume': '102', 'issn': '2', 'pages': '379--84', 'other_ids': {'DOI': ['10.1073/pnas.0406765102'], 'PMID': ['15626758']}}, 'BIBREF156': {'ref_id': 'b156', 'title': 'Evolution of canine parvovirus involved loss and 412 gain of feline host range', 'authors': [{'first': 'U', 'middle': [], 'last': 'Truyen', 'suffix': ''}, {'first': 'J', 'middle': ['F'], 'last': 'Evermann', 'suffix': ''}, {'first': 'E', 'middle': [], 'last': 'Vieler', 'suffix': ''}, {'first': 'C', 'middle': ['R'], 'last': 'Parrish', 'suffix': ''}], 'year': 1996, 'venue': 'Virology', 'volume': '215', 'issn': '2', 'pages': '186--195', 'other_ids': {'DOI': ['1996/01/15.doi:41310.1006/viro.1996.0021'], 'PMID': ['8560765']}}, 'BIBREF158': {'ref_id': 'b158', 'title': 'Adaptation of HIV-1 to its human host', 'authors': [], 'year': 2007, 'venue': 'Mol Biol Evol', 'volume': '24', 'issn': '8', 'pages': '1853--60', 'other_ids': {'DOI': ['41610.1093/molbev/msm110'], 'PMID': ['17545188']}}, 'BIBREF160': {'ref_id': 'b160', 'title': 'SARS-associated coronavirus', 'authors': [], 'year': 2005, 'venue': '', 'volume': '11', 'issn': '', 'pages': '1420--1424', 'other_ids': {'DOI': ['41910.3201/eid1109.040773'], 'PMID': ['16229772'], 'PMCID': ['PMC3310602']}}, 'BIBREF161': {'ref_id': 'b161', 'title': 'Evolutionary genomics of host adaptation in vesicular 20. Bollback JP, Huelsenbeck JP. Parallel Genetic Evolution Within and Between Bacteriophage 427 Species of Varying Degrees of Divergence', 'authors': [{'first': 'S', 'middle': ['K'], 'last': 'Remold', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Rambaut', 'suffix': ''}, {'first': 'P', 'middle': ['E'], 'last': 'Turner', 'suffix': ''}, {'first': 'D', 'middle': ['G'], 'last': 'Streicker', 'suffix': ''}, {'first': 'A', 'middle': ['S'], 'last': 'Turmelle', 'suffix': ''}, {'first': 'M', 'middle': ['J'], 'last': 'Vonhof', 'suffix': ''}, {'first': 'I', 'middle': ['V'], 'last': 'Kuzmin', 'suffix': ''}, {'first': 'G', 'middle': ['F'], 'last': 'Mccracken', 'suffix': ''}, {'first': 'C', 'middle': ['E'], 'last': 'Rupprecht', 'suffix': ''}], 'year': 2009, 'venue': 'Genetics', 'volume': '181', 'issn': '1', 'pages': '225--259', 'other_ids': {'DOI': ['42810.1534/genetics.107.085225']}}, 'BIBREF162': {'ref_id': 'b162', 'title': 'Phylogeny Constrains Cross-Species Emergence and Establishment of Rabies Virus in Bats', 'authors': [], 'year': 2010, 'venue': 'Science', 'volume': '431', 'issn': '5992', 'pages': '', 'other_ids': {'DOI': ['10.1126/science.1188836']}}, 'BIBREF163': {'ref_id': 'b163', 'title': 'Simultaneously reconstructing 433 viral cross-species transmission history and identifying the underlying constraints', 'authors': [{'first': 'N', 'middle': ['R'], 'last': 'Faria', 'suffix': ''}, {'first': 'M', 'middle': ['A'], 'last': 'Suchard', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Rambaut', 'suffix': ''}, {'first': 'D', 'middle': ['G'], 'last': 'Streicker', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Lemey', 'suffix': ''}], 'year': None, 'venue': 'Philos Trans R Soc', 'volume': '434', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF165': {'ref_id': 'b165', 'title': 'Phylogenetic host specificity 437 and understanding parasite sharing in primates', 'authors': [{'first': 'N', 'middle': [], 'last': 'Cooper', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Griffin', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Franz', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Omotayo', 'suffix': ''}, {'first': 'C', 'middle': ['L'], 'last': 'Nunn', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Fryxell', 'suffix': ''}], 'year': 2012, 'venue': 'Ecol Lett', 'volume': '15', 'issn': '12', 'pages': '', 'other_ids': {'DOI': ['10.1111/j.1461-0248.2012.01858.x'], 'PMID': ['22913776']}}, 'BIBREF166': {'ref_id': 'b166', 'title': 'Inferring host range dynamics from comparative data: the 440 protozoan parasites of new world monkeys', 'authors': [{'first': 'D', 'middle': [], 'last': 'Waxman', 'suffix': ''}, {'first': 'L', 'middle': ['A'], 'last': 'Weinert', 'suffix': ''}, {'first': 'J', 'middle': ['J'], 'last': 'Welch', 'suffix': ''}], 'year': 2014, 'venue': 'Am Nat', 'volume': '184', 'issn': '1', 'pages': '65--74', 'other_ids': {'DOI': ['44110.1086/676589'], 'PMID': ['24921601']}}, 'BIBREF167': {'ref_id': 'b167', 'title': 'Phylogenetically related 443 and ecologically similar carnivores harbor similar parasite assemblages', 'authors': [{'first': 'S', 'middle': [], 'last': 'Huang', 'suffix': ''}, {'first': 'Orp', 'middle': [], 'last': 'Bininda-Emonds', 'suffix': ''}, {'first': 'P', 'middle': ['R'], 'last': 'Stephens', 'suffix': ''}, {'first': 'J', 'middle': ['L'], 'last': 'Gittleman', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Altizer', 'suffix': ''}], 'year': 2013, 'venue': 'Journal of Animal Ecology', 'volume': '444', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1111/1365-2656.12160']}}, 'BIBREF168': {'ref_id': 'b168', 'title': 'A Tale of Two Phylogenies: Comparative 446', 'authors': [{'first': 'J', 'middle': ['D'], 'last': 'Hadfield', 'suffix': ''}, {'first': 'B', 'middle': ['R'], 'last': 'Krasnov', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Poulin', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Nakagawa', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF170': {'ref_id': 'b170', 'title': 'Hantavirus evolution in relation to its rodent and 448 insectivore hosts: no evidence for codivergence', 'authors': [{'first': 'C', 'middle': [], 'last': 'Ramsden', 'suffix': ''}, {'first': 'E', 'middle': ['C'], 'last': 'Holmes', 'suffix': ''}, {'first': 'M', 'middle': ['A'], 'last': 'Charleston', 'suffix': ''}], 'year': 2009, 'venue': 'Mol Biol Evol', 'volume': '26', 'issn': '1', 'pages': '143--53', 'other_ids': {}}, 'BIBREF172': {'ref_id': 'b172', 'title': 'Phylogenetic determinants of potential host shifts in 452 fungal pathogens', 'authors': [{'first': 'D', 'middle': ['M'], 'last': 'De Vienne', 'suffix': ''}, {'first': 'M', 'middle': ['E'], 'last': 'Hood', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Giraud', 'suffix': ''}], 'year': 2009, 'venue': 'Journal of Evolutionary Biology', 'volume': '22', 'issn': '12', 'pages': '', 'other_ids': {'DOI': ['10.1111/j.1420-4539101.2009.01878']}}, 'BIBREF173': {'ref_id': 'b173', 'title': 'Phylogenetic signal in plant pathogen-host range. Proceedings of the 455 National Academy of Sciences of the United States of America', 'authors': [{'first': 'G', 'middle': ['S'], 'last': 'Gilbert', 'suffix': ''}, {'first': 'Webb', 'middle': [], 'last': 'Co', 'suffix': ''}], 'year': 2007, 'venue': '', 'volume': '104', 'issn': '', 'pages': '4979--83', 'other_ids': {'DOI': ['45610.1073/pnas.0607968104']}}, 'BIBREF174': {'ref_id': 'b174', 'title': 'Small steps or giant leaps for male-killers? Phylogenetic 458 constraints to male-killer host shifts', 'authors': [{'first': 'M', 'middle': ['C'], 'last': 'Tinsley', 'suffix': ''}, {'first': 'Men', 'middle': [], 'last': 'Majerus', 'suffix': ''}], 'year': 2007, 'venue': 'Bmc Evolutionary Biology', 'volume': '7', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1186/1471-2148-7-459238']}}, 'BIBREF176': {'ref_id': 'b176', 'title': 'Specialization and geographic isolation among Wolbachia symbionts from ants and lycaenid 462 butterflies', 'authors': [], 'year': 2009, 'venue': 'Evolution', 'volume': '63', 'issn': '3', 'pages': '624--664', 'other_ids': {'DOI': ['10.1111/j.1558-5646.2008.00579.x']}}, 'BIBREF178': {'ref_id': 'b178', 'title': 'Infection success in novel hosts: An experimental and phylogenetic 465 study of Drosophila-parasitic nematodes', 'authors': [{'first': 'S', 'middle': ['J'], 'last': 'Perlman', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Jaenike', 'suffix': ''}], 'year': 2003, 'venue': 'Evolution', 'volume': '57', 'issn': '3', 'pages': '544--57', 'other_ids': {'PMID': ['466']}}, 'BIBREF179': {'ref_id': 'b179', 'title': 'The Causes and 468 Consequences of Changes in Virulence following Pathogen Host Shifts', 'authors': [{'first': 'B', 'middle': [], 'last': 'Longdon', 'suffix': ''}, {'first': 'J', 'middle': ['D'], 'last': 'Hadfield', 'suffix': ''}, {'first': 'J', 'middle': ['P'], 'last': 'Day', 'suffix': ''}, {'first': 'S', 'middle': ['C'], 'last': 'Smith', 'suffix': ''}, {'first': 'J', 'middle': ['E'], 'last': 'Mcgonigle', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Cogni', 'suffix': ''}], 'year': 2015, 'venue': 'PLoS Pathog', 'volume': '469', 'issn': '3', 'pages': '', 'other_ids': {'DOI': ['10.1371/journal.ppat.1004728'], 'PMID': ['470']}}, 'BIBREF180': {'ref_id': 'b180', 'title': 'Host phylogeny determines viral 472 persistence and replication in novel hosts', 'authors': [{'first': 'B', 'middle': [], 'last': 'Longdon', 'suffix': ''}, {'first': 'J', 'middle': ['D'], 'last': 'Hadfield', 'suffix': ''}, {'first': 'C', 'middle': ['L'], 'last': 'Webster', 'suffix': ''}, {'first': 'D', 'middle': ['J'], 'last': 'Obbard', 'suffix': ''}, {'first': 'F', 'middle': ['M'], 'last': 'Jiggins', 'suffix': ''}], 'year': 2011, 'venue': 'PLoS Pathogens', 'volume': '7', 'issn': '9', 'pages': '', 'other_ids': {'DOI': ['47310.1371/journal.ppat.1002260']}}, 'BIBREF181': {'ref_id': 'b181', 'title': 'Estimating 475 divergence dates and substitution rates in the Drosophila phylogeny', 'authors': [{'first': 'D', 'middle': ['J'], 'last': 'Obbard', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Maclennan', 'suffix': ''}, {'first': 'K-W', 'middle': [], 'last': 'Kim', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Rambaut', 'suffix': ''}, {'first': 'P', 'middle': ['M'], 'last': 'O&apos;grady', 'suffix': ''}, {'first': 'F', 'middle': ['M'], 'last': 'Jiggins', 'suffix': ''}], 'year': 2012, 'venue': 'Molecular Biology and 476 Evolution', 'volume': '29', 'issn': '11', 'pages': '3459--73', 'other_ids': {}}, 'BIBREF182': {'ref_id': 'b182', 'title': 'The copyright holder for this preprint (which was not peer-reviewed) is the', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/226175']}}, 'BIBREF183': {'ref_id': 'b183', 'title': 'Temporal patterns of fruit fly (Drosophila) evolution 478 revealed by mutation clocks', 'authors': [{'first': 'K', 'middle': [], 'last': 'Tamura', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Subramanian', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Kumar', 'suffix': ''}], 'year': 2004, 'venue': 'Mol Biol Evol', 'volume': '21', 'issn': '1', 'pages': '36--44', 'other_ids': {'DOI': ['47910.1093/molbev/msg236']}}, 'BIBREF185': {'ref_id': 'b185', 'title': 'A direct approach to false discovery rates', 'authors': [{'first': 'J', 'middle': ['D'], 'last': 'Storey', 'suffix': ''}], 'year': 2002, 'venue': 'J Roy Stat Soc B', 'volume': '64', 'issn': '', 'pages': '479--98', 'other_ids': {}}, 'BIBREF187': {'ref_id': 'b187', 'title': 'Divergent IRES elements in invertebrates', 'authors': [{'first': 'Jan', 'middle': ['E'], 'last': '', 'suffix': ''}], 'year': 2006, 'venue': 'Virus Res', 'volume': '119', 'issn': '1', 'pages': '16--28', 'other_ids': {'DOI': ['48510.1016/j.virusres.2005.10.011'], 'PMID': ['16307820']}}, 'BIBREF188': {'ref_id': 'b188', 'title': 'The novel genome organization of the insect picorna-like virus 487', 'authors': [{'first': 'K', 'middle': ['N'], 'last': 'Johnson', 'suffix': ''}, {'first': 'P', 'middle': ['D'], 'last': 'Christian', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF189': {'ref_id': 'b189', 'title': 'Drosophila C virus suggests this virus belongs to a previously undescribed virus family', 'authors': [], 'year': 1998, 'venue': 'J Gen Virol', 'volume': '488', 'issn': '', 'pages': '191--203', 'other_ids': {'PMID': ['9460942']}}, 'BIBREF190': {'ref_id': 'b190', 'title': 'Cleavage sites of the \"P3 region\" in the nonstructural polyprotein 490 precursor of a dicistrovirus', 'authors': [{'first': 'N', 'middle': [], 'last': 'Nakashima', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Nakamura', 'suffix': ''}], 'year': 2008, 'venue': 'Arch Virol', 'volume': '153', 'issn': '10', 'pages': '1955--60', 'other_ids': {'DOI': ['10.1007/s00705-008-0208-5']}}, 'BIBREF192': {'ref_id': 'b192', 'title': 'Functional analysis of structural motifs in dicistroviruses', 'authors': [{'first': 'N', 'middle': [], 'last': 'Nakashima', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Uchiumi', 'suffix': ''}], 'year': 2009, 'venue': 'Virus Res', 'volume': '493', 'issn': '2', 'pages': '', 'other_ids': {'DOI': ['10.1016/j.virusres.2008.06.006'], 'PMID': ['18621089']}}, 'BIBREF193': {'ref_id': 'b193', 'title': 'Bayesian phylogenetics with BEAUti and the 496 BEAST 1.7', 'authors': [{'first': 'A', 'middle': ['J'], 'last': 'Drummond', 'suffix': ''}, {'first': 'M', 'middle': ['A'], 'last': 'Suchard', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Xie', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Rambaut', 'suffix': ''}], 'year': 2012, 'venue': 'Mol Biol Evol', 'volume': '29', 'issn': '8', 'pages': '1969--73', 'other_ids': {'DOI': ['10.1093/molbev/mss075']}}, 'BIBREF195': {'ref_id': 'b195', 'title': 'Host range and emerging and reemerging pathogens', 'authors': [{'first': 'M', 'middle': ['E'], 'last': 'Woolhouse', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Gowtage-Sequeria', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF198': {'ref_id': 'b198', 'title': 'Adaptation of Vpu and Nef Function and the Evolution of Pandemic and Nonpandemic HIV-1 Strains', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF200': {'ref_id': 'b200', 'title': 'Emergence of influenza A viruses', 'authors': [{'first': 'R', 'middle': ['J'], 'last': 'Webby', 'suffix': ''}, {'first': 'R', 'middle': ['G'], 'last': 'Webster', 'suffix': ''}], 'year': None, 'venue': 'Philos Trans R Soc Lond B Biol Sci', 'volume': '505', 'issn': '', 'pages': '1817--1845', 'other_ids': {'DOI': ['10.1098/rstb.2001.0997'], 'PMID': ['11779380']}}, 'BIBREF202': {'ref_id': 'b202', 'title': 'Molecular dating of 508 human-to-bovid host jumps by Staphylococcus aureus reveals an association with the spread of 509 domestication', 'authors': [{'first': 'L', 'middle': ['A'], 'last': 'Weinert', 'suffix': ''}, {'first': 'J', 'middle': ['J'], 'last': 'Welch', 'suffix': ''}, {'first': 'M', 'middle': ['A'], 'last': 'Suchard', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Lemey', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Rambaut', 'suffix': ''}, {'first': 'J', 'middle': ['R'], 'last': 'Fitzgerald', 'suffix': ''}], 'year': 2012, 'venue': 'Biol Lett', 'volume': '8', 'issn': '5', 'pages': '829--861', 'other_ids': {'DOI': ['10.1098/rsbl.2012.0290'], 'PMID': ['22628096']}}, 'BIBREF203': {'ref_id': 'b203', 'title': 'Studies of Drosophila C and A viruses in Australian populations of Drosophila 512 melanogaster: Australian National University', 'authors': [{'first': 'P', 'middle': ['D'], 'last': 'Christian', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF204': {'ref_id': 'b204', 'title': 'The 514 discovery, distribution and evolution of viruses associated with Drosophila melanogaster', 'authors': [{'first': 'C', 'middle': ['L'], 'last': 'Webster', 'suffix': ''}, {'first': 'F', 'middle': ['M'], 'last': 'Waldron', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Robertson', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Crowson', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Ferrai', 'suffix': ''}, {'first': 'J', 'middle': ['F'], 'last': 'Quintana', 'suffix': ''}], 'year': 2015, 'venue': 'PLOS 515 Biology', 'volume': '13', 'issn': '7', 'pages': '', 'other_ids': {}}, 'BIBREF205': {'ref_id': 'b205', 'title': 'The Bacterial Symbiont Wolbachia Induces Resistance to 517 RNA Viral Infections in Drosophila melanogaster', 'authors': [{'first': 'L', 'middle': [], 'last': 'Teixeira', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Ferreira', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Ashburner', 'suffix': ''}], 'year': 2008, 'venue': 'Plos Biology', 'volume': '6', 'issn': '12', 'pages': '', 'other_ids': {'DOI': ['DOI51810.1371/journal.pbio.1000002']}}, 'BIBREF206': {'ref_id': 'b206', 'title': 'Previous Exposure to an RNA Virus Does Not 520', 'authors': [{'first': 'B', 'middle': [], 'last': 'Longdon', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Cao', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Martinez', 'suffix': ''}, {'first': 'F', 'middle': ['M'], 'last': 'Jiggins', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF207': {'ref_id': 'b207', 'title': 'Protect against Subsequent Infection in Drosophila melanogaster', 'authors': [], 'year': 2013, 'venue': 'Plos One', 'volume': '8', 'issn': '9', 'pages': '', 'other_ids': {'DOI': ['52110.1371/journal.pone.0073833']}}, 'BIBREF210': {'ref_id': 'b210', 'title': 'A simple method of estimating fifty per cent endpoints. The American 526', 'authors': [{'first': 'L', 'middle': ['J'], 'last': 'Reed', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Muench', 'suffix': ''}], 'year': 1938, 'venue': 'Journal of Hygiene', 'volume': '27', 'issn': '', 'pages': '493--500', 'other_ids': {}}, 'BIBREF211': {'ref_id': 'b211', 'title': 'The copyright holder for this preprint (which was not peer-reviewed) is the', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/226175']}}, 'BIBREF212': {'ref_id': 'b212', 'title': 'FastQC: a quality control tool for high throughput sequence data', 'authors': [{'first': 'S', 'middle': [], 'last': 'Andrews', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '529', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF213': {'ref_id': 'b213', 'title': 'Trimmomatic: a flexible trimmer for Illumina sequence data', 'authors': [{'first': 'A', 'middle': ['M'], 'last': 'Bolger', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Lohse', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Usadel', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF216': {'ref_id': 'b216', 'title': 'Alignment/Map format and SAMtools', 'authors': [], 'year': 2009, 'venue': '', 'volume': '25', 'issn': '', 'pages': '2078--2087', 'other_ids': {'DOI': ['53410.1093/bioinformatics/btp352'], 'PMID': ['19505943'], 'PMCID': ['PMC535']}}, 'BIBREF217': {'ref_id': 'b217', 'title': 'BEAST: Bayesian evolutionary analysis by sampling trees', 'authors': [{'first': 'A', 'middle': ['J'], 'last': 'Drummond', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Rambaut', 'suffix': ''}], 'year': 2007, 'venue': 'BMC 537 Evolutionary Biology', 'volume': '7', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF218': {'ref_id': 'b218', 'title': 'PubMed PMID: ISI:000253468300001. 539 58. Team RDC. R: a language and environment for statistical computing', 'authors': [], 'year': 2006, 'venue': '', 'volume': '540', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['Doi10.1186/1471-2148-7-214']}}, 'BIBREF219': {'ref_id': 'b219', 'title': 'APE: Analyses of Phylogenetics and Evolution in R language', 'authors': [{'first': 'E', 'middle': [], 'last': 'Paradis', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Claude', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Strimmer', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF221': {'ref_id': 'b221', 'title': 'Estimation of levels of gene flow from DNA sequence 543 data', 'authors': [{'first': 'R', 'middle': ['R'], 'last': 'Hudson', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Slatkin', 'suffix': ''}, {'first': 'W', 'middle': ['P'], 'last': 'Maddison', 'suffix': ''}], 'year': 1992, 'venue': 'Genetics', 'volume': '132', 'issn': '2', 'pages': '', 'other_ids': {'PMID': ['1427045']}}, 'BIBREF222': {'ref_id': 'b222', 'title': 'MCMC Methods for Multi-Response Generalized Linear Mixed Models: The 546', 'authors': [{'first': 'J', 'middle': ['D'], 'last': 'Hadfield', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF224': {'ref_id': 'b224', 'title': 'Algorithm AS 159: An Efficient Method of Generating Random R &#xd7', 'authors': [{'first': 'W', 'middle': ['M'], 'last': 'Patefield', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF225': {'ref_id': 'b225', 'title': 'Tables with Given Row and Column Totals', 'authors': [], 'year': 1981, 'venue': '', 'volume': '30', 'issn': '', 'pages': '91--98', 'other_ids': {'DOI': ['10.2307/2346669']}}, 'BIBREF226': {'ref_id': 'b226', 'title': 'The copyright holder for this preprint (which was not peer-reviewed) is the', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/226175']}}}, 'ref_entries': {'FIGREF0': {'text': '50 experimental evolution studies, where parallel genetic changes occur repeatedly when replicate viral 51 lineages adapt to a new host species in the lab. For example, when Vesicular Stomatitis Virus was 52 passaged in human or dog cells, the virus evolved parallel mutations when evolved on the same cell 53 type [18]. Likewise, a study passaging Tobacco Etch Potyvirus on four plant species found parallel 54 mutations occurred only when the virus infected the same host species [19]. These parallel 55 mutations provide compelling evidence that these genetic changes are adaptive, with the same 56 mutations evolving independently in response to natural selection [20].', 'latex': None, 'type': 'figure'}, 'FIGREF2': {'text': 'The frequency of SNPs in viral lineages that have evolved in different host species. Each', 'latex': None, 'type': 'figure'}, 'FIGREF3': {'text': '100 row represents an independent viral lineage. Viruses that evolved in different host species are 101 separated by black horizontal lines. Each column represents a polymorphic site in the DCV genome, 102 and only sites where the derived allele frequency >0.05 in at least two lineages are shown. The 103 intensity of shading represents the derived allele frequency. Sites where there are three alleles have 104 the two derived allele frequencies pooled for illustrative purposes. Sites with SNP frequencies that 105 are significantly correlated among lineages from the same host species are shown by red stars at the 106 bottom the column (permutation test; p<0.05). Open reading frames (ORFs) and viral proteins based 107 on predicted polyprotein cleavage sites [38-42] are below the x axis. Information on the distribution 108 of mutations across the genome and whether they are synonymous or non-synonymous can be 109 found in the supplementary results. Sites with missing data are shown in white. The phylogeny was 110 inferred under a relaxed molecular clock [33, 43] and the scale axis represents the approximate age 111 since divergence in millions of years (my) based on estimates from: [35, 36].', 'latex': None, 'type': 'figure'}, 'FIGREF6': {'text': 'Viral lineages from the same host species were genetically more similar to each other 114 than to lineages from different host species. The mean pairwise F ST between all possible pairs of 115 viral lineages from the same host species was calculated. The red line shows the observed value. The 116 grey bars are the null distribution of this statistic obtained by permuting the viral lineages across 117 host species 1000 times.', 'latex': None, 'type': 'figure'}, 'FIGREF7': {'text': \"125 significant positive relationship between virus F ST and host genetic distance (Figure 3B, Permutation 126 test: r=0.15, P=0.002). We lacked the statistical power to identify the specific SNPs that are causing 127 the signature of parallel evolution in Figure 3 (false discovery rate >0.49 for all SNPs). Viral lineages from more closely related host species are genetically more similar. (A) The 131 correlation between the genetic differentiation of viral lineages and the genetic distance between 132 the species they have evolved in. Linear regression line is shown in red. Genetic distances were 133 scaled so that the distance from the root to the tip of the tree was one. (B) Pearson's correlation 134 coefficient (r) of F ST between pairs of viral lineage and the genetic distance between the host species 135 they evolved in. The observed value is in red and the grey bars are the null distribution obtained by 136 permutation.\", 'latex': None, 'type': 'figure'}, 'FIGREF8': {'text': '157host[2,[21][22][23][24][25][26][27][28][29][30][31][32][33][34]. For example, in a large cross-infection experiment involving Drosophila sigma viruses 158 (Rhabdoviridae) isolated from different species of Drosophila, the viruses tended to replicate most 159 efficiently in species closely related to their natural hosts[34]. This suggests that these viruses had', 'latex': None, 'type': 'figure'}, 'FIGREF9': {'text': \"DL2) cells [50] as described in [51]. Cells were cultured at 25°C in Schneider's 191 Drosophila Medium with 10% Fetal Bovine Serum, 100 U/ml penicillin and 100 μg/ml streptomycin 192 (all Invitrogen, UK). The DCV strain used was isolated from D. melanogaster collected in Charolles, 193 France [52]. DL2 cells were seeded into two 96-well tissue culture plates at approximately 10 4 cells in 194 100 μl of media per well. Cells were allowed to adhere to the plates by incubating at 25°C for five 195 hours or over-night. Serial 1:1 dilutions of DCV were made in complete Schneider's media, giving a 196 range of final dilutions from 1:10 8 -1:4x10 14 . 100 μl of these dilutions were then added to the cells 197 and incubated for 7 days, 8 replicates were made for each DCV dilution. Each well was then 198 examined for DCV infection of the DL2 cells, and a well was scored as positive for DCV infection if 199 clear cytopathic effects were present in the majority of the cells. The media was taken from the200wells with the greatest dilution factor that were scored as infected with DCV and stored at -80°C.\", 'latex': None, 'type': 'figure'}, 'FIGREF10': {'text': \"Details of the fly stocks used can be found in the supplementary materials. 216 217 4-11 day old males were infected with DCV using a 0.0125 mm diameter stainless steel needle 218 (26002-10, Fine Science Tools, CA, USA) dipped in DCV solution. For the first passage this was the 219 cloned DCV isolate in cell culture supernatant (described above), and then subsequently was the 220 virus extracted from the previous passage (described below). The needle was pricked into the 221 pleural suture on the thorax of flies, towards the midcoxa. Each replicate was infected using a new 222 needle and strict general cleaning procedures were used to minimise any risk of cross-contamination 223 between replicates. Species were collected and inoculated in a randomised order each passage. Flies 224 were then placed into vials of cornmeal medium and kept at 22°C and 70% relative humidity. Flies 225 were snap frozen in liquid nitrogen 3 days post-infection, homogenised in Ringer's solution (2.5μl 226 per fly) and then centrifuged at 12,000g for 10 mins at 4°C. The resulting supernatant was removed 227 and frozen at -80°C to be used for infecting flies in the subsequent passage. The remaining 228 homogenate was preserved in Trizol reagent (Invitrogen) and stored at -80°C for RNA extraction.\", 'latex': None, 'type': 'figure'}, 'FIGREF11': {'text': 'The 3 day viral incubation period was chosen based on time course and pilot data showing that viral 230 load reaches a maximum at approximately 3 days post-infection. This process was repeated for 10 231 passages for all species, except D. montana where only 8 passages were carried out due to the fly 232 stocks failing to reproduce. Each lineage was injected into a mean of 11 flies at each passage (range 233 4-18). Experimental evolution studies in different tissue types have seen clear signals of adaptation 234 in 100 virus generations [18]. Based on log 2 change in RNA viral load we estimate that we have 235 passaged DCV for approximately 100-200 generations.', 'latex': None, 'type': 'figure'}, 'FIGREF12': {'text': '245 position 62-9050bp (8989bp) of the Genbank refseq (NC_001834.1) giving 97% coverage of the 246 genome. PCRs of individual genomes were pooled and purified with Ampure XP beads (Agencourt).', 'latex': None, 'type': 'figure'}, 'FIGREF13': {'text': 'Nextera XT libraries (Illumina)  were prepared for each viral lineage. In total we sequenced was used to assess read quality and primer contamination. Trimmomatic, 253 version 0.32 [55] was used to removed low quality bases and adaptor sequences, using the following254 options: MINLEN=30 (Drop the read if it is below 30 base pairs), TRAILING=15 (cut bases of the end 255 of the read if below a threshold quality of 15), SLIDINGWINDOW=4:20 (perform a sliding window 256 trimming, cutting once the average quality within a 4bp window falls below a threshold of 20), and 257 ILLUMINACLIP=TruSeq3-PE.fa:2:20:10:1:true (remove adapter contamination; the values correspond 258 in order to: input fasta file with adapter sequences to be matched, seed mismatches, palindrome clip 259 threshold, simple clip threshold, minimum adapter length and logical value to keep both reads in 260 case of read-through being detected in paired reads by palindrome mode).', 'latex': None, 'type': 'figure'}, 'FIGREF14': {'text': '263 remove unused PCR primers and dNTPs and then sequenced directly using BigDye reagents (ABI) on 264 an ABI 3730 capillary sequencer in both directions (Source Bioscience, Cambridge, UK). Sequences 265 were edited in Sequencher (version 4.8; Gene Codes), and were manually checked for errors. Fastq 266 reads were independently aligned to this reference sequence (Genbank accession: MG570143) using 267 BWA-MEM, version 0.7.10 {Li, 2009 #1605} with default options with exception of the parameter -268 M, which marks shorter split hits as secondary. 99.5% of reads had mapping phred quality scores of 269 >60. The generated SAM files were converted to their binary format (BAM) and sorted by their 270 leftmost coordinates with SAMtools, version 0.1.19 (website: http://samtools.sourceforge.net/) [56].', 'latex': None, 'type': 'figure'}, 'FIGREF15': {'text': 'Group information (RG) was added to the BAM files using the module 272 AddOrReplaceReadGroups from Picard Tools, version 1.126 (https://broadinstitute.github.io/picard).', 'latex': None, 'type': 'figure'}, 'FIGREF16': {'text': 'The variant calling was then performed for each individual BAM using UnifiedGenotyper tool from 274 GATK, version 3.3.0. As we were interested in calling low frequency variants in our viruses, we275 assumed a ploidy level of 100 (-sample_ploidy:100). The other parameters were set to their defaults 276 except --stand_call_conf:30 (minimum phred-scaled confidence threshold at which variants should 277 be called) and --downsample_to_coverage:1000 (down-sample each sample to 1000X coverage) 278 Host phylogeny 279 We used a trimmed version of a phylogeny produced previously [33]. This time-based tree (where 280 the distance from the root to the tip is equal for all taxa) was inferred using seven genes with a 281 relaxed molecular clock model in BEAST (v1.8.0) [43, 57]. The tree was pruned to the 19 species used 282 using the Ape package in R [58, 59].', 'latex': None, 'type': 'figure'}, 'FIGREF18': {'text': 'H b is the mean number of differences between pairs of sequence reads sampled from the two 299 different lineages. H w is mean number of differences between sequence reads sampled from within 300 each lineage. H b and H w were calculated separately for each polymorphic site, and the mean across 301 sites used in equation', 'latex': None, 'type': 'figure'}, 'FIGREF19': {'text': '327either genetic distance from the host DCV was isolated from (D. melanogaster) or estimated viral 328 population size (see supplementary Figures S1 and S2) using generalised linear mixed models that 329 include the phylogeny as a random effect in the MCMCglmm package in R [61] as described ST', 'latex': None, 'type': 'figure'}, 'FIGREF20': {'text': 'experiment: BL, FMJ. Lab work: JPD, SCLS, TMH, LT, BL. Bioinformatics analysis: JMA, JEM, 348 FMJ, BL. Statistical analysis: BL, FMJ. Manuscript written by BL and FMJ with input from all authors.349 350Acknowledgements351Thanks to the Drosophila species stock centre for providing fly stocks and four anonymous reviewers 352 for constructive comments.', 'latex': None, 'type': 'figure'}, 'FIGREF21': {'text': 'and FMJ are supported by a Natural Environment Research Council (NE/L004232/1 357 http://www.nerc.ac.uk/) and by an European Research Council grant (281668, DrosophilaInfection, 358 http://erc.europa.eu/). JMA was supported by a grant from the Portuguese Ministério da Ciência, 359 Tecnologia e Ensino Superior (SFRH/BD/72381/2010). BL is supported by a Sir Henry Dale Fellowship 360 jointly funded by the Wellcome Trust and the Royal Society (Grant Number 109356/Z/15/Z). 361 362 421 stomatitis virus. Mol Biol Evol. 2008;25(6):1138-47. Epub 2008/03/21. doi: 10.1093/molbev/msn059.', 'latex': None, 'type': 'figure'}, 'FIGREF23': {'text': 'Lafforgue G, Elena SF. Multihost experimental evolution of a plant RNA virus 424 reveals local adaptation and host-specific mutations. Mol Biol Evol. 2012;29(5):1481-92. Epub 425 2012/02/10. doi: 10.1093/molbev/msr314. PubMed PMID: 22319146.', 'latex': None, 'type': 'figure'}}, 'back_matter': []}\n",
      "{'paper_id': 'b23ef04fdeffdf21cf7c32a977c96d5b6a2c194c', 'metadata': {'title': 'Multiple-Input Deep Convolutional Neural Network Model for COVID-19 Forecasting in China 2', 'authors': [{'first': 'Chiou-Jye', 'middle': [], 'last': 'Huang', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Jiangxi University of Science and Technology', 'location': {'addrLine': '4 Jiangxi', 'postCode': '341000', 'settlement': 'Ganzhou', 'country': 'China'}}, 'email': 'cjhuang@jxust.edu.cn'}, {'first': 'Yung-Hsiang', 'middle': [], 'last': 'Chen', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'National Pingtung University of Science and Technology', 'location': {'addrLine': 'Pingtung 6 91201', 'country': 'Taiwan'}}, 'email': 'yhchen@mail.npust.edu.tw7'}, {'first': 'Yuxuan', 'middle': [], 'last': 'Ma', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Jiangxi University of Science and Technology', 'location': {'addrLine': '4 Jiangxi', 'postCode': '341000', 'settlement': 'Ganzhou', 'country': 'China'}}, 'email': ''}, {'first': 'Ping-Huan', 'middle': [], 'last': 'Kuo', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'National Pingtung University', 'location': {'addrLine': 'Pingtung 8 90004', 'country': 'Taiwan'}}, 'email': 'phkuo@mail.nptu.edu.tw'}]}, 'abstract': [{'text': 'COVID-19 is spreading all across the globe. Up until March 23, 2020, the confirmed cases in 173 12 countries and regions of the globe had surpassed 346,000, and more than 14,700 deaths had resulted. The 13 confirmed cases outside of China had also reached over 81,000, with over 3,200 deaths. In this study, a 14 Convolutional Neural Network (CNN) was proposed to analyze and predict the number of confirmed cases. 15 Several cities with the most confirmed cases in China were the focus of this study, and a COVID-19 16 forecasting model, based on the CNN deep neural network method, was proposed. To compare the overall 17 efficacies of different algorithms, the indicators of mean absolute error and root mean square error were 18 applied in the experiment of this study. The experiment results indicated that compared with other deep 19 learning methods, the CNN model proposed in this study has the greatest prediction efficacy. The feasibility 20 and practicality of the model in predicting the cumulative number of COVID-19 confirmed cases were also 21 verified in this study.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': '22', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}], 'body_text': [{'text': 'CoV-2), and it has, at the time of writing, spread in over 173 countries, causing more than 15,000 deaths. The 31 epidemic is still spreading, and the WHO has classified the global risk level of the disease as a \"pandemic.\" 32 Currently, the earliest known case symptoms appeared on December 1, 2019, and the first case sought treatment 33 on December 8. On December 26, 2019, Wuhan City Respiratory and Intensive care doctor Zhang Jixian first 34 discovered this pneumonia with unknown cause and suspected that it is an infectious disease. Subsequently, the 35 disease broke out in Wuhan City. On January 20, 2020, Chinese academic Zhong Nanshan publicly announced 36 that the novel coronavirus pneumonia \"definitely transmits between people.\" On January 23, 2020, the Wuhan 37 City Government announced the adoption of lockdown and quarantine measures in the infected areas, which 38 was the first case of lockdown in a major city (with a population of 11 million) in recent public health history.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'Since January 13, the disease continued to spread to Thailand, Japan, and South Korea. On January 30, three 40 countries outside China were verified as having interpersonal propagation, and the pandemic was thus 41 designated as an international public health emergency event by the WHO.', 'cite_spans': [], 'ref_spans': [], 'section': '39'}, {'text': 'According to reports on the epidemic, \"the propagation speed is faster, and the virus propagation power 43 has increased.\" Infected people can infect others with the virus while exhibiting no symptoms; and the latency 44 from infection to the presentation of symptoms is as long as 14 days. These characteristics also increase the 45 difficulty of controlling the epidemic. As the epidemic continues, there is also a problem of the global 46 undersupply of surgical masks. Currently, no vaccine and remedy for the novel coronavirus have been 47 discovered. WHO assistant director-general Bruce Aylward stated that Remdesivir is currently the only drug 48 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '42'}, {'text': 'The copyright holder for this preprint . https: //doi.org/10.1101/2020.03.23.20041608 doi: medRxiv preprint 2 of 16 that is \"considered to probably have real efficacy.\" The China-Japan Friendship Hospital in China and the 49 American National Institute of Allergy and Infectious Diseases are starting to conduct clinical trials for the drug.', 'cite_spans': [{'start': 48, 'end': 85, 'text': '//doi.org/10.1101/2020.03.23.20041608', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'At present, no sufficient knowledge regarding the disease is known, and key factors, such as virus source, virus 51 birthplace, morbid mechanism, virus pathogenicity, and propagation power are still uncertain. The WHO stated 52 on March 3 that with the novel coronavirus epidemic, the world is in an \"unknown state.\" 53 Up until March 23, 2020, 345,297 CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 320, 'end': 352, 'text': 'Up until March 23, 2020, 345,297', 'ref_id': None}], 'ref_spans': [], 'section': '50'}, {'text': 'The copyright holder for this preprint . https://doi.org/10.1101/2020.03.23.20041608 doi: medRxiv preprint 3 of 16 test based on clustered regularly interspaced short palindromic repeats (CRISPR) genome editing and selected 102 67 viruses and subspecies as well as subspecies that are highly similar to 2019-nCoV. Thus, 67 test methods 103 were designed in the ADAPT system to increase the sensitivity and speed of virus detection. 104 Riou et al. [13] ', 'cite_spans': [{'start': 436, 'end': 452, 'text': 'Riou et al. [13]', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': \"The structure of this study proceeds as follows. The first section introduces the entire study. The second 135 section is a literature review. The following third section details this study's proposed deep neural network 136 algorithm. The fourth section presents and discusses the experiment results in detail. Finally, the fifth section 137 concludes the entire study and underscores the contribution of this study and the predicted trend of the COVID-138 19 epidemic.\", 'cite_spans': [], 'ref_spans': [], 'section': '134'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. ', 'cite_spans': [], 'ref_spans': [], 'section': '139'}, {'text': 'The structure principle of the fully connected layer can be expressed by Equation (2):', 'cite_spans': [], 'ref_spans': [], 'section': '167'}, {'text': 'In (2) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '169'}, {'text': 'The copyright holder for this preprint ', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'The difference between the RNN and LSTM is that for LSTM, a cell state is added to store long-term states. In 193 the neural unit model structure in Fig. 4 (a) , the internal structure of LSTM can be divided into the input gate, 194 forget gate, and output gate. The principle of the LSTM input gate is expressed in the following formulae.', 'cite_spans': [], 'ref_spans': [{'start': 149, 'end': 159, 'text': 'Fig. 4 (a)', 'ref_id': 'FIGREF9'}], 'section': '192'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '195'}, {'text': 'The copyright holder for this preprint ', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'The forget gate of the LSTM uses a sigmoid layer and a dot product to allow information to pass through 203 selectively. Equation (7) allows the LSTM to decide whether to forget the related information of the previous 204 cell, at a certain probability, in which is the weight matrix, and is the offset term.', 'cite_spans': [], 'ref_spans': [], 'section': '202'}, {'text': '206', 'cite_spans': [], 'ref_spans': [], 'section': '205'}, {'text': 'The output gate of LSTM decides which states are required to be maintained by the input ℎ −1 and 207 according to Equations (8) and (9) ', 'cite_spans': [], 'ref_spans': [], 'section': '205'}, {'text': 'The deep LSTM structure proposed in this study is illustrated in Fig. 4 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [{'start': 65, 'end': 71, 'text': 'Fig. 4', 'ref_id': 'FIGREF9'}], 'section': '211'}, {'text': 'The copyright holder for this preprint ', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'The principle of the GRU reset gate is described in Equation (11). The reset information is reserved 230 after ℎ −1 and are separately combined with the weight matrix.', 'cite_spans': [], 'ref_spans': [], 'section': '229'}, {'text': '= ( ⋅ [ℎ −1 , ])(11)', 'cite_spans': [], 'ref_spans': [], 'section': '231'}, {'text': 'The input information of the current moment is maintained with the obtained at the weight 233 probability. The following Formula (12) is used to calculate the information of the current moment h . ', 'cite_spans': [], 'ref_spans': [], 'section': '232'}, {'text': 'The structure of the proposed deep GRU is shown in Fig. 5 ', 'cite_spans': [], 'ref_spans': [{'start': 51, 'end': 57, 'text': 'Fig. 5', 'ref_id': 'FIGREF12'}], 'section': '238'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '256'}, {'text': 'The copyright holder for this preprint . https: //doi.org/10.1101 //doi.org/10. /2020 Algorithm 1 Convolutional Neural Network Input, target, and hyperparameter setting:', 'cite_spans': [{'start': 48, 'end': 65, 'text': '//doi.org/10.1101', 'ref_id': None}, {'start': 66, 'end': 85, 'text': '//doi.org/10. /2020', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'Load the training set and testing set.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'Reshape the input into the corresponding three-dimensional matrix.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': '1. The first dimension is the sample index.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': '2. The second dimension is the time step.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': '3. The third dimension is the characteristic.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'Reshape the output into a two-dimensional matrix.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': '1. The first dimension is the sample target index. The established CNN model is used to predict the testing set. The real value is compared with the predicted value, and the prediction error is calculated.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'Data on confirmed cases of COVID-19 from January 23, 2020 to March 2, 2020, and from January 258 23, 2020 to March 2, 2020, were obtained from Surging News Network (a media outlet) [9] and WHO', 'cite_spans': [], 'ref_spans': [], 'section': '257'}, {'text': '[31], respectively. In this experiment, the two evaluation indexes of the mean absolute error (MAE) and root 260 mean square error (RMSE) were used; their formulae are presented in Equations (14) and (15) Tables I, II, III, and IV. The   264   . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [{'start': 205, 'end': 245, 'text': 'Tables I, II, III, and IV. The   264   .', 'ref_id': 'TABREF6'}], 'section': '259'}, {'text': 'The copyright holder for this preprint . https://doi.org/10.1101/2020.03.23.20041608 doi: medRxiv preprint 9 of 16 experiment results indicated that the GRU has decent efficacy, and that CNN is the best performing algorithm 265 among its counterparts tested. The experiment demonstrated that the characteristic extraction of CNN is very 266 helpful for predicting the number of confirmed cases of COVID-19. factor was adopted as the input layer: cumulative confirmed cases.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '279'}, {'text': 'The copyright holder for this preprint . https://doi.org/10.1101/2020.03.23.20041608 doi: medRxiv preprint factor was adopted as the input layer: cumulative confirmed cases.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '289'}, {'text': 'The copyright holder for this preprint . https: //doi.org/10.1101 //doi.org/10. /2020 ', 'cite_spans': [{'start': 48, 'end': 65, 'text': '//doi.org/10.1101', 'ref_id': None}, {'start': 66, 'end': 85, 'text': '//doi.org/10. /2020', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '299'}, {'text': 'The copyright holder for this preprint . https: //doi.org/10.1101 //doi.org/10. /2020 ', 'cite_spans': [{'start': 48, 'end': 65, 'text': '//doi.org/10.1101', 'ref_id': None}, {'start': 66, 'end': 85, 'text': '//doi.org/10. /2020', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '311'}, {'text': 'The copyright holder for this preprint . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'The copyright holder for this preprint . https://doi.org/10.1101/2020.03.23.20041608 doi: medRxiv preprint ', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}], 'bib_entries': {'BIBREF1': {'ref_id': 'b1', 'title': 'Risk Assessment of Novel Coronavirus 375 COVID-19 Outbreaks Outside China', 'authors': [{'first': 'P', 'middle': [], 'last': 'Boldog', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Tekeli', 'suffix': ''}, {'first': 'Z', 'middle': [], 'last': 'Vizi', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Dénes', 'suffix': ''}, {'first': 'F', 'middle': ['A'], 'last': 'Bartha', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Röst', 'suffix': ''}], 'year': None, 'venue': 'Journal of Clinical Medicine', 'volume': '2020', 'issn': '2', 'pages': '', 'other_ids': {}}, 'BIBREF2': {'ref_id': 'b2', 'title': 'Optimization Method for Forecasting', 'authors': [{'first': 'M', 'middle': ['A A'], 'last': 'Al-Qaness', 'suffix': ''}, {'first': 'A', 'middle': ['A'], 'last': 'Ewees', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Fan', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF3': {'ref_id': 'b3', 'title': 'Confirmed Cases of COVID-19 in China', 'authors': [], 'year': None, 'venue': 'J Clin Med', 'volume': '2020', 'issn': '3', 'pages': '', 'other_ids': {}}, 'BIBREF5': {'ref_id': 'b5', 'title': 'Real-Time Estimation of the Risk of Death from Novel Coronavirus 380 (COVID-19) Infection: Inference Using Exported Cases', 'authors': [{'first': 'R', 'middle': [], 'last': 'Kinoshita', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Nishiura', 'suffix': ''}], 'year': None, 'venue': 'Journal of Clinical Medicine', 'volume': '2020', 'issn': '2', 'pages': '', 'other_ids': {}}, 'BIBREF7': {'ref_id': 'b7', 'title': 'Epidemic Spread of the 2019 Novel Coronavirus Driven by Spring Festival Transportation in China: A 383 Population-Based Study', 'authors': [], 'year': 2020, 'venue': 'Int J Environ Res Public Health', 'volume': '17', 'issn': '5', 'pages': '', 'other_ids': {}}, 'BIBREF8': {'ref_id': 'b8', 'title': 'Clinical 385 characteristics and imaging manifestations of the 2019 novel coronavirus disease (COVID-19):A multi-386 center study in Wenzhou city', 'authors': [{'first': 'W', 'middle': [], 'last': 'Yang', 'suffix': ''}, {'first': 'Q', 'middle': [], 'last': 'Cao', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Qin', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'Z', 'middle': [], 'last': 'Cheng', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Pan', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Dai', 'suffix': ''}, {'first': 'Q', 'middle': [], 'last': 'Sun', 'suffix': ''}, {'first': 'F', 'middle': [], 'last': 'Zhao', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Qu', 'suffix': ''}, {'first': 'F', 'middle': [], 'last': 'Yan', 'suffix': ''}], 'year': 2020, 'venue': 'China. J Infect', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF9': {'ref_id': 'b9', 'title': 'Li Jin and Momiao Xiong Artificial Intelligence Forecasting of Covid-19 in', 'authors': [{'first': 'Zixin', 'middle': [], 'last': 'Hu', 'suffix': ''}, {'first': 'Q', 'middle': ['G'], 'last': '', 'suffix': ''}, {'first': 'Shudi', 'middle': [], 'last': 'Li', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF10': {'ref_id': 'b10', 'title': 'Host and infectivity 390 prediction of Wuhan 2019 novel coronavirus using deep learning algorithm', 'authors': [{'first': 'Q', 'middle': [], 'last': 'Guo', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'Z', 'middle': [], 'last': 'Fang', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Tan', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Wu', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Xiao', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Zhu', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '391', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF11': {'ref_id': 'b11', 'title': 'CRISPR-based 392 surveillance for COVID-19 using genomically-comprehensive machine learning design', 'authors': [{'first': 'H', 'middle': ['C'], 'last': 'Metsky', 'suffix': ''}, {'first': 'C', 'middle': ['A'], 'last': 'Freije', 'suffix': ''}, {'first': 'T.-S', 'middle': ['F'], 'last': 'Kosoko-Thoroddsen', 'suffix': ''}, {'first': 'P', 'middle': ['C'], 'last': 'Sabeti', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Myhrvold', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '393', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF15': {'ref_id': 'b15', 'title': 'Time-varying transmission dynamics of Novel Coronavirus Pneumonia in China', 'authors': [{'first': 'J', 'middle': [], 'last': 'He', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Ma', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '398', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF16': {'ref_id': 'b16', 'title': 'Breaking down of healthcare system: Mathematical modelling 399 for controlling the novel coronavirus (2019-nCoV) outbreak in Wuhan', 'authors': [{'first': 'W.-K', 'middle': [], 'last': 'Ming', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Huang', 'suffix': ''}, {'first': 'C', 'middle': ['J P'], 'last': 'Zhang', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '400', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF18': {'ref_id': 'b18', 'title': 'How to make more 402 from exposure data? An integrated machine 1 learning pipeline to predict pathogen exposure 2019', 'authors': [{'first': 'N', 'middle': [], 'last': 'Fountain-Jones', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Machado', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Carver', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Packer', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Mendoza', 'suffix': ''}, {'first': 'M', 'middle': ['E'], 'last': 'Craft', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '403', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF19': {'ref_id': 'b19', 'title': 'Application of the ARIMA model 404 on the COVID-2019 epidemic dataset', 'authors': [{'first': 'D', 'middle': [], 'last': 'Benvenuto', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Giovanetti', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Vassallo', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Angeletti', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Ciccozzi', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF20': {'ref_id': 'b20', 'title': 'Trend and forecasting of the COVID-19 outbreak in China', 'authors': [{'first': 'Q', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Feng', 'suffix': ''}, {'first': 'Y', 'middle': ['H'], 'last': 'Quan', 'suffix': ''}], 'year': 2020, 'venue': 'J Infect', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF21': {'ref_id': 'b21', 'title': 'International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not peer-reviewed) The copyright holder for this preprint', 'authors': [{'first': '', 'middle': [], 'last': 'Cc-By-Nc-Nd', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.03.23.20041608']}}, 'BIBREF23': {'ref_id': 'b23', 'title': 'Characteristics of COVID-19 infection in 408', 'authors': [{'first': 'Y', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Lian', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Niu', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Zhang', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF26': {'ref_id': 'b26', 'title': 'Clinical characteristics and intrauterine vertical transmission potential of 411', 'authors': [{'first': 'H', 'middle': [], 'last': 'Hou', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': '', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF27': {'ref_id': 'b27', 'title': 'COVID-19 infection in nine pregnant women: a retrospective review of medical records. The Lancet', 'authors': [], 'year': 2020, 'venue': '', 'volume': '412', 'issn': '', 'pages': '809--815', 'other_ids': {}}, 'BIBREF29': {'ref_id': 'b29', 'title': 'Bearing Capacity Using Artificial Neural Network and Random Forest', 'authors': [], 'year': 2020, 'venue': 'Applied Sciences', 'volume': '415', 'issn': '10', 'pages': '', 'other_ids': {}}, 'BIBREF31': {'ref_id': 'b31', 'title': 'Mechanism Using Artificial Neural Network Based Customer Profiles in Smart Grid', 'authors': [], 'year': None, 'venue': '', 'volume': '2020', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF32': {'ref_id': 'b32', 'title': 'The Influence of the Activation Function in a Convolution Neural 419', 'authors': [{'first': 'Y', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Song', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Rong', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF35': {'ref_id': 'b35', 'title': 'CNN Method Based on DRNet and RoI Align for Building Detection in Remote Sensing Images', 'authors': [], 'year': 2020, 'venue': '', 'volume': '12', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF36': {'ref_id': 'b36', 'title': 'Towards a Smart Smoking Cessation App: A', 'authors': [{'first': 'M', 'middle': [], 'last': 'Abo-Tabik', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Costen', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Darby', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Benn', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF38': {'ref_id': 'b38', 'title': 'Public Environment Emotion Prediction Model Using LSTM 426 Network', 'authors': [{'first': 'Q', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Gao', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Liu', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Zheng', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '12', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF39': {'ref_id': 'b39', 'title': 'Semi-Supervised Bidirectional Long Short-Term Memory and 428 Conditional Random Fields Model for Named-Entity Recognition Using Embeddings from Language 429', 'authors': [{'first': 'M', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Geng', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Chen', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF41': {'ref_id': 'b41', 'title': 'Hybrid Deep Learning Predictor for', 'authors': [{'first': 'X', 'middle': ['B'], 'last': 'Jin', 'suffix': ''}, {'first': 'N', 'middle': ['X'], 'last': 'Yang', 'suffix': ''}, {'first': 'X', 'middle': ['Y'], 'last': 'Wang', 'suffix': ''}, {'first': 'Y', 'middle': ['T'], 'last': 'Bai', 'suffix': ''}, {'first': 'T', 'middle': ['L'], 'last': 'Su', 'suffix': ''}, {'first': 'J', 'middle': ['L'], 'last': 'Kong', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF42': {'ref_id': 'b42', 'title': 'Smart Agriculture Sensing Based on Empirical Mode Decomposition and Gated Recurrent Unit Group 432', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF44': {'ref_id': 'b44', 'title': 'An Optimal Feature Parameter Set Based on Gated Recurrent Unit', 'authors': [{'first': 'Ö', 'middle': [], 'last': 'Batur Dinler', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Aydin', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}}, 'ref_entries': {'FIGREF0': {'text': \"December 2019, the first case of COVID-19 infection was discovered in Wuhan City in China's 26 Hubei Province [1]. In the following weeks, this disease broke out within China and continued to spread 27 extensively in other countries, causing worldwide panic. The pneumonia caused by this novel coronavirus was 28 officially named the Coronavirus disease 2019 (COVID-19) by the World Health Organization (WHO). The 29 2019 coronavirus epidemic was triggered by the Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-\", 'latex': None, 'type': 'figure'}, 'FIGREF1': {'text': '73 Moreover, the Galton-Watson branch was used to analogize virus propagation, thus determining the 74 connectivity and transmissibility between China and the destination country. The disadvantage of this method 75 is that the uncertainty regarding the efficacy of control measures against the disease results in difficulties in 76 predicting the development trajectories of the disease. By comparison, Al-Qaness et al. [6] adopted the adaptive 77 neuro-fuzzy inference system (ANFIS), an improvement from flower pollination algorithm and salp swarm 78 algorithm, to predict the number of confirmed cases in the next 10 days. The comparison with the ANFIS 79 optimized with GA, PSO, ABC, and FPA indicated that the accuracy of the FPASSA-ANFIS model reached 80 0.97, and the predicted average increase in numbers of new cases in the next 10 days was 10% more than the 81 number of present cases. Jung et al. [7] used the rate of increase together with delayed distribution estimation 82 and statistical induction to establish a mathematical model, based on COVID-2019 case data reported before 83 January 24. The predicted cumulative number of confirmed cases for up until January 24 was 6,924, and the 84 predicted death ratio was 5.3%; these figures were within the 95% confidence interval. Compared with previous 85 methods for establishing mathematical models, Fan et al. [8] adopted a statistical method for predicting the 86 floating population in Wuhan City. The experiment indicated that the floating population in Wuhan region is 87 highly correlated with the number of daily confirmed cases. The residence time of the floating population for 88 local cases was longer than that for non-local cases, which results in a lower predicted number of confirmed 89 cases in the areas around Hubei Province. The prediction results indicate that approximately 80% of the 90 epidemic will be centralized in the top 30 districts. 91 However, applications of deep learning methods have mainly used the genome of the 2019-cCoV virus to 92 predict disease propagation. Yang et al. [9] used the Susceptible-Exposed-Infectious-Removed model to 93 integrate the epidemic curve, combining it with the long short-term memory model. The maximum value was 94 reached on February 28, and the curve gradually lowered in late April. Hu et al. [10] used an improved stacked 95 autoencoder and a cluster algorithm to group the instantaneous confirmed cases in every province; they noted 96 a high accuracy in AI-based methods for COVID-19 trajectory prediction. The epidemic was predicted to end 97 in mid-April. Guo et al. [11] used deep learning-based virus host prediction to compare the gene sequence of 98 2019-nCoVs with those of Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV), bat SARS corona 99 virus, and Middle East respiratory syndrome-related coronavirus (MERS-CoV). The bat SARS coronavirus 100 was discovered to have a more similar mode of infection to COVID-19. Metsky et al. [12] used a nucleic acid 101 .', 'latex': None, 'type': 'figure'}, 'FIGREF2': {'text': 'nCoV has the potential for continual infection between people.Liu et al. [14]  compared 2019-nCoV with 107 SARS and estimated the doubling time of 2019-nCoV and SARS as well as the basic reproduction number r0 108 and time-varying instantaneous reproduction number rt through data analysis. They discovered that the 109 infectivity of 2019-nCoV may be stronger than SARS, but that disease control efforts are effective.Ming et al.   110    [15] adopted an improved version of the Susceptible-Infectious-Recovered (SIR) model to predict the actual 111 number of infection cases of 2019-nCoV as well as the actual load on the intensive care units under different 112 diagnosis rates and public health intervention efficacies. They discovered that under a 50% diagnosis rate and 113 no public health intervention, the actual number of cases will be significantly higher than the reported number, 114 whereas under a 70% public health intervention, the load on the health system will decrease substantially. Zhao115 et al. [16] used a probability prediction model to accurately predict the infection nodes from snapshots of 116 spreading. Fountain-Jones et al. [17] used machine learning to establish pathogen-risk models and compared 117 the prediction results of different machine learning methods and explained the results by using game theory. 118 Benvenuto et al. [18] adopted an autoregressive integrated moving average model to predict the epidemic 119 trend and morbidity of 2019-nCoV with a 95% confidence interval. They discovered that if the virus does not 120 mutate, the case number will reach a plateau. Li et al. [19] used a function to separately describe the daily 121 infection and death data of 2019-nCoV in Hubei and outside Hubei. The author thought that the inflection point 122 in the Hubei region was at February 6, 2020, and that the epidemic will end on March 10, 2020, with an 123 estimated infection rate of 39,000 people. However, the data did not include the data after February 12, where 124 such inclusion will increase the predicted figures by 1.4 times. Yang et al. [9] described the clinical 125 characteristics and imaging manifestation of the infected cases of COVID-19 that had been confirmed in the 126 hospitals in Wenzhou area. In addition, the patients with and without travel or residence history in Hubei were 127 compared. Tian et al. [20] analyzed the clinical and epidemiological characteristics of COVID-19 in the Beijing 128 area. The characteristics of severe confirmed cases and common confirmed cases were compared, and the 129 characteristics of COVID-19 and SARS were compared as well. Chen et al. [21] analyzed some cases of 130 pregnant women with COVID-19 and noted no evidence that COVID-19 causes intrauterine infection through 131 vertical transmission among women in late pregnancy. The major contribution of that study is its fast 132 establishment of a prediction model by using deep learning with a small dataset, making it an important 133 reference for other countries in their containment of the COVID-19 epidemic.', 'latex': None, 'type': 'figure'}, 'FIGREF3': {'text': 'Global Map of Confirmed COVID-19 Cases as of 12:00 a.m. ET, March 21, 2020 [2].', 'latex': None, 'type': 'figure'}, 'FIGREF4': {'text': 'Neural Network (ANN) [22, 23] is a mathematical model that mimics how neurons operate 144 in organisms. It is a very powerful tool for establishing nonlinear models. An early ANN structure is the 145 Multilayer Perceptron (MLP), which uses the neural network of fully connected structures; it has decent efficacy 146 and numerous applications. However, if the data is too complex, simply using the MLP structure may cause the 147 model to be unable to effectively learn how to handle every situation. At present, ANN has already developed 148 into various new structures. The main structure of this study is the Convolutional Neural Network (CNN) [24, 149 25], and three other neural network structures are compared with it. These four neural network structures are 150 detailed in the following subsections.', 'latex': None, 'type': 'figure'}, 'FIGREF5': {'text': 'convolution operation[26]  is illustrated inFig. 2. The difference between CNN and MLP is that 153 CNN uses the concept of weight sharing. As illustrated inFig. 2, the advantage of CNN is that its weight number 154 does not have to be as large as that for a fully connected structure. Consequently, the training is relatively easier, 155 and important characteristics are extracted more effectively. CNN is a feedforward neural network, which is an 156 improvement from MLP. CNN contains four levels in structure: an input layer, convolutional layer, pooling 157 layer, and fully connected layer.158The deep CNN structure proposed in this study is illustrated inFig. 2, and the structure is composed of 159 CNN and the dropout layer. The input layer structure is constituted by six time sequences composed of factors 160 influencing the cumulative number of cases. The convolutional layer structure is divided into four layers, and 161 each convolution layer has 16, 32, 32, and 64 convolutional kernels. Going through every convolutional layer, 162 the convolutional kernels of the previous layer slide in the input data matrixes. The convolution process can be 163 expressed in Equation (1), in which is the ith and jth matrix in the row and column direction, respectively, 164 corresponding to the convolution kernels in the input matrix through smooth shifting. K is the', 'latex': None, 'type': 'figure'}, 'FIGREF6': {'text': ', is the vector composed of output values, is the vector composed of input values, is the 170 vector composed of weight values, is the vector composed of threshold values, and is the activation 171 function. The output of the convolution layer can yield a 1D vector through Flatten expansion. Subsequently, 172 the vectors are connected to the fully connected layer (dense) to obtain a 1D output. At the time of writing', 'latex': None, 'type': 'figure'}, 'FIGREF7': {'text': 'is an ANN model that includes the input layer, hidden layer, and output layer. The structure of the 179 ANN is illustrated in Fig. 3. Different connection layers are all fully connected, and the principle is similar to 180 that of MLP. Each neural unit is connected with a weight coefficient ( ), and the combination of transmission 181 result and the deviation is calculated using Equation (3). In addition, current neural unit output of the proposed deep MLP is illustrated in Fig. 3. The structure is composed of two levels of 185 MLP, and each MLP layer has 16 neural units. The input structure is constituted by six factors influencing the 186 cumulative number of cases, with the time step of 5. Through a Flatten expansion, a 1 × 30 vector is derived, 187 and the output is obtained after connecting the vector to the fully connected layer (dense).188 Structure of the comparative deep neural network MLP adopted in this study.', 'latex': None, 'type': 'figure'}, 'FIGREF8': {'text': 'Long short-term memory neural network 191The long short-term memory model (LSTM)[27, 28]  is an improvement from the recurrent neural network.', 'latex': None, 'type': 'figure'}, 'FIGREF9': {'text': '(b). The structure is composed of 212 two LSTM layers, and each layer has 16 neural units. The input structure is constituted by six factors influencing 213 the cumulative number of cases, with a time step of 5. A 1D output is obtained through the fully Structure of the comparative deep neural network LSTM adopted in this study. (a) LSTM neural unit; (b) the 220 deep neural network LSTM model with an input layer consisting six input signals.', 'latex': None, 'type': 'figure'}, 'FIGREF10': {'text': \"Gate recurrent unit 222 The Gate Recurrent Unit (GRU) [29, 30] is a variation of the LSTM, as illustrated in Fig. 5 (a). GRU 223 changes the input gate and forget gate of LSTM into the update gate. GRU retains the effect of LSTM and 224 simplifies the internal structure of GRU model units. The principle of GRU's update gate is described in 225 Equation (10). Specifically, the neural unit information of the previous moment ℎ −1 and the input data of\", 'latex': None, 'type': 'figure'}, 'FIGREF12': {'text': '(b). The structure is composed of two GRU 239 layers, and each layer has 16 neural units. The input structure is constituted by six factors influencing the 240 cumulative number of cases, with a time step of 5. A 1D output is obtained through the fully connected layer. Structure of the comparative deep neural network GRU adopted in this study. (a) GRU neural unit; (b) 247 the deep neural network GRU model with an input layer consisting six input signals.', 'latex': None, 'type': 'figure'}, 'FIGREF13': {'text': 'The second dimension is the sample target. Set relevant parameters of the CNN model. Training: Initialize the weight w s → Total number of training set samples c → Batch size a → 0 for i to epoch then for k to s then j = k mod c if j = 0 then j = c end if j − a > 0 then Calculate the output using Equations (1) and (2) for training set samples in the batch.else Calculate the loss using loss function, update the weight using optimizer, and calculate the output using Equations (1) and (2) for training set samples in the next batch.', 'latex': None, 'type': 'figure'}, 'FIGREF14': {'text': '. To conduct 261 comprehensive efficacy tests, the dataset for January 23 to February 17 was chosen as the training data, and the 262 dataset for February 18 to March 2 was used as testing data. Figs. 6 to 12 present the comparative prediction 263 results of all the algorithms. The detailed MAE and RMSE values are listed in', 'latex': None, 'type': 'figure'}, 'FIGREF15': {'text': 'Comparison of prediction results for the cumulative number of confirmed cases in Wuhan City, Hubei 270 Province, China. The comparison used the deep neural networks CNN, MLP, LSTM, and GRU. (a) Six 271 important factors were adopted as the input layer: new confirmed cases, new deceased cases, new cured cases, 272 cumulative confirmed cases, cumulative deceased cases, and cumulative cured cases. (b) Only one important 273 factor was adopted as the input layer: cumulative confirmed cases. Comparison of prediction results for the cumulative number of confirmed cases in Huanggang City, 275 Hubei Province, China. The comparison used the deep neural networks CNN, MLP, LSTM, and GRU. (a) Six 276 important factors were adopted as the input layer: new confirmed cases, new deceased cases, new cured cases, 277 cumulative confirmed cases, cumulative deceased cases, and cumulative cured cases. (b) Only one important 278', 'latex': None, 'type': 'figure'}, 'FIGREF16': {'text': 'Comparison of prediction results for the cumulative number of confirmed cases in Xiaogan City, Hubei 280 Province, China. The comparison used the deep neural networks CNN, MLP, LSTM, and GRU. (a) Six 281 important factors were adopted as the input layer: new confirmed cases, new deceased cases, new cured cases, 282 cumulative confirmed cases, cumulative deceased cases, and cumulative cured cases. (b) Only one important 283 factor was adopted as the input layer: cumulative confirmed cases. Comparison of prediction results for the cumulative number of confirmed cases in Ezhou City, Hubei 285 Province, China. The comparison used the deep neural networks CNN, MLP, LSTM, and GRU. (a) Six 286 important factors were adopted as the input layer: new confirmed cases, new deceased cases, new cured cases, 287 cumulative confirmed cases, cumulative deceased cases, and cumulative cured cases. (b) Only one important 288', 'latex': None, 'type': 'figure'}, 'FIGREF17': {'text': 'Comparison of prediction results for the cumulative number of confirmed cases in Yichang City, Hubei 290 Province, China. The comparison used the deep neural networks CNN, MLP, LSTM, and GRU. (a) Six 291 important factors were adopted as the input layer: new confirmed cases, new deceased cases, new cured cases, 292 cumulative confirmed cases, cumulative deceased cases, and cumulative cured cases. (b) Only one important 293 factor was adopted as the input layer: cumulative confirmed cases. Comparison of prediction results for the cumulative number of confirmed cases in Wenzhou City, 295 Zhejiang Province, China. The comparison used the deep neural networks CNN, MLP, LSTM, and GRU. (a) 296Six important factors were adopted as the input layer: new confirmed cases, new deceased cases, new cured 297 cases, cumulative confirmed cases, cumulative deceased cases, and cumulative cured cases. (b) Only one 298 important factor was adopted as the input layer: cumulative confirmed cases.', 'latex': None, 'type': 'figure'}, 'FIGREF18': {'text': 'Comparison of prediction results for the cumulative number of confirmed cases in Shenzhen City, 300 Guangdong Province, China. The comparison used the deep neural networks CNN, MLP, LSTM, and GRU. (a) 301 Six important factors were adopted as the input layer: new confirmed cases, new deceased cases, new cured 302 cases, cumulative confirmed cases, cumulative deceased cases, and cumulative cured cases. (b) Only one 303 important factor was adopted as the input layer: cumulative confirmed cases.304', 'latex': None, 'type': 'figure'}, 'FIGREF19': {'text': 'MAE radar charts. (a) Six factors were adopted as the input layer. (b) Only one factor was adopted as the radar charts of MAE values for the comparison between the output results 329 and real values for six and one factors, respectively. Regardless of whether one or six factors were used, the 330 prediction results, in terms of MAE, of the proposed CNN model were superior to those of its counterparts. For 331 the proposed CNN model, the use of six factors yielded better prediction results, in terms of MAE, in 332 comparison to the use of only one factor, particularly for Wuhan CityRMSE radar charts. (a) Six pieces of important information were adopted as the input layer. (b) Only one piece of 335 information was adopted as the input layer.336 Fig. 14 (a) and (b) presents the radar charts of RMSE values for the comparison between the output results 337 and real values for six and one factors, respectively. The results further demonstrated the excellent predictive 338 performance of the proposed CNN model for the cumulative number of confirmed cases of COVID-19, making 339 the model a valuable reference for other countries in their establishment of country-specific COVID-19 340 prediction models. In general, the prediction performances of the models from most favorable to least favorable 341 were those for CNN, GRU, LSTM, and MLP.', 'latex': None, 'type': 'figure'}, 'TABREF0': {'text': \"people have been infected with the virus; among these people, 14,765 54 have died. The several European countries with the most severe epidemics have experienced the maximum 55 increase in confirmed cases in a single day on March 5: cases for Italy increased from 3,089 to 3,858, those for Germany increased from 262 to 482, those for France increased from 285 to 423, and those for the Netherlands 57 increased more than twofold from 38 to 82. In the WHO report on February 26, the numbers of new cases in China and Japan have decreased, but the numbers of new cases in Italy, Iran, and South Korea are still rising.At the time of writing, the numbers of confirmed cases in Germany, France, and Spain have surpassed 1,000.Fig. 1 [2] presents the global map of COVID-19 occurrence according to data from the United States' Centers for Disease Control and Prevention. The countries marked with a color are countries with confirmed 63 cases. At present, 173 of the 195 countries in the world have confirmed cases, underscoring the epidemic's 64 astounding speed of propagation from its origin in China's Hubei province. Most scholars have used 65 mathematical models to predict the spread of COVID-19. Roosa et al. [3] used the generalized logistic model, 66 the Richards model, and the sub-epidemic model to establish prediction models for the cumulative numbers of 67 confirmed cases in Guangdong Province and Zhejiang Province. They predicted that, in the next 5 days and 10 68 days from their time of writing, 65-81 and 44-354 new cases will appear in Guangdong and Zhejiang provinces, 69 respectively. In addition, Liu et al. [4] used mathematical models to simulate the propagation status of COVID-70 19 in China and noted the importance of the government's policy of limiting public activities and the movement 71 of infected people with no symptoms. Boldog et al. [5] adopted a Time-Dependent Compartmental Model of 72 the Transmission Dynamics to estimate the cumulative number of confirmed cases outside of Hubei Province.\", 'latex': None, 'type': 'table'}, 'TABREF1': {'text': 'COVID-19 epidemic has continued for only 3 months, 1 week, and 3 days. Thus, considering the small number 174 of obtained samples, this model prevents overfitting by introducing the method of dropout regularization.Fig. 2. Structure of the proposed deep CNN model.', 'latex': None, 'type': 'table'}, 'TABREF2': {'text': 'Equation (4) is used to decide which piece of information is to be added by passing ℎ −1 and through the', 'latex': None, 'type': 'table'}, 'TABREF3': {'text': '. The final output results are obtained by passing the new information', 'latex': None, 'type': 'table'}, 'TABREF6': {'text': 'EXPERIMENTAL RESULTS IN TERMS OF MEAN ABSOLUTE ERROR OF THE INPUT LAYER OF SIX NEURONS', 'latex': None, 'type': 'table'}, 'TABREF7': {'text': '', 'latex': None, 'type': 'table'}, 'TABREF8': {'text': '. https://doi.org/10.1101/2020.03.23.20041608 doi: medRxiv preprint 13 of 16 EXPERIMENTAL RESULTS IN TERMS OF ROOT MEAN SQUARE ERROR OF THE INPUT LAYER OF SIX NEURONS', 'latex': None, 'type': 'table'}, 'TABREF9': {'text': 'EXPERIMENTAL RESULTS IN TERMS OF ROOT MEAN SQUARE ERROR OF THE INPUT LAYER OF A SINGLERegarding the deep neural network prediction results for six and one-input factors, the MAE and RMSE318 values are listed in Tables 1 to 4. The MAE results from low to high are as follows: CNN (102.943, 284.573), GRU (2599.150, 1916.904), LSTM (2992.976, 3324.591), and MLP (5710.293, 5811.429). The RMSE results 320 from low to high are as follows: CNN (109.439, 325.857), GRU (2602.666, 1924.361), LSTM (2994.851, 321 3331.925), and MLP (5715.125, 5816.079). The experiment results demonstrated that GRU and LSTM have 322 decent efficacies and that CNN had the best performance. This experiment also demonstrated that the procedure 323 of initial characteristic extraction through CNN and the subsequent input of characteristic values to the CNN 324 structure greatly aid the prediction of the cumulative number of confirmed cases of COVID-19.', 'latex': None, 'type': 'table'}, 'TABREF10': {'text': \"Conclusions343A novel and multi-input CNN deep neural network model was proposed in this study to predict the 344 cumulative number of confirmed cases of COVID-19. The cumulative number of confirmed cases in the next 345 day is predicted according to the previous five days' total number of confirmed cases, total confirmed new cases, 346 total cured cases, total cured new cases, total deaths, and total new deaths. In the experiment, the datasets of 347 those seven Chinese cities with severe confirmed cases-from Hubei Province, Guangdong Province, and Zhejiang Province-were used for the models' training and prediction. Because the COVID-19 epidemic is 349 ongoing, the algorithm proposed in this study can rapidly use small datasets to establish models with high350 predictive accuracy, different from many other studies. In addition, with the algorithm, a deep learning network 351 prediction model for the number of confirmed cases of COVID-19 was established, and the verification and 352 comparison were conducted among different deep learning algorithms. The accuracy and reliability of the deep 353 learning algorithm were verified by having it predict the future trend of COVID-19. Furthermore, experiments 354 for multiple cities with more severe confirmed cases in China indicated that the prediction model of this study 355had the lowest error rate among its counterparts tested. However, at the time of writing, countries other than China have had COVID-19 outbreaks, such as Italy, South Korea, Iran, Spain, France, and Germany. In the 357 future, the establishment of more complete databases will lead to improvements to the proposed prediction 358 model. For example, deep learning networks with a mixed structure can be introduced to establish more accurate 359 models, which can be applied to more countries. The predicted trends can aid the containment of the COVID-360 19 epidemic and extend the scope of application of artificial intelligence.\", 'latex': None, 'type': 'table'}}, 'back_matter': [{'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.(which was not peer-reviewed) The copyright holder for this preprint ', 'cite_spans': [], 'ref_spans': [], 'section': '365'}, {'text': '438 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.(which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10. 1101 /2020 ', 'cite_spans': [{'start': 258, 'end': 262, 'text': '1101', 'ref_id': None}, {'start': 263, 'end': 268, 'text': '/2020', 'ref_id': None}], 'ref_spans': [], 'section': '437'}]}\n",
      "{'paper_id': 'b2de691570027e843e3af655efc42bf87ac56c7f', 'metadata': {'title': 'A prospect on the use of antiviral drugs to control local outbreaks of COVID-19', 'authors': [{'first': 'Andrea', 'middle': [], 'last': 'Torneri', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Antwerp', 'location': {'settlement': 'Antwerp', 'country': 'Belgium'}}, 'email': ''}, {'first': '†', 'middle': [], 'last': '', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Pieter', 'middle': [], 'last': 'Libin', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Hasselt University', 'location': {'settlement': 'Hasselt', 'country': 'Belgium'}}, 'email': ''}, {'first': 'Joris', 'middle': [], 'last': 'Vanderlocht', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Hasselt University', 'location': {'settlement': 'Hasselt', 'country': 'Belgium'}}, 'email': ''}, {'first': 'Anne-Mieke', 'middle': [], 'last': 'Vandamme', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Rega Institute for Medical Research', 'location': {'settlement': 'Leuven', 'country': 'Belgium'}}, 'email': ''}, {'first': 'Johan', 'middle': [], 'last': 'Neyts', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Rega Institute for Medical Research', 'location': {'settlement': 'Leuven', 'country': 'Belgium'}}, 'email': ''}, {'first': 'Niel', 'middle': [], 'last': 'Hens', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Antwerp', 'location': {'settlement': 'Antwerp', 'country': 'Belgium'}}, 'email': ''}]}, 'abstract': [{'text': 'Background: Current outbreaks of COVID-19 are threatening the health care systems of several countries around the world. Control measures, based on isolation and quarantine, have been shown to decrease and delay the burden of the ongoing epidemic. With respect to the ongoing COVID-19 epidemic, recent modelling work shows that this intervention technique may be inadequate to control local outbreaks, even when perfect isolation is assumed. Furthermore, the effect of infectiousness prior to symptom onset combined with a significant proportion of asymptomatic infectees further complicates the use of contact tracing. Antivirals, which decrease the viral load and reduce the infectiousness, could be integrated in the control measures in order to augment the feasibility of controlling the epidemic. Methods: Using a simulation-based model of viral transmission we tested the efficacy of different intervention measures for the control of COVID-19. For individuals that were identified through contact tracing, we evaluate two procedures: monitoring individuals for symptoms onset and testing of individuals. Moreover, we investigate the effect of a potent antiviral compound on the contact tracing process. Findings: The use of an antiviral drug, in combination with contact tracing, quarantine and isolation, results in a significant decrease of the final size, the peak incidence, and increases the probability that the outbreak will fade out. Interpretation: For an infectious disease in which presymptomatic infections are plausible, an intervention measure based on contact tracing performs better when realized together with testing instead of monitoring, provided that the test is able to detect infections during the incubation period. In addition, in all tested scenarios, the model highlights the benefits of the administration of an antiviral drug in addition to quarantine, isolation and contact tracing. The resulting control measure, could be an effective strategy to control local and re-emerging outbreaks of COVID-19.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}], 'body_text': [{'text': 'the model to represent the viral load decrease thereof. 23 In this manuscript we first present the effect of isolation, considering both home quarantine (for 24 individuals that are part of a contact trace network and for infected individuals with mild symptoms) 25 and hospital isolation (for severe cases). We argue that when an individual is quarantined at home, 26 this will only result in a partial reduction of contacts, accounting for contacts with household 27 members and other isolation imperfectness. To compensate for this imperfect isolation, we consider 28 the use of an antiviral compound. We test these different control measures in a simulation study 29 that aims at representing, given the available information, the current COVID-19 epidemic. While 30 at this phase of the epidemic, many countries are already beyond the point of local containment. We 31 do however expect that the methodology we propose will be key to avoid a second peak, especially 32 given the limited depletion of susceptibles. Individuals are initially susceptible (S) and once infected, they enter the exposed class (E). The on the notion of infectious contact processes. 5 First, contacts between individuals are generated.', 'cite_spans': [{'start': 56, 'end': 58, 'text': '23', 'ref_id': None}, {'start': 263, 'end': 265, 'text': '25', 'ref_id': None}, {'start': 366, 'end': 368, 'text': '26', 'ref_id': None}, {'start': 971, 'end': 973, 'text': '32', 'ref_id': None}, {'start': 1165, 'end': 1166, 'text': '5', 'ref_id': 'BIBREF4'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'When such contacts are generated between susceptible and infectious people, these can result in 45 an infection event according to a Bernoulli probability value based on the time since infection.', 'cite_spans': [{'start': 96, 'end': 98, 'text': '45', 'ref_id': None}], 'ref_spans': [], 'section': '44'}, {'text': 'This probability is computed, at a precise time point, as the product of two components: the 47 infectiousness measure, ν(t), which quantifies the level of infectiousness over time, and the total 48 amount of infectivity q, i.e. the number of infections over the contact rate. 6 The function ν(t) is 49 defined over the exposed and infectious period, or analogously over the incubation and symptomatic 50 period, along which it integrates to one. This function is scaled to have a similar shape among 51 different infectives, based on their lengths of exposed and infectious period. According to this 52 framework, an infectious individual makes effective contacts at a rate, r(t) given by:', 'cite_spans': [{'start': 277, 'end': 278, 'text': '6', 'ref_id': 'BIBREF5'}], 'ref_spans': [], 'section': '46'}, {'text': 'where λ is the contact rate. The mean number of effective contacts is an approximation of the 54 reproduction number. The two quantities are identical in an infinite and homogeneous population, 55 where the probability of making two effective contacts with the same person is zero. For the 56 considered population size, the probability of this event is extremely low. Therefore, throughout 57 the manuscript, we approximate the reproduction number with the mean number of secondary 58 cases.', 'cite_spans': [{'start': 194, 'end': 196, 'text': '55', 'ref_id': None}], 'ref_spans': [], 'section': '46'}, {'text': 'In the considered framework, isolation/quarantine is implemented by reducing the contact rate λ at 60 the time of diagnosis. To date, little is known about the difference in viral load among severe and 61 mild cases. Zhou et al. 7 indicate that in nasal and throat swabs the viral load is higher in mild cases.', 'cite_spans': [{'start': 217, 'end': 230, 'text': 'Zhou et al. 7', 'ref_id': None}], 'ref_spans': [], 'section': '59'}, {'text': 'Due to this uncertainty, we assume that the same curve is defined for all the infected individuals.', 'cite_spans': [], 'ref_spans': [], 'section': '62'}, {'text': 'In this work, we assume the population to be homogeneous, closed and finite population. These 64 assumptions relate to the control measures currently in place, e.g. in Italy, aiming at containing 65 immigration and emigration in a country with an ongoing outbreak. 66 ', 'cite_spans': [{'start': 265, 'end': 267, 'text': '66', 'ref_id': None}], 'ref_spans': [], 'section': '63'}, {'text': 'In Table 1 we report the parameters and distributions that were utilized in the simulation study.', 'cite_spans': [], 'ref_spans': [{'start': 3, 'end': 10, 'text': 'Table 1', 'ref_id': 'TABREF0'}], 'section': 'Simulation parameters and distributions'}, {'text': 'Where distributions are not reported, the parameters are assumed to be constant. In the last 68 column, when available, we report the references to the literature that justifies the choice of the 69 parameter value, or distribution, we use. 70 We assume infectious individuals to make, on average, between two and three effective contacts. nasal and throat swabs data. 7 The time to hospitalization is estimated from the data presented in 78 Backer et al. 10 We assume that the time to hospitalization coincides with the time of diagnosis. At with SARS-CoV-2, a contact tracing procedure is started. We assume that each contact in H i will 85 be traced back successfully with probability η. Depending on the considered scenario, traced-back 86 individuals will be monitored or tested. When individuals are found positive for SARS-CoV-2, they 87 will be put in quarantine/isolation. For certain scenarios, next to quarantine, we also inject infected 88 individuals with an antiviral drug (i.e., Remdesivir). 89 We assume that the test can detect infection after two days since infection. Traced individuals who 90 test negative the first time, are tested again after two days. The quarantine will result in a decreased 91 contact rate (i.e. imperfect isolation), λ q , while in case of perfect isolation the contact rate is set 92 to zero. Similarly, diagnosed individuals will also be quarantined: at home (mild symptoms), with 93 a decreased contact rate λ q , or in the hospital (severe symptoms), where we assume that perfect ', 'cite_spans': [{'start': 241, 'end': 243, 'text': '70', 'ref_id': None}, {'start': 369, 'end': 370, 'text': '7', 'ref_id': 'BIBREF6'}, {'start': 439, 'end': 441, 'text': '78', 'ref_id': None}, {'start': 456, 'end': 458, 'text': '10', 'ref_id': 'BIBREF9'}, {'start': 1007, 'end': 1009, 'text': '89', 'ref_id': None}], 'ref_spans': [], 'section': '67'}, {'text': 'To compensate for imperfect isolation we investigate the use of antiviral compounds to reduce the 98 infectiousness of an infected individual. We assume that, once the antiviral compound has been 99 administered, the infectiousness measure will exponentially decay according to an inverse Malthusian 100 growth model (shown in Figure 2 ). 17 The rate of this decay is set to represent the reduction in viral 101 load, due to Remdesevir, as reported in 4 for the MERS coronavirus. 102 4 All rights reserved. No reuse allowed without permission.', 'cite_spans': [{'start': 339, 'end': 341, 'text': '17', 'ref_id': 'BIBREF16'}, {'start': 480, 'end': 483, 'text': '102', 'ref_id': None}], 'ref_spans': [{'start': 327, 'end': 335, 'text': 'Figure 2', 'ref_id': None}], 'section': 'Antiviral compounds'}, {'text': 'author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Antiviral compounds'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.19.20038182 doi: medRxiv preprint Figure 2 : Reduction of infectiousness. The blue and the orange lines describe the infectiousness measure, respectively, before (dashed blue) and after antiviral injection (solid yellow). The red arrows indicate the injection times.', 'cite_spans': [], 'ref_spans': [{'start': 144, 'end': 152, 'text': 'Figure 2', 'ref_id': None}], 'section': 'Antiviral compounds'}, {'text': 'We assume the following parameters for the reduction of contacts because of home quarantine: λ q = 103 0·1λ, 0·25λ, 0·5λ and for the probability of tracing back a contact in history H i : η = 0·25,0·5,0·75 18 .', 'cite_spans': [], 'ref_spans': [], 'section': 'Scenarios'}, {'text': 'Scenarios: In all the considered scenarios we assume that individuals are isolated, or quarantined, 105 when diagnosed. Moreover, we assume that contact tracing starts at the time of diagnosis.', 'cite_spans': [], 'ref_spans': [], 'section': '104'}, {'text': 'IAS: Traced individuals are monitored for two weeks, and isolated/quarantined if they show symptoms during this period. This scenario is similar to the baseline scenario described by Hellewell et al. 1 with the exception that in our description only severe cases are isolated while the mild are home quarantined. This scenario reflects more realistically the current practice of containment.', 'cite_spans': [{'start': 200, 'end': 201, 'text': '1', 'ref_id': 'BIBREF0'}], 'ref_spans': [], 'section': '106'}, {'text': 'IBS: Traced individuals are isolated/quarantined, as soon as they test positive for SARS-COV-2.', 'cite_spans': [], 'ref_spans': [], 'section': '106'}, {'text': 'We assume that an individual that is infected tests positive 2 days after infection. Therefore, a traced individual is tested immediately when traced, and, if this test was negative, we test the individual again two days later.', 'cite_spans': [], 'ref_spans': [], 'section': '106'}, {'text': 'IBTBS: A diagnosed patient is immediately treated with the antiviral drug. Furthermore, traced individuals are isolated/quarantined and injected with the antiviral drug, as soon as they test positive for SARS-COV-2. We assume that an individual that is infected tests positive 2 days after infection. Therefore, a traced individual is tested immediately when traced, and, if this test was negative, we test the individual again two days later.', 'cite_spans': [], 'ref_spans': [], 'section': '106'}, {'text': 'For each scenario we run 5000 simulations. Among these, we compute the final size and the cases 107 at peak for the one in which at least the 10% of individuals have been infected. Doing this, we only 108 account for outbreaks that are most challenging to contain. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '106'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.19.20038182 doi: medRxiv preprint to successfully trace contacts and on the reduction in contact rate due to quarantine (Figure 3 ).', 'cite_spans': [], 'ref_spans': [{'start': 230, 'end': 239, 'text': '(Figure 3', 'ref_id': 'FIGREF9'}], 'section': '106'}, {'text': 'Isolation and quarantine lead to a substantial decrease in final size and peak incidence. When 114 performed prior to symptom onset their efficacy increases, which is important, as recent work 115 indicates that infectees are infectious prior to symptom onset. 19 The antiviral treatment is shown for Scenario IAS (yellow), Scenario IBS (green) and Scenario IBTBS (blue) when the quarantine contact rate is λ q =0·25λ together with the probability that a simulation leads to a number of cases smaller than the 10% of the population (purple asterisks). Right panel: mean peak incidence together with the 2·5 % and the 97·5 % percentiles.', 'cite_spans': [{'start': 193, 'end': 196, 'text': '115', 'ref_id': None}, {'start': 261, 'end': 263, 'text': '19', 'ref_id': None}], 'ref_spans': [], 'section': '106'}, {'text': 'to have a substantial impact and, together with quarantine and isolation, significantly reduces the 117 final size, the peak incidence and the number of outbreaks that are most challenging to contain.', 'cite_spans': [], 'ref_spans': [], 'section': '116'}, {'text': 'We assume that we have sufficient antiviral drugs doses to treat all individuals that are encountered 119 via the contact tracing procedure. This is motivated by the fact that we consider an emerging 120 outbreak and the required number of doses will thus be limited.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Furthermore, we assume that all the individuals will show symptoms, sooner or later, during their 122 infectious period, and therefore all infected individuals will be diagnosed. Due to the awareness of show that, for the tested scenarios, the use of an antiviral drug increases the control of the outbreak, 131 even when undiagnosed individuals are circulating in the population.', 'cite_spans': [], 'ref_spans': [], 'section': '121'}, {'text': 'In the hospital, we assume perfect isolation, meaning that infected individuals cannot spread the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '132'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the ', 'cite_spans': [], 'ref_spans': [], 'section': '132'}, {'text': 'The ongoing epidemic of COVID-19 threatens the health system of many countries. Although we consider a setting where the reproductive number between symptomatic and asymptomatic is 180 equal ( Figure 9 ) and a setting where the asymptomatic infectees have a reproduction number that is 181 55% of the reproduction number set for symptomatic infectees (Figure 10 ). 20 For both settings, we 182 show a significant advantage with respect to the reduction of the final size and the peak incidence.', 'cite_spans': [], 'ref_spans': [{'start': 193, 'end': 201, 'text': 'Figure 9', 'ref_id': None}, {'start': 351, 'end': 361, 'text': '(Figure 10', 'ref_id': 'FIGREF3'}], 'section': 'Conclusion'}, {'text': 'Yet, our analysis also shows that when the asymptomatic are as infectious as the symptomatic 184 infectees, it is important to have a high contact tracing success rate (η). 185 We remain hopeful that the ongoing clinical trials will reveal an antiviral compound that can be ', 'cite_spans': [{'start': 173, 'end': 176, 'text': '185', 'ref_id': None}], 'ref_spans': [], 'section': '183'}, {'text': 'We report in Figure 4 and 5 the sensitivity analysis for the quarantine contact rate: λ q = 0.1λ, 0.5λ.', 'cite_spans': [], 'ref_spans': [{'start': 13, 'end': 21, 'text': 'Figure 4', 'ref_id': 'FIGREF17'}], 'section': 'Appendix: sensitivity analyses'}, {'text': 'The introduction of an antiviral compound substantially contributes in reducing the final size, the 212 peak incidence and the probability of a challenging outbreak in all the considered settings. This 213 decrease, compared to the scenario in which only isolation/quarantine is implemented, increases 214 when quarantine is less effective (left panels). In Figure 6 and 7, we vary the reproduction number 215 that is set, respectively, to R 0 = 2 and R 0 = 3. The effect of the antiviral drug, in addition to 216 isolation and quarantine, increases when the reproduction number increases. In case of R 0 = 3, case of remarkable impact in both the final size and the peak incidence. 224 We tested also the effect of an asymptomatic proportion of infectives. We assume that the 35% author/funder, who has granted medRxiv a license to display the preprint in perpetuity. Figure 5 : Mean peak incidence for Scenario IAS (yellow), Scenario IBS (green) and Scenario IBTBS (blue) when the quarantine contact rate is λ q =0·5λ (left panel) and λ q =0·1λ (right panel) together with 2·5% and 97·5% percentiles. Figure 6 : Final size distribution (left panel) and mean peak incidence for Scenario IAS (yellow), Scenario IBS (green) and Scenario IBTBS (blue) when the quarantine contact rate is λ q =0·25λ, and R 0 = 2. In the left panel, for each scenario we report the probability that a simulation leads to a number of cases smaller than the 10% of the population (purple asterisks). In the right panel, together with the point estimates we report the 2·5% and 97·5 % percentiles. Figure 7 : Final size distribution (left panel) and mean peak incidence for Scenario IAS (yellow), Scenario IBS (green) and Scenario IBTBS (blue) when the quarantine contact rate is λ q =0·25λ, and R 0 = 3. In the left panel, for each scenario we report the probability that a simulation leads to a number of cases smaller than the 10% of the population (purple asterisks). In the right panel, together with the point estimates we report the 2·5% and 97·5 % percentiles.', 'cite_spans': [{'start': 683, 'end': 686, 'text': '224', 'ref_id': None}], 'ref_spans': [{'start': 358, 'end': 366, 'text': 'Figure 6', 'ref_id': None}, {'start': 869, 'end': 877, 'text': 'Figure 5', 'ref_id': None}, {'start': 1103, 'end': 1111, 'text': 'Figure 6', 'ref_id': None}, {'start': 1574, 'end': 1582, 'text': 'Figure 7', 'ref_id': None}], 'section': '211'}, {'text': 'All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': '11'}, {'text': 'author/funder, who has granted medRxiv a license to display the preprint in perpetuity. Figure 8 : Final size distribution (left panel) and mean peak incidence for Scenario IAS (yellow), Scenario IBS (green) and Scenario IBTBS (blue) when the quarantine contact rate is λ q =0·25 λ, R 0 =2·5 and the test detect positively an infectious individual after 4 days since infection. In the left panel, for each scenario we report the probability that a simulation leads to a number of cases smaller than the 10% of the population (purple asterisks). In the right panel, together with the point estimates we report the 2·5% and 97·5 % percentiles.', 'cite_spans': [], 'ref_spans': [{'start': 88, 'end': 96, 'text': 'Figure 8', 'ref_id': None}], 'section': '11'}, {'text': 'number of value R 0 = 2.5, while in Figure 10 the reproduction number for symptomatic and 229 asymptomatic is set, respectively, to R s 0 =2·5 and R a 0 =0·55 R s 0 . 20 Also in these scenarios we noticed 230 that the administration of an antiviral drug lead to a substantial decrease in the final size and peak 231 incidence compared to the values obtained when the other control measures are considered. 232 12 All rights reserved. No reuse allowed without permission.', 'cite_spans': [{'start': 406, 'end': 409, 'text': '232', 'ref_id': None}], 'ref_spans': [{'start': 36, 'end': 45, 'text': 'Figure 10', 'ref_id': 'FIGREF3'}], 'section': '11'}, {'text': 'author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '11'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the Figure 9 : Final size distribution (left panel) and mean peak incidence for Scenario IAS (yellow), Scenario IBS (green) and Scenario IBTBS (blue) when the quarantine contact rate is λ q =0·25 λ and the proportion of asymptomatic individuals is 35%. We consider the same reproduction number for asymptomatic (R a 0 ) and symptomatic (R s 0 ) individuals: R a 0 = R s 0 = 2.5. In the left panel, for each scenario we report the probability that a simulation leads to a number of cases smaller than the 10% of the population (purple asterisks). In the right panel, together with the point estimates we report the 2·5% and 97·5 % percentiles. Figure 10 : Final size distribution (left panel) and mean peak incidence for Scenario IAS (yellow), Scenario IBS (green) and Scenario IBTBS (blue) when the quarantine contact rate is λ q =0·25 λ, R 0 =2·5 and the proportion of asymptomatic individuals is 35%. We consider the a reproductive number that is different for asymptomatic (R a 0 ) and symptomatic (R s 0 ) individuals: R s 0 = 2.5 and R a 0 = 0.55 · R s 0 . In the left panel, for each scenario we report the probability that a simulation leads to a number of cases smaller than the 10% of the population (purple asterisks). In the right panel, together with the point estimates we report the 2·5% and 97·5 % percentiles.', 'cite_spans': [], 'ref_spans': [{'start': 76, 'end': 84, 'text': 'Figure 9', 'ref_id': None}, {'start': 715, 'end': 724, 'text': 'Figure 10', 'ref_id': 'FIGREF3'}], 'section': '11'}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'Feasibility of controlling COVID-19 outbreaks by isolation of cases and contacts. The Lancet Global Health', 'authors': [{'first': 'J', 'middle': [], 'last': 'Hellewell', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Abbott', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Gimma', 'suffix': ''}, {'first': 'N', 'middle': ['I'], 'last': 'Bosse', 'suffix': ''}, {'first': 'C', 'middle': ['I'], 'last': 'Jarvis', 'suffix': ''}, {'first': 'T', 'middle': ['W'], 'last': 'Russell', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF1': {'ref_id': 'b1', 'title': 'Remdesivir and chloroquine effectively inhibit the recently emerged novel coronavirus (2019-nCoV) in vitro', 'authors': [{'first': 'M', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Cao', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Yang', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Liu', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Xu', 'suffix': ''}], 'year': 2020, 'venue': 'Cell research', 'volume': '30', 'issn': '3', 'pages': '269--271', 'other_ids': {}}, 'BIBREF2': {'ref_id': 'b2', 'title': 'Therapeutic efficacy of the small molecule GS-5734 against Ebola virus in rhesus monkeys', 'authors': [{'first': 'T', 'middle': ['K'], 'last': 'Warren', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Jordan', 'suffix': ''}, {'first': 'M', 'middle': ['K'], 'last': 'Lo', 'suffix': ''}, {'first': 'A', 'middle': ['S'], 'last': 'Ray', 'suffix': ''}, {'first': 'R', 'middle': ['L'], 'last': 'Mackman', 'suffix': ''}, {'first': 'V', 'middle': [], 'last': 'Soloveva', 'suffix': ''}], 'year': 2016, 'venue': 'Nature', 'volume': '531', 'issn': '7594', 'pages': '381--385', 'other_ids': {}}, 'BIBREF3': {'ref_id': 'b3', 'title': 'Comparative therapeutic efficacy of remdesivir and combination lopinavir, ritonavir, and interferon beta against MERS-CoV', 'authors': [{'first': 'T', 'middle': ['P'], 'last': 'Sheahan', 'suffix': ''}, {'first': 'A', 'middle': ['C'], 'last': 'Sims', 'suffix': ''}, {'first': 'S', 'middle': ['R'], 'last': 'Leist', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Schäfer', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Won', 'suffix': ''}, {'first': 'A', 'middle': ['J'], 'last': 'Brown', 'suffix': ''}], 'year': 2020, 'venue': 'Nature Communications', 'volume': '11', 'issn': '1', 'pages': '', 'other_ids': {'DOI': ['10.1038/s41467-019-13940-6']}}, 'BIBREF4': {'ref_id': 'b4', 'title': 'Counting process models for infectious disease data: distinguishing exposure to infection from susceptibility', 'authors': [{'first': 'P', 'middle': ['H'], 'last': 'Rhodes', 'suffix': ''}, {'first': 'M', 'middle': ['E'], 'last': 'Halloran', 'suffix': ''}, {'first': 'Longini', 'middle': [], 'last': 'Jr', 'suffix': ''}, {'first': 'I', 'middle': ['M'], 'last': '', 'suffix': ''}], 'year': 1996, 'venue': 'Journal of the Royal Statistical Society: Series B (Methodological)', 'volume': '58', 'issn': '4', 'pages': '751--762', 'other_ids': {}}, 'BIBREF5': {'ref_id': 'b5', 'title': 'A note on generation times in epidemic models', 'authors': [{'first': '', 'middle': [], 'last': 'Svenssonå', 'suffix': ''}], 'year': 2007, 'venue': 'Mathematical biosciences', 'volume': '208', 'issn': '1', 'pages': '300--311', 'other_ids': {}}, 'BIBREF6': {'ref_id': 'b6', 'title': 'SARS-CoV-2 viral load in upper respiratory specimens of infected patients', 'authors': [{'first': 'L', 'middle': [], 'last': 'Zou', 'suffix': ''}, {'first': 'F', 'middle': [], 'last': 'Ruan', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Huang', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Liang', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Huang', 'suffix': ''}, {'first': 'Z', 'middle': [], 'last': 'Hong', 'suffix': ''}], 'year': 2020, 'venue': 'New England Journal of Medicine', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF7': {'ref_id': 'b7', 'title': 'Evolving epidemiology of novel coronavirus diseases 2019 and possible interruption of local transmission outside Hubei Province in China: a descriptive and modeling study. medRxiv', 'authors': [{'first': 'J', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Litvinova', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Deng', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Chen', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF8': {'ref_id': 'b8', 'title': 'Early Transmission Dynamics in Wuhan, China, of Novel Coronavirus -Infected Pneumonia', 'authors': [{'first': 'Q', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Guan', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Wu', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Zhou', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Tong', 'suffix': ''}], 'year': 2020, 'venue': 'New England Journal of Medicine', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1056/NEJMoa2001316']}}, 'BIBREF9': {'ref_id': 'b9', 'title': 'Incubation period of 2019 novel coronavirus (2019-nCoV) infections among travellers from Wuhan, China', 'authors': [{'first': 'J', 'middle': ['A'], 'last': 'Backer', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Klinkenberg', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Wallinga', 'suffix': ''}], 'year': 2020, 'venue': 'Eurosurveillance', 'volume': '25', 'issn': '5', 'pages': '20--28', 'other_ids': {}}, 'BIBREF10': {'ref_id': 'b10', 'title': 'Has China faced only a herald wave of SARS-CoV-2? The Lancet', 'authors': [{'first': 'A', 'middle': [], 'last': 'Flahault', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF11': {'ref_id': 'b11', 'title': 'World Health Organization. Coronavirus disease 2019 -(COVID-19)', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF12': {'ref_id': 'b12', 'title': 'Clinical Characteristics of Coronavirus Disease 2019 in China', 'authors': [{'first': 'Guan', 'middle': [], 'last': 'Wj', 'suffix': ''}, {'first': 'Ni', 'middle': [], 'last': 'Zy', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Hu', 'suffix': ''}, {'first': 'Liang', 'middle': [], 'last': 'Wh', 'suffix': ''}, {'first': 'Ou', 'middle': [], 'last': 'Cq', 'suffix': ''}, {'first': 'He', 'middle': [], 'last': 'Jx', 'suffix': ''}], 'year': 2020, 'venue': 'New England Journal of Medicine', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF13': {'ref_id': 'b13', 'title': 'SOCRATES: An online tool leveraging a social contact data sharing initiative to assess mitigation strategies for COVID-19', 'authors': [{'first': 'L', 'middle': [], 'last': 'Willem', 'suffix': ''}, {'first': 'T', 'middle': ['V'], 'last': 'Hoang', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Funk', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Coletti', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Beutels', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Hens', 'suffix': ''}], 'year': 2003, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF14': {'ref_id': 'b14', 'title': 'A pneumonia outbreak associated with a new coronavirus of probable bat origin', 'authors': [{'first': 'P', 'middle': [], 'last': 'Zhou', 'suffix': ''}, {'first': 'X', 'middle': ['L'], 'last': 'Yang', 'suffix': ''}, {'first': 'X', 'middle': ['G'], 'last': 'Wang', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Hu', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Zhang', 'suffix': ''}], 'year': 2020, 'venue': 'Nature', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1038/s41586-020-2012-7']}}, 'BIBREF15': {'ref_id': 'b15', 'title': 'Viral load of SARS-CoV-2 in clinical samples. The Lancet Infectious Diseases', 'authors': [{'first': 'Y', 'middle': [], 'last': 'Pan', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Yang', 'suffix': ''}, {'first': 'Llm', 'middle': [], 'last': 'Poon', 'suffix': ''}, {'first': 'Q', 'middle': [], 'last': 'Wang', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF16': {'ref_id': 'b16', 'title': 'An Essay on the Principle of Population', 'authors': [{'first': 'T', 'middle': ['R'], 'last': 'Malthus', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Winch', 'suffix': ''}, {'first': 'James', 'middle': ['P'], 'last': 'Malthus', 'suffix': ''}], 'year': 1992, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF17': {'ref_id': 'b17', 'title': 'The effectiveness of contact tracing in emerging', 'authors': [{'first': 'D', 'middle': [], 'last': 'Klinkenberg', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Fraser', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Heesterbeek', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}}, 'ref_entries': {'FIGREF0': {'text': 'disease dynamics are depicted in the left panel of Figure 1. The possible transitions between 34 epidemic classes are described by the arrows.', 'latex': None, 'type': 'figure'}, 'FIGREF2': {'text': '36 infection, that is at first asymptomatic (I a ), can lead to the onset of mild (I m ) or severe symptoms 37 (I s ). Symptomatic individuals are hospitalized (H), where they are isolated, or are confined in 38 home quarantine (Q), based on the severity of symptoms. Ultimately, all infectives are assumed to 39 either recover from infection or die (R). Individuals that are hospitalized are immediately isolated; 40 therefore they can no longer transmit the disease. The quarantined individuals, can still make 41 contacts, although at a decreased rate.', 'latex': None, 'type': 'figure'}, 'FIGREF3': {'text': 'Disease dynamics: Possible transitions among the different epidemic compartments (left panel). A measure describing how infectiousness is distributed over time since infection (right panel).', 'latex': None, 'type': 'figure'}, 'FIGREF4': {'text': 'this time-point, depending on the severity of the symptoms, individuals are isolated (severe cases)80    or quarantined (mild cases). The population size is set to 500 to represent a localised outbreak of 81 COVID-19. Furthermore, we assume that the contact tracing starts when individuals are diagnosed.', 'latex': None, 'type': 'figure'}, 'FIGREF5': {'text': 'Contact tracing and isolationIn order to implement contact tracing we keep track of a contact history H i for each individual 83 i for all contacts made since the time of infection. When an individual i is found to be infected 84', 'latex': None, 'type': 'figure'}, 'FIGREF6': {'text': 'isolation is possible. In all the considered scenarios, the 16% of individuals is isolated while the 95 rest is put in quarantine. The selected proportion reflects the proportion of severe cases reported 96 in Guan et al. 13 97', 'latex': None, 'type': 'figure'}, 'FIGREF8': {'text': ', isolation and antiviral treatments lead, in different levels, to the mitigation of the 110 outbreak by reducing the final size as well as by reducing the number of cases at the peak of 111 the epidemic. The containment performance depends, among all the scenarios, on the probability 112 5 All rights reserved. No reuse allowed without permission.', 'latex': None, 'type': 'figure'}, 'FIGREF9': {'text': 'Final size and Mean peak incidence. Left panel: distributions of the final size value', 'latex': None, 'type': 'figure'}, 'FIGREF10': {'text': '123COVID-19 given by media and government officials, individuals are more likely to act upon even 124 mild symptoms. This assumption is in line with the work of Hellewell et al.1 Moreover, we assume 125 that all infected individuals will eventually be diagnosed. While this is a limitation of our study,126    we argue that in the scenarios where we aggressively trace and treat the contacts of individuals,127    we are more likely to find (and constrain) cases that would otherwise go undetected. To challenge 128 this assumption, we simulate the impact of asymptomatic individuals, defined to be infectives that129 do not show symptoms during their infectious periods. Results reported in the sensitivity analysis130', 'latex': None, 'type': 'figure'}, 'FIGREF11': {'text': 'infection. While this assumption is in line with earlier works 1 , health care workers are at risk and 134 they could be infected by infected individuals in isolation.135In the IBS and IBTBS scenarios we assume that the traced individuals that test positive are isolated136 in the 16% of cases, even before showing actual severe symptoms. 137 Although this model is informed with data on the control of MERS-CoV viral load using prophylaxis 138 with Remdesivir, it stands to reason that different classes of viral inhibitors control the viral load 139 in different ways. Additionally, despite the sequence similarity of MERS-CoV and SARS-COV-', 'latex': None, 'type': 'figure'}, 'FIGREF12': {'text': '. To this end, longitudinal data of the viral load on COVID-19 infected patients treated 142 with different viral inhibitors will be informative. Furthermore, Sheahan et al. 4 demonstrated that 143 the degree of the clinical benefit of Remdesivir for MERS depends on the viral dose and also on 144 the timing of the treatment of the viral inhibitor. A lethal viral dose and delaying the initiation of 145 antiviral treatment failed to fully prevent viral pathogenesis. Although Remdesivir proved effective 146 at reducing the viral load also in these conditions, the argument for an early start of antiviral 147 treatment is evident. Presumably, reducing the viral load with an antiviral compound loses its 148 efficacy in advanced disease as the tissue damage is sustained by inflammatory processes in absence 149 of the viral initiator. In our implementation, antiviral injections are immediately administered to 150 successfully traced back individuals, mostly in their asymptomatic phase, and to the diagnosed 151 patient. Therefore, we believe that the assumptions on the use of this drug, in the considered 152 scenario, are reasonable.', 'latex': None, 'type': 'figure'}, 'FIGREF14': {'text': '154control measures such as isolation and quarantine are of great importance, relying on their exclusive 155 use could fail to contain an outbreak. In addition, when a proportion of infected individuals requires 156 hospital care, the number of cases at peak should be minimized as much as possible to avoid 157 that regional health care infrastructure is overwhelmed. In the current study we highlight the158 impact of an antiviral compound that reduces the viral load and, consequently, the infectiousness 159 of infectives. For this simulation study we utilized the data on the control of the viral load by 160 Remdesivir in a pathogen challenge experiment, however, this study can be easily extended to other 161 antivirals, given the availability of viral load data upon drug administration. Although the efficacy 162 of administering an antiviral compound, in addition to isolation and quarantine depends on the 163 effectiveness of the respective drug, it is plausible that drugs that interfere with the infection life 164 cycle show a comparable impact on the viral load. We demonstrate that the implementation of 165 such a compound together with quarantine leads to a substantial reduction of the final size and 166 the peak incidence. In addition, the number of outbreaks that are most challenging to contain 167 decreases when the antiviral is administered to diagnosed and traced individuals. Therefore, the 168 administration of an antiviral drug, together with isolation and quarantine, is expected to have 169 a major impact in the control of local COVID-19 outbreaks. Although antiviral compounds are 170 mostly evaluated in clinical trials for their capacity to control the disease manifestations and alter 171 the clinical outcome in individual patients, our study implies that antiviral compounds can be 172 implemented to alter the spread of a viral epidemic by influencing the viral load and infectiousness 173 of infectees upon quarantine. Additionally, it is expected that the implementation of such a strategy 174during an epidemic outbreak might also prove effective to prevent hospitalisation and the pressure 175 on the health care capacity as these antiviral treatment is initiated earlier in the disease.', 'latex': None, 'type': 'figure'}, 'FIGREF15': {'text': 'Currently, there remains a great deal of uncertainty with respect to the proportion of asymptomatic 177 infectees. Therefore, we performed a sensitivity analysis where we consider a proportion of 35% 178 asymptomatic individuals, which is in line with the findings of recent works. 20,21 . For this proportion 179', 'latex': None, 'type': 'figure'}, 'FIGREF17': {'text': 'the peak incidence decreases when the antiviral compound is used, compared to control measures 218 based only on isolation and quarantine after symptoms onset. Distributions of the final size value for Scenario IAS (yellow), Scenario IBS (green) and Scenario IBTBS (blue) when the quarantine contact rate is λ q =0·5λ (left panel) and λ q =0·1λ (right panel) together with the probability that a simulation leads to a number of cases smaller than the 10% of the population (purple asterisks)InFigure 8we investigate the effect of a longer time needed for the test to detect an infectious 220 individual.We assume the test is positive when performed on an infectious individual who has been 221 infected since at least 4 days. Simulations show a substantial increase, both in the final size and 222 the peak incidence for the IBS scenario. Instead, the use of an antiviral drug results also in this 223', 'latex': None, 'type': 'figure'}, 'FIGREF18': {'text': 'of individuals will not show symptoms during their infectious periods, according to result reported 226 in recent studies. 20,21 We consider two values of the reproduction number for the asymptomatic 227 individuals. In Figure 9 asymptomatic and symptomatic individuals have the same reproduction 228 All rights reserved. No reuse allowed without permission.', 'latex': None, 'type': 'figure'}, 'TABREF0': {'text': 'Model Parameters', 'latex': None, 'type': 'table'}, 'TABREF3': {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.19.', 'latex': None, 'type': 'table'}, 'TABREF4': {'text': '. https://doi.org/10.1101/2020.03.19.', 'latex': None, 'type': 'table'}}, 'back_matter': [{'text': 'All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'annex'}]}\n",
      "{'paper_id': 'b30b43ac5cf19932f2b3c3034e2f4ea34a69e901', 'metadata': {'title': 'of a densely sampled COVID19 2 outbreak in China', 'authors': [{'first': 'Erik', 'middle': ['M'], 'last': 'Volz', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Imperial College London', 'location': {'postCode': 'W2 1PG', 'settlement': 'Norfolk Place', 'country': 'United Kingdom'}}, 'email': ''}, {'first': 'Han', 'middle': [], 'last': 'Fu', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Imperial College London', 'location': {'postCode': 'W2 1PG', 'settlement': 'Norfolk Place', 'country': 'United Kingdom'}}, 'email': ''}, {'first': 'Haowei', 'middle': [], 'last': 'Wang', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Imperial College London', 'location': {'postCode': 'W2 1PG', 'settlement': 'Norfolk Place', 'country': 'United Kingdom'}}, 'email': ''}, {'first': 'Xiaoyue', 'middle': [], 'last': 'Xi', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Imperial College London', 'location': {'postCode': 'SW7 2AZ', 'settlement': 'London', 'country': 'United 9 Kingdom'}}, 'email': ''}, {'first': 'Wei', 'middle': [], 'last': 'Chen', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Dehui', 'middle': [], 'last': 'Liu', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Yingying', 'middle': [], 'last': 'Chen', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Mengmeng', 'middle': [], 'last': 'Tian', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Wei', 'middle': [], 'last': 'Tan', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Junjie', 'middle': [], 'last': 'Zai', 'suffix': '', 'affiliation': {'laboratory': 'Immunology Innovation Team', 'institution': 'Ningbo University', 'location': {'postCode': '315211', 'settlement': 'Ningbo', 'country': 'China'}}, 'email': ''}, {'first': 'Wanying', 'middle': [], 'last': 'Sun', 'suffix': '', 'affiliation': {'laboratory': 'Shenzhen Key Laboratory of 13 Unknown Pathogen Identification, BGI-Shenzhen', 'institution': '', 'location': {'postCode': '518083', 'settlement': 'Shenzhen', 'country': 'China'}}, 'email': ''}, {'first': 'Jiandong', 'middle': [], 'last': '', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Li', 'middle': [], 'last': '', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Junhua', 'middle': [], 'last': 'Li', 'suffix': '', 'affiliation': {'laboratory': 'Immunology Innovation Team', 'institution': 'Ningbo University', 'location': {'postCode': '315211', 'settlement': 'Ningbo', 'country': 'China'}}, 'email': ''}, {'first': 'Xingguang', 'middle': [], 'last': 'Li', 'suffix': '', 'affiliation': {'laboratory': 'Shenzhen Key Laboratory of 13 Unknown Pathogen Identification, BGI-Shenzhen', 'institution': '', 'location': {'postCode': '518083', 'settlement': 'Shenzhen', 'country': 'China'}}, 'email': 'xingguanglee@hotmail.com'}, {'first': 'Qing', 'middle': [], 'last': 'Nie', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': '†', 'middle': [], 'last': '', 'suffix': '', 'affiliation': {}, 'email': ''}]}, 'abstract': [{'text': \"Analysis of genetic sequence data from the pandemic SARS Coronavirus 2 can provide 18 insights into epidemic origins, worldwide dispersal, and epidemiological history. With few 19 exceptions, genomic epidemiological analysis has focused on geographically distributed data sets 20 with few isolates in any given location. Here we report an analysis of 20 whole SARS-CoV 2 21 genomes from a single relatively small and geographically constrained outbreak in Weifang, 22 People's Republic of China. Using Bayesian model-based phylodynamic methods, we estimate the 23 reproduction number for the outbreak to be 2.6 (95% CI:1.5-5). We further estimate the number of 24 infections through time and compare these estimates to confirmed diagnoses by the Weifang 25 Centers for Disease Control. We find that these estimates are consistent with reported cases and 26 there is unlikely to be a large undiagnosed burden of infection over the period we studied. 27 28 194\", 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'We gratefully acknowledge China National GeneBank at Shenzhen, China for the sequencing strategy 195 and capacity support. We also gratefully acknowledge the laboratories that have contributed', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}], 'body_text': [{'text': 'We report a genomic epidemiological analysis of one of the first geographically concentrated 30 community transmission samples of SARS-CoV 2 genetic sequences collected outside of the initial 31 1 of 9 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': \"is the (which was not peer-reviewed) The copyright holder for this preprint . https: //doi.org/10.1101 //doi.org/10. /2020 Submitted manuscript. Not peer reviewed. outbreak in Wuhan, China. These data comprise 20 whole genome sequences from confirmed 32 COVID19 infections in Weifang, Shandong Province, People's Republic of China. The data were 33 collected over the course of several weeks up to February 10, 2020 and overlap with a period 34 of intensifying public health and social distancing measures. Phylodynamic analysis allows us to 35 evaluate epidemiological trends after seeding events which took place in mid to late January, 2020. 36 The objective of our analysis is to evaluate epidemiological trends based on national surveillance 37 and response efforts by Weifang Centers for Disease Control (CDC). This analysis provides an 38 estimate of the initial rate of spread and reproduction number in Weifang City. In contrast to the 39 early spread of COVID19 in Hubei Province of China, most community transmissions within Weifang 40 took place after public health interventions and social distancing measures were put in place. We 41 therefore hypothesize that genetic data should reflect a lower growth rate and reproduction number 42 than was observed in Wuhan. A secondary aim is to estimate the total numbers infected and to 43 evaluate the possibility that there is a large unmeasured burden of infection due to imperfect case 44 ascertainment and a large proportion of infections with mild or asymptomatic illness. 45 To analyze the Weifang sequences, we have adapted model-based phylodynamic methods 46 which were previously used to estimate growth rates and reproduction numbers using sequence 47 data from Wuhan and exported international cases (Volz et al., 2020) . This analysis has several 48 constraints and requirements: 49 50 Importation of lineages from Wuhan. The outbreak in Weifang was seeded by multiple lineages 51 imported at various times from the rest of China. We use a phylodynamic model that accounts for 52 location of sampling. Migration is modeled as a bi-directional process with rates proportional to 53 epidemic size in Weifang. The larger international reservoir of COVID19 cases serves as a source of 54 new infections and is assumed to be growing exponentially over this period of time. 55 Nonlinear epidemiological dynamics in Weifang. The maximum number of daily confirmed COVID19 56 cases occurred on February 5, but it is unknown when the maximum prevalence of infection oc-57 curred. We use a susceptible-exposed-infectious-recovered (SEIR) model (Keeling and Rohani, 2011) 58 for epidemic dynamics in Weifang. The model accounts for a realistic distribution of generation 59 times and can potentially capture a nonlinear decrease in cases following epidemic peak. 60 Variance in transmission rates (Lloyd-Smith et al., 2005) .To estimate total numbers infected, the 61 phylodynamic model must account for epidemiological variables which are known to significantly 62 influence genetic diversity. Foremost among these is the variance in offspring distribution (number 63 of transmissions per primary case). We draw on previous evidence based on the previous SARS 64 epidemic which indicates that the offspring distribution is highly over-dispersed. High variance of 65 transmission rates will reduce genetic diversity of a sample and failure to account for this factor 66 will lead to highly biased estimates of epidemic size (Li et al., 2017) . Recent analyses of sequence 67 data drawn primarily from Wuhan has found that high over-dispersion was required for estimated 68 cases to be consistent with the epidemiological record (Volz et al., 2020) . Models assuming low 69 variance in transmission rates between people would generate estimates of cases that are lower . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\", 'cite_spans': [{'start': 85, 'end': 102, 'text': '//doi.org/10.1101', 'ref_id': None}, {'start': 103, 'end': 122, 'text': '//doi.org/10. /2020', 'ref_id': None}, {'start': 346, 'end': 348, 'text': '33', 'ref_id': None}, {'start': 542, 'end': 544, 'text': '35', 'ref_id': None}, {'start': 645, 'end': 647, 'text': '36', 'ref_id': None}, {'start': 1343, 'end': 1345, 'text': '43', 'ref_id': None}, {'start': 1535, 'end': 1537, 'text': '45', 'ref_id': None}, {'start': 1768, 'end': 1787, 'text': '(Volz et al., 2020)', 'ref_id': None}, {'start': 2337, 'end': 2339, 'text': '55', 'ref_id': None}, {'start': 2602, 'end': 2631, 'text': '(Keeling and Rohani, 2011) 58', 'ref_id': None}, {'start': 2820, 'end': 2822, 'text': '60', 'ref_id': None}, {'start': 2854, 'end': 2880, 'text': '(Lloyd-Smith et al., 2005)', 'ref_id': 'BIBREF16'}, {'start': 3481, 'end': 3498, 'text': '(Li et al., 2017)', 'ref_id': 'BIBREF15'}, {'start': 3685, 'end': 3704, 'text': '(Volz et al., 2020)', 'ref_id': None}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.09.20033365 doi: medRxiv preprint Submitted manuscript. Not peer reviewed.', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'found that high over-dispersion is required to reconcile estimated reproduction numbers with 72 the observed frequency of international outbreaks. In this study, we elaborate the SEIR model to 73 include a compartment( ) with higher transmission rates. The variance of the implied offspring 74 distribution is calibrated to give similar overdispersion from the SARS epidemic.', 'cite_spans': [{'start': 193, 'end': 195, 'text': '73', 'ref_id': None}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'Despite an initial rapid increase in confirmed cases in Weifang in late January and early February, the 77 number of confirmed cases by Weifang CDC show that outbreak peaked quite early and maximum 78 number of cases ocurred on February 5. Phylodynamic analysis supports the interpretation that 79 control efforts reduced epidemic growth rates and contributed to eventual control. Figure 1A 80 illustrates the phylodynamic model which was co-estimated with the phylogeny which provides 81 estimates of epidemiological parameters summarized in Table 1 . Figure We do not have sufficient data to detect a large decrease in epidemic growth rates as the 93 epidemic progressed. We estimate 0 = 2.6 (95% HPD:1.5-5). The growth rate in estimated 94 infections remained positive but decreased substantially over the sampling period. These estimates 95 correspond to growth during a period when Weifang was implementing a variety of public health 96 interventions and contact tracing to limit epidemic spread. These interventions included public 97 3 of 9', 'cite_spans': [{'start': 740, 'end': 742, 'text': '94', 'ref_id': None}], 'ref_spans': [{'start': 381, 'end': 390, 'text': 'Figure 1A', 'ref_id': 'FIGREF2'}, {'start': 543, 'end': 550, 'text': 'Table 1', 'ref_id': 'TABREF0'}, {'start': 553, 'end': 559, 'text': 'Figure', 'ref_id': None}], 'section': '76'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '76'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.09.20033365 doi: medRxiv preprint Submitted manuscript. Not peer reviewed. . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '76'}, {'text': \"is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.09.20033365 doi: medRxiv preprint Submitted manuscript. Not peer reviewed. health messaging, establishing phone hotlines, encouraging home isolation for recent vitors from 98 Wuhan (January 23-26), optimizing triage of suspected cases in hospitals (January 24), travel 99 restrictions(January 26), extending school closures, and establishing 'fever clinics' for consulatation 100 and diagnosis(January 27) (Mao, 2020) . 101 As well as providing novel epidemiological estimates, our results point to the significance of 102 realistic modeling for fidelity of phylogenetic inference. The use of a model-based structured coa-103 lescent prior had large influence over estimated molecular clock rates and inferred TMRCAs. . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\", 'cite_spans': [{'start': 282, 'end': 284, 'text': '98', 'ref_id': None}, {'start': 486, 'end': 489, 'text': '100', 'ref_id': None}, {'start': 516, 'end': 527, 'text': '(Mao, 2020)', 'ref_id': 'BIBREF17'}, {'start': 530, 'end': 533, 'text': '101', 'ref_id': None}], 'ref_spans': [], 'section': '76'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https: //doi.org/10.1101 //doi.org/10. /2020 Submitted manuscript. Not peer reviewed. size estimates. The model of epidemic dynamics within Weifang is based on a susceptible-exposedinfectious-recovered (SEIR) model. We elaborate the model with with an additional compartment which has a higher transmission rate ( -fold higher) than the compartment. Upon leaving the incubation period individuals progress to the compartment with probability ℎ , or otherwise to . 6 of 9 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 85, 'end': 102, 'text': '//doi.org/10.1101', 'ref_id': None}, {'start': 103, 'end': 122, 'text': '//doi.org/10. /2020', 'ref_id': None}], 'ref_spans': [], 'section': '76'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https: //doi.org/10.1101 //doi.org/10. /2020 Submitted manuscript. Not peer reviewed.', 'cite_spans': [{'start': 85, 'end': 102, 'text': '//doi.org/10.1101', 'ref_id': None}, {'start': 103, 'end': 122, 'text': '//doi.org/10. /2020', 'ref_id': None}], 'ref_spans': [], 'section': 'Methods and Materials'}, {'text': 'The model is implemented as a system of ordinary differential equations:', 'cite_spans': [], 'ref_spans': [], 'section': 'Methods and Materials'}, {'text': \"We also model an exponentially growing reservoir ( ) for imported lineages in to Weifang. The equation governing this population iṡ substitution model and a time-scaled tree was estimated using treedater 0.5.0 (Volz and Frost, 2017) . 176 Two outliers according to the molecular clock model were identified and removed using 'treedater' 177 which was also used to compute the root to tip regression. 178 Bayesian phylogenetic analysis was carried out using BEAST 2.6.1 (Bouckaert et al., 2019) . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\", 'cite_spans': [{'start': 211, 'end': 233, 'text': '(Volz and Frost, 2017)', 'ref_id': 'BIBREF21'}, {'start': 236, 'end': 239, 'text': '176', 'ref_id': None}, {'start': 401, 'end': 404, 'text': '178', 'ref_id': None}], 'ref_spans': [{'start': 470, 'end': 494, 'text': '(Bouckaert et al., 2019)', 'ref_id': 'FIGREF2'}], 'section': 'Methods and Materials'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint in study design, data analysis, data interpretation, or writing of the report.', 'cite_spans': [], 'ref_spans': [], 'section': 'Methods and Materials'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '193'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.09.20033365 doi: medRxiv preprint Submitted manuscript. Not peer reviewed. . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '193'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.09.20033365 doi: medRxiv preprint . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '193'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10. 1101 ', 'cite_spans': [{'start': 98, 'end': 102, 'text': '1101', 'ref_id': None}], 'ref_spans': [], 'section': '193'}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'Cryptic transmission of novel coronavirus revealed by genomic epidemiology', 'authors': [{'first': 'T', 'middle': [], 'last': 'Bedford', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '2020--2023', 'other_ids': {}}, 'BIBREF2': {'ref_id': 'b2', 'title': 'BEAST 2.5: An advanced software platform for Bayesian evolutionary analysis. PLoS 219 computational biology', 'authors': [{'first': 'D', 'middle': [], 'last': '', 'suffix': ''}, {'first': 'De', 'middle': [], 'last': 'Maio', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': '', 'suffix': ''}], 'year': 2019, 'venue': '', 'volume': '15', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF3': {'ref_id': 'b3', 'title': 'MERS-CoV spillover at the camel-human interface', 'authors': [{'first': 'G', 'middle': [], 'last': 'Dudas', 'suffix': ''}, {'first': 'L', 'middle': ['M'], 'last': 'Carvalho', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Rambaut', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Bedford', 'suffix': ''}], 'year': 2018, 'venue': 'Elife', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF4': {'ref_id': 'b4', 'title': \"disease and diplomacy: GISAID's innovative contribution to global health\", 'authors': [{'first': 'S', 'middle': [], 'last': 'Elbe', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Buckland-Merrett', 'suffix': ''}, {'first': '', 'middle': [], 'last': 'Data', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF8': {'ref_id': 'b8', 'title': 'Genomic surveillance elucidates Ebola virus origin and transmission during the 2014 outbreak', 'authors': [], 'year': 2014, 'venue': 'Science', 'volume': '345', 'issn': '6202', 'pages': '1369--1372', 'other_ids': {}}, 'BIBREF9': {'ref_id': 'b9', 'title': 'doi: medRxiv preprint Submitted manuscript. Not peer reviewed', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.03.09.20033365']}}, 'BIBREF10': {'ref_id': 'b10', 'title': 'Dispersion vs', 'authors': [{'first': 'K', 'middle': [], 'last': 'Grantz', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Metcalf', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Lessler', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF13': {'ref_id': 'b13', 'title': 'MAFFT multiple sequence alignment software version 7: improvements in performance 233 and usability. Molecular biology and evolution', 'authors': [{'first': 'K', 'middle': [], 'last': 'Katoh', 'suffix': ''}, {'first': 'D', 'middle': ['M'], 'last': 'Standley', 'suffix': ''}], 'year': 2013, 'venue': '', 'volume': '30', 'issn': '', 'pages': '772--780', 'other_ids': {}}, 'BIBREF14': {'ref_id': 'b14', 'title': 'Modeling Infectious Diseases in Humans and Animals', 'authors': [{'first': 'M', 'middle': ['J'], 'last': 'Keeling', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Rohani', 'suffix': ''}], 'year': 2011, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF15': {'ref_id': 'b15', 'title': 'Quantifying Transmission Heterogeneity Using Both Pathogen Phylogenies and 236 Incidence Time Series', 'authors': [{'first': 'L', 'middle': ['M'], 'last': 'Li', 'suffix': ''}, {'first': 'N', 'middle': ['C'], 'last': 'Grassly', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Fraser', 'suffix': ''}], 'year': 2017, 'venue': 'Mol Biol Evol', 'volume': '34', 'issn': '11', 'pages': '2982--2995', 'other_ids': {}}, 'BIBREF16': {'ref_id': 'b16', 'title': 'Superspreading and the effect of individual variation on 238 disease emergence', 'authors': [{'first': 'J', 'middle': ['O'], 'last': 'Lloyd-Smith', 'suffix': ''}, {'first': 'S', 'middle': ['J'], 'last': 'Schreiber', 'suffix': ''}, {'first': 'P', 'middle': ['E'], 'last': 'Kopp', 'suffix': ''}, {'first': 'W', 'middle': ['M'], 'last': 'Getz', 'suffix': ''}], 'year': 2005, 'venue': 'Nature', 'volume': '438', 'issn': '7066', 'pages': '355--359', 'other_ids': {}}, 'BIBREF17': {'ref_id': 'b17', 'title': 'Weifang City announces fever clinics. Weifang News Network', 'authors': [{'first': 'H', 'middle': [], 'last': 'Mao', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF18': {'ref_id': 'b18', 'title': 'IQ-TREE 2: New 241 models and efficient methods for phylogenetic inference in the genomic era', 'authors': [{'first': 'B', 'middle': ['Q'], 'last': 'Minh', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Schmidt', 'suffix': ''}, {'first': 'O', 'middle': [], 'last': 'Chernomor', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Schrempf', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Woodhams', 'suffix': ''}, {'first': 'Von', 'middle': [], 'last': 'Haeseler', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Lanfear', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': '', 'suffix': ''}], 'year': 2019, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF19': {'ref_id': 'b19', 'title': 'Phylodynamic Analyses based on 11 genomes from the Italian outbreak', 'authors': [{'first': 'T', 'middle': [], 'last': 'Stadler', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '2020--2023', 'other_ids': {}}, 'BIBREF20': {'ref_id': 'b20', 'title': 'Phylodynamic analysis of SARS-CoV-2 | Update 2020-03-06', 'authors': [{'first': 'Verity', 'middle': [], 'last': 'Hill', 'suffix': ''}, {'first': 'A', 'middle': ['R'], 'last': '', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '2020--2023', 'other_ids': {}}, 'BIBREF21': {'ref_id': 'b21', 'title': 'Scalable relaxed clock phylogenetic dating', 'authors': [{'first': 'E', 'middle': ['M'], 'last': 'Volz', 'suffix': ''}, {'first': 'Sdw', 'middle': [], 'last': 'Frost', 'suffix': ''}], 'year': 2017, 'venue': 'Virus Evol', 'volume': '3', 'issn': '2', 'pages': '', 'other_ids': {}}, 'BIBREF23': {'ref_id': 'b23', 'title': 'Phylogenetic analysis of SARS-CoV', 'authors': [{'first': 'I', 'middle': [], 'last': '', 'suffix': ''}, {'first': 'Fitzjohn', 'middle': ['R'], 'last': 'Fu', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Gaythorpe', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Ghani', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Hamlet', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Hinsley', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Imai', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Laydon', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Nedjati-Gilani', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Gemma', 'suffix': ''}, {'first': 'Riley', 'middle': ['S'], 'last': 'Okell', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Van Elsland', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '5', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF24': {'ref_id': 'b24', 'title': 'Bayesian phylodynamic inference with complex models', 'authors': [{'first': 'E', 'middle': ['M'], 'last': 'Volz', 'suffix': ''}, {'first': 'I', 'middle': [], 'last': 'Siveroni', 'suffix': ''}], 'year': 2018, 'venue': 'PLoS Comput Biol', 'volume': '14', 'issn': '11', 'pages': '', 'other_ids': {}}, 'BIBREF25': {'ref_id': 'b25', 'title': 'ggtree : an R package for visualization and annotation of phylogenetic 253 trees with their covariates and other associated data', 'authors': [{'first': 'G', 'middle': [], 'last': 'Yu', 'suffix': ''}, {'first': 'D', 'middle': ['K'], 'last': 'Smith', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Zhu', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Guan', 'suffix': ''}, {'first': 'Tty', 'middle': [], 'last': 'Lam', 'suffix': ''}], 'year': 2017, 'venue': 'Methods Ecol Evol', 'volume': '8', 'issn': '1', 'pages': '28--36', 'other_ids': {}}, 'BIBREF27': {'ref_id': 'b27', 'title': 'A pneumonia outbreak associated with a new 256 coronavirus of probable bat origin', 'authors': [{'first': 'R', 'middle': ['D'], 'last': 'Jiang', 'suffix': ''}, {'first': 'M', 'middle': ['Q'], 'last': 'Liu', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Chen', 'suffix': ''}, {'first': 'X', 'middle': ['R'], 'last': 'Shen', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'X', 'middle': ['S'], 'last': 'Zheng', 'suffix': ''}], 'year': 2020, 'venue': 'Nature', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}}, 'ref_entries': {'FIGREF0': {'text': '70than the known number of confirmed cases. Separately, Grantz et al.(Grantz et al., ????)', 'latex': None, 'type': 'figure'}, 'FIGREF1': {'text': '1B shows the estimated 82 time scaled phylogeny (maximum clade credibility) including 20 lineages sampled from distinct 83 patients in Weifang and 33 genomes sampled from Wuhan and internationally. 84 The estimated number of infections is shown Figure 1C. The time series of confirmed cases 85 should lag the estimated number of infected because of delays from infection to appearance of 86 symptoms and delays from symptoms to diagnosis. We also expect that an unknown proportion of 87 infections will be missed by the surveillance system due very mild, subclinical, or asymptomatic 88 infection. Our estimates do not support the hypothesis that there was a very large hidden burden 89 of infection in Weifang over the period that the sequence data were sampled. Indeed, our central 90 estimate for the number infected on 10th February is only 142 and the credible intervals cover the 91 44 cumulative confirmed cases at the end of February.92', 'latex': None, 'type': 'figure'}, 'FIGREF2': {'text': 'Phylodynamic estimates and epidemiological model. A. Diagram representing the structure of the epidemiological SEIR model which was fitted in tandem with the time scaled phylogeny. Colours correspond to the state of individuals sampled and represented in the tree (B). Note that infected and infectious individuals may occupy a low transmission state (I) or a high transmission rate state (J) to account for high dispersion of the reproduction number. B. A time scaled phylogeny co-estimated with epidemiological parameters. The colour of tips corresponds to location sampling. Red tips were sampled from Weifang, China. The credible interval of TMRCA is shown as a blue bar for all nodes with more than 50% posterior support. C. Cumulative estimated infections through time produced by fitting the SEIR model and the cumulative confirmed cases', 'latex': None, 'type': 'figure'}, 'FIGREF3': {'text': 'points) reported by Weifan CDC. The shaded region shows the 95% HPD and the line shows the posterior median. D. A root to tip regression showing approximately linear increase in diversity with time of sampling.', 'latex': None, 'type': 'figure'}, 'FIGREF4': {'text': 'Figure supplement 1. Maximum likelihood time tree.', 'latex': None, 'type': 'figure'}, 'FIGREF5': {'text': 'Figure supplement 2. Tree posterior density plot.', 'latex': None, 'type': 'figure'}, 'FIGREF6': {'text': 'Figure supplement 3. Tree posterior density plot. 4 of 9', 'latex': None, 'type': 'figure'}, 'FIGREF7': {'text': 'shows that maximum likelihood inference of time-scaled phylogenies produces a 105 distribution of TMRCAs which are substantially different than the Bayesian model-based analysis.106    Choice of population genetic prior will have a large influence on phylogenetic inference based on 107 sparse or poorly informative genetic sequence data. Among the 20 Weifang sequences included in 108 this analysis, there is mean pairwise difference of only three single nucleotide polymorphisms and 109 only approximately twice as much diversity observed among the remainder of the sequences we 110 studied. There is correspondingly low confidence in tree topology(Figure Supplement 2), and only 111 three clades had greater than 50% posterior support including one clade which had a monophyletic 112 compoisition of 13 Weifang lineages. The earliest Weifang sequence was sampled on January 25 113 from a patient who showed first symptoms on January 16. These dates cover a similar range as the 114 posterior TMRCA of all Weifang sequences (Figure Supplement 3). 115 Discussion 116 Our analysis of 20 SARS-CoV 2 genomes from Weifang, China has confirmed independent observa-117 tions regarding the rate of spread and burden of infection in the city. Surveillance of COVID19 is 118 rendered difficult by high proportions of illness with mild severity and an unknown proportion of 119 asymptomatic infection(Guan et al., 2020). The extent of under-reporting and case ascertainment 120 rates has been widely debated. Analysis of genetic sequence data provides an alternative source of 121 information about epidemic size which can be more robust to imperfect case ascertainment. We 122 do not find evidence for a very large hidden burden of infection within Weifang. The relatively low 123 estimate of 0 is consistent with a slower rate of spread outside of Wuhan and effective control 124 strategies implemented in late January. 125 While the value of pathogen genomic analyses is widely recognized for estimating dates of 126 emergence(Verity Hill, 2020; Gire et al., 2014) and identifying animal reservoirs(Zhou et al., 2020; 127 Dudas et al., 2018), analysis of pathogen sequences also has potential to inform epidemic surveil-128 lance and intervention efforts. With few exceptions(Stadler, 2020;Bedford, 2020), this potential 129 is currently not being realized for the international response to COVID19. It is worth noting that 130 the analysis described in this report was accomplished in approximately 48 hours and drew on 131 previously developed models and packages for BEAST2(Bouckaert et al., 2019; Volz and Siveroni,   132    2018). It is therefore feasible for phylodynamic analysis to provide a rapid supplement to epidemio-133 logical surveillance, however this requires rapid sequencing and timely sharing of data as well as 134 randomized concentrated sampling of the epidemic within localities such as individual cities.', 'latex': None, 'type': 'figure'}, 'FIGREF8': {'text': 'Mini Kit (QIAGEN 74104) by column method. RT-qPCR was carried out using 2019 141 novel coronavirus nucleic acid detection kit (BioGerm, Shanghai, China) to confirm the presence 142 of SARS-CoV-2 viral RNA with cycle threshold (Ct) values range from 17 to 37, targeting the high 143 conservative region (ORF1ab/N gene) in SARS-CoV-2 genome. Metagenomic sequencing: The 144 concentration of RNA samples was measurement by Qubit RNA HS Assay Kit (Thermo Fisher 145 Scientific, Waltham, MA, USA). DNase was used to remove host DNA. The remaining RNA was used 146 to construct the single-stranded circular DNA library with MGIEasy RNA Library preparation reagent 147 set (MGI, Shenzhen, China). Purified RNA was then fragmented. Using these short fragments as 148 templates, random hexamers were used to synthesize the first-strand cDNA, followed by the second 149 strand synthesis. Using the short double-strand DNA, a DNA library was constructed through end 150 repair, adaptor ligation, and PCR amplification. PCR products were transformed into a single strand 151 circular DNA library through DNA-denaturation and circularization. DNA nanoballs (DNBs) were 152 generated with the single-stranded circular DNA library by rolling circle replication (RCR). The DNBs 153 were loaded into the flow cell and pair-end 100bp sequencing on the DNBSEQ-T7 platform 8 (MGI, 154 Shenzhen, China). 20 genomes were assembled with length from 26,840 to 29,882 nucleotides. 155 The median age of patients was 36 (range:6-75). Two of twenty patients suffered severe or critical 156 illness. Weifang sequences were combined with a diverse selection of sequences from China 157 outside of Weifang and other countries provided by GISAIDElbe and Buckland-Merrett (2017)model. The phylodynamic model is designed to account for nonlinear epidemic dynamics in Weifang, a realistic course of infection (incubation and infectious periods), migration of lineages in and out of Weifang, and variance in transmission rates which can influence epidemic', 'latex': None, 'type': 'figure'}, 'FIGREF9': {'text': 'Migration is modeled as a bidirectional process which only depends on the size of variables in 163 the Weifang compartment and thus migration does not influence epidemic dynamics; it will only 164 influence the inferred probability that a lineage resides within Weifang. For a compartment(E,I,   165    or J), is the per lineage rate of migration out of Weifang and the total rate of migration in and model fitting and are estimated. Additionally, we estimate initial sizes168 of , , and . Other parameters are fixed based on prior information. We fix 1∕ 0 = 4.1days and 169 1∕ 1 = 3.8 days. We set ℎ = 0.20 and = 74 which yields a dispersion of the reproduction number 170 that matches a negative binomial distribution with = 0.22 if 0 = 2, similar to values estimated for 171 the 2003 SARS epidemic (Lloyd-Smith et al., 2005). 172 Phylogenetic analysis. We aligned the 20 Weifang sequences using MAFFT(Katoh and Standley, 173 2013) with a previous aligment of 35 SARS-CoV 2 sequences from outside of Weifang(Volz et al., 174 2020). Maximum likelihood analysis was carried using IQTree(Minh et al., 2019) with a HKY+G4 175', 'latex': None, 'type': 'figure'}, 'FIGREF10': {'text': 'Figure supplement 1. A time scaled phylogeny estimated using IQTree and treedater and using the same data as used for the Bayesian analysis.258', 'latex': None, 'type': 'figure'}, 'FIGREF11': {'text': 'Figure supplement 2. A tree density plot based on the posterior distribution of Figure supplement 3. The estimated posterior TMRCA among all Weifang lineages.260', 'latex': None, 'type': 'figure'}, 'TABREF0': {'text': 'Summary of primary epidemiological and evolutionary parameters, including Bayesian prior distributions and estimated posteriors. Posterior uncertainty is summerized using a 95% highest posterior density interval.', 'latex': None, 'type': 'table'}, 'TABREF1': {'text': 'using a HKY+G4 substitution model and a strict molecular clock. The phylodynamic model was implemented 180 using the PhyDyn package(Volz and Siveroni, 2018) using the QL likelihood approximation and the 181 RK ODE solver. The model was fitted by running 8 MCMC chains in parallel and combining chains 182 after removing 50% burn-in.The ggtree package was used for all phylogeny visualizations(Yu et al., 2017).Code to replicate this analysis and and BEAST XML files can be found at https://github.com/ This work was supported by Centre funding from the UK Medical Research Council under a con-188 cordat with the UK Department for International Development. NIHR. J-IDEA. This work was also', 'latex': None, 'type': 'table'}, 'TABREF2': {'text': '. https://doi.org/10.1101/2020.03.09.20033365 doi: medRxiv preprint Submitted manuscript. Not peer reviewed. supported by a grant from the Special Project for Prevention and Control of Pneumonia of New 190 Coronavirus Infection in Weifang Science and Technology Development Plan in 2020 (2020YQFK015) 191 to Associate Senior Technologist Qing Nie. Role of the Funders: All funders of the study had no role', 'latex': None, 'type': 'table'}}, 'back_matter': []}\n",
      "{'paper_id': 'b323d6bf1873bf0b94bcae0be2052d9e335bf1af', 'metadata': {'title': 'Knowledge synthesis from 100 million biomedical documents augments the deep expression profiling of coronavirus receptors', 'authors': [{'first': 'A', 'middle': ['J'], 'last': 'Venkatakrishnan', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Arjun', 'middle': [], 'last': 'Puranik', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Akash', 'middle': [], 'last': 'Anand', 'suffix': '', 'affiliation': {'laboratory': 'nference Labs, pvt. ltd', 'institution': '', 'location': {'addrLine': 'Wind Tunnel Road, Murugesh Palaya', 'postCode': '560017', 'settlement': 'Bengaluru', 'country': 'India'}}, 'email': ''}, {'first': 'David', 'middle': [], 'last': 'Zemmour', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Xiang', 'middle': [], 'last': 'Yao', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Xiaoying', 'middle': [], 'last': 'Wu', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Ramakrishna', 'middle': [], 'last': 'Chilaka', 'suffix': '', 'affiliation': {'laboratory': 'nference Labs, pvt. ltd', 'institution': '', 'location': {'addrLine': 'Wind Tunnel Road, Murugesh Palaya', 'postCode': '560017', 'settlement': 'Bengaluru', 'country': 'India'}}, 'email': ''}, {'first': 'Dariusz', 'middle': ['K'], 'last': 'Murakowski', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Kristopher', 'middle': [], 'last': 'Standish', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Bharathwaj', 'middle': [], 'last': 'Raghunathan', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Tyler', 'middle': [], 'last': 'Wagner', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Enrique', 'middle': [], 'last': 'Garcia-Rivera', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Hugo', 'middle': [], 'last': 'Solomon', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Abhinav', 'middle': [], 'last': 'Garg', 'suffix': '', 'affiliation': {'laboratory': 'nference Labs, pvt. ltd', 'institution': '', 'location': {'addrLine': 'Wind Tunnel Road, Murugesh Palaya', 'postCode': '560017', 'settlement': 'Bengaluru', 'country': 'India'}}, 'email': ''}, {'first': 'Rakesh', 'middle': [], 'last': 'Barve', 'suffix': '', 'affiliation': {'laboratory': 'nference Labs, pvt. ltd', 'institution': '', 'location': {'addrLine': 'Wind Tunnel Road, Murugesh Palaya', 'postCode': '560017', 'settlement': 'Bengaluru', 'country': 'India'}}, 'email': ''}, {'first': 'Anuli', 'middle': [], 'last': 'Anyanwu-Ofili', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Najat', 'middle': [], 'last': 'Khan', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Venky', 'middle': [], 'last': 'Soundararajan', 'suffix': '', 'affiliation': {'laboratory': 'nference Labs, pvt. ltd', 'institution': '', 'location': {'addrLine': 'Wind Tunnel Road, Murugesh Palaya', 'postCode': '560017', 'settlement': 'Bengaluru', 'country': 'India'}}, 'email': ''}]}, 'abstract': [{'text': 'The COVID-19 pandemic demands assimilation of all available biomedical knowledge to decode its mechanisms of pathogenicity and transmission. Despite the recent renaissance in unsupervised neural networks for decoding unstructured natural languages, a platform for the real-time synthesis of the exponentially growing biomedical literature and its comprehensive triangulation with deep omic insights is not available. Here, we present the nferX platform for dynamic inference from over 45 quadrillion possible conceptual associations extracted from unstructured biomedical text, and their triangulation with Single Cell RNA-sequencing based insights from over 25 tissues. Using this platform, we identify intersections between the pathologic manifestations of COVID-19 and the comprehensive expression profile of the SARS-CoV-2 receptor ACE2. We find that tongue keratinocytes, airway club cells, and ciliated cells are likely underappreciated targets of SARS-CoV-2 infection, in addition to type II pneumocytes and olfactory epithelial cells. We further identify mature small intestinal enterocytes as a possible hotspot of COVID-19 fecal-oral transmission, where an intriguing maturation-correlated transcriptional signature is shared between ACE2 and the other coronavirus receptors DPP4 (MERS-CoV) and ANPEP (ɑ-coronavirus). This study demonstrates how a holistic data science platform can leverage unprecedented quantities of structured and unstructured publicly available data to accelerate the generation of impactful biological insights and hypotheses.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'The nferX Platform Single-cell resource -https://academia.nferx.com/', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}], 'body_text': [{'text': 'Since December 2019, the SARS-CoV-2 virus has been rapidly spreading across the globe. The associated disease has been declared a pandemic by the World Health Organization (WHO), with over 350,000 confirmed cases and 15,000 deaths across nearly every country as of March 23, 2020 1 . The constellation of symptoms, ranging from acute respiratory distress syndrome (ARDS) to gastrointestinal issues, is similar to that observed in the 2002 Severe Acute Respiratory Syndrome (SARS) epidemic and the 2012 Middle East respiratory syndrome (MERS) outbreak. SARS, MERS, and COVID-19 are all caused by Coronaviruses (CoV), deriving their name from the crown-like spike proteins protruding from the viral capsid surface. Coronavirus infection is driven by the attachment of the viral spike protein to specific human cell-surface receptors: ACE2 for SARS-CoV-2 and SARS-CoV 2-4 , DPP4 for MERS-CoV 5 and ANPEP for specific ɑ-coronaviruses 6 . In addition to these receptors, the protease activity of TMPRSS2 has also been implicated in viral entry 7, 8 .', 'cite_spans': [{'start': 930, 'end': 931, 'text': '6', 'ref_id': 'BIBREF5'}, {'start': 1039, 'end': 1041, 'text': '7,', 'ref_id': 'BIBREF6'}, {'start': 1042, 'end': 1043, 'text': '8', 'ref_id': 'BIBREF7'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'In a recent clinical study of COVID-19 patients from China, 48% of the 191 infected patients studied had comorbidities such as hypertension and diabetes 9 . Epidemiological and clinical investigations on COVID-19 patients are also suggesting fecal viral shedding and gastrointestinal infection [10] [11] [12] . In the case of the earlier SARS epidemic, multiple organ damage involving lung, kidney and heart was reported 13 . The mechanisms by which various comorbidities impact the clinical course of infections and the reasons for the observed multi-organ phenotypes are still not well understood. Thus, there is an urgent need to conduct a comprehensive pan-tissue profiling of ACE2, the putative human receptor for SARS-CoV-2.', 'cite_spans': [{'start': 153, 'end': 154, 'text': '9', 'ref_id': 'BIBREF8'}, {'start': 294, 'end': 298, 'text': '[10]', 'ref_id': 'BIBREF9'}, {'start': 299, 'end': 303, 'text': '[11]', 'ref_id': 'BIBREF10'}, {'start': 304, 'end': 308, 'text': '[12]', 'ref_id': 'BIBREF11'}, {'start': 421, 'end': 423, 'text': '13', 'ref_id': 'BIBREF12'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': \"A deep profiling of ACE2 expression in the human body demands a platform that synthesizes biomedical insights encompassing multiple scales, modalities, and pathologies described across the scientific literature and various omics siloes. With the exponential growth of scientific (e.g. PubMed, preprints, grants), translational (e.g. clinicaltrials.gov), and other (e.g. patents) biomedical knowledge bases, a fundamental requirement is to recognize nuanced scientific phraseology and measure the strength of association between all possible pairs of such phrases. Such a holistic map of associations will provide insights into the knowledge harbored in the world's biomedical literature.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'While unsupervised machine learning has been advanced to study the semantic relationships between word embeddings 14, 15 and applied to the material science corpus 16 , this has not been scaled-up to extract the \"global context\" of conceptual associations from the entirety of publicly available unstructured biomedical text. Additionally, a principled way of accounting for the distances between phrases captured from the ever-growing scientific literature has not been comprehensively researched to quantify the strength of \"local context\" between pairs of biological concepts. Given the propensity for irreproducible or erroneous scientific research 17 , which reflects as truths, semi-truths, and falsities in the literature, any local or global signals extracted from this unstructured knowledge need to be seamlessly triangulated with deep biological insights emergent from various omics data silos. author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [{'start': 114, 'end': 117, 'text': '14,', 'ref_id': 'BIBREF13'}, {'start': 118, 'end': 120, 'text': '15', 'ref_id': 'BIBREF14'}, {'start': 164, 'end': 166, 'text': '16', 'ref_id': 'BIBREF15'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.24.005702 doi: bioRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'The nferX software is a cloud-based platform that enables users to dynamically query the universe of possible conceptual associations from over 100 million biomedical documents, including the COVID-19 Open Research Dataset recently announced by the White House 18 (Figure 1 ). An unsupervised neural network is used to recognize and preserve complex biomedical phraseology as 300 million searchable tokens, beyond the simpler words that have generally been explored using higher dimensional word embeddings previously 14 . Our local context score is derived from pointwise mutual information content between pairs of these tokens and can be retrieved dynamically for over 45 quadrillion possible associations. Our global context score is derived using word2vec 14 , as the cosine distance between 180 million word vectors projected in a 300 dimensional space ( Figure 1A, Figure S1 ).', 'cite_spans': [], 'ref_spans': [{'start': 264, 'end': 273, 'text': '(Figure 1', 'ref_id': 'FIGREF1'}, {'start': 861, 'end': 881, 'text': 'Figure 1A, Figure S1', 'ref_id': 'FIGREF1'}], 'section': 'Introduction'}, {'text': 'In order to assess the veracity of these conceptual associations derived from biomedical literature, it is absolutely essential to enable triangulation with structured data sources including gene and protein expression datasets. To address this need and empower the scientific community, we built a Single Cell resource which harnesses these local and global score metrics to enable seamless integration of literature-derived associations with the analysis of transcriptomes from over 1 million individual cells from over 25 human and mouse tissues ( Figure 1B) . Here we use this first-in-class resource to conduct a comprehensive expression profiling of ACE2 across host tissues and cell types and discuss how the observed expression patterns correlate with the pathogenicity and viral transmission shaping the ongoing COVID-19 pandemic ( Figure 1C ).', 'cite_spans': [], 'ref_spans': [{'start': 551, 'end': 561, 'text': 'Figure 1B)', 'ref_id': 'FIGREF1'}, {'start': 841, 'end': 850, 'text': 'Figure 1C', 'ref_id': 'FIGREF1'}], 'section': 'Introduction'}, {'text': 'To systematically profile the transcriptional expression of ACE2 across tissues and cell types, we triangulated single cell RNAseq-based measurements with literature-derived signals to automatically delineate novel, emerging, and known expression patterns ( Figure 2B ; Table S1 ). This approach immediately highlights renal proximal tubular cells and small intestinal enterocytes among the cell types that most robustly express ACE2 (detection in >40% of cells). These cell types are also moderately to strongly associated with ACE2 in the literature. The strong intestinal ACE2 expression is particularly interesting given the emerging clinical reports of fecal shedding and persistence post-recovery which may reflect a fecal-oral transmission pattern 10-12 . Conversely, pancreatic PP cells (gamma cells), pancreatic alpha cells, and keratinocytes show similarly robust ACE2 expression but have not been strongly associated with ACE2 in the literature. This combination suggests either a biological novelty or an experimental artifact. We note that the strong ACE2 expression in pancreatic cell types is derived from only one murine study (Figure S2A 19 ), while ACE2 expression is not observed in gamma or alpha cells from scRNA-seq of human pancreatic islets ( Figure S2B [20] [21] [22] ). While we cannot determine the validity of either observation, this example demonstrates how knowledge synthesis can automatically surface discordant biological signals for further evaluation. author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [{'start': 755, 'end': 762, 'text': '10-12 .', 'ref_id': None}, {'start': 1278, 'end': 1282, 'text': '[20]', 'ref_id': 'BIBREF20'}, {'start': 1283, 'end': 1287, 'text': '[21]', 'ref_id': 'BIBREF21'}, {'start': 1288, 'end': 1292, 'text': '[22]', 'ref_id': 'BIBREF22'}], 'ref_spans': [{'start': 258, 'end': 267, 'text': 'Figure 2B', 'ref_id': 'FIGREF2'}, {'start': 270, 'end': 278, 'text': 'Table S1', 'ref_id': None}, {'start': 1143, 'end': 1157, 'text': '(Figure S2A 19', 'ref_id': 'FIGREF1'}, {'start': 1267, 'end': 1277, 'text': 'Figure S2B', 'ref_id': 'FIGREF2'}], 'section': 'ACE2 has higher expression levels in multiple cell types of the Gastrointestinal (GI) tract compared to respiratory cells'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.24.005702 doi: bioRxiv preprint Surprisingly, cells from respiratory tissues were notably absent among the populations with highest ACE2 expression by scRNA-seq ( Figure 2B ). This observation was corroborated by complementary gene expression analysis of over 250,000 bulk RNA-seq samples from GTEx 23, 24 and the Gene Expression Omnibus (GEO) along with protein expression analysis from healthy tissue proteomics and immunohistochemistry (IHC) datasets [25] [26] [27] , where lung and other respiratory tissues consistently show lower ACE2 expression compared to the digestive tract and kidney (Figure S3A-D) . However, the respiratory transmission of COVID-19 along with the disease symptomatology and well-documented viral shedding in respiratory secretions 28 strongly indicates that SARS-CoV-2 indeed infects and replicates within these tissues. This would suggest that ACE2 is likely expressed in the respiratory epithelium, and so we prioritized the respiratory and digestive tracts for further knowledge synthesis-augmented scRNA-seq analysis.', 'cite_spans': [{'start': 409, 'end': 412, 'text': '23,', 'ref_id': None}, {'start': 413, 'end': 415, 'text': '24', 'ref_id': None}, {'start': 564, 'end': 568, 'text': '[25]', 'ref_id': 'BIBREF25'}, {'start': 569, 'end': 573, 'text': '[26]', 'ref_id': 'BIBREF26'}, {'start': 574, 'end': 578, 'text': '[27]', 'ref_id': 'BIBREF27'}], 'ref_spans': [{'start': 273, 'end': 282, 'text': 'Figure 2B', 'ref_id': 'FIGREF2'}, {'start': 705, 'end': 719, 'text': '(Figure S3A-D)', 'ref_id': 'FIGREF3'}], 'section': 'ACE2 has higher expression levels in multiple cell types of the Gastrointestinal (GI) tract compared to respiratory cells'}, {'text': 'We also applied the Single Cell resource to analyze several other human and mouse tissues including heart, adipose, liver, pancreas, blood, spleen, bone marrow, thymus, testis, prostate, bladder, ovary, uterus, placenta, brain, and retina. An example use case describing the functionalities of the Single Cell app and a summary of ACE2 expression across these tissues are given in the Supplemental Text and Figures S4-19 .', 'cite_spans': [], 'ref_spans': [{'start': 407, 'end': 420, 'text': 'Figures S4-19', 'ref_id': 'FIGREF1'}], 'section': 'ACE2 has higher expression levels in multiple cell types of the Gastrointestinal (GI) tract compared to respiratory cells'}, {'text': 'Next, we classified 105 respiratory cell populations from eight independent studies based on their expression of and literature-derived associations to ACE2 ( Figure 3A) . Consistent with the low levels of ACE2 in respiratory tissues by bulk RNA-seq, proteomics, and IHC ( Figure S3A-D) , we found that ACE2 expression is detected in fewer than 10% of all cell types recovered from these studies. However, as mentioned above, we believe that even low ACE2 expression levels in these respiratory cells may be sufficient for COVID-19 pathogenesis.', 'cite_spans': [], 'ref_spans': [{'start': 159, 'end': 169, 'text': 'Figure 3A)', 'ref_id': 'FIGREF3'}, {'start': 273, 'end': 286, 'text': 'Figure S3A-D)', 'ref_id': 'FIGREF3'}], 'section': 'Club cells, ciliated cells, and pneumocytes are likely targets of SARS-CoV-2 in respiratory tract'}, {'text': 'We found that club cells (formerly known as Clara cells), were consistently among the highestexpressing respiratory cell types (Figures 3A-B) . Literature-derived local and global scores suggest that this ACE2-club cell connection is underappreciated, with a few documents discussing these concepts together (nferx link). We also found that ACE2 is detected in type II pneumocytes in multiple studies, although the percentage of expressing cells ranges from only 0.5-7% (Figures 3A-B) . This relatively low expression, which may be deemed inconsequential if viewed in isolation, is strongly supported by knowledge synthesis that highlights an existing association between ACE2 and type II pneumocytes (Figures 3A-B) . Indeed, multiple studies have demonstrated ACE2 expression in these cells [29] [30] [31] [32] [33] [34] [35] . Further, ACE2 expression in bulk RNAseq of GTEx lung samples (n = 578) is strongly correlated to markers of type II pneumocytes, with all seven surfactant protein-encoding genes among the top 4% of transcriptional correlations to ACE2 (out of the ~19,000 genes expressed at > 1 TPM in GTEx lung samples; hypergeometric p-value = 1.1x10 -10 ) ( Figure S20) .', 'cite_spans': [{'start': 792, 'end': 796, 'text': '[29]', 'ref_id': 'BIBREF29'}, {'start': 797, 'end': 801, 'text': '[30]', 'ref_id': 'BIBREF30'}, {'start': 802, 'end': 806, 'text': '[31]', 'ref_id': 'BIBREF31'}, {'start': 807, 'end': 811, 'text': '[32]', 'ref_id': 'BIBREF32'}, {'start': 812, 'end': 816, 'text': '[33]', 'ref_id': 'BIBREF33'}, {'start': 817, 'end': 821, 'text': '[34]', 'ref_id': 'BIBREF34'}, {'start': 822, 'end': 826, 'text': '[35]', 'ref_id': 'BIBREF35'}], 'ref_spans': [{'start': 127, 'end': 141, 'text': '(Figures 3A-B)', 'ref_id': 'FIGREF3'}, {'start': 470, 'end': 484, 'text': '(Figures 3A-B)', 'ref_id': 'FIGREF3'}, {'start': 701, 'end': 715, 'text': '(Figures 3A-B)', 'ref_id': 'FIGREF3'}, {'start': 1173, 'end': 1184, 'text': 'Figure S20)', 'ref_id': 'FIGREF2'}], 'section': 'Club cells, ciliated cells, and pneumocytes are likely targets of SARS-CoV-2 in respiratory tract'}, {'text': 'Our scRNAseq analysis also shows that ACE2 is expressed in small fractions of ciliated airway cells and epithelial cells of the nasal cavity (Figures 3A-B) . While no staining is observed for ACE2 in nasopharynx samples from the Human Protein Atlas (HPA) IHC dataset ( Figure S21) , a author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [{'start': 141, 'end': 155, 'text': '(Figures 3A-B)', 'ref_id': 'FIGREF3'}, {'start': 269, 'end': 280, 'text': 'Figure S21)', 'ref_id': 'FIGREF1'}], 'section': 'Club cells, ciliated cells, and pneumocytes are likely targets of SARS-CoV-2 in respiratory tract'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.24.005702 doi: bioRxiv preprint previous IHC study did report the staining of ACE2 in nasal and oral mucosa and the nasopharynx 33 . This expression is consistent with the high SARS-CoV-2 viral loads detected in nasal swab samples 28 . Intriguingly, mild degeneration of olfactory epithelium was observed in an immunosuppressed animal model infected with SARS-CoV 36 . These observations are correlated with emerging reports of anosmia/hyposmia (loss of smell) in otherwise asymptomatic COVID-19 patients from South Korea and other countries 37 . Such emerging clinical evidence emphasizes the need for further investigation into olfactory ACE2 expression via scRNA-seq and other modalities.', 'cite_spans': [{'start': 238, 'end': 240, 'text': '33', 'ref_id': 'BIBREF33'}, {'start': 341, 'end': 343, 'text': '28', 'ref_id': 'BIBREF28'}], 'ref_spans': [], 'section': 'Club cells, ciliated cells, and pneumocytes are likely targets of SARS-CoV-2 in respiratory tract'}, {'text': 'Taken together, these scRNA-seq analyses and triangulation to literature synthesis confirm that type II pneumocytes are a likely target of SARS-CoV-2 infection while also highlighting club cells, ciliated cells, and olfactory epithelial cells as additional potential sites of infection.', 'cite_spans': [], 'ref_spans': [], 'section': 'Club cells, ciliated cells, and pneumocytes are likely targets of SARS-CoV-2 in respiratory tract'}, {'text': 'We then classified 136 gastrointestinal cell types from nine scRNA-seq studies based on their expression of and literature associations to ACE2 ( Figure 4A ). These studies encompassed samples from the upper, mid, and lower GI tracts including tongue, esophagus, stomach, small intestine, and colon 19, [38] [39] [40] [41] [42] .', 'cite_spans': [{'start': 299, 'end': 302, 'text': '19,', 'ref_id': 'BIBREF19'}, {'start': 303, 'end': 307, 'text': '[38]', 'ref_id': 'BIBREF38'}, {'start': 308, 'end': 312, 'text': '[39]', 'ref_id': 'BIBREF39'}, {'start': 313, 'end': 317, 'text': '[40]', 'ref_id': 'BIBREF40'}, {'start': 318, 'end': 322, 'text': '[41]', 'ref_id': 'BIBREF41'}, {'start': 323, 'end': 327, 'text': '[42]', 'ref_id': None}], 'ref_spans': [{'start': 146, 'end': 155, 'text': 'Figure 4A', 'ref_id': 'FIGREF4'}], 'section': 'Tongue keratinocytes and mature small intestinal enterocytes are potential targets of SARS-CoV-2'}, {'text': 'This analysis highlights a robust expression of ACE2 in tongue keratinocytes that has not been strongly documented in the literature, as evidenced by the weak local context score between ACE2 and keratinocytes ( Figure 4B ). In fact, we found no previous reports of ACE2 expression in keratinocytes and only one recent report suggesting ACE2 expression in the human tongue based on a combination of bulk RNA-seq and a scRNA-seq dataset which has not been made publicly accessible 43 . We propose that a subset of ACE2 + tongue keratinocytes may serve as a novel site of SARS-CoV-2 entry and highlight the need to generate additional gene and protein expression data from human tongue samples to further evaluate this hypothesis. Emerging reports of loss of taste (dysgeusia) in otherwise non-symptomatic COVID-19 patients may warrant further study of the tongue in this pathology 44 .', 'cite_spans': [{'start': 480, 'end': 482, 'text': '43', 'ref_id': 'BIBREF43'}, {'start': 880, 'end': 882, 'text': '44', 'ref_id': 'BIBREF44'}], 'ref_spans': [{'start': 212, 'end': 221, 'text': 'Figure 4B', 'ref_id': 'FIGREF4'}], 'section': 'Tongue keratinocytes and mature small intestinal enterocytes are potential targets of SARS-CoV-2'}, {'text': 'We also found that ACE2 is highly expressed in both human and murine small intestinal enterocytes, confirming an association which has been moderately appreciated in literature, as indicated by our literature derived local score between ACE2 and enterocytes. However, to our knowledge, the transcriptional heterogeneity of ACE2 among enterocyte populations has never been explored. In this context, we found that ACE2 shows an increase in expression correlated with the maturation of murine small intestinal enterocytes, with minimal expression in stem cells and transit amplifying cells in contrast to most robust expression in mature enterocytes ( Figure  4 ). To the best of our knowledge, this is the first demonstration that ACE2 expression synchronously increases over the course of enterocyte maturation. The recognition of such intratissue heterogeneity is necessary to specify the cell types which are most likely responsible for the proposed fecal-oral transmission of COVID-19 12 .', 'cite_spans': [], 'ref_spans': [{'start': 650, 'end': 659, 'text': 'Figure  4', 'ref_id': 'FIGREF4'}], 'section': 'Tongue keratinocytes and mature small intestinal enterocytes are potential targets of SARS-CoV-2'}, {'text': 'author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Tongue keratinocytes and mature small intestinal enterocytes are potential targets of SARS-CoV-2'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.24.005702 doi: bioRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'Tongue keratinocytes and mature small intestinal enterocytes are potential targets of SARS-CoV-2'}, {'text': 'To determine whether the maturation-correlated expression pattern is unique to ACE2, we computed cosine similarities between the ACE2 gene expression vector (CP10K values in ~6,000 small intestinal enterocytes) and that of the ~15,700 other genes detected in this study ( Figure  5A ). For this analysis, the vector space is constituted of the individual cells as the dimensions using the gene expression values to construct the vectors (see Methods). Interestingly, we found that ANPEP, the established entry receptor for HCoV-229E, showed the third highest cosine similarity to ACE2 ( Figure 5B ). Further, DPP4 -the entry receptor for MERS coronavirus -is also among the top 1% of similarly expressed genes by this metric (Figure 5B ). We confirmed that both of these genes do indeed show a maturation-correlated transcriptional pattern similar to that of ACE2 ( Figure 5C-D) , highlighting an unexpected shared pattern of transcriptional heterogeneity among known coronavirus receptors in a cell population which may be relevant for viral transmission.', 'cite_spans': [], 'ref_spans': [{'start': 272, 'end': 282, 'text': 'Figure  5A', 'ref_id': 'FIGREF5'}, {'start': 587, 'end': 596, 'text': 'Figure 5B', 'ref_id': 'FIGREF5'}, {'start': 725, 'end': 735, 'text': '(Figure 5B', 'ref_id': 'FIGREF5'}, {'start': 866, 'end': 878, 'text': 'Figure 5C-D)', 'ref_id': 'FIGREF5'}], 'section': 'COVID-19, SARS, MERS and HCoV-229E receptors share a transcriptional signature correlated to maturation of small intestinal enterocytes'}, {'text': 'We then asked whether this shared pattern of transcriptional heterogeneity among coronavirus receptors is observed in the human small intestine. Indeed, among all enterocytes from a human scRNA-seq study, both ANPEP and DPP4 were among the top 1% of genes with similar expression vectors to that of ACE2 ( Figure S22A-B) . We independently validated this observation by computing gene expression correlations from bulk RNA-sequencing of human small intestine samples from GTEx (n = 187), which similarly revealed that DPP4 and ANPEP are among the top 1% of correlated genes to ACE2 ( Figure S22C ). In fact, among all ~18,500 genes mean expression > 1 TPM in GTEx small intestine samples, DPP4 shows the second highest correlation to ACE2 (r = 0.95).', 'cite_spans': [], 'ref_spans': [{'start': 306, 'end': 320, 'text': 'Figure S22A-B)', 'ref_id': 'FIGREF2'}, {'start': 584, 'end': 595, 'text': 'Figure S22C', 'ref_id': 'FIGREF2'}], 'section': 'COVID-19, SARS, MERS and HCoV-229E receptors share a transcriptional signature correlated to maturation of small intestinal enterocytes'}, {'text': 'To our knowledge, this is the first demonstration that all known coronavirus entry receptors display highly coordinated and maturation-correlated transcriptional expression patterns in intestinal epithelial cells. We propose that the requisite interaction with human proteins displaying a tightly defined expression gradient on apical surfaces of epithelial cells, which is shared among known coronavirus strains, may have fundamental implications for understanding the evolution, lifecycle, and/or transmission patterns of this family of viruses.', 'cite_spans': [], 'ref_spans': [], 'section': 'COVID-19, SARS, MERS and HCoV-229E receptors share a transcriptional signature correlated to maturation of small intestinal enterocytes'}, {'text': 'Recent advances in scRNA-seq are empowering us to study tissue and cellular transcriptomes at previously unprecedented resolutions. Several single-cell RNA sequencing based efforts such as the Human Cell Atlas are underway to catalog gene expression across tissues and cell types, and the raw data from many of these studies are available on public platforms such as the Broad Institute Single Cell Portal 45 and Gene Expression Omnibus (GEO). Analyses of these datasets are of interest to a wide range of researchers but currently prove challenging for all but a few due to the need for specialized workflows and computing infrastructures. Consequently, the widespread use of this data for biomedical research is hampered, an issue which is particularly evident in the face of public health crises like the ongoing COVID-19 pandemic. To address this unmet need, the nferX platform Single Cell resource enables the rapid and interactive analysis of author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.24.005702 doi: bioRxiv preprint the continually growing scRNAseq datasets by specialists and non-specialists alike. Furthermore, the seamless triangulation of scRNA-seq insights with global and local scores derived from the synthesis of accessible biomedical literature creates a truly first-in-class resource.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': \"By making the resource available to all academic researchers, we enable scientists to not only dive deeper into insights that are aligned with existing knowledge but also to prioritize the novel insights which warrant further experimental validation. Looking forward, we plan to automate the integration of the rapidly growing number of scRNA-seq studies so that access to the entire world's knowledge of single cell transcriptomes is just one click away for any researcher. As we do so, we encourage interactive feedback from the scientific community so that this platform can evolve to optimally support the research needs across the biomedical ecosystem, beyond the COVID-19 focus on the current study.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Combined with our analyses of bulk RNA-seq, IHC, and proteomics datasets, our characterization of the known human coronavirus receptors (ACE2, DPP4, ANPEP) using the nferX platform Single Cell resource represents the most comprehensive molecular fingerprint of host factors determining coronavirus infections including COVID-19. While this serves as a primer of the deep profiling that is made possible with this resource, we also identified several interesting aspects of coronavirus receptor biology which warrant further experimental follow-up.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'We identified tongue keratinocytes and olfactory epithelia as novel ACE2-expressing cell populations and thus as important potential sites of SARS-CoV-2 infection. This molecular fingerprint is a striking correlate to emerging clinical reports of dysgeusia 44 and anosmia 37 in COVID-19 patients, which strongly implicate the gustatory and olfactory systems in SARS-CoV-2 pathogenesis and human-to-human transmission. Tongue epithelial cells have also previously been shown to uptake Epstein-Barr virus 46 , and importantly a recent study found that ACE2 is appreciably expressed in tongue based on a small number of non-tumor bulk RNA-seq samples from TCGA 43 . This study further showed by scRNA-seq that ACE2 expression is observed in a subset of the human tongue (but not other oral mucosal) epithelial cells, albeit in only ~0.5% of the recovered epithelial population. This data has unfortunately not been released for public consumption but certainly does provide preliminary support for our finding, particularly as the listed set of cluster-defining genes for this population (SFN, KRT6A, KRT10) is consistent with the tongue keratinocyte identify from the Tabula Muris data set ( Figure S23) . We thus emphasize the imminent need for further generation of multi-omic expression data from large numbers of healthy and diseased human tongue samples drawn from a cohort of wide demographic representation.', 'cite_spans': [{'start': 503, 'end': 505, 'text': '46', 'ref_id': 'BIBREF46'}, {'start': 658, 'end': 660, 'text': '43', 'ref_id': 'BIBREF43'}], 'ref_spans': [{'start': 1190, 'end': 1201, 'text': 'Figure S23)', 'ref_id': 'FIGREF2'}], 'section': 'Discussion'}, {'text': 'We also observed that expression of ACE2 and other coronavirus receptors is intimately linked to the maturation status of small intestinal enterocytes, pinpointing the more mature subsets as the most likely cells to harbor SARS-CoV-2 virus. This finding amplifies the potential for fecal-oral transmission of COVID-19 10-12 and should motivate further experimental validation to determine whether monitoring of fecal viral loads should be considered clinically for diagnostic or prognostic purposes. author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.24.005702 doi: bioRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'We further found that this transcriptional mirroring of coronavirus entry receptors was not unique to small intestine but rather also strongly present among renal proximal tubule epithelial cells, where ACE2, DPP4, and ANPEP expression tends to be observed in the same cellular subsets. These observations suggest the existence of a transcriptional network spanning tissues and cell types which may drive and/or regulate coronavirus receptor expression. The question of whether coronaviruses have evolved to exploit such a network may be relevant to pursue, particularly given that downregulation of ACE2 by SARS-CoV has been reported previously and is associated with poor clinical outcomes 31, 47 . Perhaps other coronaviruses can similarly modulate the expression of their entry receptors to impact the clinical course of the induced disease.', 'cite_spans': [{'start': 692, 'end': 695, 'text': '31,', 'ref_id': 'BIBREF31'}, {'start': 696, 'end': 698, 'text': '47', 'ref_id': 'BIBREF47'}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'The emerging picture of the coronavirus life cycle appears to be intricately interwoven with many proteins beyond the primary host receptors. For instance, a recent structural complex of the SARS-CoV-2 spike protein with ACE2 identified SLC6A19 as an interaction partner of ACE2 48 . Further, spike proteins from some coronaviruses can interact with CEACAM1 49 and sialylated glycans similar to influenza hemagglutinin 50 as host receptors. Future studies are likely to highlight several other proteins and glycans that constitute the \"interactome\" of the coronavirus proteome. Understanding the expression profiles of the interactome across tissues will provide systems level insights on the cellular dynamics of the functional partners and the regulatory machinery of the host receptor proteins. Like in the current study, the nferX platform will be an excellent resource for unraveling the purported interaction partners for coronavirus receptors and profiling their expression across different tissues and cells constituting the human body.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': \"Overall, this study evidences the utility of an integrative data science platform to enable rapid and high-throughput analysis of publicly available data to generate relevant biological insights and scientific hypotheses. We hope that by making our biomedical knowledge synthesis-augmented single cell platform publicly accessible, we help empower the research community to advance our understanding of the world's most pressing biomedical challenges such as COVID-19.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'In order to capture biomedical literature based associations, the nferX platform defines two scores: a \"local score\" and a \"global score\", as described previously 51 . Briefly, the local score represents a traditional natural language processing technique which captures the strength of association between two concepts in a selected corpus of biomedical literature based on the frequency of their co-occurrence normalized by the frequency of each individual concept throughout the corpus. A higher local score between Concept X and Concept Y indicates that these concepts are frequently mentioned in close proximity to each other more frequently than would be expected by chance. The global score, on the other hand, is based on the neural network renaissance that has recently taken place in the Natural Language Processing (NLP) field. To compute global scores, all tokens (e.g. words and phrases) are projected in a high-dimensional vector space of word embeddings. These vectors serve to represent the \"neighborhood\" of concepts which occur around a given concept. The cosine distance between any two vectors author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [{'start': 163, 'end': 165, 'text': '51', 'ref_id': 'BIBREF51'}], 'ref_spans': [], 'section': 'Unstructured biomedical knowledge synthesis and triangulation capabilities'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.24.005702 doi: bioRxiv preprint measures the similarity of these neighborhoods and is the basis for our global score metric, where concepts which are more similar in this vector space have a higher global score.', 'cite_spans': [], 'ref_spans': [], 'section': 'Unstructured biomedical knowledge synthesis and triangulation capabilities'}, {'text': 'While the global scores in this work are computed in the embedding space of word2vec model, it can also be computed in the embedding space of any deep learning model including recent transformer-based models like BERT 52 . These may have complementary benefits to word2vec embeddings since the embeddings are context sensitive having different vectors for different sentence contexts. However, despite the context sensitive nature of BERT embeddings a global score computation for a phrase may still be of value given the score is computed across sentence embeddings capturing the context sensitive nature of those phrases.', 'cite_spans': [], 'ref_spans': [], 'section': 'Unstructured biomedical knowledge synthesis and triangulation capabilities'}, {'text': 'From a visualization perspective, the local score and global score (\"Signals\") are represented in the platform using bubbles where bubble size corresponds to the local score and color intensity corresponds to the global score. This allows users to rapidly determine the strength of association between any two concepts throughout biomedical literature. We consider concepts which show both high local and global scores to be \"concordant\" and have found that these typically recapitulate well-known associations.', 'cite_spans': [], 'ref_spans': [], 'section': 'Unstructured biomedical knowledge synthesis and triangulation capabilities'}, {'text': 'The nferX platform also supports a logical thought engine that enables AND (conjunction), OR (disjunction), and NOT (negation) queries -the universal logic gates. This engine is referred to as \"dynamic adjacency\" and leverages a highly distributed main memory approach that allows the computation of local scores for any type of logical query in real time. Fundamentally, this system allows a user to extract all 100-word fragments of text which meet the specified logical query. We then calculate local scores for all other tokens occurring within these fragments, which quantifies the likelihood (i.e. odds) of each token occurring this frequently within these textual fragments by chance.', 'cite_spans': [], 'ref_spans': [], 'section': 'Unstructured biomedical knowledge synthesis and triangulation capabilities'}, {'text': 'The platform further leverages statistical inference to calculate \"enrichments\" based on structured data, thus enabling real-time triangulation of signals from the unstructured biomedical knowledge graph various other structured databases (e.g. curated ontologies, RNA-sequencing datasets, human genetic associations, protein-protein interactions). This facilitates unbiased hypothesisfree learning and faster pattern recognition, and it allows users to more holistically determine the veracity of concept associations. Finally, the platform allows the user to identify and further examine the documents and textual fragments from which the knowledge synthesis signals are derived using the Documents and Signals applications.', 'cite_spans': [], 'ref_spans': [], 'section': 'Unstructured biomedical knowledge synthesis and triangulation capabilities'}, {'text': 'If we have an automated method that, given a corpus (consisting of text and other structured and semi-structured data as is often the case with biomedical data), comes up with a strength of association score between a query entity and all the tokens or entities present in the corpus, then that score can be used to obtain a ranking of tokens or entities related to the query. There have been other motivations for ranking the association strength of tokens/entities (e.g. for use in picking among possible tokens in speech recognition or Optical Character Recognition -OCR). author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Association Scores'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.24.005702 doi: bioRxiv preprint Generally, association scores are based in some way on the co-occurrences of the tokens (or referents to the entities) in the text within small windows of text. Co-occurrences have been studied in linguistics/NLP since at least Firth\\'s maxim that \"a word is known by the company it keeps.\" One popular traditional measure for association strength between tokens in text is pointwise mutual information, or PMI 53 , which we consider in several association scores.', 'cite_spans': [], 'ref_spans': [], 'section': 'Association Scores'}, {'text': \"Formally, for a given corpus, an association score is some real-valued function S(q, t) where q is a query token/entity and t is another token/entity. We discussed association scores as if they were symmetric above, but for some convenience later on our formal notion is asymmetric: there's a query q and a token t. In particular, we later extend q to logical combinations of tokens/entities. The association score need not be symmetric (for logical queries it cannot be as t is still restricted to single tokens/entities) though it often is when q and t are both single tokens/entities.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Measures of association'}, {'text': 'Context of q -All the measures involve the notion of the \"context\" of the query q. The context of q are those locations in the corpus deemed to be \"near\" to q. For single token queries, follow the typical approach of defining context as those locations in the corpus that are within some fixed number of words w (the window size w is a tunable parameter) from an occurrence of q in the corpus. The dynamic adjacency engine generalizes this notion of context in a natural way to logical queries; the context for a logical q is a certain set of fixed-length fragments.', 'cite_spans': [], 'ref_spans': [], 'section': 'Measures of association'}, {'text': 'Co-occurrences -This is just the number of times t appears in the context of q.', 'cite_spans': [], 'ref_spans': [], 'section': 'Measures of association'}, {'text': 'Traditional PMI -This is log(p(t | q) / p(t)). Here p(t | q) is the number of times t occurs in the context of q (ie co-occurrences of t and q) divided by the total length of all q contexts in the corpus, whereas p(t) is the number of occurrences of t in the entire corpus, divided by the corpus length.', 'cite_spans': [], 'ref_spans': [], 'section': 'Measures of association'}, {'text': 'Word2vec cosine distance -The popular word2vec algorithm [5] generates a vector (we use 300dimensional vector representation) for each token in a corpus. The purpose of these vectors is usually to be used as features in downstream NLP tasks. But they can also be used for similarity. The original paper validates the vectors by testing them on word similarity tasks: the association score is the cosine between the vector for q and the vector for t. This score only applies to singletoken q.', 'cite_spans': [{'start': 57, 'end': 60, 'text': '[5]', 'ref_id': 'BIBREF4'}], 'ref_spans': [], 'section': 'Measures of association'}, {'text': \"Exponential mask PMI (ExpPMI) -This is our first new proposed score. PMI treats every position in a binary way: it's either in the context of q or not. With a window size of say 50, a token which appears 3 words from a query q and a token which appears 45 words from a query q are treated the same. We thought it might be useful to consider a measure which distinguishes positions in the context based on the number of words away that position is from an occurrence of q. We did this by weighting the positions in the context by some weight between 0 and 1. Our weighting is based on an exponential decay (which has some nice properties especially when we extend to the case of logical queries). author/funder. All rights reserved. No reuse allowed without permission.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Measures of association'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.24.005702 doi: bioRxiv preprint Local score -This is another new proposed score. We find that PMI and ExpPMI can vary a lot for small samples (i.e. small numbers of co-occurrences, occurrences). The Local Score is log(coocc) * sigmoid(PMI -0.5), constructed to correct for this; we found that this formula too works well empirically.', 'cite_spans': [], 'ref_spans': [], 'section': 'Measures of association'}, {'text': 'Exponential mask local score (ExpLocalScore) -We apply both modifications together: the exponential mask score is log(weighted_coocc) * sigmoid(expPMI -0.5). Here weighted_coocc is the sum of the weights of the positions of the corpus Evaluation of association scores are further described in the Supplementary Information.', 'cite_spans': [], 'ref_spans': [], 'section': 'Measures of association'}, {'text': 'The objective of the single cell platform is to enable dynamic visualization and analysis of single cell RNA-sequencing data. Currently, there are over 30 scRNA-seq studies available for analysis in the Single Cell app including studies from human donors/patients covering tissues such as adipose tissue, blood, bone marrow, colon, esophagus, liver, lung, kidney, ovary, nasal epithelium, pancreas, placenta, prostate, retina, small intestine, and spleen. Because no pantissue reference dataset yet exists for humans, we have manually selected individual studies to maximally cover the set of human tissues. In some cases, these studies contain cells from both healthy donors and patients affected by a specified pathology such as ulcerative colitis (colon) or asthma (lung). There are also a number of murine scRNA-seq studies covering tissues including adipose tissue, airway epithelium, blood, bone marrow, brain, breast, colon, heart, kidney, liver, lung, ovary, pancreas, placenta, prostate, skeletal muscle, skin, spleen, stomach, small intestine, testis, thymus, tongue, trachea, urinary bladder, uterus, and vasculature. Note that two of these murine studies (Tabula Muris and Mouse Cell Atlas) include ~20 tissues each.', 'cite_spans': [], 'ref_spans': [], 'section': 'Single-cell RNA-seq analysis platform'}, {'text': 'For each study, a counts matrix was downloaded from a public data repository such as the Gene Expression Omnibus (GEO) or the Broad Institute Single Cell Portal ( Table S1 ). Note that this data has not been re-processed from the raw sequencing output, and so it is likely that alignment and quantification of gene expression was performed using different tools for different studies. In some cases, multiple complementary datasets have been generated from a single publication. In these cases, we have generated separate entries in the Single Cell platform.', 'cite_spans': [], 'ref_spans': [{'start': 163, 'end': 171, 'text': 'Table S1', 'ref_id': None}], 'section': 'Single-cell data processing pipeline'}, {'text': 'While counts matrices have been generated using different technologies (e.g. Drop-Seq, 10x Genomics, etc.) and different alignment/pre-processing pipelines, all counts matrices were scaled such that each cell contains a total of 10,000 scaled counts (i.e. the sum of expression values for all genes equals 10,000 in each individual cell). All data were uniformly processed using the Seurat v3 package 54 . In short, this pipeline involves the following steps. First, we identify 2000 variable genes across the given dataset and then perform linear dimensionality reduction by principal component analysis (PCA). Using the set of principal components which contribute >80% of variance across the dataset, we then do the following: (i) perform graph-based clustering to identify groups of cells with similar expression profiles (Louvain clustering), (ii) compute UMAP and tSNE coordinates for each individual cell (used for data visualization) and (iii) annotate cell author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [{'start': 401, 'end': 403, 'text': '54', 'ref_id': 'BIBREF54'}], 'ref_spans': [], 'section': 'Single-cell data processing pipeline'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.24.005702 doi: bioRxiv preprint clusters. Note that the three human pancreatic datasets (GSE81076, GSE85241, GSE86469) were integrated together in a shared multi-dimensional space using CCA (Canonical Correlation Analysis) and the integration method in the Seurat v3 package 54 . Cell clustering and computation of dimensionality reduction coordinates were performed on this integrated dataset.', 'cite_spans': [], 'ref_spans': [], 'section': 'Single-cell data processing pipeline'}, {'text': 'In cases where publicly deposited counts matrices are accompanied by author-assigned annotations for individual cells or clusters, we have retained these cell annotations for display in the platform and accompanying analyses. For any study which was not accompanied by a metadata file containing cluster annotations, we have manually labeled clusters based on sets of canonical \"cluster-defining genes.\" In these cases, we have attempted to leverage annotations and descriptions of gene expression patterns described by study authors in the manuscript text and figures corresponding to the data being analyzed.', 'cite_spans': [], 'ref_spans': [], 'section': 'Cell cluster annotation'}, {'text': \"The platform allows users to query any gene in any selected study. The corresponding data is displayed in commonly employed formats including a series of violin plots and as a set of dimensionality reduction plots. Expression is summarized by listing the percent of cells expressing Gene G in each annotated cluster and the mean expression of Gene G in each cluster. To measure the specificity of Gene G expression to each Cluster C, we compute a cohen's D value which assesses the effect size between the mean expression of Gene G in cluster C and the mean expression of Gene G in all other clusters. Specifically, the cohen's D formula is given as follows: (MeanC -MeanA)/(sqrt(StDevC 2 + StDevA 2 )), where C represents the cluster of interest and A represents the complement of C (i.e. all other cell clusters). Note that this is functionally similar to the computation of paired fold change values and p-values between clusters which is frequently used to identify cluster-defining genes.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Metrics to Summarize Cluster-Level Gene Expression'}, {'text': 'Within the platform, we support the run-time computation of cosine similarity (i.e. 1 -cosine distance) between the queried gene and all other genes. This provides a measure of expression similarity across cells and can be used to identify co-regulated and co-expressed genes. Specifically, to perform this computation, we construct a \"gene expression vector\" for each gene G. This corresponds to the set of CP10K values for gene G in each individual cell from the selected populations in the selected study.', 'cite_spans': [], 'ref_spans': [], 'section': 'Gene-Gene Cosine Similarity'}, {'text': 'For each single-cell dataset, we examined the expression of ACE2, TMPRSS2, ANPEP, and DPP4. We generally considered a cell population to potentially express a gene if at least 5% of cells from that cluster showed non-zero expression of this gene. For each dataset, we show a figure which includes a UMAP dimensionality reduction plot colored by annotated cell type along with identical plots colored by the expression level of each coronavirus receptor in all individual cells. In some cases, we also show violin plots from the platform which automatically integrate literature-derived insights to highlight whether there exist textual associations between the queried gene and the tissue/cell types identified in the selected study. author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Profiling Expression of Coronavirus Receptors in Single-cell Datasets'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.24.005702 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.24.005702 doi: bioRxiv preprint Figure 5 Expression of all known coronavirus receptors synchronously increase with enterocyte maturation author/funder. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [{'start': 284, 'end': 292, 'text': 'Figure 5', 'ref_id': 'FIGREF5'}], 'section': 'Profiling Expression of Coronavirus Receptors in Single-cell Datasets'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.24.005702 doi: bioRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'Profiling Expression of Coronavirus Receptors in Single-cell Datasets'}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'Johns Hopkins Coronavirus Resource Center', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF1': {'ref_id': 'b1', 'title': 'A pneumonia outbreak associated with a new coronavirus of probable bat origin', 'authors': [{'first': 'P', 'middle': [], 'last': 'Zhou', 'suffix': ''}], 'year': 2020, 'venue': 'Nature', 'volume': '579', 'issn': '', 'pages': '270--273', 'other_ids': {}}, 'BIBREF2': {'ref_id': 'b2', 'title': 'Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus', 'authors': [{'first': 'W', 'middle': [], 'last': 'Li', 'suffix': ''}], 'year': 2003, 'venue': 'Nature', 'volume': '426', 'issn': '', 'pages': '450--454', 'other_ids': {}}, 'BIBREF3': {'ref_id': 'b3', 'title': 'Human coronavirus NL63 employs the severe acute respiratory syndrome coronavirus receptor for cellular entry', 'authors': [{'first': 'H', 'middle': [], 'last': 'Hofmann', 'suffix': ''}], 'year': 2005, 'venue': 'Proc. Natl. Acad. Sci. U. S. A', 'volume': '102', 'issn': '', 'pages': '7988--7993', 'other_ids': {}}, 'BIBREF4': {'ref_id': 'b4', 'title': 'Dipeptidyl peptidase 4 is a functional receptor for the emerging human coronavirus-EMC', 'authors': [{'first': 'V', 'middle': ['S'], 'last': 'Raj', 'suffix': ''}], 'year': 2013, 'venue': 'Nature', 'volume': '495', 'issn': '', 'pages': '251--254', 'other_ids': {}}, 'BIBREF5': {'ref_id': 'b5', 'title': 'Human aminopeptidase N is a receptor for human coronavirus 229E', 'authors': [{'first': 'C', 'middle': ['L'], 'last': 'Yeager', 'suffix': ''}], 'year': 1992, 'venue': 'Nature', 'volume': '357', 'issn': '', 'pages': '420--422', 'other_ids': {}}, 'BIBREF6': {'ref_id': 'b6', 'title': 'The novel coronavirus 2019 (2019-nCoV) uses the SARS-coronavirus receptor ACE2 and the cellular protease TMPRSS2 for entry into target cells', 'authors': [{'first': 'M', 'middle': [], 'last': 'Hoffmann', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.01.31.929042']}}, 'BIBREF7': {'ref_id': 'b7', 'title': 'The spike protein of the emerging betacoronavirus EMC uses a novel coronavirus receptor for entry, can be activated by TMPRSS2, and is targeted by neutralizing antibodies', 'authors': [{'first': 'S', 'middle': [], 'last': 'Gierer', 'suffix': ''}], 'year': 2013, 'venue': 'J. Virol', 'volume': '87', 'issn': '', 'pages': '5502--5511', 'other_ids': {}}, 'BIBREF8': {'ref_id': 'b8', 'title': 'Clinical course and risk factors for mortality of adult inpatients with COVID-19 in Wuhan, China: a retrospective cohort study', 'authors': [{'first': 'F', 'middle': [], 'last': 'Zhou', 'suffix': ''}], 'year': 2020, 'venue': 'The Lancet', 'volume': '', 'issn': '', 'pages': '30566--30569', 'other_ids': {'DOI': ['10.1016/s0140-6736(20']}}, 'BIBREF9': {'ref_id': 'b9', 'title': 'Characteristics of pediatric SARS-CoV-2 infection and potential evidence for persistent fecal viral shedding', 'authors': [{'first': 'Y', 'middle': [], 'last': 'Xu', 'suffix': ''}], 'year': 2020, 'venue': 'Nature Medicine', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1038/s41591-020-0817-4']}}, 'BIBREF10': {'ref_id': 'b10', 'title': 'COVID-19: Gastrointestinal manifestations and potential fecaloral transmission', 'authors': [{'first': 'J', 'middle': [], 'last': 'Gu', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Han', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Wang', 'suffix': ''}], 'year': 2020, 'venue': 'Gastroenterology', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1053/j.gastro.2020.02.054']}}, 'BIBREF11': {'ref_id': 'b11', 'title': 'Evidence for gastrointestinal infection of SARS-CoV-2', 'authors': [{'first': 'F', 'middle': [], 'last': 'Xiao', 'suffix': ''}], 'year': 2020, 'venue': 'Gastroenterology', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1053/j.gastro.2020.02.055']}}, 'BIBREF12': {'ref_id': 'b12', 'title': 'Binding of SARS coronavirus to its receptor damages islets and causes acute diabetes', 'authors': [{'first': 'J.-K', 'middle': [], 'last': 'Yang', 'suffix': ''}, {'first': 'S.-S', 'middle': [], 'last': 'Lin', 'suffix': ''}, {'first': 'X.-J', 'middle': [], 'last': 'Ji', 'suffix': ''}, {'first': 'L.-M', 'middle': [], 'last': 'Guo', 'suffix': ''}], 'year': 2010, 'venue': 'Acta Diabetol', 'volume': '47', 'issn': '', 'pages': '193--199', 'other_ids': {}}, 'BIBREF13': {'ref_id': 'b13', 'title': 'Efficient Estimation of Word Representations in Vector Space', 'authors': [{'first': 'T', 'middle': [], 'last': 'Mikolov', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Chen', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Corrado', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Dean', 'suffix': ''}], 'year': 2013, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF14': {'ref_id': 'b14', 'title': 'Deep learning', 'authors': [{'first': 'Y', 'middle': [], 'last': 'Lecun', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Bengio', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Hinton', 'suffix': ''}], 'year': 2015, 'venue': 'Nature', 'volume': '521', 'issn': '', 'pages': '436--444', 'other_ids': {}}, 'BIBREF15': {'ref_id': 'b15', 'title': 'Unsupervised word embeddings capture latent knowledge from materials science literature', 'authors': [{'first': 'V', 'middle': [], 'last': 'Tshitoyan', 'suffix': ''}], 'year': 2019, 'venue': 'Nature', 'volume': '571', 'issn': '', 'pages': '95--98', 'other_ids': {}}, 'BIBREF16': {'ref_id': 'b16', 'title': 'Reality check on reproducibility', 'authors': [], 'year': 2016, 'venue': 'Nature', 'volume': '533', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF17': {'ref_id': 'b17', 'title': 'Call to Action to the Tech Community on New Machine Readable', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF19': {'ref_id': 'b19', 'title': 'Single-cell transcriptomics of 20 mouse organs creates a Tabula Muris', 'authors': [{'first': '', 'middle': [], 'last': 'Tabula Muris Consortium', 'suffix': ''}], 'year': 2018, 'venue': 'Nature', 'volume': '562', 'issn': '', 'pages': '367--372', 'other_ids': {}}, 'BIBREF20': {'ref_id': 'b20', 'title': 'All rights reserved. No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the', 'authors': [{'first': 'Å', 'middle': [], 'last': 'Segerstolpe', 'suffix': ''}], 'year': 2016, 'venue': 'Cell Metab', 'volume': '24', 'issn': '', 'pages': '593--607', 'other_ids': {'DOI': ['10.1101/2020.03.24.005702']}}, 'BIBREF21': {'ref_id': 'b21', 'title': 'De Novo Prediction of Stem Cell Identity using Single-Cell Transcriptome Data', 'authors': [{'first': 'D', 'middle': [], 'last': 'Grün', 'suffix': ''}], 'year': 2016, 'venue': 'Cell Stem Cell', 'volume': '19', 'issn': '', 'pages': '266--277', 'other_ids': {}}, 'BIBREF22': {'ref_id': 'b22', 'title': 'A Single-Cell Transcriptome Atlas of the Human Pancreas', 'authors': [{'first': 'M', 'middle': ['J'], 'last': 'Muraro', 'suffix': ''}], 'year': 2016, 'venue': 'Cell Syst', 'volume': '3', 'issn': '', 'pages': '385--394', 'other_ids': {}}, 'BIBREF25': {'ref_id': 'b25', 'title': 'Proteomics. Tissue-based map of the human proteome', 'authors': [{'first': 'M', 'middle': [], 'last': 'Uhlén', 'suffix': ''}], 'year': 2015, 'venue': 'Science', 'volume': '347', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF26': {'ref_id': 'b26', 'title': 'A deep proteome and transcriptome abundance atlas of 29 healthy human tissues', 'authors': [{'first': 'D', 'middle': [], 'last': 'Wang', 'suffix': ''}], 'year': 2019, 'venue': 'Mol. Syst. Biol', 'volume': '15', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF27': {'ref_id': 'b27', 'title': 'A draft map of the human proteome', 'authors': [{'first': 'M.-S', 'middle': [], 'last': 'Kim', 'suffix': ''}], 'year': 2014, 'venue': 'Nature', 'volume': '509', 'issn': '', 'pages': '575--581', 'other_ids': {}}, 'BIBREF28': {'ref_id': 'b28', 'title': 'Detection of SARS-CoV-2 in Different Types of Clinical Specimens', 'authors': [{'first': 'W', 'middle': [], 'last': 'Wang', 'suffix': ''}], 'year': 2020, 'venue': 'JAMA', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1001/jama.2020.3786']}}, 'BIBREF29': {'ref_id': 'b29', 'title': 'Evidence that TMPRSS2 activates the severe acute respiratory syndrome coronavirus spike protein for membrane fusion and reduces viral control by the humoral immune response', 'authors': [{'first': 'I', 'middle': [], 'last': 'Glowacka', 'suffix': ''}], 'year': 2011, 'venue': 'J. Virol', 'volume': '85', 'issn': '', 'pages': '4122--4134', 'other_ids': {}}, 'BIBREF30': {'ref_id': 'b30', 'title': 'Pathology of experimental SARS coronavirus infection in cats and ferrets', 'authors': [{'first': 'J', 'middle': ['M A'], 'last': 'Van Den Brand', 'suffix': ''}], 'year': 2008, 'venue': 'Vet. Pathol', 'volume': '45', 'issn': '', 'pages': '551--562', 'other_ids': {}}, 'BIBREF31': {'ref_id': 'b31', 'title': 'Differential downregulation of ACE2 by the spike proteins of severe acute respiratory syndrome coronavirus and human coronavirus NL63', 'authors': [{'first': 'I', 'middle': [], 'last': 'Glowacka', 'suffix': ''}], 'year': 2010, 'venue': 'J. Virol', 'volume': '84', 'issn': '', 'pages': '1198--1205', 'other_ids': {}}, 'BIBREF32': {'ref_id': 'b32', 'title': 'Cleavage and activation of the severe acute respiratory syndrome coronavirus spike protein by human airway trypsin-like protease', 'authors': [{'first': 'S', 'middle': [], 'last': 'Bertram', 'suffix': ''}], 'year': 2011, 'venue': 'J. Virol', 'volume': '85', 'issn': '', 'pages': '13363--13372', 'other_ids': {}}, 'BIBREF33': {'ref_id': 'b33', 'title': 'Tissue distribution of ACE2 protein, the functional receptor for SARS coronavirus. A first step in understanding SARS pathogenesis', 'authors': [{'first': 'I', 'middle': [], 'last': 'Hamming', 'suffix': ''}], 'year': 2004, 'venue': 'J. Pathol', 'volume': '203', 'issn': '', 'pages': '631--637', 'other_ids': {}}, 'BIBREF34': {'ref_id': 'b34', 'title': 'SARS-CoV replicates in primary human alveolar type II cell cultures but not in type I-like cells', 'authors': [{'first': 'E', 'middle': ['C'], 'last': 'Mossel', 'suffix': ''}], 'year': 2008, 'venue': 'Virology', 'volume': '372', 'issn': '', 'pages': '127--135', 'other_ids': {}}, 'BIBREF35': {'ref_id': 'b35', 'title': 'Exploring the pathogenesis of severe acute respiratory syndrome (SARS): the tissue distribution of the coronavirus (SARS-CoV) and its putative receptor, angiotensin-converting enzyme 2 (ACE2)', 'authors': [{'first': 'K', 'middle': ['F'], 'last': 'To', 'suffix': ''}, {'first': 'A', 'middle': ['W'], 'last': 'Lo', 'suffix': ''}], 'year': 2004, 'venue': 'J. Pathol', 'volume': '203', 'issn': '', 'pages': '740--743', 'other_ids': {}}, 'BIBREF36': {'ref_id': 'b36', 'title': 'An immunosuppressed Syrian golden hamster model for SARS-CoV infection', 'authors': [{'first': 'S', 'middle': ['R'], 'last': 'Schaecher', 'suffix': ''}], 'year': 2008, 'venue': 'Virology', 'volume': '380', 'issn': '', 'pages': '312--321', 'other_ids': {}}, 'BIBREF38': {'ref_id': 'b38', 'title': 'Mapping the Mouse Cell Atlas by Microwell-Seq', 'authors': [{'first': 'X', 'middle': [], 'last': 'Han', 'suffix': ''}], 'year': 2018, 'venue': 'Cell', 'volume': '172', 'issn': '', 'pages': '1091--1107', 'other_ids': {}}, 'BIBREF39': {'ref_id': 'b39', 'title': 'Single-cell transcriptome analysis reveals differential nutrient absorption functions in human intestine', 'authors': [{'first': 'Y', 'middle': [], 'last': 'Wang', 'suffix': ''}], 'year': 2020, 'venue': 'J. Exp. Med', 'volume': '217', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF40': {'ref_id': 'b40', 'title': 'Intra-and Inter-cellular Rewiring of the Human Colon during Ulcerative Colitis', 'authors': [{'first': 'C', 'middle': ['S'], 'last': 'Smillie', 'suffix': ''}], 'year': 2019, 'venue': 'Cell', 'volume': '178', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF41': {'ref_id': 'b41', 'title': '333-author/funder. All rights reserved. No reuse allowed without permission. The copyright holder for this preprint', 'authors': [{'first': 'A', 'middle': ['L'], 'last': 'Haber', 'suffix': ''}], 'year': 2017, 'venue': 'Nature', 'volume': '551', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.03.24.005702']}}, 'BIBREF43': {'ref_id': 'b43', 'title': 'High expression of ACE2 receptor of 2019-nCoV on the epithelial cells of oral mucosa', 'authors': [{'first': 'H', 'middle': [], 'last': 'Xu', 'suffix': ''}], 'year': 2020, 'venue': 'Int. J. Oral Sci', 'volume': '12', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF44': {'ref_id': 'b44', 'title': 'Anosmia, Hyposmia, and Dysgeusia Symptoms of Coronavirus Disease. American Academy of Otolaryngology-Head and Neck Surgery', 'authors': [{'first': '', 'middle': [], 'last': 'Aao-Hns', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF46': {'ref_id': 'b46', 'title': 'Epstein-Barr virus infection of polarized tongue and nasopharyngeal epithelial cells', 'authors': [{'first': 'S', 'middle': ['M'], 'last': 'Tugizov', 'suffix': ''}, {'first': 'J', 'middle': ['W'], 'last': 'Berline', 'suffix': ''}, {'first': 'J', 'middle': ['M'], 'last': 'Palefsky', 'suffix': ''}], 'year': 2003, 'venue': 'Nat. Med', 'volume': '9', 'issn': '', 'pages': '307--314', 'other_ids': {}}, 'BIBREF47': {'ref_id': 'b47', 'title': 'Trilogy of ACE2: a peptidase in the renin-angiotensin system, a SARS receptor, and a partner for amino acid transporters', 'authors': [{'first': 'K', 'middle': [], 'last': 'Kuba', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Imai', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Ohto-Nakanishi', 'suffix': ''}, {'first': 'J', 'middle': ['M'], 'last': 'Penninger', 'suffix': ''}], 'year': 2010, 'venue': 'Pharmacol. Ther', 'volume': '128', 'issn': '', 'pages': '119--128', 'other_ids': {}}, 'BIBREF48': {'ref_id': 'b48', 'title': 'Structural basis for the recognition of the SARS-CoV-2 by full-length human ACE2', 'authors': [{'first': 'R', 'middle': [], 'last': 'Yan', 'suffix': ''}], 'year': 2020, 'venue': 'Science', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1126/science.abb2762']}}, 'BIBREF49': {'ref_id': 'b49', 'title': 'N-terminal domain of the murine coronavirus receptor CEACAM1 is responsible for fusogenic activation and conformational changes of the spike protein', 'authors': [{'first': 'H', 'middle': ['S'], 'last': 'Miura', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Nakagaki', 'suffix': ''}, {'first': 'F', 'middle': [], 'last': 'Taguchi', 'suffix': ''}], 'year': 2004, 'venue': 'J. Virol', 'volume': '78', 'issn': '', 'pages': '216--223', 'other_ids': {}}, 'BIBREF50': {'ref_id': 'b50', 'title': 'Structural basis for human coronavirus attachment to sialic acid receptors', 'authors': [{'first': 'M', 'middle': ['A'], 'last': 'Tortorici', 'suffix': ''}], 'year': 2019, 'venue': 'Nat. Struct. Mol. Biol', 'volume': '26', 'issn': '', 'pages': '481--489', 'other_ids': {}}, 'BIBREF51': {'ref_id': 'b51', 'title': 'Recapitulation and Retrospective Prediction of Biomedical Associations Using Temporally-enabled Word Embeddings', 'authors': [{'first': 'J', 'middle': [], 'last': 'Park', 'suffix': ''}], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/627513']}}, 'BIBREF52': {'ref_id': 'b52', 'title': 'Proceedings of the 2019 Conference of the North', 'authors': [{'first': 'J', 'middle': [], 'last': 'Devlin', 'suffix': ''}, {'first': 'M.-W', 'middle': [], 'last': 'Chang', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Lee', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Toutanova', 'suffix': ''}], 'year': 2019, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.18653/v1/n19-1423']}}, 'BIBREF53': {'ref_id': 'b53', 'title': 'The Statistics of Word Cooccurrences: Word Pairs and Collocations', 'authors': [{'first': 'S', 'middle': [], 'last': 'Evert', 'suffix': ''}], 'year': 2005, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF54': {'ref_id': 'b54', 'title': 'Integrating single-cell transcriptomic data across different conditions, technologies, and species', 'authors': [{'first': 'A', 'middle': [], 'last': 'Butler', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Hoffman', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Smibert', 'suffix': ''}, {'first': 'E', 'middle': [], 'last': 'Papalexi', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Satija', 'suffix': ''}], 'year': 2018, 'venue': 'Nat. Biotechnol', 'volume': '36', 'issn': '', 'pages': '411--420', 'other_ids': {}}, 'BIBREF55': {'ref_id': 'b55', 'title': 'No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.03.24.005702']}}}, 'ref_entries': {'FIGREF0': {'text': 'Legends', 'latex': None, 'type': 'figure'}, 'FIGREF1': {'text': 'Knowledge synthesis and the nferX Single Cell resource. (A) Knowledge synthesis: capturing association between concepts from over 100 million documents. Schematic shows the work-flow for generating literature-derived associations between phrases. Local score and global score are defined and the types of literature-derived associations are shown for combinations of high and low local and global scores. (B) Datasets enabling knowledge synthesis-powered scRNA-seq analysis platform. Single Cell RNAseq data was obtained from over 30 publicly available human and mouse single cell RNA-seq datasets. Bulk RNA-seq data was obtained from Gene Expression Omnibus (GEO) and the Genotype Tissue Expression (GTEx) project portal. Protein-level expression of coronavirus receptors was assessed using a collection of immunohistochemistry (IHC) images and tissue proteomics datasets from the Human Protein Atlas and the Human Proteome Map.Literature-derived association scores are obtained from over 100 million biomedical documents (C) Tissues and cell-types identified by one or more modalities to express ACE2, the putative receptor of SARS-CoV-2 spike protein.', 'latex': None, 'type': 'figure'}, 'FIGREF2': {'text': 'Triangulation of knowledge synthesis with ACE2 expression profile by scRNAseq across cells and tissues. Scatterplot shows comparison of percentage of cells with nonzero expression (x-axis) against literature-derived associations: local score (y-axis and size of circles) and global score (transparency of circles). Data includes more than 800 cell populations from over 25 murine and human tissues comprising over 1 million cells.', 'latex': None, 'type': 'figure'}, 'FIGREF3': {'text': 'Triangulation of ACE2 expression in the respiratory tract with literature-derived insights. (A) Schematic representation of the respiratory system highlighting key cell types from the nasal cavity, airway and alveoli. Scatterplot shows comparison of percentage of cells with non-zero expression (x-axis) from eight single cell studies against literature-derived associations: local score (y-axis and size of circles) and global score (transparency of circles). (B) Assessing literature-based and scRNAseq-based associations between ACE2 and respiratory tract cells. On the left, the dimensionality reduction plots show different cell populations associated with lung and olfactory epithelium. On the right, violin plots show the distribution of ACE2 expression levels in selected populations with non-zero expression. The cell-types and the literature-derived local and global associations scores are shown.author/funder. All rights reserved. No reuse allowed without permission.', 'latex': None, 'type': 'figure'}, 'FIGREF4': {'text': 'Triangulation of ACE2 expression in the gastrointestinal (GI) tract with literaturederived insights. (A) Schematic representation of the GI tract highlighting key cell types. Scatterplot shows comparison of percentage of cells with non-zero expression (x-axis) from nine single cell studies against literature-derived associations: local score (y-axis and size of circles) and global score (transparency of circles). (B) Assessing literature-based and scRNAseq-based association between ACE2 and tongue keratinocytes. Violin plot shows distributions of ACE2 expression in keratinocytes. The literature-derived local and global associations between ACE2 and keratinocytes are shown. (C) ACE2 transcriptional expression is correlated to enterocyte maturation. Violin plots show distribution of ACE2 expression levels in enterocytes at different stages of differentiation.', 'latex': None, 'type': 'figure'}, 'FIGREF5': {'text': \"Coronavirus receptors share a transcriptional signature correlated to maturation of small intestinal enterocytes. (A) Distribution of cosine distances between the 'gene expression vectors' of ACE2 and all genes in a scRNA-seq study of the murine small intestine. The gene expression vector corresponds to the set of CP10K values for a given gene in each individual cell from the selected populations in the selected study. (B) Genes similar to ACE2 (cosine similarity > 0.4) sorted by literature-derivation association. (C) Transcriptional expression of ANPEP correlated to enterocyte maturation in murine small intestine. Violin plots show distribution of ANPEP expression levels in enterocytes at different stages of differentiation. (D) Transcriptional expression of DPP4 correlated to enterocyte maturation in murine small intestine. Violin plots show distribution of DPP4 expression levels in enterocytes at different stages of differentiation.\", 'latex': None, 'type': 'figure'}, 'FIGREF6': {'text': 'Assessing literature-based and scRNAseq-based association between ACE2 and respiratory tract', 'latex': None, 'type': 'figure'}, 'FIGREF7': {'text': 'ACE2 expression in tongue keratinocytesTriangulation of ACE2 expression in GI tract from literature-based knowledge synthesis and nine single cell studies e.g. Single-cell transcriptomics of 20 mouse organs creates a Tabula Muris.', 'latex': None, 'type': 'figure'}, 'TABREF0': {'text': \"author/funder. All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.24.005702 doi: bioRxiv preprint Expression profiling of ACE2 Putative receptor of SARS-CoV-2Knowledge synthesis: capturing association between concepts from over 100 million documents 100 million+ documents e.g., PubMed, Grants, Preprints, Patents, Clinical TrialsGlobal score measures the similarity between the word vectors corresponding to two tokens, normalized by a control token collection. Word2Vec at scale.Local score measures how frequently two tokens are found within each other's local context in a particular corpus, normalized by the occurrences of those tokens in that corpus. Derivative of point-wise mutual information.Types of literature-based associations\", 'latex': None, 'type': 'table'}}, 'back_matter': [{'text': 'The authors thank Murali Aravamudan, Peter Lebowitz, Ajit Rajasekharan, and Mathai Mammen for their insightful reviews and feedback on our research. We also express our gratitude to Patrick Lenehan, Saurav Kumar Verma, Travis Hughes, and Vishy Thiagarajan who helped develop some of the scientific tools that were leveraged for this study.', 'cite_spans': [], 'ref_spans': [], 'section': 'Acknowledgments'}]}\n",
      "{'paper_id': 'b378a19034ecc758062e01cad504b9df12b07fb6', 'metadata': {'title': 'Basic prediction methodology for covid-19: estimation and sensitivity considerations', 'authors': [{'first': 'Tom', 'middle': [], 'last': 'Britton', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Stockholm University', 'location': {}}, 'email': 'tom.britton@math.su.se'}]}, 'abstract': [{'text': 'The purpose of the present paper is to present simple estimation and prediction methods for basic quantities in an emerging epidemic like the ongoing covid-10 pandemic. The simple methods have the advantage that relations between basic quantities become more transparent, thus shedding light to which quantities have biggest impact on predictions, with the additional conclusion that uncertainties in these quantities carry over to high uncertainty also in predictions.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'A simple non-parametric prediction method for future cumulative case fatalities, as well as future cumulative incidence of infections (assuming a given infection fatality risk f ), is presented. The method uses cumulative reported case fatalities up to present time as input data. It is also described how the introduction of preventive measures of a given magnitude ρ will affect the two incidence predictions, using basic theory of epidemic models. This methodology is then reversed, thus enabling estimation of the preventive magnitude ρ, and of the resulting effective reproduction number R E . However, the effects of preventive measures only start affecting case fatalities some 3-4 weeks later, so estimates are only available after this time has elapsed. The methodology is applicable in the early stage of an outbreak, before, say, 10% of the community have been infected.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'Beside giving simple estimation and prediction tools for an ongoing epidemic, another important conclusion lies in the observation that the two quantities f (infection fatality risk) and ρ (the magnitude of preventive measures) have very big impact on predictions. Further, both of these quantities currently have very high uncertainty: current estimates of f lie in the range 0.2% up to 2% ([9], [7]), and the overall effect of several combined preventive measures is clearly very uncertain.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}, {'text': 'The two main findings from the paper are hence that, a) any prediction containing f , and/or some preventive measures, contain a large amount of uncertainty (which is usually not acknowledged well enough), and b) obtaining more accurate estimates of in particular f , should be highly prioritized. Seroprevalence testing of random samples in a community where the epidemic has ended are urgently needed.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}], 'body_text': [{'text': 'Many papers use advanced models combined with extensive simulations to analyse and predict the outcome of the ongoing covid-19 pandemic (e.g. [3] , [5] [6] ). Here we complement this important literature with much simpler methods. Most likely this has the effect that estimation and prediction is less accurate than more advanced well-motivated methods. On the other hand, the methods presented here are much simpler and hence more transparent, thus allowing any reader to apply the methods to quickly obtain ball-park estimates for the most important parameters/quantities. For a more thourough statistical investigation we propose that the simple analysis presented here should be complemented with more realistic advanced models.', 'cite_spans': [{'start': 142, 'end': 145, 'text': '[3]', 'ref_id': None}, {'start': 148, 'end': 151, 'text': '[5]', 'ref_id': None}, {'start': 152, 'end': 155, 'text': '[6]', 'ref_id': None}], 'ref_spans': [], 'section': 'Methods'}, {'text': 'Suppose the data at hand is the cumulative deaths Λ D (t), starting some calendar time t 0 and ending at t 1 at which still only a small community fraction has been infected. As a rule of thumb we define \"early stage\" by meaning that no more than 10% of the community have been infected. Mathematical theory, as well as empirical evidence for many infectious diseases, suggest that Λ D (t) grows approximately according to an exponential rate meaning that Λ D (t) ≈ const * e rt for some rate factor r. The exponential rate factor r relates to the more commonly used and easy-to-estimate doubling-time d by the relation r = ln(2)/d.', 'cite_spans': [], 'ref_spans': [], 'section': 'Early stage predicting case fatalities and number of infected'}, {'text': 'The growth of Λ D (t) will continue in the same manner until either some intervention is put in place, or until a significant community fraction have been infected. Here we consider predictions up until, say, 10% have become infected. Before this population immunity is very marginal. As a consequence, unless additional preventive measures are put in place, a very natural prediction of the cumulative number of deaths some date t after the final observation point t 1 is given bŷ', 'cite_spans': [], 'ref_spans': [], 'section': 'Early stage predicting case fatalities and number of infected'}, {'text': 'A very important quantity is how the number of infected people (n.b. not only reported!) increases over time, because this quantity carries information about how immunty builds up and when the growth rate will start declining due to partial population immunity. We hence let Λ I (t) denote the total number of infected individuals up until time t. This cumulative number is never observed. What can be observed is the cumulative number of reported cases, but since this number is very sensitive to testing procedures which may vary over time, we refrain from using this data and stick to the more reliable reported deaths Λ D . If the infection fatality risk f is known, and if the typical time s d between getting infected and dying (for those who die) is known (or estimated) from some other source of information, then it is straight forward to estimate Λ I (t) by simply back-tracking the number of \"to-die\" infected s d days earlier and upscaling by the factor f :', 'cite_spans': [], 'ref_spans': [], 'section': 'Early stage predicting case fatalities and number of infected'}, {'text': '(Note that t, t 0 and t 1 denote calendar time whereas s d is days, so if e.g. t = March 4 and s d = 21, then t + s d = March 25.) It is also possible to predict the cumulative number of 2 . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Early stage predicting case fatalities and number of infected'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint .', 'cite_spans': [], 'ref_spans': [], 'section': 'Early stage predicting case fatalities and number of infected'}, {'text': 'infected later than t 1 − s d by using the same exponential growth as before:', 'cite_spans': [], 'ref_spans': [], 'section': 'Early stage predicting case fatalities and number of infected'}, {'text': 'These predictions work well up until around 10% of the community have been infected, i.e. as long asΛ I (t)/N ≤ 0.1, where N is the size of the population from which death-reports are collected.', 'cite_spans': [], 'ref_spans': [], 'section': 'Early stage predicting case fatalities and number of infected'}, {'text': 'We end this subsetion by pointing out that predictions of the number of infected are very sensitive to the infection fatality risk f . Any error in f carry over to prediction error multiplicatively.', 'cite_spans': [], 'ref_spans': [], 'section': 'Early stage predicting case fatalities and number of infected'}, {'text': 'Suppose that a new set of preventive measures are initiated on date t p , for example general social distancing, symptomatic people isolating, closing schools, and so on. Suppose further, that the overall effect of the preventive measures are such that the overall rate of contacts from infectious individuals to others is reduced by a factor ρ ∈ [0, 1].', 'cite_spans': [], 'ref_spans': [], 'section': 'Introducing preventive measures during early stage'}, {'text': 'The effect of this will be that the original basic reproduction number R 0 is reduced by a factor ρ such that the new, effective reproduction number R E is given by R E = (1 − ρ)R 0 (e.g. [4] ). In order to induce how the exponential growth is affected some results from epidemic theory are needed. The exponential growth r is a function of the basic reproduction number R 0 and the generation time G, where the latter is defined as the random variable describing the typical time between getting infected and infecting others. The complete relation is described by the Euler-Lotka equation (e.g. [2] ) but for a wide class of epidemic models the exponential growth r will lie between the two relations given by', 'cite_spans': [{'start': 188, 'end': 191, 'text': '[4]', 'ref_id': None}, {'start': 597, 'end': 600, 'text': '[2]', 'ref_id': None}], 'ref_spans': [], 'section': 'Introducing preventive measures during early stage'}, {'text': 'where g = E(G) denotes the mean generation time [8] . Relation (1) comes from the stochastic general epidemic model which has a highly variable generation time, whereas relation (2) comes from the Reed-Frost epidemic model for which all infections happen exactly g times units after getting infected, so no variation at all [2] .', 'cite_spans': [{'start': 48, 'end': 51, 'text': '[8]', 'ref_id': None}, {'start': 324, 'end': 327, 'text': '[2]', 'ref_id': None}], 'ref_spans': [], 'section': 'Introducing preventive measures during early stage'}, {'text': 'We now describe how to derive an expression, or rather a range of r E -values, to which the preventive measures will change the exponential growth to. We start by deriving an expression using relation (1) . First of all, the growth rate r prior to preventive measures, is estimated from the cumulative reported deaths as described above. Suppose further that an estimate of R 0 is available (from contact tracing, prior knowledge, or other method).', 'cite_spans': [{'start': 201, 'end': 204, 'text': '(1)', 'ref_id': None}], 'ref_spans': [], 'section': 'Introducing preventive measures during early stage'}, {'text': 'Using the estimates of r and R 0 we obtain an estimate g 1 of g using (1). If preventive measures are introduced, then the reproduction number changes from R 0 to R E = (1 − ρ)R 0 . The mean generation time remains unchanged (this might not hold if enforced 3 . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Introducing preventive measures during early stage'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.27.20045575 doi: medRxiv preprint contact tracing is included among the introduced preventive measures). As a consequence, the new exponential growth rate satisfies', 'cite_spans': [], 'ref_spans': [], 'section': 'Introducing preventive measures during early stage'}, {'text': 'If exactly the same procedure is performed, but using relation (2) we get', 'cite_spans': [], 'ref_spans': [], 'section': 'Introducing preventive measures during early stage'}, {'text': 'The latter expression is bigger than the former, so a natural bound for the new exponential rate r E , once preventive measures are put in place, is given by r', 'cite_spans': [], 'ref_spans': [], 'section': 'Introducing preventive measures during early stage'}, {'text': 'E . It is clear that the effect on the exponential growth rate depends to a great extent on the magnitude of the preventive effects ρ. It seems in general very hard to specify what overall effect ρ a given set of preventive measure has on reducing the rate of spread. It is hence recommended to consider a range of values rather than a fixed hypothesized value.', 'cite_spans': [], 'ref_spans': [], 'section': 'Introducing preventive measures during early stage'}, {'text': 'The procedure described above can be reversed, thus enabling estimation of the preventive effect ρ as follows. Suppose that the exponential growth was observed to equal r prior to preventive measures and that, once preventive measure have been put in place the growth rate declined to r E . If measured by reduced cumulative death rates we emphasize that this can only be estimated s d days after the measure has been put in place (or preferably a bit later). From this it is possible to estimate ρ by simply inverting the procedure. As before we assume that an estimate of R 0 is also available and as before we obtain two bounds using relation (1) and (2). The resulting bounds are', 'cite_spans': [], 'ref_spans': [], 'section': 'Introducing preventive measures during early stage'}, {'text': 'Here too, the latter expression is bigger, so a natural bound for the overall effect ρ is given by ρ 1 ≤ ρ ≤ ρ 2 . Here we have implicitly assumed that r E ≥ 0; if r E < 0 then the epidemic has already started to decline. In principle the equations may still be applied (then ρ 2 < ρ 1 ) but precision quickly drops when number of newly infected is on decline.', 'cite_spans': [], 'ref_spans': [], 'section': 'Introducing preventive measures during early stage'}, {'text': 'Predicting the epidemic during the main phase when most infections happen, so after the early stage has ended, is less straighforward and typically require a specific underlying epidemic model. To describe how this is done does not fit into the current paper relying mainly on non-parametric methods and robust basic results from epidemic theory.', 'cite_spans': [], 'ref_spans': [], 'section': 'Prediction during the main phase of the epidemic'}, {'text': 'For this reason we refrain from predicting the progress of the epidemic during this main phase but only give the rough picture. Once the early stage of the epidemic has passed the exponential growth rate r (or r E in case preventive measures have been introduced) starts to decline. This goes on up until the time t at which the cumulative fraction infected Λ I (t) exceeds 1 − 1/R 0 (or R E in case of preventive measures). This is the time when the expected number of infections no longer exceeds one due to reduced community 4 . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Prediction during the main phase of the epidemic'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint .', 'cite_spans': [], 'ref_spans': [], 'section': 'Prediction during the main phase of the epidemic'}, {'text': 'susceptibility. After this time the number of new infections starts dropping with time thus eventuelly leading the epidemic to a halt. The effect of introducing preventive measures reduces the initial growth rate as described in earlier sections. It also delays the timing of the peak, and particular the height of the peak, closely related the maximal healthcare burdon. The amount of peak-reduction is model dependent, but for a simple model, reducing R 0 = 3 to R E = 2 reduces the height of the peak by around 50%.', 'cite_spans': [], 'ref_spans': [], 'section': 'Prediction during the main phase of the epidemic'}, {'text': 'We end the methodology section be given predictions of the total fraction getting infected, and perhaps even more important, the expected number of case fatalities. We assume that the same set of preventive measures (if any) remain constant through main part of the epidemic. The results are not valid if the they were put in place after a few percent had been infected, nor if they were varied during the main part of the outbreak (the prediction problem is then much more complicated and better suited for simulation studies). Suppose hence that reproduction number equals R (equal to R 0 if no preventive measures are put in place, or otherwise R E ). The final fraction τ getting infected is known to solve the equation 1 − τ = e −R 0 τ [4] , a solution which has to be obtained numerically. In case the community is heterogeneous in one or several ways, the final fraction getting infected is usually 5-20% smaller than this solution. So, given that the reproduction number is known the final fraction getting infected is well-predicted. The estimate is of course sensitive to R, but if R ≥ 2 then τ ≥ 75%, so it is not too sensitive to exakt values of R in case it is above 2 or so.', 'cite_spans': [{'start': 741, 'end': 744, 'text': '[4]', 'ref_id': None}], 'ref_spans': [], 'section': 'Predicting final epidemic size and total number of case fatalities'}, {'text': 'If the infection fatality risk f is known, then the predicted number of case fatalities is N * f * τ , where as before N denotes the population size. This estimate is very sensitive to f . Since f is typically very small, possible values may well vary by a factor 10, implying that predictions of case fatalities will also vary by a factor 10.', 'cite_spans': [], 'ref_spans': [], 'section': 'Predicting final epidemic size and total number of case fatalities'}, {'text': 'We now illustrate our simple methodology to realistic covid-19 settings. We start be making predictions assuming no preventive measures whatsoever. In most European countries the doubling time prior to prevntive measures lie very close to d = 3 days. A common estimate of R 0 equals 2.5, but estimates seem to go up so we use R 0 = 3 in our illustration. As our last cumulative death number we pick t 1 = March 24 and assume that the country of interest (with N = 30million) then had experienced 50 case fatalities (Λ D (t 1 ) = 50). A prediction for the cumulative number of deaths by t = April 3 ( 10 days later) hence equalsΛ I (t) = Λ D (t 1 )2 (t−t 1 )/d = 50 * 2 10/3 = 500. We assume that s d , the typical time from getting infected to dying (assuming the individual dies from covid-19), equals 21 days, and that the infection fatality risk equals f = 0.3%. With these assumptions, the estimate of how many people that had been infected by t = March 3 (3 weeks earlier) equalsΛ I (t − s d ) = Λ D (t)/f = 16 700. If we want to estimate how infected there are at t 1 = March 24, this number has to scaled up using the doubling times: Λ I (t 1 ) = Λ D (t 1 ) * 2 s d /3 /f = 2.1 million, thus approaching the limit of 10% where predictions should stop. Nedless to say, there is of course quite a lot of uncertainty in such 5 . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Illustrations'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint .', 'cite_spans': [], 'ref_spans': [], 'section': 'Illustrations'}, {'text': 'a prediction: first we estimated the number of infected 3 weeks back using case fatalities of present time and then projected this number three weeks forward in time to present using the doubling times.', 'cite_spans': [], 'ref_spans': [], 'section': 'Illustrations'}, {'text': 'If R 0 = 3 and the doubling time equals d = 3 corresponding to an exponential growth rate of r = ln(2)/3 = 0.23. If the magnitude of the overall effect of preventive measures equals ρ = 1/3, then using relation (1) gives a mean generation time g 1 = (R 0 − 1)/r = 8.7 days. The new exponential growth rate after preventive measure are put in place hence equals r (1) E = (R E − 1)/g 1 = 0.11. If instead relation (2) is used, the mean generation time is g 2 = ln(R 0 )/r = 4.5 days. As a consequence, the new exponential growth rate assuming relation (2) equals r (2) E = ln(R E )/g 2 = 0.14. From the two bounds we hence conclude that r E should satisfy 0.11 ≤ r E ≤ 0.14.', 'cite_spans': [], 'ref_spans': [], 'section': 'Illustrations'}, {'text': 'We now include preventive measures. We still assume R 0 = 3 and a doubling time of d = 3 days without preventions. Then preventive measures are put in place, resulting in a prolonged doubling time of case fatalities starting (around) s d = 21 days later. Suppose that the new observed doubling time equals d E = 5 days. These two doubling times correspond to the exponential rates r = 0.23 and r E = ln(2)/5 = 0.14. If we first assume relation (1) to hold, this reduced exponential rate gives a lower bound the magnitude ρ given by ρ 1 = (r − r E )g 1 /R 0 = 0.27. If we instead consider relation (2) we obtain the upper bound of the magnitude as ρ 2 = 1 − e (r−r E )g 2 = 0.35. It hence follows that the magnitude ρ of the preventive measure giving rise to an increased doubling time from d = 3 to d E = 5 days is estimated to lie somewhere in the interval (0.27, 0.35). The effective reproduction R E is hence estimated to lie between R 0 (1 − 0.35) = 1.93 and R 0 (1 − 0.27) = 2.19, so a substantial reduction.', 'cite_spans': [], 'ref_spans': [], 'section': 'Illustrations'}, {'text': 'We end by predicting the final fraction infected, and the total number of case fatalities, at the end of the outbreak. This is possible from simply knowing the reproduction number, as described in the method section. For instance, if R 0 = 3 the final size equation gives τ = 94%. If preventive measures with magnitude ρ = 1/3 are put in place during the early stage and kept all trough the outbreak, then the predicted final size instead equals τ E = 78% In the more likely case of a heterogeneous community the fractions should be deflated by about 20%. Similarly, the predicted total number of case fatalities is N f τ = and N f τ E respectively. Since communities typically are heterogeneous better estimates would be to deflate these numbers by about 20%.', 'cite_spans': [], 'ref_spans': [], 'section': 'Illustrations'}, {'text': 'One of our main conclusions is that any prediction of the number of infected, using data from cumulative case fatalities, rely heavily on accurate knowledge of the infection fatality risk f . Without accurate knowledge of f , the prediction of the total case fatality once the epidemic outbreak is over, is equipped with a similar multiplicative uncertainty. It is hence of utmost importance to obtain more accurate estimates of f . The best way to obtain such estimates, now that seroprevalence tests are becoming available, is to conduct tests on a random sample of a community in which the outbreak is (close to) over. f is then simply estimated as the ratio of the total number of case fatalities and the estimated population number of people testing positive.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusions'}, {'text': '. CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '6'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint .', 'cite_spans': [], 'ref_spans': [], 'section': '6'}, {'text': 'Further, combinations of preventive measures typically have a very uncertain overall magnitude ρ, and it was shown how this uncertainty carries over to high uncertainty in estimates of the effective reproduction number R E and the corresponding decline in growth rate r E . However, it was also illustrated how to estimate ρ if cumulative deaths are collected a bit longer than s d days ahead before estimating.', 'cite_spans': [], 'ref_spans': [], 'section': '6'}, {'text': 'As mentioned earlier, the main idea with the present paper is to keep methods simple enough to make the methodology transparent. We highly recommend to complement this simple analysis with more detailed models which might give better precision (at the price of not being transparent).', 'cite_spans': [], 'ref_spans': [], 'section': '6'}, {'text': 'The methodology presented used cumulative reported deaths as the main data source. A problem with this data is that there is a delay of about 3 weeks from getting infected to dying (for those who die). As a consequence, there is a more than 3 week delay before effects of interventions may be estimated. The same methodology can be used for data on cumulative number of patients having received intensive care (IC) treatment. The time from getting infected to the start of IC treatment (for those who need it) is around 2 weeks, thus making the delay shorter. This must be weighted against the often lower quality data for IC treatment as compared to reported deaths.', 'cite_spans': [], 'ref_spans': [], 'section': '6'}, {'text': 'Finally we stress that in the current paper we have not at all treated statistical uncertainty of the estimators and predictions. Deriving standard errors for these quantities is far from easy, and are often also model-dependent. Such methodology is hence beyong the scope of the present paper where we focus on simple methodology. We also believe the uncertainty in f and ρ is orders of magnitude larger than statistical uncertainty.', 'cite_spans': [], 'ref_spans': [], 'section': '6'}], 'bib_entries': {'BIBREF8': {'ref_id': 'b8', 'title': 'WHO Covid-19 Situation report 30', 'authors': [], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}}, 'ref_entries': {}, 'back_matter': []}\n",
      "{'paper_id': 'b3a334560f9a7a4b35a60415c87d41d658a89ebb', 'metadata': {'title': 'A deductive approach to modeling the spread of COVID-19', 'authors': [{'first': 'Pranav', 'middle': [], 'last': 'Mishra', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Manipal University', 'location': {'settlement': 'Manipal, Naperville', 'region': 'KA, IL', 'country': 'India, USA'}}, 'email': ''}, {'first': 'Shekhar', 'middle': [], 'last': 'Mishra', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Manipal University', 'location': {'settlement': 'Manipal, Naperville', 'region': 'KA, IL', 'country': 'India, USA'}}, 'email': ''}]}, 'abstract': [{'text': 'Severe acute respiratory syndrome coronavirus 2 (SARS-COV-2), previously known as 2019-nCoV, is responsible for the atypical pneumonia pandemic designated as Coronavirus Disease 2019 . The number of cases continues to grow exponentially reaching 492,000 people in 175 countries as of March 25, 2020. 22,169 people (~4.5%) infected with SARS-COV-2 virus have died. We have developed an exponential regression model using the COVID-19 case data (Jan 22 -Mar 22, 2020). Our primary model uses designated Phase 1 countries, who exceed 2500 cases on Mar 22. The model is then applied to Phase 2 countries: those that escaped the initial Phase 1 global expansion of COVID-19. With the exception of stabilizing countries (South Korea, Japan, and Iran) all Phase 1 countries are growing exponentially, as per 2500 ( ) = 120.4 × 0.238 , with a rate, r = 0.238 ± 0.068. Excluding China, the BRICS developing nations and Australia are in Phase 2. Case data from Phase 2 countries are following the model derived from Phase 1 countries. In the absence of measures employed to flatten the curve including social distancing, quarantine, and healthcare expansion, our model projects over 274,000 cases and 12,300 deaths in the US by Mar 31. India can expect 123,000 cases by April 16. By flattening the curve to the growth rate of stabilizing countries (r = 0.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}], 'body_text': [{'text': 'Severe acute respiratory syndrome coronavirus 2 (SARS-COV-2), previously known as 2019-nCoV, is a novel virus in the coronaviridae family of positive sense, enveloped, RNA viruses 1 . It is responsible for the atypical pneumonia pandemic designated as Coronavirus Disease 2019 , by the World Health Organization. First identified in Wuhan, China in December 2019, COVID-19 has spread to 175 countries/territories, with over 492,000 cases as of March 25, 2020 2,3 . During this period, there has been a stepwise escalation of international, national, and regional governmental responses. The Wuhan Government confirmed that local hospitals were treating 27 cases of viral pneumonia on December 31, 2019 4 . However, approximately three weeks later, on January 20, the United States confirmed its first case 5 . Due to the rapid expansion of cases across the world, the World Health Organization (WHO) designated COVID-19 as Public Health Emergency of International Concern on January 30, 2020 6 . On the same day, India confirmed its first case in the state of Kerala 7 .', 'cite_spans': [{'start': 806, 'end': 807, 'text': '5', 'ref_id': 'BIBREF3'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'The United States Centers for Disease Control and Prevention (CDC) issuing a travel alert to the Wuhan region on January 6, 2020 8 . Various countries systematically blocked international travel from known COVID-19 hot-spots, including China, Iran, and the European Union. At the time of writing, nearly 80 nations have a global travel ban with an additional 9 implementing global quarantine measures 9 . These efforts hope to prevent local seeding of infection, each of which can rapidly increase case numbers. When examining smaller confined communities exposed to SARS-COV-2, the disease spreads unabated. Initial studies examined high-density exposed cohorts, including cruise ships, prisons, nursing homes, and healthcare workers [10] [11] [12] [13] . Therefore, it appears that containment of COVID-19 primarily be mitigated by preventing local seeding through travel restriction and reducing the population density around potential infected person.', 'cite_spans': [{'start': 129, 'end': 130, 'text': '8', 'ref_id': 'BIBREF6'}, {'start': 401, 'end': 402, 'text': '9', 'ref_id': 'BIBREF7'}, {'start': 735, 'end': 739, 'text': '[10]', 'ref_id': 'BIBREF8'}, {'start': 740, 'end': 744, 'text': '[11]', 'ref_id': 'BIBREF9'}, {'start': 745, 'end': 749, 'text': '[12]', 'ref_id': 'BIBREF10'}, {'start': 750, 'end': 754, 'text': '[13]', 'ref_id': 'BIBREF11'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'A wide array of models exists for the predicting the spread of infectious disease across various cohorts. These models may be stochastic in nature, creating probability distributions by moving about random variables, or deterministic in nature, compartmentalizing the population across various groups. The Susceptible Exposed Infectious Recovered (SEIR) model is a widely utilized mathematical model in the COVID-19 outbreak 14 . Liu et al reviewed the models of 12 studies which calculated the basic reproductive number (R0) of the SARS-COV-2 virus 15 . The studies examined employed various techniques of subdividing the population and utilizing historical data of related diseases (i.e. SARS, MERS) to estimate the potential impact of COVID-19. While these models help simulate the possible course of a disease given population and disease characteristics, each model becomes outdated as various governments implement restrictions on its population. Infectious disease dynamics are classically considered as exponential in nature. However, Maier and Brockmann demonstrate that COVID-19 is affected by \"fundamental mechanisms that are not captured by standard epidemiological models\" 16 . This is notably true in early phases of disease expansion. Consequentially, COVID-19\\'s behavior warrants the examination of models outside of inductive logic.', 'cite_spans': [{'start': 550, 'end': 552, 'text': '15', 'ref_id': 'BIBREF13'}, {'start': 1186, 'end': 1188, 'text': '16', 'ref_id': 'BIBREF14'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': \"The authors of this paper seek to understand the expansion of COVID-19 through a deductive examination of existing case data. Our study examines this pandemic through data provided by the Johns Hopkins University's Systems Science and Engineering Group (JHU CSSE) 17 . Our approach is inherently simplistic as it examines COVID-19 case numbers first. This deductive 'outside-in' observational approach allows us to derive models without consideration of ever-changing population dynamics reflective of health policy measures being implemented. We seek to apply our model to nations which have avoided the initial expansion of COVID-19. In this study, we model the pandemic's impact on the United States and BRICS developing nations. India's 657 cases, as of March 25, represent ample seeding, priming the nation for a devastating expansion of cases and fatalities.\", 'cite_spans': [{'start': 264, 'end': 266, 'text': '17', 'ref_id': 'BIBREF15'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': \"The pandemic case data is provided by JSU CSSE's GitHub repository on COVID-19. It is an aggregation of case data starting on January 22, 2020, from several sources 17 . Cases in the data set include laboratory confirmed cases and presumptive cases of COVID-19. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Raw Data -COVID-19 Case Numbers'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint .', 'cite_spans': [], 'ref_spans': [], 'section': 'Raw Data -COVID-19 Case Numbers'}, {'text': 'We include a series of countries into our data set for comparative purposes. We designate Phase 1 countries as those who exceed 2500 cases of COVID-19 as of March 23, 2020. These countries, which suffered the initial global expansion of SARS-COV-2, provide a working example for countries who recently acquired the virus. We also include members of the Group of 7 (G7) to examine differences between developed and Phase 1 countries. We include BRICS countries as an example of large, developing countries likely to become part of Phase 2. The Phase 2 countries are hypothesized to acquire SARS-COV-2 either through direct seeding from the epicenter, China, or via secondary seeding from Phase 1 countries. We include Australia as a nation of interest, due to its developed status, geographic proximity to China, and relative escape from Phase 1.', 'cite_spans': [], 'ref_spans': [], 'section': 'Raw Data -COVID-19 Case Numbers'}, {'text': 'In the early expansion of COVID-19 within each country, cases primarily expanded from international seeding from China, rather than internal human-to-human spread. The sporadic nature of these cases spreading around the world can be visualized through notable sub-exponential, erratic growth patterns, within a country. After a country reaches its 100 th case, the disease load increases exponentially ( Figure  1) . The time it takes to exceed 100 cases also demonstrates substantial variation (Figure 2 ). . CC-BY 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [{'start': 404, 'end': 414, 'text': 'Figure  1)', 'ref_id': None}, {'start': 495, 'end': 504, 'text': '(Figure 2', 'ref_id': None}], 'section': 'Excluding Data Prior to the 100 th Case Per Country'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint .', 'cite_spans': [], 'ref_spans': [], 'section': 'Excluding Data Prior to the 100 th Case Per Country'}, {'text': 'Of the countries already exceeding 2500 cases, the mean time to exceed 100 cases is 40 days, with standard deviation 6.3 days, calculated from Jan 22, 2020. The high variance in data supports excluding the first 100 cases from the modeling equations. ', 'cite_spans': [], 'ref_spans': [], 'section': 'Figure 2: Days to exceed 100 cases of COVID-19 from January 22, 2020. The JHU CSSE data set starts on January 22, 2020. From this date, we count the number of days each country takes to exceed 100 COVID-19 cases. The initial expansion is primarily from international seeding, rather than internal human-to-human spread. Countries which reached 100 cases likely have greater international travel with China. For the countries shown, the number of days to reach 100 cases has a mean of 40 days, with standard deviation 6.4 days.'}, {'text': 'We exclude four countries: China, Iran, Japan, and South Korea. China is excluded due to notably unreliable data and changed its methodology. Numerous reports exist demonstrating the suppression of case data by Chinese authorities [18] [19] [20] . Iran, Japan, and South Korea are excluded as outliers, which approached the stationary phase of COVID-19 early in the timeline (Figure 3) .', 'cite_spans': [{'start': 231, 'end': 235, 'text': '[18]', 'ref_id': 'BIBREF16'}, {'start': 236, 'end': 240, 'text': '[19]', 'ref_id': 'BIBREF17'}, {'start': 241, 'end': 245, 'text': '[20]', 'ref_id': 'BIBREF18'}], 'ref_spans': [{'start': 375, 'end': 385, 'text': '(Figure 3)', 'ref_id': 'FIGREF3'}], 'section': 'Exclusion Criteria for the Predictive Model'}, {'text': '. CC-BY 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Exclusion Criteria for the Predictive Model'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.26.20044651 doi: medRxiv preprint ', 'cite_spans': [], 'ref_spans': [], 'section': 'Exclusion Criteria for the Predictive Model'}, {'text': 'Eleven countries pass the inclusion and exclusion criteria. For each of them, an exponential regression is applied, as per the function:', 'cite_spans': [], 'ref_spans': [], 'section': 'Creating a Predictive Model'}, {'text': 'where I is the number of infected cases at a time t in days, N is the initial case load, r is the rate of growth. The results are reported in Tables 1-3Table 1. We then average the values for N and r, producing:', 'cite_spans': [], 'ref_spans': [], 'section': 'Creating a Predictive Model'}, {'text': '7 ( ) = 122 × 0.245', 'cite_spans': [], 'ref_spans': [], 'section': 'Model Equation 1: Countries Exceeding 2500 cases of COVID-19'}, {'text': 'Japan is excluded from the model due as it meets the exclusion criteria. Figure 4 presents our primary model, I2500, with 95% confidence intervals and case data of Phase 1 countries. We aligned the data from each Phase 1 country, such that t = 1 day when the number of cases is closest to, but greater than 100. The 95% confidence intervals continue to contain the trajectories of nearly all Phase 1 countries beyond Mar 22.', 'cite_spans': [], 'ref_spans': [{'start': 73, 'end': 81, 'text': 'Figure 4', 'ref_id': None}], 'section': 'Model Equation 2: Group of 7 (G7) Countries.'}, {'text': '. CC-BY 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Model Equation 2: Group of 7 (G7) Countries.'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint ', 'cite_spans': [], 'ref_spans': [], 'section': 'Model Equation 2: Group of 7 (G7) Countries.'}, {'text': \"We have excluded the SARS-COV-2 source nation of China from the BRICS countries for future modeling. We hypothesize that these major developing countries escaped the first phase of global COVID-19 expansion due to reduced seeding. With a smaller amount of air travel between China and the other BRICS countries, the time to reach the 100 th case will fall later in the pandemic's timeline. Next, we fit the equation, 2500 ( ) = 120.4 × 0.238 , to each country ( Figure 5) . For t=1 day, the model projects 152 cases. Thus, we start the model for each country on the date with number of COVID-19 cases closest to 152.\", 'cite_spans': [], 'ref_spans': [{'start': 462, 'end': 471, 'text': 'Figure 5)', 'ref_id': None}], 'section': 'Applying the Model to Developing Nations'}, {'text': 'The actual number of infected persons will be affected by statistical uncertainties. We use our fit and standard deviation in the fitted parameter to estimate the range of growth at ± 1σ and 2σ levels for India. We expect all Phase 2 countries to have a similar band of cases (Figure 6 ).', 'cite_spans': [], 'ref_spans': [{'start': 276, 'end': 285, 'text': '(Figure 6', 'ref_id': None}], 'section': 'Applying the Model to Developing Nations'}, {'text': '. CC-BY 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. ', 'cite_spans': [], 'ref_spans': [], 'section': 'Applying the Model to Developing Nations'}, {'text': 'India (Projected) +2s +1s -2s -1s', 'cite_spans': [], 'ref_spans': [], 'section': 'Projection of COVID-19 Expansion in India'}, {'text': \"The mean rate (r) of expansion of Phase 1 countries is 0.238, with standard deviation (σ) 0.0340. We plot 1σ and 2σ variations about the function 2500 ( ) = 120.4 × 0.238 . There is a 95% probability that India's case rate will fall between the outer two lines, assuming it COVID-19 expands at the same rate as Phase 1 countries. Existing case data is plotted with red markers.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Figure 6: Projection of COVID-19 Expansion in India.'}, {'text': 'For theoretical consideration, we examine what would happen if a country could instantaneously shift from the Phase 1 growth rate to that of the stabilizing countries, starting March 26, 2020. The estimated case fatality rate (CFR) of the COVID-19 is 4.51%, based on arithmetic division of fatalities to known cases 17 . We can then estimate the number of preventable deaths when if a country instantly stabilized its growth to that of South Korea, Japan, and Iran.', 'cite_spans': [{'start': 316, 'end': 318, 'text': '17', 'ref_id': 'BIBREF15'}], 'ref_spans': [], 'section': 'Modeling the Impact of Stabilizing Expansion'}, {'text': '. CC-BY 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Model Equation 5: Preventable deaths with Phase 1 growth. PD is preventable deaths. The case fatality ratio of COVID-19 (CFR) is 4.51%. It is multiplied into the difference of cases between Phase 1 growth and that of stabilizing countries.'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint .', 'cite_spans': [], 'ref_spans': [], 'section': 'Model Equation 5: Preventable deaths with Phase 1 growth. PD is preventable deaths. The case fatality ratio of COVID-19 (CFR) is 4.51%. It is multiplied into the difference of cases between Phase 1 growth and that of stabilizing countries.'}, {'text': \"Through our examination of COVID-19's expansion around the world, we draw several conclusions on the intrinsic nature of SARS-COV-2 and the impact of human intervention to curtail its spread. First, we note that most countries affected in the Phase 1 growth of disease follow exponential growth curves with comparable rates. Norway exhibits the minimum rate at 0.191, while the United States has the greatest at 0.295. The mean rate of expansion of Phase 1 countries is 0.238 (95% CI 0.170-0.306). For G7 countries, the mean rate of expansion is 0.245 (95% CI 0.184-0.306). Thus, we observe that G7 nations follow closely with other Phase 1 nations in COVID-19 cases.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'When compared with the three stabilized countries of Iran, Japan, and South Korea, we notice more troubling picture. The average rate of growth of stabilized countries is in excess of 5 times less than the rate of growth of Phase 1 and G7 countries. If the US follows I2500, we expect over 119,000 cases by Mar 31. More concerning, though, is that the US is growing one standard deviation above I2500. The cases based on ℎ 1 is over 274,000 by Mar 31. If it were possible to instantaneously change to the rate of growth calculated for stabilized countries, the expansion of COVID-19 would be limited to approximately 85,800. If we apply the CFR to ℎ 1 of the United States, we expect in excess of 12,300 fatalities resulting from COVID-19 by March 31. Applying the same CFR to , we expect about 3,800 deaths. If the United States implements the measures noted in South Korea, Japan, and Iran, approximately 8,500 of deaths could be prevented.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'The Phase 2 nations we examined, Australia plus the developing nations of Brazil, Russia, India, and South Africa, have a unique opportunity to compare the differences between the stabilized nations and expanding Phase 1 nations. India, the second most populous country in the world, has a population density over 3 times greater than that of China (454 vs 145 persons per square kilometer) 21 . This is far greater than the remaining Phase 2 nations, indicating the greatest potential for COVID-19 expansion. If India expands as per ℎ 1 , we expect approximately 123,000 cases, resulting in over 6,800 fatalities, by April 25. However, if India expands as per , we expect 1,745 cases. Given the high attack rate and rapid spread of COVID-19 in densely populated, confined areas such as cruise ships, prisons, and hospitals, I2500 may underestimate India if its dense urban population interacts normally. However, if India successfully executes its 21-day lockdown on its 1.3 billion people, we calculate approximately 5,500 lives saved by April 16.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': \"Our model may prove to be optimistic when considering the application of Phase 1 Country's largely developed healthcare systems with those of the Phase 2 developing countries. South Africa, Brazil, India, and have 2.8, 2.2, and 0.7 hospital beds per 1000 people, respectively. Though closer to Iran's 1.5 hospital beds per 1000 people, they are far below Japan and South Korea with 13.4 and 11.5, respectively 22 . Therefore, even with robust laboratory testing and quarantine measures, we must prepare for greater COVID-19 growth in developing countries. It is equally important to note that South Korea has a population density in excess of India (530 vs . CC-BY 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\", 'cite_spans': [{'start': 410, 'end': 412, 'text': '22', 'ref_id': 'BIBREF19'}], 'ref_spans': [], 'section': 'Discussion'}, {'text': \"is the (which was not peer-reviewed) The copyright holder for this preprint . 455 people per square kilometer) 21 . Japan also exceeds the remainder of the studied nations with 347 people per square kilometer. Therefore, a country's ability to manage COVID-19 is not primarily limited by population or population density.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Looking forward, we advise countries to rapidly implement policies to augment the spread of COVID-19. The exponential spread of disease is difficult to intuitively explain to the general population. In everyday life, we encounter arithmetic changes more often than exponential changes. Our instinct of examining relatively manageable changes to the case load to make policy decisions will quickly be eclipsed by the exponential expansion of this disease. Further complicating the matter is an estimation that 17.9% of COVID-19 patients are asymptomatic 23 . Looking at the data from the Diamond Princess cruise line, most of the cases occurred before or around the start of the quarantine. This reemphasizes the importance to make proactive, as opposed to reactive, policy decisions. It additionally demonstrates the importance of individual responsibility towards social distancing and personal hygiene. Should countries integrate the methods behind the success noted in South Korea, Japan, and Iran, thousands, perhaps millions, of deaths be prevented.', 'cite_spans': [{'start': 553, 'end': 555, 'text': '23', 'ref_id': 'BIBREF20'}], 'ref_spans': [], 'section': 'Discussion'}, {'text': \"It is important to consider the significant limitations of this preliminary study. First, our model assumes continuous, regular exponential growth. Disease epidemics ultimately follow a sigmoidal shape, as they approach the carrying capacity of the disease. SEIR models account for a decreasing susceptible population and increasing recovered population with immunity. These models prove to be superior in long term analysis of a disease's expansion. It is most likely that our model is only applicable for the initial exponential expansion of a disease. For COVID-19, we caution its usage beyond 60 days after initial seeding of 100 cases.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Limitations'}, {'text': 'The data collected by the JHU CSSE is impacted by the reporting capabilities of each country. Subclinical cases are improbable to detect through surveillance screening due to lack of laboratory resources in most countries. Such cases would be entirely missed as presumptive cases, as these patients would not report to a healthcare facility. As mentioned earlier, the presumptive asymptomatic SARS-COV-2 positive population is 17.9% 23 . Beyond sub-clinical cases, governments are instructing mildly symptomatic patients to quarantine at home rather than seek hospital care. These cases would also avoid detection and registration in the data set.', 'cite_spans': [], 'ref_spans': [], 'section': 'Limitations'}, {'text': \"The model equations serve to average the differences between healthcare systems, hospital bed per capita, laboratory testing capabilities, population dynamics, etc. When applying the model to any given country in initial phase of growth (<60 days after 100 cases), though, these differences are expected to exert considerable influence on COVID-19's growth rate. This further emphasizes the caution to utilize this model beyond the initial expansion of the disease.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Limitations'}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'The species Severe acute respiratory syndrome-related coronavirus: classifying 2019-nCoV and naming it SARS-CoV-2. Nat Microbiol', 'authors': [], 'year': 2020, 'venue': 'Coronaviridae Study Group of the International Committee on Taxonomy of Viruses', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1038/s41564-020-0695-z2']}}, 'BIBREF1': {'ref_id': 'b1', 'title': 'Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China. The Lancet', 'authors': [{'first': 'C', 'middle': [], 'last': 'Huang', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Li', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '395', 'issn': '', 'pages': '497--506', 'other_ids': {'DOI': ['10.1016/S0140-6736(20)30183-5']}}, 'BIBREF2': {'ref_id': 'b2', 'title': \"Wuhan Municiple Health and Health Commission's briefing on current pneumonia situation in our city\", 'authors': [], 'year': 2019, 'venue': '', 'volume': '31', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF3': {'ref_id': 'b3', 'title': 'First Case of 2019 Novel Coronavirus in the United States', 'authors': [{'first': 'M', 'middle': ['L'], 'last': 'Holshue', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Debolt', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Lindquist', 'suffix': ''}], 'year': 2020, 'venue': 'N Engl J Med', 'volume': '382', 'issn': '10', 'pages': '929--936', 'other_ids': {'DOI': ['10.1056/NEJMoa2001191']}}, 'BIBREF4': {'ref_id': 'b4', 'title': 'COVID-19) events as they happen', 'authors': [{'first': '', 'middle': [], 'last': 'Coronavirus', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF5': {'ref_id': 'b5', 'title': 'Update on Novel Coronavirus: one positive case reported in Kerala. pib.gov.in/Pressreleaseshare.aspx?PRID=1601095. Accessed', 'authors': [], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF6': {'ref_id': 'b6', 'title': \"Avoid Nonessential Travel -Travel Health Notices | Travelers' Health | CDC\", 'authors': [], 'year': 2020, 'venue': 'COVID-19 in China -Warning -Level', 'volume': '3', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF7': {'ref_id': 'b7', 'title': 'Covid19 | Global Travel Navigator', 'authors': [{'first': 'Mayer', 'middle': [], 'last': 'Brown', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '22', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF8': {'ref_id': 'b8', 'title': 'Transmission potential of the novel coronavirus (COVID-19) onboard the diamond Princess Cruises Ship', 'authors': [{'first': 'K', 'middle': [], 'last': 'Mizumoto', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Chowell', 'suffix': ''}], 'year': 2020, 'venue': 'Infect Dis Model', 'volume': '5', 'issn': '', 'pages': '264--270', 'other_ids': {'DOI': ['10.1016/j.idm.2020.02.003']}}, 'BIBREF9': {'ref_id': 'b9', 'title': 'Coronavirus Found In China Prisons, As Cases Spike In South Korea. NPR.org', 'authors': [], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF10': {'ref_id': 'b10', 'title': 'COVID-19 in a Long-Term Care Facility -King County', 'authors': [{'first': 'T', 'middle': ['M'], 'last': 'Mcmichael', 'suffix': ''}], 'year': 2020, 'venue': 'MMWR Morb Mortal Wkly Rep', 'volume': '69', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.15585/mmwr.mm6912e1']}}, 'BIBREF11': {'ref_id': 'b11', 'title': 'Over 1,700 frontline medics likely infected with coronavirus in China, presenting new crisis for the government', 'authors': [{'first': 'Cnn', 'middle': [], 'last': 'Ng', 'suffix': ''}, {'first': 'Natalie', 'middle': [], 'last': 'Thomas', 'suffix': ''}, {'first': 'David', 'middle': [], 'last': 'Culver', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF12': {'ref_id': 'b12', 'title': 'Epidemic Analysis of COVID-19 in China by Dynamical Modeling. Epidemiology', 'authors': [{'first': 'L', 'middle': [], 'last': 'Peng', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Yang', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Zhuge', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Hong', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.02.16.20023465']}}, 'BIBREF13': {'ref_id': 'b13', 'title': 'The reproductive number of COVID-19 is higher compared to SARS coronavirus', 'authors': [{'first': 'Y', 'middle': [], 'last': 'Liu', 'suffix': ''}, {'first': 'A', 'middle': ['A'], 'last': 'Gayle', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Wilder-Smith', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Rocklöv', 'suffix': ''}], 'year': 2020, 'venue': 'J Travel Med', 'volume': '27', 'issn': '2', 'pages': '', 'other_ids': {'DOI': ['10.1093/jtm/taaa021']}}, 'BIBREF14': {'ref_id': 'b14', 'title': 'Effective containment explains sub-exponential growth in confirmed cases of recent COVID-19 outbreak in Mainland China. ArXiv200207572 Phys Q-Bio', 'authors': [{'first': 'B', 'middle': ['F'], 'last': 'Maier', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Brockmann', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF15': {'ref_id': 'b15', 'title': 'Novel Coronavirus COVID-19 (2019-NCoV) Data Repository by Johns Hopkins CSSE', 'authors': [{'first': '', 'middle': [], 'last': 'Cssegisanddata', 'suffix': ''}], 'year': 2019, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF16': {'ref_id': 'b16', 'title': 'How early signs of the coronavirus were spotted, spread and throttled in China. The Straits Times', 'authors': [], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF17': {'ref_id': 'b17', 'title': 'China Silences Critics Over Deadly Virus Outbreak. The New York Times', 'authors': [{'first': 'L', 'middle': [], 'last': 'Yuan', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF18': {'ref_id': 'b18', 'title': 'China keeps changing how it counts coronavirus cases', 'authors': [{'first': 'W', 'middle': [], 'last': 'Feuer', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF19': {'ref_id': 'b19', 'title': 'Hospital beds', 'authors': [{'first': '', 'middle': [], 'last': 'Islamic Rep', 'suffix': ''}, {'first': '', 'middle': [], 'last': 'Japan', 'suffix': ''}, {'first': 'Rep', 'middle': [], 'last': 'Korea', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF20': {'ref_id': 'b20', 'title': 'Estimating the asymptomatic proportion of coronavirus disease 2019 (COVID-19) cases on board the Diamond Princess cruise ship', 'authors': [{'first': 'K', 'middle': [], 'last': 'Mizumoto', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Kagaya', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Zarebski', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Chowell', 'suffix': ''}], 'year': 2020, 'venue': 'Eurosurveillance', 'volume': '25', 'issn': '10', 'pages': '', 'other_ids': {'DOI': ['10.2807/1560-7917.ES.2020.25.10.2000180']}}}, 'ref_entries': {'FIGREF0': {'text': 'Data: • Phase 1 Countries: All countries reporting greater than 2500 cases as of March 22, 2020 • Members of the Group of 7 (G7): Canada, France, Germany, Italy, Japan, UK, US • Members of the BRICS developing nations: Brazil, Russia, India, China, South Africa • Developed nations of interest in Phase 2:', 'latex': None, 'type': 'figure'}, 'FIGREF2': {'text': 'Model 1: All countries reporting greater than 2500 cases as of March 23, 2020 • Model 2: G7 countries', 'latex': None, 'type': 'figure'}, 'FIGREF3': {'text': 'Cases of COVID-19 in Stabilizing Countries.We isolate Iran, Japan, and South Korea fromFig. 1. On a logarithmic scale graph, the rate of exponential growth is visualized by the slope of the curve. These countries obtained early COVID-19 expansion reduction, noted by a significant change in slope. They are excluded from the modeling equations.', 'latex': None, 'type': 'figure'}, 'FIGREF6': {'text': 'Equations 3 and Model Equation 4: 25− as the number of cases in a country on March 25 and t is the number of days after March 25. The growth rates are derived from Tables 1 and 3, respectively.', 'latex': None, 'type': 'figure'}, 'TABREF1': {'text': 'The mean rate of expansion of Phase 1 countries is 0.238, with standard deviation 0.034.', 'latex': None, 'type': 'table'}, 'TABREF2': {'text': \"Exponential regression modeling of Group of 7 (G7) countries, for the equation ( ) = × . An average value for 'N' and 'r' is calculated. The mean rate of expansion is 0.245, with standard deviation 0.0305. Japan, a member of the G7, is displayed as a strikeout due to being present in the exclusion criteria. However, we display it for comparison purposes. Note that Japan's rate of expansion, r, is approximately 3 times below the average for G7 nations.\", 'latex': None, 'type': 'table'}, 'TABREF3': {'text': 'author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'latex': None, 'type': 'table'}}, 'back_matter': []}\n",
      "{'paper_id': 'b3b84e6844412c5aba86d285a15a9bca706c3fe8', 'metadata': {'title': 'Title: Orthogonal genome-wide screenings in bat cells identify MTHFD1 as a target of 1 broad antiviral therapy 2 3', 'authors': [{'first': 'Danielle', 'middle': ['E'], 'last': 'Anderson', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Jin', 'middle': [], 'last': 'Cui', 'suffix': '', 'affiliation': {'laboratory': 'MOE Key Laboratory of Bioorganic Phosphorus Chemistry & Chemical Biology', 'institution': '', 'location': {'settlement': 'Beijing'}}, 'email': ''}, {'first': 'Qian', 'middle': [], 'last': 'Ye', 'suffix': '', 'affiliation': {'laboratory': 'MOE Key Laboratory of Bioorganic Phosphorus Chemistry & Chemical Biology', 'institution': '', 'location': {'settlement': 'Beijing'}}, 'email': ''}, {'first': 'Baoying', 'middle': [], 'last': 'Huang', 'suffix': '', 'affiliation': {'laboratory': 'NHC Key Laboratory of Biosafety', 'institution': 'Ministry of Health', 'location': {'settlement': 'Beijing', 'country': 'China, China'}}, 'email': ''}, {'first': 'Wenhong', 'middle': [], 'last': 'Zu', 'suffix': '', 'affiliation': {'laboratory': 'MOE Key Laboratory of Bioorganic Phosphorus Chemistry & Chemical Biology', 'institution': '', 'location': {'settlement': 'Beijing'}}, 'email': ''}, {'first': 'Jing', 'middle': [], 'last': 'Gong', 'suffix': '', 'affiliation': {'laboratory': 'MOE Key Laboratory of Bioorganic Phosphorus Chemistry & Chemical Biology', 'institution': '', 'location': {'settlement': 'Beijing'}}, 'email': ''}, {'first': 'Weiqiang', 'middle': [], 'last': 'Liu', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Chinese Academy of Science', 'location': {'settlement': 'Beijing', 'country': 'China'}}, 'email': ''}, {'first': 'So', 'middle': ['Young'], 'last': 'Kim', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Duke University', 'location': {'postCode': '18, 27708', 'settlement': 'Durham, Carolina', 'region': 'North', 'country': 'USA'}}, 'email': ''}, {'first': 'Biao', 'middle': ['Guo'], 'last': 'Yan', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Kristmundur', 'middle': [], 'last': 'Sigmundsson', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Xiao', 'middle': ['Fang'], 'last': 'Lim', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Fei', 'middle': [], 'last': 'Ye', 'suffix': '', 'affiliation': {'laboratory': 'NHC Key Laboratory of Biosafety', 'institution': 'Ministry of Health', 'location': {'settlement': 'Beijing', 'country': 'China, China'}}, 'email': ''}, {'first': 'Peihua', 'middle': [], 'last': 'Niu', 'suffix': '', 'affiliation': {'laboratory': 'NHC Key Laboratory of Biosafety', 'institution': 'Ministry of Health', 'location': {'settlement': 'Beijing', 'country': 'China, China'}}, 'email': ''}, {'first': 'Xuming', 'middle': [], 'last': 'Zhou', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Chinese Academy of Science', 'location': {'settlement': 'Beijing', 'country': 'China'}}, 'email': ''}, {'first': 'Wenjie', 'middle': [], 'last': 'Tan', 'suffix': '', 'affiliation': {'laboratory': 'NHC Key Laboratory of Biosafety', 'institution': 'Ministry of Health', 'location': {'settlement': 'Beijing', 'country': 'China, China'}}, 'email': ''}, {'first': 'Lin-Fa', 'middle': [], 'last': 'Wang', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': '1#', 'middle': [], 'last': '', 'suffix': '', 'affiliation': {}, 'email': ''}, {'first': 'Xu', 'middle': [], 'last': 'Tan', 'suffix': '', 'affiliation': {'laboratory': 'MOE Key Laboratory of Bioorganic Phosphorus Chemistry & Chemical Biology', 'institution': '', 'location': {'settlement': 'Beijing'}}, 'email': 'xutan@tsinghua.edu.cn'}]}, 'abstract': [{'text': 'bioRxiv preprint TATA box, CCAAT box, GC box and DNA stability are shown. Blue and red boxes indicate positions of those putative elements and related transcription factors predicted by GPMiner. A lack of CCAAT box was observed in both bats.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}], 'body_text': [{'text': 'Bats are responsible for the zoonotic transmission of several major viral diseases including the 2 2003 SARS outbreak and the ongoing COVID-19 pandemic. While bat genomic sequencing 3 studies have revealed characteristic adaptations of the innate immune system, functional genomic 4 studies are urgently needed to provide a foundation for the molecular dissection of the tolerance 5 of viral infections in bats. Here we report the establishment and screening of genome-wide RNAi 6 library and CRISPR library for the model megabat, Pteropus Alecto. We used the complementary 7', 'cite_spans': [], 'ref_spans': [], 'section': 'SUMMARY: 1'}, {'text': 'RNAi and CRISPR libraries to interrogate Pteropus Alecto cells for infection with two different 8 viruses, mumps virus and Influenza A virus, respectively. Screening results converged on the 9 endocytosis pathway and the protein secretory pathway as required for both viral infections. 10', 'cite_spans': [], 'ref_spans': [], 'section': 'SUMMARY: 1'}, {'text': 'Additionally, we revealed a general dependence of the C-1-tetrahydrofolate synthase gene, 11 MTHFD1, for viral replication in bat cells as well as in human cells. MTHFD1 inhibitor carolacton 12 potently blocked replication of several RNA viruses including SARS-CoV-2. Our studies provide 13 a resource for systematic inquiry into the genetic underpinnings of bat biology and a potential 14 target for developing broad spectrum antiviral therapy. viruses, causing the epidemics of the Ebola 1 , SARS 2 , Henipavirus 3 , MERS 4 and 6 . 20', 'cite_spans': [{'start': 530, 'end': 531, 'text': '6', 'ref_id': None}], 'ref_spans': [], 'section': 'SUMMARY: 1'}, {'text': 'Bats are widely considered the largest reservoir of numerous virus species but are resilient to the 21 diseases associated with many of these viruses 7, 8 . How bats tolerate the infections with these 22 viruses is of intense interest due to the scientific and public health relevance. Genomic 23 sequencing studies have revealed several genetic adaptions underlying this tolerance of viral 24', 'cite_spans': [{'start': 150, 'end': 152, 'text': '7,', 'ref_id': None}, {'start': 153, 'end': 154, 'text': '8', 'ref_id': None}], 'ref_spans': [], 'section': 'SUMMARY: 1'}, {'text': 'infections. For example, the DNA damage checkpoint pathway and the NF-kB pathway are 25 known to be positively selected in the bat genomes 9 . In addition, in the type I interferon gene 26 family, the MHC class I genes and natural killer cell receptor family have been reported to be 27 highly expanded in the bat genomes 10 . These studies provided evidence to support the hypothesis 28 that bats have evolved a highly specific genetic configuration, especially in the innate immunity 29 pathways. A major caveat is the lack of systematic examination of the functions of these genes 30 due to shortage of functional genomic tools. 31 CRISPR and RNAi technologies have enabled genome-wide screening of gene functions for a 1 variety of cell types, yielding an unprecedented wealth of information revealing the complexity of 2 biological processes 11, 12 . This is especially true in the field of virus-host interactions, where a 3 plethora of host factors have been identified for a variety of viruses [13] [14] [15] . Both technologies have 4 advantages and disadvantages and provide complementary approaches to achieve comprehensive 5 genetic perturbations [16] [17] [18] . CRISPR permits complete knockout of a gene, thus affords generation 6 of truly null genotype. A caveat is that the lethality associated with knockout of essential genes 7 prevents functional parsing of these genes in different biological processes. In addition, CRISPR 8 screening is usually used a pool-based format that requires the generation of stable cell lines 9 through long term culturing. Therefore, CRISPR is suited for studying phenotypes that require a 10 longer time to manifest. Based on previous studies, CRISPR screens demonstrated excellent 11 specificity but limited sensitivity 17 . In contrast, siRNA screening relies on temporary knockdown 12 of genes in an individual well format, a methodology suited for examining short-term phenotypic 13 changes. Due to the transient nature of the assay, siRNA screening can tolerate growth defects due 14 to gene knockdown and can elucidate hypomorphic phenotypes induced by siRNAs with varying 15 degrees of knockdown. In general, RNAi screens allow comprehensive identification of genes in 16 a biological process at the cost of more false positives due to pervasive off-target effects 17 . To fully 17 realize the power of CRISPR and RNAi screening as complementary strategies to interrogate gene 18 function in bats, here we developed a pool-based CRISPR library and a plate-based siRNA library 19', 'cite_spans': [{'start': 847, 'end': 850, 'text': '11,', 'ref_id': None}, {'start': 851, 'end': 853, 'text': '12', 'ref_id': None}, {'start': 1002, 'end': 1006, 'text': '[13]', 'ref_id': None}, {'start': 1007, 'end': 1011, 'text': '[14]', 'ref_id': None}, {'start': 1012, 'end': 1016, 'text': '[15]', 'ref_id': None}, {'start': 1159, 'end': 1163, 'text': '[16]', 'ref_id': None}, {'start': 1164, 'end': 1168, 'text': '[17]', 'ref_id': None}, {'start': 1169, 'end': 1173, 'text': '[18]', 'ref_id': None}], 'ref_spans': [], 'section': 'SUMMARY: 1'}, {'text': 'for the whole genome of the model megabat Pteropus Alecto. We screened two viruses, influenza 20 A virus and mumps virus, using the CRISPR and RNAi libraries, respectively, in the hope of 21 uncovering common host factors for different viruses. From these two screens, common pathways 22 such as the endocytosis pathway and protein secretory pathway have been identified, 23 demonstrating conserved pathways between bat and other mammals in hosting viral infections. 24', 'cite_spans': [], 'ref_spans': [], 'section': 'SUMMARY: 1'}, {'text': 'We extensively characterized a common hit from the two screens, MTHFD1, and identified it as a 25 critical factor for viral replication in both bats and humans. We further demonstrate that MTHFD1 26 is a potential target for developing broad-spectrum antiviral drugs. MTHFD1 inhibitor carolacton 27 demonstrated broad antiviral activities against Zika virus, mumps virus, and importantly, SARS-28', 'cite_spans': [], 'ref_spans': [], 'section': 'SUMMARY: 1'}, {'text': 'CoV-2. In summary, we developed two orthogonal libraries for genome-wide loss-of-function 29 screening in bats and demonstrated their utility in discovery of important host factors for viral 30 infections. 31 . CC-BY-NC 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020. 03.29.014209 doi: bioRxiv preprint Design and validation of genome-wide CRISPR of P. Alecto 1', 'cite_spans': [{'start': 398, 'end': 432, 'text': '03.29.014209 doi: bioRxiv preprint', 'ref_id': None}], 'ref_spans': [], 'section': 'SUMMARY: 1'}, {'text': 'We designed and synthesized the CRISPR and RNAi libraries based on the genome of P. alecto. 2', 'cite_spans': [], 'ref_spans': [], 'section': 'SUMMARY: 1'}, {'text': 'For the CRISPR library, we adopted the algorithm developed for the construction of the popular 3', 'cite_spans': [], 'ref_spans': [], 'section': 'SUMMARY: 1'}, {'text': 'GeCKO library to select single guide RNAs (sgRNAs) 19 . We designed 85,275 sgRNAs targeting 4 21,334 genes annotated in the P. alecto genome with 21,330 genes with 4 sgRNAs for each gene. 5', 'cite_spans': [], 'ref_spans': [], 'section': 'SUMMARY: 1'}, {'text': 'The other 34 genes have 1-3 sgRNAs due to the constraints of the gene sequences for designing 6 sgRNAs. Selection of sgRNAs was based on the calculations of off-target scores to lower potential 7 off-target effects of the sgRNAs. The sgRNAs designs were validated for 21 sgRNAs targeting 11 8 genes using T7E1 endonuclease cleavage assay or Western blotting. We observed 90.5% of 9 effectiveness in gene editing or gene knockdown, supporting the quality of the designed library 10 (Supplementary Figure 1) . 11', 'cite_spans': [], 'ref_spans': [{'start': 496, 'end': 505, 'text': 'Figure 1)', 'ref_id': None}], 'section': 'SUMMARY: 1'}, {'text': 'We performed a CRISPR screen with influenza A virus in PaKi cells to identify viral host factors 14 ( Fig. 1a) . PaKi cells stably expressing SgCas9 and sgRNA library were infected with the H1N1 15 PR8 strain of influenza A virus, in duplicate. We used a high multiplicity-of-infection (MOI) so 16 that naïve PaKi cells all died before the 9th day post infection. We then collected the surviving 17 cells and compared sgRNA abundance with mock-infected cells. The two replicates demonstrated 18 a high correlation, indicating high reproducibility (Supplementary Fig. 2) . We calculated the 19 results using the RIGER algorithm and identified 21 host factors required for the viral infection or 20 pathogenesis 20 (Fig. 1b) . Three of the top four hits belong to the V-type proton ATPase subunits 21 (ATP6V1B2, ATP6V1F and ATP6V0D1), which are involved in the acidification of endosome 22', 'cite_spans': [], 'ref_spans': [{'start': 102, 'end': 110, 'text': 'Fig. 1a)', 'ref_id': None}, {'start': 547, 'end': 569, 'text': '(Supplementary Fig. 2)', 'ref_id': None}, {'start': 713, 'end': 722, 'text': '(Fig. 1b)', 'ref_id': None}], 'section': 'CRISPR screening identified influenza A virus host factors 13'}, {'text': 'and have previously been previous identified as key host factors for influenza virus entry in several 23 screens performed in human cells (Fig. 1b) 21 . Among the other hits were two genes, COPZ1 and 24 SEC23B, involved in the protein secretory pathway, which are also known to be required for the 25 replication of many viruses including influenza virus and flavivirus (Fig. 1b) [21] [22] [23] . The enrichment 26 of previously known host factors in the identified hits in the CRISPR screen validated our library 27 and the screening methodology. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020. 03.29.014209 doi: bioRxiv preprint We synthesized the sgRNA library in an array-based oligonucleotide library and cloned them into 1 the lentiGuide-Puro lentivirus expression vector 24 . A custom bat siRNA library was designed to 2 target 18,328 genes using 40,016 siRNA duplexes. We performed RNAi screening in PaKi cells 3 for identification of host factors for the infection of mumps virus, a paramyxovirus (Fig. 1c) . The 4 screen was performed in 384-well format with a mumps virus strain expressing EGFP from an 5 additional open reading frame 25 . PaKi cells were infected 48 hours after transfection of the siRNA 6 library. High-content imaging analysis was utilized to quantitate the infection rate as well as the 7 total cell number each well, indicating cytotoxicity of the siRNA. Excluding siRNAs with 8 significant cytotoxicity, we selected 45 host dependency factors whose knockdown resulted in 9', 'cite_spans': [{'start': 148, 'end': 150, 'text': '21', 'ref_id': None}, {'start': 380, 'end': 384, 'text': '[21]', 'ref_id': None}, {'start': 385, 'end': 389, 'text': '[22]', 'ref_id': None}, {'start': 390, 'end': 394, 'text': '[23]', 'ref_id': None}, {'start': 656, 'end': 690, 'text': '03.29.014209 doi: bioRxiv preprint', 'ref_id': None}], 'ref_spans': [{'start': 138, 'end': 147, 'text': '(Fig. 1b)', 'ref_id': None}, {'start': 370, 'end': 379, 'text': '(Fig. 1b)', 'ref_id': None}, {'start': 1066, 'end': 1075, 'text': '(Fig. 1c)', 'ref_id': None}], 'section': 'CRISPR screening identified influenza A virus host factors 13'}, {'text': 'significantly reduced viral infection and 45 host restriction factors whose knockdown promoted 10 viral infection based on the Z score of infection rate (Fig. 1d) . We selected a number of genes for 11 individual validation with more replicates and obtained a high validation rate (11/12 of host 12 dependency factors and 7/12 of the host restriction factors validated with at least one siRNA) 13 ( Supplementary Fig. 3) , supporting the reproducibility of our screening methodology. 14 15', 'cite_spans': [], 'ref_spans': [{'start': 153, 'end': 162, 'text': '(Fig. 1d)', 'ref_id': None}, {'start': 399, 'end': 420, 'text': 'Supplementary Fig. 3)', 'ref_id': None}], 'section': 'CRISPR screening identified influenza A virus host factors 13'}, {'text': 'Although there is only one gene overlap between the host factors identified in both CRISPR 16', 'cite_spans': [], 'ref_spans': [], 'section': 'CRISPR screening identified influenza A virus host factors 13'}, {'text': 'and RNAi screens, we observed that the endocytosis pathway and protein secretory pathway were 17 enriched in both screens, albeit represented by different genes (Fig. 1b and 1d) . For the 18 endocytosis pathway, two genes encoding components of clathrin complex, CLTA and CLTC, 19', 'cite_spans': [], 'ref_spans': [{'start': 161, 'end': 177, 'text': '(Fig. 1b and 1d)', 'ref_id': None}], 'section': 'CRISPR screening identified influenza A virus host factors 13'}, {'text': 'were among the top three hits (Fig. 1d) . Four genes involved in the protein secretory, namely 20 ARF1, ARF4, COPG1 and SEC61B, were identified (Fig. 1d) . This is consistent with the idea that 21 both influenza A virus and mumps require the two pathways for viral replication. In addition, 22 protein translation initiation, which was previously reported to be required for influenza A 23 infection 21 , was also represented in the top hits by three genes: EIF3B, EIF3G and EIF4A3. 24', 'cite_spans': [], 'ref_spans': [{'start': 30, 'end': 39, 'text': '(Fig. 1d)', 'ref_id': None}, {'start': 144, 'end': 153, 'text': '(Fig. 1d)', 'ref_id': None}], 'section': 'CRISPR screening identified influenza A virus host factors 13'}, {'text': 'Conversely, RNAi screening revealed genes that restrict mumps infection in PaKi cells. Among 25 the top hits were several known antiviral genes, TRIM39, TRIM43 26 and NAMPT 27 . Overall, we 26 identified many candidate genes that might play a role in viral infection in bats. 27', 'cite_spans': [], 'ref_spans': [], 'section': 'CRISPR screening identified influenza A virus host factors 13'}, {'text': 'The two pathways identified by both CRISPR screen and siRNA screen, endocytosis and 29 protein secretion, have previously been implicated in human host response to a number of viral 30 pathogens 13, 21, 23 To validate the role of these pathways in the bat response to viral infection, we 31 . CC-BY-NC 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020. 03.29.014209 doi: bioRxiv preprint established PaKi stable cell lines expressing shRNAs targeting several hits from the two screens. 1', 'cite_spans': [{'start': 195, 'end': 198, 'text': '13,', 'ref_id': None}, {'start': 199, 'end': 202, 'text': '21,', 'ref_id': None}, {'start': 203, 'end': 205, 'text': '23', 'ref_id': None}, {'start': 480, 'end': 514, 'text': '03.29.014209 doi: bioRxiv preprint', 'ref_id': None}], 'ref_spans': [], 'section': '28'}, {'text': 'Stable knockdown of a V-type proton ATPase subunit and SEC23B led to a significant decrease 2 of influenza A virus infection in PaKi cells ( Fig. 1e and 1f ). In addition, the small molecule 3 inhibitor of V-type proton ATPase, bafilomycin A (Fig. 1g) , potently blocked influenza infection. 4', 'cite_spans': [], 'ref_spans': [{'start': 141, 'end': 155, 'text': 'Fig. 1e and 1f', 'ref_id': None}, {'start': 242, 'end': 251, 'text': '(Fig. 1g)', 'ref_id': None}], 'section': '28'}, {'text': 'Taken together, these results demonstrate that bat and human cells share key genes and pathways 5', 'cite_spans': [], 'ref_spans': [], 'section': '28'}, {'text': 'in hosting viral infection. 6 7 MTHFD1 is a key host factor for a broad spectrum of RNA viruses 8', 'cite_spans': [], 'ref_spans': [], 'section': '28'}, {'text': 'The gene identified as a host factor from both the CRISPR and RNAi screen was MTHFD1 ( Fig.  9 1c and 1d), which we validated by siRNA knockdown in the context of mumps infection 10 (Supplemental Fig. 3 ', 'cite_spans': [], 'ref_spans': [{'start': 87, 'end': 94, 'text': 'Fig.  9', 'ref_id': None}, {'start': 196, 'end': 202, 'text': 'Fig. 3', 'ref_id': None}], 'section': '28'}, {'text': 'with two different sgRNAs targeting MTHFD1 ( Fig. 2a-2b) . Similarly, in HEK293T cells with 12 MTHFD1 knocked out by two sgRNAs, influenza replication is also inhibited, suggesting a 13 conserved requirement for this gene by influenza virus in human cells (Fig. 2c) . Further virus 14 titration assay demonstrated that MTHFD1 knockdown significantly inhibits viral replication for 15 mumps virus in PaKi cells ( Fig. 2d and 2e) . The effect is similar to the knockdown of SEC61B, a 16 well-studied host protein required for viral replication and a top hit in our RNAi screen ( Fig. 2d  17 and 2e) 23 . We additionally tested Melaka virus (PRV3M), a pteropine orthoreovirus that causes 18 acute respiratory disease when transmitted by bats to humans 28 . PRV3M titer was also significantly 19 reduced by MTHFD1 knockdown ( Fig. 2e and 2f) . Infection of Zika virus, a member of the 20 flavivirus family, was also inhibited by MTHFD1 knockdown using siRNA (Fig. 2g) . Alternative 21 knockdown with stable expression of two shRNAs validated the requirement of MTHFD1 for the 22 replication of mumps virus and Zika virus (Fig. 2h, 2i and 2j) . Importantly, we also show that this 23 inhibition is completely rescued by overexpression of MTHFD1 from a construct resistant to the 24 shRNA, ruling out potential off-target effects (Fig. 2k) . 25 26', 'cite_spans': [], 'ref_spans': [{'start': 45, 'end': 56, 'text': 'Fig. 2a-2b)', 'ref_id': None}, {'start': 256, 'end': 265, 'text': '(Fig. 2c)', 'ref_id': None}, {'start': 412, 'end': 427, 'text': 'Fig. 2d and 2e)', 'ref_id': None}, {'start': 577, 'end': 588, 'text': 'Fig. 2d  17', 'ref_id': None}, {'start': 822, 'end': 837, 'text': 'Fig. 2e and 2f)', 'ref_id': None}, {'start': 954, 'end': 963, 'text': '(Fig. 2g)', 'ref_id': None}, {'start': 1117, 'end': 1137, 'text': '(Fig. 2h, 2i and 2j)', 'ref_id': None}, {'start': 1324, 'end': 1333, 'text': '(Fig. 2k)', 'ref_id': None}], 'section': '). We validated its function in influenza A virus replication in PaKi cells 11'}, {'text': 'MTHFD1 is a trifunctional enzyme involved in the one carbon (C1) metabolism pathway, which 28 is responsible for cellular production of purine, dTMP and methyl groups 29 . MTHFD1 has three 29 enzymatic functions including dehydrogenase and cyclohydrolase activities encoded in the N-30 terminal domain and formyl tetrahydrofolate (Formyl THF) synthetase activity encoded in the C-31 . CC-BY-NC 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020. 03.29.014209 doi: bioRxiv preprint terminal domain ( Fig. 3a and 3b) . We suspected that MTHFD1 knockdown would lead to the 1 deficiency of cellular purine levels thereby reducing RNA replication. Indeed, supplementation of 2 purine analogs, either hypoxanthine or inosine in the media almost completely rescued the 3 replication of influenza A and Zika viruses in MTHFD1-knockout or knockdown PaKi cells as 4', 'cite_spans': [{'start': 572, 'end': 606, 'text': '03.29.014209 doi: bioRxiv preprint', 'ref_id': None}], 'ref_spans': [{'start': 625, 'end': 640, 'text': 'Fig. 3a and 3b)', 'ref_id': None}], 'section': \"MTHFD1's formyl tetrahydrofolate synthetase activity is essential for RNA virus replication 27\"}, {'text': 'shown by both immunofluorescence staining and virus tittering assay (Fig. 3c, 3d and 3e) . In 5 contrast, supplementation of methyl-group donor or thymidine had little effect on viral replication, 6 evidence that it is the purine synthesis activity of MTHFD1 that is essential for the viral replication 7 (Fig. 3d) . To further pinpoint the enzymatic function of MTHFD1 in viral replication, we 8 performed gene complementation experiments in an shRNA cell line with over 90% knockdown 9 of MTHFD1 (Fig. 2i) . The block of Zika virus replication in this cell line was completely rescued 10 by overexpression of an shRNA-resistant MTHFD1 construct or bat homolog ( Fig. 3f and 3g) . 11', 'cite_spans': [], 'ref_spans': [{'start': 68, 'end': 88, 'text': '(Fig. 3c, 3d and 3e)', 'ref_id': None}, {'start': 305, 'end': 314, 'text': '(Fig. 3d)', 'ref_id': None}, {'start': 498, 'end': 507, 'text': '(Fig. 2i)', 'ref_id': None}, {'start': 664, 'end': 679, 'text': 'Fig. 3f and 3g)', 'ref_id': None}], 'section': \"MTHFD1's formyl tetrahydrofolate synthetase activity is essential for RNA virus replication 27\"}, {'text': 'Mutations of MTHFD1 that are known to disrupt its dehydrogenase and cyclohydrolase activities 12 (D125A and R173C) 30,31 did not affect this rescue (Fig. 3b, 3f and 3g) . However, mutations in the 13', 'cite_spans': [], 'ref_spans': [{'start': 148, 'end': 168, 'text': '(Fig. 3b, 3f and 3g)', 'ref_id': None}], 'section': \"MTHFD1's formyl tetrahydrofolate synthetase activity is essential for RNA virus replication 27\"}, {'text': 'Formyl THF synthetase active site (F759A/W788A, or FW/AA) 32 completely abolished this rescue, 14 further supporting that the purine synthesis activity of MTHFD1 is an essential activity for viral 15 replication (Fig. 3b, 3f and 3g) . 16', 'cite_spans': [], 'ref_spans': [{'start': 212, 'end': 232, 'text': '(Fig. 3b, 3f and 3g)', 'ref_id': None}], 'section': \"MTHFD1's formyl tetrahydrofolate synthetase activity is essential for RNA virus replication 27\"}, {'text': 'We observed a similar dependence on MTHFD1 function for influenza A virus infection in 18', 'cite_spans': [], 'ref_spans': [], 'section': '17'}, {'text': 'HEK293T cells, a human kidney epithelial cell line (Fig. 3h) . Time course experiments 19 demonstrated a viral replication following MTHFD1 knockdown steadily decreased from four 20 hours post-infection with influenza A virus, consistent with a RNA replication defect (Fig. 3i) . 21', 'cite_spans': [], 'ref_spans': [{'start': 51, 'end': 60, 'text': '(Fig. 3h)', 'ref_id': None}, {'start': 268, 'end': 277, 'text': '(Fig. 3i)', 'ref_id': None}], 'section': '17'}, {'text': 'This defect in replication was completely rescued by supplementing purine analog inosine in the 22 media, supporting a conserved function of MTHFD1 in viral infection in bats and humans (Fig.  23   3h and 3j, Supplementary Fig. 4) . 24', 'cite_spans': [], 'ref_spans': [{'start': 186, 'end': 230, 'text': '(Fig.  23   3h and 3j, Supplementary Fig. 4)', 'ref_id': None}], 'section': '17'}, {'text': 'Carolacton, a natural product derived from bacteria, has recently been identified as a potent 27 inhibitor of MTHFD1 33 . We tested the effect of this compound on viral replication. Carolacton 28 inhibited Zika virus and mumps virus replication in PaKi cells in a dose dependent fashion (Fig.  29   4a, 4b and 4c ). The inhibition was achieved at doses with little cytotoxicity (Fig. 4a) . 30', 'cite_spans': [], 'ref_spans': [{'start': 287, 'end': 312, 'text': '(Fig.  29   4a, 4b and 4c', 'ref_id': None}, {'start': 378, 'end': 387, 'text': '(Fig. 4a)', 'ref_id': None}], 'section': 'MTHFD1 inhibitor carolacton inhibits RNA virus replication 26'}, {'text': 'Mechanistically, this inhibition was completely rescued by supplementation of inosine, providing 31 . CC-BY-NC 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.29.014209 doi: bioRxiv preprint support that carolacton indeed exerts antiviral activities by inhibiting MTHFD1 (Fig. 4b and 4c ). 1', 'cite_spans': [], 'ref_spans': [{'start': 403, 'end': 418, 'text': '(Fig. 4b and 4c', 'ref_id': None}], 'section': 'MTHFD1 inhibitor carolacton inhibits RNA virus replication 26'}, {'text': 'Vero E6 cells was effectively inhibited at sub-micromolar concentrations by carolacton (IC 50 =0.14 3 µM) (Fig. 4d) . The cytotoxic effect is very moderate in comparison (Fig. 4d) , supporting a 4 therapeutic window of this compound in potential clinical applications. 5 6 Discussion 7', 'cite_spans': [], 'ref_spans': [{'start': 106, 'end': 115, 'text': '(Fig. 4d)', 'ref_id': None}, {'start': 170, 'end': 179, 'text': '(Fig. 4d)', 'ref_id': None}], 'section': 'We next tested the effect of carolacton on the infection of SARS-CoV-2. The viral infection in 2'}, {'text': \"We developed the first bat genome-wide RNAi and CRISPR libraries and performed two screens 8 with two different RNA viruses. Although the screening methodologies are quite different, we 9 uncovered similar pathways essential for viral replication, namely the endocytosis pathway and 10 protein secretory pathways. The different genes in these two pathways identified from the screens 11 reflect the complementarity of the methodologies, with RNAi screening useful for observing acute 12 effects on viral infection and CRISPR screening for long-term effects on cell survival. Other than 13 technical differences between screening methods, viral kinetics, sensitivity of the viruses to loss- 20 Both screens identified MTHFD1 as an essential viral host factor. MTHFD1 is a key enzyme 21 in the C1 metabolism responsible for production of purine, dTMP and methyl group, the first two 22 of which are building blocks for DNA and RNA synthesis. Interestingly, all previous RNAi and 23 CRISPR screens using human cells have not identified MTHFD1 as a host factor. Moreover, we 24 found bats cells and tissues have lower MTHFD1 expression levels than their human counterparts 25 (Supplementary Fig. 5a, 5b and 5c ). This might be related to the difference in the promoter 26 regions between bats and humans, notably a lack of CCAAT box 34 in the promoter of two bat 27 species (Supplementary Fig. 5d) The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020. 03.29.014209 doi: bioRxiv preprint converts the whole cellular machinery into a factory to massively produce progeny viruses, which 1 explains the vital dependence on MTHFD1. This is analogous to cancer cell's addiction to non-2 oncogenic gene functions such as many metabolic enzymes 35 . Similar to the therapeutic targeting 3 of these non-oncogenic addictions for developing cancer drugs, these viral addictions to metabolic 4 enzymes such as MTHFD1 could open doors to many new opportunities for developing broad-5 spectrum antiviral drugs 36 . These drugs could be readily applied in emerging epidemics, as 6 demonstrated by our evidence here that MTHFD1 inhibitor carolacton can suppress the infection 7 of SARS-CoV-2. We envision the functional genomics tools we provided here will be instrumental 8 to further discoveries by better understanding the bat-virus interaction. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.29.014209 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020. 03.29.014209 doi: bioRxiv preprint Influenza A virus (IAV) for 12h. The infection rates were quantified with immunofluorescence 1 staining of HA. Data shown as the mean ±standard deviation (n=3). b. Representative images 2 from a. (scale bar=100 µm). c. MTHFD1 knock out 293T cells were generated from two 3 independent sgRNAs and infected with IAV for another 12h. Then the cells were collected and 4 analyzed by Western blotting. Data shown are representative of three independent experiments. d-5\", 'cite_spans': [{'start': 1502, 'end': 1536, 'text': '03.29.014209 doi: bioRxiv preprint', 'ref_id': None}, {'start': 2633, 'end': 2667, 'text': '03.29.014209 doi: bioRxiv preprint', 'ref_id': None}], 'ref_spans': [{'start': 1172, 'end': 1205, 'text': '(Supplementary Fig. 5a, 5b and 5c', 'ref_id': None}, {'start': 1370, 'end': 1393, 'text': '(Supplementary Fig. 5d)', 'ref_id': None}], 'section': 'We next tested the effect of carolacton on the infection of SARS-CoV-2. The viral infection in 2'}, {'text': 'f. PaKi cells were transfected with non-target control siRNA (siNT) or siRNAs targeting MTHFD1 The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020. 03.29.014209 doi: bioRxiv preprint infected with SARS-CoV-2. At 48 h post infection, the SARS-CoV-2 in the supernatant was 1 measured by real-time PCR. Data shown as the mean ±standard deviation (n=3). ', 'cite_spans': [{'start': 203, 'end': 237, 'text': '03.29.014209 doi: bioRxiv preprint', 'ref_id': None}], 'ref_spans': [], 'section': 'We next tested the effect of carolacton on the infection of SARS-CoV-2. The viral infection in 2'}, {'text': 'De-multiplexed reads were trimmed by Cutadapt 2 (version 1.18) using 22 bp flanking 23 sequences around the guide sequence. Trimmed reads were mapped to the index of bat sgRNA 24 guide library using Bowtie2 3 . Samtools 4 (version 1.9) was used to process the alignment files to 25 retain perfectly mapped reads only, and a custom-made Python script was used to count the reads 26 mapped to each guide. The reads count of guide for each condition were then normalized using 27', 'cite_spans': [], 'ref_spans': [], 'section': 'Deep sequencing data analysis 22'}, {'text': 'DEseq2. The log2 ratio of counts between conditions were calculated. The log2 fold change of 28 reads count of infected samples to mock infected samples were then used as the input for RIGER 29 algorithm as previously described (RNAi Gene Enrichment Ranking, GENE-E) 1 . The log-fold-30 change metric for ranking sgRNAs and the weighted sum method (weighted sum of first two ranks 31 of sgRNAs for a gene) with 1 × 10 6 permutations were used to convert individual sgRNA to genes. 32 . CC-BY-NC 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020. 03.29.014209 doi: bioRxiv preprint In the output of RIGER analysis, a given gene was assigned a normalized enrichment score (NES) 1 to give the ranked gene list. Hits were defined as those owning p-value smaller than 0.001. 2 3 siRNA screen 4', 'cite_spans': [{'start': 673, 'end': 707, 'text': '03.29.014209 doi: bioRxiv preprint', 'ref_id': None}], 'ref_spans': [], 'section': 'Deep sequencing data analysis 22'}, {'text': 'For siRNA screen, siRNAs targeting 18,328 genes of P. alecto were arrayed in the 96-well 5 plates. The siRNAs were reverse transfected into PaKi cells, 48 h later, the cells were infected with 6 mumps virus. The virus infection level was measured at 48 h post infection by high content 7 imaging (Operetta) as previously described 5 . The siRNA targeted genes were ranked by robust Z 8 scores 6 , and the potential host factors (Z score < -3) or restriction factors (Z score > 6) were 9', 'cite_spans': [], 'ref_spans': [], 'section': 'Deep sequencing data analysis 22'}, {'text': 'identified. 10 11', 'cite_spans': [], 'ref_spans': [], 'section': 'Deep sequencing data analysis 22'}, {'text': 'Single sgRNA was cloned into lentiGuide-puro and packaged in lentivirus, which was 13 transduced into Cas9 stably expressing PaKi cells, followed by puromycin and blasticidin selection. 14 The clonal cells were generated by limiting dilution. ', 'cite_spans': [], 'ref_spans': [], 'section': 'Validation of sgRNA 12'}, {'text': 'Individual shRNAs were designed using the BLOC-iT TM RNAi Designer (Thermo Fisher 23 Scientific). The specificity was tested by BLAST (NCBI). The shRNA primes were synthesized 24 as oligos and cloned into PLKO.1. 25', 'cite_spans': [], 'ref_spans': [], 'section': 'Validation of shRNA 22'}, {'text': \"PaKi shMTHFD1-1 forward primer: 5'-ccgg GCACATGGGAATTCCTCTACCctcgag 26\", 'cite_spans': [], 'ref_spans': [], 'section': 'Validation of shRNA 22'}, {'text': \"PaKi shMTHFD1-1 reverse primer: 5'-AATTCAAAAA GCACATGGGAATTCCTCTACC 28\", 'cite_spans': [], 'ref_spans': [], 'section': \"GGTAGAGGAATTCCCATGTGC tttttg-3' 27\"}, {'text': \"PaKi shMTHFD1-2 forward primer: 5'-ccgg GCCTGCTGTCACTTAGGAAATctcgag 30 ATTTCCTAAGTGACAGCAGGC tttttg -3' 31 . CC-BY-NC 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020. 03.29.014209 doi: bioRxiv preprint PaKi shMTHFD1-2 reverse primer: 5'-AATTCAAAAA GCCTGCTGTCACTTAGGAAAT 1 CTCGAG ATTTCCTAAGTGACAGCAGGC -3' 2 3\", 'cite_spans': [{'start': 296, 'end': 330, 'text': '03.29.014209 doi: bioRxiv preprint', 'ref_id': None}], 'ref_spans': [], 'section': \"CTCGAG GGTAGAGGAATTCCCATGTGC-3' 29\"}, {'text': 'Cells were harvested and lysed in RIPA lysis buffer (Beyotime, P0013C) with a cocktail 5 of protease inhibitors (Biomake, B14001). Samples were applied to 10% or 12% SDS/PAGE 6 gels(BioRad) and transferred to PVDF membranes (millipore). Membranes were blocked with 5% 7 nonfat milk in PBS with 0.1% tween 20. Then probed with primary antibodies. The following 8 antibodies were used in this study: anti-MTHFD1 (Proteintech, 10794-1-AP), anti-beta actin 9', 'cite_spans': [], 'ref_spans': [], 'section': 'Western blot 4'}, {'text': \"(Easybio, BE0022), anti PR8 M1 (Genetex, GTX125928-S), anti HA (H1N1) (Genetex, 10 P. alecto ACTIN forward primer: 5'-gccagtctacaccgtctgcag -3' 23 P. alecto ACTIN reverse primer: 5'-cgtaggaatccttctggcccatg-3' 24 P. alecto MTHFD1 forward primer: 5'-gggagcgactgaagaaccaag-3' 25 P. alecto MTHFD1 reverse primer: 5'-tcttcagcagccttcagcttcac-3' 26 P. alecto SEC23B forward primer: 5'-cagcgtttgaccaggaggcc-3' 27 P. alecto SEC23B reverse primer: 5'-gggtcagatcctgtcgggc -3' 28 P. alecto GAPDH forward primer: 5'-ATACTTCTCATGGTTCACAC -3' 29 P. alecto GAPDH reverse primer: 5'-TCATTGACCTCAACTACATG-3' 30 P. alecto ATP6V0D1 forward primer: 5'-GTGGTAGAGTTCCGCCACAT-3' 31 . CC-BY-NC 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.29.014209 doi: bioRxiv preprint P. alecto ATP6V0D1 reverse primer: 5'-CTCAAAGCTGCCTAGTGGGT-3' 1 PR8 M1 forward primer: 5'-TTCTAACCGAGGTCGAAACGTACG-3' 2 PR8 M1 reserve primer: 5'-ACAAAGCGTCTACGCTGCAG-3' 3 PR8 vRNA reserve transcription primer: 5'-AGCRAAAGCAGG-3' 4 ZIKV NS5 forward primer: 5'-GGTCAGCGTCCTCTCTAATAAACG-3' 5 ZIKV NS5 reserve primer: 5'-GCACCCTAGTGTCCACTTTTTCC-3' 6 7 Immunofluorescence 8\", 'cite_spans': [], 'ref_spans': [], 'section': 'Western blot 4'}, {'text': 'Virus infected cells were fixed with 4% paraformaldehyde (PFA) for 10min at room 9 temperature (RT), and permeated with 0.2% Triton X-100 for another 10min at RT, the cells were 10 washed with PBS for 3 times and incubated with PR8 HA antibody or ZIKV E protein antibody 11 for 2h at RT or 4℃ overnight, then washed with PBS for 3 times and incubated with second 12 antibody AF488 (goat anti mouse). The nuclei were stained with DAPI. Images were acquired 13 with Cellomic ArrayScan VTI HCS (Thermo Scientific) 14 15', 'cite_spans': [], 'ref_spans': [], 'section': 'Western blot 4'}, {'text': 'Approximately 1x10 4 PaKi cells were plated into each well of 96-well plates. The 17 supernatant containing virus harvested from previously infected cell was subjected to ten-fold 18 serial dilution. Then dilutions were added to the cells and cytopathic effect was observed 96 h after 19 infection. Titres were determined using standard methodology 7 . The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020. 03.29.014209 doi: bioRxiv preprint ', 'cite_spans': [{'start': 461, 'end': 495, 'text': '03.29.014209 doi: bioRxiv preprint', 'ref_id': None}], 'ref_spans': [], 'section': 'Virus titration 16'}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'The blots were developed by HRP reaction', 'authors': [{'first': 'Anti-Zikv', 'middle': [], 'last': 'Gtx117951-S)', 'suffix': ''}, {'first': '', 'middle': [], 'last': 'Ns1', 'suffix': ''}], 'year': None, 'venue': 'Genetex, GTX133307), anti flavivirus group antigen antibody 11 (Millipore, MAB10216)', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF1': {'ref_id': 'b1', 'title': 'ProteinSimple FluorChem imaging system', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF2': {'ref_id': 'b2', 'title': 'RNA isolation and real-time PCR', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF4': {'ref_id': 'b4', 'title': 'The cDNA was generated by RevertAid RT Reverse Transcription Kit (thermos scientific', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF5': {'ref_id': 'b5', 'title': 'Influenza vRNA specific primer was used for vRNA transcription. The quantitative real-18', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF6': {'ref_id': 'b6', 'title': 'time PCR was performed with SYBR Green qPCR master mix (Vazyme, Q311-02) and data were 19 analyzed by Bio-Rad CFX96 system. The primers for target genes were as follows', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}}, 'ref_entries': {'FIGREF0': {'text': 'the only mammal capable of sustained flying, bats are intriguing animals with unique 18 behavioral and physiological characteristics. Bats are notorious for transmission of deadly 19', 'latex': None, 'type': 'figure'}, 'FIGREF1': {'text': 'RNAi screen results converge with CRISPR screening on key pathways 30', 'latex': None, 'type': 'figure'}, 'FIGREF2': {'text': '14 of-function of genes, differences in genome coverage within the two libraries likely contributed to 15 the different hits observed in our studies. Both libraries provide valuable and complementary 16 sources that meet the urgent need to address the biological questions of bats. Similar to human 17 genomic libraries, the utility of these bat libraries in screening different cell types of bats, 18 especially those of the immune systems, would unveil more secrets of this enigmatic species. 19', 'latex': None, 'type': 'figure'}, 'FIGREF4': {'text': 'Author contributions: 16 XT, DEA and L-FW conceived and guided the study. DEA performed and analyzed RNAi screen 17 and RNAi hit validation with assistance from SYK, BGY, KS and XFL. JC and QY performed 18 CRISPR screen and hits validation. JC performed most experiments on MTHFD1. BH performed 19 SARS-CoV-2 infection assay with help from FY and PFY and under the guidance of WT. WZ, JG, 20 WL and XZ performed bioinformatics analysis of the study. XT wrote the manuscript with input 21 from all authors. 22 23 Acknowledgements: 24 We thank Fu Chengzhang and Rolf Müller for providing carolacton. We thank Jianzhong Xi for 25 oligonucleotide synthesis. XT is supported by the the China National Funds for Excellent Young 26 Scientists (31722030) and was supported by the Beijing Advanced Innovation Center for 27 Structural Biology and the Tsinghua-Peking Joint Center for Life Sciences. DEA and L-FW are 28 supported by grants NRF2012NRF-CRP001-056 and NRF2016NRF-NSFC002-013 from the 29 Singapore National Research Foundation and National Medical Research Council Cooperative 30 Basic Research Grant NMRC/BNIG/2030', 'latex': None, 'type': 'figure'}, 'FIGREF5': {'text': 'Work flows and results of CRISPR and RNAi screens in PaKi cells. a. For CRISPR 1Cas9 knock out screen, PaKi cells stably expressing Cas9 were transduced with lentiviral sgRNA 2 library targeting 21300 genes of P. alecto. The transduced cells were selected with puromycin to 3 generate the sgRNA cell library. Then the library cells were infected with influenza A virus (IAV) 4 PR8 strain (A/PuertoRico/8/1934) at lethal dose or mock infected. The surviving cells were 5 subjected to two more rounds of infection. The surviving cells at day 9 post-infection were 6 harvested and the genomic DNA were prepared and amplified for the sgRNA region for deep 7 sequencing. b. The sgRNA abundance from sequencing was analyzed by RIGER algorithm, the 8 top hits representing host factors required for cell deaths from influenza infection were labeled. c. 9For siRNA screen, siRNAs targeting 18328 genes of P. alecto were arrayed in the 96-well plates.10 The siRNAs were reverse transfected into PaKi cells and 48 h after transfection, the cells were 11 infected with mumps virus. The percentage of infected cells was calculated at 48 h post infection 12 using high content imaging. d. The siRNA targeted genes were ranked by robust Z score, and the 13 potential host factors or restriction factors were shown in the bottom or the top, respectively. e. 14 PaKi cells expressing an shRNA targeting ATP6V0D1 or a non-targeting control shRNA were 15 infected with IAV and the virus infection rate was measured by immunofluorescence using the 16 hemagglutinin antibody at 12h post infection (left). The shRNA knockdown efficiency was 17 detected by real-time PCR (right). For immunofluorescence experiments, n=3, and for real-time 18 PCR data, n=2. Data shown as mean ±standard deviation. f. PaKi cells expressing an shRNA 19 targeting SEC23B or a non-targeting control shRNA were infected with IAV PR8 and the virus 20 infection rate was measured by immunofluorescence using the hemagglutinin antibody at 12h post 21 infection (left). The shRNA knockdown efficiency was detected by real-time PCR (right). For 22 immunofluorescence experiments, n=3, and for real-time PCR data, n=2. Data shown as mean ± 23 standard deviation. g. PaKi cells were treated with indicated doses of V-ATPase inhibitor, 24 bafilomycin A1, and infected with IAV. The infection rate was measured by immunofluorescence 25 using the hemagglutinin antibody at 12h post infection. Data shown as the mean ±standard 26 deviation (n=3). *: P<0.05, **: P<0.01, MTHFD1 is host factor for replication of multiple RNA viruses. a. MTHFD1 30 knockout clonal PaKi cells were generated from two independent sgRNAs and infected with 31', 'latex': None, 'type': 'figure'}, 'FIGREF6': {'text': \"SEC61B followed by mumps virus(d) and Melaka virus (f) infection. The virus in the 7 supernatant were tittered by limiting dilution. The knockdown efficiency was determined by real-8 time PCR and shown in (e). For d and e n=3, and for f, n=4. Data shown as the mean ±standard 9 deviation. g. PaKi cells were transfected with siNT or siRNA targeting MTHFD1, 48 h later the 10 cells were infected with Zika virus. The Zika virus RNA level was measured by real-time PCR. 11 For g, n=4. Data shown as the mean ±standard deviation. h-j. PaKi shRNA control cells (shNT) 12 and shMTHFD1 cells were infected with mumps virus (h) or Zika virus (i), the virus infection 13 level was measured by immunofluorescence. For h, n=2 and for i, n=3. Data shown as the mean 14 ± standard deviation. The shRNA knockdown efficiency was tested by Western blotting as 15 showing in (j). Data shown are representative of three independent experiments. k. Luciferase or 16 shRNA resistant MTHFD1 construct were stably expressed in shMTHFD1-2 PaKi cells, then the 17 cells were infected with Zika virus. The rescue level of Zika virus was analyzed by Western 18 blotting. Data shown are representative of three independent experiments. For all bar graphs, the 19 mean ± standard deviation values are shown. *: P<0.05, **: P<0.01, ***: P<0.001. 20 MTHFD1's formyl tetrahydrofolate synthetase activity is essential for RNA virus 22 replication, a. MTHFD1 is a trifunctional enzyme in the one carbon metabolism which is required 23 for purines, dTMP and methionine synthesis. b. Domain structures and key amino acids of 24 MTHFD1 with key residues for the three enzymatic functions indicated. c. PaKi MTHFD1 knock 25 out cells were infected with IAV with or without supplementation of hypoxanthine or thymidine 26 for 12 h and the virus infection level was detected by immunofluorescence. Data shown as the 27 mean ±standard deviation (n=3). d. PaKi shMTHFD1-2 cells or control cells with a non-targeting 28 shRNA were infected with Zika virus in regular medium or in medium supplemented with inosine, 29 thymidine or 5-CH 3 THF. Data shown as the mean ±standard deviation (n=3). e. PaKi shMTHFD1 30 cells were infected with Zika virus in regular medium or in medium supplemented with inosine. 1 At 48 h post infection, the virus in the supernatant was harvested and the titre was determined by 2 limiting dilution in PaKi cells. Data shown as the mean ±standard deviation (n=2). f. PaKi 3 shMTHFD1-2 cells or control cells with a non-targeting shRNA were transfected with luciferase 4 (negative control), human wild type MTHFD1, bat wildtype MTHFD1 or bat MTHFD mutants, 5 followed by Zika virus infection. The virus infection rate was measured by immunofluorescence. 6 Data shown as the mean ±standard deviation (n=3). g. Representative images from (f). Scale bar: 7 100 µm. h. HEK293T MTHFD1 knock out cells or control HEK293T cells with a non-targeting 8 sgRNA (sgNT) were infected with PR8 with or without the supplementation of inosine, thymidine 9 or 5-CH 3 THF. At 12 h post infection, the cells were collected and analyzed by Western blotting. 10 Data shown are representative of three independent experiments. i. HEK293T MTHFD1 knockout 11 cells (sgMTHFD1-2) or control cells (sgNT) were infected with IAV. The vRNA level was 12 determined by real-time PCR at the indicated time points. The results shown representative of 13 three independent experiments j. HEK293T MTHFD1 knockout cells (sgMTHFD1-2) or control 14 cells (sgNT) were transfected with plasmids expressing the influenza virus mini-genome including 15 polymerase subunits PB1, PB2 and PA, and nucleoprotein together with pPOLI-HA, which can 16 transcribe the hemagglutinin vRNA of influenza virus with or without inosine supplementation. 17 At 16h, 28h, 40h and 52h post transfection, the cells were collected and analyzed by Western 18 blotting. Data shown are representative of three independent experiments. For all bar graphs, the 19 mean ± standard deviation values are shown. *P<0.05. 20 MTHFD1 inhibitor carolacton can inhibit a number of RNA viruses including 22 SARS-CoV-2. a. PaKi cells were treated with different concentrations of carolacton and infected 23 with mumps virus expressing EGFP or Zika virus. Virus infection level was detected by EGFP or 24 immunofluorescence staining of Zika E protein. The cell viability was measured by MTT assay. 25 n=3. Data shown as the mean ± standard deviation. b. Representative images of 26 immunofluorescence staining of mumps infection from (a). c. PaKi cells were treated with 27 carolacton and infected with Zika virus in the presence of inosine or mock treated. Representative 28 images of immunofluorescence staining of Zika virus E protein were shown. Infection rate shown 29 as the mean ±standard deviation (n=3). d. PaKi cells were treated with carolacton, and 1h later 30\", 'latex': None, 'type': 'figure'}, 'FIGREF7': {'text': 'Vero and 293T cells were cultured in DMEM supplemented with 10% heat inactivated 9 FBS. Influenza H1N1 PR8 (A/Puerto Rico/8/1934) was propagated in embryonated eggs. Mumps 10 virus was propagated in PaKi or Vero cells. 11 12 Genome-wide CRISPR-Cas9 knockout screen 13 Cas9 stably expressed bat kidney derived PaKi cells were transduced with lentivirus of 14 sgRNA library targeting 21300 genes in the genome of P. alecto. The transduced cells were 15 selected with puromycin to generate the library cells. Then the sgRNA library cells were mock 16 infected (negative control) or changed with influenza A H1N1 PR8 strain (A/Puerto Rico/8/1934) 17 at lethal dose. The surviving cells were subjected to two more round infection. The final surviving 18 cells were applying to deep sequencing. And the sgRNA abundance was analyzed by RIGER 19 algorithm 1 . 20 21', 'latex': None, 'type': 'figure'}, 'FIGREF8': {'text': 'mini-genome assay 22 Approximately 2x10 5 of 293T control cells or 293T MTHFD1 knock out cells were plated 23 into each well of 12-well plates and the knockout cells were cultured with HT (100 µM 24 hypoxanthine and 16 µM thymidine, Gibco) medium. Then the cells were transfected with the 25 plasmids expressing viral polymerase subunits PB1, PB2 and PA, and nucleoprotein, together with 26 pPOLI-HA, transcribing hemagglutinin vRNA segment of A/WSN/33(H1N1) (WSN). At the time 27 of transfection, HT was removed from the cells and supplied with fresh medium with or without 28 inosine (100 µM). The HA protein level were analyzed at 16 h', 'latex': None, 'type': 'figure'}, 'TABREF0': {'text': '. We show that MTHFD1 knockdown blocks replication of 28 multiple RNA viruses, including influenza virus, mumps virus, Melaka virus and Zika virus. Importantly, host cells have a higher tolerance for MTHFD1 inhibition than viruses, potentially 30 providing a therapeutic window for targeting MTHFD1 with antiviral drugs. Virus infection 31', 'latex': None, 'type': 'table'}, 'TABREF1': {'text': 'P. S. et al. Comparative Flavivirus-Host Protein Interaction Mapping Reveals 22 Mechanisms of Dengue and Zika Virus Pathogenesis. Cell 175, 1931-1945 e1918,', 'latex': None, 'type': 'table'}, 'TABREF2': {'text': 'The knock out or knock down efficiency was 15 tested by Western blotting. To facilitate the growth of single clone MTHFD1 knock out cells, the 16 cells were cultured with HT medium (DMEM supplemented with 10% FBS, 100µM hypoxanthine 17 and 16µM thymidine).The knock out or knock down efficiency was tested by Western blotting. 18 Validated knockout cells were applied to virus infection assays. For MTHFD1 knock out cells, the 19 HT was removed 8h before virus infection. 20', 'latex': None, 'type': 'table'}}, 'back_matter': [{'text': 'humans. a. livers and spleens of human and two bat species were applied to RNAseq, and MTHFD1 mRNA level was analyzed. b. MTHFD1 mRNA levels in difference cell lines were compared using qRTPCR with GAPDH as a reference gene. c. MTHFD1 protein levels in different cell lines was compared by Western blotting. d. promoter analysis of MTHFD1 between human and two bat species (P. alecto and E. spelaea). The transcription factor binding sites and structure of promotors were characterized by the help of GPMiner (http://gpminer.mbc.nctu.edu.tw/) and the related information of the regulatory features such as transcription factor binding site (TFBS), G + C content,', 'cite_spans': [], 'ref_spans': [], 'section': 'Supplementary Figure 5. Bats have a lower MTHFD1 expression level compared'}]}\n",
      "{'paper_id': 'b3cb1d91dcd7f78dd9bbc88b93d2403eb6869746', 'metadata': {'title': 'Genetic control of the HDL proteome', 'authors': [{'first': 'Nathalie', 'middle': [], 'last': 'Pamir', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Oregon Health & Sciences University', 'location': {'settlement': 'Portland', 'region': 'OR'}}, 'email': ''}, {'first': 'Calvin', 'middle': [], 'last': 'Pan', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of California at Los Angeles', 'location': {'settlement': 'Los Angeles', 'region': 'CA'}}, 'email': ''}, {'first': 'Deanna', 'middle': ['L'], 'last': 'Plubell', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Oregon Health & Sciences University', 'location': {'settlement': 'Portland', 'region': 'OR'}}, 'email': ''}, {'first': 'Patrick', 'middle': ['M'], 'last': 'Hutchins', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Washington', 'location': {'postCode': '98102', 'settlement': 'Seattle', 'region': 'WA'}}, 'email': ''}, {'first': 'Chongren', 'middle': [], 'last': 'Tang', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Washington', 'location': {'postCode': '98102', 'settlement': 'Seattle', 'region': 'WA'}}, 'email': ''}, {'first': 'Jake', 'middle': [], 'last': 'Wimberger', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Washington', 'location': {'postCode': '98102', 'settlement': 'Seattle', 'region': 'WA'}}, 'email': ''}, {'first': 'Angela', 'middle': [], 'last': 'Irwin', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Washington', 'location': {'postCode': '98102', 'settlement': 'Seattle', 'region': 'WA'}}, 'email': ''}, {'first': 'Thomas', 'middle': ['Q'], 'last': 'De Aguiar Vallim', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of California at Los Angeles', 'location': {'settlement': 'Los Angeles', 'region': 'CA'}}, 'email': ''}, {'first': 'Jay', 'middle': ['W'], 'last': 'Heinecke', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of Washington', 'location': {'postCode': '98102', 'settlement': 'Seattle', 'region': 'WA'}}, 'email': ''}, {'first': 'Aldons', 'middle': ['J'], 'last': 'Lusis', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'University of California at Los Angeles', 'location': {'settlement': 'Los Angeles', 'region': 'CA'}}, 'email': ''}]}, 'abstract': [{'text': \"High-density lipoproteins (HDL) are nanoparticles with >80 associated proteins, phospholipids, cholesterol and cholesteryl esters. A comprehensive genetic analysis of the regulation of proteome of HDL isolated from a panel of 100 diverse inbred strains of mice, Hybrid Mouse Diversity Panel (HMDP), revealed widely varied HDL protein levels across the strains. Some of this variation was explained by local, cis-acting regulation, termed cis-protein quantitative trait loci. Variations in apolipoprotein A-II and apolipoprotein C-3 affected the abundance of multiple HDL proteins indicating a coordinated regulation. We identified modules of co-varying proteins and define a protein-protein interaction network describing the protein composition of the naturally occurring subspecies of HDL in mice. Sterol efflux capacity varied up to 3-fold across the strains and HDL proteins displayed distinct correlation patterns with macrophage and ABCA1 specific cholesterol efflux capacity and cholesterol exchange, suggesting that subspecies of HDL participate in discrete functions. The baseline and stimulated sterol efflux capacity phenotypes associated with distinct QTLs with smaller effect size suggesting a multi genetic regulation. Our results highlight the complexity of HDL particles by revealing high degree of heterogeneity and intercorrelation, some of which is associated with functional variation, supporting the concept that HDL-cholesterol alone is not an accurate measure of HDL's properties such as protection against CAD.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}], 'body_text': [{'text': \"as well as larger HDL species that contain a large core of cholesteryl esters (Kontush et al., 2013; Yetukuri et al., 2010) . In humans, discrete classes of HDL based on size can be identified. In mice, HDL sizes are more continuous (LeBoeuf and Lusis, 1983 ) and represent one monodisperse peak. The size and levels of HDL vary in both human and mouse populations (Joshi et al., 2016; Pamir et al., 2016) . There are clear functional differences associated with the various size classes of HDL. In particular, the small lipid-poor particles are the best acceptors of cholesterol from cells and thus should be particularly important in mediating reverse cholesterol transport, whereas larger particles, associated with proteins such as paraoxonase 1 (PON1) and APOE, are likely to be important in protecting against inflammation. HDL particles containing proteins such as serum amyloid A (SAA) species tend to lack anti-inflammatory properties (Vaisar et al., 2015) . HDL-C levels have a skewed normal distribution in the general population, and the median levels vary by sex and ethnicity. Linkage based studies from the early 1980's have tried to identify the genetic factors that influence plasma HDL-C levels, but many findings have not been replicated due to polygenic nature of this trait, with contributions from multiple small-effect gene variants. Meta-analyses and GWAS results do, however, support the association of HDL-C with variation in CETP, LIPC, LPL, ABCA1, endothelial lipase (LIPG) and LCAT (Hegele, 2009; Thompson et al., 2008) . Multiple genetic factors could be present in an individual, creating a polygenic network of HDL-C determinants (Cohen et al., 2004) . These determinants include monogenic effectors, such rare homozygous mutations in ABCA1, LCAT, and APOA1 causing extremely low HDL-C (Brooks-Wilson et al., 1999; Kuivenhoven et al., 1996; Ng et al., 1994) , and rare homozygous mutations in CETP, LIPC, and SCARB1, causing extremely elevated HDL-C . The mouse models of these variants have been supportive of the human findings (Wang and Paigen, 2005) (Wang & Paigen, 2005) . Polygenic determinants have been recently investigated using targeted next-generation sequencing in patients with extremely low and high HDL-C. About 30% of individuals at the extremes of HDL-C had rare large effect and common small effect variants explaining the trait (Dron et al., 2017) . Whereas the genetic determinants of plasma HDL-C levels have been well studied, the genetic determinants of HDL proteome and lipidome have never been previously investigated.\", 'cite_spans': [{'start': 78, 'end': 100, 'text': '(Kontush et al., 2013;', 'ref_id': 'BIBREF22'}, {'start': 101, 'end': 123, 'text': 'Yetukuri et al., 2010)', 'ref_id': None}, {'start': 233, 'end': 257, 'text': '(LeBoeuf and Lusis, 1983', 'ref_id': None}, {'start': 365, 'end': 385, 'text': '(Joshi et al., 2016;', 'ref_id': None}, {'start': 386, 'end': 405, 'text': 'Pamir et al., 2016)', 'ref_id': None}, {'start': 944, 'end': 965, 'text': '(Vaisar et al., 2015)', 'ref_id': 'BIBREF24'}, {'start': 1511, 'end': 1525, 'text': '(Hegele, 2009;', 'ref_id': None}, {'start': 1526, 'end': 1548, 'text': 'Thompson et al., 2008)', 'ref_id': 'BIBREF21'}, {'start': 1662, 'end': 1682, 'text': '(Cohen et al., 2004)', 'ref_id': 'BIBREF8'}, {'start': 1818, 'end': 1846, 'text': '(Brooks-Wilson et al., 1999;', 'ref_id': 'BIBREF3'}, {'start': 1847, 'end': 1872, 'text': 'Kuivenhoven et al., 1996;', 'ref_id': None}, {'start': 1873, 'end': 1889, 'text': 'Ng et al., 1994)', 'ref_id': None}, {'start': 2062, 'end': 2085, 'text': '(Wang and Paigen, 2005)', 'ref_id': None}, {'start': 2086, 'end': 2107, 'text': '(Wang & Paigen, 2005)', 'ref_id': None}, {'start': 2380, 'end': 2399, 'text': '(Dron et al., 2017)', 'ref_id': None}], 'ref_spans': [], 'section': ''}, {'text': 'To better define the various species of HDL at the level of protein composition and to understand their genetic regulation we used the Hybrid Mouse Diversity Panel (HMDP) with a systems biology approach (Bennett et al., 2010; Ghazalpour et al., 2012) . HMDP is a collection of 100 classical laboratory and recombinant inbred (RI) strains that have been genotyped at 135,000 SNPs (Bennett et al., 2010) . HMDP provides a confined genetic space (Bennett et al., 2010) that rely on naturally occurring genetic variation that perturbs protein abundance. We performed systems genetics approach using analytical approaches that include genome-wide association, expression quantitative trait locus (eQTL) discovery, functional outcomes and network analysis (Bennett et al., 2010) .', 'cite_spans': [{'start': 203, 'end': 225, 'text': '(Bennett et al., 2010;', 'ref_id': 'BIBREF1'}, {'start': 226, 'end': 250, 'text': 'Ghazalpour et al., 2012)', 'ref_id': None}, {'start': 379, 'end': 401, 'text': '(Bennett et al., 2010)', 'ref_id': 'BIBREF1'}, {'start': 443, 'end': 465, 'text': '(Bennett et al., 2010)', 'ref_id': 'BIBREF1'}, {'start': 750, 'end': 772, 'text': '(Bennett et al., 2010)', 'ref_id': 'BIBREF1'}], 'ref_spans': [], 'section': ''}, {'text': 'We identified HDL proteome and HDL function for each strain. Using quantitative trait locus (QTL) analyses, we performed genetic mapping. First, associations between SNPs and HDL protein levels and function were determined. Second, the effects of SNPs on gene regulation were determined by eQTL analysis using hepatic and adipose tissue-specific gene expression. Based on the genetic variation in HDL observed, we were able to identify numerous genetic factors mediating HDL composition and provide an approximation of the nature of the interactions between the proteins. Finally, we used protein-protein interaction cluster analyses to build an HDL model that describes the group of proteins that constitute the core proteins and the ones that are peripheral. We identified moderate QTLs associated with HDLs sterol efflux capacity. Our results reveal a great deal of heterogeneity and inter-correlation, some of which is of proteins that are acutely regulated by the environment (ie. Diet and inflammationhttp://homepages.uc.edu/~davidswm/HDLproteome.html). The H2 scores for such proteins are expected to be zero. Furthermore, due to our study design, the proteins that are strain specific are also expected to have a H2 of zero.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'Loci contributing to variations in protein levels (protein QTLs or pQTLs) or hepatic transcript levels of the proteins (expression QTLs or eQTLs) were mapped using the FAST LMN, an association algorithm with a mixed model component to correct for population structure. Association analysis was performed using about 200,000 informative SNPs (Bennett et al., 2010) , spaced throughout the genome (Table 2 ). Hepatic transcript levels were from a previous survey of the HMDP maintained on the same chow diet as in this study (Bennett et al., 2010) . Loci averaged 500 kb to 2 Mb in size and in most cases contained 1 to 20 genes within a linkage disequilibrium (LD) block, an improvement of more than an order of magnitude as compared to traditional linkage analysis in mice (typically a resolution of 10 to 20 Mb) (Flint et al., 2005) . Loci mapping within 1Mb of the gene are termed \"local\" QTL, suggesting that they probably act in cis. For example, promoter or enhancer variants would act in cis. Loci mapping greater than 1Mb from the gene of interest are termed \"distal\", implying that they act in trans, presumably mediated by a diffusible factor such as a transcription factor (Table 2 and  Supplemental Table 2 ). We applied a significance filter of P=10 -3 and 10 -6 to identify suggestive cis and trans QTLs respectively.', 'cite_spans': [{'start': 341, 'end': 363, 'text': '(Bennett et al., 2010)', 'ref_id': 'BIBREF1'}, {'start': 523, 'end': 545, 'text': '(Bennett et al., 2010)', 'ref_id': 'BIBREF1'}, {'start': 813, 'end': 833, 'text': '(Flint et al., 2005)', 'ref_id': None}], 'ref_spans': [{'start': 395, 'end': 403, 'text': '(Table 2', 'ref_id': 'TABREF5'}, {'start': 1183, 'end': 1217, 'text': '(Table 2 and  Supplemental Table 2', 'ref_id': 'TABREF5'}], 'section': 'Genetic regulation of HDL proteome'}, {'text': 'A total of 19 HDL proteins showed significant evidence of local regulation of hepatic transcript levels or protein levels (Table 2) . With the exception of Apoc3, all of the significant pQTL also exhibited significant eQTL, indicating that genetic variation in protein levels was largely due to regulation of gene expression. In the case of Apoc3, while there was no significant eQTL in liver, there was a highly significant eQTL in adipose tissue (P=1.8 x 10 -7 ) (Supplemental Table  3 ). Apoa2 and Saa2 exhibited much more significant pQTL than eQTL (P=1.324e-22, effect size = -0.413 vs. P=1.276e-4, effect size = 0.0210 for Apoa2, and P=2.559e-10, effect size = 0.0780 and P=1.533e-6, effect size = 0.780 for Saa2 respectively), suggesting that the pQTL were due to coding rather than regulatory variations. In the case of Apoa2, our previous studies indicated that structural variation affecting translation efficiency was largely responsible for the differences in protein levels among several common inbred strains (Bennett et al., 2010) . Also, common coding variants are present among the HMDP strains for both Apoe3 and Saa2 (www.informatics.jax.org). As shown in Table 2 , many variants significantly affecting HDL protein expression did not exhibit corresponding variations in protein levels. A likely explanation in the case of HDL is that, for some proteins, only a limited amount of the protein can be incorporated into the HDL lipid-protein complex, the remainder presumably being degraded.', 'cite_spans': [{'start': 1023, 'end': 1045, 'text': '(Bennett et al., 2010)', 'ref_id': 'BIBREF1'}], 'ref_spans': [{'start': 122, 'end': 131, 'text': '(Table 2)', 'ref_id': 'TABREF5'}, {'start': 479, 'end': 487, 'text': 'Table  3', 'ref_id': 'TABREF6'}, {'start': 1175, 'end': 1182, 'text': 'Table 2', 'ref_id': 'TABREF5'}], 'section': 'Genetic regulation of HDL proteome'}, {'text': 'We have attempted to identify distal (trans-acting) factors affecting HDL protein levels (Supplemental Table 2 ). In contrast to local eQTL, where only SNPs within 1Mb of the gene are tested for association, distal QTL analyses involve genome wide SNP tests for association, requiring a much higher threshold for significance. A number of the likely significant distal eQTL occur with several Mb of the gene and, thus, probably result from either long range (>1 Mb) linkage disequilibrium or chromosome looping interactions (for example Apoh, Apom, B2m, H2-Q10, and Pp1c) (Supplemental Table 2 ). The pQTL analysis also identified some highly significant distal (trans-acting) interactions, most notably for Apoa2 (p=4.6 x 10 -14 ). The Apoa2 locus is about 5 Mb from the structural gene and the significant association is probably the result of linkage disequilibrium or chromosome looping.', 'cite_spans': [], 'ref_spans': [{'start': 103, 'end': 110, 'text': 'Table 2', 'ref_id': 'TABREF5'}, {'start': 586, 'end': 593, 'text': 'Table 2', 'ref_id': 'TABREF5'}], 'section': 'Genetic regulation of HDL proteome'}, {'text': 'We asked if the local pQTL could be used to identify causal interactions between HDL proteins. For this, we selected the genes with significant pQTL and asked whether the peak local pQTL SNP was associated with any other HDL proteins, suggesting that the regulation of the former perturbs levels of the latter. For example, Apoe3 (on chr. 9) protein levels were controlled by a local pQTL with peak SNP 46673334 (chr. 9) and the same SNP was significantly associated with the levels of Podx1 (chr. 6, p=4x10 -3 ), Fetub (chr. 16, p=5.5x10 -3 ), Apoc2 (chr. 7, p=5.9x10 -3 ), and a number of other proteins. Likewise, a local pQTL SNP for Apoa2 was associated with Apoc3 and Itgb3 levels and an Hq-Q10 local pQTL SNP was associated with Apob and Apoe levels (Supplemental Table 4 ).', 'cite_spans': [], 'ref_spans': [{'start': 771, 'end': 778, 'text': 'Table 4', 'ref_id': 'TABREF7'}], 'section': 'Genetic regulation of HDL proteome'}, {'text': 'To understand interactions of HDL proteins with each other and with other metrics of HDL (ABCA1 specific sterol efflux, baseline -diffusion sterol efflux, HDL particle size, and HDL cholesterol), we correlated proteins that are present in more than 80% of the strains. We then applied hierarchical clustering to a matrix that contains all measured phenotypes. The clustering of the proteome and functional metrics revealed expected patterns (Figure 3 for yeast normalized and Supplemental Figure 3 for total normalized data). The complex correlation matrices represented a high number of strongly correlated variables suggesting an organized interplay among HDL proteins and between the physiological and functional metrics of HDL ( Figure. 4 and Supplemental Figure 4 for yeast and total normalized data respectively). Of the 8100 total correlations, we have focused on 2216 correlations that are |r|>0.5 with a Bonferroni-Holm adjusted P<0.05, n=2216. The correlation and P values are presented in Supplemental Table 2 .', 'cite_spans': [], 'ref_spans': [{'start': 441, 'end': 450, 'text': '(Figure 3', 'ref_id': None}, {'start': 489, 'end': 497, 'text': 'Figure 3', 'ref_id': None}, {'start': 733, 'end': 740, 'text': 'Figure.', 'ref_id': None}, {'start': 760, 'end': 768, 'text': 'Figure 4', 'ref_id': 'FIGREF3'}, {'start': 1013, 'end': 1020, 'text': 'Table 2', 'ref_id': 'TABREF5'}], 'section': 'Clustering of HDL metrics based on quantitation across strains'}, {'text': 'All sterol efflux metrics displayed up to 4-fold differences across strains (Supplementary Figure  7) . The non-proteome phenotypes clustered together; for example, unstimulated sterol efflux from two different cell types, min.cAMP and min.MF and stimulated sterol efflux (r= 0.909, P=10 -15 ), plus. cAMP and plus.MF (r=0.83, P=10 -15 ) correlated strongly and were part of the same cluster that is modestly associated with APOA1 (Figure 3 and Supplementary Table 2 ). The ABCA1 mediated sterol efflux, delta BHK and delta J774, correlated strongly (r=0.73, P=10 -15 ) and formed a cluster along with HDL-C and medium size HDL where the latter two metrics correlated strongly (r=0.78, P=1.82e-12) (Figure 3 and Supplementary Table 5) .', 'cite_spans': [], 'ref_spans': [{'start': 91, 'end': 101, 'text': 'Figure  7)', 'ref_id': None}, {'start': 431, 'end': 466, 'text': '(Figure 3 and Supplementary Table 2', 'ref_id': 'TABREF5'}, {'start': 698, 'end': 734, 'text': '(Figure 3 and Supplementary Table 5)', 'ref_id': 'TABREF6'}], 'section': 'Clustering of HDL metrics based on quantitation across strains'}, {'text': 'HDL-C was negatively associated with ABCA1 mediated sterol efflux from both BHK and J774 cells (r=-0.58, P=0.00036 and -0.49, P=0.03 respectively) and was positively associated with the diffusional (unstimulated) efflux from both cell types (r=0.62 P=6.14e-06 for BHK and r=0.64 P=2.04e-05 for J774).', 'cite_spans': [], 'ref_spans': [], 'section': 'Clustering of HDL metrics based on quantitation across strains'}, {'text': 'Mice have 85% of their HDL cholesterol distributed in the 7.6-9.8nm range (medium size) in a monodispersed peak (Pamir et al, 2016) , and therefore medium size HDL represents the majority of HDL cholesterol. This latter cluster is associated with APOA2 levels ( Figure 3 ). Further, proteins that participate in immune responses, including serum amyloids SAA1, SAA2, histocompatibility complexes H2-Q4 and H2-Q10 associated strongly and formed a distinct cluster with other immune response proteins CD97, C4BPA and APOE. All of the hemoglobin proteins, HBBA, HBB1, HBBT1 formed a distinct cluster. Lipid metabolism proteins, APOC1, APOA5, APOA4, PLTP, and GPLD1 clustered together. Most interestingly, APOC3 correlated negatively with 39 proteins on HDL. Among these were proteins with roles in immune response such as APOE, PON1, SAA1, SAA4 (Supplementary Table 5 ). insulin like growth factor binding protein, IGFALS clustered strongly but negatively with APOC3 (r=-0.53, P=0.002). Recent studies suggest a role for APOC3 in beta cell insulin resistance (Åvall et al, 2015) and according to proteome interactome by Harmonizome (a tool curated from 100 public databases (Rouillard et al, 2016) ) APOC3 is one of the 73 proteins found to interact with IGFALS. We observed significant correlations between HDL-C levels and GM94, ITGB1, APOC1, APOC2 (positive) and APOA2, SERPINA1A, ALB, and TFRC (negative) regardless of PSM normalization method used (Supplemental Table 5 ). All the New Zealand strains studied (NZB BIN/J, NZW LAC/J, KKHI/J) were in the top quintile of HDL-C distribution and bottom quintile of APOA2 distribution. The QTL analysis indicated the same genomic regions for regulation of APOA2 and HDL-C levels (Supplemental Figure 8) . APOA2 seems to be the main genetic regulator of plasma HDL-C levels in mice.', 'cite_spans': [{'start': 112, 'end': 131, 'text': '(Pamir et al, 2016)', 'ref_id': None}, {'start': 1056, 'end': 1075, 'text': '(Åvall et al, 2015)', 'ref_id': None}, {'start': 1171, 'end': 1194, 'text': '(Rouillard et al, 2016)', 'ref_id': None}], 'ref_spans': [{'start': 262, 'end': 270, 'text': 'Figure 3', 'ref_id': None}, {'start': 842, 'end': 864, 'text': '(Supplementary Table 5', 'ref_id': 'TABREF12'}, {'start': 1464, 'end': 1471, 'text': 'Table 5', 'ref_id': 'TABREF12'}, {'start': 1739, 'end': 1748, 'text': 'Figure 8)', 'ref_id': None}], 'section': 'Clustering of HDL metrics based on quantitation across strains'}, {'text': 'Among the HDL proteins, APOD, with high homology to carrier proteins such as lipocalins and with strong innate immune response roles such as antioxidative (Ganfornina et al., 2008) and neuroprotective effects (Do Carmo et al., 2008) had the most, significant correlations with other HDL proteins involved in immune response. Twenty of these correlations exhibited |r|>0.7 suggesting that it is highly interactive apolipoprotein that acts as a carrier for other proteins on HDL.', 'cite_spans': [{'start': 155, 'end': 180, 'text': '(Ganfornina et al., 2008)', 'ref_id': None}, {'start': 209, 'end': 232, 'text': '(Do Carmo et al., 2008)', 'ref_id': 'BIBREF11'}], 'ref_spans': [], 'section': 'Clustering of HDL metrics based on quantitation across strains'}, {'text': 'Another way to visualize the relationships among the proteins is to present these as a correlation network ( Figure 5 and Supplemental Figure 5 ). The network consists of multiple layers of spatial organization, a core, an outer layer and the periphery. The \"core\" proteins, including AHSG, NAPSA, PLG, SAA4, HBA, APOD, APOJ, APOM, APOH, FGA, and TFPI are surrounded by common HDL associated apolipoproteins such as APOA4, APOA5, APOC1, APOC3, APOC4 in addition proteins that have been described to be associated with HDL in most proteomic studies ( Davidson HDL proteome watch http://homepages.uc.edu/~davidswm/HDLproteome.html) such as H2-Q4, H2-Q10, SELL, ANTRX1/2, PF4 RBP4, SERPINs, GPLD1, FGA and FGB. More peripheral are known associated proteins such as PLTP, PON3, LCAT, APOE, SAA1, SAA2. HDL core proteins formed a tight network suggesting that they are co-regulated. Inflammation response and complement activation proteins such as SAA1, SAA2, H2-Q4, H2-Q10, C3, C4b, C4BPA were distal to the core co-regulatory network suggesting that they are mostly regulated by external factors such as an inflammatory stimulus. The two major structural proteins, APOA1 and APOA2 along with APOC3 were on the outer layers and periphery and are negatively correlated with other proteins suggesting that their presence requires the displacement of other proteins.', 'cite_spans': [], 'ref_spans': [{'start': 109, 'end': 117, 'text': 'Figure 5', 'ref_id': 'FIGREF4'}, {'start': 135, 'end': 143, 'text': 'Figure 5', 'ref_id': 'FIGREF4'}], 'section': 'Clustering of HDL metrics based on quantitation across strains'}, {'text': 'The sterol efflux capacity was measured in two different cell lines (J774 and BHK) in basal (no ABCA1) and stimulated (with ABCA1) conditions, with ABCA1 specific sterol efflux calculated by subtracting basal from stimulated. The QTL analysis for the efflux traits revealed shared global association profiles that are condition specific and in agreement between cell types (Supplemental Figure 6 A and B) . The significant QTLs that are shared by both cell types are presented in Table 4 . The lack of strong QTLs that associate with sterol efflux capacity of HDL suggest that either sterol efflux is under limited genetic control or its control is multigenic with small effect sizes.', 'cite_spans': [], 'ref_spans': [{'start': 387, 'end': 404, 'text': 'Figure 6 A and B)', 'ref_id': None}, {'start': 480, 'end': 487, 'text': 'Table 4', 'ref_id': 'TABREF7'}], 'section': 'Genetic Regulation of Sterol Efflux'}, {'text': 'The HMDP strains have been examined in separate studies (Lusis et al., 2016) either on chow (Bennett et al., 2010) , high fat/high sucrose (Parks et al., 2013; or high cholesterol diets (Bennett et al., 2015) 2. We compared our data with results from these studies, reasoning that while power to detect correlation will be diminished given that the data were collected in separate animals and at different times with incomplete overlap of strains, traits significantly impacted by genetics should still retain some correlation structure. The most significant correlations between HDL proteins and clinical traits are summarized in Table 3 and detailed in Supplemental Table 6. A total of 22 HDL proteins significantly and positively correlated with lesion area in the hypercholesterolemia study (Bennett et al., 2015) among which Plasminogen (PLG), Immunoglobin chain C region (IGHM), Platelet Factor 4 (PF4) were strongest (r=0.469, P=0.00026; r=0.461, P=0.00034, r=0.37, P=0.0049). We have recently shown that plasminogen is an effective sterol acceptor through ABCA1 and this could be the mechanism by which plasminogen contributes to atherosclerosis (Pamir et al., 2017) .', 'cite_spans': [{'start': 92, 'end': 114, 'text': '(Bennett et al., 2010)', 'ref_id': 'BIBREF1'}, {'start': 139, 'end': 159, 'text': '(Parks et al., 2013;', 'ref_id': None}, {'start': 186, 'end': 208, 'text': '(Bennett et al., 2015)', 'ref_id': 'BIBREF0'}, {'start': 795, 'end': 817, 'text': '(Bennett et al., 2015)', 'ref_id': 'BIBREF0'}, {'start': 1154, 'end': 1174, 'text': '(Pamir et al., 2017)', 'ref_id': None}], 'ref_spans': [{'start': 631, 'end': 638, 'text': 'Table 3', 'ref_id': 'TABREF6'}, {'start': 668, 'end': 676, 'text': 'Table 6.', 'ref_id': None}], 'section': 'Lipid and clinical trait interactions'}, {'text': 'Conforming to the recently identified role of APOC3 in insulin resistance, HOMA-IR plasma insulin levels, body weight and adiposity correlated positively with HDL associated APOC3 levels (r=0.50, P=0.00017; r=0.44 P=0.0015; r=0.23, P=0.048; r=0.26, P=0.021, respectively).', 'cite_spans': [], 'ref_spans': [], 'section': 'Lipid and clinical trait interactions'}, {'text': 'We report the analysis of the HDL proteome across a set of 93 inbred strains of mice exhibiting common genetic variation. The genetic variation across this mouse panel resembles that in human populations, based on the number of common SNPs (about 4 million). Previous studies in mice have revealed genetic variations in the levels of HDL-cholesterol, HDL apoproteins, and HDL composition (Lusis et al., 1983) . Our results are consistent with a high level of heritability of HDL proteins, including the identification of a number of novel pQTLs. One significant conclusion that has emerged is that certain proteins cluster together in response to genetic perturbations, presumably reflecting physical or regulatory interactions. These clusters then help define the heterogeneity of HDL particles, although our analyses do not address lipid heterogeneity. Another important conclusion is evidence of a relationship between protein composition and HDL function. Finally, we have identified potential links between HDL proteins and various clinical or molecular traits studied previously in the HMDP strains. We discuss each of these points in turn below.', 'cite_spans': [{'start': 334, 'end': 408, 'text': 'HDL-cholesterol, HDL apoproteins, and HDL composition (Lusis et al., 1983)', 'ref_id': None}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Normalization of shotgun proteomic data is a continuous struggle in the field (Välikangas et al., 2018) . HDL is a rather uncomplicated mixture containing only ~100 proteins. However, its proteome is driven by the 10 most abundant proteins, with 65% being made up of APOA1 and 15% of APOA2 (Toth et al., 2013) . The normalization strategy should conserve the compositional bias of the HDL. The normalization that directly adjusts scale such as Total Count (TC) and Upper Quartile (UQ) fails to accommodate compositional bias. The normalization strategies that adjust scales using landmarks in the distribution Median, (Med), Differentially expressed (DESeq), Trimmed mean (TMM) are promising approaches for HDL proteome however, detailed analyses need to be performed for their validation to be used on smaller libraries of stochastic count data from mass spectrometry. The quartile (Q) and reads per kilobase per million mapped (RPKM) (equivalent of normalizing spectral counts to protein length) have adverse effects on intra-sample variance and on distribution bias (Dillies et al., 2013) . The TC and UQ normalizations favor the most abundant proteins and are unfriendly for mixtures with a distribution bias. That said, TC normalization is often the preferred method for shotgun HDL proteomics as it controls for differences in instrument response, digestion efficiency, and amounts of loaded protein digest but fails at conserving the distribution bias as it tends to accommodate the changes in the abundant proteins (Vaisar et al., 2007) . That is partly why the HDL protein quantification by shotgun proteomics is not optimal and the correlation with immunobased assays are moderate (Hoofnagle et al., 2012) . Therefore, we opted to include a second normalization approach by spiking yeast carboxy peptidase at levels ~8 fold lower than APOA1 and to correspond to the median/mean abundance of the typical HDL proteome (Carvalho et al., 2008; Liu et al., 2004) . The QTLs identified using both proteomic information are mostly overlapping with TC normalization resulting in ~30% more significant QTLs. The relationships among inbred lines of mice was inferred from the high-density SNP map where strains cluster according to their genealogy (Cervino et al., 2005) . We employed the same approach: The 155 proteins that were present in at least 20% of the strains loosely predicted the relatedness of the strains according to their genealogy for inbred strains and according to the breeding scheme for recombinant strains . Almost half of these proteins (81) were present in greater than 80% of the strains and only 34 were shared by all the strains. The strain dependent distribution of HDL proteome across 93 strains validates our previous studies with only 5 strains (Pamir et al., 2016) . However, the comparison of the clustering patterns between microarray data or the SNPs did not reach full agreement as genetic variation explains only a fraction of the variation and a very small part of the genome is involved in regulating HDL (data not shown). The 93 strains are represented by N=1-5 with a distribution of ~4, 9, 75, 9, and 1 % for N=1,2,3,4, and 5 respectively. Even though, these N are not optimal to calculateintra and -inter strain variation, the broad sense heritability calculations captured 65 proteins that have greater than 10% heritability. Among which APOA2 has a score of 0.62 which is consistent with its strong association with HDL-C loci -a highly heritable trait.', 'cite_spans': [{'start': 78, 'end': 103, 'text': '(Välikangas et al., 2018)', 'ref_id': None}, {'start': 290, 'end': 309, 'text': '(Toth et al., 2013)', 'ref_id': 'BIBREF22'}, {'start': 1069, 'end': 1091, 'text': '(Dillies et al., 2013)', 'ref_id': 'BIBREF10'}, {'start': 1523, 'end': 1544, 'text': '(Vaisar et al., 2007)', 'ref_id': 'BIBREF23'}, {'start': 1691, 'end': 1715, 'text': '(Hoofnagle et al., 2012)', 'ref_id': None}, {'start': 1926, 'end': 1949, 'text': '(Carvalho et al., 2008;', 'ref_id': 'BIBREF4'}, {'start': 1950, 'end': 1967, 'text': 'Liu et al., 2004)', 'ref_id': None}, {'start': 2248, 'end': 2270, 'text': '(Cervino et al., 2005)', 'ref_id': 'BIBREF6'}, {'start': 2776, 'end': 2796, 'text': '(Pamir et al., 2016)', 'ref_id': None}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'The high-level heritability of the proteins is demonstrated by >20K pQTL associated SNPs that map to > 66 loci. To understand whether a pQTL results from structural or regulatory variation, we have incorporated gene expression information. A positive finding in such analysis suggests that the genotype dependent differential gene expression is the basis of most of the association. (Farber et al., 2011) . In our studies, we used adipose and liver tissue global gene expression profiles to map distinct loci in liver (5) and adipose tissue (20) and that are associated with the gene expression levels for the SNPs associated with pQTLs. Adipose tissue exhibited only inflammatory gene (Saa1, Saa2, Tfrc, Vtn) associated SNPs that were almost exclusively transacting. Liver tissue had cis and trans eQTLs associated with multiple genes including Apoc4, Apoh, Fgb, Tfrc, Saa2. The complex regulation at the protein and gene expression level dictates the protein composition of HDL and its hereditary preservation.', 'cite_spans': [{'start': 383, 'end': 404, 'text': '(Farber et al., 2011)', 'ref_id': None}], 'ref_spans': [], 'section': 'Discussion'}, {'text': \"Correlation networks such as weighted gene co-expression network analysis is a systems biology method for describing the correlation patterns among genes. The weighted correlation network analysis (WGCNA), revealed that core HDL proteins, composed of most common apoproteins are highly correlated and co-regulated. The co-regulated gene network is consistent with HDL's role in innate immunity and lipid metabolism as it reveals a tight network of co-regulation among the proteins with primary roles in immunity and lipid metabolism. The histocompatibility protein isoforms such as H2Q4 and H2-Q10 that have been shown to be associated with mouse HDL in multiple studies (S. M. Gordon et al., 2010; Pamir et al., 2016) are part of the core co-regulated proteins. In mice, histocompatibility proteins play a role in innate immunity by antigen presenting via major histocompatibility complex class 1 (MHC-1).H2-Q10 is the only murine MHC-1 protein found in the serum in appreciable concentrations (Lew et al., 1986) . While these innate immune proteins with roles in antigen presentation are part of core co-regulation network, acute phase proteins such SAA1 and SAA2 are not, as they are primarily regulated by an inflammatory stimulus.\", 'cite_spans': [{'start': 678, 'end': 698, 'text': 'Gordon et al., 2010;', 'ref_id': None}, {'start': 699, 'end': 718, 'text': 'Pamir et al., 2016)', 'ref_id': None}, {'start': 995, 'end': 1013, 'text': '(Lew et al., 1986)', 'ref_id': None}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'While up to 50% of HDL cholesterol level can be heritable, less is known about heritability of its sterol efflux function or its proteome (Bentley et al., 2014; Goode et al., 2007) . The sterol efflux capacity of HDL seems to be regulated by a multigenic architecture with small effect size. The loci captured using 93 strains of mice have moderate P values and small effect sizes. In a human cohort of 846 individuals, Villard et al tested 7 preselected SNPs with known effects in HDL metabolism such as ABCA1, CETP, APOA1 and APOA2. Seven SNPs tested accounted together for approximately 6% of total plasma efflux capacity supporting our findings of moderate strength QTLs.', 'cite_spans': [{'start': 138, 'end': 160, 'text': '(Bentley et al., 2014;', 'ref_id': 'BIBREF2'}, {'start': 161, 'end': 180, 'text': 'Goode et al., 2007)', 'ref_id': None}], 'ref_spans': [], 'section': 'Discussion'}, {'text': \"The classic linear view of HDL genesis from discoidal, lipid-poor nascent particles to spherical, cholesterol-and phospholipid-rich particles packed with a combination of over 100 different proteins has been recently challenged by the finding that HDL is secreted directly from hepatocytes in 4 distinct sizes, with little interchange between them, and representing all of the plasma HDL subparticle pools (Mendivil et al., 2016) . Although our analyses do not incorporate the lipidome of HDL which can contribute to the orchestration of the composition of HDL sub populations, and our samples weren't stored with a cryoprotectant (HDL structure, function and proteome can be affected and proteome under represented (Holzer et al, 2017) a highly inter correlated proteome reveals the complexity of HDL particle composition. We captured a remarkable 2216 correlations among the proteins that survived multiple comparison correction and that explains at least 25% of the variation (R2>0.25). The hierarchical clustering of the correlated proteins regrouped the proteins according to their biological functions emphasizing the coordinated co-regulation. However, while applying the stringent statistical approach certain biological relationships were missed. For example, APOC3 significantly and exclusively negatively correlated with 36 other HDL proteins (Supplementary Table 2 ). In humans, }increased circulating APOC3 levels are associated with cardiovascular disorders, inflammation, and insulin resistance (Chan et al., 2008; Petersen et al., 2010) . On the other hand, humans with an APOC3 mutation benefit from a favorable lipoprotein profile, increased insulin sensitivity, lower incidence of hypertension, and protection against cardiovascular diseases (Atzmon et al., 2006; Jørgensen et al., 2014; Pollin et al., 2008) . The negative correlation of APOC3 with 36 other proteins, its association with plasma insulin levels, and HOMA-IR levels conforms to its newly appreciated role as a brake on the metabolic system. Efforts to identify the proteomic, lipidomic, and functional fingerprints of HDL subspecies are of critical importance and may open paths to novel pharmacological targets. Clinical and epidemiological studies show a robust, inverse association between HDL cholesterol (HDL-C) levels and CHD risk (D. J. Gordon and Rifkind, 1989; Wilson et al., 1988) . However, pharmacological interventions aimed at raising HDL cholesterol levels in humans showed no cardiovascular benefits (AIM-HIGH Investigators et al., 2011; Barter et al., 2007; Landray et al., 2014; Schwartz et al., 2012) . Since the collapse of the HDL-C hypothesis for atherosclerosis, a new generation of HDL metrics are under investigation to be used in clinic (Fazio and Pamir, 2016) . For example, greater HDL cholesterol efflux capacity (CEC), independent of levels of HDL-C and APOA1 (the major structural protein of HDL), is associated with a lower prevalence of atherosclerotic vascular disease (Khera et al., 2011; Li et al., 2013; Rohatgi et al., 2014) . Most changes in HDL function are likely to be a reflection of changes in the HDL proteome (Green et al., 2008; Vaisar et al., 2007) . Thus, identification of the protein signature responsible for loss of sterol efflux capacity could provide biomarkers of clinical validity to assess CHD risk. The interplay between HDL sterol efflux function, particle concentration, size and HDL proteome is still poorly understood. HDL-C levels correlated strongly with all the efflux measures. While we captured strong associations between expected metrics such as diffusional or ABCA1 specific efflux from two different cell types, no single HDL protein explained the majority of the variation in sterol efflux, suggesting that it is a polygenic process. That said, APOA2 explained about 10% of ABCA1 dependent sterol efflux from both cell types and the Apoa2 locus was strongly associated with ABCA1 specific sterol efflux (data not shown). In mice, APOA2 seems to impact the sterol efflux function at the protein and gene level. It is important to note that Apoa2 locus aligns with HDL-C QTL.\", 'cite_spans': [{'start': 406, 'end': 429, 'text': '(Mendivil et al., 2016)', 'ref_id': None}, {'start': 1510, 'end': 1529, 'text': '(Chan et al., 2008;', 'ref_id': 'BIBREF7'}, {'start': 1530, 'end': 1552, 'text': 'Petersen et al., 2010)', 'ref_id': 'BIBREF14'}, {'start': 1761, 'end': 1782, 'text': '(Atzmon et al., 2006;', 'ref_id': None}, {'start': 1783, 'end': 1806, 'text': 'Jørgensen et al., 2014;', 'ref_id': None}, {'start': 1807, 'end': 1827, 'text': 'Pollin et al., 2008)', 'ref_id': 'BIBREF15'}, {'start': 2329, 'end': 2354, 'text': 'Gordon and Rifkind, 1989;', 'ref_id': None}, {'start': 2355, 'end': 2375, 'text': 'Wilson et al., 1988)', 'ref_id': None}, {'start': 2501, 'end': 2538, 'text': '(AIM-HIGH Investigators et al., 2011;', 'ref_id': None}, {'start': 2539, 'end': 2559, 'text': 'Barter et al., 2007;', 'ref_id': None}, {'start': 2560, 'end': 2581, 'text': 'Landray et al., 2014;', 'ref_id': None}, {'start': 2582, 'end': 2604, 'text': 'Schwartz et al., 2012)', 'ref_id': 'BIBREF18'}, {'start': 2748, 'end': 2771, 'text': '(Fazio and Pamir, 2016)', 'ref_id': None}, {'start': 2988, 'end': 3008, 'text': '(Khera et al., 2011;', 'ref_id': None}, {'start': 3009, 'end': 3025, 'text': 'Li et al., 2013;', 'ref_id': None}, {'start': 3026, 'end': 3047, 'text': 'Rohatgi et al., 2014)', 'ref_id': 'BIBREF17'}, {'start': 3140, 'end': 3160, 'text': '(Green et al., 2008;', 'ref_id': None}, {'start': 3161, 'end': 3181, 'text': 'Vaisar et al., 2007)', 'ref_id': 'BIBREF23'}], 'ref_spans': [{'start': 1369, 'end': 1376, 'text': 'Table 2', 'ref_id': 'TABREF5'}], 'section': 'Discussion'}, {'text': 'In summary, a systems biology approach reveals the highly complex and inter-correlated nature of HDL protein composition, heritable contributions to both HDL levels and composition and its association with disease. We show that HDL proteins preserve hereditary patterns that are likely to harbor ancestral/lineage information. It is likely that inheritance controls the production of HDL particles of a certain protein and lipid composition that have different functions. At present, we lack a model for the assembly of HDL protein and lipid cargo. Our results provide the ground work to support future studies aimed at characterization of the genetic architecture regulating HDL function and comprehensive composition in humans.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': \"Mice. All studies were approved by the Animal Care and Use Committee of the University of California, Los Angeles. Mice were housed (1-3/cage) in a pathogen-free barrier facility (22°C) with a 12 h light/dark cycle with free access to food and water. All the strains were a fed low-fat diet (Wayne Rodent BLOX 8604; Harlan Teklad Laboratory). 60-80 day old mice fasted for 16 h at 7pm and sacrificed at 9am the following morning. Mice were bled from the retro-orbital sinus into tubes containing EDTA (final concentration 1 mM), after isofluorane inhalation. Plasma was collected and stored at −80 o C until analysis. Plasma HDL-C measurements. Plasma cholesterol levels (Invitrogen) were determined biochemically following the manufacturer's guidelines. (Shao et al., 2005) . Efflux of [ 3 H]cholesterol was measured after a 4 h incubation in medium with APOB depleted serum HDL (2.8% v/v). ABCA1-specific cholesterol efflux capacity was calculated as the percentage of total [ 3 H]cholesterol (medium plus cell) released into the medium of BHK cells stimulated with mifepristone after the value obtained with cells stimulated with medium alone was subtracted. HDL isolation. Serum HDL was prepared by adding calcium (2 mM final concentration) to plasma and using PEG (polyethylene glycol, 8kDa, Sigma) to precipitate lipoproteins containing APOB (VLDL, IDL, LDL). After centrifugation at 10,000 g for 30 min at 4 o C, serum HDL was harvested from the supernatant. HDL was isolated from serum or EDTA-anticoagulated plasma, using sequential ultracentrifugation (d=1.063-1.21 mg/mL) (Vaisar et al., 2007) . HDL was stored on ice in the dark and used within 1 week of preparation. For each isolation batch, control samples from the same pooled mouse plasma were included and further processed by tryptic digest and MS to control for experimental variability. Liquid chromatography-electrospray ionization tandem mass spectrometric (LC-ESI-MS/MS) analysis. HDL (10 μg protein) isolated by ultracentrifugation and 0.5 ug of yeast carboxypeptidase were solubilized with 0.1% RapiGest (Waters) in 200 mM ammonium bicarbonate, reduced with dithiothreitol, alkylated with iodoacetamide, and digested with trypsin (1:20, w/w HDL protein; Promega) for 3 h at 37°C. After a second aliquot of trypsin (1:20, w/w HDL protein) was added, samples were incubated overnight at 37°C. After RapiGest was removed by acid hydrolysis, samples were dried and stored at −20°C until analysis. Prior to analysis, samples were reconstituted in 5% acetonitrile, 0.1% formic acid (Pamir et al., 2016) . Tryptic digests of mouse HDL (1 μg protein) were injected onto a C18 trap column (Paradigm Platinum Peptide Nanotrap, 0.15 x 50 mm; Michrom Bioresources Inc.), desalted (50 μL/min) for 5 min with 1% acetonitrile/0.1% formic acid, eluted onto an analytical reverse-phase column (0.15 x 150 mm, Magic C18AQ, 5 μm, 200 A; Michrom Bioresources Inc.), and separated on a Paradigm M4B HPLC (Michrom Bioresources Inc.) at a flow rate of 1 μL/min over 180 min, using a linear gradient of 5% to 35% buffer B (90% acetonitrile, 0.1% formic acid) in buffer A (5% acetonitrile, 0.1% formic acid). Electrospray ionization (ESI) was performed using a Captive . CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/405811 doi: bioRxiv preprint Spray source (Michrom BioResources, Inc, Auburn, CA) at 10 mL/min flow rate and 1.4 kV setting. HDL digests were introduced into the gas phase by ESI, positive ion mass spectra were acquired with a orbitrap mass spectrometer (Fusion, Thermo Electron Corp.) using datadependent acquisition (one MS survey scan followed by MS/MS scans of the 8 most abundant ions in the survey scan) with a 350-1400 m/z scan. An exclusion window of 30 s was used after 2 acquisitions of the same precursor ion (Pamir et al., 2016; Vaisar et al., 2007) . Protein identification. MS/MS spectra were matched against performed using the Comet search engine (version 2015.01 rev. 1) against a mouse UniProt database appended with yeast caboxypeptidase Y protein sequence (52,639 total sequences). The following Comet search parameters were applied: peptide mass tolerance of +/-20.0 ppm allowing for C13 isotope offsets, full tryptic digest allowing up to 2 missed cleavages, oxidized methionine variable modification, and carbamidomethyl cysteine static modification. The search results were subsequently processed though the Trans-Proteomic Pipeline (version 4.8.0) using the PeptideProphet and ProteinProphet tools using an adjusted probability of ≥0.90 for peptides and ≥0.95 for proteins. Each charge state of a peptide was considered a unique identification (Nesvizhskii et al., 2003) . We used the gene and protein names in the Entrez databases (National Center for Biotechnology Information; based on the nomenclature guidelines of the Human Gene Nomenclature Committee (http://www.gene.ucl.ca.uk/nomenlature) for human (Wain et al., 2002) and Mouse Genome Informatics (http://www.infromatics.jax.org.nomen/) guidelines (Davisson, 1994) to identify HDL proteins and to eliminate the redundant identifications of isoforms and protein fragments frequently found in databases used in proteomic analysis (Rappsilber and Mann, 2002) . This approach also permits cross-referencing of proteins from different species.\", 'cite_spans': [{'start': 755, 'end': 774, 'text': '(Shao et al., 2005)', 'ref_id': 'BIBREF19'}, {'start': 1583, 'end': 1604, 'text': '(Vaisar et al., 2007)', 'ref_id': 'BIBREF23'}, {'start': 2552, 'end': 2572, 'text': '(Pamir et al., 2016)', 'ref_id': None}, {'start': 3926, 'end': 3946, 'text': '(Pamir et al., 2016;', 'ref_id': None}, {'start': 3947, 'end': 3967, 'text': 'Vaisar et al., 2007)', 'ref_id': 'BIBREF23'}, {'start': 4775, 'end': 4801, 'text': '(Nesvizhskii et al., 2003)', 'ref_id': None}, {'start': 5039, 'end': 5058, 'text': '(Wain et al., 2002)', 'ref_id': None}, {'start': 5139, 'end': 5155, 'text': '(Davisson, 1994)', 'ref_id': 'BIBREF9'}, {'start': 5319, 'end': 5346, 'text': '(Rappsilber and Mann, 2002)', 'ref_id': 'BIBREF16'}], 'ref_spans': [], 'section': 'Methods'}, {'text': \"Protein quantification. Proteins were quantified using Peptide Spectra Matches (PSM) -the total number of MS/MS spectra detected for a protein (Vaisar et al., 2007) . Proteins considered for analysis had to be detected in ≥30 analyses (%10 of the total samples) with ≥2 unique peptides. Because only a few peptides are typically measured for a given protein, these peptides might not be sufficient to define all isoforms of the protein that are present in the sample therefore, when MS/MS spectra could not differentiate between protein isoforms, the isoform with the most unique peptides was used for further analysis. PSMs for each protein, normalized to either spiked yeast carboxypeptidase or to total PSMs for peptides from each sample, were used to calculate a normalized PSM to compare the relative protein composition of mouse strains' HDLs (Vaisar et al., 2007) . Supplemental Table 1 provides the total calculated PSMs for each protein, the individual peptides that identified each protein, the total number peptide spectra matches, and relative quantification as normalized to yeast carboxypeptidase Y total PSMs or total PSMs of one sample. HDL particle size. HDL particle size was quantified by calibrated ion mobility analysis (Hutchins et al., 2014) . Briefly, HDL isolated by ultracentrifugation from EDTA plasma is introduced into the gas-phase ions by ESI. Because electrophoretic mobility depends chiefly on size, ion mobility analysis data are expressed in terms of particle diameter (nm), which corresponds to the calculated diameter of a singly charged, spherical particle with the same electrophoretic mobility (Hutchins et al., 2014) . Association analyses. GWAS for protein levels and gene expression was performed using correction for population structure as described (Hui et al., 2015; Orozco et al., 2012) . Loci were defined as cis if the peak SNP mapped within 1 Mb of gene position and trans if it mapped outside (cis significance threshold, p < 1.4 e-3; trans threshold, p < 6.13e-6).\", 'cite_spans': [{'start': 143, 'end': 164, 'text': '(Vaisar et al., 2007)', 'ref_id': 'BIBREF23'}, {'start': 849, 'end': 870, 'text': '(Vaisar et al., 2007)', 'ref_id': 'BIBREF23'}, {'start': 1241, 'end': 1264, 'text': '(Hutchins et al., 2014)', 'ref_id': None}, {'start': 1795, 'end': 1813, 'text': '(Hui et al., 2015;', 'ref_id': 'BIBREF13'}, {'start': 1814, 'end': 1834, 'text': 'Orozco et al., 2012)', 'ref_id': None}], 'ref_spans': [{'start': 886, 'end': 893, 'text': 'Table 1', 'ref_id': 'TABREF4'}, {'start': 1634, 'end': 1657, 'text': '(Hutchins et al., 2014)', 'ref_id': 'FIGREF1'}], 'section': 'Methods'}, {'text': '. CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/405811 doi: bioRxiv preprint Heritability. Broad sense heretibiliaty scores were calculated for each protein using R package (sommer) using the formula H2= genetic variance / (genetic variance + residual variance) Statistical analyses. Data are means ± SEMs. Linear correlation among the HDL metrics of the 93 strains were assessed with pearson correlations and the association of the proteins were assessed by spearman correlations both were followed with Bonferroni-Holm post-hoc correction for multiple comparisons. Data were analyzed with Prism and R software. Data and software availability. The MS/MS datasets produced in this study are available in the PRIDE consortium (ProteomXchange submission ref: 1-20180406-10713) and in the UCLA based public database established to harbor HMDP related data (https://systems.genetics.ucla.edu/data).', 'cite_spans': [], 'ref_spans': [], 'section': 'Methods'}, {'text': 'Jay Heinecke is named as a co-inventor on patents from the US Patent Office on the use of HDL markers to predict the risk of cardiovascular disease. Dr. Heinecke has served as a consultant for Kowa, Merck, Amgen, Bristol Meyer Squibb, GSK, and Pacific Biomarkers. . Hierarchical clustering of the HDL metrics: Proteome, sterol efflux, particle concentration and size. min.MF, plus.MF, delta.BHK are unstimulated, ABCA1 upregulated and ABCA1 specific sterol efflux from BHK cells respectively. min.cAMP, plus.cAMP, delta.J774 are unstimulated, ABCA1 upregulated and ABCA1 specific sterol efflux from J774 cells respectively. m.hdl.size and l.hdl.size are medium and large HDL sizes respectively. The correlation structure was determined using pearson correlation. The protein functional groups were curated from DAVID, KEGG, Panther, and Uniprot databases. Figure 1 . Self-loops were removed and edges were bundled for clarity. Node locations are assigned using an edge-weighted spring-embedded layout algorithm using the negative log of the Benjamini-Hochberg corrected p-value, and edge transparency is directly proportional to the same value. Red: negative correlation, Blue: positive correlation. Shorter distances indicate stronger correlations.', 'cite_spans': [], 'ref_spans': [{'start': 856, 'end': 864, 'text': 'Figure 1', 'ref_id': 'FIGREF1'}], 'section': 'Disclosures'}, {'text': '. CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. Figure 3 . Hierarchical clustering of the HDL metrics: Proteome, sterol efflux, particle concentration and size. min.MF, plus.MF, delta.BHK are unstimulated, ABCA1 upregulated and ABCA1 specific sterol efflux from BHK cells respectively. min.cAMP, plus.cAMP, delta.J774 are unstimulated, ABCA1 upregulated and ABCA1 specific sterol efflux from J774 cells respectively. m.hdl.size and l.hdl.size are medium and large HDL sizes respectively. The correlation structure was determined using pearson correlation. The protein functional groups were curated from DAVID, KEGG, Panther, and Uniprot databases. Figure 1 . Self-loops were removed and edges were bundled for clarity. Node locations are assigned using an edge-weighted spring-embedded layout algorithm using the negative log of the Benjamini-Hochberg corrected p-value, and edge transparency is directly proportional to the same value. Red: negative correlation, Blue: positive correlation. Shorter distances indicate stronger correlations. Figure 1 : The heatmap visualization of the HDL protein abundances (calculated as normalized to total PSMs) across 93 strains. The proteins, their biological functions and cellular locations are represented. Logarithmic transformation of the total PSM normalized data has been performed to accommodate the abundance distribution form high (red) to very low (dark blue). White squares represent not available values. Both the proteins and the strains were clustered using Euclidean distances.', 'cite_spans': [], 'ref_spans': [{'start': 157, 'end': 165, 'text': 'Figure 3', 'ref_id': None}, {'start': 758, 'end': 766, 'text': 'Figure 1', 'ref_id': 'FIGREF1'}, {'start': 1152, 'end': 1160, 'text': 'Figure 1', 'ref_id': 'FIGREF1'}], 'section': 'Disclosures'}, {'text': 'Supplementary Figure 2 . A representative QTLs for HDL proteins normalized to total PSMs. Loci associated with A) APOA2, B), NAPSA, C) SAA2, D) SAA4.', 'cite_spans': [], 'ref_spans': [{'start': 14, 'end': 22, 'text': 'Figure 2', 'ref_id': 'FIGREF6'}], 'section': 'Disclosures'}, {'text': 'Supplementary Figure 3 Hierarchical clustering of the HDL metrics: Proteome (normalized to total PSMs), sterol efflux, particle concentration and size. min.MF, plus.MF, delta.BHK are unstimulated, ABCA1 upregulated and ABCA1 specific sterol efflux from BHK cells respectively. min.cAMP, plus.cAMP, delta.J774 are unstimulated, ABCA1 upregulated and ABCA1 specific sterol efflux from J774 cells respectively. m.hdlp and t.hdlp are medium size and total HDL particle concentration. m.hdl.size and l.hdl.size are medium and large HDL sizes respectively. The correlation structure was determined using pearson correlation. The protein functional groups were curated from DAVID, KEGG, Panther, and Uniprot databases. Figure 1 : The heatmap visualization of the HDL protein abundances (calculated as normalized to total PSMs) across 93 strains. The proteins, their biological functions and cellular locations are represented. Logarithmic transformation of the total PSM normalized data has been performed to accommodate the abundance distribution form high (red) to very low (dark blue). White squares represent not available values. Both the proteins and the strains were clustered using Euclidean distances. Figure 3 . Hierarchical clustering of the HDL metrics: Proteome (normalized to total PSMs), sterol efflux, particle concentration and size. min.MF, plus.MF, delta.BHK are unstimulated, ABCA1 upregulated and ABCA1 specific sterol efflux from BHK cells respectively. min.cAMP, plus.cAMP, delta.J774 are unstimulated, ABCA1 upregulated and ABCA1 specific sterol efflux from J774 cells respectively. m.hdlp and t.hdlp are medium size and total HDL particle concentration. m.hdl.size and l.hdl.size are medium and large HDL sizes respectively. The correlation structure was determined using pearson correlation. The protein functional groups were curated from DAVID, KEGG, Panther, and Uniprot databases. Table 3 . Local adipose eQTL and local HDL pQTL. Table 4 . Potential causal interaction based on local pQTL analyses. . CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/405811 doi: bioRxiv preprint', 'cite_spans': [], 'ref_spans': [{'start': 14, 'end': 22, 'text': 'Figure 3', 'ref_id': None}, {'start': 712, 'end': 720, 'text': 'Figure 1', 'ref_id': 'FIGREF1'}, {'start': 1204, 'end': 1212, 'text': 'Figure 3', 'ref_id': None}, {'start': 1904, 'end': 1911, 'text': 'Table 3', 'ref_id': 'TABREF6'}, {'start': 1953, 'end': 1960, 'text': 'Table 4', 'ref_id': 'TABREF7'}], 'section': 'Disclosures'}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'Genetic Architecture of Atherosclerosis in Mice: A Systems Genetics Analysis of Common Inbred Strains', 'authors': [{'first': 'B', 'middle': ['J'], 'last': 'Bennett', 'suffix': ''}, {'first': 'R', 'middle': ['C'], 'last': 'Davis', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Civelek', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Orozco', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Wu', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Qi', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Pan', 'suffix': ''}, {'first': 'R', 'middle': ['R S'], 'last': 'Packard', 'suffix': ''}, {'first': 'E', 'middle': [], 'last': 'Eskin', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Yan', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Kirchgessner', 'suffix': ''}, {'first': 'Z', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'J', 'middle': ['C'], 'last': 'Gregory', 'suffix': ''}, {'first': 'S', 'middle': ['L'], 'last': 'Hazen', 'suffix': ''}, {'first': 'P', 'middle': ['S'], 'last': 'Gargalovic', 'suffix': ''}, {'first': 'A', 'middle': ['J'], 'last': 'Lusis', 'suffix': ''}], 'year': 2015, 'venue': 'PLoS Genet', 'volume': '11', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1371/journal.pgen.1005711']}}, 'BIBREF1': {'ref_id': 'b1', 'title': 'A high-resolution association mapping panel for the dissection of complex traits in mice', 'authors': [{'first': 'B', 'middle': ['J'], 'last': 'Bennett', 'suffix': ''}, {'first': 'C', 'middle': ['R'], 'last': 'Farber', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Orozco', 'suffix': ''}, {'first': 'Min', 'middle': [], 'last': 'Kang', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Ghazalpour', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Siemers', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Neubauer', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Neuhaus', 'suffix': ''}, {'first': 'I', 'middle': [], 'last': 'Yordanova', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Guan', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Truong', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Yang', 'suffix': ''}, {'first': 'W', 'middle': ['P'], 'last': 'He', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Kayne', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Gargalovic', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Kirchgessner', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Pan', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Castellani', 'suffix': ''}, {'first': 'L', 'middle': ['W'], 'last': 'Kostem', 'suffix': ''}, {'first': 'E', 'middle': [], 'last': 'Furlotte', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Drake', 'suffix': ''}, {'first': 'T', 'middle': ['A'], 'last': 'Eskin', 'suffix': ''}, {'first': 'E', 'middle': [], 'last': 'Lusis', 'suffix': ''}, {'first': 'A', 'middle': ['J'], 'last': '', 'suffix': ''}], 'year': 2010, 'venue': 'Genome Research', 'volume': '20', 'issn': '', 'pages': '281--290', 'other_ids': {'DOI': ['10.1101/gr.099234.109']}}, 'BIBREF2': {'ref_id': 'b2', 'title': 'Gene-based sequencing identifies lipid-influencing variants with ethnicity-specific effects in African Americans', 'authors': [{'first': 'A', 'middle': ['R'], 'last': 'Bentley', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Chen', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Shriner', 'suffix': ''}, {'first': 'A', 'middle': ['P'], 'last': 'Doumatey', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Zhou', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Huang', 'suffix': ''}, {'first': 'J', 'middle': ['C'], 'last': 'Mullikin', 'suffix': ''}, {'first': 'R', 'middle': ['W'], 'last': 'Blakesley', 'suffix': ''}, {'first': 'N', 'middle': ['F'], 'last': 'Hansen', 'suffix': ''}, {'first': 'G', 'middle': ['G'], 'last': 'Bouffard', 'suffix': ''}, {'first': 'P', 'middle': ['F'], 'last': 'Cherukuri', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Maskeri', 'suffix': ''}, {'first': 'A', 'middle': ['C'], 'last': 'Young', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Adeyemo', 'suffix': ''}, {'first': 'C', 'middle': ['N'], 'last': 'Rotimi', 'suffix': ''}], 'year': 2014, 'venue': 'PLoS Genet', 'volume': '10', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1371/journal.pgen.1004190']}}, 'BIBREF3': {'ref_id': 'b3', 'title': 'Mutations in ABC1 in Tangier disease and familial high-density lipoprotein deficiency', 'authors': [{'first': 'A', 'middle': [], 'last': 'Brooks-Wilson', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Marcil', 'suffix': ''}, {'first': 'S', 'middle': ['M'], 'last': 'Clee', 'suffix': ''}, {'first': 'L.-H', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Roomp', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Van Dam', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Yu', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Brewer', 'suffix': ''}, {'first': 'J', 'middle': ['A'], 'last': 'Collins', 'suffix': ''}, {'first': 'H', 'middle': ['O F'], 'last': 'Molhuizen', 'suffix': ''}, {'first': 'O', 'middle': [], 'last': 'Loubser', 'suffix': ''}, {'first': 'B', 'middle': ['F F'], 'last': 'Ouelette', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Fichter', 'suffix': ''}, {'first': 'K', 'middle': ['J D'], 'last': 'Ashbourne-Excoffon', 'suffix': ''}, {'first': 'C', 'middle': ['W'], 'last': 'Sensen', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Scherer', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Mott', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Denis', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Martindale', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Frohlich', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Morgan', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Koop', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Pimstone', 'suffix': ''}, {'first': 'J', 'middle': ['J P'], 'last': 'Kastelein', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Genest', 'suffix': ''}, {'first': 'M', 'middle': ['R'], 'last': 'Hayden', 'suffix': ''}], 'year': 1999, 'venue': 'Nat. Genet', 'volume': '22', 'issn': '', 'pages': '336--345', 'other_ids': {'DOI': ['10.1038/11905']}}, 'BIBREF4': {'ref_id': 'b4', 'title': 'Identifying differences in protein expression levels by spectral counting and feature selection', 'authors': [{'first': 'P', 'middle': ['C'], 'last': 'Carvalho', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Hewel', 'suffix': ''}, {'first': 'V', 'middle': ['C'], 'last': 'Barbosa', 'suffix': ''}, {'first': 'J', 'middle': ['R'], 'last': 'Yates', 'suffix': ''}, {'first': '', 'middle': [], 'last': 'Iii', 'suffix': ''}], 'year': 2008, 'venue': 'Genetics and Molecular Research', 'volume': '7', 'issn': '', 'pages': '342--356', 'other_ids': {'DOI': ['10.4238/vol7-2gmr426']}}, 'BIBREF5': {'ref_id': 'b5', 'title': 'A population at risk: Prevalence of high cholesterol levels in hypertensive patients in the framingham study', 'authors': [{'first': 'W', 'middle': ['P'], 'last': 'Castelli', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Anderson', 'suffix': ''}], 'year': 1986, 'venue': 'The American Journal of Medicine', 'volume': '80', 'issn': '86', 'pages': '90157--90166', 'other_ids': {'DOI': ['10.1016/0002-9343']}}, 'BIBREF6': {'ref_id': 'b6', 'title': 'Integrating QTL and high-density SNP analyses in mice to identify Insig2 as a susceptibility gene for plasma cholesterol levels', 'authors': [{'first': 'A', 'middle': ['C'], 'last': 'Cervino', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Edwards', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Zhu', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Laurie', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Tokiwa', 'suffix': ''}, {'first': 'P', 'middle': ['Y'], 'last': 'Lum', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'L', 'middle': ['W'], 'last': 'Castellani', 'suffix': ''}, {'first': 'L', 'middle': ['W'], 'last': 'Castellini', 'suffix': ''}, {'first': 'A', 'middle': ['J'], 'last': 'Lusis', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Carlson', 'suffix': ''}, {'first': 'A', 'middle': ['B'], 'last': 'Sachs', 'suffix': ''}, {'first': 'E', 'middle': ['E'], 'last': 'Schadt', 'suffix': ''}], 'year': 2005, 'venue': 'Genomics', 'volume': '86', 'issn': '', 'pages': '505--517', 'other_ids': {'DOI': ['10.1016/j.ygeno.2005.07.010']}}, 'BIBREF7': {'ref_id': 'b7', 'title': 'Plasma Apolipoprotein C-III Transport in Centrally Obese Men: Associations with Very Low-Density Lipoprotein Apolipoprotein B and High-Density Lipoprotein Apolipoprotein A-I Metabolism', 'authors': [{'first': 'D', 'middle': ['C'], 'last': 'Chan', 'suffix': ''}, {'first': 'M', 'middle': ['N'], 'last': 'Nguyen', 'suffix': ''}, {'first': 'G', 'middle': ['F'], 'last': 'Watts', 'suffix': ''}, {'first': 'P', 'middle': ['H R'], 'last': 'Barrett', 'suffix': ''}], 'year': 2008, 'venue': 'Journal of Clinical Endocrinology & Metabolism', 'volume': '93', 'issn': '', 'pages': '557--564', 'other_ids': {'DOI': ['10.1210/jc.2006-2676']}}, 'BIBREF8': {'ref_id': 'b8', 'title': 'Multiple rare alleles contribute to low plasma levels of HDL cholesterol', 'authors': [{'first': 'J', 'middle': ['C'], 'last': 'Cohen', 'suffix': ''}, {'first': 'R', 'middle': ['S'], 'last': 'Kiss', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Pertsemlidis', 'suffix': ''}, {'first': 'Y', 'middle': ['L'], 'last': 'Marcel', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Mcpherson', 'suffix': ''}, {'first': 'H', 'middle': ['H'], 'last': 'Hobbs', 'suffix': ''}], 'year': 2004, 'venue': 'Science', 'volume': '305', 'issn': '', 'pages': '869--872', 'other_ids': {'DOI': ['10.1126/science.1099870']}}, 'BIBREF9': {'ref_id': 'b9', 'title': 'Rules and guidelines for nomenclature of mouse genes. International Committee on Standardized Genetic Nomenclature for Mice', 'authors': [{'first': 'M', 'middle': ['T'], 'last': 'Davisson', 'suffix': ''}], 'year': 1994, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF10': {'ref_id': 'b10', 'title': 'A comprehensive evaluation of normalization methods for Illumina high-throughput RNA sequencing data analysis', 'authors': [{'first': 'M.-A', 'middle': [], 'last': 'Dillies', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Rau', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Aubert', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Hennequet-Antier', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Jeanmougin', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Servant', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Keime', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Marot', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Castel', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Estelle', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Guernec', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Jagla', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Jouneau', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Laloë', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Le Gall', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Schaëffer', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Le Crom', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Guedj', 'suffix': ''}, {'first': 'F', 'middle': [], 'last': 'Jaffrézic', 'suffix': ''}], 'year': 2013, 'venue': 'Brief. Bioinformatics', 'volume': '14', 'issn': '', 'pages': '671--683', 'other_ids': {'DOI': ['10.1093/bib/bbs046']}}, 'BIBREF11': {'ref_id': 'b11', 'title': 'Neuroprotective Effect of Apolipoprotein D against Human Coronavirus OC43-Induced Encephalitis in Mice', 'authors': [{'first': 'S', 'middle': [], 'last': 'Do Carmo', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Jacomy', 'suffix': ''}, {'first': 'P', 'middle': ['J'], 'last': 'Talbot', 'suffix': ''}, {'first': 'E', 'middle': [], 'last': 'Rassart', 'suffix': ''}], 'year': 2008, 'venue': 'J. Neurosci', 'volume': '28', 'issn': '', 'pages': '10330--10338', 'other_ids': {'DOI': ['10.1523/JNEUROSCI.2644-08.2008']}}, 'BIBREF12': {'ref_id': 'b12', 'title': 'Genetic Control of Obesity and Gut Microbiota Composition in Response to High-Fat, High-Sucrose Diet in Mice', 'authors': [{'first': 'M', 'middle': [], 'last': 'Drake', 'suffix': ''}, {'first': 'T', 'middle': ['A'], 'last': 'Drevon', 'suffix': ''}, {'first': 'C', 'middle': ['A'], 'last': 'Knight', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Gargalovic', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Kirchgessner', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Eskin', 'suffix': ''}, {'first': 'E', 'middle': [], 'last': 'Lusis', 'suffix': ''}, {'first': 'A', 'middle': ['J'], 'last': '', 'suffix': ''}], 'year': 2013, 'venue': 'Cell Metabolism', 'volume': '17', 'issn': '', 'pages': '141--152', 'other_ids': {'DOI': ['10.1016/j.cmet.2012.12.007']}}, 'BIBREF13': {'ref_id': 'b13', 'title': 'Genetic Architecture of Insulin Resistance in the Mouse', 'authors': [{'first': 'B', 'middle': ['W'], 'last': 'Parks', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Sallam', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Mehrabian', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Psychogios', 'suffix': ''}, {'first': 'S', 'middle': ['T'], 'last': 'Hui', 'suffix': ''}, {'first': 'F', 'middle': [], 'last': 'Norheim', 'suffix': ''}, {'first': 'L', 'middle': ['W'], 'last': 'Castellani', 'suffix': ''}, {'first': 'C', 'middle': ['D'], 'last': 'Rau', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Pan', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Phun', 'suffix': ''}, {'first': 'Z', 'middle': [], 'last': 'Zhou', 'suffix': ''}, {'first': 'W.-P', 'middle': [], 'last': 'Yang', 'suffix': ''}, {'first': 'I', 'middle': [], 'last': 'Neuhaus', 'suffix': ''}, {'first': 'P', 'middle': ['S'], 'last': 'Gargalovic', 'suffix': ''}, {'first': 'T', 'middle': ['G'], 'last': 'Kirchgessner', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Graham', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Lee', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Tontonoz', 'suffix': ''}, {'first': 'R', 'middle': ['E'], 'last': 'Gerszten', 'suffix': ''}, {'first': 'A', 'middle': ['L'], 'last': 'Hevener', 'suffix': ''}, {'first': 'A', 'middle': ['J'], 'last': 'Lusis', 'suffix': ''}], 'year': 2015, 'venue': 'Cell Metabolism', 'volume': '21', 'issn': '', 'pages': '334--347', 'other_ids': {'DOI': ['10.1016/j.cmet.2015.01.002']}}, 'BIBREF14': {'ref_id': 'b14', 'title': 'Apolipoprotein C3 gene variants in nonalcoholic fatty liver disease', 'authors': [{'first': 'K', 'middle': ['F'], 'last': 'Petersen', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Dufour', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Hariri', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Nelson-Williams', 'suffix': ''}, {'first': 'J', 'middle': ['N'], 'last': 'Foo', 'suffix': ''}, {'first': 'X.-M', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Dziura', 'suffix': ''}, {'first': 'R', 'middle': ['P'], 'last': 'Lifton', 'suffix': ''}, {'first': 'G', 'middle': ['I'], 'last': 'Shulman', 'suffix': ''}], 'year': 2010, 'venue': 'N. Engl. J. Med', 'volume': '362', 'issn': '', 'pages': '1082--1089', 'other_ids': {'DOI': ['10.1056/NEJMoa0907295']}}, 'BIBREF15': {'ref_id': 'b15', 'title': 'A null mutation in human APOC3 confers a favorable plasma lipid profile and apparent cardioprotection', 'authors': [{'first': 'T', 'middle': ['I'], 'last': 'Pollin', 'suffix': ''}, {'first': 'C', 'middle': ['M'], 'last': 'Damcott', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Shen', 'suffix': ''}, {'first': 'S', 'middle': ['H'], 'last': 'Ott', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Shelton', 'suffix': ''}, {'first': 'R', 'middle': ['B'], 'last': 'Horenstein', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Post', 'suffix': ''}, {'first': 'J', 'middle': ['C'], 'last': 'Mclenithan', 'suffix': ''}, {'first': 'L', 'middle': ['F'], 'last': 'Bielak', 'suffix': ''}, {'first': 'P', 'middle': ['A'], 'last': 'Peyser', 'suffix': ''}, {'first': 'B', 'middle': ['D'], 'last': 'Mitchell', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Miller', 'suffix': ''}, {'first': 'J', 'middle': ['R'], 'last': 'O&apos;connell', 'suffix': ''}, {'first': 'A', 'middle': ['R'], 'last': 'Shuldiner', 'suffix': ''}], 'year': 2008, 'venue': 'Science', 'volume': '322', 'issn': '', 'pages': '1702--1705', 'other_ids': {'DOI': ['10.1126/science.1161524']}}, 'BIBREF16': {'ref_id': 'b16', 'title': 'What does it mean to identify a protein in proteomics', 'authors': [{'first': 'J', 'middle': [], 'last': 'Rappsilber', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Mann', 'suffix': ''}], 'year': 2002, 'venue': 'Trends in Biochemical Sciences', 'volume': '27', 'issn': '', 'pages': '74--78', 'other_ids': {'DOI': ['10.1016/S0968-0004(01)02021-7']}}, 'BIBREF17': {'ref_id': 'b17', 'title': 'HDL cholesterol efflux capacity and incident cardiovascular events', 'authors': [{'first': 'A', 'middle': [], 'last': 'Rohatgi', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Khera', 'suffix': ''}, {'first': 'J', 'middle': ['D'], 'last': 'Berry', 'suffix': ''}, {'first': 'E', 'middle': ['G'], 'last': 'Givens', 'suffix': ''}, {'first': 'C', 'middle': ['R'], 'last': 'Ayers', 'suffix': ''}, {'first': 'K', 'middle': ['E'], 'last': 'Wedin', 'suffix': ''}, {'first': 'I', 'middle': ['J'], 'last': 'Neeland', 'suffix': ''}, {'first': 'I', 'middle': ['S'], 'last': 'Yuhanna', 'suffix': ''}, {'first': 'D', 'middle': ['R'], 'last': 'Rader', 'suffix': ''}, {'first': 'J', 'middle': ['A'], 'last': 'De Lemos', 'suffix': ''}, {'first': 'P', 'middle': ['W'], 'last': 'Shaul', 'suffix': ''}], 'year': 2014, 'venue': 'N. Engl. J. Med', 'volume': '371', 'issn': '', 'pages': '2383--2393', 'other_ids': {'DOI': ['10.1056/NEJMoa1409065']}}, 'BIBREF18': {'ref_id': 'b18', 'title': 'Effects of dalcetrapib in patients with a recent acute coronary syndrome', 'authors': [{'first': 'G', 'middle': ['G'], 'last': 'Schwartz', 'suffix': ''}, {'first': 'A', 'middle': ['G'], 'last': 'Olsson', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Abt', 'suffix': ''}, {'first': 'C', 'middle': ['M'], 'last': 'Ballantyne', 'suffix': ''}, {'first': 'P', 'middle': ['J'], 'last': 'Barter', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Brumm', 'suffix': ''}, {'first': 'B', 'middle': ['R'], 'last': 'Chaitman', 'suffix': ''}, {'first': 'I', 'middle': ['M'], 'last': 'Holme', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Kallend', 'suffix': ''}, {'first': 'L', 'middle': ['A'], 'last': 'Leiter', 'suffix': ''}, {'first': 'E', 'middle': [], 'last': 'Leitersdorf', 'suffix': ''}, {'first': 'J', 'middle': ['J V'], 'last': 'Mcmurray', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Mundl', 'suffix': ''}, {'first': 'S', 'middle': ['J'], 'last': 'Nicholls', 'suffix': ''}, {'first': 'P', 'middle': ['K'], 'last': 'Shah', 'suffix': ''}, {'first': 'J.-C', 'middle': [], 'last': 'Tardif', 'suffix': ''}, {'first': 'R', 'middle': ['S'], 'last': 'Wright', 'suffix': ''}], 'year': 2012, 'venue': 'N. Engl. J. Med', 'volume': '367', 'issn': '', 'pages': '2089--2099', 'other_ids': {'DOI': ['10.1056/NEJMoa1206797']}}, 'BIBREF19': {'ref_id': 'b19', 'title': 'Acrolein Impairs ATP Binding Cassette Transporter A1-dependent Cholesterol Export from Cells through Site-specific Modification of Apolipoprotein A-I', 'authors': [{'first': 'B', 'middle': [], 'last': 'Shao', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Fu', 'suffix': ''}, {'first': 'T', 'middle': ['O'], 'last': 'Mcdonald', 'suffix': ''}, {'first': 'P', 'middle': ['S'], 'last': 'Green', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Uchida', 'suffix': ''}, {'first': 'K', 'middle': ['D'], 'last': 'O&apos;brien', 'suffix': ''}, {'first': 'J', 'middle': ['F'], 'last': 'Oram', 'suffix': ''}, {'first': 'J', 'middle': ['W'], 'last': 'Heinecke', 'suffix': ''}], 'year': 2005, 'venue': 'J. Biol. Chem', 'volume': '280', 'issn': '', 'pages': '36386--36396', 'other_ids': {'DOI': ['10.1074/jbc.M508169200']}}, 'BIBREF20': {'ref_id': 'b20', 'title': 'Cholesterol, inflammation and innate immunity', 'authors': [{'first': 'A', 'middle': ['R'], 'last': 'Tall', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Yvan-Charvet', 'suffix': ''}], 'year': 2015, 'venue': 'Nat Rev Immunol', 'volume': '15', 'issn': '', 'pages': '104--116', 'other_ids': {'DOI': ['10.1038/nri3793']}}, 'BIBREF21': {'ref_id': 'b21', 'title': 'Association of cholesteryl ester transfer protein genotypes with CETP mass and activity, lipid levels, and coronary risk', 'authors': [{'first': 'A', 'middle': [], 'last': 'Thompson', 'suffix': ''}, {'first': 'E', 'middle': [], 'last': 'Di Angelantonio', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Sarwar', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Erqou', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Saleheen', 'suffix': ''}, {'first': 'R', 'middle': ['P F'], 'last': 'Dullaart', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Keavney', 'suffix': ''}, {'first': 'Z', 'middle': [], 'last': 'Ye', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Danesh', 'suffix': ''}], 'year': 2008, 'venue': 'JAMA', 'volume': '299', 'issn': '', 'pages': '2777--2788', 'other_ids': {'DOI': ['10.1001/jama.299.23.2777']}}, 'BIBREF22': {'ref_id': 'b22', 'title': 'High-density lipoproteins: A consensus statement from the National Lipid Association', 'authors': [{'first': 'P', 'middle': ['P'], 'last': 'Toth', 'suffix': ''}, {'first': 'P', 'middle': ['J'], 'last': 'Barter', 'suffix': ''}, {'first': 'R', 'middle': ['S'], 'last': 'Rosenson', 'suffix': ''}, {'first': 'W', 'middle': ['E'], 'last': 'Boden', 'suffix': ''}, {'first': 'M', 'middle': ['J'], 'last': 'Chapman', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Cuchel', 'suffix': ''}, {'first': 'R', 'middle': ['B'], 'last': 'D&apos;agostino', 'suffix': ''}, {'first': '', 'middle': [], 'last': 'Sr', 'suffix': ''}, {'first': 'M', 'middle': ['H'], 'last': 'Davidson', 'suffix': ''}, {'first': 'W', 'middle': ['S'], 'last': 'Davidson', 'suffix': ''}, {'first': 'J', 'middle': ['W'], 'last': 'Heinecke', 'suffix': ''}, {'first': 'R', 'middle': ['H'], 'last': 'Karas', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Kontush', 'suffix': ''}, {'first': 'R', 'middle': ['M'], 'last': 'Krauss', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Miller', 'suffix': ''}, {'first': 'D', 'middle': ['J'], 'last': 'Rader', 'suffix': ''}], 'year': 2013, 'venue': 'Journal of Clinical Lipidology', 'volume': '7', 'issn': '', 'pages': '484--525', 'other_ids': {'DOI': ['10.1016/j.jacl.2013.08.001']}}, 'BIBREF23': {'ref_id': 'b23', 'title': 'Shotgun proteomics implicates protease inhibition and complement activation in the antiinflammatory properties of HDL', 'authors': [{'first': 'T', 'middle': [], 'last': 'Vaisar', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Pennathur', 'suffix': ''}, {'first': 'P', 'middle': ['S'], 'last': 'Green', 'suffix': ''}, {'first': 'S', 'middle': ['A'], 'last': 'Gharib', 'suffix': ''}, {'first': 'A', 'middle': ['N'], 'last': 'Hoofnagle', 'suffix': ''}, {'first': 'M', 'middle': ['C'], 'last': 'Cheung', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Byun', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Vuletic', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Kassim', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Singh', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Chea', 'suffix': ''}, {'first': 'R', 'middle': ['H'], 'last': 'Knopp', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Brunzell', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Geary', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Chait', 'suffix': ''}, {'first': 'X.-Q', 'middle': [], 'last': 'Zhao', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Elkon', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Marcovina', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Ridker', 'suffix': ''}, {'first': 'J', 'middle': ['F'], 'last': 'Oram', 'suffix': ''}, {'first': 'J', 'middle': ['W'], 'last': 'Heinecke', 'suffix': ''}], 'year': 2007, 'venue': '', 'volume': '117', 'issn': '', 'pages': '746--756', 'other_ids': {'DOI': ['10.1172/JCI26206']}}, 'BIBREF24': {'ref_id': 'b24', 'title': 'Inflammatory remodeling of the HDL proteome impairs cholesterol efflux capacity', 'authors': [{'first': 'T', 'middle': [], 'last': 'Vaisar', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Tang', 'suffix': ''}, {'first': 'I', 'middle': [], 'last': 'Babenko', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Hutchins', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Wimberger', 'suffix': ''}, {'first': 'A', 'middle': ['F'], 'last': 'Suffredini', 'suffix': ''}, {'first': 'J', 'middle': ['W'], 'last': 'Heinecke', 'suffix': ''}], 'year': 2015, 'venue': 'The Journal of Lipid Research', 'volume': '56', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1194/jlr.M059089']}}}, 'ref_entries': {'FIGREF0': {'text': 'HIGH Investigators, Boden, W.E., Probstfield, J.L., Anderson, T., Chaitman, B.R., Desvignes-Nickens, P., Koprowicz, K., McBride, R., Teo, K., Weintraub, W., 2011. Niacin in patients with low HDL cholesterol levels receiving intensive statin therapy. N. Engl. J. Med. 365, 2255-2267. doi:10.1056/NEJMoa1107579 Atzmon, G., Rincon, M., Schechter, C.B., Shuldiner, A.R., Lipton, R.B., Bergman, A., Barzilai, N., 2006. Lipoprotein genotype and conserved pathway for exceptional longevity in humans. PLoS Biol. 4, e113. doi:10.1371/journal.pbio.0040113 Barter, P.J., 2004. Antiinflammatory Properties of HDL. Circulation Research 95, 764-772. doi:10.1161/01.RES.0000146094.59640.13 Barter, P.J., Caulfield, M., Eriksson, M., Grundy, S.M., Kastelein, J.J.P., Komajda, M., Lopez-Sendon, J., Mosca, L., Tardif, J.-C., Waters, D.D., Shear, C.L., Revkin, J.H., Buhr, K.A., Fisher, M.R., Tall, A.R., Brewer, B., ILLUMINATE Investigators, 2007. Effects of torcetrapib in patients at high risk for coronary events. N. Engl. J. Med. 357, 2109-2122. doi:10.human gene nomenclature. Genomics 79, 464-470. doi:10.1006/geno.2002.6748 Wang, X., Paigen, B., 2005. Genome-wide search for new genes controlling plasma lipid concentrations in mice and humans. Current Opinion in Lipidology 16, 127-137. doi:10.1097/01.mol.0000162317.09054.9d Wilson, P.W., Abbott, R.D., Castelli, W.P., 1988. High density lipoprotein cholesterol and mortality. The Framingham Heart Study. Arteriosclerosis 8, 737-741. Yetukuri, L., Söderlund, S., Koivuniemi, A., Seppänen-Laakso, T., Niemelä, P.S., Hyvönen, M., Taskinen, M.-R., Vattulainen, I., Jauhiainen, M., Orešič, M., 2010. Composition and lipid spatial distribution of HDL particles in subjects with low and high HDL-cholesterol. J. Lipid Res. 51, 2341-2351. doi:10.', 'latex': None, 'type': 'figure'}, 'FIGREF1': {'text': 'The heatmap visualization of the HDL protein abundances across 93 strains. The proteins, their biological functions and cellular locations are represented. Logarithmic transformation of the yeast normalized data has been performed to accommodate the abundance distribution form high (red) to low (dark blue). White squares represent not available values. Both the proteins and the strains were clustered using Euclidean distances.', 'latex': None, 'type': 'figure'}, 'FIGREF2': {'text': 'A representative QTLs for HDL proteins. Loci associated with protein levels of A) APOA2, B), APOC2, C) SAA2, D) H2-Q10, E) SAA1, F) APOC3', 'latex': None, 'type': 'figure'}, 'FIGREF3': {'text': 'The relationship between HDL metrics is represented by a correlation matrix. Total of 8,100 correlations were observed. Among which 2,216 Pearson correlation with Bonferroni-Holm correction having |r| values >0.5 (positive in bleu, negative in red) that are P<0.05 are presented.', 'latex': None, 'type': 'figure'}, 'FIGREF4': {'text': 'Cytoscape visualization of all the protein-protein interactions presented in', 'latex': None, 'type': 'figure'}, 'FIGREF5': {'text': 'The heatmap visualization of the HDL protein abundances across 93 strains. The proteins, their biological functions and cellular locations are represented. Logarithmic transformation of the yeast normalized data has been performed to accommodate the abundance distribution form high (red) to low (dark blue). White squares represent not available values. Both the proteins and the strains were clustered using Euclidean distances.', 'latex': None, 'type': 'figure'}, 'FIGREF6': {'text': 'A representative QTLs for HDL proteins. Loci associated with protein levels of A) APOA2, B), APOC2, C) SAA2, D) H2-Q10 E)SAA1 F', 'latex': None, 'type': 'figure'}, 'FIGREF9': {'text': 'The relationship between HDL metrics is represented by a correlation matrix. Total of 8,100 correlations were observed. Among which 2,216 Pearson correlation with Bonferroni-Holm correction having |r| values >0.5 (positive in bleu, negative in red) that are P<0.05 are presented.', 'latex': None, 'type': 'figure'}, 'FIGREF10': {'text': 'Cytoscape visualization of all the protein-protein interactions presented in', 'latex': None, 'type': 'figure'}, 'FIGREF11': {'text': 'The relationship between HDL metrics is represented by a correlation matrix. Total of 8,100 correlations were observed. Among which 380 Pearson correlation with Bonferroni-Holm correction having |r| values >0.5 (positive in bleu, negative in red) that are P<0.05 are presented. Cytoscape visualization of all the protein-protein interactions presented in Supplemental Figure 2. Self-loops were removed and edges were bundled for clarity. Node locations are assigned using an edge-weighted spring-embedded layout algorithm using the negative log of the Benjamini-Hochberg corrected p-value, and edge transparency is directly proportional to the same value. Red: negative correlation, Blue: positive correlation. Shorter distances indicate stronger correlations.SupplementalFigure 6. The manhattan plots showing A) Global association profiles across strains for baseline (min. BHK, min.cAMP) stimulated (plus.BHK, plus.cAMP) and ABCA1 dependent (delta.J774 delta.BHK) sterol efflux; B) The sterol efflux QTLs located on chromosome 15. Sterol Efflux distribution across strains. min.cAMP, plus.cAMP, delta.J774 are unstimulated, ABCA1 upregulated and ABCA1 specific sterol efflux from J774 cells respectively (-C). min.MF, plus.MF, delta.BHK are unstimulated, ABCA1 upregulated and ABCA1 specific sterol efflux from BHK cells respectively (D-F). All the measures are in duplicate from PEG depleted plasma collected from 1-5 mice for each strain. Manhattan plot showing alignment between APOA2 QTL and HDL-C QTL.', 'latex': None, 'type': 'figure'}, 'FIGREF12': {'text': 'A representative QTLs for HDL proteins normalized to total PSMs. Loci associated with A) APOA2, B), NAPSA, C) SAA2, D) SAA4.', 'latex': None, 'type': 'figure'}, 'FIGREF13': {'text': 'The relationship between HDL metrics is represented by a correlation matrix. Total of 8,100 correlations were observed. Among which 380 Pearson correlation with Bonferroni-Holm correction having |r| values >0.5 (positive in bleu, negative in red) that are P<0.05 are presented. Cytoscape visualization of all the protein-protein interactions presented in Supplemental Figure 2. Self-loops were removed and edges were bundled for clarity. Node locations are assigned using an edge-weighted spring-embedded layout algorithm using the negative log of the Benjamini-Hochberg corrected p-value, and edge transparency is directly proportional to the same value. Red: negative correlation, Blue: positive correlation. Shorter distances indicate stronger correlations. Sterol Efflux distribution across strains. min.cAMP, plus.cAMP, delta.J774 are unstimulated, ABCA1 upregulated and ABCA1 specific sterol efflux from J774 cells respectively (-C). min.MF, plus.MF, delta.BHK are unstimulated, ABCA1 upregulated and ABCA1 specific sterol efflux from BHK cells respectively (D-F). All the measures are in duplicate from PEG depleted plasma collected from 1-5 mice for each strain. The manhattan plots showing A) Global association profiles across strains for baseline (min. BHK, min.cAMP) stimulated (plus.BHK, plus.cAMP) and ABCA1 dependent (delta.J774 delta.BHK) sterol efflux; B) The sterol efflux QTLs located on chromosome 3. Manhattan plot showing alignment between APOA2 QTL and HDL-C QTL. Overlap of the loci regulating APOA2 and HDL-C.', 'latex': None, 'type': 'figure'}, 'TABREF0': {'text': 'Cholesterol efflux assays. Macrophage cholesterol efflux capacity was assessed with J774 macrophages labeled with [ 3 H]cholesterol and stimulated with a cAMP analogue, as described by Rothblat, Rader, and colleagues (la Llera-Moya et al., 2010). Efflux via the ABCA1 pathways were measured with BHK cells expressing mifepristone-inducible human ABCA1 that were radiolabeled with [ 3 H]cholesterol', 'latex': None, 'type': 'table'}, 'TABREF4': {'text': 'List of proteins detected in mouse HDL across HMDP.', 'latex': None, 'type': 'table'}, 'TABREF5': {'text': 'Local hepatic eQTL and HDL protein pQTLLocal eQTL and pQTL with p < 0.001 are listed along with SNP identifiers and their locations', 'latex': None, 'type': 'table'}, 'TABREF6': {'text': 'Correlations between HDL proteins and clinical traits measured within HMDP', 'latex': None, 'type': 'table'}, 'TABREF7': {'text': 'The peak QTLs for sterol efflux capacity of HDL.', 'latex': None, 'type': 'table'}, 'TABREF8': {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/405811 doi: bioRxiv preprint', 'latex': None, 'type': 'table'}, 'TABREF11': {'text': 'SupplementaryTable 1. Summary of HDL proteins per strain. Distal hepatic eQTL and distal HDL pQTL.', 'latex': None, 'type': 'table'}, 'TABREF12': {'text': 'The correlations for the HDL metrics.SupplementalTable 6. Association of HDL Proteins with Clinical Traits in HMDP.', 'latex': None, 'type': 'table'}}, 'back_matter': []}\n",
      "{'paper_id': 'b3e877acb1b2ec719f6e094f7900cc3bf865457c', 'metadata': {'title': 'SARS-CoV-2, an evolutionary perspective of interaction with human ACE2 reveals undiscovered amino acids necessary for complex stability', 'authors': [{'first': 'Vinicio', 'middle': [], 'last': 'Armijos-Jaramillo', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Universidad de Las Américas', 'location': {'postCode': 'EC170125', 'settlement': 'Quito', 'country': 'Ecuador'}}, 'email': ''}, {'first': 'Justin', 'middle': [], 'last': 'Yeager', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Universidad de Las Américas', 'location': {'settlement': 'Quito', 'country': 'Ecuador'}}, 'email': ''}, {'first': 'Claire', 'middle': [], 'last': 'Muslin', 'suffix': '', 'affiliation': {'laboratory': 'One Health Research Group', 'institution': 'Universidad de Las Américas', 'location': {'settlement': 'Quito', 'country': 'Ecuador'}}, 'email': ''}, {'first': 'Yunierkis', 'middle': [], 'last': 'Perez-Castillo', 'suffix': '', 'affiliation': {'laboratory': '', 'institution': 'Universidad de Las Américas', 'location': {'postCode': 'EC170125', 'settlement': 'Quito', 'country': 'Ecuador'}}, 'email': ''}]}, 'abstract': [{'text': 'The emergence of SARS-CoV-2 has resulted in more than 200,000 infections and nearly 9,000 deaths globally so far. This novel virus is thought to have originated from an animal reservoir, and acquired the ability to infect human cells using the SARS-CoV cell receptor hACE2. In the wake of a global pandemic it is essential to improve our understanding of the evolutionary dynamics surrounding the origin and spread of a novel infectious disease. One way theory predicts selection pressures should shape viral evolution is to enhance binding with host cells. We first assessed evolutionary dynamics in select betacoronavirus spike protein genes to predict where these genomic regions are under directional or purifying selection between divergent viral lineages at various scales of relatedness. With this analysis, we determine a region inside the receptor-binding domain with putative sites under positive selection interspersed among highly conserved sites, which are implicated in structural stability of the viral spike protein and its union with human receptor hACE2. Next, to gain further insights into factors associated with coronaviruses recognition of the human host receptor, we performed modeling studies of five different coronaviruses and their potential binding to hACE2. Modeling results indicate that interfering with the salt bridges at hot spot 353 could be an effective strategy for inhibiting binding, and hence for the prevention of coronavirus infections. We also propose that a glycine residue at the receptor binding domain of the spike glycoprotein can have a critical role in permitting bat variants of the coronaviruses to infect human cells.', 'cite_spans': [], 'ref_spans': [], 'section': 'Abstract'}], 'body_text': [{'text': 'The recent emergence of the novel SARS coronavirus 2 (SARS-CoV-2) marked the third introduction of a highly pathogenic coronavirus into the human population in the twenty-first century, following the severe acute respiratory syndrome coronavirus (SARS-CoV) and the Middle East respiratory syndrome coronavirus (MERS-CoV). The first, SARS-CoV emerged in November 2002 in the Guangdong province of China and spread globally during 2002-03, infecting more than 8000 people and causing 774 deaths (Drosten et al., 2003; WHO, 2004) . MERS-CoV was the second emergence and was first detected in Saudi Arabia in 2012 and resulted in nearly 2500 human infections and 858 deaths in 27 countries (Fehr et al., 2017; Zaki et al., 2012) . In December 2019, SARS-CoV-2, a previously unknown coronavirus capable of infecting humans was discovered in the Chinese city of Wuhan, in the Hubei province Zhu et al., 2020) . SARS-CoV-2 is associated with an ongoing pandemic of atypical pneumonia, now termed coronavirus disease 19 (Covid-2019) that has affected over 209,000 people with 8778 fatalities as of March 19, 2020 (WHO, 2020 . Both SARS-CoV and MERS-CoV are thought to have originated in colonies of bats, eventually transmitted to humans, putatively facilitated by intermediate hosts such as palm civets and dromedary camels, respectively (Cui et al., 2019) . The genome of SARS-CoV-2 shares about 80% nucleotide identity with that of SARS-CoV and is 96% identical to the bat coronavirus BatCoV RaTG13 genome, reinforcing the probable bat origin of the virus . However, better assessing the evolutionary dynamics of SARS-CoV-2 is an active research priority worldwide.', 'cite_spans': [{'start': 493, 'end': 515, 'text': '(Drosten et al., 2003;', 'ref_id': None}, {'start': 516, 'end': 526, 'text': 'WHO, 2004)', 'ref_id': None}, {'start': 686, 'end': 705, 'text': '(Fehr et al., 2017;', 'ref_id': 'BIBREF8'}, {'start': 706, 'end': 724, 'text': 'Zaki et al., 2012)', 'ref_id': 'BIBREF53'}, {'start': 885, 'end': 902, 'text': 'Zhu et al., 2020)', 'ref_id': 'BIBREF57'}, {'start': 1090, 'end': 1104, 'text': 'March 19, 2020', 'ref_id': None}, {'start': 1105, 'end': 1115, 'text': '(WHO, 2020', 'ref_id': None}, {'start': 1331, 'end': 1349, 'text': '(Cui et al., 2019)', 'ref_id': None}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'SARS-CoV, MERS-CoV and SARS-CoV-2 belong to the genus Betacoronavirus within the subfamily Coronavirinae of the family Coronaviridae. Members of this family are enveloped viruses containing a single positive-strand RNA genome of 27-32 kb in length, the largest known RNA virus genome. The coronavirus spherical virion consists of four structural proteins: the spike glycoprotein (S-protein), the envelope protein, membrane protein and nucleocapsid. The transmembrane trimeric S-protein plays a critical role in virus entry into host cells (Gallagher & Buchmeier, 2001; Tortorici & Veesler, 2019) . It comprises two functional subunits: S1 subunit, where the receptor-binding domain (RBD) is found, is responsible for binding host cell surface receptors and S2 subunit mediates subsequent fusion between the viral and cellular membranes (Kirchdoerfer et al., 2016; Yuan et al., 2017) . Both SARS-CoV and SARS-CoV-2 interact directly with angiotensin-converting enzyme 2 (ACE2) to enter host target cells (Hoffmann et al., 2020; Li et al., 2003; Walls et al., 2020; Yan et al., 2020) . In the case of SARS-CoV, ACE2 binding was found to be a critical determinant for the virus host range and key amino acid residues in the RBD were identified to be essential for ACE2-mediated SARS-CoV infection and adaptation to humans (Li et al., 2006; Li et al., 2005) .', 'cite_spans': [{'start': 539, 'end': 568, 'text': '(Gallagher & Buchmeier, 2001;', 'ref_id': 'BIBREF9'}, {'start': 569, 'end': 595, 'text': 'Tortorici & Veesler, 2019)', 'ref_id': 'BIBREF37'}, {'start': 836, 'end': 863, 'text': '(Kirchdoerfer et al., 2016;', 'ref_id': 'BIBREF21'}, {'start': 864, 'end': 882, 'text': 'Yuan et al., 2017)', 'ref_id': 'BIBREF52'}, {'start': 1003, 'end': 1026, 'text': '(Hoffmann et al., 2020;', 'ref_id': 'BIBREF12'}, {'start': 1027, 'end': 1043, 'text': 'Li et al., 2003;', 'ref_id': None}, {'start': 1044, 'end': 1063, 'text': 'Walls et al., 2020;', 'ref_id': 'BIBREF39'}, {'start': 1064, 'end': 1081, 'text': 'Yan et al., 2020)', 'ref_id': 'BIBREF49'}, {'start': 1319, 'end': 1336, 'text': '(Li et al., 2006;', 'ref_id': 'BIBREF25'}, {'start': 1337, 'end': 1353, 'text': 'Li et al., 2005)', 'ref_id': 'BIBREF30'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'Understanding the dynamics that permits a virus to shift hosts is of considerable interest, and further be an essential preliminary step towards facilitating the development of vaccines and the discovery of specific drug therapies. We employ a multidisciplinary approach to look for evidence of diversifying selection on the S-protein gene, and model the interactions between human ACE2 (hACE2) and the RBD of selected coronavirus strains, which ultimately afforded us novel insights detailing virus and host cell interactions. Given the rapid pace of discovery we aim to add clarity to evolutionary dynamics of diseases strains by more precisely understand the dynamics at the S-protein and its interaction with hACE2.', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'Phylogenetic reconstruction, and analysis testing for evidence of positive/purifying selection in the coronavirus S-protein region (Katoh & Standley, 2013) . The phylogenetic reconstruction of S-protein genes was performed with PhyML (Guindon et al., 2010) , using a GTR+I+G model, using 100 non-parametric bootstrap replicates. Both, the alignment and the tree were used as input for PAML Codeml (Yang, 2007) . The presence of sites under positive selection was tested by the comparison of M2 (it allows a proportion of positive, neutral and negative selection sites in the alignment) vs M1 (it allows a proportion of neutral and negative selection sites in the alignment) and M8 (ω follows a beta distribution plus a proportion of sites with ω>1) vs M7 (ω follows a beta distribution) models using the ETE toolkit 3.0 (Huerta-Cepas et al., 2010) . The presence of tree nodes under positive selection was obtained with the free branch model and then tested by the comparison of branch free (different ω for each selected branches) vs M0 (negative selection for all sites and branchesnull model) and branch free vs branch neutral (ω=1 for selected branches) models. The presence of sites with positive selection under specific branches of the tree was tested with bsA (proportion of sites with positive selection in a specific branch of the tree) vs bsA1 (proportion of sites with neutral and purifying selection in a specific branch of the tree) models. Likelihood ratio test (LRT) was performed (p≤ 0.05) to compare the hypothesis contrasted by each model.', 'cite_spans': [{'start': 131, 'end': 155, 'text': '(Katoh & Standley, 2013)', 'ref_id': 'BIBREF19'}, {'start': 234, 'end': 256, 'text': '(Guindon et al., 2010)', 'ref_id': 'BIBREF10'}, {'start': 397, 'end': 409, 'text': '(Yang, 2007)', 'ref_id': 'BIBREF51'}, {'start': 820, 'end': 847, 'text': '(Huerta-Cepas et al., 2010)', 'ref_id': 'BIBREF16'}], 'ref_spans': [], 'section': 'Methods'}, {'text': 'We used the set of programs available in HyPhy (Kosakovsky Pond et al., 2020) , Fast Unconstrained Bayesian AppRoximation (FUBAR) to detect overall sites under positive selection, and Fixed Effects Likelihood (FEL) to detect specific sites under positive selection in specific branches. We used Mixed Effects Model of Evolution (MEME) to detect episodic positive/diversifying selection and adaptive Branch Site REL (aBSREL) to detect branches in the tree under positive selection. The web server Datamonkey (Weaver et al., 2018) was used to perform the HyPhy analyses.', 'cite_spans': [{'start': 59, 'end': 77, 'text': 'Pond et al., 2020)', 'ref_id': 'BIBREF23'}, {'start': 507, 'end': 528, 'text': '(Weaver et al., 2018)', 'ref_id': None}], 'ref_spans': [], 'section': 'Methods'}, {'text': 'Finally, TreeSAAP 3.2 (Woolley et al., 2003) was used to detect sites under adaptation (in terms of physicochemical properties). The same alignment and tree described above were used for this analysis. All these experiments were performed again using the S-protein genes of a shorter list of accessions and more distantly related (broad dataset) to SARS-COV-2 (AY304488, AY395003, DQ412043, FJ882957, KY417144, MG772933, MG772934, MN908947, NC_004718) to test the reproducibility of the predicted branches and sites under positive selection. ', 'cite_spans': [{'start': 22, 'end': 44, 'text': '(Woolley et al., 2003)', 'ref_id': 'BIBREF47'}], 'ref_spans': [], 'section': 'Methods'}, {'text': 'The crystal structure of the SARS-CoV S-protein RBD ( GeneBank ID NC_004718) in complex with hACE2 was retrieved from the Protein Data Bank (code 2AJF) (Berman et al., 2000) . Homology models were constructed using this structure as template for the RBDs of SARS-CoV-2 (SARS2, GeneBank ID MN908947), the Bat SARS-like coronavirus isolate Rm1 (Rm1, GeneBank ID DQ412043) and the Bat SARS-like coronavirus isolate Rs4231 ( Rs4231, GeneBank ID KY417146). One additional homology model for the G496D mutant of the SARS-CoV-2 RBD (SARS2-MUT) was constructed. Homology models were built with Modeller v. 9 (Webb & Sali, 2016) using its UCSF Chimera interface (Pettersen et al., 2004) . Five models were constructed for each target sequence and the one with the lowest DOPE score was selected for the final model.', 'cite_spans': [{'start': 152, 'end': 173, 'text': '(Berman et al., 2000)', 'ref_id': 'BIBREF2'}, {'start': 653, 'end': 677, 'text': '(Pettersen et al., 2004)', 'ref_id': 'BIBREF33'}], 'ref_spans': [], 'section': 'Assessing molecular structure and host-specific interactions'}, {'text': 'All non-amino acidic residues were removed from the SARS-CoV RBD-hACE2 complex to obtain a clean complex. The homology models of the SARS2, Rm1, Rs4231 RBDs and SARS2-MUT were superimposed into the SARS-CoV RBD to obtain their initial complexes with hACE2. These complexes were then subject to molecular dynamics (MD) simulations and estimation of their free energies of binding using Amber 18 (Case et al., 2018) . For the later, ACE2 was considered as the receptor and the RBDs as ligands. The protocol described below was employed for all complexes and otherwise noted default software parameters were employed.', 'cite_spans': [{'start': 394, 'end': 413, 'text': '(Case et al., 2018)', 'ref_id': None}], 'ref_spans': [], 'section': 'Assessing molecular structure and host-specific interactions'}, {'text': 'Systems preparation was performed with the tleap program of the Amber 18 suite. Each complex was enclosed in a truncated octahedron box extending 10 Å from any atom. Next, the boxes were solvated with TIP3P water molecules and Na+ ions were added to neutralize the excess charge. Systems were minimized in two steps, the first of which consisted in 500 steps of the steepest descent algorithm followed by 500 cycles of conjugate gradient with protein atoms restrained using a force constant of 500 kcal/mol.Å 2 . The PME method with a cutoff of 12 Å was used to treat long range electrostatic interactions. During the second minimization step the PME cutoff was set to 10 Å and it proceeded for 1500 steps of the steepest descent algorithm followed by 1000 cycles of conjugate gradient with no restrains. The same PME cutoff of 10 Å was used in all simulation steps from here on. Both minimization stages were performed at constant volume.', 'cite_spans': [], 'ref_spans': [], 'section': 'Assessing molecular structure and host-specific interactions'}, {'text': 'The minimized systems were heated from 0 to 300 K at constant volume constraining all protein atoms with a force constant of 10 kcal/mol.Å 2 . The SHAKE algorithm was used to constrain all bonds involving hydrogens and their interactions were omitted from this step on. Heating took place for 10000 steps, with a time step of 2 fs and a Langevin thermostat with a collision frequency of 1.0 ps -1 was employed. All subsequent MD steps utilized the same thermostat settings. Afterward, the systems were equilibrated for 100 ps at a constant temperature of 300 K and a constant pressure of 1 bar. Pressure was controlled with isotropic position scaling with a relaxation time of 2 ps. The equilibrated systems were used as input for 10 ns length production MD simulations.', 'cite_spans': [], 'ref_spans': [], 'section': 'Assessing molecular structure and host-specific interactions'}, {'text': 'The free energies of binding were computed under the MM-PBSA approach implemented in AmberTools 18 (Case et al., 2018) . A total of 100 MD snapshots were evenly selected, one every 50 ps, from the last 5 ns of the production run for MM-PBSA calculations. The ionic strength was set to 100 mM and the solute dielectric factor was set to 4 for all systems.', 'cite_spans': [{'start': 99, 'end': 118, 'text': '(Case et al., 2018)', 'ref_id': None}], 'ref_spans': [], 'section': 'Assessing molecular structure and host-specific interactions'}, {'text': \"In order to detect branches and sites under positive/negative selection, two datasets were explored. The first ('closer' dataset) harbors the most similar genomes to Wuhan-Hu-1 coronavirus (SARS-CoV-2) (MN908947). For this dataset, several genomes were excluded from the analysis because they showed minimal variation to other sequences. We used a preliminary phylogeny to select a representative isolate of each clade (Table 1 ) in order to exclude highly similar sequences. The second dataset ('broad' dataset) includes some accessions of the first dataset plus isolates less related to SARS-CoV-2, like SARS-like coronavirus isolates from different countries (see methods). We compare the results of two dataset because the phylogenetic distance between orthologues in a given dataset has been demonstrated to alter the ability to detect selection in PAML and MEME (McBee et al., 2015) .\", 'cite_spans': [{'start': 868, 'end': 888, 'text': '(McBee et al., 2015)', 'ref_id': 'BIBREF31'}], 'ref_spans': [{'start': 419, 'end': 427, 'text': '(Table 1', 'ref_id': 'TABREF0'}], 'section': 'Results and discussion'}, {'text': \"In both datasets, we observed evidence of purifying selection in the majority of nodes of the tree. Specifically, in the 'closer' dataset we identified 38 nodes with evidence of negative selection, and 4 under positive selection when free ratios model of CODEML model was applied.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Results and discussion'}, {'text': \"To confirm the four nodes under positive selection we use LTR test for contrasting hypothesis using branch free, branch neutral and M0 models of CODEML. Using these approximations, any node predicted by free ratios model with ω>1 was significantly different to the purifying (ω<1) or neutral (ω=1) models. An equivalent analysis was performed using aBSREL of HyPhy, observing episodic diversifying selection in at least 8 of 41 nodes of the phylogenetic tree reconstructed with the 'closer' dataset ( Figure 1 ). Interestingly, one of the divisions detected with diversifying selection was the branch that contains SARS-CoV-2, Pangolin coronavirus isolate MP789 and Bat coronavirus RaTG13 (called SARS-CoV-2 group) but not the specific branch that contains SARS-CoV-2. under positive selection in SARS-CoV-2 using the closer dataset without Pangolin coronavirus isolate MP789. It is interesting despite the influence of the dataset in the results, because site F486 is directly involved in hACE2-RBD interaction , explaining at least in part strong selection at this site. Moreover, the branch-site model bsA (positive selection) vs bsA1 (relaxation) of CODEML were compared to find evidence of sites under positive selection in branch of SARS-CoV-2 using the 'closer' dataset, but bsA does not show significant differences with bsA1 (p>0.05) indicating selection cannot be confidently implicated, but it was when other datasets were used (including F486). In summary, we do find evidence of sites under positive/episodic selection in branches of close related strains of Wuhan-Hu-1 isolate coronavirus. However, there is not strong evidence of specific sites under positive selection in SARS-CoV-2 using the tools mentioned in this work. This result does not disregard the presence of positive selection sites in SARS-CoV-2, nonetheless, it shows the limitation of the methods to identify with precision specific sites under positive selection in a precise taxon of a phylogenetic tree. We further warn researchers need to be conservative with interpretations of studies utilizing these methodologies, given the equivocal results can be generated by datasets varying in genetic similarity.\", 'cite_spans': [], 'ref_spans': [{'start': 501, 'end': 509, 'text': 'Figure 1', 'ref_id': 'FIGREF0'}], 'section': 'Results and discussion'}, {'text': 'To complement our analyses looking for evidence of selection among lineages, we specifically analyzed for patterns of selection across sites in the S-protein genes, we used the sites models available in CODEML and HyPhY. Model M2 of CODEML detected 0.133 % of sites under positive selection (ω>1) and models M1 and M2 detected 85% of sites under purifying selection (ω<1). Model M2 explains the significant data better (p=7e-4) than M1 model, that takes in account only sites with neutral and purifying selection. To resolve these ambiguities in positive selection sites we calculate putative selection sites with CODEML (using Bayes Empirical Bayes from M2 and M8 models) and FUBAR with different datasets reflecting the addition of novel sequences to online repositories (broad, closer, closer without MN996532 and MT084071 and closer without MT084071) and we obtain different results.', 'cite_spans': [], 'ref_spans': [], 'section': 'Results and discussion'}, {'text': 'It is becoming increasingly clear that predictions of positive selected sites are highly influenced simply by the diversity of the individual sequences included in the datasets. In any case, the majority of predicted sites converge in the region between 439 to 508, a section of the RBD. Additionally, we used TreeSaap to detect important biochemical amino acid properties changes over regions and/or sites along betacoronavirus S-protein. Using a sliding window size of 20 (increasing by 1) we detect that the region between 466 to 500 (using SARS-COV-2 S-protein as a reference) have drastic amino acid changes for alpha-helical tendencies. In addition, the section between 448 to 485 residues registers radical changes in amino acids implicated in the equilibrium constant (ionization of COOH). In the structural analysis we performed, the section between 472 to 486 forms a loop that is not present in certain S-proteins of coronavirus isolated in bats. This loop extends the interaction area between RBD of S-protein and human ACE2, in fact, the lack of these loop decreases the negative energy of interaction (increasing the binding) among these two molecules (see Table 2 ). These results obtained from independent analysis strongly highlight the importance of 439 to 508 section. Additionally, important hACE2-binding residues in the RBD from SARS-COV-2 obtained from the crystallography and structure determination performed by Shang et al. (2020) are also present in the section we highlight here. We propose that this region is the most probable to contain the sites under positive selection due to predictions by our CODEML and FUBAR models. In that sense, we refer to this section as Region under Positive Selection (RPS). It is important to additionally clarify that even inside the RPS we found at least 20 aa highly conserved between coronaviruses, several of them are predicted as sites under purifying selection. This shows that it is necessary to maintain sites without change around polymorphic sites, probably to conserve the protein structure and at the same time to have the ability to colonize more than one host.', 'cite_spans': [{'start': 1437, 'end': 1456, 'text': 'Shang et al. (2020)', 'ref_id': 'BIBREF35'}], 'ref_spans': [{'start': 1171, 'end': 1178, 'text': 'Table 2', 'ref_id': 'TABREF1'}], 'section': 'Results and discussion'}, {'text': 'Interestingly, the RPS of the pangolin coronavirus isolate MP789 differs only in one amino acid with the homologous region of SARS-CoV-2, whereas in contrast the bat coronavirus RaTG13', 'cite_spans': [], 'ref_spans': [], 'section': 'Results and discussion'}, {'text': '. CC-BY-NC-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.21.001933 doi: bioRxiv preprint (the overall most similar isolate to SARS-CoV-2 sequenced at the moment) shows 17 differences in the same region. Several explanations could derive from this observation. The hypothesis of recombination inside the pangolin between a native coronavirus strain and a bat coronavirus (like RaTG13) is congruent with our observation. This scenario was proposed and discussed as the origin of SARS-CoV-2 by (Lam et al., 2020; Wong et al., 2020; Xiao et al., 2020) , however, other explanations are possible. If the SARS-CoV-2, RaTG13 and pangolin coronavirus MP789 isolate are closely related as shown in the tree of the Figure 1 , we are observing the ancestral sequence of RPS in human and pangolin coronaviruses, and a mutated version in bat virus. Elucidating the origin of SARS-CoV-2 is beyond the scope of this work, nevertheless sequencing of new coronavirus isolates in the near future could resolve this question.', 'cite_spans': [{'start': 628, 'end': 646, 'text': '(Lam et al., 2020;', 'ref_id': 'BIBREF24'}, {'start': 647, 'end': 665, 'text': 'Wong et al., 2020;', 'ref_id': 'BIBREF46'}, {'start': 666, 'end': 684, 'text': 'Xiao et al., 2020)', 'ref_id': 'BIBREF48'}], 'ref_spans': [{'start': 842, 'end': 850, 'text': 'Figure 1', 'ref_id': 'FIGREF0'}], 'section': 'Results and discussion'}, {'text': 'With a list of broader observations related to the role of selection across viral genomes we aimed to specifically understand how these regions could affect virus/host interactions. To understand more in deep the importance of RPS in the evolution of SARS-COV-2, we quantified the relative importance of this region in the interaction between RBD and hACE2. In that sense, MD simulations were run for five complexes (listed in methods). In all cases the systems were stable with Root Mean Square Deviations (RMSD) of their backbones between 1.11 Å and 3.30 Å relative to the initial complexes structures during the last 5 ns of the production run. We first investigated the network of contacts between the ligands (coronaviruses RDB) with the receptor (hACE2). Overall, all complexes present a large number of contacts between the ligands and the receptor in at least 50% of the MD snapshots selected for MM-PBSA calculations. Common interactions with T27, F28, K31, H34, Y41, K353, G354, D355 and R357 of the receptor are observed in all systems. The full networks of interactions between the coronaviruses and the hACE2 receptor are provided as Supporting Information.', 'cite_spans': [], 'ref_spans': [], 'section': 'Results and discussion'}, {'text': \"Next we estimated the free energies of binding of the coronaviruses' RBDs to hACE2 and the results of these evaluations are summarized in Table 2 . These calculations show that the SARS2, SARS and Rs4231 viruses are predicted to favorably bind to the human hACE2 receptor, while the Rm1 and SARS2-MUT variants present unfavorable free energies of binding. The fact that the bat's coronavirus Rs4231, in addition to SARS and SARS2, presents favorable interaction with hACE2 is in accordance with the previous observation that it is able to infect human cells expressing this protein (Hu et al., 2017) . To get more insights into the contribution of the receptor and the RBDs to the binding process, we performed energy decomposition experiments. The contribution of each residue in the studied coronaviruses that interact with the hACE2 receptor are shown in Table 3 . Rows are presented in such a way that each of them contains the residues occupying the same position in the viruses RBDs structures as in the SAR2 RBD . CC-BY-NC-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.21.001933 doi: bioRxiv preprint structure. From here on, residues numeration will take that of SARS2 as reference. In general, most RBDs residues show negative values of contribution to the free energies of binding to the human receptor. All studied RBDs, except that of the Rm1 coronavirus, have amino acids with large favorable contributions to the free energies of binding that directly interact with hACE2: K417 of SARS2 and SARS2-MUT, R426 in SARS and R480 in Rs4231. On the other hand, the G476D mutation (D463 present in bat coronavirus strains) have a negative contribution to the binding of the RDB to hACE2. This site was predicted to be under purifying selection by FUBAR analyses, and is located within the RPS. Strikingly, the G496D mutation (SARS2 numeration) has a large negative influence in the free energy of binding in the two complexes that contain it. It is also worth noting that the three aspartic acid substitutions present in all systems negatively contribute to the systems stability. Taking into account that the only difference between SARS2 and SARS2-MUT is the G496D mutation, we postulate that this RBD position is critical for the human receptor recognition by coronaviruses. To the best of our knowledge, no coronavirus having aspartic acid at this position . CC-BY-NC-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.21.001933 doi: bioRxiv preprint is able to infect human cells. This result supports the prediction from FUBAR analyses indicating that the site G496D is under purifying selection. Combined, our results strongly suggest that the mutation of the D496 residue present in the coronaviruses from bats is critical for their RBDs to recognize the human hACE2 receptor. Additionally, it shows the importance of sites under purifying selection in RPS for the RBD evolution.\", 'cite_spans': [{'start': 582, 'end': 599, 'text': '(Hu et al., 2017)', 'ref_id': 'BIBREF14'}], 'ref_spans': [{'start': 138, 'end': 145, 'text': 'Table 2', 'ref_id': 'TABREF1'}, {'start': 858, 'end': 865, 'text': 'Table 3', 'ref_id': 'TABREF2'}], 'section': 'Results and discussion'}, {'text': 'To better interpret the influence of the key interactions between the coronaviruses RBDs and their hACE2 receptor, their interactions were analyzed. To select the representative structure of each system the MD snapshots employed for MM-PBSA calculations were clustered. Then, the representative structure of a system was selected as the centroid of the most populated cluster. The predicted RBD-hACE2 complexes for SARS2, SARS and SARS2-MUT are depicted in Figure 2 .', 'cite_spans': [], 'ref_spans': [{'start': 457, 'end': 465, 'text': 'Figure 2', 'ref_id': 'FIGREF2'}], 'section': 'Results and discussion'}, {'text': 'Many studies have focused on coronaviruses mutations that favor adaptations for human hosts infections. For example, it has been shown that specific substitutions at positions 455, 486, 483, 494 and 501 (442, 472, 479, 480 and 487 in SARS) of the RBD of SARS favors the interaction between the RBD of SARS and hACE2 (Cui et al., 2019) . Likewise, homology modeling studies found favorable interactions between the residues occupying these positions in the SARS2 RBD and the human receptor . The cornerstone of these favorable interactions is the complementarity of the RBDs with hot spots 31 and 353. These are salt bridges between K31 and E35 and between D38 and K353 of ACE2 which are buried in a hydrophobic environment (see Figure. 2). In the cases of SARS2 and SARS, Q493 (N479 in SARS) and N501 (T487 in SARS) add support to the hot spots according to these previous studies. These observations should also hold for the Rs4231 strain, however the N501A change in the later compared to SARS2 (A488 in Rs4231) add little support to hot spot 353. In this case, to continue permitting human infection, the large favorable contribution of R480 in Rs4231 to the free energy of binding could compensate the weak support provided by A488 to hot spot 353.', 'cite_spans': [{'start': 176, 'end': 230, 'text': '455, 486, 483, 494 and 501 (442, 472, 479, 480 and 487', 'ref_id': None}, {'start': 316, 'end': 334, 'text': '(Cui et al., 2019)', 'ref_id': None}], 'ref_spans': [{'start': 728, 'end': 735, 'text': 'Figure.', 'ref_id': None}], 'section': 'Results and discussion'}, {'text': 'Interestingly, K353 is the residue forming the largest network of contacts with the analyzed RBDs among those belonging to both hot spots. Our simulations also show that in SARS2 and SARS the RBD amino acids with the largest contribution to the free energy of binding, K417 and R426 (see Table 3 ) respectively, do not interact with any hot spot residue. Instead, they interact with D30 of hACE2 in the SARS2 complex and with E329 of the human receptor in the SARS complex. This could indicate that interactions additional to those previously identified with the hACE2 hotspots could be critical for the stabilization of the RDB-human receptor complexes. Finally, we analyzed the possible reasons for the predicted negative impact that the G496D mutation has on the predicted free energies of binding of the RBD to hACE2. As depicted in Figure 2 , G496 directly interacts with K353 in hot spot 353 and its mutation interferes with the D38-K353 salt bridge. Specifically, D496 of the RDB point to D38 of hACE yields a high electric repulsion between these amino acids. Consequently, this portion of the RBD is pushed to a position further from hACE2 than that observed in the wild type receptor, resulting in the reduction of its network of contacts with K353. As a result, the binding of the RBD to hACE2 is considerably inhibited and unlikely to occur.', 'cite_spans': [], 'ref_spans': [{'start': 288, 'end': 295, 'text': 'Table 3', 'ref_id': 'TABREF2'}, {'start': 837, 'end': 845, 'text': 'Figure 2', 'ref_id': 'FIGREF2'}], 'section': 'Results and discussion'}, {'text': 'A priority in ongoing research is to better understand coronavirus evolution, with specific interests in understanding the role of selection pressures in viral evolution, and clarifying how viral strains can infect novel hosts. Our experiments suggest that there are sites under positive selection in the S-protein gene of SARS-CoV-2 and other betacoronaviruses, particularly in a region that we called RPS (Region under Positive Selection) inside of the RBD. However, we have identified that by in large, sites in this region (and overall, in the S-protein gene) are under purifying selection. Particularly, for the site D496G, the presence of aspartic acid seems indispensable for the interaction with the hACE2. Additionally, we performed MD simulations and free energies of binding predictions for five different complexes of coronaviruses that do and do not infect human cells. Our results suggest that as long as no disrupting interference occur with both salt bridges at hot spots 31 and 353 coronaviruses are able to bind with hACE2. Modeling results suggest that interference with the hot spot 353 could be and effective strategy for inhibiting the recognition of the RBD of the SARS-COV-2 spike protein by its human host receptor ACE2 and hence prevent infections. Although additional simulations and experiments are required, all evidence suggests that the mutation of D496 in the bat variants of the coronaviruses permit infection of human cells. Giving the large contribution of SARS2 K417 to the free energy of binding of the RBD to hACE2 we propose that blocking its interaction with the receptor D30 could be a promising strategy for future drug discovery efforts.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusions'}], 'bib_entries': {'BIBREF0': {'ref_id': 'b0', 'title': 'Gapped BLAST and PSI-BLAST: A new generation of protein database search programs', 'authors': [{'first': 'S', 'middle': ['F'], 'last': 'Altschul', 'suffix': ''}, {'first': 'T', 'middle': ['L'], 'last': 'Madden', 'suffix': ''}, {'first': 'A', 'middle': ['A'], 'last': 'Schäffer', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'Z', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Miller', 'suffix': ''}, {'first': 'D', 'middle': ['J'], 'last': 'Lipman', 'suffix': ''}], 'year': 1997, 'venue': 'Nucleic Acids Research', 'volume': '25', 'issn': '17', 'pages': '3389--3402', 'other_ids': {}}, 'BIBREF1': {'ref_id': 'b1', 'title': 'The 2019-new coronavirus epidemic: Evidence for virus evolution', 'authors': [{'first': 'D', 'middle': [], 'last': 'Benvenuto', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Giovanetti', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Ciccozzi', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Spoto', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Angeletti', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Ciccozzi', 'suffix': ''}], 'year': 2020, 'venue': 'Journal of Medical Virology', 'volume': '92', 'issn': '4', 'pages': '455--459', 'other_ids': {'DOI': ['10.1002/jmv.25688']}}, 'BIBREF2': {'ref_id': 'b2', 'title': 'The Protein Data Bank', 'authors': [{'first': 'H', 'middle': ['M'], 'last': 'Berman', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Westbrook', 'suffix': ''}, {'first': 'Z', 'middle': [], 'last': 'Feng', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Gilliland', 'suffix': ''}, {'first': 'T', 'middle': ['N'], 'last': 'Bhat', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Weissig', 'suffix': ''}, {'first': 'I', 'middle': ['N'], 'last': 'Shindyalov', 'suffix': ''}, {'first': 'P', 'middle': ['E'], 'last': 'Bourne', 'suffix': ''}], 'year': 2000, 'venue': 'Nucleic Acids Research', 'volume': '28', 'issn': '1', 'pages': '235--242', 'other_ids': {}}, 'BIBREF4': {'ref_id': 'b4', 'title': 'Mauve: Multiple alignment of conserved genomic sequence with rearrangements', 'authors': [{'first': 'D', 'middle': ['A'], 'last': 'Case', 'suffix': ''}, {'first': 'I', 'middle': ['Y'], 'last': 'Ben-Shalom', 'suffix': ''}, {'first': 'S', 'middle': ['R'], 'last': 'Brozell', 'suffix': ''}, {'first': 'D', 'middle': ['S'], 'last': 'Cerutti', 'suffix': ''}, {'first': 'T', 'middle': ['E'], 'last': 'Cheatham', 'suffix': ''}, {'first': 'V', 'middle': ['W D'], 'last': 'Cruzeiro', 'suffix': ''}, {'first': 'T', 'middle': ['A'], 'last': 'Darden', 'suffix': ''}, {'first': 'R', 'middle': ['E'], 'last': 'Duke', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Ghoreishi', 'suffix': ''}, {'first': 'M', 'middle': ['K'], 'last': 'Gilson', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Gohlke', 'suffix': ''}, {'first': 'A', 'middle': ['W'], 'last': 'Goetz', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Greene', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Harris', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Homeyer', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Izadi', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Kovalenko', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Kurtzman', 'suffix': ''}, {'first': 'T', 'middle': ['S'], 'last': 'Lee', 'suffix': ''}, {'first': '', 'middle': [], 'last': 'Kollman', 'suffix': ''}, {'first': 'A', 'middle': ['C E'], 'last': 'Darling', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Mau', 'suffix': ''}, {'first': 'F', 'middle': ['R'], 'last': 'Blattner', 'suffix': ''}, {'first': 'N', 'middle': ['T'], 'last': 'Perna', 'suffix': ''}], 'year': 2004, 'venue': 'Genome Research', 'volume': '14', 'issn': '7', 'pages': '1394--1403', 'other_ids': {'DOI': ['10.1101/gr.2289704']}}, 'BIBREF6': {'ref_id': 'b6', 'title': 'Identification of a novel coronavirus in patients with severe acute respiratory syndrome', 'authors': [{'first': 'H', 'middle': ['W'], 'last': 'Doerr', 'suffix': ''}], 'year': 2003, 'venue': 'The New England Journal of Medicine', 'volume': '348', 'issn': '20', 'pages': '1967--1976', 'other_ids': {}}, 'BIBREF8': {'ref_id': 'b8', 'title': 'Middle East Respiratory Syndrome: Emergence of a Pathogenic Human Coronavirus', 'authors': [{'first': 'A', 'middle': ['R'], 'last': 'Fehr', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Channappanavar', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Perlman', 'suffix': ''}], 'year': 2017, 'venue': 'Annual Review of Medicine', 'volume': '68', 'issn': '1', 'pages': '387--399', 'other_ids': {'DOI': ['10.1146/annurev-med-051215-031152']}}, 'BIBREF9': {'ref_id': 'b9', 'title': 'Coronavirus spike proteins in viral entry and pathogenesis', 'authors': [{'first': 'T', 'middle': ['M'], 'last': 'Gallagher', 'suffix': ''}, {'first': 'M', 'middle': ['J'], 'last': 'Buchmeier', 'suffix': ''}], 'year': 2001, 'venue': 'Virology', 'volume': '279', 'issn': '2', 'pages': '371--374', 'other_ids': {'DOI': ['10.1006/viro.2000.0757']}}, 'BIBREF10': {'ref_id': 'b10', 'title': 'New algorithms and methods to estimate maximum-likelihood phylogenies: Assessing the performance of PhyML 3.0. Systematic biology', 'authors': [{'first': 'S', 'middle': [], 'last': 'Guindon', 'suffix': ''}, {'first': 'J.-F', 'middle': [], 'last': 'Dufayard', 'suffix': ''}, {'first': 'V', 'middle': [], 'last': 'Lefort', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Anisimova', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Hordijk', 'suffix': ''}, {'first': 'O', 'middle': [], 'last': 'Gascuel', 'suffix': ''}], 'year': 2010, 'venue': '', 'volume': '59', 'issn': '', 'pages': '307--321', 'other_ids': {}}, 'BIBREF12': {'ref_id': 'b12', 'title': 'SARS-CoV-2 Cell Entry Depends on ACE2 and TMPRSS2 and Is Blocked by a Clinically Proven Protease Inhibitor', 'authors': [{'first': 'M', 'middle': [], 'last': 'Hoffmann', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Kleine-Weber', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Schroeder', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Krüger', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Herrler', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Erichsen', 'suffix': ''}, {'first': 'T', 'middle': ['S'], 'last': 'Schiergens', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Herrler', 'suffix': ''}, {'first': 'N.-H', 'middle': [], 'last': 'Wu', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Nitsche', 'suffix': ''}, {'first': 'M', 'middle': ['A'], 'last': 'Müller', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Drosten', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Pöhlmann', 'suffix': ''}], 'year': 2020, 'venue': 'Cell', 'volume': '181', 'issn': '', 'pages': '1--10', 'other_ids': {}}, 'BIBREF14': {'ref_id': 'b14', 'title': 'Discovery of a rich gene pool of bat SARS-related coronaviruses provides new insights into the origin of SARS coronavirus', 'authors': [{'first': 'B', 'middle': [], 'last': 'Hu', 'suffix': ''}, {'first': 'L.-P', 'middle': [], 'last': 'Zeng', 'suffix': ''}, {'first': 'X.-L', 'middle': [], 'last': 'Yang', 'suffix': ''}, {'first': 'X.-Y', 'middle': [], 'last': 'Ge', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'J.-Z', 'middle': [], 'last': 'Xie', 'suffix': ''}, {'first': 'X.-R', 'middle': [], 'last': 'Shen', 'suffix': ''}, {'first': 'Y.-Z', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'D.-S', 'middle': [], 'last': 'Luo', 'suffix': ''}, {'first': 'X.-S', 'middle': [], 'last': 'Zheng', 'suffix': ''}, {'first': 'M.-N', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Daszak', 'suffix': ''}, {'first': 'L.-F', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Cui', 'suffix': ''}, {'first': 'Z.-L', 'middle': [], 'last': 'Shi', 'suffix': ''}], 'year': 2017, 'venue': 'PLOS Pathogens', 'volume': '13', 'issn': '11', 'pages': '', 'other_ids': {'DOI': ['10.1371/journal.ppat.1006698']}}, 'BIBREF15': {'ref_id': 'b15', 'title': 'Clinical features of patients infected with 2019 novel coronavirus in', 'authors': [{'first': 'C', 'middle': [], 'last': 'Huang', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Ren', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Zhao', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Hu', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Fan', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Xu', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Gu', 'suffix': ''}, {'first': 'Z', 'middle': [], 'last': 'Cheng', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Yu', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Xia', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Wei', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Wu', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Xie', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Yin', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Liu', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Cao', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '30183--30188', 'other_ids': {'DOI': ['10.1016/S0140-6736(20']}}, 'BIBREF16': {'ref_id': 'b16', 'title': 'ETE: A python Environment for Tree Exploration', 'authors': [{'first': 'J', 'middle': [], 'last': 'Huerta-Cepas', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Dopazo', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Gabaldón', 'suffix': ''}], 'year': 2010, 'venue': 'BMC Bioinformatics', 'volume': '11', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1186/1471-2105-11-24']}}, 'BIBREF17': {'ref_id': 'b17', 'title': 'More effective purifying selection on RNA viruses than in DNA viruses', 'authors': [{'first': 'A', 'middle': ['L'], 'last': 'Hughes', 'suffix': ''}, {'first': 'M', 'middle': ['A K'], 'last': 'Hughes', 'suffix': ''}], 'year': 2007, 'venue': 'Gene', 'volume': '404', 'issn': '1-2', 'pages': '117--125', 'other_ids': {}}, 'BIBREF19': {'ref_id': 'b19', 'title': 'MAFFT multiple sequence alignment software version 7: Improvements in performance and usability', 'authors': [{'first': 'K', 'middle': [], 'last': 'Katoh', 'suffix': ''}, {'first': 'D', 'middle': ['M'], 'last': 'Standley', 'suffix': ''}], 'year': 2013, 'venue': 'Molecular Biology and Evolution', 'volume': '30', 'issn': '4', 'pages': '772--780', 'other_ids': {'DOI': ['10.1093/molbev/mst010']}}, 'BIBREF20': {'ref_id': 'b20', 'title': 'Geneious Basic: An integrated and extendable desktop software platform for the organization and analysis of sequence data', 'authors': [{'first': 'M', 'middle': [], 'last': 'Kearse', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Moir', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Wilson', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Stones-Havas', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Cheung', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Sturrock', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Buxton', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Cooper', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Markowitz', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Duran', 'suffix': ''}, {'first': 'T', 'middle': [], 'last': 'Thierer', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Ashton', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Meintjes', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Drummond', 'suffix': ''}], 'year': 2012, 'venue': 'Bioinformatics', 'volume': '', 'issn': '12', 'pages': '1647--1649', 'other_ids': {'DOI': ['10.1093/bioinformatics/bts199']}}, 'BIBREF21': {'ref_id': 'b21', 'title': 'Pre-fusion structure of a human coronavirus spike protein', 'authors': [{'first': 'R', 'middle': ['N'], 'last': 'Kirchdoerfer', 'suffix': ''}, {'first': 'C', 'middle': ['A'], 'last': 'Cottrell', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Pallesen', 'suffix': ''}, {'first': 'H', 'middle': ['M'], 'last': 'Yassine', 'suffix': ''}, {'first': 'H', 'middle': ['L'], 'last': 'Turner', 'suffix': ''}, {'first': 'K', 'middle': ['S'], 'last': 'Corbett', 'suffix': ''}, {'first': 'B', 'middle': ['S'], 'last': 'Graham', 'suffix': ''}, {'first': 'J', 'middle': ['S'], 'last': 'Mclellan', 'suffix': ''}, {'first': 'A', 'middle': ['B'], 'last': 'Ward', 'suffix': ''}], 'year': 2016, 'venue': 'Nature', 'volume': '531', 'issn': '7592', 'pages': '118--121', 'other_ids': {}}, 'BIBREF23': {'ref_id': 'b23', 'title': 'HyPhy 2.5-A Customizable Platform for Evolutionary Hypothesis Testing Using Phylogenies', 'authors': [{'first': 'Kosakovsky', 'middle': [], 'last': 'Pond', 'suffix': ''}, {'first': 'S', 'middle': ['L'], 'last': 'Poon', 'suffix': ''}, {'first': 'A', 'middle': ['F Y'], 'last': 'Velazquez', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Weaver', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Hepler', 'suffix': ''}, {'first': 'N', 'middle': ['L'], 'last': 'Murrell', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Shank', 'suffix': ''}, {'first': 'S', 'middle': ['D'], 'last': 'Magalis', 'suffix': ''}, {'first': 'B', 'middle': ['R'], 'last': 'Bouvier', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Nekrutenko', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Wisotsky', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Spielman', 'suffix': ''}, {'first': 'S', 'middle': ['J'], 'last': 'Frost', 'suffix': ''}, {'first': 'S', 'middle': ['D W'], 'last': 'Muse', 'suffix': ''}, {'first': 'S', 'middle': ['V'], 'last': '', 'suffix': ''}], 'year': 2020, 'venue': 'Molecular Biology and Evolution', 'volume': '37', 'issn': '1', 'pages': '295--299', 'other_ids': {'DOI': ['10.1093/molbev/msz197']}}, 'BIBREF24': {'ref_id': 'b24', 'title': 'Identification of 2019-nCoV related coronaviruses in Malayan pangolins in southern China', 'authors': [{'first': 'T', 'middle': ['T'], 'last': 'Lam', 'suffix': ''}, {'first': '.-Y', 'middle': [], 'last': 'Shum', 'suffix': ''}, {'first': 'M', 'middle': ['H'], 'last': '', 'suffix': ''}, {'first': '.-H', 'middle': [], 'last': 'Zhu', 'suffix': ''}, {'first': 'H.-C', 'middle': [], 'last': 'Tong', 'suffix': ''}, {'first': 'Y.-G', 'middle': [], 'last': 'Ni', 'suffix': ''}, {'first': 'X.-B', 'middle': [], 'last': 'Liao', 'suffix': ''}, {'first': 'Y.-S', 'middle': [], 'last': 'Wei', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Cheung', 'suffix': ''}, {'first': 'W', 'middle': ['Y'], 'last': '', 'suffix': ''}, {'first': '.-M', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'W.-J', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'L.-F', 'middle': [], 'last': 'Leung', 'suffix': ''}, {'first': 'G', 'middle': ['M'], 'last': 'Holmes', 'suffix': ''}, {'first': 'E', 'middle': ['C'], 'last': 'Hu', 'suffix': ''}, {'first': 'Y.-L', 'middle': [], 'last': 'Guan', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': '', 'suffix': ''}], 'year': 2020, 'venue': 'BioRxiv', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.02.13.945485']}}, 'BIBREF25': {'ref_id': 'b25', 'title': 'Animal Origins of the Severe Acute Respiratory Syndrome Coronavirus: Insight from ACE2-S-Protein Interactions', 'authors': [{'first': 'W', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'S.-K', 'middle': [], 'last': 'Wong', 'suffix': ''}, {'first': 'F', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'J', 'middle': ['H'], 'last': 'Kuhn', 'suffix': ''}, {'first': 'I.-C', 'middle': [], 'last': 'Huang', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Choe', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Farzan', 'suffix': ''}], 'year': 2006, 'venue': 'Journal of Virology', 'volume': '80', 'issn': '9', 'pages': '4211--4219', 'other_ids': {}}, 'BIBREF28': {'ref_id': 'b28', 'title': 'Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus', 'authors': [], 'year': None, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF30': {'ref_id': 'b30', 'title': 'Receptor and viral determinants of SARS-coronavirus adaptation to human ACE2', 'authors': [{'first': 'Wenhui', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Sui', 'suffix': ''}, {'first': 'J', 'middle': ['H'], 'last': 'Kuhn', 'suffix': ''}, {'first': 'M', 'middle': ['J'], 'last': 'Moore', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Luo', 'suffix': ''}, {'first': 'S.-K', 'middle': [], 'last': 'Wong', 'suffix': ''}, {'first': 'I.-C', 'middle': [], 'last': 'Huang', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Xu', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Vasilieva', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Murakami', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'He', 'suffix': ''}, {'first': 'W', 'middle': ['A'], 'last': 'Marasco', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Guan', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Choe', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Farzan', 'suffix': ''}], 'year': 2005, 'venue': 'The EMBO journal', 'volume': '24', 'issn': '8', 'pages': '1634--1643', 'other_ids': {'DOI': ['10.1038/sj.emboj.7600640']}}, 'BIBREF31': {'ref_id': 'b31', 'title': 'The Effect of Species Representation on the Detection of Positive Selection in Primate Gene Data Sets', 'authors': [{'first': 'R', 'middle': ['M'], 'last': 'Mcbee', 'suffix': ''}, {'first': 'S', 'middle': ['A'], 'last': 'Rozmiarek', 'suffix': ''}, {'first': 'N', 'middle': ['R'], 'last': 'Meyerson', 'suffix': ''}, {'first': 'P', 'middle': ['A'], 'last': 'Rowley', 'suffix': ''}, {'first': 'S', 'middle': ['L'], 'last': 'Sawyer', 'suffix': ''}], 'year': 2015, 'venue': 'Molecular Biology and Evolution', 'volume': '32', 'issn': '4', 'pages': '1091--1096', 'other_ids': {}}, 'BIBREF33': {'ref_id': 'b33', 'title': 'UCSF Chimera-A visualization system for exploratory research and', 'authors': [{'first': 'E', 'middle': ['F'], 'last': 'Pettersen', 'suffix': ''}, {'first': 'T', 'middle': ['D'], 'last': 'Goddard', 'suffix': ''}, {'first': 'C', 'middle': ['C'], 'last': 'Huang', 'suffix': ''}, {'first': 'G', 'middle': ['S'], 'last': 'Couch', 'suffix': ''}, {'first': 'D', 'middle': ['M'], 'last': 'Greenblatt', 'suffix': ''}, {'first': 'E', 'middle': ['C'], 'last': 'Meng', 'suffix': ''}, {'first': 'T', 'middle': ['E'], 'last': 'Ferrin', 'suffix': ''}], 'year': 2004, 'venue': '', 'volume': '25', 'issn': '', 'pages': '1605--1612', 'other_ids': {}}, 'BIBREF35': {'ref_id': 'b35', 'title': 'Structural basis for receptor recognition by the novel coronavirus from Wuhan', 'authors': [{'first': 'J', 'middle': [], 'last': 'Shang', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Ye', 'suffix': ''}, {'first': 'K', 'middle': [], 'last': 'Shi', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Wan', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Luo', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Aihara', 'suffix': ''}, {'first': 'Q', 'middle': [], 'last': 'Geng', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Auerbach', 'suffix': ''}, {'first': 'F', 'middle': [], 'last': 'Li', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.21203/rs.2.24749/v1']}}, 'BIBREF36': {'ref_id': 'b36', 'title': 'On the origin and continuing evolution of SARS-CoV-2', 'authors': [{'first': 'X', 'middle': [], 'last': 'Tang', 'suffix': ''}, {'first': 'C', 'middle': [], 'last': 'Wu', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Song', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Yao', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Wu', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Duan', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'Z', 'middle': [], 'last': 'Qian', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Cui', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Lu', 'suffix': ''}], 'year': 2020, 'venue': 'National Science Review', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1093/nsr/nwaa036']}}, 'BIBREF37': {'ref_id': 'b37', 'title': 'Structural insights into coronavirus entry', 'authors': [{'first': 'M', 'middle': ['A'], 'last': 'Tortorici', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Veesler', 'suffix': ''}], 'year': 2019, 'venue': 'En Advances in Virus Research', 'volume': '105', 'issn': '', 'pages': '93--116', 'other_ids': {}}, 'BIBREF39': {'ref_id': 'b39', 'title': 'Structure, Function, and Antigenicity of the SARS-CoV-2 Spike Glycoprotein', 'authors': [{'first': 'A', 'middle': ['C'], 'last': 'Walls', 'suffix': ''}, {'first': 'Y.-J', 'middle': [], 'last': 'Park', 'suffix': ''}, {'first': 'M', 'middle': ['A'], 'last': 'Tortorici', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Wall', 'suffix': ''}, {'first': 'A', 'middle': ['T'], 'last': 'Mcguire', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Veesler', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1016/j.cell.2020.02.058']}}, 'BIBREF40': {'ref_id': 'b40', 'title': 'Receptor recognition by novel coronavirus from Wuhan: An analysis based on decade-long structural studies of SARS', 'authors': [{'first': 'Y', 'middle': [], 'last': 'Wan', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Shang', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Graham', 'suffix': ''}, {'first': 'R', 'middle': ['S'], 'last': 'Baric', 'suffix': ''}, {'first': 'F', 'middle': [], 'last': 'Li', 'suffix': ''}], 'year': 2020, 'venue': 'Journal of Virology', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1128/JVI.00127-20']}}, 'BIBREF42': {'ref_id': 'b42', 'title': 'Datamonkey 2.0: A Modern Web Application for Characterizing Selective and Other Evolutionary Processes', 'authors': [], 'year': None, 'venue': 'Molecular Biology and Evolution', 'volume': '35', 'issn': '3', 'pages': '773--777', 'other_ids': {}}, 'BIBREF44': {'ref_id': 'b44', 'title': 'Comparative Protein Structure Modeling Using MODELLER', 'authors': [{'first': 'B', 'middle': [], 'last': 'Webb', 'suffix': ''}, {'first': 'A', 'middle': [], 'last': 'Sali', 'suffix': ''}], 'year': 2016, 'venue': 'Current Protocols in Bioinformatics', 'volume': '54', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1002/cpbi.3']}}, 'BIBREF45': {'ref_id': 'b45', 'title': 'Summary of probable SARS cases with onset of illness from 1', 'authors': [], 'year': 2002, 'venue': 'Coronavirus disease 2019', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF46': {'ref_id': 'b46', 'title': 'Evidence of recombination in coronaviruses implicating pangolin origins of nCoV-2019', 'authors': [{'first': 'M', 'middle': ['C'], 'last': 'Wong', 'suffix': ''}, {'first': 'S', 'middle': ['J J'], 'last': 'Cregeen', 'suffix': ''}, {'first': 'N', 'middle': ['J'], 'last': 'Ajami', 'suffix': ''}, {'first': 'J', 'middle': ['F'], 'last': 'Petrosino', 'suffix': ''}], 'year': 2020, 'venue': 'BioRxiv', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.02.07.939207']}}, 'BIBREF47': {'ref_id': 'b47', 'title': 'TreeSAAP: Selection on amino acid properties using phylogenetic trees', 'authors': [{'first': 'S', 'middle': [], 'last': 'Woolley', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Johnson', 'suffix': ''}, {'first': 'M', 'middle': ['J'], 'last': 'Smith', 'suffix': ''}, {'first': 'K', 'middle': ['A'], 'last': 'Crandall', 'suffix': ''}, {'first': 'D', 'middle': ['A'], 'last': 'Mcclellan', 'suffix': ''}], 'year': 2003, 'venue': 'Bioinformatics', 'volume': '19', 'issn': '5', 'pages': '671--672', 'other_ids': {'DOI': ['10.1093/bioinformatics/btg043']}}, 'BIBREF48': {'ref_id': 'b48', 'title': 'Isolation and Characterization of 2019-nCoV-like Coronavirus from Malayan Pangolins', 'authors': [{'first': 'K', 'middle': [], 'last': 'Xiao', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Zhai', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Feng', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Zhou', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'J.-J', 'middle': [], 'last': 'Zou', 'suffix': ''}, {'first': 'N', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Guo', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Shen', 'suffix': ''}, {'first': 'Z', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'F', 'middle': [], 'last': 'Shu', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Huang', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'Z', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'R.-A', 'middle': [], 'last': 'Chen', 'suffix': ''}, {'first': 'Y.-J', 'middle': [], 'last': 'Wu', 'suffix': ''}, {'first': 'S.-M', 'middle': [], 'last': 'Peng', 'suffix': ''}, {'first': 'M', 'middle': [], 'last': 'Huang', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Shen', 'suffix': ''}], 'year': 2020, 'venue': 'BioRxiv', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1101/2020.02.17.951335']}}, 'BIBREF49': {'ref_id': 'b49', 'title': 'Structural basis for the recognition of the SARS-CoV-2 by full-length human ACE2', 'authors': [{'first': 'R', 'middle': [], 'last': 'Yan', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Xia', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Guo', 'suffix': ''}, {'first': 'Q', 'middle': [], 'last': 'Zhou', 'suffix': ''}], 'year': 2020, 'venue': 'Science', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF51': {'ref_id': 'b51', 'title': 'PAML 4: Phylogenetic analysis by maximum likelihood', 'authors': [{'first': 'Z', 'middle': [], 'last': 'Yang', 'suffix': ''}], 'year': 2007, 'venue': 'Molecular Biology and Evolution', 'volume': '24', 'issn': '8', 'pages': '1586--1591', 'other_ids': {'DOI': ['10.1093/molbev/msm088']}}, 'BIBREF52': {'ref_id': 'b52', 'title': 'Cryo-EM structures of MERS-CoV and SARS-CoV spike glycoproteins reveal the dynamic receptor binding domains', 'authors': [{'first': 'Y', 'middle': [], 'last': 'Yuan', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Cao', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Ma', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Qi', 'suffix': ''}, {'first': 'Q', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Lu', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Wu', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Yan', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Shi', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'G', 'middle': ['F'], 'last': 'Gao', 'suffix': ''}], 'year': 2017, 'venue': 'Nature communications', 'volume': '8', 'issn': '', 'pages': '15092--15092', 'other_ids': {'DOI': ['10.1038/ncomms15092']}}, 'BIBREF53': {'ref_id': 'b53', 'title': 'Isolation of a novel coronavirus from a man with pneumonia in Saudi Arabia', 'authors': [{'first': 'A', 'middle': ['M'], 'last': 'Zaki', 'suffix': ''}, {'first': 'S', 'middle': [], 'last': 'Van Boheemen', 'suffix': ''}, {'first': 'T', 'middle': ['M'], 'last': 'Bestebroer', 'suffix': ''}, {'first': 'A', 'middle': ['D M E'], 'last': 'Osterhaus', 'suffix': ''}, {'first': 'R', 'middle': ['A M'], 'last': 'Fouchier', 'suffix': ''}], 'year': 2012, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {}}, 'BIBREF56': {'ref_id': 'b56', 'title': 'A pneumonia outbreak associated with a new coronavirus of probable bat origin', 'authors': [{'first': 'P', 'middle': [], 'last': 'Zhou', 'suffix': ''}, {'first': 'X.-L', 'middle': [], 'last': 'Yang', 'suffix': ''}, {'first': 'X.-G', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Hu', 'suffix': ''}, {'first': 'L', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'H.-R', 'middle': [], 'last': 'Si', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Zhu', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'C.-L', 'middle': [], 'last': 'Huang', 'suffix': ''}, {'first': 'H.-D', 'middle': [], 'last': 'Chen', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Chen', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Luo', 'suffix': ''}, {'first': 'H', 'middle': [], 'last': 'Guo', 'suffix': ''}, {'first': 'R.-D', 'middle': [], 'last': 'Jiang', 'suffix': ''}, {'first': 'M.-Q', 'middle': [], 'last': 'Liu', 'suffix': ''}, {'first': 'Y', 'middle': [], 'last': 'Chen', 'suffix': ''}, {'first': 'X.-R', 'middle': [], 'last': 'Shen', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'Z.-L', 'middle': [], 'last': 'Shi', 'suffix': ''}], 'year': 2020, 'venue': '', 'volume': '', 'issn': '', 'pages': '', 'other_ids': {'DOI': ['10.1038/s41586-020-2012-7']}}, 'BIBREF57': {'ref_id': 'b57', 'title': 'A novel coronavirus from patients with pneumonia in China', 'authors': [{'first': 'N', 'middle': [], 'last': 'Zhu', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Zhang', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Li', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Yang', 'suffix': ''}, {'first': 'J', 'middle': [], 'last': 'Song', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Zhao', 'suffix': ''}, {'first': 'B', 'middle': [], 'last': 'Huang', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Shi', 'suffix': ''}, {'first': 'R', 'middle': [], 'last': 'Lu', 'suffix': ''}, {'first': 'P', 'middle': [], 'last': 'Niu', 'suffix': ''}, {'first': 'F', 'middle': [], 'last': 'Zhan', 'suffix': ''}, {'first': 'X', 'middle': [], 'last': 'Ma', 'suffix': ''}, {'first': 'D', 'middle': [], 'last': 'Wang', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Xu', 'suffix': ''}, {'first': 'G', 'middle': [], 'last': 'Wu', 'suffix': ''}, {'first': 'G', 'middle': ['F'], 'last': 'Gao', 'suffix': ''}, {'first': 'W', 'middle': [], 'last': 'Tan', 'suffix': ''}], 'year': 2019, 'venue': 'New England Journal of Medicine', 'volume': '382', 'issn': '8', 'pages': '727--733', 'other_ids': {'DOI': ['10.1056/NEJMoa2001017']}}}, 'ref_entries': {'FIGREF0': {'text': \"Phylogenetic tree of S-protein genes of selected betacoronaviruses ('closer' dataset). Predicted nodes under episodic diversifying selection (according to predictions generated by aBSREL of HyPhy) were colored in blue. Numbers in the inner nodes represent the number of non-parametric bootstrap replicates MEME of HyPhy detected 40 sites in different branches with signal of episodic positive selection, but FEL did not identify sites under positive selection (with p < 0.1) in the SARS-CoV-2 when 'closer' dataset was used. Depending on the focal dataset analyzed ('close' vs. 'broad'), FEL detected several sites under positive selection in SARS-CoV-2, demonstrating the influence of relatedness within the dataset in branch-sites model predictions. FEL predicted the site F486\", 'latex': None, 'type': 'figure'}, 'FIGREF1': {'text': 'FUBAR of HyPhy also detected 1070 sites under purifying selection and only 2 sites under positive selection (alignment size 1284). A similar analysis performed byBenvenuto et al. (2020) with FUBAR detects 1065 sites under purifying selection and sites 536 and 644 as under positive selection.These results suggests by in large strong purifying selection is acting over the vast majority of the S-protein gene, with a comparatively low proportion of sites under positive selection. The evidence of a high amount of purifying selection strengthens similar findings previously reported in RNA viruses(Hughes & Hughes, 2007). These results are congruent with the analysis ofTang et al. (2020) performed with entire coronavirus genomes. However, the sites under positive selection reported differ from our results, and from those ofBenvenuto et al. (2020). They found the sites 439, 483 and 439 (counting from the first amino acid of S-protein), our FUBAR analysis found the sites 443 and 445, and FEL found the residue 486 as specific site under positive selection in SARS-COV-2.', 'latex': None, 'type': 'figure'}, 'FIGREF2': {'text': 'Predicted interaction of SARS2 (top), SARS2-MUT (middle) and SARS (bottom) with the human ACE2 receptor. hACE2 in depicted in gray and the RBD of coronaviruses in cyan. Oxygen atoms are colored red and nitrogen blue. The numbering of the residues corresponds to that of the sequence of each spike glycoprotein.', 'latex': None, 'type': 'figure'}, 'TABREF0': {'text': 'List of coronavirus isolates used for positive selection analysis (closer dataset)', 'latex': None, 'type': 'table'}, 'TABREF1': {'text': 'Estimated free energies of binding of the coronaviruses RBDs to ACE2. Energy values are expressed in kcal/mol.', 'latex': None, 'type': 'table'}, 'TABREF2': {'text': 'Contribution of the residues in the RDB of the coronaviruses interacting with the hACE2 receptor. Energy values are presented as kcal/mol.', 'latex': None, 'type': 'table'}}, 'back_matter': [{'text': 'The authors declare that they have no conflicts of interest.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conflict of interest'}]}\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "IOPub data rate exceeded.\n",
      "The notebook server will temporarily stop sending output\n",
      "to the client in order to avoid crashing it.\n",
      "To change this limit, set the config variable\n",
      "`--NotebookApp.iopub_data_rate_limit`.\n",
      "\n",
      "Current values:\n",
      "NotebookApp.iopub_data_rate_limit=1000000.0 (bytes/sec)\n",
      "NotebookApp.rate_limit_window=3.0 (secs)\n",
      "\n"
     ]
    }
   ],
   "source": [
    "json_text = read_directory_files(dirbase)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
